<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!-- NewPage -->
<html lang="en">
<head>
<!-- Generated by javadoc (version 1.7.0_05) on Fri Nov 30 11:03:56 PST 2012 -->
<meta http-equiv="Content-Type" content="text/html" charset="UTF-8">
<title>Uses of Interface org.biojava3.core.sequence.template.CompoundSet (biojava 3.0.5 API)</title>
<meta name="date" content="2012-11-30">
<link rel="stylesheet" type="text/css" href="../../../../../../stylesheet.css" title="Style">
</head>
<body>
<script type="text/javascript"><!--
    if (location.href.indexOf('is-external=true') == -1) {
        parent.document.title="Uses of Interface org.biojava3.core.sequence.template.CompoundSet (biojava 3.0.5 API)";
    }
//-->
</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
<!-- ========= START OF TOP NAVBAR ======= -->
<div class="topNav"><a name="navbar_top">
<!--   -->
</a><a href="#skip-navbar_top" title="Skip navigation links"></a><a name="navbar_top_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../../../../../../overview-summary.html">Overview</a></li>
<li><a href="../package-summary.html">Package</a></li>
<li><a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">Class</a></li>
<li class="navBarCell1Rev">Use</li>
<li><a href="../package-tree.html">Tree</a></li>
<li><a href="../../../../../../deprecated-list.html">Deprecated</a></li>
<li><a href="../../../../../../index-all.html">Index</a></li>
<li><a href="../../../../../../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li>Prev</li>
<li>Next</li>
</ul>
<ul class="navList">
<li><a href="../../../../../../index.html?org/biojava3/core/sequence/template/class-use/CompoundSet.html" target="_top">Frames</a></li>
<li><a href="CompoundSet.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_top">
<li><a href="../../../../../../allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_top");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<a name="skip-navbar_top">
<!--   -->
</a></div>
<!-- ========= END OF TOP NAVBAR ========= -->
<div class="header">
<h2 title="Uses of Interface org.biojava3.core.sequence.template.CompoundSet" class="title">Uses of Interface<br>org.biojava3.core.sequence.template.CompoundSet</h2>
</div>
<div class="classUseContainer">
<ul class="blockList">
<li class="blockList">
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing packages, and an explanation">
<caption><span>Packages that use <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Package</th>
<th class="colLast" scope="col">Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><a href="#org.biojava3.aaproperties.xml">org.biojava3.aaproperties.xml</a></td>
<td class="colLast">
<div class="block">Set of classes that handles the reading and writing of xml files.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="#org.biojava3.alignment">org.biojava3.alignment</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="#org.biojava3.alignment.aaindex">org.biojava3.alignment.aaindex</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="#org.biojava3.alignment.template">org.biojava3.alignment.template</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="#org.biojava3.core.sequence">org.biojava3.core.sequence</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="#org.biojava3.core.sequence.compound">org.biojava3.core.sequence.compound</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="#org.biojava3.core.sequence.io">org.biojava3.core.sequence.io</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="#org.biojava3.core.sequence.io.util">org.biojava3.core.sequence.io.util</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="#org.biojava3.core.sequence.loader">org.biojava3.core.sequence.loader</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="#org.biojava3.core.sequence.storage">org.biojava3.core.sequence.storage</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="#org.biojava3.core.sequence.template">org.biojava3.core.sequence.template</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="#org.biojava3.core.sequence.transcription">org.biojava3.core.sequence.transcription</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="#org.biojava3.core.sequence.views">org.biojava3.core.sequence.views</a></td>
<td class="colLast">&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList">
<ul class="blockList">
<li class="blockList"><a name="org.biojava3.aaproperties.xml">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a> in <a href="../../../../../../org/biojava3/aaproperties/xml/package-summary.html">org.biojava3.aaproperties.xml</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing classes, and an explanation">
<caption><span>Classes in <a href="../../../../../../org/biojava3/aaproperties/xml/package-summary.html">org.biojava3.aaproperties.xml</a> that implement <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Class and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/aaproperties/xml/CaseFreeAminoAcidCompoundSet.html" title="class in org.biojava3.aaproperties.xml">CaseFreeAminoAcidCompoundSet</a></strong></code>
<div class="block">Set of proteinogenic amino acids.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/aaproperties/xml/ModifiedAminoAcidCompoundSet.html" title="class in org.biojava3.aaproperties.xml">ModifiedAminoAcidCompoundSet</a></strong></code>&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava3.alignment">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a> in <a href="../../../../../../org/biojava3/alignment/package-summary.html">org.biojava3.alignment</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/biojava3/alignment/package-summary.html">org.biojava3.alignment</a> that return <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/alignment/SimpleSubstitutionMatrix.html" title="type parameter in SimpleSubstitutionMatrix">C</a>&gt;</code></td>
<td class="colLast"><span class="strong">SimpleSubstitutionMatrix.</span><code><strong><a href="../../../../../../org/biojava3/alignment/SimpleSubstitutionMatrix.html#getCompoundSet()">getCompoundSet</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/alignment/SimpleProfile.html" title="type parameter in SimpleProfile">C</a>&gt;</code></td>
<td class="colLast"><span class="strong">SimpleProfile.</span><code><strong><a href="../../../../../../org/biojava3/alignment/SimpleProfile.html#getCompoundSet()">getCompoundSet</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/alignment/SimpleAlignedSequence.html" title="type parameter in SimpleAlignedSequence">C</a>&gt;</code></td>
<td class="colLast"><span class="strong">SimpleAlignedSequence.</span><code><strong><a href="../../../../../../org/biojava3/alignment/SimpleAlignedSequence.html#getCompoundSet()">getCompoundSet</a></strong>()</code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing constructors, and an explanation">
<caption><span>Constructors in <a href="../../../../../../org/biojava3/alignment/package-summary.html">org.biojava3.alignment</a> with parameters of type <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colOne" scope="col">Constructor and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/alignment/SimpleSubstitutionMatrix.html#SimpleSubstitutionMatrix(org.biojava3.core.sequence.template.CompoundSet, java.io.File)">SimpleSubstitutionMatrix</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/alignment/SimpleSubstitutionMatrix.html" title="type parameter in SimpleSubstitutionMatrix">C</a>&gt;&nbsp;compoundSet,
                                                <a href="http://docs.oracle.com/javase/6/docs/api/java/io/File.html?is-external=true" title="class or interface in java.io">File</a>&nbsp;fileInput)</code>
<div class="block">Creates a substitution matrix by reading in a file.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/alignment/SimpleSubstitutionMatrix.html#SimpleSubstitutionMatrix(org.biojava3.core.sequence.template.CompoundSet, java.io.Reader, java.lang.String)">SimpleSubstitutionMatrix</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/alignment/SimpleSubstitutionMatrix.html" title="type parameter in SimpleSubstitutionMatrix">C</a>&gt;&nbsp;compoundSet,
                                                <a href="http://docs.oracle.com/javase/6/docs/api/java/io/Reader.html?is-external=true" title="class or interface in java.io">Reader</a>&nbsp;input,
                                                <a href="http://docs.oracle.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a>&nbsp;name)</code>
<div class="block">Creates a substitution matrix by parsing some input.</div>
</td>
</tr>
<tr class="altColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/alignment/SimpleSubstitutionMatrix.html#SimpleSubstitutionMatrix(org.biojava3.core.sequence.template.CompoundSet, short, short)">SimpleSubstitutionMatrix</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/alignment/SimpleSubstitutionMatrix.html" title="type parameter in SimpleSubstitutionMatrix">C</a>&gt;&nbsp;compoundSet,
                                                short&nbsp;match,
                                                short&nbsp;replace)</code>
<div class="block">Creates an identity substitution matrix from match and replace values.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/alignment/SimpleSubstitutionMatrix.html#SimpleSubstitutionMatrix(org.biojava3.core.sequence.template.CompoundSet, java.lang.String, java.lang.String)">SimpleSubstitutionMatrix</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/alignment/SimpleSubstitutionMatrix.html" title="type parameter in SimpleSubstitutionMatrix">C</a>&gt;&nbsp;compoundSet,
                                                <a href="http://docs.oracle.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a>&nbsp;matrixInput,
                                                <a href="http://docs.oracle.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a>&nbsp;name)</code>
<div class="block">Creates a substitution matrix by parsing a String.</div>
</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava3.alignment.aaindex">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a> in <a href="../../../../../../org/biojava3/alignment/aaindex/package-summary.html">org.biojava3.alignment.aaindex</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/biojava3/alignment/aaindex/package-summary.html">org.biojava3.alignment.aaindex</a> that return <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;</code></td>
<td class="colLast"><span class="strong">ScaledSubstitutionMatrix.</span><code><strong><a href="../../../../../../org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html#getCompoundSet()">getCompoundSet</a></strong>()</code>&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava3.alignment.template">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a> in <a href="../../../../../../org/biojava3/alignment/template/package-summary.html">org.biojava3.alignment.template</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/biojava3/alignment/template/package-summary.html">org.biojava3.alignment.template</a> that return <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>protected <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/alignment/template/AbstractProfileProfileAligner.html" title="type parameter in AbstractProfileProfileAligner">C</a>&gt;</code></td>
<td class="colLast"><span class="strong">AbstractProfileProfileAligner.</span><code><strong><a href="../../../../../../org/biojava3/alignment/template/AbstractProfileProfileAligner.html#getCompoundSet()">getCompoundSet</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/alignment/template/SubstitutionMatrix.html" title="type parameter in SubstitutionMatrix">C</a>&gt;</code></td>
<td class="colLast"><span class="strong">SubstitutionMatrix.</span><code><strong><a href="../../../../../../org/biojava3/alignment/template/SubstitutionMatrix.html#getCompoundSet()">getCompoundSet</a></strong>()</code>
<div class="block">Returns the <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template"><code>CompoundSet</code></a> on which the matrix is defined.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/alignment/template/Profile.html" title="type parameter in Profile">C</a>&gt;</code></td>
<td class="colLast"><span class="strong">Profile.</span><code><strong><a href="../../../../../../org/biojava3/alignment/template/Profile.html#getCompoundSet()">getCompoundSet</a></strong>()</code>
<div class="block">Returns <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template"><code>CompoundSet</code></a> of all <a href="../../../../../../org/biojava3/alignment/template/AlignedSequence.html" title="interface in org.biojava3.alignment.template"><code>AlignedSequence</code></a>s</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>protected abstract <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/alignment/template/AbstractMatrixAligner.html" title="type parameter in AbstractMatrixAligner">C</a>&gt;</code></td>
<td class="colLast"><span class="strong">AbstractMatrixAligner.</span><code><strong><a href="../../../../../../org/biojava3/alignment/template/AbstractMatrixAligner.html#getCompoundSet()">getCompoundSet</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>protected <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html" title="type parameter in AbstractPairwiseSequenceAligner">C</a>&gt;</code></td>
<td class="colLast"><span class="strong">AbstractPairwiseSequenceAligner.</span><code><strong><a href="../../../../../../org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html#getCompoundSet()">getCompoundSet</a></strong>()</code>&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava3.core.sequence">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a> in <a href="../../../../../../org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a> that return <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/MultipleSequenceAlignment.html" title="type parameter in MultipleSequenceAlignment">C</a>&gt;</code></td>
<td class="colLast"><span class="strong">MultipleSequenceAlignment.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/MultipleSequenceAlignment.html#getCompoundSet()">getCompoundSet</a></strong>()</code>
<div class="block">Get the Compounds defined in the first sequence</div>
</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing constructors, and an explanation">
<caption><span>Constructors in <a href="../../../../../../org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a> with parameters of type <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colOne" scope="col">Constructor and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/BasicSequence.html#BasicSequence(java.lang.String, org.biojava3.core.sequence.template.CompoundSet)">BasicSequence</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a>&nbsp;sequence,
                          <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/BasicSequence.html" title="type parameter in BasicSequence">C</a>&gt;&nbsp;compoundSet)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/ChromosomeSequence.html#ChromosomeSequence(org.biojava3.core.sequence.template.SequenceReader, org.biojava3.core.sequence.template.CompoundSet)">ChromosomeSequence</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/SequenceReader.html" title="interface in org.biojava3.core.sequence.template">SequenceReader</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;proxyLoader,
                                    <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;compoundSet)</code>
<div class="block">Allows the creation of a ChromosomeSequence using a ProxyResequenceReader for the sequence with a custom CompoundSet</div>
</td>
</tr>
<tr class="altColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/ChromosomeSequence.html#ChromosomeSequence(java.lang.String, org.biojava3.core.sequence.template.CompoundSet)">ChromosomeSequence</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a>&nbsp;seqString,
                                    <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;compoundSet)</code>
<div class="block">Allows the creation of a ChromosomeSequence using String for the sequence with a custom CompoundSet</div>
</td>
</tr>
<tr class="rowColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/DNASequence.html#DNASequence(org.biojava3.core.sequence.template.SequenceReader, org.biojava3.core.sequence.template.CompoundSet)">DNASequence</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/SequenceReader.html" title="interface in org.biojava3.core.sequence.template">SequenceReader</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;proxyLoader,
                      <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;compoundSet)</code>
<div class="block">Create a sequence from a ProxySequencereader and user defined compound set</div>
</td>
</tr>
<tr class="altColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/DNASequence.html#DNASequence(java.lang.String, org.biojava3.core.sequence.template.CompoundSet)">DNASequence</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a>&nbsp;seqString,
                      <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;compoundSet)</code>
<div class="block">Create a sequence from a string with user defined compound set</div>
</td>
</tr>
<tr class="rowColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/ProteinSequence.html#ProteinSequence(org.biojava3.core.sequence.template.ProxySequenceReader, org.biojava3.core.sequence.template.CompoundSet)">ProteinSequence</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/ProxySequenceReader.html" title="interface in org.biojava3.core.sequence.template">ProxySequenceReader</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;proxyLoader,
                              <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;compoundSet)</code>
<div class="block">A protein sequence where the storage of the sequence is somewhere else with user defined
 set of amino acids.</div>
</td>
</tr>
<tr class="altColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/ProteinSequence.html#ProteinSequence(java.lang.String, org.biojava3.core.sequence.template.CompoundSet)">ProteinSequence</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a>&nbsp;seqString,
                              <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;compoundSet)</code>
<div class="block">Create a protein from a string with a user defined set of amino acids</div>
</td>
</tr>
<tr class="rowColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/RNASequence.html#RNASequence(org.biojava3.core.sequence.template.ProxySequenceReader, org.biojava3.core.sequence.template.CompoundSet)">RNASequence</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/ProxySequenceReader.html" title="interface in org.biojava3.core.sequence.template">ProxySequenceReader</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;proxyLoader,
                      <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;compoundSet)</code>
<div class="block">Create a RNA sequence from a proxy reader and user defined RNA compound set</div>
</td>
</tr>
<tr class="altColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/RNASequence.html#RNASequence(java.lang.String, org.biojava3.core.sequence.template.CompoundSet)">RNASequence</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a>&nbsp;seqString,
                      <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;compoundSet)</code>
<div class="block">Create a RNA sequence from a string with a user defined RNA compound set</div>
</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava3.core.sequence.compound">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a> in <a href="../../../../../../org/biojava3/core/sequence/compound/package-summary.html">org.biojava3.core.sequence.compound</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing classes, and an explanation">
<caption><span>Classes in <a href="../../../../../../org/biojava3/core/sequence/compound/package-summary.html">org.biojava3.core.sequence.compound</a> that implement <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Class and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/compound/AmbiguityDNACompoundSet.html" title="class in org.biojava3.core.sequence.compound">AmbiguityDNACompoundSet</a></strong></code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/compound/AmbiguityRNACompoundSet.html" title="class in org.biojava3.core.sequence.compound">AmbiguityRNACompoundSet</a></strong></code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompoundSet.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompoundSet</a></strong></code>
<div class="block">Set of proteinogenic amino acids.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/compound/DNACompoundSet.html" title="class in org.biojava3.core.sequence.compound">DNACompoundSet</a></strong></code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/compound/RNACompoundSet.html" title="class in org.biojava3.core.sequence.compound">RNACompoundSet</a></strong></code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/biojava3/core/sequence/compound/package-summary.html">org.biojava3.core.sequence.compound</a> that return <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;</code></td>
<td class="colLast"><span class="strong">AminoAcidCompound.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html#getCompoundSet()">getCompoundSet</a></strong>()</code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing constructors, and an explanation">
<caption><span>Constructors in <a href="../../../../../../org/biojava3/core/sequence/compound/package-summary.html">org.biojava3.core.sequence.compound</a> with parameters of type <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colOne" scope="col">Constructor and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html#NucleotideCompound(java.lang.String, org.biojava3.core.sequence.template.CompoundSet, java.lang.String)">NucleotideCompound</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a>&nbsp;base,
                                    <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;compoundSet,
                                    <a href="http://docs.oracle.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a>&nbsp;complementStr)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html#NucleotideCompound(java.lang.String, org.biojava3.core.sequence.template.CompoundSet, java.lang.String, org.biojava3.core.sequence.compound.NucleotideCompound[])">NucleotideCompound</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a>&nbsp;base,
                                    <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;compoundSet,
                                    <a href="http://docs.oracle.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a>&nbsp;complementStr,
                                    <a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>[]&nbsp;constituents)</code>&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava3.core.sequence.io">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a> in <a href="../../../../../../org/biojava3/core/sequence/io/package-summary.html">org.biojava3.core.sequence.io</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/biojava3/core/sequence/io/package-summary.html">org.biojava3.core.sequence.io</a> that return <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/transcription/Table.Codon.html" title="class in org.biojava3.core.sequence.transcription">Table.Codon</a>&gt;</code></td>
<td class="colLast"><span class="strong">IUPACParser.IUPACTable.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/io/IUPACParser.IUPACTable.html#getCodonCompoundSet(org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.template.CompoundSet)">getCodonCompoundSet</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;rnaCompounds,
                                      <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;aminoAcidCompounds)</code>
<div class="block">Returns the compound set of codons</div>
</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/biojava3/core/sequence/io/package-summary.html">org.biojava3.core.sequence.io</a> with parameters of type <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/transcription/Table.Codon.html" title="class in org.biojava3.core.sequence.transcription">Table.Codon</a>&gt;</code></td>
<td class="colLast"><span class="strong">IUPACParser.IUPACTable.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/io/IUPACParser.IUPACTable.html#getCodonCompoundSet(org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.template.CompoundSet)">getCodonCompoundSet</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;rnaCompounds,
                                      <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;aminoAcidCompounds)</code>
<div class="block">Returns the compound set of codons</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/transcription/Table.Codon.html" title="class in org.biojava3.core.sequence.transcription">Table.Codon</a>&gt;</code></td>
<td class="colLast"><span class="strong">IUPACParser.IUPACTable.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/io/IUPACParser.IUPACTable.html#getCodonCompoundSet(org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.template.CompoundSet)">getCodonCompoundSet</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;rnaCompounds,
                                      <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;aminoAcidCompounds)</code>
<div class="block">Returns the compound set of codons</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/transcription/Table.Codon.html" title="class in org.biojava3.core.sequence.transcription">Table.Codon</a>&gt;</code></td>
<td class="colLast"><span class="strong">IUPACParser.IUPACTable.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/io/IUPACParser.IUPACTable.html#getCodons(org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.template.CompoundSet)">getCodons</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;nucelotides,
                  <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;aminoAcids)</code>
<div class="block">Returns a list of codons where the source and target compounds
 are the same as those given by the parameters.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/transcription/Table.Codon.html" title="class in org.biojava3.core.sequence.transcription">Table.Codon</a>&gt;</code></td>
<td class="colLast"><span class="strong">IUPACParser.IUPACTable.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/io/IUPACParser.IUPACTable.html#getCodons(org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.template.CompoundSet)">getCodons</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;nucelotides,
                  <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;aminoAcids)</code>
<div class="block">Returns a list of codons where the source and target compounds
 are the same as those given by the parameters.</div>
</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing constructors, and an explanation">
<caption><span>Constructors in <a href="../../../../../../org/biojava3/core/sequence/io/package-summary.html">org.biojava3.core.sequence.io</a> with parameters of type <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colOne" scope="col">Constructor and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/io/CasePreservingProteinSequenceCreator.html#CasePreservingProteinSequenceCreator(org.biojava3.core.sequence.template.CompoundSet)">CasePreservingProteinSequenceCreator</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;compoundSet)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/io/DNASequenceCreator.html#DNASequenceCreator(org.biojava3.core.sequence.template.CompoundSet)">DNASequenceCreator</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;compoundSet)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/io/FileProxyDNASequenceCreator.html#FileProxyDNASequenceCreator(java.io.File, org.biojava3.core.sequence.template.CompoundSet)">FileProxyDNASequenceCreator</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/io/File.html?is-external=true" title="class or interface in java.io">File</a>&nbsp;fastaFile,
                                                      <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;compoundSet)</code>
<div class="block">Need File so that we can store full path name in SequenceFileProxyLoader for Random File access as a quick read</div>
</td>
</tr>
<tr class="rowColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/io/FileProxyProteinSequenceCreator.html#FileProxyProteinSequenceCreator(java.io.File, org.biojava3.core.sequence.template.CompoundSet)">FileProxyProteinSequenceCreator</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/io/File.html?is-external=true" title="class or interface in java.io">File</a>&nbsp;fastaFile,
                                                              <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;compoundSet)</code>
<div class="block">Need File so that we can store full path name in SequenceFileProxyLoader for Random File access as a quick read</div>
</td>
</tr>
<tr class="altColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/io/ProteinSequenceCreator.html#ProteinSequenceCreator(org.biojava3.core.sequence.template.CompoundSet)">ProteinSequenceCreator</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;compoundSet)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/io/RNASequenceCreator.html#RNASequenceCreator(org.biojava3.core.sequence.template.CompoundSet)">RNASequenceCreator</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;compoundSet)</code>&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava3.core.sequence.io.util">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a> in <a href="../../../../../../org/biojava3/core/sequence/io/util/package-summary.html">org.biojava3.core.sequence.io.util</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/biojava3/core/sequence/io/util/package-summary.html">org.biojava3.core.sequence.io.util</a> with parameters of type <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>static &lt;C extends <a href="../../../../../../org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt;&nbsp;<br><a href="http://docs.oracle.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a></code></td>
<td class="colLast"><span class="strong">IOUtils.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/io/util/IOUtils.html#getGCGType(org.biojava3.core.sequence.template.CompoundSet)">getGCGType</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;C&gt;&nbsp;cs)</code>
<div class="block">Determines GCG type</div>
</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava3.core.sequence.loader">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a> in <a href="../../../../../../org/biojava3/core/sequence/loader/package-summary.html">org.biojava3.core.sequence.loader</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/biojava3/core/sequence/loader/package-summary.html">org.biojava3.core.sequence.loader</a> that return <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html" title="type parameter in UniprotProxySequenceReader">C</a>&gt;</code></td>
<td class="colLast"><span class="strong">UniprotProxySequenceReader.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html#getCompoundSet()">getCompoundSet</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/loader/StringProxySequenceReader.html" title="type parameter in StringProxySequenceReader">C</a>&gt;</code></td>
<td class="colLast"><span class="strong">StringProxySequenceReader.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/loader/StringProxySequenceReader.html#getCompoundSet()">getCompoundSet</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html" title="type parameter in SequenceFileProxyLoader">C</a>&gt;</code></td>
<td class="colLast"><span class="strong">SequenceFileProxyLoader.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html#getCompoundSet()">getCompoundSet</a></strong>()</code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/biojava3/core/sequence/loader/package-summary.html">org.biojava3.core.sequence.loader</a> with parameters of type <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">UniprotProxySequenceReader.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html#setCompoundSet(org.biojava3.core.sequence.template.CompoundSet)">setCompoundSet</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html" title="type parameter in UniprotProxySequenceReader">C</a>&gt;&nbsp;compoundSet)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">StringProxySequenceReader.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/loader/StringProxySequenceReader.html#setCompoundSet(org.biojava3.core.sequence.template.CompoundSet)">setCompoundSet</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/loader/StringProxySequenceReader.html" title="type parameter in StringProxySequenceReader">C</a>&gt;&nbsp;compoundSet)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">SequenceFileProxyLoader.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html#setCompoundSet(org.biojava3.core.sequence.template.CompoundSet)">setCompoundSet</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html" title="type parameter in SequenceFileProxyLoader">C</a>&gt;&nbsp;compoundSet)</code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing constructors, and an explanation">
<caption><span>Constructors in <a href="../../../../../../org/biojava3/core/sequence/loader/package-summary.html">org.biojava3.core.sequence.loader</a> with parameters of type <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colOne" scope="col">Constructor and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html#SequenceFileProxyLoader(java.io.File, org.biojava3.core.sequence.io.template.SequenceParserInterface, long, int, org.biojava3.core.sequence.template.CompoundSet)">SequenceFileProxyLoader</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/io/File.html?is-external=true" title="class or interface in java.io">File</a>&nbsp;file,
                                              <a href="../../../../../../org/biojava3/core/sequence/io/template/SequenceParserInterface.html" title="interface in org.biojava3.core.sequence.io.template">SequenceParserInterface</a>&nbsp;sequenceParser,
                                              long&nbsp;sequenceStartIndex,
                                              int&nbsp;sequenceLength,
                                              <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html" title="type parameter in SequenceFileProxyLoader">C</a>&gt;&nbsp;compoundSet)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/loader/StringProxySequenceReader.html#StringProxySequenceReader(java.lang.String, org.biojava3.core.sequence.template.CompoundSet)">StringProxySequenceReader</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a>&nbsp;sequence,
                                                  <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/loader/StringProxySequenceReader.html" title="type parameter in StringProxySequenceReader">C</a>&gt;&nbsp;compoundSet)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html#UniprotProxySequenceReader(java.lang.String, org.biojava3.core.sequence.template.CompoundSet)">UniprotProxySequenceReader</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a>&nbsp;accession,
                                                    <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html" title="type parameter in UniprotProxySequenceReader">C</a>&gt;&nbsp;compoundSet)</code>
<div class="block">The uniprot id is used to retrieve the uniprot XML which is then parsed as a DOM object
 so we know everything about the protein.</div>
</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava3.core.sequence.storage">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a> in <a href="../../../../../../org/biojava3/core/sequence/storage/package-summary.html">org.biojava3.core.sequence.storage</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/biojava3/core/sequence/storage/package-summary.html">org.biojava3.core.sequence.storage</a> that return <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html" title="type parameter in SingleCompoundSequenceReader">C</a>&gt;</code></td>
<td class="colLast"><span class="strong">SingleCompoundSequenceReader.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html#getCompoundSet()">getCompoundSet</a></strong>()</code>
<div class="block">Returns the compound set given at construction</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/storage/JoiningSequenceReader.html" title="type parameter in JoiningSequenceReader">C</a>&gt;</code></td>
<td class="colLast"><span class="strong">JoiningSequenceReader.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/storage/JoiningSequenceReader.html#getCompoundSet()">getCompoundSet</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/storage/ArrayListSequenceReader.html" title="type parameter in ArrayListSequenceReader">C</a>&gt;</code></td>
<td class="colLast"><span class="strong">ArrayListSequenceReader.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/storage/ArrayListSequenceReader.html#getCompoundSet()">getCompoundSet</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/storage/BitSequenceReader.html" title="type parameter in BitSequenceReader">C</a>&gt;</code></td>
<td class="colLast"><span class="strong">BitSequenceReader.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/storage/BitSequenceReader.html#getCompoundSet()">getCompoundSet</a></strong>()</code>
<div class="block">Returns the compound set backing this store</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="type parameter in BitSequenceReader.BitArrayWorker">C</a>&gt;</code></td>
<td class="colLast"><span class="strong">BitSequenceReader.BitArrayWorker.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#getCompoundSet()">getCompoundSet</a></strong>()</code>
<div class="block">Returns the compound set backing this store</div>
</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/biojava3/core/sequence/storage/package-summary.html">org.biojava3.core.sequence.storage</a> with parameters of type <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a></code></td>
<td class="colLast"><span class="strong">SequenceAsStringHelper.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/storage/SequenceAsStringHelper.html#getSequenceAsString(java.util.List, org.biojava3.core.sequence.template.CompoundSet, java.lang.Integer, java.lang.Integer, org.biojava3.core.sequence.Strand)">getSequenceAsString</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/storage/SequenceAsStringHelper.html" title="type parameter in SequenceAsStringHelper">C</a>&gt;&nbsp;parsedCompounds,
                                      <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/storage/SequenceAsStringHelper.html" title="type parameter in SequenceAsStringHelper">C</a>&gt;&nbsp;compoundSet,
                                      <a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Integer.html?is-external=true" title="class or interface in java.lang">Integer</a>&nbsp;bioBegin,
                                      <a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Integer.html?is-external=true" title="class or interface in java.lang">Integer</a>&nbsp;bioEnd,
                                      <a href="../../../../../../org/biojava3/core/sequence/Strand.html" title="enum in org.biojava3.core.sequence">Strand</a>&nbsp;strand)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">SingleCompoundSequenceReader.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html#setCompoundSet(org.biojava3.core.sequence.template.CompoundSet)">setCompoundSet</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html" title="type parameter in SingleCompoundSequenceReader">C</a>&gt;&nbsp;compoundSet)</code>
<div class="block">Unsupported</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">JoiningSequenceReader.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/storage/JoiningSequenceReader.html#setCompoundSet(org.biojava3.core.sequence.template.CompoundSet)">setCompoundSet</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/storage/JoiningSequenceReader.html" title="type parameter in JoiningSequenceReader">C</a>&gt;&nbsp;compoundSet)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">ArrayListSequenceReader.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/storage/ArrayListSequenceReader.html#setCompoundSet(org.biojava3.core.sequence.template.CompoundSet)">setCompoundSet</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/storage/ArrayListSequenceReader.html" title="type parameter in ArrayListSequenceReader">C</a>&gt;&nbsp;compoundSet)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">BitSequenceReader.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/storage/BitSequenceReader.html#setCompoundSet(org.biojava3.core.sequence.template.CompoundSet)">setCompoundSet</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/storage/BitSequenceReader.html" title="type parameter in BitSequenceReader">C</a>&gt;&nbsp;compoundSet)</code>
<div class="block">Class is immutable & so this is unsupported</div>
</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing constructors, and an explanation">
<caption><span>Constructors in <a href="../../../../../../org/biojava3/core/sequence/storage/package-summary.html">org.biojava3.core.sequence.storage</a> with parameters of type <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colOne" scope="col">Constructor and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/storage/ArrayListSequenceReader.html#ArrayListSequenceReader(java.util.List, org.biojava3.core.sequence.template.CompoundSet)">ArrayListSequenceReader</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/storage/ArrayListSequenceReader.html" title="type parameter in ArrayListSequenceReader">C</a>&gt;&nbsp;compounds,
                                              <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/storage/ArrayListSequenceReader.html" title="type parameter in ArrayListSequenceReader">C</a>&gt;&nbsp;compoundSet)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/storage/ArrayListSequenceReader.html#ArrayListSequenceReader(java.lang.String, org.biojava3.core.sequence.template.CompoundSet)">ArrayListSequenceReader</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a>&nbsp;sequence,
                                              <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/storage/ArrayListSequenceReader.html" title="type parameter in ArrayListSequenceReader">C</a>&gt;&nbsp;compoundSet)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#BitSequenceReader.BitArrayWorker(org.biojava3.core.sequence.template.CompoundSet, int)">BitSequenceReader.BitArrayWorker</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="type parameter in BitSequenceReader.BitArrayWorker">C</a>&gt;&nbsp;compoundSet,
                                                                int&nbsp;length)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#BitSequenceReader.BitArrayWorker(org.biojava3.core.sequence.template.CompoundSet, int[])">BitSequenceReader.BitArrayWorker</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="type parameter in BitSequenceReader.BitArrayWorker">C</a>&gt;&nbsp;compoundSet,
                                                                int[]&nbsp;sequence)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#BitSequenceReader.BitArrayWorker(java.lang.String, org.biojava3.core.sequence.template.CompoundSet)">BitSequenceReader.BitArrayWorker</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a>&nbsp;sequence,
                                                                <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="type parameter in BitSequenceReader.BitArrayWorker">C</a>&gt;&nbsp;compoundSet)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html#FourBitSequenceReader.FourBitArrayWorker(org.biojava3.core.sequence.template.CompoundSet, int)">FourBitSequenceReader.FourBitArrayWorker</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html" title="type parameter in FourBitSequenceReader.FourBitArrayWorker">C</a>&gt;&nbsp;compoundSet,
                                                                                int&nbsp;length)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html#FourBitSequenceReader.FourBitArrayWorker(org.biojava3.core.sequence.template.CompoundSet, int[])">FourBitSequenceReader.FourBitArrayWorker</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html" title="type parameter in FourBitSequenceReader.FourBitArrayWorker">C</a>&gt;&nbsp;compoundSet,
                                                                                int[]&nbsp;sequence)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html#FourBitSequenceReader.FourBitArrayWorker(java.lang.String, org.biojava3.core.sequence.template.CompoundSet)">FourBitSequenceReader.FourBitArrayWorker</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a>&nbsp;sequence,
                                                                                <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html" title="type parameter in FourBitSequenceReader.FourBitArrayWorker">C</a>&gt;&nbsp;compoundSet)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/storage/FourBitSequenceReader.html#FourBitSequenceReader(java.lang.String, org.biojava3.core.sequence.template.CompoundSet)">FourBitSequenceReader</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a>&nbsp;sequence,
                                          <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/storage/FourBitSequenceReader.html" title="type parameter in FourBitSequenceReader">C</a>&gt;&nbsp;compoundSet)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/storage/FourBitSequenceReader.html#FourBitSequenceReader(java.lang.String, org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.AccessionID)">FourBitSequenceReader</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a>&nbsp;sequence,
                                          <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/storage/FourBitSequenceReader.html" title="type parameter in FourBitSequenceReader">C</a>&gt;&nbsp;compoundSet,
                                          <a href="../../../../../../org/biojava3/core/sequence/AccessionID.html" title="class in org.biojava3.core.sequence">AccessionID</a>&nbsp;accession)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/storage/JoiningSequenceReader.html#JoiningSequenceReader(org.biojava3.core.sequence.template.CompoundSet, java.util.List)">JoiningSequenceReader</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/storage/JoiningSequenceReader.html" title="type parameter in JoiningSequenceReader">C</a>&gt;&nbsp;compoundSet,
                                          <a href="http://docs.oracle.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/storage/JoiningSequenceReader.html" title="type parameter in JoiningSequenceReader">C</a>&gt;&gt;&nbsp;sequences)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/storage/JoiningSequenceReader.html#JoiningSequenceReader(org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.template.Sequence...)">JoiningSequenceReader</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/storage/JoiningSequenceReader.html" title="type parameter in JoiningSequenceReader">C</a>&gt;&nbsp;compoundSet,
                                          <a href="../../../../../../org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/storage/JoiningSequenceReader.html" title="type parameter in JoiningSequenceReader">C</a>&gt;...&nbsp;sequences)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html#SingleCompoundSequenceReader(C, org.biojava3.core.sequence.template.CompoundSet, int)">SingleCompoundSequenceReader</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html" title="type parameter in SingleCompoundSequenceReader">C</a>&nbsp;compound,
                                                        <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html" title="type parameter in SingleCompoundSequenceReader">C</a>&gt;&nbsp;compoundSet,
                                                        int&nbsp;length)</code>
<div class="block">Build the object with a compound rather than a String</div>
</td>
</tr>
<tr class="rowColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html#SingleCompoundSequenceReader(java.lang.String, org.biojava3.core.sequence.template.CompoundSet, int)">SingleCompoundSequenceReader</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a>&nbsp;compound,
                                                        <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html" title="type parameter in SingleCompoundSequenceReader">C</a>&gt;&nbsp;compoundSet,
                                                        int&nbsp;length)</code>
<div class="block">Public constructor to be used with String based constructor</div>
</td>
</tr>
<tr class="altColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html#TwoBitSequenceReader.TwoBitArrayWorker(org.biojava3.core.sequence.template.CompoundSet, int)">TwoBitSequenceReader.TwoBitArrayWorker</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html" title="type parameter in TwoBitSequenceReader.TwoBitArrayWorker">C</a>&gt;&nbsp;compoundSet,
                                                                            int&nbsp;length)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html#TwoBitSequenceReader.TwoBitArrayWorker(org.biojava3.core.sequence.template.CompoundSet, int[])">TwoBitSequenceReader.TwoBitArrayWorker</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html" title="type parameter in TwoBitSequenceReader.TwoBitArrayWorker">C</a>&gt;&nbsp;compoundSet,
                                                                            int[]&nbsp;sequence)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html#TwoBitSequenceReader.TwoBitArrayWorker(java.lang.String, org.biojava3.core.sequence.template.CompoundSet)">TwoBitSequenceReader.TwoBitArrayWorker</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a>&nbsp;sequence,
                                                                            <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html" title="type parameter in TwoBitSequenceReader.TwoBitArrayWorker">C</a>&gt;&nbsp;compoundSet)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/storage/TwoBitSequenceReader.html#TwoBitSequenceReader(java.lang.String, org.biojava3.core.sequence.template.CompoundSet)">TwoBitSequenceReader</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a>&nbsp;sequence,
                                        <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/storage/TwoBitSequenceReader.html" title="type parameter in TwoBitSequenceReader">C</a>&gt;&nbsp;compoundSet)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/storage/TwoBitSequenceReader.html#TwoBitSequenceReader(java.lang.String, org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.AccessionID)">TwoBitSequenceReader</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a>&nbsp;sequence,
                                        <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/storage/TwoBitSequenceReader.html" title="type parameter in TwoBitSequenceReader">C</a>&gt;&nbsp;compoundSet,
                                        <a href="../../../../../../org/biojava3/core/sequence/AccessionID.html" title="class in org.biojava3.core.sequence">AccessionID</a>&nbsp;accession)</code>&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava3.core.sequence.template">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a> in <a href="../../../../../../org/biojava3/core/sequence/template/package-summary.html">org.biojava3.core.sequence.template</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing classes, and an explanation">
<caption><span>Classes in <a href="../../../../../../org/biojava3/core/sequence/template/package-summary.html">org.biojava3.core.sequence.template</a> that implement <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Class and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava3.core.sequence.template">AbstractCompoundSet</a>&lt;C extends <a href="../../../../../../org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt;</strong></code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/template/AbstractNucleotideCompoundSet.html" title="class in org.biojava3.core.sequence.template">AbstractNucleotideCompoundSet</a>&lt;C extends <a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;</strong></code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/biojava3/core/sequence/template/package-summary.html">org.biojava3.core.sequence.template</a> that return <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/template/LightweightProfile.html" title="type parameter in LightweightProfile">C</a>&gt;</code></td>
<td class="colLast"><span class="strong">LightweightProfile.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/template/LightweightProfile.html#getCompoundSet()">getCompoundSet</a></strong>()</code>
<div class="block">Returns <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template"><code>CompoundSet</code></a> of all <a href="../../../../../../org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a>s</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/template/SequenceProxyView.html" title="type parameter in SequenceProxyView">C</a>&gt;</code></td>
<td class="colLast"><span class="strong">SequenceProxyView.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/template/SequenceProxyView.html#getCompoundSet()">getCompoundSet</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/template/AbstractSequence.html" title="type parameter in AbstractSequence">C</a>&gt;</code></td>
<td class="colLast"><span class="strong">AbstractSequence.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/template/AbstractSequence.html#getCompoundSet()">getCompoundSet</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/template/Sequence.html" title="type parameter in Sequence">C</a>&gt;</code></td>
<td class="colLast"><span class="strong">Sequence.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/template/Sequence.html#getCompoundSet()">getCompoundSet</a></strong>()</code>
<div class="block">Gets the compound set used to back this Sequence</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/template/AbstractCompoundTranslator.html" title="type parameter in AbstractCompoundTranslator">F</a>&gt;</code></td>
<td class="colLast"><span class="strong">AbstractCompoundTranslator.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/template/AbstractCompoundTranslator.html#getFromCompoundSet()">getFromCompoundSet</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/template/AbstractCompoundTranslator.html" title="type parameter in AbstractCompoundTranslator">T</a>&gt;</code></td>
<td class="colLast"><span class="strong">AbstractCompoundTranslator.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/template/AbstractCompoundTranslator.html#getToCompoundSet()">getToCompoundSet</a></strong>()</code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/biojava3/core/sequence/template/package-summary.html">org.biojava3.core.sequence.template</a> with parameters of type <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">AbstractSequence.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/template/AbstractSequence.html#setCompoundSet(org.biojava3.core.sequence.template.CompoundSet)">setCompoundSet</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/template/AbstractSequence.html" title="type parameter in AbstractSequence">C</a>&gt;&nbsp;compoundSet)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">SequenceReader.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/template/SequenceReader.html#setCompoundSet(org.biojava3.core.sequence.template.CompoundSet)">setCompoundSet</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/template/SequenceReader.html" title="type parameter in SequenceReader">C</a>&gt;&nbsp;compoundSet)</code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing constructors, and an explanation">
<caption><span>Constructors in <a href="../../../../../../org/biojava3/core/sequence/template/package-summary.html">org.biojava3.core.sequence.template</a> with parameters of type <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colOne" scope="col">Constructor and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/template/AbstractCompoundTranslator.html#AbstractCompoundTranslator(org.biojava3.core.sequence.io.template.SequenceCreatorInterface, org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.template.CompoundSet)">AbstractCompoundTranslator</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/io/template/SequenceCreatorInterface.html" title="interface in org.biojava3.core.sequence.io.template">SequenceCreatorInterface</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/template/AbstractCompoundTranslator.html" title="type parameter in AbstractCompoundTranslator">T</a>&gt;&nbsp;creator,
                                                    <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/template/AbstractCompoundTranslator.html" title="type parameter in AbstractCompoundTranslator">F</a>&gt;&nbsp;fromCompoundSet,
                                                    <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/template/AbstractCompoundTranslator.html" title="type parameter in AbstractCompoundTranslator">T</a>&gt;&nbsp;toCompoundSet)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/template/AbstractCompoundTranslator.html#AbstractCompoundTranslator(org.biojava3.core.sequence.io.template.SequenceCreatorInterface, org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.template.CompoundSet)">AbstractCompoundTranslator</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/io/template/SequenceCreatorInterface.html" title="interface in org.biojava3.core.sequence.io.template">SequenceCreatorInterface</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/template/AbstractCompoundTranslator.html" title="type parameter in AbstractCompoundTranslator">T</a>&gt;&nbsp;creator,
                                                    <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/template/AbstractCompoundTranslator.html" title="type parameter in AbstractCompoundTranslator">F</a>&gt;&nbsp;fromCompoundSet,
                                                    <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/template/AbstractCompoundTranslator.html" title="type parameter in AbstractCompoundTranslator">T</a>&gt;&nbsp;toCompoundSet)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/template/AbstractSequence.html#AbstractSequence(org.biojava3.core.sequence.template.SequenceReader, org.biojava3.core.sequence.template.CompoundSet)">AbstractSequence</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/SequenceReader.html" title="interface in org.biojava3.core.sequence.template">SequenceReader</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/template/AbstractSequence.html" title="type parameter in AbstractSequence">C</a>&gt;&nbsp;proxyLoader,
                                <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/template/AbstractSequence.html" title="type parameter in AbstractSequence">C</a>&gt;&nbsp;compoundSet)</code>
<div class="block">A ProxySequenceReader allows abstraction of both the storage of the sequence data and the location
 of the sequence data.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/template/AbstractSequence.html#AbstractSequence(java.lang.String, org.biojava3.core.sequence.template.CompoundSet)">AbstractSequence</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a>&nbsp;seqString,
                                <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/template/AbstractSequence.html" title="type parameter in AbstractSequence">C</a>&gt;&nbsp;compoundSet)</code>
<div class="block">Create a Sequence from a simple string where the values should be found in compoundSet</div>
</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava3.core.sequence.transcription">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a> in <a href="../../../../../../org/biojava3/core/sequence/transcription/package-summary.html">org.biojava3.core.sequence.transcription</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/biojava3/core/sequence/transcription/package-summary.html">org.biojava3.core.sequence.transcription</a> that return <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;</code></td>
<td class="colLast"><span class="strong">TranscriptionEngine.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/transcription/TranscriptionEngine.html#getAminoAcidCompounds()">getAminoAcidCompounds</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/transcription/Table.Codon.html" title="class in org.biojava3.core.sequence.transcription">Table.Codon</a>&gt;</code></td>
<td class="colLast"><span class="strong">Table.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/transcription/Table.html#getCodonCompoundSet(org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.template.CompoundSet)">getCodonCompoundSet</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;rnaCompounds,
                                      <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;aminoAcidCompounds)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;</code></td>
<td class="colLast"><span class="strong">TranscriptionEngine.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/transcription/TranscriptionEngine.html#getDnaCompounds()">getDnaCompounds</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;</code></td>
<td class="colLast"><span class="strong">TranscriptionEngine.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/transcription/TranscriptionEngine.html#getRnaCompounds()">getRnaCompounds</a></strong>()</code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/biojava3/core/sequence/transcription/package-summary.html">org.biojava3.core.sequence.transcription</a> with parameters of type <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava3.core.sequence.transcription">TranscriptionEngine.Builder</a></code></td>
<td class="colLast"><span class="strong">TranscriptionEngine.Builder.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/transcription/TranscriptionEngine.Builder.html#aminoAcidsCompounds(org.biojava3.core.sequence.template.CompoundSet)">aminoAcidsCompounds</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;compounds)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava3.core.sequence.transcription">TranscriptionEngine.Builder</a></code></td>
<td class="colLast"><span class="strong">TranscriptionEngine.Builder.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/transcription/TranscriptionEngine.Builder.html#dnaCompounds(org.biojava3.core.sequence.template.CompoundSet)">dnaCompounds</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;compounds)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/transcription/Table.Codon.html" title="class in org.biojava3.core.sequence.transcription">Table.Codon</a>&gt;</code></td>
<td class="colLast"><span class="strong">Table.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/transcription/Table.html#getCodonCompoundSet(org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.template.CompoundSet)">getCodonCompoundSet</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;rnaCompounds,
                                      <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;aminoAcidCompounds)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/transcription/Table.Codon.html" title="class in org.biojava3.core.sequence.transcription">Table.Codon</a>&gt;</code></td>
<td class="colLast"><span class="strong">Table.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/transcription/Table.html#getCodonCompoundSet(org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.template.CompoundSet)">getCodonCompoundSet</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;rnaCompounds,
                                      <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;aminoAcidCompounds)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/transcription/Table.Codon.html" title="class in org.biojava3.core.sequence.transcription">Table.Codon</a>&gt;</code></td>
<td class="colLast"><span class="strong">Table.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/transcription/Table.html#getCodons(org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.template.CompoundSet)">getCodons</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;nucelotides,
                  <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;aminoAcids)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/transcription/Table.Codon.html" title="class in org.biojava3.core.sequence.transcription">Table.Codon</a>&gt;</code></td>
<td class="colLast"><span class="strong">Table.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/transcription/Table.html#getCodons(org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.template.CompoundSet)">getCodons</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;nucelotides,
                  <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;aminoAcids)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava3.core.sequence.transcription">TranscriptionEngine.Builder</a></code></td>
<td class="colLast"><span class="strong">TranscriptionEngine.Builder.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/transcription/TranscriptionEngine.Builder.html#rnaCompounds(org.biojava3.core.sequence.template.CompoundSet)">rnaCompounds</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;compounds)</code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing constructors, and an explanation">
<caption><span>Constructors in <a href="../../../../../../org/biojava3/core/sequence/transcription/package-summary.html">org.biojava3.core.sequence.transcription</a> with parameters of type <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colOne" scope="col">Constructor and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/transcription/DNAToRNATranslator.html#DNAToRNATranslator(org.biojava3.core.sequence.io.template.SequenceCreatorInterface, org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.template.CompoundSet, boolean)">DNAToRNATranslator</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/io/template/SequenceCreatorInterface.html" title="interface in org.biojava3.core.sequence.io.template">SequenceCreatorInterface</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;rnaCreator,
                                    <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;dna,
                                    <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;rna,
                                    boolean&nbsp;shortCutTranslation)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/transcription/DNAToRNATranslator.html#DNAToRNATranslator(org.biojava3.core.sequence.io.template.SequenceCreatorInterface, org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.template.CompoundSet, boolean)">DNAToRNATranslator</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/io/template/SequenceCreatorInterface.html" title="interface in org.biojava3.core.sequence.io.template">SequenceCreatorInterface</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;rnaCreator,
                                    <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;dna,
                                    <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;rna,
                                    boolean&nbsp;shortCutTranslation)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/transcription/RNAToAminoAcidTranslator.html#RNAToAminoAcidTranslator(org.biojava3.core.sequence.io.template.SequenceCreatorInterface, org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.transcription.Table, boolean, boolean, boolean)">RNAToAminoAcidTranslator</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/io/template/SequenceCreatorInterface.html" title="interface in org.biojava3.core.sequence.io.template">SequenceCreatorInterface</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;creator,
                                                <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;nucleotides,
                                                <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/transcription/Table.Codon.html" title="class in org.biojava3.core.sequence.transcription">Table.Codon</a>&gt;&nbsp;codons,
                                                <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;aminoAcids,
                                                <a href="../../../../../../org/biojava3/core/sequence/transcription/Table.html" title="interface in org.biojava3.core.sequence.transcription">Table</a>&nbsp;table,
                                                boolean&nbsp;trimStops,
                                                boolean&nbsp;initMetOnly,
                                                boolean&nbsp;translateNCodons)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/transcription/RNAToAminoAcidTranslator.html#RNAToAminoAcidTranslator(org.biojava3.core.sequence.io.template.SequenceCreatorInterface, org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.transcription.Table, boolean, boolean, boolean)">RNAToAminoAcidTranslator</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/io/template/SequenceCreatorInterface.html" title="interface in org.biojava3.core.sequence.io.template">SequenceCreatorInterface</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;creator,
                                                <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;nucleotides,
                                                <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/transcription/Table.Codon.html" title="class in org.biojava3.core.sequence.transcription">Table.Codon</a>&gt;&nbsp;codons,
                                                <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;aminoAcids,
                                                <a href="../../../../../../org/biojava3/core/sequence/transcription/Table.html" title="interface in org.biojava3.core.sequence.transcription">Table</a>&nbsp;table,
                                                boolean&nbsp;trimStops,
                                                boolean&nbsp;initMetOnly,
                                                boolean&nbsp;translateNCodons)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/transcription/RNAToAminoAcidTranslator.html#RNAToAminoAcidTranslator(org.biojava3.core.sequence.io.template.SequenceCreatorInterface, org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.transcription.Table, boolean, boolean, boolean)">RNAToAminoAcidTranslator</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/io/template/SequenceCreatorInterface.html" title="interface in org.biojava3.core.sequence.io.template">SequenceCreatorInterface</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;creator,
                                                <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;nucleotides,
                                                <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/transcription/Table.Codon.html" title="class in org.biojava3.core.sequence.transcription">Table.Codon</a>&gt;&nbsp;codons,
                                                <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;aminoAcids,
                                                <a href="../../../../../../org/biojava3/core/sequence/transcription/Table.html" title="interface in org.biojava3.core.sequence.transcription">Table</a>&nbsp;table,
                                                boolean&nbsp;trimStops,
                                                boolean&nbsp;initMetOnly,
                                                boolean&nbsp;translateNCodons)</code>&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava3.core.sequence.views">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a> in <a href="../../../../../../org/biojava3/core/sequence/views/package-summary.html">org.biojava3.core.sequence.views</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/biojava3/core/sequence/views/package-summary.html">org.biojava3.core.sequence.views</a> with parameters of type <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">RnaSequenceView.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/views/RnaSequenceView.html#setCompoundSet(org.biojava3.core.sequence.template.CompoundSet)">setCompoundSet</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;compoundSet)</code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing constructors, and an explanation">
<caption><span>Constructors in <a href="../../../../../../org/biojava3/core/sequence/views/package-summary.html">org.biojava3.core.sequence.views</a> with parameters of type <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colOne" scope="col">Constructor and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/views/RnaSequenceView.html#RnaSequenceView(org.biojava3.core.sequence.template.Sequence, org.biojava3.core.sequence.template.CompoundSet)">RnaSequenceView</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;sourceDna,
                              <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;rnaCompounds)</code>&nbsp;</td>
</tr>
</tbody>
</table>
</li>
</ul>
</li>
</ul>
</div>
<!-- ======= START OF BOTTOM NAVBAR ====== -->
<div class="bottomNav"><a name="navbar_bottom">
<!--   -->
</a><a href="#skip-navbar_bottom" title="Skip navigation links"></a><a name="navbar_bottom_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../../../../../../overview-summary.html">Overview</a></li>
<li><a href="../package-summary.html">Package</a></li>
<li><a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">Class</a></li>
<li class="navBarCell1Rev">Use</li>
<li><a href="../package-tree.html">Tree</a></li>
<li><a href="../../../../../../deprecated-list.html">Deprecated</a></li>
<li><a href="../../../../../../index-all.html">Index</a></li>
<li><a href="../../../../../../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li>Prev</li>
<li>Next</li>
</ul>
<ul class="navList">
<li><a href="../../../../../../index.html?org/biojava3/core/sequence/template/class-use/CompoundSet.html" target="_top">Frames</a></li>
<li><a href="CompoundSet.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_bottom">
<li><a href="../../../../../../allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_bottom");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<a name="skip-navbar_bottom">
<!--   -->
</a></div>
<!-- ======== END OF BOTTOM NAVBAR ======= -->
<p class="legalCopy"><small>Copyright &#169; 2012 <a href="http://www.biojava.org">BioJava</a>. All Rights Reserved.</small></p>
</body>
</html>
