<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<html lang="en">
<head>
<title>Source code</title>
<link rel="stylesheet" type="text/css" href="../../../../../../stylesheet.css" title="Style">
</head>
<body>
<div class="sourceContainer">
<pre><span class="sourceLineNo">001</span>/*<a name="line.1"></a>
<span class="sourceLineNo">002</span> * To change this template, choose Tools | Templates<a name="line.2"></a>
<span class="sourceLineNo">003</span> * and open the template in the editor.<a name="line.3"></a>
<span class="sourceLineNo">004</span> */<a name="line.4"></a>
<span class="sourceLineNo">005</span>package org.biojava3.genome.parsers.gff;<a name="line.5"></a>
<span class="sourceLineNo">006</span><a name="line.6"></a>
<span class="sourceLineNo">007</span>import java.io.File;<a name="line.7"></a>
<span class="sourceLineNo">008</span>import java.io.FileOutputStream;<a name="line.8"></a>
<span class="sourceLineNo">009</span>import java.io.OutputStream;<a name="line.9"></a>
<span class="sourceLineNo">010</span>import java.util.ArrayList;<a name="line.10"></a>
<span class="sourceLineNo">011</span>import java.util.Collection;<a name="line.11"></a>
<span class="sourceLineNo">012</span>import java.util.Collections;<a name="line.12"></a>
<span class="sourceLineNo">013</span>import java.util.LinkedHashMap;<a name="line.13"></a>
<span class="sourceLineNo">014</span>import org.biojava3.core.sequence.CDSSequence;<a name="line.14"></a>
<span class="sourceLineNo">015</span>import org.biojava3.core.sequence.ChromosomeSequence;<a name="line.15"></a>
<span class="sourceLineNo">016</span>import org.biojava3.core.sequence.GeneSequence;<a name="line.16"></a>
<span class="sourceLineNo">017</span>import org.biojava3.core.sequence.SequenceComparator;<a name="line.17"></a>
<span class="sourceLineNo">018</span>import org.biojava3.core.sequence.TranscriptSequence;<a name="line.18"></a>
<span class="sourceLineNo">019</span>import org.biojava3.genome.GeneFeatureHelper;<a name="line.19"></a>
<span class="sourceLineNo">020</span><a name="line.20"></a>
<span class="sourceLineNo">021</span>/**<a name="line.21"></a>
<span class="sourceLineNo">022</span> *<a name="line.22"></a>
<span class="sourceLineNo">023</span> * @author Scooter Willis &lt;willishf at gmail dot com&gt;<a name="line.23"></a>
<span class="sourceLineNo">024</span> */<a name="line.24"></a>
<span class="sourceLineNo">025</span>public class GFF3Writer {<a name="line.25"></a>
<span class="sourceLineNo">026</span><a name="line.26"></a>
<span class="sourceLineNo">027</span>    /**<a name="line.27"></a>
<span class="sourceLineNo">028</span>     * Output gff3 format for a DNA Sequence<a name="line.28"></a>
<span class="sourceLineNo">029</span>     * @param fileName<a name="line.29"></a>
<span class="sourceLineNo">030</span>     * @param chromosomeSequence<a name="line.30"></a>
<span class="sourceLineNo">031</span>     * @throws Exception<a name="line.31"></a>
<span class="sourceLineNo">032</span>     */<a name="line.32"></a>
<span class="sourceLineNo">033</span>    public void write(OutputStream outputStream, LinkedHashMap&lt;String, ChromosomeSequence&gt; chromosomeSequenceList) throws Exception {<a name="line.33"></a>
<span class="sourceLineNo">034</span><a name="line.34"></a>
<span class="sourceLineNo">035</span>        outputStream.write("##gff-version 3\n".getBytes());<a name="line.35"></a>
<span class="sourceLineNo">036</span>        for (String key : chromosomeSequenceList.keySet()) {<a name="line.36"></a>
<span class="sourceLineNo">037</span>            ChromosomeSequence chromosomeSequence = chromosomeSequenceList.get(key);<a name="line.37"></a>
<span class="sourceLineNo">038</span>            String gff3line = "";<a name="line.38"></a>
<span class="sourceLineNo">039</span>   //         if(source.length() == 0){<a name="line.39"></a>
<span class="sourceLineNo">040</span>   //             Collection&lt;GeneSequence&gt; genes = chromosomeSequence.getGeneSequences().values();<a name="line.40"></a>
<span class="sourceLineNo">041</span>   //             for(GeneSequence gene : genes){<a name="line.41"></a>
<span class="sourceLineNo">042</span>   //                 source = gene.getSource();<a name="line.42"></a>
<span class="sourceLineNo">043</span>   //                 break;<a name="line.43"></a>
<span class="sourceLineNo">044</span>   //             }<a name="line.44"></a>
<span class="sourceLineNo">045</span>   //         }<a name="line.45"></a>
<span class="sourceLineNo">046</span>   //         gff3line = key + "\t" + source + "\t" + "size" + "\t" + "1" + "\t" + chromosomeSequence.getBioEnd() + "\t.\t.\t.\tName=" + key + "\r\n";<a name="line.46"></a>
<span class="sourceLineNo">047</span>   //         outputStream.write(gff3line.getBytes());<a name="line.47"></a>
<span class="sourceLineNo">048</span><a name="line.48"></a>
<span class="sourceLineNo">049</span>            for (GeneSequence geneSequence : chromosomeSequence.getGeneSequences().values()) {<a name="line.49"></a>
<span class="sourceLineNo">050</span>                gff3line = key + "\t" + geneSequence.getSource() + "\t" + "gene" + "\t" + geneSequence.getBioBegin() + "\t" + geneSequence.getBioEnd() + "\t";<a name="line.50"></a>
<span class="sourceLineNo">051</span>                Double score = geneSequence.getSequenceScore();<a name="line.51"></a>
<span class="sourceLineNo">052</span>                if (score == null) {<a name="line.52"></a>
<span class="sourceLineNo">053</span>                    gff3line = gff3line + ".\t";<a name="line.53"></a>
<span class="sourceLineNo">054</span>                } else {<a name="line.54"></a>
<span class="sourceLineNo">055</span>                    gff3line = gff3line + score + "\t";<a name="line.55"></a>
<span class="sourceLineNo">056</span>                }<a name="line.56"></a>
<span class="sourceLineNo">057</span>                gff3line = gff3line + geneSequence.getStrand().getStringRepresentation() + "\t";<a name="line.57"></a>
<span class="sourceLineNo">058</span>                gff3line = gff3line + ".\t";<a name="line.58"></a>
<span class="sourceLineNo">059</span>                gff3line = gff3line + "ID=" + geneSequence.getAccession().getID() + ";Name=" + geneSequence.getAccession().getID();<a name="line.59"></a>
<span class="sourceLineNo">060</span>                gff3line = gff3line + getGFF3Note(geneSequence.getNotesList());<a name="line.60"></a>
<span class="sourceLineNo">061</span>                gff3line = gff3line + "\n";<a name="line.61"></a>
<span class="sourceLineNo">062</span>                outputStream.write(gff3line.getBytes());<a name="line.62"></a>
<span class="sourceLineNo">063</span><a name="line.63"></a>
<span class="sourceLineNo">064</span>                int transcriptIndex = 0;<a name="line.64"></a>
<span class="sourceLineNo">065</span>                for (TranscriptSequence transcriptSequence : geneSequence.getTranscripts().values()) {<a name="line.65"></a>
<span class="sourceLineNo">066</span>                    transcriptIndex++;<a name="line.66"></a>
<span class="sourceLineNo">067</span><a name="line.67"></a>
<span class="sourceLineNo">068</span>                    gff3line = key + "\t" + transcriptSequence.getSource() + "\t" + "mRNA" + "\t" + transcriptSequence.getBioBegin() + "\t" + transcriptSequence.getBioEnd() + "\t";<a name="line.68"></a>
<span class="sourceLineNo">069</span>                    score = transcriptSequence.getSequenceScore();<a name="line.69"></a>
<span class="sourceLineNo">070</span>                    if (score == null) {<a name="line.70"></a>
<span class="sourceLineNo">071</span>                        gff3line = gff3line + ".\t";<a name="line.71"></a>
<span class="sourceLineNo">072</span>                    } else {<a name="line.72"></a>
<span class="sourceLineNo">073</span>                        gff3line = gff3line + score + "\t";<a name="line.73"></a>
<span class="sourceLineNo">074</span>                    }<a name="line.74"></a>
<span class="sourceLineNo">075</span>                    gff3line = gff3line + transcriptSequence.getStrand().getStringRepresentation() + "\t";<a name="line.75"></a>
<span class="sourceLineNo">076</span>                    gff3line = gff3line + ".\t";<a name="line.76"></a>
<span class="sourceLineNo">077</span>                    String id = geneSequence.getAccession().getID() + "." + transcriptIndex;<a name="line.77"></a>
<span class="sourceLineNo">078</span>                    gff3line = gff3line + "ID=" + id + ";Parent=" + geneSequence.getAccession().getID() + ";Name=" + id;<a name="line.78"></a>
<span class="sourceLineNo">079</span>                    gff3line = gff3line + getGFF3Note(transcriptSequence.getNotesList());<a name="line.79"></a>
<span class="sourceLineNo">080</span><a name="line.80"></a>
<span class="sourceLineNo">081</span>                    gff3line = gff3line + "\n";<a name="line.81"></a>
<span class="sourceLineNo">082</span>                    outputStream.write(gff3line.getBytes());<a name="line.82"></a>
<span class="sourceLineNo">083</span><a name="line.83"></a>
<span class="sourceLineNo">084</span>                    String transcriptParentName = geneSequence.getAccession().getID() + "." + transcriptIndex;<a name="line.84"></a>
<span class="sourceLineNo">085</span>                    ArrayList&lt;CDSSequence&gt; cdsSequenceList = new ArrayList(transcriptSequence.getCDSSequences().values());<a name="line.85"></a>
<span class="sourceLineNo">086</span>                    Collections.sort(cdsSequenceList, new SequenceComparator());<a name="line.86"></a>
<span class="sourceLineNo">087</span>                    for (CDSSequence cdsSequence : cdsSequenceList) {<a name="line.87"></a>
<span class="sourceLineNo">088</span>                        gff3line = key + "\t" + cdsSequence.getSource() + "\t" + "CDS" + "\t" + cdsSequence.getBioBegin() + "\t" + cdsSequence.getBioEnd() + "\t";<a name="line.88"></a>
<span class="sourceLineNo">089</span>                        score = cdsSequence.getSequenceScore();<a name="line.89"></a>
<span class="sourceLineNo">090</span>                        if (score == null) {<a name="line.90"></a>
<span class="sourceLineNo">091</span>                            gff3line = gff3line + ".\t";<a name="line.91"></a>
<span class="sourceLineNo">092</span>                        } else {<a name="line.92"></a>
<span class="sourceLineNo">093</span>                            gff3line = gff3line + score + "\t";<a name="line.93"></a>
<span class="sourceLineNo">094</span>                        }<a name="line.94"></a>
<span class="sourceLineNo">095</span>                        gff3line = gff3line + cdsSequence.getStrand().getStringRepresentation() + "\t";<a name="line.95"></a>
<span class="sourceLineNo">096</span>                        gff3line = gff3line + cdsSequence.getPhase() + "\t";<a name="line.96"></a>
<span class="sourceLineNo">097</span>                        gff3line = gff3line + "ID=" + cdsSequence.getAccession().getID() + ";Parent=" + transcriptParentName;<a name="line.97"></a>
<span class="sourceLineNo">098</span>                        gff3line = gff3line + getGFF3Note(cdsSequence.getNotesList());<a name="line.98"></a>
<span class="sourceLineNo">099</span><a name="line.99"></a>
<span class="sourceLineNo">100</span>                        gff3line = gff3line + "\n";<a name="line.100"></a>
<span class="sourceLineNo">101</span>                        outputStream.write(gff3line.getBytes());<a name="line.101"></a>
<span class="sourceLineNo">102</span>                    }<a name="line.102"></a>
<span class="sourceLineNo">103</span><a name="line.103"></a>
<span class="sourceLineNo">104</span>                }<a name="line.104"></a>
<span class="sourceLineNo">105</span>            }<a name="line.105"></a>
<span class="sourceLineNo">106</span><a name="line.106"></a>
<span class="sourceLineNo">107</span>        }<a name="line.107"></a>
<span class="sourceLineNo">108</span><a name="line.108"></a>
<span class="sourceLineNo">109</span><a name="line.109"></a>
<span class="sourceLineNo">110</span>    }<a name="line.110"></a>
<span class="sourceLineNo">111</span><a name="line.111"></a>
<span class="sourceLineNo">112</span>    private String getGFF3Note(ArrayList&lt;String&gt; notesList) {<a name="line.112"></a>
<span class="sourceLineNo">113</span>        String notes = "";<a name="line.113"></a>
<span class="sourceLineNo">114</span><a name="line.114"></a>
<span class="sourceLineNo">115</span>        if (notesList.size() &gt; 0) {<a name="line.115"></a>
<span class="sourceLineNo">116</span>            notes = ";Note=";<a name="line.116"></a>
<span class="sourceLineNo">117</span>            int noteindex = 1;<a name="line.117"></a>
<span class="sourceLineNo">118</span>            for (String note : notesList) {<a name="line.118"></a>
<span class="sourceLineNo">119</span>                notes = notes + note;<a name="line.119"></a>
<span class="sourceLineNo">120</span>                if (noteindex &lt; notesList.size() - 1) {<a name="line.120"></a>
<span class="sourceLineNo">121</span>                    notes = notes + " ";<a name="line.121"></a>
<span class="sourceLineNo">122</span>                }<a name="line.122"></a>
<span class="sourceLineNo">123</span>            } <a name="line.123"></a>
<span class="sourceLineNo">124</span><a name="line.124"></a>
<span class="sourceLineNo">125</span>        }<a name="line.125"></a>
<span class="sourceLineNo">126</span>        return notes;<a name="line.126"></a>
<span class="sourceLineNo">127</span>    }<a name="line.127"></a>
<span class="sourceLineNo">128</span><a name="line.128"></a>
<span class="sourceLineNo">129</span>    public static void main(String args[]) throws Exception {<a name="line.129"></a>
<span class="sourceLineNo">130</span><a name="line.130"></a>
<span class="sourceLineNo">131</span>        if (true) {<a name="line.131"></a>
<span class="sourceLineNo">132</span>            FileOutputStream fo = new FileOutputStream("/Users/Scooter/scripps/dyadic/geneid/geneid/c1-geneid.gff3");//-16<a name="line.132"></a>
<span class="sourceLineNo">133</span>            LinkedHashMap&lt;String, ChromosomeSequence&gt; dnaSequenceList = GeneFeatureHelper.loadFastaAddGeneFeaturesFromGeneIDGFF2(new File("/Users/Scooter/scripps/dyadic/analysis/454Scaffolds/454Scaffolds.fna"), new File("/Users/Scooter/scripps/dyadic/geneid/geneid/c1_geneid.gff"));<a name="line.133"></a>
<span class="sourceLineNo">134</span>            GFF3Writer gff3Writer = new GFF3Writer();<a name="line.134"></a>
<span class="sourceLineNo">135</span>            gff3Writer.write(fo, dnaSequenceList);<a name="line.135"></a>
<span class="sourceLineNo">136</span><a name="line.136"></a>
<span class="sourceLineNo">137</span><a name="line.137"></a>
<span class="sourceLineNo">138</span>     //       LinkedHashMap&lt;String, ProteinSequence&gt; proteinSequenceList = GeneFeatureHelper.getProteinSequences(chromosomeSequenceList.values());<a name="line.138"></a>
<span class="sourceLineNo">139</span>     //       for(String id : proteinSequenceList.keySet()){<a name="line.139"></a>
<span class="sourceLineNo">140</span>     //           ProteinSequence sequence = proteinSequenceList.get(id);<a name="line.140"></a>
<span class="sourceLineNo">141</span>     //           System.out.println(id + " " + sequence.getSequenceAsString());<a name="line.141"></a>
<span class="sourceLineNo">142</span><a name="line.142"></a>
<span class="sourceLineNo">143</span>     //       }<a name="line.143"></a>
<span class="sourceLineNo">144</span>            fo.close();<a name="line.144"></a>
<span class="sourceLineNo">145</span>        }<a name="line.145"></a>
<span class="sourceLineNo">146</span><a name="line.146"></a>
<span class="sourceLineNo">147</span>        if (false) {<a name="line.147"></a>
<span class="sourceLineNo">148</span>            FileOutputStream fo = new FileOutputStream("/Users/Scooter/scripps/dyadic/analysis/454Scaffolds/genemark_hmm.gff3");//-16<a name="line.148"></a>
<span class="sourceLineNo">149</span>            LinkedHashMap&lt;String, ChromosomeSequence&gt; dnaSequenceList = GeneFeatureHelper.loadFastaAddGeneFeaturesFromGeneMarkGTF(new File("/Users/Scooter/scripps/dyadic/analysis/454Scaffolds/454Scaffolds.fna"), new File("/Users/Scooter/scripps/dyadic/analysis/454Scaffolds/genemark_hmm.gtf"));<a name="line.149"></a>
<span class="sourceLineNo">150</span>            GFF3Writer gff3Writer = new GFF3Writer();<a name="line.150"></a>
<span class="sourceLineNo">151</span>            gff3Writer.write(fo, dnaSequenceList);<a name="line.151"></a>
<span class="sourceLineNo">152</span>            fo.close();<a name="line.152"></a>
<span class="sourceLineNo">153</span>        }<a name="line.153"></a>
<span class="sourceLineNo">154</span><a name="line.154"></a>
<span class="sourceLineNo">155</span>        if (false) {<a name="line.155"></a>
<span class="sourceLineNo">156</span>            LinkedHashMap&lt;String, ChromosomeSequence&gt; dnaSequenceList = GeneFeatureHelper.loadFastaAddGeneFeaturesFromGlimmerGFF3(new File("/Users/Scooter/scripps/dyadic/analysis/454Scaffolds/454Scaffolds-16.fna"), new File("/Users/Scooter/scripps/dyadic/GlimmerHMM/c1_glimmerhmm-16.gff"));<a name="line.156"></a>
<span class="sourceLineNo">157</span>            GFF3Writer gff3Writer = new GFF3Writer();<a name="line.157"></a>
<span class="sourceLineNo">158</span>            gff3Writer.write(System.out, dnaSequenceList);<a name="line.158"></a>
<span class="sourceLineNo">159</span>        }<a name="line.159"></a>
<span class="sourceLineNo">160</span>//        System.out.println(listGenes);<a name="line.160"></a>
<span class="sourceLineNo">161</span>        //      GeneMarkGTF.write( list, args[1] );<a name="line.161"></a>
<span class="sourceLineNo">162</span>    }<a name="line.162"></a>
<span class="sourceLineNo">163</span>}<a name="line.163"></a>




























































</pre>
</div>
</body>
</html>
