<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!-- NewPage -->
<html lang="en">
<head>
<!-- Generated by javadoc (version 1.7.0_05) on Fri Nov 30 10:54:59 PST 2012 -->
<meta http-equiv="Content-Type" content="text/html" charset="UTF-8">
<title>Index (biojava3-structure 3.0.5 API)</title>
<meta name="date" content="2012-11-30">
<link rel="stylesheet" type="text/css" href="./stylesheet.css" title="Style">
</head>
<body>
<script type="text/javascript"><!--
    if (location.href.indexOf('is-external=true') == -1) {
        parent.document.title="Index (biojava3-structure 3.0.5 API)";
    }
//-->
</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
<!-- ========= START OF TOP NAVBAR ======= -->
<div class="topNav"><a name="navbar_top">
<!--   -->
</a><a href="#skip-navbar_top" title="Skip navigation links"></a><a name="navbar_top_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="./overview-summary.html">Overview</a></li>
<li>Package</li>
<li>Class</li>
<li>Use</li>
<li><a href="./overview-tree.html">Tree</a></li>
<li><a href="./deprecated-list.html">Deprecated</a></li>
<li class="navBarCell1Rev">Index</li>
<li><a href="./help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li>Prev</li>
<li>Next</li>
</ul>
<ul class="navList">
<li><a href="./index.html?index-all.html" target="_top">Frames</a></li>
<li><a href="index-all.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_top">
<li><a href="./allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_top");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<a name="skip-navbar_top">
<!--   -->
</a></div>
<!-- ========= END OF TOP NAVBAR ========= -->
<div class="contentContainer"><a href="#_A_">A</a>&nbsp;<a href="#_B_">B</a>&nbsp;<a href="#_C_">C</a>&nbsp;<a href="#_D_">D</a>&nbsp;<a href="#_E_">E</a>&nbsp;<a href="#_F_">F</a>&nbsp;<a href="#_G_">G</a>&nbsp;<a href="#_H_">H</a>&nbsp;<a href="#_I_">I</a>&nbsp;<a href="#_J_">J</a>&nbsp;<a href="#_L_">L</a>&nbsp;<a href="#_M_">M</a>&nbsp;<a href="#_N_">N</a>&nbsp;<a href="#_O_">O</a>&nbsp;<a href="#_P_">P</a>&nbsp;<a href="#_Q_">Q</a>&nbsp;<a href="#_R_">R</a>&nbsp;<a href="#_S_">S</a>&nbsp;<a href="#_T_">T</a>&nbsp;<a href="#_U_">U</a>&nbsp;<a href="#_V_">V</a>&nbsp;<a href="#_W_">W</a>&nbsp;<a href="#_X_">X</a>&nbsp;<a href="#_Z_">Z</a>&nbsp;<a name="_A_">
<!--   -->
</a>
<h2 class="title">A</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html#aaScore(char, char)">aaScore(char, char)</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html" title="class in org.biojava.bio.structure.align.util">AFPAlignmentDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/AbstractBean.html" title="class in org.biojava.bio.structure.io.mmcif.model"><span class="strong">AbstractBean</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a></dt>
<dd>
<div class="block">a generic class that implements the toString method for a bean</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AbstractBean.html#AbstractBean()">AbstractBean()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AbstractBean.html" title="class in org.biojava.bio.structure.io.mmcif.model">AbstractBean</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/AbstractStructureAlignment.html" title="class in org.biojava.bio.structure.align"><span class="strong">AbstractStructureAlignment</span></a> - Class in <a href="./org/biojava/bio/structure/align/package-summary.html">org.biojava.bio.structure.align</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/AbstractStructureAlignment.html#AbstractStructureAlignment()">AbstractStructureAlignment()</a></span> - Constructor for class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/AbstractStructureAlignment.html" title="class in org.biojava.bio.structure.align">AbstractStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce"><span class="strong">AbstractUserArgumentProcessor</span></a> - Class in <a href="./org/biojava/bio/structure/align/ce/package-summary.html">org.biojava.bio.structure.align.ce</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html#AbstractUserArgumentProcessor()">AbstractUserArgumentProcessor()</a></span> - Constructor for class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce">AbstractUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/PDBFilter.html#accept(org.biojava.bio.structure.PDBHeader)">accept(PDBHeader)</a></span> - Method in interface org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/PDBFilter.html" title="interface in org.biojava.bio.structure.server">PDBFilter</a></dt>
<dd>
<div class="block">return true if this PDBHeader is passing this filter.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/CutValues.html#AD">AD</a></span> - Variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/CutValues.html" title="class in org.biojava.bio.structure.domain.pdp">CutValues</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/JointFragments.html#add(int, int, int, int)">add(int, int, int, int)</a></span> - Method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/JointFragments.html" title="class in org.biojava.bio.structure.align.helper">JointFragments</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#add(org.biojava.bio.structure.Atom, org.biojava.bio.structure.Atom)">add(Atom, Atom)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">add two atoms ( a + b).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructureAlignmentFactory.html#addAlgorithm(org.biojava.bio.structure.align.StructureAlignment)">addAlgorithm(StructureAlignment)</a></span> - Static method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructureAlignmentFactory.html" title="class in org.biojava.bio.structure.align">StructureAlignmentFactory</a></dt>
<dd>
<div class="block">Adds a new StructureAlignment algorithm to the list.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html#addAlignmentProgressListener(org.biojava.bio.structure.align.events.AlignmentProgressListener)">addAlignmentProgressListener(AlignmentProgressListener)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html" title="class in org.biojava.bio.structure.align.client">FarmJobRunnable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/FarmJob.html#addAlignmentProgressListener(org.biojava.bio.structure.align.events.AlignmentProgressListener)">addAlignmentProgressListener(AlignmentProgressListener)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/FarmJob.html" title="class in org.biojava.bio.structure.align">FarmJob</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#addAltLoc(org.biojava.bio.structure.Group)">addAltLoc(Group)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">Add a group that is an alternate location for this group.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#addAltLoc(org.biojava.bio.structure.Group)">addAltLoc(Group)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#addAtom(org.biojava.bio.structure.Atom)">addAtom(Atom)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">add an atom to this group.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#addAtom(org.biojava.bio.structure.Atom)">addAtom(Atom)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">add an atom to this group.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#addChain(org.biojava.bio.structure.Chain)">addChain(Chain)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#addChain(org.biojava.bio.structure.Chain)">addChain(Chain)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">add a new chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#addChain(org.biojava.bio.structure.Chain, int)">addChain(Chain, int)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">add a new chain, if several models are available.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#addChain(org.biojava.bio.structure.Chain)">addChain(Chain)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">add a new chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#addChain(org.biojava.bio.structure.Chain, int)">addChain(Chain, int)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">add a new chain, if several models are available.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html#addChemComp(org.biojava.bio.structure.io.mmcif.model.ChemComp)">addChemComp(ChemComp)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html" title="class in org.biojava.bio.structure.io.mmcif">ChemicalComponentDictionary</a></dt>
<dd>
<div class="block">add a new component to the dictionary</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/MMCIFFileReader.html#addExtension(java.lang.String)">addExtension(String)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/MMCIFFileReader.html" title="class in org.biojava.bio.structure.io">MMCIFFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#addExtension(java.lang.String)">addExtension(String)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>
<div class="block">define supported file extensions
 compressed extensions .Z,.gz do not need to be specified
 they are dealt with automatically.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/StructureIOFile.html#addExtension(java.lang.String)">addExtension(String)</a></span> - Method in interface org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/StructureIOFile.html" title="interface in org.biojava.bio.structure.io">StructureIOFile</a></dt>
<dd>
<div class="block">add a known File extension.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#addGroup(org.biojava.bio.structure.Group)">addGroup(Group)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">add a group to the list of ATOM record group of this chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#addGroup(org.biojava.bio.structure.Group)">addGroup(Group)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">add a group to the list of ATOM record group of this chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#addMatrixListener(org.biojava.bio.structure.align.ce.MatrixListener)">addMatrixListener(MatrixListener)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#addMatrixListener(org.biojava.bio.structure.align.ce.MatrixListener)">addMatrixListener(MatrixListener)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifParser.html#addMMcifConsumer(org.biojava.bio.structure.io.mmcif.MMcifConsumer)">addMMcifConsumer(MMcifConsumer)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifParser.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifParser</a></dt>
<dd>
<div class="block">Add a MMcifConsumer that listens to even being triggered by the parser and processes the data into a backend provided by the Consumer.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#addMMcifConsumer(org.biojava.bio.structure.io.mmcif.MMcifConsumer)">addMMcifConsumer(MMcifConsumer)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#addModel(java.util.List)">addModel(List&lt;Chain&gt;)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">add a new model.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#addModel(java.util.List)">addModel(List&lt;Chain&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">add a new model.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/FlatFileInstallation.html#addPDBFilter(org.biojava.bio.structure.server.PDBFilter)">addPDBFilter(PDBFilter)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/FlatFileInstallation.html" title="class in org.biojava.bio.structure.server">FlatFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html#addPDBFilter(org.biojava.bio.structure.server.PDBFilter)">addPDBFilter(PDBFilter)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html" title="class in org.biojava.bio.structure.server">MMCIFFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/PDBInstallation.html#addPDBFilter(org.biojava.bio.structure.server.PDBFilter)">addPDBFilter(PDBFilter)</a></span> - Method in interface org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/PDBInstallation.html" title="interface in org.biojava.bio.structure.server">PDBInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
<div class="block">add a filter for PDB files.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructurePairAligner.html#addProgressListener(org.biojava.bio.structure.align.pairwise.AlignmentProgressListener)">addProgressListener(AlignmentProgressListener)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructurePairAligner.html" title="class in org.biojava.bio.structure.align">StructurePairAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsSegment.html#addResidue(org.biojava.bio.structure.io.sifts.SiftsResidue)">addResidue(SiftsResidue)</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsSegment.html" title="class in org.biojava.bio.structure.io.sifts">SiftsSegment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsEntity.html#addSegment(org.biojava.bio.structure.io.sifts.SiftsSegment)">addSegment(SiftsSegment)</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsEntity.html" title="class in org.biojava.bio.structure.io.sifts">SiftsEntity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#addSeqResGroup(org.biojava.bio.structure.Group)">addSeqResGroup(Group)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#addSSBond(org.biojava.bio.structure.SSBond)">addSSBond(SSBond)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">add a single SSBond to this structure</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#addSSBond(org.biojava.bio.structure.SSBond)">addSSBond(SSBond)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">add a single SSBond to this structure</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/SimpleStructureServer.html#addStructureListener(org.biojava.bio.structure.server.StructureListener)">addStructureListener(StructureListener)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/SimpleStructureServer.html" title="class in org.biojava.bio.structure.server">SimpleStructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureFetcherRunnable.html#addStructureListener(org.biojava.bio.structure.server.StructureListener)">addStructureListener(StructureListener)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureFetcherRunnable.html" title="class in org.biojava.bio.structure.server">StructureFetcherRunnable</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureServer.html#addStructureListener(org.biojava.bio.structure.server.StructureListener)">addStructureListener(StructureListener)</a></span> - Method in interface org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureServer.html" title="interface in org.biojava.bio.structure.server">StructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/model/AFP.html" title="class in org.biojava.bio.structure.align.model"><span class="strong">AFP</span></a> - Class in <a href="./org/biojava/bio/structure/align/model/package-summary.html">org.biojava.bio.structure.align.model</a></dt>
<dd>
<div class="block">A class to represent a FATCAT AFP</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFP.html#AFP()">AFP()</a></span> - Constructor for class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFP.html" title="class in org.biojava.bio.structure.align.model">AFP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/AFPTwister.html#afp2Res(org.biojava.bio.structure.align.model.AFPChain, int, int[], int)">afp2Res(AFPChain, int, int[], int)</a></span> - Static method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/AFPTwister.html" title="class in org.biojava.bio.structure.align">AFPTwister</a></dt>
<dd>
<div class="block">Set the list of equivalent residues in the two proteins given a list of AFPs    
 
 WARNING: changes the values for FocusRes1, focusRes2 and FocusResn in afpChain!</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html" title="class in org.biojava.bio.structure.align.util"><span class="strong">AFPAlignmentDisplay</span></a> - Class in <a href="./org/biojava/bio/structure/align/util/package-summary.html">org.biojava.bio.structure.align.util</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html#AFPAlignmentDisplay()">AFPAlignmentDisplay()</a></span> - Constructor for class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html" title="class in org.biojava.bio.structure.align.util">AFPAlignmentDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/fatcat/calc/AFPCalculator.html" title="class in org.biojava.bio.structure.align.fatcat.calc"><span class="strong">AFPCalculator</span></a> - Class in <a href="./org/biojava/bio/structure/align/fatcat/calc/package-summary.html">org.biojava.bio.structure.align.fatcat.calc</a></dt>
<dd>
<div class="block">a class that performs calculations on AFPCHains</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/AFPCalculator.html#AFPCalculator()">AFPCalculator()</a></span> - Constructor for class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/AFPCalculator.html" title="class in org.biojava.bio.structure.align.fatcat.calc">AFPCalculator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model"><span class="strong">AFPChain</span></a> - Class in <a href="./org/biojava/bio/structure/align/model/package-summary.html">org.biojava.bio.structure.align.model</a></dt>
<dd>
<div class="block">a bean to contain the core of an alignment
 
 The FatCat aligner class is working on the AFPChain class.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#AFPChain()">AFPChain()</a></span> - Constructor for class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#AFPChain(org.biojava.bio.structure.align.model.AFPChain)">AFPChain(AFPChain)</a></span> - Constructor for class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">Copy constructor</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/fatcat/calc/AFPChainer.html" title="class in org.biojava.bio.structure.align.fatcat.calc"><span class="strong">AFPChainer</span></a> - Class in <a href="./org/biojava/bio/structure/align/fatcat/calc/package-summary.html">org.biojava.bio.structure.align.fatcat.calc</a></dt>
<dd>
<div class="block">a class to chain AFPs to an alignment</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/AFPChainer.html#AFPChainer()">AFPChainer()</a></span> - Constructor for class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/AFPChainer.html" title="class in org.biojava.bio.structure.align.fatcat.calc">AFPChainer</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/xml/AFPChainFlipper.html" title="class in org.biojava.bio.structure.align.xml"><span class="strong">AFPChainFlipper</span></a> - Class in <a href="./org/biojava/bio/structure/align/xml/package-summary.html">org.biojava.bio.structure.align.xml</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/AFPChainFlipper.html#AFPChainFlipper()">AFPChainFlipper()</a></span> - Constructor for class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/AFPChainFlipper.html" title="class in org.biojava.bio.structure.align.xml">AFPChainFlipper</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/util/AFPChainScorer.html" title="class in org.biojava.bio.structure.align.util"><span class="strong">AFPChainScorer</span></a> - Class in <a href="./org/biojava/bio/structure/align/util/package-summary.html">org.biojava.bio.structure.align.util</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AFPChainScorer.html#AFPChainScorer()">AFPChainScorer()</a></span> - Constructor for class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AFPChainScorer.html" title="class in org.biojava.bio.structure.align.util">AFPChainScorer</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html" title="class in org.biojava.bio.structure.align.model"><span class="strong">AfpChainWriter</span></a> - Class in <a href="./org/biojava/bio/structure/align/model/package-summary.html">org.biojava.bio.structure.align.model</a></dt>
<dd>
<div class="block">A class to convert the data in an AfpChain object to various String outputs.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html#AfpChainWriter()">AfpChainWriter()</a></span> - Constructor for class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html" title="class in org.biojava.bio.structure.align.model">AfpChainWriter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/xml/AFPChainXMLConverter.html" title="class in org.biojava.bio.structure.align.xml"><span class="strong">AFPChainXMLConverter</span></a> - Class in <a href="./org/biojava/bio/structure/align/xml/package-summary.html">org.biojava.bio.structure.align.xml</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/AFPChainXMLConverter.html#AFPChainXMLConverter()">AFPChainXMLConverter()</a></span> - Constructor for class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/AFPChainXMLConverter.html" title="class in org.biojava.bio.structure.align.xml">AFPChainXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/xml/AFPChainXMLParser.html" title="class in org.biojava.bio.structure.align.xml"><span class="strong">AFPChainXMLParser</span></a> - Class in <a href="./org/biojava/bio/structure/align/xml/package-summary.html">org.biojava.bio.structure.align.xml</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/AFPChainXMLParser.html#AFPChainXMLParser()">AFPChainXMLParser()</a></span> - Constructor for class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/AFPChainXMLParser.html" title="class in org.biojava.bio.structure.align.xml">AFPChainXMLParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/fatcat/calc/AFPOptimizer.html" title="class in org.biojava.bio.structure.align.fatcat.calc"><span class="strong">AFPOptimizer</span></a> - Class in <a href="./org/biojava/bio/structure/align/fatcat/calc/package-summary.html">org.biojava.bio.structure.align.fatcat.calc</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/AFPOptimizer.html#AFPOptimizer()">AFPOptimizer()</a></span> - Constructor for class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/AFPOptimizer.html" title="class in org.biojava.bio.structure.align.fatcat.calc">AFPOptimizer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/AFPChainer.html#afpPairConn(int, int, org.biojava.bio.structure.align.fatcat.calc.FatCatParameters, org.biojava.bio.structure.align.model.AFPChain)">afpPairConn(int, int, FatCatParameters, AFPChain)</a></span> - Static method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/AFPChainer.html" title="class in org.biojava.bio.structure.align.fatcat.calc">AFPChainer</a></dt>
<dd>
<div class="block">//Key function: calculate the connectivity of AFP pairs
  //no compatibility criteria is executed
  //note: afp1 is previous to afp2 in terms of the position
     //this module must be optimized</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/fatcat/calc/AFPPostProcessor.html" title="class in org.biojava.bio.structure.align.fatcat.calc"><span class="strong">AFPPostProcessor</span></a> - Class in <a href="./org/biojava/bio/structure/align/fatcat/calc/package-summary.html">org.biojava.bio.structure.align.fatcat.calc</a></dt>
<dd>
<div class="block">does post processing after alignment chaingin</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/AFPPostProcessor.html#AFPPostProcessor()">AFPPostProcessor()</a></span> - Constructor for class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/AFPPostProcessor.html" title="class in org.biojava.bio.structure.align.fatcat.calc">AFPPostProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/AFPTwister.html" title="class in org.biojava.bio.structure.align"><span class="strong">AFPTwister</span></a> - Class in <a href="./org/biojava/bio/structure/align/package-summary.html">org.biojava.bio.structure.align</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/AFPTwister.html#AFPTwister()">AFPTwister()</a></span> - Constructor for class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/AFPTwister.html" title="class in org.biojava.bio.structure.align">AFPTwister</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/BioJavaStructureAlignment.html#algorithmName">algorithmName</a></span> - Static variable in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/BioJavaStructureAlignment.html" title="class in org.biojava.bio.structure.align">BioJavaStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCPMain.html#algorithmName">algorithmName</a></span> - Static variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCPMain.html" title="class in org.biojava.bio.structure.align.ce">CeCPMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeMain.html#algorithmName">algorithmName</a></span> - Static variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeMain.html" title="class in org.biojava.bio.structure.align.ce">CeMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeSideChainMain.html#algorithmName">algorithmName</a></span> - Static variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeSideChainMain.html" title="class in org.biojava.bio.structure.align.ce">CeSideChainMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/OptimalCECPMain.html#algorithmName">algorithmName</a></span> - Static variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/OptimalCECPMain.html" title="class in org.biojava.bio.structure.align.ce">OptimalCECPMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCatFlexible.html#algorithmName">algorithmName</a></span> - Static variable in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCatFlexible.html" title="class in org.biojava.bio.structure.align.fatcat">FatCatFlexible</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCatRigid.html#algorithmName">algorithmName</a></span> - Static variable in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCatRigid.html" title="class in org.biojava.bio.structure.align.fatcat">FatCatRigid</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWaterman3Daligner.html#algorithmName">algorithmName</a></span> - Static variable in class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWaterman3Daligner.html" title="class in org.biojava.bio.structure.align.seq">SmithWaterman3Daligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/Gotoh.html#ALIGFACTOR">ALIGFACTOR</a></span> - Static variable in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/Gotoh.html" title="class in org.biojava.bio.structure.align.pairwise">Gotoh</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/helper/AligMatEl.html" title="class in org.biojava.bio.structure.align.helper"><span class="strong">AligMatEl</span></a> - Class in <a href="./org/biojava/bio/structure/align/helper/package-summary.html">org.biojava.bio.structure.align.helper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/AligMatEl.html#AligMatEl()">AligMatEl()</a></span> - Constructor for class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/AligMatEl.html" title="class in org.biojava.bio.structure.align.helper">AligMatEl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/AbstractStructureAlignment.html#align(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">align(Atom[], Atom[])</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/AbstractStructureAlignment.html" title="class in org.biojava.bio.structure.align">AbstractStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/AbstractStructureAlignment.html#align(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], java.lang.Object)">align(Atom[], Atom[], Object)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/AbstractStructureAlignment.html" title="class in org.biojava.bio.structure.align">AbstractStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/BioJavaStructureAlignment.html#align(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">align(Atom[], Atom[])</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/BioJavaStructureAlignment.html" title="class in org.biojava.bio.structure.align">BioJavaStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/BioJavaStructureAlignment.html#align(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], java.lang.Object)">align(Atom[], Atom[], Object)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/BioJavaStructureAlignment.html" title="class in org.biojava.bio.structure.align">BioJavaStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCPMain.html#align(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], java.lang.Object)">align(Atom[], Atom[], Object)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCPMain.html" title="class in org.biojava.bio.structure.align.ce">CeCPMain</a></dt>
<dd>
<div class="block">Aligns ca1 and ca2 using a heuristic to check for CPs.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeMain.html#align(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], java.lang.Object)">align(Atom[], Atom[], Object)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeMain.html" title="class in org.biojava.bio.structure.align.ce">CeMain</a></dt>
<dd>
<div class="block">Align ca2 onto ca1.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeMain.html#align(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">align(Atom[], Atom[])</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeMain.html" title="class in org.biojava.bio.structure.align.ce">CeMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/OptimalCECPMain.html#align(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], java.lang.Object)">align(Atom[], Atom[], Object)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/OptimalCECPMain.html" title="class in org.biojava.bio.structure.align.ce">OptimalCECPMain</a></dt>
<dd>
<div class="block">Finds the optimal alignment between two proteins allowing for a circular
 permutation (CP).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatAligner.html#align(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], boolean, org.biojava.bio.structure.align.fatcat.calc.FatCatParameters)">align(Atom[], Atom[], boolean, FatCatParameters)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatAligner.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCat.html#align(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], org.biojava.bio.structure.align.fatcat.calc.FatCatParameters, boolean)">align(Atom[], Atom[], FatCatParameters, boolean)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCat.html" title="class in org.biojava.bio.structure.align.fatcat">FatCat</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCatFlexible.html#align(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">align(Atom[], Atom[])</a></span> - Method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCatFlexible.html" title="class in org.biojava.bio.structure.align.fatcat">FatCatFlexible</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCatFlexible.html#align(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], java.lang.Object)">align(Atom[], Atom[], Object)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCatFlexible.html" title="class in org.biojava.bio.structure.align.fatcat">FatCatFlexible</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCatRigid.html#align(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">align(Atom[], Atom[])</a></span> - Method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCatRigid.html" title="class in org.biojava.bio.structure.align.fatcat">FatCatRigid</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCatRigid.html#align(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], java.lang.Object)">align(Atom[], Atom[], Object)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCatRigid.html" title="class in org.biojava.bio.structure.align.fatcat">FatCatRigid</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWaterman3Daligner.html#align(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">align(Atom[], Atom[])</a></span> - Method in class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWaterman3Daligner.html" title="class in org.biojava.bio.structure.align.seq">SmithWaterman3Daligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWaterman3Daligner.html#align(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], java.lang.Object)">align(Atom[], Atom[], Object)</a></span> - Method in class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWaterman3Daligner.html" title="class in org.biojava.bio.structure.align.seq">SmithWaterman3Daligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructureAlignment.html#align(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">align(Atom[], Atom[])</a></span> - Method in interface org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructureAlignment.html" title="interface in org.biojava.bio.structure.align">StructureAlignment</a></dt>
<dd>
<div class="block">Run an alignment while specifying the atoms to be aligned.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructureAlignment.html#align(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], java.lang.Object)">align(Atom[], Atom[], Object)</a></span> - Method in interface org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructureAlignment.html" title="interface in org.biojava.bio.structure.align">StructureAlignment</a></dt>
<dd>
<div class="block">run an alignment and also send a bean containing the parameters.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructurePairAligner.html#align(org.biojava.bio.structure.Structure, org.biojava.bio.structure.Structure)">align(Structure, Structure)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructurePairAligner.html" title="class in org.biojava.bio.structure.align">StructurePairAligner</a></dt>
<dd>
<div class="block">Calculate the alignment between the two full structures with default parameters</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructurePairAligner.html#align(org.biojava.bio.structure.Structure, org.biojava.bio.structure.Structure, org.biojava.bio.structure.align.StrucAligParameters)">align(Structure, Structure, StrucAligParameters)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructurePairAligner.html" title="class in org.biojava.bio.structure.align">StructurePairAligner</a></dt>
<dd>
<div class="block">Calculate the alignment between the two full structures with user provided parameters</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructurePairAligner.html#align(org.biojava.bio.structure.Structure, java.lang.String, org.biojava.bio.structure.Structure, java.lang.String)">align(Structure, String, Structure, String)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructurePairAligner.html" title="class in org.biojava.bio.structure.align">StructurePairAligner</a></dt>
<dd>
<div class="block">Align two chains from the structures.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructurePairAligner.html#align(org.biojava.bio.structure.Structure, java.lang.String, org.biojava.bio.structure.Structure, java.lang.String, org.biojava.bio.structure.align.StrucAligParameters)">align(Structure, String, Structure, String, StrucAligParameters)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructurePairAligner.html" title="class in org.biojava.bio.structure.align">StructurePairAligner</a></dt>
<dd>
<div class="block">Aligns two chains from the structures using user provided parameters.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructurePairAligner.html#align(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], org.biojava.bio.structure.align.StrucAligParameters)">align(Atom[], Atom[], StrucAligParameters)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructurePairAligner.html" title="class in org.biojava.bio.structure.align">StructurePairAligner</a></dt>
<dd>
<div class="block">calculate the  protein structure superimposition, between two sets of atoms.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/SeqRes2AtomAligner.html#align(org.biojava.bio.structure.Structure, java.util.List)">align(Structure, List&lt;Chain&gt;)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/SeqRes2AtomAligner.html" title="class in org.biojava.bio.structure.io">SeqRes2AtomAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/SeqRes2AtomAligner.html#align(java.util.List, java.util.List)">align(List&lt;Group&gt;, List&lt;Group&gt;)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/SeqRes2AtomAligner.html" title="class in org.biojava.bio.structure.io">SeqRes2AtomAligner</a></dt>
<dd>
<div class="block">aligns two chains of groups, where the first parent is representing the
 list of amino acids as obtained from the SEQRES records, and the second parent
 represents the groups obtained from the ATOM records (and containing the actual ATOM information).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AligNPE.html#align_NPE(org.biojava.bio.structure.jama.Matrix, org.biojava.bio.structure.align.StrucAligParameters)">align_NPE(Matrix, StrucAligParameters)</a></span> - Static method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AligNPE.html" title="class in org.biojava.bio.structure.align.pairwise">AligNPE</a></dt>
<dd>
<div class="block">Align w/o penalizing endpags.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/pairwise/Alignable.html" title="interface in org.biojava.bio.structure.align.pairwise"><span class="strong">Alignable</span></a> - Interface in <a href="./org/biojava/bio/structure/align/pairwise/package-summary.html">org.biojava.bio.structure.align.pairwise</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCat.html#alignFlexible(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], org.biojava.bio.structure.align.fatcat.calc.FatCatParameters)">alignFlexible(Atom[], Atom[], FatCatParameters)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCat.html" title="class in org.biojava.bio.structure.align.fatcat">FatCat</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AlignmentTools.html#alignmentAsMap(org.biojava.bio.structure.align.model.AFPChain)">alignmentAsMap(AFPChain)</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AlignmentTools.html" title="class in org.biojava.bio.structure.align.util">AlignmentTools</a></dt>
<dd>
<div class="block">Creates a Map specifying the alignment as a mapping between residue indices
 of protein 1 and residue indices of protein 2.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/CountProgressListener.html#alignmentEnded()">alignmentEnded()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/CountProgressListener.html" title="class in org.biojava.bio.structure.align.client">CountProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/events/AlignmentProgressListener.html#alignmentEnded()">alignmentEnded()</a></span> - Method in interface org.biojava.bio.structure.align.events.<a href="./org/biojava/bio/structure/align/events/AlignmentProgressListener.html" title="interface in org.biojava.bio.structure.align.events">AlignmentProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/events/AlignmentProgressListener.html" title="interface in org.biojava.bio.structure.align.events"><span class="strong">AlignmentProgressListener</span></a> - Interface in <a href="./org/biojava/bio/structure/align/events/package-summary.html">org.biojava.bio.structure.align.events</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/pairwise/AlignmentProgressListener.html" title="class in org.biojava.bio.structure.align.pairwise"><span class="strong">AlignmentProgressListener</span></a> - Class in <a href="./org/biojava/bio/structure/align/pairwise/package-summary.html">org.biojava.bio.structure.align.pairwise</a></dt>
<dd>
<div class="block">A class to listen to progress of the structure alignmnent calcualtions</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentProgressListener.html#AlignmentProgressListener()">AlignmentProgressListener()</a></span> - Constructor for class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentProgressListener.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise"><span class="strong">AlignmentResult</span></a> - Class in <a href="./org/biojava/bio/structure/align/pairwise/package-summary.html">org.biojava.bio.structure.align.pairwise</a></dt>
<dd>
<div class="block">A class to track the alignment results in a flat file</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html#AlignmentResult()">AlignmentResult()</a></span> - Constructor for class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentResult</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/CountProgressListener.html#alignmentStarted(java.lang.String, java.lang.String)">alignmentStarted(String, String)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/CountProgressListener.html" title="class in org.biojava.bio.structure.align.client">CountProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/events/AlignmentProgressListener.html#alignmentStarted(java.lang.String, java.lang.String)">alignmentStarted(String, String)</a></span> - Method in interface org.biojava.bio.structure.align.events.<a href="./org/biojava/bio/structure/align/events/AlignmentProgressListener.html" title="interface in org.biojava.bio.structure.align.events">AlignmentProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/util/AlignmentTools.html" title="class in org.biojava.bio.structure.align.util"><span class="strong">AlignmentTools</span></a> - Class in <a href="./org/biojava/bio/structure/align/util/package-summary.html">org.biojava.bio.structure.align.util</a></dt>
<dd>
<div class="block">Some utility methods for analyzing and manipulating AFPChains.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AlignmentTools.html#AlignmentTools()">AlignmentTools()</a></span> - Constructor for class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AlignmentTools.html" title="class in org.biojava.bio.structure.align.util">AlignmentTools</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/util/AlignmentTools.IdentityMap.html" title="class in org.biojava.bio.structure.align.util"><span class="strong">AlignmentTools.IdentityMap</span></a>&lt;<a href="./org/biojava/bio/structure/align/util/AlignmentTools.IdentityMap.html" title="type parameter in AlignmentTools.IdentityMap">K</a>&gt; - Class in <a href="./org/biojava/bio/structure/align/util/package-summary.html">org.biojava.bio.structure.align.util</a></dt>
<dd>
<div class="block">A Map<K,V> can be viewed as a function from K to V.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AlignmentTools.IdentityMap.html#AlignmentTools.IdentityMap()">AlignmentTools.IdentityMap()</a></span> - Constructor for class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AlignmentTools.IdentityMap.html" title="class in org.biojava.bio.structure.align.util">AlignmentTools.IdentityMap</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/SeqRes2AtomAligner.html#alignNucleotideGroups(java.util.List, java.util.List)">alignNucleotideGroups(List&lt;Group&gt;, List&lt;Group&gt;)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/SeqRes2AtomAligner.html" title="class in org.biojava.bio.structure.io">SeqRes2AtomAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/OptimalCECPMain.html#alignOptimal(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], java.lang.Object, org.biojava.bio.structure.align.model.AFPChain[])">alignOptimal(Atom[], Atom[], Object, AFPChain[])</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/OptimalCECPMain.html" title="class in org.biojava.bio.structure.align.ce">OptimalCECPMain</a></dt>
<dd>
<div class="block">Finds the optimal alignment between two proteins allowing for a circular
 permutation (CP).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html#alignPair(java.lang.String, java.lang.String)">alignPair(String, String)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html" title="class in org.biojava.bio.structure.align.client">FarmJobRunnable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html#alignPair(java.lang.String, java.lang.String, java.lang.String)">alignPair(String, String, String)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html" title="class in org.biojava.bio.structure.align.client">FarmJobRunnable</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/pairwise/AligNPE.html" title="class in org.biojava.bio.structure.align.pairwise"><span class="strong">AligNPE</span></a> - Class in <a href="./org/biojava/bio/structure/align/pairwise/package-summary.html">org.biojava.bio.structure.align.pairwise</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AligNPE.html#AligNPE()">AligNPE()</a></span> - Constructor for class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AligNPE.html" title="class in org.biojava.bio.structure.align.pairwise">AligNPE</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/OptimalCECPMain.html#alignPermuted(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], java.lang.Object, int)">alignPermuted(Atom[], Atom[], Object, int)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/OptimalCECPMain.html" title="class in org.biojava.bio.structure.align.ce">OptimalCECPMain</a></dt>
<dd>
<div class="block">Aligns ca1 with ca2 permuted by <i>cp</i> residues.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCat.html#alignRigid(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">alignRigid(Atom[], Atom[])</a></span> - Method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCat.html" title="class in org.biojava.bio.structure.align.fatcat">FatCat</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCat.html#alignRigid(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], org.biojava.bio.structure.align.fatcat.calc.FatCatParameters)">alignRigid(Atom[], Atom[], FatCatParameters)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCat.html" title="class in org.biojava.bio.structure.align.fatcat">FatCat</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoSCOP.html#alignSuperfamily()">alignSuperfamily()</a></span> - Method in class demo.<a href="./demo/DemoSCOP.html" title="class in demo">DemoSCOP</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/helper/AlignTools.html" title="class in org.biojava.bio.structure.align.helper"><span class="strong">AlignTools</span></a> - Class in <a href="./org/biojava/bio/structure/align/helper/package-summary.html">org.biojava.bio.structure.align.helper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/AlignTools.html#AlignTools()">AlignTools()</a></span> - Constructor for class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/AlignTools.html" title="class in org.biojava.bio.structure.align.helper">AlignTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/chem/PolymerType.html#ALL_POLYMER_TYPES">ALL_POLYMER_TYPES</a></span> - Static variable in enum org.biojava.bio.structure.io.mmcif.chem.<a href="./org/biojava/bio/structure/io/mmcif/chem/PolymerType.html" title="enum in org.biojava.bio.structure.io.mmcif.chem">PolymerType</a></dt>
<dd>
<div class="block">Convenience <tt>Set</tt> of all polymer types.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/AllChemCompProvider.html" title="class in org.biojava.bio.structure.io.mmcif"><span class="strong">AllChemCompProvider</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/package-summary.html">org.biojava.bio.structure.io.mmcif</a></dt>
<dd>
<div class="block">A ChemComp provider that downloads and caches the components.cif file from the wwPDB site.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/AllChemCompProvider.html#AllChemCompProvider()">AllChemCompProvider()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/AllChemCompProvider.html" title="class in org.biojava.bio.structure.io.mmcif">AllChemCompProvider</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/pairwise/AltAligComparator.html" title="class in org.biojava.bio.structure.align.pairwise"><span class="strong">AltAligComparator</span></a> - Class in <a href="./org/biojava/bio/structure/align/pairwise/package-summary.html">org.biojava.bio.structure.align.pairwise</a></dt>
<dd>
<div class="block">a comparator to sort AlternativeAlignments based on their number of equivalent residues
 and RMSD.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AltAligComparator.html#AltAligComparator()">AltAligComparator()</a></span> - Constructor for class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AltAligComparator.html" title="class in org.biojava.bio.structure.align.pairwise">AltAligComparator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise"><span class="strong">AlternativeAlignment</span></a> - Class in <a href="./org/biojava/bio/structure/align/pairwise/package-summary.html">org.biojava.bio.structure.align.pairwise</a></dt>
<dd>
<div class="block">Implements a class which handles one possible (alternative) solution.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#AlternativeAlignment()">AlternativeAlignment()</a></span> - Constructor for class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/AminoAcid.html" title="interface in org.biojava.bio.structure"><span class="strong">AminoAcid</span></a> - Interface in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">
 A <a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure"><code>Group</code></a> that represents an AminoAcid.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/GroupType.html#AMINOACID">AMINOACID</a></span> - Static variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/GroupType.html" title="class in org.biojava.bio.structure">GroupType</a></dt>
<dd>
<div class="block">the type for amino acids</div>
</dd>
<dt><a href="./org/biojava/bio/structure/AminoAcidImpl.html" title="class in org.biojava.bio.structure"><span class="strong">AminoAcidImpl</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">AminoAcid inherits most from Hetatom.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcidImpl.html#AminoAcidImpl()">AminoAcidImpl()</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcidImpl.html" title="class in org.biojava.bio.structure">AminoAcidImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#amount(org.biojava.bio.structure.Atom)">amount(Atom)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">amount.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#angle(org.biojava.bio.structure.Atom, org.biojava.bio.structure.Atom)">angle(Atom, Atom)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">angle.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentJoiner.html#angleCheckOk(org.biojava.bio.structure.align.pairwise.FragmentPair, org.biojava.bio.structure.align.pairwise.FragmentPair, float)">angleCheckOk(FragmentPair, FragmentPair, float)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentJoiner.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentJoiner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#apairs_from_idxlst(org.biojava.bio.structure.align.helper.JointFragments)">apairs_from_idxlst(JointFragments)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>
<div class="block">Set apairs according to a list of (i,j) tuples.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#apairs_from_seed(int, int, int)">apairs_from_seed(int, int, int)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>
<div class="block">Set apairs according to a seed position.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AlignmentTools.html#applyAlignment(java.util.Map, int)">applyAlignment(Map&lt;T, T&gt;, int)</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AlignmentTools.html" title="class in org.biojava.bio.structure.align.util">AlignmentTools</a></dt>
<dd>
<div class="block">Applies an alignment k times.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AlignmentTools.html#applyAlignment(java.util.Map, java.util.Map, int)">applyAlignment(Map&lt;S, T&gt;, Map&lt;T, S&gt;, int)</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AlignmentTools.html" title="class in org.biojava.bio.structure.align.util">AlignmentTools</a></dt>
<dd>
<div class="block">Applies an alignment k times.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/FlatFileInstallation.html#applyFilters()">applyFilters()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/FlatFileInstallation.html" title="class in org.biojava.bio.structure.server">FlatFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentJoiner.html#approach_ap3(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], org.biojava.bio.structure.align.pairwise.FragmentPair[], org.biojava.bio.structure.align.StrucAligParameters)">approach_ap3(Atom[], Atom[], FragmentPair[], StrucAligParameters)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentJoiner.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentJoiner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#arrayLeftDivide(org.biojava.bio.structure.jama.Matrix)">arrayLeftDivide(Matrix)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Element-by-element left division, C = A.\B</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#arrayLeftDivideEquals(org.biojava.bio.structure.jama.Matrix)">arrayLeftDivideEquals(Matrix)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Element-by-element left division in place, A = A.\B</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#arrayRightDivide(org.biojava.bio.structure.jama.Matrix)">arrayRightDivide(Matrix)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Element-by-element right division, C = A./B</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#arrayRightDivideEquals(org.biojava.bio.structure.jama.Matrix)">arrayRightDivideEquals(Matrix)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Element-by-element right division in place, A = A./B</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#arrayTimes(org.biojava.bio.structure.jama.Matrix)">arrayTimes(Matrix)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Element-by-element multiplication, C = A.*B</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#arrayTimesEquals(org.biojava.bio.structure.jama.Matrix)">arrayTimesEquals(Matrix)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Element-by-element multiplication in place, A = A.*B</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStruc.html#assign(org.biojava.bio.structure.Structure)">assign(Structure)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStruc.html" title="class in org.biojava.bio.structure.secstruc">SecStruc</a></dt>
<dd>
<div class="block">assigns the secondary structure to the groups in this Structure object
 and set the results in the group properties.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/domain/AssignmentXMLSerializer.html" title="class in org.biojava.bio.structure.domain"><span class="strong">AssignmentXMLSerializer</span></a> - Class in <a href="./org/biojava/bio/structure/domain/package-summary.html">org.biojava.bio.structure.domain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/AssignmentXMLSerializer.html#AssignmentXMLSerializer()">AssignmentXMLSerializer()</a></span> - Constructor for class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/AssignmentXMLSerializer.html" title="class in org.biojava.bio.structure.domain">AssignmentXMLSerializer</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure"><span class="strong">Atom</span></a> - Interface in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">A simple interface for an Atom.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileParsingParameters.html#ATOM_CA_THRESHOLD">ATOM_CA_THRESHOLD</a></span> - Static variable in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileParsingParameters.html" title="class in org.biojava.bio.structure.io">FileParsingParameters</a></dt>
<dd>
<div class="block">the maximum number of atoms that will be parsed before the parser switches to a CA-only
 representation of the PDB file.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util"><span class="strong">AtomCache</span></a> - Class in <a href="./org/biojava/bio/structure/align/util/package-summary.html">org.biojava.bio.structure.align.util</a></dt>
<dd>
<div class="block">A utility class that provides easy access to Structure objects.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#AtomCache()">AtomCache()</a></span> - Constructor for class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>
<div class="block">Default AtomCache constructor.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#AtomCache(java.lang.String, boolean)">AtomCache(String, boolean)</a></span> - Constructor for class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>
<div class="block">Creates an instance of an AtomCache that is pointed to the a particular
 path in the file system.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#AtomCache(org.biojava.bio.structure.align.util.UserConfiguration)">AtomCache(UserConfiguration)</a></span> - Constructor for class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>
<div class="block">Creates a new AtomCache object based on the provided UserConfiguration.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure"><span class="strong">AtomImpl</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">Implementation of an Atom of a PDB file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#AtomImpl()">AtomImpl()</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/AtomIterator.html" title="class in org.biojava.bio.structure"><span class="strong">AtomIterator</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">an iterator over all atoms of a structure / group.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomIterator.html#AtomIterator(org.biojava.bio.structure.Structure)">AtomIterator(Structure)</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomIterator.html" title="class in org.biojava.bio.structure">AtomIterator</a></dt>
<dd>
<div class="block">Constructs an AtomIterator object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomIterator.html#AtomIterator(org.biojava.bio.structure.Group)">AtomIterator(Group)</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomIterator.html" title="class in org.biojava.bio.structure">AtomIterator</a></dt>
<dd>
<div class="block">Constructs an AtomIterator object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcid.html#ATOMRECORD">ATOMRECORD</a></span> - Static variable in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcid.html" title="interface in org.biojava.bio.structure">AminoAcid</a></dt>
<dd>
<div class="block">Field to distribguish AminoAcids that have been created from SEQRES records and ATOM records.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#atoms">atoms</a></span> - Variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model"><span class="strong">AtomSite</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#AtomSite()">AtomSite()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/AuditAuthor.html" title="class in org.biojava.bio.structure.io.mmcif.model"><span class="strong">AuditAuthor</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AuditAuthor.html#AuditAuthor()">AuditAuthor()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AuditAuthor.html" title="class in org.biojava.bio.structure.io.mmcif.model">AuditAuthor</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/Author.html" title="class in org.biojava.bio.structure"><span class="strong">Author</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">Describes author attributes for author information in a PDB file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Author.html#Author()">Author()</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Author.html" title="class in org.biojava.bio.structure">Author</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_B_">
<!--   -->
</a>
<h2 class="title">B</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#backboneAtomNames">backboneAtomNames</a></span> - Static variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>
<div class="block">The names of the Atoms that form the backbone.</div>
</dd>
<dt><span class="strong"><a href="./demo/DemoChangeChemCompProvider.html#basicLoad(org.biojava.bio.structure.io.PDBFileReader, boolean, java.lang.String)">basicLoad(PDBFileReader, boolean, String)</a></span> - Method in class demo.<a href="./demo/DemoChangeChemCompProvider.html" title="class in demo">DemoChangeChemCompProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoDomainsplit.html#basicLoad(java.lang.String)">basicLoad(String)</a></span> - Method in class demo.<a href="./demo/DemoDomainsplit.html" title="class in demo">DemoDomainsplit</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoLoadStructure.html#basicLoad()">basicLoad()</a></span> - Method in class demo.<a href="./demo/DemoLoadStructure.html" title="class in demo">DemoLoadStructure</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/scop/BerkeleyScopInstallation.html" title="class in org.biojava.bio.structure.scop"><span class="strong">BerkeleyScopInstallation</span></a> - Class in <a href="./org/biojava/bio/structure/scop/package-summary.html">org.biojava.bio.structure.scop</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/BerkeleyScopInstallation.html#BerkeleyScopInstallation()">BerkeleyScopInstallation()</a></span> - Constructor for class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/BerkeleyScopInstallation.html" title="class in org.biojava.bio.structure.scop">BerkeleyScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#bestTrace1">bestTrace1</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#bestTrace1">bestTrace1</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#bestTrace2">bestTrace2</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#bestTrace2">bestTrace2</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#bestTraces1">bestTraces1</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#bestTraces1">bestTraces1</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#bestTraces2">bestTraces2</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#bestTraces2">bestTraces2</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#bestTraceScore">bestTraceScore</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#bestTraceScore">bestTraceScore</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#bestTracesN">bestTracesN</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#bestTracesN">bestTracesN</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#bestTracesScores">bestTracesScores</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#bestTracesScores">bestTracesScores</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/secstruc/BigSqrt.html" title="class in org.biojava.bio.structure.secstruc"><span class="strong">BigSqrt</span></a> - Class in <a href="./org/biojava/bio/structure/secstruc/package-summary.html">org.biojava.bio.structure.secstruc</a></dt>
<dd>
<div class="block">calculate a more precise SQRT.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/BigSqrt.html#BigSqrt()">BigSqrt()</a></span> - Constructor for class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/BigSqrt.html" title="class in org.biojava.bio.structure.secstruc">BigSqrt</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/RemoteBioUnitDataProvider.html#BIO_ASSEMBLY">BIO_ASSEMBLY</a></span> - Static variable in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/RemoteBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">RemoteBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/quaternary/BioAssemblyTools.html" title="class in org.biojava.bio.structure.quaternary"><span class="strong">BioAssemblyTools</span></a> - Class in <a href="./org/biojava/bio/structure/quaternary/package-summary.html">org.biojava.bio.structure.quaternary</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/BioAssemblyTools.html#BioAssemblyTools()">BioAssemblyTools()</a></span> - Constructor for class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/BioAssemblyTools.html" title="class in org.biojava.bio.structure.quaternary">BioAssemblyTools</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/BioJavaStructureAlignment.html" title="class in org.biojava.bio.structure.align"><span class="strong">BioJavaStructureAlignment</span></a> - Class in <a href="./org/biojava/bio/structure/align/package-summary.html">org.biojava.bio.structure.align</a></dt>
<dd>
<div class="block">Wrapper for the BioJava Structure Alignment Implementation</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/BioJavaStructureAlignment.html#BioJavaStructureAlignment()">BioJavaStructureAlignment()</a></span> - Constructor for class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/BioJavaStructureAlignment.html" title="class in org.biojava.bio.structure.align">BioJavaStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#BIOL_ASSEMBLY_IDENTIFIER">BIOL_ASSEMBLY_IDENTIFIER</a></span> - Static variable in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/quaternary/BiologicalAssemblyBuilder.html" title="class in org.biojava.bio.structure.quaternary"><span class="strong">BiologicalAssemblyBuilder</span></a> - Class in <a href="./org/biojava/bio/structure/quaternary/package-summary.html">org.biojava.bio.structure.quaternary</a></dt>
<dd>
<div class="block">Reconstructs the quaternary structure of a protein from an asymmetric unit</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/BiologicalAssemblyBuilder.html#BiologicalAssemblyBuilder()">BiologicalAssemblyBuilder()</a></span> - Constructor for class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/BiologicalAssemblyBuilder.html" title="class in org.biojava.bio.structure.quaternary">BiologicalAssemblyBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/quaternary/io/BioUnitDataProvider.html" title="interface in org.biojava.bio.structure.quaternary.io"><span class="strong">BioUnitDataProvider</span></a> - Interface in <a href="./org/biojava/bio/structure/quaternary/io/package-summary.html">org.biojava.bio.structure.quaternary.io</a></dt>
<dd>
<div class="block">Provides access to the data that is needed in order to rebuild the correct biological assembly of a protein.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/quaternary/io/BioUnitDataProviderFactory.html" title="class in org.biojava.bio.structure.quaternary.io"><span class="strong">BioUnitDataProviderFactory</span></a> - Class in <a href="./org/biojava/bio/structure/quaternary/io/package-summary.html">org.biojava.bio.structure.quaternary.io</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/AFPOptimizer.html#blockInfo(org.biojava.bio.structure.align.model.AFPChain)">blockInfo(AFPChain)</a></span> - Static method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/AFPOptimizer.html" title="class in org.biojava.bio.structure.align.fatcat.calc">AFPOptimizer</a></dt>
<dd>
<div class="block">get the afp list and residue list for each block</div>
</dd>
<dt><a href="./org/biojava/bio/structure/Bond.html" title="class in org.biojava.bio.structure"><span class="strong">Bond</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">Simple bond  - it's just an edge/vertex for two <a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure"><code>Atom</code></a> nodes.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Bond.html#Bond()">Bond()</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Bond.html" title="class in org.biojava.bio.structure">Bond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Bond.html#Bond(double, org.biojava.bio.structure.BondType, org.biojava.bio.structure.Group, org.biojava.bio.structure.Atom, org.biojava.bio.structure.Group, org.biojava.bio.structure.Atom)">Bond(double, BondType, Group, Atom, Group, Atom)</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Bond.html" title="class in org.biojava.bio.structure">Bond</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/BondType.html" title="enum in org.biojava.bio.structure"><span class="strong">BondType</span></a> - Enum in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">Work in progress - NOT final!</div>
</dd>
<dt><a href="./org/biojava/bio/structure/secstruc/BridgeType.html" title="enum in org.biojava.bio.structure.secstruc"><span class="strong">BridgeType</span></a> - Enum in <a href="./org/biojava/bio/structure/secstruc/package-summary.html">org.biojava.bio.structure.secstruc</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_C_">
<!--   -->
</a>
<h2 class="title">C</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCPMain.CPRange.html#c">c</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCPMain.CPRange.html" title="class in org.biojava.bio.structure.align.ce">CeCPMain.CPRange</a></dt>
<dd>
<div class="block">first allowed c-term</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#CA_AND_SIDE_CHAIN_ANGLE_SCORING">CA_AND_SIDE_CHAIN_ANGLE_SCORING</a></span> - Static variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStruc.html#CA_MIN_DIST">CA_MIN_DIST</a></span> - Static variable in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStruc.html" title="class in org.biojava.bio.structure.secstruc">SecStruc</a></dt>
<dd>
<div class="block">the minimal distance of two CA atoms if H-bonds are allowed to form.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#caAtomName">caAtomName</a></span> - Static variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>
<div class="block">The Atom name of C-alpha atoms.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html#cache">cache</a></span> - Variable in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html" title="class in org.biojava.bio.structure.align.client">FarmJobRunnable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/SerializableCache.html#cache(K, V)">cache(K, V)</a></span> - Method in class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/SerializableCache.html" title="class in org.biojava.bio.structure.domain">SerializableCache</a></dt>
<dd>
<div class="block">This will not cache null values.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/scop/CachedRemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop"><span class="strong">CachedRemoteScopInstallation</span></a> - Class in <a href="./org/biojava/bio/structure/scop/package-summary.html">org.biojava.bio.structure.scop</a></dt>
<dd>
<div class="block">An extension of the RemoteScopInstallation that caches some of the data locally.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/CachedRemoteScopInstallation.html#CachedRemoteScopInstallation()">CachedRemoteScopInstallation()</a></span> - Constructor for class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/CachedRemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">CachedRemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/CachedRemoteScopInstallation.html#CachedRemoteScopInstallation(boolean)">CachedRemoteScopInstallation(boolean)</a></span> - Constructor for class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/CachedRemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">CachedRemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/util/CacheFactory.html" title="class in org.biojava.bio.structure.align.util"><span class="strong">CacheFactory</span></a> - Class in <a href="./org/biojava/bio/structure/align/util/package-summary.html">org.biojava.bio.structure.align.util</a></dt>
<dd>
<div class="block">provides a SoftHashMap singleton.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/SerializableCache.html#cacheFileName">cacheFileName</a></span> - Variable in class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/SerializableCache.html" title="class in org.biojava.bio.structure.domain">SerializableCache</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/CAConverter.html" title="class in org.biojava.bio.structure.io"><span class="strong">CAConverter</span></a> - Class in <a href="./org/biojava/bio/structure/io/package-summary.html">org.biojava.bio.structure.io</a></dt>
<dd>
<div class="block">Converts full atom representations to Calpha only ones.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/CAConverter.html#CAConverter()">CAConverter()</a></span> - Constructor for class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/CAConverter.html" title="class in org.biojava.bio.structure.io">CAConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/AFPChainer.html#calAfpRmsd(int, int[], int, org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">calAfpRmsd(int, int[], int, AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/AFPChainer.html" title="class in org.biojava.bio.structure.align.fatcat.calc">AFPChainer</a></dt>
<dd>
<div class="block">//return the rmsd of the residues from the segments that form the given AFP list
  //this value can be a measurement (1) for the connectivity of the AFPs</div>
</dd>
<dt><a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure"><span class="strong">Calc</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">utility operations on Atoms, AminoAcids, etc.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#Calc()">Calc()</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#calc_rmsd(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], int, boolean, boolean)">calc_rmsd(Atom[], Atom[], int, boolean, boolean)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>
<div class="block">superimpose and get rmsd</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#calc_rmsd(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], int, boolean, boolean)">calc_rmsd(Atom[], Atom[], int, boolean, boolean)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>
<div class="block">superimpose and get rmsd</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html#calcIdSimilarity(char[], char[], int)">calcIdSimilarity(char[], char[], int)</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html" title="class in org.biojava.bio.structure.align.util">AFPAlignmentDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#calcScores(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">calcScores(Atom[], Atom[])</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>
<div class="block">calculates  scores for this alignment ( %id )</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#calcSimilarity()">calcSimilarity()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentProgressListener.html#calculatedFragmentPairs(java.util.List)">calculatedFragmentPairs(List&lt;FragmentPair&gt;)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentProgressListener.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStruc.html#calculateHBondEnergy(org.biojava.bio.structure.secstruc.SecStrucGroup, org.biojava.bio.structure.secstruc.SecStrucGroup)">calculateHBondEnergy(SecStrucGroup, SecStrucGroup)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStruc.html" title="class in org.biojava.bio.structure.secstruc">SecStruc</a></dt>
<dd>
<div class="block">calculate HBond energy of two groups in cal/mol ...</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCPMain.html#calculateMinCP(int[], int, int, int)">calculateMinCP(int[], int, int, int)</a></span> - Static method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCPMain.html" title="class in org.biojava.bio.structure.align.ce">CeCPMain</a></dt>
<dd>
<div class="block">Finds the alignment index of the residues minCPlength before and after
 the duplication.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#calculateSuperpositionByIdx(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">calculateSuperpositionByIdx(Atom[], Atom[])</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeMain.html#calculator">calculator</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeMain.html" title="class in org.biojava.bio.structure.align.ce">CeMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html#call()">call()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html" title="class in org.biojava.bio.structure.align">CallableStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html" title="class in org.biojava.bio.structure.align"><span class="strong">CallableStructureAlignment</span></a> - Class in <a href="./org/biojava/bio/structure/align/package-summary.html">org.biojava.bio.structure.align</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html#CallableStructureAlignment()">CallableStructureAlignment()</a></span> - Constructor for class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html" title="class in org.biojava.bio.structure.align">CallableStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/SigEva.html#calNS(org.biojava.bio.structure.align.fatcat.calc.FatCatParameters, org.biojava.bio.structure.align.model.AFPChain)">calNS(FatCatParameters, AFPChain)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/SigEva.html" title="class in org.biojava.bio.structure.align.fatcat.calc">SigEva</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/SigEva.html#calSigAll(org.biojava.bio.structure.align.fatcat.calc.FatCatParameters, org.biojava.bio.structure.align.model.AFPChain)">calSigAll(FatCatParameters, AFPChain)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/SigEva.html" title="class in org.biojava.bio.structure.align.fatcat.calc">SigEva</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/quaternary/CartesianProduct.html" title="class in org.biojava.bio.structure.quaternary"><span class="strong">CartesianProduct</span></a>&lt;<a href="./org/biojava/bio/structure/quaternary/CartesianProduct.html" title="type parameter in CartesianProduct">T</a>&gt; - Class in <a href="./org/biojava/bio/structure/quaternary/package-summary.html">org.biojava.bio.structure.quaternary</a></dt>
<dd>
<div class="block">A cartesian product between two lists A and B is the set of all ordered pairs 
 of the elements of both sets.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/CartesianProduct.html#CartesianProduct(java.util.List, java.util.List)">CartesianProduct(List&lt;T&gt;, List&lt;T&gt;)</a></span> - Constructor for class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/CartesianProduct.html" title="class in org.biojava.bio.structure.quaternary">CartesianProduct</a></dt>
<dd>
<div class="block">Class constructor specifying the two lists of a cartesian product.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#CATH_DOWNLOAD">CATH_DOWNLOAD</a></span> - Static variable in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/cath/CathCategory.html" title="enum in org.biojava.bio.structure.cath"><span class="strong">CathCategory</span></a> - Enum in <a href="./org/biojava/bio/structure/cath/package-summary.html">org.biojava.bio.structure.cath</a></dt>
<dd>
<div class="block">The categories found within CATH.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/cath/CathDatabase.html" title="interface in org.biojava.bio.structure.cath"><span class="strong">CathDatabase</span></a> - Interface in <a href="./org/biojava/bio/structure/cath/package-summary.html">org.biojava.bio.structure.cath</a></dt>
<dd>
<div class="block">General API for interacting with CATH.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath"><span class="strong">CathDomain</span></a> - Class in <a href="./org/biojava/bio/structure/cath/package-summary.html">org.biojava.bio.structure.cath</a></dt>
<dd>
<div class="block">A class which represents a single CATH domain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#CathDomain()">CathDomain()</a></span> - Constructor for class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/cath/CathFragment.html" title="class in org.biojava.bio.structure.cath"><span class="strong">CathFragment</span></a> - Class in <a href="./org/biojava/bio/structure/cath/package-summary.html">org.biojava.bio.structure.cath</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathFragment.html#CathFragment()">CathFragment()</a></span> - Constructor for class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathFragment.html" title="class in org.biojava.bio.structure.cath">CathFragment</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath"><span class="strong">CathInstallation</span></a> - Class in <a href="./org/biojava/bio/structure/cath/package-summary.html">org.biojava.bio.structure.cath</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#CathInstallation(java.lang.String, boolean, boolean)">CathInstallation(String, boolean, boolean)</a></span> - Constructor for class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#CathInstallation(java.lang.String)">CathInstallation(String)</a></span> - Constructor for class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#CathInstallation()">CathInstallation()</a></span> - Constructor for class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/cath/CathNode.html" title="class in org.biojava.bio.structure.cath"><span class="strong">CathNode</span></a> - Class in <a href="./org/biojava/bio/structure/cath/package-summary.html">org.biojava.bio.structure.cath</a></dt>
<dd>
<div class="block">Represents a node in the CATH hierarchy.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathNode.html#CathNode()">CathNode()</a></span> - Constructor for class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathNode.html" title="class in org.biojava.bio.structure.cath">CathNode</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/cath/CathSegment.html" title="class in org.biojava.bio.structure.cath"><span class="strong">CathSegment</span></a> - Class in <a href="./org/biojava/bio/structure/cath/package-summary.html">org.biojava.bio.structure.cath</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathSegment.html#CathSegment()">CathSegment()</a></span> - Constructor for class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathSegment.html" title="class in org.biojava.bio.structure.cath">CathSegment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#cbAtomName">cbAtomName</a></span> - Static variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce"><span class="strong">CECalculator</span></a> - Class in <a href="./org/biojava/bio/structure/align/ce/package-summary.html">org.biojava.bio.structure.align.ce</a></dt>
<dd>
<div class="block">This is based on the original Combinatorial Extension (CE) source code from 2003 or 2004 (CE version 2.3),
 as has been originally developed by I.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#CECalculator(org.biojava.bio.structure.align.ce.CeParameters)">CECalculator(CeParameters)</a></span> - Constructor for class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce"><span class="strong">CeCalculatorEnhanced</span></a> - Class in <a href="./org/biojava/bio/structure/align/ce/package-summary.html">org.biojava.bio.structure.align.ce</a></dt>
<dd>
<div class="block">This is based on the original Combinatorial Extension (CE) source code from 2003 or 2004 (CE version 2.3),
 as has been originally developed by I.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#CeCalculatorEnhanced(org.biojava.bio.structure.align.ce.CeParameters)">CeCalculatorEnhanced(CeParameters)</a></span> - Constructor for class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/ce/CeCPMain.html" title="class in org.biojava.bio.structure.align.ce"><span class="strong">CeCPMain</span></a> - Class in <a href="./org/biojava/bio/structure/align/ce/package-summary.html">org.biojava.bio.structure.align.ce</a></dt>
<dd>
<div class="block">A wrapper for <a href="./org/biojava/bio/structure/align/ce/CeMain.html" title="class in org.biojava.bio.structure.align.ce"><code>CeMain</code></a> which sets default parameters to be appropriate for finding
 circular permutations.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCPMain.html#CeCPMain()">CeCPMain()</a></span> - Constructor for class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCPMain.html" title="class in org.biojava.bio.structure.align.ce">CeCPMain</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/ce/CeCPMain.CPRange.html" title="class in org.biojava.bio.structure.align.ce"><span class="strong">CeCPMain.CPRange</span></a> - Class in <a href="./org/biojava/bio/structure/align/ce/package-summary.html">org.biojava.bio.structure.align.ce</a></dt>
<dd>
<div class="block">Tiny wrapper for the disallowed regions of an alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCPMain.CPRange.html#CeCPMain.CPRange()">CeCPMain.CPRange()</a></span> - Constructor for class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCPMain.CPRange.html" title="class in org.biojava.bio.structure.align.ce">CeCPMain.CPRange</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/ce/CeCPUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce"><span class="strong">CeCPUserArgumentProcessor</span></a> - Class in <a href="./org/biojava/bio/structure/align/ce/package-summary.html">org.biojava.bio.structure.align.ce</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCPUserArgumentProcessor.html#CeCPUserArgumentProcessor()">CeCPUserArgumentProcessor()</a></span> - Constructor for class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCPUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce">CeCPUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/ce/CeMain.html" title="class in org.biojava.bio.structure.align.ce"><span class="strong">CeMain</span></a> - Class in <a href="./org/biojava/bio/structure/align/ce/package-summary.html">org.biojava.bio.structure.align.ce</a></dt>
<dd>
<div class="block">The main class of the Java implementation of the Combinatorial Extension Algorithm (CE),
 as has been originally developed by I.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeMain.html#CeMain()">CeMain()</a></span> - Constructor for class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeMain.html" title="class in org.biojava.bio.structure.align.ce">CeMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#centerAtoms(org.biojava.bio.structure.Atom[])">centerAtoms(Atom[])</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">Center the atoms at the Centroid.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#centerAtoms(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom)">centerAtoms(Atom[], Atom)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">Center the atoms at the Centroid, if the centroid is already know.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#centerOfMass(org.biojava.bio.structure.Atom[])">centerOfMass(Atom[])</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce"><span class="strong">CeParameters</span></a> - Class in <a href="./org/biojava/bio/structure/align/ce/package-summary.html">org.biojava.bio.structure.align.ce</a></dt>
<dd>
<div class="block">Contains the parameters that can be sent to CE</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#CeParameters()">CeParameters()</a></span> - Constructor for class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/ce/CeSideChainMain.html" title="class in org.biojava.bio.structure.align.ce"><span class="strong">CeSideChainMain</span></a> - Class in <a href="./org/biojava/bio/structure/align/ce/package-summary.html">org.biojava.bio.structure.align.ce</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeSideChainMain.html#CeSideChainMain()">CeSideChainMain()</a></span> - Constructor for class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeSideChainMain.html" title="class in org.biojava.bio.structure.align.ce">CeSideChainMain</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/ce/CeSideChainUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce"><span class="strong">CeSideChainUserArgumentProcessor</span></a> - Class in <a href="./org/biojava/bio/structure/align/ce/package-summary.html">org.biojava.bio.structure.align.ce</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeSideChainUserArgumentProcessor.html#CeSideChainUserArgumentProcessor()">CeSideChainUserArgumentProcessor()</a></span> - Constructor for class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeSideChainUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce">CeSideChainUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/ce/CeUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce"><span class="strong">CeUserArgumentProcessor</span></a> - Class in <a href="./org/biojava/bio/structure/align/ce/package-summary.html">org.biojava.bio.structure.align.ce</a></dt>
<dd>
<div class="block">process the arguments from command line</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeUserArgumentProcessor.html#CeUserArgumentProcessor()">CeUserArgumentProcessor()</a></span> - Constructor for class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce">CeUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure"><span class="strong">Chain</span></a> - Interface in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">
 Defines the interface for a Chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#CHAIN_NR_SYMBOL">CHAIN_NR_SYMBOL</a></span> - Static variable in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#CHAIN_SPLIT_SYMBOL">CHAIN_SPLIT_SYMBOL</a></span> - Static variable in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/StructureName.html#chainId">chainId</a></span> - Variable in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/StructureName.html" title="class in org.biojava.bio.structure.align.client">StructureName</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure"><span class="strong">ChainImpl</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">A Chain in a PDB file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#ChainImpl()">ChainImpl()</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Constructs a ChainImpl object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/DownloadChemCompProvider.html#checkDoFirstInstall()">checkDoFirstInstall()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/DownloadChemCompProvider.html" title="class in org.biojava.bio.structure.io.mmcif">DownloadChemCompProvider</a></dt>
<dd>
<div class="block">checks if the chemical components already have been installed into the PDB directory.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/AllChemCompProvider.html#checkPath()">checkPath()</a></span> - Static method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/AllChemCompProvider.html" title="class in org.biojava.bio.structure.io.mmcif">AllChemCompProvider</a></dt>
<dd>
<div class="block">make sure all paths are initialized correctly</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/DownloadChemCompProvider.html#CHEM_COMP_CACHE_DIRECTORY">CHEM_COMP_CACHE_DIRECTORY</a></span> - Static variable in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/DownloadChemCompProvider.html" title="class in org.biojava.bio.structure.io.mmcif">DownloadChemCompProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/chem/ResidueType.html#chem_comp_type">chem_comp_type</a></span> - Variable in enum org.biojava.bio.structure.io.mmcif.chem.<a href="./org/biojava/bio/structure/io/mmcif/chem/ResidueType.html" title="enum in org.biojava.bio.structure.io.mmcif.chem">ResidueType</a></dt>
<dd>
<div class="block">String value of the type</div>
</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model"><span class="strong">ChemComp</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a></dt>
<dd>
<div class="block">A definition for a Chemical Component, as maintained by the wwPDB.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#ChemComp()">ChemComp()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif"><span class="strong">ChemCompConsumer</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/package-summary.html">org.biojava.bio.structure.io.mmcif</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#ChemCompConsumer()">ChemCompConsumer()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html" title="class in org.biojava.bio.structure.io.mmcif.model"><span class="strong">ChemCompDescriptor</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a></dt>
<dd>
<div class="block">Container object for _pdbx_chem_comp_descriptor</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html#ChemCompDescriptor()">ChemCompDescriptor()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemCompDescriptor</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./demo/ChemCompDistribution.html" title="class in demo"><span class="strong">ChemCompDistribution</span></a> - Class in <a href="./demo/package-summary.html">demo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/ChemCompDistribution.html#ChemCompDistribution()">ChemCompDistribution()</a></span> - Constructor for class demo.<a href="./demo/ChemCompDistribution.html" title="class in demo">ChemCompDistribution</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/ChemCompGroupFactory.html" title="class in org.biojava.bio.structure.io.mmcif"><span class="strong">ChemCompGroupFactory</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/package-summary.html">org.biojava.bio.structure.io.mmcif</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompGroupFactory.html#ChemCompGroupFactory()">ChemCompGroupFactory()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompGroupFactory.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompGroupFactory</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/ChemCompProvider.html" title="interface in org.biojava.bio.structure.io.mmcif"><span class="strong">ChemCompProvider</span></a> - Interface in <a href="./org/biojava/bio/structure/io/mmcif/package-summary.html">org.biojava.bio.structure.io.mmcif</a></dt>
<dd>
<div class="block">Interface that is implemented by all classes that can provide <a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model"><code>ChemComp</code></a> definitions.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/chem/ChemCompTools.html" title="class in org.biojava.bio.structure.io.mmcif.chem"><span class="strong">ChemCompTools</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/chem/package-summary.html">org.biojava.bio.structure.io.mmcif.chem</a></dt>
<dd>
<div class="block">Some tools for working with chemical compounds.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/chem/ChemCompTools.html#ChemCompTools()">ChemCompTools()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.chem.<a href="./org/biojava/bio/structure/io/mmcif/chem/ChemCompTools.html" title="class in org.biojava.bio.structure.io.mmcif.chem">ChemCompTools</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html" title="class in org.biojava.bio.structure.io.mmcif"><span class="strong">ChemicalComponentDictionary</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/package-summary.html">org.biojava.bio.structure.io.mmcif</a></dt>
<dd>
<div class="block">A representation of the Chemical Component Dictionary.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html#ChemicalComponentDictionary()">ChemicalComponentDictionary()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html" title="class in org.biojava.bio.structure.io.mmcif">ChemicalComponentDictionary</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#chol()">chol()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Cholesky Decomposition</div>
</dd>
<dt><a href="./org/biojava/bio/structure/jama/CholeskyDecomposition.html" title="class in org.biojava.bio.structure.jama"><span class="strong">CholeskyDecomposition</span></a> - Class in <a href="./org/biojava/bio/structure/jama/package-summary.html">org.biojava.bio.structure.jama</a></dt>
<dd>
<div class="block">Cholesky Decomposition.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/CholeskyDecomposition.html#CholeskyDecomposition(org.biojava.bio.structure.jama.Matrix)">CholeskyDecomposition(Matrix)</a></span> - Constructor for class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/CholeskyDecomposition.html" title="class in org.biojava.bio.structure.jama">CholeskyDecomposition</a></dt>
<dd>
<div class="block">Cholesky algorithm for symmetric and positive definite matrix.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/BerkeleyScopInstallation.html#claFileName">claFileName</a></span> - Static variable in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/BerkeleyScopInstallation.html" title="class in org.biojava.bio.structure.scop">BerkeleyScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#claFileName">claFileName</a></span> - Static variable in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/ShortSegmentRemover.html#cleanup(java.util.List)">cleanup(List&lt;Domain&gt;)</a></span> - Static method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/ShortSegmentRemover.html" title="class in org.biojava.bio.structure.domain.pdp">ShortSegmentRemover</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructureAlignmentFactory.html#clearAlgorithms()">clearAlgorithms()</a></span> - Static method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructureAlignmentFactory.html" title="class in org.biojava.bio.structure.align">StructureAlignmentFactory</a></dt>
<dd>
<div class="block">Removes all algorithms from the list</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#clearAtoms()">clearAtoms()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">Remove all atoms from this group.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#clearAtoms()">clearAtoms()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">remove all atoms</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBStatus.html#clearCache()">clearCache()</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBStatus.html" title="class in org.biojava.bio.structure">PDBStatus</a></dt>
<dd>
<div class="block">The status of PDB IDs are cached to reduce server overload.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifParser.html#clearConsumers()">clearConsumers()</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifParser.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifParser</a></dt>
<dd>
<div class="block">Remove all consumers from the parser.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#clearConsumers()">clearConsumers()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/MMCIFFileReader.html#clearExtensions()">clearExtensions()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/MMCIFFileReader.html" title="class in org.biojava.bio.structure.io">MMCIFFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#clearExtensions()">clearExtensions()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>
<div class="block">clear the supported file extensions</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/StructureIOFile.html#clearExtensions()">clearExtensions()</a></span> - Method in interface org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/StructureIOFile.html" title="interface in org.biojava.bio.structure.io">StructureIOFile</a></dt>
<dd>
<div class="block">clear all file extensions</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/FlatFileInstallation.html#clearFilters()">clearFilters()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/FlatFileInstallation.html" title="class in org.biojava.bio.structure.server">FlatFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html#clearFilters()">clearFilters()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html" title="class in org.biojava.bio.structure.server">MMCIFFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/PDBInstallation.html#clearFilters()">clearFilters()</a></span> - Method in interface org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/PDBInstallation.html" title="interface in org.biojava.bio.structure.server">PDBInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
<div class="block">remove all filters, next time getAll is called, it will return all available PDBs</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html#clearListeners()">clearListeners()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html" title="class in org.biojava.bio.structure.align.client">FarmJobRunnable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/FarmJob.html#clearListeners()">clearListeners()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/FarmJob.html" title="class in org.biojava.bio.structure.align">FarmJob</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructurePairAligner.html#clearListeners()">clearListeners()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructurePairAligner.html" title="class in org.biojava.bio.structure.align">StructurePairAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureFetcherRunnable.html#clearListeners()">clearListeners()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureFetcherRunnable.html" title="class in org.biojava.bio.structure.server">StructureFetcherRunnable</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/SimpleStructureServer.html#clearStructureListeners()">clearStructureListeners()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/SimpleStructureServer.html" title="class in org.biojava.bio.structure.server">SimpleStructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureServer.html#clearStructureListeners()">clearStructureListeners()</a></span> - Method in interface org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureServer.html" title="interface in org.biojava.bio.structure.server">StructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/util/CliTools.html" title="class in org.biojava.bio.structure.align.util"><span class="strong">CliTools</span></a> - Class in <a href="./org/biojava/bio/structure/align/util/package-summary.html">org.biojava.bio.structure.align.util</a></dt>
<dd>
<div class="block">Utilities for autoconfiguring javabeans based on command line arguments.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#clone()">clone()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">Creates and returns a copy of this object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#clone()">clone()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcidImpl.html#clone()">clone()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcidImpl.html" title="class in org.biojava.bio.structure">AminoAcidImpl</a></dt>
<dd>
<div class="block">returns and identical copy of this Group object .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#clone()">clone()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">returns and identical copy of this  object .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#clone()">clone()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>
<div class="block">returns and identical copy of this  object .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#clone()">clone()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">returns an identical copy of this Chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#clone()">clone()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Returns an identical copy of this Chain .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#clone()">clone()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#clone()">clone()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">returns and identical copy of this Group object .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/GroupIterator.html#clone()">clone()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/GroupIterator.html" title="class in org.biojava.bio.structure">GroupIterator</a></dt>
<dd>
<div class="block">Creates and returns a copy of this object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#clone()">clone()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">returns and identical copy of this Group object .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#clone()">clone()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Clone the Matrix object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/NucleotideImpl.html#clone()">clone()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/NucleotideImpl.html" title="class in org.biojava.bio.structure">NucleotideImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html#clone()">clone()</a></span> - Method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html" title="class in org.biojava.bio.structure.quaternary">ModelTransformationMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDescription.html#clone()">clone()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDescription.html" title="class in org.biojava.bio.structure.scop">ScopDescription</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#clone()">clone()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/HBond.html#clone()">clone()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/HBond.html" title="class in org.biojava.bio.structure.secstruc">HBond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SSBond.html#clone()">clone()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SSBond.html" title="class in org.biojava.bio.structure">SSBond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#clone()">clone()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">returns an identical copy of this Structure object</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#clone()">clone()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">returns an identical copy of this structure .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#cloneCAArray(org.biojava.bio.structure.Atom[])">cloneCAArray(Atom[])</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>
<div class="block">Provides an equivalent copy of Atoms in a new array.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#cloneGroups(org.biojava.bio.structure.Atom[])">cloneGroups(Atom[])</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>
<div class="block">Clone a set of CA Atoms, but returns the parent groups</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/SynchronizedOutFile.html#close()">close()</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/SynchronizedOutFile.html" title="class in org.biojava.bio.structure.align.util">SynchronizedOutFile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ClusterAltAligs.html#cluster(org.biojava.bio.structure.align.pairwise.AlternativeAlignment[])">cluster(AlternativeAlignment[])</a></span> - Static method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/ClusterAltAligs.html" title="class in org.biojava.bio.structure.align">ClusterAltAligs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ClusterAltAligs.html#cluster(org.biojava.bio.structure.align.pairwise.AlternativeAlignment[], int)">cluster(AlternativeAlignment[], int)</a></span> - Static method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/ClusterAltAligs.html" title="class in org.biojava.bio.structure.align">ClusterAltAligs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/ClusterDomains.html#cluster(java.util.List, org.biojava.bio.structure.domain.pdp.PDPDistanceMatrix)">cluster(List&lt;Domain&gt;, PDPDistanceMatrix)</a></span> - Static method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/ClusterDomains.html" title="class in org.biojava.bio.structure.domain.pdp">ClusterDomains</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/ClusterAltAligs.html" title="class in org.biojava.bio.structure.align"><span class="strong">ClusterAltAligs</span></a> - Class in <a href="./org/biojava/bio/structure/align/package-summary.html">org.biojava.bio.structure.align</a></dt>
<dd>
<div class="block">A class that clusters alternative alignments according to their
 similarity.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ClusterAltAligs.html#ClusterAltAligs()">ClusterAltAligs()</a></span> - Constructor for class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/ClusterAltAligs.html" title="class in org.biojava.bio.structure.align">ClusterAltAligs</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/domain/pdp/ClusterDomains.html" title="class in org.biojava.bio.structure.domain.pdp"><span class="strong">ClusterDomains</span></a> - Class in <a href="./org/biojava/bio/structure/domain/pdp/package-summary.html">org.biojava.bio.structure.domain.pdp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/ClusterDomains.html#ClusterDomains()">ClusterDomains()</a></span> - Constructor for class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/ClusterDomains.html" title="class in org.biojava.bio.structure.domain.pdp">ClusterDomains</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html#code">code</a></span> - Variable in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html" title="class in org.biojava.bio.structure.quaternary">ModelTransformationMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/util/CollectionTools.html" title="class in org.biojava.bio.structure.align.util"><span class="strong">CollectionTools</span></a> - Class in <a href="./org/biojava/bio/structure/align/util/package-summary.html">org.biojava.bio.structure.align.util</a></dt>
<dd>
<div class="block">Utilities for working with collections.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/CollectionTools.html#CollectionTools()">CollectionTools()</a></span> - Constructor for class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/CollectionTools.html" title="class in org.biojava.bio.structure.align.util">CollectionTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html#commaSpaces">commaSpaces</a></span> - Static variable in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html" title="class in org.biojava.bio.structure.quaternary">ModelTransformationMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/IdxComparator.html#compare(int[], int[])">compare(int[], int[])</a></span> - Method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/IdxComparator.html" title="class in org.biojava.bio.structure.align.helper">IdxComparator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AltAligComparator.html#compare(org.biojava.bio.structure.align.pairwise.AlternativeAlignment, org.biojava.bio.structure.align.pairwise.AlternativeAlignment)">compare(AlternativeAlignment, AlternativeAlignment)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AltAligComparator.html" title="class in org.biojava.bio.structure.align.pairwise">AltAligComparator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/SegmentComparator.html#compare(org.biojava.bio.structure.domain.pdp.Segment, org.biojava.bio.structure.domain.pdp.Segment)">compare(Segment, Segment)</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/SegmentComparator.html" title="class in org.biojava.bio.structure.domain.pdp">SegmentComparator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/PdbPair.html#compareTo(org.biojava.bio.structure.align.client.PdbPair)">compareTo(PdbPair)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/PdbPair.html" title="class in org.biojava.bio.structure.align.client">PdbPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/StructureName.html#compareTo(org.biojava.bio.structure.align.client.StructureName)">compareTo(StructureName)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/StructureName.html" title="class in org.biojava.bio.structure.align.client">StructureName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Domain.html#compareTo(org.biojava.bio.structure.domain.pdp.Domain)">compareTo(Domain)</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Domain.html" title="class in org.biojava.bio.structure.domain.pdp">Domain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Segment.html#compareTo(org.biojava.bio.structure.domain.pdp.Segment)">compareTo(Segment)</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Segment.html" title="class in org.biojava.bio.structure.domain.pdp">Segment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#compareTo(org.biojava.bio.structure.io.mmcif.model.ChemComp)">compareTo(ChemComp)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ResidueNumber.html#compareTo(org.biojava.bio.structure.ResidueNumber)">compareTo(ResidueNumber)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ResidueNumber.html" title="class in org.biojava.bio.structure">ResidueNumber</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Site.html#compareTo(org.biojava.bio.structure.Site)">compareTo(Site)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Site.html" title="class in org.biojava.bio.structure">Site</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure"><span class="strong">Compound</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">Created by IntelliJ IDEA.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#Compound()">Compound()</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#cond()">cond()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Matrix condition (2 norm)</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/SingularValueDecomposition.html#cond()">cond()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/SingularValueDecomposition.html" title="class in org.biojava.bio.structure.jama">SingularValueDecomposition</a></dt>
<dd>
<div class="block">Two norm condition number</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/ce/ConfigStrucAligParams.html" title="interface in org.biojava.bio.structure.align.ce"><span class="strong">ConfigStrucAligParams</span></a> - Interface in <a href="./org/biojava/bio/structure/align/ce/package-summary.html">org.biojava.bio.structure.align.ce</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/util/ConfigurationException.html" title="class in org.biojava.bio.structure.align.util"><span class="strong">ConfigurationException</span></a> - Exception in <a href="./org/biojava/bio/structure/align/util/package-summary.html">org.biojava.bio.structure.align.util</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/ConfigurationException.html#ConfigurationException(java.lang.String)">ConfigurationException(String)</a></span> - Constructor for exception org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/ConfigurationException.html" title="class in org.biojava.bio.structure.align.util">ConfigurationException</a></dt>
<dd>
<div class="block">Constructs a ConfigurationException object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/ConfigurationException.html#ConfigurationException(java.lang.Throwable)">ConfigurationException(Throwable)</a></span> - Constructor for exception org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/ConfigurationException.html" title="class in org.biojava.bio.structure.align.util">ConfigurationException</a></dt>
<dd>
<div class="block">Constructs a ConfigurationException object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/CliTools.html#configureBean(java.lang.Object, java.lang.String[])">configureBean(Object, String[])</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/CliTools.html" title="class in org.biojava.bio.structure.align.util">CliTools</a></dt>
<dd>
<div class="block">Configure a JavaBean based on a set of command line arguments.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#constructWithCopy(double[][])">constructWithCopy(double[][])</a></span> - Static method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Construct a matrix from a copy of a 2-D array.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AlignmentTools.IdentityMap.html#containsKey(java.lang.Object)">containsKey(Object)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AlignmentTools.IdentityMap.html" title="class in org.biojava.bio.structure.align.util">AlignmentTools.IdentityMap</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#convert_3code_1code(java.lang.String)">convert_3code_1code(String)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>
<div class="block">convert three character amino acid codes into single character
  e.g.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#convertAfpChain(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">convertAfpChain(AFPChain, Atom[], Atom[])</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>
<div class="block">copy data from this class into AFPChain container object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#convertAfpChain(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">convertAfpChain(AFPChain, Atom[], Atom[])</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>
<div class="block">copy data from this class into AFPChain container object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#convertAtomsToSeq(org.biojava.bio.structure.Atom[])">convertAtomsToSeq(Atom[])</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/PdbPairXMLConverter.html#convertPairsToXML(java.util.SortedSet, java.lang.String)">convertPairsToXML(SortedSet&lt;PdbPair&gt;, String)</a></span> - Static method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/PdbPairXMLConverter.html" title="class in org.biojava.bio.structure.align.xml">PdbPairXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/JFatCatClient.html#convertStreamToString(java.io.InputStream)">convertStreamToString(InputStream)</a></span> - Static method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/JFatCatClient.html" title="class in org.biojava.bio.structure.align.client">JFatCatClient</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/PdbPairXMLConverter.html#convertXMLtoPairs(java.lang.String)">convertXMLtoPairs(String)</a></span> - Static method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/PdbPairXMLConverter.html" title="class in org.biojava.bio.structure.align.xml">PdbPairXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/util/FileDownloadUtils.html#copy(java.io.File, java.io.File)">copy(File, File)</a></span> - Static method in class org.biojava.bio.structure.io.util.<a href="./org/biojava/bio/structure/io/util/FileDownloadUtils.html" title="class in org.biojava.bio.structure.io.util">FileDownloadUtils</a></dt>
<dd>
<div class="block">Copy the content of file A to B</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#copy()">copy()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Make a deep copy of a matrix</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/client/CountProgressListener.html" title="class in org.biojava.bio.structure.align.client"><span class="strong">CountProgressListener</span></a> - Class in <a href="./org/biojava/bio/structure/align/client/package-summary.html">org.biojava.bio.structure.align.client</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/CountProgressListener.html#CountProgressListener()">CountProgressListener()</a></span> - Constructor for class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/CountProgressListener.html" title="class in org.biojava.bio.structure.align.client">CountProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/OptimalCECPParameters.html#cpPoint">cpPoint</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/OptimalCECPParameters.html" title="class in org.biojava.bio.structure.align.ce">OptimalCECPParameters</a></dt>
<dd>
<div class="block">The CP point, specified as a residue index</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AlignmentTools.html#createAFPChain(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], org.biojava.bio.structure.ResidueNumber[], org.biojava.bio.structure.ResidueNumber[])">createAFPChain(Atom[], Atom[], ResidueNumber[], ResidueNumber[])</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AlignmentTools.html" title="class in org.biojava.bio.structure.align.util">AlignmentTools</a></dt>
<dd>
<div class="block">Fundimentally, an alignment is just a list of aligned residues in each
 protein.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html#createArtificalStructure(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">createArtificalStructure(AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html" title="class in org.biojava.bio.structure.align.util">AFPAlignmentDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/PrepareMMcifIndexFile.html#createMMcifInfoList(java.io.File[], java.io.File, java.io.File)">createMMcifInfoList(File[], File, File)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/PrepareMMcifIndexFile.html" title="class in org.biojava.bio.structure.server">PrepareMMcifIndexFile</a></dt>
<dd>
<div class="block">parses a set of PDB files and writes info into a file
 the file is tab separated and has the following columns:
 name length  resolution depositionDate modificationDate  technique title classification filename

 binaryDirectory: a directory in which binary files containing the atoms will be places, to provide a speedup

 This method needs to be run, before a DBSearch can be performed, since the files created by this method
 are required for the DBSearch</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/PrepareIndexFile.html#createPDBInfoList(java.io.File[], java.io.File, java.io.File)">createPDBInfoList(File[], File, File)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/PrepareIndexFile.html" title="class in org.biojava.bio.structure.server">PrepareIndexFile</a></dt>
<dd>
<div class="block">parses a set of PDB files and writes info into a file
 the file is tab separated and has the following columns:
 name length  resolution depositionDate modificationDate  technique title classification filename

 binaryDirectory: a directory in which binary files containing the atoms will be places, to provide a speedup

 This method needs to be run, before a DBSearch can be performed, since the files created by this method
 are required for the DBSearch</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#createVirtualCBAtom(org.biojava.bio.structure.AminoAcid)">createVirtualCBAtom(AminoAcid)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">creates a virtual C-beta atom.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/domain/pdp/Cut.html" title="class in org.biojava.bio.structure.domain.pdp"><span class="strong">Cut</span></a> - Class in <a href="./org/biojava/bio/structure/domain/pdp/package-summary.html">org.biojava.bio.structure.domain.pdp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Cut.html#Cut()">Cut()</a></span> - Constructor for class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Cut.html" title="class in org.biojava.bio.structure.domain.pdp">Cut</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Cut.html#cut(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.domain.pdp.Domain, org.biojava.bio.structure.domain.pdp.CutValues, int[][], org.biojava.bio.structure.domain.pdp.PDPDistanceMatrix)">cut(Atom[], Domain, CutValues, int[][], PDPDistanceMatrix)</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Cut.html" title="class in org.biojava.bio.structure.domain.pdp">Cut</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html#CUT_OFF_VALUE">CUT_OFF_VALUE</a></span> - Static variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html" title="class in org.biojava.bio.structure.domain.pdp">PDPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html#CUT_OFF_VALUE1">CUT_OFF_VALUE1</a></span> - Static variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html" title="class in org.biojava.bio.structure.domain.pdp">PDPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html#CUT_OFF_VALUE1M">CUT_OFF_VALUE1M</a></span> - Static variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html" title="class in org.biojava.bio.structure.domain.pdp">PDPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html#CUT_OFF_VALUE1S">CUT_OFF_VALUE1S</a></span> - Static variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html" title="class in org.biojava.bio.structure.domain.pdp">PDPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html#CUT_OFF_VALUE2">CUT_OFF_VALUE2</a></span> - Static variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html" title="class in org.biojava.bio.structure.domain.pdp">PDPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/domain/pdp/CutDomain.html" title="class in org.biojava.bio.structure.domain.pdp"><span class="strong">CutDomain</span></a> - Class in <a href="./org/biojava/bio/structure/domain/pdp/package-summary.html">org.biojava.bio.structure.domain.pdp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/CutDomain.html#CutDomain(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.domain.pdp.PDPDistanceMatrix)">CutDomain(Atom[], PDPDistanceMatrix)</a></span> - Constructor for class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/CutDomain.html" title="class in org.biojava.bio.structure.domain.pdp">CutDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/CutDomain.html#cutDomain(org.biojava.bio.structure.domain.pdp.Domain, org.biojava.bio.structure.domain.pdp.CutSites, org.biojava.bio.structure.domain.pdp.PDPDistanceMatrix)">cutDomain(Domain, CutSites, PDPDistanceMatrix)</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/CutDomain.html" title="class in org.biojava.bio.structure.domain.pdp">CutDomain</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/domain/pdp/CutSites.html" title="class in org.biojava.bio.structure.domain.pdp"><span class="strong">CutSites</span></a> - Class in <a href="./org/biojava/bio/structure/domain/pdp/package-summary.html">org.biojava.bio.structure.domain.pdp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/CutSites.html#CutSites()">CutSites()</a></span> - Constructor for class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/CutSites.html" title="class in org.biojava.bio.structure.domain.pdp">CutSites</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/domain/pdp/CutValues.html" title="class in org.biojava.bio.structure.domain.pdp"><span class="strong">CutValues</span></a> - Class in <a href="./org/biojava/bio/structure/domain/pdp/package-summary.html">org.biojava.bio.structure.domain.pdp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/CutValues.html#CutValues()">CutValues()</a></span> - Constructor for class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/CutValues.html" title="class in org.biojava.bio.structure.domain.pdp">CutValues</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_D_">
<!--   -->
</a>
<h2 class="title">D</h2>
<dl>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBremark.html" title="class in org.biojava.bio.structure.io.mmcif.model"><span class="strong">DatabasePDBremark</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBremark.html#DatabasePDBremark()">DatabasePDBremark()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBremark.html" title="class in org.biojava.bio.structure.io.mmcif.model">DatabasePDBremark</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html" title="class in org.biojava.bio.structure.io.mmcif.model"><span class="strong">DatabasePDBrev</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html#DatabasePDBrev()">DatabasePDBrev()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html" title="class in org.biojava.bio.structure.io.mmcif.model">DatabasePDBrev</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/FlatFileInstallation.html#dateFormat">dateFormat</a></span> - Static variable in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/FlatFileInstallation.html" title="class in org.biojava.bio.structure.server">FlatFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html#DBL">DBL</a></span> - Static variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html" title="class in org.biojava.bio.structure.domain.pdp">PDPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure"><span class="strong">DBRef</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">A class to represent database cross references.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#DBRef()">DBRef()</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/AFPCalculator.html#debug">debug</a></span> - Static variable in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/AFPCalculator.html" title="class in org.biojava.bio.structure.align.fatcat.calc">AFPCalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/AFPChainer.html#debug">debug</a></span> - Static variable in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/AFPChainer.html" title="class in org.biojava.bio.structure.align.fatcat.calc">AFPChainer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/AFPOptimizer.html#debug">debug</a></span> - Static variable in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/AFPOptimizer.html" title="class in org.biojava.bio.structure.align.fatcat.calc">AFPOptimizer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/AFPPostProcessor.html#debug">debug</a></span> - Static variable in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/AFPPostProcessor.html" title="class in org.biojava.bio.structure.align.fatcat.calc">AFPPostProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatAligner.html#debug">debug</a></span> - Static variable in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatAligner.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AlignmentTools.html#debug">debug</a></span> - Static variable in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AlignmentTools.html" title="class in org.biojava.bio.structure.align.util">AlignmentTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/FlatFileInstallation.html#DEFAUL_CHAIN_FILE">DEFAUL_CHAIN_FILE</a></span> - Static variable in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/FlatFileInstallation.html" title="class in org.biojava.bio.structure.server">FlatFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#DEFAULT_ALGORITHM_NAME">DEFAULT_ALGORITHM_NAME</a></span> - Static variable in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">the default algorithm used in the RCSB PDB all vs.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#DEFAULT_BATCH_SIZE">DEFAULT_BATCH_SIZE</a></span> - Static variable in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#DEFAULT_CHAIN_ID">DEFAULT_CHAIN_ID</a></span> - Static variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">The default chain identifier is an empty space.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ClusterAltAligs.html#DEFAULT_CLUSTER_CUTOFF">DEFAULT_CLUSTER_CUTOFF</a></span> - Static variable in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/ClusterAltAligs.html" title="class in org.biojava.bio.structure.align">ClusterAltAligs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/HTTPConnectionTools.html#DEFAULT_CONNECTION_TIMEOUT">DEFAULT_CONNECTION_TIMEOUT</a></span> - Static variable in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/HTTPConnectionTools.html" title="class in org.biojava.bio.structure.align.util">HTTPConnectionTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#DEFAULT_DIR_SPLIT">DEFAULT_DIR_SPLIT</a></span> - Static variable in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#DEFAULT_FRAGLEN">DEFAULT_FRAGLEN</a></span> - Static variable in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#DEFAULT_GAP_EXTENSION">DEFAULT_GAP_EXTENSION</a></span> - Static variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#DEFAULT_GAP_OPEN">DEFAULT_GAP_OPEN</a></span> - Static variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/FlatFileInstallation.html#DEFAULT_INDEX_FILE">DEFAULT_INDEX_FILE</a></span> - Static variable in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/FlatFileInstallation.html" title="class in org.biojava.bio.structure.server">FlatFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#DEFAULT_JOB_TIME">DEFAULT_JOB_TIME</a></span> - Static variable in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/BigSqrt.html#DEFAULT_MAX_ITERATIONS">DEFAULT_MAX_ITERATIONS</a></span> - Static variable in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/BigSqrt.html" title="class in org.biojava.bio.structure.secstruc">BigSqrt</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/PdbPairXMLConverter.html#DEFAULT_METHOD_NAME">DEFAULT_METHOD_NAME</a></span> - Static variable in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/PdbPairXMLConverter.html" title="class in org.biojava.bio.structure.align.xml">PdbPairXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCPMain.html#DEFAULT_MIN_CP_LENGTH">DEFAULT_MIN_CP_LENGTH</a></span> - Static variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCPMain.html" title="class in org.biojava.bio.structure.align.ce">CeCPMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#DEFAULT_NR_ALIGNMENTS">DEFAULT_NR_ALIGNMENTS</a></span> - Static variable in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#DEFAULT_NR_THREADS">DEFAULT_NR_THREADS</a></span> - Static variable in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#DEFAULT_oRmsdThr">DEFAULT_oRmsdThr</a></span> - Static variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/PDBDomainProvider.html#DEFAULT_PDB_API_URL">DEFAULT_PDB_API_URL</a></span> - Static variable in class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/PDBDomainProvider.html" title="class in org.biojava.bio.structure.domain">PDBDomainProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#DEFAULT_PDB_FILE_SERVER">DEFAULT_PDB_FILE_SERVER</a></span> - Static variable in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/PDBDomainProvider.html#DEFAULT_PDB_HOST">DEFAULT_PDB_HOST</a></span> - Static variable in class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/PDBDomainProvider.html" title="class in org.biojava.bio.structure.domain">PDBDomainProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#DEFAULT_PDB_PATH">DEFAULT_PDB_PATH</a></span> - Static variable in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBStatus.html#DEFAULT_PDB_SERVER">DEFAULT_PDB_SERVER</a></span> - Static variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBStatus.html" title="class in org.biojava.bio.structure">PDBStatus</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/BioUnitDataProviderFactory.html#DEFAULT_PROVIDER_CLASSNAME">DEFAULT_PROVIDER_CLASSNAME</a></span> - Static variable in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/BioUnitDataProviderFactory.html" title="class in org.biojava.bio.structure.quaternary.io">BioUnitDataProviderFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#DEFAULT_RESOLUTION">DEFAULT_RESOLUTION</a></span> - Static variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/BigSqrt.html#DEFAULT_SCALE">DEFAULT_SCALE</a></span> - Static variable in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/BigSqrt.html" title="class in org.biojava.bio.structure.secstruc">BigSqrt</a></dt>
<dd>
<div class="block">we take 3 aftercomma</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#DEFAULT_SCORING_STRATEGY">DEFAULT_SCORING_STRATEGY</a></span> - Static variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/RemotePDPProvider.html#DEFAULT_SERVER">DEFAULT_SERVER</a></span> - Static variable in class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/RemotePDPProvider.html" title="class in org.biojava.bio.structure.domain">RemotePDPProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html#DEFAULT_SERVER">DEFAULT_SERVER</a></span> - Static variable in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">RemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#DEFAULT_SERVER_URL">DEFAULT_SERVER_URL</a></span> - Static variable in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/RemoteBioUnitDataProvider.html#DEFAULT_SERVERNAME">DEFAULT_SERVERNAME</a></span> - Static variable in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/RemoteBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">RemoteBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/RemoteRawBioUnitDataProvider.html#DEFAULT_SERVERNAME">DEFAULT_SERVERNAME</a></span> - Static variable in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/RemoteRawBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">RemoteRawBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#DEFAULT_VERSION">DEFAULT_VERSION</a></span> - Static variable in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#DEFAULT_VERSION">DEFAULT_VERSION</a></span> - Static variable in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#degreesPerRadian">degreesPerRadian</a></span> - Static variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">Degrees per radian.</div>
</dd>
<dt><a href="./demo/package-summary.html">demo</a> - package demo</dt>
<dd>&nbsp;</dd>
<dt><a href="./demo/DemoAtomCache.html" title="class in demo"><span class="strong">DemoAtomCache</span></a> - Class in <a href="./demo/package-summary.html">demo</a></dt>
<dd>
<div class="block">Example of how to load PDB files using the AtomCache class.</div>
</dd>
<dt><span class="strong"><a href="./demo/DemoAtomCache.html#DemoAtomCache()">DemoAtomCache()</a></span> - Constructor for class demo.<a href="./demo/DemoAtomCache.html" title="class in demo">DemoAtomCache</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./demo/DemoBerkeleyScop.html" title="class in demo"><span class="strong">DemoBerkeleyScop</span></a> - Class in <a href="./demo/package-summary.html">demo</a></dt>
<dd>
<div class="block">A demo for how to use the Berkeley version of SCOP instead of the default UK-SCOP</div>
</dd>
<dt><span class="strong"><a href="./demo/DemoBerkeleyScop.html#DemoBerkeleyScop()">DemoBerkeleyScop()</a></span> - Constructor for class demo.<a href="./demo/DemoBerkeleyScop.html" title="class in demo">DemoBerkeleyScop</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./demo/DemoCATH.html" title="class in demo"><span class="strong">DemoCATH</span></a> - Class in <a href="./demo/package-summary.html">demo</a></dt>
<dd>
<div class="block">An example for how to access CATH data.</div>
</dd>
<dt><span class="strong"><a href="./demo/DemoCATH.html#DemoCATH()">DemoCATH()</a></span> - Constructor for class demo.<a href="./demo/DemoCATH.html" title="class in demo">DemoCATH</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./demo/DemoCE.html" title="class in demo"><span class="strong">DemoCE</span></a> - Class in <a href="./demo/package-summary.html">demo</a></dt>
<dd>
<div class="block">Example of how to run a structure alignment using the CE algorithm.</div>
</dd>
<dt><span class="strong"><a href="./demo/DemoCE.html#DemoCE()">DemoCE()</a></span> - Constructor for class demo.<a href="./demo/DemoCE.html" title="class in demo">DemoCE</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./demo/DemoChangeChemCompProvider.html" title="class in demo"><span class="strong">DemoChangeChemCompProvider</span></a> - Class in <a href="./demo/package-summary.html">demo</a></dt>
<dd>
<div class="block">This demo shows how to use an alternative ChemCompProvider.</div>
</dd>
<dt><span class="strong"><a href="./demo/DemoChangeChemCompProvider.html#DemoChangeChemCompProvider()">DemoChangeChemCompProvider()</a></span> - Constructor for class demo.<a href="./demo/DemoChangeChemCompProvider.html" title="class in demo">DemoChangeChemCompProvider</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./demo/DemoCommandLineStartup.html" title="class in demo"><span class="strong">DemoCommandLineStartup</span></a> - Class in <a href="./demo/package-summary.html">demo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoCommandLineStartup.html#DemoCommandLineStartup()">DemoCommandLineStartup()</a></span> - Constructor for class demo.<a href="./demo/DemoCommandLineStartup.html" title="class in demo">DemoCommandLineStartup</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./demo/DemoDomainsplit.html" title="class in demo"><span class="strong">DemoDomainsplit</span></a> - Class in <a href="./demo/package-summary.html">demo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoDomainsplit.html#DemoDomainsplit()">DemoDomainsplit()</a></span> - Constructor for class demo.<a href="./demo/DemoDomainsplit.html" title="class in demo">DemoDomainsplit</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./demo/DemoFATCAT.html" title="class in demo"><span class="strong">DemoFATCAT</span></a> - Class in <a href="./demo/package-summary.html">demo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoFATCAT.html#DemoFATCAT()">DemoFATCAT()</a></span> - Constructor for class demo.<a href="./demo/DemoFATCAT.html" title="class in demo">DemoFATCAT</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./demo/DemoLoadStructure.html" title="class in demo"><span class="strong">DemoLoadStructure</span></a> - Class in <a href="./demo/package-summary.html">demo</a></dt>
<dd>
<div class="block">Example for how to load protein structures (from PDB files).</div>
</dd>
<dt><span class="strong"><a href="./demo/DemoLoadStructure.html#DemoLoadStructure()">DemoLoadStructure()</a></span> - Constructor for class demo.<a href="./demo/DemoLoadStructure.html" title="class in demo">DemoLoadStructure</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./demo/DemoMMCIFReader.html" title="class in demo"><span class="strong">DemoMMCIFReader</span></a> - Class in <a href="./demo/package-summary.html">demo</a></dt>
<dd>
<div class="block">An example of how to read MMcif files</div>
</dd>
<dt><span class="strong"><a href="./demo/DemoMMCIFReader.html#DemoMMCIFReader()">DemoMMCIFReader()</a></span> - Constructor for class demo.<a href="./demo/DemoMMCIFReader.html" title="class in demo">DemoMMCIFReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./demo/DemoSCOP.html" title="class in demo"><span class="strong">DemoSCOP</span></a> - Class in <a href="./demo/package-summary.html">demo</a></dt>
<dd>
<div class="block">A class demonstrating the use of the SCOP parsing tools</div>
</dd>
<dt><span class="strong"><a href="./demo/DemoSCOP.html#DemoSCOP()">DemoSCOP()</a></span> - Constructor for class demo.<a href="./demo/DemoSCOP.html" title="class in demo">DemoSCOP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html#deserialize(java.io.File)">deserialize(File)</a></span> - Static method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentResult</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/BerkeleyScopInstallation.html#desFileName">desFileName</a></span> - Static variable in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/BerkeleyScopInstallation.html" title="class in org.biojava.bio.structure.scop">BerkeleyScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#desFileName">desFileName</a></span> - Static variable in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/LUDecomposition.html#det()">det()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/LUDecomposition.html" title="class in org.biojava.bio.structure.jama">LUDecomposition</a></dt>
<dd>
<div class="block">Determinant</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#det()">det()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Matrix determinant</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/SerializableCache.html#disableCache()">disableCache()</a></span> - Method in class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/SerializableCache.html" title="class in org.biojava.bio.structure.domain">SerializableCache</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/GuiWrapper.html#display(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">display(AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/GuiWrapper.html" title="class in org.biojava.bio.structure.align.ce">GuiWrapper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#DISTANCE_INCREMENT">DISTANCE_INCREMENT</a></span> - Static variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#distanceIncrement">distanceIncrement</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/secstruc/DistEn.html" title="class in org.biojava.bio.structure.secstruc"><span class="strong">DistEn</span></a> - Class in <a href="./org/biojava/bio/structure/secstruc/package-summary.html">org.biojava.bio.structure.secstruc</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/DistEn.html#DistEn()">DistEn()</a></span> - Constructor for class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/DistEn.html" title="class in org.biojava.bio.structure.secstruc">DistEn</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/chem/PolymerType.html#DNA_ONLY">DNA_ONLY</a></span> - Static variable in enum org.biojava.bio.structure.io.mmcif.chem.<a href="./org/biojava/bio/structure/io/mmcif/chem/PolymerType.html" title="enum in org.biojava.bio.structure.io.mmcif.chem">PolymerType</a></dt>
<dd>
<div class="block">Convenience <tt>Set</tt> of polymer types classified as DNA.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/AFPChainer.html#doChainAfp(org.biojava.bio.structure.align.fatcat.calc.FatCatParameters, org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">doChainAfp(FatCatParameters, AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/AFPChainer.html" title="class in org.biojava.bio.structure.align.fatcat.calc">AFPChainer</a></dt>
<dd>
<div class="block">// Key function: chain (assembly) the AFPs
  // a AFP (k) is defined as (i, j, k), with i and j are staring points
  // AFP extension (eg.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#documentEnd()">documentEnd()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#documentEnd()">documentEnd()</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>
<div class="block">called at end of document</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#documentEnd()">documentEnd()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#documentStart()">documentStart()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#documentStart()">documentStart()</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>
<div class="block">called at start of document</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#documentStart()">documentStart()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>
<div class="block">Start the parsing</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#doIter0(int, double, double)">doIter0(int, double, double)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#doIter0(int, double, double)">doIter0(int, double, double)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/domain/pdp/Domain.html" title="class in org.biojava.bio.structure.domain.pdp"><span class="strong">Domain</span></a> - Class in <a href="./org/biojava/bio/structure/domain/pdp/package-summary.html">org.biojava.bio.structure.domain.pdp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Domain.html#Domain()">Domain()</a></span> - Constructor for class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Domain.html" title="class in org.biojava.bio.structure.domain.pdp">Domain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#domainDescriptionFileName">domainDescriptionFileName</a></span> - Static variable in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#domainListFileName">domainListFileName</a></span> - Static variable in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/domain/DomainProvider.html" title="interface in org.biojava.bio.structure.domain"><span class="strong">DomainProvider</span></a> - Interface in <a href="./org/biojava/bio/structure/domain/package-summary.html">org.biojava.bio.structure.domain</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/domain/DomainProviderFactory.html" title="class in org.biojava.bio.structure.domain"><span class="strong">DomainProviderFactory</span></a> - Class in <a href="./org/biojava/bio/structure/domain/package-summary.html">org.biojava.bio.structure.domain</a></dt>
<dd>
<div class="block">A simple factory object that returns the system wide default DomainProvider</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#domallFileName">domallFileName</a></span> - Static variable in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/HTTPConnectionTools.html#doPOST(java.net.URL, java.lang.String)">doPOST(URL, String)</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/HTTPConnectionTools.html" title="class in org.biojava.bio.structure.align.util">HTTPConnectionTools</a></dt>
<dd>
<div class="block">do a POST to a URL and return the response stream for further processing elsewhere.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/HTTPConnectionTools.html#doPOST(java.net.URL, java.lang.String, int)">doPOST(URL, String, int)</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/HTTPConnectionTools.html" title="class in org.biojava.bio.structure.align.util">HTTPConnectionTools</a></dt>
<dd>
<div class="block">do a POST to a URL and return the response stream for further processing elsewhere.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileConvert.html#doPrintConnections()">doPrintConnections()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileConvert.html" title="class in org.biojava.bio.structure.io">FileConvert</a></dt>
<dd>
<div class="block">returns if the Connections should be added
 default is true;</div>
</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/DownloadChemCompProvider.html" title="class in org.biojava.bio.structure.io.mmcif"><span class="strong">DownloadChemCompProvider</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/package-summary.html">org.biojava.bio.structure.io.mmcif</a></dt>
<dd>
<div class="block">This provider of chemical components can download and cache chemical component definition files from the RCSB PDB web site.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/DownloadChemCompProvider.html#DownloadChemCompProvider()">DownloadChemCompProvider()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/DownloadChemCompProvider.html" title="class in org.biojava.bio.structure.io.mmcif">DownloadChemCompProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html#downloadCIF(java.lang.String)">downloadCIF(String)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html" title="class in org.biojava.bio.structure.server">MMCIFFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/BerkeleyScopInstallation.html#downloadClaFile()">downloadClaFile()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/BerkeleyScopInstallation.html" title="class in org.biojava.bio.structure.scop">BerkeleyScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#downloadClaFile()">downloadClaFile()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/BerkeleyScopInstallation.html#downloadDesFile()">downloadDesFile()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/BerkeleyScopInstallation.html" title="class in org.biojava.bio.structure.scop">BerkeleyScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#downloadDesFile()">downloadDesFile()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#downloadDomainDescriptionFile()">downloadDomainDescriptionFile()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#downloadDomainListFile()">downloadDomainListFile()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#downloadDomallFile()">downloadDomallFile()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/AllChemCompProvider.html#downloadFile()">downloadFile()</a></span> - Static method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/AllChemCompProvider.html" title="class in org.biojava.bio.structure.io.mmcif">AllChemCompProvider</a></dt>
<dd>
<div class="block">Downloads the components.cif.gz file from the wwPDB site.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#downloadFileFromRemote(java.net.URL, java.io.File)">downloadFileFromRemote(URL, File)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#downloadFileFromRemote(java.net.URL, java.io.File)">downloadFileFromRemote(URL, File)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/util/FileDownloadUtils.html#downloadFileIfAvailable(java.net.URL, java.io.File)">downloadFileIfAvailable(URL, File)</a></span> - Static method in class org.biojava.bio.structure.io.util.<a href="./org/biojava/bio/structure/io/util/FileDownloadUtils.html" title="class in org.biojava.bio.structure.io.util">FileDownloadUtils</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/util/FileDownloadUtils.html#downloadGzipCompressedFile(java.net.URL, java.io.File)">downloadGzipCompressedFile(URL, File)</a></span> - Static method in class org.biojava.bio.structure.io.util.<a href="./org/biojava/bio/structure/io/util/FileDownloadUtils.html" title="class in org.biojava.bio.structure.io.util">FileDownloadUtils</a></dt>
<dd>
<div class="block">Download the content provided at URL url and stores the result to a local file</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/BerkeleyScopInstallation.html#downloadHieFile()">downloadHieFile()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/BerkeleyScopInstallation.html" title="class in org.biojava.bio.structure.scop">BerkeleyScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#downloadHieFile()">downloadHieFile()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/CountProgressListener.html#downloadingStructures(java.lang.String)">downloadingStructures(String)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/CountProgressListener.html" title="class in org.biojava.bio.structure.align.client">CountProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/events/AlignmentProgressListener.html#downloadingStructures(java.lang.String)">downloadingStructures(String)</a></span> - Method in interface org.biojava.bio.structure.align.events.<a href="./org/biojava/bio/structure/align/events/AlignmentProgressListener.html" title="interface in org.biojava.bio.structure.align.events">AlignmentProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#downloadNodeListFile()">downloadNodeListFile()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/MMCIFFileReader.html#downloadPDB(java.lang.String)">downloadPDB(String)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/MMCIFFileReader.html" title="class in org.biojava.bio.structure.io">MMCIFFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#duplicateCA2(org.biojava.bio.structure.Atom[])">duplicateCA2(Atom[])</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>
<div class="block">Utility method for working with circular permutations.</div>
</dd>
</dl>
<a name="_E_">
<!--   -->
</a>
<h2 class="title">E</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#eig()">eig()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Eigenvalue Decomposition</div>
</dd>
<dt><a href="./org/biojava/bio/structure/jama/EigenvalueDecomposition.html" title="class in org.biojava.bio.structure.jama"><span class="strong">EigenvalueDecomposition</span></a> - Class in <a href="./org/biojava/bio/structure/jama/package-summary.html">org.biojava.bio.structure.jama</a></dt>
<dd>
<div class="block">Eigenvalues and eigenvectors of a real matrix.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/EigenvalueDecomposition.html#EigenvalueDecomposition(org.biojava.bio.structure.jama.Matrix)">EigenvalueDecomposition(Matrix)</a></span> - Constructor for class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/EigenvalueDecomposition.html" title="class in org.biojava.bio.structure.jama">EigenvalueDecomposition</a></dt>
<dd>
<div class="block">Check for symmetry, then construct the eigenvalue decomposition
       provides a  way  to access D and V.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure"><span class="strong">Element</span></a> - Enum in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">Element is an enumeration of the elements of the periodic table.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/ElementType.html" title="enum in org.biojava.bio.structure"><span class="strong">ElementType</span></a> - Enum in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">ElementType is an enumeration of the types of elements found in the periodic table.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/SerializableCache.html#enableCache()">enableCache()</a></span> - Method in class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/SerializableCache.html" title="class in org.biojava.bio.structure.domain">SerializableCache</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html#ENDS">ENDS</a></span> - Static variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html" title="class in org.biojava.bio.structure.domain.pdp">PDPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html#ENDSEND">ENDSEND</a></span> - Static variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html" title="class in org.biojava.bio.structure.domain.pdp">PDPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#ensureClaInstalled()">ensureClaInstalled()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#ensureDesInstalled()">ensureDesInstalled()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#ensureDomainDescriptionInstalled()">ensureDomainDescriptionInstalled()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#ensureDomainListInstalled()">ensureDomainListInstalled()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#ensureDomallInstalled()">ensureDomallInstalled()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#ensureHieInstalled()">ensureHieInstalled()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#ensureNodeListInstalled()">ensureNodeListInstalled()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model"><span class="strong">Entity</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a></dt>
<dd>
<div class="block">A simple class to represent Entity records in mmCif files</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html#Entity()">Entity()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model">Entity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/chem/PolymerType.html#entity_poly_type">entity_poly_type</a></span> - Variable in enum org.biojava.bio.structure.io.mmcif.chem.<a href="./org/biojava/bio/structure/io/mmcif/chem/PolymerType.html" title="enum in org.biojava.bio.structure.io.mmcif.chem">PolymerType</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/EntityPolySeq.html" title="class in org.biojava.bio.structure.io.mmcif.model"><span class="strong">EntityPolySeq</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a></dt>
<dd>
<div class="block">Container for _entity_poly_seq records</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/EntityPolySeq.html#EntityPolySeq()">EntityPolySeq()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/EntityPolySeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">EntityPolySeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AlignmentTools.IdentityMap.html#entrySet()">entrySet()</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AlignmentTools.IdentityMap.html" title="class in org.biojava.bio.structure.align.util">AlignmentTools.IdentityMap</a></dt>
<dd>
<div class="block">Always returns the empty set</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/PdbPair.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/PdbPair.html" title="class in org.biojava.bio.structure.align.client">PdbPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/StructureName.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/StructureName.html" title="class in org.biojava.bio.structure.align.client">StructureName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">A week equality metric.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Author.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Author.html" title="class in org.biojava.bio.structure">Author</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemCompDescriptor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsEntity.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsEntity.html" title="class in org.biojava.bio.structure.io.sifts">SiftsEntity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsSegment.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsSegment.html" title="class in org.biojava.bio.structure.io.sifts">SiftsSegment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#equals(org.biojava.bio.structure.PDBHeader)">equals(PDBHeader)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>
<div class="block">Compare two PDBHeader objects</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ResidueNumber.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ResidueNumber.html" title="class in org.biojava.bio.structure">ResidueNumber</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDescription.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDescription.html" title="class in org.biojava.bio.structure.scop">ScopDescription</a></dt>
<dd>
<div class="block">Compares the fields sunID, category, classificationId, and name for equality</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Site.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Site.html" title="class in org.biojava.bio.structure">Site</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html" title="class in org.biojava.bio.structure.io.mmcif.model"><span class="strong">Exptl</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html#Exptl()">Exptl()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html" title="class in org.biojava.bio.structure.io.mmcif.model">Exptl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentJoiner.html#extendFragments(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], org.biojava.bio.structure.align.helper.JointFragments[], org.biojava.bio.structure.align.StrucAligParameters)">extendFragments(Atom[], Atom[], JointFragments[], StrucAligParameters)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentJoiner.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentJoiner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentJoiner.html#extendFragments(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], org.biojava.bio.structure.align.helper.JointFragments, org.biojava.bio.structure.align.StrucAligParameters)">extendFragments(Atom[], Atom[], JointFragments, StrucAligParameters)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentJoiner.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentJoiner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/AFPCalculator.html#extractAFPChains(org.biojava.bio.structure.align.fatcat.calc.FatCatParameters, org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">extractAFPChains(FatCatParameters, AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/AFPCalculator.html" title="class in org.biojava.bio.structure.align.fatcat.calc">AFPCalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#extractFragments(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">extractFragments(AFPChain, Atom[], Atom[])</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#extractFragments(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">extractFragments(AFPChain, Atom[], Atom[])</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_F_">
<!--   -->
</a>
<h2 class="title">F</h2>
<dl>
<dt><a href="./org/biojava/bio/structure/align/FarmJob.html" title="class in org.biojava.bio.structure.align"><span class="strong">FarmJob</span></a> - Class in <a href="./org/biojava/bio/structure/align/package-summary.html">org.biojava.bio.structure.align</a></dt>
<dd>
<div class="block">A job as it can be run on the farm.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/FarmJob.html#FarmJob()">FarmJob()</a></span> - Constructor for class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/FarmJob.html" title="class in org.biojava.bio.structure.align">FarmJob</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client"><span class="strong">FarmJobParameters</span></a> - Class in <a href="./org/biojava/bio/structure/align/client/package-summary.html">org.biojava.bio.structure.align.client</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#FarmJobParameters()">FarmJobParameters()</a></span> - Constructor for class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html" title="class in org.biojava.bio.structure.align.client"><span class="strong">FarmJobRunnable</span></a> - Class in <a href="./org/biojava/bio/structure/align/client/package-summary.html">org.biojava.bio.structure.align.client</a></dt>
<dd>
<div class="block">Contains the single thread for a job that can run multiple alignments.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html#FarmJobRunnable(org.biojava.bio.structure.align.client.FarmJobParameters)">FarmJobRunnable(FarmJobParameters)</a></span> - Constructor for class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html" title="class in org.biojava.bio.structure.align.client">FarmJobRunnable</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/FastaStructureParser.html" title="class in org.biojava.bio.structure.io"><span class="strong">FastaStructureParser</span></a> - Class in <a href="./org/biojava/bio/structure/io/package-summary.html">org.biojava.bio.structure.io</a></dt>
<dd>
<div class="block">Reads a protein sequence from a fasta file and attempts to match it to a
 3D structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FastaStructureParser.html#FastaStructureParser(java.io.InputStream, org.biojava3.core.sequence.io.template.FastaHeaderParserInterface, org.biojava3.core.sequence.io.template.SequenceCreatorInterface, org.biojava.bio.structure.align.util.AtomCache)">FastaStructureParser(InputStream, FastaHeaderParserInterface&lt;ProteinSequence, AminoAcidCompound&gt;, SequenceCreatorInterface&lt;AminoAcidCompound&gt;, AtomCache)</a></span> - Constructor for class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FastaStructureParser.html" title="class in org.biojava.bio.structure.io">FastaStructureParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FastaStructureParser.html#FastaStructureParser(java.io.File, org.biojava3.core.sequence.io.template.FastaHeaderParserInterface, org.biojava3.core.sequence.io.template.SequenceCreatorInterface, org.biojava.bio.structure.align.util.AtomCache)">FastaStructureParser(File, FastaHeaderParserInterface&lt;ProteinSequence, AminoAcidCompound&gt;, SequenceCreatorInterface&lt;AminoAcidCompound&gt;, AtomCache)</a></span> - Constructor for class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FastaStructureParser.html" title="class in org.biojava.bio.structure.io">FastaStructureParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FastaStructureParser.html#FastaStructureParser(org.biojava3.core.sequence.io.FastaReader, org.biojava.bio.structure.align.util.AtomCache)">FastaStructureParser(FastaReader&lt;ProteinSequence, AminoAcidCompound&gt;, AtomCache)</a></span> - Constructor for class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FastaStructureParser.html" title="class in org.biojava.bio.structure.io">FastaStructureParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/fatcat/FatCat.html" title="class in org.biojava.bio.structure.align.fatcat"><span class="strong">FatCat</span></a> - Class in <a href="./org/biojava/bio/structure/align/fatcat/package-summary.html">org.biojava.bio.structure.align.fatcat</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCat.html#FatCat()">FatCat()</a></span> - Constructor for class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCat.html" title="class in org.biojava.bio.structure.align.fatcat">FatCat</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatAligner.html" title="class in org.biojava.bio.structure.align.fatcat.calc"><span class="strong">FatCatAligner</span></a> - Class in <a href="./org/biojava/bio/structure/align/fatcat/calc/package-summary.html">org.biojava.bio.structure.align.fatcat.calc</a></dt>
<dd>
<div class="block">A class that does calculations on an AFPChain</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatAligner.html#FatCatAligner()">FatCatAligner()</a></span> - Constructor for class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatAligner.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatAligner</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/fatcat/FatCatFlexible.html" title="class in org.biojava.bio.structure.align.fatcat"><span class="strong">FatCatFlexible</span></a> - Class in <a href="./org/biojava/bio/structure/align/fatcat/package-summary.html">org.biojava.bio.structure.align.fatcat</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCatFlexible.html#FatCatFlexible()">FatCatFlexible()</a></span> - Constructor for class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCatFlexible.html" title="class in org.biojava.bio.structure.align.fatcat">FatCatFlexible</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc"><span class="strong">FatCatParameters</span></a> - Class in <a href="./org/biojava/bio/structure/align/fatcat/calc/package-summary.html">org.biojava.bio.structure.align.fatcat.calc</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#FatCatParameters()">FatCatParameters()</a></span> - Constructor for class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/fatcat/FatCatRigid.html" title="class in org.biojava.bio.structure.align.fatcat"><span class="strong">FatCatRigid</span></a> - Class in <a href="./org/biojava/bio/structure/align/fatcat/package-summary.html">org.biojava.bio.structure.align.fatcat</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCatRigid.html#FatCatRigid()">FatCatRigid()</a></span> - Constructor for class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCatRigid.html" title="class in org.biojava.bio.structure.align.fatcat">FatCatRigid</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/fatcat/FatCatUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.fatcat"><span class="strong">FatCatUserArgumentProcessor</span></a> - Class in <a href="./org/biojava/bio/structure/align/fatcat/package-summary.html">org.biojava.bio.structure.align.fatcat</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCatUserArgumentProcessor.html#FatCatUserArgumentProcessor()">FatCatUserArgumentProcessor()</a></span> - Constructor for class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCatUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.fatcat">FatCatUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/fatcat/calc/FCAlignHelper.html" title="class in org.biojava.bio.structure.align.fatcat.calc"><span class="strong">FCAlignHelper</span></a> - Class in <a href="./org/biojava/bio/structure/align/fatcat/calc/package-summary.html">org.biojava.bio.structure.align.fatcat.calc</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FCAlignHelper.html#FCAlignHelper(double[][], int, int, double, double)">FCAlignHelper(double[][], int, int, double, double)</a></span> - Constructor for class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FCAlignHelper.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FCAlignHelper</a></dt>
<dd>
<div class="block">do an alignment given the provided matrix sij0</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#FIELD_LINE">FIELD_LINE</a></span> - Static variable in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/quaternary/io/FileBasedPDBBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io"><span class="strong">FileBasedPDBBioUnitDataProvider</span></a> - Class in <a href="./org/biojava/bio/structure/quaternary/io/package-summary.html">org.biojava.bio.structure.quaternary.io</a></dt>
<dd>
<div class="block">A biounit provider that loads the biol assembly from thepublic PDB file, rather than re-creating it on the fly out of the default PDB file for the asym unit</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/FileBasedPDBBioUnitDataProvider.html#FileBasedPDBBioUnitDataProvider()">FileBasedPDBBioUnitDataProvider()</a></span> - Constructor for class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/FileBasedPDBBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">FileBasedPDBBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/BioUnitDataProviderFactory.html#fileBasedProviderClassName">fileBasedProviderClassName</a></span> - Static variable in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/BioUnitDataProviderFactory.html" title="class in org.biojava.bio.structure.quaternary.io">BioUnitDataProviderFactory</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/FileConvert.html" title="class in org.biojava.bio.structure.io"><span class="strong">FileConvert</span></a> - Class in <a href="./org/biojava/bio/structure/io/package-summary.html">org.biojava.bio.structure.io</a></dt>
<dd>
<div class="block">Methods to convert a structure object into different file formats.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileConvert.html#FileConvert(org.biojava.bio.structure.Structure)">FileConvert(Structure)</a></span> - Constructor for class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileConvert.html" title="class in org.biojava.bio.structure.io">FileConvert</a></dt>
<dd>
<div class="block">Constructs a FileConvert object.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/io/util/FileDownloadUtils.html" title="class in org.biojava.bio.structure.io.util"><span class="strong">FileDownloadUtils</span></a> - Class in <a href="./org/biojava/bio/structure/io/util/package-summary.html">org.biojava.bio.structure.io.util</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/util/FileDownloadUtils.html#FileDownloadUtils()">FileDownloadUtils()</a></span> - Constructor for class org.biojava.bio.structure.io.util.<a href="./org/biojava/bio/structure/io/util/FileDownloadUtils.html" title="class in org.biojava.bio.structure.io.util">FileDownloadUtils</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/FileParsingParameters.html" title="class in org.biojava.bio.structure.io"><span class="strong">FileParsingParameters</span></a> - Class in <a href="./org/biojava/bio/structure/io/package-summary.html">org.biojava.bio.structure.io</a></dt>
<dd>
<div class="block">A class that configures parameters that can be sent to the PDB file parsers
 
 
  <a href="./org/biojava/bio/structure/io/FileParsingParameters.html#setParseCAOnly(boolean)"><code>FileParsingParameters.setParseCAOnly(boolean)</code></a> - parse only the Atom records for C-alpha atoms
  <a href="./org/biojava/bio/structure/io/FileParsingParameters.html#setParseSecStruc(boolean)"><code>FileParsingParameters.setParseSecStruc(boolean)</code></a> - a flag if the secondary structure information from the PDB file (author's assignment) should be parsed.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileParsingParameters.html#FileParsingParameters()">FileParsingParameters()</a></span> - Constructor for class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileParsingParameters.html" title="class in org.biojava.bio.structure.io">FileParsingParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/SandboxStyleStructureProvider.html#fileSeparator">fileSeparator</a></span> - Static variable in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/SandboxStyleStructureProvider.html" title="class in org.biojava.bio.structure.io">SandboxStyleStructureProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#FILESPLIT">FILESPLIT</a></span> - Static variable in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#FILESPLIT">FILESPLIT</a></span> - Static variable in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDatabase.html#filterByCathCode(java.lang.String)">filterByCathCode(String)</a></span> - Method in interface org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDatabase.html" title="interface in org.biojava.bio.structure.cath">CathDatabase</a></dt>
<dd>
<div class="block">Return list of CATH descriptions whose CATH codes (e.g.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#filterByCathCode(java.lang.String)">filterByCathCode(String)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/CachedRemoteScopInstallation.html#filterByClassificationId(java.lang.String)">filterByClassificationId(String)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/CachedRemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">CachedRemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html#filterByClassificationId(java.lang.String)">filterByClassificationId(String)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">RemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDatabase.html#filterByClassificationId(java.lang.String)">filterByClassificationId(String)</a></span> - Method in interface org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDatabase.html" title="interface in org.biojava.bio.structure.scop">ScopDatabase</a></dt>
<dd>
<div class="block">Get all scop descriptions that start with a classifcation ID, e.g.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#filterByClassificationId(java.lang.String)">filterByClassificationId(String)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDatabase.html#filterByDescription(java.lang.String)">filterByDescription(String)</a></span> - Method in interface org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDatabase.html" title="interface in org.biojava.bio.structure.cath">CathDatabase</a></dt>
<dd>
<div class="block">Return list of CATH descriptions whose descriptions (name field) starts with the query.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#filterByDescription(java.lang.String)">filterByDescription(String)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/CachedRemoteScopInstallation.html#filterByDescription(java.lang.String)">filterByDescription(String)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/CachedRemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">CachedRemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html#filterByDescription(java.lang.String)">filterByDescription(String)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">RemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDatabase.html#filterByDescription(java.lang.String)">filterByDescription(String)</a></span> - Method in interface org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDatabase.html" title="interface in org.biojava.bio.structure.scop">ScopDatabase</a></dt>
<dd>
<div class="block">Get all scop descriptions that start with a certain name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#filterByDescription(java.lang.String)">filterByDescription(String)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/CachedRemoteScopInstallation.html#filterByDomainName(java.lang.String)">filterByDomainName(String)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/CachedRemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">CachedRemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html#filterByDomainName(java.lang.String)">filterByDomainName(String)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">RemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDatabase.html#filterByDomainName(java.lang.String)">filterByDomainName(String)</a></span> - Method in interface org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDatabase.html" title="interface in org.biojava.bio.structure.scop">ScopDatabase</a></dt>
<dd>
<div class="block">search through SCOP and filter based on domain name</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#filterByDomainName(java.lang.String)">filterByDomainName(String)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDatabase.html#filterByNodeName(java.lang.String)">filterByNodeName(String)</a></span> - Method in interface org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDatabase.html" title="interface in org.biojava.bio.structure.cath">CathDatabase</a></dt>
<dd>
<div class="block">Return list of CATH domains whose node name (e.g.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#filterByNodeName(java.lang.String)">filterByNodeName(String)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCPMain.html#filterDuplicateAFPs(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.align.ce.CECalculator, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">filterDuplicateAFPs(AFPChain, CECalculator, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCPMain.html" title="class in org.biojava.bio.structure.align.ce">CeCPMain</a></dt>
<dd>
<div class="block">Takes as input an AFPChain where ca2 has been artificially duplicated.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCPMain.html#filterDuplicateAFPs(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.align.ce.CECalculator, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], int)">filterDuplicateAFPs(AFPChain, CECalculator, Atom[], Atom[], int)</a></span> - Static method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCPMain.html" title="class in org.biojava.bio.structure.align.ce">CeCPMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBBioAssemblyParser.html#finalizeCurrentBioMolecule()">finalizeCurrentBioMolecule()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBBioAssemblyParser.html" title="class in org.biojava.bio.structure.io">PDBBioAssemblyParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#findBonds(org.biojava.bio.structure.Group, java.util.List)">findBonds(Group, List&lt;Group&gt;)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#findChain(java.lang.String)">findChain(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">request a particular chain from a structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#findChain(java.lang.String, int)">findChain(String, int)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">request a particular chain from a particular model</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#findChain(java.lang.String, int)">findChain(String, int)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#findChain(java.lang.String)">findChain(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#findGroup(java.lang.String, java.lang.String)">findGroup(String, String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">request a particular group from a structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#findGroup(java.lang.String, java.lang.String, int)">findGroup(String, String, int)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">request a particular group from a structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#findGroup(java.lang.String, java.lang.String, int)">findGroup(String, String, int)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#findGroup(java.lang.String, java.lang.String)">findGroup(String, String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#finish(org.biojava.bio.structure.align.StrucAligParameters, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">finish(StrucAligParameters, Atom[], Atom[])</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/CutValues.html#first_cut">first_cut</a></span> - Variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/CutValues.html" title="class in org.biojava.bio.structure.domain.pdp">CutValues</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/server/FlatFileInstallation.html" title="class in org.biojava.bio.structure.server"><span class="strong">FlatFileInstallation</span></a> - Class in <a href="./org/biojava/bio/structure/server/package-summary.html">org.biojava.bio.structure.server</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>&nbsp;</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/FlatFileInstallation.html#FlatFileInstallation(java.io.File)">FlatFileInstallation(File)</a></span> - Constructor for class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/FlatFileInstallation.html" title="class in org.biojava.bio.structure.server">FlatFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
<div class="block">create a new FlatFile Installation and point it to the directory that contains all PDB files</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/AFPChainXMLParser.html#flipAlignment(java.lang.String)">flipAlignment(String)</a></span> - Static method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/AFPChainXMLParser.html" title="class in org.biojava.bio.structure.align.xml">AFPChainXMLParser</a></dt>
<dd>
<div class="block">Takes an XML representation of the alignment and flips the positions of name1 and name2</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/AFPChainFlipper.html#flipChain(org.biojava.bio.structure.align.model.AFPChain)">flipChain(AFPChain)</a></span> - Static method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/AFPChainFlipper.html" title="class in org.biojava.bio.structure.align.xml">AFPChainFlipper</a></dt>
<dd>
<div class="block">Flip the position of name1 and name2 (as well as all underlying data) in an AFPChain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/RemoteDomainProvider.html#flushCache()">flushCache()</a></span> - Method in class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/RemoteDomainProvider.html" title="class in org.biojava.bio.structure.domain">RemoteDomainProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/SerializableCache.html#flushCache()">flushCache()</a></span> - Method in class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/SerializableCache.html" title="class in org.biojava.bio.structure.domain">SerializableCache</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/CachedRemoteScopInstallation.html#flushCache()">flushCache()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/CachedRemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">CachedRemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentJoiner.html#frag_pairwise_compat(org.biojava.bio.structure.align.pairwise.FragmentPair[], int, float, int)">frag_pairwise_compat(FragmentPair[], int, float, int)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentJoiner.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentJoiner</a></dt>
<dd>
<div class="block">Calculate the pairwise compatibility of fpairs.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/pairwise/FragmentJoiner.html" title="class in org.biojava.bio.structure.align.pairwise"><span class="strong">FragmentJoiner</span></a> - Class in <a href="./org/biojava/bio/structure/align/pairwise/package-summary.html">org.biojava.bio.structure.align.pairwise</a></dt>
<dd>
<div class="block">Joins the initial Fragments together to larger Fragments</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentJoiner.html#FragmentJoiner()">FragmentJoiner()</a></span> - Constructor for class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentJoiner.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentJoiner</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise"><span class="strong">FragmentPair</span></a> - Class in <a href="./org/biojava/bio/structure/align/pairwise/package-summary.html">org.biojava.bio.structure.align.pairwise</a></dt>
<dd>
<div class="block">a pair of fragments of two protein structures</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#FragmentPair(int, int, int)">FragmentPair(int, int, int)</a></span> - Constructor for class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathCategory.html#fromCathCode(java.lang.String)">fromCathCode(String)</a></span> - Static method in enum org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathCategory.html" title="enum in org.biojava.bio.structure.cath">CathCategory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/BridgeType.html#fromCharacter(java.lang.Character)">fromCharacter(Character)</a></span> - Static method in enum org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/BridgeType.html" title="enum in org.biojava.bio.structure.secstruc">BridgeType</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucType.html#fromCharacter(java.lang.Character)">fromCharacter(Character)</a></span> - Static method in enum org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucType.html" title="enum in org.biojava.bio.structure.secstruc">SecStrucType</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html#fromMultiXML(java.lang.String)">fromMultiXML(String)</a></span> - Static method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html" title="class in org.biojava.bio.structure.quaternary">ModelTransformationMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/UserConfiguration.html#fromStartupParams(org.biojava.bio.structure.align.ce.StartupParameters)">fromStartupParams(StartupParameters)</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/UserConfiguration.html" title="class in org.biojava.bio.structure.align.util">UserConfiguration</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathCategory.html#fromString(java.lang.String)">fromString(String)</a></span> - Static method in enum org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathCategory.html" title="enum in org.biojava.bio.structure.cath">CathCategory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBStatus.Status.html#fromString(java.lang.String)">fromString(String)</a></span> - Static method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBStatus.Status.html" title="enum in org.biojava.bio.structure">PDBStatus.Status</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ResidueNumber.html#fromString(java.lang.String)">fromString(String)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ResidueNumber.html" title="class in org.biojava.bio.structure">ResidueNumber</a></dt>
<dd>
<div class="block">Convert a string representation of a residue number to a residue number object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopCategory.html#fromString(java.lang.String)">fromString(String)</a></span> - Static method in enum org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopCategory.html" title="enum in org.biojava.bio.structure.scop">ScopCategory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/AFPChainXMLParser.html#fromXML(java.lang.String, java.lang.String, java.lang.String, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">fromXML(String, String, String, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/AFPChainXMLParser.html" title="class in org.biojava.bio.structure.align.xml">AFPChainXMLParser</a></dt>
<dd>
<div class="block">new utility method that checks that the order of the pair in the XML alignment is correct and flips the direction if needed</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/AFPChainXMLParser.html#fromXML(java.lang.String, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">fromXML(String, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/AFPChainXMLParser.html" title="class in org.biojava.bio.structure.align.xml">AFPChainXMLParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/HasResultXMLConverter.html#fromXML(java.lang.String)">fromXML(String)</a></span> - Method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/HasResultXMLConverter.html" title="class in org.biojava.bio.structure.align.xml">HasResultXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/PositionInQueueXMLConverter.html#fromXML(java.lang.String)">fromXML(String)</a></span> - Method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/PositionInQueueXMLConverter.html" title="class in org.biojava.bio.structure.align.xml">PositionInQueueXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/RepresentativeXMLConverter.html#fromXML(java.lang.String)">fromXML(String)</a></span> - Static method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/RepresentativeXMLConverter.html" title="class in org.biojava.bio.structure.align.xml">RepresentativeXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/AssignmentXMLSerializer.html#fromXML(java.lang.String)">fromXML(String)</a></span> - Static method in class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/AssignmentXMLSerializer.html" title="class in org.biojava.bio.structure.domain">AssignmentXMLSerializer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssemblyGenXMLContainer.html#fromXML(java.lang.String)">fromXML(String)</a></span> - Static method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssemblyGenXMLContainer.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructAssemblyGenXMLContainer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssemblyXMLContainer.html#fromXML(java.lang.String)">fromXML(String)</a></span> - Static method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssemblyXMLContainer.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructAssemblyXMLContainer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperListXMLContainer.html#fromXML(java.lang.String)">fromXML(String)</a></span> - Static method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperListXMLContainer.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructOperListXMLContainer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html#fromXML(java.lang.String)">fromXML(String)</a></span> - Static method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html" title="class in org.biojava.bio.structure.quaternary">ModelTransformationMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/ScopDescriptions.html#fromXML(java.lang.String)">fromXML(String)</a></span> - Static method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/ScopDescriptions.html" title="class in org.biojava.bio.structure.scop.server">ScopDescriptions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/ScopDomains.html#fromXML(java.lang.String)">fromXML(String)</a></span> - Static method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/ScopDomains.html" title="class in org.biojava.bio.structure.scop.server">ScopDomains</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/ScopNodes.html#fromXML(java.lang.String)">fromXML(String)</a></span> - Static method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/ScopNodes.html" title="class in org.biojava.bio.structure.scop.server">ScopNodes</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/TreeSetStringWrapper.html#fromXML(java.lang.String)">fromXML(String)</a></span> - Static method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/TreeSetStringWrapper.html" title="class in org.biojava.bio.structure.scop.server">TreeSetStringWrapper</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_G_">
<!--   -->
</a>
<h2 class="title">G</h2>
<dl>
<dt><a href="./org/biojava/bio/structure/align/helper/GapArray.html" title="class in org.biojava.bio.structure.align.helper"><span class="strong">GapArray</span></a> - Class in <a href="./org/biojava/bio/structure/align/helper/package-summary.html">org.biojava.bio.structure.align.helper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/GapArray.html#GapArray()">GapArray()</a></span> - Constructor for class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/GapArray.html" title="class in org.biojava.bio.structure.align.helper">GapArray</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#gapExtension">gapExtension</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#gapOpen">gapOpen</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AlignmentTools.IdentityMap.html#get(java.lang.Object)">get(Object)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AlignmentTools.IdentityMap.html" title="class in org.biojava.bio.structure.align.util">AlignmentTools.IdentityMap</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/SerializableCache.html#get(K)">get(K)</a></span> - Method in class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/SerializableCache.html" title="class in org.biojava.bio.structure.domain">SerializableCache</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#get(int, int)">get(int, int)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Get a single element.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#get1LetterCode(java.lang.String)">get1LetterCode(String)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>
<div class="block">convert a three letter code into single character.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/RemoteRawBioUnitDataProvider.html#GET_ASSEMBLY">GET_ASSEMBLY</a></span> - Static variable in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/RemoteRawBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">RemoteRawBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/RemoteRawBioUnitDataProvider.html#GET_ASSEMBLY_GENS">GET_ASSEMBLY_GENS</a></span> - Static variable in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/RemoteRawBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">RemoteRawBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/RemoteRawBioUnitDataProvider.html#GET_STRUCT_OPER">GET_STRUCT_OPER</a></span> - Static variable in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/RemoteRawBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">RemoteRawBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBCrystallographicInfo.html#getA()">getA()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBCrystallographicInfo.html" title="class in org.biojava.bio.structure">PDBCrystallographicInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html#getAbsorpt_coefficient_mu()">getAbsorpt_coefficient_mu()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html" title="class in org.biojava.bio.structure.io.mmcif.model">Exptl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html#getAbsorpt_correction_T_max()">getAbsorpt_correction_T_max()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html" title="class in org.biojava.bio.structure.io.mmcif.model">Exptl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html#getAbsorpt_correction_T_min()">getAbsorpt_correction_T_min()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html" title="class in org.biojava.bio.structure.io.mmcif.model">Exptl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html#getAbsorpt_correction_type()">getAbsorpt_correction_type()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html" title="class in org.biojava.bio.structure.io.mmcif.model">Exptl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html#getAbsorpt_process_details()">getAbsorpt_process_details()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html" title="class in org.biojava.bio.structure.io.mmcif.model">Exptl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucState.html#getAccept1()">getAccept1()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucState.html" title="class in org.biojava.bio.structure.secstruc">SecStrucState</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucState.html#getAccept2()">getAccept2()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucState.html" title="class in org.biojava.bio.structure.secstruc">SecStrucState</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileParsingParameters.html#getAcceptedAtomNames()">getAcceptedAtomNames()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileParsingParameters.html" title="class in org.biojava.bio.structure.io">FileParsingParameters</a></dt>
<dd>
<div class="block">by default the parser will read in all atoms (unless using the CAonly switch).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FastaStructureParser.html#getAccessions()">getAccessions()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FastaStructureParser.html" title="class in org.biojava.bio.structure.io">FastaStructureParser</a></dt>
<dd>
<div class="block">Gets the protein accessions mapped from the Fasta file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getAfpAftIndex()">getAfpAftIndex()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getAfpBefIndex()">getAfpBefIndex()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatAligner.html#getAfpChain()">getAfpChain()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatAligner.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/JFatCatClient.html#getAFPChainFromServer(java.lang.String, java.lang.String, java.lang.String, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">getAFPChainFromServer(String, String, String, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/JFatCatClient.html" title="class in org.biojava.bio.structure.align.client">JFatCatClient</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/JFatCatClient.html#getAFPChainFromServer(java.lang.String, java.lang.String, java.lang.String, java.lang.String, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], int)">getAFPChainFromServer(String, String, String, String, Atom[], Atom[], int)</a></span> - Static method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/JFatCatClient.html" title="class in org.biojava.bio.structure.align.client">JFatCatClient</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getAfpChainLen()">getAfpChainLen()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getAfpChainList()">getAfpChainList()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getAfpChainTwiBin()">getAfpChainTwiBin()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getAfpChainTwiList()">getAfpChainTwiList()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getAfpChainTwiNum()">getAfpChainTwiNum()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getAfpDisCut()">getAfpDisCut()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getAfpDisCut0()">getAfpDisCut0()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getAfpIndex()">getAfpIndex()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getAfpSet()">getAfpSet()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">Get the set of AFPs for this alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html#getAlgorithm()">getAlgorithm()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce">AbstractUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCPUserArgumentProcessor.html#getAlgorithm()">getAlgorithm()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCPUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce">CeCPUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeSideChainUserArgumentProcessor.html#getAlgorithm()">getAlgorithm()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeSideChainUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce">CeSideChainUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeUserArgumentProcessor.html#getAlgorithm()">getAlgorithm()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce">CeUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCatUserArgumentProcessor.html#getAlgorithm()">getAlgorithm()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCatUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.fatcat">FatCatUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWatermanUserArgumentProcessor.html#getAlgorithm()">getAlgorithm()</a></span> - Method in class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWatermanUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.seq">SmithWatermanUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructureAlignmentFactory.html#getAlgorithm(java.lang.String)">getAlgorithm(String)</a></span> - Static method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructureAlignmentFactory.html" title="class in org.biojava.bio.structure.align">StructureAlignmentFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/AbstractStructureAlignment.html#getAlgorithmName()">getAlgorithmName()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/AbstractStructureAlignment.html" title="class in org.biojava.bio.structure.align">AbstractStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/BioJavaStructureAlignment.html#getAlgorithmName()">getAlgorithmName()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/BioJavaStructureAlignment.html" title="class in org.biojava.bio.structure.align">BioJavaStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCPMain.html#getAlgorithmName()">getAlgorithmName()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCPMain.html" title="class in org.biojava.bio.structure.align.ce">CeCPMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeMain.html#getAlgorithmName()">getAlgorithmName()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeMain.html" title="class in org.biojava.bio.structure.align.ce">CeMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeSideChainMain.html#getAlgorithmName()">getAlgorithmName()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeSideChainMain.html" title="class in org.biojava.bio.structure.align.ce">CeSideChainMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/OptimalCECPMain.html#getAlgorithmName()">getAlgorithmName()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/OptimalCECPMain.html" title="class in org.biojava.bio.structure.align.ce">OptimalCECPMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCatFlexible.html#getAlgorithmName()">getAlgorithmName()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCatFlexible.html" title="class in org.biojava.bio.structure.align.fatcat">FatCatFlexible</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCatRigid.html#getAlgorithmName()">getAlgorithmName()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCatRigid.html" title="class in org.biojava.bio.structure.align.fatcat">FatCatRigid</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getAlgorithmName()">getAlgorithmName()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWaterman3Daligner.html#getAlgorithmName()">getAlgorithmName()</a></span> - Method in class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWaterman3Daligner.html" title="class in org.biojava.bio.structure.align.seq">SmithWaterman3Daligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructureAlignment.html#getAlgorithmName()">getAlgorithmName()</a></span> - Method in interface org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructureAlignment.html" title="interface in org.biojava.bio.structure.align">StructureAlignment</a></dt>
<dd>
<div class="block">Get the name of the Algorithm</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/Alignable.html#getAligMat()">getAligMat()</a></span> - Method in interface org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/Alignable.html" title="interface in org.biojava.bio.structure.align.pairwise">Alignable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html#getAligMat(int, int)">getAligMat(int, int)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">StrCompAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html#getAligMat()">getAligMat()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">StrCompAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html#getAlign(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">getAlign(AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html" title="class in org.biojava.bio.structure.align.util">AFPAlignmentDisplay</a></dt>
<dd>
<div class="block">Extract the alignment output</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html#getAlign(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], boolean)">getAlign(AFPChain, Atom[], Atom[], boolean)</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html" title="class in org.biojava.bio.structure.align.util">AFPAlignmentDisplay</a></dt>
<dd>
<div class="block">Sets the following properties:
 
 The alignment strings <a href="./org/biojava/bio/structure/align/model/AFPChain.html#setAlnseq1(char[])"><code>alnseq1</code></a>,
  <a href="./org/biojava/bio/structure/align/model/AFPChain.html#setAlnseq2(char[])"><code>alnseq2</code></a>,
  and <a href="./org/biojava/bio/structure/align/model/AFPChain.html#setAlnsymb(char[])"><code>alnsymb</code></a>
 <a href="./org/biojava/bio/structure/align/model/AFPChain.html#setAlnbeg1(int)"><code>alnbeg1</code></a> and 2
 <a href="./org/biojava/bio/structure/align/model/AFPChain.html#setAlnLength(int)"><code>alnLength</code></a> and 
  <a href="./org/biojava/bio/structure/align/model/AFPChain.html#setGapLen(int)"><code>gapLen</code></a>
 </div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#getAlign_id()">getAlign_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#getAlign_se1()">getAlign_se1()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#getAlign_se1()">getAlign_se1()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#getAlign_se2()">getAlign_se2()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#getAlign_se2()">getAlign_se2()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html#getAlignedAtoms1(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[])">getAlignedAtoms1(AFPChain, Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html" title="class in org.biojava.bio.structure.align.util">AFPAlignmentDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html#getAlignedAtoms2(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[])">getAlignedAtoms2(AFPChain, Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html" title="class in org.biojava.bio.structure.align.util">AFPAlignmentDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/GuiWrapper.html#getAlignedStructure(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">getAlignedStructure(Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/GuiWrapper.html" title="class in org.biojava.bio.structure.align.ce">GuiWrapper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#getAlignedStructure(org.biojava.bio.structure.Structure, org.biojava.bio.structure.Structure)">getAlignedStructure(Structure, Structure)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>
<div class="block">create an artifical Structure object that contains the two
 structures superimposed onto each other.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructurePairAligner.html#getAlignmentAtoms(org.biojava.bio.structure.Structure)">getAlignmentAtoms(Structure)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructurePairAligner.html" title="class in org.biojava.bio.structure.align">StructurePairAligner</a></dt>
<dd>
<div class="block">Returns the atoms that are being used for the alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html#getAlignmentPairsFromServer()">getAlignmentPairsFromServer()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html" title="class in org.biojava.bio.structure.align.client">FarmJobRunnable</a></dt>
<dd>
<div class="block">talk to centralized server and fetch all alignments to run.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html#getAlignments()">getAlignments()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentResult</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructurePairAligner.html#getAlignments()">getAlignments()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructurePairAligner.html" title="class in org.biojava.bio.structure.align">StructurePairAligner</a></dt>
<dd>
<div class="block">return the alternative alignments that can be found for the two structures</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/SeqRes2AtomAligner.html#getAlignmentString()">getAlignmentString()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/SeqRes2AtomAligner.html" title="class in org.biojava.bio.structure.io">SeqRes2AtomAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#getAlignPairs()">getAlignPairs()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FCAlignHelper.html#getAlignPos(int[][])">getAlignPos(int[][])</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FCAlignHelper.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FCAlignHelper</a></dt>
<dd>
<div class="block">record the aligned pairs in alignList[][0], alignList[][1];
 return the number of aligned pairs</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getAlignScore()">getAlignScore()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getAlignScoreUpdate()">getAlignScoreUpdate()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/FlatFileInstallation.html#getAll()">getAll()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/FlatFileInstallation.html" title="class in org.biojava.bio.structure.server">FlatFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html#getAll()">getAll()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html" title="class in org.biojava.bio.structure.server">MMCIFFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/PDBInstallation.html#getAll()">getAll()</a></span> - Method in interface org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/PDBInstallation.html" title="interface in org.biojava.bio.structure.server">PDBInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
<div class="block">get all PDBHeaders that pass the added Filters, if no filters have been added returns all available PDBs</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructureAlignmentFactory.html#getAllAlgorithmNames()">getAllAlgorithmNames()</a></span> - Static method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructureAlignmentFactory.html" title="class in org.biojava.bio.structure.align">StructureAlignmentFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructureAlignmentFactory.html#getAllAlgorithms()">getAllAlgorithms()</a></span> - Static method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructureAlignmentFactory.html" title="class in org.biojava.bio.structure.align">StructureAlignmentFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#getAllAtomArray(org.biojava.bio.structure.Structure)">getAllAtomArray(Structure)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>
<div class="block">Convert all atoms of the structure (first model) into an Atom array</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/PrepareMMcifIndexFile.html#getAllMMcif(java.io.File)">getAllMMcif(File)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/PrepareMMcifIndexFile.html" title="class in org.biojava.bio.structure.server">PrepareMMcifIndexFile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/PrepareIndexFile.html#getAllPDB(java.io.File)">getAllPDB(File)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/PrepareIndexFile.html" title="class in org.biojava.bio.structure.server">PrepareIndexFile</a></dt>
<dd>
<div class="block">get all PDBfiles from a directory</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/SandboxStyleStructureProvider.html#getAllPDBIDs()">getAllPDBIDs()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/SandboxStyleStructureProvider.html" title="class in org.biojava.bio.structure.io">SandboxStyleStructureProvider</a></dt>
<dd>
<div class="block">Returns a list of all PDB IDs that are available in this installation</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getAlnbeg1()">getAlnbeg1()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getAlnbeg2()">getAlnbeg2()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getAlnLength()">getAlnLength()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getAlnseq1()">getAlnseq1()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getAlnseq2()">getAlnseq2()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getAlnsymb()">getAlnsymb()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBCrystallographicInfo.html#getAlpha()">getAlpha()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBCrystallographicInfo.html" title="class in org.biojava.bio.structure">PDBCrystallographicInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#getAltAligNumber()">getAltAligNumber()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>
<div class="block">returns the sequential number of this alternative alignment</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#getAltLoc()">getAltLoc()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">get set alternate Location.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#getAltLoc()">getAltLoc()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>
<div class="block">get alternate Location.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#getAltLocs()">getAltLocs()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">Get the list of alternate locations.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#getAltLocs()">getAltLocs()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StandardAminoAcid.html#getAminoAcid(java.lang.String)">getAminoAcid(String)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StandardAminoAcid.html" title="class in org.biojava.bio.structure">StandardAminoAcid</a></dt>
<dd>
<div class="block">get a standard amino acid.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/chem/ChemCompTools.html#getAminoOneLetter(java.lang.String)">getAminoOneLetter(String)</a></span> - Static method in class org.biojava.bio.structure.io.mmcif.chem.<a href="./org/biojava/bio/structure/io/mmcif/chem/ChemCompTools.html" title="class in org.biojava.bio.structure.io.mmcif.chem">ChemCompTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/chem/ChemCompTools.html#getAminoThreeLetter(java.lang.Character)">getAminoThreeLetter(Character)</a></span> - Static method in class org.biojava.bio.structure.io.mmcif.chem.<a href="./org/biojava/bio/structure/io/mmcif/chem/ChemCompTools.html" title="class in org.biojava.bio.structure.io.mmcif.chem">ChemCompTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcid.html#getAminoType()">getAminoType()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcid.html" title="interface in org.biojava.bio.structure">AminoAcid</a></dt>
<dd>
<div class="block">returns the name of the AA, in single letter code.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcidImpl.html#getAminoType()">getAminoType()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcidImpl.html" title="class in org.biojava.bio.structure">AminoAcidImpl</a></dt>
<dd>
<div class="block">returns the name of the AA, in single letter code.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#getAngleDiff()">getAngleDiff()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getAniso_B()">getAniso_B()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#getArchitectureId()">getArchitectureId()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#getArray()">getArray()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Access the internal two-dimensional array.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#getArrayCopy()">getArrayCopy()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Copy the internal two-dimensional array.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssemblyGen.html#getAssembly_id()">getAssembly_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssemblyGen.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructAssemblyGen</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/AssignmentXMLSerializer.html#getAssignments()">getAssignments()</a></span> - Method in class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/AssignmentXMLSerializer.html" title="class in org.biojava.bio.structure.domain">AssignmentXMLSerializer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#getAsym_id()">getAsym_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#getAsym_id()">getAsym_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssemblyGen.html#getAsym_id_list()">getAsym_id_list()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssemblyGen.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructAssemblyGen</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/BioUnitDataProvider.html#getAsymUnit(java.lang.String)">getAsymUnit(String)</a></span> - Method in interface org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/BioUnitDataProvider.html" title="interface in org.biojava.bio.structure.quaternary.io">BioUnitDataProvider</a></dt>
<dd>
<div class="block">load the asym unit, but set the info how to re-create the bio unit in the PdbHeader object</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/FileBasedPDBBioUnitDataProvider.html#getAsymUnit(java.lang.String)">getAsymUnit(String)</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/FileBasedPDBBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">FileBasedPDBBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/MmCifBiolAssemblyProvider.html#getAsymUnit(java.lang.String)">getAsymUnit(String)</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/MmCifBiolAssemblyProvider.html" title="class in org.biojava.bio.structure.quaternary.io">MmCifBiolAssemblyProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/MmCifPDBBiolAssemblyProvider.html#getAsymUnit()">getAsymUnit()</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/MmCifPDBBiolAssemblyProvider.html" title="class in org.biojava.bio.structure.quaternary.io">MmCifPDBBiolAssemblyProvider</a></dt>
<dd>
<div class="block">get the asym unit for this PDB ID</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/PDBBioUnitDataProvider.html#getAsymUnit(java.lang.String)">getAsymUnit(String)</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/PDBBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">PDBBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/RemoteBioUnitDataProvider.html#getAsymUnit(java.lang.String)">getAsymUnit(String)</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/RemoteBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">RemoteBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getAtcc()">getAtcc()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#getAtom(java.lang.String)">getAtom(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">Get an atom.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#getAtom(int)">getAtom(int)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">Get at atom by position.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#getAtom(java.lang.String)">getAtom(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">get an atom throws StructureException if atom not found.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#getAtom(int)">getAtom(int)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">return an atom by its position in the internal List.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Bond.html#getAtomA()">getAtomA()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Bond.html" title="class in org.biojava.bio.structure">Bond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/GuiWrapper.html#getAtomArray(org.biojava.bio.structure.Atom[], java.util.List, java.util.List)">getAtomArray(Atom[], List&lt;Group&gt;, List&lt;Group&gt;)</a></span> - Static method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/GuiWrapper.html" title="class in org.biojava.bio.structure.align.ce">GuiWrapper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#getAtomArray(org.biojava.bio.structure.Structure, java.lang.String[])">getAtomArray(Structure, String[])</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>
<div class="block">Returns an array of the requested Atoms from the Structure object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#getAtomArray(org.biojava.bio.structure.Chain, java.lang.String[])">getAtomArray(Chain, String[])</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>
<div class="block">Returns an array of the requested Atoms from the Structure object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#getAtomArrayAllModels(org.biojava.bio.structure.Structure, java.lang.String[])">getAtomArrayAllModels(Structure, String[])</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>
<div class="block">Returns an array of the requested Atoms from the Structure object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Bond.html#getAtomB()">getAtomB()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Bond.html" title="class in org.biojava.bio.structure">Bond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#getAtomByPDBname(java.lang.String)">getAtomByPDBname(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">Get an atom by the full PDB name e.g.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#getAtomByPDBname(java.lang.String)">getAtomByPDBname(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">Get an atom by the full PDB name e.g.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#getAtomCAArray(org.biojava.bio.structure.Chain)">getAtomCAArray(Chain)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>
<div class="block">Returns an Atom array of the CA atoms.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#getAtomCAArray(org.biojava.bio.structure.Structure)">getAtomCAArray(Structure)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>
<div class="block">Returns an Atom array of the CA atoms.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/BioUnitDataProvider.html#getAtomCache()">getAtomCache()</a></span> - Method in interface org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/BioUnitDataProvider.html" title="interface in org.biojava.bio.structure.quaternary.io">BioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/FileBasedPDBBioUnitDataProvider.html#getAtomCache()">getAtomCache()</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/FileBasedPDBBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">FileBasedPDBBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/MmCifBiolAssemblyProvider.html#getAtomCache()">getAtomCache()</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/MmCifBiolAssemblyProvider.html" title="class in org.biojava.bio.structure.quaternary.io">MmCifBiolAssemblyProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/MmCifPDBBiolAssemblyProvider.html#getAtomCache()">getAtomCache()</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/MmCifPDBBiolAssemblyProvider.html" title="class in org.biojava.bio.structure.quaternary.io">MmCifPDBBiolAssemblyProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/PDBBioUnitDataProvider.html#getAtomCache()">getAtomCache()</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/PDBBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">PDBBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/RemoteBioUnitDataProvider.html#getAtomCache()">getAtomCache()</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/RemoteBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">RemoteBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileParsingParameters.html#getAtomCaThreshold()">getAtomCaThreshold()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileParsingParameters.html" title="class in org.biojava.bio.structure.io">FileParsingParameters</a></dt>
<dd>
<div class="block">the maximum number of atoms that will be parsed before the parser switches to a CA-only
 representation of the PDB file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/BioAssemblyTools.html#getAtomCoordinateBounds(org.biojava.bio.structure.Structure)">getAtomCoordinateBounds(Structure)</a></span> - Static method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/BioAssemblyTools.html" title="class in org.biojava.bio.structure.quaternary">BioAssemblyTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getAtomGroup(int)">getAtomGroup(int)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">return the Group at position X.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getAtomGroup(int)">getAtomGroup(int)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">return the group at position .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getAtomGroups()">getAtomGroups()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Return all groups that have been specified in the ATOM section of this chain .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getAtomGroups(java.lang.String)">getAtomGroups(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Return a List of all groups of a special type (e.g.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getAtomGroups(java.lang.String)">getAtomGroups(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Get the Groups of a certain type, that are listed int the ATOM records of the PDB file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getAtomGroups()">getAtomGroups()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Return all groups that have been specified in the ATOM section of this chain .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#getAtomicMass()">getAtomicMass()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns the atomic mass for this Element.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#getAtomicNumber()">getAtomicNumber()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns the atomic number of this Element.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getAtomLength()">getAtomLength()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Return the number of Groups in the ATOM records of the chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getAtomLength()">getAtomLength()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Return the number of Groups in the ATOM records of the chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getAtomLigands()">getAtomLigands()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Get all groups that are not polymer groups and that are not solvent groups.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getAtomLigands()">getAtomLigands()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Get all groups that are not polymer groups and that are not solvent groups.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#getAtoms(java.lang.String)">getAtoms(String)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>
<div class="block">Returns the CA atoms for the provided name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#getAtoms(java.lang.String, boolean)">getAtoms(String, boolean)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>does the same as <a href="./org/biojava/bio/structure/align/util/AtomCache.html#getAtoms(java.lang.String)"><code>AtomCache.getAtoms(String)</code></a> ;</i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#getAtoms()">getAtoms()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">Get list of atoms.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#getAtoms()">getAtoms()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">get all atoms of this group .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getAtomSequence()">getAtomSequence()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Return the sequence of amino acids as it has been provided in the ATOM records.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getAtomSequence()">getAtomSequence()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Return the sequence of amino acids as it has been provided in the ATOM records.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getAuth_asym_id()">getAuth_asym_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getAuth_atom_id()">getAuth_atom_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getAuth_comp_id()">getAuth_comp_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#getAuth_mon_id()">getAuth_mon_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#getAuth_mon_id()">getAuth_mon_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getAuth_seq_id()">getAuth_seq_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#getAuth_seq_num()">getAuth_seq_num()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#getAuth_seq_num()">getAuth_seq_num()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#getAuthorList()">getAuthorList()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>
<div class="block">Get the list of Authors of the JournalArticle</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#getAuthors()">getAuthors()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>
<div class="block">Returns the names of the authors as listed in the AUTHORS section of a PDB file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/UserConfiguration.html#getAutoFetch()">getAutoFetch()</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/UserConfiguration.html" title="class in org.biojava.bio.structure.align.util">UserConfiguration</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBCrystallographicInfo.html#getB()">getB()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBCrystallographicInfo.html" title="class in org.biojava.bio.structure">PDBCrystallographicInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getB_iso_max()">getB_iso_max()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getB_iso_mean()">getB_iso_mean()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getB_iso_min()">getB_iso_min()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getB_iso_or_equiv()">getB_iso_or_equiv()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getB_iso_or_equiv_esd()">getB_iso_or_equiv_esd()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#getBackboneAtomArray(org.biojava.bio.structure.Structure)">getBackboneAtomArray(Structure)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>
<div class="block">Returns an Atom array of the MainChain atoms.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getBadRmsd()">getBadRmsd()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBCrystallographicInfo.html#getBeta()">getBeta()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBCrystallographicInfo.html" title="class in org.biojava.bio.structure">PDBCrystallographicInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/OperatorResolver.html#getBinaryOperators()">getBinaryOperators()</a></span> - Method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/OperatorResolver.html" title="class in org.biojava.bio.structure.quaternary">OperatorResolver</a></dt>
<dd>
<div class="block">Returns a list of operators for this assembly.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html#getBiol_id()">getBiol_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRef</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/MmCifBiolAssemblyProvider.html#getBiolAssembly(java.lang.String, int)">getBiolAssembly(String, int)</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/MmCifBiolAssemblyProvider.html" title="class in org.biojava.bio.structure.quaternary.io">MmCifBiolAssemblyProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#getBiologicalAssembly(java.lang.String, int, boolean)">getBiologicalAssembly(String, int, boolean)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>
<div class="block">Loads the biological assembly for a given PDB ID and bioAssemblyId.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/structure/StructureIO.html#getBiologicalAssembly(java.lang.String)">getBiologicalAssembly(String)</a></span> - Static method in class org.biojava3.structure.<a href="./org/biojava3/structure/StructureIO.html" title="class in org.biojava3.structure">StructureIO</a></dt>
<dd>
<div class="block">Returns the first biologicalAssembly that is available for a protein structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/structure/StructureIO.html#getBiologicalAssembly(java.lang.String, int)">getBiologicalAssembly(String, int)</a></span> - Static method in class org.biojava3.structure.<a href="./org/biojava3/structure/StructureIO.html" title="class in org.biojava3.structure">StructureIO</a></dt>
<dd>
<div class="block">By default the getStructure method loads asym units.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/BioAssemblyTools.html#getBiologicalMoleculeBounds(org.biojava.bio.structure.Structure, java.util.List)">getBiologicalMoleculeBounds(Structure, List&lt;ModelTransformationMatrix&gt;)</a></span> - Static method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/BioAssemblyTools.html" title="class in org.biojava.bio.structure.quaternary">BioAssemblyTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/BioAssemblyTools.html#getBiologicalMoleculeCentroid(org.biojava.bio.structure.Structure, java.util.List)">getBiologicalMoleculeCentroid(Structure, List&lt;ModelTransformationMatrix&gt;)</a></span> - Static method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/BioAssemblyTools.html" title="class in org.biojava.bio.structure.quaternary">BioAssemblyTools</a></dt>
<dd>
<div class="block">Returns the centroid of the biological molecule.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/BioAssemblyTools.html#getBiologicalMoleculeMaximumExtend(org.biojava.bio.structure.Structure, java.util.List)">getBiologicalMoleculeMaximumExtend(Structure, List&lt;ModelTransformationMatrix&gt;)</a></span> - Static method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/BioAssemblyTools.html" title="class in org.biojava.bio.structure.quaternary">BioAssemblyTools</a></dt>
<dd>
<div class="block">Returns the maximum extend of the biological molecule in the x, y, or z direction.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#getBiologicalUnit(java.lang.String)">getBiologicalUnit(String)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>
<div class="block">Loads the default biological unit (*.pdb1.gz) file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getBiologicalUnit()">getBiologicalUnit()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/LocalCacheStructureProvider.html#getBiologicalUnit(java.lang.String)">getBiologicalUnit(String)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/LocalCacheStructureProvider.html" title="class in org.biojava.bio.structure.io">LocalCacheStructureProvider</a></dt>
<dd>
<div class="block">Returns the first available biological unit for the PDB ID.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/SandboxStyleStructureProvider.html#getBiologicalUnit(java.lang.String)">getBiologicalUnit(String)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/SandboxStyleStructureProvider.html" title="class in org.biojava.bio.structure.io">SandboxStyleStructureProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/StructureProvider.html#getBiologicalUnit(java.lang.String)">getBiologicalUnit(String)</a></span> - Method in interface org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/StructureProvider.html" title="interface in org.biojava.bio.structure.io">StructureProvider</a></dt>
<dd>
<div class="block">get the biological unit for a file</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/FileBasedPDBBioUnitDataProvider.html#getBioUnit(java.lang.String, int)">getBioUnit(String, int)</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/FileBasedPDBBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">FileBasedPDBBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/BioUnitDataProviderFactory.html#getBioUnitDataProvider()">getBioUnitDataProvider()</a></span> - Static method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/BioUnitDataProviderFactory.html" title="class in org.biojava.bio.structure.quaternary.io">BioUnitDataProviderFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#getBioUnitTranformationMap()">getBioUnitTranformationMap()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/BiologicalAssemblyBuilder.html#getBioUnitTransformationList(org.biojava.bio.structure.io.mmcif.model.PdbxStructAssembly, java.util.List, java.util.List)">getBioUnitTransformationList(PdbxStructAssembly, List&lt;PdbxStructAssemblyGen&gt;, List&lt;PdbxStructOperList&gt;)</a></span> - Method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/BiologicalAssemblyBuilder.html" title="class in org.biojava.bio.structure.quaternary">BiologicalAssemblyBuilder</a></dt>
<dd>
<div class="block">Returns a list of transformation matrices for the generation of a macromolecular
 assembly for the specified assembly Id.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/BioUnitDataProvider.html#getBioUnitTransformationList(java.lang.String, int)">getBioUnitTransformationList(String, int)</a></span> - Method in interface org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/BioUnitDataProvider.html" title="interface in org.biojava.bio.structure.quaternary.io">BioUnitDataProvider</a></dt>
<dd>
<div class="block">get the data for a particular assembly, counting starts at 1...</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/FileBasedPDBBioUnitDataProvider.html#getBioUnitTransformationList(java.lang.String, int)">getBioUnitTransformationList(String, int)</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/FileBasedPDBBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">FileBasedPDBBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/MmCifBiolAssemblyProvider.html#getBioUnitTransformationList(java.lang.String, int)">getBioUnitTransformationList(String, int)</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/MmCifBiolAssemblyProvider.html" title="class in org.biojava.bio.structure.quaternary.io">MmCifBiolAssemblyProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/PDBBioUnitDataProvider.html#getBioUnitTransformationList(java.lang.String, int)">getBioUnitTransformationList(String, int)</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/PDBBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">PDBBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/RemoteBioUnitDataProvider.html#getBioUnitTransformationList(java.lang.String, int)">getBioUnitTransformationList(String, int)</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/RemoteBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">RemoteBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getBJSequence()">getBJSequence()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Convert the SEQRES groups of a Chain to a Biojava Sequence object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getBJSequence()">getBJSequence()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Convert the SEQRES groups of a Chain to a Biojava Sequence object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getBlock2Afp()">getBlock2Afp()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getBlockGap()">getBlockGap()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html#getBlockNrForAlignPos(org.biojava.bio.structure.align.model.AFPChain, int)">getBlockNrForAlignPos(AFPChain, int)</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html" title="class in org.biojava.bio.structure.align.util">AFPAlignmentDisplay</a></dt>
<dd>
<div class="block">get the block number for an aligned position</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getBlockNum()">getBlockNum()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">The number of blocks in the alignment</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getBlockNumClu()">getBlockNumClu()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getBlockNumIni()">getBlockNumIni()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getBlockNumSpt()">getBlockNumSpt()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getBlockResList()">getBlockResList()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">tracks the residues of the initial blocks (before optimization)</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getBlockResSize()">getBlockResSize()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getBlockRmsd()">getBlockRmsd()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getBlockRotationMatrix()">getBlockRotationMatrix()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getBlockScore()">getBlockScore()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getBlockShiftVector()">getBlockShiftVector()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getBlockSize()">getBlockSize()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/Ladder.html#getBtype()">getBtype()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/Ladder.html" title="class in org.biojava.bio.structure.secstruc">Ladder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDatabase.html#getByCategory(org.biojava.bio.structure.cath.CathCategory)">getByCategory(CathCategory)</a></span> - Method in interface org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDatabase.html" title="interface in org.biojava.bio.structure.cath">CathDatabase</a></dt>
<dd>
<div class="block">Return list of CATH descriptions for node representatives at a CATH category (e.g.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#getByCategory(org.biojava.bio.structure.cath.CathCategory)">getByCategory(CathCategory)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/CachedRemoteScopInstallation.html#getByCategory(org.biojava.bio.structure.scop.ScopCategory)">getByCategory(ScopCategory)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/CachedRemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">CachedRemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html#getByCategory(org.biojava.bio.structure.scop.ScopCategory)">getByCategory(ScopCategory)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">RemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDatabase.html#getByCategory(org.biojava.bio.structure.scop.ScopCategory)">getByCategory(ScopCategory)</a></span> - Method in interface org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDatabase.html" title="interface in org.biojava.bio.structure.scop">ScopDatabase</a></dt>
<dd>
<div class="block">Get all records of a particular classification.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#getByCategory(org.biojava.bio.structure.scop.ScopCategory)">getByCategory(ScopCategory)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcid.html#getC()">getC()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcid.html" title="interface in org.biojava.bio.structure">AminoAcid</a></dt>
<dd>
<div class="block">get C atom.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcidImpl.html#getC()">getC()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcidImpl.html" title="class in org.biojava.bio.structure">AminoAcidImpl</a></dt>
<dd>
<div class="block">get C atom.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBCrystallographicInfo.html#getC()">getC()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBCrystallographicInfo.html" title="class in org.biojava.bio.structure">PDBCrystallographicInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html#getC()">getC()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html" title="class in org.biojava.bio.structure.secstruc">SecStrucGroup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcid.html#getCA()">getCA()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcid.html" title="interface in org.biojava.bio.structure">AminoAcid</a></dt>
<dd>
<div class="block">get CA atom.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcidImpl.html#getCA()">getCA()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcidImpl.html" title="class in org.biojava.bio.structure">AminoAcidImpl</a></dt>
<dd>
<div class="block">get CA atom.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html#getCA()">getCA()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html" title="class in org.biojava.bio.structure.secstruc">SecStrucGroup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html#getCa1()">getCa1()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html" title="class in org.biojava.bio.structure.align">CallableStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getCa1Length()">getCa1Length()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getCa2Length()">getCa2Length()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html#getCache()">getCache()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html" title="class in org.biojava.bio.structure.align">CallableStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/CacheFactory.html#getCache()">getCache()</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/CacheFactory.html" title="class in org.biojava.bio.structure.align.util">CacheFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#getCacheLocation()">getCacheLocation()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#getCacheLocation()">getCacheLocation()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/SimpleStructureServer.html#getCacheSize()">getCacheSize()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/SimpleStructureServer.html" title="class in org.biojava.bio.structure.server">SimpleStructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureServer.html#getCacheSize()">getCacheSize()</a></span> - Method in interface org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureServer.html" title="interface in org.biojava.bio.structure.server">StructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getCalculationTime()">getCalculationTime()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html#getCalculationTime()">getCalculationTime()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentResult</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/CAConverter.html#getCAOnly(java.util.List)">getCAOnly(List&lt;Chain&gt;)</a></span> - Static method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/CAConverter.html" title="class in org.biojava.bio.structure.io">CAConverter</a></dt>
<dd>
<div class="block">Convert a List of chain objects to another List of chains, containing C-alpha atoms only.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/CAConverter.html#getCAOnly(org.biojava.bio.structure.Chain)">getCAOnly(Chain)</a></span> - Static method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/CAConverter.html" title="class in org.biojava.bio.structure.io">CAConverter</a></dt>
<dd>
<div class="block">Convert a Chain to a new Chain containing C-alpha atoms only.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getCartn_x()">getCartn_x()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getCartn_x_esd()">getCartn_x_esd()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getCartn_y()">getCartn_y()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getCartn_y_esd()">getCartn_y_esd()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getCartn_z()">getCartn_z()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getCartn_z_esd()">getCartn_z_esd()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoSCOP.html#getCategories()">getCategories()</a></span> - Method in class demo.<a href="./demo/DemoSCOP.html" title="class in demo">DemoSCOP</a></dt>
<dd>
<div class="block">Get various categories</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathNode.html#getCategory()">getCategory()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathNode.html" title="class in org.biojava.bio.structure.cath">CathNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDescription.html#getCategory()">getCategory()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDescription.html" title="class in org.biojava.bio.structure.scop">ScopDescription</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#getCATH()">getCATH()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#getCathDownloadUrl()">getCathDownloadUrl()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDatabase.html#getCathNode(java.lang.String)">getCathNode(String)</a></span> - Method in interface org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDatabase.html" title="interface in org.biojava.bio.structure.cath">CathDatabase</a></dt>
<dd>
<div class="block">Return the CathNode for a node ID.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#getCathNode(java.lang.String)">getCathNode(String)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDatabase.html#getCathVersion()">getCathVersion()</a></span> - Method in interface org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDatabase.html" title="interface in org.biojava.bio.structure.cath">CathDatabase</a></dt>
<dd>
<div class="block">Return the CATH release version.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#getCathVersion()">getCathVersion()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcid.html#getCB()">getCB()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcid.html" title="interface in org.biojava.bio.structure">AminoAcid</a></dt>
<dd>
<div class="block">get CB atom.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcidImpl.html#getCB()">getCB()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcidImpl.html" title="class in org.biojava.bio.structure">AminoAcidImpl</a></dt>
<dd>
<div class="block">get CB atom.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeMain.html#getCECalculator()">getCECalculator()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeMain.html" title="class in org.biojava.bio.structure.align.ce">CeMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getCell()">getCell()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getCellLine()">getCellLine()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getCellularLocation()">getCellularLocation()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/AlignTools.html#getCenter(org.biojava.bio.structure.Atom[], int, int)">getCenter(Atom[], int, int)</a></span> - Static method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/AlignTools.html" title="class in org.biojava.bio.structure.align.helper">AlignTools</a></dt>
<dd>
<div class="block">get the centroid for the set of atoms starting fromposition pos, length fragmentLenght</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#getCenter1()">getCenter1()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#getCenter2()">getCenter2()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#getCenterVector(org.biojava.bio.structure.Atom[])">getCenterVector(Atom[])</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">Returns the Vector that needs to be applied to shift a set of atoms
 to the Centroid.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#getCenterVector(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom)">getCenterVector(Atom[], Atom)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">Returns the Vector that needs to be applied to shift a set of atoms
 to the Centroid, if the centroid is already known</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#getCentroid(org.biojava.bio.structure.Atom[])">getCentroid(Atom[])</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">Returns the center  of mass of the set of atoms.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#getChain()">getChain()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">Returns the parent Chain of the Group.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#getChain()">getChain()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">Returns the parent Chain of the Group.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#getChain(int)">getChain(int)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">retrieve a chain by it's position within the Structure .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#getChain(int, int)">getChain(int, int)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">retrieve a chain by it's position within the Structure and model number.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#getChain(int)">getChain(int)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">retrieve a chain by it's position within the Structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#getChain(int, int)">getChain(int, int)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">retrieve a chain by it's position within the Structure and model number.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html#getChain1()">getChain1()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentResult</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html#getChain2()">getChain2()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentResult</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#getChainByPDB(java.lang.String)">getChainByPDB(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">request a chain by it's PDB code
 by default takes only the first model</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#getChainByPDB(java.lang.String, int)">getChainByPDB(String, int)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">request a chain by it's PDB code
 by default takes only the first model</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#getChainByPDB(java.lang.String, int)">getChainByPDB(String, int)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#getChainByPDB(java.lang.String)">getChainByPDB(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/StructureName.html#getChainId()">getChainId()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/StructureName.html" title="class in org.biojava.bio.structure.align.client">StructureName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getChainID()">getChainID()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">get and set the name of this chain (Chain id in PDB file ).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getChainID()">getChainID()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">get and set the name of this chain (Chain id in PDB file ).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getChainId()">getChainId()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>
<div class="block">Returns the chain id value.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#getChainId()">getChainId()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">The chain ID of the corresponding chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#getChainId()">getChainId()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">Utility method for returning the chainId of the Group or null if no
 Chain has been set.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#getChainId()">getChainId()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">Utility method for returning the chainId of the Group or null if no
 Chain has been set.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html#getChainId()">getChainId()</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html" title="class in org.biojava.bio.structure.io.sifts">SiftsResidue</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ResidueNumber.html#getChainId()">getChainId()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ResidueNumber.html" title="class in org.biojava.bio.structure">ResidueNumber</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/PdbPair.html#getChainId1()">getChainId1()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/PdbPair.html" title="class in org.biojava.bio.structure.align.client">PdbPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SSBond.html#getChainID1()">getChainID1()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SSBond.html" title="class in org.biojava.bio.structure">SSBond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/PdbPair.html#getChainId2()">getChainId2()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/PdbPair.html" title="class in org.biojava.bio.structure.align.client">PdbPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SSBond.html#getChainID2()">getChainID2()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SSBond.html" title="class in org.biojava.bio.structure">SSBond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/FlatFileInstallation.html#getChainInfoFile()">getChainInfoFile()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/FlatFileInstallation.html" title="class in org.biojava.bio.structure.server">FlatFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getChainLen()">getChainLen()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getChainRmsd()">getChainRmsd()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getChains()">getChains()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>
<div class="block">get the chains that are part of this Compound</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#getChains()">getChains()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">retrieve all chains - if it is a NMR structure will return the chains of the first model.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#getChains(int)">getChains(int)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">retrieve all chains of a model.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#getChains(int)">getChains(int)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">retrieve all chains of a model.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#getChains()">getChains()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#getChemComp()">getChemComp()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">Get the chemical component that closer describes this group.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#getChemComp()">getChemComp()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/AllChemCompProvider.html#getChemComp(java.lang.String)">getChemComp(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/AllChemCompProvider.html" title="class in org.biojava.bio.structure.io.mmcif">AllChemCompProvider</a></dt>
<dd>
<div class="block">Returns a new instance of a chemical component definition.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompGroupFactory.html#getChemComp(java.lang.String)">getChemComp(String)</a></span> - Static method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompGroupFactory.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompGroupFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompProvider.html#getChemComp(java.lang.String)">getChemComp(String)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompProvider.html" title="interface in org.biojava.bio.structure.io.mmcif">ChemCompProvider</a></dt>
<dd>
<div class="block">Returns a new instance of a chemical component definition.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html#getChemComp(java.lang.String)">getChemComp(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html" title="class in org.biojava.bio.structure.io.mmcif">ChemicalComponentDictionary</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/DownloadChemCompProvider.html#getChemComp(java.lang.String)">getChemComp(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/DownloadChemCompProvider.html" title="class in org.biojava.bio.structure.io.mmcif">DownloadChemCompProvider</a></dt>
<dd>
<div class="block">Loads the definitions for this <a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model"><code>ChemComp</code></a> from a local file and instantiates a new object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ReducedChemCompProvider.html#getChemComp(java.lang.String)">getChemComp(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ReducedChemCompProvider.html" title="class in org.biojava.bio.structure.io.mmcif">ReducedChemCompProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompGroupFactory.html#getChemCompProvider()">getChemCompProvider()</a></span> - Static method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompGroupFactory.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompGroupFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopNode.html#getChildren()">getChildren()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopNode.html" title="class in org.biojava.bio.structure.scop">ScopNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#getClaFilename()">getClaFilename()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#getClassId()">getClassId()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#getClassId()">getClassId()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#getClassification()">getClassification()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#getClassificationId(org.biojava.bio.structure.cath.CathCategory)">getClassificationId(CathCategory)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDescription.html#getClassificationId()">getClassificationId()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDescription.html" title="class in org.biojava.bio.structure.scop">ScopDescription</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDescription.html#getClassificationId(org.biojava.bio.structure.scop.ScopCategory)">getClassificationId(ScopCategory)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDescription.html" title="class in org.biojava.bio.structure.scop">ScopDescription</a></dt>
<dd>
<div class="block">Return a portion of the classificationID corresponding to the specified
 category (class, fold, superfamily, family).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#getClassificationId()">getClassificationId()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#getCluster()">getCluster()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>
<div class="block">get the number of the cluster this alignment belongs to</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#getCluster()">getCluster()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html#getCode()">getCode()</a></span> - Method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html" title="class in org.biojava.bio.structure.quaternary">ModelTransformationMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/IndexPair.html#getCol()">getCol()</a></span> - Method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/IndexPair.html" title="class in org.biojava.bio.structure.align.helper">IndexPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/Alignable.html#getCols()">getCols()</a></span> - Method in interface org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/Alignable.html" title="interface in org.biojava.bio.structure.align.pairwise">Alignable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html#getCols()">getCols()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">StrCompAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#getColumnDimension()">getColumnDimension()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Get column dimension.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#getColumnPackedCopy()">getColumnPackedCopy()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Make a one-dimensional column packed copy of the internal array.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#getComment()">getComment()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#getCommonValence()">getCommonValence()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns the common valence for this Element.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html#getComp_id()">getComp_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemCompDescriptor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxEntityNonPoly.html#getComp_id()">getComp_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxEntityNonPoly.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxEntityNonPoly</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#getCompoundById(java.lang.String)">getCompoundById(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">request a particular compound by its id</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#getCompoundById(java.lang.String)">getCompoundById(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#getCompounds()">getCompounds()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">get all the Compounds that are defined in the PDB Header</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#getCompounds()">getCompounds()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getConn()">getConn()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/Ladder.html#getConnectedFrom()">getConnectedFrom()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/Ladder.html" title="class in org.biojava.bio.structure.secstruc">Ladder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/Ladder.html#getConnectedTo()">getConnectedTo()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/Ladder.html" title="class in org.biojava.bio.structure.secstruc">Ladder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#getConnections()">getConnections()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">Returns the connections value.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#getConnections()">getConnections()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">Returns the connections value.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#getContacts()">getContacts()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/AligMatEl.html#getContig()">getContig()</a></span> - Method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/AligMatEl.html" title="class in org.biojava.bio.structure.align.helper">AligMatEl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#getCoords()">getCoords()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">get the coordinates.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#getCoords()">getCoords()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>
<div class="block">get the coordinates as a double[3] array .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#getCoreElectronCount()">getCoreElectronCount()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns the number of core electrons for this Element.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getCorrelation_coeff_Fo_to_Fc()">getCorrelation_coeff_Fo_to_Fc()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getCorrelation_coeff_Fo_to_Fc_free()">getCorrelation_coeff_Fo_to_Fc_free()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#getCovalentRadius()">getCovalentRadius()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns the covalent radius of this Element.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getCoverage1()">getCoverage1()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">Get the coverage of protein 1 with the alignment</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getCoverage2()">getCoverage2()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">Get the coverage of protein 2 with the alignment</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#getCovered()">getCovered()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/OptimalCECPParameters.html#getCPPoint()">getCPPoint()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/OptimalCECPParameters.html" title="class in org.biojava.bio.structure.align.ce">OptimalCECPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#getCreate_co()">getCreate_co()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#getCrystallographicInfo()">getCrystallographicInfo()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">Gets crystallographic information for this structure</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#getCrystallographicInfo()">getCrystallographicInfo()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">Gets crystallographic information for this structure</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html#getCrystals_number()">getCrystals_number()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html" title="class in org.biojava.bio.structure.io.mmcif.model">Exptl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBStatus.html#getCurrent(java.lang.String)">getCurrent(String)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBStatus.html" title="class in org.biojava.bio.structure">PDBStatus</a></dt>
<dd>
<div class="block">Gets the current version of a PDB ID.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomIterator.html#getCurrentChain()">getCurrentChain()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomIterator.html" title="class in org.biojava.bio.structure">AtomIterator</a></dt>
<dd>
<div class="block">Get the  chain that contains the current atom.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/GroupIterator.html#getCurrentChain()">getCurrentChain()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/GroupIterator.html" title="class in org.biojava.bio.structure">GroupIterator</a></dt>
<dd>
<div class="block">Get the current Chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomIterator.html#getCurrentModel()">getCurrentModel()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomIterator.html" title="class in org.biojava.bio.structure">AtomIterator</a></dt>
<dd>
<div class="block">Get the model number of the model containing the current atom.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/GroupIterator.html#getCurrentModel()">getCurrentModel()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/GroupIterator.html" title="class in org.biojava.bio.structure">GroupIterator</a></dt>
<dd>
<div class="block">Get the model number of the current model.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBStatus.html#getCurrentPDBIds()">getCurrentPDBIds()</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBStatus.html" title="class in org.biojava.bio.structure">PDBStatus</a></dt>
<dd>
<div class="block">Returns a list of current PDB IDs</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/CutSites.html#getCut_sites()">getCut_sites()</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/CutSites.html" title="class in org.biojava.bio.structure.domain.pdp">CutSites</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/EigenvalueDecomposition.html#getD()">getD()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/EigenvalueDecomposition.html" title="class in org.biojava.bio.structure.jama">EigenvalueDecomposition</a></dt>
<dd>
<div class="block">Return the block diagonal eigenvalue matrix</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/TreeSetStringWrapper.html#getData()">getData()</a></span> - Method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/TreeSetStringWrapper.html" title="class in org.biojava.bio.structure.scop.server">TreeSetStringWrapper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#getDatabase()">getDatabase()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">The database of the db-ref.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#getDate()">getDate()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html#getDate()">getDate()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html" title="class in org.biojava.bio.structure.io.mmcif.model">DatabasePDBrev</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html#getDate_original()">getDate_original()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html" title="class in org.biojava.bio.structure.io.mmcif.model">DatabasePDBrev</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#getDb_align_beg()">getDb_align_beg()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#getDb_align_end()">getDb_align_end()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html#getDb_code()">getDb_code()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRef</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html#getDb_name()">getDb_name()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRef</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#getDbAccession()">getDbAccession()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Sequence database accession code.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#getDbIdCode()">getDbIdCode()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Sequence database          identification code.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#getDBRefs()">getDBRefs()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">get the list of database references</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#getDBRefs()">getDBRefs()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html#getDbSearchLegend()">getDbSearchLegend()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce">AbstractUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeSideChainUserArgumentProcessor.html#getDbSearchLegend()">getDbSearchLegend()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeSideChainUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce">CeSideChainUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeUserArgumentProcessor.html#getDbSearchLegend()">getDbSearchLegend()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce">CeUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCatUserArgumentProcessor.html#getDbSearchLegend()">getDbSearchLegend()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCatUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.fatcat">FatCatUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWatermanUserArgumentProcessor.html#getDbSearchLegend()">getDbSearchLegend()</a></span> - Method in class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWatermanUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.seq">SmithWatermanUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#getDBSearchParameters()">getDBSearchParameters()</a></span> - Static method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html#getDbSearchResult(org.biojava.bio.structure.align.model.AFPChain)">getDbSearchResult(AFPChain)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce">AbstractUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#getDbSeqBegin()">getDbSeqBegin()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Initial sequence number of the
    database seqment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#getDbSeqEnd()">getDbSeqEnd()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Ending sequence position  of the database segment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#getDefaultParameters()">getDefaultParameters()</a></span> - Static method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#getDensityCutoff()">getDensityCutoff()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#getDepDate()">getDepDate()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathNode.html#getDescription()">getDescription()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathNode.html" title="class in org.biojava.bio.structure.cath">CathNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#getDescription()">getDescription()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDescription.html#getDescription()">getDescription()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDescription.html" title="class in org.biojava.bio.structure.scop">ScopDescription</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Site.html#getDescription()">getDescription()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Site.html" title="class in org.biojava.bio.structure">Site</a></dt>
<dd>
<div class="block">gets the REMARK 800 description of the site</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getDescription2()">getDescription2()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">Get a textual description for the protein 2 of the alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDatabase.html#getDescriptionByCathId(java.lang.String)">getDescriptionByCathId(String)</a></span> - Method in interface org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDatabase.html" title="interface in org.biojava.bio.structure.cath">CathDatabase</a></dt>
<dd>
<div class="block">Return CATH description for CATH domain ID.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#getDescriptionByCathId(java.lang.String)">getDescriptionByCathId(String)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDatabase.html#getDescriptionByNodeId(java.lang.String)">getDescriptionByNodeId(String)</a></span> - Method in interface org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDatabase.html" title="interface in org.biojava.bio.structure.cath">CathDatabase</a></dt>
<dd>
<div class="block">Return CATH description for node representative by node ID.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#getDescriptionByNodeId(java.lang.String)">getDescriptionByNodeId(String)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html#getDescriptor()">getDescriptor()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemCompDescriptor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getDescriptors()">getDescriptors()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#getDesFilename()">getDesFilename()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getDetails()">getDetails()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html#getDetails()">getDetails()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model">Entity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html#getDetails()">getDetails()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html" title="class in org.biojava.bio.structure.io.mmcif.model">Exptl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssembly.html#getDetails()">getDetails()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssembly.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructAssembly</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getDetails()">getDetails()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html#getDetails()">getDetails()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructAsym</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#getDetails()">getDetails()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/AlignTools.html#getDiagonalAtK(org.biojava.bio.structure.Atom[], int)">getDiagonalAtK(Atom[], int)</a></span> - Static method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/AlignTools.html" title="class in org.biojava.bio.structure.align.helper">AlignTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#getDiagonalDistance()">getDiagonalDistance()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#getDiagonalDistance2()">getDiagonalDistance2()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#getDictionary()">getDictionary()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getDisCut()">getDisCut()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>
<div class="block">Get the distance cutoff used during AFP chain connectivity checks</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getDisFilter()">getDisFilter()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getDisSmooth()">getDisSmooth()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPDistanceMatrix.html#getDist()">getDist()</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPDistanceMatrix.html" title="class in org.biojava.bio.structure.domain.pdp">PDPDistanceMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#getDist1()">getDist1()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#getDist1()">getDist1()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#getDist2()">getDist2()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#getDist2()">getDist2()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getDisTable1()">getDisTable1()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getDisTable2()">getDisTable2()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#getDistance(org.biojava.bio.structure.Atom, org.biojava.bio.structure.Atom)">getDistance(Atom, Atom)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">calculate distance between two atoms.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/DistEn.html#getDistance()">getDistance()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/DistEn.html" title="class in org.biojava.bio.structure.secstruc">DistEn</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#getDistanceFast(org.biojava.bio.structure.Atom, org.biojava.bio.structure.Atom)">getDistanceFast(Atom, Atom)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">Will calculate the *square* of distances between two atoms.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#getDistanceIncrement()">getDistanceIncrement()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/AlignTools.html#getDistanceMatrix(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">getDistanceMatrix(Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/AlignTools.html" title="class in org.biojava.bio.structure.align.helper">AlignTools</a></dt>
<dd>
<div class="block">matrix of all distances between two sets of 3d coords"</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getDistanceMatrix()">getDistanceMatrix()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">A matrix with <i>ca1length</i> rows and <i>ca2length</i> columns.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#getDistanceMatrix(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">getDistanceMatrix(Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#getDistanceMatrix()">getDistanceMatrix()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>
<div class="block">The distance matrix this alignment is based on</div>
</dd>
<dt><a href="./org/biojava/bio/structure/domain/pdp/GetDistanceMatrix.html" title="class in org.biojava.bio.structure.domain.pdp"><span class="strong">GetDistanceMatrix</span></a> - Class in <a href="./org/biojava/bio/structure/domain/pdp/package-summary.html">org.biojava.bio.structure.domain.pdp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/GetDistanceMatrix.html#GetDistanceMatrix()">GetDistanceMatrix()</a></span> - Constructor for class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/GetDistanceMatrix.html" title="class in org.biojava.bio.structure.domain.pdp">GetDistanceMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/GetDistanceMatrix.html#getDistanceMatrix(org.biojava.bio.structure.Atom[])">getDistanceMatrix(Atom[])</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/GetDistanceMatrix.html" title="class in org.biojava.bio.structure.domain.pdp">GetDistanceMatrix</a></dt>
<dd>
<div class="block">A set of Calpha atoms that are representing the protein</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructurePairAligner.html#getDistMat()">getDistMat()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructurePairAligner.html" title="class in org.biojava.bio.structure.align">StructurePairAligner</a></dt>
<dd>
<div class="block">return the difference of distance matrix between the two structures</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/chem/ChemCompTools.html#getDNAOneLetter(java.lang.String)">getDNAOneLetter(String)</a></span> - Static method in class org.biojava.bio.structure.io.mmcif.chem.<a href="./org/biojava/bio/structure/io/mmcif/chem/ChemCompTools.html" title="class in org.biojava.bio.structure.io.mmcif.chem">ChemCompTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/chem/ChemCompTools.html#getDNATwoLetter(java.lang.Character)">getDNATwoLetter(Character)</a></span> - Static method in class org.biojava.bio.structure.io.mmcif.chem.<a href="./org/biojava/bio/structure/io/mmcif/chem/ChemCompTools.html" title="class in org.biojava.bio.structure.io.mmcif.chem">ChemCompTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#getDoi()">getDoi()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>
<div class="block">Get the value of DOI field.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/PDPProvider.html#getDomain(java.lang.String, org.biojava.bio.structure.align.util.AtomCache)">getDomain(String, AtomCache)</a></span> - Method in interface org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/PDPProvider.html" title="interface in org.biojava.bio.structure.domain">PDPProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/RemotePDPProvider.html#getDomain(java.lang.String, org.biojava.bio.structure.align.util.AtomCache)">getDomain(String, AtomCache)</a></span> - Method in class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/RemotePDPProvider.html" title="class in org.biojava.bio.structure.domain">RemotePDPProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDatabase.html#getDomainByCathId(java.lang.String)">getDomainByCathId(String)</a></span> - Method in interface org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDatabase.html" title="interface in org.biojava.bio.structure.cath">CathDatabase</a></dt>
<dd>
<div class="block">Return CATH domain for CATH domain ID.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#getDomainByCathId(java.lang.String)">getDomainByCathId(String)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/CachedRemoteScopInstallation.html#getDomainByScopID(java.lang.String)">getDomainByScopID(String)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/CachedRemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">CachedRemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html#getDomainByScopID(java.lang.String)">getDomainByScopID(String)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">RemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDatabase.html#getDomainByScopID(java.lang.String)">getDomainByScopID(String)</a></span> - Method in interface org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDatabase.html" title="interface in org.biojava.bio.structure.scop">ScopDatabase</a></dt>
<dd>
<div class="block">get a ScopDomain by its SCOP ID (warning, they are not stable between releases!)</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#getDomainByScopID(java.lang.String)">getDomainByScopID(String)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#getDomainCounter()">getDomainCounter()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#getDomainDescriptionFileName()">getDomainDescriptionFileName()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#getDomainId()">getDomainId()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#getDomainId()">getDomainId()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#getDomainListFileName()">getDomainListFileName()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#getDomainName()">getDomainName()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/DomainProvider.html#getDomainNames(java.lang.String)">getDomainNames(String)</a></span> - Method in interface org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/DomainProvider.html" title="interface in org.biojava.bio.structure.domain">DomainProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/PDBDomainProvider.html#getDomainNames(java.lang.String)">getDomainNames(String)</a></span> - Method in class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/PDBDomainProvider.html" title="class in org.biojava.bio.structure.domain">PDBDomainProvider</a></dt>
<dd>
<div class="block">Gets a list of domain representatives for a given PDB ID.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/RemoteDomainProvider.html#getDomainNames(java.lang.String)">getDomainNames(String)</a></span> - Method in class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/RemoteDomainProvider.html" title="class in org.biojava.bio.structure.domain">RemoteDomainProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/DomainProviderFactory.html#getDomainProvider()">getDomainProvider()</a></span> - Static method in class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/DomainProviderFactory.html" title="class in org.biojava.bio.structure.domain">DomainProviderFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/XMLUtil.html#getDomainRangesFromXML(java.lang.String)">getDomainRangesFromXML(String)</a></span> - Static method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/XMLUtil.html" title="class in org.biojava.bio.structure.scop.server">XMLUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/XMLUtil.html#getDomainRangesXML(java.util.SortedSet)">getDomainRangesXML(SortedSet&lt;String&gt;)</a></span> - Static method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/XMLUtil.html" title="class in org.biojava.bio.structure.scop.server">XMLUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/CutDomain.html#getDomains()">getDomains()</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/CutDomain.html" title="class in org.biojava.bio.structure.domain.pdp">CutDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDatabase.html#getDomainsByNodeId(java.lang.String)">getDomainsByNodeId(String)</a></span> - Method in interface org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDatabase.html" title="interface in org.biojava.bio.structure.cath">CathDatabase</a></dt>
<dd>
<div class="block">Return all CATH domains for a particular CATH node.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#getDomainsByNodeId(java.lang.String)">getDomainsByNodeId(String)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDatabase.html#getDomainsForPdb(java.lang.String)">getDomainsForPdb(String)</a></span> - Method in interface org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDatabase.html" title="interface in org.biojava.bio.structure.cath">CathDatabase</a></dt>
<dd>
<div class="block">Return all CATH domains for a PDB ID.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#getDomainsForPdb(java.lang.String)">getDomainsForPdb(String)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/CachedRemoteScopInstallation.html#getDomainsForPDB(java.lang.String)">getDomainsForPDB(String)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/CachedRemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">CachedRemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html#getDomainsForPDB(java.lang.String)">getDomainsForPDB(String)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">RemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDatabase.html#getDomainsForPDB(java.lang.String)">getDomainsForPDB(String)</a></span> - Method in interface org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDatabase.html" title="interface in org.biojava.bio.structure.scop">ScopDatabase</a></dt>
<dd>
<div class="block">Get a list of ScopDomains that have been assigned to a PDB ID</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#getDomainsForPDB(java.lang.String)">getDomainsForPDB(String)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/XMLUtil.html#getDomainsFromXML(java.lang.String)">getDomainsFromXML(String)</a></span> - Static method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/XMLUtil.html" title="class in org.biojava.bio.structure.scop.server">XMLUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/XMLUtil.html#getDomainsXML(java.util.SortedSet)">getDomainsXML(SortedSet&lt;Domain&gt;)</a></span> - Static method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/XMLUtil.html" title="class in org.biojava.bio.structure.scop.server">XMLUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#getDomallFileName()">getDomallFileName()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucState.html#getDonor1()">getDonor1()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucState.html" title="class in org.biojava.bio.structure.secstruc">SecStrucState</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucState.html#getDonor2()">getDonor2()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucState.html" title="class in org.biojava.bio.structure.secstruc">SecStrucState</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/LUDecomposition.html#getDoublePivot()">getDoublePivot()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/LUDecomposition.html" title="class in org.biojava.bio.structure.jama">LUDecomposition</a></dt>
<dd>
<div class="block">Return pivot permutation vector as a one-dimensional double array</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getDVar()">getDVar()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getEcNums()">getEcNums()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#getEditorList()">getEditorList()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>
<div class="block">Get the list of editors of the JournalArticle</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#getElement()">getElement()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">get element of the atom, e.g.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#getElement()">getElement()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/OrderedPair.html#getElement1()">getElement1()</a></span> - Method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/OrderedPair.html" title="class in org.biojava.bio.structure.quaternary">OrderedPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/OrderedPair.html#getElement2()">getElement2()</a></span> - Method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/OrderedPair.html" title="class in org.biojava.bio.structure.quaternary">OrderedPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#getElementFromHillIndex(int)">getElementFromHillIndex(int)</a></span> - Static method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns the Element that corresponds to the specified Hill Order.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#getElementType()">getElementType()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns the Element Type for this Element.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ReducedChemCompProvider.html#getEmptyChemComp()">getEmptyChemComp()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ReducedChemCompProvider.html" title="class in org.biojava.bio.structure.io.mmcif">ReducedChemCompProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsSegment.html#getEnd()">getEnd()</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsSegment.html" title="class in org.biojava.bio.structure.io.sifts">SiftsSegment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/DistEn.html#getEnergy()">getEnergy()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/DistEn.html" title="class in org.biojava.bio.structure.secstruc">DistEn</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/HBond.html#getEnergy()">getEnergy()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/HBond.html" title="class in org.biojava.bio.structure.secstruc">HBond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getEngineered()">getEngineered()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsXMLParser.html#getEntities()">getEntities()</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsXMLParser.html" title="class in org.biojava.bio.structure.io.sifts">SiftsXMLParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/EntityPolySeq.html#getEntity_id()">getEntity_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/EntityPolySeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">EntityPolySeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxEntityNonPoly.html#getEntity_id()">getEntity_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxEntityNonPoly.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxEntityNonPoly</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#getEntity_id()">getEntity_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#getEntity_id()">getEntity_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html#getEntity_id()">getEntity_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructAsym</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html#getEntity_id()">getEntity_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRef</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsEntity.html#getEntityId()">getEntityId()</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsEntity.html" title="class in org.biojava.bio.structure.io.sifts">SiftsEntity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html#getEntry_id()">getEntry_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html" title="class in org.biojava.bio.structure.io.mmcif.model">Exptl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getEntry_id()">getEntry_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html#getEntry_id()">getEntry_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html" title="class in org.biojava.bio.structure.io.mmcif.model">Struct</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructKeywords.html#getEntry_id()">getEntry_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructKeywords.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructKeywords</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#getEqr()">getEqr()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>
<div class="block">returns the number of euqivalent residues in this alignment</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/BigSqrt.html#getError()">getError()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/BigSqrt.html" title="class in org.biojava.bio.structure.secstruc">BigSqrt</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#getEvalCutoff()">getEvalCutoff()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Site.html#getEvCode()">getEvCode()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Site.html" title="class in org.biojava.bio.structure">Site</a></dt>
<dd>
<div class="block">gets the REMARK 800 EVIDENCE CODE for the site.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getExpressionSystem()">getExpressionSystem()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getExpressionSystemAtccNumber()">getExpressionSystemAtccNumber()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getExpressionSystemCell()">getExpressionSystemCell()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getExpressionSystemCellLine()">getExpressionSystemCellLine()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getExpressionSystemCellularLocation()">getExpressionSystemCellularLocation()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getExpressionSystemGene()">getExpressionSystemGene()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getExpressionSystemOrgan()">getExpressionSystemOrgan()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getExpressionSystemOrganelle()">getExpressionSystemOrganelle()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getExpressionSystemOtherDetails()">getExpressionSystemOtherDetails()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getExpressionSystemPlasmid()">getExpressionSystemPlasmid()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getExpressionSystemStrain()">getExpressionSystemStrain()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getExpressionSystemTaxId()">getExpressionSystemTaxId()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getExpressionSystemTissue()">getExpressionSystemTissue()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getExpressionSystemVariant()">getExpressionSystemVariant()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getExpressionSystemVector()">getExpressionSystemVector()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getExpressionSystemVectorType()">getExpressionSystemVectorType()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#getFamilyId()">getFamilyId()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCat.html#getFatCatAligner()">getFatCatAligner()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCat.html" title="class in org.biojava.bio.structure.align.fatcat">FatCat</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#getFile1()">getFile1()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#getFile2()">getFile2()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/util/FileDownloadUtils.html#getFileExtension(java.io.File)">getFileExtension(File)</a></span> - Static method in class org.biojava.bio.structure.io.util.<a href="./org/biojava/bio/structure/io/util/FileDownloadUtils.html" title="class in org.biojava.bio.structure.io.util">FileDownloadUtils</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/UserConfiguration.html#getFileFormat()">getFileFormat()</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/UserConfiguration.html" title="class in org.biojava.bio.structure.align.util">UserConfiguration</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/LocalCacheStructureProvider.html#getFileParsingParameters()">getFileParsingParameters()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/LocalCacheStructureProvider.html" title="class in org.biojava.bio.structure.io">LocalCacheStructureProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#getFileParsingParameters()">getFileParsingParameters()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#getFileParsingParameters()">getFileParsingParameters()</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#getFileParsingParameters()">getFileParsingParameters()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/MMCIFFileReader.html#getFileParsingParameters()">getFileParsingParameters()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/MMCIFFileReader.html" title="class in org.biojava.bio.structure.io">MMCIFFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileParser.html#getFileParsingParameters()">getFileParsingParameters()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileParser.html" title="class in org.biojava.bio.structure.io">PDBFileParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#getFileParsingParameters()">getFileParsingParameters()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/SandboxStyleStructureProvider.html#getFileParsingParameters()">getFileParsingParameters()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/SandboxStyleStructureProvider.html" title="class in org.biojava.bio.structure.io">SandboxStyleStructureProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/StructureIOFile.html#getFileParsingParameters()">getFileParsingParameters()</a></span> - Method in interface org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/StructureIOFile.html" title="interface in org.biojava.bio.structure.io">StructureIOFile</a></dt>
<dd>
<div class="block">Get the parameters that should be used for file parsing</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/StructureProvider.html#getFileParsingParameters()">getFileParsingParameters()</a></span> - Method in interface org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/StructureProvider.html" title="interface in org.biojava.bio.structure.io">StructureProvider</a></dt>
<dd>
<div class="block">Get the parameters that should be used for file parsing</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#getFileParsingParams()">getFileParsingParams()</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/FlatFileInstallation.html#getFilePath()">getFilePath()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/FlatFileInstallation.html" title="class in org.biojava.bio.structure.server">FlatFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/util/FileDownloadUtils.html#getFilePrefix(java.io.File)">getFilePrefix(File)</a></span> - Static method in class org.biojava.bio.structure.io.util.<a href="./org/biojava/bio/structure/io/util/FileDownloadUtils.html" title="class in org.biojava.bio.structure.io.util">FileDownloadUtils</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getFocusAfpList()">getFocusAfpList()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getFocusAfpn()">getFocusAfpn()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getFocusRes1()">getFocusRes1()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getFocusRes2()">getFocusRes2()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getFocusResn()">getFocusResn()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#getFoldId()">getFoldId()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#getFormat()">getFormat()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getFormula()">getFormula()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getFormula_weight()">getFormula_weight()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html#getFormula_weight()">getFormula_weight()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model">Entity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#getFragCompat()">getFragCompat()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getFragLen()">getFragLen()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFP.html#getFragLen()">getFragLen()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFP.html" title="class in org.biojava.bio.structure.align.model">AFP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getFragLenSq()">getFragLenSq()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/AlignTools.html#getFragment(org.biojava.bio.structure.Atom[], int, int)">getFragment(Atom[], int, int)</a></span> - Static method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/AlignTools.html" title="class in org.biojava.bio.structure.align.helper">AlignTools</a></dt>
<dd>
<div class="block">get a continue subset of Atoms based by the starting position and the length</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getFragment()">getFragment()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/AlignTools.html#getFragmentFromIdxList(org.biojava.bio.structure.Atom[], int[])">getFragmentFromIdxList(Atom[], int[])</a></span> - Static method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/AlignTools.html" title="class in org.biojava.bio.structure.align.helper">AlignTools</a></dt>
<dd>
<div class="block">get a subset of Atoms based by their positions</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathFragment.html#getFragmentId()">getFragmentId()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathFragment.html" title="class in org.biojava.bio.structure.cath">CathFragment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#getFragmentLength()">getFragmentLength()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#getFragmentMiniDistance()">getFragmentMiniDistance()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/AlignTools.html#getFragmentNoClone(org.biojava.bio.structure.Atom[], int, int)">getFragmentNoClone(Atom[], int, int)</a></span> - Static method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/AlignTools.html" title="class in org.biojava.bio.structure.align.helper">AlignTools</a></dt>
<dd>
<div class="block">get a continue subset of Atoms based by the starting position and the length
 does not clone the original atoms.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructurePairAligner.html#getFragmentPairs()">getFragmentPairs()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructurePairAligner.html" title="class in org.biojava.bio.structure.align">StructurePairAligner</a></dt>
<dd>
<div class="block">get the results of step 1 - the FragmentPairs used for seeding the alignment</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDatabase.html#getFragmentsByPdbId(java.lang.String)">getFragmentsByPdbId(String)</a></span> - Method in interface org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDatabase.html" title="interface in org.biojava.bio.structure.cath">CathDatabase</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#getFragmentsByPdbId(java.lang.String)">getFragmentsByPdbId(String)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getFragScore()">getFragScore()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Segment.html#getFrom()">getFrom()</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Segment.html" title="class in org.biojava.bio.structure.domain.pdp">Segment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/Ladder.html#getFrom()">getFrom()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/Ladder.html" title="class in org.biojava.bio.structure.secstruc">Ladder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/SeqRes2AtomAligner.html#getFullAtomSequence(java.util.List, java.util.Map)">getFullAtomSequence(List&lt;Group&gt;, Map&lt;Integer, Integer&gt;)</a></span> - Static method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/SeqRes2AtomAligner.html" title="class in org.biojava.bio.structure.io">SeqRes2AtomAligner</a></dt>
<dd>
<div class="block">returns the full sequence of the Atom records of a parent
 with X instead of HETATMSs.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#getFullName()">getFullName()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">get full name of atom e.g.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#getFullName()">getFullName()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>
<div class="block">get full name of atom e.g.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBCrystallographicInfo.html#getGamma()">getGamma()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBCrystallographicInfo.html" title="class in org.biojava.bio.structure">PDBCrystallographicInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getGapCreate()">getGapCreate()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/Alignable.html#getGapExtCol()">getGapExtCol()</a></span> - Method in interface org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/Alignable.html" title="interface in org.biojava.bio.structure.align.pairwise">Alignable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html#getGapExtCol()">getGapExtCol()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">StrCompAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getGapExtend()">getGapExtend()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWaterman3DParameters.html#getGapExtend()">getGapExtend()</a></span> - Method in class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWaterman3DParameters.html" title="class in org.biojava.bio.structure.align.seq">SmithWaterman3DParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#getGapExtension()">getGapExtension()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#getGapExtension()">getGapExtension()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/Alignable.html#getGapExtRow()">getGapExtRow()</a></span> - Method in interface org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/Alignable.html" title="interface in org.biojava.bio.structure.align.pairwise">Alignable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html#getGapExtRow()">getGapExtRow()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">StrCompAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getGapLen()">getGapLen()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#getGapOpen()">getGapOpen()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWaterman3DParameters.html#getGapOpen()">getGapOpen()</a></span> - Method in class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWaterman3DParameters.html" title="class in org.biojava.bio.structure.align.seq">SmithWaterman3DParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#getGapOpen()">getGapOpen()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/Alignable.html#getGapOpenCol()">getGapOpenCol()</a></span> - Method in interface org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/Alignable.html" title="interface in org.biojava.bio.structure.align.pairwise">Alignable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html#getGapOpenCol()">getGapOpenCol()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">StrCompAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/Alignable.html#getGapOpenRow()">getGapOpenRow()</a></span> - Method in interface org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/Alignable.html" title="interface in org.biojava.bio.structure.align.pairwise">Alignable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html#getGapOpenRow()">getGapOpenRow()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">StrCompAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#getGaps()">getGaps()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>
<div class="block">return the number of gaps in this alignment</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getGene()">getGene()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#getGroup()">getGroup()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">Returns the parent Group of the Atom.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#getGroup()">getGroup()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getGroup(int)">getGroup(int)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>use getAtomGroup or getSeqResGroup instead</i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getGroup(int)">getGroup(int)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>use getAtomGroup or getSeqResGroup instead</i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getGroup_PDB()">getGroup_PDB()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Bond.html#getGroupA()">getGroupA()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Bond.html" title="class in org.biojava.bio.structure">Bond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Bond.html#getGroupB()">getGroupB()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Bond.html" title="class in org.biojava.bio.structure">Bond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getGroupByPDB(java.lang.String)">getGroupByPDB(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>replaced by <a href="./org/biojava/bio/structure/Chain.html#getGroupByPDB(org.biojava.bio.structure.ResidueNumber)"><code>Chain.getGroupByPDB(ResidueNumber)</code></a></i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getGroupByPDB(org.biojava.bio.structure.ResidueNumber)">getGroupByPDB(ResidueNumber)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Get a group by its PDB residue numbering.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getGroupByPDB(java.lang.String)">getGroupByPDB(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">get a group by its PDB residue numbering.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getGroupByPDB(org.biojava.bio.structure.ResidueNumber)">getGroupByPDB(ResidueNumber)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Get a group by its PDB residue numbering.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#getGroupByPDBResidueNumber(org.biojava.bio.structure.Structure, org.biojava.bio.structure.ResidueNumber)">getGroupByPDBResidueNumber(Structure, ResidueNumber)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>
<div class="block">Get a group represented by a ResidueNumber.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompGroupFactory.html#getGroupFromChemCompDictionary(java.lang.String)">getGroupFromChemCompDictionary(String)</a></span> - Static method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompGroupFactory.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompGroupFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getGroups(java.lang.String)">getGroups(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>use getAtomGroups or getSeqResGroups instead</i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getGroups()">getGroups()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>use getAtomGroups or getSeqResGroups instead</i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getGroups(java.lang.String)">getGroups(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>use getAtomGroups instead</i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getGroups()">getGroups()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>use getAtomGroups instead</i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStruc.html#getGroups()">getGroups()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStruc.html" title="class in org.biojava.bio.structure.secstruc">SecStruc</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Site.html#getGroups()">getGroups()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Site.html" title="class in org.biojava.bio.structure">Site</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getGroupsByPDB(java.lang.String, java.lang.String)">getGroupsByPDB(String, String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>replaced by <a href="./org/biojava/bio/structure/Chain.html#getGroupsByPDB(org.biojava.bio.structure.ResidueNumber, org.biojava.bio.structure.ResidueNumber)"><code>Chain.getGroupsByPDB(ResidueNumber, ResidueNumber)</code></a></i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getGroupsByPDB(org.biojava.bio.structure.ResidueNumber, org.biojava.bio.structure.ResidueNumber)">getGroupsByPDB(ResidueNumber, ResidueNumber)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Get all groups that are located between two PDB residue numbers.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getGroupsByPDB(java.lang.String, java.lang.String, boolean)">getGroupsByPDB(String, String, boolean)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>replaced by #<a href="./org/biojava/bio/structure/Chain.html#getGroupsByPDB(org.biojava.bio.structure.ResidueNumber, org.biojava.bio.structure.ResidueNumber, boolean)"><code>Chain.getGroupsByPDB(ResidueNumber, ResidueNumber, boolean)</code></a></i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getGroupsByPDB(org.biojava.bio.structure.ResidueNumber, org.biojava.bio.structure.ResidueNumber, boolean)">getGroupsByPDB(ResidueNumber, ResidueNumber, boolean)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Get all groups that are located between two PDB residue numbers.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getGroupsByPDB(java.lang.String, java.lang.String, boolean)">getGroupsByPDB(String, String, boolean)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Get all groups that are located between two PDB residue numbers.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getGroupsByPDB(org.biojava.bio.structure.ResidueNumber, org.biojava.bio.structure.ResidueNumber, boolean)">getGroupsByPDB(ResidueNumber, ResidueNumber, boolean)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getGroupsByPDB(java.lang.String, java.lang.String)">getGroupsByPDB(String, String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Get all groups that are located between two PDB residue numbers.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getGroupsByPDB(org.biojava.bio.structure.ResidueNumber, org.biojava.bio.structure.ResidueNumber)">getGroupsByPDB(ResidueNumber, ResidueNumber)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Get all groups that are located between two PDB residue numbers.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#getGroupsWithinShell(org.biojava.bio.structure.Structure, org.biojava.bio.structure.Group, double, boolean)">getGroupsWithinShell(Structure, Group, double, boolean)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/QRDecomposition.html#getH()">getH()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/QRDecomposition.html" title="class in org.biojava.bio.structure.jama">QRDecomposition</a></dt>
<dd>
<div class="block">Return the Householder vectors</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html#getH()">getH()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html" title="class in org.biojava.bio.structure.secstruc">SecStrucGroup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getHeader()">getHeader()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Returns the Header ( a Compound object) for this chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getHeader()">getHeader()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Returns the Header ( a Compound object) for this chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#getHeader()">getHeader()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>use getPDBHeader instead</i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#getHeader()">getHeader()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">get Header data.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getHeaderVars()">getHeaderVars()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/EntityPolySeq.html#getHetero()">getHetero()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/EntityPolySeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">EntityPolySeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#getHetero()">getHetero()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#getHetGroups()">getHetGroups()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#getHetGroups()">getHetGroups()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#getHieFilename()">getHieFilename()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#getHillOrder()">getHillOrder()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns the Hill Order of this Element.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#getHomologyId()">getHomologyId()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPDistanceMatrix.html#getIclose()">getIclose()</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPDistanceMatrix.html" title="class in org.biojava.bio.structure.domain.pdp">PDPDistanceMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFP.html#getId()">getId()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFP.html" title="class in org.biojava.bio.structure.align.model">AFP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getId()">getId()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#getId()">getId()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>
<div class="block">Get the Hibernate database ID.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getId()">getId()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Get the ID used by Hibernate.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getId()">getId()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Get the ID used by Hibernate.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getId()">getId()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>
<div class="block">get the ID used by Hibernate</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#getId()">getId()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Get the ID used by Hibernate.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Domain.html#getId()">getId()</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Domain.html" title="class in org.biojava.bio.structure.domain.pdp">Domain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#getId()">getId()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">the Hibernate database ID</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getId()">getId()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getId()">getId()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBremark.html#getId()">getId()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBremark.html" title="class in org.biojava.bio.structure.io.mmcif.model">DatabasePDBremark</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html#getId()">getId()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model">Entity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssembly.html#getId()">getId()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssembly.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructAssembly</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html#getId()">getId()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructOperList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html#getId()">getId()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructAsym</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html#getId()">getId()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRef</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#getId()">getId()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>
<div class="block">Get the ID used by Hibernate.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html#getId()">getId()</a></span> - Method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html" title="class in org.biojava.bio.structure.quaternary">ModelTransformationMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#getId()">getId()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">get the ID used by Hibernate</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#getId()">getId()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">get the ID used by Hibernate</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#getIdbnsBegin()">getIdbnsBegin()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Insertion code of initial residue of the segment, if PDB is the
    reference.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#getIdbnsEnd()">getIdbnsEnd()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Insertion code of the ending
    residue of the segment, if PDB is
    the reference.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#getIdCode()">getIdCode()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">get the idCode for this entry</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#getIdCode()">getIdCode()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>
<div class="block">The PDB code for this protein structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#getIdenticalSequenceFamilyId()">getIdenticalSequenceFamilyId()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getIdentity()">getIdentity()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">The percent of residues that are sequence-identical in the alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#getIdx1()">getIdx1()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>
<div class="block">the positions of the structure equivalent positions in atom set 1</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#getIdx2()">getIdx2()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>
<div class="block">the positions of the structure equivalent atoms in atom set 2</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/JointFragments.html#getIdxlist()">getIdxlist()</a></span> - Method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/JointFragments.html" title="class in org.biojava.bio.structure.align.helper">JointFragments</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/EigenvalueDecomposition.html#getImagEigenvalues()">getImagEigenvalues()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/EigenvalueDecomposition.html" title="class in org.biojava.bio.structure.jama">EigenvalueDecomposition</a></dt>
<dd>
<div class="block">Return the imaginary parts of the eigenvalues</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/GapArray.html#getIndex()">getIndex()</a></span> - Method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/GapArray.html" title="class in org.biojava.bio.structure.align.helper">GapArray</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#getInitialK()">getInitialK()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Author.html#getInitials()">getInitials()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Author.html" title="class in org.biojava.bio.structure">Author</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/HTTPConnectionTools.html#getInputStream(java.net.URL, int)">getInputStream(URL, int)</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/HTTPConnectionTools.html" title="class in org.biojava.bio.structure.align.util">HTTPConnectionTools</a></dt>
<dd>
<div class="block">connect to DAS server and return result as an InputStream.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/HTTPConnectionTools.html#getInputStream(java.net.URL)">getInputStream(URL)</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/HTTPConnectionTools.html" title="class in org.biojava.bio.structure.align.util">HTTPConnectionTools</a></dt>
<dd>
<div class="block">connect to DAS server and return result as an InputStream.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/HTTPConnectionTools.html#getInputStream(java.net.URL, boolean, int)">getInputStream(URL, boolean, int)</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/HTTPConnectionTools.html" title="class in org.biojava.bio.structure.align.util">HTTPConnectionTools</a></dt>
<dd>
<div class="block">open a URL and return an InputStream to it
  if acceptGzipEncoding == true, use GZIPEncoding to
  compress communication</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ResidueNumber.html#getInsCode()">getInsCode()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ResidueNumber.html" title="class in org.biojava.bio.structure">ResidueNumber</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SSBond.html#getInsCode1()">getInsCode1()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SSBond.html" title="class in org.biojava.bio.structure">SSBond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SSBond.html#getInsCode2()">getInsCode2()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SSBond.html" title="class in org.biojava.bio.structure">SSBond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#getInsertBegin()">getInsertBegin()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Initial insertion code of the PDB sequence segment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#getInsertEnd()">getInsertEnd()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Ending insertion code of the PDB sequence segment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#getInstalledDomainDescription()">getInstalledDomainDescription()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#getInstalledDomainList()">getInstalledDomainList()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#getInstalledDomall()">getInstalledDomall()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#getInstalledNodeList()">getInstalledNodeList()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getInternalChainID()">getInternalChainID()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">If available, returns the internal chain ID that is used in mmCif files, otherwise null</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getInternalChainID()">getInternalChainID()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getIoTime()">getIoTime()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html#getIoTime()">getIoTime()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentResult</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/BigSqrt.html#getIterations()">getIterations()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/BigSqrt.html" title="class in org.biojava.bio.structure.secstruc">BigSqrt</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPDistanceMatrix.html#getJclose()">getJclose()</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPDistanceMatrix.html" title="class in org.biojava.bio.structure.domain.pdp">PDPDistanceMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#getJoinRMSCutoff()">getJoinRMSCutoff()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#getJournalArticle()">getJournalArticle()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">get the associated publication as defined by the JRNL records in a PDB
 file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#getJournalArticle()">getJournalArticle()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">get the associated publication as defined by the JRNL records in a PDB
 file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#getJournalName()">getJournalName()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucState.html#getKappa()">getKappa()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucState.html" title="class in org.biojava.bio.structure.secstruc">SecStrucState</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/CholeskyDecomposition.html#getL()">getL()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/CholeskyDecomposition.html" title="class in org.biojava.bio.structure.jama">CholeskyDecomposition</a></dt>
<dd>
<div class="block">Return triangular factor.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/LUDecomposition.html#getL()">getL()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/LUDecomposition.html" title="class in org.biojava.bio.structure.jama">LUDecomposition</a></dt>
<dd>
<div class="block">Return lower triangular factor</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getLabel_alt_id()">getLabel_alt_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getLabel_asym_id()">getLabel_asym_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getLabel_atom_id()">getLabel_atom_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getLabel_comp_id()">getLabel_comp_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getLabel_entity_id()">getLabel_entity_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getLabel_seq_id()">getLabel_seq_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#getLcmp()">getLcmp()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#getLcmp()">getLcmp()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#getLength()">getLength()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Bond.html#getLength()">getLength()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Bond.html" title="class in org.biojava.bio.structure">Bond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#getLength()">getLength()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathFragment.html#getLength()">getLength()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathFragment.html" title="class in org.biojava.bio.structure.cath">CathFragment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathSegment.html#getLength()">getLength()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathSegment.html" title="class in org.biojava.bio.structure.cath">CathSegment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getLength()">getLength()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>please use getAtomLength or getLengthSeqRes instead</i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getLength()">getLength()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>use getAtomLength instead</i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html#getLength1()">getLength1()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentResult</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html#getLength2()">getLength2()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentResult</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getLengthAminos()">getLengthAminos()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>use getAtomGroups("amino").size() instead.</i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getLengthAminos()">getLengthAminos()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">returns the length of the AminoAcids in the ATOM records of this chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/Ladder.html#getLfrom()">getLfrom()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/Ladder.html" title="class in org.biojava.bio.structure.secstruc">Ladder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#getLikeSequenceFamilyId()">getLikeSequenceFamilyId()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/DownloadChemCompProvider.html#getLocalFileName(java.lang.String)">getLocalFileName(String)</a></span> - Static method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/DownloadChemCompProvider.html" title="class in org.biojava.bio.structure.io.mmcif">DownloadChemCompProvider</a></dt>
<dd>
<div class="block">Returns the file name that contains the definition for this <a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model"><code>ChemComp</code></a></div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getLs_d_res_high()">getLs_d_res_high()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getLs_d_res_low()">getLs_d_res_low()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getLs_number_parameters()">getLs_number_parameters()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getLs_number_reflns_all()">getLs_number_reflns_all()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getLs_number_reflns_obs()">getLs_number_reflns_obs()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getLs_number_reflns_R_free()">getLs_number_reflns_R_free()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getLs_number_restraints()">getLs_number_restraints()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getLs_percent_reflns_obs()">getLs_percent_reflns_obs()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getLs_percent_reflns_R_free()">getLs_percent_reflns_R_free()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getLs_R_factor_all()">getLs_R_factor_all()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getLs_R_factor_obs()">getLs_R_factor_obs()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getLs_R_factor_R_free()">getLs_R_factor_R_free()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getLs_R_factor_R_free_error()">getLs_R_factor_R_free_error()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getLs_R_factor_R_free_error_details()">getLs_R_factor_R_free_error_details()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getLs_R_factor_R_work()">getLs_R_factor_R_work()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getLs_redundancy_reflns_obs()">getLs_redundancy_reflns_obs()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getLs_wR_factor_R_free()">getLs_wR_factor_R_free()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getLs_wR_factor_R_work()">getLs_wR_factor_R_work()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/Ladder.html#getLto()">getLto()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/Ladder.html" title="class in org.biojava.bio.structure.secstruc">Ladder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFP.html#getM()">getM()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFP.html" title="class in org.biojava.bio.structure.align.model">AFP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html#getMandatoryArgs()">getMandatoryArgs()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce">AbstractUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/SeqRes2AtomAligner.html#getMatchingAtomRes(org.biojava.bio.structure.Chain, java.util.List)">getMatchingAtomRes(Chain, List&lt;Chain&gt;)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/SeqRes2AtomAligner.html" title="class in org.biojava.bio.structure.io">SeqRes2AtomAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#getMatMatrix()">getMatMatrix()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>
<div class="block">Caution: this matrix is overwriten with very different data at several
 points in the alignment algorithm.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#getMatMatrix()">getMatMatrix()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>
<div class="block">Caution: this matrix is overwriten with very different data at several
 points in the alignment algorithm.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html#getMatrix()">getMatrix()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructOperList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#getMatrix(int, int, int, int)">getMatrix(int, int, int, int)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Get a submatrix.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#getMatrix(int[], int[])">getMatrix(int[], int[])</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Get a submatrix.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#getMatrix(int, int, int[])">getMatrix(int, int, int[])</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Get a submatrix.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#getMatrix(int[], int, int)">getMatrix(int[], int, int)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Get a submatrix.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html#getMatrix()">getMatrix()</a></span> - Method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html" title="class in org.biojava.bio.structure.quaternary">ModelTransformationMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html#getMatrix11()">getMatrix11()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructOperList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html#getMatrix12()">getMatrix12()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructOperList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html#getMatrix13()">getMatrix13()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructOperList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html#getMatrix21()">getMatrix21()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructOperList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html#getMatrix22()">getMatrix22()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructOperList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html#getMatrix23()">getMatrix23()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructOperList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html#getMatrix31()">getMatrix31()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructOperList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html#getMatrix32()">getMatrix32()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructOperList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html#getMatrix33()">getMatrix33()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructOperList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileParsingParameters.html#getMaxAtoms()">getMaxAtoms()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileParsingParameters.html" title="class in org.biojava.bio.structure.io">FileParsingParameters</a></dt>
<dd>
<div class="block">the maximum numbers of atoms to load in a protein structure (prevents memory overflows)</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getMaxGap()">getMaxGap()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getMaxGapFrag()">getMaxGapFrag()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#getMaxGapSize()">getMaxGapSize()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>
<div class="block">the Max gap size parameter G .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#getMaxGapSize()">getMaxGapSize()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>
<div class="block">CE specific parameter: set the Max gap size parameter G (during AFP extension).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#getMaximumCovalentValence()">getMaximumCovalentValence()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns the maximum valence for this Element.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/BioAssemblyTools.html#getMaximumExtend(org.biojava.bio.structure.Structure)">getMaximumExtend(Structure)</a></span> - Static method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/BioAssemblyTools.html" title="class in org.biojava.bio.structure.quaternary">BioAssemblyTools</a></dt>
<dd>
<div class="block">Returns the maximum extend of the structure in the x, y, or z direction.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#getMaximumValence()">getMaximumValence()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns the maximum valence for this Element.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#getMaxIter()">getMaxIter()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/BigSqrt.html#getMaxIterations()">getMaxIterations()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/BigSqrt.html" title="class in org.biojava.bio.structure.secstruc">BigSqrt</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#getMaxNrIterationsForOptimization()">getMaxNrIterationsForOptimization()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>
<div class="block">Get the maximum nr of times the (slow) optimiziation of alignment should iterate.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#getMaxOptRMSD()">getMaxOptRMSD()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>
<div class="block">Returns the maximum RMSD cutoff to be applied during alignment optimization (default: 99 = unlimited)</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#getMaxOptRMSD()">getMaxOptRMSD()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>
<div class="block">(jCE specific): maximum RMSD that shall be calculated for the alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getMaxPenalty()">getMaxPenalty()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#getMaxrefine()">getMaxrefine()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getMaxTra()">getMaxTra()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>
<div class="block">get the maximum number of Twists that are allowed...</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getMaxTra()">getMaxTra()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">get the maximum nr of Twists that are allowed...</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/PdbPairsMessage.html#getMethod()">getMethod()</a></span> - Method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/PdbPairsMessage.html" title="class in org.biojava.bio.structure.align.xml">PdbPairsMessage</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html#getMethod()">getMethod()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html" title="class in org.biojava.bio.structure.io.mmcif.model">Exptl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#getMethod()">getMethod()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html#getMethod_details()">getMethod_details()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html" title="class in org.biojava.bio.structure.io.mmcif.model">Exptl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssembly.html#getMethod_details()">getMethod_details()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssembly.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructAssembly</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#getMinimumValence()">getMinimumValence()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns the minimum valence for this Element.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getMinLen()">getMinLen()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getMisCut()">getMisCut()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getMisLen()">getMisLen()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getMisScore()">getMisScore()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/MMCIFFileReader.html#getMMcifConsumer()">getMMcifConsumer()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/MMCIFFileReader.html" title="class in org.biojava.bio.structure.io">MMCIFFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html#getMod_type()">getMod_type()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html" title="class in org.biojava.bio.structure.io.mmcif.model">DatabasePDBrev</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#getModDate()">getModDate()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#getModel(int)">getModel(int)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">retrieve all Chains belonging to a model .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#getModel(int)">getModel(int)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">retrieve all Chains belonging to a model .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getMolId()">getMolId()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>
<div class="block">Returns the mol id value.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getMolName()">getMolName()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/EntityPolySeq.html#getMon_id()">getMon_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/EntityPolySeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">EntityPolySeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#getMon_id()">getMon_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#getMon_id()">getMon_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getMon_nstd_flag()">getMon_nstd_flag()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getMon_nstd_parent_comp_id()">getMon_nstd_parent_comp_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getMutation()">getMutation()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcid.html#getN()">getN()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcid.html" title="interface in org.biojava.bio.structure">AminoAcid</a></dt>
<dd>
<div class="block">get N atom.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcidImpl.html#getN()">getN()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcidImpl.html" title="class in org.biojava.bio.structure">AminoAcidImpl</a></dt>
<dd>
<div class="block">get N atom.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html#getN()">getN()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html" title="class in org.biojava.bio.structure.secstruc">SecStrucGroup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/StructureName.html#getName()">getName()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/StructureName.html" title="class in org.biojava.bio.structure.align.client">StructureName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#getName()">getName()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">get trimmed version of atom name, e.g.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#getName()">getName()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>
<div class="block">Gets this object's name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#getName()">getName()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getName()">getName()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>replaced by <a href="./org/biojava/bio/structure/Chain.html#getChainID()"><code>Chain.getChainID()</code></a></i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getName()">getName()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AuditAuthor.html#getName()">getName()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AuditAuthor.html" title="class in org.biojava.bio.structure.io.mmcif.model">AuditAuthor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getName()">getName()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxEntityNonPoly.html#getName()">getName()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxEntityNonPoly.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxEntityNonPoly</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDescription.html#getName()">getName()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDescription.html" title="class in org.biojava.bio.structure.scop">ScopDescription</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#getName()">getName()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">get biological name of Structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#getName()">getName()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">get biological name of Structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/PdbPair.html#getName1()">getName1()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/PdbPair.html" title="class in org.biojava.bio.structure.align.client">PdbPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getName1()">getName1()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/PdbPair.html#getName2()">getName2()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/PdbPair.html" title="class in org.biojava.bio.structure.align.client">PdbPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getName2()">getName2()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#getnAtom()">getnAtom()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#getnAtom()">getnAtom()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html#getNaturalPos()">getNaturalPos()</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html" title="class in org.biojava.bio.structure.io.sifts">SiftsResidue</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPDistanceMatrix.html#getNclose()">getNclose()</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPDistanceMatrix.html" title="class in org.biojava.bio.structure.domain.pdp">PDPDistanceMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/CutSites.html#getNcuts()">getNcuts()</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/CutSites.html" title="class in org.biojava.bio.structure.domain.pdp">CutSites</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#getNdb_seq_num()">getNdb_seq_num()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#getNdb_seq_num()">getNdb_seq_num()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html#getNdbChainId()">getNdbChainId()</a></span> - Method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html" title="class in org.biojava.bio.structure.quaternary">ModelTransformationMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathNode.html#getNodeId()">getNodeId()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathNode.html" title="class in org.biojava.bio.structure.cath">CathNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#getNodeListFileName()">getNodeListFileName()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getNormAlignScore()">getNormAlignScore()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html#getNotObserved()">getNotObserved()</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html" title="class in org.biojava.bio.structure.io.sifts">SiftsResidue</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#getNrAlignments()">getNrAlignments()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#getNrAtoms(org.biojava.bio.structure.Structure)">getNrAtoms(Structure)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>
<div class="block">Count how many number of Atoms are contained within a Structure object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBBioAssemblyParser.html#getNrBioAssemblies()">getNrBioAssemblies()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBBioAssemblyParser.html" title="class in org.biojava.bio.structure.io">PDBBioAssemblyParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#getNrBioAssemblies()">getNrBioAssemblies()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/BioUnitDataProvider.html#getNrBiolAssemblies(java.lang.String)">getNrBiolAssemblies(String)</a></span> - Method in interface org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/BioUnitDataProvider.html" title="interface in org.biojava.bio.structure.quaternary.io">BioUnitDataProvider</a></dt>
<dd>
<div class="block">Returns the number of available biological assemblies.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/FileBasedPDBBioUnitDataProvider.html#getNrBiolAssemblies(java.lang.String)">getNrBiolAssemblies(String)</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/FileBasedPDBBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">FileBasedPDBBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/MmCifBiolAssemblyProvider.html#getNrBiolAssemblies(java.lang.String)">getNrBiolAssemblies(String)</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/MmCifBiolAssemblyProvider.html" title="class in org.biojava.bio.structure.quaternary.io">MmCifBiolAssemblyProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/MmCifPDBBiolAssemblyProvider.html#getNrBiolAssemblies()">getNrBiolAssemblies()</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/MmCifPDBBiolAssemblyProvider.html" title="class in org.biojava.bio.structure.quaternary.io">MmCifPDBBiolAssemblyProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/PDBBioUnitDataProvider.html#getNrBiolAssemblies(java.lang.String)">getNrBiolAssemblies(String)</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/PDBBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">PDBBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/RawBioUnitDataProvider.html#getNrBiolAssemblies()">getNrBiolAssemblies()</a></span> - Method in interface org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/RawBioUnitDataProvider.html" title="interface in org.biojava.bio.structure.quaternary.io">RawBioUnitDataProvider</a></dt>
<dd>
<div class="block">Returns the number of available biological assemblies.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/RemoteBioUnitDataProvider.html#getNrBiolAssemblies(java.lang.String)">getNrBiolAssemblies(String)</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/RemoteBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">RemoteBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/RemoteRawBioUnitDataProvider.html#getNrBiolAssemblies()">getNrBiolAssemblies()</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/RemoteRawBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">RemoteRawBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/structure/StructureIO.html#getNrBiologicalAssemblies(java.lang.String)">getNrBiologicalAssemblies(String)</a></span> - Static method in class org.biojava3.structure.<a href="./org/biojava3/structure/StructureIO.html" title="class in org.biojava3.structure">StructureIO</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/SimpleStructureServer.html#getNrCPUs()">getNrCPUs()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/SimpleStructureServer.html" title="class in org.biojava.bio.structure.server">SimpleStructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getNrEQR()">getNrEQR()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">Get the number of structurally equivalent residues</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#getNrGroups(org.biojava.bio.structure.Structure)">getNrGroups(Structure)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>
<div class="block">Count how many groups are contained within a structure object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Domain.html#getNseg()">getNseg()</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Domain.html" title="class in org.biojava.bio.structure.domain.pdp">Domain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html#getNum()">getNum()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html" title="class in org.biojava.bio.structure.io.mmcif.model">DatabasePDBrev</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/EntityPolySeq.html#getNum()">getNum()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/EntityPolySeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">EntityPolySeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getNumRes()">getNumRes()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcid.html#getO()">getO()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcid.html" title="interface in org.biojava.bio.structure">AminoAcid</a></dt>
<dd>
<div class="block">get O atom.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcidImpl.html#getO()">getO()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcidImpl.html" title="class in org.biojava.bio.structure">AminoAcidImpl</a></dt>
<dd>
<div class="block">get O atom.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html#getO()">getO()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html" title="class in org.biojava.bio.structure.secstruc">SecStrucGroup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#getOccupancy()">getOccupancy()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">get occupancy.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#getOccupancy()">getOccupancy()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getOccupancy()">getOccupancy()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getOccupancy_esd()">getOccupancy_esd()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getOccupancy_max()">getOccupancy_max()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getOccupancy_min()">getOccupancy_min()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssembly.html#getOligomeric_count()">getOligomeric_count()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssembly.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructAssembly</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssembly.html#getOligomeric_details()">getOligomeric_details()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssembly.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructAssembly</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucState.html#getOmega()">getOmega()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucState.html" title="class in org.biojava.bio.structure.secstruc">SecStrucState</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getOne_letter_code()">getOne_letter_code()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/chem/ChemCompTools.html#getOneLetterCode(org.biojava.bio.structure.io.mmcif.model.ChemComp, org.biojava.bio.structure.io.mmcif.ChemicalComponentDictionary)">getOneLetterCode(ChemComp, ChemicalComponentDictionary)</a></span> - Static method in class org.biojava.bio.structure.io.mmcif.chem.<a href="./org/biojava/bio/structure/io/mmcif/chem/ChemCompTools.html" title="class in org.biojava.bio.structure.io.mmcif.chem">ChemCompTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompGroupFactory.html#getOneLetterCode(org.biojava.bio.structure.io.mmcif.model.ChemComp)">getOneLetterCode(ChemComp)</a></span> - Static method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompGroupFactory.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompGroupFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssemblyGen.html#getOper_expression()">getOper_expression()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssemblyGen.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructAssemblyGen</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getOptAln()">getOptAln()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">Tracks the Atom positions in the optimal alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AlignmentTools.html#getOptAlnAsList(org.biojava.bio.structure.align.model.AFPChain)">getOptAlnAsList(AFPChain)</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AlignmentTools.html" title="class in org.biojava.bio.structure.align.util">AlignmentTools</a></dt>
<dd>
<div class="block">Retrieves the optimum alignment from an AFPChain and returns it as a
 java collection.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getOptLen()">getOptLen()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">The length of each block</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getOptLength()">getOptLength()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getOptRmsd()">getOptRmsd()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/CartesianProduct.html#getOrderedPairs()">getOrderedPairs()</a></span> - Method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/CartesianProduct.html" title="class in org.biojava.bio.structure.quaternary">CartesianProduct</a></dt>
<dd>
<div class="block">Generates the list of ordered pair between two sets.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getOrgan()">getOrgan()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getOrganelle()">getOrganelle()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getOrganismCommon()">getOrganismCommon()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getOrganismScientific()">getOrganismScientific()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getOrganismTaxId()">getOrganismTaxId()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html#getOriginal()">getOriginal()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html" title="class in org.biojava.bio.structure.secstruc">SecStrucGroup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#getORmsdThr()">getORmsdThr()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>
<div class="block">Get the Original RMSD threshold from which the alignment optimization is started</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#getOrthologousSequenceFamilyId()">getOrthologousSequenceFamilyId()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html#getOutFile()">getOutFile()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html" title="class in org.biojava.bio.structure.align">CallableStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#getOutFile()">getOutFile()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getOverall_FOM_free_R_set()">getOverall_FOM_free_R_set()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getOverall_FOM_work_R_set()">getOverall_FOM_work_R_set()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getOverall_SU_B()">getOverall_SU_B()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getOverall_SU_ML()">getOverall_SU_ML()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getOverall_SU_R_Cruickshank_DPI()">getOverall_SU_R_Cruickshank_DPI()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getOverall_SU_R_free()">getOverall_SU_R_free()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#getOxidationState()">getOxidationState()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns a typical oxidation state for this Element.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFP.html#getP1()">getP1()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFP.html" title="class in org.biojava.bio.structure.align.model">AFP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFP.html#getP2()">getP2()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFP.html" title="class in org.biojava.bio.structure.align.model">AFP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html#getPair()">getPair()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html" title="class in org.biojava.bio.structure.align">CallableStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/PdbPairsMessage.html#getPairs()">getPairs()</a></span> - Method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/PdbPairsMessage.html" title="class in org.biojava.bio.structure.align.xml">PdbPairsMessage</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/SigEva.html#getPara(int, int)">getPara(int, int)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/SigEva.html" title="class in org.biojava.bio.structure.align.fatcat.calc">SigEva</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/AbstractStructureAlignment.html#getParameters()">getParameters()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/AbstractStructureAlignment.html" title="class in org.biojava.bio.structure.align">AbstractStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/BioJavaStructureAlignment.html#getParameters()">getParameters()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/BioJavaStructureAlignment.html" title="class in org.biojava.bio.structure.align">BioJavaStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html#getParameters()">getParameters()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce">AbstractUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeMain.html#getParameters()">getParameters()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeMain.html" title="class in org.biojava.bio.structure.align.ce">CeMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeSideChainMain.html#getParameters()">getParameters()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeSideChainMain.html" title="class in org.biojava.bio.structure.align.ce">CeSideChainMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeSideChainUserArgumentProcessor.html#getParameters()">getParameters()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeSideChainUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce">CeSideChainUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeUserArgumentProcessor.html#getParameters()">getParameters()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce">CeUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/OptimalCECPMain.html#getParameters()">getParameters()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/OptimalCECPMain.html" title="class in org.biojava.bio.structure.align.ce">OptimalCECPMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCatFlexible.html#getParameters()">getParameters()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCatFlexible.html" title="class in org.biojava.bio.structure.align.fatcat">FatCatFlexible</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCatRigid.html#getParameters()">getParameters()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCatRigid.html" title="class in org.biojava.bio.structure.align.fatcat">FatCatRigid</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCatUserArgumentProcessor.html#getParameters()">getParameters()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCatUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.fatcat">FatCatUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWaterman3Daligner.html#getParameters()">getParameters()</a></span> - Method in class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWaterman3Daligner.html" title="class in org.biojava.bio.structure.align.seq">SmithWaterman3Daligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWatermanUserArgumentProcessor.html#getParameters()">getParameters()</a></span> - Method in class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWatermanUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.seq">SmithWatermanUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructureAlignment.html#getParameters()">getParameters()</a></span> - Method in interface org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructureAlignment.html" title="interface in org.biojava.bio.structure.align">StructureAlignment</a></dt>
<dd>
<div class="block">Return the paramers for this algorithm.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/FarmJob.html#getParams()">getParams()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/FarmJob.html" title="class in org.biojava.bio.structure.align">FarmJob</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructurePairAligner.html#getParams()">getParams()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructurePairAligner.html" title="class in org.biojava.bio.structure.align">StructurePairAligner</a></dt>
<dd>
<div class="block">get the parameters.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#getParent()">getParent()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>replaced by <a href="./org/biojava/bio/structure/Atom.html#getGroup()"><code>Atom.getGroup()</code></a></i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#getParent()">getParent()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getParent()">getParent()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Returns the parent Structure of this chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getParent()">getParent()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Returns the parent Structure of this chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#getParent()">getParent()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Get the structure object that this DBRef relates to.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#getParent()">getParent()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>replaced by <a href="./org/biojava/bio/structure/Group.html#getChain()"><code>Group.getChain()</code></a></i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#getParent()">getParent()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">Returns the parent Chain of the Group</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html#getParent(org.biojava.bio.structure.io.mmcif.model.ChemComp)">getParent(ChemComp)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html" title="class in org.biojava.bio.structure.io.mmcif">ChemicalComponentDictionary</a></dt>
<dd>
<div class="block">Get the parent of a component.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathNode.html#getParentId()">getParentId()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathNode.html" title="class in org.biojava.bio.structure.cath">CathNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopNode.html#getParentSunid()">getParentSunid()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopNode.html" title="class in org.biojava.bio.structure.scop">ScopNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/HBond.html#getPartner()">getPartner()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/HBond.html" title="class in org.biojava.bio.structure.secstruc">HBond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/Alignable.html#getPath()">getPath()</a></span> - Method in interface org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/Alignable.html" title="interface in org.biojava.bio.structure.align.pairwise">Alignable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#getPath()">getPath()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html#getPath()">getPath()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">StrCompAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#getPath()">getPath()</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>
<div class="block">Get the path that is used to cache PDB files.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/LocalCacheStructureProvider.html#getPath()">getPath()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/LocalCacheStructureProvider.html" title="class in org.biojava.bio.structure.io">LocalCacheStructureProvider</a></dt>
<dd>
<div class="block">Returns the path value.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/MMCIFFileReader.html#getPath()">getPath()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/MMCIFFileReader.html" title="class in org.biojava.bio.structure.io">MMCIFFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#getPath()">getPath()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>
<div class="block">Returns the path value.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/StructureIOFile.html#getPath()">getPath()</a></span> - Method in interface org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/StructureIOFile.html" title="interface in org.biojava.bio.structure.io">StructureIOFile</a></dt>
<dd>
<div class="block">get the directory path to the files</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/Alignable.html#getPathSize()">getPathSize()</a></span> - Method in interface org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/Alignable.html" title="interface in org.biojava.bio.structure.align.pairwise">Alignable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html#getPathSize()">getPathSize()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">StrCompAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#getPaulingElectronegativity()">getPaulingElectronegativity()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns the Pauling electronegativity for this Element.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#getPdb1()">getPdb1()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html#getPdb1()">getPdb1()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentResult</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#getPdb2()">getPdb2()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html#getPdb2()">getPdb2()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentResult</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#getPdb_ins_code()">getPdb_ins_code()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#getPdb_ins_code()">getPdb_ins_code()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#getPdb_mon_id()">getPdb_mon_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#getPdb_mon_id()">getPdb_mon_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#getPdb_seq_num()">getPdb_seq_num()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#getPdb_seq_num()">getPdb_seq_num()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#getPdb_strand_id()">getPdb_strand_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#getPdb_strand_id()">getPdb_strand_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getPdbAln()">getPdbAln()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">used temporarily during XML serialization to track the PDB positions of the alignmnet</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#getPDBCode()">getPDBCode()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>replaced by <a href="./org/biojava/bio/structure/Group.html#getResidueNumber()"><code>Group.getResidueNumber()</code></a></i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#getPDBCode()">getPDBCode()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>replaced by #getSeqNum</i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureEvent.html#getPDBCode()">getPDBCode()</a></span> - Method in interface org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureEvent.html" title="interface in org.biojava.bio.structure.server">StructureEvent</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureEventImpl.html#getPDBCode()">getPDBCode()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureEventImpl.html" title="class in org.biojava.bio.structure.server">StructureEventImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#getPDBCode()">getPDBCode()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">get PDB code of structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#getPDBCode()">getPDBCode()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">get PDB code of structure .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/PdbPair.html#getPDBCode1()">getPDBCode1()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/PdbPair.html" title="class in org.biojava.bio.structure.align.client">PdbPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/PdbPair.html#getPDBCode2()">getPDBCode2()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/PdbPair.html" title="class in org.biojava.bio.structure.align.client">PdbPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#getPdbFilePath()">getPdbFilePath()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#getPdbFilePath()">getPdbFilePath()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/UserConfiguration.html#getPdbFilePath()">getPdbFilePath()</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/UserConfiguration.html" title="class in org.biojava.bio.structure.align.util">UserConfiguration</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/FlatFileInstallation.html#getPDBHeader(java.lang.String)">getPDBHeader(String)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/FlatFileInstallation.html" title="class in org.biojava.bio.structure.server">FlatFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html#getPDBHeader(java.lang.String)">getPDBHeader(String)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html" title="class in org.biojava.bio.structure.server">MMCIFFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/PDBInstallation.html#getPDBHeader(java.lang.String)">getPDBHeader(String)</a></span> - Method in interface org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/PDBInstallation.html" title="interface in org.biojava.bio.structure.server">PDBInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
<div class="block">get the PDB header for a single protein structure</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#getPDBHeader()">getPDBHeader()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">return the header information for this PDB file</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#getPDBHeader()">getPDBHeader()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/StructureName.html#getPdbId()">getPdbId()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/StructureName.html" title="class in org.biojava.bio.structure.align.client">StructureName</a></dt>
<dd>
<div class="block">PDB IDs are always returned as upper case</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#getPdbId()">getPdbId()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html#getPdbId()">getPdbId()</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html" title="class in org.biojava.bio.structure.io.sifts">SiftsResidue</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/MmCifPDBBiolAssemblyProvider.html#getPdbId()">getPdbId()</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/MmCifPDBBiolAssemblyProvider.html" title="class in org.biojava.bio.structure.quaternary.io">MmCifPDBBiolAssemblyProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#getPdbId()">getPdbId()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/FlatFileInstallation.html#getPDBInfoFile()">getPDBInfoFile()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/FlatFileInstallation.html" title="class in org.biojava.bio.structure.server">FlatFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/SimpleStructureServer.html#getPDBInstallation()">getPDBInstallation()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/SimpleStructureServer.html" title="class in org.biojava.bio.structure.server">SimpleStructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureServer.html#getPDBInstallation()">getPDBInstallation()</a></span> - Method in interface org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureServer.html" title="interface in org.biojava.bio.structure.server">StructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#getPDBline()">getPDBline()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>@deprecated replaced by <a href="./org/biojava/bio/structure/PDBRecord.html#toPDB()"><code>PDBRecord.toPDB()</code></a></i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#getPDBline()">getPDBline()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>&nbsp;</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#getPDBName()">getPDBName()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">Get the PDB 3 character name for this group.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#getPDBName()">getPDBName()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">Returns the PDBName.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/JFatCatClient.html#getPdbPairs(java.lang.String, int, java.lang.String)">getPdbPairs(String, int, String)</a></span> - Static method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/JFatCatClient.html" title="class in org.biojava.bio.structure.align.client">JFatCatClient</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#getPDBResidueNumber(org.biojava.bio.structure.Group)">getPDBResidueNumber(Group)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>replaced by  Group.getResidueNumber()</i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html#getPdbResName()">getPdbResName()</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html" title="class in org.biojava.bio.structure.io.sifts">SiftsResidue</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html#getPdbResNum()">getPdbResNum()</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html" title="class in org.biojava.bio.structure.io.sifts">SiftsResidue</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#getPDBresnum1()">getPDBresnum1()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#getPDBresnum2()">getPDBresnum2()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#getPDBserial()">getPDBserial()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">get PDB atom number.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#getPDBserial()">getPDBserial()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>
<div class="block">get PDB atom number.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html#getPdbx_align_begin()">getPdbx_align_begin()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRef</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getPdbx_ambiguous_flag()">getPdbx_ambiguous_flag()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#getPdbx_auth_seq_align_beg()">getPdbx_auth_seq_align_beg()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#getPdbx_auth_seq_align_end()">getPdbx_auth_seq_align_end()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html#getPdbx_blank_PDB_chainid_flag()">getPdbx_blank_PDB_chainid_flag()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructAsym</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html#getPdbx_CASP_flag()">getPdbx_CASP_flag()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html" title="class in org.biojava.bio.structure.io.mmcif.model">Struct</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getPdbx_data_cutoff_high_absF()">getPdbx_data_cutoff_high_absF()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getPdbx_data_cutoff_high_rms_absF()">getPdbx_data_cutoff_high_rms_absF()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getPdbx_data_cutoff_low_absF()">getPdbx_data_cutoff_low_absF()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html#getPdbx_db_accession()">getPdbx_db_accession()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRef</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#getPdbx_db_accession()">getPdbx_db_accession()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#getPdbx_db_align_beg_ins_code()">getPdbx_db_align_beg_ins_code()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#getPdbx_db_align_end_ins_code()">getPdbx_db_align_end_ins_code()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html#getPdbx_description()">getPdbx_description()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model">Entity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html#getPdbx_descriptor()">getPdbx_descriptor()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html" title="class in org.biojava.bio.structure.io.mmcif.model">Struct</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getPdbx_diffrn_id()">getPdbx_diffrn_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html#getPdbx_ec()">getPdbx_ec()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model">Entity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getPdbx_formal_charge()">getPdbx_formal_charge()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getPdbx_formal_charge()">getPdbx_formal_charge()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html#getPdbx_fragment()">getPdbx_fragment()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model">Entity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getPdbx_ideal_coordinates_details()">getPdbx_ideal_coordinates_details()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getPdbx_ideal_coordinates_missing_flag()">getPdbx_ideal_coordinates_missing_flag()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getPdbx_initial_date()">getPdbx_initial_date()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getPdbx_isotropic_thermal_model()">getPdbx_isotropic_thermal_model()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructKeywords.html#getPdbx_keywords()">getPdbx_keywords()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructKeywords.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructKeywords</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getPdbx_ls_cross_valid_method()">getPdbx_ls_cross_valid_method()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getPdbx_ls_sigma_F()">getPdbx_ls_sigma_F()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getPdbx_ls_sigma_I()">getPdbx_ls_sigma_I()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getPdbx_method_to_determine_struct()">getPdbx_method_to_determine_struct()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getPdbx_model_coordinates_db_code()">getPdbx_model_coordinates_db_code()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getPdbx_model_coordinates_details()">getPdbx_model_coordinates_details()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getPdbx_model_coordinates_missing_flag()">getPdbx_model_coordinates_missing_flag()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html#getPdbx_model_details()">getPdbx_model_details()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html" title="class in org.biojava.bio.structure.io.mmcif.model">Struct</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html#getPdbx_model_type_details()">getPdbx_model_type_details()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html" title="class in org.biojava.bio.structure.io.mmcif.model">Struct</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html#getPdbx_modified()">getPdbx_modified()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructAsym</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getPdbx_modified_date()">getPdbx_modified_date()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html#getPdbx_mutation()">getPdbx_mutation()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model">Entity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html#getPdbx_number_of_molecules()">getPdbx_number_of_molecules()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model">Entity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AuditAuthor.html#getPdbx_ordinal()">getPdbx_ordinal()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AuditAuthor.html" title="class in org.biojava.bio.structure.io.mmcif.model">AuditAuthor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getPdbx_overall_ESU_R()">getPdbx_overall_ESU_R()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getPdbx_overall_ESU_R_Free()">getPdbx_overall_ESU_R_Free()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getPdbx_overall_phase_error()">getPdbx_overall_phase_error()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getPdbx_overall_SU_R_Blow_DPI()">getPdbx_overall_SU_R_Blow_DPI()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getPdbx_overall_SU_R_free_Blow_DPI()">getPdbx_overall_SU_R_free_Blow_DPI()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getPdbx_overall_SU_R_free_Cruickshank_DPI()">getPdbx_overall_SU_R_free_Cruickshank_DPI()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#getPdbx_PDB_id_code()">getPdbx_PDB_id_code()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getPdbx_PDB_ins_code()">getPdbx_PDB_ins_code()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getPdbx_PDB_model_num()">getPdbx_PDB_model_num()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getPdbx_processing_site()">getPdbx_processing_site()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getPdbx_R_Free_selection_details()">getPdbx_R_Free_selection_details()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getPdbx_refine_id()">getPdbx_refine_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getPdbx_release_status()">getPdbx_release_status()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getPdbx_replaced_by()">getPdbx_replaced_by()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getPdbx_replaces()">getPdbx_replaces()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#getPdbx_seq_align_beg_ins_code()">getPdbx_seq_align_beg_ins_code()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#getPdbx_seq_align_end_ins_code()">getPdbx_seq_align_end_ins_code()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html#getPdbx_seq_one_letter_code()">getPdbx_seq_one_letter_code()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRef</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getPdbx_solvent_ion_probe_radii()">getPdbx_solvent_ion_probe_radii()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getPdbx_solvent_shrinkage_radii()">getPdbx_solvent_shrinkage_radii()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getPdbx_solvent_vdw_probe_radii()">getPdbx_solvent_vdw_probe_radii()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getPdbx_starting_model()">getPdbx_starting_model()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getPdbx_stereochem_target_val_spec_case()">getPdbx_stereochem_target_val_spec_case()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getPdbx_stereochemistry_target_values()">getPdbx_stereochemistry_target_values()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#getPdbx_strand_id()">getPdbx_strand_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getPdbx_subcomponent_list()">getPdbx_subcomponent_list()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getPdbx_synonyms()">getPdbx_synonyms()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getPdbx_TLS_residual_ADP_flag()">getPdbx_TLS_residual_ADP_flag()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getPdbx_type()">getPdbx_type()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssemblyXMLContainer.html#getPdbxStructAssemblies()">getPdbxStructAssemblies()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssemblyXMLContainer.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructAssemblyXMLContainer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/MmCifPDBBiolAssemblyProvider.html#getPdbxStructAssemblies()">getPdbxStructAssemblies()</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/MmCifPDBBiolAssemblyProvider.html" title="class in org.biojava.bio.structure.quaternary.io">MmCifPDBBiolAssemblyProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/RawBioUnitDataProvider.html#getPdbxStructAssemblies()">getPdbxStructAssemblies()</a></span> - Method in interface org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/RawBioUnitDataProvider.html" title="interface in org.biojava.bio.structure.quaternary.io">RawBioUnitDataProvider</a></dt>
<dd>
<div class="block">Data access method for list describing all assemblies</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/RemoteRawBioUnitDataProvider.html#getPdbxStructAssemblies()">getPdbxStructAssemblies()</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/RemoteRawBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">RemoteRawBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/MmCifPDBBiolAssemblyProvider.html#getPdbxStructAssembly(int)">getPdbxStructAssembly(int)</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/MmCifPDBBiolAssemblyProvider.html" title="class in org.biojava.bio.structure.quaternary.io">MmCifPDBBiolAssemblyProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/RawBioUnitDataProvider.html#getPdbxStructAssembly(int)">getPdbxStructAssembly(int)</a></span> - Method in interface org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/RawBioUnitDataProvider.html" title="interface in org.biojava.bio.structure.quaternary.io">RawBioUnitDataProvider</a></dt>
<dd>
<div class="block">get the data for a particular pdbxStructAssembly.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/RemoteRawBioUnitDataProvider.html#getPdbxStructAssembly(int)">getPdbxStructAssembly(int)</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/RemoteRawBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">RemoteRawBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/MmCifPDBBiolAssemblyProvider.html#getPdbxStructAssemblyGen(int)">getPdbxStructAssemblyGen(int)</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/MmCifPDBBiolAssemblyProvider.html" title="class in org.biojava.bio.structure.quaternary.io">MmCifPDBBiolAssemblyProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/RawBioUnitDataProvider.html#getPdbxStructAssemblyGen(int)">getPdbxStructAssemblyGen(int)</a></span> - Method in interface org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/RawBioUnitDataProvider.html" title="interface in org.biojava.bio.structure.quaternary.io">RawBioUnitDataProvider</a></dt>
<dd>
<div class="block">get the data for a particular pdbxStructAssemblyGen.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/RemoteRawBioUnitDataProvider.html#getPdbxStructAssemblyGen(int)">getPdbxStructAssemblyGen(int)</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/RemoteRawBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">RemoteRawBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssemblyGenXMLContainer.html#getPdbxStructAssemblyGens()">getPdbxStructAssemblyGens()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssemblyGenXMLContainer.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructAssemblyGenXMLContainer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/MmCifPDBBiolAssemblyProvider.html#getPdbxStructAssemblyGens()">getPdbxStructAssemblyGens()</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/MmCifPDBBiolAssemblyProvider.html" title="class in org.biojava.bio.structure.quaternary.io">MmCifPDBBiolAssemblyProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/RawBioUnitDataProvider.html#getPdbxStructAssemblyGens()">getPdbxStructAssemblyGens()</a></span> - Method in interface org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/RawBioUnitDataProvider.html" title="interface in org.biojava.bio.structure.quaternary.io">RawBioUnitDataProvider</a></dt>
<dd>
<div class="block">Data access method for list describing all assemblies</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/RemoteRawBioUnitDataProvider.html#getPdbxStructAssemblyGens()">getPdbxStructAssemblyGens()</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/RemoteRawBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">RemoteRawBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/MmCifPDBBiolAssemblyProvider.html#getPdbxStructOperList()">getPdbxStructOperList()</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/MmCifPDBBiolAssemblyProvider.html" title="class in org.biojava.bio.structure.quaternary.io">MmCifPDBBiolAssemblyProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/RawBioUnitDataProvider.html#getPdbxStructOperList()">getPdbxStructOperList()</a></span> - Method in interface org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/RawBioUnitDataProvider.html" title="interface in org.biojava.bio.structure.quaternary.io">RawBioUnitDataProvider</a></dt>
<dd>
<div class="block">Get all the possible operators</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/RemoteRawBioUnitDataProvider.html#getPdbxStructOperList()">getPdbxStructOperList()</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/RemoteRawBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">RemoteRawBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperListXMLContainer.html#getPdbxStructOperLists()">getPdbxStructOperLists()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperListXMLContainer.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructOperListXMLContainer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/PDPProvider.html#getPDPDomainNamesForPDB(java.lang.String)">getPDPDomainNamesForPDB(String)</a></span> - Method in interface org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/PDPProvider.html" title="interface in org.biojava.bio.structure.domain">PDPProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/RemotePDPProvider.html#getPDPDomainNamesForPDB(java.lang.String)">getPDPDomainNamesForPDB(String)</a></span> - Method in class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/RemotePDPProvider.html" title="class in org.biojava.bio.structure.domain">RemotePDPProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#getPdpprovider()">getPdpprovider()</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#getPercId()">getPercId()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#getPeriod()">getPeriod()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns the period in the periodic table of this Element.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#getPermutationSize()">getPermutationSize()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#getPhi(org.biojava.bio.structure.AminoAcid, org.biojava.bio.structure.AminoAcid)">getPhi(AminoAcid, AminoAcid)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">phi angle.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucState.html#getPhi()">getPhi()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucState.html" title="class in org.biojava.bio.structure.secstruc">SecStrucState</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/LUDecomposition.html#getPivot()">getPivot()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/LUDecomposition.html" title="class in org.biojava.bio.structure.jama">LUDecomposition</a></dt>
<dd>
<div class="block">Return pivot permutation vector</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#getPmid()">getPmid()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>
<div class="block">Get the value of PMID field.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getPolymerType()">getPolymerType()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#getPos1()">getPos1()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#getPos2()">getPos2()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/JFatCatClient.html#getPositionInQueue(java.lang.String, java.lang.String, java.lang.String, java.lang.String)">getPositionInQueue(String, String, String, String)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/JFatCatClient.html" title="class in org.biojava.bio.structure.align.client">JFatCatClient</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/JFatCatClient.html#getPositionInQueue(java.lang.String, java.lang.String, java.lang.String, java.lang.String, int)">getPositionInQueue(String, String, String, String, int)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/JFatCatClient.html" title="class in org.biojava.bio.structure.align.client">JFatCatClient</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStruc.html#getPreciseDistance(org.biojava.bio.structure.Atom, org.biojava.bio.structure.Atom)">getPreciseDistance(Atom, Atom)</a></span> - Static method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStruc.html" title="class in org.biojava.bio.structure.secstruc">SecStruc</a></dt>
<dd>
<div class="block">calculate distance between two atoms.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getProbability()">getProbability()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">The probability (FATCAT) or Z-score (CE) of the alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html#getProgram()">getProgram()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemCompDescriptor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html#getProgram_version()">getProgram_version()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemCompDescriptor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#getProperties()">getProperties()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">return properties.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#getProperties()">getProperties()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">return properties.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#getProperty(java.lang.String)">getProperty(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">get a single property .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#getProperty(java.lang.String)">getProperty(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">get a single property .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/StructureSequenceMatcher.html#getProteinSequenceForStructure(org.biojava.bio.structure.Structure, java.util.Map)">getProteinSequenceForStructure(Structure, Map&lt;Integer, Group&gt;)</a></span> - Static method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/StructureSequenceMatcher.html" title="class in org.biojava.bio.structure.io">StructureSequenceMatcher</a></dt>
<dd>
<div class="block">Generates a ProteinSequence corresponding to the sequence of struct,
 and maintains a mapping from the sequence back to the original groups.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#getPsi(org.biojava.bio.structure.AminoAcid, org.biojava.bio.structure.AminoAcid)">getPsi(AminoAcid, AminoAcid)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">psi angle.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucState.html#getPsi()">getPsi()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucState.html" title="class in org.biojava.bio.structure.secstruc">SecStrucState</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#getPublicationDate()">getPublicationDate()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#getPublisher()">getPublisher()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#getPx()">getPx()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/QRDecomposition.html#getQ()">getQ()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/QRDecomposition.html" title="class in org.biojava.bio.structure.jama">QRDecomposition</a></dt>
<dd>
<div class="block">Generate and return the (economy-sized) orthogonal factor</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/QRDecomposition.html#getR()">getR()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/QRDecomposition.html" title="class in org.biojava.bio.structure.jama">QRDecomposition</a></dt>
<dd>
<div class="block">Return the upper triangular factor</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/JFatCatClient.html#getRandomSleepTime()">getRandomSleepTime()</a></span> - Static method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/JFatCatClient.html" title="class in org.biojava.bio.structure.align.client">JFatCatClient</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html#getRandomUsername()">getRandomUsername()</a></span> - Static method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html" title="class in org.biojava.bio.structure.align.client">FarmJobRunnable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#getRanges()">getRanges()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/EigenvalueDecomposition.html#getRealEigenvalues()">getRealEigenvalues()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/EigenvalueDecomposition.html" title="class in org.biojava.bio.structure.jama">EigenvalueDecomposition</a></dt>
<dd>
<div class="block">Return the real parts of the eigenvalues</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcid.html#getRecordType()">getRecordType()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcid.html" title="interface in org.biojava.bio.structure">AminoAcid</a></dt>
<dd>
<div class="block">Allows to distinguish between amino acids that are provided
                as ATOM records and a SEQRES records.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcidImpl.html#getRecordType()">getRecordType()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcidImpl.html" title="class in org.biojava.bio.structure">AminoAcidImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/BioAssemblyTools.html#getReducedCAStructure(org.biojava.bio.structure.Structure)">getReducedCAStructure(Structure)</a></span> - Static method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/BioAssemblyTools.html" title="class in org.biojava.bio.structure.quaternary">BioAssemblyTools</a></dt>
<dd>
<div class="block">reduce a structure to a Calpha representation only</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#getReducedStructure(org.biojava.bio.structure.Structure, java.lang.String)">getReducedStructure(Structure, String)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>
<div class="block">Reduce a structure to provide a smaller representation .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#getReducedStructure(org.biojava.bio.structure.Structure, int)">getReducedStructure(Structure, int)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>
<div class="block">Reduce a structure to provide a smaller representation.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#getRef()">getRef()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>
<div class="block">Get the value of REF field.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#getRef_id()">getRef_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getRefChainId()">getRefChainId()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>
<div class="block">Returns the ref chain id value.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#getRefn()">getRefn()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>
<div class="block">Get the value of REFN field.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html#getReplaced(org.biojava.bio.structure.io.mmcif.model.ChemComp)">getReplaced(ChemComp)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html" title="class in org.biojava.bio.structure.io.mmcif">ChemicalComponentDictionary</a></dt>
<dd>
<div class="block">if ChemComp is replacing another one, get the old version
 otherwise return the same ChemComp again.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html#getReplaced(java.lang.String)">getReplaced(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html" title="class in org.biojava.bio.structure.io.mmcif">ChemicalComponentDictionary</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBStatus.html#getReplacement(java.lang.String, boolean, boolean)">getReplacement(String, boolean, boolean)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBStatus.html" title="class in org.biojava.bio.structure">PDBStatus</a></dt>
<dd>
<div class="block">Gets the PDB which superseded oldPdbId.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html#getReplacer(org.biojava.bio.structure.io.mmcif.model.ChemComp)">getReplacer(ChemComp)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html" title="class in org.biojava.bio.structure.io.mmcif">ChemicalComponentDictionary</a></dt>
<dd>
<div class="block">if ChemComp is replaced by another one, get the newer version
 otherwise return the same ChemComp again.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html#getReplacer(java.lang.String)">getReplacer(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html" title="class in org.biojava.bio.structure.io.mmcif">ChemicalComponentDictionary</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html#getReplaces()">getReplaces()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html" title="class in org.biojava.bio.structure.io.mmcif.model">DatabasePDBrev</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBStatus.html#getReplaces(java.lang.String, boolean)">getReplaces(String, boolean)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBStatus.html" title="class in org.biojava.bio.structure">PDBStatus</a></dt>
<dd>
<div class="block">Get the ID of the protein which was made obsolete by newPdbId.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathNode.html#getRepresentative()">getRepresentative()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathNode.html" title="class in org.biojava.bio.structure.cath">CathNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/DomainProvider.html#getRepresentativeDomains()">getRepresentativeDomains()</a></span> - Method in interface org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/DomainProvider.html" title="interface in org.biojava.bio.structure.domain">DomainProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/PDBDomainProvider.html#getRepresentativeDomains()">getRepresentativeDomains()</a></span> - Method in class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/PDBDomainProvider.html" title="class in org.biojava.bio.structure.domain">PDBDomainProvider</a></dt>
<dd>
<div class="block">Gets a list of all domain representatives</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/RemoteDomainProvider.html#getRepresentativeDomains()">getRepresentativeDomains()</a></span> - Method in class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/RemoteDomainProvider.html" title="class in org.biojava.bio.structure.domain">RemoteDomainProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/JFatCatClient.html#getRepresentatives(java.lang.String, int)">getRepresentatives(String, int)</a></span> - Static method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/JFatCatClient.html" title="class in org.biojava.bio.structure.align.client">JFatCatClient</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#getResidueNumber()">getResidueNumber()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">returns a dynamically created ResidueNumber for the group - this
 contains the chainId, resNum and insCode of the group.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#getResidueNumber()">getResidueNumber()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">returns a dynamically created ResidueNumber for the group - this
 contains the chainId, resNum and insCode of the group.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FastaStructureParser.html#getResidues()">getResidues()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FastaStructureParser.html" title="class in org.biojava.bio.structure.io">FastaStructureParser</a></dt>
<dd>
<div class="block">For each residue in the fasta file, return the ResidueNumber in the
 corresponding structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsSegment.html#getResidues()">getResidues()</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsSegment.html" title="class in org.biojava.bio.structure.io.sifts">SiftsSegment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getResidueType()">getResidueType()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/chem/ResidueType.html#getResidueTypeFromString(java.lang.String)">getResidueTypeFromString(String)</a></span> - Static method in enum org.biojava.bio.structure.io.mmcif.chem.<a href="./org/biojava/bio/structure/io/mmcif/chem/ResidueType.html" title="enum in org.biojava.bio.structure.io.mmcif.chem">ResidueType</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getResNames()">getResNames()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SSBond.html#getResnum1()">getResnum1()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SSBond.html" title="class in org.biojava.bio.structure">SSBond</a></dt>
<dd>
<div class="block">get residue number for first CYS.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SSBond.html#getResnum2()">getResnum2()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SSBond.html" title="class in org.biojava.bio.structure">SSBond</a></dt>
<dd>
<div class="block">get residue number for second CYS.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#getResolution()">getResolution()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#getResolution()">getResolution()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/ResourceManager.html#getResourceManager(java.lang.String)">getResourceManager(String)</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/ResourceManager.html" title="class in org.biojava.bio.structure.align.util">ResourceManager</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getResScore()">getResScore()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/PdbPair.html#getReverse()">getReverse()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/PdbPair.html" title="class in org.biojava.bio.structure.align.client">PdbPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/JointFragments.html#getRms()">getRms()</a></span> - Method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/JointFragments.html" title="class in org.biojava.bio.structure.align.helper">JointFragments</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentJoiner.html#getRMS(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], org.biojava.bio.structure.align.helper.JointFragments)">getRMS(Atom[], Atom[], JointFragments)</a></span> - Static method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentJoiner.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentJoiner</a></dt>
<dd>
<div class="block">get the RMS of the JointFragments pair frag</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#getRms()">getRms()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SVDSuperimposer.html#getRMS(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">getRMS(Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SVDSuperimposer.html" title="class in org.biojava.bio.structure">SVDSuperimposer</a></dt>
<dd>
<div class="block">Calculate the RMS (root mean square) deviation of two sets of atoms.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFP.html#getRmsd()">getRmsd()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFP.html" title="class in org.biojava.bio.structure.align.model">AFP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#getRmsd()">getRmsd()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getRmsdCut()">getRmsdCut()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>
<div class="block">The cutoff to be used during AFP detection</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#getRmsdThr()">getRmsdThr()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>
<div class="block">RMSD Threshold</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#getRmsdThrJoin()">getRmsdThrJoin()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>
<div class="block">RMSD threshold for joining of AFPs</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#getRot()">getRot()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SVDSuperimposer.html#getRotation()">getRotation()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SVDSuperimposer.html" title="class in org.biojava.bio.structure">SVDSuperimposer</a></dt>
<dd>
<div class="block">Get the Rotation matrix that is required to superimpose the two atom sets.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#getRotationMatrix()">getRotationMatrix()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>
<div class="block">Gets the rotation matrix from the last call to 
 <a href="./org/biojava/bio/structure/align/ce/CECalculator.html#calc_rmsd(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], int, boolean, boolean)"><code>calc_rmsd</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#getRotationMatrix()">getRotationMatrix()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>
<div class="block">Gets the rotation matrix from the last call to 
 <a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#calc_rmsd(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], int, boolean, boolean)"><code>calc_rmsd</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#getRotationMatrix()">getRotationMatrix()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>
<div class="block">returns the rotation matrix that needs to be applied to structure 2 to rotate on structure 1</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html#getRotMax(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">getRotMax(AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html" title="class in org.biojava.bio.structure.align.util">AFPAlignmentDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/IndexPair.html#getRow()">getRow()</a></span> - Method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/IndexPair.html" title="class in org.biojava.bio.structure.align.helper">IndexPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#getRowDimension()">getRowDimension()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Get row dimension.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#getRowPackedCopy()">getRowPackedCopy()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Make a one-dimensional row packed copy of the internal array.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/Alignable.html#getRows()">getRows()</a></span> - Method in interface org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/Alignable.html" title="interface in org.biojava.bio.structure.align.pairwise">Alignable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html#getRows()">getRows()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">StrCompAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/SingularValueDecomposition.html#getS()">getS()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/SingularValueDecomposition.html" title="class in org.biojava.bio.structure.jama">SingularValueDecomposition</a></dt>
<dd>
<div class="block">Return the diagonal matrix of singular values</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#getSaveOutputDir()">getSaveOutputDir()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/BigSqrt.html#getScale()">getScale()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/BigSqrt.html" title="class in org.biojava.bio.structure.secstruc">BigSqrt</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/GuiWrapper.html#getScaleableMatrixPanel(org.biojava.bio.structure.jama.Matrix)">getScaleableMatrixPanel(Matrix)</a></span> - Static method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/GuiWrapper.html" title="class in org.biojava.bio.structure.align.ce">GuiWrapper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopFactory.html#getSCOP()">getSCOP()</a></span> - Static method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopFactory.html" title="class in org.biojava.bio.structure.scop">ScopFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/ScopDescriptions.html#getScopDescription()">getScopDescription()</a></span> - Method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/ScopDescriptions.html" title="class in org.biojava.bio.structure.scop.server">ScopDescriptions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/CachedRemoteScopInstallation.html#getScopDescriptionBySunid(int)">getScopDescriptionBySunid(int)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/CachedRemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">CachedRemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html#getScopDescriptionBySunid(int)">getScopDescriptionBySunid(int)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">RemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDatabase.html#getScopDescriptionBySunid(int)">getScopDescriptionBySunid(int)</a></span> - Method in interface org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDatabase.html" title="interface in org.biojava.bio.structure.scop">ScopDatabase</a></dt>
<dd>
<div class="block">Return the SCOP description for a node in the hierarchy by its "sunid" id.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#getScopDescriptionBySunid(int)">getScopDescriptionBySunid(int)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/XMLUtil.html#getScopDescriptionFromXML(java.lang.String)">getScopDescriptionFromXML(String)</a></span> - Static method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/XMLUtil.html" title="class in org.biojava.bio.structure.scop.server">XMLUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/XMLUtil.html#getScopDescriptionsXML(java.util.List)">getScopDescriptionsXML(List&lt;ScopDescription&gt;)</a></span> - Static method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/XMLUtil.html" title="class in org.biojava.bio.structure.scop.server">XMLUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/XMLUtil.html#getScopDescriptionXML(org.biojava.bio.structure.scop.ScopDescription)">getScopDescriptionXML(ScopDescription)</a></span> - Static method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/XMLUtil.html" title="class in org.biojava.bio.structure.scop.server">XMLUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/ScopDomains.html#getScopDomain()">getScopDomain()</a></span> - Method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/ScopDomains.html" title="class in org.biojava.bio.structure.scop.server">ScopDomains</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/XMLUtil.html#getScopDomainFromXML(java.lang.String)">getScopDomainFromXML(String)</a></span> - Static method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/XMLUtil.html" title="class in org.biojava.bio.structure.scop.server">XMLUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/CachedRemoteScopInstallation.html#getScopDomainsBySunid(java.lang.Integer)">getScopDomainsBySunid(Integer)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/CachedRemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">CachedRemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html#getScopDomainsBySunid(java.lang.Integer)">getScopDomainsBySunid(Integer)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">RemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDatabase.html#getScopDomainsBySunid(java.lang.Integer)">getScopDomainsBySunid(Integer)</a></span> - Method in interface org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDatabase.html" title="interface in org.biojava.bio.structure.scop">ScopDatabase</a></dt>
<dd>
<div class="block">Get a SCOP domain by its sunid</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#getScopDomainsBySunid(java.lang.Integer)">getScopDomainsBySunid(Integer)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/XMLUtil.html#getScopDomainsXML(java.util.List)">getScopDomainsXML(List&lt;ScopDomain&gt;)</a></span> - Static method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/XMLUtil.html" title="class in org.biojava.bio.structure.scop.server">XMLUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/XMLUtil.html#getScopDomainXML(org.biojava.bio.structure.scop.ScopDomain)">getScopDomainXML(ScopDomain)</a></span> - Static method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/XMLUtil.html" title="class in org.biojava.bio.structure.scop.server">XMLUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#getScopDownloadURL()">getScopDownloadURL()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#getScopId()">getScopId()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#getScopInstallation()">getScopInstallation()</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/CachedRemoteScopInstallation.html#getScopNode(int)">getScopNode(int)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/CachedRemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">CachedRemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html#getScopNode(int)">getScopNode(int)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">RemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDatabase.html#getScopNode(int)">getScopNode(int)</a></span> - Method in interface org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDatabase.html" title="interface in org.biojava.bio.structure.scop">ScopDatabase</a></dt>
<dd>
<div class="block">Access a particular ScopNode.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#getScopNode(int)">getScopNode(int)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/ScopNodes.html#getScopNode()">getScopNode()</a></span> - Method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/ScopNodes.html" title="class in org.biojava.bio.structure.scop.server">ScopNodes</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/XMLUtil.html#getScopNodeFromXML(java.lang.String)">getScopNodeFromXML(String)</a></span> - Static method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/XMLUtil.html" title="class in org.biojava.bio.structure.scop.server">XMLUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/XMLUtil.html#getScopNodesXML(java.util.List)">getScopNodesXML(List&lt;ScopNode&gt;)</a></span> - Static method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/XMLUtil.html" title="class in org.biojava.bio.structure.scop.server">XMLUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/XMLUtil.html#getScopNodeXML(org.biojava.bio.structure.scop.ScopNode)">getScopNodeXML(ScopNode)</a></span> - Static method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/XMLUtil.html" title="class in org.biojava.bio.structure.scop.server">XMLUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/CachedRemoteScopInstallation.html#getScopVersion()">getScopVersion()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/CachedRemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">CachedRemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html#getScopVersion()">getScopVersion()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">RemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDatabase.html#getScopVersion()">getScopVersion()</a></span> - Method in interface org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDatabase.html" title="interface in org.biojava.bio.structure.scop">ScopDatabase</a></dt>
<dd>
<div class="block">Returns the SCOP version</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#getScopVersion()">getScopVersion()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFP.html#getScore()">getScore()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFP.html" title="class in org.biojava.bio.structure.align.model">AFP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/Alignable.html#getScore()">getScore()</a></span> - Method in interface org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/Alignable.html" title="interface in org.biojava.bio.structure.align.pairwise">Alignable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#getScore()">getScore()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>
<div class="block">the alignment score</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html#getScore()">getScore()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">StrCompAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Domain.html#getScore()">getScore()</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Domain.html" title="class in org.biojava.bio.structure.domain.pdp">Domain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Segment.html#getScore()">getScore()</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Segment.html" title="class in org.biojava.bio.structure.domain.pdp">Segment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#getScore2(int, int, double[][], int, int[], int, int, double, double)">getScore2(int, int, double[][], int, int[], int, int, double, double)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#getScore2(int, int, double[][], int, int[], int, int, double, double)">getScore2(int, int, double[][], int, int[], int, int, double, double)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#getScoreFromDistanceMatrices(int, int, int)">getScoreFromDistanceMatrices(int, int, int)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#getScoreFromDistanceMatrices(int, int, int)">getScoreFromDistanceMatrices(int, int, int)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#getScoringStrategy()">getScoringStrategy()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getSecretion()">getSecretion()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcid.html#getSecStruc()">getSecStruc()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcid.html" title="interface in org.biojava.bio.structure">AminoAcid</a></dt>
<dd>
<div class="block">get secondary structure data .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcidImpl.html#getSecStruc()">getSecStruc()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcidImpl.html" title="class in org.biojava.bio.structure">AminoAcidImpl</a></dt>
<dd>
<div class="block">get secondary structure data .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucState.html#getSecStruc()">getSecStruc()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucState.html" title="class in org.biojava.bio.structure.secstruc">SecStrucState</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#getSeedFragmentLength()">getSeedFragmentLength()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#getSeedRmsdCutoff()">getSeedRmsdCutoff()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsSegment.html#getSegId()">getSegId()</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsSegment.html" title="class in org.biojava.bio.structure.io.sifts">SiftsSegment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Domain.html#getSegmentAtPos(int)">getSegmentAtPos(int)</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Domain.html" title="class in org.biojava.bio.structure.domain.pdp">Domain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathSegment.html#getSegmentId()">getSegmentId()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathSegment.html" title="class in org.biojava.bio.structure.cath">CathSegment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#getSegments()">getSegments()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Domain.html#getSegments()">getSegments()</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Domain.html" title="class in org.biojava.bio.structure.domain.pdp">Domain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsEntity.html#getSegments()">getSegments()</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsEntity.html" title="class in org.biojava.bio.structure.io.sifts">SiftsEntity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#getSeq_align_beg()">getSeq_align_beg()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#getSeq_align_end()">getSeq_align_end()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#getSeq_id()">getSeq_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#getSeq_id()">getSeq_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#getSeqBegin()">getSeqBegin()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Initial sequence number of the PDB sequence segment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#getSeqEnd()">getSeqEnd()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Ending sequence number   of the PDB sequence segment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ResidueNumber.html#getSeqNum()">getSeqNum()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ResidueNumber.html" title="class in org.biojava.bio.structure">ResidueNumber</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getSeqResGroup(int)">getSeqResGroup(int)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">return the Group at position X.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getSeqResGroup(int)">getSeqResGroup(int)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">return the Group at position X.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getSeqResGroups(java.lang.String)">getSeqResGroups(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Return a List of all groups of a special type (e.g.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getSeqResGroups()">getSeqResGroups()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Return all groups of this chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getSeqResGroups(java.lang.String)">getSeqResGroups(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Return a List of all groups of a special type (e.g.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getSeqResGroups()">getSeqResGroups()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Return all groups of this chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getSeqResLength()">getSeqResLength()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Returns the number of groups in the SEQRES records of the chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getSeqResLength()">getSeqResLength()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html#getSeqResName()">getSeqResName()</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html" title="class in org.biojava.bio.structure.io.sifts">SiftsResidue</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getSeqResSequence()">getSeqResSequence()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Get the sequence for all amino acids as it is specified in the SEQRES residues.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getSeqResSequence()">getSeqResSequence()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Returns the PDB SEQRES sequence as a one-letter sequence string.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#getSequence()">getSequence()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathSegment.html#getSequence()">getSequence()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathSegment.html" title="class in org.biojava.bio.structure.cath">CathSegment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getSequence()">getSequence()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>use getAtomSequence instead</i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getSequence()">getSequence()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>use getAtomSequence instead</i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#getSequenceFamilyId()">getSequenceFamilyId()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#getSequenceHeader()">getSequenceHeader()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathSegment.html#getSequenceHeader()">getSequenceHeader()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathSegment.html" title="class in org.biojava.bio.structure.cath">CathSegment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FastaStructureParser.html#getSequences()">getSequences()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FastaStructureParser.html" title="class in org.biojava.bio.structure.io">FastaStructureParser</a></dt>
<dd>
<div class="block">Gets the protein sequences read from the Fasta file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#getSeqWeight()">getSeqWeight()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>
<div class="block">Should sequence conservation be considered as part of the alignment? If yes, this weight factor allows to determine how much.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SSBond.html#getSerNum()">getSerNum()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SSBond.html" title="class in org.biojava.bio.structure">SSBond</a></dt>
<dd>
<div class="block">set serial number of this SSBOND in PDB file</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#getServer()">getServer()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/RemotePDPProvider.html#getServer()">getServer()</a></span> - Method in class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/RemotePDPProvider.html" title="class in org.biojava.bio.structure.domain">RemotePDPProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html#getServer()">getServer()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">RemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#getShift()">getShift()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>
<div class="block">Gets the shift from the last call to 
 <a href="./org/biojava/bio/structure/align/ce/CECalculator.html#calc_rmsd(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], int, boolean, boolean)"><code>calc_rmsd</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#getShift()">getShift()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>
<div class="block">Gets the shift from the last call to 
 <a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#calc_rmsd(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], int, boolean, boolean)"><code>calc_rmsd</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#getShift()">getShift()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>
<div class="block">returns the shift vector that has to be applied on structure to to shift on structure one</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#getShowDBresult()">getShowDBresult()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsMappingProvider.html#getSiftsMapping(java.lang.String)">getSiftsMapping(String)</a></span> - Static method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsMappingProvider.html" title="class in org.biojava.bio.structure.io.sifts">SiftsMappingProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getSimilarity()">getSimilarity()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">Returns the similarity score for the alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getSimilarity1()">getSimilarity1()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>use getCoverage1() instead</i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getSimilarity2()">getSimilarity2()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>use getCoverage2() instead</i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/SingularValueDecomposition.html#getSingularValues()">getSingularValues()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/SingularValueDecomposition.html" title="class in org.biojava.bio.structure.jama">SingularValueDecomposition</a></dt>
<dd>
<div class="block">Return the one-dimensional array of singular values</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Site.html#getSiteID()">getSiteID()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Site.html" title="class in org.biojava.bio.structure">Site</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#getSites()">getSites()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#getSites()">getSites()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Domain.html#getSize()">getSize()</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Domain.html" title="class in org.biojava.bio.structure.domain.pdp">Domain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#getSOILD()">getSOILD()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getSolvent_model_details()">getSolvent_model_details()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getSolvent_model_param_bsol()">getSolvent_model_param_bsol()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getSolvent_model_param_ksol()">getSolvent_model_param_ksol()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#getSource()">getSource()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBCrystallographicInfo.html#getSpaceGroup()">getSpaceGroup()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBCrystallographicInfo.html" title="class in org.biojava.bio.structure">PDBCrystallographicInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getSparse()">getSparse()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#getSpeciesId()">getSpeciesId()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html#getSrc_method()">getSrc_method()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model">Entity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#getSSBonds()">getSSBonds()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">get the list of SSBonds as they have been defined in the PDB files</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#getSSBonds()">getSSBonds()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">get the list of SSBonds as they have been defined in the PDB files</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathFragment.html#getStart()">getStart()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathFragment.html" title="class in org.biojava.bio.structure.cath">CathFragment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathSegment.html#getStart()">getStart()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathSegment.html" title="class in org.biojava.bio.structure.cath">CathSegment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsSegment.html#getStart()">getStart()</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsSegment.html" title="class in org.biojava.bio.structure.io.sifts">SiftsSegment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#getStartPage()">getStartPage()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html#getStatus()">getStatus()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html" title="class in org.biojava.bio.structure.io.mmcif.model">DatabasePDBrev</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBStatus.html#getStatus(java.lang.String)">getStatus(String)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBStatus.html" title="class in org.biojava.bio.structure">PDBStatus</a></dt>
<dd>
<div class="block">Get the status of the PDB in question.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBStatus.html#getStatus(java.lang.String[])">getStatus(String[])</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBStatus.html" title="class in org.biojava.bio.structure">PDBStatus</a></dt>
<dd>
<div class="block">Get the status of the a collection of PDBs in question in a single query.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#getStepSize()">getStepSize()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>
<div class="block">how many pairs should be requested for alignment from server?</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathFragment.html#getStop()">getStop()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathFragment.html" title="class in org.biojava.bio.structure.cath">CathFragment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathSegment.html#getStop()">getStop()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathSegment.html" title="class in org.biojava.bio.structure.cath">CathSegment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileParsingParameters.html#getStoreEmptySeqRes()">getStoreEmptySeqRes()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileParsingParameters.html" title="class in org.biojava.bio.structure.io">FileParsingParameters</a></dt>
<dd>
<div class="block">a flag to detrermine if SEQRES should be stored, even if alignSeqREs is disabled.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getStrain()">getStrain()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/ResourceManager.html#getString(java.lang.String)">getString(String)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/ResourceManager.html" title="class in org.biojava.bio.structure.align.util">ResourceManager</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#getStructAssemblies()">getStructAssemblies()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#getStructAssemblyGens()">getStructAssemblyGens()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#getStructOpers()">getStructOpers()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#getStructure(java.lang.String)">getStructure(String)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>
<div class="block">Request a Structure based on a <i>name</i>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#getStructure()">getStructure()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>
<div class="block">This method will return the parsed protein structure, once the parsing has been finished</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/MMCIFFileReader.html#getStructure(java.lang.String)">getStructure(String)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/MMCIFFileReader.html" title="class in org.biojava.bio.structure.io">MMCIFFileReader</a></dt>
<dd>
<div class="block">Opens filename, parses it and returns
 a Structure object .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/MMCIFFileReader.html#getStructure(java.io.File)">getStructure(File)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/MMCIFFileReader.html" title="class in org.biojava.bio.structure.io">MMCIFFileReader</a></dt>
<dd>
<div class="block">Opens filename, parses it and returns a Structure object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#getStructure(java.lang.String)">getStructure(String)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>
<div class="block">opens filename, parses it and returns
 aStructure object .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#getStructure(java.io.File)">getStructure(File)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>
<div class="block">opens filename, parses it and returns a Structure object</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#getStructure(java.net.URL)">getStructure(URL)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/StructureIOFile.html#getStructure(java.lang.String)">getStructure(String)</a></span> - Method in interface org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/StructureIOFile.html" title="interface in org.biojava.bio.structure.io">StructureIOFile</a></dt>
<dd>
<div class="block">open filename and returns
 a Structure object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/StructureIOFile.html#getStructure(java.io.File)">getStructure(File)</a></span> - Method in interface org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/StructureIOFile.html" title="interface in org.biojava.bio.structure.io">StructureIOFile</a></dt>
<dd>
<div class="block">read file from File and returns
 a Structure object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/FlatFileInstallation.html#getStructure(java.lang.String)">getStructure(String)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/FlatFileInstallation.html" title="class in org.biojava.bio.structure.server">FlatFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html#getStructure(java.lang.String)">getStructure(String)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html" title="class in org.biojava.bio.structure.server">MMCIFFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/PDBInstallation.html#getStructure(java.lang.String)">getStructure(String)</a></span> - Method in interface org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/PDBInstallation.html" title="interface in org.biojava.bio.structure.server">PDBInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
<div class="block">request a structure by its PDB identifier</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureEvent.html#getStructure()">getStructure()</a></span> - Method in interface org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureEvent.html" title="interface in org.biojava.bio.structure.server">StructureEvent</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureEventImpl.html#getStructure()">getStructure()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureEventImpl.html" title="class in org.biojava.bio.structure.server">StructureEventImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/structure/StructureIO.html#getStructure(java.lang.String)">getStructure(String)</a></span> - Static method in class org.biojava3.structure.<a href="./org/biojava3/structure/StructureIO.html" title="class in org.biojava3.structure">StructureIO</a></dt>
<dd>
<div class="block">Loads a structure based on a name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/LocalCacheStructureProvider.html#getStructureById(java.lang.String)">getStructureById(String)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/LocalCacheStructureProvider.html" title="class in org.biojava.bio.structure.io">LocalCacheStructureProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/MMCIFFileReader.html#getStructureById(java.lang.String)">getStructureById(String)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/MMCIFFileReader.html" title="class in org.biojava.bio.structure.io">MMCIFFileReader</a></dt>
<dd>
<div class="block">Get a structure by PDB code.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#getStructureById(java.lang.String)">getStructureById(String)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>
<div class="block">load a structure from local file system and return a PDBStructure object</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBSRSReader.html#getStructureById(java.lang.String)">getStructureById(String)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBSRSReader.html" title="class in org.biojava.bio.structure.io">PDBSRSReader</a></dt>
<dd>
<div class="block">load a structure from from SRS installation using wgetz
 returns null if no structure found</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/SandboxStyleStructureProvider.html#getStructureById(java.lang.String)">getStructureById(String)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/SandboxStyleStructureProvider.html" title="class in org.biojava.bio.structure.io">SandboxStyleStructureProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/StructureIO.html#getStructureById(java.lang.String)">getStructureById(String)</a></span> - Method in interface org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/StructureIO.html" title="interface in org.biojava.bio.structure.io">StructureIO</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
<div class="block">Get a structure by providing a PDB code.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/StructureIOFile.html#getStructureById(java.lang.String)">getStructureById(String)</a></span> - Method in interface org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/StructureIOFile.html" title="interface in org.biojava.bio.structure.io">StructureIOFile</a></dt>
<dd>
<div class="block">Get a Structure based on its PDB id.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/StructureProvider.html#getStructureById(java.lang.String)">getStructureById(String)</a></span> - Method in interface org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/StructureProvider.html" title="interface in org.biojava.bio.structure.io">StructureProvider</a></dt>
<dd>
<div class="block">get the structure for a PDB ID</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#getStructureForDomain(org.biojava.bio.structure.scop.ScopDomain)">getStructureForDomain(ScopDomain)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>
<div class="block">Returns the representation of a ScopDomain as a BioJava Structure object</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureFetcherRunnable.html#getStructureListeners()">getStructureListeners()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureFetcherRunnable.html" title="class in org.biojava.bio.structure.server">StructureFetcherRunnable</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FastaStructureParser.html#getStructures()">getStructures()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FastaStructureParser.html" title="class in org.biojava.bio.structure.io">FastaStructureParser</a></dt>
<dd>
<div class="block">Gets the protein structures mapped from the Fasta file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#getSubRanges(org.biojava.bio.structure.Structure, java.lang.String)">getSubRanges(Structure, String)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>
<div class="block">In addition to the functionality provided by getReducedStructure also provides a way to specify sub-regions of a structure with the following 
 specification:
 
 
 ranges can be surrounded by ( and ).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#getSubstitutionMatrix()">getSubstitutionMatrix()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>
<div class="block">Sets the  substitution matrix to be used for influencing the alignment with sequence conservation information.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDescription.html#getSunID()">getSunID()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDescription.html" title="class in org.biojava.bio.structure.scop">ScopDescription</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#getSunid()">getSunid()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopNode.html#getSunid()">getSunid()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopNode.html" title="class in org.biojava.bio.structure.scop">ScopNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#getSuperfamilyId()">getSuperfamilyId()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Author.html#getSurname()">getSurname()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Author.html" title="class in org.biojava.bio.structure">Author</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getSwissprotId()">getSwissprotId()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Get the Swissprot id of this chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getSwissprotId()">getSwissprotId()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">get the Swissprot id of this chains .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AlignmentTools.html#getSymmetryOrder(java.util.Map, int, float)">getSymmetryOrder(Map&lt;Integer, Integer&gt;, int, float)</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AlignmentTools.html" title="class in org.biojava.bio.structure.align.util">AlignmentTools</a></dt>
<dd>
<div class="block">Helper for <a href="./org/biojava/bio/structure/align/util/AlignmentTools.html#getSymmetryOrder(java.util.Map, java.util.Map, int, float)"><code>AlignmentTools.getSymmetryOrder(Map, Map, int, float)</code></a> with a true
 identity function (X->X).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AlignmentTools.html#getSymmetryOrder(java.util.Map, java.util.Map, int, float)">getSymmetryOrder(Map&lt;Integer, Integer&gt;, Map&lt;Integer, Integer&gt;, int, float)</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AlignmentTools.html" title="class in org.biojava.bio.structure.align.util">AlignmentTools</a></dt>
<dd>
<div class="block">Tries to detect symmetry in an alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AlignmentTools.html#getSymmetryOrder(org.biojava.bio.structure.align.model.AFPChain, int, float)">getSymmetryOrder(AFPChain, int, float)</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AlignmentTools.html" title="class in org.biojava.bio.structure.align.util">AlignmentTools</a></dt>
<dd>
<div class="block">Guesses the order of symmetry in an alignment</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html#getSymmetryShorthand()">getSymmetryShorthand()</a></span> - Method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html" title="class in org.biojava.bio.structure.quaternary">ModelTransformationMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getSynonyms()">getSynonyms()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getSynthetic()">getSynthetic()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFP.html#getT()">getT()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFP.html" title="class in org.biojava.bio.structure.align.model">AFP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#getTechnique()">getTechnique()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#getTempFactor()">getTempFactor()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">get set temp factor.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#getTempFactor()">getTempFactor()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBremark.html#getText()">getText()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBremark.html" title="class in org.biojava.bio.structure.io.mmcif.model">DatabasePDBremark</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructKeywords.html#getText()">getText()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructKeywords.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructKeywords</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#getThreads()">getThreads()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getThree_letter_code()">getThree_letter_code()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucState.html#getThreeState()">getThreeState()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucState.html" title="class in org.biojava.bio.structure.secstruc">SecStrucState</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#getTime()">getTime()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/JFatCatClient.html#getTimeout()">getTimeout()</a></span> - Static method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/JFatCatClient.html" title="class in org.biojava.bio.structure.align.client">JFatCatClient</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/RemoteRawBioUnitDataProvider.html#getTimeout()">getTimeout()</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/RemoteRawBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">RemoteRawBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileParser.html#getTimeStamp()">getTimeStamp()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileParser.html" title="class in org.biojava.bio.structure.io">PDBFileParser</a></dt>
<dd>
<div class="block">Returns a time stamp.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getTissue()">getTissue()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getTitle()">getTitle()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html#getTitle()">getTitle()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html" title="class in org.biojava.bio.structure.io.mmcif.model">Struct</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#getTitle()">getTitle()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>
<div class="block">Get the value of title</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#getTitle()">getTitle()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getTMScore()">getTMScore()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">Returns the tmScore of the alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AFPChainScorer.html#getTMScore(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">getTMScore(AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AFPChainScorer.html" title="class in org.biojava.bio.structure.align.util">AFPChainScorer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SVDSuperimposer.html#getTMScore(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], int, int)">getTMScore(Atom[], Atom[], int, int)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SVDSuperimposer.html" title="class in org.biojava.bio.structure">SVDSuperimposer</a></dt>
<dd>
<div class="block">Calculate the TM-Score for the superposition.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Segment.html#getTo()">getTo()</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Segment.html" title="class in org.biojava.bio.structure.domain.pdp">Segment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/Ladder.html#getTo()">getTo()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/Ladder.html" title="class in org.biojava.bio.structure.secstruc">Ladder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#getTopologyId()">getTopologyId()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getTorsionPenalty()">getTorsionPenalty()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getTotalLenIni()">getTotalLenIni()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getTotalLenOpt()">getTotalLenOpt()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getTotalRmsdIni()">getTotalRmsdIni()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">this is the init-RMSD, not the final RMSD after refinement.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getTotalRmsdOpt()">getTotalRmsdOpt()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">The RMSD of the final alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/BigSqrt.html#getTraceFlag()">getTraceFlag()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/BigSqrt.html" title="class in org.biojava.bio.structure.secstruc">BigSqrt</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#getTrans()">getTrans()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBBioAssemblyParser.html#getTransformationMap()">getTransformationMap()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBBioAssemblyParser.html" title="class in org.biojava.bio.structure.io">PDBBioAssemblyParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html#getTranslation(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">getTranslation(AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html" title="class in org.biojava.bio.structure.align.util">AFPAlignmentDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SVDSuperimposer.html#getTranslation()">getTranslation()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SVDSuperimposer.html" title="class in org.biojava.bio.structure">SVDSuperimposer</a></dt>
<dd>
<div class="block">Get the shift vector.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDatabase.html#getTree(org.biojava.bio.structure.cath.CathDomain)">getTree(CathDomain)</a></span> - Method in interface org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDatabase.html" title="interface in org.biojava.bio.structure.cath">CathDatabase</a></dt>
<dd>
<div class="block">Return the CATH sub-tree for a particular domain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#getTree(org.biojava.bio.structure.cath.CathDomain)">getTree(CathDomain)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/CachedRemoteScopInstallation.html#getTree(org.biojava.bio.structure.scop.ScopDomain)">getTree(ScopDomain)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/CachedRemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">CachedRemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html#getTree(org.biojava.bio.structure.scop.ScopDomain)">getTree(ScopDomain)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">RemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDatabase.html#getTree(org.biojava.bio.structure.scop.ScopDomain)">getTree(ScopDomain)</a></span> - Method in interface org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDatabase.html" title="interface in org.biojava.bio.structure.scop">ScopDatabase</a></dt>
<dd>
<div class="block">get the SCOP sub-tree for a particular domain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#getTree(org.biojava.bio.structure.scop.ScopDomain)">getTree(ScopDomain)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucState.html#getTurn()">getTurn()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucState.html" title="class in org.biojava.bio.structure.secstruc">SecStrucState</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getTwi()">getTwi()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatAligner.html#getTwistedGroups()">getTwistedGroups()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatAligner.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcidImpl.html#getType()">getType()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcidImpl.html" title="class in org.biojava.bio.structure">AminoAcidImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Bond.html#getType()">getType()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Bond.html" title="class in org.biojava.bio.structure">Bond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#getType()">getType()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">get Type of group, e.g.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#getType()">getType()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">Returns the type value.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getType()">getType()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html#getType()">getType()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemCompDescriptor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html#getType()">getType()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model">Entity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html#getType()">getType()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructOperList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsEntity.html#getType()">getType()</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsEntity.html" title="class in org.biojava.bio.structure.io.sifts">SiftsEntity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/NucleotideImpl.html#getType()">getType()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/NucleotideImpl.html" title="class in org.biojava.bio.structure">NucleotideImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getType_symbol()">getType_symbol()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/LUDecomposition.html#getU()">getU()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/LUDecomposition.html" title="class in org.biojava.bio.structure.jama">LUDecomposition</a></dt>
<dd>
<div class="block">Return upper triangular factor</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/SingularValueDecomposition.html#getU()">getU()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/SingularValueDecomposition.html" title="class in org.biojava.bio.structure.jama">SingularValueDecomposition</a></dt>
<dd>
<div class="block">Return the left singular vectors</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/OperatorResolver.html#getUnaryOperators()">getUnaryOperators()</a></span> - Method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/OperatorResolver.html" title="class in org.biojava.bio.structure.quaternary">OperatorResolver</a></dt>
<dd>
<div class="block">Returns a list of operators for this assembly.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html#getUniProtAccessionId()">getUniProtAccessionId()</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html" title="class in org.biojava.bio.structure.io.sifts">SiftsResidue</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html#getUniProtPos()">getUniProtPos()</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html" title="class in org.biojava.bio.structure.io.sifts">SiftsResidue</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html#getUniProtResName()">getUniProtResName()</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html" title="class in org.biojava.bio.structure.io.sifts">SiftsResidue</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#getUnitv()">getUnitv()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#getUsed()">getUsed()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#getUsedAtomNames()">getUsedAtomNames()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#getUserConfigHelp()">getUserConfigHelp()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/ConfigStrucAligParams.html#getUserConfigHelp()">getUserConfigHelp()</a></span> - Method in interface org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/ConfigStrucAligParams.html" title="interface in org.biojava.bio.structure.align.ce">ConfigStrucAligParams</a></dt>
<dd>
<div class="block">The help text for each of these parameters.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/OptimalCECPParameters.html#getUserConfigHelp()">getUserConfigHelp()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/OptimalCECPParameters.html" title="class in org.biojava.bio.structure.align.ce">OptimalCECPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getUserConfigHelp()">getUserConfigHelp()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWaterman3DParameters.html#getUserConfigHelp()">getUserConfigHelp()</a></span> - Method in class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWaterman3DParameters.html" title="class in org.biojava.bio.structure.align.seq">SmithWaterman3DParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#getUserConfigParameterNames()">getUserConfigParameterNames()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/ConfigStrucAligParams.html#getUserConfigParameterNames()">getUserConfigParameterNames()</a></span> - Method in interface org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/ConfigStrucAligParams.html" title="interface in org.biojava.bio.structure.align.ce">ConfigStrucAligParams</a></dt>
<dd>
<div class="block">The labels to be displayed to the user for each parameter</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/OptimalCECPParameters.html#getUserConfigParameterNames()">getUserConfigParameterNames()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/OptimalCECPParameters.html" title="class in org.biojava.bio.structure.align.ce">OptimalCECPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getUserConfigParameterNames()">getUserConfigParameterNames()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWaterman3DParameters.html#getUserConfigParameterNames()">getUserConfigParameterNames()</a></span> - Method in class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWaterman3DParameters.html" title="class in org.biojava.bio.structure.align.seq">SmithWaterman3DParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#getUserConfigParameters()">getUserConfigParameters()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/ConfigStrucAligParams.html#getUserConfigParameters()">getUserConfigParameters()</a></span> - Method in interface org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/ConfigStrucAligParams.html" title="interface in org.biojava.bio.structure.align.ce">ConfigStrucAligParams</a></dt>
<dd>
<div class="block">get the list of parameters that the user can change through the user interface.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/OptimalCECPParameters.html#getUserConfigParameters()">getUserConfigParameters()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/OptimalCECPParameters.html" title="class in org.biojava.bio.structure.align.ce">OptimalCECPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getUserConfigParameters()">getUserConfigParameters()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWaterman3DParameters.html#getUserConfigParameters()">getUserConfigParameters()</a></span> - Method in class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWaterman3DParameters.html" title="class in org.biojava.bio.structure.align.seq">SmithWaterman3DParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#getUserConfigTypes()">getUserConfigTypes()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/ConfigStrucAligParams.html#getUserConfigTypes()">getUserConfigTypes()</a></span> - Method in interface org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/ConfigStrucAligParams.html" title="interface in org.biojava.bio.structure.align.ce">ConfigStrucAligParams</a></dt>
<dd>
<div class="block">Get the data types of the parameters</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/OptimalCECPParameters.html#getUserConfigTypes()">getUserConfigTypes()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/OptimalCECPParameters.html" title="class in org.biojava.bio.structure.align.ce">OptimalCECPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getUserConfigTypes()">getUserConfigTypes()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWaterman3DParameters.html#getUserConfigTypes()">getUserConfigTypes()</a></span> - Method in class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWaterman3DParameters.html" title="class in org.biojava.bio.structure.align.seq">SmithWaterman3DParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#getUsername()">getUsername()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/EigenvalueDecomposition.html#getV()">getV()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/EigenvalueDecomposition.html" title="class in org.biojava.bio.structure.jama">EigenvalueDecomposition</a></dt>
<dd>
<div class="block">Return the eigenvector matrix</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/SingularValueDecomposition.html#getV()">getV()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/SingularValueDecomposition.html" title="class in org.biojava.bio.structure.jama">SingularValueDecomposition</a></dt>
<dd>
<div class="block">Return the right singular vectors</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#getValenceElectronCount()">getValenceElectronCount()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns the number of valence electrons for this Element.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/AligMatEl.html#getValue()">getValue()</a></span> - Method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/AligMatEl.html" title="class in org.biojava.bio.structure.align.helper">AligMatEl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/GapArray.html#getValue()">getValue()</a></span> - Method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/GapArray.html" title="class in org.biojava.bio.structure.align.helper">GapArray</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getVariant()">getVariant()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#getVDWRadius()">getVDWRadius()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns the van der Waals radius of this Element.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html#getVector()">getVector()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructOperList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html#getVector()">getVector()</a></span> - Method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html" title="class in org.biojava.bio.structure.quaternary">ModelTransformationMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/AbstractStructureAlignment.html#getVersion()">getVersion()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/AbstractStructureAlignment.html" title="class in org.biojava.bio.structure.align">AbstractStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/BioJavaStructureAlignment.html#getVersion()">getVersion()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/BioJavaStructureAlignment.html" title="class in org.biojava.bio.structure.align">BioJavaStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCPMain.html#getVersion()">getVersion()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCPMain.html" title="class in org.biojava.bio.structure.align.ce">CeCPMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeMain.html#getVersion()">getVersion()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeMain.html" title="class in org.biojava.bio.structure.align.ce">CeMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeSideChainMain.html#getVersion()">getVersion()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeSideChainMain.html" title="class in org.biojava.bio.structure.align.ce">CeSideChainMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/OptimalCECPMain.html#getVersion()">getVersion()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/OptimalCECPMain.html" title="class in org.biojava.bio.structure.align.ce">OptimalCECPMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCatFlexible.html#getVersion()">getVersion()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCatFlexible.html" title="class in org.biojava.bio.structure.align.fatcat">FatCatFlexible</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCatRigid.html#getVersion()">getVersion()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCatRigid.html" title="class in org.biojava.bio.structure.align.fatcat">FatCatRigid</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getVersion()">getVersion()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWaterman3Daligner.html#getVersion()">getVersion()</a></span> - Method in class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWaterman3Daligner.html" title="class in org.biojava.bio.structure.align.seq">SmithWaterman3Daligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructureAlignment.html#getVersion()">getVersion()</a></span> - Method in interface org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructureAlignment.html" title="interface in org.biojava.bio.structure.align">StructureAlignment</a></dt>
<dd>
<div class="block">Get the Version information for this Algorithm.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#getVersion()">getVersion()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#getVolume()">getVolume()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#getWinSize()">getWinSize()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>
<div class="block">The window size to look at</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#getX()">getX()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">Get coordinate X.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#getX()">getX()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>
<div class="block">Get the X coordinate.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#getXYZEuler(org.biojava.bio.structure.jama.Matrix)">getXYZEuler(Matrix)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">Convert a rotation Matrix to Euler angles.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#getY()">getY()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">Get coordinate Y.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#getY()">getY()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>
<div class="block">Get the Y coordinate.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#getZ()">getZ()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">Get coordinate Z.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#getZ()">getZ()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>
<div class="block">Get the Z coordinate.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBCrystallographicInfo.html#getZ()">getZ()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBCrystallographicInfo.html" title="class in org.biojava.bio.structure">PDBCrystallographicInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#getZYZEuler(org.biojava.bio.structure.jama.Matrix)">getZYZEuler(Matrix)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">Gget euler angles for a matrix given in ZYZ convention.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#GLOBAL_ALIGN1">GLOBAL_ALIGN1</a></span> - Static variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#GLOBAL_ALIGN2">GLOBAL_ALIGN2</a></span> - Static variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/pairwise/Gotoh.html" title="class in org.biojava.bio.structure.align.pairwise"><span class="strong">Gotoh</span></a> - Class in <a href="./org/biojava/bio/structure/align/pairwise/package-summary.html">org.biojava.bio.structure.align.pairwise</a></dt>
<dd>
<div class="block">a class to perform Gotoh algorithm</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/Gotoh.html#Gotoh(org.biojava.bio.structure.align.pairwise.Alignable)">Gotoh(Alignable)</a></span> - Constructor for class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/Gotoh.html" title="class in org.biojava.bio.structure.align.pairwise">Gotoh</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure"><span class="strong">Group</span></a> - Interface in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">This is the data structure for a single Group of atoms.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/GroupIterator.html" title="class in org.biojava.bio.structure"><span class="strong">GroupIterator</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">An iterator over all groups of a structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/GroupIterator.html#GroupIterator(org.biojava.bio.structure.Structure)">GroupIterator(Structure)</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/GroupIterator.html" title="class in org.biojava.bio.structure">GroupIterator</a></dt>
<dd>
<div class="block">Constructs a GroupIterator object.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/GroupType.html" title="class in org.biojava.bio.structure"><span class="strong">GroupType</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">contains only the static declaration of which types of Groups are available</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/GroupType.html#GroupType()">GroupType()</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/GroupType.html" title="class in org.biojava.bio.structure">GroupType</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AlignmentTools.html#guessSequentialAlignment(java.util.Map, boolean)">guessSequentialAlignment(Map&lt;Integer, Integer&gt;, boolean)</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AlignmentTools.html" title="class in org.biojava.bio.structure.align.util">AlignmentTools</a></dt>
<dd>
<div class="block">Takes a potentially non-sequential alignment and guesses a sequential
 version of it.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/ce/GuiWrapper.html" title="class in org.biojava.bio.structure.align.ce"><span class="strong">GuiWrapper</span></a> - Class in <a href="./org/biojava/bio/structure/align/ce/package-summary.html">org.biojava.bio.structure.align.ce</a></dt>
<dd>
<div class="block">A class to wrap some of the strucutre.gui classes using Reflection</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/GuiWrapper.html#GuiWrapper()">GuiWrapper()</a></span> - Constructor for class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/GuiWrapper.html" title="class in org.biojava.bio.structure.align.ce">GuiWrapper</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_H_">
<!--   -->
</a>
<h2 class="title">H</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#has3D()">has3D()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">returns true or false, depending if this group has 3D coordinates or not.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#has3D()">has3D()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">returns true or false, depending if this group has 3D coordinates or not.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#hasAltLoc()">hasAltLoc()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">Test if this group has alternate locations.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#hasAltLoc()">hasAltLoc()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#hasAminoAtoms()">hasAminoAtoms()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">calculate if a groups has all atoms required for an amino acid.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#hasAminoAtoms()">hasAminoAtoms()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">calculate if a groups has all atoms required for an amino acid
     this allows to include chemically modified amino acids that
     are labeled hetatoms into some computations ...</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#hasAtom(java.lang.String)">hasAtom(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">Teturns flag whether a particular atom is existing within this group .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#hasAtom(java.lang.String)">hasAtom(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">test is an Atom with name is existing.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/BioUnitDataProvider.html#hasBiolAssembly(java.lang.String)">hasBiolAssembly(String)</a></span> - Method in interface org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/BioUnitDataProvider.html" title="interface in org.biojava.bio.structure.quaternary.io">BioUnitDataProvider</a></dt>
<dd>
<div class="block">Does the PDB ID have biological assembly information?</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/FileBasedPDBBioUnitDataProvider.html#hasBiolAssembly(java.lang.String)">hasBiolAssembly(String)</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/FileBasedPDBBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">FileBasedPDBBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/MmCifBiolAssemblyProvider.html#hasBiolAssembly(java.lang.String)">hasBiolAssembly(String)</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/MmCifBiolAssemblyProvider.html" title="class in org.biojava.bio.structure.quaternary.io">MmCifBiolAssemblyProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/MmCifPDBBiolAssemblyProvider.html#hasBiolAssembly()">hasBiolAssembly()</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/MmCifPDBBiolAssemblyProvider.html" title="class in org.biojava.bio.structure.quaternary.io">MmCifPDBBiolAssemblyProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/PDBBioUnitDataProvider.html#hasBiolAssembly(java.lang.String)">hasBiolAssembly(String)</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/PDBBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">PDBBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/RawBioUnitDataProvider.html#hasBiolAssembly()">hasBiolAssembly()</a></span> - Method in interface org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/RawBioUnitDataProvider.html" title="interface in org.biojava.bio.structure.quaternary.io">RawBioUnitDataProvider</a></dt>
<dd>
<div class="block">Does the PDB ID have biological assembly information?</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/RemoteBioUnitDataProvider.html#hasBiolAssembly(java.lang.String)">hasBiolAssembly(String)</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/RemoteBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">RemoteBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/RemoteRawBioUnitDataProvider.html#hasBiolAssembly()">hasBiolAssembly()</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/RemoteRawBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">RemoteRawBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/structure/StructureIO.html#hasBiologicalAssembly(java.lang.String)">hasBiologicalAssembly(String)</a></span> - Static method in class org.biojava3.structure.<a href="./org/biojava3/structure/StructureIO.html" title="class in org.biojava3.structure">StructureIO</a></dt>
<dd>
<div class="block">Does the provider PDB ID have a biological assembly?</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#hasChain(java.lang.String)">hasChain(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">check if a chain with the id chainId is contained in this structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#hasChain(java.lang.String)">hasChain(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/StructureName.html#hasChainID()">hasChainID()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/StructureName.html" title="class in org.biojava.bio.structure.align.client">StructureName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/PdbPair.html#hashCode()">hashCode()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/PdbPair.html" title="class in org.biojava.bio.structure.align.client">PdbPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/StructureName.html#hashCode()">hashCode()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/StructureName.html" title="class in org.biojava.bio.structure.align.client">StructureName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#hashCode()">hashCode()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Author.html#hashCode()">hashCode()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Author.html" title="class in org.biojava.bio.structure">Author</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#hashCode()">hashCode()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html#hashCode()">hashCode()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemCompDescriptor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsEntity.html#hashCode()">hashCode()</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsEntity.html" title="class in org.biojava.bio.structure.io.sifts">SiftsEntity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsSegment.html#hashCode()">hashCode()</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsSegment.html" title="class in org.biojava.bio.structure.io.sifts">SiftsSegment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ResidueNumber.html#hashCode()">hashCode()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ResidueNumber.html" title="class in org.biojava.bio.structure">ResidueNumber</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDescription.html#hashCode()">hashCode()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDescription.html" title="class in org.biojava.bio.structure.scop">ScopDescription</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Site.html#hashCode()">hashCode()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Site.html" title="class in org.biojava.bio.structure">Site</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#hasJournalArticle()">hasJournalArticle()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">return whether or not the entry has an associated journal article
 or publication.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#hasJournalArticle()">hasJournalArticle()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">Return whether or not the entry has an associated journal article
 or publication.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomIterator.html#hasNext()">hasNext()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomIterator.html" title="class in org.biojava.bio.structure">AtomIterator</a></dt>
<dd>
<div class="block">Is there a next atom ?</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/GroupIterator.html#hasNext()">hasNext()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/GroupIterator.html" title="class in org.biojava.bio.structure">GroupIterator</a></dt>
<dd>
<div class="block">is there a group after the current one in the structure?</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/FlatFileInstallation.html#hasNext()">hasNext()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/FlatFileInstallation.html" title="class in org.biojava.bio.structure.server">FlatFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html#hasNext()">hasNext()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html" title="class in org.biojava.bio.structure.server">MMCIFFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/PDBInstallation.html#hasNext()">hasNext()</a></span> - Method in interface org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/PDBInstallation.html" title="interface in org.biojava.bio.structure.server">PDBInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
<div class="block">return if the iteration over all structures will return another structure</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/SimpleStructureServer.html#hasNextStructure()">hasNextStructure()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/SimpleStructureServer.html" title="class in org.biojava.bio.structure.server">SimpleStructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureServer.html#hasNextStructure()">hasNextStructure()</a></span> - Method in interface org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureServer.html" title="interface in org.biojava.bio.structure.server">StructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#hasParent()">hasParent()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/JFatCatClient.html#hasPrecalculatedResult(java.lang.String, java.lang.String, java.lang.String, java.lang.String)">hasPrecalculatedResult(String, String, String, String)</a></span> - Static method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/JFatCatClient.html" title="class in org.biojava.bio.structure.align.client">JFatCatClient</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/JFatCatClient.html#hasPrecalculatedResult(java.lang.String, java.lang.String, java.lang.String, java.lang.String, int)">hasPrecalculatedResult(String, String, String, String, int)</a></span> - Static method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/JFatCatClient.html" title="class in org.biojava.bio.structure.align.client">JFatCatClient</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/xml/HasResultXMLConverter.html" title="class in org.biojava.bio.structure.align.xml"><span class="strong">HasResultXMLConverter</span></a> - Class in <a href="./org/biojava/bio/structure/align/xml/package-summary.html">org.biojava.bio.structure.align.xml</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/HasResultXMLConverter.html#HasResultXMLConverter()">HasResultXMLConverter()</a></span> - Constructor for class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/HasResultXMLConverter.html" title="class in org.biojava.bio.structure.align.xml">HasResultXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/secstruc/HBond.html" title="class in org.biojava.bio.structure.secstruc"><span class="strong">HBond</span></a> - Class in <a href="./org/biojava/bio/structure/secstruc/package-summary.html">org.biojava.bio.structure.secstruc</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/HBond.html#HBond()">HBond()</a></span> - Constructor for class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/HBond.html" title="class in org.biojava.bio.structure.secstruc">HBond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStruc.html#HBONDHIGHENERGY">HBONDHIGHENERGY</a></span> - Static variable in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStruc.html" title="class in org.biojava.bio.structure.secstruc">SecStruc</a></dt>
<dd>
<div class="block">higher limit for H-bond energy</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStruc.html#HBONDLOWENERGY">HBONDLOWENERGY</a></span> - Static variable in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStruc.html" title="class in org.biojava.bio.structure.secstruc">SecStruc</a></dt>
<dd>
<div class="block">Minimal H-bond energy in cal / mol</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileParser.html#HELIX">HELIX</a></span> - Static variable in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileParser.html" title="class in org.biojava.bio.structure.io">PDBFileParser</a></dt>
<dd>
<div class="block">Helix secondary structure assignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/GroupType.html#HETATM">HETATM</a></span> - Static variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/GroupType.html" title="class in org.biojava.bio.structure">GroupType</a></dt>
<dd>
<div class="block">the type for hetero groups</div>
</dd>
<dt><a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure"><span class="strong">HetatomImpl</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">Generic Implementation of a Group interface.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#HetatomImpl()">HetatomImpl()</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/BerkeleyScopInstallation.html#hieFileName">hieFileName</a></span> - Static variable in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/BerkeleyScopInstallation.html" title="class in org.biojava.bio.structure.scop">BerkeleyScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#hieFileName">hieFileName</a></span> - Static variable in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/util/HTTPConnectionTools.html" title="class in org.biojava.bio.structure.align.util"><span class="strong">HTTPConnectionTools</span></a> - Class in <a href="./org/biojava/bio/structure/align/util/package-summary.html">org.biojava.bio.structure.align.util</a></dt>
<dd>
<div class="block">a class that takes care about opening HttpURLConnections and sets the proper timeouts</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/HTTPConnectionTools.html#HTTPConnectionTools()">HTTPConnectionTools()</a></span> - Constructor for class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/HTTPConnectionTools.html" title="class in org.biojava.bio.structure.align.util">HTTPConnectionTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Maths.html#hypot(double, double)">hypot(double, double)</a></span> - Static method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Maths.html" title="class in org.biojava.bio.structure.jama">Maths</a></dt>
<dd>
<div class="block">sqrt(a^2 + b^2) without under/overflow.</div>
</dd>
</dl>
<a name="_I_">
<!--   -->
</a>
<h2 class="title">I</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html#id">id</a></span> - Variable in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html" title="class in org.biojava.bio.structure.quaternary">ModelTransformationMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#identity(int, int)">identity(int, int)</a></span> - Static method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Generate identity matrix</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/helper/IdxComparator.html" title="class in org.biojava.bio.structure.align.helper"><span class="strong">IdxComparator</span></a> - Class in <a href="./org/biojava/bio/structure/align/helper/package-summary.html">org.biojava.bio.structure.align.helper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/IdxComparator.html#IdxComparator()">IdxComparator()</a></span> - Constructor for class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/IdxComparator.html" title="class in org.biojava.bio.structure.align.helper">IdxComparator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/GapArray.html#index">index</a></span> - Variable in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/GapArray.html" title="class in org.biojava.bio.structure.align.helper">GapArray</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/helper/IndexPair.html" title="class in org.biojava.bio.structure.align.helper"><span class="strong">IndexPair</span></a> - Class in <a href="./org/biojava/bio/structure/align/helper/package-summary.html">org.biojava.bio.structure.align.helper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/IndexPair.html#IndexPair()">IndexPair()</a></span> - Constructor for class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/IndexPair.html" title="class in org.biojava.bio.structure.align.helper">IndexPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/IndexPair.html#IndexPair(short, short)">IndexPair(short, short)</a></span> - Constructor for class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/IndexPair.html" title="class in org.biojava.bio.structure.align.helper">IndexPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html#init()">init()</a></span> - Method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html" title="class in org.biojava.bio.structure.quaternary">ModelTransformationMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/SimpleStructureServer.html#initCache()">initCache()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/SimpleStructureServer.html" title="class in org.biojava.bio.structure.server">SimpleStructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/MatrixListener.html#initializeBreakFlag(boolean[][])">initializeBreakFlag(boolean[][])</a></span> - Method in interface org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/MatrixListener.html" title="interface in org.biojava.bio.structure.align.ce">MatrixListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#initSumOfDistances(int, int, int, int, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">initSumOfDistances(int, int, int, int, Atom[], Atom[])</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#initSumOfDistances(int, int, int, int, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">initSumOfDistances(int, int, int, int, Atom[], Atom[])</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#inverse()">inverse()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Matrix inverse or pseudoinverse</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html#inverse3()">inverse3()</a></span> - Method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html" title="class in org.biojava.bio.structure.quaternary">ModelTransformationMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#invert(org.biojava.bio.structure.Atom)">invert(Atom)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileParsingParameters.html#isAlignSeqRes()">isAlignSeqRes()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileParsingParameters.html" title="class in org.biojava.bio.structure.io">FileParsingParameters</a></dt>
<dd>
<div class="block">Flag if the SEQRES amino acids should be aligned with the ATOM amino acids.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#isAutoFetch()">isAutoFetch()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#isAutoFetch()">isAutoFetch()</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>
<div class="block">Does the cache automatically download files that are missing from the local installation from the PDB FTP site?</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/MMCIFFileReader.html#isAutoFetch()">isAutoFetch()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/MMCIFFileReader.html" title="class in org.biojava.bio.structure.io">MMCIFFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#isAutoFetch()">isAutoFetch()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/StructureIOFile.html#isAutoFetch()">isAutoFetch()</a></span> - Method in interface org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/StructureIOFile.html" title="interface in org.biojava.bio.structure.io">StructureIOFile</a></dt>
<dd>
<div class="block">Fetch files automatically from FTP server.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html#isAutoFetch()">isAutoFetch()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html" title="class in org.biojava.bio.structure.server">MMCIFFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
<div class="block">should the parser to fetch missing mmCif files from the RCSB FTP server automatically?
  default is false</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucState.html#isBend()">isBend()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucState.html" title="class in org.biojava.bio.structure.secstruc">SecStrucState</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#isBiologicalAssembly()">isBiologicalAssembly()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">Gets flag that indicates if this structure is a biological assembly</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#isBiologicalAssembly()">isBiologicalAssembly()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">Gets flag that indicates if this structure is a biological assembly</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/SerializableCache.html#isCacheEnabled()">isCacheEnabled()</a></span> - Method in class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/SerializableCache.html" title="class in org.biojava.bio.structure.domain">SerializableCache</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/StructureName.html#isCathID()">isCathID()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/StructureName.html" title="class in org.biojava.bio.structure.align.client">StructureName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#isChalcogen()">isChalcogen()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns <CODE>true</CODE> if Element is a chalcogen (O, S, Se, Te, Po).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#isConnected(org.biojava.bio.structure.AminoAcid, org.biojava.bio.structure.AminoAcid)">isConnected(AminoAcid, AminoAcid)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">test if two amino acids are connected, i.e.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBCrystallographicInfo.html#isCrystallographic()">isCrystallographic()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBCrystallographicInfo.html" title="class in org.biojava.bio.structure">PDBCrystallographicInfo</a></dt>
<dd>
<div class="block">Returns true if structure was solved by a crystallographic method, i.e.,
 the values returned by this class are meaningful.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructurePairAligner.html#isDebug()">isDebug()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructurePairAligner.html" title="class in org.biojava.bio.structure.align">StructurePairAligner</a></dt>
<dd>
<div class="block">check if debug mode is set on</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/SerializableCache.html#isDebug()">isDebug()</a></span> - Method in class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/SerializableCache.html" title="class in org.biojava.bio.structure.domain">SerializableCache</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/SeqRes2AtomAligner.html#isDEBUG()">isDEBUG()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/SeqRes2AtomAligner.html" title="class in org.biojava.bio.structure.io">SeqRes2AtomAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#isDoAngleCheck()">isDoAngleCheck()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#isDoDensityCheck()">isDoDensityCheck()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#isDoDistanceCheck()">isDoDistanceCheck()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#isDoRMSCheck()">isDoRMSCheck()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/DownloadChemCompProvider.html#isDownloadAll()">isDownloadAll()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/DownloadChemCompProvider.html" title="class in org.biojava.bio.structure.io.mmcif">DownloadChemCompProvider</a></dt>
<dd>
<div class="block">By default this provider will download only some of the <a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model"><code>ChemComp</code></a> files.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/AFPChainXMLParser.html#isErrorXML(java.lang.String)">isErrorXML(String)</a></span> - Static method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/AFPChainXMLParser.html" title="class in org.biojava.bio.structure.align.xml">AFPChainXMLParser</a></dt>
<dd>
<div class="block">returns true if the alignment XML contains an error message</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#isFetchCurrent()">isFetchCurrent()</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>
<div class="block"><b>N.B.</b> This feature won't work unless the structure wasn't found & autoFetch is set to <code>true</code>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#isFetchCurrent()">isFetchCurrent()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>
<div class="block"><b>N.B.</b> This feature won't work unless the structure wasn't found & autoFetch is set to <code>true</code>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#isFetchFileEvenIfObsolete()">isFetchFileEvenIfObsolete()</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>
<div class="block">forces the cache to fetch the file if its status is OBSOLETE.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#isFetchFileEvenIfObsolete()">isFetchFileEvenIfObsolete()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>
<div class="block">forces the reader to fetch the file if its status is OBSOLETE.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#isFlexible()">isFlexible()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/QRDecomposition.html#isFullRank()">isFullRank()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/QRDecomposition.html" title="class in org.biojava.bio.structure.jama">QRDecomposition</a></dt>
<dd>
<div class="block">Is the matrix full rank?</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/GuiWrapper.html#isGuiModuleInstalled()">isGuiModuleInstalled()</a></span> - Static method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/GuiWrapper.html" title="class in org.biojava.bio.structure.align.ce">GuiWrapper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#isHalogen()">isHalogen()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns <CODE>true</CODE> if Element is a halogen (F, Cl, Br, I, At).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileParsingParameters.html#isHeaderOnly()">isHeaderOnly()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileParsingParameters.html" title="class in org.biojava.bio.structure.io">FileParsingParameters</a></dt>
<dd>
<div class="block">Parse only the PDB file header out of the files</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#isHeavyAtom()">isHeavyAtom()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns <CODE>true</CODE> is the Element is an not Hydrogen.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucType.html#isHelixType()">isHelixType()</a></span> - Method in enum org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucType.html" title="enum in org.biojava.bio.structure.secstruc">SecStrucType</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#isHeteroAtom()">isHeteroAtom()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns <CODE>true</CODE> if Element is not Hydrogen and not Carbon.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#isHydrogen()">isHydrogen()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns true if this Element is Hydrogen.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#isJoinFast()">isJoinFast()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#isJoinPlo()">isJoinPlo()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileParsingParameters.html#isLoadChemCompInfo()">isLoadChemCompInfo()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileParsingParameters.html" title="class in org.biojava.bio.structure.io">FileParsingParameters</a></dt>
<dd>
<div class="block">Should the chemical component information be automatically be downloade from the web?
 If set to false, a limited set of ChemComps is being used.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#isMetal()">isMetal()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns <CODE>true</CODE> if ElementType is a metal.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ElementType.html#isMetal()">isMetal()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ElementType.html" title="enum in org.biojava.bio.structure">ElementType</a></dt>
<dd>
<div class="block">Returns <CODE>true</CODE> if ElementType is a metal.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#isMetalloid()">isMetalloid()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns <CODE>true</CODE> if ElementType is a metalloid.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ElementType.html#isMetalloid()">isMetalloid()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ElementType.html" title="enum in org.biojava.bio.structure">ElementType</a></dt>
<dd>
<div class="block">Returns <CODE>true</CODE> if ElementType is a metalloid.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#isNmr()">isNmr()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">test if this structure is an nmr structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#isNmr()">isNmr()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">is this structure an nmr structure ?</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#isNonMetal()">isNonMetal()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns <CODE>true</CODE> if ElementType is a non-metal.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ElementType.html#isNonMetal()">isNonMetal()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ElementType.html" title="enum in org.biojava.bio.structure">ElementType</a></dt>
<dd>
<div class="block">Returns <CODE>true</CODE> if ElementType is a non-metal.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/LUDecomposition.html#isNonsingular()">isNonsingular()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/LUDecomposition.html" title="class in org.biojava.bio.structure.jama">LUDecomposition</a></dt>
<dd>
<div class="block">Is the matrix nonsingular?</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#isNucleotide(java.lang.String)">isNucleotide(String)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#isOutputPDB()">isOutputPDB()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>
<div class="block">When writing the results to a file, don;t write as XML but write aligned PDB file</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileParsingParameters.html#isParseBioAssembly()">isParseBioAssembly()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileParsingParameters.html" title="class in org.biojava.bio.structure.io">FileParsingParameters</a></dt>
<dd>
<div class="block">Should the biological assembly info (REMARK 350) be parsed from the PDB file?</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileParsingParameters.html#isParseCAOnly()">isParseCAOnly()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileParsingParameters.html" title="class in org.biojava.bio.structure.io">FileParsingParameters</a></dt>
<dd>
<div class="block">the flag if only the C-alpha atoms of the structure should be parsed.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileParsingParameters.html#isParseSecStruc()">isParseSecStruc()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileParsingParameters.html" title="class in org.biojava.bio.structure.io">FileParsingParameters</a></dt>
<dd>
<div class="block">is secondary structure assignment being parsed from the file?
 default is null</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/MMCIFFileReader.html#isPdbDirectorySplit()">isPdbDirectorySplit()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/MMCIFFileReader.html" title="class in org.biojava.bio.structure.io">MMCIFFileReader</a></dt>
<dd>
<div class="block">Flag that defines if the PDB directory is containing all PDB files or is split into sub dirs (like the FTP site).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#isPdbDirectorySplit()">isPdbDirectorySplit()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>
<div class="block">Flag that defines if the PDB directory is containing all PDB files or is split into sub dirs (like the FTP site).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/StructureIOFile.html#isPdbDirectorySplit()">isPdbDirectorySplit()</a></span> - Method in interface org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/StructureIOFile.html" title="interface in org.biojava.bio.structure.io">StructureIOFile</a></dt>
<dd>
<div class="block">The PDB files are organized hierarchically (as on the PDB - FTP server.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#isPdbDirSplit()">isPdbDirSplit()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#isPdbDirSplit()">isPdbDirSplit()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/StructureName.html#isPDPDomain()">isPDPDomain()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/StructureName.html" title="class in org.biojava.bio.structure.align.client">StructureName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#isPrint">isPrint</a></span> - Static variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#isPrint">isPrint</a></span> - Static variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#isPrintCE()">isPrintCE()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>
<div class="block">Display the output string in CE style</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#isPrintFatCat()">isPrintFatCat()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#isPrintXML()">isPrintXML()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#isPublished()">isPublished()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html#isReplaced(org.biojava.bio.structure.io.mmcif.model.ChemComp)">isReplaced(ChemComp)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html" title="class in org.biojava.bio.structure.io.mmcif">ChemicalComponentDictionary</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html#isReplaced(java.lang.String)">isReplaced(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html" title="class in org.biojava.bio.structure.io.mmcif">ChemicalComponentDictionary</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html#isReplacer(org.biojava.bio.structure.io.mmcif.model.ChemComp)">isReplacer(ChemComp)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html" title="class in org.biojava.bio.structure.io.mmcif">ChemicalComponentDictionary</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html#isReplacer(java.lang.String)">isReplacer(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html" title="class in org.biojava.bio.structure.io.mmcif">ChemicalComponentDictionary</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#isRunBackground()">isRunBackground()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>
<div class="block">Flag if a job that only runs one parallell job should be run in its own thread or in the main thread.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/StructureName.html#isScopName()">isScopName()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/StructureName.html" title="class in org.biojava.bio.structure.align.client">StructureName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#isSequentialAlignment()">isSequentialAlignment()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">Get whether this alignment has the normal topology, ie the residues
 aligned in each block increase sequentially over the original protein.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AlignmentTools.html#isSequentialAlignment(org.biojava.bio.structure.align.model.AFPChain, boolean)">isSequentialAlignment(AFPChain, boolean)</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AlignmentTools.html" title="class in org.biojava.bio.structure.align.util">AlignmentTools</a></dt>
<dd>
<div class="block">Checks that the alignment given by afpChain is sequential.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#isShortAlign()">isShortAlign()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#isShow3d()">isShow3d()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#isShowAFPRanges()">isShowAFPRanges()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#isShowAFPRanges()">isShowAFPRanges()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#isShowMenu()">isShowMenu()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#isSignificantResult()">isSignificantResult()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/CholeskyDecomposition.html#isSPD()">isSPD()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/CholeskyDecomposition.html" title="class in org.biojava.bio.structure.jama">CholeskyDecomposition</a></dt>
<dd>
<div class="block">Is the matrix symmetric and positive definite?</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#isSplit()">isSplit()</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>
<div class="block">Is the organization of files within the directory split, as on the PDB FTP servers,
 or are all files contained in one directory.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/UserConfiguration.html#isSplit()">isSplit()</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/UserConfiguration.html" title="class in org.biojava.bio.structure.align.util">UserConfiguration</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#isStandard()">isStandard()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/chem/ChemCompTools.html#isStandardChemComp(org.biojava.bio.structure.io.mmcif.model.ChemComp)">isStandardChemComp(ChemComp)</a></span> - Static method in class org.biojava.bio.structure.io.mmcif.chem.<a href="./org/biojava/bio/structure/io/mmcif/chem/ChemCompTools.html" title="class in org.biojava.bio.structure.io.mmcif.chem">ChemCompTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#isStrictSCOP()">isStrictSCOP()</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>
<div class="block">Reports whether strict scop naming will be enforced, or whether this AtomCache
 should try to guess some simple variants on scop domains.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/OptimalCECPParameters.html#isTryAllCPs()">isTryAllCPs()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/OptimalCECPParameters.html" title="class in org.biojava.bio.structure.align.ce">OptimalCECPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/BioAssemblyTools.html#isUnaryExpression(java.lang.String)">isUnaryExpression(String)</a></span> - Static method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/BioAssemblyTools.html" title="class in org.biojava.bio.structure.quaternary">BioAssemblyTools</a></dt>
<dd>
<div class="block">Checks if the passed in expression is a unary operator expression
 Example: (1,2,3) or (1-60) are unary operator expressions
          (1-60)(61-88) is a binary operator expression, representing
          a cartesian product of the two parenthesised lists</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileParsingParameters.html#isUpdateRemediatedFiles()">isUpdateRemediatedFiles()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileParsingParameters.html" title="class in org.biojava.bio.structure.io">FileParsingParameters</a></dt>
<dd>
<div class="block">A flag if local files should be replaced with the latest version of remediated PDB files.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#isVerbose()">isVerbose()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>
<div class="block">Flag if the job should be run in verbose mode.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html#isWaitForAlignments()">isWaitForAlignments()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html" title="class in org.biojava.bio.structure.align.client">FarmJobRunnable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#iterator()">iterator()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">get an Atom Iterator.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#iterator()">iterator()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">return an AtomIterator.</div>
</dd>
</dl>
<a name="_J_">
<!--   -->
</a>
<h2 class="title">J</h2>
<dl>
<dt><a href="./org/biojava/bio/structure/align/client/JFatCatClient.html" title="class in org.biojava.bio.structure.align.client"><span class="strong">JFatCatClient</span></a> - Class in <a href="./org/biojava/bio/structure/align/client/package-summary.html">org.biojava.bio.structure.align.client</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/JFatCatClient.html#JFatCatClient()">JFatCatClient()</a></span> - Constructor for class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/JFatCatClient.html" title="class in org.biojava.bio.structure.align.client">JFatCatClient</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/client/JobKillException.html" title="class in org.biojava.bio.structure.align.client"><span class="strong">JobKillException</span></a> - Exception in <a href="./org/biojava/bio/structure/align/client/package-summary.html">org.biojava.bio.structure.align.client</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/JobKillException.html#JobKillException(java.lang.String)">JobKillException(String)</a></span> - Constructor for exception org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/JobKillException.html" title="class in org.biojava.bio.structure.align.client">JobKillException</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/helper/JointFragments.html" title="class in org.biojava.bio.structure.align.helper"><span class="strong">JointFragments</span></a> - Class in <a href="./org/biojava/bio/structure/align/helper/package-summary.html">org.biojava.bio.structure.align.helper</a></dt>
<dd>
<div class="block">A utility class that defines which set of atoms are considered
 to be on equivalent positions.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/JointFragments.html#JointFragments()">JointFragments()</a></span> - Constructor for class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/JointFragments.html" title="class in org.biojava.bio.structure.align.helper">JointFragments</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentProgressListener.html#jointFragments(org.biojava.bio.structure.align.helper.JointFragments[])">jointFragments(JointFragments[])</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentProgressListener.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure"><span class="strong">JournalArticle</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">PDB-specific</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#JournalArticle()">JournalArticle()</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_L_">
<!--   -->
</a>
<h2 class="title">L</h2>
<dl>
<dt><a href="./org/biojava/bio/structure/secstruc/Ladder.html" title="class in org.biojava.bio.structure.secstruc"><span class="strong">Ladder</span></a> - Class in <a href="./org/biojava/bio/structure/secstruc/package-summary.html">org.biojava.bio.structure.secstruc</a></dt>
<dd>
<div class="block">Connects two connected strand regions to form larger sheets</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/Ladder.html#Ladder()">Ladder()</a></span> - Constructor for class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/Ladder.html" title="class in org.biojava.bio.structure.secstruc">Ladder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#lastRemediationDate">lastRemediationDate</a></span> - Static variable in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/LocalCacheStructureProvider.html#lineSplit">lineSplit</a></span> - Static variable in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/LocalCacheStructureProvider.html" title="class in org.biojava.bio.structure.io">LocalCacheStructureProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/MMCIFFileReader.html#lineSplit">lineSplit</a></span> - Static variable in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/MMCIFFileReader.html" title="class in org.biojava.bio.structure.io">MMCIFFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#lineSplit">lineSplit</a></span> - Static variable in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileParser.html#linkChains2Compound(org.biojava.bio.structure.Structure)">linkChains2Compound(Structure)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileParser.html" title="class in org.biojava.bio.structure.io">PDBFileParser</a></dt>
<dd>
<div class="block">After the parsing of a PDB file the <a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure"><code>Chain</code></a> and  <a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure"><code>Compound</code></a>
 objects need to be linked to each other.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#LOAD_CHEM_COMP_PROPERTY">LOAD_CHEM_COMP_PROPERTY</a></span> - Static variable in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/FileBasedPDBBioUnitDataProvider.html#loadPDB(java.lang.String, int)">loadPDB(String, int)</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/FileBasedPDBBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">FileBasedPDBBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/PDBBioUnitDataProvider.html#loadPDB(java.lang.String)">loadPDB(String)</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/PDBBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">PDBBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoLoadStructure.html#loadStructureFromCache()">loadStructureFromCache()</a></span> - Method in class demo.<a href="./demo/DemoLoadStructure.html" title="class in demo">DemoLoadStructure</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/LocalCacheStructureProvider.html" title="class in org.biojava.bio.structure.io"><span class="strong">LocalCacheStructureProvider</span></a> - Class in <a href="./org/biojava/bio/structure/io/package-summary.html">org.biojava.bio.structure.io</a></dt>
<dd>
<div class="block">Provides structures based on locally cached PDB files.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/LocalCacheStructureProvider.html#LocalCacheStructureProvider()">LocalCacheStructureProvider()</a></span> - Constructor for class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/LocalCacheStructureProvider.html" title="class in org.biojava.bio.structure.io">LocalCacheStructureProvider</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/domain/LocalProteinDomainParser.html" title="class in org.biojava.bio.structure.domain"><span class="strong">LocalProteinDomainParser</span></a> - Class in <a href="./org/biojava/bio/structure/domain/package-summary.html">org.biojava.bio.structure.domain</a></dt>
<dd>
<div class="block">Protein Domain Parser is a an algorithm that attempts at assigning domains for 3D protein structures.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html#log(java.lang.String)">log(String)</a></span> - Static method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html" title="class in org.biojava.bio.structure.align.client">FarmJobRunnable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#logger">logger</a></span> - Static variable in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentJoiner.html#logger">logger</a></span> - Static variable in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentJoiner.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentJoiner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#logger">logger</a></span> - Static variable in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#logger">logger</a></span> - Static variable in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/FlatFileInstallation.html#logger">logger</a></span> - Static variable in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/FlatFileInstallation.html" title="class in org.biojava.bio.structure.server">FlatFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html#logger">logger</a></span> - Static variable in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html" title="class in org.biojava.bio.structure.server">MMCIFFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/PrepareMMcifIndexFile.html#logger">logger</a></span> - Static variable in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/PrepareMMcifIndexFile.html" title="class in org.biojava.bio.structure.server">PrepareMMcifIndexFile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/SimpleStructureServer.html#logger">logger</a></span> - Static variable in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/SimpleStructureServer.html" title="class in org.biojava.bio.structure.server">SimpleStructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#logger">logger</a></span> - Static variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/PrepareIndexFile.html#logPDBInfoFile(java.io.PrintWriter, java.io.PrintWriter, org.biojava.bio.structure.io.StructureIOFile, java.io.File[])">logPDBInfoFile(PrintWriter, PrintWriter, StructureIOFile, File[])</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/PrepareIndexFile.html" title="class in org.biojava.bio.structure.server">PrepareIndexFile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/CountProgressListener.html#logStatus(java.lang.String)">logStatus(String)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/CountProgressListener.html" title="class in org.biojava.bio.structure.align.client">CountProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/events/AlignmentProgressListener.html#logStatus(java.lang.String)">logStatus(String)</a></span> - Method in interface org.biojava.bio.structure.align.events.<a href="./org/biojava/bio/structure/align/events/AlignmentProgressListener.html" title="interface in org.biojava.bio.structure.align.events">AlignmentProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#LOOP_END">LOOP_END</a></span> - Static variable in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#LOOP_START">LOOP_START</a></span> - Static variable in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#lu()">lu()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">LU Decomposition</div>
</dd>
<dt><a href="./org/biojava/bio/structure/jama/LUDecomposition.html" title="class in org.biojava.bio.structure.jama"><span class="strong">LUDecomposition</span></a> - Class in <a href="./org/biojava/bio/structure/jama/package-summary.html">org.biojava.bio.structure.jama</a></dt>
<dd>
<div class="block">LU Decomposition.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/LUDecomposition.html#LUDecomposition(org.biojava.bio.structure.jama.Matrix)">LUDecomposition(Matrix)</a></span> - Constructor for class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/LUDecomposition.html" title="class in org.biojava.bio.structure.jama">LUDecomposition</a></dt>
<dd>
<div class="block">LU Decomposition provides a data structure  to access L, U and piv.</div>
</dd>
</dl>
<a name="_M_">
<!--   -->
</a>
<h2 class="title">M</h2>
<dl>
<dt><span class="strong"><a href="./demo/ChemCompDistribution.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class demo.<a href="./demo/ChemCompDistribution.html" title="class in demo">ChemCompDistribution</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoAtomCache.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class demo.<a href="./demo/DemoAtomCache.html" title="class in demo">DemoAtomCache</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoBerkeleyScop.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class demo.<a href="./demo/DemoBerkeleyScop.html" title="class in demo">DemoBerkeleyScop</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoCATH.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class demo.<a href="./demo/DemoCATH.html" title="class in demo">DemoCATH</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoCE.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class demo.<a href="./demo/DemoCE.html" title="class in demo">DemoCE</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoChangeChemCompProvider.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class demo.<a href="./demo/DemoChangeChemCompProvider.html" title="class in demo">DemoChangeChemCompProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoCommandLineStartup.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class demo.<a href="./demo/DemoCommandLineStartup.html" title="class in demo">DemoCommandLineStartup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoDomainsplit.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class demo.<a href="./demo/DemoDomainsplit.html" title="class in demo">DemoDomainsplit</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoFATCAT.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class demo.<a href="./demo/DemoFATCAT.html" title="class in demo">DemoFATCAT</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoLoadStructure.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class demo.<a href="./demo/DemoLoadStructure.html" title="class in demo">DemoLoadStructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoMMCIFReader.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class demo.<a href="./demo/DemoMMCIFReader.html" title="class in demo">DemoMMCIFReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoSCOP.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class demo.<a href="./demo/DemoSCOP.html" title="class in demo">DemoSCOP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCPMain.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCPMain.html" title="class in org.biojava.bio.structure.align.ce">CeCPMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeMain.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeMain.html" title="class in org.biojava.bio.structure.align.ce">CeMain</a></dt>
<dd>
<div class="block">Example Parameters:
 
 -pdbFilePath /tmp -autoFetch -printCE -pdb1 1cnv -pdb2 3cna</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeSideChainMain.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeSideChainMain.html" title="class in org.biojava.bio.structure.align.ce">CeSideChainMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/OptimalCECPMain.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/OptimalCECPMain.html" title="class in org.biojava.bio.structure.align.ce">OptimalCECPMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/JFatCatClient.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/JFatCatClient.html" title="class in org.biojava.bio.structure.align.client">JFatCatClient</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/FarmJob.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/FarmJob.html" title="class in org.biojava.bio.structure.align">FarmJob</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCat.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCat.html" title="class in org.biojava.bio.structure.align.fatcat">FatCat</a></dt>
<dd>
<div class="block">See demo/FatCatDemo.java for an example how to run.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWaterman3Daligner.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWaterman3Daligner.html" title="class in org.biojava.bio.structure.align.seq">SmithWaterman3Daligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructurePairAligner.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructurePairAligner.html" title="class in org.biojava.bio.structure.align">StructurePairAligner</a></dt>
<dd>
<div class="block">example usage of this class</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/PDBDomainProvider.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/PDBDomainProvider.html" title="class in org.biojava.bio.structure.domain">PDBDomainProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/RemoteDomainProvider.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/RemoteDomainProvider.html" title="class in org.biojava.bio.structure.domain">RemoteDomainProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/RemotePDPProvider.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/RemotePDPProvider.html" title="class in org.biojava.bio.structure.domain">RemotePDPProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/MMCIFFileReader.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/MMCIFFileReader.html" title="class in org.biojava.bio.structure.io">MMCIFFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsMappingProvider.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsMappingProvider.html" title="class in org.biojava.bio.structure.io.sifts">SiftsMappingProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/RemoteRawBioUnitDataProvider.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/RemoteRawBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">RemoteRawBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">RemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStruc.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStruc.html" title="class in org.biojava.bio.structure.secstruc">SecStruc</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html#mandatoryArgs">mandatoryArgs</a></span> - Static variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce">AbstractUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/SeqRes2AtomAligner.html#mapSeqresRecords(org.biojava.bio.structure.Chain, org.biojava.bio.structure.Chain)">mapSeqresRecords(Chain, Chain)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/SeqRes2AtomAligner.html" title="class in org.biojava.bio.structure.io">SeqRes2AtomAligner</a></dt>
<dd>
<div class="block">map the seqres groups to the atomRes chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#mat">mat</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#mat">mat</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/StructureSequenceMatcher.html#matchSequenceToStructure(org.biojava3.core.sequence.ProteinSequence, org.biojava.bio.structure.Structure)">matchSequenceToStructure(ProteinSequence, Structure)</a></span> - Static method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/StructureSequenceMatcher.html" title="class in org.biojava.bio.structure.io">StructureSequenceMatcher</a></dt>
<dd>
<div class="block">Given a sequence and the corresponding Structure, get the ResidueNumber
 for each residue in the sequence.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/jama/Maths.html" title="class in org.biojava.bio.structure.jama"><span class="strong">Maths</span></a> - Class in <a href="./org/biojava/bio/structure/jama/package-summary.html">org.biojava.bio.structure.jama</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Maths.html#Maths()">Maths()</a></span> - Constructor for class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Maths.html" title="class in org.biojava.bio.structure.jama">Maths</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama"><span class="strong">Matrix</span></a> - Class in <a href="./org/biojava/bio/structure/jama/package-summary.html">org.biojava.bio.structure.jama</a></dt>
<dd>
<div class="block">Jama = Java Matrix class.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#Matrix(int, int)">Matrix(int, int)</a></span> - Constructor for class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Construct an m-by-n matrix of zeros.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#Matrix(int, int, double)">Matrix(int, int, double)</a></span> - Constructor for class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Construct an m-by-n constant matrix.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#Matrix(double[][])">Matrix(double[][])</a></span> - Constructor for class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Construct a matrix from a 2-D array.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#Matrix(double[][], int, int)">Matrix(double[][], int, int)</a></span> - Constructor for class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Construct a matrix quickly without checking arguments.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#Matrix(double[], int)">Matrix(double[], int)</a></span> - Constructor for class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Construct a matrix from a one-dimensional packed array</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#matrixFromEuler(double, double, double)">matrixFromEuler(double, double, double)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">This conversion uses NASA standard aeroplane conventions as described on page:
   http://www.euclideanspace.com/maths/geometry/rotations/euler/index.htm
   Coordinate System: right hand
   Positive angle: right hand
   Order of euler angles: heading first, then attitude, then bank.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/MatrixListener.html#matrixInOptimizer(double[][])">matrixInOptimizer(double[][])</a></span> - Method in interface org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/MatrixListener.html" title="interface in org.biojava.bio.structure.align.ce">MatrixListener</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/ce/MatrixListener.html" title="interface in org.biojava.bio.structure.align.ce"><span class="strong">MatrixListener</span></a> - Interface in <a href="./org/biojava/bio/structure/align/ce/package-summary.html">org.biojava.bio.structure.align.ce</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileParsingParameters.html#MAX_ATOMS">MAX_ATOMS</a></span> - Static variable in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileParsingParameters.html" title="class in org.biojava.bio.structure.io">FileParsingParameters</a></dt>
<dd>
<div class="block">the maximum number of atoms we will add to a structure
    this protects from memory overflows in the few really big protein structures.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html#MAX_CUTS">MAX_CUTS</a></span> - Static variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html" title="class in org.biojava.bio.structure.domain.pdp">PDPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html#MAXCONT">MAXCONT</a></span> - Static variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html" title="class in org.biojava.bio.structure.domain.pdp">PDPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html#MAXDOM">MAXDOM</a></span> - Static variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html" title="class in org.biojava.bio.structure.domain.pdp">PDPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#maxGapSize">maxGapSize</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html#MAXLEN">MAXLEN</a></span> - Static variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html" title="class in org.biojava.bio.structure.domain.pdp">PDPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#maxNrIterationsForOptimization">maxNrIterationsForOptimization</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#maxOptRMSD">maxOptRMSD</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html#MAXSIZE">MAXSIZE</a></span> - Static variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html" title="class in org.biojava.bio.structure.domain.pdp">PDPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCPMain.CPRange.html#mid">mid</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCPMain.CPRange.html" title="class in org.biojava.bio.structure.align.ce">CeCPMain.CPRange</a></dt>
<dd>
<div class="block">midpoint of the alignment</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html#MIN_DOMAIN_LENGTH">MIN_DOMAIN_LENGTH</a></span> - Static variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html" title="class in org.biojava.bio.structure.domain.pdp">PDPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStruc.html#MINDIST">MINDIST</a></span> - Static variable in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStruc.html" title="class in org.biojava.bio.structure.secstruc">SecStruc</a></dt>
<dd>
<div class="block">the minimal distance between two residues</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#minus(org.biojava.bio.structure.jama.Matrix)">minus(Matrix)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">C = A - B</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#minusEquals(org.biojava.bio.structure.jama.Matrix)">minusEquals(Matrix)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">A = A - B</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/UserConfiguration.html#MMCIF_FORMAT">MMCIF_FORMAT</a></span> - Static variable in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/UserConfiguration.html" title="class in org.biojava.bio.structure.align.util">UserConfiguration</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/quaternary/io/MmCifBiolAssemblyProvider.html" title="class in org.biojava.bio.structure.quaternary.io"><span class="strong">MmCifBiolAssemblyProvider</span></a> - Class in <a href="./org/biojava/bio/structure/quaternary/io/package-summary.html">org.biojava.bio.structure.quaternary.io</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/MmCifBiolAssemblyProvider.html#MmCifBiolAssemblyProvider()">MmCifBiolAssemblyProvider()</a></span> - Constructor for class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/MmCifBiolAssemblyProvider.html" title="class in org.biojava.bio.structure.quaternary.io">MmCifBiolAssemblyProvider</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif"><span class="strong">MMcifConsumer</span></a> - Interface in <a href="./org/biojava/bio/structure/io/mmcif/package-summary.html">org.biojava.bio.structure.io.mmcif</a></dt>
<dd>
<div class="block">An interface for the events triggered by a MMcifParser.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html" title="class in org.biojava.bio.structure.server"><span class="strong">MMCIFFileInstallation</span></a> - Class in <a href="./org/biojava/bio/structure/server/package-summary.html">org.biojava.bio.structure.server</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>&nbsp;</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html#MMCIFFileInstallation(java.io.File)">MMCIFFileInstallation(File)</a></span> - Constructor for class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html" title="class in org.biojava.bio.structure.server">MMCIFFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/MMCIFFileReader.html" title="class in org.biojava.bio.structure.io"><span class="strong">MMCIFFileReader</span></a> - Class in <a href="./org/biojava/bio/structure/io/package-summary.html">org.biojava.bio.structure.io</a></dt>
<dd>
<div class="block">How to parse an mmCif file:</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/MMCIFFileReader.html#MMCIFFileReader()">MMCIFFileReader()</a></span> - Constructor for class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/MMCIFFileReader.html" title="class in org.biojava.bio.structure.io">MMCIFFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/MMcifParser.html" title="interface in org.biojava.bio.structure.io.mmcif"><span class="strong">MMcifParser</span></a> - Interface in <a href="./org/biojava/bio/structure/io/mmcif/package-summary.html">org.biojava.bio.structure.io.mmcif</a></dt>
<dd>
<div class="block">Interface that needs to be implemented by an MMcifParser</div>
</dd>
<dt><a href="./org/biojava/bio/structure/quaternary/io/MmCifPDBBiolAssemblyProvider.html" title="class in org.biojava.bio.structure.quaternary.io"><span class="strong">MmCifPDBBiolAssemblyProvider</span></a> - Class in <a href="./org/biojava/bio/structure/quaternary/io/package-summary.html">org.biojava.bio.structure.quaternary.io</a></dt>
<dd>
<div class="block">A provider for information about biological units for PDB files that is based on reading local MMcif files.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/MmCifPDBBiolAssemblyProvider.html#MmCifPDBBiolAssemblyProvider()">MmCifPDBBiolAssemblyProvider()</a></span> - Constructor for class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/MmCifPDBBiolAssemblyProvider.html" title="class in org.biojava.bio.structure.quaternary.io">MmCifPDBBiolAssemblyProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/BioUnitDataProviderFactory.html#mmcifProviderClassName">mmcifProviderClassName</a></span> - Static variable in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/BioUnitDataProviderFactory.html" title="class in org.biojava.bio.structure.quaternary.io">BioUnitDataProviderFactory</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html" title="class in org.biojava.bio.structure.quaternary"><span class="strong">ModelTransformationMatrix</span></a> - Class in <a href="./org/biojava/bio/structure/quaternary/package-summary.html">org.biojava.bio.structure.quaternary</a></dt>
<dd>
<div class="block">Isolation issue -
 
 The original version of this (renamed as GLTransformationMatrix) uses FloatBuffer objects obtained from
 GL to store and operate on the matrices - doing so destroys the ability
 to isolate the model, because the loaders use this to generate and store biological unit and non-crystallographic
 transformations.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html#ModelTransformationMatrix()">ModelTransformationMatrix()</a></span> - Constructor for class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html" title="class in org.biojava.bio.structure.quaternary">ModelTransformationMatrix</a></dt>
<dd>
<div class="block">Default Constructor</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html#ModelTransformationMatrix(org.biojava.bio.structure.quaternary.ModelTransformationMatrix)">ModelTransformationMatrix(ModelTransformationMatrix)</a></span> - Constructor for class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html" title="class in org.biojava.bio.structure.quaternary">ModelTransformationMatrix</a></dt>
<dd>
<div class="block">Copy Constructor</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/SimpleStructureServer.html#modifiedStructure(org.biojava.bio.structure.server.StructureEvent)">modifiedStructure(StructureEvent)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/SimpleStructureServer.html" title="class in org.biojava.bio.structure.server">SimpleStructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureListener.html#modifiedStructure(org.biojava.bio.structure.server.StructureEvent)">modifiedStructure(StructureEvent)</a></span> - Method in interface org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureListener.html" title="interface in org.biojava.bio.structure.server">StructureListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html#multiply4square_x_4square2(org.biojava.bio.structure.quaternary.ModelTransformationMatrix, org.biojava.bio.structure.quaternary.ModelTransformationMatrix)">multiply4square_x_4square2(ModelTransformationMatrix, ModelTransformationMatrix)</a></span> - Static method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html" title="class in org.biojava.bio.structure.quaternary">ModelTransformationMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Mutator.html#mutate(org.biojava.bio.structure.Structure, java.lang.String, java.lang.String, java.lang.String)">mutate(Structure, String, String, String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Mutator.html" title="class in org.biojava.bio.structure">Mutator</a></dt>
<dd>
<div class="block">creates a new structure which is identical with the original one.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Mutator.html#mutateResidue(org.biojava.bio.structure.AminoAcid, java.lang.String)">mutateResidue(AminoAcid, String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Mutator.html" title="class in org.biojava.bio.structure">Mutator</a></dt>
<dd>
<div class="block">create a new residue which is of the new type.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/Mutator.html" title="class in org.biojava.bio.structure"><span class="strong">Mutator</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">A class that can change one amino acid to another.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Mutator.html#Mutator()">Mutator()</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Mutator.html" title="class in org.biojava.bio.structure">Mutator</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_N_">
<!--   -->
</a>
<h2 class="title">N</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCPMain.CPRange.html#n">n</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCPMain.CPRange.html" title="class in org.biojava.bio.structure.align.ce">CeCPMain.CPRange</a></dt>
<dd>
<div class="block">last allowed n-term</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/StructureName.html#name">name</a></span> - Variable in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/StructureName.html" title="class in org.biojava.bio.structure.align.client">StructureName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/BridgeType.html#name">name</a></span> - Variable in enum org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/BridgeType.html" title="enum in org.biojava.bio.structure.secstruc">BridgeType</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucType.html#name">name</a></span> - Variable in enum org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucType.html" title="enum in org.biojava.bio.structure.secstruc">SecStrucType</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#nAtomName">nAtomName</a></span> - Static variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#nBestTrace">nBestTrace</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#nBestTrace">nBestTrace</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#nBestTraces">nBestTraces</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#nBestTraces">nBestTraces</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html#ndbChainId">ndbChainId</a></span> - Variable in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html" title="class in org.biojava.bio.structure.quaternary">ModelTransformationMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#newAtomSite(org.biojava.bio.structure.io.mmcif.model.AtomSite)">newAtomSite(AtomSite)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#newAtomSite(org.biojava.bio.structure.io.mmcif.model.AtomSite)">newAtomSite(AtomSite)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>
<div class="block">A new AtomSite record has been read.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#newAtomSite(org.biojava.bio.structure.io.mmcif.model.AtomSite)">newAtomSite(AtomSite)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#newAuditAuthor(org.biojava.bio.structure.io.mmcif.model.AuditAuthor)">newAuditAuthor(AuditAuthor)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#newAuditAuthor(org.biojava.bio.structure.io.mmcif.model.AuditAuthor)">newAuditAuthor(AuditAuthor)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>
<div class="block">AuditAuthor contains the info from the PDB-AUTHOR records.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#newAuditAuthor(org.biojava.bio.structure.io.mmcif.model.AuditAuthor)">newAuditAuthor(AuditAuthor)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#newChemComp(org.biojava.bio.structure.io.mmcif.model.ChemComp)">newChemComp(ChemComp)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#newChemComp(org.biojava.bio.structure.io.mmcif.model.ChemComp)">newChemComp(ChemComp)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#newChemComp(org.biojava.bio.structure.io.mmcif.model.ChemComp)">newChemComp(ChemComp)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#newChemCompDescriptor(org.biojava.bio.structure.io.mmcif.model.ChemCompDescriptor)">newChemCompDescriptor(ChemCompDescriptor)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#newChemCompDescriptor(org.biojava.bio.structure.io.mmcif.model.ChemCompDescriptor)">newChemCompDescriptor(ChemCompDescriptor)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#newChemCompDescriptor(org.biojava.bio.structure.io.mmcif.model.ChemCompDescriptor)">newChemCompDescriptor(ChemCompDescriptor)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#newDatabasePDBremark(org.biojava.bio.structure.io.mmcif.model.DatabasePDBremark)">newDatabasePDBremark(DatabasePDBremark)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#newDatabasePDBremark(org.biojava.bio.structure.io.mmcif.model.DatabasePDBremark)">newDatabasePDBremark(DatabasePDBremark)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#newDatabasePDBremark(org.biojava.bio.structure.io.mmcif.model.DatabasePDBremark)">newDatabasePDBremark(DatabasePDBremark)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#newDatabasePDBrev(org.biojava.bio.structure.io.mmcif.model.DatabasePDBrev)">newDatabasePDBrev(DatabasePDBrev)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#newDatabasePDBrev(org.biojava.bio.structure.io.mmcif.model.DatabasePDBrev)">newDatabasePDBrev(DatabasePDBrev)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#newDatabasePDBrev(org.biojava.bio.structure.io.mmcif.model.DatabasePDBrev)">newDatabasePDBrev(DatabasePDBrev)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#newEntity(org.biojava.bio.structure.io.mmcif.model.Entity)">newEntity(Entity)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#newEntity(org.biojava.bio.structure.io.mmcif.model.Entity)">newEntity(Entity)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#newEntity(org.biojava.bio.structure.io.mmcif.model.Entity)">newEntity(Entity)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#newEntityPolySeq(org.biojava.bio.structure.io.mmcif.model.EntityPolySeq)">newEntityPolySeq(EntityPolySeq)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#newEntityPolySeq(org.biojava.bio.structure.io.mmcif.model.EntityPolySeq)">newEntityPolySeq(EntityPolySeq)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#newEntityPolySeq(org.biojava.bio.structure.io.mmcif.model.EntityPolySeq)">newEntityPolySeq(EntityPolySeq)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>
<div class="block">The EntityPolySeq object provide the amino acid sequence objects for the Entities.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#newExptl(org.biojava.bio.structure.io.mmcif.model.Exptl)">newExptl(Exptl)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#newExptl(org.biojava.bio.structure.io.mmcif.model.Exptl)">newExptl(Exptl)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#newExptl(org.biojava.bio.structure.io.mmcif.model.Exptl)">newExptl(Exptl)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#newGenericData(java.lang.String, java.util.List, java.util.List)">newGenericData(String, List&lt;String&gt;, List&lt;String&gt;)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#newGenericData(java.lang.String, java.util.List, java.util.List)">newGenericData(String, List&lt;String&gt;, List&lt;String&gt;)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>
<div class="block">This method is called if no particular handler for the provided cif category
 has been implemented so far.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#newGenericData(java.lang.String, java.util.List, java.util.List)">newGenericData(String, List&lt;String&gt;, List&lt;String&gt;)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/AbstractStructureAlignment.html#newline">newline</a></span> - Static variable in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/AbstractStructureAlignment.html" title="class in org.biojava.bio.structure.align">AbstractStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html#newline">newline</a></span> - Static variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce">AbstractUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCat.html#newline">newline</a></span> - Static variable in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCat.html" title="class in org.biojava.bio.structure.align.fatcat">FatCat</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#newline">newline</a></span> - Static variable in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html#newline">newline</a></span> - Static variable in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html" title="class in org.biojava.bio.structure.align.model">AfpChainWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#NEWLINE">NEWLINE</a></span> - Static variable in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#newline">newline</a></span> - Static variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#NEWLINE">NEWLINE</a></span> - Static variable in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#newPdbxEntityNonPoly(org.biojava.bio.structure.io.mmcif.model.PdbxEntityNonPoly)">newPdbxEntityNonPoly(PdbxEntityNonPoly)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#newPdbxEntityNonPoly(org.biojava.bio.structure.io.mmcif.model.PdbxEntityNonPoly)">newPdbxEntityNonPoly(PdbxEntityNonPoly)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#newPdbxEntityNonPoly(org.biojava.bio.structure.io.mmcif.model.PdbxEntityNonPoly)">newPdbxEntityNonPoly(PdbxEntityNonPoly)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#newPdbxNonPolyScheme(org.biojava.bio.structure.io.mmcif.model.PdbxNonPolyScheme)">newPdbxNonPolyScheme(PdbxNonPolyScheme)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#newPdbxNonPolyScheme(org.biojava.bio.structure.io.mmcif.model.PdbxNonPolyScheme)">newPdbxNonPolyScheme(PdbxNonPolyScheme)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#newPdbxNonPolyScheme(org.biojava.bio.structure.io.mmcif.model.PdbxNonPolyScheme)">newPdbxNonPolyScheme(PdbxNonPolyScheme)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#newPdbxPolySeqScheme(org.biojava.bio.structure.io.mmcif.model.PdbxPolySeqScheme)">newPdbxPolySeqScheme(PdbxPolySeqScheme)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#newPdbxPolySeqScheme(org.biojava.bio.structure.io.mmcif.model.PdbxPolySeqScheme)">newPdbxPolySeqScheme(PdbxPolySeqScheme)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#newPdbxPolySeqScheme(org.biojava.bio.structure.io.mmcif.model.PdbxPolySeqScheme)">newPdbxPolySeqScheme(PdbxPolySeqScheme)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#newPdbxStrucAssembly(org.biojava.bio.structure.io.mmcif.model.PdbxStructAssembly)">newPdbxStrucAssembly(PdbxStructAssembly)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#newPdbxStrucAssembly(org.biojava.bio.structure.io.mmcif.model.PdbxStructAssembly)">newPdbxStrucAssembly(PdbxStructAssembly)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#newPdbxStrucAssembly(org.biojava.bio.structure.io.mmcif.model.PdbxStructAssembly)">newPdbxStrucAssembly(PdbxStructAssembly)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#newPdbxStrucAssemblyGen(org.biojava.bio.structure.io.mmcif.model.PdbxStructAssemblyGen)">newPdbxStrucAssemblyGen(PdbxStructAssemblyGen)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#newPdbxStrucAssemblyGen(org.biojava.bio.structure.io.mmcif.model.PdbxStructAssemblyGen)">newPdbxStrucAssemblyGen(PdbxStructAssemblyGen)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#newPdbxStrucAssemblyGen(org.biojava.bio.structure.io.mmcif.model.PdbxStructAssemblyGen)">newPdbxStrucAssemblyGen(PdbxStructAssemblyGen)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#newPdbxStructOperList(org.biojava.bio.structure.io.mmcif.model.PdbxStructOperList)">newPdbxStructOperList(PdbxStructOperList)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#newPdbxStructOperList(org.biojava.bio.structure.io.mmcif.model.PdbxStructOperList)">newPdbxStructOperList(PdbxStructOperList)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#newPdbxStructOperList(org.biojava.bio.structure.io.mmcif.model.PdbxStructOperList)">newPdbxStructOperList(PdbxStructOperList)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#newRefine(org.biojava.bio.structure.io.mmcif.model.Refine)">newRefine(Refine)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#newRefine(org.biojava.bio.structure.io.mmcif.model.Refine)">newRefine(Refine)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#newRefine(org.biojava.bio.structure.io.mmcif.model.Refine)">newRefine(Refine)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#newStructAsym(org.biojava.bio.structure.io.mmcif.model.StructAsym)">newStructAsym(StructAsym)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#newStructAsym(org.biojava.bio.structure.io.mmcif.model.StructAsym)">newStructAsym(StructAsym)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#newStructAsym(org.biojava.bio.structure.io.mmcif.model.StructAsym)">newStructAsym(StructAsym)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#newStructKeywords(org.biojava.bio.structure.io.mmcif.model.StructKeywords)">newStructKeywords(StructKeywords)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#newStructKeywords(org.biojava.bio.structure.io.mmcif.model.StructKeywords)">newStructKeywords(StructKeywords)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#newStructKeywords(org.biojava.bio.structure.io.mmcif.model.StructKeywords)">newStructKeywords(StructKeywords)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#newStructRef(org.biojava.bio.structure.io.mmcif.model.StructRef)">newStructRef(StructRef)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#newStructRef(org.biojava.bio.structure.io.mmcif.model.StructRef)">newStructRef(StructRef)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#newStructRef(org.biojava.bio.structure.io.mmcif.model.StructRef)">newStructRef(StructRef)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#newStructRefSeq(org.biojava.bio.structure.io.mmcif.model.StructRefSeq)">newStructRefSeq(StructRefSeq)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#newStructRefSeq(org.biojava.bio.structure.io.mmcif.model.StructRefSeq)">newStructRefSeq(StructRefSeq)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#newStructRefSeq(org.biojava.bio.structure.io.mmcif.model.StructRefSeq)">newStructRefSeq(StructRefSeq)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>
<div class="block">create a DBRef record from the StrucRefSeq record:</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/SimpleStructureServer.html#newStructure(org.biojava.bio.structure.server.StructureEvent)">newStructure(StructureEvent)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/SimpleStructureServer.html" title="class in org.biojava.bio.structure.server">SimpleStructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureListener.html#newStructure(org.biojava.bio.structure.server.StructureEvent)">newStructure(StructureEvent)</a></span> - Method in interface org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureListener.html" title="interface in org.biojava.bio.structure.server">StructureListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomIterator.html#next()">next()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomIterator.html" title="class in org.biojava.bio.structure">AtomIterator</a></dt>
<dd>
<div class="block">Return next atom.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/GroupIterator.html#next()">next()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/GroupIterator.html" title="class in org.biojava.bio.structure">GroupIterator</a></dt>
<dd>
<div class="block">get next Group.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/FlatFileInstallation.html#next()">next()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/FlatFileInstallation.html" title="class in org.biojava.bio.structure.server">FlatFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html#next()">next()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html" title="class in org.biojava.bio.structure.server">MMCIFFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/PDBInstallation.html#next()">next()</a></span> - Method in interface org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/PDBInstallation.html" title="interface in org.biojava.bio.structure.server">PDBInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
<div class="block">iterate over all structures in this Installation that pass the provided filters and 
 return the next one in the list.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#nextStep(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">nextStep(AFPChain, Atom[], Atom[])</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#nextStep(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">nextStep(AFPChain, Atom[], Atom[])</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#nIter">nIter</a></span> - Static variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#nIter">nIter</a></span> - Static variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#nodeListFileName">nodeListFileName</a></span> - Static variable in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#norm1()">norm1()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">One norm</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#norm2()">norm2()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Two norm</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/SingularValueDecomposition.html#norm2()">norm2()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/SingularValueDecomposition.html" title="class in org.biojava.bio.structure.jama">SingularValueDecomposition</a></dt>
<dd>
<div class="block">Two norm</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#normF()">normF()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Frobenius norm</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#normInf()">normInf()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Infinity norm</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#notifyShutdown()">notifyShutdown()</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>
<div class="block">Send a signal to the cache that the system is shutting down.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/RemoteBioUnitDataProvider.html#NR_BIOL_APPEND">NR_BIOL_APPEND</a></span> - Static variable in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/RemoteBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">RemoteBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/RemoteRawBioUnitDataProvider.html#NR_BIOL_APPEND">NR_BIOL_APPEND</a></span> - Static variable in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/RemoteRawBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">RemoteRawBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#nrModels()">nrModels()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">return number of models .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#nrModels()">nrModels()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">return number of models.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#nTrace">nTrace</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#nTrace">nTrace</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#nTraces">nTraces</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#nTraces">nTraces</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/GroupType.html#NUCLEOTIDE">NUCLEOTIDE</a></span> - Static variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/GroupType.html" title="class in org.biojava.bio.structure">GroupType</a></dt>
<dd>
<div class="block">the type for nucelotide groups</div>
</dd>
<dt><a href="./org/biojava/bio/structure/NucleotideImpl.html" title="class in org.biojava.bio.structure"><span class="strong">NucleotideImpl</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">A nucleotide group is almost the same as a Hetatm group.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/NucleotideImpl.html#NucleotideImpl()">NucleotideImpl()</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/NucleotideImpl.html" title="class in org.biojava.bio.structure">NucleotideImpl</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_O_">
<!--   -->
</a>
<h2 class="title">O</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#oAtomName">oAtomName</a></span> - Static variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/SimpleStructureServer.html#obsoleteStructure(org.biojava.bio.structure.server.StructureEvent)">obsoleteStructure(StructureEvent)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/SimpleStructureServer.html" title="class in org.biojava.bio.structure.server">SimpleStructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureListener.html#obsoleteStructure(org.biojava.bio.structure.server.StructureEvent)">obsoleteStructure(StructureEvent)</a></span> - Method in interface org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureListener.html" title="interface in org.biojava.bio.structure.server">StructureListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/HTTPConnectionTools.html#openHttpURLConnection(java.net.URL, int)">openHttpURLConnection(URL, int)</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/HTTPConnectionTools.html" title="class in org.biojava.bio.structure.align.util">HTTPConnectionTools</a></dt>
<dd>
<div class="block">open HttpURLConnection.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/HTTPConnectionTools.html#openHttpURLConnection(java.net.URL)">openHttpURLConnection(URL)</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/HTTPConnectionTools.html" title="class in org.biojava.bio.structure.align.util">HTTPConnectionTools</a></dt>
<dd>
<div class="block">open HttpURLConnection.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/quaternary/OperatorResolver.html" title="class in org.biojava.bio.structure.quaternary"><span class="strong">OperatorResolver</span></a> - Class in <a href="./org/biojava/bio/structure/quaternary/package-summary.html">org.biojava.bio.structure.quaternary</a></dt>
<dd>
<div class="block">A class to resolve the operators for transformations</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/OperatorResolver.html#OperatorResolver()">OperatorResolver()</a></span> - Constructor for class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/OperatorResolver.html" title="class in org.biojava.bio.structure.quaternary">OperatorResolver</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/ce/OptimalCECPMain.html" title="class in org.biojava.bio.structure.align.ce"><span class="strong">OptimalCECPMain</span></a> - Class in <a href="./org/biojava/bio/structure/align/ce/package-summary.html">org.biojava.bio.structure.align.ce</a></dt>
<dd>
<div class="block">A wrapper for <a href="./org/biojava/bio/structure/align/ce/CeMain.html" title="class in org.biojava.bio.structure.align.ce"><code>CeMain</code></a> which sets default parameters to be appropriate for finding
 circular permutations.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/OptimalCECPMain.html#OptimalCECPMain()">OptimalCECPMain()</a></span> - Constructor for class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/OptimalCECPMain.html" title="class in org.biojava.bio.structure.align.ce">OptimalCECPMain</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/ce/OptimalCECPParameters.html" title="class in org.biojava.bio.structure.align.ce"><span class="strong">OptimalCECPParameters</span></a> - Class in <a href="./org/biojava/bio/structure/align/ce/package-summary.html">org.biojava.bio.structure.align.ce</a></dt>
<dd>
<div class="block">Contains the parameters that can be sent to CE</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/OptimalCECPParameters.html#OptimalCECPParameters()">OptimalCECPParameters()</a></span> - Constructor for class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/OptimalCECPParameters.html" title="class in org.biojava.bio.structure.align.ce">OptimalCECPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/AFPOptimizer.html#optimizeAln(org.biojava.bio.structure.align.fatcat.calc.FatCatParameters, org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">optimizeAln(FatCatParameters, AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/AFPOptimizer.html" title="class in org.biojava.bio.structure.align.fatcat.calc">AFPOptimizer</a></dt>
<dd>
<div class="block">//optimize the alignment by dynamic programming</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/StructureAlignmentOptimizer.html#optimizeResult(int[], int, int[][])">optimizeResult(int[], int, int[][])</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/StructureAlignmentOptimizer.html" title="class in org.biojava.bio.structure.align.fatcat.calc">StructureAlignmentOptimizer</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/quaternary/OrderedPair.html" title="class in org.biojava.bio.structure.quaternary"><span class="strong">OrderedPair</span></a>&lt;<a href="./org/biojava/bio/structure/quaternary/OrderedPair.html" title="type parameter in OrderedPair">T</a>&gt; - Class in <a href="./org/biojava/bio/structure/quaternary/package-summary.html">org.biojava.bio.structure.quaternary</a></dt>
<dd>
<div class="block">An ordered pair represents a component of a cartesian product.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a> - package org.biojava.bio.structure</dt>
<dd>
<div class="block">
Interfaces and classes for protein structure (PDB).</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/package-summary.html">org.biojava.bio.structure.align</a> - package org.biojava.bio.structure.align</dt>
<dd>
<div class="block">Classes for the alignment of structures.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/ce/package-summary.html">org.biojava.bio.structure.align.ce</a> - package org.biojava.bio.structure.align.ce</dt>
<dd>
<div class="block">Classes related to the implementation of the CE alignment algorithm, here called jCE.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/client/package-summary.html">org.biojava.bio.structure.align.client</a> - package org.biojava.bio.structure.align.client</dt>
<dd>
<div class="block">This package deals with the server communication for auto-downloading pre-calculated alignments.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/events/package-summary.html">org.biojava.bio.structure.align.events</a> - package org.biojava.bio.structure.align.events</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/fatcat/package-summary.html">org.biojava.bio.structure.align.fatcat</a> - package org.biojava.bio.structure.align.fatcat</dt>
<dd>
<div class="block">Classes related to the implementation of the FATCAT alignment algorithm, here called jFATCAT.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/fatcat/calc/package-summary.html">org.biojava.bio.structure.align.fatcat.calc</a> - package org.biojava.bio.structure.align.fatcat.calc</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/helper/package-summary.html">org.biojava.bio.structure.align.helper</a> - package org.biojava.bio.structure.align.helper</dt>
<dd>
<div class="block">Helper classes for structural alignment.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/model/package-summary.html">org.biojava.bio.structure.align.model</a> - package org.biojava.bio.structure.align.model</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/pairwise/package-summary.html">org.biojava.bio.structure.align.pairwise</a> - package org.biojava.bio.structure.align.pairwise</dt>
<dd>
<div class="block">Classes for the pairwise alignment of structures.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/seq/package-summary.html">org.biojava.bio.structure.align.seq</a> - package org.biojava.bio.structure.align.seq</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/util/package-summary.html">org.biojava.bio.structure.align.util</a> - package org.biojava.bio.structure.align.util</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/xml/package-summary.html">org.biojava.bio.structure.align.xml</a> - package org.biojava.bio.structure.align.xml</dt>
<dd>
<div class="block">Serialization and deserialization for structure alignment results (as AFPChain object) to XML.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/cath/package-summary.html">org.biojava.bio.structure.cath</a> - package org.biojava.bio.structure.cath</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/domain/package-summary.html">org.biojava.bio.structure.domain</a> - package org.biojava.bio.structure.domain</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/domain/pdp/package-summary.html">org.biojava.bio.structure.domain.pdp</a> - package org.biojava.bio.structure.domain.pdp</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/package-summary.html">org.biojava.bio.structure.io</a> - package org.biojava.bio.structure.io</dt>
<dd>
<div class="block">Input and Output of Structures</div>
</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/package-summary.html">org.biojava.bio.structure.io.mmcif</a> - package org.biojava.bio.structure.io.mmcif</dt>
<dd>
<div class="block">Input and Output of mmcif files</div>
</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/chem/package-summary.html">org.biojava.bio.structure.io.mmcif.chem</a> - package org.biojava.bio.structure.io.mmcif.chem</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a> - package org.biojava.bio.structure.io.mmcif.model</dt>
<dd>
<div class="block">Datamodel objects used for processing mmcif files.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/io/sifts/package-summary.html">org.biojava.bio.structure.io.sifts</a> - package org.biojava.bio.structure.io.sifts</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/util/package-summary.html">org.biojava.bio.structure.io.util</a> - package org.biojava.bio.structure.io.util</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/jama/package-summary.html">org.biojava.bio.structure.jama</a> - package org.biojava.bio.structure.jama</dt>
<dd>
<div class="block">Matrix package for from <a href="http://math.nist.gov/javanumerics/jama/">JAMA</a></div>
</dd>
<dt><a href="./org/biojava/bio/structure/quaternary/package-summary.html">org.biojava.bio.structure.quaternary</a> - package org.biojava.bio.structure.quaternary</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/quaternary/io/package-summary.html">org.biojava.bio.structure.quaternary.io</a> - package org.biojava.bio.structure.quaternary.io</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/scop/package-summary.html">org.biojava.bio.structure.scop</a> - package org.biojava.bio.structure.scop</dt>
<dd>
<div class="block">Parsers and API for <a href="http://scop.mrc-lmb.cam.ac.uk/scop/">SCOP, Structural Classification of Proteins</a>.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/scop/server/package-summary.html">org.biojava.bio.structure.scop.server</a> - package org.biojava.bio.structure.scop.server</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/secstruc/package-summary.html">org.biojava.bio.structure.secstruc</a> - package org.biojava.bio.structure.secstruc</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/server/package-summary.html">org.biojava.bio.structure.server</a> - package org.biojava.bio.structure.server</dt>
<dd>
<div class="block">
classes for easier management of PDB installations

As of BioJava 3.0.2 this package is deprecated.</div>
</dd>
<dt><a href="./org/biojava3/structure/package-summary.html">org.biojava3.structure</a> - package org.biojava3.structure</dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#oRmsdThr">oRmsdThr</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_P_">
<!--   -->
</a>
<h2 class="title">P</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html#params">params</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce">AbstractUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#params">params</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#params">params</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeMain.html#params">params</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeMain.html" title="class in org.biojava.bio.structure.align.ce">CeMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/OptimalCECPMain.html#params">params</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/OptimalCECPMain.html" title="class in org.biojava.bio.structure.align.ce">OptimalCECPMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/OptimalCECPParameters.html#params">params</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/OptimalCECPParameters.html" title="class in org.biojava.bio.structure.align.ce">OptimalCECPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifParser.html#parse(java.io.BufferedReader)">parse(BufferedReader)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifParser.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifParser</a></dt>
<dd>
<div class="block">Start the actual parsing.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifParser.html#parse(java.io.InputStream)">parse(InputStream)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifParser.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifParser</a></dt>
<dd>
<div class="block">Start the actual parsing.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#parse(java.io.InputStream)">parse(InputStream)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#parse(java.io.BufferedReader)">parse(BufferedReader)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/BioAssemblyTools.html#parseBinaryOperatorExpression(java.lang.String)">parseBinaryOperatorExpression(String)</a></span> - Static method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/BioAssemblyTools.html" title="class in org.biojava.bio.structure.quaternary">BioAssemblyTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/AFPChainXMLParser.html#parseMultiXML(java.lang.String)">parseMultiXML(String)</a></span> - Static method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/AFPChainXMLParser.html" title="class in org.biojava.bio.structure.align.xml">AFPChainXMLParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/OperatorResolver.html#parseOperatorExpressionString(java.lang.String)">parseOperatorExpressionString(String)</a></span> - Method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/OperatorResolver.html" title="class in org.biojava.bio.structure.quaternary">OperatorResolver</a></dt>
<dd>
<div class="block">Parses the operator expression and save the operators as a list
 of unary or binary operators (i.e.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileParser.html#parsePDBFile(java.io.InputStream)">parsePDBFile(InputStream)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileParser.html" title="class in org.biojava.bio.structure.io">PDBFileParser</a></dt>
<dd>
<div class="block">parse a PDB file and return a datastructure implementing
 PDBStructure interface.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileParser.html#parsePDBFile(java.io.BufferedReader)">parsePDBFile(BufferedReader)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileParser.html" title="class in org.biojava.bio.structure.io">PDBFileParser</a></dt>
<dd>
<div class="block">parse a PDB file and return a datastructure implementing
 PDBStructure interface.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/BioAssemblyTools.html#parseUnaryOperatorExpression(java.lang.String)">parseUnaryOperatorExpression(String)</a></span> - Static method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/BioAssemblyTools.html" title="class in org.biojava.bio.structure.quaternary">BioAssemblyTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsXMLParser.html#parseXmlFile(java.io.InputStream)">parseXmlFile(InputStream)</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsXMLParser.html" title="class in org.biojava.bio.structure.io.sifts">SiftsXMLParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileParser.html#PDB_AUTHOR_ASSIGNMENT">PDB_AUTHOR_ASSIGNMENT</a></span> - Static variable in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileParser.html" title="class in org.biojava.bio.structure.io">PDBFileParser</a></dt>
<dd>
<div class="block">Secondary strucuture assigned by the PDB author/</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html#PDB_DIR">PDB_DIR</a></span> - Static variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce">AbstractUserArgumentProcessor</a></dt>
<dd>
<div class="block">the system property PDB_DIR can be used to configure the 
 default location for PDB files.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/UserConfiguration.html#PDB_DIR">PDB_DIR</a></span> - Static variable in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/UserConfiguration.html" title="class in org.biojava.bio.structure.align.util">UserConfiguration</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#PDB_FILE_SERVER_PROPERTY">PDB_FILE_SERVER_PROPERTY</a></span> - Static variable in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#pdb_flag">pdb_flag</a></span> - Variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/UserConfiguration.html#PDB_FORMAT">PDB_FORMAT</a></span> - Static variable in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/UserConfiguration.html" title="class in org.biojava.bio.structure.align.util">UserConfiguration</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#pdb_name">pdb_name</a></span> - Variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBBioAssemblyParser.html#pdb_REMARK_350_Handler(java.lang.String)">pdb_REMARK_350_Handler(String)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBBioAssemblyParser.html" title="class in org.biojava.bio.structure.io">PDBBioAssemblyParser</a></dt>
<dd>
<div class="block">See here for the current spec:
 
 http://www.wwpdb.org/documentation/format33/remarks2.html</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBStatus.html#PDB_SERVER_PROPERTY">PDB_SERVER_PROPERTY</a></span> - Static variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBStatus.html" title="class in org.biojava.bio.structure">PDBStatus</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/PDBBioAssemblyParser.html" title="class in org.biojava.bio.structure.io"><span class="strong">PDBBioAssemblyParser</span></a> - Class in <a href="./org/biojava/bio/structure/io/package-summary.html">org.biojava.bio.structure.io</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBBioAssemblyParser.html#PDBBioAssemblyParser()">PDBBioAssemblyParser()</a></span> - Constructor for class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBBioAssemblyParser.html" title="class in org.biojava.bio.structure.io">PDBBioAssemblyParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/quaternary/io/PDBBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io"><span class="strong">PDBBioUnitDataProvider</span></a> - Class in <a href="./org/biojava/bio/structure/quaternary/io/package-summary.html">org.biojava.bio.structure.quaternary.io</a></dt>
<dd>
<div class="block">A BioUnitDataProvider that extracts the necessary info from PDB files</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/PDBBioUnitDataProvider.html#PDBBioUnitDataProvider()">PDBBioUnitDataProvider()</a></span> - Constructor for class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/PDBBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">PDBBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/PDBCrystallographicInfo.html" title="class in org.biojava.bio.structure"><span class="strong">PDBCrystallographicInfo</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">A class to hold crystallographic information about a PDB structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBCrystallographicInfo.html#PDBCrystallographicInfo()">PDBCrystallographicInfo()</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBCrystallographicInfo.html" title="class in org.biojava.bio.structure">PDBCrystallographicInfo</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/domain/PDBDomainProvider.html" title="class in org.biojava.bio.structure.domain"><span class="strong">PDBDomainProvider</span></a> - Class in <a href="./org/biojava/bio/structure/domain/package-summary.html">org.biojava.bio.structure.domain</a></dt>
<dd>
<div class="block">Class to fetch domains through the RCSB's REST API.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/PDBDomainProvider.html#PDBDomainProvider()">PDBDomainProvider()</a></span> - Constructor for class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/PDBDomainProvider.html" title="class in org.biojava.bio.structure.domain">PDBDomainProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/PDBDomainProvider.html#PDBDomainProvider(java.lang.String, int)">PDBDomainProvider(String, int)</a></span> - Constructor for class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/PDBDomainProvider.html" title="class in org.biojava.bio.structure.domain">PDBDomainProvider</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/PDBFileParser.html" title="class in org.biojava.bio.structure.io"><span class="strong">PDBFileParser</span></a> - Class in <a href="./org/biojava/bio/structure/io/package-summary.html">org.biojava.bio.structure.io</a></dt>
<dd>
<div class="block">This class implements the actual PDB file parsing.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileParser.html#PDBFileParser()">PDBFileParser()</a></span> - Constructor for class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileParser.html" title="class in org.biojava.bio.structure.io">PDBFileParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io"><span class="strong">PDBFileReader</span></a> - Class in <a href="./org/biojava/bio/structure/io/package-summary.html">org.biojava.bio.structure.io</a></dt>
<dd>
<div class="block">
  The wrapper class for parsing a PDB file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#PDBFileReader()">PDBFileReader()</a></span> - Constructor for class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/server/PDBFilter.html" title="interface in org.biojava.bio.structure.server"><span class="strong">PDBFilter</span></a> - Interface in <a href="./org/biojava/bio/structure/server/package-summary.html">org.biojava.bio.structure.server</a></dt>
<dd>
<div class="block">defines how a filter of PDB files has to behave</div>
</dd>
<dt><a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure"><span class="strong">PDBHeader</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">A class that contains PDB Header information.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#PDBHeader()">PDBHeader()</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/StructureName.html#pdbId">pdbId</a></span> - Variable in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/StructureName.html" title="class in org.biojava.bio.structure.align.client">StructureName</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/server/PDBInstallation.html" title="interface in org.biojava.bio.structure.server"><span class="strong">PDBInstallation</span></a> - Interface in <a href="./org/biojava/bio/structure/server/package-summary.html">org.biojava.bio.structure.server</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>&nbsp;</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#pdbNumRangeRegex">pdbNumRangeRegex</a></span> - Static variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>
<div class="block">Pattern to describe subranges.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/client/PdbPair.html" title="class in org.biojava.bio.structure.align.client"><span class="strong">PdbPair</span></a> - Class in <a href="./org/biojava/bio/structure/align/client/package-summary.html">org.biojava.bio.structure.align.client</a></dt>
<dd>
<div class="block">A pair for structure alignment</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/PdbPair.html#PdbPair(java.lang.String, java.lang.String)">PdbPair(String, String)</a></span> - Constructor for class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/PdbPair.html" title="class in org.biojava.bio.structure.align.client">PdbPair</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/xml/PdbPairsMessage.html" title="class in org.biojava.bio.structure.align.xml"><span class="strong">PdbPairsMessage</span></a> - Class in <a href="./org/biojava/bio/structure/align/xml/package-summary.html">org.biojava.bio.structure.align.xml</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/PdbPairsMessage.html#PdbPairsMessage()">PdbPairsMessage()</a></span> - Constructor for class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/PdbPairsMessage.html" title="class in org.biojava.bio.structure.align.xml">PdbPairsMessage</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/xml/PdbPairXMLConverter.html" title="class in org.biojava.bio.structure.align.xml"><span class="strong">PdbPairXMLConverter</span></a> - Class in <a href="./org/biojava/bio/structure/align/xml/package-summary.html">org.biojava.bio.structure.align.xml</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/PdbPairXMLConverter.html#PdbPairXMLConverter()">PdbPairXMLConverter()</a></span> - Constructor for class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/PdbPairXMLConverter.html" title="class in org.biojava.bio.structure.align.xml">PdbPairXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/PDBParseException.html" title="class in org.biojava.bio.structure.io"><span class="strong">PDBParseException</span></a> - Exception in <a href="./org/biojava/bio/structure/io/package-summary.html">org.biojava.bio.structure.io</a></dt>
<dd>
<div class="block">An exception during the parsing of a PDB file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBParseException.html#PDBParseException(java.lang.String)">PDBParseException(String)</a></span> - Constructor for exception org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBParseException.html" title="class in org.biojava.bio.structure.io">PDBParseException</a></dt>
<dd>
<div class="block">Constructs a PDBParseException object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBParseException.html#PDBParseException(java.lang.String, java.lang.Throwable)">PDBParseException(String, Throwable)</a></span> - Constructor for exception org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBParseException.html" title="class in org.biojava.bio.structure.io">PDBParseException</a></dt>
<dd>
<div class="block">Constructs a PDBParseException object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBParseException.html#PDBParseException(java.lang.Throwable)">PDBParseException(Throwable)</a></span> - Constructor for exception org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBParseException.html" title="class in org.biojava.bio.structure.io">PDBParseException</a></dt>
<dd>
<div class="block">Constructs a PDBParseException object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/BioUnitDataProviderFactory.html#pdbProviderClassName">pdbProviderClassName</a></span> - Static variable in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/BioUnitDataProviderFactory.html" title="class in org.biojava.bio.structure.quaternary.io">BioUnitDataProviderFactory</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/PDBRecord.html" title="interface in org.biojava.bio.structure"><span class="strong">PDBRecord</span></a> - Interface in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">An interface implemented by all classes that represent PDB records</div>
</dd>
<dt><a href="./org/biojava/bio/structure/io/PDBSRSReader.html" title="class in org.biojava.bio.structure.io"><span class="strong">PDBSRSReader</span></a> - Class in <a href="./org/biojava/bio/structure/io/package-summary.html">org.biojava.bio.structure.io</a></dt>
<dd>
<div class="block">reads a PDB file from a local SRS installation using getz Actually
 is the same as PDBFileReader, but instead of reading from a file stream, reads from a
 buffered stream.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBSRSReader.html#PDBSRSReader()">PDBSRSReader()</a></span> - Constructor for class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBSRSReader.html" title="class in org.biojava.bio.structure.io">PDBSRSReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/PDBStatus.html" title="class in org.biojava.bio.structure"><span class="strong">PDBStatus</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">Methods for getting the status of a PDB file (current, obsolete, etc)
 and for accessing different versions of the structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBStatus.html#PDBStatus()">PDBStatus()</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBStatus.html" title="class in org.biojava.bio.structure">PDBStatus</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/PDBStatus.Status.html" title="enum in org.biojava.bio.structure"><span class="strong">PDBStatus.Status</span></a> - Enum in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">Represents the status of PDB IDs.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxEntityNonPoly.html" title="class in org.biojava.bio.structure.io.mmcif.model"><span class="strong">PdbxEntityNonPoly</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a></dt>
<dd>
<div class="block">A bean for the Pdbx_entity_nonpoly category.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxEntityNonPoly.html#PdbxEntityNonPoly()">PdbxEntityNonPoly()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxEntityNonPoly.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxEntityNonPoly</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model"><span class="strong">PdbxNonPolyScheme</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a></dt>
<dd>
<div class="block">A bean for the PDBX_NONPOLY_SCHEME category, which provides residue level nomenclature
 mapping for non-polymer entities.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#PdbxNonPolyScheme()">PdbxNonPolyScheme()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model"><span class="strong">PdbxPolySeqScheme</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a></dt>
<dd>
<div class="block">A bean for the PDBX_POLY_SEQ_SCHEME category, which provides residue level nomenclature
 mapping for polymer entities.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#PdbxPolySeqScheme()">PdbxPolySeqScheme()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssembly.html" title="class in org.biojava.bio.structure.io.mmcif.model"><span class="strong">PdbxStructAssembly</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssembly.html#PdbxStructAssembly()">PdbxStructAssembly()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssembly.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructAssembly</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssemblyGen.html" title="class in org.biojava.bio.structure.io.mmcif.model"><span class="strong">PdbxStructAssemblyGen</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssemblyGen.html#PdbxStructAssemblyGen()">PdbxStructAssemblyGen()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssemblyGen.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructAssemblyGen</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssemblyGenXMLContainer.html" title="class in org.biojava.bio.structure.io.mmcif.model"><span class="strong">PdbxStructAssemblyGenXMLContainer</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssemblyGenXMLContainer.html#PdbxStructAssemblyGenXMLContainer()">PdbxStructAssemblyGenXMLContainer()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssemblyGenXMLContainer.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructAssemblyGenXMLContainer</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssemblyXMLContainer.html" title="class in org.biojava.bio.structure.io.mmcif.model"><span class="strong">PdbxStructAssemblyXMLContainer</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssemblyXMLContainer.html#PdbxStructAssemblyXMLContainer()">PdbxStructAssemblyXMLContainer()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssemblyXMLContainer.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructAssemblyXMLContainer</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html" title="class in org.biojava.bio.structure.io.mmcif.model"><span class="strong">PdbxStructOperList</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html#PdbxStructOperList()">PdbxStructOperList()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructOperList</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperListXMLContainer.html" title="class in org.biojava.bio.structure.io.mmcif.model"><span class="strong">PdbxStructOperListXMLContainer</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperListXMLContainer.html#PdbxStructOperListXMLContainer()">PdbxStructOperListXMLContainer()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperListXMLContainer.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructOperListXMLContainer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#PDP_DOMAIN_IDENTIFIER">PDP_DOMAIN_IDENTIFIER</a></span> - Static variable in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/domain/pdp/PDPDistanceMatrix.html" title="class in org.biojava.bio.structure.domain.pdp"><span class="strong">PDPDistanceMatrix</span></a> - Class in <a href="./org/biojava/bio/structure/domain/pdp/package-summary.html">org.biojava.bio.structure.domain.pdp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPDistanceMatrix.html#PDPDistanceMatrix()">PDPDistanceMatrix()</a></span> - Constructor for class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPDistanceMatrix.html" title="class in org.biojava.bio.structure.domain.pdp">PDPDistanceMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html" title="class in org.biojava.bio.structure.domain.pdp"><span class="strong">PDPParameters</span></a> - Class in <a href="./org/biojava/bio/structure/domain/pdp/package-summary.html">org.biojava.bio.structure.domain.pdp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html#PDPParameters()">PDPParameters()</a></span> - Constructor for class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html" title="class in org.biojava.bio.structure.domain.pdp">PDPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#pdpprovider">pdpprovider</a></span> - Variable in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/domain/PDPProvider.html" title="interface in org.biojava.bio.structure.domain"><span class="strong">PDPProvider</span></a> - Interface in <a href="./org/biojava/bio/structure/domain/package-summary.html">org.biojava.bio.structure.domain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#plus(org.biojava.bio.structure.Structure, org.biojava.bio.structure.jama.Matrix)">plus(Structure, Matrix)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">calculate structure + Matrix coodinates ...</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#plus(org.biojava.bio.structure.jama.Matrix)">plus(Matrix)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">C = A + B</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#plusEquals(org.biojava.bio.structure.jama.Matrix)">plusEquals(Matrix)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">A = A + B</div>
</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/chem/PolymerType.html" title="enum in org.biojava.bio.structure.io.mmcif.chem"><span class="strong">PolymerType</span></a> - Enum in <a href="./org/biojava/bio/structure/io/mmcif/chem/package-summary.html">org.biojava.bio.structure.io.mmcif.chem</a></dt>
<dd>
<div class="block">Enumerates the classification of polymers.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/chem/ResidueType.html#polymerType">polymerType</a></span> - Variable in enum org.biojava.bio.structure.io.mmcif.chem.<a href="./org/biojava/bio/structure/io/mmcif/chem/ResidueType.html" title="enum in org.biojava.bio.structure.io.mmcif.chem">ResidueType</a></dt>
<dd>
<div class="block">The associated <a href="./org/biojava/bio/structure/io/mmcif/chem/PolymerType.html" title="enum in org.biojava.bio.structure.io.mmcif.chem"><code>PolymerType</code></a></div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/chem/PolymerType.html#polymerTypeFromString(java.lang.String)">polymerTypeFromString(String)</a></span> - Static method in enum org.biojava.bio.structure.io.mmcif.chem.<a href="./org/biojava/bio/structure/io/mmcif/chem/PolymerType.html" title="enum in org.biojava.bio.structure.io.mmcif.chem">PolymerType</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/chem/PolymerType.html#POLYNUCLEOTIDE_ONLY">POLYNUCLEOTIDE_ONLY</a></span> - Static variable in enum org.biojava.bio.structure.io.mmcif.chem.<a href="./org/biojava/bio/structure/io/mmcif/chem/PolymerType.html" title="enum in org.biojava.bio.structure.io.mmcif.chem">PolymerType</a></dt>
<dd>
<div class="block">Convenience <tt>Set</tt> of polymer types classified as DNA.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/xml/PositionInQueueXMLConverter.html" title="class in org.biojava.bio.structure.align.xml"><span class="strong">PositionInQueueXMLConverter</span></a> - Class in <a href="./org/biojava/bio/structure/align/xml/package-summary.html">org.biojava.bio.structure.align.xml</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/PositionInQueueXMLConverter.html#PositionInQueueXMLConverter()">PositionInQueueXMLConverter()</a></span> - Constructor for class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/PositionInQueueXMLConverter.html" title="class in org.biojava.bio.structure.align.xml">PositionInQueueXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/AFPPostProcessor.html#postProcess(org.biojava.bio.structure.align.fatcat.calc.FatCatParameters, org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">postProcess(FatCatParameters, AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/AFPPostProcessor.html" title="class in org.biojava.bio.structure.align.fatcat.calc">AFPPostProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCPMain.html#postProcessAlignment(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], org.biojava.bio.structure.align.ce.CECalculator)">postProcessAlignment(AFPChain, Atom[], Atom[], CECalculator)</a></span> - Static method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCPMain.html" title="class in org.biojava.bio.structure.align.ce">CeCPMain</a></dt>
<dd>
<div class="block">Circular permutation specific code to be run after the standard CE alignment</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/GuiWrapper.html#prepareGroupsForDisplay(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">prepareGroupsForDisplay(AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/GuiWrapper.html" title="class in org.biojava.bio.structure.align.ce">GuiWrapper</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/server/PrepareIndexFile.html" title="class in org.biojava.bio.structure.server"><span class="strong">PrepareIndexFile</span></a> - Class in <a href="./org/biojava/bio/structure/server/package-summary.html">org.biojava.bio.structure.server</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/PrepareIndexFile.html#PrepareIndexFile()">PrepareIndexFile()</a></span> - Constructor for class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/PrepareIndexFile.html" title="class in org.biojava.bio.structure.server">PrepareIndexFile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/PrepareIndexFile.html#prepareIndexFileForInstallation(org.biojava.bio.structure.server.FlatFileInstallation)">prepareIndexFileForInstallation(FlatFileInstallation)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/PrepareIndexFile.html" title="class in org.biojava.bio.structure.server">PrepareIndexFile</a></dt>
<dd>
<div class="block">prepare the index file for this installation</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/PrepareMMcifIndexFile.html#prepareIndexFileForInstallation(org.biojava.bio.structure.server.FlatFileInstallation)">prepareIndexFileForInstallation(FlatFileInstallation)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/PrepareMMcifIndexFile.html" title="class in org.biojava.bio.structure.server">PrepareMMcifIndexFile</a></dt>
<dd>
<div class="block">prepare the index file for this installation</div>
</dd>
<dt><a href="./org/biojava/bio/structure/server/PrepareMMcifIndexFile.html" title="class in org.biojava.bio.structure.server"><span class="strong">PrepareMMcifIndexFile</span></a> - Class in <a href="./org/biojava/bio/structure/server/package-summary.html">org.biojava.bio.structure.server</a></dt>
<dd>
<div class="block">A class to index all files in a PDB installation</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/PrepareMMcifIndexFile.html#PrepareMMcifIndexFile()">PrepareMMcifIndexFile()</a></span> - Constructor for class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/PrepareMMcifIndexFile.html" title="class in org.biojava.bio.structure.server">PrepareMMcifIndexFile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#print(int, int)">print(int, int)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Print the matrix to stdout.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#print(java.io.PrintWriter, int, int)">print(PrintWriter, int, int)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Print the matrix to the output stream.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#print(java.text.NumberFormat, int)">print(NumberFormat, int)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Print the matrix to stdout.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#print(java.io.PrintWriter, java.text.NumberFormat, int)">print(PrintWriter, NumberFormat, int)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Print the matrix to the output stream.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html#printAboutMe()">printAboutMe()</a></span> - Static method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce">AbstractUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoSCOP.html#printDomainsForPDB()">printDomainsForPDB()</a></span> - Method in class demo.<a href="./demo/DemoSCOP.html" title="class in demo">DemoSCOP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/AbstractStructureAlignment.html#printHelp()">printHelp()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/AbstractStructureAlignment.html" title="class in org.biojava.bio.structure.align">AbstractStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/BioJavaStructureAlignment.html#printHelp()">printHelp()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/BioJavaStructureAlignment.html" title="class in org.biojava.bio.structure.align">BioJavaStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/FarmJob.html#printHelp()">printHelp()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/FarmJob.html" title="class in org.biojava.bio.structure.align">FarmJob</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCat.html#printHelp()">printHelp()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCat.html" title="class in org.biojava.bio.structure.align.fatcat">FatCat</a></dt>
<dd>
<div class="block">print the -h help text.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructureAlignment.html#printHelp()">printHelp()</a></span> - Method in interface org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructureAlignment.html" title="interface in org.biojava.bio.structure.align">StructureAlignment</a></dt>
<dd>
<div class="block">Returns some documentation on the command line arguments for this algorithm.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html#printMatrix(java.lang.String)">printMatrix(String)</a></span> - Method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html" title="class in org.biojava.bio.structure.quaternary">ModelTransformationMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SVDSuperimposer.html#printMatrix(org.biojava.bio.structure.jama.Matrix)">printMatrix(Matrix)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SVDSuperimposer.html" title="class in org.biojava.bio.structure">SVDSuperimposer</a></dt>
<dd>
<div class="block">Simple debug method to print a Matrix object on System.out.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatAligner.html#printTimeStamps">printTimeStamps</a></span> - Static variable in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatAligner.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/AFPChainXMLConverter.html#printXMLEQRInferPositions(org.biojava3.core.util.PrettyXMLWriter, org.biojava.bio.structure.align.model.AFPChain, int, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">printXMLEQRInferPositions(PrettyXMLWriter, AFPChain, int, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/AFPChainXMLConverter.html" title="class in org.biojava.bio.structure.align.xml">AFPChainXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/AFPChainXMLConverter.html#printXMLHeader(org.biojava3.core.util.PrettyXMLWriter, org.biojava.bio.structure.align.model.AFPChain)">printXMLHeader(PrettyXMLWriter, AFPChain)</a></span> - Static method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/AFPChainXMLConverter.html" title="class in org.biojava.bio.structure.align.xml">AFPChainXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html#process(java.lang.String[])">process(String[])</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce">AbstractUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/UserArgumentProcessor.html#process(java.lang.String[])">process(String[])</a></span> - Method in interface org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/UserArgumentProcessor.html" title="interface in org.biojava.bio.structure.align.ce">UserArgumentProcessor</a></dt>
<dd>
<div class="block">Process user arguments that have been provided from the command line</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FastaStructureParser.html#process()">process()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FastaStructureParser.html" title="class in org.biojava.bio.structure.io">FastaStructureParser</a></dt>
<dd>
<div class="block">Parses the fasta file and loads it into memory.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/chem/PolymerType.html#PROTEIN_ONLY">PROTEIN_ONLY</a></span> - Static variable in enum org.biojava.bio.structure.io.mmcif.chem.<a href="./org/biojava/bio/structure/io/mmcif/chem/PolymerType.html" title="enum in org.biojava.bio.structure.io.mmcif.chem">PolymerType</a></dt>
<dd>
<div class="block">Convenience <tt>Set</tt> of polymer types classified as protein.</div>
</dd>
</dl>
<a name="_Q_">
<!--   -->
</a>
<h2 class="title">Q</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStruc.html#Q">Q</a></span> - Static variable in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStruc.html" title="class in org.biojava.bio.structure.secstruc">SecStruc</a></dt>
<dd>
<div class="block">constant for electrostatic energy</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#qr()">qr()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">QR Decomposition</div>
</dd>
<dt><a href="./org/biojava/bio/structure/jama/QRDecomposition.html" title="class in org.biojava.bio.structure.jama"><span class="strong">QRDecomposition</span></a> - Class in <a href="./org/biojava/bio/structure/jama/package-summary.html">org.biojava.bio.structure.jama</a></dt>
<dd>
<div class="block">QR Decomposition.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/QRDecomposition.html#QRDecomposition(org.biojava.bio.structure.jama.Matrix)">QRDecomposition(Matrix)</a></span> - Constructor for class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/QRDecomposition.html" title="class in org.biojava.bio.structure.jama">QRDecomposition</a></dt>
<dd>
<div class="block">QR Decomposition, computed by Householder reflections.</div>
</dd>
</dl>
<a name="_R_">
<!--   -->
</a>
<h2 class="title">R</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#radiansPerDegree">radiansPerDegree</a></span> - Static variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">Radians per degree.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#random(int, int)">random(int, int)</a></span> - Static method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Generate matrix with random elements</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/CollectionTools.html#randomPick(java.util.Collection)">randomPick(Collection&lt;Number&gt;)</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/CollectionTools.html" title="class in org.biojava.bio.structure.align.util">CollectionTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#rank()">rank()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Matrix rank</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/SingularValueDecomposition.html#rank()">rank()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/SingularValueDecomposition.html" title="class in org.biojava.bio.structure.jama">SingularValueDecomposition</a></dt>
<dd>
<div class="block">Effective numerical matrix rank</div>
</dd>
<dt><a href="./org/biojava/bio/structure/quaternary/io/RawBioUnitDataProvider.html" title="interface in org.biojava.bio.structure.quaternary.io"><span class="strong">RawBioUnitDataProvider</span></a> - Interface in <a href="./org/biojava/bio/structure/quaternary/io/package-summary.html">org.biojava.bio.structure.quaternary.io</a></dt>
<dd>
<div class="block">Defines the methods that have to be implemented by a class that provides the data that is necessary to recreate the correct biological assembly of a protein.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#read(java.io.BufferedReader)">read(BufferedReader)</a></span> - Static method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Read a matrix from a stream.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/AFPChainXMLParser.html#rebuildAFPChain(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">rebuildAFPChain(AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/AFPChainXMLParser.html" title="class in org.biojava.bio.structure.align.xml">AFPChainXMLParser</a></dt>
<dd>
<div class="block">replace the PDB res nums with atom positions:</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/BiologicalAssemblyBuilder.html#rebuildQuaternaryStructure(org.biojava.bio.structure.Structure, java.util.List)">rebuildQuaternaryStructure(Structure, List&lt;ModelTransformationMatrix&gt;)</a></span> - Method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/BiologicalAssemblyBuilder.html" title="class in org.biojava.bio.structure.quaternary">BiologicalAssemblyBuilder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/ReducedChemCompProvider.html" title="class in org.biojava.bio.structure.io.mmcif"><span class="strong">ReducedChemCompProvider</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/package-summary.html">org.biojava.bio.structure.io.mmcif</a></dt>
<dd>
<div class="block">Unlike the <a href="./org/biojava/bio/structure/io/mmcif/DownloadChemCompProvider.html" title="class in org.biojava.bio.structure.io.mmcif"><code>DownloadChemCompProvider</code></a>, this  <a href="./org/biojava/bio/structure/io/mmcif/ChemCompProvider.html" title="interface in org.biojava.bio.structure.io.mmcif"><code>ChemCompProvider</code></a> does not download any chem comp definitions.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ReducedChemCompProvider.html#ReducedChemCompProvider()">ReducedChemCompProvider()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ReducedChemCompProvider.html" title="class in org.biojava.bio.structure.io.mmcif">ReducedChemCompProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentJoiner.html#reduceFragments(java.util.List, org.biojava.bio.structure.align.pairwise.FragmentPair, org.biojava.bio.structure.jama.Matrix)">reduceFragments(List&lt;FragmentPair&gt;, FragmentPair, Matrix)</a></span> - Static method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentJoiner.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentJoiner</a></dt>
<dd>
<div class="block">In helices often many similar fragments can be found.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#reduceInitialFragments()">reduceInitialFragments()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>
<div class="block">if this is set to false, the time spent to joint the initial fragments (step 2)
 is increased.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#refine(org.biojava.bio.structure.align.StrucAligParameters, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">refine(StrucAligParameters, Atom[], Atom[])</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>
<div class="block">Refinement procedure based on superposition and dynamic programming.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model"><span class="strong">Refine</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#Refine()">Refine()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/SerializableCache.html#reloadFromFile()">reloadFromFile()</a></span> - Method in class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/SerializableCache.html" title="class in org.biojava.bio.structure.domain">SerializableCache</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Site.html#remark800toPDB(java.lang.StringBuffer)">remark800toPDB(StringBuffer)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Site.html" title="class in org.biojava.bio.structure">Site</a></dt>
<dd>
<div class="block">Appends the REMARK 800 section pertaining to the site onto the end of the
 StringBuffer provided.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Site.html#remark800toPDB()">remark800toPDB()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Site.html" title="class in org.biojava.bio.structure">Site</a></dt>
<dd>
<div class="block">Provides REMARK 800 section pertaining to the site as a string.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/quaternary/io/RemoteBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io"><span class="strong">RemoteBioUnitDataProvider</span></a> - Class in <a href="./org/biojava/bio/structure/quaternary/io/package-summary.html">org.biojava.bio.structure.quaternary.io</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/RemoteBioUnitDataProvider.html#RemoteBioUnitDataProvider()">RemoteBioUnitDataProvider()</a></span> - Constructor for class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/RemoteBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">RemoteBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/domain/RemoteDomainProvider.html" title="class in org.biojava.bio.structure.domain"><span class="strong">RemoteDomainProvider</span></a> - Class in <a href="./org/biojava/bio/structure/domain/package-summary.html">org.biojava.bio.structure.domain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/RemoteDomainProvider.html#RemoteDomainProvider()">RemoteDomainProvider()</a></span> - Constructor for class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/RemoteDomainProvider.html" title="class in org.biojava.bio.structure.domain">RemoteDomainProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/RemoteDomainProvider.html#RemoteDomainProvider(boolean)">RemoteDomainProvider(boolean)</a></span> - Constructor for class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/RemoteDomainProvider.html" title="class in org.biojava.bio.structure.domain">RemoteDomainProvider</a></dt>
<dd>
<div class="block">initialize this provider with caching enabled</div>
</dd>
<dt><a href="./org/biojava/bio/structure/domain/RemotePDPProvider.html" title="class in org.biojava.bio.structure.domain"><span class="strong">RemotePDPProvider</span></a> - Class in <a href="./org/biojava/bio/structure/domain/package-summary.html">org.biojava.bio.structure.domain</a></dt>
<dd>
<div class="block">A class that provided PDP assignments that are loaded from a remote web server</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/RemotePDPProvider.html#RemotePDPProvider()">RemotePDPProvider()</a></span> - Constructor for class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/RemotePDPProvider.html" title="class in org.biojava.bio.structure.domain">RemotePDPProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/RemotePDPProvider.html#RemotePDPProvider(boolean)">RemotePDPProvider(boolean)</a></span> - Constructor for class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/RemotePDPProvider.html" title="class in org.biojava.bio.structure.domain">RemotePDPProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/BioUnitDataProviderFactory.html#remoteProviderClassName">remoteProviderClassName</a></span> - Static variable in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/BioUnitDataProviderFactory.html" title="class in org.biojava.bio.structure.quaternary.io">BioUnitDataProviderFactory</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/quaternary/io/RemoteRawBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io"><span class="strong">RemoteRawBioUnitDataProvider</span></a> - Class in <a href="./org/biojava/bio/structure/quaternary/io/package-summary.html">org.biojava.bio.structure.quaternary.io</a></dt>
<dd>
<div class="block">A BioUnitDataProvider that fetches the symmetry operations via remote calls to servers from RCSB PDB</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/RemoteRawBioUnitDataProvider.html#RemoteRawBioUnitDataProvider()">RemoteRawBioUnitDataProvider()</a></span> - Constructor for class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/RemoteRawBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">RemoteRawBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop"><span class="strong">RemoteScopInstallation</span></a> - Class in <a href="./org/biojava/bio/structure/scop/package-summary.html">org.biojava.bio.structure.scop</a></dt>
<dd>
<div class="block">A class that fetches information about SCOP from a remote data-source.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html#RemoteScopInstallation()">RemoteScopInstallation()</a></span> - Constructor for class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">RemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomIterator.html#remove()">remove()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomIterator.html" title="class in org.biojava.bio.structure">AtomIterator</a></dt>
<dd>
<div class="block">does nothing.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/GroupIterator.html#remove()">remove()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/GroupIterator.html" title="class in org.biojava.bio.structure">GroupIterator</a></dt>
<dd>
<div class="block">does nothing .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructureAlignmentFactory.html#removeAlgorithm(java.lang.String)">removeAlgorithm(String)</a></span> - Static method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructureAlignmentFactory.html" title="class in org.biojava.bio.structure.align">StructureAlignmentFactory</a></dt>
<dd>
<div class="block">Removes the specified algorithm from the list of options</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/StructureSequenceMatcher.html#removeGaps(org.biojava3.core.sequence.ProteinSequence)">removeGaps(ProteinSequence)</a></span> - Static method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/StructureSequenceMatcher.html" title="class in org.biojava.bio.structure.io">StructureSequenceMatcher</a></dt>
<dd>
<div class="block">Removes all gaps ('-') from a protein sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/StructureSequenceMatcher.html#removeGaps(T[][])">removeGaps(T[][])</a></span> - Static method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/StructureSequenceMatcher.html" title="class in org.biojava.bio.structure.io">StructureSequenceMatcher</a></dt>
<dd>
<div class="block">Creates a new list consisting of all columns of gapped where no row 
 contained a null value.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifParser.html#removeMMcifConsumer(org.biojava.bio.structure.io.mmcif.MMcifConsumer)">removeMMcifConsumer(MMcifConsumer)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifParser.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifParser</a></dt>
<dd>
<div class="block">remove a single consumer from the parser</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#removeMMcifConsumer(org.biojava.bio.structure.io.mmcif.MMcifConsumer)">removeMMcifConsumer(MMcifConsumer)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#removeModels(org.biojava.bio.structure.Structure)">removeModels(Structure)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>
<div class="block">Remove all models from a Structure and keep only the first</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/xml/RepresentativeXMLConverter.html" title="class in org.biojava.bio.structure.align.xml"><span class="strong">RepresentativeXMLConverter</span></a> - Class in <a href="./org/biojava/bio/structure/align/xml/package-summary.html">org.biojava.bio.structure.align.xml</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/RepresentativeXMLConverter.html#RepresentativeXMLConverter()">RepresentativeXMLConverter()</a></span> - Constructor for class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/RepresentativeXMLConverter.html" title="class in org.biojava.bio.structure.align.xml">RepresentativeXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/CountProgressListener.html#requestingAlignmentsFromServer(int)">requestingAlignmentsFromServer(int)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/CountProgressListener.html" title="class in org.biojava.bio.structure.align.client">CountProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/events/AlignmentProgressListener.html#requestingAlignmentsFromServer(int)">requestingAlignmentsFromServer(int)</a></span> - Method in interface org.biojava.bio.structure.align.events.<a href="./org/biojava/bio/structure/align/events/AlignmentProgressListener.html" title="interface in org.biojava.bio.structure.align.events">AlignmentProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/SimpleStructureServer.html#requestNextStructure(org.biojava.bio.structure.server.StructureListener)">requestNextStructure(StructureListener)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/SimpleStructureServer.html" title="class in org.biojava.bio.structure.server">SimpleStructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureServer.html#requestNextStructure(org.biojava.bio.structure.server.StructureListener)">requestNextStructure(StructureListener)</a></span> - Method in interface org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureServer.html" title="interface in org.biojava.bio.structure.server">StructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#reset()">reset()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/ConfigStrucAligParams.html#reset()">reset()</a></span> - Method in interface org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/ConfigStrucAligParams.html" title="interface in org.biojava.bio.structure.align.ce">ConfigStrucAligParams</a></dt>
<dd>
<div class="block">Set the parameters to the default.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/OptimalCECPParameters.html#reset()">reset()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/OptimalCECPParameters.html" title="class in org.biojava.bio.structure.align.ce">OptimalCECPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#reset()">reset()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWaterman3DParameters.html#reset()">reset()</a></span> - Method in class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWaterman3DParameters.html" title="class in org.biojava.bio.structure.align.seq">SmithWaterman3DParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#residueNumber">residueNumber</a></span> - Variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/ResidueNumber.html" title="class in org.biojava.bio.structure"><span class="strong">ResidueNumber</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">Everything that is needed to uniquely describe a residue position</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ResidueNumber.html#ResidueNumber()">ResidueNumber()</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ResidueNumber.html" title="class in org.biojava.bio.structure">ResidueNumber</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ResidueNumber.html#ResidueNumber(java.lang.String, java.lang.Integer, java.lang.Character)">ResidueNumber(String, Integer, Character)</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ResidueNumber.html" title="class in org.biojava.bio.structure">ResidueNumber</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/chem/ResidueType.html" title="enum in org.biojava.bio.structure.io.mmcif.chem"><span class="strong">ResidueType</span></a> - Enum in <a href="./org/biojava/bio/structure/io/mmcif/chem/package-summary.html">org.biojava.bio.structure.io.mmcif.chem</a></dt>
<dd>
<div class="block">Enumerates the possible classifications of residues.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentJoiner.html#resizeArray(java.lang.Object, int)">resizeArray(Object, int)</a></span> - Static method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentJoiner.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentJoiner</a></dt>
<dd>
<div class="block">Reallocates an array with a new size, and copies the contents
 of the old array to the new array.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/util/ResourceManager.html" title="class in org.biojava.bio.structure.align.util"><span class="strong">ResourceManager</span></a> - Class in <a href="./org/biojava/bio/structure/align/util/package-summary.html">org.biojava.bio.structure.align.util</a></dt>
<dd>
<div class="block">A class that manages the Strings that are defined in the spice.properties file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/ResourceManager.html#ResourceManager()">ResourceManager()</a></span> - Constructor for class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/ResourceManager.html" title="class in org.biojava.bio.structure.align.util">ResourceManager</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html#RG">RG</a></span> - Static variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html" title="class in org.biojava.bio.structure.domain.pdp">PDPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html#RG1">RG1</a></span> - Static variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html" title="class in org.biojava.bio.structure.domain.pdp">PDPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/AlignTools.html#rms_dk_diag(double[], double[], int, int, int, int)">rms_dk_diag(double[], double[], int, int, int, int)</a></span> - Static method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/AlignTools.html" title="class in org.biojava.bio.structure.align.helper">AlignTools</a></dt>
<dd>
<div class="block">Given distance matrix diagonals dk1, dk2, get the rmsd of a fpair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#rmsdThr">rmsdThr</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#rmsdThrJoin">rmsdThrJoin</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/chem/PolymerType.html#RNA_ONLY">RNA_ONLY</a></span> - Static variable in enum org.biojava.bio.structure.io.mmcif.chem.<a href="./org/biojava/bio/structure/io/mmcif/chem/PolymerType.html" title="enum in org.biojava.bio.structure.io.mmcif.chem">PolymerType</a></dt>
<dd>
<div class="block">Convenience <tt>Set</tt> of polymer types classified as RNA.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#rotate(org.biojava.bio.structure.Atom, double[][])">rotate(Atom, double[][])</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">rotate a single atom aroud a rotation matrix.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#rotate(org.biojava.bio.structure.Structure, double[][])">rotate(Structure, double[][])</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">Rotate a structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#rotate(org.biojava.bio.structure.Group, double[][])">rotate(Group, double[][])</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">rotate a structure .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#rotate(org.biojava.bio.structure.Atom, org.biojava.bio.structure.jama.Matrix)">rotate(Atom, Matrix)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">Rotate an atom around a Matrix object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#rotate(org.biojava.bio.structure.Group, org.biojava.bio.structure.jama.Matrix)">rotate(Group, Matrix)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">Rotate a group object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#rotate(org.biojava.bio.structure.Structure, org.biojava.bio.structure.jama.Matrix)">rotate(Structure, Matrix)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">Rotate a structure object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html#run()">run()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html" title="class in org.biojava.bio.structure.align.client">FarmJobRunnable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/FarmJob.html#run()">run()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/FarmJob.html" title="class in org.biojava.bio.structure.align">FarmJob</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/AllChemCompProvider.html#run()">run()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/AllChemCompProvider.html" title="class in org.biojava.bio.structure.io.mmcif">AllChemCompProvider</a></dt>
<dd>
<div class="block">Do the actual loading of the dictionary in a thread.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureFetcherRunnable.html#run()">run()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureFetcherRunnable.html" title="class in org.biojava.bio.structure.server">StructureFetcherRunnable</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/StructureAlignmentOptimizer.html#runOptimization(int)">runOptimization(int)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/StructureAlignmentOptimizer.html" title="class in org.biojava.bio.structure.align.fatcat.calc">StructureAlignmentOptimizer</a></dt>
<dd>
<div class="block">run the optimization</div>
</dd>
</dl>
<a name="_S_">
<!--   -->
</a>
<h2 class="title">S</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/CutValues.html#s_min">s_min</a></span> - Variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/CutValues.html" title="class in org.biojava.bio.structure.domain.pdp">CutValues</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/SandboxStyleStructureProvider.html" title="class in org.biojava.bio.structure.io"><span class="strong">SandboxStyleStructureProvider</span></a> - Class in <a href="./org/biojava/bio/structure/io/package-summary.html">org.biojava.bio.structure.io</a></dt>
<dd>
<div class="block">The "Sandbox" style of organizing files is  to have a directory structure like below, i.e.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/SandboxStyleStructureProvider.html#SandboxStyleStructureProvider()">SandboxStyleStructureProvider()</a></span> - Constructor for class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/SandboxStyleStructureProvider.html" title="class in org.biojava.bio.structure.io">SandboxStyleStructureProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#scaleAdd(float, org.biojava.bio.structure.Atom, org.biojava.bio.structure.Atom)">scaleAdd(float, Atom, Atom)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#SCOP_DOWNLOAD">SCOP_DOWNLOAD</a></span> - Static variable in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/scop/ScopCategory.html" title="enum in org.biojava.bio.structure.scop"><span class="strong">ScopCategory</span></a> - Enum in <a href="./org/biojava/bio/structure/scop/package-summary.html">org.biojava.bio.structure.scop</a></dt>
<dd>
<div class="block">The various categories provided by SCOP.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/scop/ScopDatabase.html" title="interface in org.biojava.bio.structure.scop"><span class="strong">ScopDatabase</span></a> - Interface in <a href="./org/biojava/bio/structure/scop/package-summary.html">org.biojava.bio.structure.scop</a></dt>
<dd>
<div class="block">General API how to interact with SCOP</div>
</dd>
<dt><a href="./org/biojava/bio/structure/scop/ScopDescription.html" title="class in org.biojava.bio.structure.scop"><span class="strong">ScopDescription</span></a> - Class in <a href="./org/biojava/bio/structure/scop/package-summary.html">org.biojava.bio.structure.scop</a></dt>
<dd>
<div class="block">Contains data from
 dir.des.scop.txt_1.75</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDescription.html#ScopDescription()">ScopDescription()</a></span> - Constructor for class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDescription.html" title="class in org.biojava.bio.structure.scop">ScopDescription</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/scop/server/ScopDescriptions.html" title="class in org.biojava.bio.structure.scop.server"><span class="strong">ScopDescriptions</span></a> - Class in <a href="./org/biojava/bio/structure/scop/server/package-summary.html">org.biojava.bio.structure.scop.server</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/ScopDescriptions.html#ScopDescriptions()">ScopDescriptions()</a></span> - Constructor for class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/ScopDescriptions.html" title="class in org.biojava.bio.structure.scop.server">ScopDescriptions</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop"><span class="strong">ScopDomain</span></a> - Class in <a href="./org/biojava/bio/structure/scop/package-summary.html">org.biojava.bio.structure.scop</a></dt>
<dd>
<div class="block">Container for the information for a domain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#ScopDomain()">ScopDomain()</a></span> - Constructor for class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/scop/server/ScopDomains.html" title="class in org.biojava.bio.structure.scop.server"><span class="strong">ScopDomains</span></a> - Class in <a href="./org/biojava/bio/structure/scop/server/package-summary.html">org.biojava.bio.structure.scop.server</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/ScopDomains.html#ScopDomains()">ScopDomains()</a></span> - Constructor for class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/ScopDomains.html" title="class in org.biojava.bio.structure.scop.server">ScopDomains</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#scopDownloadURL">scopDownloadURL</a></span> - Variable in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/scop/ScopFactory.html" title="class in org.biojava.bio.structure.scop"><span class="strong">ScopFactory</span></a> - Class in <a href="./org/biojava/bio/structure/scop/package-summary.html">org.biojava.bio.structure.scop</a></dt>
<dd>
<div class="block">Controls the global ScopDatabase being used.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopFactory.html#ScopFactory()">ScopFactory()</a></span> - Constructor for class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopFactory.html" title="class in org.biojava.bio.structure.scop">ScopFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#scopIDregex">scopIDregex</a></span> - Static variable in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop"><span class="strong">ScopInstallation</span></a> - Class in <a href="./org/biojava/bio/structure/scop/package-summary.html">org.biojava.bio.structure.scop</a></dt>
<dd>
<div class="block">This class provides access to the SCOP protein structure classification.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#ScopInstallation(java.lang.String)">ScopInstallation(String)</a></span> - Constructor for class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>
<div class="block">Create a new SCOP installation.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#ScopInstallation()">ScopInstallation()</a></span> - Constructor for class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>
<div class="block">Create a new SCOP installation, downloading the file to "the right place".</div>
</dd>
<dt><a href="./org/biojava/bio/structure/scop/ScopNode.html" title="class in org.biojava.bio.structure.scop"><span class="strong">ScopNode</span></a> - Class in <a href="./org/biojava/bio/structure/scop/package-summary.html">org.biojava.bio.structure.scop</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopNode.html#ScopNode()">ScopNode()</a></span> - Constructor for class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopNode.html" title="class in org.biojava.bio.structure.scop">ScopNode</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/scop/server/ScopNodes.html" title="class in org.biojava.bio.structure.scop.server"><span class="strong">ScopNodes</span></a> - Class in <a href="./org/biojava/bio/structure/scop/server/package-summary.html">org.biojava.bio.structure.scop.server</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/ScopNodes.html#ScopNodes()">ScopNodes()</a></span> - Constructor for class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/ScopNodes.html" title="class in org.biojava.bio.structure.scop.server">ScopNodes</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#scopVersion">scopVersion</a></span> - Variable in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#SCORING_STRATEGY">SCORING_STRATEGY</a></span> - Static variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#scoringStrategy">scoringStrategy</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/secstruc/SecStruc.html" title="class in org.biojava.bio.structure.secstruc"><span class="strong">SecStruc</span></a> - Class in <a href="./org/biojava/bio/structure/secstruc/package-summary.html">org.biojava.bio.structure.secstruc</a></dt>
<dd>
<div class="block">Assign secondary structure to a Structure object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStruc.html#SecStruc()">SecStruc()</a></span> - Constructor for class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStruc.html" title="class in org.biojava.bio.structure.secstruc">SecStruc</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html" title="class in org.biojava.bio.structure.secstruc"><span class="strong">SecStrucGroup</span></a> - Class in <a href="./org/biojava/bio/structure/secstruc/package-summary.html">org.biojava.bio.structure.secstruc</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html#SecStrucGroup()">SecStrucGroup()</a></span> - Constructor for class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html" title="class in org.biojava.bio.structure.secstruc">SecStrucGroup</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/secstruc/SecStrucState.html" title="class in org.biojava.bio.structure.secstruc"><span class="strong">SecStrucState</span></a> - Class in <a href="./org/biojava/bio/structure/secstruc/package-summary.html">org.biojava.bio.structure.secstruc</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucState.html#SecStrucState()">SecStrucState()</a></span> - Constructor for class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucState.html" title="class in org.biojava.bio.structure.secstruc">SecStrucState</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/secstruc/SecStrucType.html" title="enum in org.biojava.bio.structure.secstruc"><span class="strong">SecStrucType</span></a> - Enum in <a href="./org/biojava/bio/structure/secstruc/package-summary.html">org.biojava.bio.structure.secstruc</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/domain/pdp/Segment.html" title="class in org.biojava.bio.structure.domain.pdp"><span class="strong">Segment</span></a> - Class in <a href="./org/biojava/bio/structure/domain/pdp/package-summary.html">org.biojava.bio.structure.domain.pdp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Segment.html#Segment()">Segment()</a></span> - Constructor for class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Segment.html" title="class in org.biojava.bio.structure.domain.pdp">Segment</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/domain/pdp/SegmentComparator.html" title="class in org.biojava.bio.structure.domain.pdp"><span class="strong">SegmentComparator</span></a> - Class in <a href="./org/biojava/bio/structure/domain/pdp/package-summary.html">org.biojava.bio.structure.domain.pdp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/SegmentComparator.html#SegmentComparator()">SegmentComparator()</a></span> - Constructor for class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/SegmentComparator.html" title="class in org.biojava.bio.structure.domain.pdp">SegmentComparator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/JFatCatClient.html#sendAFPChainToServer(java.lang.String, org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">sendAFPChainToServer(String, AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/JFatCatClient.html" title="class in org.biojava.bio.structure.align.client">JFatCatClient</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/JFatCatClient.html#sendMultiAFPChainToServer(java.lang.String, java.lang.String, java.lang.String)">sendMultiAFPChainToServer(String, String, String)</a></span> - Static method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/JFatCatClient.html" title="class in org.biojava.bio.structure.align.client">JFatCatClient</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/JFatCatClient.html#sendMultiAFPChainToServer(java.lang.String, java.lang.String, java.lang.String, java.lang.String)">sendMultiAFPChainToServer(String, String, String, String)</a></span> - Static method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/JFatCatClient.html" title="class in org.biojava.bio.structure.align.client">JFatCatClient</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html#sendResultsToServer(java.util.List)">sendResultsToServer(List&lt;String&gt;)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html" title="class in org.biojava.bio.structure.align.client">FarmJobRunnable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/CountProgressListener.html#sentResultsToServer(int, java.lang.String)">sentResultsToServer(int, String)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/CountProgressListener.html" title="class in org.biojava.bio.structure.align.client">CountProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/events/AlignmentProgressListener.html#sentResultsToServer(int, java.lang.String)">sentResultsToServer(int, String)</a></span> - Method in interface org.biojava.bio.structure.align.events.<a href="./org/biojava/bio/structure/align/events/AlignmentProgressListener.html" title="interface in org.biojava.bio.structure.align.events">AlignmentProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/SeqRes2AtomAligner.html" title="class in org.biojava.bio.structure.io"><span class="strong">SeqRes2AtomAligner</span></a> - Class in <a href="./org/biojava/bio/structure/io/package-summary.html">org.biojava.bio.structure.io</a></dt>
<dd>
<div class="block">Aligns the SEQRES residues to the ATOM residues.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/SeqRes2AtomAligner.html#SeqRes2AtomAligner()">SeqRes2AtomAligner()</a></span> - Constructor for class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/SeqRes2AtomAligner.html" title="class in org.biojava.bio.structure.io">SeqRes2AtomAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#seqResGroups">seqResGroups</a></span> - Variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcid.html#SEQRESRECORD">SEQRESRECORD</a></span> - Static variable in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcid.html" title="interface in org.biojava.bio.structure">AminoAcid</a></dt>
<dd>
<div class="block">Field to distribguish AminoAcids that have been created from SEQRES records and ATOM records.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#SEQUENCE_CONSERVATION">SEQUENCE_CONSERVATION</a></span> - Static variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#seqWeight">seqWeight</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/domain/SerializableCache.html" title="class in org.biojava.bio.structure.domain"><span class="strong">SerializableCache</span></a>&lt;<a href="./org/biojava/bio/structure/domain/SerializableCache.html" title="type parameter in SerializableCache">K</a>,<a href="./org/biojava/bio/structure/domain/SerializableCache.html" title="type parameter in SerializableCache">V</a>&gt; - Class in <a href="./org/biojava/bio/structure/domain/package-summary.html">org.biojava.bio.structure.domain</a></dt>
<dd>
<div class="block">A class that provides all that is necessary to create a Serializable Cache</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/SerializableCache.html#SerializableCache(java.lang.String)">SerializableCache(String)</a></span> - Constructor for class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/SerializableCache.html" title="class in org.biojava.bio.structure.domain">SerializableCache</a></dt>
<dd>
<div class="block">set cacheFileName to null to disable caching</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html#serialize(java.io.File)">serialize(File)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentResult</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/SerializableCache.html#serializedCache">serializedCache</a></span> - Variable in class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/SerializableCache.html" title="class in org.biojava.bio.structure.domain">SerializableCache</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#serialVersionUID">serialVersionUID</a></span> - Static variable in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathFragment.html#serialVersionUID">serialVersionUID</a></span> - Static variable in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathFragment.html" title="class in org.biojava.bio.structure.cath">CathFragment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathNode.html#serialVersionUID">serialVersionUID</a></span> - Static variable in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathNode.html" title="class in org.biojava.bio.structure.cath">CathNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathSegment.html#serialVersionUID">serialVersionUID</a></span> - Static variable in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathSegment.html" title="class in org.biojava.bio.structure.cath">CathSegment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBParseException.html#serialVersionUID">serialVersionUID</a></span> - Static variable in exception org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBParseException.html" title="class in org.biojava.bio.structure.io">PDBParseException</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureException.html#serialVersionUID">serialVersionUID</a></span> - Static variable in exception org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureException.html" title="class in org.biojava.bio.structure">StructureException</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#set(int, int, double)">set(int, int, double)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Set a single element.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBCrystallographicInfo.html#setA(float)">setA(float)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBCrystallographicInfo.html" title="class in org.biojava.bio.structure">PDBCrystallographicInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html#setAbsorpt_coefficient_mu(java.lang.String)">setAbsorpt_coefficient_mu(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html" title="class in org.biojava.bio.structure.io.mmcif.model">Exptl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html#setAbsorpt_correction_T_max(java.lang.String)">setAbsorpt_correction_T_max(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html" title="class in org.biojava.bio.structure.io.mmcif.model">Exptl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html#setAbsorpt_correction_T_min(java.lang.String)">setAbsorpt_correction_T_min(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html" title="class in org.biojava.bio.structure.io.mmcif.model">Exptl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html#setAbsorpt_correction_type(java.lang.String)">setAbsorpt_correction_type(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html" title="class in org.biojava.bio.structure.io.mmcif.model">Exptl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html#setAbsorpt_process_details(java.lang.String)">setAbsorpt_process_details(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html" title="class in org.biojava.bio.structure.io.mmcif.model">Exptl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucState.html#setAccept1(org.biojava.bio.structure.secstruc.HBond)">setAccept1(HBond)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucState.html" title="class in org.biojava.bio.structure.secstruc">SecStrucState</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucState.html#setAccept2(org.biojava.bio.structure.secstruc.HBond)">setAccept2(HBond)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucState.html" title="class in org.biojava.bio.structure.secstruc">SecStrucState</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileParsingParameters.html#setAcceptedAtomNames(java.lang.String[])">setAcceptedAtomNames(String[])</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileParsingParameters.html" title="class in org.biojava.bio.structure.io">FileParsingParameters</a></dt>
<dd>
<div class="block">by default the parser will read in all atoms (unless using the CAonly switch).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setAfpAftIndex(int[][])">setAfpAftIndex(int[][])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setAfpBefIndex(int[][])">setAfpBefIndex(int[][])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setAfpChainLen(int)">setAfpChainLen(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setAfpChainList(int[])">setAfpChainList(int[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setAfpChainTwiBin(double[])">setAfpChainTwiBin(double[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setAfpChainTwiList(double[])">setAfpChainTwiList(double[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setAfpChainTwiNum(int)">setAfpChainTwiNum(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#setAfpDisCut(double)">setAfpDisCut(double)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#setAfpDisCut0(double)">setAfpDisCut0(double)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setAfpIndex(int[][])">setAfpIndex(int[][])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setAfpSet(java.util.List)">setAfpSet(List&lt;AFP&gt;)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">Set the set of AFPs for this alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html#setAlgorithmName(java.lang.String)">setAlgorithmName(String)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html" title="class in org.biojava.bio.structure.align">CallableStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setAlgorithmName(java.lang.String)">setAlgorithmName(String)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/Alignable.html#setAligMat(org.biojava.bio.structure.align.helper.AligMatEl[][])">setAligMat(AligMatEl[][])</a></span> - Method in interface org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/Alignable.html" title="interface in org.biojava.bio.structure.align.pairwise">Alignable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html#setAligMat(int, int, org.biojava.bio.structure.align.helper.AligMatEl)">setAligMat(int, int, AligMatEl)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">StrCompAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html#setAligMat(org.biojava.bio.structure.align.helper.AligMatEl[][])">setAligMat(AligMatEl[][])</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">StrCompAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#setAlign_id(java.lang.String)">setAlign_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#setAlign_se1(int[])">setAlign_se1(int[])</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#setAlign_se1(int[])">setAlign_se1(int[])</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#setAlign_se2(int[])">setAlign_se2(int[])</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#setAlign_se2(int[])">setAlign_se2(int[])</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html#setAlignments(org.biojava.bio.structure.align.pairwise.AlternativeAlignment[])">setAlignments(AlternativeAlignment[])</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentResult</a></dt>
<dd>
<div class="block">we only keep the first alternative...</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#setAlignPairs(java.lang.String)">setAlignPairs(String)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setAlignScore(double)">setAlignScore(double)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setAlignScoreUpdate(double)">setAlignScoreUpdate(double)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileParsingParameters.html#setAlignSeqRes(boolean)">setAlignSeqRes(boolean)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileParsingParameters.html" title="class in org.biojava.bio.structure.io">FileParsingParameters</a></dt>
<dd>
<div class="block">define if the SEQRES in the structure should be aligned with the ATOM records
 if yes, the AminoAcids in structure.getSeqRes will have the coordinates set.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setAlnbeg1(int)">setAlnbeg1(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setAlnbeg2(int)">setAlnbeg2(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setAlnLength(int)">setAlnLength(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setAlnseq1(char[])">setAlnseq1(char[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setAlnseq2(char[])">setAlnseq2(char[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setAlnsymb(char[])">setAlnsymb(char[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBCrystallographicInfo.html#setAlpha(float)">setAlpha(float)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBCrystallographicInfo.html" title="class in org.biojava.bio.structure">PDBCrystallographicInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#setAltAligNumber(int)">setAltAligNumber(int)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#setAltLoc(java.lang.Character)">setAltLoc(Character)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">get set alternate Location.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#setAltLoc(java.lang.Character)">setAltLoc(Character)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>
<div class="block">set alternate Location.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcid.html#setAminoType(java.lang.Character)">setAminoType(Character)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcid.html" title="interface in org.biojava.bio.structure">AminoAcid</a></dt>
<dd>
<div class="block">set the name of the AA, in single letter code .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcidImpl.html#setAminoType(java.lang.Character)">setAminoType(Character)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcidImpl.html" title="class in org.biojava.bio.structure">AminoAcidImpl</a></dt>
<dd>
<div class="block">set the name of the AA, in single letter code .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setAngleDiff(int)">setAngleDiff(int)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setAniso_B(java.lang.String[][])">setAniso_B(String[][])</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#setArchitectureId(java.lang.Integer)">setArchitectureId(Integer)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssemblyGen.html#setAssembly_id(java.lang.String)">setAssembly_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssemblyGen.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructAssemblyGen</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/AssignmentXMLSerializer.html#setAssignments(java.util.Map)">setAssignments(Map&lt;String, String&gt;)</a></span> - Method in class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/AssignmentXMLSerializer.html" title="class in org.biojava.bio.structure.domain">AssignmentXMLSerializer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#setAsym_id(java.lang.String)">setAsym_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#setAsym_id(java.lang.String)">setAsym_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssemblyGen.html#setAsym_id_list(java.lang.String)">setAsym_id_list(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssemblyGen.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructAssemblyGen</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/BioUnitDataProvider.html#setAsymUnit(org.biojava.bio.structure.Structure)">setAsymUnit(Structure)</a></span> - Method in interface org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/BioUnitDataProvider.html" title="interface in org.biojava.bio.structure.quaternary.io">BioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/FileBasedPDBBioUnitDataProvider.html#setAsymUnit(org.biojava.bio.structure.Structure)">setAsymUnit(Structure)</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/FileBasedPDBBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">FileBasedPDBBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/MmCifBiolAssemblyProvider.html#setAsymUnit(org.biojava.bio.structure.Structure)">setAsymUnit(Structure)</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/MmCifBiolAssemblyProvider.html" title="class in org.biojava.bio.structure.quaternary.io">MmCifBiolAssemblyProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/MmCifPDBBiolAssemblyProvider.html#setAsymUnit(org.biojava.bio.structure.Structure)">setAsymUnit(Structure)</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/MmCifPDBBiolAssemblyProvider.html" title="class in org.biojava.bio.structure.quaternary.io">MmCifPDBBiolAssemblyProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/PDBBioUnitDataProvider.html#setAsymUnit(org.biojava.bio.structure.Structure)">setAsymUnit(Structure)</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/PDBBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">PDBBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/RemoteBioUnitDataProvider.html#setAsymUnit(org.biojava.bio.structure.Structure)">setAsymUnit(Structure)</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/RemoteBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">RemoteBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setAtcc(java.lang.String)">setAtcc(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/BioUnitDataProvider.html#setAtomCache(org.biojava.bio.structure.align.util.AtomCache)">setAtomCache(AtomCache)</a></span> - Method in interface org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/BioUnitDataProvider.html" title="interface in org.biojava.bio.structure.quaternary.io">BioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/FileBasedPDBBioUnitDataProvider.html#setAtomCache(org.biojava.bio.structure.align.util.AtomCache)">setAtomCache(AtomCache)</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/FileBasedPDBBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">FileBasedPDBBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/MmCifBiolAssemblyProvider.html#setAtomCache(org.biojava.bio.structure.align.util.AtomCache)">setAtomCache(AtomCache)</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/MmCifBiolAssemblyProvider.html" title="class in org.biojava.bio.structure.quaternary.io">MmCifBiolAssemblyProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/MmCifPDBBiolAssemblyProvider.html#setAtomCache(org.biojava.bio.structure.align.util.AtomCache)">setAtomCache(AtomCache)</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/MmCifPDBBiolAssemblyProvider.html" title="class in org.biojava.bio.structure.quaternary.io">MmCifPDBBiolAssemblyProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/PDBBioUnitDataProvider.html#setAtomCache(org.biojava.bio.structure.align.util.AtomCache)">setAtomCache(AtomCache)</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/PDBBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">PDBBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/RemoteBioUnitDataProvider.html#setAtomCache(org.biojava.bio.structure.align.util.AtomCache)">setAtomCache(AtomCache)</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/RemoteBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">RemoteBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/structure/StructureIO.html#setAtomCache(org.biojava.bio.structure.align.util.AtomCache)">setAtomCache(AtomCache)</a></span> - Static method in class org.biojava3.structure.<a href="./org/biojava3/structure/StructureIO.html" title="class in org.biojava3.structure">StructureIO</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileParsingParameters.html#setAtomCaThreshold(int)">setAtomCaThreshold(int)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileParsingParameters.html" title="class in org.biojava.bio.structure.io">FileParsingParameters</a></dt>
<dd>
<div class="block">the maximum number of atoms that will be parsed before the parser switches to a CA-only
 representation of the PDB file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#setAtomGroups(java.util.List)">setAtomGroups(List&lt;Group&gt;)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Set all groups that have been specified in the ATOM section of this chain .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#setAtomGroups(java.util.List)">setAtomGroups(List&lt;Group&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Set all groups that have been specified in the ATOM section of this chain .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#setAtoms(java.util.List)">setAtoms(List&lt;Atom&gt;)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">Set the atoms of this group.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#setAtoms(java.util.List)">setAtoms(List&lt;Atom&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">set the atoms of this group</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setAuth_asym_id(java.lang.String)">setAuth_asym_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setAuth_atom_id(java.lang.String)">setAuth_atom_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setAuth_comp_id(java.lang.String)">setAuth_comp_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#setAuth_mon_id(java.lang.String)">setAuth_mon_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#setAuth_mon_id(java.lang.String)">setAuth_mon_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setAuth_seq_id(java.lang.String)">setAuth_seq_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#setAuth_seq_num(java.lang.String)">setAuth_seq_num(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#setAuth_seq_num(java.lang.String)">setAuth_seq_num(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#setAuthorList(java.util.List)">setAuthorList(List&lt;Author&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#setAuthors(java.lang.String)">setAuthors(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#setAutoFetch(boolean)">setAutoFetch(boolean)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#setAutoFetch(boolean)">setAutoFetch(boolean)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>
<div class="block">Does the cache automatically download files that are missing from the local installation from the PDB FTP site?</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/UserConfiguration.html#setAutoFetch(boolean)">setAutoFetch(boolean)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/UserConfiguration.html" title="class in org.biojava.bio.structure.align.util">UserConfiguration</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/MMCIFFileReader.html#setAutoFetch(boolean)">setAutoFetch(boolean)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/MMCIFFileReader.html" title="class in org.biojava.bio.structure.io">MMCIFFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#setAutoFetch(boolean)">setAutoFetch(boolean)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/StructureIOFile.html#setAutoFetch(boolean)">setAutoFetch(boolean)</a></span> - Method in interface org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/StructureIOFile.html" title="interface in org.biojava.bio.structure.io">StructureIOFile</a></dt>
<dd>
<div class="block">Tell the parser to fetch missing PDB files from the FTP server automatically.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html#setAutoFetch(boolean)">setAutoFetch(boolean)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html" title="class in org.biojava.bio.structure.server">MMCIFFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
<div class="block">tell the parser to fetch missing mmCif files from the RCSB FTP server automatically.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBCrystallographicInfo.html#setB(float)">setB(float)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBCrystallographicInfo.html" title="class in org.biojava.bio.structure">PDBCrystallographicInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setB_iso_max(java.lang.String)">setB_iso_max(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setB_iso_mean(java.lang.String)">setB_iso_mean(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setB_iso_min(java.lang.String)">setB_iso_min(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setB_iso_or_equiv(java.lang.String)">setB_iso_or_equiv(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setB_iso_or_equiv_esd(java.lang.String)">setB_iso_or_equiv_esd(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#setBadRmsd(double)">setBadRmsd(double)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucState.html#setBend(boolean)">setBend(boolean)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucState.html" title="class in org.biojava.bio.structure.secstruc">SecStrucState</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBCrystallographicInfo.html#setBeta(float)">setBeta(float)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBCrystallographicInfo.html" title="class in org.biojava.bio.structure">PDBCrystallographicInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/OperatorResolver.html#setBinaryOperators(java.util.List)">setBinaryOperators(List&lt;OrderedPair&lt;String&gt;&gt;)</a></span> - Method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/OperatorResolver.html" title="class in org.biojava.bio.structure.quaternary">OperatorResolver</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#setBioAssemblyFallback(boolean)">setBioAssemblyFallback(boolean)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>
<div class="block">Set bioAssemblyFallback to true, to download the original PDB file in cases that a biological assembly file is not available.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#setBioAssemblyId(int)">setBioAssemblyId(int)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>
<div class="block">Sets the ID of the biological assembly to be loaded.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html#setBiol_id(java.lang.String)">setBiol_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRef</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#setBiologicalAssembly(boolean)">setBiologicalAssembly(boolean)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">Sets a flag to indicate if this structure is a biological assembly</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#setBiologicalAssembly(boolean)">setBiologicalAssembly(boolean)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">Sets a flag to indicate if this structure is a biological assembly</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setBiologicalUnit(java.lang.String)">setBiologicalUnit(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/BioUnitDataProviderFactory.html#setBioUnitDataProvider(java.lang.String)">setBioUnitDataProvider(String)</a></span> - Static method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/BioUnitDataProviderFactory.html" title="class in org.biojava.bio.structure.quaternary.io">BioUnitDataProviderFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#setBioUnitTranformationMap(java.util.Map)">setBioUnitTranformationMap(Map&lt;Integer, List&lt;ModelTransformationMatrix&gt;&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setBlock2Afp(int[])">setBlock2Afp(int[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setBlockGap(int[])">setBlockGap(int[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setBlockNum(int)">setBlockNum(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setBlockNumClu(int)">setBlockNumClu(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setBlockNumIni(int)">setBlockNumIni(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setBlockNumSpt(int)">setBlockNumSpt(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setBlockResList(int[][][])">setBlockResList(int[][][])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setBlockResSize(int[])">setBlockResSize(int[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setBlockRmsd(double[])">setBlockRmsd(double[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setBlockRotationMatrix(org.biojava.bio.structure.jama.Matrix[])">setBlockRotationMatrix(Matrix[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setBlockScore(double[])">setBlockScore(double[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setBlockShiftVector(org.biojava.bio.structure.Atom[])">setBlockShiftVector(Atom[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setBlockSize(int[])">setBlockSize(int[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/Ladder.html#setBtype(org.biojava.bio.structure.secstruc.BridgeType)">setBtype(BridgeType)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/Ladder.html" title="class in org.biojava.bio.structure.secstruc">Ladder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBCrystallographicInfo.html#setC(float)">setC(float)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBCrystallographicInfo.html" title="class in org.biojava.bio.structure">PDBCrystallographicInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html#setC(org.biojava.bio.structure.Atom)">setC(Atom)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html" title="class in org.biojava.bio.structure.secstruc">SecStrucGroup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html#setCA(org.biojava.bio.structure.Atom)">setCA(Atom)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html" title="class in org.biojava.bio.structure.secstruc">SecStrucGroup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html#setCa1(org.biojava.bio.structure.Atom[])">setCa1(Atom[])</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html" title="class in org.biojava.bio.structure.align">CallableStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setCa1Length(int)">setCa1Length(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setCa2Length(int)">setCa2Length(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html#setCache(org.biojava.bio.structure.align.util.AtomCache)">setCache(AtomCache)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html" title="class in org.biojava.bio.structure.align">CallableStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#setCacheLocation(java.lang.String)">setCacheLocation(String)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#setCacheLocation(java.lang.String)">setCacheLocation(String)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/SimpleStructureServer.html#setCacheSize(int)">setCacheSize(int)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/SimpleStructureServer.html" title="class in org.biojava.bio.structure.server">SimpleStructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureServer.html#setCacheSize(int)">setCacheSize(int)</a></span> - Method in interface org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureServer.html" title="interface in org.biojava.bio.structure.server">StructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
<div class="block">set how many structures should be kept in memory - for quicker access</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setCalculationTime(long)">setCalculationTime(long)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html#setCalculationTime(long)">setCalculationTime(long)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentResult</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setCartn_x(java.lang.String)">setCartn_x(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setCartn_x_esd(java.lang.String)">setCartn_x_esd(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setCartn_y(java.lang.String)">setCartn_y(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setCartn_y_esd(java.lang.String)">setCartn_y_esd(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setCartn_z(java.lang.String)">setCartn_z(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setCartn_z_esd(java.lang.String)">setCartn_z_esd(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDescription.html#setCategory(org.biojava.bio.structure.scop.ScopCategory)">setCategory(ScopCategory)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDescription.html" title="class in org.biojava.bio.structure.scop">ScopDescription</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#setCATH(java.lang.String)">setCATH(String)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#setCathDownloadUrl(java.lang.String)">setCathDownloadUrl(String)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setCell(java.lang.String)">setCell(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setCellLine(java.lang.String)">setCellLine(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setCellularLocation(java.lang.String)">setCellularLocation(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#setCenter1(org.biojava.bio.structure.Atom)">setCenter1(Atom)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#setCenter2(org.biojava.bio.structure.Atom)">setCenter2(Atom)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#setChain(org.biojava.bio.structure.Chain)">setChain(Chain)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">Sets the back-reference to its parent Chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#setChain(org.biojava.bio.structure.Chain)">setChain(Chain)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">Sets the back-reference to its parent Chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html#setChain1(java.lang.String)">setChain1(String)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentResult</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html#setChain2(java.lang.String)">setChain2(String)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentResult</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#setChainID(java.lang.String)">setChainID(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">get and set the name of this chain (Chain id in PDB file ).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#setChainID(java.lang.String)">setChainID(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">get and set the name of this chain (Chain id in PDB file ).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setChainId(java.util.List)">setChainId(List&lt;String&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>
<div class="block">Sets the list of chain IDs.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#setChainId(java.lang.Character)">setChainId(Character)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">The chain ID of the corresponding chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html#setChainId(java.lang.String)">setChainId(String)</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html" title="class in org.biojava.bio.structure.io.sifts">SiftsResidue</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ResidueNumber.html#setChainId(java.lang.String)">setChainId(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ResidueNumber.html" title="class in org.biojava.bio.structure">ResidueNumber</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SSBond.html#setChainID1(java.lang.String)">setChainID1(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SSBond.html" title="class in org.biojava.bio.structure">SSBond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SSBond.html#setChainID2(java.lang.String)">setChainID2(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SSBond.html" title="class in org.biojava.bio.structure">SSBond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/FlatFileInstallation.html#setChainInfoFile(java.io.File)">setChainInfoFile(File)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/FlatFileInstallation.html" title="class in org.biojava.bio.structure.server">FlatFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setChainLen(int)">setChainLen(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setChainRmsd(double)">setChainRmsd(double)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">The RMSD of the chain of AFPs.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setChains(java.util.List)">setChains(List&lt;Chain&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#setChains(java.util.List)">setChains(List&lt;Chain&gt;)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">set the chains of a structure, if this is a NMR structure,
 this will only set model 0.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#setChains(int, java.util.List)">setChains(int, List&lt;Chain&gt;)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">set the chains for a model</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#setChains(java.util.List)">setChains(List&lt;Chain&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#setChains(int, java.util.List)">setChains(int, List&lt;Chain&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#setChemComp(org.biojava.bio.structure.io.mmcif.model.ChemComp)">setChemComp(ChemComp)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">Set the Chemical Component that closer describes this group.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#setChemComp(org.biojava.bio.structure.io.mmcif.model.ChemComp)">setChemComp(ChemComp)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompGroupFactory.html#setChemCompProvider(org.biojava.bio.structure.io.mmcif.ChemCompProvider)">setChemCompProvider(ChemCompProvider)</a></span> - Static method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompGroupFactory.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompGroupFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopNode.html#setChildren(java.util.List)">setChildren(List&lt;Integer&gt;)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopNode.html" title="class in org.biojava.bio.structure.scop">ScopNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#setClassId(java.lang.Integer)">setClassId(Integer)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#setClassId(int)">setClassId(int)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#setClassification(java.lang.String)">setClassification(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDescription.html#setClassificationId(java.lang.String)">setClassificationId(String)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDescription.html" title="class in org.biojava.bio.structure.scop">ScopDescription</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#setClassificationId(java.lang.String)">setClassificationId(String)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#setCluster(int)">setCluster(int)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>
<div class="block">set the number of the cluster this alignment belongs to.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#setCluster(int)">setCluster(int)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html#setCode(java.lang.String)">setCode(String)</a></span> - Method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html" title="class in org.biojava.bio.structure.quaternary">ModelTransformationMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/IndexPair.html#setCol(short)">setCol(short)</a></span> - Method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/IndexPair.html" title="class in org.biojava.bio.structure.align.helper">IndexPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#setComment(java.lang.String)">setComment(String)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html#setComp_id(java.lang.String)">setComp_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemCompDescriptor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxEntityNonPoly.html#setComp_id(java.lang.String)">setComp_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxEntityNonPoly.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxEntityNonPoly</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#setCompounds(java.util.List)">setCompounds(List&lt;Compound&gt;)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">set the compounts</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#setCompounds(java.util.List)">setCompounds(List&lt;Compound&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setConn(java.lang.Double)">setConn(Double)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/Ladder.html#setConnectedFrom(int)">setConnectedFrom(int)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/Ladder.html" title="class in org.biojava.bio.structure.secstruc">Ladder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/Ladder.html#setConnectedTo(int)">setConnectedTo(int)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/Ladder.html" title="class in org.biojava.bio.structure.secstruc">Ladder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#setConnections(java.util.List)">setConnections(List&lt;Map&lt;String, Integer&gt;&gt;)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">sets/gets an List of  Maps which corresponds to the CONECT lines in the PDB file:</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#setConnections(java.util.List)">setConnections(List&lt;Map&lt;String, Integer&gt;&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#setContacts(int)">setContacts(int)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/AligMatEl.html#setContig(int)">setContig(int)</a></span> - Method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/AligMatEl.html" title="class in org.biojava.bio.structure.align.helper">AligMatEl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#setCoords(double[])">setCoords(double[])</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">set the coordinates.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#setCoords(double[])">setCoords(double[])</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>
<div class="block">the coordinates.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setCorrelation_coeff_Fo_to_Fc(java.lang.String)">setCorrelation_coeff_Fo_to_Fc(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setCorrelation_coeff_Fo_to_Fc_free(java.lang.String)">setCorrelation_coeff_Fo_to_Fc_free(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#setCovered(int)">setCovered(int)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/OptimalCECPParameters.html#setCPPoint(java.lang.Integer)">setCPPoint(Integer)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/OptimalCECPParameters.html" title="class in org.biojava.bio.structure.align.ce">OptimalCECPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setCreate_co(float)">setCreate_co(float)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#setCrystallographicInfo(org.biojava.bio.structure.PDBCrystallographicInfo)">setCrystallographicInfo(PDBCrystallographicInfo)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">Sets crystallographic information for this structure</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#setCrystallographicInfo(org.biojava.bio.structure.PDBCrystallographicInfo)">setCrystallographicInfo(PDBCrystallographicInfo)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">Sets crystallographic information for this structure</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html#setCrystals_number(java.lang.String)">setCrystals_number(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html" title="class in org.biojava.bio.structure.io.mmcif.model">Exptl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/CutSites.html#setCut_sites(int[])">setCut_sites(int[])</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/CutSites.html" title="class in org.biojava.bio.structure.domain.pdp">CutSites</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/TreeSetStringWrapper.html#setData(java.util.TreeSet)">setData(TreeSet&lt;String&gt;)</a></span> - Method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/TreeSetStringWrapper.html" title="class in org.biojava.bio.structure.scop.server">TreeSetStringWrapper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#setDatabase(java.lang.String)">setDatabase(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Specifies the database value.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#setDate(java.util.Date)">setDate(Date)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html#setDate(java.lang.String)">setDate(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html" title="class in org.biojava.bio.structure.io.mmcif.model">DatabasePDBrev</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html#setDate_original(java.lang.String)">setDate_original(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html" title="class in org.biojava.bio.structure.io.mmcif.model">DatabasePDBrev</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#setDb_align_beg(java.lang.String)">setDb_align_beg(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#setDb_align_end(java.lang.String)">setDb_align_end(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html#setDb_code(java.lang.String)">setDb_code(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRef</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html#setDb_name(java.lang.String)">setDb_name(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRef</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#setDbAccession(java.lang.String)">setDbAccession(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Sequence database accession code.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#setDbIdCode(java.lang.String)">setDbIdCode(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Sequence database          identification code.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#setDBRefs(java.util.List)">setDBRefs(List&lt;DBRef&gt;)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">set the list of database references for this structure</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#setDBRefs(java.util.List)">setDBRefs(List&lt;DBRef&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#setDbSeqBegin(int)">setDbSeqBegin(int)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Initial sequence number of the
    database seqment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#setDbSeqEnd(int)">setDbSeqEnd(int)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">The begin of the sequence position in the database</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructurePairAligner.html#setDebug(boolean)">setDebug(boolean)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructurePairAligner.html" title="class in org.biojava.bio.structure.align">StructurePairAligner</a></dt>
<dd>
<div class="block">set the debug flag</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/SerializableCache.html#setDebug(boolean)">setDebug(boolean)</a></span> - Method in class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/SerializableCache.html" title="class in org.biojava.bio.structure.domain">SerializableCache</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/SeqRes2AtomAligner.html#setDEBUG(boolean)">setDEBUG(boolean)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/SeqRes2AtomAligner.html" title="class in org.biojava.bio.structure.io">SeqRes2AtomAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileParsingParameters.html#setDefault()">setDefault()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileParsingParameters.html" title="class in org.biojava.bio.structure.io">FileParsingParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setDensityCutoff(float)">setDensityCutoff(float)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#setDepDate(java.util.Date)">setDepDate(Date)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathNode.html#setDescription(java.lang.String)">setDescription(String)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathNode.html" title="class in org.biojava.bio.structure.cath">CathNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#setDescription(java.lang.String)">setDescription(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDescription.html#setDescription(java.lang.String)">setDescription(String)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDescription.html" title="class in org.biojava.bio.structure.scop">ScopDescription</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Site.html#setDescription(java.lang.String)">setDescription(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Site.html" title="class in org.biojava.bio.structure">Site</a></dt>
<dd>
<div class="block">sets the REMARK 800 description of the site</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setDescription2(java.lang.String)">setDescription2(String)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">Set the textual description for protein 2.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html#setDescriptor(java.lang.String)">setDescriptor(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemCompDescriptor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setDescriptors(java.util.List)">setDescriptors(List&lt;ChemCompDescriptor&gt;)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setDetails(java.lang.String)">setDetails(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html#setDetails(java.lang.String)">setDetails(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model">Entity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html#setDetails(java.lang.String)">setDetails(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html" title="class in org.biojava.bio.structure.io.mmcif.model">Exptl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssembly.html#setDetails(java.lang.String)">setDetails(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssembly.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructAssembly</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setDetails(java.lang.String)">setDetails(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html#setDetails(java.lang.String)">setDetails(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructAsym</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#setDetails(java.lang.String)">setDetails(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setDiagonalDistance(int)">setDiagonalDistance(int)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setDiagonalDistance2(int)">setDiagonalDistance2(int)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#setDisCut(java.lang.Double)">setDisCut(Double)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#setDisFilter(double)">setDisFilter(double)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#setDisSmooth(double)">setDisSmooth(double)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPDistanceMatrix.html#setDist(int[][])">setDist(int[][])</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPDistanceMatrix.html" title="class in org.biojava.bio.structure.domain.pdp">PDPDistanceMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#setDist1(double[][])">setDist1(double[][])</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#setDist1(double[][])">setDist1(double[][])</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#setDist2(double[][])">setDist2(double[][])</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#setDist2(double[][])">setDist2(double[][])</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setDisTable1(org.biojava.bio.structure.jama.Matrix)">setDisTable1(Matrix)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setDisTable2(org.biojava.bio.structure.jama.Matrix)">setDisTable2(Matrix)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/DistEn.html#setDistance(double)">setDistance(double)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/DistEn.html" title="class in org.biojava.bio.structure.secstruc">DistEn</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#setDistanceIncrement(java.lang.Double)">setDistanceIncrement(Double)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setDistanceMatrix(org.biojava.bio.structure.jama.Matrix)">setDistanceMatrix(Matrix)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">A matrix with <i>ca1length</i> rows and <i>ca2length</i> columns.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#setDistanceMatrix(org.biojava.bio.structure.jama.Matrix)">setDistanceMatrix(Matrix)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>
<div class="block">The distance matrix this alignment is based on</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setDoAngleCheck(boolean)">setDoAngleCheck(boolean)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setDoDensityCheck(boolean)">setDoDensityCheck(boolean)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setDoDistanceCheck(boolean)">setDoDistanceCheck(boolean)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#setDoi(java.lang.String)">setDoi(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>
<div class="block">Set the value of doi</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#setDomainCounter(java.lang.Integer)">setDomainCounter(Integer)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#setDomainId(int)">setDomainId(int)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#setDomainName(java.lang.String)">setDomainName(String)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/DomainProviderFactory.html#setDomainProvider(org.biojava.bio.structure.domain.DomainProvider)">setDomainProvider(DomainProvider)</a></span> - Static method in class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/DomainProviderFactory.html" title="class in org.biojava.bio.structure.domain">DomainProviderFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucState.html#setDonor1(org.biojava.bio.structure.secstruc.HBond)">setDonor1(HBond)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucState.html" title="class in org.biojava.bio.structure.secstruc">SecStrucState</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucState.html#setDonor2(org.biojava.bio.structure.secstruc.HBond)">setDonor2(HBond)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucState.html" title="class in org.biojava.bio.structure.secstruc">SecStrucState</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setDoRMSCheck(boolean)">setDoRMSCheck(boolean)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/DownloadChemCompProvider.html#setDownloadAll(boolean)">setDownloadAll(boolean)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/DownloadChemCompProvider.html" title="class in org.biojava.bio.structure.io.mmcif">DownloadChemCompProvider</a></dt>
<dd>
<div class="block">By default this provider will download only some of the <a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model"><code>ChemComp</code></a> files.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setDVar(java.lang.Double)">setDVar(Double)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setEcNums(java.util.List)">setEcNums(List&lt;String&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#setEditorList(java.util.List)">setEditorList(List&lt;Author&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#setElement(org.biojava.bio.structure.Element)">setElement(Element)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">set element of the atom name, e.g.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#setElement(org.biojava.bio.structure.Element)">setElement(Element)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/OrderedPair.html#setElement1(T)">setElement1(T)</a></span> - Method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/OrderedPair.html" title="class in org.biojava.bio.structure.quaternary">OrderedPair</a></dt>
<dd>
<div class="block">Sets the first element of an ordered pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/OrderedPair.html#setElement2(T)">setElement2(T)</a></span> - Method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/OrderedPair.html" title="class in org.biojava.bio.structure.quaternary">OrderedPair</a></dt>
<dd>
<div class="block">Sets the second element of an ordered pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsSegment.html#setEnd(java.lang.String)">setEnd(String)</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsSegment.html" title="class in org.biojava.bio.structure.io.sifts">SiftsSegment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/DistEn.html#setEnergy(double)">setEnergy(double)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/DistEn.html" title="class in org.biojava.bio.structure.secstruc">DistEn</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/HBond.html#setEnergy(double)">setEnergy(double)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/HBond.html" title="class in org.biojava.bio.structure.secstruc">HBond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setEngineered(java.lang.String)">setEngineered(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/EntityPolySeq.html#setEntity_id(java.lang.String)">setEntity_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/EntityPolySeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">EntityPolySeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxEntityNonPoly.html#setEntity_id(java.lang.String)">setEntity_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxEntityNonPoly.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxEntityNonPoly</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#setEntity_id(java.lang.String)">setEntity_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#setEntity_id(java.lang.String)">setEntity_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html#setEntity_id(java.lang.String)">setEntity_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructAsym</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html#setEntity_id(java.lang.String)">setEntity_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRef</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsEntity.html#setEntityId(java.lang.String)">setEntityId(String)</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsEntity.html" title="class in org.biojava.bio.structure.io.sifts">SiftsEntity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html#setEntry_id(java.lang.String)">setEntry_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html" title="class in org.biojava.bio.structure.io.mmcif.model">Exptl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setEntry_id(java.lang.String)">setEntry_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html#setEntry_id(java.lang.String)">setEntry_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html" title="class in org.biojava.bio.structure.io.mmcif.model">Struct</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructKeywords.html#setEntry_id(java.lang.String)">setEntry_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructKeywords.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructKeywords</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setEvalCutoff(float)">setEvalCutoff(float)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Site.html#setEvCode(java.lang.String)">setEvCode(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Site.html" title="class in org.biojava.bio.structure">Site</a></dt>
<dd>
<div class="block">sets the REMARK 800 EVIDENCE CODE for the site.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setExpressionSystem(java.lang.String)">setExpressionSystem(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setExpressionSystemAtccNumber(java.lang.String)">setExpressionSystemAtccNumber(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setExpressionSystemCell(java.lang.String)">setExpressionSystemCell(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setExpressionSystemCellLine(java.lang.String)">setExpressionSystemCellLine(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setExpressionSystemCellularLocation(java.lang.String)">setExpressionSystemCellularLocation(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setExpressionSystemGene(java.lang.String)">setExpressionSystemGene(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setExpressionSystemOrgan(java.lang.String)">setExpressionSystemOrgan(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setExpressionSystemOrganelle(java.lang.String)">setExpressionSystemOrganelle(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setExpressionSystemOtherDetails(java.lang.String)">setExpressionSystemOtherDetails(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setExpressionSystemPlasmid(java.lang.String)">setExpressionSystemPlasmid(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setExpressionSystemStrain(java.lang.String)">setExpressionSystemStrain(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setExpressionSystemTaxId(java.lang.String)">setExpressionSystemTaxId(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setExpressionSystemTissue(java.lang.String)">setExpressionSystemTissue(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setExpressionSystemVariant(java.lang.String)">setExpressionSystemVariant(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setExpressionSystemVector(java.lang.String)">setExpressionSystemVector(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setExpressionSystemVectorType(java.lang.String)">setExpressionSystemVectorType(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#setFamilyId(int)">setFamilyId(int)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#setFetchCurrent(boolean)">setFetchCurrent(boolean)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>
<div class="block">if enabled, the reader searches for the newest possible PDB ID, if not present in he local installation.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#setFetchCurrent(boolean)">setFetchCurrent(boolean)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>
<div class="block">if enabled, the reader searches for the newest possible PDB ID, if not present in he local installation.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#setFetchFileEvenIfObsolete(boolean)">setFetchFileEvenIfObsolete(boolean)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>
<div class="block"><b>N.B.</b> This feature won't work unless the structure wasn't found & autoFetch is set to <code>true</code>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#setFetchFileEvenIfObsolete(boolean)">setFetchFileEvenIfObsolete(boolean)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>
<div class="block"><b>N.B.</b> This feature won't work unless the structure wasn't found & autoFetch is set to <code>true</code>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#setFile1(java.lang.String)">setFile1(String)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#setFile2(java.lang.String)">setFile2(String)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/UserConfiguration.html#setFileFormat(java.lang.String)">setFileFormat(String)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/UserConfiguration.html" title="class in org.biojava.bio.structure.align.util">UserConfiguration</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsMappingProvider.html#setFileLocation(java.lang.String)">setFileLocation(String)</a></span> - Static method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsMappingProvider.html" title="class in org.biojava.bio.structure.io.sifts">SiftsMappingProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/LocalCacheStructureProvider.html#setFileParsingParameters(org.biojava.bio.structure.io.FileParsingParameters)">setFileParsingParameters(FileParsingParameters)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/LocalCacheStructureProvider.html" title="class in org.biojava.bio.structure.io">LocalCacheStructureProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#setFileParsingParameters(org.biojava.bio.structure.io.FileParsingParameters)">setFileParsingParameters(FileParsingParameters)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#setFileParsingParameters(org.biojava.bio.structure.io.FileParsingParameters)">setFileParsingParameters(FileParsingParameters)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#setFileParsingParameters(org.biojava.bio.structure.io.FileParsingParameters)">setFileParsingParameters(FileParsingParameters)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/MMCIFFileReader.html#setFileParsingParameters(org.biojava.bio.structure.io.FileParsingParameters)">setFileParsingParameters(FileParsingParameters)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/MMCIFFileReader.html" title="class in org.biojava.bio.structure.io">MMCIFFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileParser.html#setFileParsingParameters(org.biojava.bio.structure.io.FileParsingParameters)">setFileParsingParameters(FileParsingParameters)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileParser.html" title="class in org.biojava.bio.structure.io">PDBFileParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#setFileParsingParameters(org.biojava.bio.structure.io.FileParsingParameters)">setFileParsingParameters(FileParsingParameters)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/SandboxStyleStructureProvider.html#setFileParsingParameters(org.biojava.bio.structure.io.FileParsingParameters)">setFileParsingParameters(FileParsingParameters)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/SandboxStyleStructureProvider.html" title="class in org.biojava.bio.structure.io">SandboxStyleStructureProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/StructureIOFile.html#setFileParsingParameters(org.biojava.bio.structure.io.FileParsingParameters)">setFileParsingParameters(FileParsingParameters)</a></span> - Method in interface org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/StructureIOFile.html" title="interface in org.biojava.bio.structure.io">StructureIOFile</a></dt>
<dd>
<div class="block">Set the parameters that should be used for file parsing</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/StructureProvider.html#setFileParsingParameters(org.biojava.bio.structure.io.FileParsingParameters)">setFileParsingParameters(FileParsingParameters)</a></span> - Method in interface org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/StructureProvider.html" title="interface in org.biojava.bio.structure.io">StructureProvider</a></dt>
<dd>
<div class="block">Set the parameters that should be used for file parsing</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#setFileParsingParams(org.biojava.bio.structure.io.FileParsingParameters)">setFileParsingParams(FileParsingParameters)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/FlatFileInstallation.html#setFilePath(java.io.File)">setFilePath(File)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/FlatFileInstallation.html" title="class in org.biojava.bio.structure.server">FlatFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#setFlexible(boolean)">setFlexible(boolean)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setFocusAfpList(int[])">setFocusAfpList(int[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setFocusAfpn(int)">setFocusAfpn(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setFocusRes1(int[])">setFocusRes1(int[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setFocusRes2(int[])">setFocusRes2(int[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setFocusResn(int)">setFocusResn(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#setFoldId(int)">setFoldId(int)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#setFormat(java.lang.String)">setFormat(String)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setFormula(java.lang.String)">setFormula(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setFormula_weight(java.lang.String)">setFormula_weight(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html#setFormula_weight(java.lang.String)">setFormula_weight(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model">Entity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setFragCompat(float)">setFragCompat(float)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#setFragLen(java.lang.Integer)">setFragLen(Integer)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFP.html#setFragLen(int)">setFragLen(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFP.html" title="class in org.biojava.bio.structure.align.model">AFP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#setFragLenSq(int)">setFragLenSq(int)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setFragment(java.lang.String)">setFragment(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathFragment.html#setFragmentId(java.lang.Integer)">setFragmentId(Integer)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathFragment.html" title="class in org.biojava.bio.structure.cath">CathFragment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setFragmentLength(int)">setFragmentLength(int)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setFragmentMiniDistance(float)">setFragmentMiniDistance(float)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructurePairAligner.html#setFragmentPairs(org.biojava.bio.structure.align.pairwise.FragmentPair[])">setFragmentPairs(FragmentPair[])</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructurePairAligner.html" title="class in org.biojava.bio.structure.align">StructurePairAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#setFragScore(double)">setFragScore(double)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Segment.html#setFrom(java.lang.Integer)">setFrom(Integer)</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Segment.html" title="class in org.biojava.bio.structure.domain.pdp">Segment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/Ladder.html#setFrom(int)">setFrom(int)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/Ladder.html" title="class in org.biojava.bio.structure.secstruc">Ladder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#setFullName(java.lang.String)">setFullName(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">set full name of atom e.g.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#setFullName(java.lang.String)">setFullName(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>
<div class="block">set full name of atom e.g.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html#setFullSymmetryOperation(java.lang.String)">setFullSymmetryOperation(String)</a></span> - Method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html" title="class in org.biojava.bio.structure.quaternary">ModelTransformationMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBCrystallographicInfo.html#setGamma(float)">setGamma(float)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBCrystallographicInfo.html" title="class in org.biojava.bio.structure">PDBCrystallographicInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#setGapCreate(double)">setGapCreate(double)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/Alignable.html#setGapExtCol(float)">setGapExtCol(float)</a></span> - Method in interface org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/Alignable.html" title="interface in org.biojava.bio.structure.align.pairwise">Alignable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html#setGapExtCol(float)">setGapExtCol(float)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">StrCompAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#setGapExtend(double)">setGapExtend(double)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWaterman3DParameters.html#setGapExtend(java.lang.Short)">setGapExtend(Short)</a></span> - Method in class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWaterman3DParameters.html" title="class in org.biojava.bio.structure.align.seq">SmithWaterman3DParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#setGapExtension(java.lang.Double)">setGapExtension(Double)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setGapExtension(float)">setGapExtension(float)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/Alignable.html#setGapExtRow(float)">setGapExtRow(float)</a></span> - Method in interface org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/Alignable.html" title="interface in org.biojava.bio.structure.align.pairwise">Alignable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html#setGapExtRow(float)">setGapExtRow(float)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">StrCompAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setGapLen(int)">setGapLen(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#setGapOpen(java.lang.Double)">setGapOpen(Double)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWaterman3DParameters.html#setGapOpen(java.lang.Short)">setGapOpen(Short)</a></span> - Method in class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWaterman3DParameters.html" title="class in org.biojava.bio.structure.align.seq">SmithWaterman3DParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setGapOpen(float)">setGapOpen(float)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/Alignable.html#setGapOpenCol(float)">setGapOpenCol(float)</a></span> - Method in interface org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/Alignable.html" title="interface in org.biojava.bio.structure.align.pairwise">Alignable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html#setGapOpenCol(float)">setGapOpenCol(float)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">StrCompAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/Alignable.html#setGapOpenRow(float)">setGapOpenRow(float)</a></span> - Method in interface org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/Alignable.html" title="interface in org.biojava.bio.structure.align.pairwise">Alignable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html#setGapOpenRow(float)">setGapOpenRow(float)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">StrCompAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setGene(java.lang.String)">setGene(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#setGroup(org.biojava.bio.structure.Group)">setGroup(Group)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">Sets the back-reference to its parent Group.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#setGroup(org.biojava.bio.structure.Group)">setGroup(Group)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setGroup_PDB(java.lang.String)">setGroup_PDB(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Site.html#setGroups(java.util.List)">setGroups(List&lt;Group&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Site.html" title="class in org.biojava.bio.structure">Site</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html#setH(org.biojava.bio.structure.Atom)">setH(Atom)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html" title="class in org.biojava.bio.structure.secstruc">SecStrucGroup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#setHeader(org.biojava.bio.structure.Compound)">setHeader(Compound)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Set the Header from the PDB file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#setHeader(org.biojava.bio.structure.Compound)">setHeader(Compound)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Set the Header from the PDB file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#setHeader(java.util.Map)">setHeader(Map&lt;String, Object&gt;)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>&nbsp;</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#setHeader(java.util.Map)">setHeader(Map&lt;String, Object&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">set the Header data.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileParsingParameters.html#setHeaderOnly(boolean)">setHeaderOnly(boolean)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileParsingParameters.html" title="class in org.biojava.bio.structure.io">FileParsingParameters</a></dt>
<dd>
<div class="block">Parse only the PDB file header out of the files</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setHeaderVars(java.lang.String)">setHeaderVars(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/EntityPolySeq.html#setHetero(java.lang.String)">setHetero(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/EntityPolySeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">EntityPolySeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#setHetero(java.lang.String)">setHetero(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#setHomologyId(java.lang.Integer)">setHomologyId(Integer)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPDistanceMatrix.html#setIclose(int[])">setIclose(int[])</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPDistanceMatrix.html" title="class in org.biojava.bio.structure.domain.pdp">PDPDistanceMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFP.html#setId(long)">setId(long)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFP.html" title="class in org.biojava.bio.structure.align.model">AFP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setId(long)">setId(long)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#setId(long)">setId(long)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>
<div class="block">Set the Hibernate database ID.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#setId(java.lang.Long)">setId(Long)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Set the ID used by Hibernate.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#setId(java.lang.Long)">setId(Long)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Set the ID used by Hibernate.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setId(java.lang.Long)">setId(Long)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>
<div class="block">set the ID used by Hibernate</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#setId(java.lang.Long)">setId(Long)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Set the ID used by Hibernate.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Domain.html#setId(java.lang.String)">setId(String)</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Domain.html" title="class in org.biojava.bio.structure.domain.pdp">Domain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#setId(long)">setId(long)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">the Hibernate database ID</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setId(java.lang.String)">setId(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setId(java.lang.String)">setId(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBremark.html#setId(java.lang.String)">setId(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBremark.html" title="class in org.biojava.bio.structure.io.mmcif.model">DatabasePDBremark</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html#setId(java.lang.String)">setId(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model">Entity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssembly.html#setId(java.lang.String)">setId(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssembly.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructAssembly</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html#setId(java.lang.String)">setId(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructOperList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html#setId(java.lang.String)">setId(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructAsym</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html#setId(java.lang.String)">setId(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRef</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html#setId(java.lang.String)">setId(String)</a></span> - Method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html" title="class in org.biojava.bio.structure.quaternary">ModelTransformationMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#setId(java.lang.Long)">setId(Long)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">set the ID used by Hibernate</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#setId(java.lang.Long)">setId(Long)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">set the ID used by Hibernate</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#setIdbnsBegin(char)">setIdbnsBegin(char)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Insertion code of initial residue of the segment, if PDB is the
    reference.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#setIdbnsEnd(char)">setIdbnsEnd(char)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Insertion code of the ending
    residue of the segment, if PDB is
    the reference.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#setIdCode(java.lang.String)">setIdCode(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Set the idCode for this entry.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#setIdCode(java.lang.String)">setIdCode(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>
<div class="block">The PDB code for this protein structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#setIdenticalSequenceFamilyId(java.lang.Integer)">setIdenticalSequenceFamilyId(Integer)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setIdentity(double)">setIdentity(double)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html#setIdentity()">setIdentity()</a></span> - Method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html" title="class in org.biojava.bio.structure.quaternary">ModelTransformationMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/JointFragments.html#setIdxlist(java.util.List)">setIdxlist(List&lt;int[]&gt;)</a></span> - Method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/JointFragments.html" title="class in org.biojava.bio.structure.align.helper">JointFragments</a></dt>
<dd>
<div class="block">Stores the alignment between the residues of several fragments.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/GapArray.html#setIndex(int)">setIndex(int)</a></span> - Method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/GapArray.html" title="class in org.biojava.bio.structure.align.helper">GapArray</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setInitialK(int)">setInitialK(int)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Author.html#setInitials(java.lang.String)">setInitials(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Author.html" title="class in org.biojava.bio.structure">Author</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ResidueNumber.html#setInsCode(java.lang.Character)">setInsCode(Character)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ResidueNumber.html" title="class in org.biojava.bio.structure">ResidueNumber</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SSBond.html#setInsCode1(java.lang.String)">setInsCode1(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SSBond.html" title="class in org.biojava.bio.structure">SSBond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SSBond.html#setInsCode2(java.lang.String)">setInsCode2(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SSBond.html" title="class in org.biojava.bio.structure">SSBond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#setInsertBegin(char)">setInsertBegin(char)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Initial insertion code of the PDB sequence segment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#setInsertEnd(char)">setInsertEnd(char)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Ending insertion code of the PDB sequence segment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#setInstalledDomainDescription(java.util.concurrent.atomic.AtomicBoolean)">setInstalledDomainDescription(AtomicBoolean)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#setInstalledDomainList(java.util.concurrent.atomic.AtomicBoolean)">setInstalledDomainList(AtomicBoolean)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#setInstalledDomall(java.util.concurrent.atomic.AtomicBoolean)">setInstalledDomall(AtomicBoolean)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathInstallation.html#setInstalledNodeList(java.util.concurrent.atomic.AtomicBoolean)">setInstalledNodeList(AtomicBoolean)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathInstallation.html" title="class in org.biojava.bio.structure.cath">CathInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#setInternalChainID(java.lang.String)">setInternalChainID(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Sets the internal chain ID that is used in mmCif files</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#setInternalChainID(java.lang.String)">setInternalChainID(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setIoTime(long)">setIoTime(long)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html#setIoTime(long)">setIoTime(long)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentResult</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#setIsPublished(java.lang.Boolean)">setIsPublished(Boolean)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>
<div class="block">Sets the publication state of a JournalArticle - TO BE PUBLISHED == false</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPDistanceMatrix.html#setJclose(int[])">setJclose(int[])</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPDistanceMatrix.html" title="class in org.biojava.bio.structure.domain.pdp">PDPDistanceMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setJoinFast(boolean)">setJoinFast(boolean)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setJoinPlo(boolean)">setJoinPlo(boolean)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setJoinRMSCutoff(double)">setJoinRMSCutoff(double)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#setJournalArticle(org.biojava.bio.structure.JournalArticle)">setJournalArticle(JournalArticle)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">set the associated publication as defined by the JRNL records in a PDB
 file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#setJournalArticle(org.biojava.bio.structure.JournalArticle)">setJournalArticle(JournalArticle)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">set the associated publication as defined by the JRNL records in a PDB
 file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#setJournalName(java.lang.String)">setJournalName(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucState.html#setKappa(float)">setKappa(float)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucState.html" title="class in org.biojava.bio.structure.secstruc">SecStrucState</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setLabel_alt_id(java.lang.String)">setLabel_alt_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setLabel_asym_id(java.lang.String)">setLabel_asym_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setLabel_atom_id(java.lang.String)">setLabel_atom_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setLabel_comp_id(java.lang.String)">setLabel_comp_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setLabel_entity_id(java.lang.String)">setLabel_entity_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setLabel_seq_id(java.lang.String)">setLabel_seq_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#setLcmp(int)">setLcmp(int)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#setLcmp(int)">setLcmp(int)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#setLength(int)">setLength(int)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#setLength(java.lang.Integer)">setLength(Integer)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathFragment.html#setLength(java.lang.Integer)">setLength(Integer)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathFragment.html" title="class in org.biojava.bio.structure.cath">CathFragment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathSegment.html#setLength(java.lang.Integer)">setLength(Integer)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathSegment.html" title="class in org.biojava.bio.structure.cath">CathSegment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html#setLength1(int)">setLength1(int)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentResult</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html#setLength2(int)">setLength2(int)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentResult</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/Ladder.html#setLfrom(int)">setLfrom(int)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/Ladder.html" title="class in org.biojava.bio.structure.secstruc">Ladder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#setLikeSequenceFamilyId(java.lang.Integer)">setLikeSequenceFamilyId(Integer)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileParsingParameters.html#setLoadChemCompInfo(boolean)">setLoadChemCompInfo(boolean)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileParsingParameters.html" title="class in org.biojava.bio.structure.io">FileParsingParameters</a></dt>
<dd>
<div class="block">Sets if chemical component defintions should be loaded from the web</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setLs_d_res_high(java.lang.String)">setLs_d_res_high(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setLs_d_res_low(java.lang.String)">setLs_d_res_low(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setLs_number_parameters(java.lang.String)">setLs_number_parameters(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setLs_number_reflns_all(java.lang.String)">setLs_number_reflns_all(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setLs_number_reflns_obs(java.lang.String)">setLs_number_reflns_obs(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setLs_number_reflns_R_free(java.lang.String)">setLs_number_reflns_R_free(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setLs_number_restraints(java.lang.String)">setLs_number_restraints(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setLs_percent_reflns_obs(java.lang.String)">setLs_percent_reflns_obs(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setLs_percent_reflns_R_free(java.lang.String)">setLs_percent_reflns_R_free(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setLs_R_factor_all(java.lang.String)">setLs_R_factor_all(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setLs_R_factor_obs(java.lang.String)">setLs_R_factor_obs(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setLs_R_factor_R_free(java.lang.String)">setLs_R_factor_R_free(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setLs_R_factor_R_free_error(java.lang.String)">setLs_R_factor_R_free_error(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setLs_R_factor_R_free_error_details(java.lang.String)">setLs_R_factor_R_free_error_details(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setLs_R_factor_R_work(java.lang.String)">setLs_R_factor_R_work(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setLs_redundancy_reflns_obs(java.lang.String)">setLs_redundancy_reflns_obs(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setLs_wR_factor_R_free(java.lang.String)">setLs_wR_factor_R_free(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setLs_wR_factor_R_work(java.lang.String)">setLs_wR_factor_R_work(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/Ladder.html#setLto(int)">setLto(int)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/Ladder.html" title="class in org.biojava.bio.structure.secstruc">Ladder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFP.html#setM(org.biojava.bio.structure.jama.Matrix)">setM(Matrix)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFP.html" title="class in org.biojava.bio.structure.align.model">AFP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#setMatMatrix(double[][])">setMatMatrix(double[][])</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#setMatMatrix(double[][])">setMatMatrix(double[][])</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html#setMatrix(org.biojava.bio.structure.jama.Matrix)">setMatrix(Matrix)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructOperList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#setMatrix(int, int, int, int, org.biojava.bio.structure.jama.Matrix)">setMatrix(int, int, int, int, Matrix)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Set a submatrix.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#setMatrix(int[], int[], org.biojava.bio.structure.jama.Matrix)">setMatrix(int[], int[], Matrix)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Set a submatrix.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#setMatrix(int[], int, int, org.biojava.bio.structure.jama.Matrix)">setMatrix(int[], int, int, Matrix)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Set a submatrix.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#setMatrix(int, int, int[], org.biojava.bio.structure.jama.Matrix)">setMatrix(int, int, int[], Matrix)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Set a submatrix.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html#setMatrix(org.biojava.bio.structure.jama.Matrix)">setMatrix(Matrix)</a></span> - Method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html" title="class in org.biojava.bio.structure.quaternary">ModelTransformationMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html#setMatrix11(double)">setMatrix11(double)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructOperList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html#setMatrix12(double)">setMatrix12(double)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructOperList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html#setMatrix13(double)">setMatrix13(double)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructOperList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html#setMatrix21(double)">setMatrix21(double)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructOperList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html#setMatrix22(double)">setMatrix22(double)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructOperList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html#setMatrix23(double)">setMatrix23(double)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructOperList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html#setMatrix31(double)">setMatrix31(double)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructOperList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html#setMatrix32(double)">setMatrix32(double)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructOperList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html#setMatrix33(double)">setMatrix33(double)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructOperList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileParsingParameters.html#setMaxAtoms(int)">setMaxAtoms(int)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileParsingParameters.html" title="class in org.biojava.bio.structure.io">FileParsingParameters</a></dt>
<dd>
<div class="block">the maximum numbers of atoms to load in a protein structure (prevents memory overflows)</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#setMaxGap(int)">setMaxGap(int)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#setMaxGapFrag(int)">setMaxGapFrag(int)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#setMaxGapSize(java.lang.Integer)">setMaxGapSize(Integer)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>
<div class="block">Set the Max gap size parameter.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#setMaxGapSize(int)">setMaxGapSize(int)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>
<div class="block">CE specific parameter: set the Max gap size parameter G (during AFP extension).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setMaxIter(int)">setMaxIter(int)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/BigSqrt.html#setMaxIterations(int)">setMaxIterations(int)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/BigSqrt.html" title="class in org.biojava.bio.structure.secstruc">BigSqrt</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#setMaxNrIterationsForOptimization(int)">setMaxNrIterationsForOptimization(int)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>
<div class="block">Set the maximum nr of times the (slow) optimiziation of alignment should iterate.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#setMaxOptRMSD(java.lang.Double)">setMaxOptRMSD(Double)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>
<div class="block">set the maximum RMSD cutoff to be applied during alignment optimization.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#setMaxOptRMSD(java.lang.Double)">setMaxOptRMSD(Double)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>
<div class="block">(jCE specific): maximum RMSD that shall be calculated for the alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#setMaxPenalty(double)">setMaxPenalty(double)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setMaxrefine(int)">setMaxrefine(int)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#setMaxTra(java.lang.Integer)">setMaxTra(Integer)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>
<div class="block">set the maximum number of Twists that are allowed...</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setMaxTra(int)">setMaxTra(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">Set the maximum number of Twists that are allowed...</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/PdbPairsMessage.html#setMethod(java.lang.String)">setMethod(String)</a></span> - Method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/PdbPairsMessage.html" title="class in org.biojava.bio.structure.align.xml">PdbPairsMessage</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html#setMethod(java.lang.String)">setMethod(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html" title="class in org.biojava.bio.structure.io.mmcif.model">Exptl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#setMethod(java.lang.String)">setMethod(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html#setMethod_details(java.lang.String)">setMethod_details(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html" title="class in org.biojava.bio.structure.io.mmcif.model">Exptl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssembly.html#setMethod_details(java.lang.String)">setMethod_details(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssembly.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructAssembly</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setMinLen(int)">setMinLen(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#setMisCut(int)">setMisCut(int)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setMisLen(int)">setMisLen(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#setMisScore(double)">setMisScore(double)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/MMCIFFileReader.html#setMMCifConsumer(org.biojava.bio.structure.io.mmcif.SimpleMMcifConsumer)">setMMCifConsumer(SimpleMMcifConsumer)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/MMCIFFileReader.html" title="class in org.biojava.bio.structure.io">MMCIFFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html#setMod_type(java.lang.String)">setMod_type(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html" title="class in org.biojava.bio.structure.io.mmcif.model">DatabasePDBrev</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#setModDate(java.util.Date)">setModDate(Date)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#setModel(int, java.util.List)">setModel(int, List&lt;Chain&gt;)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">a convenience function if one wants to edit and replace the
 models in a structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#setModel(int, java.util.List)">setModel(int, List&lt;Chain&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setMolId(java.lang.String)">setMolId(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>
<div class="block">Set the mol id value.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setMolName(java.lang.String)">setMolName(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/EntityPolySeq.html#setMon_id(java.lang.String)">setMon_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/EntityPolySeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">EntityPolySeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#setMon_id(java.lang.String)">setMon_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#setMon_id(java.lang.String)">setMon_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setMon_nstd_flag(java.lang.String)">setMon_nstd_flag(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setMon_nstd_parent_comp_id(java.lang.String)">setMon_nstd_parent_comp_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setMutation(java.lang.String)">setMutation(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html#setN(org.biojava.bio.structure.Atom)">setN(Atom)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html" title="class in org.biojava.bio.structure.secstruc">SecStrucGroup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#setName(java.lang.String)">setName(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">set trimmed version of atom name, e.g.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#setName(java.lang.String)">setName(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>
<div class="block">trimmed version of atom name, e.g.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#setName(java.lang.String)">setName(String)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#setName(java.lang.String)">setName(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>replaced by <a href="./org/biojava/bio/structure/Chain.html#setChainID(java.lang.String)"><code>Chain.setChainID(String)</code></a></i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#setName(java.lang.String)">setName(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AuditAuthor.html#setName(java.lang.String)">setName(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AuditAuthor.html" title="class in org.biojava.bio.structure.io.mmcif.model">AuditAuthor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setName(java.lang.String)">setName(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxEntityNonPoly.html#setName(java.lang.String)">setName(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxEntityNonPoly.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxEntityNonPoly</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDescription.html#setName(java.lang.String)">setName(String)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDescription.html" title="class in org.biojava.bio.structure.scop">ScopDescription</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#setName(java.lang.String)">setName(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">set biological name of Structure .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#setName(java.lang.String)">setName(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">set biological name of Structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/PdbPair.html#setName1(java.lang.String)">setName1(String)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/PdbPair.html" title="class in org.biojava.bio.structure.align.client">PdbPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setName1(java.lang.String)">setName1(String)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/PdbPair.html#setName2(java.lang.String)">setName2(String)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/PdbPair.html" title="class in org.biojava.bio.structure.align.client">PdbPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setName2(java.lang.String)">setName2(String)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#setnAtom(int)">setnAtom(int)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#setnAtom(int)">setnAtom(int)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html#setNaturalPos(java.lang.Integer)">setNaturalPos(Integer)</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html" title="class in org.biojava.bio.structure.io.sifts">SiftsResidue</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPDistanceMatrix.html#setNclose(int)">setNclose(int)</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPDistanceMatrix.html" title="class in org.biojava.bio.structure.domain.pdp">PDPDistanceMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/CutSites.html#setNcuts(int)">setNcuts(int)</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/CutSites.html" title="class in org.biojava.bio.structure.domain.pdp">CutSites</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#setNdb_seq_num(java.lang.String)">setNdb_seq_num(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#setNdb_seq_num(java.lang.String)">setNdb_seq_num(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html#setNdbChainId(java.lang.String)">setNdbChainId(String)</a></span> - Method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html" title="class in org.biojava.bio.structure.quaternary">ModelTransformationMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#setNmr(boolean)">setNmr(boolean)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">set NMR flag.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#setNmr(boolean)">setNmr(boolean)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathNode.html#setNodeId(java.lang.String)">setNodeId(String)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathNode.html" title="class in org.biojava.bio.structure.cath">CathNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setNormAlignScore(double)">setNormAlignScore(double)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html#setNotObserved(java.lang.Boolean)">setNotObserved(Boolean)</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html" title="class in org.biojava.bio.structure.io.sifts">SiftsResidue</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#setNrAlignments(int)">setNrAlignments(int)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#setNrBioAssemblies(int)">setNrBioAssemblies(int)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Domain.html#setNseg(int)">setNseg(int)</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Domain.html" title="class in org.biojava.bio.structure.domain.pdp">Domain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html#setNum(java.lang.String)">setNum(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html" title="class in org.biojava.bio.structure.io.mmcif.model">DatabasePDBrev</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/EntityPolySeq.html#setNum(java.lang.String)">setNum(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/EntityPolySeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">EntityPolySeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setNumRes(java.lang.String)">setNumRes(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html#setO(org.biojava.bio.structure.Atom)">setO(Atom)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html" title="class in org.biojava.bio.structure.secstruc">SecStrucGroup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#setOccupancy(double)">setOccupancy(double)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">set occupancy.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#setOccupancy(double)">setOccupancy(double)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setOccupancy(java.lang.String)">setOccupancy(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setOccupancy_esd(java.lang.String)">setOccupancy_esd(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setOccupancy_max(java.lang.String)">setOccupancy_max(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setOccupancy_min(java.lang.String)">setOccupancy_min(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssembly.html#setOligomeric_count(java.lang.String)">setOligomeric_count(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssembly.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructAssembly</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssembly.html#setOligomeric_details(java.lang.String)">setOligomeric_details(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssembly.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructAssembly</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucState.html#setOmega(double)">setOmega(double)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucState.html" title="class in org.biojava.bio.structure.secstruc">SecStrucState</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setOne_letter_code(java.lang.String)">setOne_letter_code(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssemblyGen.html#setOper_expression(java.lang.String)">setOper_expression(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssemblyGen.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructAssemblyGen</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setOptAln(int[][][])">setOptAln(int[][][])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setOptLen(int[])">setOptLen(int[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setOptLength(int)">setOptLength(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">The length of the optimal alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setOptRmsd(double[])">setOptRmsd(double[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setOrgan(java.lang.String)">setOrgan(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setOrganelle(java.lang.String)">setOrganelle(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setOrganismCommon(java.lang.String)">setOrganismCommon(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setOrganismScientific(java.lang.String)">setOrganismScientific(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setOrganismTaxId(java.lang.String)">setOrganismTaxId(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html#setOriginal(org.biojava.bio.structure.Group)">setOriginal(Group)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html" title="class in org.biojava.bio.structure.secstruc">SecStrucGroup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#setORmsdThr(java.lang.Double)">setORmsdThr(Double)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>
<div class="block">Set the Original RMSD threshold from which the alignment optimization is started</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#setOrthologousSequenceFamilyId(java.lang.Integer)">setOrthologousSequenceFamilyId(Integer)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html#setOutFile(org.biojava.bio.structure.align.util.SynchronizedOutFile)">setOutFile(SynchronizedOutFile)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html" title="class in org.biojava.bio.structure.align">CallableStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#setOutFile(java.lang.String)">setOutFile(String)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html#setOutputDir(java.io.File)">setOutputDir(File)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html" title="class in org.biojava.bio.structure.align">CallableStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#setOutputPDB(boolean)">setOutputPDB(boolean)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>
<div class="block">When writing the results to a file, don;t write as XML but write aligned PDB file</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setOverall_FOM_free_R_set(java.lang.String)">setOverall_FOM_free_R_set(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setOverall_FOM_work_R_set(java.lang.String)">setOverall_FOM_work_R_set(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setOverall_SU_B(java.lang.String)">setOverall_SU_B(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setOverall_SU_ML(java.lang.String)">setOverall_SU_ML(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setOverall_SU_R_Cruickshank_DPI(java.lang.String)">setOverall_SU_R_Cruickshank_DPI(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setOverall_SU_R_free(java.lang.String)">setOverall_SU_R_free(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFP.html#setP1(int)">setP1(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFP.html" title="class in org.biojava.bio.structure.align.model">AFP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFP.html#setP2(int)">setP2(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFP.html" title="class in org.biojava.bio.structure.align.model">AFP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html#setPair(org.biojava.bio.structure.align.client.PdbPair)">setPair(PdbPair)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html" title="class in org.biojava.bio.structure.align">CallableStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/PdbPairsMessage.html#setPairs(java.util.SortedSet)">setPairs(SortedSet&lt;PdbPair&gt;)</a></span> - Method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/PdbPairsMessage.html" title="class in org.biojava.bio.structure.align.xml">PdbPairsMessage</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/AbstractStructureAlignment.html#setParameters(org.biojava.bio.structure.align.ce.ConfigStrucAligParams)">setParameters(ConfigStrucAligParams)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/AbstractStructureAlignment.html" title="class in org.biojava.bio.structure.align">AbstractStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/BioJavaStructureAlignment.html#setParameters(org.biojava.bio.structure.align.ce.ConfigStrucAligParams)">setParameters(ConfigStrucAligParams)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/BioJavaStructureAlignment.html" title="class in org.biojava.bio.structure.align">BioJavaStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html#setParameters(org.biojava.bio.structure.align.ce.ConfigStrucAligParams)">setParameters(ConfigStrucAligParams)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html" title="class in org.biojava.bio.structure.align">CallableStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeMain.html#setParameters(org.biojava.bio.structure.align.ce.ConfigStrucAligParams)">setParameters(ConfigStrucAligParams)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeMain.html" title="class in org.biojava.bio.structure.align.ce">CeMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeSideChainMain.html#setParameters(org.biojava.bio.structure.align.ce.ConfigStrucAligParams)">setParameters(ConfigStrucAligParams)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeSideChainMain.html" title="class in org.biojava.bio.structure.align.ce">CeSideChainMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/OptimalCECPMain.html#setParameters(org.biojava.bio.structure.align.ce.ConfigStrucAligParams)">setParameters(ConfigStrucAligParams)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/OptimalCECPMain.html" title="class in org.biojava.bio.structure.align.ce">OptimalCECPMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCatFlexible.html#setParameters(org.biojava.bio.structure.align.ce.ConfigStrucAligParams)">setParameters(ConfigStrucAligParams)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCatFlexible.html" title="class in org.biojava.bio.structure.align.fatcat">FatCatFlexible</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCatRigid.html#setParameters(org.biojava.bio.structure.align.ce.ConfigStrucAligParams)">setParameters(ConfigStrucAligParams)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCatRigid.html" title="class in org.biojava.bio.structure.align.fatcat">FatCatRigid</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWaterman3Daligner.html#setParameters(org.biojava.bio.structure.align.ce.ConfigStrucAligParams)">setParameters(ConfigStrucAligParams)</a></span> - Method in class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWaterman3Daligner.html" title="class in org.biojava.bio.structure.align.seq">SmithWaterman3Daligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructureAlignment.html#setParameters(org.biojava.bio.structure.align.ce.ConfigStrucAligParams)">setParameters(ConfigStrucAligParams)</a></span> - Method in interface org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructureAlignment.html" title="interface in org.biojava.bio.structure.align">StructureAlignment</a></dt>
<dd>
<div class="block">Set the default parameters for this algorithm to use</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/FarmJob.html#setParams(org.biojava.bio.structure.align.client.FarmJobParameters)">setParams(FarmJobParameters)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/FarmJob.html" title="class in org.biojava.bio.structure.align">FarmJob</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructurePairAligner.html#setParams(org.biojava.bio.structure.align.StrucAligParameters)">setParams(StrucAligParameters)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructurePairAligner.html" title="class in org.biojava.bio.structure.align">StructurePairAligner</a></dt>
<dd>
<div class="block">set the parameters to be used for the algorithm</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#setParent(org.biojava.bio.structure.Group)">setParent(Group)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>replaced by <a href="./org/biojava/bio/structure/Atom.html#setGroup(org.biojava.bio.structure.Group)"><code>Atom.setGroup(Group)</code></a></i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#setParent(org.biojava.bio.structure.Group)">setParent(Group)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#setParent(org.biojava.bio.structure.Structure)">setParent(Structure)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Set the back-reference to its parent Structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#setParent(org.biojava.bio.structure.Structure)">setParent(Structure)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Set the back-reference to its parent Structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#setParent(org.biojava.bio.structure.Structure)">setParent(Structure)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Set the structure object that this DBRef relates to.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#setParent(org.biojava.bio.structure.Chain)">setParent(Chain)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#setParent(org.biojava.bio.structure.Chain)">setParent(Chain)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">Set the back-reference (to its parent Chain)</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathNode.html#setParentId(java.lang.String)">setParentId(String)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathNode.html" title="class in org.biojava.bio.structure.cath">CathNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopNode.html#setParentSunid(int)">setParentSunid(int)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopNode.html" title="class in org.biojava.bio.structure.scop">ScopNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileParsingParameters.html#setParseBioAssembly(boolean)">setParseBioAssembly(boolean)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileParsingParameters.html" title="class in org.biojava.bio.structure.io">FileParsingParameters</a></dt>
<dd>
<div class="block">Should the biological assembly info (REMARK 350) be parsed from the PDB file?</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileParsingParameters.html#setParseCAOnly(boolean)">setParseCAOnly(boolean)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileParsingParameters.html" title="class in org.biojava.bio.structure.io">FileParsingParameters</a></dt>
<dd>
<div class="block">the flag if only the C-alpha atoms of the structure should be parsed.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileParsingParameters.html#setParseSecStruc(boolean)">setParseSecStruc(boolean)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileParsingParameters.html" title="class in org.biojava.bio.structure.io">FileParsingParameters</a></dt>
<dd>
<div class="block">a flag to tell the parser to parse the Author's secondary structure assignment from the file
 default is set to false, i.e.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/HBond.html#setPartner(int)">setPartner(int)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/HBond.html" title="class in org.biojava.bio.structure.secstruc">HBond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/Alignable.html#setPath(org.biojava.bio.structure.align.helper.IndexPair[])">setPath(IndexPair[])</a></span> - Method in interface org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/Alignable.html" title="interface in org.biojava.bio.structure.align.pairwise">Alignable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html#setPath(org.biojava.bio.structure.align.helper.IndexPair[])">setPath(IndexPair[])</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">StrCompAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#setPath(java.lang.String)">setPath(String)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>
<div class="block">Set the path that is used to cache PDB files.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/LocalCacheStructureProvider.html#setPath(java.lang.String)">setPath(String)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/LocalCacheStructureProvider.html" title="class in org.biojava.bio.structure.io">LocalCacheStructureProvider</a></dt>
<dd>
<div class="block">directory where to find PDB files</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/DownloadChemCompProvider.html#setPath(java.lang.String)">setPath(String)</a></span> - Static method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/DownloadChemCompProvider.html" title="class in org.biojava.bio.structure.io.mmcif">DownloadChemCompProvider</a></dt>
<dd>
<div class="block">making sure we use the same path for the PDB installation as is used by the PdbFileReader</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/MMCIFFileReader.html#setPath(java.lang.String)">setPath(String)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/MMCIFFileReader.html" title="class in org.biojava.bio.structure.io">MMCIFFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#setPath(java.lang.String)">setPath(String)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>
<div class="block">directory where to find PDB files</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/SandboxStyleStructureProvider.html#setPath(java.lang.String)">setPath(String)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/SandboxStyleStructureProvider.html" title="class in org.biojava.bio.structure.io">SandboxStyleStructureProvider</a></dt>
<dd>
<div class="block">directory where to find PDB files</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/StructureIOFile.html#setPath(java.lang.String)">setPath(String)</a></span> - Method in interface org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/StructureIOFile.html" title="interface in org.biojava.bio.structure.io">StructureIOFile</a></dt>
<dd>
<div class="block">Set path to file / connection string to db.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/Alignable.html#setPathSize(int)">setPathSize(int)</a></span> - Method in interface org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/Alignable.html" title="interface in org.biojava.bio.structure.align.pairwise">Alignable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html#setPathSize(int)">setPathSize(int)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">StrCompAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#setPdb1(java.lang.String)">setPdb1(String)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>
<div class="block">mandatory argument to set the first PDB (and optionally chain ID) to be aligned.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html#setPdb1(java.lang.String)">setPdb1(String)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentResult</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#setPdb2(java.lang.String)">setPdb2(String)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>
<div class="block">mandatory argument to set the second PDB (and optionally chain ID) to be aligned.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html#setPdb2(java.lang.String)">setPdb2(String)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentResult</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#setPdb_ins_code(java.lang.String)">setPdb_ins_code(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#setPdb_ins_code(java.lang.String)">setPdb_ins_code(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#setPdb_mon_id(java.lang.String)">setPdb_mon_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#setPdb_mon_id(java.lang.String)">setPdb_mon_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#setPdb_seq_num(java.lang.String)">setPdb_seq_num(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#setPdb_seq_num(java.lang.String)">setPdb_seq_num(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#setPdb_strand_id(java.lang.String)">setPdb_strand_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#setPdb_strand_id(java.lang.String)">setPdb_strand_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setPdbAln(java.lang.String[][][])">setPdbAln(String[][][])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#setPDBCode(java.lang.String)">setPDBCode(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>replaced by <a href="./org/biojava/bio/structure/Group.html#setResidueNumber(org.biojava.bio.structure.ResidueNumber)"><code>Group.setResidueNumber(ResidueNumber)</code></a></i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#setPDBCode(java.lang.String)">setPDBCode(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>replaced by <a href="./org/biojava/bio/structure/HetatomImpl.html#setResidueNumber(org.biojava.bio.structure.ResidueNumber)"><code>HetatomImpl.setResidueNumber(ResidueNumber)</code></a></i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureEvent.html#setPDBCode(java.lang.String)">setPDBCode(String)</a></span> - Method in interface org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureEvent.html" title="interface in org.biojava.bio.structure.server">StructureEvent</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureEventImpl.html#setPDBCode(java.lang.String)">setPDBCode(String)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureEventImpl.html" title="class in org.biojava.bio.structure.server">StructureEventImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#setPDBCode(java.lang.String)">setPDBCode(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">set PDB code of structure .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#setPDBCode(java.lang.String)">setPDBCode(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">set PDB code of structure .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/MMCIFFileReader.html#setPdbDirectorySplit(boolean)">setPdbDirectorySplit(boolean)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/MMCIFFileReader.html" title="class in org.biojava.bio.structure.io">MMCIFFileReader</a></dt>
<dd>
<div class="block">Flag that defines if the PDB directory is containing all PDB files or is split into sub dirs (like the FTP site).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#setPdbDirectorySplit(boolean)">setPdbDirectorySplit(boolean)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>
<div class="block">Flag that defines if the PDB directory is containing all PDB files or is split into sub dirs (like the FTP site).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/StructureIOFile.html#setPdbDirectorySplit(boolean)">setPdbDirectorySplit(boolean)</a></span> - Method in interface org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/StructureIOFile.html" title="interface in org.biojava.bio.structure.io">StructureIOFile</a></dt>
<dd>
<div class="block">The PDB files are organized hierarchically (as on the PDB - FTP server.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#setPdbDirSplit(boolean)">setPdbDirSplit(boolean)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#setPdbDirSplit(boolean)">setPdbDirSplit(boolean)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#setPdbFilePath(java.lang.String)">setPdbFilePath(String)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>
<div class="block">mandatory argument to set the location of PDB files.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#setPdbFilePath(java.lang.String)">setPdbFilePath(String)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/UserConfiguration.html#setPdbFilePath(java.lang.String)">setPdbFilePath(String)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/UserConfiguration.html" title="class in org.biojava.bio.structure.align.util">UserConfiguration</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#setPDBFlag(boolean)">setPDBFlag(boolean)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">flag if group has 3D data .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#setPDBFlag(boolean)">setPDBFlag(boolean)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">flag if group has 3D data.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#setPDBHeader(org.biojava.bio.structure.PDBHeader)">setPDBHeader(PDBHeader)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">the the header information for this PDB file</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#setPDBHeader(org.biojava.bio.structure.PDBHeader)">setPDBHeader(PDBHeader)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html#setPdbId(java.lang.String)">setPdbId(String)</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html" title="class in org.biojava.bio.structure.io.sifts">SiftsResidue</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/MmCifPDBBiolAssemblyProvider.html#setPdbId(java.lang.String)">setPdbId(String)</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/MmCifPDBBiolAssemblyProvider.html" title="class in org.biojava.bio.structure.quaternary.io">MmCifPDBBiolAssemblyProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/RawBioUnitDataProvider.html#setPdbId(java.lang.String)">setPdbId(String)</a></span> - Method in interface org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/RawBioUnitDataProvider.html" title="interface in org.biojava.bio.structure.quaternary.io">RawBioUnitDataProvider</a></dt>
<dd>
<div class="block">Tell the provider for which PDB ID the quaternary structure should be returned.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/RemoteRawBioUnitDataProvider.html#setPdbId(java.lang.String)">setPdbId(String)</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/RemoteRawBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">RemoteRawBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#setPdbId(java.lang.String)">setPdbId(String)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/FlatFileInstallation.html#setPDBInfoFile(java.io.File)">setPDBInfoFile(File)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/FlatFileInstallation.html" title="class in org.biojava.bio.structure.server">FlatFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/SimpleStructureServer.html#setPDBInstallation(org.biojava.bio.structure.server.PDBInstallation)">setPDBInstallation(PDBInstallation)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/SimpleStructureServer.html" title="class in org.biojava.bio.structure.server">SimpleStructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureServer.html#setPDBInstallation(org.biojava.bio.structure.server.PDBInstallation)">setPDBInstallation(PDBInstallation)</a></span> - Method in interface org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureServer.html" title="interface in org.biojava.bio.structure.server">StructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#setPDBline(java.lang.String)">setPDBline(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>replaced by <a href="./org/biojava/bio/structure/PDBRecord.html#toPDB()"><code>PDBRecord.toPDB()</code></a></i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#setPDBline(java.lang.String)">setPDBline(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>&nbsp;</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcidImpl.html#setPDBName(java.lang.String)">setPDBName(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcidImpl.html" title="class in org.biojava.bio.structure">AminoAcidImpl</a></dt>
<dd>
<div class="block">set three character name of AminoAcid.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#setPDBName(java.lang.String)">setPDBName(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">Set the PDB 3 letter name for this group.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#setPDBName(java.lang.String)">setPDBName(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">set three character name of Group .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/structure/StructureIO.html#setPdbPath(java.lang.String)">setPdbPath(String)</a></span> - Static method in class org.biojava3.structure.<a href="./org/biojava3/structure/StructureIO.html" title="class in org.biojava3.structure">StructureIO</a></dt>
<dd>
<div class="block">Utility method to set the location where PDB files can be found</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html#setPdbResName(java.lang.String)">setPdbResName(String)</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html" title="class in org.biojava.bio.structure.io.sifts">SiftsResidue</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html#setPdbResNum(java.lang.String)">setPdbResNum(String)</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html" title="class in org.biojava.bio.structure.io.sifts">SiftsResidue</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#setPDBresnum1(java.lang.String[])">setPDBresnum1(String[])</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#setPDBresnum2(java.lang.String[])">setPDBresnum2(String[])</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#setPDBserial(int)">setPDBserial(int)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">set PDB atom number.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#setPDBserial(int)">setPDBserial(int)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>
<div class="block">set PDB atom number.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html#setPdbx_align_begin(java.lang.String)">setPdbx_align_begin(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRef</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setPdbx_ambiguous_flag(java.lang.String)">setPdbx_ambiguous_flag(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#setPdbx_auth_seq_align_beg(java.lang.String)">setPdbx_auth_seq_align_beg(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#setPdbx_auth_seq_align_end(java.lang.String)">setPdbx_auth_seq_align_end(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html#setPdbx_blank_PDB_chainid_flag(java.lang.String)">setPdbx_blank_PDB_chainid_flag(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructAsym</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html#setPdbx_CASP_flag(java.lang.String)">setPdbx_CASP_flag(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html" title="class in org.biojava.bio.structure.io.mmcif.model">Struct</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setPdbx_data_cutoff_high_absF(java.lang.String)">setPdbx_data_cutoff_high_absF(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setPdbx_data_cutoff_high_rms_absF(java.lang.String)">setPdbx_data_cutoff_high_rms_absF(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setPdbx_data_cutoff_low_absF(java.lang.String)">setPdbx_data_cutoff_low_absF(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html#setPdbx_db_accession(java.lang.String)">setPdbx_db_accession(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRef</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#setPdbx_db_accession(java.lang.String)">setPdbx_db_accession(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#setPdbx_db_align_beg_ins_code(java.lang.String)">setPdbx_db_align_beg_ins_code(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#setPdbx_db_align_end_ins_code(java.lang.String)">setPdbx_db_align_end_ins_code(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html#setPdbx_description(java.lang.String)">setPdbx_description(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model">Entity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html#setPdbx_descriptor(java.lang.String)">setPdbx_descriptor(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html" title="class in org.biojava.bio.structure.io.mmcif.model">Struct</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setPdbx_diffrn_id(java.lang.String)">setPdbx_diffrn_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html#setPdbx_ec(java.lang.String)">setPdbx_ec(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model">Entity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setPdbx_formal_charge(java.lang.String)">setPdbx_formal_charge(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setPdbx_formal_charge(java.lang.String)">setPdbx_formal_charge(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html#setPdbx_fragment(java.lang.String)">setPdbx_fragment(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model">Entity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setPdbx_ideal_coordinates_details(java.lang.String)">setPdbx_ideal_coordinates_details(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setPdbx_ideal_coordinates_missing_flag(java.lang.String)">setPdbx_ideal_coordinates_missing_flag(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setPdbx_initial_date(java.lang.String)">setPdbx_initial_date(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setPdbx_isotropic_thermal_model(java.lang.String)">setPdbx_isotropic_thermal_model(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructKeywords.html#setPdbx_keywords(java.lang.String)">setPdbx_keywords(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructKeywords.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructKeywords</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setPdbx_ls_cross_valid_method(java.lang.String)">setPdbx_ls_cross_valid_method(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setPdbx_ls_sigma_F(java.lang.String)">setPdbx_ls_sigma_F(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setPdbx_ls_sigma_I(java.lang.String)">setPdbx_ls_sigma_I(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setPdbx_method_to_determine_struct(java.lang.String)">setPdbx_method_to_determine_struct(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setPdbx_model_coordinates_db_code(java.lang.String)">setPdbx_model_coordinates_db_code(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setPdbx_model_coordinates_details(java.lang.String)">setPdbx_model_coordinates_details(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setPdbx_model_coordinates_missing_flag(java.lang.String)">setPdbx_model_coordinates_missing_flag(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html#setPdbx_model_details(java.lang.String)">setPdbx_model_details(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html" title="class in org.biojava.bio.structure.io.mmcif.model">Struct</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html#setPdbx_model_type_details(java.lang.String)">setPdbx_model_type_details(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html" title="class in org.biojava.bio.structure.io.mmcif.model">Struct</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html#setPdbx_modified(java.lang.String)">setPdbx_modified(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructAsym</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setPdbx_modified_date(java.lang.String)">setPdbx_modified_date(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html#setPdbx_mutation(java.lang.String)">setPdbx_mutation(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model">Entity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html#setPdbx_number_of_molecules(java.lang.String)">setPdbx_number_of_molecules(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model">Entity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AuditAuthor.html#setPdbx_ordinal(java.lang.String)">setPdbx_ordinal(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AuditAuthor.html" title="class in org.biojava.bio.structure.io.mmcif.model">AuditAuthor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setPdbx_overall_ESU_R(java.lang.String)">setPdbx_overall_ESU_R(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setPdbx_overall_ESU_R_Free(java.lang.String)">setPdbx_overall_ESU_R_Free(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setPdbx_overall_phase_error(java.lang.String)">setPdbx_overall_phase_error(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setPdbx_overall_SU_R_Blow_DPI(java.lang.String)">setPdbx_overall_SU_R_Blow_DPI(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setPdbx_overall_SU_R_free_Blow_DPI(java.lang.String)">setPdbx_overall_SU_R_free_Blow_DPI(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setPdbx_overall_SU_R_free_Cruickshank_DPI(java.lang.String)">setPdbx_overall_SU_R_free_Cruickshank_DPI(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#setPdbx_PDB_id_code(java.lang.String)">setPdbx_PDB_id_code(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setPdbx_PDB_ins_code(java.lang.String)">setPdbx_PDB_ins_code(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setPdbx_PDB_model_num(java.lang.String)">setPdbx_PDB_model_num(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setPdbx_processing_site(java.lang.String)">setPdbx_processing_site(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setPdbx_R_Free_selection_details(java.lang.String)">setPdbx_R_Free_selection_details(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setPdbx_refine_id(java.lang.String)">setPdbx_refine_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setPdbx_release_status(java.lang.String)">setPdbx_release_status(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setPdbx_replaced_by(java.lang.String)">setPdbx_replaced_by(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setPdbx_replaces(java.lang.String)">setPdbx_replaces(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#setPdbx_seq_align_beg_ins_code(java.lang.String)">setPdbx_seq_align_beg_ins_code(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#setPdbx_seq_align_end_ins_code(java.lang.String)">setPdbx_seq_align_end_ins_code(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html#setPdbx_seq_one_letter_code(java.lang.String)">setPdbx_seq_one_letter_code(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRef</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setPdbx_solvent_ion_probe_radii(java.lang.String)">setPdbx_solvent_ion_probe_radii(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setPdbx_solvent_shrinkage_radii(java.lang.String)">setPdbx_solvent_shrinkage_radii(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setPdbx_solvent_vdw_probe_radii(java.lang.String)">setPdbx_solvent_vdw_probe_radii(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setPdbx_starting_model(java.lang.String)">setPdbx_starting_model(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setPdbx_stereochem_target_val_spec_case(java.lang.String)">setPdbx_stereochem_target_val_spec_case(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setPdbx_stereochemistry_target_values(java.lang.String)">setPdbx_stereochemistry_target_values(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#setPdbx_strand_id(java.lang.String)">setPdbx_strand_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setPdbx_subcomponent_list(java.lang.String)">setPdbx_subcomponent_list(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setPdbx_synonyms(java.lang.String)">setPdbx_synonyms(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setPdbx_TLS_residual_ADP_flag(java.lang.String)">setPdbx_TLS_residual_ADP_flag(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setPdbx_type(java.lang.String)">setPdbx_type(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssemblyGenXMLContainer.html#setPdbxStructAssemblies(java.util.List)">setPdbxStructAssemblies(List&lt;PdbxStructAssemblyGen&gt;)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssemblyGenXMLContainer.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructAssemblyGenXMLContainer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssemblyXMLContainer.html#setPdbxStructAssemblies(java.util.List)">setPdbxStructAssemblies(List&lt;PdbxStructAssembly&gt;)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssemblyXMLContainer.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructAssemblyXMLContainer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperListXMLContainer.html#setPdbxStructOperLists(java.util.List)">setPdbxStructOperLists(List&lt;PdbxStructOperList&gt;)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperListXMLContainer.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructOperListXMLContainer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#setPdpprovider(org.biojava.bio.structure.domain.PDPProvider)">setPdpprovider(PDPProvider)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#setPercId(int)">setPercId(int)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setPermutationSize(int)">setPermutationSize(int)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucState.html#setPhi(double)">setPhi(double)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucState.html" title="class in org.biojava.bio.structure.secstruc">SecStrucState</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#setPmid(java.lang.String)">setPmid(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>
<div class="block">Set the value of pmid</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setPolymerType(org.biojava.bio.structure.io.mmcif.chem.PolymerType)">setPolymerType(PolymerType)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#setPos1(int)">setPos1(int)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#setPos2(int)">setPos2(int)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#setPrintCE(boolean)">setPrintCE(boolean)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>
<div class="block">Display the output string in CE style</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileConvert.html#setPrintConnections(boolean)">setPrintConnections(boolean)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileConvert.html" title="class in org.biojava.bio.structure.io">FileConvert</a></dt>
<dd>
<div class="block">enable/disable printing of connections
 connections are sometimes buggy in PDB files
 so there are some cases where one might turn this off.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#setPrintFatCat(boolean)">setPrintFatCat(boolean)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#setPrintXML(boolean)">setPrintXML(boolean)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setProbability(double)">setProbability(double)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html#setProgram(java.lang.String)">setProgram(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemCompDescriptor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html#setProgram_version(java.lang.String)">setProgram_version(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemCompDescriptor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#setProperties(java.util.Map)">setProperties(Map&lt;String, Object&gt;)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">properties of this amino acid.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#setProperties(java.util.Map)">setProperties(Map&lt;String, Object&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">properties of this amino acid.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#setProperty(java.lang.String, java.lang.Object)">setProperty(String, Object)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">set a single property .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#setProperty(java.lang.String, java.lang.Object)">setProperty(String, Object)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">set a single property .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucState.html#setPsi(double)">setPsi(double)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucState.html" title="class in org.biojava.bio.structure.secstruc">SecStrucState</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#setPublicationDate(int)">setPublicationDate(int)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#setPublished(boolean)">setPublished(boolean)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#setPublisher(java.lang.String)">setPublisher(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#setPx(int)">setPx(int)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#setRanges(java.util.List)">setRanges(List&lt;String&gt;)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcid.html#setRecordType(java.lang.String)">setRecordType(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcid.html" title="interface in org.biojava.bio.structure">AminoAcid</a></dt>
<dd>
<div class="block">Allows to distinguish between amino acids that are provided
                as ATOM records and a SEQRES records.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcidImpl.html#setRecordType(java.lang.String)">setRecordType(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcidImpl.html" title="class in org.biojava.bio.structure">AminoAcidImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setReduceInitialFragments(boolean)">setReduceInitialFragments(boolean)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#setRef(java.lang.String)">setRef(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>
<div class="block">Set the value of the ref.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#setRef_id(java.lang.String)">setRef_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setRefChainId(java.lang.String)">setRefChainId(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>
<div class="block">Returns the ref chain id value.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#setRefn(java.lang.String)">setRefn(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>
<div class="block">Set the value of the refn</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html#setReplaces(java.lang.String)">setReplaces(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html" title="class in org.biojava.bio.structure.io.mmcif.model">DatabasePDBrev</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathNode.html#setRepresentative(java.lang.String)">setRepresentative(String)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathNode.html" title="class in org.biojava.bio.structure.cath">CathNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#setResidueNumber(org.biojava.bio.structure.ResidueNumber)">setResidueNumber(ResidueNumber)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">sets the ResidueNumber for this Group</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#setResidueNumber(java.lang.String, java.lang.Integer, java.lang.Character)">setResidueNumber(String, Integer, Character)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">Utility method to temporarily set a chainID for a group, if a parent chain object does not exist yet.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#setResidueNumber(org.biojava.bio.structure.ResidueNumber)">setResidueNumber(ResidueNumber)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#setResidueNumber(java.lang.String, java.lang.Integer, java.lang.Character)">setResidueNumber(String, Integer, Character)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsSegment.html#setResidues(java.util.List)">setResidues(List&lt;SiftsResidue&gt;)</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsSegment.html" title="class in org.biojava.bio.structure.io.sifts">SiftsSegment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setResidueType(org.biojava.bio.structure.io.mmcif.chem.ResidueType)">setResidueType(ResidueType)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setResNames(java.lang.String)">setResNames(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SSBond.html#setResnum1(java.lang.String)">setResnum1(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SSBond.html" title="class in org.biojava.bio.structure">SSBond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SSBond.html#setResnum2(java.lang.String)">setResnum2(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SSBond.html" title="class in org.biojava.bio.structure">SSBond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#setResolution(java.lang.Double)">setResolution(Double)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#setResolution(float)">setResolution(float)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#setResScore(double)">setResScore(double)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/JointFragments.html#setRms(double)">setRms(double)</a></span> - Method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/JointFragments.html" title="class in org.biojava.bio.structure.align.helper">JointFragments</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#setRms(double)">setRms(double)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>
<div class="block">the rms in the structurally equivalent regions</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#setRms(double)">setRms(double)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFP.html#setRmsd(double)">setRmsd(double)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFP.html" title="class in org.biojava.bio.structure.align.model">AFP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#setRmsdCut(java.lang.Double)">setRmsdCut(Double)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>
<div class="block">The cutoff to be used during AFP detection</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#setRmsdThr(java.lang.Double)">setRmsdThr(Double)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#setRmsdThrJoin(java.lang.Double)">setRmsdThrJoin(Double)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#setRot(org.biojava.bio.structure.jama.Matrix)">setRot(Matrix)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/IndexPair.html#setRow(short)">setRow(short)</a></span> - Method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/IndexPair.html" title="class in org.biojava.bio.structure.align.helper">IndexPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#setRunBackground(boolean)">setRunBackground(boolean)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#setSaveOutputDir(java.lang.String)">setSaveOutputDir(String)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/BigSqrt.html#setScale(int)">setScale(int)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/BigSqrt.html" title="class in org.biojava.bio.structure.secstruc">BigSqrt</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopFactory.html#setScopDatabase(org.biojava.bio.structure.scop.ScopDatabase)">setScopDatabase(ScopDatabase)</a></span> - Static method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopFactory.html" title="class in org.biojava.bio.structure.scop">ScopFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/ScopDescriptions.html#setScopDescription(java.util.List)">setScopDescription(List&lt;ScopDescription&gt;)</a></span> - Method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/ScopDescriptions.html" title="class in org.biojava.bio.structure.scop.server">ScopDescriptions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/ScopDomains.html#setScopDomain(java.util.List)">setScopDomain(List&lt;ScopDomain&gt;)</a></span> - Method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/ScopDomains.html" title="class in org.biojava.bio.structure.scop.server">ScopDomains</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#setScopDownloadURL(java.lang.String)">setScopDownloadURL(String)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#setScopId(java.lang.String)">setScopId(String)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/ScopNodes.html#setScopNode(java.util.List)">setScopNode(List&lt;ScopNode&gt;)</a></span> - Method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/ScopNodes.html" title="class in org.biojava.bio.structure.scop.server">ScopNodes</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#setScopVersion(java.lang.String)">setScopVersion(String)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFP.html#setScore(double)">setScore(double)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFP.html" title="class in org.biojava.bio.structure.align.model">AFP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/Alignable.html#setScore(float)">setScore(float)</a></span> - Method in interface org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/Alignable.html" title="interface in org.biojava.bio.structure.align.pairwise">Alignable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#setScore(float)">setScore(float)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html#setScore(float)">setScore(float)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">StrCompAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Domain.html#setScore(double)">setScore(double)</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Domain.html" title="class in org.biojava.bio.structure.domain.pdp">Domain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Segment.html#setScore(double)">setScore(double)</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Segment.html" title="class in org.biojava.bio.structure.domain.pdp">Segment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#setScoringStrategy(java.lang.Integer)">setScoringStrategy(Integer)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>
<div class="block">Set the scoring strategy to use.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setSecretion(java.lang.String)">setSecretion(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcid.html#setSecStruc(java.util.Map)">setSecStruc(Map&lt;String, String&gt;)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcid.html" title="interface in org.biojava.bio.structure">AminoAcid</a></dt>
<dd>
<div class="block">Specifies the secondary structure as a Map.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcidImpl.html#setSecStruc(java.util.Map)">setSecStruc(Map&lt;String, String&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcidImpl.html" title="class in org.biojava.bio.structure">AminoAcidImpl</a></dt>
<dd>
<div class="block">set the secondary structure data for this amino acid.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucState.html#setSecStruc(org.biojava.bio.structure.secstruc.SecStrucType)">setSecStruc(SecStrucType)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucState.html" title="class in org.biojava.bio.structure.secstruc">SecStrucState</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setSeedFragmentLength(int)">setSeedFragmentLength(int)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setSeedRmsdCutoff(float)">setSeedRmsdCutoff(float)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsSegment.html#setSegId(java.lang.String)">setSegId(String)</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsSegment.html" title="class in org.biojava.bio.structure.io.sifts">SiftsSegment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathSegment.html#setSegmentId(java.lang.Integer)">setSegmentId(Integer)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathSegment.html" title="class in org.biojava.bio.structure.cath">CathSegment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#setSegments(java.util.List)">setSegments(List&lt;CathSegment&gt;)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Domain.html#setSegments(java.util.List)">setSegments(List&lt;Segment&gt;)</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Domain.html" title="class in org.biojava.bio.structure.domain.pdp">Domain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsEntity.html#setSegments(java.util.List)">setSegments(List&lt;SiftsSegment&gt;)</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsEntity.html" title="class in org.biojava.bio.structure.io.sifts">SiftsEntity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#setSeq_align_beg(java.lang.String)">setSeq_align_beg(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#setSeq_align_end(java.lang.String)">setSeq_align_end(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#setSeq_id(java.lang.String)">setSeq_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#setSeq_id(java.lang.String)">setSeq_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#setSeqBegin(int)">setSeqBegin(int)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Initial sequence number of the PDB sequence segment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#setSeqEnd(int)">setSeqEnd(int)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Ending sequence number   of the PDB sequence segment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ResidueNumber.html#setSeqNum(java.lang.Integer)">setSeqNum(Integer)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ResidueNumber.html" title="class in org.biojava.bio.structure">ResidueNumber</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#setSeqResGroups(java.util.List)">setSeqResGroups(List&lt;Group&gt;)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Set the list of SeqResGroups for this chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#setSeqResGroups(java.util.List)">setSeqResGroups(List&lt;Group&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Set the list of SeqResGroups for this chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html#setSeqResName(java.lang.String)">setSeqResName(String)</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html" title="class in org.biojava.bio.structure.io.sifts">SiftsResidue</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#setSequence(java.lang.String)">setSequence(String)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathSegment.html#setSequence(java.lang.String)">setSequence(String)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathSegment.html" title="class in org.biojava.bio.structure.cath">CathSegment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#setSequenceFamilyId(java.lang.Integer)">setSequenceFamilyId(Integer)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#setSequenceHeader(java.lang.String)">setSequenceHeader(String)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathSegment.html#setSequenceHeader(java.lang.String)">setSequenceHeader(String)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathSegment.html" title="class in org.biojava.bio.structure.cath">CathSegment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setSequentialAlignment(boolean)">setSequentialAlignment(boolean)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">Set whether this alignment has the normal topology, ie the residues
 aligned in each block increase sequentially over the original protein.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#setSeqWeight(double)">setSeqWeight(double)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>
<div class="block">Should sequence conservation be considered as part of the alignment? If yes, this weight factor allows to determine how much.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SSBond.html#setSerNum(int)">setSerNum(int)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SSBond.html" title="class in org.biojava.bio.structure">SSBond</a></dt>
<dd>
<div class="block">get serial number of this SSBOND in PDB file</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#setServer(java.lang.String)">setServer(String)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/RemotePDPProvider.html#setServer(java.lang.String)">setServer(String)</a></span> - Method in class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/RemotePDPProvider.html" title="class in org.biojava.bio.structure.domain">RemotePDPProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html#setServer(java.lang.String)">setServer(String)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">RemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setShortAlign(boolean)">setShortAlign(boolean)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#setShow3d(boolean)">setShow3d(boolean)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#setShowAFPRanges(boolean)">setShowAFPRanges(boolean)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#setShowAFPRanges(boolean)">setShowAFPRanges(boolean)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#setShowDBresult(java.lang.String)">setShowDBresult(String)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#setShowMenu(boolean)">setShowMenu(boolean)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setSimilarity(double)">setSimilarity(double)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Site.html#setSiteID(java.lang.String)">setSiteID(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Site.html" title="class in org.biojava.bio.structure">Site</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#setSites(java.util.List)">setSites(List&lt;Site&gt;)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#setSites(java.util.List)">setSites(List&lt;Site&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Domain.html#setSize(int)">setSize(int)</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Domain.html" title="class in org.biojava.bio.structure.domain.pdp">Domain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#setSOLID(java.lang.String)">setSOLID(String)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setSolvent_model_details(java.lang.String)">setSolvent_model_details(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setSolvent_model_param_bsol(java.lang.String)">setSolvent_model_param_bsol(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setSolvent_model_param_ksol(java.lang.String)">setSolvent_model_param_ksol(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#setSource(java.lang.String)">setSource(String)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBCrystallographicInfo.html#setSpaceGroup(java.lang.String)">setSpaceGroup(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBCrystallographicInfo.html" title="class in org.biojava.bio.structure">PDBCrystallographicInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#setSparse(int)">setSparse(int)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#setSpeciesId(int)">setSpeciesId(int)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#setSplit(boolean)">setSplit(boolean)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>
<div class="block">Is the organization of files within the directory split, as on the PDB FTP servers,
 or are all files contained in one directory.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/UserConfiguration.html#setSplit(boolean)">setSplit(boolean)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/UserConfiguration.html" title="class in org.biojava.bio.structure.align.util">UserConfiguration</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html#setSrc_method(java.lang.String)">setSrc_method(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model">Entity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#setSSBonds(java.util.List)">setSSBonds(List&lt;SSBond&gt;)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">set the list of SSBonds for this structure</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#setSSBonds(java.util.List)">setSSBonds(List&lt;SSBond&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">set the list of SSBonds for this structure</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setStandard(boolean)">setStandard(boolean)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathFragment.html#setStart(java.lang.String)">setStart(String)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathFragment.html" title="class in org.biojava.bio.structure.cath">CathFragment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathSegment.html#setStart(java.lang.String)">setStart(String)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathSegment.html" title="class in org.biojava.bio.structure.cath">CathSegment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsSegment.html#setStart(java.lang.String)">setStart(String)</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsSegment.html" title="class in org.biojava.bio.structure.io.sifts">SiftsSegment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#setStartPage(java.lang.String)">setStartPage(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html#setStatus(java.lang.String)">setStatus(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html" title="class in org.biojava.bio.structure.io.mmcif.model">DatabasePDBrev</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#setStepSize(int)">setStepSize(int)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathFragment.html#setStop(java.lang.String)">setStop(String)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathFragment.html" title="class in org.biojava.bio.structure.cath">CathFragment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathSegment.html#setStop(java.lang.String)">setStop(String)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathSegment.html" title="class in org.biojava.bio.structure.cath">CathSegment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileParsingParameters.html#setStoreEmptySeqRes(boolean)">setStoreEmptySeqRes(boolean)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileParsingParameters.html" title="class in org.biojava.bio.structure.io">FileParsingParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setStrain(java.lang.String)">setStrain(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#setStrictSCOP(boolean)">setStrictSCOP(boolean)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>
<div class="block">When strictSCOP is enabled, SCOP domain identifiers (eg 'd1gbga_') are
 matched literally to the SCOP database.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#setStruct(org.biojava.bio.structure.io.mmcif.model.Struct)">setStruct(Struct)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#setStruct(org.biojava.bio.structure.io.mmcif.model.Struct)">setStruct(Struct)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#setStruct(org.biojava.bio.structure.io.mmcif.model.Struct)">setStruct(Struct)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#setSubstitutionMatrix(org.biojava3.alignment.template.SubstitutionMatrix)">setSubstitutionMatrix(SubstitutionMatrix&lt;AminoAcidCompound&gt;)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>
<div class="block">Sets the  substitution matrix to be used for influencing the alignment with sequence conservation information.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDescription.html#setSunID(int)">setSunID(int)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDescription.html" title="class in org.biojava.bio.structure.scop">ScopDescription</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#setSunid(java.lang.Integer)">setSunid(Integer)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopNode.html#setSunid(int)">setSunid(int)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopNode.html" title="class in org.biojava.bio.structure.scop">ScopNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#setSuperfamilyId(int)">setSuperfamilyId(int)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Author.html#setSurname(java.lang.String)">setSurname(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Author.html" title="class in org.biojava.bio.structure">Author</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#setSwissprotId(java.lang.String)">setSwissprotId(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Set the Swissprot id of this chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#setSwissprotId(java.lang.String)">setSwissprotId(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">set the Swissprot id of this chains .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html#setSymmetryShorthand(java.lang.String)">setSymmetryShorthand(String)</a></span> - Method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html" title="class in org.biojava.bio.structure.quaternary">ModelTransformationMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setSynonyms(java.util.List)">setSynonyms(List&lt;String&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setSynthetic(java.lang.String)">setSynthetic(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFP.html#setT(double[])">setT(double[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFP.html" title="class in org.biojava.bio.structure.align.model">AFP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#setTechnique(java.lang.String)">setTechnique(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#setTempFactor(double)">setTempFactor(double)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">get set temp factor .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#setTempFactor(double)">setTempFactor(double)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBremark.html#setText(java.lang.String)">setText(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBremark.html" title="class in org.biojava.bio.structure.io.mmcif.model">DatabasePDBremark</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructKeywords.html#setText(java.lang.String)">setText(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructKeywords.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructKeywords</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#setThreads(int)">setThreads(int)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setThree_letter_code(java.lang.String)">setThree_letter_code(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucState.html#setThreeState(org.biojava.bio.structure.secstruc.SecStrucType)">setThreeState(SecStrucType)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucState.html" title="class in org.biojava.bio.structure.secstruc">SecStrucState</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#setTime(int)">setTime(int)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/io/RemoteRawBioUnitDataProvider.html#setTimeout(int)">setTimeout(int)</a></span> - Method in class org.biojava.bio.structure.quaternary.io.<a href="./org/biojava/bio/structure/quaternary/io/RemoteRawBioUnitDataProvider.html" title="class in org.biojava.bio.structure.quaternary.io">RemoteRawBioUnitDataProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setTissue(java.lang.String)">setTissue(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setTitle(java.lang.String)">setTitle(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html#setTitle(java.lang.String)">setTitle(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html" title="class in org.biojava.bio.structure.io.mmcif.model">Struct</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#setTitle(java.lang.String)">setTitle(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>
<div class="block">Set the value of title</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#setTitle(java.lang.String)">setTitle(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setTMScore(double)">setTMScore(double)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Segment.html#setTo(java.lang.Integer)">setTo(Integer)</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Segment.html" title="class in org.biojava.bio.structure.domain.pdp">Segment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/Ladder.html#setTo(int)">setTo(int)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/Ladder.html" title="class in org.biojava.bio.structure.secstruc">Ladder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#setTopologyId(java.lang.Integer)">setTopologyId(Integer)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#setTorsionPenalty(double)">setTorsionPenalty(double)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setTotalLenIni(int)">setTotalLenIni(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setTotalLenOpt(int)">setTotalLenOpt(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setTotalRmsdIni(double)">setTotalRmsdIni(double)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">this is the init-RMSD, not the final RMSD after refinement.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setTotalRmsdOpt(double)">setTotalRmsdOpt(double)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">The RMSD of the final alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/BigSqrt.html#setTraceFlag(boolean)">setTraceFlag(boolean)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/BigSqrt.html" title="class in org.biojava.bio.structure.secstruc">BigSqrt</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#setTrans(org.biojava.bio.structure.Atom)">setTrans(Atom)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBBioAssemblyParser.html#setTransformationMap(java.util.Map)">setTransformationMap(Map&lt;Integer, List&lt;ModelTransformationMatrix&gt;&gt;)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBBioAssemblyParser.html" title="class in org.biojava.bio.structure.io">PDBBioAssemblyParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html#setTransformationMatrix(org.biojava.bio.structure.jama.Matrix, double[])">setTransformationMatrix(Matrix, double[])</a></span> - Method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html" title="class in org.biojava.bio.structure.quaternary">ModelTransformationMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html#setTransformationMatrix(float, float, float, float, float, float, float, float, float, float, float, float)">setTransformationMatrix(float, float, float, float, float, float, float, float, float, float, float, float)</a></span> - Method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html" title="class in org.biojava.bio.structure.quaternary">ModelTransformationMatrix</a></dt>
<dd>
<div class="block">The provided rotation matrix is:
 m00 m01 m02
 m10 m11 m12
 m20 m21 m22
 
 And the provided translation vector is <v0 v1 v2></div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/OptimalCECPParameters.html#setTryAllCPs(java.lang.Boolean)">setTryAllCPs(Boolean)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/OptimalCECPParameters.html" title="class in org.biojava.bio.structure.align.ce">OptimalCECPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucState.html#setTurn(boolean[])">setTurn(boolean[])</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucState.html" title="class in org.biojava.bio.structure.secstruc">SecStrucState</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setTwi(int[])">setTwi(int[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatAligner.html#setTwistedGroups(org.biojava.bio.structure.Group[])">setTwistedGroups(Group[])</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatAligner.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setType(java.lang.String)">setType(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html#setType(java.lang.String)">setType(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemCompDescriptor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html#setType(java.lang.String)">setType(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model">Entity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html#setType(java.lang.String)">setType(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructOperList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsEntity.html#setType(java.lang.String)">setType(String)</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsEntity.html" title="class in org.biojava.bio.structure.io.sifts">SiftsEntity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setType_symbol(java.lang.String)">setType_symbol(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/OperatorResolver.html#setUnaryOperators(java.util.List)">setUnaryOperators(List&lt;String&gt;)</a></span> - Method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/OperatorResolver.html" title="class in org.biojava.bio.structure.quaternary">OperatorResolver</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html#setUniProtAccessionId(java.lang.String)">setUniProtAccessionId(String)</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html" title="class in org.biojava.bio.structure.io.sifts">SiftsResidue</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html#setUniProtPos(java.lang.Integer)">setUniProtPos(Integer)</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html" title="class in org.biojava.bio.structure.io.sifts">SiftsResidue</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html#setUniProtResName(java.lang.String)">setUniProtResName(String)</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html" title="class in org.biojava.bio.structure.io.sifts">SiftsResidue</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#setUnitv(org.biojava.bio.structure.Atom)">setUnitv(Atom)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileParsingParameters.html#setUpdateRemediatedFiles(boolean)">setUpdateRemediatedFiles(boolean)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileParsingParameters.html" title="class in org.biojava.bio.structure.io">FileParsingParameters</a></dt>
<dd>
<div class="block">A flag if local files should be replaced with the latest version of remediated PDB files.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#setUsed(int)">setUsed(int)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setUsedAtomNames(java.lang.String[])">setUsedAtomNames(String[])</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#setUsername(java.lang.String)">setUsername(String)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/AligMatEl.html#setValue(int)">setValue(int)</a></span> - Method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/AligMatEl.html" title="class in org.biojava.bio.structure.align.helper">AligMatEl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/GapArray.html#setValue(int)">setValue(int)</a></span> - Method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/GapArray.html" title="class in org.biojava.bio.structure.align.helper">GapArray</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setVariant(java.lang.String)">setVariant(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html#setVector(double[])">setVector(double[])</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructOperList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html#setVector(double[])">setVector(double[])</a></span> - Method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html" title="class in org.biojava.bio.structure.quaternary">ModelTransformationMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#setVerbose(boolean)">setVerbose(boolean)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setVersion(java.lang.String)">setVersion(String)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#setVersion(java.lang.String)">setVersion(String)</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#setVolume(java.lang.String)">setVolume(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html#setWaitForAlignments(boolean)">setWaitForAlignments(boolean)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html" title="class in org.biojava.bio.structure.align.client">FarmJobRunnable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#setWinSize(java.lang.Integer)">setWinSize(Integer)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#setX(double)">setX(double)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">Set the X coordinate.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#setX(double)">setX(double)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#setY(double)">setY(double)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">Set the Y coordinate.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#setY(double)">setY(double)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#setZ(double)">setZ(double)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">Set the Z coordinate.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#setZ(double)">setZ(double)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBCrystallographicInfo.html#setZ(int)">setZ(int)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBCrystallographicInfo.html" title="class in org.biojava.bio.structure">PDBCrystallographicInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#shift(org.biojava.bio.structure.Structure, org.biojava.bio.structure.Atom)">shift(Structure, Atom)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">shift a structure with a vector.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#shift(org.biojava.bio.structure.Atom, org.biojava.bio.structure.Atom)">shift(Atom, Atom)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">Shift a vector.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#shift(org.biojava.bio.structure.Group, org.biojava.bio.structure.Atom)">shift(Group, Atom)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">Shift a Group with a vector.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/domain/pdp/ShortSegmentRemover.html" title="class in org.biojava.bio.structure.domain.pdp"><span class="strong">ShortSegmentRemover</span></a> - Class in <a href="./org/biojava/bio/structure/domain/pdp/package-summary.html">org.biojava.bio.structure.domain.pdp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/ShortSegmentRemover.html#ShortSegmentRemover()">ShortSegmentRemover()</a></span> - Constructor for class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/ShortSegmentRemover.html" title="class in org.biojava.bio.structure.domain.pdp">ShortSegmentRemover</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#showAFPRanges">showAFPRanges</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/GuiWrapper.html#showAlignmentGUI()">showAlignmentGUI()</a></span> - Static method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/GuiWrapper.html" title="class in org.biojava.bio.structure.align.ce">GuiWrapper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/GuiWrapper.html#showAlignmentImage(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], java.lang.Object)">showAlignmentImage(AFPChain, Atom[], Atom[], Object)</a></span> - Static method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/GuiWrapper.html" title="class in org.biojava.bio.structure.align.ce">GuiWrapper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#showCompound()">showCompound()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/GuiWrapper.html#showDBResults(org.biojava.bio.structure.align.ce.StartupParameters)">showDBResults(StartupParameters)</a></span> - Static method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/GuiWrapper.html" title="class in org.biojava.bio.structure.align.ce">GuiWrapper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#showHeader()">showHeader()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>
<div class="block">Print some debug statements to System.out</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#showSource()">showSource()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/GuiWrapper.html#showStructure(org.biojava.bio.structure.Structure)">showStructure(Structure)</a></span> - Static method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/GuiWrapper.html" title="class in org.biojava.bio.structure.align.ce">GuiWrapper</a></dt>
<dd>
<div class="block">Shows a structure in Jmol</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#SIDE_CHAIN_ANGLE_SCORING">SIDE_CHAIN_ANGLE_SCORING</a></span> - Static variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#SIDE_CHAIN_SCORING">SIDE_CHAIN_SCORING</a></span> - Static variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#sideChainScoringType">sideChainScoringType</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/sifts/SiftsEntity.html" title="class in org.biojava.bio.structure.io.sifts"><span class="strong">SiftsEntity</span></a> - Class in <a href="./org/biojava/bio/structure/io/sifts/package-summary.html">org.biojava.bio.structure.io.sifts</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsEntity.html#SiftsEntity()">SiftsEntity()</a></span> - Constructor for class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsEntity.html" title="class in org.biojava.bio.structure.io.sifts">SiftsEntity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsEntity.html#SiftsEntity(java.lang.String, java.lang.String)">SiftsEntity(String, String)</a></span> - Constructor for class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsEntity.html" title="class in org.biojava.bio.structure.io.sifts">SiftsEntity</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/sifts/SiftsMappingProvider.html" title="class in org.biojava.bio.structure.io.sifts"><span class="strong">SiftsMappingProvider</span></a> - Class in <a href="./org/biojava/bio/structure/io/sifts/package-summary.html">org.biojava.bio.structure.io.sifts</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsMappingProvider.html#SiftsMappingProvider()">SiftsMappingProvider()</a></span> - Constructor for class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsMappingProvider.html" title="class in org.biojava.bio.structure.io.sifts">SiftsMappingProvider</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html" title="class in org.biojava.bio.structure.io.sifts"><span class="strong">SiftsResidue</span></a> - Class in <a href="./org/biojava/bio/structure/io/sifts/package-summary.html">org.biojava.bio.structure.io.sifts</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html#SiftsResidue()">SiftsResidue()</a></span> - Constructor for class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html" title="class in org.biojava.bio.structure.io.sifts">SiftsResidue</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/sifts/SiftsSegment.html" title="class in org.biojava.bio.structure.io.sifts"><span class="strong">SiftsSegment</span></a> - Class in <a href="./org/biojava/bio/structure/io/sifts/package-summary.html">org.biojava.bio.structure.io.sifts</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsSegment.html#SiftsSegment()">SiftsSegment()</a></span> - Constructor for class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsSegment.html" title="class in org.biojava.bio.structure.io.sifts">SiftsSegment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsSegment.html#SiftsSegment(java.lang.String, java.lang.String, java.lang.String)">SiftsSegment(String, String, String)</a></span> - Constructor for class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsSegment.html" title="class in org.biojava.bio.structure.io.sifts">SiftsSegment</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/sifts/SiftsXMLParser.html" title="class in org.biojava.bio.structure.io.sifts"><span class="strong">SiftsXMLParser</span></a> - Class in <a href="./org/biojava/bio/structure/io/sifts/package-summary.html">org.biojava.bio.structure.io.sifts</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsXMLParser.html#SiftsXMLParser()">SiftsXMLParser()</a></span> - Constructor for class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsXMLParser.html" title="class in org.biojava.bio.structure.io.sifts">SiftsXMLParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/fatcat/calc/SigEva.html" title="class in org.biojava.bio.structure.align.fatcat.calc"><span class="strong">SigEva</span></a> - Class in <a href="./org/biojava/bio/structure/align/fatcat/calc/package-summary.html">org.biojava.bio.structure.align.fatcat.calc</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/SigEva.html#SigEva()">SigEva()</a></span> - Constructor for class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/SigEva.html" title="class in org.biojava.bio.structure.align.fatcat.calc">SigEva</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif"><span class="strong">SimpleMMcifConsumer</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/package-summary.html">org.biojava.bio.structure.io.mmcif</a></dt>
<dd>
<div class="block">A MMcifConsumer implementation that build a in-memory representation of the
 content of a mmcif file as a BioJava Structure object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#SimpleMMcifConsumer()">SimpleMMcifConsumer()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif"><span class="strong">SimpleMMcifParser</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/package-summary.html">org.biojava.bio.structure.io.mmcif</a></dt>
<dd>
<div class="block">A simple mmCif file parser</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#SimpleMMcifParser()">SimpleMMcifParser()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/server/SimpleStructureServer.html" title="class in org.biojava.bio.structure.server"><span class="strong">SimpleStructureServer</span></a> - Class in <a href="./org/biojava/bio/structure/server/package-summary.html">org.biojava.bio.structure.server</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>&nbsp;</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/SimpleStructureServer.html#SimpleStructureServer()">SimpleStructureServer()</a></span> - Constructor for class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/SimpleStructureServer.html" title="class in org.biojava.bio.structure.server">SimpleStructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/jama/SingularValueDecomposition.html" title="class in org.biojava.bio.structure.jama"><span class="strong">SingularValueDecomposition</span></a> - Class in <a href="./org/biojava/bio/structure/jama/package-summary.html">org.biojava.bio.structure.jama</a></dt>
<dd>
<div class="block">Singular Value Decomposition.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/SingularValueDecomposition.html#SingularValueDecomposition(org.biojava.bio.structure.jama.Matrix)">SingularValueDecomposition(Matrix)</a></span> - Constructor for class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/SingularValueDecomposition.html" title="class in org.biojava.bio.structure.jama">SingularValueDecomposition</a></dt>
<dd>
<div class="block">Construct the singular value decomposition.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/Site.html" title="class in org.biojava.bio.structure"><span class="strong">Site</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">Holds the data of sites presented in PDB files.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Site.html#Site()">Site()</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Site.html" title="class in org.biojava.bio.structure">Site</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Site.html#Site(java.lang.String, java.util.List)">Site(String, List&lt;Group&gt;)</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Site.html" title="class in org.biojava.bio.structure">Site</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/CutValues.html#site2">site2</a></span> - Variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/CutValues.html" title="class in org.biojava.bio.structure.domain.pdp">CutValues</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#size()">size()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">getnumber of atoms.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#size()">size()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">getnumber of atoms.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#size()">size()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">return number of Chains in this Structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#size(int)">size(int)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">return number of chains of model.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#size()">size()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">return number of chains , if NMR return number of chains of first model .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#size(int)">size(int)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">return number of chains  of model.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#skalarProduct(org.biojava.bio.structure.Atom, org.biojava.bio.structure.Atom)">skalarProduct(Atom, Atom)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">skalar product.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html#slash">slash</a></span> - Static variable in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html" title="class in org.biojava.bio.structure.quaternary">ModelTransformationMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/seq/SmithWaterman3Daligner.html" title="class in org.biojava.bio.structure.align.seq"><span class="strong">SmithWaterman3Daligner</span></a> - Class in <a href="./org/biojava/bio/structure/align/seq/package-summary.html">org.biojava.bio.structure.align.seq</a></dt>
<dd>
<div class="block">provides a 3D superimposition based on the sequence alignment</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWaterman3Daligner.html#SmithWaterman3Daligner()">SmithWaterman3Daligner()</a></span> - Constructor for class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWaterman3Daligner.html" title="class in org.biojava.bio.structure.align.seq">SmithWaterman3Daligner</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/seq/SmithWaterman3DParameters.html" title="class in org.biojava.bio.structure.align.seq"><span class="strong">SmithWaterman3DParameters</span></a> - Class in <a href="./org/biojava/bio/structure/align/seq/package-summary.html">org.biojava.bio.structure.align.seq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWaterman3DParameters.html#SmithWaterman3DParameters()">SmithWaterman3DParameters()</a></span> - Constructor for class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWaterman3DParameters.html" title="class in org.biojava.bio.structure.align.seq">SmithWaterman3DParameters</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/seq/SmithWatermanUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.seq"><span class="strong">SmithWatermanUserArgumentProcessor</span></a> - Class in <a href="./org/biojava/bio/structure/align/seq/package-summary.html">org.biojava.bio.structure.align.seq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWatermanUserArgumentProcessor.html#SmithWatermanUserArgumentProcessor()">SmithWatermanUserArgumentProcessor()</a></span> - Constructor for class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWatermanUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.seq">SmithWatermanUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/CholeskyDecomposition.html#solve(org.biojava.bio.structure.jama.Matrix)">solve(Matrix)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/CholeskyDecomposition.html" title="class in org.biojava.bio.structure.jama">CholeskyDecomposition</a></dt>
<dd>
<div class="block">Solve A*X = B</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/LUDecomposition.html#solve(org.biojava.bio.structure.jama.Matrix)">solve(Matrix)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/LUDecomposition.html" title="class in org.biojava.bio.structure.jama">LUDecomposition</a></dt>
<dd>
<div class="block">Solve A*X = B</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#solve(org.biojava.bio.structure.jama.Matrix)">solve(Matrix)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Solve A*X = B</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/QRDecomposition.html#solve(org.biojava.bio.structure.jama.Matrix)">solve(Matrix)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/QRDecomposition.html" title="class in org.biojava.bio.structure.jama">QRDecomposition</a></dt>
<dd>
<div class="block">Least squares solution of A*X = B</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#solveTranspose(org.biojava.bio.structure.jama.Matrix)">solveTranspose(Matrix)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Solve X*A = B, which is also A'*X' = B'</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/AFPCalculator.html#sortAfps(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">sortAfps(AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/AFPCalculator.html" title="class in org.biojava.bio.structure.align.fatcat.calc">AFPCalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html#spaces">spaces</a></span> - Static variable in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html" title="class in org.biojava.bio.structure.quaternary">ModelTransformationMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/BigSqrt.html#sqrt(java.math.BigInteger)">sqrt(BigInteger)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/BigSqrt.html" title="class in org.biojava.bio.structure.secstruc">BigSqrt</a></dt>
<dd>
<div class="block">Get square root</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/BigSqrt.html#sqrt(java.math.BigDecimal)">sqrt(BigDecimal)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/BigSqrt.html" title="class in org.biojava.bio.structure.secstruc">BigSqrt</a></dt>
<dd>
<div class="block">Get square root</div>
</dd>
<dt><a href="./org/biojava/bio/structure/SSBond.html" title="class in org.biojava.bio.structure"><span class="strong">SSBond</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">A simple bean to store disulfid bridge information, the SSBOND records in the PDB files.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SSBond.html#SSBond()">SSBond()</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SSBond.html" title="class in org.biojava.bio.structure">SSBond</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/StandardAminoAcid.html" title="class in org.biojava.bio.structure"><span class="strong">StandardAminoAcid</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">A class that provides a set of standard amino acids.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentProgressListener.html#startingAlignment(java.lang.String, org.biojava.bio.structure.Atom[], java.lang.String, org.biojava.bio.structure.Atom[])">startingAlignment(String, Atom[], String, Atom[])</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentProgressListener.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce"><span class="strong">StartupParameters</span></a> - Class in <a href="./org/biojava/bio/structure/align/ce/package-summary.html">org.biojava.bio.structure.align.ce</a></dt>
<dd>
<div class="block">a simple bean that contains the parameters that can get set at startup</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#StartupParameters()">StartupParameters()</a></span> - Constructor for class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileParser.html#STRAND">STRAND</a></span> - Static variable in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileParser.html" title="class in org.biojava.bio.structure.io">PDBFileParser</a></dt>
<dd>
<div class="block">Strand secondary structure assignment.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html" title="class in org.biojava.bio.structure.align.pairwise"><span class="strong">StrCompAlignment</span></a> - Class in <a href="./org/biojava/bio/structure/align/pairwise/package-summary.html">org.biojava.bio.structure.align.pairwise</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html#StrCompAlignment(int, int)">StrCompAlignment(int, int)</a></span> - Constructor for class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">StrCompAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#STRING_LIMIT">STRING_LIMIT</a></span> - Static variable in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align"><span class="strong">StrucAligParameters</span></a> - Class in <a href="./org/biojava/bio/structure/align/package-summary.html">org.biojava.bio.structure.align</a></dt>
<dd>
<div class="block">A class that contains all the parameters of the structure alignment algorithm.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#StrucAligParameters()">StrucAligParameters()</a></span> - Constructor for class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html" title="class in org.biojava.bio.structure.io.mmcif.model"><span class="strong">Struct</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a></dt>
<dd>
<div class="block">a bean to contain the data of the _struct lines</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html#Struct()">Struct()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html" title="class in org.biojava.bio.structure.io.mmcif.model">Struct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html" title="class in org.biojava.bio.structure.io.mmcif.model"><span class="strong">StructAsym</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a></dt>
<dd>
<div class="block">Contains the data for _struct_asym</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html#StructAsym()">StructAsym()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructAsym</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/StructKeywords.html" title="class in org.biojava.bio.structure.io.mmcif.model"><span class="strong">StructKeywords</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructKeywords.html#StructKeywords()">StructKeywords()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructKeywords.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructKeywords</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html" title="class in org.biojava.bio.structure.io.mmcif.model"><span class="strong">StructRef</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a></dt>
<dd>
<div class="block">A class to containt the _struct_ref field data</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html#StructRef()">StructRef()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRef</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model"><span class="strong">StructRefSeq</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#StructRefSeq()">StructRefSeq()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure"><span class="strong">Structure</span></a> - Interface in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">Interface for a structure object.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/StructureAlignment.html" title="interface in org.biojava.bio.structure.align"><span class="strong">StructureAlignment</span></a> - Interface in <a href="./org/biojava/bio/structure/align/package-summary.html">org.biojava.bio.structure.align</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/StructureAlignmentFactory.html" title="class in org.biojava.bio.structure.align"><span class="strong">StructureAlignmentFactory</span></a> - Class in <a href="./org/biojava/bio/structure/align/package-summary.html">org.biojava.bio.structure.align</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructureAlignmentFactory.html#StructureAlignmentFactory()">StructureAlignmentFactory()</a></span> - Constructor for class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructureAlignmentFactory.html" title="class in org.biojava.bio.structure.align">StructureAlignmentFactory</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/fatcat/calc/StructureAlignmentOptimizer.html" title="class in org.biojava.bio.structure.align.fatcat.calc"><span class="strong">StructureAlignmentOptimizer</span></a> - Class in <a href="./org/biojava/bio/structure/align/fatcat/calc/package-summary.html">org.biojava.bio.structure.align.fatcat.calc</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/StructureAlignmentOptimizer.html#StructureAlignmentOptimizer(int, int, org.biojava.bio.structure.Atom[], int, int, org.biojava.bio.structure.Atom[], int, int[][])">StructureAlignmentOptimizer(int, int, Atom[], int, int, Atom[], int, int[][])</a></span> - Constructor for class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/StructureAlignmentOptimizer.html" title="class in org.biojava.bio.structure.align.fatcat.calc">StructureAlignmentOptimizer</a></dt>
<dd>
<div class="block">optimize the structural alignment by update the equivalent residues
      and then run dynamic programming
      input: len1 the length of structure 1; c1: the structure information of 1
             len2 the length of structure 2; c2: the structure information of 2
             iniLen and iniSet is the length and list of initial equivalent residues</div>
</dd>
<dt><a href="./org/biojava/bio/structure/server/StructureEvent.html" title="interface in org.biojava.bio.structure.server"><span class="strong">StructureEvent</span></a> - Interface in <a href="./org/biojava/bio/structure/server/package-summary.html">org.biojava.bio.structure.server</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/server/StructureEventImpl.html" title="class in org.biojava.bio.structure.server"><span class="strong">StructureEventImpl</span></a> - Class in <a href="./org/biojava/bio/structure/server/package-summary.html">org.biojava.bio.structure.server</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureEventImpl.html#StructureEventImpl(org.biojava.bio.structure.Structure)">StructureEventImpl(Structure)</a></span> - Constructor for class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureEventImpl.html" title="class in org.biojava.bio.structure.server">StructureEventImpl</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/StructureException.html" title="class in org.biojava.bio.structure"><span class="strong">StructureException</span></a> - Exception in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">An exception during the parsing of a PDB file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureException.html#StructureException(java.lang.String)">StructureException(String)</a></span> - Constructor for exception org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureException.html" title="class in org.biojava.bio.structure">StructureException</a></dt>
<dd>
<div class="block">Constructs a StructureException object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureException.html#StructureException(java.lang.String, java.lang.Throwable)">StructureException(String, Throwable)</a></span> - Constructor for exception org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureException.html" title="class in org.biojava.bio.structure">StructureException</a></dt>
<dd>
<div class="block">Constructs a StructureException object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureException.html#StructureException(java.lang.Throwable)">StructureException(Throwable)</a></span> - Constructor for exception org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureException.html" title="class in org.biojava.bio.structure">StructureException</a></dt>
<dd>
<div class="block">Constructs a StructureException object.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/server/StructureFetcherRunnable.html" title="class in org.biojava.bio.structure.server"><span class="strong">StructureFetcherRunnable</span></a> - Class in <a href="./org/biojava/bio/structure/server/package-summary.html">org.biojava.bio.structure.server</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>&nbsp;</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureFetcherRunnable.html#StructureFetcherRunnable(org.biojava.bio.structure.server.PDBInstallation)">StructureFetcherRunnable(PDBInstallation)</a></span> - Constructor for class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureFetcherRunnable.html" title="class in org.biojava.bio.structure.server">StructureFetcherRunnable</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure"><span class="strong">StructureImpl</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">Implementation of a PDB Structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#StructureImpl()">StructureImpl()</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">Constructs a StructureImpl object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#StructureImpl(org.biojava.bio.structure.Group)">StructureImpl(Group)</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">construct a Structure object that only contains a single group</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#StructureImpl(org.biojava.bio.structure.Chain)">StructureImpl(Chain)</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">construct a Structure object that contains a particular chain</div>
</dd>
<dt><a href="./org/biojava/bio/structure/io/StructureIO.html" title="interface in org.biojava.bio.structure.io"><span class="strong">StructureIO</span></a> - Interface in <a href="./org/biojava/bio/structure/io/package-summary.html">org.biojava.bio.structure.io</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>Do no use any longer, StructureIO is now a static utility calss on the org.biojava3.structure level..</i></div>
</div>
</dd>
<dt><a href="./org/biojava3/structure/StructureIO.html" title="class in org.biojava3.structure"><span class="strong">StructureIO</span></a> - Class in <a href="./org/biojava3/structure/package-summary.html">org.biojava3.structure</a></dt>
<dd>
<div class="block">A class that provides static access methods for easy lookup of protein structure related components</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/structure/StructureIO.html#StructureIO()">StructureIO()</a></span> - Constructor for class org.biojava3.structure.<a href="./org/biojava3/structure/StructureIO.html" title="class in org.biojava3.structure">StructureIO</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/StructureIOFile.html" title="interface in org.biojava.bio.structure.io"><span class="strong">StructureIOFile</span></a> - Interface in <a href="./org/biojava/bio/structure/io/package-summary.html">org.biojava.bio.structure.io</a></dt>
<dd>
<div class="block">interface StructureIOFile extends the StructureIO interface
 and adds a few File specific methods.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/server/StructureListener.html" title="interface in org.biojava.bio.structure.server"><span class="strong">StructureListener</span></a> - Interface in <a href="./org/biojava/bio/structure/server/package-summary.html">org.biojava.bio.structure.server</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/client/StructureName.html" title="class in org.biojava.bio.structure.align.client"><span class="strong">StructureName</span></a> - Class in <a href="./org/biojava/bio/structure/align/client/package-summary.html">org.biojava.bio.structure.align.client</a></dt>
<dd>
<div class="block">A utility class that makes working with names of structures, domains and ranges easier.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/StructureName.html#StructureName(java.lang.String)">StructureName(String)</a></span> - Constructor for class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/StructureName.html" title="class in org.biojava.bio.structure.align.client">StructureName</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/StructurePairAligner.html" title="class in org.biojava.bio.structure.align"><span class="strong">StructurePairAligner</span></a> - Class in <a href="./org/biojava/bio/structure/align/package-summary.html">org.biojava.bio.structure.align</a></dt>
<dd>
<div class="block">Perform a pairwise protein structure superimposition.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructurePairAligner.html#StructurePairAligner()">StructurePairAligner()</a></span> - Constructor for class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructurePairAligner.html" title="class in org.biojava.bio.structure.align">StructurePairAligner</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/StructureProvider.html" title="interface in org.biojava.bio.structure.io"><span class="strong">StructureProvider</span></a> - Interface in <a href="./org/biojava/bio/structure/io/package-summary.html">org.biojava.bio.structure.io</a></dt>
<dd>
<div class="block">A class that can provide a protein structure object from somewhere.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/io/StructureSequenceMatcher.html" title="class in org.biojava.bio.structure.io"><span class="strong">StructureSequenceMatcher</span></a> - Class in <a href="./org/biojava/bio/structure/io/package-summary.html">org.biojava.bio.structure.io</a></dt>
<dd>
<div class="block">A utility class with methods for matching ProteinSequences with
 Structures.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/StructureSequenceMatcher.html#StructureSequenceMatcher()">StructureSequenceMatcher()</a></span> - Constructor for class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/StructureSequenceMatcher.html" title="class in org.biojava.bio.structure.io">StructureSequenceMatcher</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/server/StructureServer.html" title="interface in org.biojava.bio.structure.server"><span class="strong">StructureServer</span></a> - Interface in <a href="./org/biojava/bio/structure/server/package-summary.html">org.biojava.bio.structure.server</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>&nbsp;</div>
</dd>
<dt><a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure"><span class="strong">StructureTools</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">A class that provides some tool methods.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#StructureTools()">StructureTools()</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#substitutionMatrix">substitutionMatrix</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#substract(org.biojava.bio.structure.Atom, org.biojava.bio.structure.Atom)">substract(Atom, Atom)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>use <code>subtract</code> instead.</i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#subtract(org.biojava.bio.structure.Atom, org.biojava.bio.structure.Atom)">subtract(Atom, Atom)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">subtract two atoms ( a - b).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/LocalProteinDomainParser.html#suggestDomains(org.biojava.bio.structure.Structure)">suggestDomains(Structure)</a></span> - Static method in class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/LocalProteinDomainParser.html" title="class in org.biojava.bio.structure.domain">LocalProteinDomainParser</a></dt>
<dd>
<div class="block">Suggest domains for a protein structure</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/LocalProteinDomainParser.html#suggestDomains(org.biojava.bio.structure.Atom[])">suggestDomains(Atom[])</a></span> - Static method in class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/LocalProteinDomainParser.html" title="class in org.biojava.bio.structure.domain">LocalProteinDomainParser</a></dt>
<dd>
<div class="block">Suggest domains for a set of Calpha atoms</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#svd()">svd()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Singular Value Decomposition</div>
</dd>
<dt><a href="./org/biojava/bio/structure/SVDSuperimposer.html" title="class in org.biojava.bio.structure"><span class="strong">SVDSuperimposer</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">A class that calculates the superimposition between two sets of atoms
 inspired by the biopython SVDSuperimposer class...</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SVDSuperimposer.html#SVDSuperimposer(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">SVDSuperimposer(Atom[], Atom[])</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SVDSuperimposer.html" title="class in org.biojava.bio.structure">SVDSuperimposer</a></dt>
<dd>
<div class="block">Create a SVDSuperimposer object and calculate a SVD superimposition of two sets of atoms.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html#symmetryShorthand">symmetryShorthand</a></span> - Variable in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html" title="class in org.biojava.bio.structure.quaternary">ModelTransformationMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/util/SynchronizedOutFile.html" title="class in org.biojava.bio.structure.align.util"><span class="strong">SynchronizedOutFile</span></a> - Class in <a href="./org/biojava/bio/structure/align/util/package-summary.html">org.biojava.bio.structure.align.util</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/SynchronizedOutFile.html#SynchronizedOutFile(java.io.File, boolean)">SynchronizedOutFile(File, boolean)</a></span> - Constructor for class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/SynchronizedOutFile.html" title="class in org.biojava.bio.structure.align.util">SynchronizedOutFile</a></dt>
<dd>
<div class="block">Create a thread safe wrapper for writing to this file, the file will be gzip compressed.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/SynchronizedOutFile.html#SynchronizedOutFile(java.io.File)">SynchronizedOutFile(File)</a></span> - Constructor for class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/SynchronizedOutFile.html" title="class in org.biojava.bio.structure.align.util">SynchronizedOutFile</a></dt>
<dd>
<div class="block">create a thread safe wrapper for working with this file</div>
</dd>
</dl>
<a name="_T_">
<!--   -->
</a>
<h2 class="title">T</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html#TD">TD</a></span> - Static variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html" title="class in org.biojava.bio.structure.domain.pdp">PDPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html#TD1">TD1</a></span> - Static variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html" title="class in org.biojava.bio.structure.domain.pdp">PDPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html#terminate()">terminate()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html" title="class in org.biojava.bio.structure.align.client">FarmJobRunnable</a></dt>
<dd>
<div class="block">Send signal to terminate calculations</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/FarmJob.html#terminate()">terminate()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/FarmJob.html" title="class in org.biojava.bio.structure.align">FarmJob</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#times(double)">times(double)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Multiply a matrix by a scalar, C = s*A</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#times(org.biojava.bio.structure.jama.Matrix)">times(Matrix)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Linear algebraic matrix multiplication, A * B</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#timesEquals(double)">timesEquals(double)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Multiply a matrix by a scalar in place, A = s*A</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/UserConfiguration.html#TMP_DIR">TMP_DIR</a></span> - Static variable in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/UserConfiguration.html" title="class in org.biojava.bio.structure.align.util">UserConfiguration</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html#toAlignedPairs(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">toAlignedPairs(AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html" title="class in org.biojava.bio.structure.align.model">AfpChainWriter</a></dt>
<dd>
<div class="block">Prints the alignment in the simplest form: a list of aligned residues.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#toCE(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">toCE(Atom[], Atom[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html#toCE(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">toCE(AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html" title="class in org.biojava.bio.structure.align.model">AfpChainWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileConvert.html#toDASStructure(org.biojava3.core.util.XMLWriter)">toDASStructure(XMLWriter)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileConvert.html" title="class in org.biojava.bio.structure.io">FileConvert</a></dt>
<dd>
<div class="block">convert a protein Structure to a DAS Structure XML response .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#toDBSearchResult()">toDBSearchResult()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html#toDBSearchResult(org.biojava.bio.structure.align.model.AFPChain)">toDBSearchResult(AFPChain)</a></span> - Static method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html" title="class in org.biojava.bio.structure.align.model">AfpChainWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/CollectionTools.html#toDoubleArray(java.util.Collection)">toDoubleArray(Collection&lt;Number&gt;)</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/CollectionTools.html" title="class in org.biojava.bio.structure.align.util">CollectionTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#toFatcat(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">toFatcat(Atom[], Atom[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html#toFatCat(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">toFatCat(AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html" title="class in org.biojava.bio.structure.align.model">AfpChainWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html#toFatCatCore(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], boolean, boolean, boolean, boolean)">toFatCatCore(AFPChain, Atom[], Atom[], boolean, boolean, boolean, boolean)</a></span> - Static method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html" title="class in org.biojava.bio.structure.align.model">AfpChainWriter</a></dt>
<dd>
<div class="block">Output in FatCatCore format</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/CollectionTools.html#toIntArray(java.util.Collection)">toIntArray(Collection&lt;Number&gt;)</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/CollectionTools.html" title="class in org.biojava.bio.structure.align.util">CollectionTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#toPDB(org.biojava.bio.structure.Structure, org.biojava.bio.structure.Structure)">toPDB(Structure, Structure)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>
<div class="block">converts the alignment to a PDB file
 each of the structures will be represented as a model.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#toPDB()">toPDB()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#toPDB(java.lang.StringBuffer)">toPDB(StringBuffer)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#toPDB()">toPDB()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Convert the DBRef object to a DBREF record as it is used in PDB files</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#toPDB(java.lang.StringBuffer)">toPDB(StringBuffer)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Append the PDB representation of this DBRef to the provided StringBuffer</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileConvert.html#toPDB()">toPDB()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileConvert.html" title="class in org.biojava.bio.structure.io">FileConvert</a></dt>
<dd>
<div class="block">Convert a structure into a PDB file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileConvert.html#toPDB(org.biojava.bio.structure.Atom)">toPDB(Atom)</a></span> - Static method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileConvert.html" title="class in org.biojava.bio.structure.io">FileConvert</a></dt>
<dd>
<div class="block">Prints the content of an Atom object as a PDB formatted line.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileConvert.html#toPDB(org.biojava.bio.structure.Atom, java.lang.StringBuffer)">toPDB(Atom, StringBuffer)</a></span> - Static method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileConvert.html" title="class in org.biojava.bio.structure.io">FileConvert</a></dt>
<dd>
<div class="block">print ATOM record in the following syntax</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#toPDB()">toPDB()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>
<div class="block">Return a PDB representation of the PDB Header</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#toPDB(java.lang.StringBuffer)">toPDB(StringBuffer)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>
<div class="block">Appends a PDB representation of the PDB header to the provided StringBuffer</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBRecord.html#toPDB()">toPDB()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBRecord.html" title="interface in org.biojava.bio.structure">PDBRecord</a></dt>
<dd>
<div class="block">Returns a PDB file like representation of this record.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBRecord.html#toPDB(java.lang.StringBuffer)">toPDB(StringBuffer)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBRecord.html" title="interface in org.biojava.bio.structure">PDBRecord</a></dt>
<dd>
<div class="block">Appends a PDB file like representation of this record to the provided StringBuffer.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ResidueNumber.html#toPDB()">toPDB()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ResidueNumber.html" title="class in org.biojava.bio.structure">ResidueNumber</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Site.html#toPDB()">toPDB()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Site.html" title="class in org.biojava.bio.structure">Site</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Site.html#toPDB(java.lang.StringBuffer)">toPDB(StringBuffer)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Site.html" title="class in org.biojava.bio.structure">Site</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SSBond.html#toPDB()">toPDB()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SSBond.html" title="class in org.biojava.bio.structure">SSBond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SSBond.html#toPDB(java.lang.StringBuffer)">toPDB(StringBuffer)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SSBond.html" title="class in org.biojava.bio.structure">SSBond</a></dt>
<dd>
<div class="block">append the PDB representation of this SSBOND to the provided StringBUffer</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#toPDB()">toPDB()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">create a String that contains the contents of a PDB file .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#toPDB()">toPDB()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">create a String that contains the contents of a PDB file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#toRotMat()">toRotMat()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html#toRotMat(org.biojava.bio.structure.align.model.AFPChain)">toRotMat(AFPChain)</a></span> - Static method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html" title="class in org.biojava.bio.structure.align.model">AfpChainWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#torsionAngle(org.biojava.bio.structure.Atom, org.biojava.bio.structure.Atom, org.biojava.bio.structure.Atom, org.biojava.bio.structure.Atom)">torsionAngle(Atom, Atom, Atom, Atom)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">torsion angle 
 = angle between the normal vectors of the 
 two plains a-b-c and b-c-d.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html#toScoresList(org.biojava.bio.structure.align.model.AFPChain)">toScoresList(AFPChain)</a></span> - Static method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html" title="class in org.biojava.bio.structure.align.model">AfpChainWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/OptimalCECPParameters.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/OptimalCECPParameters.html" title="class in org.biojava.bio.structure.align.ce">OptimalCECPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/CountProgressListener.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/CountProgressListener.html" title="class in org.biojava.bio.structure.align.client">CountProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/PdbPair.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/PdbPair.html" title="class in org.biojava.bio.structure.align.client">PdbPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/StructureName.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/StructureName.html" title="class in org.biojava.bio.structure.align.client">StructureName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCat.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCat.html" title="class in org.biojava.bio.structure.align.fatcat">FatCat</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/AligMatEl.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/AligMatEl.html" title="class in org.biojava.bio.structure.align.helper">AligMatEl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/IndexPair.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/IndexPair.html" title="class in org.biojava.bio.structure.align.helper">IndexPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/JointFragments.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/JointFragments.html" title="class in org.biojava.bio.structure.align.helper">JointFragments</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFP.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFP.html" title="class in org.biojava.bio.structure.align.model">AFP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentResult</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>
<div class="block">print the idx positions of this alignment</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/PdbPairsMessage.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/PdbPairsMessage.html" title="class in org.biojava.bio.structure.align.xml">PdbPairsMessage</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcid.html#toString()">toString()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcid.html" title="interface in org.biojava.bio.structure">AminoAcid</a></dt>
<dd>
<div class="block">string representation.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcidImpl.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcidImpl.html" title="class in org.biojava.bio.structure">AminoAcidImpl</a></dt>
<dd>
<div class="block">string representation.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>
<div class="block">string representation.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Author.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Author.html" title="class in org.biojava.bio.structure">Author</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Bond.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Bond.html" title="class in org.biojava.bio.structure">Bond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathCategory.html#toString()">toString()</a></span> - Method in enum org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathCategory.html" title="enum in org.biojava.bio.structure.cath">CathCategory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathDomain.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathDomain.html" title="class in org.biojava.bio.structure.cath">CathDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathNode.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathNode.html" title="class in org.biojava.bio.structure.cath">CathNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathSegment.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathSegment.html" title="class in org.biojava.bio.structure.cath">CathSegment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#toString()">toString()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">string representation.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">String representation.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">String representation of a DBRef.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/CutSites.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/CutSites.html" title="class in org.biojava.bio.structure.domain.pdp">CutSites</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/CutValues.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/CutValues.html" title="class in org.biojava.bio.structure.domain.pdp">CutValues</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Domain.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Domain.html" title="class in org.biojava.bio.structure.domain.pdp">Domain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Segment.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Segment.html" title="class in org.biojava.bio.structure.domain.pdp">Segment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AbstractBean.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AbstractBean.html" title="class in org.biojava.bio.structure.io.mmcif.model">AbstractBean</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemCompDescriptor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html" title="class in org.biojava.bio.structure.io.mmcif.model">DatabasePDBrev</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model">Entity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssembly.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssembly.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructAssembly</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssemblyGen.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssemblyGen.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructAssemblyGen</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperList.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructOperList</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html" title="class in org.biojava.bio.structure.io.mmcif.model">Struct</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsEntity.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsEntity.html" title="class in org.biojava.bio.structure.io.sifts">SiftsEntity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsResidue.html" title="class in org.biojava.bio.structure.io.sifts">SiftsResidue</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/sifts/SiftsSegment.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.io.sifts.<a href="./org/biojava/bio/structure/io/sifts/SiftsSegment.html" title="class in org.biojava.bio.structure.io.sifts">SiftsSegment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/NucleotideImpl.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/NucleotideImpl.html" title="class in org.biojava.bio.structure">NucleotideImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>
<div class="block">String representation</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html" title="class in org.biojava.bio.structure.quaternary">ModelTransformationMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/OrderedPair.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/OrderedPair.html" title="class in org.biojava.bio.structure.quaternary">OrderedPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ResidueNumber.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ResidueNumber.html" title="class in org.biojava.bio.structure">ResidueNumber</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopCategory.html#toString()">toString()</a></span> - Method in enum org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopCategory.html" title="enum in org.biojava.bio.structure.scop">ScopCategory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDescription.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDescription.html" title="class in org.biojava.bio.structure.scop">ScopDescription</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopNode.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopNode.html" title="class in org.biojava.bio.structure.scop">ScopNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/BridgeType.html#toString()">toString()</a></span> - Method in enum org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/BridgeType.html" title="enum in org.biojava.bio.structure.secstruc">BridgeType</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/HBond.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/HBond.html" title="class in org.biojava.bio.structure.secstruc">HBond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/Ladder.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/Ladder.html" title="class in org.biojava.bio.structure.secstruc">Ladder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStruc.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStruc.html" title="class in org.biojava.bio.structure.secstruc">SecStruc</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html" title="class in org.biojava.bio.structure.secstruc">SecStrucGroup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucState.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucState.html" title="class in org.biojava.bio.structure.secstruc">SecStrucState</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucType.html#toString()">toString()</a></span> - Method in enum org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucType.html" title="enum in org.biojava.bio.structure.secstruc">SecStrucType</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Site.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Site.html" title="class in org.biojava.bio.structure">Site</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#toString()">toString()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">String representation of object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">string representation.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/util/FileDownloadUtils.html#toUnixPath(java.lang.String)">toUnixPath(String)</a></span> - Static method in class org.biojava.bio.structure.io.util.<a href="./org/biojava/bio/structure/io/util/FileDownloadUtils.html" title="class in org.biojava.bio.structure.io.util">FileDownloadUtils</a></dt>
<dd>
<div class="block">Converts path to Unix convention and adds a terminating slash if it was omitted</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html#toWebSiteDisplay(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">toWebSiteDisplay(AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html" title="class in org.biojava.bio.structure.align.model">AfpChainWriter</a></dt>
<dd>
<div class="block">Prints the afpChain as a nicely formatted alignment, including alignment
 statistics, the aligned sequences themselves, and information about the 
 superposition.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html#toWebSiteDisplay(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], boolean)">toWebSiteDisplay(AFPChain, Atom[], Atom[], boolean)</a></span> - Static method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html" title="class in org.biojava.bio.structure.align.model">AfpChainWriter</a></dt>
<dd>
<div class="block">Prints the afpChain as a nicely formatted alignment, including alignment
 statistics, the aligned sequences themselves, and information about the 
 superposition.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/UserConfiguration.html#toXML(java.io.PrintWriter)">toXML(PrintWriter)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/UserConfiguration.html" title="class in org.biojava.bio.structure.align.util">UserConfiguration</a></dt>
<dd>
<div class="block">convert Configuration to an XML file so it can be serialized</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/UserConfiguration.html#toXML(org.biojava3.core.util.XMLWriter)">toXML(XMLWriter)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/UserConfiguration.html" title="class in org.biojava.bio.structure.align.util">UserConfiguration</a></dt>
<dd>
<div class="block">convert Configuration to an XML file so it can be serialized
 add to an already existing xml file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/AFPChainXMLConverter.html#toXML(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">toXML(AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/AFPChainXMLConverter.html" title="class in org.biojava.bio.structure.align.xml">AFPChainXMLConverter</a></dt>
<dd>
<div class="block">Convert an afpChain to a simple XML representation</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/AFPChainXMLConverter.html#toXML(org.biojava.bio.structure.align.model.AFPChain, java.io.StringWriter, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">toXML(AFPChain, StringWriter, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/AFPChainXMLConverter.html" title="class in org.biojava.bio.structure.align.xml">AFPChainXMLConverter</a></dt>
<dd>
<div class="block">Write the XML representation to a StringWriter</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/AFPChainXMLConverter.html#toXML(org.biojava.bio.structure.align.model.AFPChain)">toXML(AFPChain)</a></span> - Static method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/AFPChainXMLConverter.html" title="class in org.biojava.bio.structure.align.xml">AFPChainXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/HasResultXMLConverter.html#toXML(boolean)">toXML(boolean)</a></span> - Method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/HasResultXMLConverter.html" title="class in org.biojava.bio.structure.align.xml">HasResultXMLConverter</a></dt>
<dd>
<div class="block">return flag if the server has a result</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/PositionInQueueXMLConverter.html#toXML(int)">toXML(int)</a></span> - Method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/PositionInQueueXMLConverter.html" title="class in org.biojava.bio.structure.align.xml">PositionInQueueXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/RepresentativeXMLConverter.html#toXML(java.util.SortedSet)">toXML(SortedSet&lt;String&gt;)</a></span> - Static method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/RepresentativeXMLConverter.html" title="class in org.biojava.bio.structure.align.xml">RepresentativeXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/AssignmentXMLSerializer.html#toXML()">toXML()</a></span> - Method in class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/AssignmentXMLSerializer.html" title="class in org.biojava.bio.structure.domain">AssignmentXMLSerializer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssemblyGenXMLContainer.html#toXML()">toXML()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssemblyGenXMLContainer.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructAssemblyGenXMLContainer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssemblyXMLContainer.html#toXML()">toXML()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructAssemblyXMLContainer.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructAssemblyXMLContainer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperListXMLContainer.html#toXML()">toXML()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxStructOperListXMLContainer.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxStructOperListXMLContainer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html#toXML()">toXML()</a></span> - Method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html" title="class in org.biojava.bio.structure.quaternary">ModelTransformationMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html#toXML(org.biojava3.core.util.PrettyXMLWriter)">toXML(PrettyXMLWriter)</a></span> - Method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html" title="class in org.biojava.bio.structure.quaternary">ModelTransformationMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/ScopDescriptions.html#toXML()">toXML()</a></span> - Method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/ScopDescriptions.html" title="class in org.biojava.bio.structure.scop.server">ScopDescriptions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/ScopDomains.html#toXML()">toXML()</a></span> - Method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/ScopDomains.html" title="class in org.biojava.bio.structure.scop.server">ScopDomains</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/ScopNodes.html#toXML()">toXML()</a></span> - Method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/ScopNodes.html" title="class in org.biojava.bio.structure.scop.server">ScopNodes</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/TreeSetStringWrapper.html#toXML()">toXML()</a></span> - Method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/TreeSetStringWrapper.html" title="class in org.biojava.bio.structure.scop.server">TreeSetStringWrapper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#trace()">trace()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Matrix trace.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#trace1">trace1</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#trace1">trace1</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#trace2">trace2</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#trace2">trace2</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#traceFragmentMatrix(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">traceFragmentMatrix(AFPChain, Atom[], Atom[])</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#traceFragmentMatrix(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">traceFragmentMatrix(AFPChain, Atom[], Atom[])</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html#transformPoint(double[], double[])">transformPoint(double[], double[])</a></span> - Method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html" title="class in org.biojava.bio.structure.quaternary">ModelTransformationMatrix</a></dt>
<dd>
<div class="block">This function will change the contents of result, but will not change point.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#transpose()">transpose()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Matrix transpose.</div>
</dd>
<dt><span class="strong"><a href="./demo/DemoSCOP.html#traverseHierarchy()">traverseHierarchy()</a></span> - Method in class demo.<a href="./demo/DemoSCOP.html" title="class in demo">DemoSCOP</a></dt>
<dd>
<div class="block">Traverse throught the SCOP hierarchy</div>
</dd>
<dt><a href="./org/biojava/bio/structure/scop/server/TreeSetStringWrapper.html" title="class in org.biojava.bio.structure.scop.server"><span class="strong">TreeSetStringWrapper</span></a> - Class in <a href="./org/biojava/bio/structure/scop/server/package-summary.html">org.biojava.bio.structure.scop.server</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/TreeSetStringWrapper.html#TreeSetStringWrapper()">TreeSetStringWrapper()</a></span> - Constructor for class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/TreeSetStringWrapper.html" title="class in org.biojava.bio.structure.scop.server">TreeSetStringWrapper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#triggerDocumentEnd()">triggerDocumentEnd()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#triggerDocumentStart()">triggerDocumentStart()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#triggerGeneric(java.lang.String, java.util.List, java.util.List)">triggerGeneric(String, List&lt;String&gt;, List&lt;String&gt;)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#triggerNewChemComp(org.biojava.bio.structure.io.mmcif.model.ChemComp)">triggerNewChemComp(ChemComp)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#triggerNewChemCompDescriptor(org.biojava.bio.structure.io.mmcif.model.ChemCompDescriptor)">triggerNewChemCompDescriptor(ChemCompDescriptor)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#triggerNewEntity(org.biojava.bio.structure.io.mmcif.model.Entity)">triggerNewEntity(Entity)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#triggerNewEntityPolySeq(org.biojava.bio.structure.io.mmcif.model.EntityPolySeq)">triggerNewEntityPolySeq(EntityPolySeq)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#triggerNewPdbxEntityNonPoly(org.biojava.bio.structure.io.mmcif.model.PdbxEntityNonPoly)">triggerNewPdbxEntityNonPoly(PdbxEntityNonPoly)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#triggerNewPdbxStructOper(org.biojava.bio.structure.io.mmcif.model.PdbxStructOperList)">triggerNewPdbxStructOper(PdbxStructOperList)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#triggerNewRefine(org.biojava.bio.structure.io.mmcif.model.Refine)">triggerNewRefine(Refine)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#triggerNewStructAsym(org.biojava.bio.structure.io.mmcif.model.StructAsym)">triggerNewStructAsym(StructAsym)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#triggerNewStructKeywords(org.biojava.bio.structure.io.mmcif.model.StructKeywords)">triggerNewStructKeywords(StructKeywords)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/OptimalCECPParameters.html#tryAllCPs">tryAllCPs</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/OptimalCECPParameters.html" title="class in org.biojava.bio.structure.align.ce">OptimalCECPParameters</a></dt>
<dd>
<div class="block">If true, ignores <a href="./org/biojava/bio/structure/align/ce/OptimalCECPParameters.html#cpPoint"><code>OptimalCECPParameters.cpPoint</code></a> and tries all possible cp points.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/SeqRes2AtomAligner.html#trySimpleMatch(java.util.List, java.util.List)">trySimpleMatch(List&lt;Group&gt;, List&lt;Group&gt;)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/SeqRes2AtomAligner.html" title="class in org.biojava.bio.structure.io">SeqRes2AtomAligner</a></dt>
<dd>
<div class="block">a simple matching approach that tries to do a 1:1 mapping between SEQRES and ATOM records
  returns true if this simple matching approach worked fine</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileParser.html#TURN">TURN</a></span> - Static variable in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileParser.html" title="class in org.biojava.bio.structure.io">PDBFileParser</a></dt>
<dd>
<div class="block">Turn secondary structure assignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/AFPTwister.html#twistOptimized(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">twistOptimized(AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/AFPTwister.html" title="class in org.biojava.bio.structure.align">AFPTwister</a></dt>
<dd>
<div class="block">superimposing according to the optimized alignment</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/AFPTwister.html#twistPDB(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">twistPDB(AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/AFPTwister.html" title="class in org.biojava.bio.structure.align">AFPTwister</a></dt>
<dd>
<div class="block">calculate the total rmsd of the blocks
 output a merged pdb file for both proteins                                                                                                                                       
     protein 1, in chain A                                                                                                                                                            
    protein 2 is twisted according to the twists detected, in chain B</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcidImpl.html#type">type</a></span> - Static variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcidImpl.html" title="class in org.biojava.bio.structure">AminoAcidImpl</a></dt>
<dd>
<div class="block">this is an Amino acid.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#type">type</a></span> - Static variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">this is a "hetatm".</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/NucleotideImpl.html#type">type</a></span> - Static variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/NucleotideImpl.html" title="class in org.biojava.bio.structure">NucleotideImpl</a></dt>
<dd>
<div class="block">this is a "nucleotide", a special occurance of a Hetatom.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/BridgeType.html#type">type</a></span> - Variable in enum org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/BridgeType.html" title="enum in org.biojava.bio.structure.secstruc">BridgeType</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucType.html#type">type</a></span> - Variable in enum org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucType.html" title="enum in org.biojava.bio.structure.secstruc">SecStrucType</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_U_">
<!--   -->
</a>
<h2 class="title">U</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#uminus()">uminus()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Unary minus</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#UNDERSCORE">UNDERSCORE</a></span> - Static variable in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#unitVector(org.biojava.bio.structure.Atom)">unitVector(Atom)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">return the unit vector of vector a .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#UNKNOWN_GROUP_LABEL">UNKNOWN_GROUP_LABEL</a></span> - Static variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/UnknownPdbAminoAcidException.html" title="class in org.biojava.bio.structure"><span class="strong">UnknownPdbAminoAcidException</span></a> - Exception in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">An exception for use during translating amino acids in a PDB file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/UnknownPdbAminoAcidException.html#UnknownPdbAminoAcidException(java.lang.String)">UnknownPdbAminoAcidException(String)</a></span> - Constructor for exception org.biojava.bio.structure.<a href="./org/biojava/bio/structure/UnknownPdbAminoAcidException.html" title="class in org.biojava.bio.structure">UnknownPdbAminoAcidException</a></dt>
<dd>
<div class="block">Constructs a PDBParseException object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/UnknownPdbAminoAcidException.html#UnknownPdbAminoAcidException(java.lang.String, java.lang.Throwable)">UnknownPdbAminoAcidException(String, Throwable)</a></span> - Constructor for exception org.biojava.bio.structure.<a href="./org/biojava/bio/structure/UnknownPdbAminoAcidException.html" title="class in org.biojava.bio.structure">UnknownPdbAminoAcidException</a></dt>
<dd>
<div class="block">Constructs a UnknownPdbAminoAcidException object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/UnknownPdbAminoAcidException.html#UnknownPdbAminoAcidException(java.lang.Throwable)">UnknownPdbAminoAcidException(Throwable)</a></span> - Constructor for exception org.biojava.bio.structure.<a href="./org/biojava/bio/structure/UnknownPdbAminoAcidException.html" title="class in org.biojava.bio.structure">UnknownPdbAminoAcidException</a></dt>
<dd>
<div class="block">Constructs a UnknownPdbAminoAcidException object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html#updateFullSymmetryDataWithInverseFractionalTransform(org.biojava.bio.structure.quaternary.ModelTransformationMatrix, org.biojava.bio.structure.quaternary.ModelTransformationMatrix)">updateFullSymmetryDataWithInverseFractionalTransform(ModelTransformationMatrix, ModelTransformationMatrix)</a></span> - Method in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html" title="class in org.biojava.bio.structure.quaternary">ModelTransformationMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#updateMatrixWithSequenceConservation(double[][], org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], org.biojava.bio.structure.align.ce.CeParameters)">updateMatrixWithSequenceConservation(double[][], Atom[], Atom[], CeParameters)</a></span> - Static method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>
<div class="block">Modifies an alignment matrix by favoring the alignment of similar and identical amino acids and penalizing the alignment of unrelated ones.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#updateMatrixWithSequenceConservation(double[][], org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], org.biojava.bio.structure.align.ce.CeParameters)">updateMatrixWithSequenceConservation(double[][], Atom[], Atom[], CeParameters)</a></span> - Static method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>
<div class="block">Modifies an alignment matrix by favoring the alignment of similar and identical amino acids and penalizing the alignment of unrelated ones.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/AFPOptimizer.html#updateScore(org.biojava.bio.structure.align.fatcat.calc.FatCatParameters, org.biojava.bio.structure.align.model.AFPChain)">updateScore(FatCatParameters, AFPChain)</a></span> - Static method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/AFPOptimizer.html" title="class in org.biojava.bio.structure.align.fatcat.calc">AFPOptimizer</a></dt>
<dd>
<div class="block">to update the chaining score after block delete and merge processed
 the blockScore value is important for significance evaluation</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AlignmentTools.html#updateSuperposition(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">updateSuperposition(AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AlignmentTools.html" title="class in org.biojava.bio.structure.align.util">AlignmentTools</a></dt>
<dd>
<div class="block">After the alignment changes (optAln, optLen, blockNum, at a minimum),
 many other properties which depend on the superposition will be invalid.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/RemoteDomainProvider.html#url">url</a></span> - Variable in class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/RemoteDomainProvider.html" title="class in org.biojava.bio.structure.domain">RemoteDomainProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/HTTPConnectionTools.html#USERAGENT">USERAGENT</a></span> - Static variable in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/HTTPConnectionTools.html" title="class in org.biojava.bio.structure.align.util">HTTPConnectionTools</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/ce/UserArgumentProcessor.html" title="interface in org.biojava.bio.structure.align.ce"><span class="strong">UserArgumentProcessor</span></a> - Interface in <a href="./org/biojava/bio/structure/align/ce/package-summary.html">org.biojava.bio.structure.align.ce</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/util/UserConfiguration.html" title="class in org.biojava.bio.structure.align.util"><span class="strong">UserConfiguration</span></a> - Class in <a href="./org/biojava/bio/structure/align/util/package-summary.html">org.biojava.bio.structure.align.util</a></dt>
<dd>
<div class="block">A container to persist config to the file system</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/UserConfiguration.html#UserConfiguration()">UserConfiguration()</a></span> - Constructor for class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/UserConfiguration.html" title="class in org.biojava.bio.structure.align.util">UserConfiguration</a></dt>
<dd>
<div class="block">Default UserConfiguration:
 
 split directory
 autofetch files
 default download location.</div>
</dd>
</dl>
<a name="_V_">
<!--   -->
</a>
<h2 class="title">V</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/GapArray.html#value">value</a></span> - Variable in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/GapArray.html" title="class in org.biojava.bio.structure.align.helper">GapArray</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/BondType.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/BondType.html" title="enum in org.biojava.bio.structure">BondType</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathCategory.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathCategory.html" title="enum in org.biojava.bio.structure.cath">CathCategory</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ElementType.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ElementType.html" title="enum in org.biojava.bio.structure">ElementType</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/chem/PolymerType.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava.bio.structure.io.mmcif.chem.<a href="./org/biojava/bio/structure/io/mmcif/chem/PolymerType.html" title="enum in org.biojava.bio.structure.io.mmcif.chem">PolymerType</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/chem/ResidueType.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava.bio.structure.io.mmcif.chem.<a href="./org/biojava/bio/structure/io/mmcif/chem/ResidueType.html" title="enum in org.biojava.bio.structure.io.mmcif.chem">ResidueType</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBStatus.Status.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBStatus.Status.html" title="enum in org.biojava.bio.structure">PDBStatus.Status</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopCategory.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopCategory.html" title="enum in org.biojava.bio.structure.scop">ScopCategory</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/BridgeType.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/BridgeType.html" title="enum in org.biojava.bio.structure.secstruc">BridgeType</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucType.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucType.html" title="enum in org.biojava.bio.structure.secstruc">SecStrucType</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#valueOfIgnoreCase(java.lang.String)">valueOfIgnoreCase(String)</a></span> - Static method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns the Element that corresponds to the specified element symbol.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/BondType.html#values()">values()</a></span> - Static method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/BondType.html" title="enum in org.biojava.bio.structure">BondType</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/cath/CathCategory.html#values()">values()</a></span> - Static method in enum org.biojava.bio.structure.cath.<a href="./org/biojava/bio/structure/cath/CathCategory.html" title="enum in org.biojava.bio.structure.cath">CathCategory</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#values()">values()</a></span> - Static method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ElementType.html#values()">values()</a></span> - Static method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ElementType.html" title="enum in org.biojava.bio.structure">ElementType</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/chem/PolymerType.html#values()">values()</a></span> - Static method in enum org.biojava.bio.structure.io.mmcif.chem.<a href="./org/biojava/bio/structure/io/mmcif/chem/PolymerType.html" title="enum in org.biojava.bio.structure.io.mmcif.chem">PolymerType</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/chem/ResidueType.html#values()">values()</a></span> - Static method in enum org.biojava.bio.structure.io.mmcif.chem.<a href="./org/biojava/bio/structure/io/mmcif/chem/ResidueType.html" title="enum in org.biojava.bio.structure.io.mmcif.chem">ResidueType</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBStatus.Status.html#values()">values()</a></span> - Static method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBStatus.Status.html" title="enum in org.biojava.bio.structure">PDBStatus.Status</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html#values">values</a></span> - Variable in class org.biojava.bio.structure.quaternary.<a href="./org/biojava/bio/structure/quaternary/ModelTransformationMatrix.html" title="class in org.biojava.bio.structure.quaternary">ModelTransformationMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopCategory.html#values()">values()</a></span> - Static method in enum org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopCategory.html" title="enum in org.biojava.bio.structure.scop">ScopCategory</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/BridgeType.html#values()">values()</a></span> - Static method in enum org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/BridgeType.html" title="enum in org.biojava.bio.structure.secstruc">BridgeType</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucType.html#values()">values()</a></span> - Static method in enum org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucType.html" title="enum in org.biojava.bio.structure.secstruc">SecStrucType</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#vectorProduct(org.biojava.bio.structure.Atom, org.biojava.bio.structure.Atom)">vectorProduct(Atom, Atom)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">Vector product .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/CutDomain.html#verbose">verbose</a></span> - Static variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/CutDomain.html" title="class in org.biojava.bio.structure.domain.pdp">CutDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCPMain.html#version">version</a></span> - Static variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCPMain.html" title="class in org.biojava.bio.structure.align.ce">CeCPMain</a></dt>
<dd>
<div class="block">version history:
  1.3 - Short CPs are now discarded
  1.2 - now supports check AlignmentTools.isSequentialAlignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeMain.html#version">version</a></span> - Static variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeMain.html" title="class in org.biojava.bio.structure.align.ce">CeMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/OptimalCECPMain.html#version">version</a></span> - Static variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/OptimalCECPMain.html" title="class in org.biojava.bio.structure.align.ce">OptimalCECPMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCat.html#VERSION">VERSION</a></span> - Static variable in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCat.html" title="class in org.biojava.bio.structure.align.fatcat">FatCat</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_W_">
<!--   -->
</a>
<h2 class="title">W</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#winSize">winSize</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/SynchronizedOutFile.html#write(java.lang.String)">write(String)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/SynchronizedOutFile.html" title="class in org.biojava.bio.structure.align.util">SynchronizedOutFile</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_X_">
<!--   -->
</a>
<h2 class="title">X</h2>
<dl>
<dt><a href="./org/biojava/bio/structure/scop/server/XMLUtil.html" title="class in org.biojava.bio.structure.scop.server"><span class="strong">XMLUtil</span></a> - Class in <a href="./org/biojava/bio/structure/scop/server/package-summary.html">org.biojava.bio.structure.scop.server</a></dt>
<dd>
<div class="block">Utility classes for the XML serialization and de-serialization of SCOP.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/XMLUtil.html#XMLUtil()">XMLUtil()</a></span> - Constructor for class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/XMLUtil.html" title="class in org.biojava.bio.structure.scop.server">XMLUtil</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_Z_">
<!--   -->
</a>
<h2 class="title">Z</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#zStrAlign(int, int, double, int)">zStrAlign(int, int, double, int)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#zStrAlign(int, int, double, int)">zStrAlign(int, int, double, int)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#zThr">zThr</a></span> - Static variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html#zThr">zThr</a></span> - Static variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCalculatorEnhanced.html" title="class in org.biojava.bio.structure.align.ce">CeCalculatorEnhanced</a></dt>
<dd>&nbsp;</dd>
</dl>
<a href="#_A_">A</a>&nbsp;<a href="#_B_">B</a>&nbsp;<a href="#_C_">C</a>&nbsp;<a href="#_D_">D</a>&nbsp;<a href="#_E_">E</a>&nbsp;<a href="#_F_">F</a>&nbsp;<a href="#_G_">G</a>&nbsp;<a href="#_H_">H</a>&nbsp;<a href="#_I_">I</a>&nbsp;<a href="#_J_">J</a>&nbsp;<a href="#_L_">L</a>&nbsp;<a href="#_M_">M</a>&nbsp;<a href="#_N_">N</a>&nbsp;<a href="#_O_">O</a>&nbsp;<a href="#_P_">P</a>&nbsp;<a href="#_Q_">Q</a>&nbsp;<a href="#_R_">R</a>&nbsp;<a href="#_S_">S</a>&nbsp;<a href="#_T_">T</a>&nbsp;<a href="#_U_">U</a>&nbsp;<a href="#_V_">V</a>&nbsp;<a href="#_W_">W</a>&nbsp;<a href="#_X_">X</a>&nbsp;<a href="#_Z_">Z</a>&nbsp;</div>
<!-- ======= START OF BOTTOM NAVBAR ====== -->
<div class="bottomNav"><a name="navbar_bottom">
<!--   -->
</a><a href="#skip-navbar_bottom" title="Skip navigation links"></a><a name="navbar_bottom_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="./overview-summary.html">Overview</a></li>
<li>Package</li>
<li>Class</li>
<li>Use</li>
<li><a href="./overview-tree.html">Tree</a></li>
<li><a href="./deprecated-list.html">Deprecated</a></li>
<li class="navBarCell1Rev">Index</li>
<li><a href="./help-doc.html">Help</a></li>
</ul>
<div class="aboutLanguage"><em><script src="http://www.google-analytics.com/urchin.js" type="text/javascript"></script><script type="text/javascript">_uacct = "UA-1326640-1";urchinTracker();</script></em></div>
</div>
<div class="subNav">
<ul class="navList">
<li>Prev</li>
<li>Next</li>
</ul>
<ul class="navList">
<li><a href="./index.html?index-all.html" target="_top">Frames</a></li>
<li><a href="index-all.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_bottom">
<li><a href="./allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_bottom");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<a name="skip-navbar_bottom">
<!--   -->
</a></div>
<!-- ======== END OF BOTTOM NAVBAR ======= -->
<p class="legalCopy"><small>Copyright &#169; 2012 <a href="http://www.biojava.org">BioJava</a>. All Rights Reserved.</small></p>
</body>
</html>
