<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!-- NewPage -->
<html lang="en">
<head>
<!-- Generated by javadoc (version 1.7.0_05) on Fri Nov 30 10:53:27 PST 2012 -->
<meta http-equiv="Content-Type" content="text/html" charset="UTF-8">
<title>Uses of Class org.biojava3.core.sequence.compound.AminoAcidCompound (biojava3-core 3.0.5 API)</title>
<meta name="date" content="2012-11-30">
<link rel="stylesheet" type="text/css" href="../../../../../../stylesheet.css" title="Style">
</head>
<body>
<script type="text/javascript"><!--
    if (location.href.indexOf('is-external=true') == -1) {
        parent.document.title="Uses of Class org.biojava3.core.sequence.compound.AminoAcidCompound (biojava3-core 3.0.5 API)";
    }
//-->
</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
<!-- ========= START OF TOP NAVBAR ======= -->
<div class="topNav"><a name="navbar_top">
<!--   -->
</a><a href="#skip-navbar_top" title="Skip navigation links"></a><a name="navbar_top_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../../../../../../overview-summary.html">Overview</a></li>
<li><a href="../package-summary.html">Package</a></li>
<li><a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">Class</a></li>
<li class="navBarCell1Rev">Use</li>
<li><a href="../package-tree.html">Tree</a></li>
<li><a href="../../../../../../deprecated-list.html">Deprecated</a></li>
<li><a href="../../../../../../index-all.html">Index</a></li>
<li><a href="../../../../../../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li>Prev</li>
<li>Next</li>
</ul>
<ul class="navList">
<li><a href="../../../../../../index.html?org/biojava3/core/sequence/compound/class-use/AminoAcidCompound.html" target="_top">Frames</a></li>
<li><a href="AminoAcidCompound.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_top">
<li><a href="../../../../../../allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_top");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<a name="skip-navbar_top">
<!--   -->
</a></div>
<!-- ========= END OF TOP NAVBAR ========= -->
<div class="header">
<h2 title="Uses of Class org.biojava3.core.sequence.compound.AminoAcidCompound" class="title">Uses of Class<br>org.biojava3.core.sequence.compound.AminoAcidCompound</h2>
</div>
<div class="classUseContainer">
<ul class="blockList">
<li class="blockList">
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing packages, and an explanation">
<caption><span>Packages that use <a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Package</th>
<th class="colLast" scope="col">Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><a href="#org.biojava3.core.sequence">org.biojava3.core.sequence</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="#org.biojava3.core.sequence.compound">org.biojava3.core.sequence.compound</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="#org.biojava3.core.sequence.io">org.biojava3.core.sequence.io</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="#org.biojava3.core.sequence.transcription">org.biojava3.core.sequence.transcription</a></td>
<td class="colLast">&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList">
<ul class="blockList">
<li class="blockList"><a name="org.biojava3.core.sequence">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a> in <a href="../../../../../../org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing constructors, and an explanation">
<caption><span>Constructor parameters in <a href="../../../../../../org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a> with type arguments of type <a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colOne" scope="col">Constructor and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/ProteinSequence.html#ProteinSequence(org.biojava3.core.sequence.template.ProxySequenceReader)">ProteinSequence</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/ProxySequenceReader.html" title="interface in org.biojava3.core.sequence.template">ProxySequenceReader</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;proxyLoader)</code>
<div class="block">A protein sequence where the storage of the sequence is somewhere else.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/ProteinSequence.html#ProteinSequence(org.biojava3.core.sequence.template.ProxySequenceReader, org.biojava3.core.sequence.template.CompoundSet)">ProteinSequence</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/ProxySequenceReader.html" title="interface in org.biojava3.core.sequence.template">ProxySequenceReader</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;proxyLoader,
               <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;compoundSet)</code>
<div class="block">A protein sequence where the storage of the sequence is somewhere else with user defined
 set of amino acids.</div>
</td>
</tr>
<tr class="altColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/ProteinSequence.html#ProteinSequence(org.biojava3.core.sequence.template.ProxySequenceReader, org.biojava3.core.sequence.template.CompoundSet)">ProteinSequence</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/ProxySequenceReader.html" title="interface in org.biojava3.core.sequence.template">ProxySequenceReader</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;proxyLoader,
               <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;compoundSet)</code>
<div class="block">A protein sequence where the storage of the sequence is somewhere else with user defined
 set of amino acids.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/ProteinSequence.html#ProteinSequence(java.lang.String, org.biojava3.core.sequence.template.CompoundSet)">ProteinSequence</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a>&nbsp;seqString,
               <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;compoundSet)</code>
<div class="block">Create a protein from a string with a user defined set of amino acids</div>
</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava3.core.sequence.compound">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a> in <a href="../../../../../../org/biojava3/core/sequence/compound/package-summary.html">org.biojava3.core.sequence.compound</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/biojava3/core/sequence/compound/package-summary.html">org.biojava3.core.sequence.compound</a> that return <a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a></code></td>
<td class="colLast"><span class="strong">AminoAcidCompoundSet.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompoundSet.html#getCompoundForString(java.lang.String)">getCompoundForString</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a>&nbsp;string)</code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/biojava3/core/sequence/compound/package-summary.html">org.biojava3.core.sequence.compound</a> that return types with arguments of type <a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;</code></td>
<td class="colLast"><span class="strong">AminoAcidCompoundSet.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompoundSet.html#getAllCompounds()">getAllCompounds</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;</code></td>
<td class="colLast"><span class="strong">AminoAcidCompound.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html#getCompoundSet()">getCompoundSet</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/util/Set.html?is-external=true" title="class or interface in java.util">Set</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;</code></td>
<td class="colLast"><span class="strong">AminoAcidCompoundSet.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompoundSet.html#getEquivalentCompounds(org.biojava3.core.sequence.compound.AminoAcidCompound)">getEquivalentCompounds</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&nbsp;compound)</code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/biojava3/core/sequence/compound/package-summary.html">org.biojava3.core.sequence.compound</a> with parameters of type <a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>boolean</code></td>
<td class="colLast"><span class="strong">AminoAcidCompoundSet.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompoundSet.html#compoundsEquivalent(org.biojava3.core.sequence.compound.AminoAcidCompound, org.biojava3.core.sequence.compound.AminoAcidCompound)">compoundsEquivalent</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&nbsp;compoundOne,
                   <a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&nbsp;compoundTwo)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/util/Set.html?is-external=true" title="class or interface in java.util">Set</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;</code></td>
<td class="colLast"><span class="strong">AminoAcidCompoundSet.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompoundSet.html#getEquivalentCompounds(org.biojava3.core.sequence.compound.AminoAcidCompound)">getEquivalentCompounds</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&nbsp;compound)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a></code></td>
<td class="colLast"><span class="strong">AminoAcidCompoundSet.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompoundSet.html#getStringForCompound(org.biojava3.core.sequence.compound.AminoAcidCompound)">getStringForCompound</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&nbsp;compound)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>boolean</code></td>
<td class="colLast"><span class="strong">AminoAcidCompoundSet.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompoundSet.html#hasCompound(org.biojava3.core.sequence.compound.AminoAcidCompound)">hasCompound</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&nbsp;compound)</code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Method parameters in <a href="../../../../../../org/biojava3/core/sequence/compound/package-summary.html">org.biojava3.core.sequence.compound</a> with type arguments of type <a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">AminoAcidCompoundSet.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompoundSet.html#verifySequence(org.biojava3.core.sequence.template.Sequence)">verifySequence</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;sequence)</code>&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava3.core.sequence.io">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a> in <a href="../../../../../../org/biojava3/core/sequence/io/package-summary.html">org.biojava3.core.sequence.io</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/biojava3/core/sequence/io/package-summary.html">org.biojava3.core.sequence.io</a> that return types with arguments of type <a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;</code></td>
<td class="colLast"><span class="strong">CasePreservingProteinSequenceCreator.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/io/CasePreservingProteinSequenceCreator.html#getSequence(java.util.List)">getSequence</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;list)</code>
<div class="block">Assumes all compounds were uppercase</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;</code></td>
<td class="colLast"><span class="strong">FileProxyProteinSequenceCreator.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/io/FileProxyProteinSequenceCreator.html#getSequence(java.util.List)">getSequence</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;list)</code>
<div class="block">Not sure of use case and currently not supported</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;</code></td>
<td class="colLast"><span class="strong">ProteinSequenceCreator.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/io/ProteinSequenceCreator.html#getSequence(java.util.List)">getSequence</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;list)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;</code></td>
<td class="colLast"><span class="strong">CasePreservingProteinSequenceCreator.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/io/CasePreservingProteinSequenceCreator.html#getSequence(org.biojava3.core.sequence.template.ProxySequenceReader, long)">getSequence</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/ProxySequenceReader.html" title="interface in org.biojava3.core.sequence.template">ProxySequenceReader</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;proxyLoader,
           long&nbsp;index)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;</code></td>
<td class="colLast"><span class="strong">FileProxyProteinSequenceCreator.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/io/FileProxyProteinSequenceCreator.html#getSequence(org.biojava3.core.sequence.template.ProxySequenceReader, long)">getSequence</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/ProxySequenceReader.html" title="interface in org.biojava3.core.sequence.template">ProxySequenceReader</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;proxyLoader,
           long&nbsp;index)</code>
<div class="block">Should be able to extend the same concept to a remote URL call or database connection.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;</code></td>
<td class="colLast"><span class="strong">ProteinSequenceCreator.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/io/ProteinSequenceCreator.html#getSequence(org.biojava3.core.sequence.template.ProxySequenceReader, long)">getSequence</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/ProxySequenceReader.html" title="interface in org.biojava3.core.sequence.template">ProxySequenceReader</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;proxyLoader,
           long&nbsp;index)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;</code></td>
<td class="colLast"><span class="strong">CasePreservingProteinSequenceCreator.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/io/CasePreservingProteinSequenceCreator.html#getSequence(java.lang.String, long)">getSequence</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a>&nbsp;sequence,
           long&nbsp;index)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;</code></td>
<td class="colLast"><span class="strong">FileProxyProteinSequenceCreator.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/io/FileProxyProteinSequenceCreator.html#getSequence(java.lang.String, long)">getSequence</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a>&nbsp;sequence,
           long&nbsp;index)</code>
<div class="block">Even though we are passing in the sequence we really only care about the length of the sequence and the offset
 index in the fasta file.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;</code></td>
<td class="colLast"><span class="strong">ProteinSequenceCreator.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/io/ProteinSequenceCreator.html#getSequence(java.lang.String, long)">getSequence</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a>&nbsp;sequence,
           long&nbsp;index)</code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/biojava3/core/sequence/io/package-summary.html">org.biojava3.core.sequence.io</a> with parameters of type <a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>boolean</code></td>
<td class="colLast"><span class="strong">IUPACParser.IUPACTable.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/io/IUPACParser.IUPACTable.html#isStart(org.biojava3.core.sequence.compound.AminoAcidCompound)">isStart</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&nbsp;compound)</code>
<div class="block">Returns true if the given compound was a start codon in this
 codon table.</div>
</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Method parameters in <a href="../../../../../../org/biojava3/core/sequence/io/package-summary.html">org.biojava3.core.sequence.io</a> with type arguments of type <a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/transcription/Table.Codon.html" title="class in org.biojava3.core.sequence.transcription">Table.Codon</a>&gt;</code></td>
<td class="colLast"><span class="strong">IUPACParser.IUPACTable.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/io/IUPACParser.IUPACTable.html#getCodonCompoundSet(org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.template.CompoundSet)">getCodonCompoundSet</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;rnaCompounds,
                   <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;aminoAcidCompounds)</code>
<div class="block">Returns the compound set of codons</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/transcription/Table.Codon.html" title="class in org.biojava3.core.sequence.transcription">Table.Codon</a>&gt;</code></td>
<td class="colLast"><span class="strong">IUPACParser.IUPACTable.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/io/IUPACParser.IUPACTable.html#getCodons(org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.template.CompoundSet)">getCodons</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;nucelotides,
         <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;aminoAcids)</code>
<div class="block">Returns a list of codons where the source and target compounds
 are the same as those given by the parameters.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;</code></td>
<td class="colLast"><span class="strong">CasePreservingProteinSequenceCreator.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/io/CasePreservingProteinSequenceCreator.html#getSequence(java.util.List)">getSequence</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;list)</code>
<div class="block">Assumes all compounds were uppercase</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;</code></td>
<td class="colLast"><span class="strong">FileProxyProteinSequenceCreator.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/io/FileProxyProteinSequenceCreator.html#getSequence(java.util.List)">getSequence</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;list)</code>
<div class="block">Not sure of use case and currently not supported</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;</code></td>
<td class="colLast"><span class="strong">ProteinSequenceCreator.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/io/ProteinSequenceCreator.html#getSequence(java.util.List)">getSequence</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;list)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;</code></td>
<td class="colLast"><span class="strong">CasePreservingProteinSequenceCreator.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/io/CasePreservingProteinSequenceCreator.html#getSequence(org.biojava3.core.sequence.template.ProxySequenceReader, long)">getSequence</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/ProxySequenceReader.html" title="interface in org.biojava3.core.sequence.template">ProxySequenceReader</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;proxyLoader,
           long&nbsp;index)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;</code></td>
<td class="colLast"><span class="strong">FileProxyProteinSequenceCreator.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/io/FileProxyProteinSequenceCreator.html#getSequence(org.biojava3.core.sequence.template.ProxySequenceReader, long)">getSequence</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/ProxySequenceReader.html" title="interface in org.biojava3.core.sequence.template">ProxySequenceReader</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;proxyLoader,
           long&nbsp;index)</code>
<div class="block">Should be able to extend the same concept to a remote URL call or database connection.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;</code></td>
<td class="colLast"><span class="strong">ProteinSequenceCreator.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/io/ProteinSequenceCreator.html#getSequence(org.biojava3.core.sequence.template.ProxySequenceReader, long)">getSequence</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/ProxySequenceReader.html" title="interface in org.biojava3.core.sequence.template">ProxySequenceReader</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;proxyLoader,
           long&nbsp;index)</code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing constructors, and an explanation">
<caption><span>Constructor parameters in <a href="../../../../../../org/biojava3/core/sequence/io/package-summary.html">org.biojava3.core.sequence.io</a> with type arguments of type <a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colOne" scope="col">Constructor and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/io/CasePreservingProteinSequenceCreator.html#CasePreservingProteinSequenceCreator(org.biojava3.core.sequence.template.CompoundSet)">CasePreservingProteinSequenceCreator</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;compoundSet)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/io/FileProxyProteinSequenceCreator.html#FileProxyProteinSequenceCreator(java.io.File, org.biojava3.core.sequence.template.CompoundSet)">FileProxyProteinSequenceCreator</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/io/File.html?is-external=true" title="class or interface in java.io">File</a>&nbsp;fastaFile,
                               <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;compoundSet)</code>
<div class="block">Need File so that we can store full path name in SequenceFileProxyLoader for Random File access as a quick read</div>
</td>
</tr>
<tr class="altColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/io/ProteinSequenceCreator.html#ProteinSequenceCreator(org.biojava3.core.sequence.template.CompoundSet)">ProteinSequenceCreator</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;compoundSet)</code>&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava3.core.sequence.transcription">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a> in <a href="../../../../../../org/biojava3/core/sequence/transcription/package-summary.html">org.biojava3.core.sequence.transcription</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/biojava3/core/sequence/transcription/package-summary.html">org.biojava3.core.sequence.transcription</a> that return <a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a></code></td>
<td class="colLast"><span class="strong">Table.Codon.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/transcription/Table.Codon.html#getAminoAcid()">getAminoAcid</a></strong>()</code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/biojava3/core/sequence/transcription/package-summary.html">org.biojava3.core.sequence.transcription</a> that return types with arguments of type <a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&gt;</code></td>
<td class="colLast"><span class="strong">RNAToAminoAcidTranslator.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/transcription/RNAToAminoAcidTranslator.html#createSequences(org.biojava3.core.sequence.template.Sequence)">createSequences</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;originalSequence)</code>
<div class="block">Performs the core conversion of RNA to Peptide.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;</code></td>
<td class="colLast"><span class="strong">TranscriptionEngine.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/transcription/TranscriptionEngine.html#getAminoAcidCompounds()">getAminoAcidCompounds</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/io/template/SequenceCreatorInterface.html" title="interface in org.biojava3.core.sequence.io.template">SequenceCreatorInterface</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;</code></td>
<td class="colLast"><span class="strong">TranscriptionEngine.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/transcription/TranscriptionEngine.html#getProteinSequenceCreator()">getProteinSequenceCreator</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/util/Map.html?is-external=true" title="class or interface in java.util">Map</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/transcription/Frame.html" title="enum in org.biojava3.core.sequence.transcription">Frame</a>,<a href="../../../../../../org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&gt;</code></td>
<td class="colLast"><span class="strong">TranscriptionEngine.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/transcription/TranscriptionEngine.html#multipleFrameTranslation(org.biojava3.core.sequence.template.Sequence, org.biojava3.core.sequence.transcription.Frame...)">multipleFrameTranslation</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;dna,
                        <a href="../../../../../../org/biojava3/core/sequence/transcription/Frame.html" title="enum in org.biojava3.core.sequence.transcription">Frame</a>...&nbsp;frames)</code>
<div class="block">A way of translating DNA in a number of frames</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;</code></td>
<td class="colLast"><span class="strong">TranscriptionEngine.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/transcription/TranscriptionEngine.html#translate(org.biojava3.core.sequence.template.Sequence)">translate</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;dna)</code>
<div class="block">Quick method to let you go from a CDS to a Peptide quickly.</div>
</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/biojava3/core/sequence/transcription/package-summary.html">org.biojava3.core.sequence.transcription</a> with parameters of type <a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>boolean</code></td>
<td class="colLast"><span class="strong">Table.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/transcription/Table.html#isStart(org.biojava3.core.sequence.compound.AminoAcidCompound)">isStart</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&nbsp;compound)</code>
<div class="block">Returns true if the given compound could have been a start amino acid;
 this does not assert if the codon that actually coded for the amino
 acid was a start codon.</div>
</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Method parameters in <a href="../../../../../../org/biojava3/core/sequence/transcription/package-summary.html">org.biojava3.core.sequence.transcription</a> with type arguments of type <a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava3.core.sequence.transcription">TranscriptionEngine.Builder</a></code></td>
<td class="colLast"><span class="strong">TranscriptionEngine.Builder.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/transcription/TranscriptionEngine.Builder.html#aminoAcidsCompounds(org.biojava3.core.sequence.template.CompoundSet)">aminoAcidsCompounds</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;compounds)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/transcription/Table.Codon.html" title="class in org.biojava3.core.sequence.transcription">Table.Codon</a>&gt;</code></td>
<td class="colLast"><span class="strong">Table.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/transcription/Table.html#getCodonCompoundSet(org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.template.CompoundSet)">getCodonCompoundSet</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;rnaCompounds,
                   <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;aminoAcidCompounds)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/transcription/Table.Codon.html" title="class in org.biojava3.core.sequence.transcription">Table.Codon</a>&gt;</code></td>
<td class="colLast"><span class="strong">Table.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/transcription/Table.html#getCodons(org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.template.CompoundSet)">getCodons</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;nucelotides,
         <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;aminoAcids)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>protected void</code></td>
<td class="colLast"><span class="strong">RNAToAminoAcidTranslator.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/transcription/RNAToAminoAcidTranslator.html#postProcessCompoundLists(java.util.List)">postProcessCompoundLists</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;<a href="http://docs.oracle.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&gt;&nbsp;compoundLists)</code>
<div class="block">Performs the trimming of stop codons and the conversion of a valid start
 amino acid to M</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/biojava3/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava3.core.sequence.transcription">TranscriptionEngine.Builder</a></code></td>
<td class="colLast"><span class="strong">TranscriptionEngine.Builder.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/transcription/TranscriptionEngine.Builder.html#proteinCreator(org.biojava3.core.sequence.io.template.SequenceCreatorInterface)">proteinCreator</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/io/template/SequenceCreatorInterface.html" title="interface in org.biojava3.core.sequence.io.template">SequenceCreatorInterface</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;creator)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>protected void</code></td>
<td class="colLast"><span class="strong">RNAToAminoAcidTranslator.</span><code><strong><a href="../../../../../../org/biojava3/core/sequence/transcription/RNAToAminoAcidTranslator.html#trimStop(java.util.List)">trimStop</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;sequence)</code>
<div class="block">Imperfect code.</div>
</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing constructors, and an explanation">
<caption><span>Constructors in <a href="../../../../../../org/biojava3/core/sequence/transcription/package-summary.html">org.biojava3.core.sequence.transcription</a> with parameters of type <a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colOne" scope="col">Constructor and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/transcription/Table.Codon.html#Table.Codon(org.biojava3.core.sequence.transcription.Table.CaseInsensitiveTriplet, org.biojava3.core.sequence.compound.AminoAcidCompound, boolean, boolean)">Table.Codon</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/transcription/Table.CaseInsensitiveTriplet.html" title="class in org.biojava3.core.sequence.transcription">Table.CaseInsensitiveTriplet</a>&nbsp;triplet,
           <a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&nbsp;aminoAcid,
           boolean&nbsp;start,
           boolean&nbsp;stop)</code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing constructors, and an explanation">
<caption><span>Constructor parameters in <a href="../../../../../../org/biojava3/core/sequence/transcription/package-summary.html">org.biojava3.core.sequence.transcription</a> with type arguments of type <a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colOne" scope="col">Constructor and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/transcription/RNAToAminoAcidTranslator.html#RNAToAminoAcidTranslator(org.biojava3.core.sequence.io.template.SequenceCreatorInterface, org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.transcription.Table, boolean, boolean, boolean)">RNAToAminoAcidTranslator</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/io/template/SequenceCreatorInterface.html" title="interface in org.biojava3.core.sequence.io.template">SequenceCreatorInterface</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;creator,
                        <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;nucleotides,
                        <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/transcription/Table.Codon.html" title="class in org.biojava3.core.sequence.transcription">Table.Codon</a>&gt;&nbsp;codons,
                        <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;aminoAcids,
                        <a href="../../../../../../org/biojava3/core/sequence/transcription/Table.html" title="interface in org.biojava3.core.sequence.transcription">Table</a>&nbsp;table,
                        boolean&nbsp;trimStops,
                        boolean&nbsp;initMetOnly,
                        boolean&nbsp;translateNCodons)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colLast"><code><strong><a href="../../../../../../org/biojava3/core/sequence/transcription/RNAToAminoAcidTranslator.html#RNAToAminoAcidTranslator(org.biojava3.core.sequence.io.template.SequenceCreatorInterface, org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.transcription.Table, boolean, boolean, boolean)">RNAToAminoAcidTranslator</a></strong>(<a href="../../../../../../org/biojava3/core/sequence/io/template/SequenceCreatorInterface.html" title="interface in org.biojava3.core.sequence.io.template">SequenceCreatorInterface</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;creator,
                        <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt;&nbsp;nucleotides,
                        <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/transcription/Table.Codon.html" title="class in org.biojava3.core.sequence.transcription">Table.Codon</a>&gt;&nbsp;codons,
                        <a href="../../../../../../org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a>&lt;<a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;aminoAcids,
                        <a href="../../../../../../org/biojava3/core/sequence/transcription/Table.html" title="interface in org.biojava3.core.sequence.transcription">Table</a>&nbsp;table,
                        boolean&nbsp;trimStops,
                        boolean&nbsp;initMetOnly,
                        boolean&nbsp;translateNCodons)</code>&nbsp;</td>
</tr>
</tbody>
</table>
</li>
</ul>
</li>
</ul>
</div>
<!-- ======= START OF BOTTOM NAVBAR ====== -->
<div class="bottomNav"><a name="navbar_bottom">
<!--   -->
</a><a href="#skip-navbar_bottom" title="Skip navigation links"></a><a name="navbar_bottom_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../../../../../../overview-summary.html">Overview</a></li>
<li><a href="../package-summary.html">Package</a></li>
<li><a href="../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">Class</a></li>
<li class="navBarCell1Rev">Use</li>
<li><a href="../package-tree.html">Tree</a></li>
<li><a href="../../../../../../deprecated-list.html">Deprecated</a></li>
<li><a href="../../../../../../index-all.html">Index</a></li>
<li><a href="../../../../../../help-doc.html">Help</a></li>
</ul>
<div class="aboutLanguage"><em><script src="http://www.google-analytics.com/urchin.js" type="text/javascript"></script><script type="text/javascript">_uacct = "UA-1326640-1";urchinTracker();</script></em></div>
</div>
<div class="subNav">
<ul class="navList">
<li>Prev</li>
<li>Next</li>
</ul>
<ul class="navList">
<li><a href="../../../../../../index.html?org/biojava3/core/sequence/compound/class-use/AminoAcidCompound.html" target="_top">Frames</a></li>
<li><a href="AminoAcidCompound.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_bottom">
<li><a href="../../../../../../allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_bottom");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<a name="skip-navbar_bottom">
<!--   -->
</a></div>
<!-- ======== END OF BOTTOM NAVBAR ======= -->
<p class="legalCopy"><small>Copyright &#169; 2012 <a href="http://www.biojava.org">BioJava</a>. All Rights Reserved.</small></p>
</body>
</html>
