<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!-- NewPage -->
<html lang="en">
<head>
<!-- Generated by javadoc (version 1.7.0) on Fri Sep 02 09:21:05 PDT 2011 -->
<meta http-equiv="Content-Type" content="text/html" charset="UTF-8">
<title>Index (biojava 3.0.2 API)</title>
<meta name="date" content="2011-09-02">
<link rel="stylesheet" type="text/css" href="./stylesheet.css" title="Style">
</head>
<body>
<script type="text/javascript"><!--
    if (location.href.indexOf('is-external=true') == -1) {
        parent.document.title="Index (biojava 3.0.2 API)";
    }
//-->
</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
<!-- ========= START OF TOP NAVBAR ======= -->
<div class="topNav"><a name="navbar_top">
<!--   -->
</a><a href="#skip-navbar_top" title="Skip navigation links"></a><a name="navbar_top_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="./overview-summary.html">Overview</a></li>
<li>Package</li>
<li>Class</li>
<li>Use</li>
<li><a href="./overview-tree.html">Tree</a></li>
<li><a href="./deprecated-list.html">Deprecated</a></li>
<li class="navBarCell1Rev">Index</li>
<li><a href="./help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li>Prev</li>
<li>Next</li>
</ul>
<ul class="navList">
<li><a href="./index.html?index-all.html" target="_top">Frames</a></li>
<li><a href="index-all.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_top">
<li><a href="./allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_top");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<a name="skip-navbar_top">
<!--   -->
</a></div>
<!-- ========= END OF TOP NAVBAR ========= -->
<div class="contentContainer"><a href="#_A_">A</a>&nbsp;<a href="#_B_">B</a>&nbsp;<a href="#_C_">C</a>&nbsp;<a href="#_D_">D</a>&nbsp;<a href="#_E_">E</a>&nbsp;<a href="#_F_">F</a>&nbsp;<a href="#_G_">G</a>&nbsp;<a href="#_H_">H</a>&nbsp;<a href="#_I_">I</a>&nbsp;<a href="#_J_">J</a>&nbsp;<a href="#_K_">K</a>&nbsp;<a href="#_L_">L</a>&nbsp;<a href="#_M_">M</a>&nbsp;<a href="#_N_">N</a>&nbsp;<a href="#_O_">O</a>&nbsp;<a href="#_P_">P</a>&nbsp;<a href="#_Q_">Q</a>&nbsp;<a href="#_R_">R</a>&nbsp;<a href="#_S_">S</a>&nbsp;<a href="#_T_">T</a>&nbsp;<a href="#_U_">U</a>&nbsp;<a href="#_V_">V</a>&nbsp;<a href="#_W_">W</a>&nbsp;<a href="#_X_">X</a>&nbsp;<a href="#_Y_">Y</a>&nbsp;<a href="#_Z_">Z</a>&nbsp;<a name="_A_">
<!--   -->
</a>
<h2 class="title">A</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/Constraints.html#A">A</a></span> - Static variable in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/Constraints.html" title="class in org.biojava3.aaproperties">Constraints</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/data/sequence/SequenceUtil.html#AA">AA</a></span> - Static variable in class org.biojava3.data.sequence.<a href="./org/biojava3/data/sequence/SequenceUtil.html" title="class in org.biojava3.data.sequence">SequenceUtil</a></dt>
<dd>
<div class="block">Valid Amino acids</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#aa">aa</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/Constraints.html#aa2ExtinctionCoefficient">aa2ExtinctionCoefficient</a></span> - Static variable in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/Constraints.html" title="class in org.biojava3.aaproperties">Constraints</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/Constraints.html#aa2Hydrophathicity">aa2Hydrophathicity</a></span> - Static variable in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/Constraints.html" title="class in org.biojava3.aaproperties">Constraints</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/Constraints.html#aa2MolecularWeight">aa2MolecularWeight</a></span> - Static variable in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/Constraints.html" title="class in org.biojava3.aaproperties">Constraints</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/Constraints.html#aa2PKa">aa2PKa</a></span> - Static variable in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/Constraints.html" title="class in org.biojava3.aaproperties">Constraints</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#aa2Triplet">aa2Triplet</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#aa3Hash">aa3Hash</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#aaIndex">aaIndex</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/alignment/aaindex/AAindexFactory.html" title="class in org.biojava3.alignment.aaindex"><span class="strong">AAindexFactory</span></a> - Class in <a href="./org/biojava3/alignment/aaindex/package-summary.html">org.biojava3.alignment.aaindex</a></dt>
<dd>
<div class="block">Factory class to get Providers for substitution matrices the are provided by the AAINDEX database.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/aaindex/AAindexFactory.html#AAindexFactory()">AAindexFactory()</a></span> - Constructor for class org.biojava3.alignment.aaindex.<a href="./org/biojava3/alignment/aaindex/AAindexFactory.html" title="class in org.biojava3.alignment.aaindex">AAindexFactory</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/alignment/aaindex/AAIndexFileParser.html" title="class in org.biojava3.alignment.aaindex"><span class="strong">AAIndexFileParser</span></a> - Class in <a href="./org/biojava3/alignment/aaindex/package-summary.html">org.biojava3.alignment.aaindex</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/aaindex/AAIndexFileParser.html#AAIndexFileParser()">AAIndexFileParser()</a></span> - Constructor for class org.biojava3.alignment.aaindex.<a href="./org/biojava3/alignment/aaindex/AAIndexFileParser.html" title="class in org.biojava3.alignment.aaindex">AAIndexFileParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/alignment/aaindex/AAIndexProvider.html" title="interface in org.biojava3.alignment.aaindex"><span class="strong">AAIndexProvider</span></a> - Interface in <a href="./org/biojava3/alignment/aaindex/package-summary.html">org.biojava3.alignment.aaindex</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html#aaScore(char, char)">aaScore(char, char)</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html" title="class in org.biojava.bio.structure.align.util">AFPAlignmentDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/gui/AboutDialog.html" title="class in org.biojava.bio.structure.align.gui"><span class="strong">AboutDialog</span></a> - Class in <a href="./org/biojava/bio/structure/align/gui/package-summary.html">org.biojava.bio.structure.align.gui</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/AboutDialog.html#AboutDialog()">AboutDialog()</a></span> - Constructor for class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/AboutDialog.html" title="class in org.biojava.bio.structure.align.gui">AboutDialog</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/AbstractBean.html" title="class in org.biojava.bio.structure.io.mmcif.model"><span class="strong">AbstractBean</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a></dt>
<dd>
<div class="block">a generic class that implements the toString method for a bean</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AbstractBean.html#AbstractBean()">AbstractBean()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AbstractBean.html" title="class in org.biojava.bio.structure.io.mmcif.model">AbstractBean</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/template/AbstractCompound.html" title="class in org.biojava3.core.sequence.template"><span class="strong">AbstractCompound</span></a> - Class in <a href="./org/biojava3/core/sequence/template/package-summary.html">org.biojava3.core.sequence.template</a></dt>
<dd>
<div class="block">The details of a Compound</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompound.html#AbstractCompound(java.lang.String)">AbstractCompound(String)</a></span> - Constructor for class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompound.html" title="class in org.biojava3.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava3.core.sequence.template"><span class="strong">AbstractCompoundSet</span></a>&lt;<a href="./org/biojava3/core/sequence/template/AbstractCompoundSet.html" title="type parameter in AbstractCompoundSet">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/core/sequence/template/package-summary.html">org.biojava3.core.sequence.template</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompoundSet.html#AbstractCompoundSet()">AbstractCompoundSet()</a></span> - Constructor for class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava3.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/template/AbstractCompoundTranslator.html" title="class in org.biojava3.core.sequence.template"><span class="strong">AbstractCompoundTranslator</span></a>&lt;<a href="./org/biojava3/core/sequence/template/AbstractCompoundTranslator.html" title="type parameter in AbstractCompoundTranslator">F</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>,<a href="./org/biojava3/core/sequence/template/AbstractCompoundTranslator.html" title="type parameter in AbstractCompoundTranslator">T</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/core/sequence/template/package-summary.html">org.biojava3.core.sequence.template</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompoundTranslator.html#AbstractCompoundTranslator(org.biojava3.core.sequence.io.template.SequenceCreatorInterface, org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.template.CompoundSet)">AbstractCompoundTranslator(SequenceCreatorInterface&lt;T&gt;, CompoundSet&lt;F&gt;, CompoundSet&lt;T&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompoundTranslator.html" title="class in org.biojava3.core.sequence.template">AbstractCompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/features/AbstractFeature.html" title="class in org.biojava3.core.sequence.features"><span class="strong">AbstractFeature</span></a>&lt;<a href="./org/biojava3/core/sequence/features/AbstractFeature.html" title="type parameter in AbstractFeature">S</a> extends <a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a>&lt;<a href="./org/biojava3/core/sequence/features/AbstractFeature.html" title="type parameter in AbstractFeature">C</a>&gt;,<a href="./org/biojava3/core/sequence/features/AbstractFeature.html" title="type parameter in AbstractFeature">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/core/sequence/features/package-summary.html">org.biojava3.core.sequence.features</a></dt>
<dd>
<div class="block">A feature is currently any descriptive item that can be associated with a sequence position(s)
 A feature has a type and a source which is currently a string to allow flexibility for the user
 Ideally well defined features should have a class to describe attributes of that feature</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/AbstractFeature.html#AbstractFeature(java.lang.String, java.lang.String)">AbstractFeature(String, String)</a></span> - Constructor for class org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/AbstractFeature.html" title="class in org.biojava3.core.sequence.features">AbstractFeature</a></dt>
<dd>
<div class="block">A feature has a type and a source</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava3.core.sequence.location.template"><span class="strong">AbstractLocation</span></a> - Class in <a href="./org/biojava3/core/sequence/location/template/package-summary.html">org.biojava3.core.sequence.location.template</a></dt>
<dd>
<div class="block">Base abstraction of a location which encodes for the majority of important
 features about a location such as the start, end and strand</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html#AbstractLocation()">AbstractLocation()</a></span> - Constructor for class org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava3.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html#AbstractLocation(org.biojava3.core.sequence.location.template.Point, org.biojava3.core.sequence.location.template.Point, org.biojava3.core.sequence.Strand, boolean, boolean, java.util.List)">AbstractLocation(Point, Point, Strand, boolean, boolean, List&lt;Location&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava3.core.sequence.location.template">AbstractLocation</a></dt>
<dd>
<div class="block">Default constructor</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html#AbstractLocation(org.biojava3.core.sequence.location.template.Point, org.biojava3.core.sequence.location.template.Point, org.biojava3.core.sequence.Strand, boolean, boolean, org.biojava3.core.sequence.AccessionID, java.util.List)">AbstractLocation(Point, Point, Strand, boolean, boolean, AccessionID, List&lt;Location&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava3.core.sequence.location.template">AbstractLocation</a></dt>
<dd>
<div class="block">Default constructor</div>
</dd>
<dt><a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava3.alignment.template"><span class="strong">AbstractMatrixAligner</span></a>&lt;<a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html" title="type parameter in AbstractMatrixAligner">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html" title="type parameter in AbstractMatrixAligner">C</a>&gt;,<a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html" title="type parameter in AbstractMatrixAligner">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/alignment/template/package-summary.html">org.biojava3.alignment.template</a></dt>
<dd>
<div class="block">Implements common code for an <a href="./org/biojava3/alignment/template/Aligner.html" title="interface in org.biojava3.alignment.template"><code>Aligner</code></a> which builds a score matrix during computation.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#AbstractMatrixAligner()">AbstractMatrixAligner()</a></span> - Constructor for class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava3.alignment.template">AbstractMatrixAligner</a></dt>
<dd>
<div class="block">Before running an alignment, data must be sent in via calls to <a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#setGapPenalty(org.biojava3.alignment.template.GapPenalty)"><code>AbstractMatrixAligner.setGapPenalty(GapPenalty)</code></a> and
 <a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#setSubstitutionMatrix(org.biojava3.alignment.template.SubstitutionMatrix)"><code>AbstractMatrixAligner.setSubstitutionMatrix(SubstitutionMatrix)</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#AbstractMatrixAligner(org.biojava3.alignment.template.GapPenalty, org.biojava3.alignment.template.SubstitutionMatrix)">AbstractMatrixAligner(GapPenalty, SubstitutionMatrix&lt;C&gt;)</a></span> - Constructor for class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava3.alignment.template">AbstractMatrixAligner</a></dt>
<dd>
<div class="block">Prepares for an alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#AbstractMatrixAligner(org.biojava3.alignment.template.GapPenalty, org.biojava3.alignment.template.SubstitutionMatrix, boolean)">AbstractMatrixAligner(GapPenalty, SubstitutionMatrix&lt;C&gt;, boolean)</a></span> - Constructor for class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava3.alignment.template">AbstractMatrixAligner</a></dt>
<dd>
<div class="block">Prepares for an alignment.</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/template/AbstractNucleotideCompoundSet.html" title="class in org.biojava3.core.sequence.template"><span class="strong">AbstractNucleotideCompoundSet</span></a>&lt;<a href="./org/biojava3/core/sequence/template/AbstractNucleotideCompoundSet.html" title="type parameter in AbstractNucleotideCompoundSet">C</a> extends <a href="./org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt; - Class in <a href="./org/biojava3/core/sequence/template/package-summary.html">org.biojava3.core.sequence.template</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractNucleotideCompoundSet.html#AbstractNucleotideCompoundSet()">AbstractNucleotideCompoundSet()</a></span> - Constructor for class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractNucleotideCompoundSet.html" title="class in org.biojava3.core.sequence.template">AbstractNucleotideCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html" title="class in org.biojava3.alignment.template"><span class="strong">AbstractPairwiseSequenceAligner</span></a>&lt;<a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html" title="type parameter in AbstractPairwiseSequenceAligner">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html" title="type parameter in AbstractPairwiseSequenceAligner">C</a>&gt;,<a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html" title="type parameter in AbstractPairwiseSequenceAligner">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/alignment/template/package-summary.html">org.biojava3.alignment.template</a></dt>
<dd>
<div class="block">Implements common code for an <a href="./org/biojava3/alignment/template/Aligner.html" title="interface in org.biojava3.alignment.template"><code>Aligner</code></a> for a pair of <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a>s.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html#AbstractPairwiseSequenceAligner()">AbstractPairwiseSequenceAligner()</a></span> - Constructor for class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html" title="class in org.biojava3.alignment.template">AbstractPairwiseSequenceAligner</a></dt>
<dd>
<div class="block">Before running a pairwise global sequence alignment, data must be sent in via calls to
 <a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html#setQuery(S)"><code>AbstractPairwiseSequenceAligner.setQuery(Sequence)</code></a>, <a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html#setTarget(S)"><code>AbstractPairwiseSequenceAligner.setTarget(Sequence)</code></a>, <a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#setGapPenalty(org.biojava3.alignment.template.GapPenalty)"><code>AbstractMatrixAligner.setGapPenalty(GapPenalty)</code></a>, and
 <a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#setSubstitutionMatrix(org.biojava3.alignment.template.SubstitutionMatrix)"><code>AbstractMatrixAligner.setSubstitutionMatrix(SubstitutionMatrix)</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html#AbstractPairwiseSequenceAligner(S, S, org.biojava3.alignment.template.GapPenalty, org.biojava3.alignment.template.SubstitutionMatrix)">AbstractPairwiseSequenceAligner(S, S, GapPenalty, SubstitutionMatrix&lt;C&gt;)</a></span> - Constructor for class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html" title="class in org.biojava3.alignment.template">AbstractPairwiseSequenceAligner</a></dt>
<dd>
<div class="block">Prepares for a pairwise global sequence alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html#AbstractPairwiseSequenceAligner(S, S, org.biojava3.alignment.template.GapPenalty, org.biojava3.alignment.template.SubstitutionMatrix, boolean)">AbstractPairwiseSequenceAligner(S, S, GapPenalty, SubstitutionMatrix&lt;C&gt;, boolean)</a></span> - Constructor for class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html" title="class in org.biojava3.alignment.template">AbstractPairwiseSequenceAligner</a></dt>
<dd>
<div class="block">Prepares for a pairwise sequence alignment.</div>
</dd>
<dt><a href="./org/biojava3/alignment/template/AbstractProfileProfileAligner.html" title="class in org.biojava3.alignment.template"><span class="strong">AbstractProfileProfileAligner</span></a>&lt;<a href="./org/biojava3/alignment/template/AbstractProfileProfileAligner.html" title="type parameter in AbstractProfileProfileAligner">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava3/alignment/template/AbstractProfileProfileAligner.html" title="type parameter in AbstractProfileProfileAligner">C</a>&gt;,<a href="./org/biojava3/alignment/template/AbstractProfileProfileAligner.html" title="type parameter in AbstractProfileProfileAligner">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/alignment/template/package-summary.html">org.biojava3.alignment.template</a></dt>
<dd>
<div class="block">Implements common code for an <a href="./org/biojava3/alignment/template/Aligner.html" title="interface in org.biojava3.alignment.template"><code>Aligner</code></a> for a pair of <a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template"><code>Profile</code></a>s.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractProfileProfileAligner.html#AbstractProfileProfileAligner()">AbstractProfileProfileAligner()</a></span> - Constructor for class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractProfileProfileAligner.html" title="class in org.biojava3.alignment.template">AbstractProfileProfileAligner</a></dt>
<dd>
<div class="block">Before running a profile-profile alignment, data must be sent in via calls to
 <a href="./org/biojava3/alignment/template/AbstractProfileProfileAligner.html#setQuery(org.biojava3.alignment.template.Profile)"><code>AbstractProfileProfileAligner.setQuery(Profile)</code></a>, <a href="./org/biojava3/alignment/template/AbstractProfileProfileAligner.html#setTarget(org.biojava3.alignment.template.Profile)"><code>AbstractProfileProfileAligner.setTarget(Profile)</code></a>, <a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#setGapPenalty(org.biojava3.alignment.template.GapPenalty)"><code>AbstractMatrixAligner.setGapPenalty(GapPenalty)</code></a>, and
 <a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#setSubstitutionMatrix(org.biojava3.alignment.template.SubstitutionMatrix)"><code>AbstractMatrixAligner.setSubstitutionMatrix(SubstitutionMatrix)</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractProfileProfileAligner.html#AbstractProfileProfileAligner(org.biojava3.alignment.template.Profile, org.biojava3.alignment.template.Profile, org.biojava3.alignment.template.GapPenalty, org.biojava3.alignment.template.SubstitutionMatrix)">AbstractProfileProfileAligner(Profile&lt;S, C&gt;, Profile&lt;S, C&gt;, GapPenalty, SubstitutionMatrix&lt;C&gt;)</a></span> - Constructor for class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractProfileProfileAligner.html" title="class in org.biojava3.alignment.template">AbstractProfileProfileAligner</a></dt>
<dd>
<div class="block">Prepares for a profile-profile alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractProfileProfileAligner.html#AbstractProfileProfileAligner(java.util.concurrent.Future, java.util.concurrent.Future, org.biojava3.alignment.template.GapPenalty, org.biojava3.alignment.template.SubstitutionMatrix)">AbstractProfileProfileAligner(Future&lt;ProfilePair&lt;S, C&gt;&gt;, Future&lt;ProfilePair&lt;S, C&gt;&gt;, GapPenalty, SubstitutionMatrix&lt;C&gt;)</a></span> - Constructor for class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractProfileProfileAligner.html" title="class in org.biojava3.alignment.template">AbstractProfileProfileAligner</a></dt>
<dd>
<div class="block">Prepares for a profile-profile alignment run concurrently.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractProfileProfileAligner.html#AbstractProfileProfileAligner(org.biojava3.alignment.template.Profile, java.util.concurrent.Future, org.biojava3.alignment.template.GapPenalty, org.biojava3.alignment.template.SubstitutionMatrix)">AbstractProfileProfileAligner(Profile&lt;S, C&gt;, Future&lt;ProfilePair&lt;S, C&gt;&gt;, GapPenalty, SubstitutionMatrix&lt;C&gt;)</a></span> - Constructor for class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractProfileProfileAligner.html" title="class in org.biojava3.alignment.template">AbstractProfileProfileAligner</a></dt>
<dd>
<div class="block">Prepares for a profile-profile alignment run concurrently.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractProfileProfileAligner.html#AbstractProfileProfileAligner(java.util.concurrent.Future, org.biojava3.alignment.template.Profile, org.biojava3.alignment.template.GapPenalty, org.biojava3.alignment.template.SubstitutionMatrix)">AbstractProfileProfileAligner(Future&lt;ProfilePair&lt;S, C&gt;&gt;, Profile&lt;S, C&gt;, GapPenalty, SubstitutionMatrix&lt;C&gt;)</a></span> - Constructor for class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractProfileProfileAligner.html" title="class in org.biojava3.alignment.template">AbstractProfileProfileAligner</a></dt>
<dd>
<div class="block">Prepares for a profile-profile alignment run concurrently.</div>
</dd>
<dt><a href="./org/biojava3/alignment/template/AbstractScorer.html" title="class in org.biojava3.alignment.template"><span class="strong">AbstractScorer</span></a> - Class in <a href="./org/biojava3/alignment/template/package-summary.html">org.biojava3.alignment.template</a></dt>
<dd>
<div class="block">Implements common code for algorithms which compute a score.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractScorer.html#AbstractScorer()">AbstractScorer()</a></span> - Constructor for class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractScorer.html" title="class in org.biojava3.alignment.template">AbstractScorer</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template"><span class="strong">AbstractSequence</span></a>&lt;<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="type parameter in AbstractSequence">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/core/sequence/template/package-summary.html">org.biojava3.core.sequence.template</a></dt>
<dd>
<div class="block">The base class for DNA, RNA and Protein sequences.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#AbstractSequence()">AbstractSequence()</a></span> - Constructor for class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#AbstractSequence(java.lang.String, org.biojava3.core.sequence.template.CompoundSet)">AbstractSequence(String, CompoundSet&lt;C&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>
<div class="block">Create a Sequence from a simple string where the values should be found in compoundSet</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#AbstractSequence(org.biojava3.core.sequence.template.SequenceReader, org.biojava3.core.sequence.template.CompoundSet)">AbstractSequence(SequenceReader&lt;C&gt;, CompoundSet&lt;C&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>
<div class="block">A ProxySequenceReader allows abstraction of both the storage of the sequence data and the location
 of the sequence data.</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/template/AbstractSequence.AnnotationType.html" title="enum in org.biojava3.core.sequence.template"><span class="strong">AbstractSequence.AnnotationType</span></a> - Enum in <a href="./org/biojava3/core/sequence/template/package-summary.html">org.biojava3.core.sequence.template</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/AbstractStructureAlignment.html" title="class in org.biojava.bio.structure.align"><span class="strong">AbstractStructureAlignment</span></a> - Class in <a href="./org/biojava/bio/structure/align/package-summary.html">org.biojava.bio.structure.align</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/AbstractStructureAlignment.html#AbstractStructureAlignment()">AbstractStructureAlignment()</a></span> - Constructor for class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/AbstractStructureAlignment.html" title="class in org.biojava.bio.structure.align">AbstractStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce"><span class="strong">AbstractUserArgumentProcessor</span></a> - Class in <a href="./org/biojava/bio/structure/align/ce/package-summary.html">org.biojava.bio.structure.align.ce</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html#AbstractUserArgumentProcessor()">AbstractUserArgumentProcessor()</a></span> - Constructor for class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce">AbstractUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/PDBFilter.html#accept(org.biojava.bio.structure.PDBHeader)">accept(PDBHeader)</a></span> - Method in interface org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/PDBFilter.html" title="interface in org.biojava.bio.structure.server">PDBFilter</a></dt>
<dd>
<div class="block">return true if this PDBHeader is passing this filter.</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/location/template/AccesionedLocation.html" title="interface in org.biojava3.core.sequence.location.template"><span class="strong">AccesionedLocation</span></a> - Interface in <a href="./org/biojava3/core/sequence/location/template/package-summary.html">org.biojava3.core.sequence.location.template</a></dt>
<dd>
<div class="block">A location which is bound to an AccessionID.</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/template/Accessioned.html" title="interface in org.biojava3.core.sequence.template"><span class="strong">Accessioned</span></a> - Interface in <a href="./org/biojava3/core/sequence/template/package-summary.html">org.biojava3.core.sequence.template</a></dt>
<dd>
<div class="block">Indicates an entity is accessioned</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/AccessionID.html" title="class in org.biojava3.core.sequence"><span class="strong">AccessionID</span></a> - Class in <a href="./org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a></dt>
<dd>
<div class="block">Used in Sequences as the unique indentifier.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/AccessionID.html#AccessionID()">AccessionID()</a></span> - Constructor for class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/AccessionID.html" title="class in org.biojava3.core.sequence">AccessionID</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/AccessionID.html#AccessionID(java.lang.String)">AccessionID(String)</a></span> - Constructor for class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/AccessionID.html" title="class in org.biojava3.core.sequence">AccessionID</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/AccessionID.html#AccessionID(java.lang.String, org.biojava3.core.sequence.DataSource)">AccessionID(String, DataSource)</a></span> - Constructor for class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/AccessionID.html" title="class in org.biojava3.core.sequence">AccessionID</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html#actionPerformed(java.awt.event.ActionEvent)">actionPerformed(ActionEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AligPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/ChooseDirAction.html#actionPerformed(java.awt.event.ActionEvent)">actionPerformed(ActionEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/ChooseDirAction.html" title="class in org.biojava.bio.structure.align.gui">ChooseDirAction</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/DBResultTable.html#actionPerformed(java.awt.event.ActionEvent)">actionPerformed(ActionEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/DBResultTable.html" title="class in org.biojava.bio.structure.align.gui">DBResultTable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/GUIAlignmentProgressListener.html#actionPerformed(java.awt.event.ActionEvent)">actionPerformed(ActionEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/GUIAlignmentProgressListener.html" title="class in org.biojava.bio.structure.align.gui">GUIAlignmentProgressListener</a></dt>
<dd>
<div class="block">Invoked when the user presses the stop button.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/JmolPanel.html#actionPerformed(java.awt.event.ActionEvent)">actionPerformed(ActionEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/JmolPanel.html" title="class in org.biojava.bio.structure.align.gui.jmol">JmolPanel</a></dt>
<dd>
<div class="block">The user selected one of the Combo boxes...</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/RasmolCommandListener.html#actionPerformed(java.awt.event.ActionEvent)">actionPerformed(ActionEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/RasmolCommandListener.html" title="class in org.biojava.bio.structure.align.gui.jmol">RasmolCommandListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html#actionPerformed(java.awt.event.ActionEvent)">actionPerformed(ActionEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html" title="class in org.biojava.bio.structure.align.gui.jmol">StructureAlignmentJmol</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/JPrintPanel.html#actionPerformed(java.awt.event.ActionEvent)">actionPerformed(ActionEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/JPrintPanel.html" title="class in org.biojava.bio.structure.align.gui">JPrintPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/MyAlignmentLoadListener.html#actionPerformed(java.awt.event.ActionEvent)">actionPerformed(ActionEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/MyAlignmentLoadListener.html" title="class in org.biojava.bio.structure.align.gui">MyAlignmentLoadListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/MyDistMaxListener.html#actionPerformed(java.awt.event.ActionEvent)">actionPerformed(ActionEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/MyDistMaxListener.html" title="class in org.biojava.bio.structure.align.gui">MyDistMaxListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/MyExportListener.html#actionPerformed(java.awt.event.ActionEvent)">actionPerformed(ActionEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/MyExportListener.html" title="class in org.biojava.bio.structure.align.gui">MyExportListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/MyOpenPdbFileListener.html#actionPerformed(java.awt.event.ActionEvent)">actionPerformed(ActionEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/MyOpenPdbFileListener.html" title="class in org.biojava.bio.structure.align.gui">MyOpenPdbFileListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/MySaveFileListener.html#actionPerformed(java.awt.event.ActionEvent)">actionPerformed(ActionEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/MySaveFileListener.html" title="class in org.biojava.bio.structure.align.gui">MySaveFileListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/ShowPDBIDListener.html#actionPerformed(java.awt.event.ActionEvent)">actionPerformed(ActionEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/ShowPDBIDListener.html" title="class in org.biojava.bio.structure.align.gui">ShowPDBIDListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/RasmolCommandListener.html#actionPerformed(java.awt.event.ActionEvent)">actionPerformed(ActionEvent)</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/RasmolCommandListener.html" title="class in org.biojava.bio.structure.gui">RasmolCommandListener</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html#actionPerformed(java.awt.event.ActionEvent)">actionPerformed(ActionEvent)</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html" title="class in org.biojava.bio.structure.gui">ScaleableMatrixPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/CutValues.html#AD">AD</a></span> - Variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/CutValues.html" title="class in org.biojava.bio.structure.domain.pdp">CutValues</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/BiojavaJmol.html#adapter">adapter</a></span> - Static variable in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/BiojavaJmol.html" title="class in org.biojava.bio.structure.gui">BiojavaJmol</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/structure/gui/JmolViewerImpl.html#adapter">adapter</a></span> - Static variable in class org.biojava3.structure.gui.<a href="./org/biojava3/structure/gui/JmolViewerImpl.html" title="class in org.biojava3.structure.gui">JmolViewerImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/JointFragments.html#add(int, int, int, int)">add(int, int, int, int)</a></span> - Method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/JointFragments.html" title="class in org.biojava.bio.structure.align.helper">JointFragments</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#add(org.biojava.bio.structure.Atom, org.biojava.bio.structure.Atom)">add(Atom, Atom)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">add two atoms ( a + b).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/FeatureList.html#add(org.biojava3.genome.parsers.gff.FeatureI)">add(FeatureI)</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/FeatureList.html" title="class in org.biojava3.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block">Add specified feature to the end of the list.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/FeatureList.html#add(java.util.Collection)">add(Collection&lt;FeatureI&gt;)</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/FeatureList.html" title="class in org.biojava3.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block">Add all features in the specified list or collection to this list.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructureAlignmentFactory.html#addAlgorithm(org.biojava.bio.structure.align.StructureAlignment)">addAlgorithm(StructureAlignment)</a></span> - Static method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructureAlignmentFactory.html" title="class in org.biojava.bio.structure.align">StructureAlignmentFactory</a></dt>
<dd>
<div class="block">Adds a new StructureAlignment algorithm to the list.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/MultipleSequenceAlignment.html#addAlignedSequence(S)">addAlignedSequence(S)</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/MultipleSequenceAlignment.html" title="class in org.biojava3.core.sequence">MultipleSequenceAlignment</a></dt>
<dd>
<div class="block">A sequence that has been aligned to other sequences will have inserts.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html#addAlignmentPositionListener(org.biojava.bio.structure.gui.events.AlignmentPositionListener)">addAlignmentPositionListener(AlignmentPositionListener)</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AligPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/SequenceDisplay.html#addAlignmentPositionListener(org.biojava.bio.structure.gui.events.AlignmentPositionListener)">addAlignmentPositionListener(AlignmentPositionListener)</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/SequenceDisplay.html" title="class in org.biojava.bio.structure.gui">SequenceDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/SequenceMouseListener.html#addAlignmentPositionListener(org.biojava.bio.structure.gui.events.AlignmentPositionListener)">addAlignmentPositionListener(AlignmentPositionListener)</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/SequenceMouseListener.html" title="class in org.biojava.bio.structure.gui.util">SequenceMouseListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html#addAlignmentProgressListener(org.biojava.bio.structure.align.events.AlignmentProgressListener)">addAlignmentProgressListener(AlignmentProgressListener)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html" title="class in org.biojava.bio.structure.align.client">FarmJobRunnable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/FarmJob.html#addAlignmentProgressListener(org.biojava.bio.structure.align.events.AlignmentProgressListener)">addAlignmentProgressListener(AlignmentProgressListener)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/FarmJob.html" title="class in org.biojava.bio.structure.align">FarmJob</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanelMouseMotionListener.html#addAligPosListener(org.biojava.bio.structure.gui.events.AlignmentPositionListener)">addAligPosListener(AlignmentPositionListener)</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanelMouseMotionListener.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AligPanelMouseMotionListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#addAltLoc(org.biojava.bio.structure.Group)">addAltLoc(Group)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">Add a group that is an alternate location for this group.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#addAltLoc(org.biojava.bio.structure.Group)">addAltLoc(Group)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/routines/AlignerHelper.html#addAnchors(org.biojava3.alignment.routines.AlignerHelper.Cut[], short[], boolean, int[])">addAnchors(AlignerHelper.Cut[], short[], boolean, int[])</a></span> - Static method in class org.biojava3.alignment.routines.<a href="./org/biojava3/alignment/routines/AlignerHelper.html" title="class in org.biojava3.alignment.routines">AlignerHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#addAtom(org.biojava.bio.structure.Atom)">addAtom(Atom)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">add an atom to this group.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#addAtom(org.biojava.bio.structure.Atom)">addAtom(Atom)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">add an atom to this group.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ModifiedCompound.html#addAtomLinkage(org.biojava3.protmod.structure.StructureAtomLinkage)">addAtomLinkage(StructureAtomLinkage)</a></span> - Method in interface org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ModifiedCompound.html" title="interface in org.biojava3.protmod.structure">ModifiedCompound</a></dt>
<dd>
<div class="block">Add a linkage.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ModifiedCompoundImpl.html#addAtomLinkage(org.biojava3.protmod.structure.StructureAtomLinkage)">addAtomLinkage(StructureAtomLinkage)</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ModifiedCompoundImpl.html" title="class in org.biojava3.protmod.structure">ModifiedCompoundImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ModifiedCompound.html#addAtomLinkages(java.util.Collection)">addAtomLinkages(Collection&lt;StructureAtomLinkage&gt;)</a></span> - Method in interface org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ModifiedCompound.html" title="interface in org.biojava3.protmod.structure">ModifiedCompound</a></dt>
<dd>
<div class="block">Add a collections of linkages.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ModifiedCompoundImpl.html#addAtomLinkages(java.util.Collection)">addAtomLinkages(Collection&lt;StructureAtomLinkage&gt;)</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ModifiedCompoundImpl.html" title="class in org.biojava3.protmod.structure">ModifiedCompoundImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/TranscriptSequence.html#addCDS(org.biojava3.core.sequence.AccessionID, int, int, int)">addCDS(AccessionID, int, int, int)</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/TranscriptSequence.html" title="class in org.biojava3.core.sequence">TranscriptSequence</a></dt>
<dd>
<div class="block">Add a Coding Sequence region with phase to the transcript sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#addChain(org.biojava.bio.structure.Chain)">addChain(Chain)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#addChain(org.biojava.bio.structure.Chain)">addChain(Chain)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">add a new chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#addChain(org.biojava.bio.structure.Chain, int)">addChain(Chain, int)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">add a new chain, if several models are available.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#addChain(org.biojava.bio.structure.Chain)">addChain(Chain)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">add a new chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#addChain(org.biojava.bio.structure.Chain, int)">addChain(Chain, int)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">add a new chain, if several models are available.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html#addChemComp(org.biojava.bio.structure.io.mmcif.model.ChemComp)">addChemComp(ChemComp)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html" title="class in org.biojava.bio.structure.io.mmcif">ChemicalComponentDictionary</a></dt>
<dd>
<div class="block">add a new component to the dictionary</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/XMLHelper.html#addChildElement(org.w3c.dom.Element, java.lang.String)">addChildElement(Element, String)</a></span> - Static method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/XMLHelper.html" title="class in org.biojava3.core.util">XMLHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompoundSet.html#addCompound(C, C, java.lang.Iterable)">addCompound(C, C, Iterable&lt;C&gt;)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava3.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompoundSet.html#addCompound(C, C, C...)">addCompound(C, C, C...)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava3.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompoundSet.html#addCompound(C)">addCompound(C)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava3.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompoundTranslator.html#addCompounds(F, T...)">addCompounds(F, T...)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompoundTranslator.html" title="class in org.biojava3.core.sequence.template">AbstractCompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompoundTranslator.html#addCompoundsToList(java.util.List, java.util.List)">addCompoundsToList(List&lt;T&gt;, List&lt;List&lt;T&gt;&gt;)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompoundTranslator.html" title="class in org.biojava3.core.sequence.template">AbstractCompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompoundTranslator.html#addCompoundToLists(java.util.List, T)">addCompoundToLists(List&lt;List&lt;T&gt;&gt;, T)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompoundTranslator.html" title="class in org.biojava3.core.sequence.template">AbstractCompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompoundSet.html#addEquivalent(C, C)">addEquivalent(C, C)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava3.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/GeneSequence.html#addExon(org.biojava3.core.sequence.AccessionID, int, int)">addExon(AccessionID, int, int)</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/GeneSequence.html" title="class in org.biojava3.core.sequence">GeneSequence</a></dt>
<dd>
<div class="block">Add an ExonSequence mainly used to mark as a feature</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/MMCIFFileReader.html#addExtension(java.lang.String)">addExtension(String)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/MMCIFFileReader.html" title="class in org.biojava.bio.structure.io">MMCIFFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#addExtension(java.lang.String)">addExtension(String)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>
<div class="block">define supported file extensions
 compressed extensions .Z,.gz do not need to be specified
 they are dealt with automatically.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/StructureIOFile.html#addExtension(java.lang.String)">addExtension(String)</a></span> - Method in interface org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/StructureIOFile.html" title="interface in org.biojava.bio.structure.io">StructureIOFile</a></dt>
<dd>
<div class="block">add a known File extension.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#addFeature(int, int, org.biojava3.core.sequence.features.FeatureInterface)">addFeature(int, int, FeatureInterface&lt;AbstractSequence&lt;C&gt;, C&gt;)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>
<div class="block">Method to help set the proper details for a feature as it relates to a sequence
 where the feature needs to have a location on the sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#addFeature(org.biojava3.core.sequence.features.FeatureInterface)">addFeature(FeatureInterface&lt;AbstractSequence&lt;C&gt;, C&gt;)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>
<div class="block">Add a feature to this sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/ChromosomeSequence.html#addGene(org.biojava3.core.sequence.AccessionID, int, int, org.biojava3.core.sequence.Strand)">addGene(AccessionID, int, int, Strand)</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/ChromosomeSequence.html" title="class in org.biojava3.core.sequence">ChromosomeSequence</a></dt>
<dd>
<div class="block">Add a gene to the chromosome sequence using bioIndexing starts at 1 instead of 0.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/GeneFeatureHelper.html#addGeneIDGFF2GeneFeatures(java.util.LinkedHashMap, org.biojava3.genome.parsers.gff.FeatureList)">addGeneIDGFF2GeneFeatures(LinkedHashMap&lt;String, ChromosomeSequence&gt;, FeatureList)</a></span> - Static method in class org.biojava3.genome.<a href="./org/biojava3/genome/GeneFeatureHelper.html" title="class in org.biojava3.genome">GeneFeatureHelper</a></dt>
<dd>
<div class="block">Load GFF2 feature file generated from the geneid prediction algorithm and map features onto the chromosome sequences</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/GeneFeatureHelper.html#addGeneMarkGTFGeneFeatures(java.util.LinkedHashMap, org.biojava3.genome.parsers.gff.FeatureList)">addGeneMarkGTFGeneFeatures(LinkedHashMap&lt;String, ChromosomeSequence&gt;, FeatureList)</a></span> - Static method in class org.biojava3.genome.<a href="./org/biojava3/genome/GeneFeatureHelper.html" title="class in org.biojava3.genome">GeneFeatureHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/GeneFeatureHelper.html#addGlimmerGFF3GeneFeatures(java.util.LinkedHashMap, org.biojava3.genome.parsers.gff.FeatureList)">addGlimmerGFF3GeneFeatures(LinkedHashMap&lt;String, ChromosomeSequence&gt;, FeatureList)</a></span> - Static method in class org.biojava3.genome.<a href="./org/biojava3/genome/GeneFeatureHelper.html" title="class in org.biojava3.genome">GeneFeatureHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/GeneFeatureHelper.html#addGmodGFF3GeneFeatures(java.util.LinkedHashMap, org.biojava3.genome.parsers.gff.FeatureList)">addGmodGFF3GeneFeatures(LinkedHashMap&lt;String, ChromosomeSequence&gt;, FeatureList)</a></span> - Static method in class org.biojava3.genome.<a href="./org/biojava3/genome/GeneFeatureHelper.html" title="class in org.biojava3.genome">GeneFeatureHelper</a></dt>
<dd>
<div class="block">Load GFF3 file using mRNA as the gene feature as not all GFF3 files are complete</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#addGroup(org.biojava.bio.structure.Group)">addGroup(Group)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">add a group to the list of ATOM record group of this chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#addGroup(org.biojava.bio.structure.Group)">addGroup(Group)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">add a group to the list of ATOM record group of this chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/GeneSequence.html#addIntron(org.biojava3.core.sequence.AccessionID, int, int)">addIntron(AccessionID, int, int)</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/GeneSequence.html" title="class in org.biojava3.core.sequence">GeneSequence</a></dt>
<dd>
<div class="block">Add an Intron Currently used to mark an IntronSequence as a feature</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/GeneSequence.html#addIntronsUsingExons()">addIntronsUsingExons()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/GeneSequence.html" title="class in org.biojava3.core.sequence">GeneSequence</a></dt>
<dd>
<div class="block">Once everything has been added to the gene sequence where you might have added exon sequences only then you
 can infer the intron sequences and add them.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationImpl.Builder.html#addKeyword(java.lang.String)">addKeyword(String)</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationImpl.Builder.html" title="class in org.biojava3.protmod">ProteinModificationImpl.Builder</a></dt>
<dd>
<div class="block">Add a keyword associate with the PTM.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationImpl.Builder.html#addKeywords(java.util.Collection)">addKeywords(Collection&lt;String&gt;)</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationImpl.Builder.html" title="class in org.biojava3.protmod">ProteinModificationImpl.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#addMatrixListener(org.biojava.bio.structure.align.ce.MatrixListener)">addMatrixListener(MatrixListener)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifParser.html#addMMcifConsumer(org.biojava.bio.structure.io.mmcif.MMcifConsumer)">addMMcifConsumer(MMcifConsumer)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifParser.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifParser</a></dt>
<dd>
<div class="block">Add a MMcifConsumer that listens to even being triggered by the parser and processes the data into a backend provided by the Consumer.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#addMMcifConsumer(org.biojava.bio.structure.io.mmcif.MMcifConsumer)">addMMcifConsumer(MMcifConsumer)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#addModel(java.util.List)">addModel(List&lt;Chain&gt;)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">add a new model.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#addModel(java.util.List)">addModel(List&lt;Chain&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">add a new model.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#addNote(java.lang.String)">addNote(String)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>
<div class="block">Add notes about this sequence that will get exported for GFF3</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractNucleotideCompoundSet.html#addNucleotideCompound(java.lang.String, java.lang.String, java.lang.String...)">addNucleotideCompound(String, String, String...)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractNucleotideCompoundSet.html" title="class in org.biojava3.core.sequence.template">AbstractNucleotideCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/SystemInfo.html#addOneSystemProperty(java.lang.String)">addOneSystemProperty(String)</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/SystemInfo.html" title="class in org.biojava.bio.structure.align.gui">SystemInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/FlatFileInstallation.html#addPDBFilter(org.biojava.bio.structure.server.PDBFilter)">addPDBFilter(PDBFilter)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/FlatFileInstallation.html" title="class in org.biojava.bio.structure.server">FlatFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html#addPDBFilter(org.biojava.bio.structure.server.PDBFilter)">addPDBFilter(PDBFilter)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html" title="class in org.biojava.bio.structure.server">MMCIFFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/PDBInstallation.html#addPDBFilter(org.biojava.bio.structure.server.PDBFilter)">addPDBFilter(PDBFilter)</a></span> - Method in interface org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/PDBInstallation.html" title="interface in org.biojava.bio.structure.server">PDBInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
<div class="block">add a filter for PDB files.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/TreeConstructor.html#addProgessListener(org.biojava3.phylo.NJTreeProgressListener)">addProgessListener(NJTreeProgressListener)</a></span> - Method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/TreeConstructor.html" title="class in org.biojava3.phylo">TreeConstructor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructurePairAligner.html#addProgressListener(org.biojava.bio.structure.align.pairwise.AlignmentProgressListener)">addProgressListener(AlignmentProgressListener)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructurePairAligner.html" title="class in org.biojava.bio.structure.align">StructurePairAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/DBReferenceInfo.html#addProperty(java.lang.String, java.lang.String)">addProperty(String, String)</a></span> - Method in class org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/DBReferenceInfo.html" title="class in org.biojava3.core.sequence.features">DBReferenceInfo</a></dt>
<dd>
<div class="block">Add a property and type to associate with this DBReferenceInfo</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/QuantityFeature.html#addQuantity(java.lang.Number)">addQuantity(Number)</a></span> - Method in class org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/QuantityFeature.html" title="class in org.biojava3.core.sequence.features">QuantityFeature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#addSSBond(org.biojava.bio.structure.SSBond)">addSSBond(SSBond)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">add a single SSBond to this structure</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#addSSBond(org.biojava.bio.structure.SSBond)">addSSBond(SSBond)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">add a single SSBond to this structure</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/TranscriptSequence.html#addStartCodonSequence(org.biojava3.core.sequence.AccessionID, int, int)">addStartCodonSequence(AccessionID, int, int)</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/TranscriptSequence.html" title="class in org.biojava3.core.sequence">TranscriptSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/TranscriptSequence.html#addStopCodonSequence(org.biojava3.core.sequence.AccessionID, int, int)">addStopCodonSequence(AccessionID, int, int)</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/TranscriptSequence.html" title="class in org.biojava3.core.sequence">TranscriptSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompoundTranslator.html#addStrings(java.lang.String, java.lang.String...)">addStrings(String, String...)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompoundTranslator.html" title="class in org.biojava3.core.sequence.template">AbstractCompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/SimpleStructureServer.html#addStructureListener(org.biojava.bio.structure.server.StructureListener)">addStructureListener(StructureListener)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/SimpleStructureServer.html" title="class in org.biojava.bio.structure.server">SimpleStructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureFetcherRunnable.html#addStructureListener(org.biojava.bio.structure.server.StructureListener)">addStructureListener(StructureListener)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureFetcherRunnable.html" title="class in org.biojava.bio.structure.server">StructureFetcherRunnable</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureServer.html#addStructureListener(org.biojava.bio.structure.server.StructureListener)">addStructureListener(StructureListener)</a></span> - Method in interface org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureServer.html" title="interface in org.biojava.bio.structure.server">StructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/FlatFileCache.html#addToCache(java.lang.String, java.io.File)">addToCache(String, File)</a></span> - Static method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/FlatFileCache.html" title="class in org.biojava3.core.util">FlatFileCache</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/GeneSequence.html#addTranscript(org.biojava3.core.sequence.AccessionID, int, int)">addTranscript(AccessionID, int, int)</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/GeneSequence.html" title="class in org.biojava3.core.sequence">GeneSequence</a></dt>
<dd>
<div class="block">Add a transcription sequence to a gene which describes a ProteinSequence</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/model/AFP.html" title="class in org.biojava.bio.structure.align.model"><span class="strong">AFP</span></a> - Class in <a href="./org/biojava/bio/structure/align/model/package-summary.html">org.biojava.bio.structure.align.model</a></dt>
<dd>
<div class="block">A class to represent a FATCAT AFP</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFP.html#AFP()">AFP()</a></span> - Constructor for class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFP.html" title="class in org.biojava.bio.structure.align.model">AFP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/AFPTwister.html#afp2Res(org.biojava.bio.structure.align.model.AFPChain, int, int[], int)">afp2Res(AFPChain, int, int[], int)</a></span> - Static method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/AFPTwister.html" title="class in org.biojava.bio.structure.align">AFPTwister</a></dt>
<dd>
<div class="block">Set the list of equivalent residues in the two proteins given a list of AFPs    
 
 WARNING: changes the values for FocusRes1, focusRes2 and FocusResn in afpChain!</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html" title="class in org.biojava.bio.structure.align.util"><span class="strong">AFPAlignmentDisplay</span></a> - Class in <a href="./org/biojava/bio/structure/align/util/package-summary.html">org.biojava.bio.structure.align.util</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html#AFPAlignmentDisplay()">AFPAlignmentDisplay()</a></span> - Constructor for class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html" title="class in org.biojava.bio.structure.align.util">AFPAlignmentDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/fatcat/calc/AFPCalculator.html" title="class in org.biojava.bio.structure.align.fatcat.calc"><span class="strong">AFPCalculator</span></a> - Class in <a href="./org/biojava/bio/structure/align/fatcat/calc/package-summary.html">org.biojava.bio.structure.align.fatcat.calc</a></dt>
<dd>
<div class="block">a class that performs calculations on AFPCHains</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/AFPCalculator.html#AFPCalculator()">AFPCalculator()</a></span> - Constructor for class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/AFPCalculator.html" title="class in org.biojava.bio.structure.align.fatcat.calc">AFPCalculator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model"><span class="strong">AFPChain</span></a> - Class in <a href="./org/biojava/bio/structure/align/model/package-summary.html">org.biojava.bio.structure.align.model</a></dt>
<dd>
<div class="block">a bean to contain the core of an alignment
 
 The FatCat aligner class is working on the AFPChain class.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#AFPChain()">AFPChain()</a></span> - Constructor for class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#AFPChain(org.biojava.bio.structure.align.model.AFPChain)">AFPChain(AFPChain)</a></span> - Constructor for class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">Copy constructor</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/gui/aligpanel/AFPChainCoordManager.html" title="class in org.biojava.bio.structure.align.gui.aligpanel"><span class="strong">AFPChainCoordManager</span></a> - Class in <a href="./org/biojava/bio/structure/align/gui/aligpanel/package-summary.html">org.biojava.bio.structure.align.gui.aligpanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AFPChainCoordManager.html#AFPChainCoordManager()">AFPChainCoordManager()</a></span> - Constructor for class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AFPChainCoordManager.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AFPChainCoordManager</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/fatcat/calc/AFPChainer.html" title="class in org.biojava.bio.structure.align.fatcat.calc"><span class="strong">AFPChainer</span></a> - Class in <a href="./org/biojava/bio/structure/align/fatcat/calc/package-summary.html">org.biojava.bio.structure.align.fatcat.calc</a></dt>
<dd>
<div class="block">a class to chain AFPs to an alignment</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/AFPChainer.html#AFPChainer()">AFPChainer()</a></span> - Constructor for class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/AFPChainer.html" title="class in org.biojava.bio.structure.align.fatcat.calc">AFPChainer</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/xml/AFPChainFlipper.html" title="class in org.biojava.bio.structure.align.xml"><span class="strong">AFPChainFlipper</span></a> - Class in <a href="./org/biojava/bio/structure/align/xml/package-summary.html">org.biojava.bio.structure.align.xml</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/AFPChainFlipper.html#AFPChainFlipper()">AFPChainFlipper()</a></span> - Constructor for class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/AFPChainFlipper.html" title="class in org.biojava.bio.structure.align.xml">AFPChainFlipper</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/util/AFPChainScorer.html" title="class in org.biojava.bio.structure.align.util"><span class="strong">AFPChainScorer</span></a> - Class in <a href="./org/biojava/bio/structure/align/util/package-summary.html">org.biojava.bio.structure.align.util</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AFPChainScorer.html#AFPChainScorer()">AFPChainScorer()</a></span> - Constructor for class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AFPChainScorer.html" title="class in org.biojava.bio.structure.align.util">AFPChainScorer</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html" title="class in org.biojava.bio.structure.align.model"><span class="strong">AfpChainWriter</span></a> - Class in <a href="./org/biojava/bio/structure/align/model/package-summary.html">org.biojava.bio.structure.align.model</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html#AfpChainWriter()">AfpChainWriter()</a></span> - Constructor for class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html" title="class in org.biojava.bio.structure.align.model">AfpChainWriter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/xml/AFPChainXMLConverter.html" title="class in org.biojava.bio.structure.align.xml"><span class="strong">AFPChainXMLConverter</span></a> - Class in <a href="./org/biojava/bio/structure/align/xml/package-summary.html">org.biojava.bio.structure.align.xml</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/AFPChainXMLConverter.html#AFPChainXMLConverter()">AFPChainXMLConverter()</a></span> - Constructor for class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/AFPChainXMLConverter.html" title="class in org.biojava.bio.structure.align.xml">AFPChainXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/xml/AFPChainXMLParser.html" title="class in org.biojava.bio.structure.align.xml"><span class="strong">AFPChainXMLParser</span></a> - Class in <a href="./org/biojava/bio/structure/align/xml/package-summary.html">org.biojava.bio.structure.align.xml</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/AFPChainXMLParser.html#AFPChainXMLParser()">AFPChainXMLParser()</a></span> - Constructor for class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/AFPChainXMLParser.html" title="class in org.biojava.bio.structure.align.xml">AFPChainXMLParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/fatcat/calc/AFPOptimizer.html" title="class in org.biojava.bio.structure.align.fatcat.calc"><span class="strong">AFPOptimizer</span></a> - Class in <a href="./org/biojava/bio/structure/align/fatcat/calc/package-summary.html">org.biojava.bio.structure.align.fatcat.calc</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/AFPOptimizer.html#AFPOptimizer()">AFPOptimizer()</a></span> - Constructor for class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/AFPOptimizer.html" title="class in org.biojava.bio.structure.align.fatcat.calc">AFPOptimizer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/AFPChainer.html#afpPairConn(int, int, org.biojava.bio.structure.align.fatcat.calc.FatCatParameters, org.biojava.bio.structure.align.model.AFPChain)">afpPairConn(int, int, FatCatParameters, AFPChain)</a></span> - Static method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/AFPChainer.html" title="class in org.biojava.bio.structure.align.fatcat.calc">AFPChainer</a></dt>
<dd>
<div class="block">//Key function: calculate the connectivity of AFP pairs
  //no compatibility criteria is executed
  //note: afp1 is previous to afp2 in terms of the position
     //this module must be optimized</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/fatcat/calc/AFPPostProcessor.html" title="class in org.biojava.bio.structure.align.fatcat.calc"><span class="strong">AFPPostProcessor</span></a> - Class in <a href="./org/biojava/bio/structure/align/fatcat/calc/package-summary.html">org.biojava.bio.structure.align.fatcat.calc</a></dt>
<dd>
<div class="block">does post processing after alignment chaingin</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/AFPPostProcessor.html#AFPPostProcessor()">AFPPostProcessor()</a></span> - Constructor for class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/AFPPostProcessor.html" title="class in org.biojava.bio.structure.align.fatcat.calc">AFPPostProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/AFPTwister.html" title="class in org.biojava.bio.structure.align"><span class="strong">AFPTwister</span></a> - Class in <a href="./org/biojava/bio/structure/align/package-summary.html">org.biojava.bio.structure.align</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/AFPTwister.html#AFPTwister()">AFPTwister()</a></span> - Constructor for class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/AFPTwister.html" title="class in org.biojava.bio.structure.align">AFPTwister</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#Ala">Ala</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/BioJavaStructureAlignment.html#algorithmName">algorithmName</a></span> - Static variable in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/BioJavaStructureAlignment.html" title="class in org.biojava.bio.structure.align">BioJavaStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCPMain.html#algorithmName">algorithmName</a></span> - Static variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCPMain.html" title="class in org.biojava.bio.structure.align.ce">CeCPMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeMain.html#algorithmName">algorithmName</a></span> - Static variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeMain.html" title="class in org.biojava.bio.structure.align.ce">CeMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeSideChainMain.html#algorithmName">algorithmName</a></span> - Static variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeSideChainMain.html" title="class in org.biojava.bio.structure.align.ce">CeSideChainMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCatFlexible.html#algorithmName">algorithmName</a></span> - Static variable in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCatFlexible.html" title="class in org.biojava.bio.structure.align.fatcat">FatCatFlexible</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCatRigid.html#algorithmName">algorithmName</a></span> - Static variable in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCatRigid.html" title="class in org.biojava.bio.structure.align.fatcat">FatCatRigid</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWaterman3Daligner.html#algorithmName">algorithmName</a></span> - Static variable in class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWaterman3Daligner.html" title="class in org.biojava.bio.structure.align.seq">SmithWaterman3Daligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/Gotoh.html#ALIGFACTOR">ALIGFACTOR</a></span> - Static variable in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/Gotoh.html" title="class in org.biojava.bio.structure.align.pairwise">Gotoh</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/helper/AligMatEl.html" title="class in org.biojava.bio.structure.align.helper"><span class="strong">AligMatEl</span></a> - Class in <a href="./org/biojava/bio/structure/align/helper/package-summary.html">org.biojava.bio.structure.align.helper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/AligMatEl.html#AligMatEl()">AligMatEl()</a></span> - Constructor for class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/AligMatEl.html" title="class in org.biojava.bio.structure.align.helper">AligMatEl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/AbstractStructureAlignment.html#align(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">align(Atom[], Atom[])</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/AbstractStructureAlignment.html" title="class in org.biojava.bio.structure.align">AbstractStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/AbstractStructureAlignment.html#align(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], java.lang.Object)">align(Atom[], Atom[], Object)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/AbstractStructureAlignment.html" title="class in org.biojava.bio.structure.align">AbstractStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/BioJavaStructureAlignment.html#align(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">align(Atom[], Atom[])</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/BioJavaStructureAlignment.html" title="class in org.biojava.bio.structure.align">BioJavaStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/BioJavaStructureAlignment.html#align(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], java.lang.Object)">align(Atom[], Atom[], Object)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/BioJavaStructureAlignment.html" title="class in org.biojava.bio.structure.align">BioJavaStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCPMain.html#align(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], java.lang.Object)">align(Atom[], Atom[], Object)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCPMain.html" title="class in org.biojava.bio.structure.align.ce">CeCPMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeMain.html#align(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], java.lang.Object)">align(Atom[], Atom[], Object)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeMain.html" title="class in org.biojava.bio.structure.align.ce">CeMain</a></dt>
<dd>
<div class="block">Align ca2 onto ca1.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeMain.html#align(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">align(Atom[], Atom[])</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeMain.html" title="class in org.biojava.bio.structure.align.ce">CeMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatAligner.html#align(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], boolean, org.biojava.bio.structure.align.fatcat.calc.FatCatParameters)">align(Atom[], Atom[], boolean, FatCatParameters)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatAligner.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCat.html#align(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], org.biojava.bio.structure.align.fatcat.calc.FatCatParameters, boolean)">align(Atom[], Atom[], FatCatParameters, boolean)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCat.html" title="class in org.biojava.bio.structure.align.fatcat">FatCat</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCatFlexible.html#align(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">align(Atom[], Atom[])</a></span> - Method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCatFlexible.html" title="class in org.biojava.bio.structure.align.fatcat">FatCatFlexible</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCatFlexible.html#align(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], java.lang.Object)">align(Atom[], Atom[], Object)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCatFlexible.html" title="class in org.biojava.bio.structure.align.fatcat">FatCatFlexible</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCatRigid.html#align(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">align(Atom[], Atom[])</a></span> - Method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCatRigid.html" title="class in org.biojava.bio.structure.align.fatcat">FatCatRigid</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCatRigid.html#align(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], java.lang.Object)">align(Atom[], Atom[], Object)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCatRigid.html" title="class in org.biojava.bio.structure.align.fatcat">FatCatRigid</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWaterman3Daligner.html#align(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">align(Atom[], Atom[])</a></span> - Method in class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWaterman3Daligner.html" title="class in org.biojava.bio.structure.align.seq">SmithWaterman3Daligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWaterman3Daligner.html#align(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], java.lang.Object)">align(Atom[], Atom[], Object)</a></span> - Method in class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWaterman3Daligner.html" title="class in org.biojava.bio.structure.align.seq">SmithWaterman3Daligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructureAlignment.html#align(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">align(Atom[], Atom[])</a></span> - Method in interface org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructureAlignment.html" title="interface in org.biojava.bio.structure.align">StructureAlignment</a></dt>
<dd>
<div class="block">Run an alignment while specifying the atoms to be aligned.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructureAlignment.html#align(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], java.lang.Object)">align(Atom[], Atom[], Object)</a></span> - Method in interface org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructureAlignment.html" title="interface in org.biojava.bio.structure.align">StructureAlignment</a></dt>
<dd>
<div class="block">run an alignment and also send a bean containing the parameters.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructurePairAligner.html#align(org.biojava.bio.structure.Structure, org.biojava.bio.structure.Structure)">align(Structure, Structure)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructurePairAligner.html" title="class in org.biojava.bio.structure.align">StructurePairAligner</a></dt>
<dd>
<div class="block">Calculate the alignment between the two full structures with default parameters</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructurePairAligner.html#align(org.biojava.bio.structure.Structure, org.biojava.bio.structure.Structure, org.biojava.bio.structure.align.StrucAligParameters)">align(Structure, Structure, StrucAligParameters)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructurePairAligner.html" title="class in org.biojava.bio.structure.align">StructurePairAligner</a></dt>
<dd>
<div class="block">Calculate the alignment between the two full structures with user provided parameters</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructurePairAligner.html#align(org.biojava.bio.structure.Structure, java.lang.String, org.biojava.bio.structure.Structure, java.lang.String)">align(Structure, String, Structure, String)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructurePairAligner.html" title="class in org.biojava.bio.structure.align">StructurePairAligner</a></dt>
<dd>
<div class="block">Align two chains from the structures.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructurePairAligner.html#align(org.biojava.bio.structure.Structure, java.lang.String, org.biojava.bio.structure.Structure, java.lang.String, org.biojava.bio.structure.align.StrucAligParameters)">align(Structure, String, Structure, String, StrucAligParameters)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructurePairAligner.html" title="class in org.biojava.bio.structure.align">StructurePairAligner</a></dt>
<dd>
<div class="block">Aligns two chains from the structures using user provided parameters.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructurePairAligner.html#align(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], org.biojava.bio.structure.align.StrucAligParameters)">align(Atom[], Atom[], StrucAligParameters)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructurePairAligner.html" title="class in org.biojava.bio.structure.align">StructurePairAligner</a></dt>
<dd>
<div class="block">calculate the  protein structure superimposition, between two sets of atoms.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/SeqRes2AtomAligner.html#align(org.biojava.bio.structure.Structure, java.util.List)">align(Structure, List&lt;Chain&gt;)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/SeqRes2AtomAligner.html" title="class in org.biojava.bio.structure.io">SeqRes2AtomAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/SeqRes2AtomAligner.html#align(java.util.List, java.util.List)">align(List&lt;Group&gt;, List&lt;Group&gt;)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/SeqRes2AtomAligner.html" title="class in org.biojava.bio.structure.io">SeqRes2AtomAligner</a></dt>
<dd>
<div class="block">aligns two chains of groups, where the first parent is representing the
 list of amino acids as obtained from the SEQRES records, and the second parent
 represents the groups obtained from the ATOM records (and containing the actual ATOM information).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#align()">align()</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava3.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AligNPE.html#align_NPE(org.biojava.bio.structure.jama.Matrix, org.biojava.bio.structure.align.StrucAligParameters)">align_NPE(Matrix, StrucAligParameters)</a></span> - Static method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AligNPE.html" title="class in org.biojava.bio.structure.align.pairwise">AligNPE</a></dt>
<dd>
<div class="block">Align w/o penalizing endpags.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/pairwise/Alignable.html" title="interface in org.biojava.bio.structure.align.pairwise"><span class="strong">Alignable</span></a> - Interface in <a href="./org/biojava/bio/structure/align/pairwise/package-summary.html">org.biojava.bio.structure.align.pairwise</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/gui/util/AlignedPosition.html" title="class in org.biojava.bio.structure.gui.util"><span class="strong">AlignedPosition</span></a> - Class in <a href="./org/biojava/bio/structure/gui/util/package-summary.html">org.biojava.bio.structure.gui.util</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/AlignedPosition.html#AlignedPosition()">AlignedPosition()</a></span> - Constructor for class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/AlignedPosition.html" title="class in org.biojava.bio.structure.gui.util">AlignedPosition</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/alignment/template/AlignedSequence.html" title="interface in org.biojava3.alignment.template"><span class="strong">AlignedSequence</span></a>&lt;<a href="./org/biojava3/alignment/template/AlignedSequence.html" title="type parameter in AlignedSequence">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava3/alignment/template/AlignedSequence.html" title="type parameter in AlignedSequence">C</a>&gt;,<a href="./org/biojava3/alignment/template/AlignedSequence.html" title="type parameter in AlignedSequence">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava3/alignment/template/package-summary.html">org.biojava3.alignment.template</a></dt>
<dd>
<div class="block">Defines a data structure for a <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a> within an alignment.</div>
</dd>
<dt><a href="./org/biojava3/alignment/template/AlignedSequence.Step.html" title="enum in org.biojava3.alignment.template"><span class="strong">AlignedSequence.Step</span></a> - Enum in <a href="./org/biojava3/alignment/template/package-summary.html">org.biojava3.alignment.template</a></dt>
<dd>
<div class="block">Defines an alignment step in order to pass alignment information from an <a href="./org/biojava3/alignment/template/Aligner.html" title="interface in org.biojava3.alignment.template"><code>Aligner</code></a> to a constructor.</div>
</dd>
<dt><a href="./org/biojava3/alignment/template/Aligner.html" title="interface in org.biojava3.alignment.template"><span class="strong">Aligner</span></a>&lt;<a href="./org/biojava3/alignment/template/Aligner.html" title="type parameter in Aligner">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava3/alignment/template/Aligner.html" title="type parameter in Aligner">C</a>&gt;,<a href="./org/biojava3/alignment/template/Aligner.html" title="type parameter in Aligner">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava3/alignment/template/package-summary.html">org.biojava3.alignment.template</a></dt>
<dd>
<div class="block">Defines an algorithm which computes an alignment <a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template"><code>Profile</code></a> from a list of <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a>s.</div>
</dd>
<dt><a href="./org/biojava3/alignment/routines/AlignerHelper.html" title="class in org.biojava3.alignment.routines"><span class="strong">AlignerHelper</span></a> - Class in <a href="./org/biojava3/alignment/routines/package-summary.html">org.biojava3.alignment.routines</a></dt>
<dd>
<div class="block">Static utility to construct alignment routines from a common library of methods.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/routines/AlignerHelper.html#AlignerHelper()">AlignerHelper()</a></span> - Constructor for class org.biojava3.alignment.routines.<a href="./org/biojava3/alignment/routines/AlignerHelper.html" title="class in org.biojava3.alignment.routines">AlignerHelper</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/alignment/routines/AlignerHelper.Cut.html" title="class in org.biojava3.alignment.routines"><span class="strong">AlignerHelper.Cut</span></a> - Class in <a href="./org/biojava3/alignment/routines/package-summary.html">org.biojava3.alignment.routines</a></dt>
<dd>
<div class="block">Defines a 'cut' row for divide-and-conquer alignment in which a new anchor is found.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/routines/AlignerHelper.Cut.html#AlignerHelper.Cut(int, int[])">AlignerHelper.Cut(int, int[])</a></span> - Constructor for class org.biojava3.alignment.routines.<a href="./org/biojava3/alignment/routines/AlignerHelper.Cut.html" title="class in org.biojava3.alignment.routines">AlignerHelper.Cut</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/alignment/routines/AlignerHelper.Last.html" title="enum in org.biojava3.alignment.routines"><span class="strong">AlignerHelper.Last</span></a> - Enum in <a href="./org/biojava3/alignment/routines/package-summary.html">org.biojava3.alignment.routines</a></dt>
<dd>
<div class="block">Defines a traceback pointer for the three edit operations: substitution (match/replacement of a query compound
 with a target compound), deletion (removal of a query compound leaving a gap in the target sequence), and
 insertion (addition of a target compound opening a gap in the query sequence).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCat.html#alignFlexible(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], org.biojava.bio.structure.align.fatcat.calc.FatCatParameters)">alignFlexible(Atom[], Atom[], FatCatParameters)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCat.html" title="class in org.biojava.bio.structure.align.fatcat">FatCat</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/MenuCreator.html#ALIGNMENT_PANEL">ALIGNMENT_PANEL</a></span> - Static variable in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/MenuCreator.html" title="class in org.biojava.bio.structure.align.gui">MenuCreator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/gui/AlignmentCalc.html" title="class in org.biojava.bio.structure.align.gui"><span class="strong">AlignmentCalc</span></a> - Class in <a href="./org/biojava/bio/structure/align/gui/package-summary.html">org.biojava.bio.structure.align.gui</a></dt>
<dd>
<div class="block">A class that obtains two structures via DAS and aligns them
  This is done in a separate thread.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/AlignmentCalc.html#AlignmentCalc(org.biojava.bio.structure.align.gui.AlignmentGui, org.biojava.bio.structure.Structure, org.biojava.bio.structure.Structure, java.lang.String, java.lang.String)">AlignmentCalc(AlignmentGui, Structure, Structure, String, String)</a></span> - Constructor for class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/AlignmentCalc.html" title="class in org.biojava.bio.structure.align.gui">AlignmentCalc</a></dt>
<dd>
<div class="block">requests an alignment of pdb1 vs pdb 2.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/gui/util/AlignmentCalc.html" title="class in org.biojava.bio.structure.gui.util"><span class="strong">AlignmentCalc</span></a> - Class in <a href="./org/biojava/bio/structure/gui/util/package-summary.html">org.biojava.bio.structure.gui.util</a></dt>
<dd>
<div class="block">A class that obtains two structures via DAS and aligns them
  This is done in a separate thread.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/AlignmentCalc.html#AlignmentCalc(org.biojava.bio.structure.align.gui.AlignmentGui, org.biojava.bio.structure.Structure, org.biojava.bio.structure.Structure)">AlignmentCalc(AlignmentGui, Structure, Structure)</a></span> - Constructor for class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/AlignmentCalc.html" title="class in org.biojava.bio.structure.gui.util">AlignmentCalc</a></dt>
<dd>
<div class="block">requests an alignment of pdb1 vs pdb 2.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/gui/AlignmentCalcDB.html" title="class in org.biojava.bio.structure.align.gui"><span class="strong">AlignmentCalcDB</span></a> - Class in <a href="./org/biojava/bio/structure/align/gui/package-summary.html">org.biojava.bio.structure.align.gui</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/AlignmentCalcDB.html#AlignmentCalcDB(org.biojava.bio.structure.align.gui.AlignmentGui, org.biojava.bio.structure.Structure, java.lang.String, org.biojava.bio.structure.align.util.UserConfiguration, java.lang.String)">AlignmentCalcDB(AlignmentGui, Structure, String, UserConfiguration, String)</a></span> - Constructor for class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/AlignmentCalcDB.html" title="class in org.biojava.bio.structure.align.gui">AlignmentCalcDB</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/gui/AlignmentCalculationRunnable.html" title="interface in org.biojava.bio.structure.align.gui"><span class="strong">AlignmentCalculationRunnable</span></a> - Interface in <a href="./org/biojava/bio/structure/align/gui/package-summary.html">org.biojava.bio.structure.align.gui</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/CountProgressListener.html#alignmentEnded()">alignmentEnded()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/CountProgressListener.html" title="class in org.biojava.bio.structure.align.client">CountProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/events/AlignmentProgressListener.html#alignmentEnded()">alignmentEnded()</a></span> - Method in interface org.biojava.bio.structure.align.events.<a href="./org/biojava/bio/structure/align/events/AlignmentProgressListener.html" title="interface in org.biojava.bio.structure.align.events">AlignmentProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/GUIAlignmentProgressListener.html#alignmentEnded()">alignmentEnded()</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/GUIAlignmentProgressListener.html" title="class in org.biojava.bio.structure.align.gui">GUIAlignmentProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/gui/AlignmentGui.html" title="class in org.biojava.bio.structure.align.gui"><span class="strong">AlignmentGui</span></a> - Class in <a href="./org/biojava/bio/structure/align/gui/package-summary.html">org.biojava.bio.structure.align.gui</a></dt>
<dd>
<div class="block">A JFrame that allows to trigger a pairwise structure alignment,
 either from files in a directory,
 or after manual upload.</div>
</dd>
<dt><a href="./demo/AlignmentGuiDemo.html" title="class in demo"><span class="strong">AlignmentGuiDemo</span></a> - Class in <a href="./demo/package-summary.html">demo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/AlignmentGuiDemo.html#AlignmentGuiDemo()">AlignmentGuiDemo()</a></span> - Constructor for class demo.<a href="./demo/AlignmentGuiDemo.html" title="class in demo">AlignmentGuiDemo</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/gui/events/AlignmentPositionListener.html" title="interface in org.biojava.bio.structure.gui.events"><span class="strong">AlignmentPositionListener</span></a> - Interface in <a href="./org/biojava/bio/structure/gui/events/package-summary.html">org.biojava.bio.structure.gui.events</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/events/AlignmentProgressListener.html" title="interface in org.biojava.bio.structure.align.events"><span class="strong">AlignmentProgressListener</span></a> - Interface in <a href="./org/biojava/bio/structure/align/events/package-summary.html">org.biojava.bio.structure.align.events</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/pairwise/AlignmentProgressListener.html" title="class in org.biojava.bio.structure.align.pairwise"><span class="strong">AlignmentProgressListener</span></a> - Class in <a href="./org/biojava/bio/structure/align/pairwise/package-summary.html">org.biojava.bio.structure.align.pairwise</a></dt>
<dd>
<div class="block">A class to listen to progress of the structure alignmnent calcualtions</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentProgressListener.html#AlignmentProgressListener()">AlignmentProgressListener()</a></span> - Constructor for class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentProgressListener.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise"><span class="strong">AlignmentResult</span></a> - Class in <a href="./org/biojava/bio/structure/align/pairwise/package-summary.html">org.biojava.bio.structure.align.pairwise</a></dt>
<dd>
<div class="block">A class to track the alignment results in a flat file</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html#AlignmentResult()">AlignmentResult()</a></span> - Constructor for class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentResult</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/alignment/Alignments.html" title="class in org.biojava3.alignment"><span class="strong">Alignments</span></a> - Class in <a href="./org/biojava3/alignment/package-summary.html">org.biojava3.alignment</a></dt>
<dd>
<div class="block">Static utility to easily run alignment routines.</div>
</dd>
<dt><a href="./org/biojava3/alignment/Alignments.PairInProfileScorerType.html" title="enum in org.biojava3.alignment"><span class="strong">Alignments.PairInProfileScorerType</span></a> - Enum in <a href="./org/biojava3/alignment/package-summary.html">org.biojava3.alignment</a></dt>
<dd>
<div class="block">List of implemented sequence pair in a profile scoring routines.</div>
</dd>
<dt><a href="./org/biojava3/alignment/Alignments.PairwiseSequenceAlignerType.html" title="enum in org.biojava3.alignment"><span class="strong">Alignments.PairwiseSequenceAlignerType</span></a> - Enum in <a href="./org/biojava3/alignment/package-summary.html">org.biojava3.alignment</a></dt>
<dd>
<div class="block">List of implemented pairwise sequence alignment routines.</div>
</dd>
<dt><a href="./org/biojava3/alignment/Alignments.PairwiseSequenceScorerType.html" title="enum in org.biojava3.alignment"><span class="strong">Alignments.PairwiseSequenceScorerType</span></a> - Enum in <a href="./org/biojava3/alignment/package-summary.html">org.biojava3.alignment</a></dt>
<dd>
<div class="block">List of implemented pairwise sequence scoring routines.</div>
</dd>
<dt><a href="./org/biojava3/alignment/Alignments.ProfileProfileAlignerType.html" title="enum in org.biojava3.alignment"><span class="strong">Alignments.ProfileProfileAlignerType</span></a> - Enum in <a href="./org/biojava3/alignment/package-summary.html">org.biojava3.alignment</a></dt>
<dd>
<div class="block">List of implemented profile-profile alignment routines.</div>
</dd>
<dt><a href="./org/biojava3/alignment/Alignments.RefinerType.html" title="enum in org.biojava3.alignment"><span class="strong">Alignments.RefinerType</span></a> - Enum in <a href="./org/biojava3/alignment/package-summary.html">org.biojava3.alignment</a></dt>
<dd>
<div class="block">List of implemented profile refinement routines.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/CountProgressListener.html#alignmentStarted(java.lang.String, java.lang.String)">alignmentStarted(String, String)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/CountProgressListener.html" title="class in org.biojava.bio.structure.align.client">CountProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/events/AlignmentProgressListener.html#alignmentStarted(java.lang.String, java.lang.String)">alignmentStarted(String, String)</a></span> - Method in interface org.biojava.bio.structure.align.events.<a href="./org/biojava/bio/structure/align/events/AlignmentProgressListener.html" title="interface in org.biojava.bio.structure.align.events">AlignmentProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/GUIAlignmentProgressListener.html#alignmentStarted(java.lang.String, java.lang.String)">alignmentStarted(String, String)</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/GUIAlignmentProgressListener.html" title="class in org.biojava.bio.structure.align.gui">GUIAlignmentProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/gui/AlignmentTextPanel.html" title="class in org.biojava.bio.structure.align.gui"><span class="strong">AlignmentTextPanel</span></a> - Class in <a href="./org/biojava/bio/structure/align/gui/package-summary.html">org.biojava.bio.structure.align.gui</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/AlignmentTextPanel.html#AlignmentTextPanel()">AlignmentTextPanel()</a></span> - Constructor for class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/AlignmentTextPanel.html" title="class in org.biojava.bio.structure.align.gui">AlignmentTextPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html#alignPair(java.lang.String, java.lang.String)">alignPair(String, String)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html" title="class in org.biojava.bio.structure.align.client">FarmJobRunnable</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/pairwise/AligNPE.html" title="class in org.biojava.bio.structure.align.pairwise"><span class="strong">AligNPE</span></a> - Class in <a href="./org/biojava/bio/structure/align/pairwise/package-summary.html">org.biojava.bio.structure.align.pairwise</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AligNPE.html#AligNPE()">AligNPE()</a></span> - Constructor for class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AligNPE.html" title="class in org.biojava.bio.structure.align.pairwise">AligNPE</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCat.html#alignRigid(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">alignRigid(Atom[], Atom[])</a></span> - Method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCat.html" title="class in org.biojava.bio.structure.align.fatcat">FatCat</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCat.html#alignRigid(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], org.biojava.bio.structure.align.fatcat.calc.FatCatParameters)">alignRigid(Atom[], Atom[], FatCatParameters)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCat.html" title="class in org.biojava.bio.structure.align.fatcat">FatCat</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoSCOP.html#alignSuperfamily()">alignSuperfamily()</a></span> - Method in class demo.<a href="./demo/DemoSCOP.html" title="class in demo">DemoSCOP</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/helper/AlignTools.html" title="class in org.biojava.bio.structure.align.helper"><span class="strong">AlignTools</span></a> - Class in <a href="./org/biojava/bio/structure/align/helper/package-summary.html">org.biojava.bio.structure.align.helper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/AlignTools.html#AlignTools()">AlignTools()</a></span> - Constructor for class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/AlignTools.html" title="class in org.biojava.bio.structure.align.helper">AlignTools</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html" title="class in org.biojava.bio.structure.align.gui.aligpanel"><span class="strong">AligPanel</span></a> - Class in <a href="./org/biojava/bio/structure/align/gui/aligpanel/package-summary.html">org.biojava.bio.structure.align.gui.aligpanel</a></dt>
<dd>
<div class="block">A JPanel that can display an AFPChain in a nice way and interact with Jmol.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html#AligPanel()">AligPanel()</a></span> - Constructor for class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AligPanel</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanelMouseMotionListener.html" title="class in org.biojava.bio.structure.align.gui.aligpanel"><span class="strong">AligPanelMouseMotionListener</span></a> - Class in <a href="./org/biojava/bio/structure/align/gui/aligpanel/package-summary.html">org.biojava.bio.structure.align.gui.aligpanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanelMouseMotionListener.html#AligPanelMouseMotionListener(org.biojava.bio.structure.align.gui.aligpanel.AligPanel)">AligPanelMouseMotionListener(AligPanel)</a></span> - Constructor for class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanelMouseMotionListener.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AligPanelMouseMotionListener</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/webstart/AligUIManager.html" title="class in org.biojava.bio.structure.align.webstart"><span class="strong">AligUIManager</span></a> - Class in <a href="./org/biojava/bio/structure/align/webstart/package-summary.html">org.biojava.bio.structure.align.webstart</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/webstart/AligUIManager.html#AligUIManager()">AligUIManager()</a></span> - Constructor for class org.biojava.bio.structure.align.webstart.<a href="./org/biojava/bio/structure/align/webstart/AligUIManager.html" title="class in org.biojava.bio.structure.align.webstart">AligUIManager</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#aliphatic">aliphatic</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/chem/PolymerType.html#ALL_POLYMER_TYPES">ALL_POLYMER_TYPES</a></span> - Static variable in enum org.biojava.bio.structure.io.mmcif.chem.<a href="./org/biojava/bio/structure/io/mmcif/chem/PolymerType.html" title="enum in org.biojava.bio.structure.io.mmcif.chem">PolymerType</a></dt>
<dd>
<div class="block">Convenience <tt>Set</tt> of all polymer types.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/AllChemCompProvider.html" title="class in org.biojava.bio.structure.io.mmcif"><span class="strong">AllChemCompProvider</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/package-summary.html">org.biojava.bio.structure.io.mmcif</a></dt>
<dd>
<div class="block">A ChemComp provider that downloads and caches the components.cif file from the wwPDB site.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/AllChemCompProvider.html#AllChemCompProvider()">AllChemCompProvider()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/AllChemCompProvider.html" title="class in org.biojava.bio.structure.io.mmcif">AllChemCompProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/Component.html#allComponents()">allComponents()</a></span> - Static method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/Component.html" title="class in org.biojava3.protmod">Component</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationRegistry.html#allComponents()">allComponents()</a></span> - Static method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationRegistry.html" title="class in org.biojava3.protmod">ProteinModificationRegistry</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationRegistry.html#allIds()">allIds()</a></span> - Static method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationRegistry.html" title="class in org.biojava3.protmod">ProteinModificationRegistry</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationRegistry.html#allKeywords()">allKeywords()</a></span> - Static method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationRegistry.html" title="class in org.biojava3.protmod">ProteinModificationRegistry</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationRegistry.html#allModifications()">allModifications()</a></span> - Static method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationRegistry.html" title="class in org.biojava3.protmod">ProteinModificationRegistry</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationRegistry.html#allPdbccIds()">allPdbccIds()</a></span> - Static method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationRegistry.html" title="class in org.biojava3.protmod">ProteinModificationRegistry</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationRegistry.html#allPsimodIds()">allPsimodIds()</a></span> - Static method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationRegistry.html" title="class in org.biojava3.protmod">ProteinModificationRegistry</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationRegistry.html#allResidIds()">allResidIds()</a></span> - Static method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationRegistry.html" title="class in org.biojava3.protmod">ProteinModificationRegistry</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/pairwise/AltAligComparator.html" title="class in org.biojava.bio.structure.align.pairwise"><span class="strong">AltAligComparator</span></a> - Class in <a href="./org/biojava/bio/structure/align/pairwise/package-summary.html">org.biojava.bio.structure.align.pairwise</a></dt>
<dd>
<div class="block">a comparator to sort AlternativeAlignments based on their number of equivalent residues
 and RMSD.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AltAligComparator.html#AltAligComparator()">AltAligComparator()</a></span> - Constructor for class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AltAligComparator.html" title="class in org.biojava.bio.structure.align.pairwise">AltAligComparator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise"><span class="strong">AlternativeAlignment</span></a> - Class in <a href="./org/biojava/bio/structure/align/pairwise/package-summary.html">org.biojava.bio.structure.align.pairwise</a></dt>
<dd>
<div class="block">Implements a class which handles one possible (alternative) solution.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#AlternativeAlignment()">AlternativeAlignment()</a></span> - Constructor for class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/gui/util/AlternativeAlignmentFrame.html" title="class in org.biojava.bio.structure.gui.util"><span class="strong">AlternativeAlignmentFrame</span></a> - Class in <a href="./org/biojava/bio/structure/gui/util/package-summary.html">org.biojava.bio.structure.gui.util</a></dt>
<dd>
<div class="block">a frame showing the alternative alignments, which are the result of a structure superimposition</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/AlternativeAlignmentFrame.html#AlternativeAlignmentFrame(org.biojava.bio.structure.Structure, org.biojava.bio.structure.Structure)">AlternativeAlignmentFrame(Structure, Structure)</a></span> - Constructor for class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/AlternativeAlignmentFrame.html" title="class in org.biojava.bio.structure.gui.util">AlternativeAlignmentFrame</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/compound/AmbiguityDNACompoundSet.html" title="class in org.biojava3.core.sequence.compound"><span class="strong">AmbiguityDNACompoundSet</span></a> - Class in <a href="./org/biojava3/core/sequence/compound/package-summary.html">org.biojava3.core.sequence.compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/AmbiguityDNACompoundSet.html#AmbiguityDNACompoundSet()">AmbiguityDNACompoundSet()</a></span> - Constructor for class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/AmbiguityDNACompoundSet.html" title="class in org.biojava3.core.sequence.compound">AmbiguityDNACompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/compound/AmbiguityRNACompoundSet.html" title="class in org.biojava3.core.sequence.compound"><span class="strong">AmbiguityRNACompoundSet</span></a> - Class in <a href="./org/biojava3/core/sequence/compound/package-summary.html">org.biojava3.core.sequence.compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/AmbiguityRNACompoundSet.html#AmbiguityRNACompoundSet()">AmbiguityRNACompoundSet()</a></span> - Constructor for class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/AmbiguityRNACompoundSet.html" title="class in org.biojava3.core.sequence.compound">AmbiguityRNACompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/data/sequence/SequenceUtil.html#AMBIGUOUS_AA">AMBIGUOUS_AA</a></span> - Static variable in class org.biojava3.data.sequence.<a href="./org/biojava3/data/sequence/SequenceUtil.html" title="class in org.biojava3.data.sequence">SequenceUtil</a></dt>
<dd>
<div class="block">Same as AA pattern but with two additional letters - XU</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/data/sequence/SequenceUtil.html#AMBIGUOUS_NUCLEOTIDE">AMBIGUOUS_NUCLEOTIDE</a></span> - Static variable in class org.biojava3.data.sequence.<a href="./org/biojava3/data/sequence/SequenceUtil.html" title="class in org.biojava3.data.sequence">SequenceUtil</a></dt>
<dd>
<div class="block">Ambiguous nucleotide</div>
</dd>
<dt><a href="./org/biojava/bio/structure/AminoAcid.html" title="interface in org.biojava.bio.structure"><span class="strong">AminoAcid</span></a> - Interface in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">
 A <a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure"><code>Group</code></a> that represents an AminoAcid.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/GroupType.html#AMINOACID">AMINOACID</a></span> - Static variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/GroupType.html" title="class in org.biojava.bio.structure">GroupType</a></dt>
<dd>
<div class="block">the type for amino acids</div>
</dd>
<dt><a href="./org/biojava3/aaproperties/xml/AminoAcidComposition.html" title="class in org.biojava3.aaproperties.xml"><span class="strong">AminoAcidComposition</span></a> - Class in <a href="./org/biojava3/aaproperties/xml/package-summary.html">org.biojava3.aaproperties.xml</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/AminoAcidComposition.html#AminoAcidComposition()">AminoAcidComposition()</a></span> - Constructor for class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/AminoAcidComposition.html" title="class in org.biojava3.aaproperties.xml">AminoAcidComposition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/AminoAcidComposition.html#AminoAcidComposition(java.lang.String, java.lang.String, java.lang.String, java.util.List, java.util.List)">AminoAcidComposition(String, String, String, List&lt;Name2Count&gt;, List&lt;Name2Count&gt;)</a></span> - Constructor for class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/AminoAcidComposition.html" title="class in org.biojava3.aaproperties.xml">AminoAcidComposition</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/aaproperties/xml/AminoAcidCompositionTable.html" title="class in org.biojava3.aaproperties.xml"><span class="strong">AminoAcidCompositionTable</span></a> - Class in <a href="./org/biojava3/aaproperties/xml/package-summary.html">org.biojava3.aaproperties.xml</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/AminoAcidCompositionTable.html#AminoAcidCompositionTable()">AminoAcidCompositionTable()</a></span> - Constructor for class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/AminoAcidCompositionTable.html" title="class in org.biojava3.aaproperties.xml">AminoAcidCompositionTable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/AminoAcidCompositionTable.html#AminoAcidCompositionTable(java.util.List)">AminoAcidCompositionTable(List&lt;AminoAcidComposition&gt;)</a></span> - Constructor for class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/AminoAcidCompositionTable.html" title="class in org.biojava3.aaproperties.xml">AminoAcidCompositionTable</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound"><span class="strong">AminoAcidCompound</span></a> - Class in <a href="./org/biojava3/core/sequence/compound/package-summary.html">org.biojava3.core.sequence.compound</a></dt>
<dd>
<div class="block">Used to describe an Amino Acid.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/AminoAcidCompound.html#AminoAcidCompound(org.biojava3.core.sequence.compound.AminoAcidCompoundSet, java.lang.String, java.lang.String, java.lang.String, java.lang.Float)">AminoAcidCompound(AminoAcidCompoundSet, String, String, String, Float)</a></span> - Constructor for class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/compound/AminoAcidCompoundSet.html" title="class in org.biojava3.core.sequence.compound"><span class="strong">AminoAcidCompoundSet</span></a> - Class in <a href="./org/biojava3/core/sequence/compound/package-summary.html">org.biojava3.core.sequence.compound</a></dt>
<dd>
<div class="block">Set of proteinogenic amino acids.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/AminoAcidCompoundSet.html#AminoAcidCompoundSet()">AminoAcidCompoundSet()</a></span> - Constructor for class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/AminoAcidCompoundSet.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/AminoAcidImpl.html" title="class in org.biojava.bio.structure"><span class="strong">AminoAcidImpl</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">AminoAcid inherits most from Hetatom.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcidImpl.html#AminoAcidImpl()">AminoAcidImpl()</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcidImpl.html" title="class in org.biojava.bio.structure">AminoAcidImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.Builder.html#aminoAcidsCompounds(org.biojava3.core.sequence.template.CompoundSet)">aminoAcidsCompounds(CompoundSet&lt;AminoAcidCompound&gt;)</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava3.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#amount(org.biojava.bio.structure.Atom)">amount(Atom)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">amount.</div>
</dd>
<dt><a href="./org/biojava3/alignment/routines/AnchoredPairwiseSequenceAligner.html" title="class in org.biojava3.alignment.routines"><span class="strong">AnchoredPairwiseSequenceAligner</span></a>&lt;<a href="./org/biojava3/alignment/routines/AnchoredPairwiseSequenceAligner.html" title="type parameter in AnchoredPairwiseSequenceAligner">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava3/alignment/routines/AnchoredPairwiseSequenceAligner.html" title="type parameter in AnchoredPairwiseSequenceAligner">C</a>&gt;,<a href="./org/biojava3/alignment/routines/AnchoredPairwiseSequenceAligner.html" title="type parameter in AnchoredPairwiseSequenceAligner">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/alignment/routines/package-summary.html">org.biojava3.alignment.routines</a></dt>
<dd>
<div class="block">This algorithm uses a divide-and-conquer approach to find optimal pairwise global sequence alignments (from the
 first until the last <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template"><code>Compound</code></a> of each <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a>) with the restriction that any alignment produced
 will connect the query sequence to the target sequence at the <em>anchors</em>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/routines/AnchoredPairwiseSequenceAligner.html#AnchoredPairwiseSequenceAligner()">AnchoredPairwiseSequenceAligner()</a></span> - Constructor for class org.biojava3.alignment.routines.<a href="./org/biojava3/alignment/routines/AnchoredPairwiseSequenceAligner.html" title="class in org.biojava3.alignment.routines">AnchoredPairwiseSequenceAligner</a></dt>
<dd>
<div class="block">Before running a pairwise global sequence alignment, data must be sent in via calls to
 <a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html#setQuery(S)"><code>AbstractPairwiseSequenceAligner.setQuery(Sequence)</code></a>, <a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html#setTarget(S)"><code>AbstractPairwiseSequenceAligner.setTarget(Sequence)</code></a>, <a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#setGapPenalty(org.biojava3.alignment.template.GapPenalty)"><code>AbstractMatrixAligner.setGapPenalty(GapPenalty)</code></a>, and
 <a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#setSubstitutionMatrix(org.biojava3.alignment.template.SubstitutionMatrix)"><code>AbstractMatrixAligner.setSubstitutionMatrix(SubstitutionMatrix)</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/routines/AnchoredPairwiseSequenceAligner.html#AnchoredPairwiseSequenceAligner(S, S, org.biojava3.alignment.template.GapPenalty, org.biojava3.alignment.template.SubstitutionMatrix)">AnchoredPairwiseSequenceAligner(S, S, GapPenalty, SubstitutionMatrix&lt;C&gt;)</a></span> - Constructor for class org.biojava3.alignment.routines.<a href="./org/biojava3/alignment/routines/AnchoredPairwiseSequenceAligner.html" title="class in org.biojava3.alignment.routines">AnchoredPairwiseSequenceAligner</a></dt>
<dd>
<div class="block">Prepares for a pairwise global sequence alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/routines/AnchoredPairwiseSequenceAligner.html#AnchoredPairwiseSequenceAligner(S, S, org.biojava3.alignment.template.GapPenalty, org.biojava3.alignment.template.SubstitutionMatrix, int)">AnchoredPairwiseSequenceAligner(S, S, GapPenalty, SubstitutionMatrix&lt;C&gt;, int)</a></span> - Constructor for class org.biojava3.alignment.routines.<a href="./org/biojava3/alignment/routines/AnchoredPairwiseSequenceAligner.html" title="class in org.biojava3.alignment.routines">AnchoredPairwiseSequenceAligner</a></dt>
<dd>
<div class="block">Prepares for a pairwise global sequence alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/routines/AnchoredPairwiseSequenceAligner.html#AnchoredPairwiseSequenceAligner(S, S, org.biojava3.alignment.template.GapPenalty, org.biojava3.alignment.template.SubstitutionMatrix, int, int[])">AnchoredPairwiseSequenceAligner(S, S, GapPenalty, SubstitutionMatrix&lt;C&gt;, int, int[])</a></span> - Constructor for class org.biojava3.alignment.routines.<a href="./org/biojava3/alignment/routines/AnchoredPairwiseSequenceAligner.html" title="class in org.biojava3.alignment.routines">AnchoredPairwiseSequenceAligner</a></dt>
<dd>
<div class="block">Prepares for a pairwise global sequence alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#anchors">anchors</a></span> - Variable in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava3.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#angle(org.biojava.bio.structure.Atom, org.biojava.bio.structure.Atom)">angle(Atom, Atom)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">angle.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentJoiner.html#angleCheckOk(org.biojava.bio.structure.align.pairwise.FragmentPair, org.biojava.bio.structure.align.pairwise.FragmentPair, float)">angleCheckOk(FragmentPair, FragmentPair, float)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentJoiner.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentJoiner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#apairs_from_idxlst(org.biojava.bio.structure.align.helper.JointFragments)">apairs_from_idxlst(JointFragments)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>
<div class="block">Set apairs according to a list of (i,j) tuples.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#apairs_from_seed(int, int, int)">apairs_from_seed(int, int, int)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>
<div class="block">Set apairs according to a seed position.</div>
</dd>
<dt><a href="./org/biojava3/genome/App.html" title="class in org.biojava3.genome"><span class="strong">App</span></a> - Class in <a href="./org/biojava3/genome/package-summary.html">org.biojava3.genome</a></dt>
<dd>
<div class="block">Hello world!</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/App.html#App()">App()</a></span> - Constructor for class org.biojava3.genome.<a href="./org/biojava3/genome/App.html" title="class in org.biojava3.genome">App</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/phylo/App.html" title="class in org.biojava3.phylo"><span class="strong">App</span></a> - Class in <a href="./org/biojava3/phylo/package-summary.html">org.biojava3.phylo</a></dt>
<dd>
<div class="block">Hello world!</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/App.html#App()">App()</a></span> - Constructor for class org.biojava3.phylo.<a href="./org/biojava3/phylo/App.html" title="class in org.biojava3.phylo">App</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/FlatFileInstallation.html#applyFilters()">applyFilters()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/FlatFileInstallation.html" title="class in org.biojava.bio.structure.server">FlatFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/ConfigPDBInstallPanel.html#applyValues()">applyValues()</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/ConfigPDBInstallPanel.html" title="class in org.biojava.bio.structure.align.gui">ConfigPDBInstallPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentJoiner.html#approach_ap3(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], org.biojava.bio.structure.align.pairwise.FragmentPair[], org.biojava.bio.structure.align.StrucAligParameters)">approach_ap3(Atom[], Atom[], FragmentPair[], StrucAligParameters)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentJoiner.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentJoiner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#Arg">Arg</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#aromatic">aromatic</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#arrayLeftDivide(org.biojava.bio.structure.jama.Matrix)">arrayLeftDivide(Matrix)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Element-by-element left division, C = A.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#arrayLeftDivideEquals(org.biojava.bio.structure.jama.Matrix)">arrayLeftDivideEquals(Matrix)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Element-by-element left division in place, A = A.</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/loader/ArrayListProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader"><span class="strong">ArrayListProxySequenceReader</span></a>&lt;<a href="./org/biojava3/core/sequence/loader/ArrayListProxySequenceReader.html" title="type parameter in ArrayListProxySequenceReader">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/core/sequence/loader/package-summary.html">org.biojava3.core.sequence.loader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/ArrayListProxySequenceReader.html#ArrayListProxySequenceReader()">ArrayListProxySequenceReader()</a></span> - Constructor for class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/ArrayListProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader">ArrayListProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava3.core.sequence.storage"><span class="strong">ArrayListSequenceReader</span></a>&lt;<a href="./org/biojava3/core/sequence/storage/ArrayListSequenceReader.html" title="type parameter in ArrayListSequenceReader">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/core/sequence/storage/package-summary.html">org.biojava3.core.sequence.storage</a></dt>
<dd>
<div class="block">Stores a Sequence as a collection of compounds in an ArrayList</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/ArrayListSequenceReader.html#ArrayListSequenceReader()">ArrayListSequenceReader()</a></span> - Constructor for class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava3.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/ArrayListSequenceReader.html#ArrayListSequenceReader(java.util.List, org.biojava3.core.sequence.template.CompoundSet)">ArrayListSequenceReader(List&lt;C&gt;, CompoundSet&lt;C&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava3.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/ArrayListSequenceReader.html#ArrayListSequenceReader(java.lang.String, org.biojava3.core.sequence.template.CompoundSet)">ArrayListSequenceReader(String, CompoundSet&lt;C&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava3.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#arrayRightDivide(org.biojava.bio.structure.jama.Matrix)">arrayRightDivide(Matrix)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Element-by-element right division, C = A.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#arrayRightDivideEquals(org.biojava.bio.structure.jama.Matrix)">arrayRightDivideEquals(Matrix)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Element-by-element right division in place, A = A.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#arrayTimes(org.biojava.bio.structure.jama.Matrix)">arrayTimes(Matrix)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Element-by-element multiplication, C = A.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#arrayTimesEquals(org.biojava.bio.structure.jama.Matrix)">arrayTimesEquals(Matrix)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Element-by-element multiplication in place, A = A.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#Asn">Asn</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#Asp">Asp</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/InsdcLocations.BondLocation.html#assertLocation()">assertLocation()</a></span> - Method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/InsdcLocations.BondLocation.html" title="class in org.biojava3.core.sequence.location">InsdcLocations.BondLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html#assertLocation()">assertLocation()</a></span> - Method in class org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava3.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStruc.html#assign(org.biojava.bio.structure.Structure)">assign(Structure)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStruc.html" title="class in org.biojava.bio.structure.secstruc">SecStruc</a></dt>
<dd>
<div class="block">assigns the secondary structure to the groups in this Structure object
 and set the results in the group properties.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure"><span class="strong">Atom</span></a> - Interface in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">A simple interface for an Atom.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileParser.html#ATOM_CA_THRESHOLD">ATOM_CA_THRESHOLD</a></span> - Static variable in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileParser.html" title="class in org.biojava.bio.structure.io">PDBFileParser</a></dt>
<dd>
<div class="block">the maximum number of atoms that will be parsed before the parser switches to a CA-only
 representation of the PDB file.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util"><span class="strong">AtomCache</span></a> - Class in <a href="./org/biojava/bio/structure/align/util/package-summary.html">org.biojava.bio.structure.align.util</a></dt>
<dd>
<div class="block">A utility class that provides easy access to Structure objects.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#AtomCache()">AtomCache()</a></span> - Constructor for class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>
<div class="block">Default AtomCache constructor.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#AtomCache(java.lang.String, boolean)">AtomCache(String, boolean)</a></span> - Constructor for class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>
<div class="block">Creates an instance of an AtomCache that is pointed to the a particular
 path in the file system.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#AtomCache(org.biojava.bio.structure.align.util.UserConfiguration)">AtomCache(UserConfiguration)</a></span> - Constructor for class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>
<div class="block">Creates a new AtomCache object based on the provided UserConfiguration.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure"><span class="strong">AtomImpl</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">Implementation of an Atom of a PDB file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#AtomImpl()">AtomImpl()</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/gui/jmol/AtomInfo.html" title="class in org.biojava.bio.structure.align.gui.jmol"><span class="strong">AtomInfo</span></a> - Class in <a href="./org/biojava/bio/structure/align/gui/jmol/package-summary.html">org.biojava.bio.structure.align.gui.jmol</a></dt>
<dd>
<div class="block">This class uniquely describes an atom</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/AtomInfo.html#AtomInfo()">AtomInfo()</a></span> - Constructor for class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/AtomInfo.html" title="class in org.biojava.bio.structure.align.gui.jmol">AtomInfo</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/gui/jmol/AtomInfoParser.html" title="class in org.biojava.bio.structure.align.gui.jmol"><span class="strong">AtomInfoParser</span></a> - Class in <a href="./org/biojava/bio/structure/align/gui/jmol/package-summary.html">org.biojava.bio.structure.align.gui.jmol</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/AtomInfoParser.html#AtomInfoParser()">AtomInfoParser()</a></span> - Constructor for class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/AtomInfoParser.html" title="class in org.biojava.bio.structure.align.gui.jmol">AtomInfoParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/AtomIterator.html" title="class in org.biojava.bio.structure"><span class="strong">AtomIterator</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">an iterator over all atoms of a structure / group.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomIterator.html#AtomIterator(org.biojava.bio.structure.Structure)">AtomIterator(Structure)</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomIterator.html" title="class in org.biojava.bio.structure">AtomIterator</a></dt>
<dd>
<div class="block">Constructs an AtomIterator object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomIterator.html#AtomIterator(org.biojava.bio.structure.Group)">AtomIterator(Group)</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomIterator.html" title="class in org.biojava.bio.structure">AtomIterator</a></dt>
<dd>
<div class="block">Constructs an AtomIterator object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcid.html#ATOMRECORD">ATOMRECORD</a></span> - Static variable in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcid.html" title="interface in org.biojava.bio.structure">AminoAcid</a></dt>
<dd>
<div class="block">Field to distribguish AminoAcids that have been created from SEQRES records and ATOM records.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#atoms">atoms</a></span> - Variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model"><span class="strong">AtomSite</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#AtomSite()">AtomSite()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/PrettyXMLWriter.html#attribute(java.lang.String, java.lang.String, java.lang.String)">attribute(String, String, String)</a></span> - Method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/PrettyXMLWriter.html" title="class in org.biojava3.core.util">PrettyXMLWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/PrettyXMLWriter.html#attribute(java.lang.String, java.lang.String)">attribute(String, String)</a></span> - Method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/PrettyXMLWriter.html" title="class in org.biojava3.core.util">PrettyXMLWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/XMLWriter.html#attribute(java.lang.String, java.lang.String, java.lang.String)">attribute(String, String, String)</a></span> - Method in interface org.biojava3.core.util.<a href="./org/biojava3/core/util/XMLWriter.html" title="interface in org.biojava3.core.util">XMLWriter</a></dt>
<dd>
<div class="block">Add an attribute to an element.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/XMLWriter.html#attribute(java.lang.String, java.lang.String)">attribute(String, String)</a></span> - Method in interface org.biojava3.core.util.<a href="./org/biojava3/core/util/XMLWriter.html" title="interface in org.biojava3.core.util">XMLWriter</a></dt>
<dd>
<div class="block">Add an un-qualified attribute to an element.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Feature.html#attributes()">attributes()</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Feature.html" title="class in org.biojava3.genome.parsers.gff">Feature</a></dt>
<dd>
<div class="block">Get the string of key/value attributes.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/FeatureList.html#attributeValues(java.lang.String)">attributeValues(String)</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/FeatureList.html" title="class in org.biojava3.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block">Create a collection of the unique values for the specified key.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/AuditAuthor.html" title="class in org.biojava.bio.structure.io.mmcif.model"><span class="strong">AuditAuthor</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AuditAuthor.html#AuditAuthor()">AuditAuthor()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AuditAuthor.html" title="class in org.biojava.bio.structure.io.mmcif.model">AuditAuthor</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/Author.html" title="class in org.biojava.bio.structure"><span class="strong">Author</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">Describes author attributes for author information in a PDB file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Author.html#Author()">Author()</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Author.html" title="class in org.biojava.bio.structure">Author</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/UncompressInputStream.html#available()">available()</a></span> - Method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/UncompressInputStream.html" title="class in org.biojava3.core.util">UncompressInputStream</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_B_">
<!--   -->
</a>
<h2 class="title">B</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#backboneAtomNames">backboneAtomNames</a></span> - Static variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>
<div class="block">The names of the Atoms that form the backbone.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html#BACKGROUND_COLOR">BACKGROUND_COLOR</a></span> - Static variable in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html" title="class in org.biojava.bio.structure.gui.util">SequenceScalePanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoChangeChemCompProvider.html#basicLoad(org.biojava.bio.structure.io.PDBFileReader, boolean, java.lang.String)">basicLoad(PDBFileReader, boolean, String)</a></span> - Method in class demo.<a href="./demo/DemoChangeChemCompProvider.html" title="class in demo">DemoChangeChemCompProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoDomainsplit.html#basicLoad(java.lang.String)">basicLoad(String)</a></span> - Method in class demo.<a href="./demo/DemoDomainsplit.html" title="class in demo">DemoDomainsplit</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoLoadStructure.html#basicLoad()">basicLoad()</a></span> - Method in class demo.<a href="./demo/DemoLoadStructure.html" title="class in demo">DemoLoadStructure</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/BasicSequence.html" title="class in org.biojava3.core.sequence"><span class="strong">BasicSequence</span></a>&lt;<a href="./org/biojava3/core/sequence/BasicSequence.html" title="type parameter in BasicSequence">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a></dt>
<dd>
<div class="block">Bare bones version of the Sequence object to be used sparingly.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/BasicSequence.html#BasicSequence(java.lang.String, org.biojava3.core.sequence.template.CompoundSet)">BasicSequence(String, CompoundSet&lt;C&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/BasicSequence.html" title="class in org.biojava3.core.sequence">BasicSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/BasicSequence.html#BasicSequence(org.biojava3.core.sequence.template.ProxySequenceReader)">BasicSequence(ProxySequenceReader&lt;C&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/BasicSequence.html" title="class in org.biojava3.core.sequence">BasicSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/secstruc/BigSqrt.html" title="class in org.biojava.bio.structure.secstruc"><span class="strong">BigSqrt</span></a> - Class in <a href="./org/biojava/bio/structure/secstruc/package-summary.html">org.biojava.bio.structure.secstruc</a></dt>
<dd>
<div class="block">calculate a more precise SQRT.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/BigSqrt.html#BigSqrt()">BigSqrt()</a></span> - Constructor for class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/BigSqrt.html" title="class in org.biojava.bio.structure.secstruc">BigSqrt</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Location.html#bioEnd()">bioEnd()</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Location.html" title="class in org.biojava3.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Get end index, in biocoordinates.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/gui/BiojavaJmol.html" title="class in org.biojava.bio.structure.gui"><span class="strong">BiojavaJmol</span></a> - Class in <a href="./org/biojava/bio/structure/gui/package-summary.html">org.biojava.bio.structure.gui</a></dt>
<dd>
<div class="block">A class that provides a simple GUI for Jmol</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/BiojavaJmol.html#BiojavaJmol()">BiojavaJmol()</a></span> - Constructor for class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/BiojavaJmol.html" title="class in org.biojava.bio.structure.gui">BiojavaJmol</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/BioJavaStructureAlignment.html" title="class in org.biojava.bio.structure.align"><span class="strong">BioJavaStructureAlignment</span></a> - Class in <a href="./org/biojava/bio/structure/align/package-summary.html">org.biojava.bio.structure.align</a></dt>
<dd>
<div class="block">Wrapper for the BioJava Structure Alignment Implementation</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/BioJavaStructureAlignment.html#BioJavaStructureAlignment()">BioJavaStructureAlignment()</a></span> - Constructor for class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/BioJavaStructureAlignment.html" title="class in org.biojava.bio.structure.align">BioJavaStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Location.html#bioStart()">bioStart()</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Location.html" title="class in org.biojava3.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Get start index, in biocoordinates.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Location.html#bioStrand()">bioStrand()</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Location.html" title="class in org.biojava3.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Get character representation of strand.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#bitMask()">bitMask()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava3.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>
<div class="block">This method should return the bit mask to be used to extract the
 bytes you are interested in working with.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html#bitMask()">bitMask()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html" title="class in org.biojava3.core.sequence.storage">FourBitSequenceReader.FourBitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html#bitMask()">bitMask()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html" title="class in org.biojava3.core.sequence.storage">TwoBitSequenceReader.TwoBitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava3.core.sequence.storage"><span class="strong">BitSequenceReader</span></a>&lt;<a href="./org/biojava3/core/sequence/storage/BitSequenceReader.html" title="type parameter in BitSequenceReader">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/core/sequence/storage/package-summary.html">org.biojava3.core.sequence.storage</a></dt>
<dd>
<div class="block">An implementation of the popular bit encodings.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/BitSequenceReader.html#BitSequenceReader(org.biojava3.core.sequence.storage.BitSequenceReader.BitArrayWorker, org.biojava3.core.sequence.AccessionID)">BitSequenceReader(BitSequenceReader.BitArrayWorker&lt;C&gt;, AccessionID)</a></span> - Constructor for class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava3.core.sequence.storage">BitSequenceReader</a></dt>
<dd>
<div class="block">Instance which allows you to supply a different @{BitArrayWorker}
 object.</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava3.core.sequence.storage"><span class="strong">BitSequenceReader.BitArrayWorker</span></a>&lt;<a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="type parameter in BitSequenceReader.BitArrayWorker">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/core/sequence/storage/package-summary.html">org.biojava3.core.sequence.storage</a></dt>
<dd>
<div class="block">The logic of working with a bit has been separated out into this class
 to help developers create the bit data structures without having to
 put the code into an intermediate format and to also use the format
 without the need to copy this code.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#BitSequenceReader.BitArrayWorker(org.biojava3.core.sequence.template.Sequence)">BitSequenceReader.BitArrayWorker(Sequence&lt;C&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava3.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#BitSequenceReader.BitArrayWorker(java.lang.String, org.biojava3.core.sequence.template.CompoundSet)">BitSequenceReader.BitArrayWorker(String, CompoundSet&lt;C&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava3.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#BitSequenceReader.BitArrayWorker(org.biojava3.core.sequence.template.CompoundSet, int)">BitSequenceReader.BitArrayWorker(CompoundSet&lt;C&gt;, int)</a></span> - Constructor for class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava3.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#BitSequenceReader.BitArrayWorker(org.biojava3.core.sequence.template.CompoundSet, int[])">BitSequenceReader.BitArrayWorker(CompoundSet&lt;C&gt;, int[])</a></span> - Constructor for class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava3.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#bitsPerCompound()">bitsPerCompound()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava3.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>
<div class="block">Returns how many bits are used to represent a compound e.g. 2 if using
 2bit encoding.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html#BLACK_WHITE_GRADIENT">BLACK_WHITE_GRADIENT</a></span> - Static variable in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html" title="class in org.biojava.bio.structure.gui.util.color">GradientMapper</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/genome/homology/BlastHomologyHits.html" title="class in org.biojava3.genome.homology"><span class="strong">BlastHomologyHits</span></a> - Class in <a href="./org/biojava3/genome/homology/package-summary.html">org.biojava3.genome.homology</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/homology/BlastHomologyHits.html#BlastHomologyHits()">BlastHomologyHits()</a></span> - Constructor for class org.biojava3.genome.homology.<a href="./org/biojava3/genome/homology/BlastHomologyHits.html" title="class in org.biojava3.genome.homology">BlastHomologyHits</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/genome/query/BlastXMLQuery.html" title="class in org.biojava3.genome.query"><span class="strong">BlastXMLQuery</span></a> - Class in <a href="./org/biojava3/genome/query/package-summary.html">org.biojava3.genome.query</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/query/BlastXMLQuery.html#BlastXMLQuery(java.lang.String)">BlastXMLQuery(String)</a></span> - Constructor for class org.biojava3.genome.query.<a href="./org/biojava3/genome/query/BlastXMLQuery.html" title="class in org.biojava3.genome.query">BlastXMLQuery</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/AFPOptimizer.html#blockInfo(org.biojava.bio.structure.align.model.AFPChain)">blockInfo(AFPChain)</a></span> - Static method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/AFPOptimizer.html" title="class in org.biojava.bio.structure.align.fatcat.calc">AFPOptimizer</a></dt>
<dd>
<div class="block">get the afp list and residue list for each block</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ronn/RonnConstraint.html#Blosum62">Blosum62</a></span> - Static variable in class org.biojava3.ronn.<a href="./org/biojava3/ronn/RonnConstraint.html" title="class in org.biojava3.ronn">RonnConstraint</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/Bond.html" title="class in org.biojava.bio.structure"><span class="strong">Bond</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">Simple bond  - it's just an edge/vertex for two <a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure"><code>Atom</code></a> nodes.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Bond.html#Bond()">Bond()</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Bond.html" title="class in org.biojava.bio.structure">Bond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Bond.html#Bond(double, org.biojava.bio.structure.BondType, org.biojava.bio.structure.Group, org.biojava.bio.structure.Atom, org.biojava.bio.structure.Group, org.biojava.bio.structure.Atom)">Bond(double, BondType, Group, Atom, Group, Atom)</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Bond.html" title="class in org.biojava.bio.structure">Bond</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/BondType.html" title="enum in org.biojava.bio.structure"><span class="strong">BondType</span></a> - Enum in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">Work in progress - NOT final!</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/FeatureList.html#bounds()">bounds()</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/FeatureList.html" title="class in org.biojava3.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block">The union of all locations of all features in this list, mapped to the positive strand.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/TreeConstructor.html#broadcastComplete()">broadcastComplete()</a></span> - Method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/TreeConstructor.html" title="class in org.biojava3.phylo">TreeConstructor</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/webstart/BrowserOpener.html" title="class in org.biojava.bio.structure.align.webstart"><span class="strong">BrowserOpener</span></a> - Class in <a href="./org/biojava/bio/structure/align/webstart/package-summary.html">org.biojava.bio.structure.align.webstart</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/webstart/BrowserOpener.html#BrowserOpener()">BrowserOpener()</a></span> - Constructor for class org.biojava.bio.structure.align.webstart.<a href="./org/biojava/bio/structure/align/webstart/BrowserOpener.html" title="class in org.biojava.bio.structure.align.webstart">BrowserOpener</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/io/BufferedReaderBytesRead.html" title="class in org.biojava3.core.sequence.io"><span class="strong">BufferedReaderBytesRead</span></a> - Class in <a href="./org/biojava3/core/sequence/io/package-summary.html">org.biojava3.core.sequence.io</a></dt>
<dd>
<div class="block">Need to keep track of actual bytes read and take advantage of buffered reader
 performance.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/BufferedReaderBytesRead.html#BufferedReaderBytesRead(java.io.Reader, int)">BufferedReaderBytesRead(Reader, int)</a></span> - Constructor for class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/BufferedReaderBytesRead.html" title="class in org.biojava3.core.sequence.io">BufferedReaderBytesRead</a></dt>
<dd>
<div class="block">Creates a buffering character-input stream that uses an input buffer of
 the specified size.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/BufferedReaderBytesRead.html#BufferedReaderBytesRead(java.io.Reader)">BufferedReaderBytesRead(Reader)</a></span> - Constructor for class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/BufferedReaderBytesRead.html" title="class in org.biojava3.core.sequence.io">BufferedReaderBytesRead</a></dt>
<dd>
<div class="block">Creates a buffering character-input stream that uses a default-sized
 input buffer.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.Builder.html#build()">build()</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava3.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>
<div class="block">The method to finish any calls to the builder with which returns
 a transcription engine.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationImpl.Builder.html#build()">build()</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationImpl.Builder.html" title="class in org.biojava3.protmod">ProteinModificationImpl.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/FeatureHelper.html#buildFeatureAtrributeIndex(java.lang.String, org.biojava3.genome.parsers.gff.FeatureList)">buildFeatureAtrributeIndex(String, FeatureList)</a></span> - Static method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/FeatureHelper.html" title="class in org.biojava3.genome.parsers.gff">FeatureHelper</a></dt>
<dd>
<div class="block">Build a list of individual features to allow easy indexing and to avoid iterating through large genome gff3 files
 The index for the returned HashMap is the value of the attribute used to build the index</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/views/RnaSequenceView.html#buildTranslators()">buildTranslators()</a></span> - Method in class org.biojava3.core.sequence.views.<a href="./org/biojava3/core/sequence/views/RnaSequenceView.html" title="class in org.biojava3.core.sequence.views">RnaSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#buried">buried</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#buriedmax">buriedmax</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#buriedmin">buriedmin</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#BYTES_PER_INT">BYTES_PER_INT</a></span> - Static variable in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava3.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_C_">
<!--   -->
</a>
<h2 class="title">C</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/Constraints.html#C">C</a></span> - Static variable in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/Constraints.html" title="class in org.biojava3.aaproperties">Constraints</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#CA_AND_SIDE_CHAIN_ANGLE_SCORING">CA_AND_SIDE_CHAIN_ANGLE_SCORING</a></span> - Static variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStruc.html#CA_MIN_DIST">CA_MIN_DIST</a></span> - Static variable in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStruc.html" title="class in org.biojava.bio.structure.secstruc">SecStruc</a></dt>
<dd>
<div class="block">the minimal distance of two CA atoms if H-bonds are allowed to form.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#caAtomName">caAtomName</a></span> - Static variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>
<div class="block">The Atom name of C-alpha atoms.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/InputStreamProvider.html#CACHE_PROPERTY">CACHE_PROPERTY</a></span> - Static variable in class org.biojava3.core.util.<a href="./org/biojava3/core/util/InputStreamProvider.html" title="class in org.biojava3.core.util">InputStreamProvider</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/util/CacheFactory.html" title="class in org.biojava.bio.structure.align.util"><span class="strong">CacheFactory</span></a> - Class in <a href="./org/biojava/bio/structure/align/util/package-summary.html">org.biojava.bio.structure.align.util</a></dt>
<dd>
<div class="block">provides a SoftHashMap singleton.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/io/CAConverter.html" title="class in org.biojava.bio.structure.io"><span class="strong">CAConverter</span></a> - Class in <a href="./org/biojava/bio/structure/io/package-summary.html">org.biojava.bio.structure.io</a></dt>
<dd>
<div class="block">Converts full atom representations to Calpha only ones.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/CAConverter.html#CAConverter()">CAConverter()</a></span> - Constructor for class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/CAConverter.html" title="class in org.biojava.bio.structure.io">CAConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/AFPChainer.html#calAfpRmsd(int, int[], int, org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">calAfpRmsd(int, int[], int, AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/AFPChainer.html" title="class in org.biojava.bio.structure.align.fatcat.calc">AFPChainer</a></dt>
<dd>
<div class="block">//return the rmsd of the residues from the segments that form the given AFP list
  //this value can be a measurement (1) for the connectivity of the AFPs</div>
</dd>
<dt><a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure"><span class="strong">Calc</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">utility operations on Atoms, AminoAcids, etc.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#Calc()">Calc()</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#calc_rmsd(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], int, boolean, boolean)">calc_rmsd(Atom[], Atom[], int, boolean, boolean)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>
<div class="block">superimpose and get rmsd</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html#calcIdSimilarity(char[], char[], int)">calcIdSimilarity(char[], char[], int)</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html" title="class in org.biojava.bio.structure.align.util">AFPAlignmentDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/SequenceDisplay.html#calcScale(int)">calcScale(int)</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/SequenceDisplay.html" title="class in org.biojava.bio.structure.gui">SequenceDisplay</a></dt>
<dd>
<div class="block">a value of 100 means that the whole sequence should be displayed in the current visible window
 a factor of 1 means that one amino acid shoud be drawn as big as possible</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#calcScores(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">calcScores(Atom[], Atom[])</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>
<div class="block">calculates  scores for this alignment ( %id )</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#calcSimilarity()">calcSimilarity()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentProgressListener.html#calculatedFragmentPairs(java.util.List)">calculatedFragmentPairs(List&lt;FragmentPair&gt;)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentProgressListener.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStruc.html#calculateHBondEnergy(org.biojava.bio.structure.secstruc.SecStrucGroup, org.biojava.bio.structure.secstruc.SecStrucGroup)">calculateHBondEnergy(SecStrucGroup, SecStrucGroup)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStruc.html" title="class in org.biojava.bio.structure.secstruc">SecStruc</a></dt>
<dd>
<div class="block">calculate HBond energy of two groups in cal/mol ...</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractNucleotideCompoundSet.html#calculateIndirectAmbiguities()">calculateIndirectAmbiguities()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractNucleotideCompoundSet.html" title="class in org.biojava3.core.sequence.template">AbstractNucleotideCompoundSet</a></dt>
<dd>
<div class="block">Loops through all known nucelotides and attempts to find which are
 equivalent to each other.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#calculateSuperpositionByIdx(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">calculateSuperpositionByIdx(Atom[], Atom[])</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeMain.html#calculator">calculator</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeMain.html" title="class in org.biojava.bio.structure.align.ce">CeMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html#call()">call()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html" title="class in org.biojava.bio.structure.align">CallableStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/CallablePairwiseSequenceAligner.html#call()">call()</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/CallablePairwiseSequenceAligner.html" title="class in org.biojava3.alignment.template">CallablePairwiseSequenceAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/CallablePairwiseSequenceScorer.html#call()">call()</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/CallablePairwiseSequenceScorer.html" title="class in org.biojava3.alignment.template">CallablePairwiseSequenceScorer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/CallableProfileProfileAligner.html#call()">call()</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/CallableProfileProfileAligner.html" title="class in org.biojava3.alignment.template">CallableProfileProfileAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/ronn/ORonn.html#call()">call()</a></span> - Method in class org.biojava3.ronn.<a href="./org/biojava3/ronn/ORonn.html" title="class in org.biojava3.ronn">ORonn</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/alignment/template/CallablePairwiseSequenceAligner.html" title="class in org.biojava3.alignment.template"><span class="strong">CallablePairwiseSequenceAligner</span></a>&lt;<a href="./org/biojava3/alignment/template/CallablePairwiseSequenceAligner.html" title="type parameter in CallablePairwiseSequenceAligner">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava3/alignment/template/CallablePairwiseSequenceAligner.html" title="type parameter in CallablePairwiseSequenceAligner">C</a>&gt;,<a href="./org/biojava3/alignment/template/CallablePairwiseSequenceAligner.html" title="type parameter in CallablePairwiseSequenceAligner">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/alignment/template/package-summary.html">org.biojava3.alignment.template</a></dt>
<dd>
<div class="block">Implements a concurrency wrapper for a <a href="./org/biojava3/alignment/template/PairwiseSequenceAligner.html" title="interface in org.biojava3.alignment.template"><code>PairwiseSequenceAligner</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/CallablePairwiseSequenceAligner.html#CallablePairwiseSequenceAligner(org.biojava3.alignment.template.PairwiseSequenceAligner)">CallablePairwiseSequenceAligner(PairwiseSequenceAligner&lt;S, C&gt;)</a></span> - Constructor for class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/CallablePairwiseSequenceAligner.html" title="class in org.biojava3.alignment.template">CallablePairwiseSequenceAligner</a></dt>
<dd>
<div class="block">Creates a pairwise sequence alignment task for simplified parallel execution.</div>
</dd>
<dt><a href="./org/biojava3/alignment/template/CallablePairwiseSequenceScorer.html" title="class in org.biojava3.alignment.template"><span class="strong">CallablePairwiseSequenceScorer</span></a>&lt;<a href="./org/biojava3/alignment/template/CallablePairwiseSequenceScorer.html" title="type parameter in CallablePairwiseSequenceScorer">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava3/alignment/template/CallablePairwiseSequenceScorer.html" title="type parameter in CallablePairwiseSequenceScorer">C</a>&gt;,<a href="./org/biojava3/alignment/template/CallablePairwiseSequenceScorer.html" title="type parameter in CallablePairwiseSequenceScorer">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/alignment/template/package-summary.html">org.biojava3.alignment.template</a></dt>
<dd>
<div class="block">Implements a concurrency wrapper for a <a href="./org/biojava3/alignment/template/PairwiseSequenceScorer.html" title="interface in org.biojava3.alignment.template"><code>PairwiseSequenceScorer</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/CallablePairwiseSequenceScorer.html#CallablePairwiseSequenceScorer(org.biojava3.alignment.template.PairwiseSequenceScorer)">CallablePairwiseSequenceScorer(PairwiseSequenceScorer&lt;S, C&gt;)</a></span> - Constructor for class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/CallablePairwiseSequenceScorer.html" title="class in org.biojava3.alignment.template">CallablePairwiseSequenceScorer</a></dt>
<dd>
<div class="block">Creates a pairwise sequence scoring task for simplified parallel execution.</div>
</dd>
<dt><a href="./org/biojava3/alignment/template/CallableProfileProfileAligner.html" title="class in org.biojava3.alignment.template"><span class="strong">CallableProfileProfileAligner</span></a>&lt;<a href="./org/biojava3/alignment/template/CallableProfileProfileAligner.html" title="type parameter in CallableProfileProfileAligner">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava3/alignment/template/CallableProfileProfileAligner.html" title="type parameter in CallableProfileProfileAligner">C</a>&gt;,<a href="./org/biojava3/alignment/template/CallableProfileProfileAligner.html" title="type parameter in CallableProfileProfileAligner">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/alignment/template/package-summary.html">org.biojava3.alignment.template</a></dt>
<dd>
<div class="block">Implements a concurrency wrapper for a <a href="./org/biojava3/alignment/template/ProfileProfileAligner.html" title="interface in org.biojava3.alignment.template"><code>ProfileProfileAligner</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/CallableProfileProfileAligner.html#CallableProfileProfileAligner(org.biojava3.alignment.template.ProfileProfileAligner)">CallableProfileProfileAligner(ProfileProfileAligner&lt;S, C&gt;)</a></span> - Constructor for class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/CallableProfileProfileAligner.html" title="class in org.biojava3.alignment.template">CallableProfileProfileAligner</a></dt>
<dd>
<div class="block">Creates a profile-profile alignment task for simplified parallel execution.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html" title="class in org.biojava.bio.structure.align"><span class="strong">CallableStructureAlignment</span></a> - Class in <a href="./org/biojava/bio/structure/align/package-summary.html">org.biojava.bio.structure.align</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html#CallableStructureAlignment()">CallableStructureAlignment()</a></span> - Constructor for class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html" title="class in org.biojava.bio.structure.align">CallableStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/SigEva.html#calNS(org.biojava.bio.structure.align.fatcat.calc.FatCatParameters, org.biojava.bio.structure.align.model.AFPChain)">calNS(FatCatParameters, AFPChain)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/SigEva.html" title="class in org.biojava.bio.structure.align.fatcat.calc">SigEva</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/SigEva.html#calSigAll(org.biojava.bio.structure.align.fatcat.calc.FatCatParameters, org.biojava.bio.structure.align.model.AFPChain)">calSigAll(FatCatParameters, AFPChain)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/SigEva.html" title="class in org.biojava.bio.structure.align.fatcat.calc">SigEva</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/TreeConstructor.html#cancel()">cancel()</a></span> - Method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/TreeConstructor.html" title="class in org.biojava3.phylo">TreeConstructor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/NJTreeProgressListener.html#canceled(java.lang.Object)">canceled(Object)</a></span> - Method in interface org.biojava3.phylo.<a href="./org/biojava3/phylo/NJTreeProgressListener.html" title="interface in org.biojava3.phylo">NJTreeProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ProgessListenerStub.html#canceled(java.lang.Object)">canceled(Object)</a></span> - Method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ProgessListenerStub.html" title="class in org.biojava3.phylo">ProgessListenerStub</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html#canComplement(org.biojava3.core.sequence.template.Sequence)">canComplement(Sequence&lt;C&gt;)</a></span> - Method in class org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava3.core.sequence.location.template">AbstractLocation</a></dt>
<dd>
<div class="block">Uses the Sequence's CompoundSet to decide if a compound can
 be assgined to ComplementCompound meaning it can complement</div>
</dd>
<dt><a href="./org/biojava3/aaproperties/xml/CaseFreeAminoAcidCompoundSet.html" title="class in org.biojava3.aaproperties.xml"><span class="strong">CaseFreeAminoAcidCompoundSet</span></a> - Class in <a href="./org/biojava3/aaproperties/xml/package-summary.html">org.biojava3.aaproperties.xml</a></dt>
<dd>
<div class="block">Set of proteinogenic amino acids.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/CaseFreeAminoAcidCompoundSet.html#CaseFreeAminoAcidCompoundSet()">CaseFreeAminoAcidCompoundSet()</a></span> - Constructor for class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/CaseFreeAminoAcidCompoundSet.html" title="class in org.biojava3.aaproperties.xml">CaseFreeAminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava3.core.sequence.transcription"><span class="strong">CaseInsensitiveCompound</span></a> - Class in <a href="./org/biojava3/core/sequence/transcription/package-summary.html">org.biojava3.core.sequence.transcription</a></dt>
<dd>
<div class="block">Attempts to wrap compounds so it is possible to view them
 in a case insensitive manner</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/CaseInsensitiveCompound.html#CaseInsensitiveCompound(org.biojava3.core.sequence.compound.NucleotideCompound)">CaseInsensitiveCompound(NucleotideCompound)</a></span> - Constructor for class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava3.core.sequence.transcription">CaseInsensitiveCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#cbAtomName">cbAtomName</a></span> - Static variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/CDSComparator.html" title="class in org.biojava3.core.sequence"><span class="strong">CDSComparator</span></a> - Class in <a href="./org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/CDSComparator.html#CDSComparator()">CDSComparator()</a></span> - Constructor for class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/CDSComparator.html" title="class in org.biojava3.core.sequence">CDSComparator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/CDSSequence.html" title="class in org.biojava3.core.sequence"><span class="strong">CDSSequence</span></a> - Class in <a href="./org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a></dt>
<dd>
<div class="block">Represents a exon or coding sequence in a gene.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/CDSSequence.html#CDSSequence(org.biojava3.core.sequence.TranscriptSequence, int, int, int)">CDSSequence(TranscriptSequence, int, int, int)</a></span> - Constructor for class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/CDSSequence.html" title="class in org.biojava3.core.sequence">CDSSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce"><span class="strong">CECalculator</span></a> - Class in <a href="./org/biojava/bio/structure/align/ce/package-summary.html">org.biojava.bio.structure.align.ce</a></dt>
<dd>
<div class="block">This is based on the original Combinatorial Extension (CE) source code from 2003 or 2004 (CE version 2.3),
 as has been originally developed by I.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#CECalculator(org.biojava.bio.structure.align.ce.CeParameters)">CECalculator(CeParameters)</a></span> - Constructor for class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/DBResultTable.html#ceColumnNames">ceColumnNames</a></span> - Static variable in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/DBResultTable.html" title="class in org.biojava.bio.structure.align.gui">DBResultTable</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/ce/CeCPMain.html" title="class in org.biojava.bio.structure.align.ce"><span class="strong">CeCPMain</span></a> - Class in <a href="./org/biojava/bio/structure/align/ce/package-summary.html">org.biojava.bio.structure.align.ce</a></dt>
<dd>
<div class="block">A wrapper for <a href="./org/biojava/bio/structure/align/ce/CeMain.html" title="class in org.biojava.bio.structure.align.ce"><code>CeMain</code></a> which sets default parameters to be appropriate for finding
 circular permutations</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCPMain.html#CeCPMain()">CeCPMain()</a></span> - Constructor for class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCPMain.html" title="class in org.biojava.bio.structure.align.ce">CeCPMain</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/ce/CeMain.html" title="class in org.biojava.bio.structure.align.ce"><span class="strong">CeMain</span></a> - Class in <a href="./org/biojava/bio/structure/align/ce/package-summary.html">org.biojava.bio.structure.align.ce</a></dt>
<dd>
<div class="block">The main class of the Java implementation of the Combinatorial Extension Algorithm (CE),
 as has been originally developed by I.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeMain.html#CeMain()">CeMain()</a></span> - Constructor for class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeMain.html" title="class in org.biojava.bio.structure.align.ce">CeMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#centerAtoms(org.biojava.bio.structure.Atom[])">centerAtoms(Atom[])</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">Center the atoms at the Centroid.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#centerAtoms(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom)">centerAtoms(Atom[], Atom)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">Center the atoms at the Centroid, if the centroid is already know.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#centerOfMass(org.biojava.bio.structure.Atom[])">centerOfMass(Atom[])</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce"><span class="strong">CeParameters</span></a> - Class in <a href="./org/biojava/bio/structure/align/ce/package-summary.html">org.biojava.bio.structure.align.ce</a></dt>
<dd>
<div class="block">Contains the parameters that can be sent to CE</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#CeParameters()">CeParameters()</a></span> - Constructor for class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/ce/CeSideChainMain.html" title="class in org.biojava.bio.structure.align.ce"><span class="strong">CeSideChainMain</span></a> - Class in <a href="./org/biojava/bio/structure/align/ce/package-summary.html">org.biojava.bio.structure.align.ce</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeSideChainMain.html#CeSideChainMain()">CeSideChainMain()</a></span> - Constructor for class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeSideChainMain.html" title="class in org.biojava.bio.structure.align.ce">CeSideChainMain</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/ce/CeSideChainUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce"><span class="strong">CeSideChainUserArgumentProcessor</span></a> - Class in <a href="./org/biojava/bio/structure/align/ce/package-summary.html">org.biojava.bio.structure.align.ce</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeSideChainUserArgumentProcessor.html#CeSideChainUserArgumentProcessor()">CeSideChainUserArgumentProcessor()</a></span> - Constructor for class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeSideChainUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce">CeSideChainUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/ce/CeUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce"><span class="strong">CeUserArgumentProcessor</span></a> - Class in <a href="./org/biojava/bio/structure/align/ce/package-summary.html">org.biojava.bio.structure.align.ce</a></dt>
<dd>
<div class="block">process the arguments from command line</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeUserArgumentProcessor.html#CeUserArgumentProcessor()">CeUserArgumentProcessor()</a></span> - Constructor for class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce">CeUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure"><span class="strong">Chain</span></a> - Interface in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">
 Defines the interface for a Chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#CHAIN_NR_SYMBOL">CHAIN_NR_SYMBOL</a></span> - Static variable in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#CHAIN_SPLIT_SYMBOL">CHAIN_SPLIT_SYMBOL</a></span> - Static variable in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure"><span class="strong">ChainImpl</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">A Chain in a PDB file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#ChainImpl()">ChainImpl()</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Constructs a ChainImpl object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#charged">charged</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/DownloadChemCompProvider.html#checkDoFirstInstall()">checkDoFirstInstall()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/DownloadChemCompProvider.html" title="class in org.biojava.bio.structure.io.mmcif">DownloadChemCompProvider</a></dt>
<dd>
<div class="block">checks if the chemical components already have been installed into the PDB directory.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/AllChemCompProvider.html#checkPath()">checkPath()</a></span> - Static method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/AllChemCompProvider.html" title="class in org.biojava.bio.structure.io.mmcif">AllChemCompProvider</a></dt>
<dd>
<div class="block">make sure all paths are initialized correctly</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ronn/Timer.html#checkPoint()">checkPoint()</a></span> - Method in class org.biojava3.ronn.<a href="./org/biojava3/ronn/Timer.html" title="class in org.biojava3.ronn">Timer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/Utils.html#checkSequence(java.lang.String)">checkSequence(String)</a></span> - Static method in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/Utils.html" title="class in org.biojava3.aaproperties">Utils</a></dt>
<dd>
<div class="block">Checks if the sequence contains invalid characters.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/Utils.html#checkSequence(java.lang.String, java.util.Set)">checkSequence(String, Set&lt;Character&gt;)</a></span> - Static method in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/Utils.html" title="class in org.biojava3.aaproperties">Utils</a></dt>
<dd>
<div class="block">Checks if the sequence contains invalid characters.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceMixin.html#checksum(org.biojava3.core.sequence.template.Sequence)">checksum(Sequence&lt;C&gt;)</a></span> - Static method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceMixin.html" title="class in org.biojava3.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">Performs a simple CRC64 checksum on any given sequence.</div>
</dd>
<dt><a href="./org/biojava3/phylo/CheckTreeAccuracy.html" title="class in org.biojava3.phylo"><span class="strong">CheckTreeAccuracy</span></a> - Class in <a href="./org/biojava3/phylo/package-summary.html">org.biojava3.phylo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/CheckTreeAccuracy.html#CheckTreeAccuracy()">CheckTreeAccuracy()</a></span> - Constructor for class org.biojava3.phylo.<a href="./org/biojava3/phylo/CheckTreeAccuracy.html" title="class in org.biojava3.phylo">CheckTreeAccuracy</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/DownloadChemCompProvider.html#CHEM_COMP_CACHE_DIRECTORY">CHEM_COMP_CACHE_DIRECTORY</a></span> - Static variable in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/DownloadChemCompProvider.html" title="class in org.biojava.bio.structure.io.mmcif">DownloadChemCompProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/chem/ResidueType.html#chem_comp_type">chem_comp_type</a></span> - Variable in enum org.biojava.bio.structure.io.mmcif.chem.<a href="./org/biojava/bio/structure/io/mmcif/chem/ResidueType.html" title="enum in org.biojava.bio.structure.io.mmcif.chem">ResidueType</a></dt>
<dd>
<div class="block">String value of the type</div>
</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model"><span class="strong">ChemComp</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a></dt>
<dd>
<div class="block">A definition for a Chemical Component, as maintained by the wwPDB.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#ChemComp()">ChemComp()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif"><span class="strong">ChemCompConsumer</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/package-summary.html">org.biojava.bio.structure.io.mmcif</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#ChemCompConsumer()">ChemCompConsumer()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html" title="class in org.biojava.bio.structure.io.mmcif.model"><span class="strong">ChemCompDescriptor</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a></dt>
<dd>
<div class="block">Container object for _pdbx_chem_comp_descriptor</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html#ChemCompDescriptor()">ChemCompDescriptor()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemCompDescriptor</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./demo/ChemCompDistribution.html" title="class in demo"><span class="strong">ChemCompDistribution</span></a> - Class in <a href="./demo/package-summary.html">demo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/ChemCompDistribution.html#ChemCompDistribution()">ChemCompDistribution()</a></span> - Constructor for class demo.<a href="./demo/ChemCompDistribution.html" title="class in demo">ChemCompDistribution</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/ChemCompGroupFactory.html" title="class in org.biojava.bio.structure.io.mmcif"><span class="strong">ChemCompGroupFactory</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/package-summary.html">org.biojava.bio.structure.io.mmcif</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompGroupFactory.html#ChemCompGroupFactory()">ChemCompGroupFactory()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompGroupFactory.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompGroupFactory</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/ChemCompProvider.html" title="interface in org.biojava.bio.structure.io.mmcif"><span class="strong">ChemCompProvider</span></a> - Interface in <a href="./org/biojava/bio/structure/io/mmcif/package-summary.html">org.biojava.bio.structure.io.mmcif</a></dt>
<dd>
<div class="block">Interface that is implemented by all classes that can provide <a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model"><code>ChemComp</code></a> definitions.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/chem/ChemCompTools.html" title="class in org.biojava.bio.structure.io.mmcif.chem"><span class="strong">ChemCompTools</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/chem/package-summary.html">org.biojava.bio.structure.io.mmcif.chem</a></dt>
<dd>
<div class="block">Some tools for working with chemical compounds.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/chem/ChemCompTools.html#ChemCompTools()">ChemCompTools()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.chem.<a href="./org/biojava/bio/structure/io/mmcif/chem/ChemCompTools.html" title="class in org.biojava.bio.structure.io.mmcif.chem">ChemCompTools</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html" title="class in org.biojava.bio.structure.io.mmcif"><span class="strong">ChemicalComponentDictionary</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/package-summary.html">org.biojava.bio.structure.io.mmcif</a></dt>
<dd>
<div class="block">A representation of the Chemical Component Dictionary.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html#ChemicalComponentDictionary()">ChemicalComponentDictionary()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html" title="class in org.biojava.bio.structure.io.mmcif">ChemicalComponentDictionary</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/GuideTree.Node.html#children()">children()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/GuideTree.Node.html" title="class in org.biojava3.alignment">GuideTree.Node</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#chol()">chol()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Cholesky Decomposition</div>
</dd>
<dt><a href="./org/biojava/bio/structure/jama/CholeskyDecomposition.html" title="class in org.biojava.bio.structure.jama"><span class="strong">CholeskyDecomposition</span></a> - Class in <a href="./org/biojava/bio/structure/jama/package-summary.html">org.biojava.bio.structure.jama</a></dt>
<dd>
<div class="block">Cholesky Decomposition.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/CholeskyDecomposition.html#CholeskyDecomposition(org.biojava.bio.structure.jama.Matrix)">CholeskyDecomposition(Matrix)</a></span> - Constructor for class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/CholeskyDecomposition.html" title="class in org.biojava.bio.structure.jama">CholeskyDecomposition</a></dt>
<dd>
<div class="block">Cholesky algorithm for symmetric and positive definite matrix.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/gui/ChooseDirAction.html" title="class in org.biojava.bio.structure.align.gui"><span class="strong">ChooseDirAction</span></a> - Class in <a href="./org/biojava/bio/structure/align/gui/package-summary.html">org.biojava.bio.structure.align.gui</a></dt>
<dd>
<div class="block">Ask the user to provide a directory containting PDB files.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/ChooseDirAction.html#ChooseDirAction(javax.swing.JTextField, org.biojava.bio.structure.align.util.UserConfiguration)">ChooseDirAction(JTextField, UserConfiguration)</a></span> - Constructor for class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/ChooseDirAction.html" title="class in org.biojava.bio.structure.align.gui">ChooseDirAction</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/ChromosomeSequence.html" title="class in org.biojava3.core.sequence"><span class="strong">ChromosomeSequence</span></a> - Class in <a href="./org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a></dt>
<dd>
<div class="block">A ChromosomeSequence is a DNASequence but keeps track of geneSequences</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/ChromosomeSequence.html#ChromosomeSequence()">ChromosomeSequence()</a></span> - Constructor for class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/ChromosomeSequence.html" title="class in org.biojava3.core.sequence">ChromosomeSequence</a></dt>
<dd>
<div class="block">Empty constructor used by tools that need a proper Bean that allows the actual
 sequence data to be set after construction.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/ChromosomeSequence.html#ChromosomeSequence(java.lang.String)">ChromosomeSequence(String)</a></span> - Constructor for class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/ChromosomeSequence.html" title="class in org.biojava3.core.sequence">ChromosomeSequence</a></dt>
<dd>
<div class="block">String is king and assume DNA</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/ChromosomeSequence.html#ChromosomeSequence(org.biojava3.core.sequence.template.SequenceReader)">ChromosomeSequence(SequenceReader&lt;NucleotideCompound&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/ChromosomeSequence.html" title="class in org.biojava3.core.sequence">ChromosomeSequence</a></dt>
<dd>
<div class="block">Fairly important constructor given the size of a ChromsomeSequence where the
 ProxySequenceReader could load from disk via RandomAccessFile so that the sequence
 doesn't need to be kept in memory.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/ChromosomeSequence.html#ChromosomeSequence(java.lang.String, org.biojava3.core.sequence.template.CompoundSet)">ChromosomeSequence(String, CompoundSet&lt;NucleotideCompound&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/ChromosomeSequence.html" title="class in org.biojava3.core.sequence">ChromosomeSequence</a></dt>
<dd>
<div class="block">Allows the creation of a ChromosomeSequence using String for the sequence with a custom CompoundSet</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/ChromosomeSequence.html#ChromosomeSequence(org.biojava3.core.sequence.template.SequenceReader, org.biojava3.core.sequence.template.CompoundSet)">ChromosomeSequence(SequenceReader&lt;NucleotideCompound&gt;, CompoundSet&lt;NucleotideCompound&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/ChromosomeSequence.html" title="class in org.biojava3.core.sequence">ChromosomeSequence</a></dt>
<dd>
<div class="block">Allows the creation of a ChromosomeSequence using a ProxyResequenceReader for the sequence with a custom CompoundSet</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/LocationHelper.html#circularLocation(int, int, org.biojava3.core.sequence.Strand, int)">circularLocation(int, int, Strand, int)</a></span> - Static method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/LocationHelper.html" title="class in org.biojava3.core.sequence.location">LocationHelper</a></dt>
<dd>
<div class="block">Converts a location which defines the outer bounds of a circular
 location and splits it into the required portions.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/Location.Tools.html#circularLocation(int, int, org.biojava3.core.sequence.Strand, int)">circularLocation(int, int, Strand, int)</a></span> - Static method in class org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/Location.Tools.html" title="class in org.biojava3.core.sequence.location.template">Location.Tools</a></dt>
<dd>
<div class="block">Converts a location which defines the outer bounds of a circular
 location and splits it into the required portions.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#claFileName">claFileName</a></span> - Static variable in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/Equals.html#classEqual(java.lang.Object, java.lang.Object)">classEqual(Object, Object)</a></span> - Static method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/Equals.html" title="class in org.biojava3.core.util">Equals</a></dt>
<dd>
<div class="block">This method should be called before beginning any equals methods.</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/io/util/ClasspathResource.html" title="class in org.biojava3.core.sequence.io.util"><span class="strong">ClasspathResource</span></a> - Class in <a href="./org/biojava3/core/sequence/io/util/package-summary.html">org.biojava3.core.sequence.io.util</a></dt>
<dd>
<div class="block">This object represents a classpath resource on the local system.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/util/ClasspathResource.html#ClasspathResource(java.lang.String)">ClasspathResource(String)</a></span> - Constructor for class org.biojava3.core.sequence.io.util.<a href="./org/biojava3/core/sequence/io/util/ClasspathResource.html" title="class in org.biojava3.core.sequence.io.util">ClasspathResource</a></dt>
<dd>
<div class="block">Basic constructor only allowing you to specify where to find the file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/util/ClasspathResource.html#ClasspathResource(java.lang.String, boolean)">ClasspathResource(String, boolean)</a></span> - Constructor for class org.biojava3.core.sequence.io.util.<a href="./org/biojava3/core/sequence/io/util/ClasspathResource.html" title="class in org.biojava3.core.sequence.io.util">ClasspathResource</a></dt>
<dd>
<div class="block">Advanced constructor which allows you to optionally pre-cache the
 data</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/util/ClasspathResource.html#ClasspathResource(java.lang.String, boolean, boolean)">ClasspathResource(String, boolean, boolean)</a></span> - Constructor for class org.biojava3.core.sequence.io.util.<a href="./org/biojava3/core/sequence/io/util/ClasspathResource.html" title="class in org.biojava3.core.sequence.io.util">ClasspathResource</a></dt>
<dd>
<div class="block">Advanced constructor which lets you set the preCache variable and to
 force the type of file we are decompressing.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/Utils.html#cleanSequence(java.lang.String, java.util.Set)">cleanSequence(String, Set&lt;Character&gt;)</a></span> - Static method in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/Utils.html" title="class in org.biojava3.aaproperties">Utils</a></dt>
<dd>
<div class="block">Returns a new sequence with all invalid characters being replaced by '-'.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/data/sequence/SequenceUtil.html#cleanSequence(java.lang.String)">cleanSequence(String)</a></span> - Static method in class org.biojava3.data.sequence.<a href="./org/biojava3/data/sequence/SequenceUtil.html" title="class in org.biojava3.data.sequence">SequenceUtil</a></dt>
<dd>
<div class="block">Removes all whitespace chars in the sequence string</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/AlignmentCalc.html#cleanup()">cleanup()</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/AlignmentCalc.html" title="class in org.biojava.bio.structure.align.gui">AlignmentCalc</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/AlignmentCalcDB.html#cleanup()">cleanup()</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/AlignmentCalcDB.html" title="class in org.biojava.bio.structure.align.gui">AlignmentCalcDB</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/AlignmentCalculationRunnable.html#cleanup()">cleanup()</a></span> - Method in interface org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/AlignmentCalculationRunnable.html" title="interface in org.biojava.bio.structure.align.gui">AlignmentCalculationRunnable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/AlignmentGui.html#cleanUp()">cleanUp()</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/AlignmentGui.html" title="class in org.biojava.bio.structure.align.gui">AlignmentGui</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/ShortSegmentRemover.html#cleanup(java.util.List)">cleanup(List&lt;Domain&gt;)</a></span> - Static method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/ShortSegmentRemover.html" title="class in org.biojava.bio.structure.domain.pdp">ShortSegmentRemover</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/AlignmentCalc.html#cleanup()">cleanup()</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/AlignmentCalc.html" title="class in org.biojava.bio.structure.gui.util">AlignmentCalc</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html#clear()">clear()</a></span> - Method in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html" title="class in org.biojava.bio.structure.gui.util.color">GradientMapper</a></dt>
<dd>
<div class="block">Clears all finite endpoints</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/FlatFileCache.html#clear()">clear()</a></span> - Method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/FlatFileCache.html" title="class in org.biojava3.core.util">FlatFileCache</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/SoftHashMap.html#clear()">clear()</a></span> - Method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/SoftHashMap.html" title="class in org.biojava3.core.util">SoftHashMap</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/structure/gui/JmolViewerImpl.html#clear()">clear()</a></span> - Method in class org.biojava3.structure.gui.<a href="./org/biojava3/structure/gui/JmolViewerImpl.html" title="class in org.biojava3.structure.gui">JmolViewerImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/structure/gui/Selection.html#clear()">clear()</a></span> - Method in interface org.biojava3.structure.gui.<a href="./org/biojava3/structure/gui/Selection.html" title="interface in org.biojava3.structure.gui">Selection</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/structure/gui/SelectionImpl.html#clear()">clear()</a></span> - Method in class org.biojava3.structure.gui.<a href="./org/biojava3/structure/gui/SelectionImpl.html" title="class in org.biojava3.structure.gui">SelectionImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/structure/gui/StructureViewer.html#clear()">clear()</a></span> - Method in interface org.biojava3.structure.gui.<a href="./org/biojava3/structure/gui/StructureViewer.html" title="interface in org.biojava3.structure.gui">StructureViewer</a></dt>
<dd>
<div class="block">Clear the current display</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructureAlignmentFactory.html#clearAlgorithms()">clearAlgorithms()</a></span> - Static method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructureAlignmentFactory.html" title="class in org.biojava.bio.structure.align">StructureAlignmentFactory</a></dt>
<dd>
<div class="block">Removes all algorithms from the list</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#clearAtoms()">clearAtoms()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">Remove all atoms from this group.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#clearAtoms()">clearAtoms()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">remove all atoms</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBStatus.html#clearCache()">clearCache()</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBStatus.html" title="class in org.biojava.bio.structure">PDBStatus</a></dt>
<dd>
<div class="block">The status of PDB IDs are cached to reduce server overload.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleAlignedSequence.html#clearCache()">clearCache()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleAlignedSequence.html" title="class in org.biojava3.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AlignedSequence.html#clearCache()">clearCache()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AlignedSequence.html" title="interface in org.biojava3.alignment.template">AlignedSequence</a></dt>
<dd>
<div class="block">Nullifies cached arrays/objects.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifParser.html#clearConsumers()">clearConsumers()</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifParser.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifParser</a></dt>
<dd>
<div class="block">Remove all consumers from the parser.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#clearConsumers()">clearConsumers()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/MMCIFFileReader.html#clearExtensions()">clearExtensions()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/MMCIFFileReader.html" title="class in org.biojava.bio.structure.io">MMCIFFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#clearExtensions()">clearExtensions()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>
<div class="block">clear the supported file extensions</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/StructureIOFile.html#clearExtensions()">clearExtensions()</a></span> - Method in interface org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/StructureIOFile.html" title="interface in org.biojava.bio.structure.io">StructureIOFile</a></dt>
<dd>
<div class="block">clear all file extensions</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/FlatFileInstallation.html#clearFilters()">clearFilters()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/FlatFileInstallation.html" title="class in org.biojava.bio.structure.server">FlatFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html#clearFilters()">clearFilters()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html" title="class in org.biojava.bio.structure.server">MMCIFFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/PDBInstallation.html#clearFilters()">clearFilters()</a></span> - Method in interface org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/PDBInstallation.html" title="interface in org.biojava.bio.structure.server">PDBInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
<div class="block">remove all filters, next time getAll is called, it will return all available PDBs</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html#clearListeners()">clearListeners()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html" title="class in org.biojava.bio.structure.align.client">FarmJobRunnable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/FarmJob.html#clearListeners()">clearListeners()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/FarmJob.html" title="class in org.biojava.bio.structure.align">FarmJob</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructurePairAligner.html#clearListeners()">clearListeners()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructurePairAligner.html" title="class in org.biojava.bio.structure.align">StructurePairAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/SequenceDisplay.html#clearListeners()">clearListeners()</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/SequenceDisplay.html" title="class in org.biojava.bio.structure.gui">SequenceDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/SequenceMouseListener.html#clearListeners()">clearListeners()</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/SequenceMouseListener.html" title="class in org.biojava.bio.structure.gui.util">SequenceMouseListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureFetcherRunnable.html#clearListeners()">clearListeners()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureFetcherRunnable.html" title="class in org.biojava.bio.structure.server">StructureFetcherRunnable</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/SimpleStructureServer.html#clearStructureListeners()">clearStructureListeners()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/SimpleStructureServer.html" title="class in org.biojava.bio.structure.server">SimpleStructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureServer.html#clearStructureListeners()">clearStructureListeners()</a></span> - Method in interface org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureServer.html" title="interface in org.biojava.bio.structure.server">StructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/util/CliTools.html" title="class in org.biojava.bio.structure.align.util"><span class="strong">CliTools</span></a> - Class in <a href="./org/biojava/bio/structure/align/util/package-summary.html">org.biojava.bio.structure.align.util</a></dt>
<dd>
<div class="block">Utilities for autoconfiguring javabeans based on command line arguments.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#clone()">clone()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">Creates and returns a copy of this object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#clone()">clone()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcidImpl.html#clone()">clone()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcidImpl.html" title="class in org.biojava.bio.structure">AminoAcidImpl</a></dt>
<dd>
<div class="block">returns and identical copy of this Group object .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#clone()">clone()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">returns and identical copy of this  object .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#clone()">clone()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>
<div class="block">returns and identical copy of this  object .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#clone()">clone()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">returns an identical copy of this Chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#clone()">clone()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Returns an identical copy of this Chain .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#clone()">clone()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#clone()">clone()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">returns and identical copy of this Group object .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/GroupIterator.html#clone()">clone()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/GroupIterator.html" title="class in org.biojava.bio.structure">GroupIterator</a></dt>
<dd>
<div class="block">Creates and returns a copy of this object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#clone()">clone()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">returns and identical copy of this Group object .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#clone()">clone()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Clone the Matrix object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/NucleotideImpl.html#clone()">clone()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/NucleotideImpl.html" title="class in org.biojava.bio.structure">NucleotideImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/HBond.html#clone()">clone()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/HBond.html" title="class in org.biojava.bio.structure.secstruc">HBond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SSBond.html#clone()">clone()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SSBond.html" title="class in org.biojava.bio.structure">SSBond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#clone()">clone()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">returns an identical copy of this Structure object</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#clone()">clone()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">returns an identical copy of this structure .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#cloneCAArray(org.biojava.bio.structure.Atom[])">cloneCAArray(Atom[])</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>
<div class="block">Provides an equivalent copy of Atoms in a new array.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#cloneGroups(org.biojava.bio.structure.Atom[])">cloneGroups(Atom[])</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>
<div class="block">Clone a set of CA Atoms, but returns the parent groups</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/SimplePoint.html#clonePoint()">clonePoint()</a></span> - Method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/SimplePoint.html" title="class in org.biojava3.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/Point.html#clonePoint()">clonePoint()</a></span> - Method in interface org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/Point.html" title="interface in org.biojava3.core.sequence.location.template">Point</a></dt>
<dd>
<div class="block">Returns a copy of this point</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/SynchronizedOutFile.html#close()">close()</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/SynchronizedOutFile.html" title="class in org.biojava.bio.structure.align.util">SynchronizedOutFile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/BufferedReaderBytesRead.html#close()">close()</a></span> - Method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/BufferedReaderBytesRead.html" title="class in org.biojava3.core.sequence.io">BufferedReaderBytesRead</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/util/IOUtils.html#close(java.io.Closeable)">close(Closeable)</a></span> - Static method in class org.biojava3.core.sequence.io.util.<a href="./org/biojava3/core/sequence/io/util/IOUtils.html" title="class in org.biojava3.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">Closes any Object which implements the interface <a href="http://java.sun.com/j2se/1.6.0/docs/api/java/io/Closeable.html?is-external=true" title="class or interface in java.io"><code>Closeable</code></a> and
 sending any error to the logger but not forcing any explicit catching of
 stream errors.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/PrettyXMLWriter.html#close()">close()</a></span> - Method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/PrettyXMLWriter.html" title="class in org.biojava3.core.util">PrettyXMLWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/XMLWriter.html#close()">close()</a></span> - Method in interface org.biojava3.core.util.<a href="./org/biojava3/core/util/XMLWriter.html" title="interface in org.biojava3.core.util">XMLWriter</a></dt>
<dd>
<div class="block">Close this XMLWriter, and it's underlying stream.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/PrettyXMLWriter.html#closeTag(java.lang.String, java.lang.String)">closeTag(String, String)</a></span> - Method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/PrettyXMLWriter.html" title="class in org.biojava3.core.util">PrettyXMLWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/PrettyXMLWriter.html#closeTag(java.lang.String)">closeTag(String)</a></span> - Method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/PrettyXMLWriter.html" title="class in org.biojava3.core.util">PrettyXMLWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/XMLWriter.html#closeTag(java.lang.String, java.lang.String)">closeTag(String, String)</a></span> - Method in interface org.biojava3.core.util.<a href="./org/biojava3/core/util/XMLWriter.html" title="interface in org.biojava3.core.util">XMLWriter</a></dt>
<dd>
<div class="block">Closes an element</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/XMLWriter.html#closeTag(java.lang.String)">closeTag(String)</a></span> - Method in interface org.biojava3.core.util.<a href="./org/biojava3/core/util/XMLWriter.html" title="interface in org.biojava3.core.util">XMLWriter</a></dt>
<dd>
<div class="block">Closes an un-qualified element.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ClusterAltAligs.html#cluster(org.biojava.bio.structure.align.pairwise.AlternativeAlignment[])">cluster(AlternativeAlignment[])</a></span> - Static method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/ClusterAltAligs.html" title="class in org.biojava.bio.structure.align">ClusterAltAligs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ClusterAltAligs.html#cluster(org.biojava.bio.structure.align.pairwise.AlternativeAlignment[], int)">cluster(AlternativeAlignment[], int)</a></span> - Static method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/ClusterAltAligs.html" title="class in org.biojava.bio.structure.align">ClusterAltAligs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/ClusterDomains.html#cluster(java.util.List, org.biojava.bio.structure.domain.pdp.PDPDistanceMatrix)">cluster(List&lt;Domain&gt;, PDPDistanceMatrix)</a></span> - Static method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/ClusterDomains.html" title="class in org.biojava.bio.structure.domain.pdp">ClusterDomains</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/ClusterAltAligs.html" title="class in org.biojava.bio.structure.align"><span class="strong">ClusterAltAligs</span></a> - Class in <a href="./org/biojava/bio/structure/align/package-summary.html">org.biojava.bio.structure.align</a></dt>
<dd>
<div class="block">A class that clusters alternative alignments according to their
 similarity.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ClusterAltAligs.html#ClusterAltAligs()">ClusterAltAligs()</a></span> - Constructor for class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/ClusterAltAligs.html" title="class in org.biojava.bio.structure.align">ClusterAltAligs</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/domain/pdp/ClusterDomains.html" title="class in org.biojava.bio.structure.domain.pdp"><span class="strong">ClusterDomains</span></a> - Class in <a href="./org/biojava/bio/structure/domain/pdp/package-summary.html">org.biojava.bio.structure.domain.pdp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/ClusterDomains.html#ClusterDomains()">ClusterDomains()</a></span> - Constructor for class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/ClusterDomains.html" title="class in org.biojava.bio.structure.domain.pdp">ClusterDomains</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/compound/CodonCompound.html" title="class in org.biojava3.core.sequence.compound"><span class="strong">CodonCompound</span></a> - Class in <a href="./org/biojava3/core/sequence/compound/package-summary.html">org.biojava3.core.sequence.compound</a></dt>
<dd>
<div class="block">Define a codon</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/CodonCompound.html#CodonCompound(org.biojava3.core.sequence.compound.NucleotideCompound, org.biojava3.core.sequence.compound.NucleotideCompound, org.biojava3.core.sequence.compound.NucleotideCompound, boolean)">CodonCompound(NucleotideCompound, NucleotideCompound, NucleotideCompound, boolean)</a></span> - Constructor for class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/CodonCompound.html" title="class in org.biojava3.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#codonHash">codonHash</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#codonHash2">codonHash2</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#codonTranslate(java.lang.String)">codonTranslate(String)</a></span> - Static method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/util/CollectionTools.html" title="class in org.biojava.bio.structure.align.util"><span class="strong">CollectionTools</span></a> - Class in <a href="./org/biojava/bio/structure/align/util/package-summary.html">org.biojava.bio.structure.align.util</a></dt>
<dd>
<div class="block">Utilities for working with collections.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/CollectionTools.html#CollectionTools()">CollectionTools()</a></span> - Constructor for class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/CollectionTools.html" title="class in org.biojava.bio.structure.align.util">CollectionTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html#coloring">coloring</a></span> - Variable in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html" title="class in org.biojava.bio.structure.gui">ScaleableMatrixPanel</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/gui/util/color/ColorInterpolator.html" title="interface in org.biojava.bio.structure.gui.util.color"><span class="strong">ColorInterpolator</span></a> - Interface in <a href="./org/biojava/bio/structure/gui/util/color/package-summary.html">org.biojava.bio.structure.gui.util.color</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/gui/util/color/ColorUtils.html" title="class in org.biojava.bio.structure.gui.util.color"><span class="strong">ColorUtils</span></a> - Class in <a href="./org/biojava/bio/structure/gui/util/color/package-summary.html">org.biojava.bio.structure.gui.util.color</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/ColorUtils.html#ColorUtils()">ColorUtils()</a></span> - Constructor for class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/ColorUtils.html" title="class in org.biojava.bio.structure.gui.util.color">ColorUtils</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/ColorUtils.html#colorWheel">colorWheel</a></span> - Static variable in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/ColorUtils.html" title="class in org.biojava.bio.structure.gui.util.color">ColorUtils</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html#COMMAND_LINE_HELP">COMMAND_LINE_HELP</a></span> - Static variable in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html" title="class in org.biojava.bio.structure.align.gui.jmol">StructureAlignmentJmol</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/aaproperties/CommandPrompt.html" title="class in org.biojava3.aaproperties"><span class="strong">CommandPrompt</span></a> - Class in <a href="./org/biojava3/aaproperties/package-summary.html">org.biojava3.aaproperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/CommandPrompt.html#CommandPrompt()">CommandPrompt()</a></span> - Constructor for class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/CommandPrompt.html" title="class in org.biojava3.aaproperties">CommandPrompt</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/aaproperties/CommandPrompt.PropertyName.html" title="enum in org.biojava3.aaproperties"><span class="strong">CommandPrompt.PropertyName</span></a> - Enum in <a href="./org/biojava3/aaproperties/package-summary.html">org.biojava3.aaproperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/IdxComparator.html#compare(int[], int[])">compare(int[], int[])</a></span> - Method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/IdxComparator.html" title="class in org.biojava.bio.structure.align.helper">IdxComparator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AltAligComparator.html#compare(org.biojava.bio.structure.align.pairwise.AlternativeAlignment, org.biojava.bio.structure.align.pairwise.AlternativeAlignment)">compare(AlternativeAlignment, AlternativeAlignment)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AltAligComparator.html" title="class in org.biojava.bio.structure.align.pairwise">AltAligComparator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/SegmentComparator.html#compare(org.biojava.bio.structure.domain.pdp.Segment, org.biojava.bio.structure.domain.pdp.Segment)">compare(Segment, Segment)</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/SegmentComparator.html" title="class in org.biojava.bio.structure.domain.pdp">SegmentComparator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/CDSComparator.html#compare(org.biojava3.core.sequence.CDSSequence, org.biojava3.core.sequence.CDSSequence)">compare(CDSSequence, CDSSequence)</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/CDSComparator.html" title="class in org.biojava3.core.sequence">CDSComparator</a></dt>
<dd>
<div class="block">Used to sort two CDSSequences where Negative Strand makes it tough</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/ExonComparator.html#compare(org.biojava3.core.sequence.ExonSequence, org.biojava3.core.sequence.ExonSequence)">compare(ExonSequence, ExonSequence)</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/ExonComparator.html" title="class in org.biojava3.core.sequence">ExonComparator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/SequenceComparator.html#compare(org.biojava3.core.sequence.template.AbstractSequence, org.biojava3.core.sequence.template.AbstractSequence)">compare(AbstractSequence&lt;?&gt;, AbstractSequence&lt;?&gt;)</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/SequenceComparator.html" title="class in org.biojava3.core.sequence">SequenceComparator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/PdbPair.html#compareTo(org.biojava.bio.structure.align.client.PdbPair)">compareTo(PdbPair)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/PdbPair.html" title="class in org.biojava.bio.structure.align.client">PdbPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/StructureName.html#compareTo(org.biojava.bio.structure.align.client.StructureName)">compareTo(StructureName)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/StructureName.html" title="class in org.biojava.bio.structure.align.client">StructureName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Domain.html#compareTo(org.biojava.bio.structure.domain.pdp.Domain)">compareTo(Domain)</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Domain.html" title="class in org.biojava.bio.structure.domain.pdp">Domain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Segment.html#compareTo(org.biojava.bio.structure.domain.pdp.Segment)">compareTo(Segment)</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Segment.html" title="class in org.biojava.bio.structure.domain.pdp">Segment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#compareTo(org.biojava.bio.structure.io.mmcif.model.ChemComp)">compareTo(ChemComp)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Site.html#compareTo(org.biojava.bio.structure.Site)">compareTo(Site)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Site.html" title="class in org.biojava.bio.structure">Site</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/FuzzyPoint.html#compareTo(org.biojava3.core.sequence.location.template.Point)">compareTo(Point)</a></span> - Method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/FuzzyPoint.html" title="class in org.biojava3.core.sequence.location">FuzzyPoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/SimplePoint.html#compareTo(org.biojava3.core.sequence.location.template.Point)">compareTo(Point)</a></span> - Method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/SimplePoint.html" title="class in org.biojava3.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/data/sequence/FastaSequence.html#compareTo(org.biojava3.data.sequence.FastaSequence)">compareTo(FastaSequence)</a></span> - Method in class org.biojava3.data.sequence.<a href="./org/biojava3/data/sequence/FastaSequence.html" title="class in org.biojava3.data.sequence">FastaSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationImpl.html#compareTo(org.biojava3.protmod.ProteinModification)">compareTo(ProteinModification)</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationImpl.html" title="class in org.biojava3.protmod">ProteinModificationImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ModifiedCompoundImpl.html#compareTo(org.biojava3.protmod.structure.ModifiedCompound)">compareTo(ModifiedCompound)</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ModifiedCompoundImpl.html" title="class in org.biojava3.protmod.structure">ModifiedCompoundImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/StructureGroup.html#compareTo(org.biojava3.protmod.structure.StructureGroup)">compareTo(StructureGroup)</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/StructureGroup.html" title="class in org.biojava3.protmod.structure">StructureGroup</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/phylo/Comparison.html" title="class in org.biojava3.phylo"><span class="strong">Comparison</span></a> - Class in <a href="./org/biojava3/phylo/package-summary.html">org.biojava3.phylo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/Comparison.html#Comparison()">Comparison()</a></span> - Constructor for class org.biojava3.phylo.<a href="./org/biojava3/phylo/Comparison.html" title="class in org.biojava3.phylo">Comparison</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/template/ComplementCompound.html" title="interface in org.biojava3.core.sequence.template"><span class="strong">ComplementCompound</span></a> - Interface in <a href="./org/biojava3/core/sequence/template/package-summary.html">org.biojava3.core.sequence.template</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/views/ComplementSequenceView.html" title="class in org.biojava3.core.sequence.views"><span class="strong">ComplementSequenceView</span></a>&lt;<a href="./org/biojava3/core/sequence/views/ComplementSequenceView.html" title="type parameter in ComplementSequenceView">C</a> extends <a href="./org/biojava3/core/sequence/template/ComplementCompound.html" title="interface in org.biojava3.core.sequence.template">ComplementCompound</a>&gt; - Class in <a href="./org/biojava3/core/sequence/views/package-summary.html">org.biojava3.core.sequence.views</a></dt>
<dd>
<div class="block">For a given sequence this class will create a view over the top of it
 and for every request the code will return the complement of the underlying
 base e.g. base A will become base T</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/views/ComplementSequenceView.html#ComplementSequenceView(org.biojava3.core.sequence.template.Sequence)">ComplementSequenceView(Sequence&lt;C&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.views.<a href="./org/biojava3/core/sequence/views/ComplementSequenceView.html" title="class in org.biojava3.core.sequence.views">ComplementSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/NJTreeProgressListener.html#complete(java.lang.Object)">complete(Object)</a></span> - Method in interface org.biojava3.phylo.<a href="./org/biojava3/phylo/NJTreeProgressListener.html" title="interface in org.biojava3.phylo">NJTreeProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ProgessListenerStub.html#complete(java.lang.Object)">complete(Object)</a></span> - Method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ProgessListenerStub.html" title="class in org.biojava3.phylo">ProgessListenerStub</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/LocationHelper.html#completeCircularPasses(int, int)">completeCircularPasses(int, int)</a></span> - Static method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/LocationHelper.html" title="class in org.biojava3.core.sequence.location">LocationHelper</a></dt>
<dd>
<div class="block">Works in a similar way to modulateCircularLocation but returns
 the number of complete passes over a Sequence length a circular
 location makes i.e. if we have a sequence of length 10 and the
 location 3..52 we make 4 complete passes through the genome to
 go from position 3 to position 52.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/Location.Tools.html#completeCircularPasses(int, int)">completeCircularPasses(int, int)</a></span> - Static method in class org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/Location.Tools.html" title="class in org.biojava3.core.sequence.location.template">Location.Tools</a></dt>
<dd>
<div class="block">Works in a similar way to modulateCircularLocation but returns
 the number of complete passes over a Sequence length a circular
 location makes i.e. if we have a sequence of length 10 and the
 location 3..52 we make 4 complete passes through the genome to
 go from position 3 to position 52.</div>
</dd>
<dt><a href="./org/biojava3/protmod/Component.html" title="class in org.biojava3.protmod"><span class="strong">Component</span></a> - Class in <a href="./org/biojava3/protmod/package-summary.html">org.biojava3.protmod</a></dt>
<dd>
<div class="block">contains information about a certain Component.</div>
</dd>
<dt><a href="./org/biojava3/protmod/io/ComponentXMLConverter.html" title="class in org.biojava3.protmod.io"><span class="strong">ComponentXMLConverter</span></a> - Class in <a href="./org/biojava3/protmod/io/package-summary.html">org.biojava3.protmod.io</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/io/ComponentXMLConverter.html#ComponentXMLConverter()">ComponentXMLConverter()</a></span> - Constructor for class org.biojava3.protmod.io.<a href="./org/biojava3/protmod/io/ComponentXMLConverter.html" title="class in org.biojava3.protmod.io">ComponentXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure"><span class="strong">Compound</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">Created by IntelliJ IDEA.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#Compound()">Compound()</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template"><span class="strong">Compound</span></a> - Interface in <a href="./org/biojava3/core/sequence/template/package-summary.html">org.biojava3.core.sequence.template</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/exceptions/CompoundNotFoundError.html" title="class in org.biojava3.core.exceptions"><span class="strong">CompoundNotFoundError</span></a> - Error in <a href="./org/biojava3/core/exceptions/package-summary.html">org.biojava3.core.exceptions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/exceptions/CompoundNotFoundError.html#CompoundNotFoundError(java.lang.CharSequence)">CompoundNotFoundError(CharSequence)</a></span> - Constructor for error org.biojava3.core.exceptions.<a href="./org/biojava3/core/exceptions/CompoundNotFoundError.html" title="class in org.biojava3.core.exceptions">CompoundNotFoundError</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompoundSet.html#compoundsEqual(C, C)">compoundsEqual(C, C)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava3.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/CaseFreeAminoAcidCompoundSet.html#compoundsEquivalent(org.biojava3.core.sequence.compound.AminoAcidCompound, org.biojava3.core.sequence.compound.AminoAcidCompound)">compoundsEquivalent(AminoAcidCompound, AminoAcidCompound)</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/CaseFreeAminoAcidCompoundSet.html" title="class in org.biojava3.aaproperties.xml">CaseFreeAminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/ModifiedAminoAcidCompoundSet.html#compoundsEquivalent(org.biojava3.core.sequence.compound.AminoAcidCompound, org.biojava3.core.sequence.compound.AminoAcidCompound)">compoundsEquivalent(AminoAcidCompound, AminoAcidCompound)</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/ModifiedAminoAcidCompoundSet.html" title="class in org.biojava3.aaproperties.xml">ModifiedAminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/AminoAcidCompoundSet.html#compoundsEquivalent(org.biojava3.core.sequence.compound.AminoAcidCompound, org.biojava3.core.sequence.compound.AminoAcidCompound)">compoundsEquivalent(AminoAcidCompound, AminoAcidCompound)</a></span> - Method in class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/AminoAcidCompoundSet.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompoundSet.html#compoundsEquivalent(C, C)">compoundsEquivalent(C, C)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava3.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/CompoundSet.html#compoundsEquivalent(C, C)">compoundsEquivalent(C, C)</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template"><span class="strong">CompoundSet</span></a>&lt;<a href="./org/biojava3/core/sequence/template/CompoundSet.html" title="type parameter in CompoundSet">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava3/core/sequence/template/package-summary.html">org.biojava3.core.sequence.template</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#compoundsPerDatatype()">compoundsPerDatatype()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava3.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>
<div class="block">Should return the maximum amount of compounds we can encode per int</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html#compoundsPerDatatype()">compoundsPerDatatype()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html" title="class in org.biojava3.core.sequence.storage">FourBitSequenceReader.FourBitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html#compoundsPerDatatype()">compoundsPerDatatype()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html" title="class in org.biojava3.core.sequence.storage">TwoBitSequenceReader.TwoBitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/Table.CaseInsensitiveTriplet.html#compoundToInt(org.biojava3.core.sequence.compound.NucleotideCompound)">compoundToInt(NucleotideCompound)</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/Table.CaseInsensitiveTriplet.html" title="class in org.biojava3.core.sequence.transcription">Table.CaseInsensitiveTriplet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/template/CompoundTranslator.html" title="interface in org.biojava3.core.sequence.template"><span class="strong">CompoundTranslator</span></a>&lt;<a href="./org/biojava3/core/sequence/template/CompoundTranslator.html" title="type parameter in CompoundTranslator">F</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>,<a href="./org/biojava3/core/sequence/template/CompoundTranslator.html" title="type parameter in CompoundTranslator">T</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava3/core/sequence/template/package-summary.html">org.biojava3.core.sequence.template</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/AminoAcidCompositionTable.html#computeMolecularWeight(org.biojava3.aaproperties.xml.ElementTable)">computeMolecularWeight(ElementTable)</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/AminoAcidCompositionTable.html" title="class in org.biojava3.aaproperties.xml">AminoAcidCompositionTable</a></dt>
<dd>
<div class="block">Computes and store the molecular weight of each amino acid by its symbol in aaSymbol2MolecularWeight.</div>
</dd>
<dt><a href="./org/biojava3/core/util/ConcurrencyTools.html" title="class in org.biojava3.core.util"><span class="strong">ConcurrencyTools</span></a> - Class in <a href="./org/biojava3/core/util/package-summary.html">org.biojava3.core.util</a></dt>
<dd>
<div class="block">Static utility to easily share a thread pool for concurrent/parallel/lazy execution.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#cond()">cond()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Matrix condition (2 norm)</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/SingularValueDecomposition.html#cond()">cond()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/SingularValueDecomposition.html" title="class in org.biojava.bio.structure.jama">SingularValueDecomposition</a></dt>
<dd>
<div class="block">Two norm condition number</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/gui/ConfigPDBInstallPanel.html" title="class in org.biojava.bio.structure.align.gui"><span class="strong">ConfigPDBInstallPanel</span></a> - Class in <a href="./org/biojava/bio/structure/align/gui/package-summary.html">org.biojava.bio.structure.align.gui</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/ce/ConfigStrucAligParams.html" title="interface in org.biojava.bio.structure.align.ce"><span class="strong">ConfigStrucAligParams</span></a> - Interface in <a href="./org/biojava/bio/structure/align/ce/package-summary.html">org.biojava.bio.structure.align.ce</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/util/ConfigurationException.html" title="class in org.biojava.bio.structure.align.util"><span class="strong">ConfigurationException</span></a> - Exception in <a href="./org/biojava/bio/structure/align/util/package-summary.html">org.biojava.bio.structure.align.util</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/ConfigurationException.html#ConfigurationException(java.lang.String)">ConfigurationException(String)</a></span> - Constructor for exception org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/ConfigurationException.html" title="class in org.biojava.bio.structure.align.util">ConfigurationException</a></dt>
<dd>
<div class="block">Constructs a ConfigurationException object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/ConfigurationException.html#ConfigurationException(java.lang.Throwable)">ConfigurationException(Throwable)</a></span> - Constructor for exception org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/ConfigurationException.html" title="class in org.biojava.bio.structure.align.util">ConfigurationException</a></dt>
<dd>
<div class="block">Constructs a ConfigurationException object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/CliTools.html#configureBean(java.lang.Object, java.lang.String[])">configureBean(Object, String[])</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/CliTools.html" title="class in org.biojava.bio.structure.align.util">CliTools</a></dt>
<dd>
<div class="block">Configure a JavaBean based on a set of command line arguments.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/AlignmentGui.html#configureParameters()">configureParameters()</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/AlignmentGui.html" title="class in org.biojava.bio.structure.align.gui">AlignmentGui</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/webstart/ConfigXMLHandler.html" title="class in org.biojava.bio.structure.align.webstart"><span class="strong">ConfigXMLHandler</span></a> - Class in <a href="./org/biojava/bio/structure/align/webstart/package-summary.html">org.biojava.bio.structure.align.webstart</a></dt>
<dd>
<div class="block">XML content handler for serialisation of RegistryConfiguration class</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/webstart/ConfigXMLHandler.html#ConfigXMLHandler()">ConfigXMLHandler()</a></span> - Constructor for class org.biojava.bio.structure.align.webstart.<a href="./org/biojava/bio/structure/align/webstart/ConfigXMLHandler.html" title="class in org.biojava.bio.structure.align.webstart">ConfigXMLHandler</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/LocationHelper.html#consistentAccessions(java.util.List)">consistentAccessions(List&lt;Location&gt;)</a></span> - Static method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/LocationHelper.html" title="class in org.biojava3.core.sequence.location">LocationHelper</a></dt>
<dd>
<div class="block">Scans a list of locations and returns true if all the given locations
 are linked to the same sequence.</div>
</dd>
<dt><a href="./org/biojava3/aaproperties/Constraints.html" title="class in org.biojava3.aaproperties"><span class="strong">Constraints</span></a> - Class in <a href="./org/biojava3/aaproperties/package-summary.html">org.biojava3.aaproperties</a></dt>
<dd>
<div class="block">This class is used to support the implementation of properties stated in IPeptideProperties.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/Constraints.html#Constraints()">Constraints()</a></span> - Constructor for class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/Constraints.html" title="class in org.biojava3.aaproperties">Constraints</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#constructWithCopy(double[][])">constructWithCopy(double[][])</a></span> - Static method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Construct a matrix from a copy of a 2-D array.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Location.html#contains(org.biojava3.genome.parsers.gff.Location)">contains(Location)</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Location.html" title="class in org.biojava3.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Check if this location contains the other.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html#containsKey(java.lang.Object)">containsKey(Object)</a></span> - Method in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html" title="class in org.biojava.bio.structure.gui.util.color">GradientMapper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html#containsValue(java.lang.Object)">containsValue(Object)</a></span> - Method in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html" title="class in org.biojava.bio.structure.gui.util.color">GradientMapper</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/gui/util/color/ContinuousColorMapper.html" title="interface in org.biojava.bio.structure.gui.util.color"><span class="strong">ContinuousColorMapper</span></a> - Interface in <a href="./org/biojava/bio/structure/gui/util/color/package-summary.html">org.biojava.bio.structure.gui.util.color</a></dt>
<dd>
<div class="block">Provides a mapping between real numbers and Colors.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/gui/util/color/ContinuousColorMapperTransform.html" title="class in org.biojava.bio.structure.gui.util.color"><span class="strong">ContinuousColorMapperTransform</span></a> - Class in <a href="./org/biojava/bio/structure/gui/util/color/package-summary.html">org.biojava.bio.structure.gui.util.color</a></dt>
<dd>
<div class="block">Maps colors by performing a transform of the input data and then passing 
 the transformed value to a ContinuousColorMapper for rendering.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/ContinuousColorMapperTransform.html#ContinuousColorMapperTransform(org.biojava.bio.structure.gui.util.color.ContinuousColorMapper)">ContinuousColorMapperTransform(ContinuousColorMapper)</a></span> - Constructor for class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/ContinuousColorMapperTransform.html" title="class in org.biojava.bio.structure.gui.util.color">ContinuousColorMapperTransform</a></dt>
<dd>
<div class="block">Creates a transform.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/convertor/Convert2Charge.html#convert(char)">convert(char)</a></span> - Method in class org.biojava3.aaproperties.profeat.convertor.<a href="./org/biojava3/aaproperties/profeat/convertor/Convert2Charge.html" title="class in org.biojava3.aaproperties.profeat.convertor">Convert2Charge</a></dt>
<dd>
<div class="block">Class for the conversion of protein sequence into charge</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/convertor/Convert2Hydrophobicity.html#convert(char)">convert(char)</a></span> - Method in class org.biojava3.aaproperties.profeat.convertor.<a href="./org/biojava3/aaproperties/profeat/convertor/Convert2Hydrophobicity.html" title="class in org.biojava3.aaproperties.profeat.convertor">Convert2Hydrophobicity</a></dt>
<dd>
<div class="block">Class for the conversion of protein sequence into hydrophobicity</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/convertor/Convert2NormalizedVanDerWaalsVolume.html#convert(char)">convert(char)</a></span> - Method in class org.biojava3.aaproperties.profeat.convertor.<a href="./org/biojava3/aaproperties/profeat/convertor/Convert2NormalizedVanDerWaalsVolume.html" title="class in org.biojava3.aaproperties.profeat.convertor">Convert2NormalizedVanDerWaalsVolume</a></dt>
<dd>
<div class="block">Class for the conversion of protein sequence into normalized van der waals volume</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/convertor/Convert2Polarity.html#convert(char)">convert(char)</a></span> - Method in class org.biojava3.aaproperties.profeat.convertor.<a href="./org/biojava3/aaproperties/profeat/convertor/Convert2Polarity.html" title="class in org.biojava3.aaproperties.profeat.convertor">Convert2Polarity</a></dt>
<dd>
<div class="block">Class for the conversion of protein sequence into polarity</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/convertor/Convert2Polarizability.html#convert(char)">convert(char)</a></span> - Method in class org.biojava3.aaproperties.profeat.convertor.<a href="./org/biojava3/aaproperties/profeat/convertor/Convert2Polarizability.html" title="class in org.biojava3.aaproperties.profeat.convertor">Convert2Polarizability</a></dt>
<dd>
<div class="block">Class for the conversion of protein sequence into polarizability</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/convertor/Convert2SecondaryStructure.html#convert(char)">convert(char)</a></span> - Method in class org.biojava3.aaproperties.profeat.convertor.<a href="./org/biojava3/aaproperties/profeat/convertor/Convert2SecondaryStructure.html" title="class in org.biojava3.aaproperties.profeat.convertor">Convert2SecondaryStructure</a></dt>
<dd>
<div class="block">Class for the conversion of protein sequence into secondary structure</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/convertor/Convert2SolventAccessibility.html#convert(char)">convert(char)</a></span> - Method in class org.biojava3.aaproperties.profeat.convertor.<a href="./org/biojava3/aaproperties/profeat/convertor/Convert2SolventAccessibility.html" title="class in org.biojava3.aaproperties.profeat.convertor">Convert2SolventAccessibility</a></dt>
<dd>
<div class="block">Class for the conversion of protein sequence into solvent accessibility</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/convertor/Convertor.html#convert(char)">convert(char)</a></span> - Method in class org.biojava3.aaproperties.profeat.convertor.<a href="./org/biojava3/aaproperties/profeat/convertor/Convertor.html" title="class in org.biojava3.aaproperties.profeat.convertor">Convertor</a></dt>
<dd>
<div class="block">Returns the grouping of the amino acid character.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/convertor/Convertor.html#convert(org.biojava3.core.sequence.ProteinSequence)">convert(ProteinSequence)</a></span> - Method in class org.biojava3.aaproperties.profeat.convertor.<a href="./org/biojava3/aaproperties/profeat/convertor/Convertor.html" title="class in org.biojava3.aaproperties.profeat.convertor">Convertor</a></dt>
<dd>
<div class="block">Returns the converted sequence.</div>
</dd>
<dt><a href="./org/biojava3/aaproperties/profeat/convertor/Convert2Charge.html" title="class in org.biojava3.aaproperties.profeat.convertor"><span class="strong">Convert2Charge</span></a> - Class in <a href="./org/biojava3/aaproperties/profeat/convertor/package-summary.html">org.biojava3.aaproperties.profeat.convertor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/convertor/Convert2Charge.html#Convert2Charge()">Convert2Charge()</a></span> - Constructor for class org.biojava3.aaproperties.profeat.convertor.<a href="./org/biojava3/aaproperties/profeat/convertor/Convert2Charge.html" title="class in org.biojava3.aaproperties.profeat.convertor">Convert2Charge</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/aaproperties/profeat/convertor/Convert2Hydrophobicity.html" title="class in org.biojava3.aaproperties.profeat.convertor"><span class="strong">Convert2Hydrophobicity</span></a> - Class in <a href="./org/biojava3/aaproperties/profeat/convertor/package-summary.html">org.biojava3.aaproperties.profeat.convertor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/convertor/Convert2Hydrophobicity.html#Convert2Hydrophobicity()">Convert2Hydrophobicity()</a></span> - Constructor for class org.biojava3.aaproperties.profeat.convertor.<a href="./org/biojava3/aaproperties/profeat/convertor/Convert2Hydrophobicity.html" title="class in org.biojava3.aaproperties.profeat.convertor">Convert2Hydrophobicity</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/aaproperties/profeat/convertor/Convert2NormalizedVanDerWaalsVolume.html" title="class in org.biojava3.aaproperties.profeat.convertor"><span class="strong">Convert2NormalizedVanDerWaalsVolume</span></a> - Class in <a href="./org/biojava3/aaproperties/profeat/convertor/package-summary.html">org.biojava3.aaproperties.profeat.convertor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/convertor/Convert2NormalizedVanDerWaalsVolume.html#Convert2NormalizedVanDerWaalsVolume()">Convert2NormalizedVanDerWaalsVolume()</a></span> - Constructor for class org.biojava3.aaproperties.profeat.convertor.<a href="./org/biojava3/aaproperties/profeat/convertor/Convert2NormalizedVanDerWaalsVolume.html" title="class in org.biojava3.aaproperties.profeat.convertor">Convert2NormalizedVanDerWaalsVolume</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/aaproperties/profeat/convertor/Convert2Polarity.html" title="class in org.biojava3.aaproperties.profeat.convertor"><span class="strong">Convert2Polarity</span></a> - Class in <a href="./org/biojava3/aaproperties/profeat/convertor/package-summary.html">org.biojava3.aaproperties.profeat.convertor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/convertor/Convert2Polarity.html#Convert2Polarity()">Convert2Polarity()</a></span> - Constructor for class org.biojava3.aaproperties.profeat.convertor.<a href="./org/biojava3/aaproperties/profeat/convertor/Convert2Polarity.html" title="class in org.biojava3.aaproperties.profeat.convertor">Convert2Polarity</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/aaproperties/profeat/convertor/Convert2Polarizability.html" title="class in org.biojava3.aaproperties.profeat.convertor"><span class="strong">Convert2Polarizability</span></a> - Class in <a href="./org/biojava3/aaproperties/profeat/convertor/package-summary.html">org.biojava3.aaproperties.profeat.convertor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/convertor/Convert2Polarizability.html#Convert2Polarizability()">Convert2Polarizability()</a></span> - Constructor for class org.biojava3.aaproperties.profeat.convertor.<a href="./org/biojava3/aaproperties/profeat/convertor/Convert2Polarizability.html" title="class in org.biojava3.aaproperties.profeat.convertor">Convert2Polarizability</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/aaproperties/profeat/convertor/Convert2SecondaryStructure.html" title="class in org.biojava3.aaproperties.profeat.convertor"><span class="strong">Convert2SecondaryStructure</span></a> - Class in <a href="./org/biojava3/aaproperties/profeat/convertor/package-summary.html">org.biojava3.aaproperties.profeat.convertor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/convertor/Convert2SecondaryStructure.html#Convert2SecondaryStructure()">Convert2SecondaryStructure()</a></span> - Constructor for class org.biojava3.aaproperties.profeat.convertor.<a href="./org/biojava3/aaproperties/profeat/convertor/Convert2SecondaryStructure.html" title="class in org.biojava3.aaproperties.profeat.convertor">Convert2SecondaryStructure</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/aaproperties/profeat/convertor/Convert2SolventAccessibility.html" title="class in org.biojava3.aaproperties.profeat.convertor"><span class="strong">Convert2SolventAccessibility</span></a> - Class in <a href="./org/biojava3/aaproperties/profeat/convertor/package-summary.html">org.biojava3.aaproperties.profeat.convertor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/convertor/Convert2SolventAccessibility.html#Convert2SolventAccessibility()">Convert2SolventAccessibility()</a></span> - Constructor for class org.biojava3.aaproperties.profeat.convertor.<a href="./org/biojava3/aaproperties/profeat/convertor/Convert2SolventAccessibility.html" title="class in org.biojava3.aaproperties.profeat.convertor">Convert2SolventAccessibility</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#convert_3code_1code(java.lang.String)">convert_3code_1code(String)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>
<div class="block">convert three character amino acid codes into single character
  e.g. convert CYS to C</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#convertAfpChain(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">convertAfpChain(AFPChain, Atom[], Atom[])</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>
<div class="block">copy data from this class into AFPChain container object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#convertAtomsToSeq(org.biojava.bio.structure.Atom[])">convertAtomsToSeq(Atom[])</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/aaproperties/profeat/convertor/Convertor.html" title="class in org.biojava3.aaproperties.profeat.convertor"><span class="strong">Convertor</span></a> - Class in <a href="./org/biojava3/aaproperties/profeat/convertor/package-summary.html">org.biojava3.aaproperties.profeat.convertor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/convertor/Convertor.html#Convertor()">Convertor()</a></span> - Constructor for class org.biojava3.aaproperties.profeat.convertor.<a href="./org/biojava3/aaproperties/profeat/convertor/Convertor.html" title="class in org.biojava3.aaproperties.profeat.convertor">Convertor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/PdbPairXMLConverter.html#convertPairsToXML(java.util.SortedSet)">convertPairsToXML(SortedSet&lt;PdbPair&gt;)</a></span> - Static method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/PdbPairXMLConverter.html" title="class in org.biojava.bio.structure.align.xml">PdbPairXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/JFatCatClient.html#convertStreamToString(java.io.InputStream)">convertStreamToString(InputStream)</a></span> - Static method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/JFatCatClient.html" title="class in org.biojava.bio.structure.align.client">JFatCatClient</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/PdbPairXMLConverter.html#convertXMLtoPairs(java.lang.String)">convertXMLtoPairs(String)</a></span> - Static method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/PdbPairXMLConverter.html" title="class in org.biojava.bio.structure.align.xml">PdbPairXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./demo/CookBook.html" title="class in demo"><span class="strong">CookBook</span></a> - Class in <a href="./demo/package-summary.html">demo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/CookBook.html#CookBook()">CookBook()</a></span> - Constructor for class demo.<a href="./demo/CookBook.html" title="class in demo">CookBook</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/gui/util/CoordManager.html" title="class in org.biojava.bio.structure.gui.util"><span class="strong">CoordManager</span></a> - Class in <a href="./org/biojava/bio/structure/gui/util/package-summary.html">org.biojava.bio.structure.gui.util</a></dt>
<dd>
<div class="block">a class that manages the conversion of sequence coordinate system to 
 JPanel drawing coordinates</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/CoordManager.html#CoordManager()">CoordManager()</a></span> - Constructor for class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/CoordManager.html" title="class in org.biojava.bio.structure.gui.util">CoordManager</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#copy()">copy()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Make a deep copy of a matrix</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/util/IOUtils.html#copy(java.io.InputStream, java.io.OutputStream)">copy(InputStream, OutputStream)</a></span> - Static method in class org.biojava3.core.sequence.io.util.<a href="./org/biojava3/core/sequence/io/util/IOUtils.html" title="class in org.biojava3.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">Moves the bytes from input to output using a 4KB byte array.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/CheckTreeAccuracy.html#copyMatrix(org.forester.phylogenyinference.DistanceMatrix)">copyMatrix(DistanceMatrix)</a></span> - Static method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/CheckTreeAccuracy.html" title="class in org.biojava3.phylo">CheckTreeAccuracy</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceMixin.html#countAT(org.biojava3.core.sequence.template.Sequence)">countAT(Sequence&lt;NucleotideCompound&gt;)</a></span> - Static method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceMixin.html" title="class in org.biojava3.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">Returns the count of AT in the given sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleAlignedSequence.html#countCompounds(C...)">countCompounds(C...)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleAlignedSequence.html" title="class in org.biojava3.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html#countCompounds(C...)">countCompounds(C...)</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava3.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/StringProxySequenceReader.html#countCompounds(C...)">countCompounds(C...)</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html#countCompounds(C...)">countCompounds(C...)</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/ArrayListSequenceReader.html#countCompounds(C...)">countCompounds(C...)</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava3.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/BitSequenceReader.html#countCompounds(C...)">countCompounds(C...)</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava3.core.sequence.storage">BitSequenceReader</a></dt>
<dd>
<div class="block">Counts the number of times a compound appears in this sequence store</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/JoiningSequenceReader.html#countCompounds(C...)">countCompounds(C...)</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava3.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html#countCompounds(C...)">countCompounds(C...)</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava3.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Delegates to <code>SequenceMixin#countCompounds(org.biojava3.core.sequence.template.Sequence, C[]) </code></div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#countCompounds(C...)">countCompounds(C...)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/Sequence.html#countCompounds(C...)">countCompounds(C...)</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a></dt>
<dd>
<div class="block">Returns the number of times we found a compound in the Sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceMixin.html#countCompounds(org.biojava3.core.sequence.template.Sequence, C...)">countCompounds(Sequence&lt;C&gt;, C...)</a></span> - Static method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceMixin.html" title="class in org.biojava3.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">For the given vargs of compounds this method counts the number of
 times those compounds appear in the given sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceProxyView.html#countCompounds(C...)">countCompounds(C...)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceProxyView.html" title="class in org.biojava3.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceMixin.html#countGC(org.biojava3.core.sequence.template.Sequence)">countGC(Sequence&lt;NucleotideCompound&gt;)</a></span> - Static method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceMixin.html" title="class in org.biojava3.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">Returns the count of GC in the given sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/data/sequence/FastaSequence.html#countMatchesInSequence(java.lang.String, java.lang.String)">countMatchesInSequence(String, String)</a></span> - Static method in class org.biojava3.data.sequence.<a href="./org/biojava3/data/sequence/FastaSequence.html" title="class in org.biojava3.data.sequence">FastaSequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/client/CountProgressListener.html" title="class in org.biojava.bio.structure.align.client"><span class="strong">CountProgressListener</span></a> - Class in <a href="./org/biojava/bio/structure/align/client/package-summary.html">org.biojava.bio.structure.align.client</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/CountProgressListener.html#CountProgressListener()">CountProgressListener()</a></span> - Constructor for class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/CountProgressListener.html" title="class in org.biojava.bio.structure.align.client">CountProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/util/CRC64Checksum.html" title="class in org.biojava3.core.util"><span class="strong">CRC64Checksum</span></a> - Class in <a href="./org/biojava3/core/util/package-summary.html">org.biojava3.core.util</a></dt>
<dd>
<div class="block">Utility class that calculates a CRC64 checksum on a stream of bytes.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/CRC64Checksum.html#CRC64Checksum()">CRC64Checksum()</a></span> - Constructor for class org.biojava3.core.util.<a href="./org/biojava3/core/util/CRC64Checksum.html" title="class in org.biojava3.core.util">CRC64Checksum</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/DisplayAFP.html#createArtificalStructure(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">createArtificalStructure(AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/DisplayAFP.html" title="class in org.biojava.bio.structure.align.gui">DisplayAFP</a></dt>
<dd>
<div class="block">Create a "fake" Structure objects that contains the two sets of atoms aligned on top of each other.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html#createArtificalStructure(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">createArtificalStructure(AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html" title="class in org.biojava.bio.structure.align.util">AFPAlignmentDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html#createGradients()">createGradients()</a></span> - Static method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html" title="class in org.biojava.bio.structure.gui">ScaleableMatrixPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/MyJmolStatusListener.html#createImage(java.lang.String, java.lang.String, java.lang.Object, int)">createImage(String, String, Object, int)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/MyJmolStatusListener.html" title="class in org.biojava.bio.structure.align.gui.jmol">MyJmolStatusListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/MenuCreator.html#createImageIcon(java.lang.String)">createImageIcon(String)</a></span> - Static method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/MenuCreator.html" title="class in org.biojava.bio.structure.align.gui">MenuCreator</a></dt>
<dd>
<div class="block">Returns an ImageIcon, or null if the path was invalid.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceMixin.html#createIterator(org.biojava3.core.sequence.template.Sequence)">createIterator(Sequence&lt;C&gt;)</a></span> - Static method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceMixin.html" title="class in org.biojava3.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">Creates a simple sequence iterator which moves through a sequence going
 from 1 to the length of the Sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/PrepareMMcifIndexFile.html#createMMcifInfoList(java.io.File[], java.io.File, java.io.File)">createMMcifInfoList(File[], File, File)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/PrepareMMcifIndexFile.html" title="class in org.biojava.bio.structure.server">PrepareMMcifIndexFile</a></dt>
<dd>
<div class="block">parses a set of PDB files and writes info into a file
 the file is tab separated and has the following columns:
 name length  resolution depositionDate modificationDate  technique title classification filename

 binaryDirectory: a directory in which binary files containing the atoms will be places, to provide a speedup

 This method needs to be run, before a DBSearch can be performed, since the files created by this method
 are required for the DBSearch</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/SchemaGenerator.html#createOutput(java.lang.String, java.lang.String)">createOutput(String, String)</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/SchemaGenerator.html" title="class in org.biojava3.aaproperties.xml">SchemaGenerator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/PrepareIndexFile.html#createPDBInfoList(java.io.File[], java.io.File, java.io.File)">createPDBInfoList(File[], File, File)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/PrepareIndexFile.html" title="class in org.biojava.bio.structure.server">PrepareIndexFile</a></dt>
<dd>
<div class="block">parses a set of PDB files and writes info into a file
 the file is tab separated and has the following columns:
 name length  resolution depositionDate modificationDate  technique title classification filename

 binaryDirectory: a directory in which binary files containing the atoms will be places, to provide a speedup

 This method needs to be run, before a DBSearch can be performed, since the files created by this method
 are required for the DBSearch</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompoundTranslator.html#createSequence(org.biojava3.core.sequence.template.Sequence)">createSequence(Sequence&lt;F&gt;)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompoundTranslator.html" title="class in org.biojava3.core.sequence.template">AbstractCompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/CompoundTranslator.html#createSequence(org.biojava3.core.sequence.template.Sequence)">createSequence(Sequence&lt;F&gt;)</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/CompoundTranslator.html" title="interface in org.biojava3.core.sequence.template">CompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/DNAToRNATranslator.html#createSequence(org.biojava3.core.sequence.template.Sequence, org.biojava3.core.sequence.transcription.Frame)">createSequence(Sequence&lt;NucleotideCompound&gt;, Frame)</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/DNAToRNATranslator.html" title="class in org.biojava3.core.sequence.transcription">DNAToRNATranslator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/DNAToRNATranslator.html#createSequence(org.biojava3.core.sequence.template.Sequence)">createSequence(Sequence&lt;NucleotideCompound&gt;)</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/DNAToRNATranslator.html" title="class in org.biojava3.core.sequence.transcription">DNAToRNATranslator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompoundTranslator.html#createSequences(org.biojava3.core.sequence.template.Sequence)">createSequences(Sequence&lt;F&gt;)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompoundTranslator.html" title="class in org.biojava3.core.sequence.template">AbstractCompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/CompoundTranslator.html#createSequences(org.biojava3.core.sequence.template.Sequence)">createSequences(Sequence&lt;F&gt;)</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/CompoundTranslator.html" title="interface in org.biojava3.core.sequence.template">CompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/DNAToRNATranslator.html#createSequences(org.biojava3.core.sequence.template.Sequence)">createSequences(Sequence&lt;NucleotideCompound&gt;)</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/DNAToRNATranslator.html" title="class in org.biojava3.core.sequence.transcription">DNAToRNATranslator</a></dt>
<dd>
<div class="block">Overloaded local version which delegates to an optional translator
 when told to (specified during construction).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/RNAToAminoAcidTranslator.html#createSequences(org.biojava3.core.sequence.template.Sequence)">createSequences(Sequence&lt;NucleotideCompound&gt;)</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/RNAToAminoAcidTranslator.html" title="class in org.biojava3.core.sequence.transcription">RNAToAminoAcidTranslator</a></dt>
<dd>
<div class="block">Performs the core conversion of RNA to Peptide.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceMixin.html#createSubSequence(org.biojava3.core.sequence.template.Sequence, int, int)">createSubSequence(Sequence&lt;C&gt;, int, int)</a></span> - Static method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceMixin.html" title="class in org.biojava3.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">Creates a simple sub sequence view delimited by the given start and end.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#createVirtualCBAtom(org.biojava.bio.structure.AminoAcid)">createVirtualCBAtom(AminoAcid)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">creates a virtual C-beta atom. this might be needed when working with GLY
 
 thanks to Peter Lackner for a python template of this method.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ModifiedCompound.html#crossChains()">crossChains()</a></span> - Method in interface org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ModifiedCompound.html" title="interface in org.biojava3.protmod.structure">ModifiedCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ModifiedCompoundImpl.html#crossChains()">crossChains()</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ModifiedCompoundImpl.html" title="class in org.biojava3.protmod.structure">ModifiedCompoundImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/HSVColorSpace.html#CS_HSV">CS_HSV</a></span> - Static variable in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/HSVColorSpace.html" title="class in org.biojava.bio.structure.gui.util.color">HSVColorSpace</a></dt>
<dd>
<div class="block">The HSV color space</div>
</dd>
<dt><a href="./org/biojava/bio/structure/domain/pdp/Cut.html" title="class in org.biojava.bio.structure.domain.pdp"><span class="strong">Cut</span></a> - Class in <a href="./org/biojava/bio/structure/domain/pdp/package-summary.html">org.biojava.bio.structure.domain.pdp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Cut.html#Cut()">Cut()</a></span> - Constructor for class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Cut.html" title="class in org.biojava.bio.structure.domain.pdp">Cut</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Cut.html#cut(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.domain.pdp.Domain, org.biojava.bio.structure.domain.pdp.CutValues, int[][], org.biojava.bio.structure.domain.pdp.PDPDistanceMatrix)">cut(Atom[], Domain, CutValues, int[][], PDPDistanceMatrix)</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Cut.html" title="class in org.biojava.bio.structure.domain.pdp">Cut</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html#CUT_OFF_VALUE">CUT_OFF_VALUE</a></span> - Static variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html" title="class in org.biojava.bio.structure.domain.pdp">PDPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html#CUT_OFF_VALUE1">CUT_OFF_VALUE1</a></span> - Static variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html" title="class in org.biojava.bio.structure.domain.pdp">PDPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html#CUT_OFF_VALUE1M">CUT_OFF_VALUE1M</a></span> - Static variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html" title="class in org.biojava.bio.structure.domain.pdp">PDPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html#CUT_OFF_VALUE1S">CUT_OFF_VALUE1S</a></span> - Static variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html" title="class in org.biojava.bio.structure.domain.pdp">PDPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html#CUT_OFF_VALUE2">CUT_OFF_VALUE2</a></span> - Static variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html" title="class in org.biojava.bio.structure.domain.pdp">PDPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/domain/pdp/CutDomain.html" title="class in org.biojava.bio.structure.domain.pdp"><span class="strong">CutDomain</span></a> - Class in <a href="./org/biojava/bio/structure/domain/pdp/package-summary.html">org.biojava.bio.structure.domain.pdp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/CutDomain.html#CutDomain(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.domain.pdp.PDPDistanceMatrix)">CutDomain(Atom[], PDPDistanceMatrix)</a></span> - Constructor for class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/CutDomain.html" title="class in org.biojava.bio.structure.domain.pdp">CutDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/CutDomain.html#cutDomain(org.biojava.bio.structure.domain.pdp.Domain, org.biojava.bio.structure.domain.pdp.CutSites, org.biojava.bio.structure.domain.pdp.PDPDistanceMatrix)">cutDomain(Domain, CutSites, PDPDistanceMatrix)</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/CutDomain.html" title="class in org.biojava.bio.structure.domain.pdp">CutDomain</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/domain/pdp/CutSites.html" title="class in org.biojava.bio.structure.domain.pdp"><span class="strong">CutSites</span></a> - Class in <a href="./org/biojava/bio/structure/domain/pdp/package-summary.html">org.biojava.bio.structure.domain.pdp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/CutSites.html#CutSites()">CutSites()</a></span> - Constructor for class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/CutSites.html" title="class in org.biojava.bio.structure.domain.pdp">CutSites</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#cutsPerSection">cutsPerSection</a></span> - Variable in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava3.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/domain/pdp/CutValues.html" title="class in org.biojava.bio.structure.domain.pdp"><span class="strong">CutValues</span></a> - Class in <a href="./org/biojava/bio/structure/domain/pdp/package-summary.html">org.biojava.bio.structure.domain.pdp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/CutValues.html#CutValues()">CutValues()</a></span> - Constructor for class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/CutValues.html" title="class in org.biojava.bio.structure.domain.pdp">CutValues</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/ColorUtils.html#cyan">cyan</a></span> - Static variable in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/ColorUtils.html" title="class in org.biojava.bio.structure.gui.util.color">ColorUtils</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#Cys">Cys</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_D_">
<!--   -->
</a>
<h2 class="title">D</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/Constraints.html#D">D</a></span> - Static variable in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/Constraints.html" title="class in org.biojava3.aaproperties">Constraints</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/ColorUtils.html#darker(java.awt.Color, double)">darker(Color, double)</a></span> - Static method in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/ColorUtils.html" title="class in org.biojava.bio.structure.gui.util.color">ColorUtils</a></dt>
<dd>
<div class="block">Make a color darker.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBremark.html" title="class in org.biojava.bio.structure.io.mmcif.model"><span class="strong">DatabasePDBremark</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBremark.html#DatabasePDBremark()">DatabasePDBremark()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBremark.html" title="class in org.biojava.bio.structure.io.mmcif.model">DatabasePDBremark</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html" title="class in org.biojava.bio.structure.io.mmcif.model"><span class="strong">DatabasePDBrev</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html#DatabasePDBrev()">DatabasePDBrev()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html" title="class in org.biojava.bio.structure.io.mmcif.model">DatabasePDBrev</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/features/DatabaseReferenceInterface.html" title="interface in org.biojava3.core.sequence.features"><span class="strong">DatabaseReferenceInterface</span></a> - Interface in <a href="./org/biojava3/core/sequence/features/package-summary.html">org.biojava3.core.sequence.features</a></dt>
<dd>
<div class="block">If a SequenceProxyReader implements this interface then that external source
 has a list of cross reference id(s)</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/DataSource.html" title="enum in org.biojava3.core.sequence"><span class="strong">DataSource</span></a> - Enum in <a href="./org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a></dt>
<dd>
<div class="block">GenBank                           gi|gi-number|gb|accession|locus
 ENA Data Library                  gi|gi-number|emb|accession|locus
 DDBJ, DNA Database of Japan       gi|gi-number|dbj|accession|locus
 NBRF PIR                          pir||entry
 Protein Research Foundation       prf||name
 SWISS-PROT UNIPROT                sp|accession|name
 Brookhaven Protein Data Bank (1)  pdb|entry|chain
 Brookhaven Protein Data Bank (2)  entry:chain|PDBID|CHAIN|SEQUENCE
 Patents                           pat|country|number
 GenInfo Backbone Id               bbs|number
 General database identifier       gnl|database|identifier
 NCBI Reference Sequence           ref|accession|locus
 Local Sequence identifier         lcl|identifier</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/FlatFileInstallation.html#dateFormat">dateFormat</a></span> - Static variable in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/FlatFileInstallation.html" title="class in org.biojava.bio.structure.server">FlatFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html#DBL">DBL</a></span> - Static variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html" title="class in org.biojava.bio.structure.domain.pdp">PDPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure"><span class="strong">DBRef</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">A class to represent database cross references.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#DBRef()">DBRef()</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/features/DBReferenceInfo.html" title="class in org.biojava3.core.sequence.features"><span class="strong">DBReferenceInfo</span></a> - Class in <a href="./org/biojava3/core/sequence/features/package-summary.html">org.biojava3.core.sequence.features</a></dt>
<dd>
<div class="block">If you have a uniprot ID then it is possible to get a collection
 of other id(s) that the protein is known by.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/DBReferenceInfo.html#DBReferenceInfo(java.lang.String, java.lang.String)">DBReferenceInfo(String, String)</a></span> - Constructor for class org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/DBReferenceInfo.html" title="class in org.biojava3.core.sequence.features">DBReferenceInfo</a></dt>
<dd>
<div class="block">The source database and id</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/gui/DBResultTable.html" title="class in org.biojava.bio.structure.align.gui"><span class="strong">DBResultTable</span></a> - Class in <a href="./org/biojava/bio/structure/align/gui/package-summary.html">org.biojava.bio.structure.align.gui</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/DBResultTable.html#DBResultTable()">DBResultTable()</a></span> - Constructor for class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/DBResultTable.html" title="class in org.biojava.bio.structure.align.gui">DBResultTable</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/gui/DBSearchGUI.html" title="class in org.biojava.bio.structure.align.gui"><span class="strong">DBSearchGUI</span></a> - Class in <a href="./org/biojava/bio/structure/align/gui/package-summary.html">org.biojava.bio.structure.align.gui</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/DBSearchGUI.html#DBSearchGUI()">DBSearchGUI()</a></span> - Constructor for class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/DBSearchGUI.html" title="class in org.biojava.bio.structure.align.gui">DBSearchGUI</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/AFPCalculator.html#debug">debug</a></span> - Static variable in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/AFPCalculator.html" title="class in org.biojava.bio.structure.align.fatcat.calc">AFPCalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/AFPChainer.html#debug">debug</a></span> - Static variable in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/AFPChainer.html" title="class in org.biojava.bio.structure.align.fatcat.calc">AFPChainer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/AFPOptimizer.html#debug">debug</a></span> - Static variable in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/AFPOptimizer.html" title="class in org.biojava.bio.structure.align.fatcat.calc">AFPOptimizer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/AFPPostProcessor.html#debug">debug</a></span> - Static variable in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/AFPPostProcessor.html" title="class in org.biojava.bio.structure.align.fatcat.calc">AFPPostProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatAligner.html#debug">debug</a></span> - Static variable in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatAligner.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/DisplayAFP.html#debug">debug</a></span> - Static variable in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/DisplayAFP.html" title="class in org.biojava.bio.structure.align.gui">DisplayAFP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/SoftHashMap.html#DEBUG">DEBUG</a></span> - Static variable in class org.biojava3.core.util.<a href="./org/biojava3/core/util/SoftHashMap.html" title="class in org.biojava3.core.util">SoftHashMap</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/PrettyXMLWriter.html#declareNamespace(java.lang.String, java.lang.String)">declareNamespace(String, String)</a></span> - Method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/PrettyXMLWriter.html" title="class in org.biojava3.core.util">PrettyXMLWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/XMLWriter.html#declareNamespace(java.lang.String, java.lang.String)">declareNamespace(String, String)</a></span> - Method in interface org.biojava3.core.util.<a href="./org/biojava3/core/util/XMLWriter.html" title="interface in org.biojava3.core.util">XMLWriter</a></dt>
<dd>
<div class="block">Hints that a namespace is going to be used in a sub-tree.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.Builder.html#decorateRna(boolean)">decorateRna(boolean)</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava3.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>
<div class="block">Performs an optimisation where RNASequences are not translated into
 their own objects but are views onto the base DNA sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/data/sequence/SequenceUtil.html#deepCleanSequence(java.lang.String)">deepCleanSequence(String)</a></span> - Static method in class org.biojava3.data.sequence.<a href="./org/biojava3/data/sequence/SequenceUtil.html" title="class in org.biojava3.data.sequence">SequenceUtil</a></dt>
<dd>
<div class="block">Removes all special characters and digits as well as whitespace chars
 from the sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/FlatFileInstallation.html#DEFAUL_CHAIN_FILE">DEFAUL_CHAIN_FILE</a></span> - Static variable in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/FlatFileInstallation.html" title="class in org.biojava.bio.structure.server">FlatFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#DEFAULT_ALGORITHM_NAME">DEFAULT_ALGORITHM_NAME</a></span> - Static variable in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">the default algorithm used in the RCSB PDB all vs. all database searches</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#DEFAULT_BATCH_SIZE">DEFAULT_BATCH_SIZE</a></span> - Static variable in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#DEFAULT_CHAIN_ID">DEFAULT_CHAIN_ID</a></span> - Static variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">The default chain identifier is an empty space.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AFPChainCoordManager.html#DEFAULT_CHAR_SIZE">DEFAULT_CHAR_SIZE</a></span> - Static variable in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AFPChainCoordManager.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AFPChainCoordManager</a></dt>
<dd>
<div class="block">size per character</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ClusterAltAligs.html#DEFAULT_CLUSTER_CUTOFF">DEFAULT_CLUSTER_CUTOFF</a></span> - Static variable in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/ClusterAltAligs.html" title="class in org.biojava.bio.structure.align">ClusterAltAligs</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/HTTPConnectionTools.html#DEFAULT_CONNECTION_TIMEOUT">DEFAULT_CONNECTION_TIMEOUT</a></span> - Static variable in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/HTTPConnectionTools.html" title="class in org.biojava.bio.structure.align.util">HTTPConnectionTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#DEFAULT_DIR_SPLIT">DEFAULT_DIR_SPLIT</a></span> - Static variable in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/ronn/RonnConstraint.html#DEFAULT_DISORDER">DEFAULT_DISORDER</a></span> - Static variable in class org.biojava3.ronn.<a href="./org/biojava3/ronn/RonnConstraint.html" title="class in org.biojava3.ronn">RonnConstraint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#DEFAULT_FRAGLEN">DEFAULT_FRAGLEN</a></span> - Static variable in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/FlatFileInstallation.html#DEFAULT_INDEX_FILE">DEFAULT_INDEX_FILE</a></span> - Static variable in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/FlatFileInstallation.html" title="class in org.biojava.bio.structure.server">FlatFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#DEFAULT_JOB_TIME">DEFAULT_JOB_TIME</a></span> - Static variable in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/SoftHashMap.html#DEFAULT_LIMIT">DEFAULT_LIMIT</a></span> - Static variable in class org.biojava3.core.util.<a href="./org/biojava3/core/util/SoftHashMap.html" title="class in org.biojava3.core.util">SoftHashMap</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AFPChainCoordManager.html#DEFAULT_LINE_LENGTH">DEFAULT_LINE_LENGTH</a></span> - Static variable in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AFPChainCoordManager.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AFPChainCoordManager</a></dt>
<dd>
<div class="block">number of chars per line</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AFPChainCoordManager.html#DEFAULT_LINE_SEPARATION">DEFAULT_LINE_SEPARATION</a></span> - Static variable in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AFPChainCoordManager.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AFPChainCoordManager</a></dt>
<dd>
<div class="block">separation of line 1 and 2 in alignment</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/BigSqrt.html#DEFAULT_MAX_ITERATIONS">DEFAULT_MAX_ITERATIONS</a></span> - Static variable in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/BigSqrt.html" title="class in org.biojava.bio.structure.secstruc">BigSqrt</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#DEFAULT_NR_ALIGNMENTS">DEFAULT_NR_ALIGNMENTS</a></span> - Static variable in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#DEFAULT_NR_THREADS">DEFAULT_NR_THREADS</a></span> - Static variable in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/ronn/RonnConstraint.html#DEFAULT_ORDER">DEFAULT_ORDER</a></span> - Static variable in class org.biojava3.ronn.<a href="./org/biojava3/ronn/RonnConstraint.html" title="class in org.biojava3.ronn">RonnConstraint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#DEFAULT_PDB_FILE_SERVER">DEFAULT_PDB_FILE_SERVER</a></span> - Static variable in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#DEFAULT_PDB_PATH">DEFAULT_PDB_PATH</a></span> - Static variable in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBStatus.html#DEFAULT_PDB_SERVER">DEFAULT_PDB_SERVER</a></span> - Static variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBStatus.html" title="class in org.biojava.bio.structure">PDBStatus</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/ronn/RonnConstraint.html#DEFAULT_RANGE_PROBABILITY_THRESHOLD">DEFAULT_RANGE_PROBABILITY_THRESHOLD</a></span> - Static variable in class org.biojava3.ronn.<a href="./org/biojava3/ronn/RonnConstraint.html" title="class in org.biojava3.ronn">RonnConstraint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#DEFAULT_RESOLUTION">DEFAULT_RESOLUTION</a></span> - Static variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AFPChainCoordManager.html#DEFAULT_RIGHT_SPACER">DEFAULT_RIGHT_SPACER</a></span> - Static variable in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AFPChainCoordManager.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AFPChainCoordManager</a></dt>
<dd>
<div class="block">Space on the right side between sequence and legend.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/BigSqrt.html#DEFAULT_SCALE">DEFAULT_SCALE</a></span> - Static variable in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/BigSqrt.html" title="class in org.biojava.bio.structure.secstruc">BigSqrt</a></dt>
<dd>
<div class="block">we take 3 aftercomma</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#DEFAULT_SCORING_STRATEGY">DEFAULT_SCORING_STRATEGY</a></span> - Static variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html#DEFAULT_SCRIPT">DEFAULT_SCRIPT</a></span> - Static variable in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html" title="class in org.biojava.bio.structure.align.gui.jmol">StructureAlignmentJmol</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html#DEFAULT_SERVER">DEFAULT_SERVER</a></span> - Static variable in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">RemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#DEFAULT_SERVER_URL">DEFAULT_SERVER_URL</a></span> - Static variable in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#DEFAULT_VERSION">DEFAULT_VERSION</a></span> - Static variable in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html#DEFAULT_X_RIGHT_BORDER">DEFAULT_X_RIGHT_BORDER</a></span> - Static variable in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html" title="class in org.biojava.bio.structure.gui.util">SequenceScalePanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AFPChainCoordManager.html#DEFAULT_X_SPACE">DEFAULT_X_SPACE</a></span> - Static variable in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AFPChainCoordManager.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AFPChainCoordManager</a></dt>
<dd>
<div class="block">left boundary</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html#DEFAULT_X_START">DEFAULT_X_START</a></span> - Static variable in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html" title="class in org.biojava.bio.structure.gui.util">SequenceScalePanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html#DEFAULT_Y_BOTTOM">DEFAULT_Y_BOTTOM</a></span> - Static variable in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html" title="class in org.biojava.bio.structure.gui.util">SequenceScalePanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html#DEFAULT_Y_HEIGHT">DEFAULT_Y_HEIGHT</a></span> - Static variable in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html" title="class in org.biojava.bio.structure.gui.util">SequenceScalePanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AFPChainCoordManager.html#DEFAULT_Y_SPACE">DEFAULT_Y_SPACE</a></span> - Static variable in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AFPChainCoordManager.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AFPChainCoordManager</a></dt>
<dd>
<div class="block">top boundary</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html#DEFAULT_Y_START">DEFAULT_Y_START</a></span> - Static variable in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html" title="class in org.biojava.bio.structure.gui.util">SequenceScalePanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AFPChainCoordManager.html#DEFAULT_Y_STEP">DEFAULT_Y_STEP</a></span> - Static variable in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AFPChainCoordManager.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AFPChainCoordManager</a></dt>
<dd>
<div class="block">size of space between rows</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html#DEFAULT_Y_STEP">DEFAULT_Y_STEP</a></span> - Static variable in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html" title="class in org.biojava.bio.structure.gui.util">SequenceScalePanel</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/alignment/aaindex/DefaultAAIndexProvider.html" title="class in org.biojava3.alignment.aaindex"><span class="strong">DefaultAAIndexProvider</span></a> - Class in <a href="./org/biojava3/alignment/aaindex/package-summary.html">org.biojava3.alignment.aaindex</a></dt>
<dd>
<div class="block">The default provider for AAINDEX loads substituion matrices from the AAINDEX file in the resources directory</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/aaindex/DefaultAAIndexProvider.html#DefaultAAIndexProvider()">DefaultAAIndexProvider()</a></span> - Constructor for class org.biojava3.alignment.aaindex.<a href="./org/biojava3/alignment/aaindex/DefaultAAIndexProvider.html" title="class in org.biojava3.alignment.aaindex">DefaultAAIndexProvider</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/gui/util/color/DefaultMatrixMapper.html" title="class in org.biojava.bio.structure.gui.util.color"><span class="strong">DefaultMatrixMapper</span></a> - Class in <a href="./org/biojava/bio/structure/gui/util/color/package-summary.html">org.biojava.bio.structure.gui.util.color</a></dt>
<dd>
<div class="block">Color Mapper which mimics the default coloring of JMatrixPanel pixels.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/DefaultMatrixMapper.html#DefaultMatrixMapper(double, float)">DefaultMatrixMapper(double, float)</a></span> - Constructor for class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/DefaultMatrixMapper.html" title="class in org.biojava.bio.structure.gui.util.color">DefaultMatrixMapper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/SystemInfo.html#defaultProperties">defaultProperties</a></span> - Static variable in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/SystemInfo.html" title="class in org.biojava.bio.structure.align.gui">SystemInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#degreesPerRadian">degreesPerRadian</a></span> - Static variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">Degrees per radian.</div>
</dd>
<dt><a href="./demo/package-summary.html">demo</a> - package demo</dt>
<dd>&nbsp;</dd>
<dt><a href="./demo/DemoAtomCache.html" title="class in demo"><span class="strong">DemoAtomCache</span></a> - Class in <a href="./demo/package-summary.html">demo</a></dt>
<dd>
<div class="block">Example of how to load PDB files using the AtomCache class.</div>
</dd>
<dt><span class="strong"><a href="./demo/DemoAtomCache.html#DemoAtomCache()">DemoAtomCache()</a></span> - Constructor for class demo.<a href="./demo/DemoAtomCache.html" title="class in demo">DemoAtomCache</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./demo/DemoCE.html" title="class in demo"><span class="strong">DemoCE</span></a> - Class in <a href="./demo/package-summary.html">demo</a></dt>
<dd>
<div class="block">Example of how to run a structure alignment using the CE algorithm.</div>
</dd>
<dt><span class="strong"><a href="./demo/DemoCE.html#DemoCE()">DemoCE()</a></span> - Constructor for class demo.<a href="./demo/DemoCE.html" title="class in demo">DemoCE</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./demo/DemoChangeChemCompProvider.html" title="class in demo"><span class="strong">DemoChangeChemCompProvider</span></a> - Class in <a href="./demo/package-summary.html">demo</a></dt>
<dd>
<div class="block">This demo shows how to use an alternative ChemCompProvider.</div>
</dd>
<dt><span class="strong"><a href="./demo/DemoChangeChemCompProvider.html#DemoChangeChemCompProvider()">DemoChangeChemCompProvider()</a></span> - Constructor for class demo.<a href="./demo/DemoChangeChemCompProvider.html" title="class in demo">DemoChangeChemCompProvider</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./demo/DemoCommandLineStartup.html" title="class in demo"><span class="strong">DemoCommandLineStartup</span></a> - Class in <a href="./demo/package-summary.html">demo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoCommandLineStartup.html#DemoCommandLineStartup()">DemoCommandLineStartup()</a></span> - Constructor for class demo.<a href="./demo/DemoCommandLineStartup.html" title="class in demo">DemoCommandLineStartup</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./demo/DemoDomainsplit.html" title="class in demo"><span class="strong">DemoDomainsplit</span></a> - Class in <a href="./demo/package-summary.html">demo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoDomainsplit.html#DemoDomainsplit()">DemoDomainsplit()</a></span> - Constructor for class demo.<a href="./demo/DemoDomainsplit.html" title="class in demo">DemoDomainsplit</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./demo/DemoFATCAT.html" title="class in demo"><span class="strong">DemoFATCAT</span></a> - Class in <a href="./demo/package-summary.html">demo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoFATCAT.html#DemoFATCAT()">DemoFATCAT()</a></span> - Constructor for class demo.<a href="./demo/DemoFATCAT.html" title="class in demo">DemoFATCAT</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./demo/DemoLoadStructure.html" title="class in demo"><span class="strong">DemoLoadStructure</span></a> - Class in <a href="./demo/package-summary.html">demo</a></dt>
<dd>
<div class="block">Example for how to load protein structures (from PDB files).</div>
</dd>
<dt><span class="strong"><a href="./demo/DemoLoadStructure.html#DemoLoadStructure()">DemoLoadStructure()</a></span> - Constructor for class demo.<a href="./demo/DemoLoadStructure.html" title="class in demo">DemoLoadStructure</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./demo/DemoMMCIFReader.html" title="class in demo"><span class="strong">DemoMMCIFReader</span></a> - Class in <a href="./demo/package-summary.html">demo</a></dt>
<dd>
<div class="block">An example of how to read MMcif files</div>
</dd>
<dt><span class="strong"><a href="./demo/DemoMMCIFReader.html#DemoMMCIFReader()">DemoMMCIFReader()</a></span> - Constructor for class demo.<a href="./demo/DemoMMCIFReader.html" title="class in demo">DemoMMCIFReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./demo/DemoSCOP.html" title="class in demo"><span class="strong">DemoSCOP</span></a> - Class in <a href="./demo/package-summary.html">demo</a></dt>
<dd>
<div class="block">A class demonstrating the use of the SCOP parsing tools</div>
</dd>
<dt><span class="strong"><a href="./demo/DemoSCOP.html#DemoSCOP()">DemoSCOP()</a></span> - Constructor for class demo.<a href="./demo/DemoSCOP.html" title="class in demo">DemoSCOP</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./demo/DemoSW3DAligner.html" title="class in demo"><span class="strong">DemoSW3DAligner</span></a> - Class in <a href="./demo/package-summary.html">demo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoSW3DAligner.html#DemoSW3DAligner()">DemoSW3DAligner()</a></span> - Constructor for class demo.<a href="./demo/DemoSW3DAligner.html" title="class in demo">DemoSW3DAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ModificationCategory.html#description()">description()</a></span> - Method in enum org.biojava3.protmod.<a href="./org/biojava3/protmod/ModificationCategory.html" title="enum in org.biojava3.protmod">ModificationCategory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html#deserialize(java.io.File)">deserialize(File)</a></span> - Static method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentResult</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#desFileName">desFileName</a></span> - Static variable in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html#destroy()">destroy()</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AligPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanelMouseMotionListener.html#destroy()">destroy()</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanelMouseMotionListener.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AligPanelMouseMotionListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/StatusDisplay.html#destroy()">destroy()</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/StatusDisplay.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">StatusDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html#destroy()">destroy()</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html" title="class in org.biojava.bio.structure.align.gui.jmol">StructureAlignmentJmol</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/FlatFileCache.html#destroy()">destroy()</a></span> - Static method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/FlatFileCache.html" title="class in org.biojava3.core.util">FlatFileCache</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ProteinModificationIdentifier.html#destroy()">destroy()</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ProteinModificationIdentifier.html" title="class in org.biojava3.protmod.structure">ProteinModificationIdentifier</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/LUDecomposition.html#det()">det()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/LUDecomposition.html" title="class in org.biojava.bio.structure.jama">LUDecomposition</a></dt>
<dd>
<div class="block">Determinant</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#det()">det()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Matrix determinant</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/LocationHelper.html#detectCicular(java.util.List)">detectCicular(List&lt;Location&gt;)</a></span> - Static method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/LocationHelper.html" title="class in org.biojava3.core.sequence.location">LocationHelper</a></dt>
<dd>
<div class="block">Loops through the given list of locations and returns true if it looks
 like they represent a circular location.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/LocationHelper.html#detectEnd(java.util.List, boolean)">detectEnd(List&lt;Location&gt;, boolean)</a></span> - Static method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/LocationHelper.html" title="class in org.biojava3.core.sequence.location">LocationHelper</a></dt>
<dd>
<div class="block">This will attempt to find what the last point is and returns that
 position.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/LocationHelper.html#detectStart(java.util.List)">detectStart(List&lt;Location&gt;)</a></span> - Static method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/LocationHelper.html" title="class in org.biojava3.core.sequence.location">LocationHelper</a></dt>
<dd>
<div class="block">Assumes that the first element is the start & clones it</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/LocationHelper.html#detectStrand(java.util.List)">detectStrand(List&lt;Location&gt;)</a></span> - Static method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/LocationHelper.html" title="class in org.biojava3.core.sequence.location">LocationHelper</a></dt>
<dd>
<div class="block">Loops through the given list of locations and returns the consensus
 Strand class.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/Constraints.html#diAA2Instability">diAA2Instability</a></span> - Static variable in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/Constraints.html" title="class in org.biojava3.aaproperties">Constraints</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/MyJmolStatusListener.html#dialogAsk(java.lang.String, java.lang.String)">dialogAsk(String, String)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/MyJmolStatusListener.html" title="class in org.biojava.bio.structure.align.gui.jmol">MyJmolStatusListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/data/sequence/SequenceUtil.html#DIGIT">DIGIT</a></span> - Static variable in class org.biojava3.data.sequence.<a href="./org/biojava3/data/sequence/SequenceUtil.html" title="class in org.biojava3.data.sequence">SequenceUtil</a></dt>
<dd>
<div class="block">A digit</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/GuiWrapper.html#display(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">display(AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/GuiWrapper.html" title="class in org.biojava.bio.structure.align.ce">GuiWrapper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/DisplayAFP.html#display(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Group[], org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], java.util.List, java.util.List, java.util.List, java.util.List)">display(AFPChain, Group[], Atom[], Atom[], List&lt;Group&gt;, List&lt;Group&gt;, List&lt;Group&gt;, List&lt;Group&gt;)</a></span> - Static method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/DisplayAFP.html" title="class in org.biojava.bio.structure.align.gui">DisplayAFP</a></dt>
<dd>
<div class="block">Note: ca2, hetatoms2 and nucleotides2 should not be rotated.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/StructureAlignmentDisplay.html#display(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">display(AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/StructureAlignmentDisplay.html" title="class in org.biojava.bio.structure.align.gui">StructureAlignmentDisplay</a></dt>
<dd>
<div class="block">Display the alignment</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/gui/DisplayAFP.html" title="class in org.biojava.bio.structure.align.gui"><span class="strong">DisplayAFP</span></a> - Class in <a href="./org/biojava/bio/structure/align/gui/package-summary.html">org.biojava.bio.structure.align.gui</a></dt>
<dd>
<div class="block">A utility class for visualistion of structure alignments</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/DisplayAFP.html#DisplayAFP()">DisplayAFP()</a></span> - Constructor for class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/DisplayAFP.html" title="class in org.biojava.bio.structure.align.gui">DisplayAFP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Location.html#distance(org.biojava3.genome.parsers.gff.Location)">distance(Location)</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Location.html" title="class in org.biojava3.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Return distance between this location and the other location.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/secstruc/DistEn.html" title="class in org.biojava.bio.structure.secstruc"><span class="strong">DistEn</span></a> - Class in <a href="./org/biojava/bio/structure/secstruc/package-summary.html">org.biojava.bio.structure.secstruc</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/DistEn.html#DistEn()">DistEn()</a></span> - Constructor for class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/DistEn.html" title="class in org.biojava.bio.structure.secstruc">DistEn</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/chem/PolymerType.html#DNA_ONLY">DNA_ONLY</a></span> - Static variable in enum org.biojava.bio.structure.io.mmcif.chem.<a href="./org/biojava/bio/structure/io/mmcif/chem/PolymerType.html" title="enum in org.biojava.bio.structure.io.mmcif.chem">PolymerType</a></dt>
<dd>
<div class="block">Convenience <tt>Set</tt> of polymer types classified as DNA.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.Builder.html#dnaCompounds(org.biojava3.core.sequence.template.CompoundSet)">dnaCompounds(CompoundSet&lt;NucleotideCompound&gt;)</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava3.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/compound/DNACompoundSet.html" title="class in org.biojava3.core.sequence.compound"><span class="strong">DNACompoundSet</span></a> - Class in <a href="./org/biojava3/core/sequence/compound/package-summary.html">org.biojava3.core.sequence.compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/DNACompoundSet.html#DNACompoundSet()">DNACompoundSet()</a></span> - Constructor for class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/DNACompoundSet.html" title="class in org.biojava3.core.sequence.compound">DNACompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.Builder.html#dnaRnaTranslator(org.biojava3.core.sequence.transcription.DNAToRNATranslator)">dnaRnaTranslator(DNAToRNATranslator)</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava3.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/DNASequence.html" title="class in org.biojava3.core.sequence"><span class="strong">DNASequence</span></a> - Class in <a href="./org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a></dt>
<dd>
<div class="block">This is class should model the attributes associated with a DNA sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/DNASequence.html#DNASequence()">DNASequence()</a></span> - Constructor for class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/DNASequence.html" title="class in org.biojava3.core.sequence">DNASequence</a></dt>
<dd>
<div class="block">Shouldn't be used but makes it bean happy</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/DNASequence.html#DNASequence(java.lang.String)">DNASequence(String)</a></span> - Constructor for class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/DNASequence.html" title="class in org.biojava3.core.sequence">DNASequence</a></dt>
<dd>
<div class="block">String is king and create a sequence from DNA with default DNA compound set</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/DNASequence.html#DNASequence(org.biojava3.core.sequence.template.SequenceReader)">DNASequence(SequenceReader&lt;NucleotideCompound&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/DNASequence.html" title="class in org.biojava3.core.sequence">DNASequence</a></dt>
<dd>
<div class="block">Create a sequence where the actual storage of the sequence data is somewhere else</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/DNASequence.html#DNASequence(java.lang.String, org.biojava3.core.sequence.template.CompoundSet)">DNASequence(String, CompoundSet&lt;NucleotideCompound&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/DNASequence.html" title="class in org.biojava3.core.sequence">DNASequence</a></dt>
<dd>
<div class="block">Create a sequence from a string with user defined compound set</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/DNASequence.html#DNASequence(org.biojava3.core.sequence.template.SequenceReader, org.biojava3.core.sequence.template.CompoundSet)">DNASequence(SequenceReader&lt;NucleotideCompound&gt;, CompoundSet&lt;NucleotideCompound&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/DNASequence.html" title="class in org.biojava3.core.sequence">DNASequence</a></dt>
<dd>
<div class="block">Create a sequence from a ProxySequencereader and user defined compound set</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/DNASequence.DNAType.html" title="enum in org.biojava3.core.sequence"><span class="strong">DNASequence.DNAType</span></a> - Enum in <a href="./org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a></dt>
<dd>
<div class="block">The type of DNA sequence</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/io/DNASequenceCreator.html" title="class in org.biojava3.core.sequence.io"><span class="strong">DNASequenceCreator</span></a> - Class in <a href="./org/biojava3/core/sequence/io/package-summary.html">org.biojava3.core.sequence.io</a></dt>
<dd>
<div class="block">A helper class that allows different ways to read a string and create a DNA sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/DNASequenceCreator.html#DNASequenceCreator(org.biojava3.core.sequence.template.CompoundSet)">DNASequenceCreator(CompoundSet&lt;NucleotideCompound&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/DNASequenceCreator.html" title="class in org.biojava3.core.sequence.io">DNASequenceCreator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/transcription/DNAToRNATranslator.html" title="class in org.biojava3.core.sequence.transcription"><span class="strong">DNAToRNATranslator</span></a> - Class in <a href="./org/biojava3/core/sequence/transcription/package-summary.html">org.biojava3.core.sequence.transcription</a></dt>
<dd>
<div class="block">Performs the first stage of transcription by going from DNA to RNA.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/DNAToRNATranslator.html#DNAToRNATranslator(org.biojava3.core.sequence.io.template.SequenceCreatorInterface, org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.template.CompoundSet, boolean)">DNAToRNATranslator(SequenceCreatorInterface&lt;NucleotideCompound&gt;, CompoundSet&lt;NucleotideCompound&gt;, CompoundSet&lt;NucleotideCompound&gt;, boolean)</a></span> - Constructor for class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/DNAToRNATranslator.html" title="class in org.biojava3.core.sequence.transcription">DNAToRNATranslator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/AFPChainer.html#doChainAfp(org.biojava.bio.structure.align.fatcat.calc.FatCatParameters, org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">doChainAfp(FatCatParameters, AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/AFPChainer.html" title="class in org.biojava.bio.structure.align.fatcat.calc">AFPChainer</a></dt>
<dd>
<div class="block">// Key function: chain (assembly) the AFPs
  // a AFP (k) is defined as (i, j, k), with i and j are staring points
  // AFP extension (eg.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#documentEnd()">documentEnd()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#documentEnd()">documentEnd()</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>
<div class="block">called at end of document</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#documentEnd()">documentEnd()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#documentStart()">documentStart()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#documentStart()">documentStart()</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>
<div class="block">called at start of document</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#documentStart()">documentStart()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>
<div class="block">Start the parsing</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/Utils.html#doesSequenceContainInvalidChar(java.lang.String, java.util.Set)">doesSequenceContainInvalidChar(String, Set&lt;Character&gt;)</a></span> - Static method in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/Utils.html" title="class in org.biojava3.aaproperties">Utils</a></dt>
<dd>
<div class="block">Checks if given sequence contains invalid characters.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/domain/pdp/Domain.html" title="class in org.biojava.bio.structure.domain.pdp"><span class="strong">Domain</span></a> - Class in <a href="./org/biojava/bio/structure/domain/pdp/package-summary.html">org.biojava.bio.structure.domain.pdp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Domain.html#Domain()">Domain()</a></span> - Constructor for class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Domain.html" title="class in org.biojava.bio.structure.domain.pdp">Domain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/HTTPConnectionTools.html#doPOST(java.net.URL, java.lang.String)">doPOST(URL, String)</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/HTTPConnectionTools.html" title="class in org.biojava.bio.structure.align.util">HTTPConnectionTools</a></dt>
<dd>
<div class="block">do a POST to a URL and return the response stream for further processing elsewhere.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/HTTPConnectionTools.html#doPOST(java.net.URL, java.lang.String, int)">doPOST(URL, String, int)</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/HTTPConnectionTools.html" title="class in org.biojava.bio.structure.align.util">HTTPConnectionTools</a></dt>
<dd>
<div class="block">do a POST to a URL and return the response stream for further processing elsewhere.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileConvert.html#doPrintConnections()">doPrintConnections()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileConvert.html" title="class in org.biojava.bio.structure.io">FileConvert</a></dt>
<dd>
<div class="block">returns if the Connections should be added
 default is true;</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/gui/DotPlotPanel.html" title="class in org.biojava.bio.structure.align.gui"><span class="strong">DotPlotPanel</span></a> - Class in <a href="./org/biojava/bio/structure/align/gui/package-summary.html">org.biojava.bio.structure.align.gui</a></dt>
<dd>
<div class="block">Displays the dot plot trace for an alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/DotPlotPanel.html#DotPlotPanel(org.biojava.bio.structure.align.model.AFPChain)">DotPlotPanel(AFPChain)</a></span> - Constructor for class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/DotPlotPanel.html" title="class in org.biojava.bio.structure.align.gui">DotPlotPanel</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/DownloadChemCompProvider.html" title="class in org.biojava.bio.structure.io.mmcif"><span class="strong">DownloadChemCompProvider</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/package-summary.html">org.biojava.bio.structure.io.mmcif</a></dt>
<dd>
<div class="block">This provider of chemical components can download and cache chemical component definition files from the RCSB PDB web site.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/DownloadChemCompProvider.html#DownloadChemCompProvider()">DownloadChemCompProvider()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/DownloadChemCompProvider.html" title="class in org.biojava.bio.structure.io.mmcif">DownloadChemCompProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html#downloadCIF(java.lang.String)">downloadCIF(String)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html" title="class in org.biojava.bio.structure.server">MMCIFFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/AllChemCompProvider.html#downloadFile()">downloadFile()</a></span> - Static method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/AllChemCompProvider.html" title="class in org.biojava.bio.structure.io.mmcif">AllChemCompProvider</a></dt>
<dd>
<div class="block">Downloads the components.cif.gz file from the wwPDB site.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/CountProgressListener.html#downloadingStructures(java.lang.String)">downloadingStructures(String)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/CountProgressListener.html" title="class in org.biojava.bio.structure.align.client">CountProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/events/AlignmentProgressListener.html#downloadingStructures(java.lang.String)">downloadingStructures(String)</a></span> - Method in interface org.biojava.bio.structure.align.events.<a href="./org/biojava/bio/structure/align/events/AlignmentProgressListener.html" title="interface in org.biojava.bio.structure.align.events">AlignmentProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/GUIAlignmentProgressListener.html#downloadingStructures(java.lang.String)">downloadingStructures(String)</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/GUIAlignmentProgressListener.html" title="class in org.biojava.bio.structure.align.gui">GUIAlignmentProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/MMCIFFileReader.html#downloadPDB(java.lang.String)">downloadPDB(String)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/MMCIFFileReader.html" title="class in org.biojava.bio.structure.io">MMCIFFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Location.html#downstream(int)">downstream(int)</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Location.html" title="class in org.biojava3.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Return the adjacent location of specified length directly downstream of this location.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/JMatrixPanel.html#drawBoxes(java.awt.Graphics)">drawBoxes(Graphics)</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/JMatrixPanel.html" title="class in org.biojava.bio.structure.gui">JMatrixPanel</a></dt>
<dd>
<div class="block">draw high scoring fragments that are used for the initial alignment seed 
 selection</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/JMatrixPanel.html#drawDistances(java.awt.Graphics)">drawDistances(Graphics)</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/JMatrixPanel.html" title="class in org.biojava.bio.structure.gui">JMatrixPanel</a></dt>
<dd>
<div class="block">For each element in matrix, draw it as a colored square or pixel.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html#drawIdx(java.awt.Graphics2D, int)">drawIdx(Graphics2D, int)</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html" title="class in org.biojava.bio.structure.gui.util">SequenceScalePanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/JMatrixPanel.html#drawPairs(java.awt.Graphics)">drawPairs(Graphics)</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/JMatrixPanel.html" title="class in org.biojava.bio.structure.gui">JMatrixPanel</a></dt>
<dd>
<div class="block">draw alternative alignments</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html#drawScale(java.awt.Graphics2D, int)">drawScale(Graphics2D, int)</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html" title="class in org.biojava.bio.structure.gui.util">SequenceScalePanel</a></dt>
<dd>
<div class="block">draw the Scale</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html#drawSequence(java.awt.Graphics2D, int)">drawSequence(Graphics2D, int)</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html" title="class in org.biojava.bio.structure.gui.util">SequenceScalePanel</a></dt>
<dd>
<div class="block">draw the Amino acid sequence</div>
</dd>
</dl>
<a name="_E_">
<!--   -->
</a>
<h2 class="title">E</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/Constraints.html#E">E</a></span> - Static variable in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/Constraints.html" title="class in org.biojava3.aaproperties">Constraints</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/edits/Edit.html" title="interface in org.biojava3.core.sequence.edits"><span class="strong">Edit</span></a>&lt;<a href="./org/biojava3/core/sequence/edits/Edit.html" title="type parameter in Edit">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava3/core/sequence/edits/package-summary.html">org.biojava3.core.sequence.edits</a></dt>
<dd>
<div class="block">Interface for carrying out edit operations on a Sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/edits/Edit.AbstractEdit.html#edit(org.biojava3.core.sequence.template.Sequence)">edit(Sequence&lt;C&gt;)</a></span> - Method in class org.biojava3.core.sequence.edits.<a href="./org/biojava3/core/sequence/edits/Edit.AbstractEdit.html" title="class in org.biojava3.core.sequence.edits">Edit.AbstractEdit</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/edits/Edit.html#edit(org.biojava3.core.sequence.template.Sequence)">edit(Sequence&lt;C&gt;)</a></span> - Method in interface org.biojava3.core.sequence.edits.<a href="./org/biojava3/core/sequence/edits/Edit.html" title="interface in org.biojava3.core.sequence.edits">Edit</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/edits/Edit.AbstractEdit.html" title="class in org.biojava3.core.sequence.edits"><span class="strong">Edit.AbstractEdit</span></a>&lt;<a href="./org/biojava3/core/sequence/edits/Edit.AbstractEdit.html" title="type parameter in Edit.AbstractEdit">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/core/sequence/edits/package-summary.html">org.biojava3.core.sequence.edits</a></dt>
<dd>
<div class="block">Abstract class which defines all edit operations as a call to discover
 what 5' and 3' ends of an editing Sequence should be joined together
 with a target Sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/edits/Edit.AbstractEdit.html#Edit.AbstractEdit(int)">Edit.AbstractEdit(int)</a></span> - Constructor for class org.biojava3.core.sequence.edits.<a href="./org/biojava3/core/sequence/edits/Edit.AbstractEdit.html" title="class in org.biojava3.core.sequence.edits">Edit.AbstractEdit</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/edits/Edit.AbstractEdit.html#Edit.AbstractEdit(int, int)">Edit.AbstractEdit(int, int)</a></span> - Constructor for class org.biojava3.core.sequence.edits.<a href="./org/biojava3/core/sequence/edits/Edit.AbstractEdit.html" title="class in org.biojava3.core.sequence.edits">Edit.AbstractEdit</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/edits/Edit.Delete.html" title="class in org.biojava3.core.sequence.edits"><span class="strong">Edit.Delete</span></a>&lt;<a href="./org/biojava3/core/sequence/edits/Edit.Delete.html" title="type parameter in Edit.Delete">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/core/sequence/edits/package-summary.html">org.biojava3.core.sequence.edits</a></dt>
<dd>
<div class="block">Implementation which allows for the deletion of bases from a Sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/edits/Edit.Delete.html#Edit.Delete(int)">Edit.Delete(int)</a></span> - Constructor for class org.biojava3.core.sequence.edits.<a href="./org/biojava3/core/sequence/edits/Edit.Delete.html" title="class in org.biojava3.core.sequence.edits">Edit.Delete</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/edits/Edit.Delete.html#Edit.Delete(int, int)">Edit.Delete(int, int)</a></span> - Constructor for class org.biojava3.core.sequence.edits.<a href="./org/biojava3/core/sequence/edits/Edit.Delete.html" title="class in org.biojava3.core.sequence.edits">Edit.Delete</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/edits/Edit.Insert.html" title="class in org.biojava3.core.sequence.edits"><span class="strong">Edit.Insert</span></a>&lt;<a href="./org/biojava3/core/sequence/edits/Edit.Insert.html" title="type parameter in Edit.Insert">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/core/sequence/edits/package-summary.html">org.biojava3.core.sequence.edits</a></dt>
<dd>
<div class="block">Edit implementation which allows us to insert a base at any position
 in a Sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/edits/Edit.Insert.html#Edit.Insert(java.lang.String, int)">Edit.Insert(String, int)</a></span> - Constructor for class org.biojava3.core.sequence.edits.<a href="./org/biojava3/core/sequence/edits/Edit.Insert.html" title="class in org.biojava3.core.sequence.edits">Edit.Insert</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/edits/Edit.Insert.html#Edit.Insert(org.biojava3.core.sequence.template.Sequence, int)">Edit.Insert(Sequence&lt;C&gt;, int)</a></span> - Constructor for class org.biojava3.core.sequence.edits.<a href="./org/biojava3/core/sequence/edits/Edit.Insert.html" title="class in org.biojava3.core.sequence.edits">Edit.Insert</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/edits/Edit.Insert.html#Edit.Insert(java.lang.String, int, int)">Edit.Insert(String, int, int)</a></span> - Constructor for class org.biojava3.core.sequence.edits.<a href="./org/biojava3/core/sequence/edits/Edit.Insert.html" title="class in org.biojava3.core.sequence.edits">Edit.Insert</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/edits/Edit.Insert.html#Edit.Insert(org.biojava3.core.sequence.template.Sequence, int, int)">Edit.Insert(Sequence&lt;C&gt;, int, int)</a></span> - Constructor for class org.biojava3.core.sequence.edits.<a href="./org/biojava3/core/sequence/edits/Edit.Insert.html" title="class in org.biojava3.core.sequence.edits">Edit.Insert</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/edits/Edit.Substitute.html" title="class in org.biojava3.core.sequence.edits"><span class="strong">Edit.Substitute</span></a>&lt;<a href="./org/biojava3/core/sequence/edits/Edit.Substitute.html" title="type parameter in Edit.Substitute">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/core/sequence/edits/package-summary.html">org.biojava3.core.sequence.edits</a></dt>
<dd>
<div class="block">Allows for the substitution of bases into an existing Sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/edits/Edit.Substitute.html#Edit.Substitute(java.lang.String, int)">Edit.Substitute(String, int)</a></span> - Constructor for class org.biojava3.core.sequence.edits.<a href="./org/biojava3/core/sequence/edits/Edit.Substitute.html" title="class in org.biojava3.core.sequence.edits">Edit.Substitute</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/edits/Edit.Substitute.html#Edit.Substitute(org.biojava3.core.sequence.template.Sequence, int)">Edit.Substitute(Sequence&lt;C&gt;, int)</a></span> - Constructor for class org.biojava3.core.sequence.edits.<a href="./org/biojava3/core/sequence/edits/Edit.Substitute.html" title="class in org.biojava3.core.sequence.edits">Edit.Substitute</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#eig()">eig()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Eigenvalue Decomposition</div>
</dd>
<dt><a href="./org/biojava/bio/structure/jama/EigenvalueDecomposition.html" title="class in org.biojava.bio.structure.jama"><span class="strong">EigenvalueDecomposition</span></a> - Class in <a href="./org/biojava/bio/structure/jama/package-summary.html">org.biojava.bio.structure.jama</a></dt>
<dd>
<div class="block">Eigenvalues and eigenvectors of a real matrix.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/EigenvalueDecomposition.html#EigenvalueDecomposition(org.biojava.bio.structure.jama.Matrix)">EigenvalueDecomposition(Matrix)</a></span> - Constructor for class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/EigenvalueDecomposition.html" title="class in org.biojava.bio.structure.jama">EigenvalueDecomposition</a></dt>
<dd>
<div class="block">Check for symmetry, then construct the eigenvalue decomposition
       provides a  way  to access D and V.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure"><span class="strong">Element</span></a> - Enum in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">Element is an enumeration of the elements of the periodic table.</div>
</dd>
<dt><a href="./org/biojava3/aaproperties/xml/Element.html" title="class in org.biojava3.aaproperties.xml"><span class="strong">Element</span></a> - Class in <a href="./org/biojava3/aaproperties/xml/package-summary.html">org.biojava3.aaproperties.xml</a></dt>
<dd>
<div class="block">One way to model the elements</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/Element.html#Element()">Element()</a></span> - Constructor for class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/Element.html" title="class in org.biojava3.aaproperties.xml">Element</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/Element.html#Element(java.lang.String, java.lang.String, int, java.util.List, double)">Element(String, String, int, List&lt;Isotope&gt;, double)</a></span> - Constructor for class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/Element.html" title="class in org.biojava3.aaproperties.xml">Element</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/aaproperties/xml/ElementTable.html" title="class in org.biojava3.aaproperties.xml"><span class="strong">ElementTable</span></a> - Class in <a href="./org/biojava3/aaproperties/xml/package-summary.html">org.biojava3.aaproperties.xml</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/ElementTable.html#ElementTable()">ElementTable()</a></span> - Constructor for class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/ElementTable.html" title="class in org.biojava3.aaproperties.xml">ElementTable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/ElementTable.html#ElementTable(java.util.List)">ElementTable(List&lt;Element&gt;)</a></span> - Constructor for class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/ElementTable.html" title="class in org.biojava3.aaproperties.xml">ElementTable</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/ElementType.html" title="enum in org.biojava.bio.structure"><span class="strong">ElementType</span></a> - Enum in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">ElementType is an enumeration of the types of elements found in the periodic table.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/Location.html#EMPTY">EMPTY</a></span> - Static variable in interface org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/Location.html" title="interface in org.biojava3.core.sequence.location.template">Location</a></dt>
<dd>
<div class="block">Basic location which is set to the minimum and maximum bounds of
 <a href="http://java.sun.com/j2se/1.6.0/docs/api/java/lang/Integer.html?is-external=true" title="class or interface in java.lang"><code>Integer</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Location.html#end()">end()</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Location.html" title="class in org.biojava3.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Get the ending index.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html#ENDS">ENDS</a></span> - Static variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html" title="class in org.biojava.bio.structure.domain.pdp">PDPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Location.html#endsAfter(org.biojava3.genome.parsers.gff.Location)">endsAfter(Location)</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Location.html" title="class in org.biojava3.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Check if this location ends after other location ends.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Location.html#endsBefore(org.biojava3.genome.parsers.gff.Location)">endsBefore(Location)</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Location.html" title="class in org.biojava3.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Check if this location ends before other location ends.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html#ENDSEND">ENDSEND</a></span> - Static variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html" title="class in org.biojava.bio.structure.domain.pdp">PDPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#ensureClaInstalled()">ensureClaInstalled()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#ensureDesInstalled()">ensureDesInstalled()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#ensureHieInstalled()">ensureHieInstalled()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model"><span class="strong">Entity</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a></dt>
<dd>
<div class="block">A simple class to represent Entity records in mmCif files</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html#Entity()">Entity()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model">Entity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/chem/PolymerType.html#entity_poly_type">entity_poly_type</a></span> - Variable in enum org.biojava.bio.structure.io.mmcif.chem.<a href="./org/biojava/bio/structure/io/mmcif/chem/PolymerType.html" title="enum in org.biojava.bio.structure.io.mmcif.chem">PolymerType</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/EntityPolySeq.html" title="class in org.biojava.bio.structure.io.mmcif.model"><span class="strong">EntityPolySeq</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a></dt>
<dd>
<div class="block">Container for _entity_poly_seq records


  Field Name                    mmCIF Data Item
  Section                       n.a.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/EntityPolySeq.html#EntityPolySeq()">EntityPolySeq()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/EntityPolySeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">EntityPolySeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html#entrySet()">entrySet()</a></span> - Method in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html" title="class in org.biojava.bio.structure.gui.util.color">GradientMapper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/SoftHashMap.html#entrySet()">entrySet()</a></span> - Method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/SoftHashMap.html" title="class in org.biojava3.core.util">SoftHashMap</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/MenuCreator.html#EQR_COLOR">EQR_COLOR</a></span> - Static variable in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/MenuCreator.html" title="class in org.biojava.bio.structure.align.gui">MenuCreator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/Equals.html#equal(int, int)">equal(int, int)</a></span> - Static method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/Equals.html" title="class in org.biojava3.core.util">Equals</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/Equals.html#equal(long, long)">equal(long, long)</a></span> - Static method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/Equals.html" title="class in org.biojava3.core.util">Equals</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/Equals.html#equal(boolean, boolean)">equal(boolean, boolean)</a></span> - Static method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/Equals.html" title="class in org.biojava3.core.util">Equals</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/Equals.html#equal(java.lang.Object, java.lang.Object)">equal(Object, Object)</a></span> - Static method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/Equals.html" title="class in org.biojava3.core.util">Equals</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/PdbPair.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/PdbPair.html" title="class in org.biojava.bio.structure.align.client">PdbPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/StructureName.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/StructureName.html" title="class in org.biojava.bio.structure.align.client">StructureName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Author.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Author.html" title="class in org.biojava.bio.structure">Author</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemCompDescriptor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#equals(org.biojava.bio.structure.PDBHeader)">equals(PDBHeader)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>
<div class="block">Compare two PDBHeader objects</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ResidueNumber.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ResidueNumber.html" title="class in org.biojava.bio.structure">ResidueNumber</a></dt>
<dd>
<div class="block">Require chainId, insCode, and seqNum to be equal</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDescription.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDescription.html" title="class in org.biojava.bio.structure.scop">ScopDescription</a></dt>
<dd>
<div class="block">Compares the fields sunID, category, classificationId, and name for equality</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Site.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Site.html" title="class in org.biojava.bio.structure">Site</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/AccessionID.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/AccessionID.html" title="class in org.biojava3.core.sequence">AccessionID</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/AminoAcidCompound.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/CodonCompound.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/CodonCompound.html" title="class in org.biojava3.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/NucleotideCompound.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/FuzzyPoint.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/FuzzyPoint.html" title="class in org.biojava3.core.sequence.location">FuzzyPoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/SimplePoint.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/SimplePoint.html" title="class in org.biojava3.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava3.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/ArrayListSequenceReader.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava3.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava3.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/BitSequenceReader.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava3.core.sequence.storage">BitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava3.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompound.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompound.html" title="class in org.biojava3.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompoundSet.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava3.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/CaseInsensitiveCompound.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava3.core.sequence.transcription">CaseInsensitiveCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/Table.CaseInsensitiveTriplet.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/Table.CaseInsensitiveTriplet.html" title="class in org.biojava3.core.sequence.transcription">Table.CaseInsensitiveTriplet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/Table.Codon.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/Table.Codon.html" title="class in org.biojava3.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/util/Equals.html" title="class in org.biojava3.core.util"><span class="strong">Equals</span></a> - Class in <a href="./org/biojava3/core/util/package-summary.html">org.biojava3.core.util</a></dt>
<dd>
<div class="block">A set of helper methods which return true if the two parameters are
 equal to each other.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/Equals.html#Equals()">Equals()</a></span> - Constructor for class org.biojava3.core.util.<a href="./org/biojava3/core/util/Equals.html" title="class in org.biojava3.core.util">Equals</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/data/sequence/FastaSequence.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava3.data.sequence.<a href="./org/biojava3/data/sequence/FastaSequence.html" title="class in org.biojava3.data.sequence">FastaSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Location.html#equals(org.biojava3.genome.parsers.gff.Location)">equals(Location)</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Location.html" title="class in org.biojava3.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Compare locations for equality.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationImpl.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationImpl.html" title="class in org.biojava3.protmod">ProteinModificationImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ModifiedCompoundImpl.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ModifiedCompoundImpl.html" title="class in org.biojava3.protmod.structure">ModifiedCompoundImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/StructureAtom.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/StructureAtom.html" title="class in org.biojava3.protmod.structure">StructureAtom</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/StructureAtomLinkage.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/StructureAtomLinkage.html" title="class in org.biojava3.protmod.structure">StructureAtomLinkage</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/StructureGroup.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/StructureGroup.html" title="class in org.biojava3.protmod.structure">StructureGroup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/ronn/Jronn.Range.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava3.ronn.<a href="./org/biojava3/ronn/Jronn.Range.html" title="class in org.biojava3.ronn">Jronn.Range</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/ronn/ModelLoader.Model.html#equals(java.lang.Object)">equals(Object)</a></span> - Method in class org.biojava3.ronn.<a href="./org/biojava3/ronn/ModelLoader.Model.html" title="class in org.biojava3.ronn">ModelLoader.Model</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/AminoAcidCompound.html#equalsIgnoreCase(org.biojava3.core.sequence.template.Compound)">equalsIgnoreCase(Compound)</a></span> - Method in class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/CodonCompound.html#equalsIgnoreCase(org.biojava3.core.sequence.template.Compound)">equalsIgnoreCase(Compound)</a></span> - Method in class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/CodonCompound.html" title="class in org.biojava3.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/NucleotideCompound.html#equalsIgnoreCase(org.biojava3.core.sequence.template.Compound)">equalsIgnoreCase(Compound)</a></span> - Method in class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompound.html#equalsIgnoreCase(org.biojava3.core.sequence.template.Compound)">equalsIgnoreCase(Compound)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompound.html" title="class in org.biojava3.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/Compound.html#equalsIgnoreCase(org.biojava3.core.sequence.template.Compound)">equalsIgnoreCase(Compound)</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/CaseInsensitiveCompound.html#equalsIgnoreCase(org.biojava3.core.sequence.template.Compound)">equalsIgnoreCase(Compound)</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava3.core.sequence.transcription">CaseInsensitiveCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/Table.Codon.html#equalsIgnoreCase(org.biojava3.core.sequence.template.Compound)">equalsIgnoreCase(Compound)</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/Table.Codon.html" title="class in org.biojava3.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/AlignedPosition.html#EQUIVALENT">EQUIVALENT</a></span> - Static variable in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/AlignedPosition.html" title="class in org.biojava.bio.structure.gui.util">AlignedPosition</a></dt>
<dd>
<div class="block">flag if this position is equivalent</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/MyJmolStatusListener.html#eval(java.lang.String)">eval(String)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/MyJmolStatusListener.html" title="class in org.biojava.bio.structure.align.gui.jmol">MyJmolStatusListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/JmolPanel.html#evalString(java.lang.String)">evalString(String)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/JmolPanel.html" title="class in org.biojava.bio.structure.align.gui.jmol">JmolPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html#evalString(java.lang.String)">evalString(String)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html" title="class in org.biojava.bio.structure.align.gui.jmol">StructureAlignmentJmol</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/BiojavaJmol.html#evalString(java.lang.String)">evalString(String)</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/BiojavaJmol.html" title="class in org.biojava.bio.structure.gui">BiojavaJmol</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/JmolPanel.html#executeCmd(java.lang.String)">executeCmd(String)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/JmolPanel.html" title="class in org.biojava.bio.structure.align.gui.jmol">JmolPanel</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/ExonComparator.html" title="class in org.biojava3.core.sequence"><span class="strong">ExonComparator</span></a> - Class in <a href="./org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a></dt>
<dd>
<div class="block">Sort Exon where it is a little confusing if exons shoud always be ordered left to right
 where a negative stranded gene should go the other direction.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/ExonComparator.html#ExonComparator()">ExonComparator()</a></span> - Constructor for class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/ExonComparator.html" title="class in org.biojava3.core.sequence">ExonComparator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/ExonSequence.html" title="class in org.biojava3.core.sequence"><span class="strong">ExonSequence</span></a> - Class in <a href="./org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a></dt>
<dd>
<div class="block">A gene contains a collection of Exon sequences</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/ExonSequence.html#ExonSequence(org.biojava3.core.sequence.GeneSequence, int, int)">ExonSequence(GeneSequence, int, int)</a></span> - Constructor for class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/ExonSequence.html" title="class in org.biojava3.core.sequence">ExonSequence</a></dt>
<dd>
<div class="block">Need a parent gene sequence and the bioBegin and bioEnd.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html" title="class in org.biojava.bio.structure.io.mmcif.model"><span class="strong">Exptl</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html#Exptl()">Exptl()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html" title="class in org.biojava.bio.structure.io.mmcif.model">Exptl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentJoiner.html#extendFragments(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], org.biojava.bio.structure.align.helper.JointFragments[], org.biojava.bio.structure.align.StrucAligParameters)">extendFragments(Atom[], Atom[], JointFragments[], StrucAligParameters)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentJoiner.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentJoiner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentJoiner.html#extendFragments(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], org.biojava.bio.structure.align.helper.JointFragments, org.biojava.bio.structure.align.StrucAligParameters)">extendFragments(Atom[], Atom[], JointFragments, StrucAligParameters)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentJoiner.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentJoiner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/AFPCalculator.html#extractAFPChains(org.biojava.bio.structure.align.fatcat.calc.FatCatParameters, org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">extractAFPChains(FatCatParameters, AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/AFPCalculator.html" title="class in org.biojava.bio.structure.align.fatcat.calc">AFPCalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#extractFragments(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">extractFragments(AFPChain, Atom[], Atom[])</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_F_">
<!--   -->
</a>
<h2 class="title">F</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/Constraints.html#F">F</a></span> - Static variable in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/Constraints.html" title="class in org.biojava3.aaproperties">Constraints</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/FarmJob.html" title="class in org.biojava.bio.structure.align"><span class="strong">FarmJob</span></a> - Class in <a href="./org/biojava/bio/structure/align/package-summary.html">org.biojava.bio.structure.align</a></dt>
<dd>
<div class="block">A job as it can be run on the farm.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/FarmJob.html#FarmJob()">FarmJob()</a></span> - Constructor for class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/FarmJob.html" title="class in org.biojava.bio.structure.align">FarmJob</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client"><span class="strong">FarmJobParameters</span></a> - Class in <a href="./org/biojava/bio/structure/align/client/package-summary.html">org.biojava.bio.structure.align.client</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#FarmJobParameters()">FarmJobParameters()</a></span> - Constructor for class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html" title="class in org.biojava.bio.structure.align.client"><span class="strong">FarmJobRunnable</span></a> - Class in <a href="./org/biojava/bio/structure/align/client/package-summary.html">org.biojava.bio.structure.align.client</a></dt>
<dd>
<div class="block">Contains the single thread for a job that can run multiple alignments.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html#FarmJobRunnable(org.biojava.bio.structure.align.client.FarmJobParameters)">FarmJobRunnable(FarmJobParameters)</a></span> - Constructor for class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html" title="class in org.biojava.bio.structure.align.client">FarmJobRunnable</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/io/FastaGeneWriter.html" title="class in org.biojava3.core.sequence.io"><span class="strong">FastaGeneWriter</span></a> - Class in <a href="./org/biojava3/core/sequence/io/package-summary.html">org.biojava3.core.sequence.io</a></dt>
<dd>
<div class="block">A Gene sequence has a Positive or Negative Strand where we want to write out to a stream the 5 to 3 prime version.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/FastaGeneWriter.html#FastaGeneWriter(java.io.OutputStream, java.util.Collection, org.biojava3.core.sequence.io.template.FastaHeaderFormatInterface, boolean)">FastaGeneWriter(OutputStream, Collection&lt;GeneSequence&gt;, FastaHeaderFormatInterface&lt;GeneSequence, NucleotideCompound&gt;, boolean)</a></span> - Constructor for class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/FastaGeneWriter.html" title="class in org.biojava3.core.sequence.io">FastaGeneWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/FastaGeneWriter.html#FastaGeneWriter(java.io.OutputStream, java.util.Collection, org.biojava3.core.sequence.io.template.FastaHeaderFormatInterface, boolean, int)">FastaGeneWriter(OutputStream, Collection&lt;GeneSequence&gt;, FastaHeaderFormatInterface&lt;GeneSequence, NucleotideCompound&gt;, boolean, int)</a></span> - Constructor for class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/FastaGeneWriter.html" title="class in org.biojava3.core.sequence.io">FastaGeneWriter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/io/template/FastaHeaderFormatInterface.html" title="interface in org.biojava3.core.sequence.io.template"><span class="strong">FastaHeaderFormatInterface</span></a>&lt;<a href="./org/biojava3/core/sequence/io/template/FastaHeaderFormatInterface.html" title="type parameter in FastaHeaderFormatInterface">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;?&gt;,<a href="./org/biojava3/core/sequence/io/template/FastaHeaderFormatInterface.html" title="type parameter in FastaHeaderFormatInterface">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava3/core/sequence/io/template/package-summary.html">org.biojava3.core.sequence.io.template</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/io/template/FastaHeaderParserInterface.html" title="interface in org.biojava3.core.sequence.io.template"><span class="strong">FastaHeaderParserInterface</span></a>&lt;<a href="./org/biojava3/core/sequence/io/template/FastaHeaderParserInterface.html" title="type parameter in FastaHeaderParserInterface">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;?&gt;,<a href="./org/biojava3/core/sequence/io/template/FastaHeaderParserInterface.html" title="type parameter in FastaHeaderParserInterface">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava3/core/sequence/io/template/package-summary.html">org.biojava3.core.sequence.io.template</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/io/FastaReader.html" title="class in org.biojava3.core.sequence.io"><span class="strong">FastaReader</span></a>&lt;<a href="./org/biojava3/core/sequence/io/FastaReader.html" title="type parameter in FastaReader">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;?&gt;,<a href="./org/biojava3/core/sequence/io/FastaReader.html" title="type parameter in FastaReader">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/core/sequence/io/package-summary.html">org.biojava3.core.sequence.io</a></dt>
<dd>
<div class="block">Use FastaReaderHelper as an example of how to use this class where FastaReaderHelper should be the
 primary class used to read Fasta files</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/FastaReader.html#FastaReader(java.io.InputStream, org.biojava3.core.sequence.io.template.FastaHeaderParserInterface, org.biojava3.core.sequence.io.template.SequenceCreatorInterface)">FastaReader(InputStream, FastaHeaderParserInterface&lt;S, C&gt;, SequenceCreatorInterface&lt;C&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/FastaReader.html" title="class in org.biojava3.core.sequence.io">FastaReader</a></dt>
<dd>
<div class="block">If you are going to use FileProxyProteinSequenceCreator then do not use this constructor because we need details about
 local file offsets for quick reads.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/FastaReader.html#FastaReader(java.io.File, org.biojava3.core.sequence.io.template.FastaHeaderParserInterface, org.biojava3.core.sequence.io.template.SequenceCreatorInterface)">FastaReader(File, FastaHeaderParserInterface&lt;S, C&gt;, SequenceCreatorInterface&lt;C&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/FastaReader.html" title="class in org.biojava3.core.sequence.io">FastaReader</a></dt>
<dd>
<div class="block">If you are going to use the FileProxyProteinSequenceCreator then you need to use this constructor because we need details about
 the location of the file.</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/io/FastaReaderHelper.html" title="class in org.biojava3.core.sequence.io"><span class="strong">FastaReaderHelper</span></a> - Class in <a href="./org/biojava3/core/sequence/io/package-summary.html">org.biojava3.core.sequence.io</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/FastaReaderHelper.html#FastaReaderHelper()">FastaReaderHelper()</a></span> - Constructor for class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/FastaReaderHelper.html" title="class in org.biojava3.core.sequence.io">FastaReaderHelper</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/data/sequence/FastaSequence.html" title="class in org.biojava3.data.sequence"><span class="strong">FastaSequence</span></a> - Class in <a href="./org/biojava3/data/sequence/package-summary.html">org.biojava3.data.sequence</a></dt>
<dd>
<div class="block">A FASTA formatted sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/data/sequence/FastaSequence.html#FastaSequence(java.lang.String, java.lang.String)">FastaSequence(String, String)</a></span> - Constructor for class org.biojava3.data.sequence.<a href="./org/biojava3/data/sequence/FastaSequence.html" title="class in org.biojava3.data.sequence">FastaSequence</a></dt>
<dd>
<div class="block">Upon construction the any whitespace characters are removed from the
 sequence</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/io/FastaSequenceParser.html" title="class in org.biojava3.core.sequence.io"><span class="strong">FastaSequenceParser</span></a> - Class in <a href="./org/biojava3/core/sequence/io/package-summary.html">org.biojava3.core.sequence.io</a></dt>
<dd>
<div class="block">Used to parse a stream of a fasta file to get the sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/FastaSequenceParser.html#FastaSequenceParser()">FastaSequenceParser()</a></span> - Constructor for class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/FastaSequenceParser.html" title="class in org.biojava3.core.sequence.io">FastaSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/io/FastaWriter.html" title="class in org.biojava3.core.sequence.io"><span class="strong">FastaWriter</span></a>&lt;<a href="./org/biojava3/core/sequence/io/FastaWriter.html" title="type parameter in FastaWriter">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;?&gt;,<a href="./org/biojava3/core/sequence/io/FastaWriter.html" title="type parameter in FastaWriter">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/core/sequence/io/package-summary.html">org.biojava3.core.sequence.io</a></dt>
<dd>
<div class="block">The FastaWriter writes a collection of sequences to an outputStream.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/FastaWriter.html#FastaWriter(java.io.OutputStream, java.util.Collection, org.biojava3.core.sequence.io.template.FastaHeaderFormatInterface)">FastaWriter(OutputStream, Collection&lt;S&gt;, FastaHeaderFormatInterface&lt;S, C&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/FastaWriter.html" title="class in org.biojava3.core.sequence.io">FastaWriter</a></dt>
<dd>
<div class="block">Use default line length of 60</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/FastaWriter.html#FastaWriter(java.io.OutputStream, java.util.Collection, org.biojava3.core.sequence.io.template.FastaHeaderFormatInterface, int)">FastaWriter(OutputStream, Collection&lt;S&gt;, FastaHeaderFormatInterface&lt;S, C&gt;, int)</a></span> - Constructor for class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/FastaWriter.html" title="class in org.biojava3.core.sequence.io">FastaWriter</a></dt>
<dd>
<div class="block">Set custom lineLength</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/io/FastaWriterHelper.html" title="class in org.biojava3.core.sequence.io"><span class="strong">FastaWriterHelper</span></a> - Class in <a href="./org/biojava3/core/sequence/io/package-summary.html">org.biojava3.core.sequence.io</a></dt>
<dd>
<div class="block">The class that should be used to write out fasta file of a sequence collection</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/FastaWriterHelper.html#FastaWriterHelper()">FastaWriterHelper()</a></span> - Constructor for class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/FastaWriterHelper.html" title="class in org.biojava3.core.sequence.io">FastaWriterHelper</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/fatcat/FatCat.html" title="class in org.biojava.bio.structure.align.fatcat"><span class="strong">FatCat</span></a> - Class in <a href="./org/biojava/bio/structure/align/fatcat/package-summary.html">org.biojava.bio.structure.align.fatcat</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCat.html#FatCat()">FatCat()</a></span> - Constructor for class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCat.html" title="class in org.biojava.bio.structure.align.fatcat">FatCat</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/MenuCreator.html#FATCAT_BLOCK">FATCAT_BLOCK</a></span> - Static variable in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/MenuCreator.html" title="class in org.biojava.bio.structure.align.gui">MenuCreator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/MenuCreator.html#FATCAT_TEXT">FATCAT_TEXT</a></span> - Static variable in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/MenuCreator.html" title="class in org.biojava.bio.structure.align.gui">MenuCreator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatAligner.html" title="class in org.biojava.bio.structure.align.fatcat.calc"><span class="strong">FatCatAligner</span></a> - Class in <a href="./org/biojava/bio/structure/align/fatcat/calc/package-summary.html">org.biojava.bio.structure.align.fatcat.calc</a></dt>
<dd>
<div class="block">A class that does calculations on an AFPChain</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatAligner.html#FatCatAligner()">FatCatAligner()</a></span> - Constructor for class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatAligner.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatAligner</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/fatcat/FatCatFlexible.html" title="class in org.biojava.bio.structure.align.fatcat"><span class="strong">FatCatFlexible</span></a> - Class in <a href="./org/biojava/bio/structure/align/fatcat/package-summary.html">org.biojava.bio.structure.align.fatcat</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCatFlexible.html#FatCatFlexible()">FatCatFlexible()</a></span> - Constructor for class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCatFlexible.html" title="class in org.biojava.bio.structure.align.fatcat">FatCatFlexible</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc"><span class="strong">FatCatParameters</span></a> - Class in <a href="./org/biojava/bio/structure/align/fatcat/calc/package-summary.html">org.biojava.bio.structure.align.fatcat.calc</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#FatCatParameters()">FatCatParameters()</a></span> - Constructor for class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/fatcat/FatCatRigid.html" title="class in org.biojava.bio.structure.align.fatcat"><span class="strong">FatCatRigid</span></a> - Class in <a href="./org/biojava/bio/structure/align/fatcat/package-summary.html">org.biojava.bio.structure.align.fatcat</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCatRigid.html#FatCatRigid()">FatCatRigid()</a></span> - Constructor for class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCatRigid.html" title="class in org.biojava.bio.structure.align.fatcat">FatCatRigid</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/fatcat/FatCatUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.fatcat"><span class="strong">FatCatUserArgumentProcessor</span></a> - Class in <a href="./org/biojava/bio/structure/align/fatcat/package-summary.html">org.biojava.bio.structure.align.fatcat</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCatUserArgumentProcessor.html#FatCatUserArgumentProcessor()">FatCatUserArgumentProcessor()</a></span> - Constructor for class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCatUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.fatcat">FatCatUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/DBResultTable.html#fatColumnNames">fatColumnNames</a></span> - Static variable in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/DBResultTable.html" title="class in org.biojava.bio.structure.align.gui">DBResultTable</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/fatcat/calc/FCAlignHelper.html" title="class in org.biojava.bio.structure.align.fatcat.calc"><span class="strong">FCAlignHelper</span></a> - Class in <a href="./org/biojava/bio/structure/align/fatcat/calc/package-summary.html">org.biojava.bio.structure.align.fatcat.calc</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FCAlignHelper.html#FCAlignHelper(double[][], int, int, double, double)">FCAlignHelper(double[][], int, int, double, double)</a></span> - Constructor for class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FCAlignHelper.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FCAlignHelper</a></dt>
<dd>
<div class="block">do an alignment given the provided matrix sij0</div>
</dd>
<dt><a href="./org/biojava3/genome/parsers/gff/Feature.html" title="class in org.biojava3.genome.parsers.gff"><span class="strong">Feature</span></a> - Class in <a href="./org/biojava3/genome/parsers/gff/package-summary.html">org.biojava3.genome.parsers.gff</a></dt>
<dd>
<div class="block">A Feature corresponds to a single row in a GFF file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Feature.html#Feature(org.biojava3.genome.parsers.gff.Feature)">Feature(Feature)</a></span> - Constructor for class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Feature.html" title="class in org.biojava3.genome.parsers.gff">Feature</a></dt>
<dd>
<div class="block">Make a copy of the specified feature.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Feature.html#Feature(java.lang.String, java.lang.String, java.lang.String, org.biojava3.genome.parsers.gff.Location, java.lang.Double, int, java.lang.String)">Feature(String, String, String, Location, Double, int, String)</a></span> - Constructor for class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Feature.html" title="class in org.biojava3.genome.parsers.gff">Feature</a></dt>
<dd>
<div class="block">Construct a new Feature from raw data (usually a GFF row).</div>
</dd>
<dt><a href="./org/biojava3/genome/parsers/gff/FeatureHelper.html" title="class in org.biojava3.genome.parsers.gff"><span class="strong">FeatureHelper</span></a> - Class in <a href="./org/biojava3/genome/parsers/gff/package-summary.html">org.biojava3.genome.parsers.gff</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/FeatureHelper.html#FeatureHelper()">FeatureHelper()</a></span> - Constructor for class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/FeatureHelper.html" title="class in org.biojava3.genome.parsers.gff">FeatureHelper</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/genome/parsers/gff/FeatureI.html" title="interface in org.biojava3.genome.parsers.gff"><span class="strong">FeatureI</span></a> - Interface in <a href="./org/biojava3/genome/parsers/gff/package-summary.html">org.biojava3.genome.parsers.gff</a></dt>
<dd>
<div class="block">A feature on a sequence (for example, an exon or a gene), defined by a location
 and a set of attributes encoded as key/value pairs.</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/features/FeatureInterface.html" title="interface in org.biojava3.core.sequence.features"><span class="strong">FeatureInterface</span></a>&lt;<a href="./org/biojava3/core/sequence/features/FeatureInterface.html" title="type parameter in FeatureInterface">S</a> extends <a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a>&lt;<a href="./org/biojava3/core/sequence/features/FeatureInterface.html" title="type parameter in FeatureInterface">C</a>&gt;,<a href="./org/biojava3/core/sequence/features/FeatureInterface.html" title="type parameter in FeatureInterface">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava3/core/sequence/features/package-summary.html">org.biojava3.core.sequence.features</a></dt>
<dd>
<div class="block">Interface class to handle describing arbitrary features.</div>
</dd>
<dt><a href="./org/biojava3/genome/parsers/gff/FeatureList.html" title="class in org.biojava3.genome.parsers.gff"><span class="strong">FeatureList</span></a> - Class in <a href="./org/biojava3/genome/parsers/gff/package-summary.html">org.biojava3.genome.parsers.gff</a></dt>
<dd>
<div class="block">A list of FeatureI objects implemented using a Java ArrayList; corresponds to a GFF file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/FeatureList.html#FeatureList()">FeatureList()</a></span> - Constructor for class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/FeatureList.html" title="class in org.biojava3.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block">Construct an empty list.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/FeatureList.html#FeatureList(java.util.Collection)">FeatureList(Collection&lt;FeatureI&gt;)</a></span> - Constructor for class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/FeatureList.html" title="class in org.biojava3.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block">Construct a new list containing the same features
 as the specified list.</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/features/FeaturesKeyWordInterface.html" title="interface in org.biojava3.core.sequence.features"><span class="strong">FeaturesKeyWordInterface</span></a> - Interface in <a href="./org/biojava3/core/sequence/features/package-summary.html">org.biojava3.core.sequence.features</a></dt>
<dd>
<div class="block">Models the keywords that are annotated for a protein sequence at Uniprot.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#FIELD_LINE">FIELD_LINE</a></span> - Static variable in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/exceptions/FileAccessError.html" title="class in org.biojava3.core.exceptions"><span class="strong">FileAccessError</span></a> - Error in <a href="./org/biojava3/core/exceptions/package-summary.html">org.biojava3.core.exceptions</a></dt>
<dd>
<div class="block">Thrown from SequenceFileProxyLoader if error opening file using RandomnAccessFile</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/exceptions/FileAccessError.html#FileAccessError(java.lang.String)">FileAccessError(String)</a></span> - Constructor for error org.biojava3.core.exceptions.<a href="./org/biojava3/core/exceptions/FileAccessError.html" title="class in org.biojava3.core.exceptions">FileAccessError</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/FileConvert.html" title="class in org.biojava.bio.structure.io"><span class="strong">FileConvert</span></a> - Class in <a href="./org/biojava/bio/structure/io/package-summary.html">org.biojava.bio.structure.io</a></dt>
<dd>
<div class="block">Methods to convert a structure object into different file formats.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileConvert.html#FileConvert(org.biojava.bio.structure.Structure)">FileConvert(Structure)</a></span> - Constructor for class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileConvert.html" title="class in org.biojava.bio.structure.io">FileConvert</a></dt>
<dd>
<div class="block">Constructs a FileConvert object.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/io/FileParsingParameters.html" title="class in org.biojava.bio.structure.io"><span class="strong">FileParsingParameters</span></a> - Class in <a href="./org/biojava/bio/structure/io/package-summary.html">org.biojava.bio.structure.io</a></dt>
<dd>
<div class="block">A class that configures parameters that can be sent to the PDB file parsers
 
 
  <a href="./org/biojava/bio/structure/io/FileParsingParameters.html#setParseCAOnly(boolean)"><code>FileParsingParameters.setParseCAOnly(boolean)</code></a> - parse only the Atom records for C-alpha atoms
  <a href="./org/biojava/bio/structure/io/FileParsingParameters.html#setParseSecStruc(boolean)"><code>FileParsingParameters.setParseSecStruc(boolean)</code></a> - a flag if the secondary structure information from the PDB file (author's assignment) should be parsed.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileParsingParameters.html#FileParsingParameters()">FileParsingParameters()</a></span> - Constructor for class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileParsingParameters.html" title="class in org.biojava.bio.structure.io">FileParsingParameters</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/io/FileProxyDNASequenceCreator.html" title="class in org.biojava3.core.sequence.io"><span class="strong">FileProxyDNASequenceCreator</span></a> - Class in <a href="./org/biojava3/core/sequence/io/package-summary.html">org.biojava3.core.sequence.io</a></dt>
<dd>
<div class="block">This class is a good example of using the SequenceCreatorInterface where during parsing of the stream
 the sequence and the offset index are passed to create a Protein sequence that will be loaded in lazily.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/FileProxyDNASequenceCreator.html#FileProxyDNASequenceCreator(java.io.File, org.biojava3.core.sequence.template.CompoundSet)">FileProxyDNASequenceCreator(File, CompoundSet&lt;NucleotideCompound&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/FileProxyDNASequenceCreator.html" title="class in org.biojava3.core.sequence.io">FileProxyDNASequenceCreator</a></dt>
<dd>
<div class="block">Need File so that we can store full path name in SequenceFileProxyLoader for Random File access as a quick read</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/io/FileProxyProteinSequenceCreator.html" title="class in org.biojava3.core.sequence.io"><span class="strong">FileProxyProteinSequenceCreator</span></a> - Class in <a href="./org/biojava3/core/sequence/io/package-summary.html">org.biojava3.core.sequence.io</a></dt>
<dd>
<div class="block">This class is a good example of using the SequenceCreatorInterface where during parsing of the stream
 the sequence and the offset index are passed to create a Protein sequence that will be loaded in lazily.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/FileProxyProteinSequenceCreator.html#FileProxyProteinSequenceCreator(java.io.File, org.biojava3.core.sequence.template.CompoundSet)">FileProxyProteinSequenceCreator(File, CompoundSet&lt;AminoAcidCompound&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/FileProxyProteinSequenceCreator.html" title="class in org.biojava3.core.sequence.io">FileProxyProteinSequenceCreator</a></dt>
<dd>
<div class="block">Need File so that we can store full path name in SequenceFileProxyLoader for Random File access as a quick read</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/SandboxStyleStructureProvider.html#fileSeparator">fileSeparator</a></span> - Static variable in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/SandboxStyleStructureProvider.html" title="class in org.biojava.bio.structure.io">SandboxStyleStructureProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#FILESPLIT">FILESPLIT</a></span> - Static variable in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html#filterByClassificationId(java.lang.String)">filterByClassificationId(String)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">RemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDatabase.html#filterByClassificationId(java.lang.String)">filterByClassificationId(String)</a></span> - Method in interface org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDatabase.html" title="interface in org.biojava.bio.structure.scop">ScopDatabase</a></dt>
<dd>
<div class="block">Get all scop descriptions that start with a classifcation ID, e.g. b.1.18</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#filterByClassificationId(java.lang.String)">filterByClassificationId(String)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html#filterByDescription(java.lang.String)">filterByDescription(String)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">RemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDatabase.html#filterByDescription(java.lang.String)">filterByDescription(String)</a></span> - Method in interface org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDatabase.html" title="interface in org.biojava.bio.structure.scop">ScopDatabase</a></dt>
<dd>
<div class="block">Get all scop descriptions that start with a certain name. e.g.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#filterByDescription(java.lang.String)">filterByDescription(String)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html#filterByDomainName(java.lang.String)">filterByDomainName(String)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">RemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDatabase.html#filterByDomainName(java.lang.String)">filterByDomainName(String)</a></span> - Method in interface org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDatabase.html" title="interface in org.biojava.bio.structure.scop">ScopDatabase</a></dt>
<dd>
<div class="block">search through SCOP and filter based on domain name</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#filterByDomainName(java.lang.String)">filterByDomainName(String)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCPMain.html#filterDuplicateAFPs(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.align.ce.CECalculator, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">filterDuplicateAFPs(AFPChain, CECalculator, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCPMain.html" title="class in org.biojava.bio.structure.align.ce">CeCPMain</a></dt>
<dd>
<div class="block">Takes as input an AFPChain where ca2 has been artificially duplicated.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/StructureUtil.html#findAtomLinkages(org.biojava.bio.structure.Group, org.biojava.bio.structure.Group, boolean, double)">findAtomLinkages(Group, Group, boolean, double)</a></span> - Static method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/StructureUtil.html" title="class in org.biojava3.protmod.structure">StructureUtil</a></dt>
<dd>
<div class="block">Find linkages between two groups within tolerance of bond length,
 from potential atoms.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/StructureUtil.html#findAtomLinkages(org.biojava.bio.structure.Group, org.biojava.bio.structure.Group, java.util.List, java.util.List, boolean, double)">findAtomLinkages(Group, Group, List&lt;String&gt;, List&lt;String&gt;, boolean, double)</a></span> - Static method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/StructureUtil.html" title="class in org.biojava3.protmod.structure">StructureUtil</a></dt>
<dd>
<div class="block">Find linkages between two groups within tolerance of bond length,
 from potential atoms.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#findBonds(org.biojava.bio.structure.Group, java.util.List)">findBonds(Group, List&lt;Group&gt;)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#findChain(java.lang.String)">findChain(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">request a particular chain from a structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#findChain(java.lang.String, int)">findChain(String, int)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">request a particular chain from a particular model</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#findChain(java.lang.String, int)">findChain(String, int)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#findChain(java.lang.String)">findChain(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/CheckTreeAccuracy.html#findCommonParent(org.forester.phylogeny.PhylogenyNode)">findCommonParent(PhylogenyNode)</a></span> - Method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/CheckTreeAccuracy.html" title="class in org.biojava3.phylo">CheckTreeAccuracy</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#findGroup(java.lang.String, java.lang.String)">findGroup(String, String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">request a particular group from a structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#findGroup(java.lang.String, java.lang.String, int)">findGroup(String, String, int)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">request a particular group from a structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#findGroup(java.lang.String, java.lang.String, int)">findGroup(String, String, int)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#findGroup(java.lang.String, java.lang.String)">findGroup(String, String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/StructureUtil.html#findLinkage(org.biojava.bio.structure.Group, org.biojava.bio.structure.Group, java.lang.String, java.lang.String, double)">findLinkage(Group, Group, String, String, double)</a></span> - Static method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/StructureUtil.html" title="class in org.biojava3.protmod.structure">StructureUtil</a></dt>
<dd>
<div class="block">Find a linkage between two groups within tolerance of bond length.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/StructureUtil.html#findNearestAtomLinkage(org.biojava.bio.structure.Group, org.biojava.bio.structure.Group, java.util.List, java.util.List, boolean, double)">findNearestAtomLinkage(Group, Group, List&lt;String&gt;, List&lt;String&gt;, boolean, double)</a></span> - Static method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/StructureUtil.html" title="class in org.biojava3.protmod.structure">StructureUtil</a></dt>
<dd>
<div class="block">Find a linkage between two groups within tolerance of bond length,
 from potential atoms.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#finish(org.biojava.bio.structure.align.StrucAligParameters, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">finish(StrucAligParameters, Atom[], Atom[])</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/CutValues.html#first_cut">first_cut</a></span> - Variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/CutValues.html" title="class in org.biojava.bio.structure.domain.pdp">CutValues</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/util/FlatFileCache.html" title="class in org.biojava3.core.util"><span class="strong">FlatFileCache</span></a> - Class in <a href="./org/biojava3/core/util/package-summary.html">org.biojava3.core.util</a></dt>
<dd>
<div class="block">Provides a cache for storing multiple small files in memory.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/server/FlatFileInstallation.html" title="class in org.biojava.bio.structure.server"><span class="strong">FlatFileInstallation</span></a> - Class in <a href="./org/biojava/bio/structure/server/package-summary.html">org.biojava.bio.structure.server</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>&nbsp;</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/FlatFileInstallation.html#FlatFileInstallation(java.io.File)">FlatFileInstallation(File)</a></span> - Constructor for class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/FlatFileInstallation.html" title="class in org.biojava.bio.structure.server">FlatFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
<div class="block">create a new FlatFile Installation and point it to the directory that contains all PDB files</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/AFPChainXMLParser.html#flipAlignment(java.lang.String)">flipAlignment(String)</a></span> - Static method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/AFPChainXMLParser.html" title="class in org.biojava.bio.structure.align.xml">AFPChainXMLParser</a></dt>
<dd>
<div class="block">Takes an XML representation of the alignment and flips the positions of name1 and name2</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/AFPChainFlipper.html#flipChain(org.biojava.bio.structure.align.model.AFPChain)">flipChain(AFPChain)</a></span> - Static method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/AFPChainFlipper.html" title="class in org.biojava.bio.structure.align.xml">AFPChainFlipper</a></dt>
<dd>
<div class="block">Flip the position of name1 and name2 (as well as all underlying data) in an AFPChain.</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/storage/FourBitSequenceReader.html" title="class in org.biojava3.core.sequence.storage"><span class="strong">FourBitSequenceReader</span></a>&lt;<a href="./org/biojava3/core/sequence/storage/FourBitSequenceReader.html" title="type parameter in FourBitSequenceReader">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/core/sequence/storage/package-summary.html">org.biojava3.core.sequence.storage</a></dt>
<dd>
<div class="block">Four bit encoding of the bit formats.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/FourBitSequenceReader.html#FourBitSequenceReader(org.biojava3.core.sequence.template.Sequence)">FourBitSequenceReader(Sequence&lt;C&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/FourBitSequenceReader.html" title="class in org.biojava3.core.sequence.storage">FourBitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/FourBitSequenceReader.html#FourBitSequenceReader(java.lang.String, org.biojava3.core.sequence.template.CompoundSet)">FourBitSequenceReader(String, CompoundSet&lt;C&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/FourBitSequenceReader.html" title="class in org.biojava3.core.sequence.storage">FourBitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/FourBitSequenceReader.html#FourBitSequenceReader(java.lang.String, org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.AccessionID)">FourBitSequenceReader(String, CompoundSet&lt;C&gt;, AccessionID)</a></span> - Constructor for class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/FourBitSequenceReader.html" title="class in org.biojava3.core.sequence.storage">FourBitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/FourBitSequenceReader.html#FourBitSequenceReader(org.biojava3.core.sequence.storage.FourBitSequenceReader.FourBitArrayWorker)">FourBitSequenceReader(FourBitSequenceReader.FourBitArrayWorker&lt;C&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/FourBitSequenceReader.html" title="class in org.biojava3.core.sequence.storage">FourBitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/FourBitSequenceReader.html#FourBitSequenceReader(org.biojava3.core.sequence.storage.FourBitSequenceReader.FourBitArrayWorker, org.biojava3.core.sequence.AccessionID)">FourBitSequenceReader(FourBitSequenceReader.FourBitArrayWorker&lt;C&gt;, AccessionID)</a></span> - Constructor for class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/FourBitSequenceReader.html" title="class in org.biojava3.core.sequence.storage">FourBitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html" title="class in org.biojava3.core.sequence.storage"><span class="strong">FourBitSequenceReader.FourBitArrayWorker</span></a>&lt;<a href="./org/biojava3/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html" title="type parameter in FourBitSequenceReader.FourBitArrayWorker">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/core/sequence/storage/package-summary.html">org.biojava3.core.sequence.storage</a></dt>
<dd>
<div class="block">A four bit per compound implementation of the bit array worker code.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html#FourBitSequenceReader.FourBitArrayWorker(org.biojava3.core.sequence.template.CompoundSet, int)">FourBitSequenceReader.FourBitArrayWorker(CompoundSet&lt;C&gt;, int)</a></span> - Constructor for class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html" title="class in org.biojava3.core.sequence.storage">FourBitSequenceReader.FourBitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html#FourBitSequenceReader.FourBitArrayWorker(org.biojava3.core.sequence.template.CompoundSet, int[])">FourBitSequenceReader.FourBitArrayWorker(CompoundSet&lt;C&gt;, int[])</a></span> - Constructor for class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html" title="class in org.biojava3.core.sequence.storage">FourBitSequenceReader.FourBitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html#FourBitSequenceReader.FourBitArrayWorker(org.biojava3.core.sequence.template.Sequence)">FourBitSequenceReader.FourBitArrayWorker(Sequence&lt;C&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html" title="class in org.biojava3.core.sequence.storage">FourBitSequenceReader.FourBitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html#FourBitSequenceReader.FourBitArrayWorker(java.lang.String, org.biojava3.core.sequence.template.CompoundSet)">FourBitSequenceReader.FourBitArrayWorker(String, CompoundSet&lt;C&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html" title="class in org.biojava3.core.sequence.storage">FourBitSequenceReader.FourBitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/alignment/FractionalIdentityInProfileScorer.html" title="class in org.biojava3.alignment"><span class="strong">FractionalIdentityInProfileScorer</span></a>&lt;<a href="./org/biojava3/alignment/FractionalIdentityInProfileScorer.html" title="type parameter in FractionalIdentityInProfileScorer">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava3/alignment/FractionalIdentityInProfileScorer.html" title="type parameter in FractionalIdentityInProfileScorer">C</a>&gt;,<a href="./org/biojava3/alignment/FractionalIdentityInProfileScorer.html" title="type parameter in FractionalIdentityInProfileScorer">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/alignment/package-summary.html">org.biojava3.alignment</a></dt>
<dd>
<div class="block">Implements an algorithm which computes a score for a sequence alignment pair picked from an alignment
 <a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template"><code>Profile</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/FractionalIdentityInProfileScorer.html#FractionalIdentityInProfileScorer(org.biojava3.alignment.template.Profile, int, int)">FractionalIdentityInProfileScorer(Profile&lt;S, C&gt;, int, int)</a></span> - Constructor for class org.biojava3.alignment.<a href="./org/biojava3/alignment/FractionalIdentityInProfileScorer.html" title="class in org.biojava3.alignment">FractionalIdentityInProfileScorer</a></dt>
<dd>
<div class="block">Creates a fractional identity scorer for an aligned pair of sequences in the given alignment profile.</div>
</dd>
<dt><a href="./org/biojava3/alignment/FractionalIdentityScorer.html" title="class in org.biojava3.alignment"><span class="strong">FractionalIdentityScorer</span></a>&lt;<a href="./org/biojava3/alignment/FractionalIdentityScorer.html" title="type parameter in FractionalIdentityScorer">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava3/alignment/FractionalIdentityScorer.html" title="type parameter in FractionalIdentityScorer">C</a>&gt;,<a href="./org/biojava3/alignment/FractionalIdentityScorer.html" title="type parameter in FractionalIdentityScorer">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/alignment/package-summary.html">org.biojava3.alignment</a></dt>
<dd>
<div class="block">Implements an algorithm which computes a score for a sequence alignment pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/FractionalIdentityScorer.html#FractionalIdentityScorer(org.biojava3.alignment.template.PairwiseSequenceAligner)">FractionalIdentityScorer(PairwiseSequenceAligner&lt;S, C&gt;)</a></span> - Constructor for class org.biojava3.alignment.<a href="./org/biojava3/alignment/FractionalIdentityScorer.html" title="class in org.biojava3.alignment">FractionalIdentityScorer</a></dt>
<dd>
<div class="block">Creates a fractional identity scorer for a pair of sequences aligned by the given pairwise sequence aligner.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/FractionalIdentityScorer.html#FractionalIdentityScorer(org.biojava3.alignment.template.SequencePair)">FractionalIdentityScorer(SequencePair&lt;S, C&gt;)</a></span> - Constructor for class org.biojava3.alignment.<a href="./org/biojava3/alignment/FractionalIdentityScorer.html" title="class in org.biojava3.alignment">FractionalIdentityScorer</a></dt>
<dd>
<div class="block">Creates a fractional identity scorer for an aligned pair of sequences.</div>
</dd>
<dt><a href="./org/biojava3/alignment/FractionalSimilarityInProfileScorer.html" title="class in org.biojava3.alignment"><span class="strong">FractionalSimilarityInProfileScorer</span></a>&lt;<a href="./org/biojava3/alignment/FractionalSimilarityInProfileScorer.html" title="type parameter in FractionalSimilarityInProfileScorer">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava3/alignment/FractionalSimilarityInProfileScorer.html" title="type parameter in FractionalSimilarityInProfileScorer">C</a>&gt;,<a href="./org/biojava3/alignment/FractionalSimilarityInProfileScorer.html" title="type parameter in FractionalSimilarityInProfileScorer">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/alignment/package-summary.html">org.biojava3.alignment</a></dt>
<dd>
<div class="block">Implements an algorithm which computes a score for a sequence alignment pair picked from an alignment
 <a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template"><code>Profile</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/FractionalSimilarityInProfileScorer.html#FractionalSimilarityInProfileScorer(org.biojava3.alignment.template.Profile, int, int)">FractionalSimilarityInProfileScorer(Profile&lt;S, C&gt;, int, int)</a></span> - Constructor for class org.biojava3.alignment.<a href="./org/biojava3/alignment/FractionalSimilarityInProfileScorer.html" title="class in org.biojava3.alignment">FractionalSimilarityInProfileScorer</a></dt>
<dd>
<div class="block">Creates a fractional similarity scorer for an aligned pair of sequences in the given alignment profile.</div>
</dd>
<dt><a href="./org/biojava3/alignment/FractionalSimilarityScorer.html" title="class in org.biojava3.alignment"><span class="strong">FractionalSimilarityScorer</span></a>&lt;<a href="./org/biojava3/alignment/FractionalSimilarityScorer.html" title="type parameter in FractionalSimilarityScorer">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava3/alignment/FractionalSimilarityScorer.html" title="type parameter in FractionalSimilarityScorer">C</a>&gt;,<a href="./org/biojava3/alignment/FractionalSimilarityScorer.html" title="type parameter in FractionalSimilarityScorer">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/alignment/package-summary.html">org.biojava3.alignment</a></dt>
<dd>
<div class="block">Implements an algorithm which computes a score for a sequence alignment pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/FractionalSimilarityScorer.html#FractionalSimilarityScorer(org.biojava3.alignment.template.PairwiseSequenceAligner)">FractionalSimilarityScorer(PairwiseSequenceAligner&lt;S, C&gt;)</a></span> - Constructor for class org.biojava3.alignment.<a href="./org/biojava3/alignment/FractionalSimilarityScorer.html" title="class in org.biojava3.alignment">FractionalSimilarityScorer</a></dt>
<dd>
<div class="block">Creates a fractional similarity scorer for a pair of sequences aligned by the given pairwise sequence aligner.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/FractionalSimilarityScorer.html#FractionalSimilarityScorer(org.biojava3.alignment.template.SequencePair)">FractionalSimilarityScorer(SequencePair&lt;S, C&gt;)</a></span> - Constructor for class org.biojava3.alignment.<a href="./org/biojava3/alignment/FractionalSimilarityScorer.html" title="class in org.biojava3.alignment">FractionalSimilarityScorer</a></dt>
<dd>
<div class="block">Creates a fractional similarity scorer for an aligned pair of sequences.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentJoiner.html#frag_pairwise_compat(org.biojava.bio.structure.align.pairwise.FragmentPair[], int, float, int)">frag_pairwise_compat(FragmentPair[], int, float, int)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentJoiner.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentJoiner</a></dt>
<dd>
<div class="block">Calculate the pairwise compatibility of fpairs.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/pairwise/FragmentJoiner.html" title="class in org.biojava.bio.structure.align.pairwise"><span class="strong">FragmentJoiner</span></a> - Class in <a href="./org/biojava/bio/structure/align/pairwise/package-summary.html">org.biojava.bio.structure.align.pairwise</a></dt>
<dd>
<div class="block">Joins the initial Fragments together to larger Fragments</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentJoiner.html#FragmentJoiner()">FragmentJoiner()</a></span> - Constructor for class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentJoiner.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentJoiner</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise"><span class="strong">FragmentPair</span></a> - Class in <a href="./org/biojava/bio/structure/align/pairwise/package-summary.html">org.biojava.bio.structure.align.pairwise</a></dt>
<dd>
<div class="block">a pair of fragments of two protein structures</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#FragmentPair(int, int, int)">FragmentPair(int, int, int)</a></span> - Constructor for class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/transcription/Frame.html" title="enum in org.biojava3.core.sequence.transcription"><span class="strong">Frame</span></a> - Enum in <a href="./org/biojava3/core/sequence/transcription/package-summary.html">org.biojava3.core.sequence.transcription</a></dt>
<dd>
<div class="block">Indicates a way of translating a sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Feature.html#frame()">frame()</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Feature.html" title="class in org.biojava3.genome.parsers.gff">Feature</a></dt>
<dd>
<div class="block">Get frame (aka phase).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Location.html#fromBio(int, int, char)">fromBio(int, int, char)</a></span> - Static method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Location.html" title="class in org.biojava3.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Create location from "biocoordinates", as in GFF file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Location.html#fromBioExt(int, int, char, int)">fromBioExt(int, int, char, int)</a></span> - Static method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Location.html" title="class in org.biojava3.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Create a location from MAF file coordinates, which represent negative
 strand locations as the distance from the end of the sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/HSVColorSpace.html#fromCIEXYZ(float[])">fromCIEXYZ(float[])</a></span> - Method in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/HSVColorSpace.html" title="class in org.biojava.bio.structure.gui.util.color">HSVColorSpace</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/HSVColorSpace.html#fromRGB(float[])">fromRGB(float[])</a></span> - Method in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/HSVColorSpace.html" title="class in org.biojava.bio.structure.gui.util.color">HSVColorSpace</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/UserConfiguration.html#fromStartupParams(org.biojava.bio.structure.align.ce.StartupParameters)">fromStartupParams(StartupParameters)</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/UserConfiguration.html" title="class in org.biojava.bio.structure.align.util">UserConfiguration</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/AtomInfo.html#fromString(java.lang.String)">fromString(String)</a></span> - Static method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/AtomInfo.html" title="class in org.biojava.bio.structure.align.gui.jmol">AtomInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBStatus.Status.html#fromString(java.lang.String)">fromString(String)</a></span> - Static method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBStatus.Status.html" title="enum in org.biojava.bio.structure">PDBStatus.Status</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ResidueNumber.html#fromString(java.lang.String)">fromString(String)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ResidueNumber.html" title="class in org.biojava.bio.structure">ResidueNumber</a></dt>
<dd>
<div class="block">Convert a string representation of a residue number to a residue number object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopCategory.html#fromString(java.lang.String)">fromString(String)</a></span> - Static method in enum org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopCategory.html" title="enum in org.biojava.bio.structure.scop">ScopCategory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/AFPChainXMLParser.html#fromXML(java.lang.String, java.lang.String, java.lang.String, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">fromXML(String, String, String, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/AFPChainXMLParser.html" title="class in org.biojava.bio.structure.align.xml">AFPChainXMLParser</a></dt>
<dd>
<div class="block">new utility method that checks that the order of the pair in the XML alignment is correct and flips the direction if needed</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/AFPChainXMLParser.html#fromXML(java.lang.String, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">fromXML(String, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/AFPChainXMLParser.html" title="class in org.biojava.bio.structure.align.xml">AFPChainXMLParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/HasResultXMLConverter.html#fromXML(java.lang.String)">fromXML(String)</a></span> - Method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/HasResultXMLConverter.html" title="class in org.biojava.bio.structure.align.xml">HasResultXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/PositionInQueueXMLConverter.html#fromXML(java.lang.String)">fromXML(String)</a></span> - Method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/PositionInQueueXMLConverter.html" title="class in org.biojava.bio.structure.align.xml">PositionInQueueXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/RepresentativeXMLConverter.html#fromXML(java.lang.String)">fromXML(String)</a></span> - Static method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/RepresentativeXMLConverter.html" title="class in org.biojava.bio.structure.align.xml">RepresentativeXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/ScopDescriptions.html#fromXML(java.lang.String)">fromXML(String)</a></span> - Static method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/ScopDescriptions.html" title="class in org.biojava.bio.structure.scop.server">ScopDescriptions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/ScopDomains.html#fromXML(java.lang.String)">fromXML(String)</a></span> - Static method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/ScopDomains.html" title="class in org.biojava.bio.structure.scop.server">ScopDomains</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/ScopNodes.html#fromXML(java.lang.String)">fromXML(String)</a></span> - Static method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/ScopNodes.html" title="class in org.biojava.bio.structure.scop.server">ScopNodes</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/io/ComponentXMLConverter.html#fromXML(java.lang.String)">fromXML(String)</a></span> - Static method in class org.biojava3.protmod.io.<a href="./org/biojava3/protmod/io/ComponentXMLConverter.html" title="class in org.biojava3.protmod.io">ComponentXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/io/ComponentXMLConverter.html#fromXML(org.w3c.dom.Node)">fromXML(Node)</a></span> - Static method in class org.biojava3.protmod.io.<a href="./org/biojava3/protmod/io/ComponentXMLConverter.html" title="class in org.biojava3.protmod.io">ComponentXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/io/ModifiedCompoundXMLConverter.html#fromXML(java.lang.String)">fromXML(String)</a></span> - Static method in class org.biojava3.protmod.io.<a href="./org/biojava3/protmod/io/ModifiedCompoundXMLConverter.html" title="class in org.biojava3.protmod.io">ModifiedCompoundXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/io/StructureAtomXMLConverter.html#fromXML(org.w3c.dom.Node)">fromXML(Node)</a></span> - Static method in class org.biojava3.protmod.io.<a href="./org/biojava3/protmod/io/StructureAtomXMLConverter.html" title="class in org.biojava3.protmod.io">StructureAtomXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/io/StructureGroupXMLConverter.html#fromXML(org.w3c.dom.Node)">fromXML(Node)</a></span> - Static method in class org.biojava3.protmod.io.<a href="./org/biojava3/protmod/io/StructureGroupXMLConverter.html" title="class in org.biojava3.protmod.io">StructureGroupXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/MyJmolStatusListener.html#functionXY(java.lang.String, int, int)">functionXY(String, int, int)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/MyJmolStatusListener.html" title="class in org.biojava.bio.structure.align.gui.jmol">MyJmolStatusListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/MyJmolStatusListener.html#functionXYZ(java.lang.String, int, int, int)">functionXYZ(String, int, int, int)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/MyJmolStatusListener.html" title="class in org.biojava.bio.structure.align.gui.jmol">MyJmolStatusListener</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/location/FuzzyPoint.html" title="class in org.biojava3.core.sequence.location"><span class="strong">FuzzyPoint</span></a> - Class in <a href="./org/biojava3/core/sequence/location/package-summary.html">org.biojava3.core.sequence.location</a></dt>
<dd>
<div class="block">Implementation for resolving fuzzy locations.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/FuzzyPoint.html#FuzzyPoint(int, int)">FuzzyPoint(int, int)</a></span> - Constructor for class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/FuzzyPoint.html" title="class in org.biojava3.core.sequence.location">FuzzyPoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/FuzzyPoint.html#FuzzyPoint(int, int, org.biojava3.core.sequence.location.template.Point.Resolver)">FuzzyPoint(int, int, Point.Resolver&lt;FuzzyPoint&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/FuzzyPoint.html" title="class in org.biojava3.core.sequence.location">FuzzyPoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/FuzzyPoint.html#FuzzyPoint(int, int, org.biojava3.core.sequence.location.template.Point.Resolver, boolean, boolean)">FuzzyPoint(int, int, Point.Resolver&lt;FuzzyPoint&gt;, boolean, boolean)</a></span> - Constructor for class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/FuzzyPoint.html" title="class in org.biojava3.core.sequence.location">FuzzyPoint</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_G_">
<!--   -->
</a>
<h2 class="title">G</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/Constraints.html#G">G</a></span> - Static variable in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/Constraints.html" title="class in org.biojava3.aaproperties">Constraints</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html#GAP_COLOR">GAP_COLOR</a></span> - Static variable in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html" title="class in org.biojava.bio.structure.gui.util">SequenceScalePanel</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/helper/GapArray.html" title="class in org.biojava.bio.structure.align.helper"><span class="strong">GapArray</span></a> - Class in <a href="./org/biojava/bio/structure/align/helper/package-summary.html">org.biojava.bio.structure.align.helper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/GapArray.html#GapArray()">GapArray()</a></span> - Constructor for class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/GapArray.html" title="class in org.biojava.bio.structure.align.helper">GapArray</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#gapPenalty">gapPenalty</a></span> - Variable in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava3.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/alignment/template/GapPenalty.html" title="interface in org.biojava3.alignment.template"><span class="strong">GapPenalty</span></a> - Interface in <a href="./org/biojava3/alignment/template/package-summary.html">org.biojava3.alignment.template</a></dt>
<dd>
<div class="block">Defines a data structure for the gap penalties used during a sequence alignment routine.</div>
</dd>
<dt><a href="./org/biojava3/alignment/template/GapPenalty.Type.html" title="enum in org.biojava3.alignment.template"><span class="strong">GapPenalty.Type</span></a> - Enum in <a href="./org/biojava3/alignment/template/package-summary.html">org.biojava3.alignment.template</a></dt>
<dd>
<div class="block">Defines the possible types of gap penalties.</div>
</dd>
<dt><a href="./org/biojava3/genome/parsers/gff/GCStats.html" title="class in org.biojava3.genome.parsers.gff"><span class="strong">GCStats</span></a> - Class in <a href="./org/biojava3/genome/parsers/gff/package-summary.html">org.biojava3.genome.parsers.gff</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/GCStats.html#GCStats()">GCStats()</a></span> - Constructor for class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/GCStats.html" title="class in org.biojava3.genome.parsers.gff">GCStats</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/genome/GeneFeatureHelper.html" title="class in org.biojava3.genome"><span class="strong">GeneFeatureHelper</span></a> - Class in <a href="./org/biojava3/genome/package-summary.html">org.biojava3.genome</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/GeneFeatureHelper.html#GeneFeatureHelper()">GeneFeatureHelper()</a></span> - Constructor for class org.biojava3.genome.<a href="./org/biojava3/genome/GeneFeatureHelper.html" title="class in org.biojava3.genome">GeneFeatureHelper</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/genome/parsers/gff/GeneIDGFF2Reader.html" title="class in org.biojava3.genome.parsers.gff"><span class="strong">GeneIDGFF2Reader</span></a> - Class in <a href="./org/biojava3/genome/parsers/gff/package-summary.html">org.biojava3.genome.parsers.gff</a></dt>
<dd>
<div class="block">http://www.bioperl.org/wiki/GTF
 Read and write FeatureLists as GFF/GTF formatted files.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/GeneIDGFF2Reader.html#GeneIDGFF2Reader()">GeneIDGFF2Reader()</a></span> - Constructor for class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/GeneIDGFF2Reader.html" title="class in org.biojava3.genome.parsers.gff">GeneIDGFF2Reader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/genome/parsers/geneid/GeneIDXMLReader.html" title="class in org.biojava3.genome.parsers.geneid"><span class="strong">GeneIDXMLReader</span></a> - Class in <a href="./org/biojava3/genome/parsers/geneid/package-summary.html">org.biojava3.genome.parsers.geneid</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/geneid/GeneIDXMLReader.html#GeneIDXMLReader(java.lang.String)">GeneIDXMLReader(String)</a></span> - Constructor for class org.biojava3.genome.parsers.geneid.<a href="./org/biojava3/genome/parsers/geneid/GeneIDXMLReader.html" title="class in org.biojava3.genome.parsers.geneid">GeneIDXMLReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/genome/parsers/gff/GeneMarkGTFReader.html" title="class in org.biojava3.genome.parsers.gff"><span class="strong">GeneMarkGTFReader</span></a> - Class in <a href="./org/biojava3/genome/parsers/gff/package-summary.html">org.biojava3.genome.parsers.gff</a></dt>
<dd>
<div class="block">http://www.bioperl.org/wiki/GTF
 Read and write FeatureLists as GFF/GTF formatted files.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/GeneMarkGTFReader.html#GeneMarkGTFReader()">GeneMarkGTFReader()</a></span> - Constructor for class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/GeneMarkGTFReader.html" title="class in org.biojava3.genome.parsers.gff">GeneMarkGTFReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#generateCompoundsToIndex()">generateCompoundsToIndex()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava3.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>
<div class="block">Returns what the value of a compound is in the backing bit storage i.e.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html#generateCompoundsToIndex()">generateCompoundsToIndex()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html" title="class in org.biojava3.core.sequence.storage">FourBitSequenceReader.FourBitArrayWorker</a></dt>
<dd>
<div class="block">Returns a Map which encodes the contents of CompoundSet.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html#generateCompoundsToIndex()">generateCompoundsToIndex()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html" title="class in org.biojava3.core.sequence.storage">TwoBitSequenceReader.TwoBitArrayWorker</a></dt>
<dd>
<div class="block">Returns a Map which encodes TCAG into positions 0,1,2,3.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#generateIndexToCompounds()">generateIndexToCompounds()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava3.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>
<div class="block">Should return the inverse information that <a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#generateCompoundsToIndex()"><code>BitSequenceReader.BitArrayWorker.generateCompoundsToIndex()</code></a>
 returns i.e. if the Compound C returns 1 from compoundsToIndex then we
 should find that compound here in position 1</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html#generateIndexToCompounds()">generateIndexToCompounds()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/FourBitSequenceReader.FourBitArrayWorker.html" title="class in org.biojava3.core.sequence.storage">FourBitSequenceReader.FourBitArrayWorker</a></dt>
<dd>
<div class="block">Returns a List which reverse encodes the Compound, Integer map</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html#generateIndexToCompounds()">generateIndexToCompounds()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html" title="class in org.biojava3.core.sequence.storage">TwoBitSequenceReader.TwoBitArrayWorker</a></dt>
<dd>
<div class="block">Returns a List which encodes TCAG into positions 0,1,2,3.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/InsdcParser.html#generatePoint(java.lang.String, java.lang.String)">generatePoint(String, String)</a></span> - Method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/InsdcParser.html" title="class in org.biojava3.core.sequence.location">InsdcParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/io/GenericFastaHeaderFormat.html" title="class in org.biojava3.core.sequence.io"><span class="strong">GenericFastaHeaderFormat</span></a>&lt;<a href="./org/biojava3/core/sequence/io/GenericFastaHeaderFormat.html" title="type parameter in GenericFastaHeaderFormat">S</a> extends <a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a>&lt;?&gt;,<a href="./org/biojava3/core/sequence/io/GenericFastaHeaderFormat.html" title="type parameter in GenericFastaHeaderFormat">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/core/sequence/io/package-summary.html">org.biojava3.core.sequence.io</a></dt>
<dd>
<div class="block">We store the original header if the sequence is parsed from a fasta file and will use that exact
 sequence if we write out the sequences to a fasta file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/GenericFastaHeaderFormat.html#GenericFastaHeaderFormat()">GenericFastaHeaderFormat()</a></span> - Constructor for class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/GenericFastaHeaderFormat.html" title="class in org.biojava3.core.sequence.io">GenericFastaHeaderFormat</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/io/GenericFastaHeaderParser.html" title="class in org.biojava3.core.sequence.io"><span class="strong">GenericFastaHeaderParser</span></a>&lt;<a href="./org/biojava3/core/sequence/io/GenericFastaHeaderParser.html" title="type parameter in GenericFastaHeaderParser">S</a> extends <a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a>&lt;<a href="./org/biojava3/core/sequence/io/GenericFastaHeaderParser.html" title="type parameter in GenericFastaHeaderParser">C</a>&gt;,<a href="./org/biojava3/core/sequence/io/GenericFastaHeaderParser.html" title="type parameter in GenericFastaHeaderParser">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/core/sequence/io/package-summary.html">org.biojava3.core.sequence.io</a></dt>
<dd>
<div class="block">The default fasta header parser where some headers are well defined based on the source
 database which allows us to set the source of the protein sequence and the identifier
 that can be used in future implementations to load features from external sources
 
 If the user has a custom header with local data then they can create their own implementation
 of a FastaHeaderParserInterface

 GenBank                           gi|gi-number|gb|accession|locus
 ENA Data Library                 gi|gi-number|emb|accession|locus
 DDBJ, DNA Database of Japan       gi|gi-number|dbj|accession|locus
 NBRF PIR                          pir||entry
 Protein Research Foundation       prf||name
 SWISS-PROT                        sp|accession|name
 Brookhaven Protein Data Bank (1)  pdb|entry|chain
 Brookhaven Protein Data Bank (2)  entry:chain|PDBID|CHAIN|SEQUENCE
 PDB EBI                           PDB:1ECY_A mol:protein length:142  ECOTIN
 Patents                           pat|country|number
 GenInfo Backbone Id               bbs|number
 General database identifier       gnl|database|identifier
 NCBI Reference Sequence           ref|accession|locus
 Local Sequence identifier         lcl|identifier</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/GenericFastaHeaderParser.html#GenericFastaHeaderParser()">GenericFastaHeaderParser()</a></span> - Constructor for class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/GenericFastaHeaderParser.html" title="class in org.biojava3.core.sequence.io">GenericFastaHeaderParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/GeneSequence.html" title="class in org.biojava3.core.sequence"><span class="strong">GeneSequence</span></a> - Class in <a href="./org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/GeneSequence.html#GeneSequence(org.biojava3.core.sequence.ChromosomeSequence, int, int, org.biojava3.core.sequence.Strand)">GeneSequence(ChromosomeSequence, int, int, Strand)</a></span> - Constructor for class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/GeneSequence.html" title="class in org.biojava3.core.sequence">GeneSequence</a></dt>
<dd>
<div class="block">A class that keeps track of the details of a GeneSequence which is difficult to properly model.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html#get(java.lang.Object)">get(Object)</a></span> - Method in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html" title="class in org.biojava.bio.structure.gui.util.color">GradientMapper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#get(int, int)">get(int, int)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Get a single element.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/views/WindowedSequence.html#get(int)">get(int)</a></span> - Method in class org.biojava3.core.sequence.views.<a href="./org/biojava3/core/sequence/views/WindowedSequence.html" title="class in org.biojava3.core.sequence.views">WindowedSequence</a></dt>
<dd>
<div class="block">Returns the window specified at the given index in offsets i.e. asking
 for position 2 in a moving window sequence of size 3 will get you
 the window starting at position 4.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/SoftHashMap.html#get(java.lang.Object)">get(Object)</a></span> - Method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/SoftHashMap.html" title="class in org.biojava3.core.util">SoftHashMap</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#get1LetterCode(java.lang.String)">get1LetterCode(String)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>
<div class="block">convert a three letter code into single character.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#getAA3Hash()">getAA3Hash()</a></span> - Static method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/IPeptideProperties.html#getAAComposition(org.biojava3.core.sequence.ProteinSequence)">getAAComposition(ProteinSequence)</a></span> - Method in interface org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/IPeptideProperties.html" title="interface in org.biojava3.aaproperties">IPeptideProperties</a></dt>
<dd>
<div class="block">Returns the composition of the 20 standard amino acid in the sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/PeptideProperties.html#getAAComposition(java.lang.String)">getAAComposition(String)</a></span> - Static method in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/PeptideProperties.html" title="class in org.biojava3.aaproperties">PeptideProperties</a></dt>
<dd>
<div class="block">An adaptor method to return the composition of the 20 standard amino acid in the sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/PeptidePropertiesImpl.html#getAAComposition(org.biojava3.core.sequence.ProteinSequence)">getAAComposition(ProteinSequence)</a></span> - Method in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/PeptidePropertiesImpl.html" title="class in org.biojava3.aaproperties">PeptidePropertiesImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/PeptideProperties.html#getAACompositionChar(java.lang.String)">getAACompositionChar(String)</a></span> - Static method in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/PeptideProperties.html" title="class in org.biojava3.aaproperties">PeptideProperties</a></dt>
<dd>
<div class="block">An adaptor method to return the composition of the 20 standard amino acid in the sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/PeptideProperties.html#getAACompositionString(java.lang.String)">getAACompositionString(String)</a></span> - Static method in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/PeptideProperties.html" title="class in org.biojava3.aaproperties">PeptideProperties</a></dt>
<dd>
<div class="block">An adaptor method to return the composition of the 20 standard amino acid in the sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/aaindex/AAindexFactory.html#getAAIndexProvider()">getAAIndexProvider()</a></span> - Static method in class org.biojava3.alignment.aaindex.<a href="./org/biojava3/alignment/aaindex/AAindexFactory.html" title="class in org.biojava3.alignment.aaindex">AAindexFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/MenuCreator.html#getAboutMenuItem()">getAboutMenuItem()</a></span> - Static method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/MenuCreator.html" title="class in org.biojava.bio.structure.align.gui">MenuCreator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/IPeptideProperties.html#getAbsorbance(org.biojava3.core.sequence.ProteinSequence, boolean)">getAbsorbance(ProteinSequence, boolean)</a></span> - Method in interface org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/IPeptideProperties.html" title="interface in org.biojava3.aaproperties">IPeptideProperties</a></dt>
<dd>
<div class="block">Returns the absorbance (optical density) of sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/PeptideProperties.html#getAbsorbance(java.lang.String, boolean)">getAbsorbance(String, boolean)</a></span> - Static method in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/PeptideProperties.html" title="class in org.biojava3.aaproperties">PeptideProperties</a></dt>
<dd>
<div class="block">An adaptor method to returns the absorbance (optical density) of sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/PeptidePropertiesImpl.html#getAbsorbance(org.biojava3.core.sequence.ProteinSequence, boolean)">getAbsorbance(ProteinSequence, boolean)</a></span> - Method in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/PeptidePropertiesImpl.html" title="class in org.biojava3.aaproperties">PeptidePropertiesImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html#getAbsorpt_coefficient_mu()">getAbsorpt_coefficient_mu()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html" title="class in org.biojava.bio.structure.io.mmcif.model">Exptl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html#getAbsorpt_correction_T_max()">getAbsorpt_correction_T_max()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html" title="class in org.biojava.bio.structure.io.mmcif.model">Exptl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html#getAbsorpt_correction_T_min()">getAbsorpt_correction_T_min()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html" title="class in org.biojava.bio.structure.io.mmcif.model">Exptl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html#getAbsorpt_correction_type()">getAbsorpt_correction_type()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html" title="class in org.biojava.bio.structure.io.mmcif.model">Exptl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html#getAbsorpt_process_details()">getAbsorpt_process_details()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html" title="class in org.biojava.bio.structure.io.mmcif.model">Exptl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleAlignedSequence.html#getAccession()">getAccession()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleAlignedSequence.html" title="class in org.biojava3.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html#getAccession()">getAccession()</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava3.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/StringProxySequenceReader.html#getAccession()">getAccession()</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html#getAccession()">getAccession()</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/InsdcParser.html#getAccession(java.lang.String)">getAccession(String)</a></span> - Method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/InsdcParser.html" title="class in org.biojava3.core.sequence.location">InsdcParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html#getAccession()">getAccession()</a></span> - Method in class org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava3.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/ArrayListSequenceReader.html#getAccession()">getAccession()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava3.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/BitSequenceReader.html#getAccession()">getAccession()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava3.core.sequence.storage">BitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/JoiningSequenceReader.html#getAccession()">getAccession()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava3.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html#getAccession()">getAccession()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava3.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Unsupoorted</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#getAccession()">getAccession()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/Accessioned.html#getAccession()">getAccession()</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/Accessioned.html" title="interface in org.biojava3.core.sequence.template">Accessioned</a></dt>
<dd>
<div class="block">Returns the AccessionID this location is currently bound with</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceProxyView.html#getAccession()">getAccession()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceProxyView.html" title="class in org.biojava3.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/JmolPanel.html#getAdapter()">getAdapter()</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/JmolPanel.html" title="class in org.biojava.bio.structure.align.gui.jmol">JmolPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getAfpAftIndex()">getAfpAftIndex()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getAfpBefIndex()">getAfpBefIndex()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatAligner.html#getAfpChain()">getAfpChain()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatAligner.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html#getAFPChain()">getAFPChain()</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AligPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/StatusDisplay.html#getAfpChain()">getAfpChain()</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/StatusDisplay.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">StatusDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/JFatCatClient.html#getAFPChainFromServer(java.lang.String, java.lang.String, java.lang.String, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">getAFPChainFromServer(String, String, String, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/JFatCatClient.html" title="class in org.biojava.bio.structure.align.client">JFatCatClient</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/JFatCatClient.html#getAFPChainFromServer(java.lang.String, java.lang.String, java.lang.String, java.lang.String, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], int)">getAFPChainFromServer(String, String, String, String, Atom[], Atom[], int)</a></span> - Static method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/JFatCatClient.html" title="class in org.biojava.bio.structure.align.client">JFatCatClient</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getAfpChainLen()">getAfpChainLen()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getAfpChainList()">getAfpChainList()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getAfpChainTwiBin()">getAfpChainTwiBin()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getAfpChainTwiList()">getAfpChainTwiList()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getAfpChainTwiNum()">getAfpChainTwiNum()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getAfpDisCut()">getAfpDisCut()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getAfpDisCut0()">getAfpDisCut0()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getAfpIndex()">getAfpIndex()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getAfpSet()">getAfpSet()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">Get the set of AFPs for this alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html#getAlgorithm()">getAlgorithm()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce">AbstractUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeSideChainUserArgumentProcessor.html#getAlgorithm()">getAlgorithm()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeSideChainUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce">CeSideChainUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeUserArgumentProcessor.html#getAlgorithm()">getAlgorithm()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce">CeUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCatUserArgumentProcessor.html#getAlgorithm()">getAlgorithm()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCatUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.fatcat">FatCatUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWatermanUserArgumentProcessor.html#getAlgorithm()">getAlgorithm()</a></span> - Method in class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWatermanUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.seq">SmithWatermanUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructureAlignmentFactory.html#getAlgorithm(java.lang.String)">getAlgorithm(String)</a></span> - Static method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructureAlignmentFactory.html" title="class in org.biojava.bio.structure.align">StructureAlignmentFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/AbstractStructureAlignment.html#getAlgorithmName()">getAlgorithmName()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/AbstractStructureAlignment.html" title="class in org.biojava.bio.structure.align">AbstractStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/BioJavaStructureAlignment.html#getAlgorithmName()">getAlgorithmName()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/BioJavaStructureAlignment.html" title="class in org.biojava.bio.structure.align">BioJavaStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCPMain.html#getAlgorithmName()">getAlgorithmName()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCPMain.html" title="class in org.biojava.bio.structure.align.ce">CeCPMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeMain.html#getAlgorithmName()">getAlgorithmName()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeMain.html" title="class in org.biojava.bio.structure.align.ce">CeMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeSideChainMain.html#getAlgorithmName()">getAlgorithmName()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeSideChainMain.html" title="class in org.biojava.bio.structure.align.ce">CeSideChainMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCatFlexible.html#getAlgorithmName()">getAlgorithmName()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCatFlexible.html" title="class in org.biojava.bio.structure.align.fatcat">FatCatFlexible</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCatRigid.html#getAlgorithmName()">getAlgorithmName()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCatRigid.html" title="class in org.biojava.bio.structure.align.fatcat">FatCatRigid</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getAlgorithmName()">getAlgorithmName()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWaterman3Daligner.html#getAlgorithmName()">getAlgorithmName()</a></span> - Method in class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWaterman3Daligner.html" title="class in org.biojava.bio.structure.align.seq">SmithWaterman3Daligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructureAlignment.html#getAlgorithmName()">getAlgorithmName()</a></span> - Method in interface org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructureAlignment.html" title="interface in org.biojava.bio.structure.align">StructureAlignment</a></dt>
<dd>
<div class="block">Get the name of the Algorithm</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/SequenceDisplay.html#getAligMap()">getAligMap()</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/SequenceDisplay.html" title="class in org.biojava.bio.structure.gui">SequenceDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/Alignable.html#getAligMat()">getAligMat()</a></span> - Method in interface org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/Alignable.html" title="interface in org.biojava.bio.structure.align.pairwise">Alignable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html#getAligMat(int, int)">getAligMat(int, int)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">StrCompAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html#getAligMat()">getAligMat()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">StrCompAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html#getAlign(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">getAlign(AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html" title="class in org.biojava.bio.structure.align.util">AFPAlignmentDisplay</a></dt>
<dd>
<div class="block">//extract the alignment output
   // eg
//       STWNTWACTWHLKQP--WSTILILA
//       111111111111     22222222
//       SQNNTYACSWKLKSWNNNSTILILG
   // those position pairs labeled by 1 and 2 are equivalent positions, belongs to
   // two blocks 1 and 2.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html#getAlign(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], boolean)">getAlign(AFPChain, Atom[], Atom[], boolean)</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html" title="class in org.biojava.bio.structure.align.util">AFPAlignmentDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#getAlign_id()">getAlign_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#getAlign_se1()">getAlign_se1()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#getAlign_se2()">getAlign_se2()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html#getAlignedAtoms1(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[])">getAlignedAtoms1(AFPChain, Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html" title="class in org.biojava.bio.structure.align.util">AFPAlignmentDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html#getAlignedAtoms2(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[])">getAlignedAtoms2(AFPChain, Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html" title="class in org.biojava.bio.structure.align.util">AFPAlignmentDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleProfile.html#getAlignedSequence(int)">getAlignedSequence(int)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleProfile.html" title="class in org.biojava3.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleProfile.html#getAlignedSequence(S)">getAlignedSequence(S)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleProfile.html" title="class in org.biojava3.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/Profile.html#getAlignedSequence(int)">getAlignedSequence(int)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns <a href="./org/biojava3/alignment/template/AlignedSequence.html" title="interface in org.biojava3.alignment.template"><code>AlignedSequence</code></a> at given index.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/Profile.html#getAlignedSequence(S)">getAlignedSequence(S)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template">Profile</a></dt>
<dd>
<div class="block">Searches for the given <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a> within this alignment profile.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/MultipleSequenceAlignment.html#getAlignedSequence(int)">getAlignedSequence(int)</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/MultipleSequenceAlignment.html" title="class in org.biojava3.core.sequence">MultipleSequenceAlignment</a></dt>
<dd>
<div class="block">Uses bioIndex starting at 1 instead of 0</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/LightweightProfile.html#getAlignedSequence(int)">getAlignedSequence(int)</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/LightweightProfile.html" title="interface in org.biojava3.core.sequence.template">LightweightProfile</a></dt>
<dd>
<div class="block">Returns <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a> at given index.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleProfile.html#getAlignedSequences()">getAlignedSequences()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleProfile.html" title="class in org.biojava3.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleProfile.html#getAlignedSequences(int...)">getAlignedSequences(int...)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleProfile.html" title="class in org.biojava3.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleProfile.html#getAlignedSequences(S...)">getAlignedSequences(S...)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleProfile.html" title="class in org.biojava3.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/Profile.html#getAlignedSequences()">getAlignedSequences()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns a <a href="http://java.sun.com/j2se/1.6.0/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util"><code>List</code></a> containing the individual <a href="./org/biojava3/alignment/template/AlignedSequence.html" title="interface in org.biojava3.alignment.template"><code>AlignedSequence</code></a>s of this alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/Profile.html#getAlignedSequences(int...)">getAlignedSequences(int...)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns a <a href="http://java.sun.com/j2se/1.6.0/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util"><code>List</code></a> containing some of the individual <a href="./org/biojava3/alignment/template/AlignedSequence.html" title="interface in org.biojava3.alignment.template"><code>AlignedSequence</code></a>s of this alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/Profile.html#getAlignedSequences(S...)">getAlignedSequences(S...)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns a <a href="http://java.sun.com/j2se/1.6.0/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util"><code>List</code></a> containing some of the individual <a href="./org/biojava3/alignment/template/AlignedSequence.html" title="interface in org.biojava3.alignment.template"><code>AlignedSequence</code></a>s of this alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/MultipleSequenceAlignment.html#getAlignedSequences()">getAlignedSequences()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/MultipleSequenceAlignment.html" title="class in org.biojava3.core.sequence">MultipleSequenceAlignment</a></dt>
<dd>
<div class="block">Get the list of sequences</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/LightweightProfile.html#getAlignedSequences()">getAlignedSequences()</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/LightweightProfile.html" title="interface in org.biojava3.core.sequence.template">LightweightProfile</a></dt>
<dd>
<div class="block">Returns a <a href="http://java.sun.com/j2se/1.6.0/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util"><code>List</code></a> containing the individual <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a>s of this alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/GuiWrapper.html#getAlignedStructure(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">getAlignedStructure(Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/GuiWrapper.html" title="class in org.biojava.bio.structure.align.ce">GuiWrapper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/DisplayAFP.html#getAlignedStructure(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">getAlignedStructure(Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/DisplayAFP.html" title="class in org.biojava.bio.structure.align.gui">DisplayAFP</a></dt>
<dd>
<div class="block">Get an artifical Structure containing both chains.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#getAlignedStructure(org.biojava.bio.structure.Structure, org.biojava.bio.structure.Structure)">getAlignedStructure(Structure, Structure)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>
<div class="block">create an artifical Structure object that contains the two
 structures superimposed onto each other.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructurePairAligner.html#getAlignmentAtoms(org.biojava.bio.structure.Structure)">getAlignmentAtoms(Structure)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructurePairAligner.html" title="class in org.biojava.bio.structure.align">StructurePairAligner</a></dt>
<dd>
<div class="block">Returns the atoms that are being used for the alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleAlignedSequence.html#getAlignmentIndexAt(int)">getAlignmentIndexAt(int)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleAlignedSequence.html" title="class in org.biojava3.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AlignedSequence.html#getAlignmentIndexAt(int)">getAlignmentIndexAt(int)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AlignedSequence.html" title="interface in org.biojava3.alignment.template">AlignedSequence</a></dt>
<dd>
<div class="block">Returns the column index within an alignment corresponding to the given index in the original <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastOutputProperties.html#getAlignmentNumber()">getAlignmentNumber()</a></span> - Method in class org.biojava3.ws.alignment.qblast.<a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastOutputProperties.html" title="class in org.biojava3.ws.alignment.qblast">NCBIQBlastOutputProperties</a></dt>
<dd>
<div class="block">A method that simply returns the number of alignments fetched with this RemoteQBlastOutputProperties object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastAlignmentProperties.html#getAlignmentOption(java.lang.String)">getAlignmentOption(String)</a></span> - Method in class org.biojava3.ws.alignment.qblast.<a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastAlignmentProperties.html" title="class in org.biojava3.ws.alignment.qblast">NCBIQBlastAlignmentProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/RemotePairwiseAlignmentProperties.html#getAlignmentOption(java.lang.String)">getAlignmentOption(String)</a></span> - Method in interface org.biojava3.ws.alignment.<a href="./org/biojava3/ws/alignment/RemotePairwiseAlignmentProperties.html" title="interface in org.biojava3.ws.alignment">RemotePairwiseAlignmentProperties</a></dt>
<dd>
<div class="block">Method that returns the value associated with the key given in parameter.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastAlignmentProperties.html#getAlignmentOptions()">getAlignmentOptions()</a></span> - Method in class org.biojava3.ws.alignment.qblast.<a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastAlignmentProperties.html" title="class in org.biojava3.ws.alignment.qblast">NCBIQBlastAlignmentProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/RemotePairwiseAlignmentProperties.html#getAlignmentOptions()">getAlignmentOptions()</a></span> - Method in interface org.biojava3.ws.alignment.<a href="./org/biojava3/ws/alignment/RemotePairwiseAlignmentProperties.html" title="interface in org.biojava3.ws.alignment">RemotePairwiseAlignmentProperties</a></dt>
<dd>
<div class="block">Method to get all keys to the information stored in this object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastOutputProperties.html#getAlignmentOutputFormat()">getAlignmentOutputFormat()</a></span> - Method in class org.biojava3.ws.alignment.qblast.<a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastOutputProperties.html" title="class in org.biojava3.ws.alignment.qblast">NCBIQBlastOutputProperties</a></dt>
<dd>
<div class="block">Method that returns the alignment output format for this actual RemoteQBlastOutputProperties object</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html#getAlignmentPairsFromServer()">getAlignmentPairsFromServer()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html" title="class in org.biojava.bio.structure.align.client">FarmJobRunnable</a></dt>
<dd>
<div class="block">talk to centralized server and fetch all alignments to run.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastService.html#getAlignmentResults(java.lang.String, org.biojava3.ws.alignment.RemotePairwiseAlignmentOutputProperties)">getAlignmentResults(String, RemotePairwiseAlignmentOutputProperties)</a></span> - Method in class org.biojava3.ws.alignment.qblast.<a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastService.html" title="class in org.biojava3.ws.alignment.qblast">NCBIQBlastService</a></dt>
<dd>
<div class="block">
 This method extracts the actual Blast report for this request ID.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/RemotePairwiseAlignmentService.html#getAlignmentResults(java.lang.String, org.biojava3.ws.alignment.RemotePairwiseAlignmentOutputProperties)">getAlignmentResults(String, RemotePairwiseAlignmentOutputProperties)</a></span> - Method in interface org.biojava3.ws.alignment.<a href="./org/biojava3/ws/alignment/RemotePairwiseAlignmentService.html" title="interface in org.biojava3.ws.alignment">RemotePairwiseAlignmentService</a></dt>
<dd>
<div class="block">Getting the actual alignment results from this instantiated service for a given ID with specific
 formatting parameters held in a RemotePairwiseAlignmentOutputProperties-implemented object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html#getAlignments()">getAlignments()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentResult</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructurePairAligner.html#getAlignments()">getAlignments()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructurePairAligner.html" title="class in org.biojava.bio.structure.align">StructurePairAligner</a></dt>
<dd>
<div class="block">return the alternative alignments that can be found for the two structures</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/SeqRes2AtomAligner.html#getAlignmentString()">getAlignmentString()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/SeqRes2AtomAligner.html" title="class in org.biojava.bio.structure.io">SeqRes2AtomAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/MenuCreator.html#getAlignmentTextMenu(javax.swing.JFrame, java.awt.event.ActionListener, org.biojava.bio.structure.align.model.AFPChain)">getAlignmentTextMenu(JFrame, ActionListener, AFPChain)</a></span> - Static method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/MenuCreator.html" title="class in org.biojava.bio.structure.align.gui">MenuCreator</a></dt>
<dd>
<div class="block">Create the menu for the "Text Only" representation of alignments</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#getAlignPairs()">getAlignPairs()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FCAlignHelper.html#getAlignPos(int[][])">getAlignPos(int[][])</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FCAlignHelper.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FCAlignHelper</a></dt>
<dd>
<div class="block">record the aligned pairs in alignList[][0], alignList[][1];
 return the number of aligned pairs</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getAlignScore()">getAlignScore()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getAlignScoreUpdate()">getAlignScoreUpdate()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AFPChainCoordManager.html#getAligSeq(java.awt.Point)">getAligSeq(Point)</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AFPChainCoordManager.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AFPChainCoordManager</a></dt>
<dd>
<div class="block">returns the AligSeq (0 or 1) for a point
 returns -1 if not over an alig seq.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/FlatFileInstallation.html#getAll()">getAll()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/FlatFileInstallation.html" title="class in org.biojava.bio.structure.server">FlatFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html#getAll()">getAll()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html" title="class in org.biojava.bio.structure.server">MMCIFFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/PDBInstallation.html#getAll()">getAll()</a></span> - Method in interface org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/PDBInstallation.html" title="interface in org.biojava.bio.structure.server">PDBInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
<div class="block">get all PDBHeaders that pass the added Filters, if no filters have been added returns all available PDBs</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructureAlignmentFactory.html#getAllAlgorithmNames()">getAllAlgorithmNames()</a></span> - Static method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructureAlignmentFactory.html" title="class in org.biojava.bio.structure.align">StructureAlignmentFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructureAlignmentFactory.html#getAllAlgorithms()">getAllAlgorithms()</a></span> - Static method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructureAlignmentFactory.html" title="class in org.biojava.bio.structure.align">StructureAlignmentFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/CaseFreeAminoAcidCompoundSet.html#getAllCompounds()">getAllCompounds()</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/CaseFreeAminoAcidCompoundSet.html" title="class in org.biojava3.aaproperties.xml">CaseFreeAminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/ModifiedAminoAcidCompoundSet.html#getAllCompounds()">getAllCompounds()</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/ModifiedAminoAcidCompoundSet.html" title="class in org.biojava3.aaproperties.xml">ModifiedAminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/AminoAcidCompoundSet.html#getAllCompounds()">getAllCompounds()</a></span> - Method in class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/AminoAcidCompoundSet.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompoundSet.html#getAllCompounds()">getAllCompounds()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava3.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/CompoundSet.html#getAllCompounds()">getAllCompounds()</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/Frame.html#getAllFrames()">getAllFrames()</a></span> - Static method in enum org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/Frame.html" title="enum in org.biojava3.core.sequence.transcription">Frame</a></dt>
<dd>
<div class="block">Delegates to <a href="./org/biojava3/core/sequence/transcription/Frame.html#values()"><code>Frame.values()</code></a></div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/PrepareMMcifIndexFile.html#getAllMMcif(java.io.File)">getAllMMcif(File)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/PrepareMMcifIndexFile.html" title="class in org.biojava.bio.structure.server">PrepareMMcifIndexFile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/GuideTree.Node.html#getAllowsChildren()">getAllowsChildren()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/GuideTree.Node.html" title="class in org.biojava3.alignment">GuideTree.Node</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/Alignments.html#getAllPairsAlignments(java.util.List, org.biojava3.alignment.Alignments.PairwiseSequenceAlignerType, org.biojava3.alignment.template.GapPenalty, org.biojava3.alignment.template.SubstitutionMatrix)">getAllPairsAlignments(List&lt;S&gt;, Alignments.PairwiseSequenceAlignerType, GapPenalty, SubstitutionMatrix&lt;C&gt;)</a></span> - Static method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/Alignments.html" title="class in org.biojava3.alignment">Alignments</a></dt>
<dd>
<div class="block">Factory method which computes a sequence alignment for all <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a> pairs in the given <a href="http://java.sun.com/j2se/1.6.0/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util"><code>List</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/Alignments.html#getAllPairsScorers(java.util.List, org.biojava3.alignment.Alignments.PairwiseSequenceScorerType, org.biojava3.alignment.template.GapPenalty, org.biojava3.alignment.template.SubstitutionMatrix)">getAllPairsScorers(List&lt;S&gt;, Alignments.PairwiseSequenceScorerType, GapPenalty, SubstitutionMatrix&lt;C&gt;)</a></span> - Static method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/Alignments.html" title="class in org.biojava3.alignment">Alignments</a></dt>
<dd>
<div class="block">Factory method which sets up a sequence pair scorer for all <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a> pairs in the given <a href="http://java.sun.com/j2se/1.6.0/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util"><code>List</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/Alignments.html#getAllPairsScores(java.util.List, org.biojava3.alignment.Alignments.PairwiseSequenceScorerType, org.biojava3.alignment.template.GapPenalty, org.biojava3.alignment.template.SubstitutionMatrix)">getAllPairsScores(List&lt;S&gt;, Alignments.PairwiseSequenceScorerType, GapPenalty, SubstitutionMatrix&lt;C&gt;)</a></span> - Static method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/Alignments.html" title="class in org.biojava3.alignment">Alignments</a></dt>
<dd>
<div class="block">Factory method which computes a sequence pair score for all <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a> pairs in the given <a href="http://java.sun.com/j2se/1.6.0/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util"><code>List</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/GuideTree.html#getAllPairsScores()">getAllPairsScores()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/GuideTree.html" title="class in org.biojava3.alignment">GuideTree</a></dt>
<dd>
<div class="block">Returns a sequence pair score for all <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a> pairs in the given <a href="http://java.sun.com/j2se/1.6.0/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util"><code>List</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/PrepareIndexFile.html#getAllPDB(java.io.File)">getAllPDB(File)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/PrepareIndexFile.html" title="class in org.biojava.bio.structure.server">PrepareIndexFile</a></dt>
<dd>
<div class="block">get all PDBfiles from a directory</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/SandboxStyleStructureProvider.html#getAllPDBIDs()">getAllPDBIDs()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/SandboxStyleStructureProvider.html" title="class in org.biojava.bio.structure.io">SandboxStyleStructureProvider</a></dt>
<dd>
<div class="block">Returns a list of all PDB IDs that are available in this installation</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getAlnbeg1()">getAlnbeg1()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getAlnbeg2()">getAlnbeg2()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getAlnLength()">getAlnLength()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getAlnseq1()">getAlnseq1()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getAlnseq2()">getAlnseq2()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getAlnsymb()">getAlnsymb()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#getAltAligNumber()">getAltAligNumber()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>
<div class="block">returns the sequential number of this alternative alignment</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/JMatrixPanel.html#getAlternativeAligs()">getAlternativeAligs()</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/JMatrixPanel.html" title="class in org.biojava.bio.structure.gui">JMatrixPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html#getAlternativeAligs()">getAlternativeAligs()</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html" title="class in org.biojava.bio.structure.gui">ScaleableMatrixPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#getAltLoc()">getAltLoc()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">get set alternate Location.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#getAltLoc()">getAltLoc()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>
<div class="block">get alternate Location.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#getAltLocs()">getAltLocs()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">Get the list of alternate locations.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#getAltLocs()">getAltLocs()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractNucleotideCompoundSet.html#getAmbiguity(org.biojava3.core.sequence.compound.NucleotideCompound...)">getAmbiguity(NucleotideCompound...)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractNucleotideCompoundSet.html" title="class in org.biojava3.core.sequence.template">AbstractNucleotideCompoundSet</a></dt>
<dd>
<div class="block">Calculates the best symbol for a collection of compounds.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StandardAminoAcid.html#getAminoAcid(java.lang.String)">getAminoAcid(String)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StandardAminoAcid.html" title="class in org.biojava.bio.structure">StandardAminoAcid</a></dt>
<dd>
<div class="block">get a standard amino acid.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/AminoAcidCompositionTable.html#getAminoacid()">getAminoacid()</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/AminoAcidCompositionTable.html" title="class in org.biojava3.aaproperties.xml">AminoAcidCompositionTable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/Table.Codon.html#getAminoAcid()">getAminoAcid()</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/Table.Codon.html" title="class in org.biojava3.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.html#getAminoAcidCompounds()">getAminoAcidCompounds()</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.html" title="class in org.biojava3.core.sequence.transcription">TranscriptionEngine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/AminoAcidCompositionTable.html#getAminoAcidCompoundSet()">getAminoAcidCompoundSet()</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/AminoAcidCompositionTable.html" title="class in org.biojava3.aaproperties.xml">AminoAcidCompositionTable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/CaseFreeAminoAcidCompoundSet.html#getAminoAcidCompoundSet()">getAminoAcidCompoundSet()</a></span> - Static method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/CaseFreeAminoAcidCompoundSet.html" title="class in org.biojava3.aaproperties.xml">CaseFreeAminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/AminoAcidCompoundSet.html#getAminoAcidCompoundSet()">getAminoAcidCompoundSet()</a></span> - Static method in class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/AminoAcidCompoundSet.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/StructureUtil.html#getAminoAcids(org.biojava.bio.structure.Chain)">getAminoAcids(Chain)</a></span> - Static method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/StructureUtil.html" title="class in org.biojava3.protmod.structure">StructureUtil</a></dt>
<dd>
<div class="block">Get all amino acids in a chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/chem/ChemCompTools.html#getAminoOneLetter(java.lang.String)">getAminoOneLetter(String)</a></span> - Static method in class org.biojava.bio.structure.io.mmcif.chem.<a href="./org/biojava/bio/structure/io/mmcif/chem/ChemCompTools.html" title="class in org.biojava.bio.structure.io.mmcif.chem">ChemCompTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/chem/ChemCompTools.html#getAminoThreeLetter(java.lang.Character)">getAminoThreeLetter(Character)</a></span> - Static method in class org.biojava.bio.structure.io.mmcif.chem.<a href="./org/biojava/bio/structure/io/mmcif/chem/ChemCompTools.html" title="class in org.biojava.bio.structure.io.mmcif.chem">ChemCompTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcid.html#getAminoType()">getAminoType()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcid.html" title="interface in org.biojava.bio.structure">AminoAcid</a></dt>
<dd>
<div class="block">returns the name of the AA, in single letter code.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcidImpl.html#getAminoType()">getAminoType()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcidImpl.html" title="class in org.biojava.bio.structure">AminoAcidImpl</a></dt>
<dd>
<div class="block">returns the name of the AA, in single letter code.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/routines/AnchoredPairwiseSequenceAligner.html#getAnchors()">getAnchors()</a></span> - Method in class org.biojava3.alignment.routines.<a href="./org/biojava3/alignment/routines/AnchoredPairwiseSequenceAligner.html" title="class in org.biojava3.alignment.routines">AnchoredPairwiseSequenceAligner</a></dt>
<dd>
<div class="block">Returns the list of anchors.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#getAngleDiff()">getAngleDiff()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getAniso_B()">getAniso_B()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#getAnnotationType()">getAnnotationType()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/IPeptideProperties.html#getApliphaticIndex(org.biojava3.core.sequence.ProteinSequence)">getApliphaticIndex(ProteinSequence)</a></span> - Method in interface org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/IPeptideProperties.html" title="interface in org.biojava3.aaproperties">IPeptideProperties</a></dt>
<dd>
<div class="block">Returns the apliphatic index of sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/PeptideProperties.html#getApliphaticIndex(java.lang.String)">getApliphaticIndex(String)</a></span> - Static method in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/PeptideProperties.html" title="class in org.biojava3.aaproperties">PeptideProperties</a></dt>
<dd>
<div class="block">An adaptor method to return the apliphatic index of sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/PeptidePropertiesImpl.html#getApliphaticIndex(org.biojava3.core.sequence.ProteinSequence)">getApliphaticIndex(ProteinSequence)</a></span> - Method in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/PeptidePropertiesImpl.html" title="class in org.biojava3.aaproperties">PeptidePropertiesImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#getArray()">getArray()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Access the internal two-dimensional array.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#getArrayCopy()">getArrayCopy()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Copy the internal two-dimensional array.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleAlignedSequence.html#getAsList()">getAsList()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleAlignedSequence.html" title="class in org.biojava3.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html#getAsList()">getAsList()</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava3.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/StringProxySequenceReader.html#getAsList()">getAsList()</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html#getAsList()">getAsList()</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/ArrayListSequenceReader.html#getAsList()">getAsList()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava3.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/BitSequenceReader.html#getAsList()">getAsList()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava3.core.sequence.storage">BitSequenceReader</a></dt>
<dd>
<div class="block">Returns this Sequence store as a List</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/JoiningSequenceReader.html#getAsList()">getAsList()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava3.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html#getAsList()">getAsList()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava3.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Delegates to <a href="./org/biojava3/core/sequence/template/SequenceMixin.html#toList(org.biojava3.core.sequence.template.Sequence)"><code>SequenceMixin.toList(org.biojava3.core.sequence.template.Sequence)</code></a></div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#getAsList()">getAsList()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/Sequence.html#getAsList()">getAsList()</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a></dt>
<dd>
<div class="block">Returns the Sequence as a List of compounds</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceProxyView.html#getAsList()">getAsList()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceProxyView.html" title="class in org.biojava3.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#getAsym_id()">getAsym_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#getAsym_id()">getAsym_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getAtcc()">getAtcc()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#getAtom(java.lang.String)">getAtom(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">Get an atom.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#getAtom(int)">getAtom(int)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">Get at atom by position.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#getAtom(java.lang.String)">getAtom(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">get an atom throws StructureException if atom not found.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#getAtom(int)">getAtom(int)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">return an atom by its position in the internal List.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/StructureAtomLinkage.html#getAtom1()">getAtom1()</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/StructureAtomLinkage.html" title="class in org.biojava3.protmod.structure">StructureAtomLinkage</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/StructureAtomLinkage.html#getAtom2()">getAtom2()</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/StructureAtomLinkage.html" title="class in org.biojava3.protmod.structure">StructureAtomLinkage</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Bond.html#getAtomA()">getAtomA()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Bond.html" title="class in org.biojava.bio.structure">Bond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/GuiWrapper.html#getAtomArray(org.biojava.bio.structure.Atom[], java.util.List, java.util.List)">getAtomArray(Atom[], List&lt;Group&gt;, List&lt;Group&gt;)</a></span> - Static method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/GuiWrapper.html" title="class in org.biojava.bio.structure.align.ce">GuiWrapper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/DisplayAFP.html#getAtomArray(org.biojava.bio.structure.Atom[], java.util.List, java.util.List)">getAtomArray(Atom[], List&lt;Group&gt;, List&lt;Group&gt;)</a></span> - Static method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/DisplayAFP.html" title="class in org.biojava.bio.structure.align.gui">DisplayAFP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#getAtomArray(org.biojava.bio.structure.Structure, java.lang.String[])">getAtomArray(Structure, String[])</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>
<div class="block">Returns an array of the requested Atoms from the Structure object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#getAtomArray(org.biojava.bio.structure.Chain, java.lang.String[])">getAtomArray(Chain, String[])</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>
<div class="block">Returns an array of the requested Atoms from the Structure object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#getAtomArrayAllModels(org.biojava.bio.structure.Structure, java.lang.String[])">getAtomArrayAllModels(Structure, String[])</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>
<div class="block">Returns an array of the requested Atoms from the Structure object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Bond.html#getAtomB()">getAtomB()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Bond.html" title="class in org.biojava.bio.structure">Bond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#getAtomByPDBname(java.lang.String)">getAtomByPDBname(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">Get an atom by the full PDB name e.g. " N  " for N.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#getAtomByPDBname(java.lang.String)">getAtomByPDBname(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">Get an atom by the full PDB name e.g. " N  " for N.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#getAtomCAArray(org.biojava.bio.structure.Chain)">getAtomCAArray(Chain)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>
<div class="block">Returns an Atom array of the CA atoms.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#getAtomCAArray(org.biojava.bio.structure.Structure)">getAtomCAArray(Structure)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>
<div class="block">Returns an Atom array of the CA atoms.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/StructureUtil.html#getAtomDistance(org.biojava.bio.structure.Atom, org.biojava.bio.structure.Atom)">getAtomDistance(Atom, Atom)</a></span> - Static method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/StructureUtil.html" title="class in org.biojava3.protmod.structure">StructureUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/DisplayAFP.html#getAtomForAligPos(org.biojava.bio.structure.align.model.AFPChain, int, int, org.biojava.bio.structure.Atom[], boolean)">getAtomForAligPos(AFPChain, int, int, Atom[], boolean)</a></span> - Static method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/DisplayAFP.html" title="class in org.biojava.bio.structure.align.gui">DisplayAFP</a></dt>
<dd>
<div class="block">return the atom at alignment position aligPos. at the present only works with block 0</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getAtomGroup(int)">getAtomGroup(int)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">return the Group at position X.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getAtomGroup(int)">getAtomGroup(int)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">return the group at position .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getAtomGroups()">getAtomGroups()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Return all groups that have been specified in the ATOM section of this chain .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getAtomGroups(java.lang.String)">getAtomGroups(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Return a List of all groups of a special type (e.g. amino,
 hetatm, nucleotide).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getAtomGroups(java.lang.String)">getAtomGroups(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Get the Groups of a certain type, that are listed int the ATOM records of the PDB file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getAtomGroups()">getAtomGroups()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Return all groups that have been specified in the ATOM section of this chain .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#getAtomicMass()">getAtomicMass()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns the atomic mass for this Element.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#getAtomicNumber()">getAtomicNumber()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns the atomic number of this Element.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getAtomLength()">getAtomLength()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Return the number of Groups in the ATOM records of the chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getAtomLength()">getAtomLength()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Return the number of Groups in the ATOM records of the chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getAtomLigands()">getAtomLigands()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Get all groups that are not polymer groups and that are not solvent groups.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getAtomLigands()">getAtomLigands()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Get all groups that are not polymer groups and that are not solvent groups.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ModifiedCompound.html#getAtomLinkages()">getAtomLinkages()</a></span> - Method in interface org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ModifiedCompound.html" title="interface in org.biojava3.protmod.structure">ModifiedCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ModifiedCompoundImpl.html#getAtomLinkages()">getAtomLinkages()</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ModifiedCompoundImpl.html" title="class in org.biojava3.protmod.structure">ModifiedCompoundImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/AtomInfo.html#getAtomName()">getAtomName()</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/AtomInfo.html" title="class in org.biojava.bio.structure.align.gui.jmol">AtomInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/StructureAtom.html#getAtomName()">getAtomName()</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/StructureAtom.html" title="class in org.biojava3.protmod.structure">StructureAtom</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/StructureUtil.html#getAtomNames(org.biojava.bio.structure.Group)">getAtomNames(Group)</a></span> - Static method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/StructureUtil.html" title="class in org.biojava3.protmod.structure">StructureUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#getAtoms(java.lang.String)">getAtoms(String)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>
<div class="block">Returns the CA atoms for the provided name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#getAtoms(java.lang.String, boolean)">getAtoms(String, boolean)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>does the same as <a href="./org/biojava/bio/structure/align/util/AtomCache.html#getAtoms(java.lang.String)"><code>AtomCache.getAtoms(String)</code></a> ;</i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#getAtoms()">getAtoms()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">Get list of atoms.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#getAtoms()">getAtoms()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">get all atoms of this group .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getAtomSequence()">getAtomSequence()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Return the sequence of amino acids as it has been provided in the ATOM records.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getAtomSequence()">getAtomSequence()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Return the sequence of amino acids as it has been provided in the ATOM records.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/convertor/Convert2Charge.html#getAttribute()">getAttribute()</a></span> - Method in class org.biojava3.aaproperties.profeat.convertor.<a href="./org/biojava3/aaproperties/profeat/convertor/Convert2Charge.html" title="class in org.biojava3.aaproperties.profeat.convertor">Convert2Charge</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/convertor/Convert2Hydrophobicity.html#getAttribute()">getAttribute()</a></span> - Method in class org.biojava3.aaproperties.profeat.convertor.<a href="./org/biojava3/aaproperties/profeat/convertor/Convert2Hydrophobicity.html" title="class in org.biojava3.aaproperties.profeat.convertor">Convert2Hydrophobicity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/convertor/Convert2NormalizedVanDerWaalsVolume.html#getAttribute()">getAttribute()</a></span> - Method in class org.biojava3.aaproperties.profeat.convertor.<a href="./org/biojava3/aaproperties/profeat/convertor/Convert2NormalizedVanDerWaalsVolume.html" title="class in org.biojava3.aaproperties.profeat.convertor">Convert2NormalizedVanDerWaalsVolume</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/convertor/Convert2Polarity.html#getAttribute()">getAttribute()</a></span> - Method in class org.biojava3.aaproperties.profeat.convertor.<a href="./org/biojava3/aaproperties/profeat/convertor/Convert2Polarity.html" title="class in org.biojava3.aaproperties.profeat.convertor">Convert2Polarity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/convertor/Convert2Polarizability.html#getAttribute()">getAttribute()</a></span> - Method in class org.biojava3.aaproperties.profeat.convertor.<a href="./org/biojava3/aaproperties/profeat/convertor/Convert2Polarizability.html" title="class in org.biojava3.aaproperties.profeat.convertor">Convert2Polarizability</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/convertor/Convert2SecondaryStructure.html#getAttribute()">getAttribute()</a></span> - Method in class org.biojava3.aaproperties.profeat.convertor.<a href="./org/biojava3/aaproperties/profeat/convertor/Convert2SecondaryStructure.html" title="class in org.biojava3.aaproperties.profeat.convertor">Convert2SecondaryStructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/convertor/Convert2SolventAccessibility.html#getAttribute()">getAttribute()</a></span> - Method in class org.biojava3.aaproperties.profeat.convertor.<a href="./org/biojava3/aaproperties/profeat/convertor/Convert2SolventAccessibility.html" title="class in org.biojava3.aaproperties.profeat.convertor">Convert2SolventAccessibility</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/convertor/Convertor.html#getAttribute()">getAttribute()</a></span> - Method in class org.biojava3.aaproperties.profeat.convertor.<a href="./org/biojava3/aaproperties/profeat/convertor/Convertor.html" title="class in org.biojava3.aaproperties.profeat.convertor">Convertor</a></dt>
<dd>
<div class="block">Return the attribute of the grouping</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Feature.html#getAttribute(java.lang.String)">getAttribute(String)</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Feature.html" title="class in org.biojava3.genome.parsers.gff">Feature</a></dt>
<dd>
<div class="block">Get value of specified attribute key.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/FeatureI.html#getAttribute(java.lang.String)">getAttribute(String)</a></span> - Method in interface org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/FeatureI.html" title="interface in org.biojava3.genome.parsers.gff">FeatureI</a></dt>
<dd>
<div class="block">Get the attribute value for this key.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Feature.html#getAttributeOld(java.lang.String)">getAttributeOld(String)</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Feature.html" title="class in org.biojava3.genome.parsers.gff">Feature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getAuth_asym_id()">getAuth_asym_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getAuth_atom_id()">getAuth_atom_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getAuth_comp_id()">getAuth_comp_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#getAuth_mon_id()">getAuth_mon_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#getAuth_mon_id()">getAuth_mon_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getAuth_seq_id()">getAuth_seq_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#getAuth_seq_num()">getAuth_seq_num()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#getAuth_seq_num()">getAuth_seq_num()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#getAuthorList()">getAuthorList()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>
<div class="block">Get the list of Authors of the JournalArticle</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#getAuthors()">getAuthors()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>
<div class="block">Returns the names of the authors as listed in the AUTHORS section of a PDB file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/UserConfiguration.html#getAutoFetch()">getAutoFetch()</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/UserConfiguration.html" title="class in org.biojava.bio.structure.align.util">UserConfiguration</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/IPeptideProperties.html#getAvgHydropathy(org.biojava3.core.sequence.ProteinSequence)">getAvgHydropathy(ProteinSequence)</a></span> - Method in interface org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/IPeptideProperties.html" title="interface in org.biojava3.aaproperties">IPeptideProperties</a></dt>
<dd>
<div class="block">Returns the average hydropathy value of sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/PeptideProperties.html#getAvgHydropathy(java.lang.String)">getAvgHydropathy(String)</a></span> - Static method in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/PeptideProperties.html" title="class in org.biojava3.aaproperties">PeptideProperties</a></dt>
<dd>
<div class="block">An adaptor method to return the average hydropathy value of sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/PeptidePropertiesImpl.html#getAvgHydropathy(org.biojava3.core.sequence.ProteinSequence)">getAvgHydropathy(ProteinSequence)</a></span> - Method in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/PeptidePropertiesImpl.html" title="class in org.biojava3.aaproperties">PeptidePropertiesImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getB_iso_max()">getB_iso_max()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getB_iso_mean()">getB_iso_mean()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getB_iso_min()">getB_iso_min()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getB_iso_or_equiv()">getB_iso_or_equiv()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getB_iso_or_equiv_esd()">getB_iso_or_equiv_esd()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#getBackboneAtomArray(org.biojava.bio.structure.Structure)">getBackboneAtomArray(Structure)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>
<div class="block">Returns an Atom array of the MainChain atoms.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/views/WindowedSequence.html#getBackingSequence()">getBackingSequence()</a></span> - Method in class org.biojava3.core.sequence.views.<a href="./org/biojava3/core/sequence/views/WindowedSequence.html" title="class in org.biojava3.core.sequence.views">WindowedSequence</a></dt>
<dd>
<div class="block">Access the sequence which backs this window</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getBadRmsd()">getBadRmsd()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompound.html#getBase()">getBase()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompound.html" title="class in org.biojava3.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Location.html#getBegin()">getBegin()</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Location.html" title="class in org.biojava3.genome.parsers.gff">Location</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#getBioBegin()">getBioBegin()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#getBioEnd()">getBioEnd()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceProxyView.html#getBioEnd()">getBioEnd()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceProxyView.html" title="class in org.biojava3.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceView.html#getBioEnd()">getBioEnd()</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceView.html" title="interface in org.biojava3.core.sequence.template">SequenceView</a></dt>
<dd>
<div class="block">1-indexed, inclusive.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html#getBiol_id()">getBiol_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRef</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#getBiologicalAssembly(java.lang.String, int, boolean)">getBiologicalAssembly(String, int, boolean)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>
<div class="block">Loads the biological assembly for a given PDB ID and bioAssemblyId.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#getBiologicalUnit(java.lang.String)">getBiologicalUnit(String)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>
<div class="block">Loads the default biological unit (*.pdb1.gz) file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getBiologicalUnit()">getBiologicalUnit()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/LocalCacheStructureProvider.html#getBiologicalUnit(java.lang.String)">getBiologicalUnit(String)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/LocalCacheStructureProvider.html" title="class in org.biojava.bio.structure.io">LocalCacheStructureProvider</a></dt>
<dd>
<div class="block">Returns the first available biological unit for the PDB ID.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/SandboxStyleStructureProvider.html#getBiologicalUnit(java.lang.String)">getBiologicalUnit(String)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/SandboxStyleStructureProvider.html" title="class in org.biojava.bio.structure.io">SandboxStyleStructureProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/StructureProvider.html#getBiologicalUnit(java.lang.String)">getBiologicalUnit(String)</a></span> - Method in interface org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/StructureProvider.html" title="interface in org.biojava.bio.structure.io">StructureProvider</a></dt>
<dd>
<div class="block">get the biological unit for a file</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceProxyView.html#getBioStart()">getBioStart()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceProxyView.html" title="class in org.biojava3.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceView.html#getBioStart()">getBioStart()</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceView.html" title="interface in org.biojava3.core.sequence.template">SequenceView</a></dt>
<dd>
<div class="block">1-indexed, inclusive.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getBJSequence()">getBJSequence()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Convert the SEQRES groups of a Chain to a Biojava Sequence object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getBJSequence()">getBJSequence()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Convert the SEQRES groups of a Chain to a Biojava Sequence object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastAlignmentProperties.html#getBlastAdvancedOptions()">getBlastAdvancedOptions()</a></span> - Method in class org.biojava3.ws.alignment.qblast.<a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastAlignmentProperties.html" title="class in org.biojava3.ws.alignment.qblast">NCBIQBlastAlignmentProperties</a></dt>
<dd>
<div class="block">Simply return the string given as argument via setBlastAdvancedOptions</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastAlignmentProperties.html#getBlastDatabase()">getBlastDatabase()</a></span> - Method in class org.biojava3.ws.alignment.qblast.<a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastAlignmentProperties.html" title="class in org.biojava3.ws.alignment.qblast">NCBIQBlastAlignmentProperties</a></dt>
<dd>
<div class="block">This method returns the value of the database used for this particular
 blast run.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastAlignmentProperties.html#getBlastExpect()">getBlastExpect()</a></span> - Method in class org.biojava3.ws.alignment.qblast.<a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastAlignmentProperties.html" title="class in org.biojava3.ws.alignment.qblast">NCBIQBlastAlignmentProperties</a></dt>
<dd>
<div class="block">This method returns the value of EXPECT parameter used for this particular
 blast run.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastAlignmentProperties.html#getBlastFromPosition()">getBlastFromPosition()</a></span> - Method in class org.biojava3.ws.alignment.qblast.<a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastAlignmentProperties.html" title="class in org.biojava3.ws.alignment.qblast">NCBIQBlastAlignmentProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastAlignmentProperties.html#getBlastProgram()">getBlastProgram()</a></span> - Method in class org.biojava3.ws.alignment.qblast.<a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastAlignmentProperties.html" title="class in org.biojava3.ws.alignment.qblast">NCBIQBlastAlignmentProperties</a></dt>
<dd>
<div class="block">This method returns the value of the program used for this particular
 blast run.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastAlignmentProperties.html#getBlastToPosition()">getBlastToPosition()</a></span> - Method in class org.biojava3.ws.alignment.qblast.<a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastAlignmentProperties.html" title="class in org.biojava3.ws.alignment.qblast">NCBIQBlastAlignmentProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastAlignmentProperties.html#getBlastWordSize()">getBlastWordSize()</a></span> - Method in class org.biojava3.ws.alignment.qblast.<a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastAlignmentProperties.html" title="class in org.biojava3.ws.alignment.qblast">NCBIQBlastAlignmentProperties</a></dt>
<dd>
<div class="block">This method returns the value of the WORD_SIZE parameter used for this particular
 blast run.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getBlock2Afp()">getBlock2Afp()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getBlockGap()">getBlockGap()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/DisplayAFP.html#getBlockNrForAlignPos(org.biojava.bio.structure.align.model.AFPChain, int)">getBlockNrForAlignPos(AFPChain, int)</a></span> - Static method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/DisplayAFP.html" title="class in org.biojava.bio.structure.align.gui">DisplayAFP</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>use AFPAlignmentDisplay.getBlockNrForAlignPos instead...</i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html#getBlockNrForAlignPos(org.biojava.bio.structure.align.model.AFPChain, int)">getBlockNrForAlignPos(AFPChain, int)</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html" title="class in org.biojava.bio.structure.align.util">AFPAlignmentDisplay</a></dt>
<dd>
<div class="block">get the block number for an aligned position</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getBlockNum()">getBlockNum()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">The number of blocks in the alignment</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getBlockNumClu()">getBlockNumClu()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getBlockNumIni()">getBlockNumIni()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getBlockNumSpt()">getBlockNumSpt()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getBlockResList()">getBlockResList()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">tracks the residues of the initial blocks (before optimization)</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getBlockResSize()">getBlockResSize()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getBlockRmsd()">getBlockRmsd()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getBlockRotationMatrix()">getBlockRotationMatrix()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getBlockScore()">getBlockScore()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getBlockShiftVector()">getBlockShiftVector()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getBlockSize()">getBlockSize()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SubstitutionMatrixHelper.html#getBlosum100()">getBlosum100()</a></span> - Static method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SubstitutionMatrixHelper.html" title="class in org.biojava3.alignment">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 100 matrix by Henikoff & Henikoff</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SubstitutionMatrixHelper.html#getBlosum30()">getBlosum30()</a></span> - Static method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SubstitutionMatrixHelper.html" title="class in org.biojava3.alignment">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 30 matrix by Henikoff & Henikoff</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SubstitutionMatrixHelper.html#getBlosum35()">getBlosum35()</a></span> - Static method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SubstitutionMatrixHelper.html" title="class in org.biojava3.alignment">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 35 matrix by Henikoff & Henikoff</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SubstitutionMatrixHelper.html#getBlosum40()">getBlosum40()</a></span> - Static method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SubstitutionMatrixHelper.html" title="class in org.biojava3.alignment">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 40 matrix by Henikoff & Henikoff</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SubstitutionMatrixHelper.html#getBlosum45()">getBlosum45()</a></span> - Static method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SubstitutionMatrixHelper.html" title="class in org.biojava3.alignment">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 45 matrix by Henikoff & Henikoff</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SubstitutionMatrixHelper.html#getBlosum50()">getBlosum50()</a></span> - Static method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SubstitutionMatrixHelper.html" title="class in org.biojava3.alignment">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 50 matrix by Henikoff & Henikoff</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SubstitutionMatrixHelper.html#getBlosum55()">getBlosum55()</a></span> - Static method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SubstitutionMatrixHelper.html" title="class in org.biojava3.alignment">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 55 matrix by Henikoff & Henikoff</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SubstitutionMatrixHelper.html#getBlosum60()">getBlosum60()</a></span> - Static method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SubstitutionMatrixHelper.html" title="class in org.biojava3.alignment">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 60 matrix by Henikoff & Henikoff</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SubstitutionMatrixHelper.html#getBlosum62()">getBlosum62()</a></span> - Static method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SubstitutionMatrixHelper.html" title="class in org.biojava3.alignment">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 62 matrix by Henikoff & Henikoff</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#getBLOSUM62()">getBLOSUM62()</a></span> - Static method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#getBLOSUM62(char, char)">getBLOSUM62(char, char)</a></span> - Static method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SubstitutionMatrixHelper.html#getBlosum65()">getBlosum65()</a></span> - Static method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SubstitutionMatrixHelper.html" title="class in org.biojava3.alignment">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 65 matrix by Henikoff & Henikoff</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SubstitutionMatrixHelper.html#getBlosum70()">getBlosum70()</a></span> - Static method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SubstitutionMatrixHelper.html" title="class in org.biojava3.alignment">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 70 matrix by Henikoff & Henikoff</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SubstitutionMatrixHelper.html#getBlosum75()">getBlosum75()</a></span> - Static method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SubstitutionMatrixHelper.html" title="class in org.biojava3.alignment">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 75 matrix by Henikoff & Henikoff</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SubstitutionMatrixHelper.html#getBlosum80()">getBlosum80()</a></span> - Static method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SubstitutionMatrixHelper.html" title="class in org.biojava3.alignment">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 80 matrix by Henikoff & Henikoff</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SubstitutionMatrixHelper.html#getBlosum85()">getBlosum85()</a></span> - Static method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SubstitutionMatrixHelper.html" title="class in org.biojava3.alignment">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 85 matrix by Henikoff & Henikoff</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SubstitutionMatrixHelper.html#getBlosum90()">getBlosum90()</a></span> - Static method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SubstitutionMatrixHelper.html" title="class in org.biojava3.alignment">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Blosum 90 matrix by Henikoff & Henikoff</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/util/ClasspathResource.html#getBufferedReader()">getBufferedReader()</a></span> - Method in class org.biojava3.core.sequence.io.util.<a href="./org/biojava3/core/sequence/io/util/ClasspathResource.html" title="class in org.biojava3.core.sequence.io.util">ClasspathResource</a></dt>
<dd>
<div class="block">Returns the reader representation of this classpath resource</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html#getByCategory(org.biojava.bio.structure.scop.ScopCategory)">getByCategory(ScopCategory)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">RemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDatabase.html#getByCategory(org.biojava.bio.structure.scop.ScopCategory)">getByCategory(ScopCategory)</a></span> - Method in interface org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDatabase.html" title="interface in org.biojava.bio.structure.scop">ScopDatabase</a></dt>
<dd>
<div class="block">Get all records of a particular classification.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#getByCategory(org.biojava.bio.structure.scop.ScopCategory)">getByCategory(ScopCategory)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationRegistry.html#getByCategory(org.biojava3.protmod.ModificationCategory)">getByCategory(ModificationCategory)</a></span> - Static method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationRegistry.html" title="class in org.biojava3.protmod">ProteinModificationRegistry</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationRegistry.html#getByComponent(org.biojava3.protmod.Component, org.biojava3.protmod.Component...)">getByComponent(Component, Component...)</a></span> - Static method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationRegistry.html" title="class in org.biojava3.protmod">ProteinModificationRegistry</a></dt>
<dd>
<div class="block">Get ProteinModifications that involves one or more components.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationRegistry.html#getById(java.lang.String)">getById(String)</a></span> - Static method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationRegistry.html" title="class in org.biojava3.protmod">ProteinModificationRegistry</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationRegistry.html#getByKeyword(java.lang.String)">getByKeyword(String)</a></span> - Static method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationRegistry.html" title="class in org.biojava3.protmod">ProteinModificationRegistry</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ModificationCategory.html#getByLabel(java.lang.String)">getByLabel(String)</a></span> - Static method in enum org.biojava3.protmod.<a href="./org/biojava3/protmod/ModificationCategory.html" title="enum in org.biojava3.protmod">ModificationCategory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ModificationOccurrenceType.html#getByLabel(java.lang.String)">getByLabel(String)</a></span> - Static method in enum org.biojava3.protmod.<a href="./org/biojava3/protmod/ModificationOccurrenceType.html" title="enum in org.biojava3.protmod">ModificationOccurrenceType</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationRegistry.html#getByOccurrenceType(org.biojava3.protmod.ModificationOccurrenceType)">getByOccurrenceType(ModificationOccurrenceType)</a></span> - Static method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationRegistry.html" title="class in org.biojava3.protmod">ProteinModificationRegistry</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationRegistry.html#getByPdbccId(java.lang.String)">getByPdbccId(String)</a></span> - Static method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationRegistry.html" title="class in org.biojava3.protmod">ProteinModificationRegistry</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationRegistry.html#getByPsimodId(java.lang.String)">getByPsimodId(String)</a></span> - Static method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationRegistry.html" title="class in org.biojava3.protmod">ProteinModificationRegistry</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationRegistry.html#getByResidId(java.lang.String)">getByResidId(String)</a></span> - Static method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationRegistry.html" title="class in org.biojava3.protmod">ProteinModificationRegistry</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/BufferedReaderBytesRead.html#getBytesRead()">getBytesRead()</a></span> - Method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/BufferedReaderBytesRead.html" title="class in org.biojava3.core.sequence.io">BufferedReaderBytesRead</a></dt>
<dd>
<div class="block">Keep track of bytesread via ReadLine to account for CR-LF in the stream.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcid.html#getC()">getC()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcid.html" title="interface in org.biojava.bio.structure">AminoAcid</a></dt>
<dd>
<div class="block">get C atom.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcidImpl.html#getC()">getC()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcidImpl.html" title="class in org.biojava.bio.structure">AminoAcidImpl</a></dt>
<dd>
<div class="block">get C atom.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html#getC()">getC()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html" title="class in org.biojava.bio.structure.secstruc">SecStrucGroup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcid.html#getCA()">getCA()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcid.html" title="interface in org.biojava.bio.structure">AminoAcid</a></dt>
<dd>
<div class="block">get CA atom.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcidImpl.html#getCA()">getCA()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcidImpl.html" title="class in org.biojava.bio.structure">AminoAcidImpl</a></dt>
<dd>
<div class="block">get CA atom.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html#getCA()">getCA()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html" title="class in org.biojava.bio.structure.secstruc">SecStrucGroup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html#getCa1()">getCa1()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html" title="class in org.biojava.bio.structure.align">CallableStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html#getCa1()">getCa1()</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AligPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/StatusDisplay.html#getCa1()">getCa1()</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/StatusDisplay.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">StatusDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getCa1Length()">getCa1Length()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html#getCa2()">getCa2()</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AligPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/StatusDisplay.html#getCa2()">getCa2()</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/StatusDisplay.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">StatusDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getCa2Length()">getCa2Length()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html#getCache()">getCache()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html" title="class in org.biojava.bio.structure.align">CallableStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/CacheFactory.html#getCache()">getCache()</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/CacheFactory.html" title="class in org.biojava.bio.structure.align.util">CacheFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#getCacheLocation()">getCacheLocation()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/SimpleStructureServer.html#getCacheSize()">getCacheSize()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/SimpleStructureServer.html" title="class in org.biojava.bio.structure.server">SimpleStructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureServer.html#getCacheSize()">getCacheSize()</a></span> - Method in interface org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureServer.html" title="interface in org.biojava.bio.structure.server">StructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getCalculationTime()">getCalculationTime()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html#getCalculationTime()">getCalculationTime()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentResult</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/CAConverter.html#getCAOnly(java.util.List)">getCAOnly(List&lt;Chain&gt;)</a></span> - Static method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/CAConverter.html" title="class in org.biojava.bio.structure.io">CAConverter</a></dt>
<dd>
<div class="block">Convert a List of chain objects to another List of chains, containing C-alpha atoms only.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/CAConverter.html#getCAOnly(org.biojava.bio.structure.Chain)">getCAOnly(Chain)</a></span> - Static method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/CAConverter.html" title="class in org.biojava.bio.structure.io">CAConverter</a></dt>
<dd>
<div class="block">Convert a Chain to a new Chain containing C-alpha atoms only.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getCartn_x()">getCartn_x()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getCartn_x_esd()">getCartn_x_esd()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getCartn_y()">getCartn_y()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getCartn_y_esd()">getCartn_y_esd()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getCartn_z()">getCartn_z()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getCartn_z_esd()">getCartn_z_esd()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoSCOP.html#getCategories()">getCategories()</a></span> - Method in class demo.<a href="./demo/DemoSCOP.html" title="class in demo">DemoSCOP</a></dt>
<dd>
<div class="block">Get various categories</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDescription.html#getCategory()">getCategory()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDescription.html" title="class in org.biojava.bio.structure.scop">ScopDescription</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModification.html#getCategory()">getCategory()</a></span> - Method in interface org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModification.html" title="interface in org.biojava3.protmod">ProteinModification</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationImpl.html#getCategory()">getCategory()</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationImpl.html" title="class in org.biojava3.protmod">ProteinModificationImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcid.html#getCB()">getCB()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcid.html" title="interface in org.biojava.bio.structure">AminoAcid</a></dt>
<dd>
<div class="block">get CB atom.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcidImpl.html#getCB()">getCB()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcidImpl.html" title="class in org.biojava.bio.structure">AminoAcidImpl</a></dt>
<dd>
<div class="block">get CB atom.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/TranscriptSequence.html#getCDSSequences()">getCDSSequences()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/TranscriptSequence.html" title="class in org.biojava3.core.sequence">TranscriptSequence</a></dt>
<dd>
<div class="block">Get the CDS sequences that have been added to the TranscriptSequences</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeMain.html#getCECalculator()">getCECalculator()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeMain.html" title="class in org.biojava.bio.structure.align.ce">CeMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getCell()">getCell()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/JMatrixPanel.html#getCellColor()">getCellColor()</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/JMatrixPanel.html" title="class in org.biojava.bio.structure.gui">JMatrixPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html#getCellColor()">getCellColor()</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html" title="class in org.biojava.bio.structure.gui">ScaleableMatrixPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getCellLine()">getCellLine()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getCellularLocation()">getCellularLocation()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/AlignTools.html#getCenter(org.biojava.bio.structure.Atom[], int, int)">getCenter(Atom[], int, int)</a></span> - Static method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/AlignTools.html" title="class in org.biojava.bio.structure.align.helper">AlignTools</a></dt>
<dd>
<div class="block">get the centroid for the set of atoms starting fromposition pos, length fragmentLenght</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#getCenter1()">getCenter1()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#getCenter2()">getCenter2()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#getCenterVector(org.biojava.bio.structure.Atom[])">getCenterVector(Atom[])</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">Returns the Vector that needs to be applied to shift a set of atoms
 to the Centroid.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#getCenterVector(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom)">getCenterVector(Atom[], Atom)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">Returns the Vector that needs to be applied to shift a set of atoms
 to the Centroid, if the centroid is already known</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#getCentroid(org.biojava.bio.structure.Atom[])">getCentroid(Atom[])</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">Returns the center  of mass of the set of atoms.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#getChain()">getChain()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">Returns the parent Chain of the Group.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html#getChain()">getChain()</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html" title="class in org.biojava.bio.structure.gui.util">SequenceScalePanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#getChain()">getChain()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">Returns the parent Chain of the Group.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#getChain(int)">getChain(int)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">retrieve a chain by it's position within the Structure .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#getChain(int, int)">getChain(int, int)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">retrieve a chain by it's position within the Structure and model number.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#getChain(int)">getChain(int)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">retrieve a chain by it's position within the Structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#getChain(int, int)">getChain(int, int)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">retrieve a chain by it's position within the Structure and model number.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html#getChain1()">getChain1()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentResult</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/PDBUploadPanel.html#getChain1()">getChain1()</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/PDBUploadPanel.html" title="class in org.biojava.bio.structure.gui.util">PDBUploadPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html#getChain2()">getChain2()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentResult</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#getChainByPDB(java.lang.String)">getChainByPDB(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">request a chain by it's PDB code
 by default takes only the first model</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#getChainByPDB(java.lang.String, int)">getChainByPDB(String, int)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">request a chain by it's PDB code
 by default takes only the first model</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#getChainByPDB(java.lang.String, int)">getChainByPDB(String, int)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#getChainByPDB(java.lang.String)">getChainByPDB(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/StructureName.html#getChainId()">getChainId()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/StructureName.html" title="class in org.biojava.bio.structure.align.client">StructureName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/AtomInfo.html#getChainId()">getChainId()</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/AtomInfo.html" title="class in org.biojava.bio.structure.align.gui.jmol">AtomInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getChainID()">getChainID()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">get and set the name of this chain (Chain id in PDB file ).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getChainID()">getChainID()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">get and set the name of this chain (Chain id in PDB file ).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getChainId()">getChainId()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>
<div class="block">Returns the chain id value.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#getChainId()">getChainId()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">The chain ID of the corresponding chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#getChainId()">getChainId()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">Utility method for returning the chainId of the Group or null if no
 Chain has been set.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#getChainId()">getChainId()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">Utility method for returning the chainId of the Group or null if no
 Chain has been set.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ResidueNumber.html#getChainId()">getChainId()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ResidueNumber.html" title="class in org.biojava.bio.structure">ResidueNumber</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/StructureGroup.html#getChainId()">getChainId()</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/StructureGroup.html" title="class in org.biojava3.protmod.structure">StructureGroup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/PdbPair.html#getChainId1()">getChainId1()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/PdbPair.html" title="class in org.biojava.bio.structure.align.client">PdbPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SSBond.html#getChainID1()">getChainID1()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SSBond.html" title="class in org.biojava.bio.structure">SSBond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/PdbPair.html#getChainId2()">getChainId2()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/PdbPair.html" title="class in org.biojava.bio.structure.align.client">PdbPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SSBond.html#getChainID2()">getChainID2()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SSBond.html" title="class in org.biojava.bio.structure">SSBond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/FlatFileInstallation.html#getChainInfoFile()">getChainInfoFile()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/FlatFileInstallation.html" title="class in org.biojava.bio.structure.server">FlatFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getChainLen()">getChainLen()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getChainRmsd()">getChainRmsd()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getChains()">getChains()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>
<div class="block">get the chains that are part of this Compound</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#getChains()">getChains()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">retrieve all chains - if it is a NMR structure will return the chains of the first model.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#getChains(int)">getChains(int)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">retrieve all chains of a model.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#getChains(int)">getChains(int)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">retrieve all chains of a model.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#getChains()">getChains()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#getChemComp()">getChemComp()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">Get the chemical component that closer describes this group.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#getChemComp()">getChemComp()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/AllChemCompProvider.html#getChemComp(java.lang.String)">getChemComp(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/AllChemCompProvider.html" title="class in org.biojava.bio.structure.io.mmcif">AllChemCompProvider</a></dt>
<dd>
<div class="block">Returns a new instance of a chemical component definition.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompGroupFactory.html#getChemComp(java.lang.String)">getChemComp(String)</a></span> - Static method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompGroupFactory.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompGroupFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompProvider.html#getChemComp(java.lang.String)">getChemComp(String)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompProvider.html" title="interface in org.biojava.bio.structure.io.mmcif">ChemCompProvider</a></dt>
<dd>
<div class="block">Returns a new instance of a chemical component definition.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html#getChemComp(java.lang.String)">getChemComp(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html" title="class in org.biojava.bio.structure.io.mmcif">ChemicalComponentDictionary</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/DownloadChemCompProvider.html#getChemComp(java.lang.String)">getChemComp(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/DownloadChemCompProvider.html" title="class in org.biojava.bio.structure.io.mmcif">DownloadChemCompProvider</a></dt>
<dd>
<div class="block">Loads the definitions for this <a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model"><code>ChemComp</code></a> from a local file and instantiates a new object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ReducedChemCompProvider.html#getChemComp(java.lang.String)">getChemComp(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ReducedChemCompProvider.html" title="class in org.biojava.bio.structure.io.mmcif">ReducedChemCompProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompGroupFactory.html#getChemCompProvider()">getChemCompProvider()</a></span> - Static method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompGroupFactory.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompGroupFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/GuideTree.Node.html#getChild1()">getChild1()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/GuideTree.Node.html" title="class in org.biojava3.alignment">GuideTree.Node</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/GuideTreeNode.html#getChild1()">getChild1()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/GuideTreeNode.html" title="interface in org.biojava3.alignment.template">GuideTreeNode</a></dt>
<dd>
<div class="block">Returns the first child node of this node.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/GuideTree.Node.html#getChild2()">getChild2()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/GuideTree.Node.html" title="class in org.biojava3.alignment">GuideTree.Node</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/GuideTreeNode.html#getChild2()">getChild2()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/GuideTreeNode.html" title="interface in org.biojava3.alignment.template">GuideTreeNode</a></dt>
<dd>
<div class="block">Returns the second child node of this node.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/GuideTree.Node.html#getChildAt(int)">getChildAt(int)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/GuideTree.Node.html" title="class in org.biojava3.alignment">GuideTree.Node</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/GuideTree.Node.html#getChildCount()">getChildCount()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/GuideTree.Node.html" title="class in org.biojava3.alignment">GuideTree.Node</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopNode.html#getChildren()">getChildren()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopNode.html" title="class in org.biojava.bio.structure.scop">ScopNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/AbstractFeature.html#getChildrenFeatures()">getChildrenFeatures()</a></span> - Method in class org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/AbstractFeature.html" title="class in org.biojava3.core.sequence.features">AbstractFeature</a></dt>
<dd>
<div class="block">Get the children features</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/FeatureInterface.html#getChildrenFeatures()">getChildrenFeatures()</a></span> - Method in interface org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/FeatureInterface.html" title="interface in org.biojava3.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">Get the features contained by this feature</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/ChromosomeSequence.html#getChromosomeNumber()">getChromosomeNumber()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/ChromosomeSequence.html" title="class in org.biojava3.core.sequence">ChromosomeSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/GeneFeatureHelper.html#getChromosomeSequenceFromDNASequence(java.util.LinkedHashMap)">getChromosomeSequenceFromDNASequence(LinkedHashMap&lt;String, DNASequence&gt;)</a></span> - Static method in class org.biojava3.genome.<a href="./org/biojava3/genome/GeneFeatureHelper.html" title="class in org.biojava3.genome">GeneFeatureHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#getClassId()">getClassId()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#getClassification()">getClassification()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDescription.html#getClassificationId()">getClassificationId()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDescription.html" title="class in org.biojava.bio.structure.scop">ScopDescription</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDescription.html#getClassificationId(org.biojava.bio.structure.scop.ScopCategory)">getClassificationId(ScopCategory)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDescription.html" title="class in org.biojava.bio.structure.scop">ScopDescription</a></dt>
<dd>
<div class="block">Return a portion of the classificationID corresponding to the specified
 category (class, fold, superfamily, family).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#getClassificationId()">getClassificationId()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/MenuCreator.html#getCloseMenuItem(javax.swing.JFrame)">getCloseMenuItem(JFrame)</a></span> - Static method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/MenuCreator.html" title="class in org.biojava.bio.structure.align.gui">MenuCreator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#getCluster()">getCluster()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>
<div class="block">get the number of the cluster this alignment belongs to</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#getCluster()">getCluster()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/CDSSequence.html#getCodingSequence()">getCodingSequence()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/CDSSequence.html" title="class in org.biojava3.core.sequence">CDSSequence</a></dt>
<dd>
<div class="block">A CDS sequence if negative stranded needs to be reverse complement
 to represent the actual coding sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/IUPACParser.IUPACTable.html#getCodonCompoundSet(org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.template.CompoundSet)">getCodonCompoundSet(CompoundSet&lt;NucleotideCompound&gt;, CompoundSet&lt;AminoAcidCompound&gt;)</a></span> - Method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/IUPACParser.IUPACTable.html" title="class in org.biojava3.core.sequence.io">IUPACParser.IUPACTable</a></dt>
<dd>
<div class="block">Returns the compound set of codons</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/Table.html#getCodonCompoundSet(org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.template.CompoundSet)">getCodonCompoundSet(CompoundSet&lt;NucleotideCompound&gt;, CompoundSet&lt;AminoAcidCompound&gt;)</a></span> - Method in interface org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/Table.html" title="interface in org.biojava3.core.sequence.transcription">Table</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/IUPACParser.IUPACTable.html#getCodons(org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.template.CompoundSet)">getCodons(CompoundSet&lt;NucleotideCompound&gt;, CompoundSet&lt;AminoAcidCompound&gt;)</a></span> - Method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/IUPACParser.IUPACTable.html" title="class in org.biojava3.core.sequence.io">IUPACParser.IUPACTable</a></dt>
<dd>
<div class="block">Returns a list of codons where the source and target compounds
 are the same as those given by the parameters.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/Table.html#getCodons(org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.template.CompoundSet)">getCodons(CompoundSet&lt;NucleotideCompound&gt;, CompoundSet&lt;AminoAcidCompound&gt;)</a></span> - Method in interface org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/Table.html" title="interface in org.biojava3.core.sequence.transcription">Table</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#getCodons(java.lang.String)">getCodons(String)</a></span> - Static method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/IndexPair.html#getCol()">getCol()</a></span> - Method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/IndexPair.html" title="class in org.biojava.bio.structure.align.helper">IndexPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/ContinuousColorMapper.html#getColor(double)">getColor(double)</a></span> - Method in interface org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/ContinuousColorMapper.html" title="interface in org.biojava.bio.structure.gui.util.color">ContinuousColorMapper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/ContinuousColorMapperTransform.html#getColor(double)">getColor(double)</a></span> - Method in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/ContinuousColorMapperTransform.html" title="class in org.biojava.bio.structure.gui.util.color">ContinuousColorMapperTransform</a></dt>
<dd>
<div class="block">Return the color corresponding to submapper.getColor(transform(value)).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/DefaultMatrixMapper.html#getColor(double)">getColor(double)</a></span> - Method in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/DefaultMatrixMapper.html" title="class in org.biojava.bio.structure.gui.util.color">DefaultMatrixMapper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html#getColor(double)">getColor(double)</a></span> - Method in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html" title="class in org.biojava.bio.structure.gui.util.color">GradientMapper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/structure/gui/JmolViewerImpl.html#getColor()">getColor()</a></span> - Method in class org.biojava3.structure.gui.<a href="./org/biojava3/structure/gui/JmolViewerImpl.html" title="class in org.biojava3.structure.gui">JmolViewerImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/structure/gui/StructureViewer.html#getColor()">getColor()</a></span> - Method in interface org.biojava3.structure.gui.<a href="./org/biojava3/structure/gui/StructureViewer.html" title="interface in org.biojava3.structure.gui">StructureViewer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/Alignable.html#getCols()">getCols()</a></span> - Method in interface org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/Alignable.html" title="interface in org.biojava.bio.structure.align.pairwise">Alignable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html#getCols()">getCols()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">StrCompAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html#getCols()">getCols()</a></span> - Method in class org.biojava3.alignment.aaindex.<a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava3.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#getColumnDimension()">getColumnDimension()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Get column dimension.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#getColumnPackedCopy()">getColumnPackedCopy()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Make a one-dimensional column packed copy of the internal array.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html#getComment()">getComment()</a></span> - Static method in class org.biojava3.alignment.aaindex.<a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava3.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#getCommonValence()">getCommonValence()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns the common valence for this Element.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html#getComp_id()">getComp_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemCompDescriptor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxEntityNonPoly.html#getComp_id()">getComp_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxEntityNonPoly.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxEntityNonPoly</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/MyTableRowSorter.html#getComparator(int)">getComparator(int)</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/MyTableRowSorter.html" title="class in org.biojava.bio.structure.align.gui">MyTableRowSorter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/NucleotideCompound.html#getComplement()">getComplement()</a></span> - Method in class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/DNASequence.html#getComplement()">getComplement()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/DNASequence.html" title="class in org.biojava3.core.sequence">DNASequence</a></dt>
<dd>
<div class="block">Returns a Sequence which will complement every base</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/RNASequence.html#getComplement()">getComplement()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/RNASequence.html" title="class in org.biojava3.core.sequence">RNASequence</a></dt>
<dd>
<div class="block">Get the complement view of the RNA sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/ComplementCompound.html#getComplement()">getComplement()</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/ComplementCompound.html" title="interface in org.biojava3.core.sequence.template">ComplementCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ModificationLinkage.html#getComponent1()">getComponent1()</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ModificationLinkage.html" title="class in org.biojava3.protmod">ModificationLinkage</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ModificationLinkage.html#getComponent2()">getComponent2()</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ModificationLinkage.html" title="class in org.biojava3.protmod">ModificationLinkage</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ModificationCondition.html#getComponents()">getComponents()</a></span> - Method in interface org.biojava3.protmod.<a href="./org/biojava3/protmod/ModificationCondition.html" title="interface in org.biojava3.protmod">ModificationCondition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ModificationConditionImpl.html#getComponents()">getComponents()</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ModificationConditionImpl.html" title="class in org.biojava3.protmod">ModificationConditionImpl</a></dt>
<dd>
<div class="block">}</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/IProfeatProperties.html#getComposition(org.biojava3.core.sequence.ProteinSequence, org.biojava3.aaproperties.profeat.IProfeatProperties.ATTRIBUTE, org.biojava3.aaproperties.profeat.IProfeatProperties.GROUPING)">getComposition(ProteinSequence, IProfeatProperties.ATTRIBUTE, IProfeatProperties.GROUPING)</a></span> - Method in interface org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/IProfeatProperties.html" title="interface in org.biojava3.aaproperties.profeat">IProfeatProperties</a></dt>
<dd>
<div class="block">Returns the composition of the specific grouping for the given attribute.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/IProfeatProperties.html#getComposition(org.biojava3.core.sequence.ProteinSequence, org.biojava3.aaproperties.profeat.IProfeatProperties.ATTRIBUTE)">getComposition(ProteinSequence, IProfeatProperties.ATTRIBUTE)</a></span> - Method in interface org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/IProfeatProperties.html" title="interface in org.biojava3.aaproperties.profeat">IProfeatProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/IProfeatProperties.html#getComposition(org.biojava3.core.sequence.ProteinSequence)">getComposition(ProteinSequence)</a></span> - Method in interface org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/IProfeatProperties.html" title="interface in org.biojava3.aaproperties.profeat">IProfeatProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/ProfeatProperties.html#getComposition(org.biojava3.core.sequence.ProteinSequence, org.biojava3.aaproperties.profeat.IProfeatProperties.ATTRIBUTE, org.biojava3.aaproperties.profeat.IProfeatProperties.GROUPING)">getComposition(ProteinSequence, IProfeatProperties.ATTRIBUTE, IProfeatProperties.GROUPING)</a></span> - Static method in class org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/ProfeatProperties.html" title="class in org.biojava3.aaproperties.profeat">ProfeatProperties</a></dt>
<dd>
<div class="block">An adaptor method which returns the composition of the specific grouping for the given attribute.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/ProfeatProperties.html#getComposition(org.biojava3.core.sequence.ProteinSequence, org.biojava3.aaproperties.profeat.IProfeatProperties.ATTRIBUTE)">getComposition(ProteinSequence, IProfeatProperties.ATTRIBUTE)</a></span> - Static method in class org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/ProfeatProperties.html" title="class in org.biojava3.aaproperties.profeat">ProfeatProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/ProfeatProperties.html#getComposition(org.biojava3.core.sequence.ProteinSequence)">getComposition(ProteinSequence)</a></span> - Static method in class org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/ProfeatProperties.html" title="class in org.biojava3.aaproperties.profeat">ProfeatProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/ProfeatProperties.html#getComposition(java.lang.String, org.biojava3.aaproperties.profeat.IProfeatProperties.ATTRIBUTE, org.biojava3.aaproperties.profeat.IProfeatProperties.GROUPING)">getComposition(String, IProfeatProperties.ATTRIBUTE, IProfeatProperties.GROUPING)</a></span> - Static method in class org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/ProfeatProperties.html" title="class in org.biojava3.aaproperties.profeat">ProfeatProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/ProfeatProperties.html#getComposition(java.lang.String, org.biojava3.aaproperties.profeat.IProfeatProperties.ATTRIBUTE)">getComposition(String, IProfeatProperties.ATTRIBUTE)</a></span> - Static method in class org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/ProfeatProperties.html" title="class in org.biojava3.aaproperties.profeat">ProfeatProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/ProfeatProperties.html#getComposition(java.lang.String)">getComposition(String)</a></span> - Static method in class org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/ProfeatProperties.html" title="class in org.biojava3.aaproperties.profeat">ProfeatProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/ProfeatPropertiesImpl.html#getComposition(org.biojava3.core.sequence.ProteinSequence, org.biojava3.aaproperties.profeat.IProfeatProperties.ATTRIBUTE, org.biojava3.aaproperties.profeat.IProfeatProperties.GROUPING)">getComposition(ProteinSequence, IProfeatProperties.ATTRIBUTE, IProfeatProperties.GROUPING)</a></span> - Method in class org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/ProfeatPropertiesImpl.html" title="class in org.biojava3.aaproperties.profeat">ProfeatPropertiesImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/ProfeatPropertiesImpl.html#getComposition(org.biojava3.core.sequence.ProteinSequence, org.biojava3.aaproperties.profeat.IProfeatProperties.ATTRIBUTE)">getComposition(ProteinSequence, IProfeatProperties.ATTRIBUTE)</a></span> - Method in class org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/ProfeatPropertiesImpl.html" title="class in org.biojava3.aaproperties.profeat">ProfeatPropertiesImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/ProfeatPropertiesImpl.html#getComposition(org.biojava3.core.sequence.ProteinSequence)">getComposition(ProteinSequence)</a></span> - Method in class org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/ProfeatPropertiesImpl.html" title="class in org.biojava3.aaproperties.profeat">ProfeatPropertiesImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceMixin.html#getComposition(org.biojava3.core.sequence.template.Sequence)">getComposition(Sequence&lt;C&gt;)</a></span> - Static method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceMixin.html" title="class in org.biojava3.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">Does a linear scan over the given Sequence and records the number of
 times each base appears.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleAlignedSequence.html#getCompoundAt(int)">getCompoundAt(int)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleAlignedSequence.html" title="class in org.biojava3.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleProfile.html#getCompoundAt(int, int)">getCompoundAt(int, int)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleProfile.html" title="class in org.biojava3.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleProfile.html#getCompoundAt(S, int)">getCompoundAt(S, int)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleProfile.html" title="class in org.biojava3.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/Profile.html#getCompoundAt(int, int)">getCompoundAt(int, int)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns the <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template"><code>Compound</code></a> at row of given sequence and column of alignment index.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/Profile.html#getCompoundAt(S, int)">getCompoundAt(S, int)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns the <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template"><code>Compound</code></a> at row of given sequence and column of alignment index.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html#getCompoundAt(int)">getCompoundAt(int)</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava3.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/StringProxySequenceReader.html#getCompoundAt(int)">getCompoundAt(int)</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html#getCompoundAt(int)">getCompoundAt(int)</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/ArrayListSequenceReader.html#getCompoundAt(int)">getCompoundAt(int)</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava3.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#getCompoundAt(int)">getCompoundAt(int)</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava3.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>
<div class="block">Returns the compound at the specified biological index</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/BitSequenceReader.html#getCompoundAt(int)">getCompoundAt(int)</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava3.core.sequence.storage">BitSequenceReader</a></dt>
<dd>
<div class="block">Returns the compound at the specified biological index</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/JoiningSequenceReader.html#getCompoundAt(int)">getCompoundAt(int)</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava3.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html#getCompoundAt(int)">getCompoundAt(int)</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava3.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Always returns the compound given at construction</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#getCompoundAt(int)">getCompoundAt(int)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/Sequence.html#getCompoundAt(int)">getCompoundAt(int)</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a></dt>
<dd>
<div class="block">Returns the Compound at the given biological index</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceProxyView.html#getCompoundAt(int)">getCompoundAt(int)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceProxyView.html" title="class in org.biojava3.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/views/ComplementSequenceView.html#getCompoundAt(int)">getCompoundAt(int)</a></span> - Method in class org.biojava3.core.sequence.views.<a href="./org/biojava3/core/sequence/views/ComplementSequenceView.html" title="class in org.biojava3.core.sequence.views">ComplementSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/views/ReversedSequenceView.html#getCompoundAt(int)">getCompoundAt(int)</a></span> - Method in class org.biojava3.core.sequence.views.<a href="./org/biojava3/core/sequence/views/ReversedSequenceView.html" title="class in org.biojava3.core.sequence.views">ReversedSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/views/RnaSequenceView.html#getCompoundAt(int)">getCompoundAt(int)</a></span> - Method in class org.biojava3.core.sequence.views.<a href="./org/biojava3/core/sequence/views/RnaSequenceView.html" title="class in org.biojava3.core.sequence.views">RnaSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#getCompoundById(java.lang.String)">getCompoundById(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">request a particular compound by its id</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#getCompoundById(java.lang.String)">getCompoundById(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleProfile.html#getCompoundCountsAt(int)">getCompoundCountsAt(int)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleProfile.html" title="class in org.biojava3.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleProfile.html#getCompoundCountsAt(int, java.util.List)">getCompoundCountsAt(int, List&lt;C&gt;)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleProfile.html" title="class in org.biojava3.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/Profile.html#getCompoundCountsAt(int)">getCompoundCountsAt(int)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns the number of each <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template"><code>Compound</code></a> in the given column for all compounds in <a href="./org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template"><code>CompoundSet</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/Profile.html#getCompoundCountsAt(int, java.util.List)">getCompoundCountsAt(int, List&lt;C&gt;)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns the number of each <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template"><code>Compound</code></a> in the given column only for compounds in the given list.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/CaseFreeAminoAcidCompoundSet.html#getCompoundForString(java.lang.String)">getCompoundForString(String)</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/CaseFreeAminoAcidCompoundSet.html" title="class in org.biojava3.aaproperties.xml">CaseFreeAminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/ModifiedAminoAcidCompoundSet.html#getCompoundForString(java.lang.String)">getCompoundForString(String)</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/ModifiedAminoAcidCompoundSet.html" title="class in org.biojava3.aaproperties.xml">ModifiedAminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/AminoAcidCompoundSet.html#getCompoundForString(java.lang.String)">getCompoundForString(String)</a></span> - Method in class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/AminoAcidCompoundSet.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompoundSet.html#getCompoundForString(java.lang.String)">getCompoundForString(String)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava3.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/CompoundSet.html#getCompoundForString(java.lang.String)">getCompoundForString(String)</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a></dt>
<dd>
<div class="block">Return null if not recognised.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleSequencePair.html#getCompoundInQueryAt(int)">getCompoundInQueryAt(int)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleSequencePair.html" title="class in org.biojava3.alignment">SimpleSequencePair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/SequencePair.html#getCompoundInQueryAt(int)">getCompoundInQueryAt(int)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/SequencePair.html" title="interface in org.biojava3.alignment.template">SequencePair</a></dt>
<dd>
<div class="block">Returns the <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template"><code>Compound</code></a> in query sequence at given column index in alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleSequencePair.html#getCompoundInTargetAt(int)">getCompoundInTargetAt(int)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleSequencePair.html" title="class in org.biojava3.alignment">SimpleSequencePair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/SequencePair.html#getCompoundInTargetAt(int)">getCompoundInTargetAt(int)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/SequencePair.html" title="interface in org.biojava3.alignment.template">SequencePair</a></dt>
<dd>
<div class="block">Returns the <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template"><code>Compound</code></a> in target sequence at given column index in alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#getCompounds()">getCompounds()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">get all the Compounds that are defined in the PDB Header</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#getCompounds()">getCompounds()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/views/WindowedSequence.html#getCompounds(int)">getCompounds(int)</a></span> - Method in class org.biojava3.core.sequence.views.<a href="./org/biojava3/core/sequence/views/WindowedSequence.html" title="class in org.biojava3.core.sequence.views">WindowedSequence</a></dt>
<dd>
<div class="block">For a given position into the windowed view this will return those
 compounds we can see in the window. i.e. in the sequence AGGCCT requesting
 index 1 returns AGG and requesting index 2 return CCT.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleProfile.html#getCompoundsAt(int)">getCompoundsAt(int)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleProfile.html" title="class in org.biojava3.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/Profile.html#getCompoundsAt(int)">getCompoundsAt(int)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns the <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template"><code>Compound</code></a> elements of the original <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a>s at the given column.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/MultipleSequenceAlignment.html#getCompoundsAt(int)">getCompoundsAt(int)</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/MultipleSequenceAlignment.html" title="class in org.biojava3.core.sequence">MultipleSequenceAlignment</a></dt>
<dd>
<div class="block">Get a list of compounds at a sequence position</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/LightweightProfile.html#getCompoundsAt(int)">getCompoundsAt(int)</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/LightweightProfile.html" title="interface in org.biojava3.core.sequence.template">LightweightProfile</a></dt>
<dd>
<div class="block">Returns the <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template"><code>Compound</code></a> elements of the original <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a>s at the given column.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html#getCompoundSet()">getCompoundSet()</a></span> - Method in class org.biojava3.alignment.aaindex.<a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava3.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleAlignedSequence.html#getCompoundSet()">getCompoundSet()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleAlignedSequence.html" title="class in org.biojava3.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleProfile.html#getCompoundSet()">getCompoundSet()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleProfile.html" title="class in org.biojava3.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleSubstitutionMatrix.html#getCompoundSet()">getCompoundSet()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleSubstitutionMatrix.html" title="class in org.biojava3.alignment">SimpleSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#getCompoundSet()">getCompoundSet()</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava3.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html#getCompoundSet()">getCompoundSet()</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html" title="class in org.biojava3.alignment.template">AbstractPairwiseSequenceAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractProfileProfileAligner.html#getCompoundSet()">getCompoundSet()</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractProfileProfileAligner.html" title="class in org.biojava3.alignment.template">AbstractProfileProfileAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/Profile.html#getCompoundSet()">getCompoundSet()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns <a href="./org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template"><code>CompoundSet</code></a> of all <a href="./org/biojava3/alignment/template/AlignedSequence.html" title="interface in org.biojava3.alignment.template"><code>AlignedSequence</code></a>s</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/SubstitutionMatrix.html#getCompoundSet()">getCompoundSet()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/SubstitutionMatrix.html" title="interface in org.biojava3.alignment.template">SubstitutionMatrix</a></dt>
<dd>
<div class="block">Returns the <a href="./org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template"><code>CompoundSet</code></a> on which the matrix is defined.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/AminoAcidCompound.html#getCompoundSet()">getCompoundSet()</a></span> - Method in class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html#getCompoundSet()">getCompoundSet()</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava3.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/StringProxySequenceReader.html#getCompoundSet()">getCompoundSet()</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html#getCompoundSet()">getCompoundSet()</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/MultipleSequenceAlignment.html#getCompoundSet()">getCompoundSet()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/MultipleSequenceAlignment.html" title="class in org.biojava3.core.sequence">MultipleSequenceAlignment</a></dt>
<dd>
<div class="block">Get the Compounds defined in the first sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/ArrayListSequenceReader.html#getCompoundSet()">getCompoundSet()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava3.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#getCompoundSet()">getCompoundSet()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava3.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>
<div class="block">Returns the compound set backing this store</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/BitSequenceReader.html#getCompoundSet()">getCompoundSet()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava3.core.sequence.storage">BitSequenceReader</a></dt>
<dd>
<div class="block">Returns the compound set backing this store</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/JoiningSequenceReader.html#getCompoundSet()">getCompoundSet()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava3.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html#getCompoundSet()">getCompoundSet()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava3.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Returns the compound set given at construction</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#getCompoundSet()">getCompoundSet()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/LightweightProfile.html#getCompoundSet()">getCompoundSet()</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/LightweightProfile.html" title="interface in org.biojava3.core.sequence.template">LightweightProfile</a></dt>
<dd>
<div class="block">Returns <a href="./org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template"><code>CompoundSet</code></a> of all <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a>s</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/Sequence.html#getCompoundSet()">getCompoundSet()</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a></dt>
<dd>
<div class="block">Gets the compound set used to back this Sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceProxyView.html#getCompoundSet()">getCompoundSet()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceProxyView.html" title="class in org.biojava3.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#getCompoundsOfQuery()">getCompoundsOfQuery()</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava3.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html#getCompoundsOfQuery()">getCompoundsOfQuery()</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html" title="class in org.biojava3.alignment.template">AbstractPairwiseSequenceAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractProfileProfileAligner.html#getCompoundsOfQuery()">getCompoundsOfQuery()</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractProfileProfileAligner.html" title="class in org.biojava3.alignment.template">AbstractProfileProfileAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#getCompoundsOfTarget()">getCompoundsOfTarget()</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava3.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html#getCompoundsOfTarget()">getCompoundsOfTarget()</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html" title="class in org.biojava3.alignment.template">AbstractPairwiseSequenceAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractProfileProfileAligner.html#getCompoundsOfTarget()">getCompoundsOfTarget()</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractProfileProfileAligner.html" title="class in org.biojava3.alignment.template">AbstractProfileProfileAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#getCompoundsToIndexLookup()">getCompoundsToIndexLookup()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava3.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>
<div class="block">Returns a map which converts from compound to an integer representation</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleProfile.html#getCompoundWeightsAt(int)">getCompoundWeightsAt(int)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleProfile.html" title="class in org.biojava3.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleProfile.html#getCompoundWeightsAt(int, java.util.List)">getCompoundWeightsAt(int, List&lt;C&gt;)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleProfile.html" title="class in org.biojava3.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/Profile.html#getCompoundWeightsAt(int)">getCompoundWeightsAt(int)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns the fraction of each <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template"><code>Compound</code></a> in the given column for all compounds in <a href="./org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template"><code>CompoundSet</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/Profile.html#getCompoundWeightsAt(int, java.util.List)">getCompoundWeightsAt(int, List&lt;C&gt;)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns the fraction of each <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template"><code>Compound</code></a> in the given column only for compounds in the given list.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/StandardRescoreRefiner.html#getComputationTime()">getComputationTime()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/StandardRescoreRefiner.html" title="class in org.biojava3.alignment">StandardRescoreRefiner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#getComputationTime()">getComputationTime()</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava3.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/Aligner.html#getComputationTime()">getComputationTime()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/Aligner.html" title="interface in org.biojava3.alignment.template">Aligner</a></dt>
<dd>
<div class="block">Returns the computation time needed for an alignment computed in nanoseconds.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModification.html#getCondition()">getCondition()</a></span> - Method in interface org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModification.html" title="interface in org.biojava3.protmod">ProteinModification</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationImpl.html#getCondition()">getCondition()</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationImpl.html" title="class in org.biojava3.protmod">ProteinModificationImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/webstart/ConfigXMLHandler.html#getConfig()">getConfig()</a></span> - Method in class org.biojava.bio.structure.align.webstart.<a href="./org/biojava/bio/structure/align/webstart/ConfigXMLHandler.html" title="class in org.biojava.bio.structure.align.webstart">ConfigXMLHandler</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getConn()">getConn()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#getConnections()">getConnections()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">Returns the connections value.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#getConnections()">getConnections()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">Returns the connections value.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/NucleotideCompound.html#getConsituents()">getConsituents()</a></span> - Method in class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#getContacts()">getContacts()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/AligMatEl.html#getContig()">getContig()</a></span> - Method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/AligMatEl.html" title="class in org.biojava.bio.structure.align.helper">AligMatEl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html#getCoordManager()">getCoordManager()</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AligPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#getCoords()">getCoords()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">get the coordinates.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#getCoords()">getCoords()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>
<div class="block">get the coordinates as a double[3] array .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#getCoreElectronCount()">getCoreElectronCount()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns the number of core electrons for this Element.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getCorrelation_coeff_Fo_to_Fc()">getCorrelation_coeff_Fo_to_Fc()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getCorrelation_coeff_Fo_to_Fc_free()">getCorrelation_coeff_Fo_to_Fc_free()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/Name2Count.html#getCount()">getCount()</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/Name2Count.html" title="class in org.biojava3.aaproperties.xml">Name2Count</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#getCovalentRadius()">getCovalentRadius()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns the covalent radius of this Element.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getCoverage1()">getCoverage1()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">Get the coverage of protein 1 with the alignment</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getCoverage2()">getCoverage2()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">Get the coverage of protein 2 with the alignment</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#getCovered()">getCovered()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#getCreate_co()">getCreate_co()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompoundTranslator.html#getCreator()">getCreator()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompoundTranslator.html" title="class in org.biojava3.core.sequence.template">AbstractCompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html#getCrystals_number()">getCrystals_number()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html" title="class in org.biojava.bio.structure.io.mmcif.model">Exptl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBStatus.html#getCurrent(java.lang.String)">getCurrent(String)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBStatus.html" title="class in org.biojava.bio.structure">PDBStatus</a></dt>
<dd>
<div class="block">Gets the current version of a PDB ID.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomIterator.html#getCurrentChain()">getCurrentChain()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomIterator.html" title="class in org.biojava.bio.structure">AtomIterator</a></dt>
<dd>
<div class="block">Get the  chain that contains the current atom.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/GroupIterator.html#getCurrentChain()">getCurrentChain()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/GroupIterator.html" title="class in org.biojava.bio.structure">GroupIterator</a></dt>
<dd>
<div class="block">Get the current Chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomIterator.html#getCurrentModel()">getCurrentModel()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomIterator.html" title="class in org.biojava.bio.structure">AtomIterator</a></dt>
<dd>
<div class="block">Get the model number of the model containing the current atom.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/GroupIterator.html#getCurrentModel()">getCurrentModel()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/GroupIterator.html" title="class in org.biojava.bio.structure">GroupIterator</a></dt>
<dd>
<div class="block">Get the model number of the current model.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/CutSites.html#getCut_sites()">getCut_sites()</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/CutSites.html" title="class in org.biojava.bio.structure.domain.pdp">CutSites</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/routines/AlignerHelper.html#getCuts(int, int[], int[], boolean)">getCuts(int, int[], int[], boolean)</a></span> - Static method in class org.biojava3.alignment.routines.<a href="./org/biojava3/alignment/routines/AlignerHelper.html" title="class in org.biojava3.alignment.routines">AlignerHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/routines/AnchoredPairwiseSequenceAligner.html#getCutsPerSection()">getCutsPerSection()</a></span> - Method in class org.biojava3.alignment.routines.<a href="./org/biojava3/alignment/routines/AnchoredPairwiseSequenceAligner.html" title="class in org.biojava3.alignment.routines">AnchoredPairwiseSequenceAligner</a></dt>
<dd>
<div class="block">Returns the number of cuts added to each section during each pass.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/EigenvalueDecomposition.html#getD()">getD()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/EigenvalueDecomposition.html" title="class in org.biojava.bio.structure.jama">EigenvalueDecomposition</a></dt>
<dd>
<div class="block">Return the block diagonal eigenvalue matrix</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#getDatabase()">getDatabase()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">The database of the db-ref.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/DBReferenceInfo.html#getDatabase()">getDatabase()</a></span> - Method in class org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/DBReferenceInfo.html" title="class in org.biojava3.core.sequence.features">DBReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/DatabaseReferenceInterface.html#getDatabaseReferences()">getDatabaseReferences()</a></span> - Method in interface org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/DatabaseReferenceInterface.html" title="interface in org.biojava3.core.sequence.features">DatabaseReferenceInterface</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html#getDatabaseReferences()">getDatabaseReferences()</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>
<div class="block">The Uniprot mappings to other database identifiers for this sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#getDatabaseReferences()">getDatabaseReferences()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/AccessionID.html#getDataSource()">getDataSource()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/AccessionID.html" title="class in org.biojava3.core.sequence">AccessionID</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/InsdcParser.html#getDataSource()">getDataSource()</a></span> - Method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/InsdcParser.html" title="class in org.biojava3.core.sequence.location">InsdcParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/TaxonomyID.html#getDataSource()">getDataSource()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/TaxonomyID.html" title="class in org.biojava3.core.sequence">TaxonomyID</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html#getDate()">getDate()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html" title="class in org.biojava.bio.structure.io.mmcif.model">DatabasePDBrev</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html#getDate_original()">getDate_original()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html" title="class in org.biojava.bio.structure.io.mmcif.model">DatabasePDBrev</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#getDb_align_beg()">getDb_align_beg()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#getDb_align_end()">getDb_align_end()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html#getDb_code()">getDb_code()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRef</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html#getDb_name()">getDb_name()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRef</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#getDbAccession()">getDbAccession()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Sequence database accession code.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#getDbIdCode()">getDbIdCode()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Sequence database          identification code.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#getDBRefs()">getDBRefs()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">get the list of database references</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#getDBRefs()">getDBRefs()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/MenuCreator.html#getDBResultMenuItem()">getDBResultMenuItem()</a></span> - Static method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/MenuCreator.html" title="class in org.biojava.bio.structure.align.gui">MenuCreator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/AlignmentGui.html#getDBSearch()">getDBSearch()</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/AlignmentGui.html" title="class in org.biojava.bio.structure.align.gui">AlignmentGui</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html#getDbSearchLegend()">getDbSearchLegend()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce">AbstractUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeSideChainUserArgumentProcessor.html#getDbSearchLegend()">getDbSearchLegend()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeSideChainUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce">CeSideChainUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeUserArgumentProcessor.html#getDbSearchLegend()">getDbSearchLegend()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce">CeUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCatUserArgumentProcessor.html#getDbSearchLegend()">getDbSearchLegend()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCatUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.fatcat">FatCatUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWatermanUserArgumentProcessor.html#getDbSearchLegend()">getDbSearchLegend()</a></span> - Method in class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWatermanUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.seq">SmithWatermanUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#getDBSearchParameters()">getDBSearchParameters()</a></span> - Static method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html#getDbSearchResult(org.biojava.bio.structure.align.model.AFPChain)">getDbSearchResult(AFPChain)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce">AbstractUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#getDbSeqBegin()">getDbSeqBegin()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Initial sequence number of the
    database seqment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#getDbSeqEnd()">getDbSeqEnd()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Ending sequence position  of the database segment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.html#getDefault()">getDefault()</a></span> - Static method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.html" title="class in org.biojava3.core.sequence.transcription">TranscriptionEngine</a></dt>
<dd>
<div class="block">Default instance to use when Transcribing from DNA -&gt; RNA -&gt;
 Protein.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/webstart/WebStartMain.html#getDefaultConfig()">getDefaultConfig()</a></span> - Static method in class org.biojava.bio.structure.align.webstart.<a href="./org/biojava/bio/structure/align/webstart/WebStartMain.html" title="class in org.biojava.bio.structure.align.webstart">WebStartMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#getDefaultDnaMatrix()">getDefaultDnaMatrix()</a></span> - Static method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/Frame.html#getDefaultFrame()">getDefaultFrame()</a></span> - Static method in enum org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/Frame.html" title="enum in org.biojava3.core.sequence.transcription">Frame</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#getDefaultParameters()">getDefaultParameters()</a></span> - Static method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#getDefaultPeptideMatrix()">getDefaultPeptideMatrix()</a></span> - Static method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#getDensityCutoff()">getDensityCutoff()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#getDepDate()">getDepDate()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#getDescription()">getDescription()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDescription.html#getDescription()">getDescription()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDescription.html" title="class in org.biojava.bio.structure.scop">ScopDescription</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Site.html#getDescription()">getDescription()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Site.html" title="class in org.biojava.bio.structure">Site</a></dt>
<dd>
<div class="block">gets the REMARK 800 description of the site</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html#getDescription()">getDescription()</a></span> - Method in class org.biojava3.alignment.aaindex.<a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava3.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleSubstitutionMatrix.html#getDescription()">getDescription()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleSubstitutionMatrix.html" title="class in org.biojava3.alignment">SimpleSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/SubstitutionMatrix.html#getDescription()">getDescription()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/SubstitutionMatrix.html" title="interface in org.biojava3.alignment.template">SubstitutionMatrix</a></dt>
<dd>
<div class="block">Returns the description of this matrix.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/CodonCompound.html#getDescription()">getDescription()</a></span> - Method in class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/CodonCompound.html" title="class in org.biojava3.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/AbstractFeature.html#getDescription()">getDescription()</a></span> - Method in class org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/AbstractFeature.html" title="class in org.biojava3.core.sequence.features">AbstractFeature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/FeatureInterface.html#getDescription()">getDescription()</a></span> - Method in interface org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/FeatureInterface.html" title="interface in org.biojava3.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">Get the description that can be used to describe the feature</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompound.html#getDescription()">getDescription()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompound.html" title="class in org.biojava3.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#getDescription()">getDescription()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/Compound.html#getDescription()">getDescription()</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/CaseInsensitiveCompound.html#getDescription()">getDescription()</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava3.core.sequence.transcription">CaseInsensitiveCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/Table.Codon.html#getDescription()">getDescription()</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/Table.Codon.html" title="class in org.biojava3.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModification.html#getDescription()">getDescription()</a></span> - Method in interface org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModification.html" title="interface in org.biojava3.protmod">ProteinModification</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationImpl.html#getDescription()">getDescription()</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationImpl.html" title="class in org.biojava3.protmod">ProteinModificationImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ModifiedCompound.html#getDescription()">getDescription()</a></span> - Method in interface org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ModifiedCompound.html" title="interface in org.biojava3.protmod.structure">ModifiedCompound</a></dt>
<dd>
<div class="block">return a description of this compound</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ModifiedCompoundImpl.html#getDescription()">getDescription()</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ModifiedCompoundImpl.html" title="class in org.biojava3.protmod.structure">ModifiedCompoundImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastOutputProperties.html#getDescriptionNumber()">getDescriptionNumber()</a></span> - Method in class org.biojava3.ws.alignment.qblast.<a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastOutputProperties.html" title="class in org.biojava3.ws.alignment.qblast">NCBIQBlastOutputProperties</a></dt>
<dd>
<div class="block">A method that simply returns the number of descriptions fetched with this RemoteQBlastOutputProperties object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html#getDescriptor()">getDescriptor()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemCompDescriptor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getDescriptors()">getDescriptors()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getDetails()">getDetails()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html#getDetails()">getDetails()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model">Entity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html#getDetails()">getDetails()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html" title="class in org.biojava.bio.structure.io.mmcif.model">Exptl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getDetails()">getDetails()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html#getDetails()">getDetails()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructAsym</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#getDetails()">getDetails()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/AlignTools.html#getDiagonalAtK(org.biojava.bio.structure.Atom[], int)">getDiagonalAtK(Atom[], int)</a></span> - Static method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/AlignTools.html" title="class in org.biojava.bio.structure.align.helper">AlignTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#getDiagonalDistance()">getDiagonalDistance()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#getDiagonalDistance2()">getDiagonalDistance2()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#getDictionary()">getDictionary()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getDisCut()">getDisCut()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>
<div class="block">Get the distance cutoff used during AFP chain connectivity checks</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getDisFilter()">getDisFilter()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/ronn/Jronn.html#getDisorder(org.biojava3.data.sequence.FastaSequence)">getDisorder(FastaSequence)</a></span> - Static method in class org.biojava3.ronn.<a href="./org/biojava3/ronn/Jronn.html" title="class in org.biojava3.ronn">Jronn</a></dt>
<dd>
<div class="block">Calculates the disordered regions of the sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ronn/Jronn.html#getDisorder(java.util.List)">getDisorder(List&lt;FastaSequence&gt;)</a></span> - Static method in class org.biojava3.ronn.<a href="./org/biojava3/ronn/Jronn.html" title="class in org.biojava3.ronn">Jronn</a></dt>
<dd>
<div class="block">Calculates the disordered regions of the sequence for many sequences in the input.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ronn/Jronn.html#getDisorder(java.lang.String)">getDisorder(String)</a></span> - Static method in class org.biojava3.ronn.<a href="./org/biojava3/ronn/Jronn.html" title="class in org.biojava3.ronn">Jronn</a></dt>
<dd>
<div class="block">Calculates the disordered regions of the protein sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ronn/Jronn.html#getDisorderScores(org.biojava3.data.sequence.FastaSequence)">getDisorderScores(FastaSequence)</a></span> - Static method in class org.biojava3.ronn.<a href="./org/biojava3/ronn/Jronn.html" title="class in org.biojava3.ronn">Jronn</a></dt>
<dd>
<div class="block">Calculates the probability value for each residue in the protein sequence, 
 telling the probability that the residue belongs to disordered region.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ronn/Jronn.html#getDisorderScores(java.util.List)">getDisorderScores(List&lt;FastaSequence&gt;)</a></span> - Static method in class org.biojava3.ronn.<a href="./org/biojava3/ronn/Jronn.html" title="class in org.biojava3.ronn">Jronn</a></dt>
<dd>
<div class="block">Calculates the probability of disorder scores for each residue in the sequence for 
 many sequences in the input.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getDisSmooth()">getDisSmooth()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPDistanceMatrix.html#getDist()">getDist()</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPDistanceMatrix.html" title="class in org.biojava.bio.structure.domain.pdp">PDPDistanceMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#getDist1()">getDist1()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#getDist2()">getDist2()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getDisTable1()">getDisTable1()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getDisTable2()">getDisTable2()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#getDistance(org.biojava.bio.structure.Atom, org.biojava.bio.structure.Atom)">getDistance(Atom, Atom)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">calculate distance between two atoms.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/DistEn.html#getDistance()">getDistance()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/DistEn.html" title="class in org.biojava.bio.structure.secstruc">DistEn</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractScorer.html#getDistance()">getDistance()</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractScorer.html" title="class in org.biojava3.alignment.template">AbstractScorer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractScorer.html#getDistance(double)">getDistance(double)</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractScorer.html" title="class in org.biojava3.alignment.template">AbstractScorer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/Scorer.html#getDistance()">getDistance()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/Scorer.html" title="interface in org.biojava3.alignment.template">Scorer</a></dt>
<dd>
<div class="block">Returns score as a distance between 0.0 and 1.0.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/Scorer.html#getDistance(double)">getDistance(double)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/Scorer.html" title="interface in org.biojava3.alignment.template">Scorer</a></dt>
<dd>
<div class="block">Returns score as a distance between 0.0 and scale.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/StructureAtomLinkage.html#getDistance()">getDistance()</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/StructureAtomLinkage.html" title="class in org.biojava3.protmod.structure">StructureAtomLinkage</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#getDistanceFast(org.biojava.bio.structure.Atom, org.biojava.bio.structure.Atom)">getDistanceFast(Atom, Atom)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">Will calculate the *square* of distances between two atoms.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#getDistanceIncrement()">getDistanceIncrement()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/AlignTools.html#getDistanceMatrix(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">getDistanceMatrix(Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/AlignTools.html" title="class in org.biojava.bio.structure.align.helper">AlignTools</a></dt>
<dd>
<div class="block">matrix of all distances between two sets of 3d coords"</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getDistanceMatrix()">getDistanceMatrix()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">A ca1length x ca2length matrix.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#getDistanceMatrix(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">getDistanceMatrix(Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#getDistanceMatrix()">getDistanceMatrix()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>
<div class="block">The distance matrix this alignment is based on</div>
</dd>
<dt><a href="./org/biojava/bio/structure/domain/pdp/GetDistanceMatrix.html" title="class in org.biojava.bio.structure.domain.pdp"><span class="strong">GetDistanceMatrix</span></a> - Class in <a href="./org/biojava/bio/structure/domain/pdp/package-summary.html">org.biojava.bio.structure.domain.pdp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/GetDistanceMatrix.html#GetDistanceMatrix()">GetDistanceMatrix()</a></span> - Constructor for class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/GetDistanceMatrix.html" title="class in org.biojava.bio.structure.domain.pdp">GetDistanceMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/GetDistanceMatrix.html#getDistanceMatrix(org.biojava.bio.structure.Atom[])">getDistanceMatrix(Atom[])</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/GetDistanceMatrix.html" title="class in org.biojava.bio.structure.domain.pdp">GetDistanceMatrix</a></dt>
<dd>
<div class="block">A set of Calpha atoms that are representing the protein</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/GuideTree.html#getDistanceMatrix()">getDistanceMatrix()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/GuideTree.html" title="class in org.biojava3.alignment">GuideTree</a></dt>
<dd>
<div class="block">Returns the distance matrix used to construct this guide tree.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/HierarchicalClusterer.html#getDistanceMatrix()">getDistanceMatrix()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/HierarchicalClusterer.html" title="interface in org.biojava3.alignment.template">HierarchicalClusterer</a></dt>
<dd>
<div class="block">Returns the distance matrix used in clustering.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/TreeConstructor.html#getDistanceMatrix()">getDistanceMatrix()</a></span> - Method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/TreeConstructor.html" title="class in org.biojava3.phylo">TreeConstructor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/GuideTree.Node.html#getDistanceToParent()">getDistanceToParent()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/GuideTree.Node.html" title="class in org.biojava3.alignment">GuideTree.Node</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/GuideTreeNode.html#getDistanceToParent()">getDistanceToParent()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/GuideTreeNode.html" title="interface in org.biojava3.alignment.template">GuideTreeNode</a></dt>
<dd>
<div class="block">Returns the difference in height of this node and it's parent node.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructurePairAligner.html#getDistMat()">getDistMat()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructurePairAligner.html" title="class in org.biojava.bio.structure.align">StructurePairAligner</a></dt>
<dd>
<div class="block">return the difference of distance matrix between the two structures</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceMixin.html#getDistribution(org.biojava3.core.sequence.template.Sequence)">getDistribution(Sequence&lt;C&gt;)</a></span> - Static method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceMixin.html" title="class in org.biojava3.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">Analogous to <a href="./org/biojava3/core/sequence/template/SequenceMixin.html#getComposition(org.biojava3.core.sequence.template.Sequence)"><code>SequenceMixin.getComposition(Sequence)</code></a> but returns the
 distribution of that <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template"><code>Compound</code></a> over the given sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/IProfeatProperties.html#getDistributionPosition(org.biojava3.core.sequence.ProteinSequence, org.biojava3.aaproperties.profeat.IProfeatProperties.ATTRIBUTE, org.biojava3.aaproperties.profeat.IProfeatProperties.GROUPING, org.biojava3.aaproperties.profeat.IProfeatProperties.DISTRIBUTION)">getDistributionPosition(ProteinSequence, IProfeatProperties.ATTRIBUTE, IProfeatProperties.GROUPING, IProfeatProperties.DISTRIBUTION)</a></span> - Method in interface org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/IProfeatProperties.html" title="interface in org.biojava3.aaproperties.profeat">IProfeatProperties</a></dt>
<dd>
<div class="block">Computes and return the position with respect to the sequence where the given distribution of the grouping can be found.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/IProfeatProperties.html#getDistributionPosition(org.biojava3.core.sequence.ProteinSequence, org.biojava3.aaproperties.profeat.IProfeatProperties.ATTRIBUTE, org.biojava3.aaproperties.profeat.IProfeatProperties.GROUPING)">getDistributionPosition(ProteinSequence, IProfeatProperties.ATTRIBUTE, IProfeatProperties.GROUPING)</a></span> - Method in interface org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/IProfeatProperties.html" title="interface in org.biojava3.aaproperties.profeat">IProfeatProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/IProfeatProperties.html#getDistributionPosition(org.biojava3.core.sequence.ProteinSequence, org.biojava3.aaproperties.profeat.IProfeatProperties.ATTRIBUTE)">getDistributionPosition(ProteinSequence, IProfeatProperties.ATTRIBUTE)</a></span> - Method in interface org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/IProfeatProperties.html" title="interface in org.biojava3.aaproperties.profeat">IProfeatProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/IProfeatProperties.html#getDistributionPosition(org.biojava3.core.sequence.ProteinSequence)">getDistributionPosition(ProteinSequence)</a></span> - Method in interface org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/IProfeatProperties.html" title="interface in org.biojava3.aaproperties.profeat">IProfeatProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/ProfeatProperties.html#getDistributionPosition(org.biojava3.core.sequence.ProteinSequence, org.biojava3.aaproperties.profeat.IProfeatProperties.ATTRIBUTE, org.biojava3.aaproperties.profeat.IProfeatProperties.GROUPING, org.biojava3.aaproperties.profeat.IProfeatProperties.DISTRIBUTION)">getDistributionPosition(ProteinSequence, IProfeatProperties.ATTRIBUTE, IProfeatProperties.GROUPING, IProfeatProperties.DISTRIBUTION)</a></span> - Static method in class org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/ProfeatProperties.html" title="class in org.biojava3.aaproperties.profeat">ProfeatProperties</a></dt>
<dd>
<div class="block">An adaptor method which computes and return the position with respect to the sequence where the given distribution of the grouping can be found.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/ProfeatProperties.html#getDistributionPosition(org.biojava3.core.sequence.ProteinSequence, org.biojava3.aaproperties.profeat.IProfeatProperties.ATTRIBUTE, org.biojava3.aaproperties.profeat.IProfeatProperties.GROUPING)">getDistributionPosition(ProteinSequence, IProfeatProperties.ATTRIBUTE, IProfeatProperties.GROUPING)</a></span> - Static method in class org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/ProfeatProperties.html" title="class in org.biojava3.aaproperties.profeat">ProfeatProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/ProfeatProperties.html#getDistributionPosition(org.biojava3.core.sequence.ProteinSequence, org.biojava3.aaproperties.profeat.IProfeatProperties.ATTRIBUTE)">getDistributionPosition(ProteinSequence, IProfeatProperties.ATTRIBUTE)</a></span> - Static method in class org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/ProfeatProperties.html" title="class in org.biojava3.aaproperties.profeat">ProfeatProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/ProfeatProperties.html#getDistributionPosition(org.biojava3.core.sequence.ProteinSequence)">getDistributionPosition(ProteinSequence)</a></span> - Static method in class org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/ProfeatProperties.html" title="class in org.biojava3.aaproperties.profeat">ProfeatProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/ProfeatProperties.html#getDistributionPosition(java.lang.String, org.biojava3.aaproperties.profeat.IProfeatProperties.ATTRIBUTE, org.biojava3.aaproperties.profeat.IProfeatProperties.GROUPING, org.biojava3.aaproperties.profeat.IProfeatProperties.DISTRIBUTION)">getDistributionPosition(String, IProfeatProperties.ATTRIBUTE, IProfeatProperties.GROUPING, IProfeatProperties.DISTRIBUTION)</a></span> - Static method in class org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/ProfeatProperties.html" title="class in org.biojava3.aaproperties.profeat">ProfeatProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/ProfeatProperties.html#getDistributionPosition(java.lang.String, org.biojava3.aaproperties.profeat.IProfeatProperties.ATTRIBUTE, org.biojava3.aaproperties.profeat.IProfeatProperties.GROUPING)">getDistributionPosition(String, IProfeatProperties.ATTRIBUTE, IProfeatProperties.GROUPING)</a></span> - Static method in class org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/ProfeatProperties.html" title="class in org.biojava3.aaproperties.profeat">ProfeatProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/ProfeatProperties.html#getDistributionPosition(java.lang.String, org.biojava3.aaproperties.profeat.IProfeatProperties.ATTRIBUTE)">getDistributionPosition(String, IProfeatProperties.ATTRIBUTE)</a></span> - Static method in class org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/ProfeatProperties.html" title="class in org.biojava3.aaproperties.profeat">ProfeatProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/ProfeatProperties.html#getDistributionPosition(java.lang.String)">getDistributionPosition(String)</a></span> - Static method in class org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/ProfeatProperties.html" title="class in org.biojava3.aaproperties.profeat">ProfeatProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/ProfeatPropertiesImpl.html#getDistributionPosition(org.biojava3.core.sequence.ProteinSequence, org.biojava3.aaproperties.profeat.IProfeatProperties.ATTRIBUTE, org.biojava3.aaproperties.profeat.IProfeatProperties.GROUPING, org.biojava3.aaproperties.profeat.IProfeatProperties.DISTRIBUTION)">getDistributionPosition(ProteinSequence, IProfeatProperties.ATTRIBUTE, IProfeatProperties.GROUPING, IProfeatProperties.DISTRIBUTION)</a></span> - Method in class org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/ProfeatPropertiesImpl.html" title="class in org.biojava3.aaproperties.profeat">ProfeatPropertiesImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/ProfeatPropertiesImpl.html#getDistributionPosition(org.biojava3.core.sequence.ProteinSequence, org.biojava3.aaproperties.profeat.IProfeatProperties.ATTRIBUTE, org.biojava3.aaproperties.profeat.IProfeatProperties.GROUPING)">getDistributionPosition(ProteinSequence, IProfeatProperties.ATTRIBUTE, IProfeatProperties.GROUPING)</a></span> - Method in class org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/ProfeatPropertiesImpl.html" title="class in org.biojava3.aaproperties.profeat">ProfeatPropertiesImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/ProfeatPropertiesImpl.html#getDistributionPosition(org.biojava3.core.sequence.ProteinSequence, org.biojava3.aaproperties.profeat.IProfeatProperties.ATTRIBUTE)">getDistributionPosition(ProteinSequence, IProfeatProperties.ATTRIBUTE)</a></span> - Method in class org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/ProfeatPropertiesImpl.html" title="class in org.biojava3.aaproperties.profeat">ProfeatPropertiesImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/ProfeatPropertiesImpl.html#getDistributionPosition(org.biojava3.core.sequence.ProteinSequence)">getDistributionPosition(ProteinSequence)</a></span> - Method in class org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/ProfeatPropertiesImpl.html" title="class in org.biojava3.aaproperties.profeat">ProfeatPropertiesImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#getDNA()">getDNA()</a></span> - Static method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/TranscriptSequence.html#getDNACodingSequence()">getDNACodingSequence()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/TranscriptSequence.html" title="class in org.biojava3.core.sequence">TranscriptSequence</a></dt>
<dd>
<div class="block">Get the stitched together CDS sequences then maps to the cDNA</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/geneid/GeneIDXMLReader.html#getDNACodingSequences()">getDNACodingSequences()</a></span> - Method in class org.biojava3.genome.parsers.geneid.<a href="./org/biojava3/genome/parsers/geneid/GeneIDXMLReader.html" title="class in org.biojava3.genome.parsers.geneid">GeneIDXMLReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.html#getDnaCompounds()">getDnaCompounds()</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.html" title="class in org.biojava3.core.sequence.transcription">TranscriptionEngine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/AmbiguityDNACompoundSet.html#getDNACompoundSet()">getDNACompoundSet()</a></span> - Static method in class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/AmbiguityDNACompoundSet.html" title="class in org.biojava3.core.sequence.compound">AmbiguityDNACompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/AmbiguityRNACompoundSet.html#getDNACompoundSet()">getDNACompoundSet()</a></span> - Static method in class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/AmbiguityRNACompoundSet.html" title="class in org.biojava3.core.sequence.compound">AmbiguityRNACompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/DNACompoundSet.html#getDNACompoundSet()">getDNACompoundSet()</a></span> - Static method in class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/DNACompoundSet.html" title="class in org.biojava3.core.sequence.compound">DNACompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/chem/ChemCompTools.html#getDNAOneLetter(java.lang.String)">getDNAOneLetter(String)</a></span> - Static method in class org.biojava.bio.structure.io.mmcif.chem.<a href="./org/biojava/bio/structure/io/mmcif/chem/ChemCompTools.html" title="class in org.biojava.bio.structure.io.mmcif.chem">ChemCompTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.html#getDnaRnaTranslator()">getDnaRnaTranslator()</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.html" title="class in org.biojava3.core.sequence.transcription">TranscriptionEngine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/views/RnaSequenceView.html#getDnaToRna()">getDnaToRna()</a></span> - Method in class org.biojava3.core.sequence.views.<a href="./org/biojava3/core/sequence/views/RnaSequenceView.html" title="class in org.biojava3.core.sequence.views">RnaSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/chem/ChemCompTools.html#getDNATwoLetter(java.lang.Character)">getDNATwoLetter(Character)</a></span> - Static method in class org.biojava.bio.structure.io.mmcif.chem.<a href="./org/biojava/bio/structure/io/mmcif/chem/ChemCompTools.html" title="class in org.biojava.bio.structure.io.mmcif.chem">ChemCompTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/DNASequence.html#getDNAType()">getDNAType()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/DNASequence.html" title="class in org.biojava3.core.sequence">DNASequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#getDoi()">getDoi()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>
<div class="block">Get the value of DOI field.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html#getDomainByScopID(java.lang.String)">getDomainByScopID(String)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">RemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDatabase.html#getDomainByScopID(java.lang.String)">getDomainByScopID(String)</a></span> - Method in interface org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDatabase.html" title="interface in org.biojava.bio.structure.scop">ScopDatabase</a></dt>
<dd>
<div class="block">get a ScopDomain by its SCOP ID (warning, they are not stable between releases!)</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#getDomainByScopID(java.lang.String)">getDomainByScopID(String)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#getDomainId()">getDomainId()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/CutDomain.html#getDomains()">getDomains()</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/CutDomain.html" title="class in org.biojava.bio.structure.domain.pdp">CutDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html#getDomainsForPDB(java.lang.String)">getDomainsForPDB(String)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">RemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDatabase.html#getDomainsForPDB(java.lang.String)">getDomainsForPDB(String)</a></span> - Method in interface org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDatabase.html" title="interface in org.biojava.bio.structure.scop">ScopDatabase</a></dt>
<dd>
<div class="block">Get a list of ScopDomains that have been assigned to a PDB ID</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#getDomainsForPDB(java.lang.String)">getDomainsForPDB(String)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/LUDecomposition.html#getDoublePivot()">getDoublePivot()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/LUDecomposition.html" title="class in org.biojava.bio.structure.jama">LUDecomposition</a></dt>
<dd>
<div class="block">Return pivot permutation vector as a one-dimensional double array</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#getDssp3state(java.lang.String)">getDssp3state(String)</a></span> - Static method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>
<div class="block">translate from other dssp secondary structure alphabets to 3-state</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getDVar()">getDVar()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getEcNums()">getEcNums()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#getEditorList()">getEditorList()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>
<div class="block">Get the list of editors of the JournalArticle</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#getElement()">getElement()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">get element of the atom, e.g.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#getElement()">getElement()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/ElementTable.html#getElement()">getElement()</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/ElementTable.html" title="class in org.biojava3.aaproperties.xml">ElementTable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/ElementTable.html#getElement(java.lang.String)">getElement(String)</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/ElementTable.html" title="class in org.biojava3.aaproperties.xml">ElementTable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#getElementFromHillIndex(int)">getElementFromHillIndex(int)</a></span> - Static method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns the Element that corresponds to the specified Hill Order.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/AminoAcidComposition.html#getElementList()">getElementList()</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/AminoAcidComposition.html" title="class in org.biojava3.aaproperties.xml">AminoAcidComposition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#getElementType()">getElementType()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns the Element Type for this Element.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastService.html#getEmail()">getEmail()</a></span> - Method in class org.biojava3.ws.alignment.qblast.<a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastService.html" title="class in org.biojava3.ws.alignment.qblast">NCBIQBlastService</a></dt>
<dd>
<div class="block">Get the email for QBlast.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ReducedChemCompProvider.html#getEmptyChemComp()">getEmptyChemComp()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ReducedChemCompProvider.html" title="class in org.biojava.bio.structure.io.mmcif">ReducedChemCompProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/edits/Edit.AbstractEdit.html#getEmptySequence(org.biojava3.core.sequence.template.Sequence)">getEmptySequence(Sequence&lt;C&gt;)</a></span> - Method in class org.biojava3.core.sequence.edits.<a href="./org/biojava3/core/sequence/edits/Edit.AbstractEdit.html" title="class in org.biojava3.core.sequence.edits">Edit.AbstractEdit</a></dt>
<dd>
<div class="block">Returns an empty sequence with the given compound set of the editing
 sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleAlignedSequence.html#getEnd()">getEnd()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleAlignedSequence.html" title="class in org.biojava3.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AlignedSequence.html#getEnd()">getEnd()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AlignedSequence.html" title="interface in org.biojava3.alignment.template">AlignedSequence</a></dt>
<dd>
<div class="block">Returns the <a href="./org/biojava3/core/sequence/location/template/Point.html" title="interface in org.biojava3.core.sequence.location.template"><code>Point</code></a> within an alignment of the last element of the original <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/ProfileView.html#getEnd()">getEnd()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/ProfileView.html" title="interface in org.biojava3.alignment.template">ProfileView</a></dt>
<dd>
<div class="block">Returns the column index of the viewed <a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template"><code>Profile</code></a> corresponding to the final element in this view</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/edits/Edit.AbstractEdit.html#getEnd()">getEnd()</a></span> - Method in class org.biojava3.core.sequence.edits.<a href="./org/biojava3/core/sequence/edits/Edit.AbstractEdit.html" title="class in org.biojava3.core.sequence.edits">Edit.AbstractEdit</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/edits/Edit.Substitute.html#getEnd(org.biojava3.core.sequence.template.Sequence)">getEnd(Sequence&lt;C&gt;)</a></span> - Method in class org.biojava3.core.sequence.edits.<a href="./org/biojava3/core/sequence/edits/Edit.Substitute.html" title="class in org.biojava3.core.sequence.edits">Edit.Substitute</a></dt>
<dd>
<div class="block">Must use this rather than the no-args getEnd as this can return
 -1 and the length of a sub is dependent on the length of the
 Sequence; we cannot assume 1:1 mapping between characters in a
 String and the number of compounds we will have to insert.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html#getEnd()">getEnd()</a></span> - Method in class org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava3.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/Location.html#getEnd()">getEnd()</a></span> - Method in interface org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/Location.html" title="interface in org.biojava3.core.sequence.location.template">Location</a></dt>
<dd>
<div class="block">End of the location</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Location.html#getEnd()">getEnd()</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Location.html" title="class in org.biojava3.genome.parsers.gff">Location</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AFPChainCoordManager.html#getEndLegendPosition(int, int)">getEndLegendPosition(int, int)</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AFPChainCoordManager.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AFPChainCoordManager</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/DistEn.html#getEnergy()">getEnergy()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/DistEn.html" title="class in org.biojava.bio.structure.secstruc">DistEn</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/HBond.html#getEnergy()">getEnergy()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/HBond.html" title="class in org.biojava.bio.structure.secstruc">HBond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getEngineered()">getEngineered()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/IPeptideProperties.html#getEnrichment(org.biojava3.core.sequence.ProteinSequence, org.biojava3.core.sequence.compound.AminoAcidCompound)">getEnrichment(ProteinSequence, AminoAcidCompound)</a></span> - Method in interface org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/IPeptideProperties.html" title="interface in org.biojava3.aaproperties">IPeptideProperties</a></dt>
<dd>
<div class="block">Returns the composition of specified amino acid in the sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/PeptideProperties.html#getEnrichment(java.lang.String, org.biojava3.aaproperties.PeptideProperties.SingleLetterAACode)">getEnrichment(String, PeptideProperties.SingleLetterAACode)</a></span> - Static method in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/PeptideProperties.html" title="class in org.biojava3.aaproperties">PeptideProperties</a></dt>
<dd>
<div class="block">An adaptor method to return the composition of specified amino acid in the sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/PeptideProperties.html#getEnrichment(java.lang.String, char)">getEnrichment(String, char)</a></span> - Static method in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/PeptideProperties.html" title="class in org.biojava3.aaproperties">PeptideProperties</a></dt>
<dd>
<div class="block">An adaptor method to return the composition of specified amino acid in the sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/PeptideProperties.html#getEnrichment(java.lang.String, java.lang.String)">getEnrichment(String, String)</a></span> - Static method in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/PeptideProperties.html" title="class in org.biojava3.aaproperties">PeptideProperties</a></dt>
<dd>
<div class="block">An adaptor method to return the composition of specified amino acid in the sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/PeptidePropertiesImpl.html#getEnrichment(org.biojava3.core.sequence.ProteinSequence, org.biojava3.core.sequence.compound.AminoAcidCompound)">getEnrichment(ProteinSequence, AminoAcidCompound)</a></span> - Method in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/PeptidePropertiesImpl.html" title="class in org.biojava3.aaproperties">PeptidePropertiesImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/EntityPolySeq.html#getEntity_id()">getEntity_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/EntityPolySeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">EntityPolySeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxEntityNonPoly.html#getEntity_id()">getEntity_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxEntityNonPoly.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxEntityNonPoly</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#getEntity_id()">getEntity_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#getEntity_id()">getEntity_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html#getEntity_id()">getEntity_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructAsym</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html#getEntity_id()">getEntity_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRef</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html#getEntry_id()">getEntry_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html" title="class in org.biojava.bio.structure.io.mmcif.model">Exptl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getEntry_id()">getEntry_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html#getEntry_id()">getEntry_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html" title="class in org.biojava.bio.structure.io.mmcif.model">Struct</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructKeywords.html#getEntry_id()">getEntry_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructKeywords.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructKeywords</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#getEqr()">getEqr()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>
<div class="block">returns the number of euqivalent residues in this alignment</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/DisplayAFP.html#getEQRAlignmentPos(org.biojava.bio.structure.align.model.AFPChain)">getEQRAlignmentPos(AFPChain)</a></span> - Static method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/DisplayAFP.html" title="class in org.biojava.bio.structure.align.gui">DisplayAFP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/AlignedPosition.html#getEquivalent()">getEquivalent()</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/AlignedPosition.html" title="class in org.biojava.bio.structure.gui.util">AlignedPosition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/CaseFreeAminoAcidCompoundSet.html#getEquivalentCompounds(org.biojava3.core.sequence.compound.AminoAcidCompound)">getEquivalentCompounds(AminoAcidCompound)</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/CaseFreeAminoAcidCompoundSet.html" title="class in org.biojava3.aaproperties.xml">CaseFreeAminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/ModifiedAminoAcidCompoundSet.html#getEquivalentCompounds(org.biojava3.core.sequence.compound.AminoAcidCompound)">getEquivalentCompounds(AminoAcidCompound)</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/ModifiedAminoAcidCompoundSet.html" title="class in org.biojava3.aaproperties.xml">ModifiedAminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/AminoAcidCompoundSet.html#getEquivalentCompounds(org.biojava3.core.sequence.compound.AminoAcidCompound)">getEquivalentCompounds(AminoAcidCompound)</a></span> - Method in class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/AminoAcidCompoundSet.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompoundSet.html#getEquivalentCompounds(C)">getEquivalentCompounds(C)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava3.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/CompoundSet.html#getEquivalentCompounds(C)">getEquivalentCompounds(C)</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/BigSqrt.html#getError()">getError()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/BigSqrt.html" title="class in org.biojava.bio.structure.secstruc">BigSqrt</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#getEvalCutoff()">getEvalCutoff()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Site.html#getEvCode()">getEvCode()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Site.html" title="class in org.biojava.bio.structure">Site</a></dt>
<dd>
<div class="block">gets the REMARK 800 EVIDENCE CODE for the site.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/MenuCreator.html#getExitMenuItem()">getExitMenuItem()</a></span> - Static method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/MenuCreator.html" title="class in org.biojava.bio.structure.align.gui">MenuCreator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/GeneSequence.html#getExonSequences()">getExonSequences()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/GeneSequence.html" title="class in org.biojava3.core.sequence">GeneSequence</a></dt>
<dd>
<div class="block">Get the exons as an ArrayList</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/MenuCreator.html#getExportPDBMenuItem(org.biojava.bio.structure.align.gui.jmol.StructureAlignmentJmol)">getExportPDBMenuItem(StructureAlignmentJmol)</a></span> - Static method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/MenuCreator.html" title="class in org.biojava.bio.structure.align.gui">MenuCreator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getExpressionSystem()">getExpressionSystem()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getExpressionSystemAtccNumber()">getExpressionSystemAtccNumber()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getExpressionSystemCell()">getExpressionSystemCell()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getExpressionSystemCellLine()">getExpressionSystemCellLine()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getExpressionSystemCellularLocation()">getExpressionSystemCellularLocation()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getExpressionSystemGene()">getExpressionSystemGene()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getExpressionSystemOrgan()">getExpressionSystemOrgan()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getExpressionSystemOrganelle()">getExpressionSystemOrganelle()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getExpressionSystemOtherDetails()">getExpressionSystemOtherDetails()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getExpressionSystemPlasmid()">getExpressionSystemPlasmid()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getExpressionSystemStrain()">getExpressionSystemStrain()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getExpressionSystemTaxId()">getExpressionSystemTaxId()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getExpressionSystemTissue()">getExpressionSystemTissue()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getExpressionSystemVariant()">getExpressionSystemVariant()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getExpressionSystemVector()">getExpressionSystemVector()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getExpressionSystemVectorType()">getExpressionSystemVectorType()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleGapPenalty.html#getExtensionPenalty()">getExtensionPenalty()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleGapPenalty.html" title="class in org.biojava3.alignment">SimpleGapPenalty</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/GapPenalty.html#getExtensionPenalty()">getExtensionPenalty()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/GapPenalty.html" title="interface in org.biojava3.alignment.template">GapPenalty</a></dt>
<dd>
<div class="block">Returns penalty given when an already open gap elongates by a single element</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/IPeptideProperties.html#getExtinctionCoefficient(org.biojava3.core.sequence.ProteinSequence, boolean)">getExtinctionCoefficient(ProteinSequence, boolean)</a></span> - Method in interface org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/IPeptideProperties.html" title="interface in org.biojava3.aaproperties">IPeptideProperties</a></dt>
<dd>
<div class="block">Returns the extinction coefficient of sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/PeptideProperties.html#getExtinctionCoefficient(java.lang.String, boolean)">getExtinctionCoefficient(String, boolean)</a></span> - Static method in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/PeptideProperties.html" title="class in org.biojava3.aaproperties">PeptideProperties</a></dt>
<dd>
<div class="block">An adaptor method to return the extinction coefficient of sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/PeptidePropertiesImpl.html#getExtinctionCoefficient(org.biojava3.core.sequence.ProteinSequence, boolean)">getExtinctionCoefficient(ProteinSequence, boolean)</a></span> - Method in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/PeptidePropertiesImpl.html" title="class in org.biojava3.aaproperties">PeptidePropertiesImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#getFamilyId()">getFamilyId()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/GUIAlignmentProgressListener.html#getFarmJob()">getFarmJob()</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/GUIAlignmentProgressListener.html" title="class in org.biojava.bio.structure.align.gui">GUIAlignmentProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCat.html#getFatCatAligner()">getFatCatAligner()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCat.html" title="class in org.biojava.bio.structure.align.fatcat">FatCat</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#getFeatures(java.lang.String, int)">getFeatures(String, int)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>
<div class="block">Return features at a sequence position by type</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#getFeatures(int)">getFeatures(int)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>
<div class="block">Return features at a sequence position</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#getFeatures()">getFeatures()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#getFeaturesByType(java.lang.String)">getFeaturesByType(String)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#getFeaturesKeyWord()">getFeaturesKeyWord()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#getFile1()">getFile1()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#getFile2()">getFile2()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/UserConfiguration.html#getFileFormat()">getFileFormat()</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/UserConfiguration.html" title="class in org.biojava.bio.structure.align.util">UserConfiguration</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/PDBUploadPanel.html#getFileFormatSelect()">getFileFormatSelect()</a></span> - Static method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/PDBUploadPanel.html" title="class in org.biojava.bio.structure.gui.util">PDBUploadPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/LocalCacheStructureProvider.html#getFileParsingParameters()">getFileParsingParameters()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/LocalCacheStructureProvider.html" title="class in org.biojava.bio.structure.io">LocalCacheStructureProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#getFileParsingParameters()">getFileParsingParameters()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#getFileParsingParameters()">getFileParsingParameters()</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#getFileParsingParameters()">getFileParsingParameters()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/MMCIFFileReader.html#getFileParsingParameters()">getFileParsingParameters()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/MMCIFFileReader.html" title="class in org.biojava.bio.structure.io">MMCIFFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileParser.html#getFileParsingParameters()">getFileParsingParameters()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileParser.html" title="class in org.biojava.bio.structure.io">PDBFileParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#getFileParsingParameters()">getFileParsingParameters()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/SandboxStyleStructureProvider.html#getFileParsingParameters()">getFileParsingParameters()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/SandboxStyleStructureProvider.html" title="class in org.biojava.bio.structure.io">SandboxStyleStructureProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/StructureIOFile.html#getFileParsingParameters()">getFileParsingParameters()</a></span> - Method in interface org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/StructureIOFile.html" title="interface in org.biojava.bio.structure.io">StructureIOFile</a></dt>
<dd>
<div class="block">Get the parameters that should be used for file parsing</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/StructureProvider.html#getFileParsingParameters()">getFileParsingParameters()</a></span> - Method in interface org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/StructureProvider.html" title="interface in org.biojava.bio.structure.io">StructureProvider</a></dt>
<dd>
<div class="block">Get the parameters that should be used for file parsing</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#getFileParsingParams()">getFileParsingParams()</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/FlatFileInstallation.html#getFilePath()">getFilePath()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/FlatFileInstallation.html" title="class in org.biojava.bio.structure.server">FlatFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/PDBUploadPanel.html#getFilePath1()">getFilePath1()</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/PDBUploadPanel.html" title="class in org.biojava.bio.structure.gui.util">PDBUploadPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/edits/Edit.AbstractEdit.html#getFivePrime(org.biojava3.core.sequence.template.Sequence)">getFivePrime(Sequence&lt;C&gt;)</a></span> - Method in class org.biojava3.core.sequence.edits.<a href="./org/biojava3/core/sequence/edits/Edit.AbstractEdit.html" title="class in org.biojava3.core.sequence.edits">Edit.AbstractEdit</a></dt>
<dd>
<div class="block">Should return the 5-prime end of the given Sequence according to
 the edit.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/edits/Edit.Delete.html#getFivePrime(org.biojava3.core.sequence.template.Sequence)">getFivePrime(Sequence&lt;C&gt;)</a></span> - Method in class org.biojava3.core.sequence.edits.<a href="./org/biojava3/core/sequence/edits/Edit.Delete.html" title="class in org.biojava3.core.sequence.edits">Edit.Delete</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/edits/Edit.Insert.html#getFivePrime(org.biojava3.core.sequence.template.Sequence)">getFivePrime(Sequence&lt;C&gt;)</a></span> - Method in class org.biojava3.core.sequence.edits.<a href="./org/biojava3/core/sequence/edits/Edit.Insert.html" title="class in org.biojava3.core.sequence.edits">Edit.Insert</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/edits/Edit.Substitute.html#getFivePrime(org.biojava3.core.sequence.template.Sequence)">getFivePrime(Sequence&lt;C&gt;)</a></span> - Method in class org.biojava3.core.sequence.edits.<a href="./org/biojava3/core/sequence/edits/Edit.Substitute.html" title="class in org.biojava3.core.sequence.edits">Edit.Substitute</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getFocusAfpList()">getFocusAfpList()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getFocusAfpn()">getFocusAfpn()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getFocusRes1()">getFocusRes1()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getFocusRes2()">getFocusRes2()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getFocusResn()">getFocusResn()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#getFoldId()">getFoldId()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/data/sequence/FastaSequence.html#getFormatedSequence(int)">getFormatedSequence(int)</a></span> - Method in class org.biojava3.data.sequence.<a href="./org/biojava3/data/sequence/FastaSequence.html" title="class in org.biojava3.data.sequence">FastaSequence</a></dt>
<dd>
<div class="block">Format sequence per width letter in one string.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/data/sequence/FastaSequence.html#getFormattedFasta()">getFormattedFasta()</a></span> - Method in class org.biojava3.data.sequence.<a href="./org/biojava3/data/sequence/FastaSequence.html" title="class in org.biojava3.data.sequence">FastaSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getFormula()">getFormula()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModification.html#getFormula()">getFormula()</a></span> - Method in interface org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModification.html" title="interface in org.biojava3.protmod">ProteinModification</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationImpl.html#getFormula()">getFormula()</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationImpl.html" title="class in org.biojava3.protmod">ProteinModificationImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getFormula_weight()">getFormula_weight()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html#getFormula_weight()">getFormula_weight()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model">Entity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/Frame.html#getForwardFrames()">getForwardFrames()</a></span> - Static method in enum org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/Frame.html" title="enum in org.biojava3.core.sequence.transcription">Frame</a></dt>
<dd>
<div class="block">Returns all frames in the forward orientation</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#getFragCompat()">getFragCompat()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getFragLen()">getFragLen()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFP.html#getFragLen()">getFragLen()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFP.html" title="class in org.biojava.bio.structure.align.model">AFP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getFragLenSq()">getFragLenSq()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/AlignTools.html#getFragment(org.biojava.bio.structure.Atom[], int, int)">getFragment(Atom[], int, int)</a></span> - Static method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/AlignTools.html" title="class in org.biojava.bio.structure.align.helper">AlignTools</a></dt>
<dd>
<div class="block">get a continue subset of Atoms based by the starting position and the length</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getFragment()">getFragment()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/AlignTools.html#getFragmentFromIdxList(org.biojava.bio.structure.Atom[], int[])">getFragmentFromIdxList(Atom[], int[])</a></span> - Static method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/AlignTools.html" title="class in org.biojava.bio.structure.align.helper">AlignTools</a></dt>
<dd>
<div class="block">get a subset of Atoms based by their positions</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#getFragmentLength()">getFragmentLength()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#getFragmentMiniDistance()">getFragmentMiniDistance()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/AlignTools.html#getFragmentNoClone(org.biojava.bio.structure.Atom[], int, int)">getFragmentNoClone(Atom[], int, int)</a></span> - Static method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/AlignTools.html" title="class in org.biojava.bio.structure.align.helper">AlignTools</a></dt>
<dd>
<div class="block">get a continue subset of Atoms based by the starting position and the length
 does not clone the original atoms.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructurePairAligner.html#getFragmentPairs()">getFragmentPairs()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructurePairAligner.html" title="class in org.biojava.bio.structure.align">StructurePairAligner</a></dt>
<dd>
<div class="block">get the results of step 1 - the FragmentPairs used for seeding the alignment</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/JMatrixPanel.html#getFragmentPairs()">getFragmentPairs()</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/JMatrixPanel.html" title="class in org.biojava.bio.structure.gui">JMatrixPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html#getFragmentPairs()">getFragmentPairs()</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html" title="class in org.biojava.bio.structure.gui">ScaleableMatrixPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getFragScore()">getFragScore()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/BiojavaJmol.html#getFrame()">getFrame()</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/BiojavaJmol.html" title="class in org.biojava.bio.structure.gui">BiojavaJmol</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Segment.html#getFrom()">getFrom()</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Segment.html" title="class in org.biojava.bio.structure.domain.pdp">Segment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompoundTranslator.html#getFromCompoundSet()">getFromCompoundSet()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompoundTranslator.html" title="class in org.biojava3.core.sequence.template">AbstractCompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/ConfigPDBInstallPanel.html#getFromFtp()">getFromFtp()</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/ConfigPDBInstallPanel.html" title="class in org.biojava.bio.structure.align.gui">ConfigPDBInstallPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/SeqRes2AtomAligner.html#getFullAtomSequence(java.util.List, java.util.Map)">getFullAtomSequence(List&lt;Group&gt;, Map&lt;Integer, Integer&gt;)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/SeqRes2AtomAligner.html" title="class in org.biojava.bio.structure.io">SeqRes2AtomAligner</a></dt>
<dd>
<div class="block">returns the full sequence of the Atom records of a parent
 with X instead of HETATMSs.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#getFullName()">getFullName()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">get full name of atom e.g. " CA ".</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#getFullName()">getFullName()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>
<div class="block">get full name of atom e.g. " CA ".</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getGapCreate()">getGapCreate()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/Alignable.html#getGapExtCol()">getGapExtCol()</a></span> - Method in interface org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/Alignable.html" title="interface in org.biojava.bio.structure.align.pairwise">Alignable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html#getGapExtCol()">getGapExtCol()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">StrCompAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getGapExtend()">getGapExtend()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWaterman3DParameters.html#getGapExtend()">getGapExtend()</a></span> - Method in class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWaterman3DParameters.html" title="class in org.biojava.bio.structure.align.seq">SmithWaterman3DParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#getGapExtension()">getGapExtension()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#getGapExtension()">getGapExtension()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/Alignable.html#getGapExtRow()">getGapExtRow()</a></span> - Method in interface org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/Alignable.html" title="interface in org.biojava.bio.structure.align.pairwise">Alignable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html#getGapExtRow()">getGapExtRow()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">StrCompAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getGapLen()">getGapLen()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#getGapOpen()">getGapOpen()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWaterman3DParameters.html#getGapOpen()">getGapOpen()</a></span> - Method in class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWaterman3DParameters.html" title="class in org.biojava.bio.structure.align.seq">SmithWaterman3DParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#getGapOpen()">getGapOpen()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/Alignable.html#getGapOpenCol()">getGapOpenCol()</a></span> - Method in interface org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/Alignable.html" title="interface in org.biojava.bio.structure.align.pairwise">Alignable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html#getGapOpenCol()">getGapOpenCol()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">StrCompAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/Alignable.html#getGapOpenRow()">getGapOpenRow()</a></span> - Method in interface org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/Alignable.html" title="interface in org.biojava.bio.structure.align.pairwise">Alignable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html#getGapOpenRow()">getGapOpenRow()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">StrCompAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#getGapPenalty()">getGapPenalty()</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava3.alignment.template">AbstractMatrixAligner</a></dt>
<dd>
<div class="block">Returns the gap penalties.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#getGaps()">getGaps()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>
<div class="block">return the number of gaps in this alignment</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/RNASequence.html#getGC()">getGC()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/RNASequence.html" title="class in org.biojava3.core.sequence">RNASequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/DNASequence.html#getGCCount()">getGCCount()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/DNASequence.html" title="class in org.biojava3.core.sequence">DNASequence</a></dt>
<dd>
<div class="block">Get the GC count in the DNA Sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/util/IOUtils.html#getGCGChecksum(java.util.List)">getGCGChecksum(List&lt;S&gt;)</a></span> - Static method in class org.biojava3.core.sequence.io.util.<a href="./org/biojava3/core/sequence/io/util/IOUtils.html" title="class in org.biojava3.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">Calculates GCG checksum for entire list of sequences</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/util/IOUtils.html#getGCGChecksum(S)">getGCGChecksum(S)</a></span> - Static method in class org.biojava3.core.sequence.io.util.<a href="./org/biojava3/core/sequence/io/util/IOUtils.html" title="class in org.biojava3.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">Calculates GCG checksum for a given sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/util/IOUtils.html#getGCGHeader(java.util.List)">getGCGHeader(List&lt;S&gt;)</a></span> - Static method in class org.biojava3.core.sequence.io.util.<a href="./org/biojava3/core/sequence/io/util/IOUtils.html" title="class in org.biojava3.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">Assembles a GCG file header</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/util/IOUtils.html#getGCGType(org.biojava3.core.sequence.template.CompoundSet)">getGCGType(CompoundSet&lt;C&gt;)</a></span> - Static method in class org.biojava3.core.sequence.io.util.<a href="./org/biojava3/core/sequence/io/util/IOUtils.html" title="class in org.biojava3.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">Determines GCG type</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/GCStats.html#getGCStats(java.util.Collection)">getGCStats(Collection&lt;DNASequence&gt;)</a></span> - Static method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/GCStats.html" title="class in org.biojava3.genome.parsers.gff">GCStats</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/GCStats.html#getGCStatsString(java.util.Collection)">getGCStatsString(Collection&lt;String&gt;)</a></span> - Static method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/GCStats.html" title="class in org.biojava3.genome.parsers.gff">GCStats</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getGene()">getGene()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/ChromosomeSequence.html#getGene(java.lang.String)">getGene(String)</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/ChromosomeSequence.html" title="class in org.biojava3.core.sequence">ChromosomeSequence</a></dt>
<dd>
<div class="block">Get the gene based on accession.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html#getGeneName()">getGeneName()</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>
<div class="block">Get the gene name associated with this sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/ChromosomeSequence.html#getGeneSequences()">getGeneSequences()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/ChromosomeSequence.html" title="class in org.biojava3.core.sequence">ChromosomeSequence</a></dt>
<dd>
<div class="block">Get the list of genes that have been added to the ChromosomeSequence where accession.toString is the key.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/GeneFeatureHelper.html#getGeneSequences(java.util.Collection)">getGeneSequences(Collection&lt;ChromosomeSequence&gt;)</a></span> - Static method in class org.biojava3.genome.<a href="./org/biojava3/genome/GeneFeatureHelper.html" title="class in org.biojava3.genome">GeneFeatureHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SubstitutionMatrixHelper.html#getGonnet250()">getGonnet250()</a></span> - Static method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SubstitutionMatrixHelper.html" title="class in org.biojava3.alignment">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns PAM 250 matrix by Gonnet, Cohen & Benner</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html#getGradientMapper(int, double, double)">getGradientMapper(int, double, double)</a></span> - Static method in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html" title="class in org.biojava.bio.structure.gui.util.color">GradientMapper</a></dt>
<dd>
<div class="block">Constructs a gradientMapper to draw one of the pre-defined gradients
 
 For example,
 GradientMapper.getGradientMapper(GradientMapper.RAINBOW_GRADIENT, 0, 10)</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#getGroup()">getGroup()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">Returns the parent Group of the Atom.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#getGroup()">getGroup()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getGroup(int)">getGroup(int)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>use getAtomGroup or getSeqResGroup instead</i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getGroup(int)">getGroup(int)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>use getAtomGroup or getSeqResGroup instead</i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/StructureAtom.html#getGroup()">getGroup()</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/StructureAtom.html" title="class in org.biojava3.protmod.structure">StructureAtom</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getGroup_PDB()">getGroup_PDB()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Bond.html#getGroupA()">getGroupA()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Bond.html" title="class in org.biojava.bio.structure">Bond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Bond.html#getGroupB()">getGroupB()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Bond.html" title="class in org.biojava.bio.structure">Bond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getGroupByPDB(java.lang.String)">getGroupByPDB(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>replaced by <a href="./org/biojava/bio/structure/Chain.html#getGroupByPDB(org.biojava.bio.structure.ResidueNumber)"><code>Chain.getGroupByPDB(ResidueNumber)</code></a></i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getGroupByPDB(org.biojava.bio.structure.ResidueNumber)">getGroupByPDB(ResidueNumber)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Get a group by its PDB residue numbering. if the PDB residue number is not know,
 throws a StructureException.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getGroupByPDB(java.lang.String)">getGroupByPDB(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">get a group by its PDB residue numbering. if the PDB residue number is not know,
 throws a StructureException.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getGroupByPDB(org.biojava.bio.structure.ResidueNumber)">getGroupByPDB(ResidueNumber)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Get a group by its PDB residue numbering. if the PDB residue number is not know,
 throws a StructureException.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#getGroupByPDBResidueNumber(org.biojava.bio.structure.Structure, org.biojava.bio.structure.ResidueNumber)">getGroupByPDBResidueNumber(Structure, ResidueNumber)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>
<div class="block">Get a group represented by a ResidueNumber.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompGroupFactory.html#getGroupFromChemCompDictionary(java.lang.String)">getGroupFromChemCompDictionary(String)</a></span> - Static method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompGroupFactory.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompGroupFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/convertor/Convert2Charge.html#getGrouping()">getGrouping()</a></span> - Method in class org.biojava3.aaproperties.profeat.convertor.<a href="./org/biojava3/aaproperties/profeat/convertor/Convert2Charge.html" title="class in org.biojava3.aaproperties.profeat.convertor">Convert2Charge</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/convertor/Convert2Hydrophobicity.html#getGrouping()">getGrouping()</a></span> - Method in class org.biojava3.aaproperties.profeat.convertor.<a href="./org/biojava3/aaproperties/profeat/convertor/Convert2Hydrophobicity.html" title="class in org.biojava3.aaproperties.profeat.convertor">Convert2Hydrophobicity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/convertor/Convert2NormalizedVanDerWaalsVolume.html#getGrouping()">getGrouping()</a></span> - Method in class org.biojava3.aaproperties.profeat.convertor.<a href="./org/biojava3/aaproperties/profeat/convertor/Convert2NormalizedVanDerWaalsVolume.html" title="class in org.biojava3.aaproperties.profeat.convertor">Convert2NormalizedVanDerWaalsVolume</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/convertor/Convert2Polarity.html#getGrouping()">getGrouping()</a></span> - Method in class org.biojava3.aaproperties.profeat.convertor.<a href="./org/biojava3/aaproperties/profeat/convertor/Convert2Polarity.html" title="class in org.biojava3.aaproperties.profeat.convertor">Convert2Polarity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/convertor/Convert2Polarizability.html#getGrouping()">getGrouping()</a></span> - Method in class org.biojava3.aaproperties.profeat.convertor.<a href="./org/biojava3/aaproperties/profeat/convertor/Convert2Polarizability.html" title="class in org.biojava3.aaproperties.profeat.convertor">Convert2Polarizability</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/convertor/Convert2SecondaryStructure.html#getGrouping()">getGrouping()</a></span> - Method in class org.biojava3.aaproperties.profeat.convertor.<a href="./org/biojava3/aaproperties/profeat/convertor/Convert2SecondaryStructure.html" title="class in org.biojava3.aaproperties.profeat.convertor">Convert2SecondaryStructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/convertor/Convert2SolventAccessibility.html#getGrouping()">getGrouping()</a></span> - Method in class org.biojava3.aaproperties.profeat.convertor.<a href="./org/biojava3/aaproperties/profeat/convertor/Convert2SolventAccessibility.html" title="class in org.biojava3.aaproperties.profeat.convertor">Convert2SolventAccessibility</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/convertor/Convertor.html#getGrouping()">getGrouping()</a></span> - Method in class org.biojava3.aaproperties.profeat.convertor.<a href="./org/biojava3/aaproperties/profeat/convertor/Convertor.html" title="class in org.biojava3.aaproperties.profeat.convertor">Convertor</a></dt>
<dd>
<div class="block">Returns the groupings of the attribute</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getGroups(java.lang.String)">getGroups(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>use getAtomGroups or getSeqResGroups instead</i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getGroups()">getGroups()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>use getAtomGroups or getSeqResGroups instead</i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getGroups(java.lang.String)">getGroups(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>use getAtomGroups instead</i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getGroups()">getGroups()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>use getAtomGroups instead</i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Site.html#getGroups()">getGroups()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Site.html" title="class in org.biojava.bio.structure">Site</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ModifiedCompound.html#getGroups()">getGroups()</a></span> - Method in interface org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ModifiedCompound.html" title="interface in org.biojava3.protmod.structure">ModifiedCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ModifiedCompound.html#getGroups(boolean)">getGroups(boolean)</a></span> - Method in interface org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ModifiedCompound.html" title="interface in org.biojava3.protmod.structure">ModifiedCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ModifiedCompoundImpl.html#getGroups()">getGroups()</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ModifiedCompoundImpl.html" title="class in org.biojava3.protmod.structure">ModifiedCompoundImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ModifiedCompoundImpl.html#getGroups(boolean)">getGroups(boolean)</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ModifiedCompoundImpl.html" title="class in org.biojava3.protmod.structure">ModifiedCompoundImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getGroupsByPDB(java.lang.String, java.lang.String)">getGroupsByPDB(String, String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>replaced by <a href="./org/biojava/bio/structure/Chain.html#getGroupsByPDB(org.biojava.bio.structure.ResidueNumber, org.biojava.bio.structure.ResidueNumber)"><code>Chain.getGroupsByPDB(ResidueNumber, ResidueNumber)</code></a></i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getGroupsByPDB(org.biojava.bio.structure.ResidueNumber, org.biojava.bio.structure.ResidueNumber)">getGroupsByPDB(ResidueNumber, ResidueNumber)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Get all groups that are located between two PDB residue numbers.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getGroupsByPDB(java.lang.String, java.lang.String, boolean)">getGroupsByPDB(String, String, boolean)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>replaced by #<a href="./org/biojava/bio/structure/Chain.html#getGroupsByPDB(org.biojava.bio.structure.ResidueNumber, org.biojava.bio.structure.ResidueNumber, boolean)"><code>Chain.getGroupsByPDB(ResidueNumber, ResidueNumber, boolean)</code></a></i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getGroupsByPDB(org.biojava.bio.structure.ResidueNumber, org.biojava.bio.structure.ResidueNumber, boolean)">getGroupsByPDB(ResidueNumber, ResidueNumber, boolean)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Get all groups that are located between two PDB residue numbers.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getGroupsByPDB(java.lang.String, java.lang.String, boolean)">getGroupsByPDB(String, String, boolean)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Get all groups that are located between two PDB residue numbers.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getGroupsByPDB(org.biojava.bio.structure.ResidueNumber, org.biojava.bio.structure.ResidueNumber, boolean)">getGroupsByPDB(ResidueNumber, ResidueNumber, boolean)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getGroupsByPDB(java.lang.String, java.lang.String)">getGroupsByPDB(String, String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Get all groups that are located between two PDB residue numbers.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getGroupsByPDB(org.biojava.bio.structure.ResidueNumber, org.biojava.bio.structure.ResidueNumber)">getGroupsByPDB(ResidueNumber, ResidueNumber)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Get all groups that are located between two PDB residue numbers.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#getGroupsWithinShell(org.biojava.bio.structure.Structure, org.biojava.bio.structure.Group, double, boolean)">getGroupsWithinShell(Structure, Group, double, boolean)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/QRDecomposition.html#getH()">getH()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/QRDecomposition.html" title="class in org.biojava.bio.structure.jama">QRDecomposition</a></dt>
<dd>
<div class="block">Return the Householder vectors</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html#getH()">getH()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html" title="class in org.biojava.bio.structure.secstruc">SecStrucGroup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getHeader()">getHeader()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Returns the Header ( a Compound object) for this chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getHeader()">getHeader()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Returns the Header ( a Compound object) for this chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#getHeader()">getHeader()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>use getPDBHeader instead</i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#getHeader()">getHeader()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">get Header data.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/GenericFastaHeaderFormat.html#getHeader(S)">getHeader(S)</a></span> - Method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/GenericFastaHeaderFormat.html" title="class in org.biojava3.core.sequence.io">GenericFastaHeaderFormat</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/template/FastaHeaderFormatInterface.html#getHeader(S)">getHeader(S)</a></span> - Method in interface org.biojava3.core.sequence.io.template.<a href="./org/biojava3/core/sequence/io/template/FastaHeaderFormatInterface.html" title="interface in org.biojava3.core.sequence.io.template">FastaHeaderFormatInterface</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getHeaderVars()">getHeaderVars()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/MenuCreator.html#getHelpMenuItem()">getHelpMenuItem()</a></span> - Static method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/MenuCreator.html" title="class in org.biojava.bio.structure.align.gui">MenuCreator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/EntityPolySeq.html#getHetero()">getHetero()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/EntityPolySeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">EntityPolySeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#getHetero()">getHetero()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#getHetGroups()">getHetGroups()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#getHetGroups()">getHetGroups()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#getHillOrder()">getHillOrder()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns the Hill Order of this Element.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/query/BlastXMLQuery.html#getHitsQueryDef(double)">getHitsQueryDef(double)</a></span> - Method in class org.biojava3.genome.query.<a href="./org/biojava3/genome/query/BlastXMLQuery.html" title="class in org.biojava3.genome.query">BlastXMLQuery</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/HSVColorSpace.html#getHSVColorSpace()">getHSVColorSpace()</a></span> - Static method in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/HSVColorSpace.html" title="class in org.biojava.bio.structure.gui.util.color">HSVColorSpace</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#getHyd()">getHyd()</a></span> - Static method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#getHydmax()">getHydmax()</a></span> - Static method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#getHydmin()">getHydmin()</a></span> - Static method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPDistanceMatrix.html#getIclose()">getIclose()</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPDistanceMatrix.html" title="class in org.biojava.bio.structure.domain.pdp">PDPDistanceMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFP.html#getId()">getId()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFP.html" title="class in org.biojava.bio.structure.align.model">AFP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getId()">getId()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#getId()">getId()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>
<div class="block">Get the Hibernate database ID.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getId()">getId()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Get the ID used by Hibernate.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getId()">getId()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Get the ID used by Hibernate.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getId()">getId()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>
<div class="block">get the ID used by Hibernate</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#getId()">getId()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Get the ID used by Hibernate.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Domain.html#getId()">getId()</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Domain.html" title="class in org.biojava.bio.structure.domain.pdp">Domain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#getId()">getId()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">the Hibernate database ID</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getId()">getId()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getId()">getId()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBremark.html#getId()">getId()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBremark.html" title="class in org.biojava.bio.structure.io.mmcif.model">DatabasePDBremark</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html#getId()">getId()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model">Entity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html#getId()">getId()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructAsym</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html#getId()">getId()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRef</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#getId()">getId()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>
<div class="block">Get the ID used by Hibernate.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#getId()">getId()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">get the ID used by Hibernate</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#getId()">getId()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">get the ID used by Hibernate</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/AccessionID.html#getID()">getID()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/AccessionID.html" title="class in org.biojava3.core.sequence">AccessionID</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/DBReferenceInfo.html#getId()">getId()</a></span> - Method in class org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/DBReferenceInfo.html" title="class in org.biojava3.core.sequence.features">DBReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/IUPACParser.IUPACTable.html#getId()">getId()</a></span> - Method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/IUPACParser.IUPACTable.html" title="class in org.biojava3.core.sequence.io">IUPACParser.IUPACTable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/TaxonomyID.html#getID()">getID()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/TaxonomyID.html" title="class in org.biojava3.core.sequence">TaxonomyID</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/data/sequence/FastaSequence.html#getId()">getId()</a></span> - Method in class org.biojava3.data.sequence.<a href="./org/biojava3/data/sequence/FastaSequence.html" title="class in org.biojava3.data.sequence">FastaSequence</a></dt>
<dd>
<div class="block">Gets the value of id</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModification.html#getId()">getId()</a></span> - Method in interface org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModification.html" title="interface in org.biojava3.protmod">ProteinModification</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationImpl.html#getId()">getId()</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationImpl.html" title="class in org.biojava3.protmod">ProteinModificationImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#getIdbnsBegin()">getIdbnsBegin()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Insertion code of initial residue of the segment, if PDB is the
    reference.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#getIdbnsEnd()">getIdbnsEnd()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Insertion code of the ending
    residue of the segment, if PDB is
    the reference.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#getIdCode()">getIdCode()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">get the idCode for this entry</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#getIdCode()">getIdCode()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>
<div class="block">The PDB code for this protein structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ProteinModificationIdentifier.html#getIdentifiedModifiedCompound()">getIdentifiedModifiedCompound()</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ProteinModificationIdentifier.html" title="class in org.biojava3.protmod.structure">ProteinModificationIdentifier</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getIdentity()">getIdentity()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">The percent of residues that are sequence-identical in the alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/util/IOUtils.html#getIDFormat(java.util.List)">getIDFormat(List&lt;S&gt;)</a></span> - Static method in class org.biojava3.core.sequence.io.util.<a href="./org/biojava3/core/sequence/io/util/IOUtils.html" title="class in org.biojava3.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">Creates format String for accession IDs</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#getIdx1()">getIdx1()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>
<div class="block">the positions of the structure equivalent positions in atom set 1</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/SequenceDisplay.html#getIdx1()">getIdx1()</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/SequenceDisplay.html" title="class in org.biojava.bio.structure.gui">SequenceDisplay</a></dt>
<dd>
<div class="block">get the identical position in the alignment</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#getIdx2()">getIdx2()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>
<div class="block">the positions of the structure equivalent atoms in atom set 2</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/SequenceDisplay.html#getIdx2()">getIdx2()</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/SequenceDisplay.html" title="class in org.biojava.bio.structure.gui">SequenceDisplay</a></dt>
<dd>
<div class="block">get the identical position in the alignment</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/JointFragments.html#getIdxlist()">getIdxlist()</a></span> - Method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/JointFragments.html" title="class in org.biojava.bio.structure.align.helper">JointFragments</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/EigenvalueDecomposition.html#getImagEigenvalues()">getImagEigenvalues()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/EigenvalueDecomposition.html" title="class in org.biojava.bio.structure.jama">EigenvalueDecomposition</a></dt>
<dd>
<div class="block">Return the imaginary parts of the eigenvalues</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/GapArray.html#getIndex()">getIndex()</a></span> - Method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/GapArray.html" title="class in org.biojava.bio.structure.align.helper">GapArray</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/GuideTree.Node.html#getIndex(javax.swing.tree.TreeNode)">getIndex(TreeNode)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/GuideTree.Node.html" title="class in org.biojava3.alignment">GuideTree.Node</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleSequencePair.html#getIndexInQueryAt(int)">getIndexInQueryAt(int)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleSequencePair.html" title="class in org.biojava3.alignment">SimpleSequencePair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/SequencePair.html#getIndexInQueryAt(int)">getIndexInQueryAt(int)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/SequencePair.html" title="interface in org.biojava3.alignment.template">SequencePair</a></dt>
<dd>
<div class="block">Returns the query index corresponding to a given alignment column.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleSequencePair.html#getIndexInQueryForTargetAt(int)">getIndexInQueryForTargetAt(int)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleSequencePair.html" title="class in org.biojava3.alignment">SimpleSequencePair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/SequencePair.html#getIndexInQueryForTargetAt(int)">getIndexInQueryForTargetAt(int)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/SequencePair.html" title="interface in org.biojava3.alignment.template">SequencePair</a></dt>
<dd>
<div class="block">Returns the query index corresponding to a given target index.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleSequencePair.html#getIndexInTargetAt(int)">getIndexInTargetAt(int)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleSequencePair.html" title="class in org.biojava3.alignment">SimpleSequencePair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/SequencePair.html#getIndexInTargetAt(int)">getIndexInTargetAt(int)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/SequencePair.html" title="interface in org.biojava3.alignment.template">SequencePair</a></dt>
<dd>
<div class="block">Returns the target index corresponding to a given alignment column.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleSequencePair.html#getIndexInTargetForQueryAt(int)">getIndexInTargetForQueryAt(int)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleSequencePair.html" title="class in org.biojava3.alignment">SimpleSequencePair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/SequencePair.html#getIndexInTargetForQueryAt(int)">getIndexInTargetForQueryAt(int)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/SequencePair.html" title="interface in org.biojava3.alignment.template">SequencePair</a></dt>
<dd>
<div class="block">Returns the target index corresponding to a given query index.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleAlignedSequence.html#getIndexOf(C)">getIndexOf(C)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleAlignedSequence.html" title="class in org.biojava3.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleProfile.html#getIndexOf(C)">getIndexOf(C)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleProfile.html" title="class in org.biojava3.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/Profile.html#getIndexOf(C)">getIndexOf(C)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template">Profile</a></dt>
<dd>
<div class="block">Searches for the given <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template"><code>Compound</code></a> within this alignment profile.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html#getIndexOf(C)">getIndexOf(C)</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava3.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/StringProxySequenceReader.html#getIndexOf(C)">getIndexOf(C)</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html#getIndexOf(C)">getIndexOf(C)</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/ArrayListSequenceReader.html#getIndexOf(C)">getIndexOf(C)</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava3.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/BitSequenceReader.html#getIndexOf(C)">getIndexOf(C)</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava3.core.sequence.storage">BitSequenceReader</a></dt>
<dd>
<div class="block">Returns the first occurrence of the given compound in this store; performs
 a linear search</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/JoiningSequenceReader.html#getIndexOf(C)">getIndexOf(C)</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava3.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html#getIndexOf(C)">getIndexOf(C)</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava3.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Returns 1 if the given compound is equal to the one given during
 construction; otherwise will return -1.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#getIndexOf(C)">getIndexOf(C)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/Sequence.html#getIndexOf(C)">getIndexOf(C)</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a></dt>
<dd>
<div class="block">Scans through the Sequence looking for the first occurrence of the given
 compound</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceProxyView.html#getIndexOf(C)">getIndexOf(C)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceProxyView.html" title="class in org.biojava3.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/views/ComplementSequenceView.html#getIndexOf(C)">getIndexOf(C)</a></span> - Method in class org.biojava3.core.sequence.views.<a href="./org/biojava3/core/sequence/views/ComplementSequenceView.html" title="class in org.biojava3.core.sequence.views">ComplementSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/views/RnaSequenceView.html#getIndexOf(org.biojava3.core.sequence.compound.NucleotideCompound)">getIndexOf(NucleotideCompound)</a></span> - Method in class org.biojava3.core.sequence.views.<a href="./org/biojava3/core/sequence/views/RnaSequenceView.html" title="class in org.biojava3.core.sequence.views">RnaSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ModificationLinkage.html#getIndexOfComponent1()">getIndexOfComponent1()</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ModificationLinkage.html" title="class in org.biojava3.protmod">ModificationLinkage</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ModificationLinkage.html#getIndexOfComponent2()">getIndexOfComponent2()</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ModificationLinkage.html" title="class in org.biojava3.protmod">ModificationLinkage</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#getIndexToCompoundsLookup()">getIndexToCompoundsLookup()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava3.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>
<div class="block">Returns a list of compounds the index position of which is used
 to translate from the byte representation into a compound.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleProfile.html#getIndicesAt(int)">getIndicesAt(int)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleProfile.html" title="class in org.biojava3.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/Profile.html#getIndicesAt(int)">getIndicesAt(int)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns the indices in the original <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a>s corresponding to the given column.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#getInitialK()">getInitialK()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Author.html#getInitials()">getInitials()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Author.html" title="class in org.biojava.bio.structure">Author</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/HTTPConnectionTools.html#getInputStream(java.net.URL, int)">getInputStream(URL, int)</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/HTTPConnectionTools.html" title="class in org.biojava.bio.structure.align.util">HTTPConnectionTools</a></dt>
<dd>
<div class="block">connect to DAS server and return result as an InputStream.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/HTTPConnectionTools.html#getInputStream(java.net.URL)">getInputStream(URL)</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/HTTPConnectionTools.html" title="class in org.biojava.bio.structure.align.util">HTTPConnectionTools</a></dt>
<dd>
<div class="block">connect to DAS server and return result as an InputStream.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/HTTPConnectionTools.html#getInputStream(java.net.URL, boolean, int)">getInputStream(URL, boolean, int)</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/HTTPConnectionTools.html" title="class in org.biojava.bio.structure.align.util">HTTPConnectionTools</a></dt>
<dd>
<div class="block">open a URL and return an InputStream to it
  if acceptGzipEncoding == true, use GZIPEncoding to
  compress communication</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/util/ClasspathResource.html#getInputStream()">getInputStream()</a></span> - Method in class org.biojava3.core.sequence.io.util.<a href="./org/biojava3/core/sequence/io/util/ClasspathResource.html" title="class in org.biojava3.core.sequence.io.util">ClasspathResource</a></dt>
<dd>
<div class="block">Returns the InputStream instance of this classpath resource</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/FlatFileCache.html#getInputStream(java.lang.String)">getInputStream(String)</a></span> - Static method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/FlatFileCache.html" title="class in org.biojava3.core.util">FlatFileCache</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/InputStreamProvider.html#getInputStream(java.lang.String)">getInputStream(String)</a></span> - Method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/InputStreamProvider.html" title="class in org.biojava3.core.util">InputStreamProvider</a></dt>
<dd>
<div class="block">get an InputStream for this file</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/InputStreamProvider.html#getInputStream(java.net.URL)">getInputStream(URL)</a></span> - Method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/InputStreamProvider.html" title="class in org.biojava3.core.util">InputStreamProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/InputStreamProvider.html#getInputStream(java.io.File)">getInputStream(File)</a></span> - Method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/InputStreamProvider.html" title="class in org.biojava3.core.util">InputStreamProvider</a></dt>
<dd>
<div class="block">get an InputStream for the file</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/aaindex/DefaultAAIndexProvider.html#getInputStreamToAAindexFile()">getInputStreamToAAindexFile()</a></span> - Method in class org.biojava3.alignment.aaindex.<a href="./org/biojava3/alignment/aaindex/DefaultAAIndexProvider.html" title="class in org.biojava3.alignment.aaindex">DefaultAAIndexProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ResidueNumber.html#getInsCode()">getInsCode()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ResidueNumber.html" title="class in org.biojava.bio.structure">ResidueNumber</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/StructureGroup.html#getInsCode()">getInsCode()</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/StructureGroup.html" title="class in org.biojava3.protmod.structure">StructureGroup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SSBond.html#getInsCode1()">getInsCode1()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SSBond.html" title="class in org.biojava.bio.structure">SSBond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SSBond.html#getInsCode2()">getInsCode2()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SSBond.html" title="class in org.biojava.bio.structure">SSBond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#getInsertBegin()">getInsertBegin()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Initial insertion code of the PDB sequence segment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#getInsertEnd()">getInsertEnd()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Ending insertion code of the PDB sequence segment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/IPeptideProperties.html#getInstabilityIndex(org.biojava3.core.sequence.ProteinSequence)">getInstabilityIndex(ProteinSequence)</a></span> - Method in interface org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/IPeptideProperties.html" title="interface in org.biojava3.aaproperties">IPeptideProperties</a></dt>
<dd>
<div class="block">Returns the instability index of sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/PeptideProperties.html#getInstabilityIndex(java.lang.String)">getInstabilityIndex(String)</a></span> - Static method in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/PeptideProperties.html" title="class in org.biojava3.aaproperties">PeptideProperties</a></dt>
<dd>
<div class="block">An adaptor method to return the instability index of sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/PeptidePropertiesImpl.html#getInstabilityIndex(org.biojava3.core.sequence.ProteinSequence)">getInstabilityIndex(ProteinSequence)</a></span> - Method in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/PeptidePropertiesImpl.html" title="class in org.biojava3.aaproperties">PeptidePropertiesImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/AlignmentGui.html#getInstance()">getInstance()</a></span> - Static method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/AlignmentGui.html" title="class in org.biojava.bio.structure.align.gui">AlignmentGui</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/HSVColorSpace.html#getInstance(int)">getInstance(int)</a></span> - Static method in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/HSVColorSpace.html" title="class in org.biojava.bio.structure.gui.util.color">HSVColorSpace</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/ScopInstallationInstance.html#getInstance()">getInstance()</a></span> - Static method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/ScopInstallationInstance.html" title="class in org.biojava.bio.structure.gui.util">ScopInstallationInstance</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/IUPACParser.html#getInstance()">getInstance()</a></span> - Static method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/IUPACParser.html" title="class in org.biojava3.core.sequence.io">IUPACParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/FlatFileCache.html#getInstance()">getInstance()</a></span> - Static method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/FlatFileCache.html" title="class in org.biojava3.core.util">FlatFileCache</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/AlignmentGui.html#getInstanceNoVisibilityChange()">getInstanceNoVisibilityChange()</a></span> - Static method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/AlignmentGui.html" title="class in org.biojava.bio.structure.align.gui">AlignmentGui</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/ColorUtils.html#getIntermediate(java.awt.Color, java.awt.Color, int, int)">getIntermediate(Color, Color, int, int)</a></span> - Static method in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/ColorUtils.html" title="class in org.biojava.bio.structure.gui.util.color">ColorUtils</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html#getInterpolator()">getInterpolator()</a></span> - Method in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html" title="class in org.biojava.bio.structure.gui.util.color">GradientMapper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/GeneSequence.html#getIntronSequences()">getIntronSequences()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/GeneSequence.html" title="class in org.biojava3.core.sequence">GeneSequence</a></dt>
<dd>
<div class="block">Get the introns as an ArrayList</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleAlignedSequence.html#getInverse()">getInverse()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleAlignedSequence.html" title="class in org.biojava3.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html#getInverse()">getInverse()</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava3.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/StringProxySequenceReader.html#getInverse()">getInverse()</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html#getInverse()">getInverse()</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/RNASequence.html#getInverse()">getInverse()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/RNASequence.html" title="class in org.biojava3.core.sequence">RNASequence</a></dt>
<dd>
<div class="block">Get the inverse view of the sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/ArrayListSequenceReader.html#getInverse()">getInverse()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava3.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/BitSequenceReader.html#getInverse()">getInverse()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava3.core.sequence.storage">BitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/JoiningSequenceReader.html#getInverse()">getInverse()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava3.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html#getInverse()">getInverse()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava3.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#getInverse()">getInverse()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/Sequence.html#getInverse()">getInverse()</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a></dt>
<dd>
<div class="block">Does the <em>right thing</em> to get the inverse of the current
 Sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceProxyView.html#getInverse()">getInverse()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceProxyView.html" title="class in org.biojava3.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getIoTime()">getIoTime()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html#getIoTime()">getIoTime()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentResult</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/IPeptideProperties.html#getIsoelectricPoint(org.biojava3.core.sequence.ProteinSequence)">getIsoelectricPoint(ProteinSequence)</a></span> - Method in interface org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/IPeptideProperties.html" title="interface in org.biojava3.aaproperties">IPeptideProperties</a></dt>
<dd>
<div class="block">Returns the isoelectric point of sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/PeptideProperties.html#getIsoelectricPoint(java.lang.String)">getIsoelectricPoint(String)</a></span> - Static method in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/PeptideProperties.html" title="class in org.biojava3.aaproperties">PeptideProperties</a></dt>
<dd>
<div class="block">An adaptor method to return the isoelectric point of sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/PeptidePropertiesImpl.html#getIsoelectricPoint(org.biojava3.core.sequence.ProteinSequence)">getIsoelectricPoint(ProteinSequence)</a></span> - Method in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/PeptidePropertiesImpl.html" title="class in org.biojava3.aaproperties">PeptidePropertiesImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/ElementTable.html#getIsotope(java.lang.String)">getIsotope(String)</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/ElementTable.html" title="class in org.biojava3.aaproperties.xml">ElementTable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/AminoAcidComposition.html#getIsotopeList()">getIsotopeList()</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/AminoAcidComposition.html" title="class in org.biojava3.aaproperties.xml">AminoAcidComposition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/Element.html#getIsotopes()">getIsotopes()</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/Element.html" title="class in org.biojava3.aaproperties.xml">Element</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/BigSqrt.html#getIterations()">getIterations()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/BigSqrt.html" title="class in org.biojava.bio.structure.secstruc">BigSqrt</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPDistanceMatrix.html#getJclose()">getJclose()</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPDistanceMatrix.html" title="class in org.biojava.bio.structure.domain.pdp">PDPDistanceMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html#getJmolPanel()">getJmolPanel()</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html" title="class in org.biojava.bio.structure.align.gui.jmol">StructureAlignmentJmol</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html#getJmolScript4Block(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], int)">getJmolScript4Block(AFPChain, Atom[], Atom[], int)</a></span> - Static method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html" title="class in org.biojava.bio.structure.align.gui.jmol">StructureAlignmentJmol</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html#getJmolString(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">getJmolString(AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html" title="class in org.biojava.bio.structure.align.gui.jmol">StructureAlignmentJmol</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#getJoinRMSCutoff()">getJoinRMSCutoff()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#getJournalArticle()">getJournalArticle()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">get the associated publication as defined by the JRNL records in a PDB
 file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#getJournalArticle()">getJournalArticle()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">get the associated publication as defined by the JRNL records in a PDB
 file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#getJournalName()">getJournalName()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/FeaturesKeyWordInterface.html#getKeyWords()">getKeyWords()</a></span> - Method in interface org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/FeaturesKeyWordInterface.html" title="interface in org.biojava3.core.sequence.features">FeaturesKeyWordInterface</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html#getKeyWords()">getKeyWords()</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>
<div class="block">Pull uniprot key words which is a mixed bag of words associated with this sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModification.html#getKeywords()">getKeywords()</a></span> - Method in interface org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModification.html" title="interface in org.biojava3.protmod">ProteinModification</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationImpl.html#getKeywords()">getKeywords()</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationImpl.html" title="class in org.biojava3.protmod">ProteinModificationImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/CholeskyDecomposition.html#getL()">getL()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/CholeskyDecomposition.html" title="class in org.biojava.bio.structure.jama">CholeskyDecomposition</a></dt>
<dd>
<div class="block">Return triangular factor.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/LUDecomposition.html#getL()">getL()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/LUDecomposition.html" title="class in org.biojava.bio.structure.jama">LUDecomposition</a></dt>
<dd>
<div class="block">Return lower triangular factor</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getLabel_alt_id()">getLabel_alt_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getLabel_asym_id()">getLabel_asym_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getLabel_atom_id()">getLabel_atom_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getLabel_comp_id()">getLabel_comp_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getLabel_entity_id()">getLabel_entity_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getLabel_seq_id()">getLabel_seq_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ModificationLinkage.html#getLabelOfAtomOnComponent1()">getLabelOfAtomOnComponent1()</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ModificationLinkage.html" title="class in org.biojava3.protmod">ModificationLinkage</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ModificationLinkage.html#getLabelOfAtomOnComponent2()">getLabelOfAtomOnComponent2()</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ModificationLinkage.html" title="class in org.biojava3.protmod">ModificationLinkage</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleAlignedSequence.html#getLastIndexOf(C)">getLastIndexOf(C)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleAlignedSequence.html" title="class in org.biojava3.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleProfile.html#getLastIndexOf(C)">getLastIndexOf(C)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleProfile.html" title="class in org.biojava3.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/Profile.html#getLastIndexOf(C)">getLastIndexOf(C)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template">Profile</a></dt>
<dd>
<div class="block">Searches for the given <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template"><code>Compound</code></a> within this alignment profile.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html#getLastIndexOf(C)">getLastIndexOf(C)</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava3.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/StringProxySequenceReader.html#getLastIndexOf(C)">getLastIndexOf(C)</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html#getLastIndexOf(C)">getLastIndexOf(C)</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/ArrayListSequenceReader.html#getLastIndexOf(C)">getLastIndexOf(C)</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava3.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/BitSequenceReader.html#getLastIndexOf(C)">getLastIndexOf(C)</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava3.core.sequence.storage">BitSequenceReader</a></dt>
<dd>
<div class="block">Returns the last occurrence of the given compound in this store; performs
 a linear search</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/JoiningSequenceReader.html#getLastIndexOf(C)">getLastIndexOf(C)</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava3.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html#getLastIndexOf(C)">getLastIndexOf(C)</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava3.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Returns the length of the Sequence if the given compound was equal to
 the one given during construction.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#getLastIndexOf(C)">getLastIndexOf(C)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/Sequence.html#getLastIndexOf(C)">getLastIndexOf(C)</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a></dt>
<dd>
<div class="block">Scans through the Sequence looking for the last occurrence of the given
 compound</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceProxyView.html#getLastIndexOf(C)">getLastIndexOf(C)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceProxyView.html" title="class in org.biojava3.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/views/ComplementSequenceView.html#getLastIndexOf(C)">getLastIndexOf(C)</a></span> - Method in class org.biojava3.core.sequence.views.<a href="./org/biojava3/core/sequence/views/ComplementSequenceView.html" title="class in org.biojava3.core.sequence.views">ComplementSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/views/RnaSequenceView.html#getLastIndexOf(org.biojava3.core.sequence.compound.NucleotideCompound)">getLastIndexOf(NucleotideCompound)</a></span> - Method in class org.biojava3.core.sequence.views.<a href="./org/biojava3/core/sequence/views/RnaSequenceView.html" title="class in org.biojava3.core.sequence.views">RnaSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#getLcmp()">getLcmp()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AFPChainCoordManager.html#getLegendPosition(int, int)">getLegendPosition(int, int)</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AFPChainCoordManager.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AFPChainCoordManager</a></dt>
<dd>
<div class="block">provide the coordinates for where to draw the legend for line X and if it is chain 1 or 2</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#getLength()">getLength()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Bond.html#getLength()">getLength()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Bond.html" title="class in org.biojava.bio.structure">Bond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getLength()">getLength()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>please use getAtomLength or getLengthSeqRes instead</i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getLength()">getLength()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>use getAtomLength instead</i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleAlignedSequence.html#getLength()">getLength()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleAlignedSequence.html" title="class in org.biojava3.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleProfile.html#getLength()">getLength()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleProfile.html" title="class in org.biojava3.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/Profile.html#getLength()">getLength()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns the number of columns in the alignment profile.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/CDSSequence.html#getLength()">getLength()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/CDSSequence.html" title="class in org.biojava3.core.sequence">CDSSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/ExonSequence.html#getLength()">getLength()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/ExonSequence.html" title="class in org.biojava3.core.sequence">ExonSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/GeneSequence.html#getLength()">getLength()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/GeneSequence.html" title="class in org.biojava3.core.sequence">GeneSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/IntronSequence.html#getLength()">getLength()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/IntronSequence.html" title="class in org.biojava3.core.sequence">IntronSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html#getLength()">getLength()</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava3.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/StringProxySequenceReader.html#getLength()">getLength()</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html#getLength()">getLength()</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>
<div class="block">The sequence length</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html#getLength()">getLength()</a></span> - Method in class org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava3.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/Location.html#getLength()">getLength()</a></span> - Method in interface org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/Location.html" title="interface in org.biojava3.core.sequence.location.template">Location</a></dt>
<dd>
<div class="block">Returns the length of the outer bounds of this location</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/MultipleSequenceAlignment.html#getLength()">getLength()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/MultipleSequenceAlignment.html" title="class in org.biojava3.core.sequence">MultipleSequenceAlignment</a></dt>
<dd>
<div class="block">Get the length of the MSA where it is assumed that
 all sequence position</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/StartCodonSequence.html#getLength()">getLength()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/StartCodonSequence.html" title="class in org.biojava3.core.sequence">StartCodonSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/StopCodonSequence.html#getLength()">getLength()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/StopCodonSequence.html" title="class in org.biojava3.core.sequence">StopCodonSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/ArrayListSequenceReader.html#getLength()">getLength()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava3.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#getLength()">getLength()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava3.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/BitSequenceReader.html#getLength()">getLength()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava3.core.sequence.storage">BitSequenceReader</a></dt>
<dd>
<div class="block">Returns the length of the sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/JoiningSequenceReader.html#getLength()">getLength()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava3.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html#getLength()">getLength()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava3.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Returns the length given during construction</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#getLength()">getLength()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/LightweightProfile.html#getLength()">getLength()</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/LightweightProfile.html" title="interface in org.biojava3.core.sequence.template">LightweightProfile</a></dt>
<dd>
<div class="block">Returns the number of columns in the alignment profile.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/Sequence.html#getLength()">getLength()</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a></dt>
<dd>
<div class="block">Returns the length of the Sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceProxyView.html#getLength()">getLength()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceProxyView.html" title="class in org.biojava3.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/TranscriptSequence.html#getLength()">getLength()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/TranscriptSequence.html" title="class in org.biojava3.core.sequence">TranscriptSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/views/WindowedSequence.html#getLength()">getLength()</a></span> - Method in class org.biojava3.core.sequence.views.<a href="./org/biojava3/core/sequence/views/WindowedSequence.html" title="class in org.biojava3.core.sequence.views">WindowedSequence</a></dt>
<dd>
<div class="block">Returns the size of the windowed sequence which is the length by the
 window size.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/data/sequence/FastaSequence.html#getLength()">getLength()</a></span> - Method in class org.biojava3.data.sequence.<a href="./org/biojava3/data/sequence/FastaSequence.html" title="class in org.biojava3.data.sequence">FastaSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html#getLength1()">getLength1()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentResult</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html#getLength2()">getLength2()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentResult</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getLengthAminos()">getLengthAminos()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>use getAtomGroups("amino").size() instead.</i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getLengthAminos()">getLengthAminos()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">returns the length of the AminoAcids in the ATOM records of this chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/FastaGeneWriter.html#getLineLength()">getLineLength()</a></span> - Method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/FastaGeneWriter.html" title="class in org.biojava3.core.sequence.io">FastaGeneWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/FastaWriter.html#getLineLength()">getLineLength()</a></span> - Method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/FastaWriter.html" title="class in org.biojava3.core.sequence.io">FastaWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ModificationCondition.html#getLinkages()">getLinkages()</a></span> - Method in interface org.biojava3.protmod.<a href="./org/biojava3/protmod/ModificationCondition.html" title="interface in org.biojava3.protmod">ModificationCondition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ModificationConditionImpl.html#getLinkages()">getLinkages()</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ModificationConditionImpl.html" title="class in org.biojava3.protmod">ModificationConditionImpl</a></dt>
<dd>
<div class="block">}</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/util/ClasspathResource.html#getList()">getList()</a></span> - Method in class org.biojava3.core.sequence.io.util.<a href="./org/biojava3/core/sequence/io/util/ClasspathResource.html" title="class in org.biojava3.core.sequence.io.util">ClasspathResource</a></dt>
<dd>
<div class="block">Returns this resource as a list of Strings</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/util/IOUtils.html#getList(java.io.BufferedReader)">getList(BufferedReader)</a></span> - Static method in class org.biojava3.core.sequence.io.util.<a href="./org/biojava3/core/sequence/io/util/IOUtils.html" title="class in org.biojava3.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">Returns the contents of a buffered reader as a list of strings</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/util/IOUtils.html#getList(java.io.InputStream)">getList(InputStream)</a></span> - Static method in class org.biojava3.core.sequence.io.util.<a href="./org/biojava3/core/sequence/io/util/IOUtils.html" title="class in org.biojava3.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">Delegates to <a href="./org/biojava3/core/sequence/io/util/IOUtils.html#getList(java.io.BufferedReader)"><code>IOUtils.getList(BufferedReader)</code></a> by wrapping the InputStream
 in a valid reader.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/util/IOUtils.html#getList(java.io.File)">getList(File)</a></span> - Static method in class org.biojava3.core.sequence.io.util.<a href="./org/biojava3/core/sequence/io/util/IOUtils.html" title="class in org.biojava3.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">Delegates to <a href="./org/biojava3/core/sequence/io/util/IOUtils.html#getList(java.io.InputStream)"><code>IOUtils.getList(InputStream)</code></a> by wrapping the File
 in a valid stream.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.GradientRenderer.html#getListCellRendererComponent(javax.swing.JList, java.lang.Object, int, boolean, boolean)">getListCellRendererComponent(JList, Object, int, boolean, boolean)</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.GradientRenderer.html" title="class in org.biojava.bio.structure.gui">ScaleableMatrixPanel.GradientRenderer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/MenuCreator.html#getLoadMenuItem()">getLoadMenuItem()</a></span> - Static method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/MenuCreator.html" title="class in org.biojava.bio.structure.align.gui">MenuCreator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/DownloadChemCompProvider.html#getLocalFileName(java.lang.String)">getLocalFileName(String)</a></span> - Static method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/DownloadChemCompProvider.html" title="class in org.biojava.bio.structure.io.mmcif">DownloadChemCompProvider</a></dt>
<dd>
<div class="block">Returns the file name that contains the definition for this <a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model"><code>ChemComp</code></a></div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleAlignedSequence.html#getLocationInAlignment()">getLocationInAlignment()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleAlignedSequence.html" title="class in org.biojava3.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AlignedSequence.html#getLocationInAlignment()">getLocationInAlignment()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AlignedSequence.html" title="interface in org.biojava3.alignment.template">AlignedSequence</a></dt>
<dd>
<div class="block">Returns the <a href="./org/biojava3/core/sequence/location/template/Location.html" title="interface in org.biojava3.core.sequence.location.template"><code>Location</code></a> of the original <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a> within an alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/AbstractFeature.html#getLocations()">getLocations()</a></span> - Method in class org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/AbstractFeature.html" title="class in org.biojava3.core.sequence.features">AbstractFeature</a></dt>
<dd>
<div class="block">A feature could be a single sequence position like a mutation or a post translational modification of an amino acid.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/FeatureInterface.html#getLocations()">getLocations()</a></span> - Method in interface org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/FeatureInterface.html" title="interface in org.biojava3.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">The location(s) of this feature where the location should contain a reference to parent and sequence etc.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/CodonCompound.html#getLongName()">getLongName()</a></span> - Method in class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/CodonCompound.html" title="class in org.biojava3.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompound.html#getLongName()">getLongName()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompound.html" title="class in org.biojava3.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/Compound.html#getLongName()">getLongName()</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/CaseInsensitiveCompound.html#getLongName()">getLongName()</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava3.core.sequence.transcription">CaseInsensitiveCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/Table.Codon.html#getLongName()">getLongName()</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/Table.Codon.html" title="class in org.biojava3.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getLs_d_res_high()">getLs_d_res_high()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getLs_d_res_low()">getLs_d_res_low()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getLs_number_parameters()">getLs_number_parameters()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getLs_number_reflns_all()">getLs_number_reflns_all()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getLs_number_reflns_obs()">getLs_number_reflns_obs()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getLs_number_reflns_R_free()">getLs_number_reflns_R_free()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getLs_number_restraints()">getLs_number_restraints()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getLs_percent_reflns_obs()">getLs_percent_reflns_obs()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getLs_percent_reflns_R_free()">getLs_percent_reflns_R_free()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getLs_R_factor_all()">getLs_R_factor_all()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getLs_R_factor_obs()">getLs_R_factor_obs()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getLs_R_factor_R_free()">getLs_R_factor_R_free()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getLs_R_factor_R_free_error()">getLs_R_factor_R_free_error()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getLs_R_factor_R_free_error_details()">getLs_R_factor_R_free_error_details()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getLs_R_factor_R_work()">getLs_R_factor_R_work()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getLs_redundancy_reflns_obs()">getLs_redundancy_reflns_obs()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getLs_wR_factor_R_free()">getLs_wR_factor_R_free()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getLs_wR_factor_R_work()">getLs_wR_factor_R_work()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFP.html#getM()">getM()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFP.html" title="class in org.biojava.bio.structure.align.model">AFP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html#getMandatoryArgs()">getMandatoryArgs()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce">AbstractUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/Element.html#getMass()">getMass()</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/Element.html" title="class in org.biojava3.aaproperties.xml">Element</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/Isotope.html#getMass()">getMass()</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/Isotope.html" title="class in org.biojava3.aaproperties.xml">Isotope</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/homology/BlastHomologyHits.html#getMatches(java.io.File, double)">getMatches(File, double)</a></span> - Static method in class org.biojava3.genome.homology.<a href="./org/biojava3/genome/homology/BlastHomologyHits.html" title="class in org.biojava3.genome.homology">BlastHomologyHits</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/SeqRes2AtomAligner.html#getMatchingAtomRes(org.biojava.bio.structure.Chain, java.util.List)">getMatchingAtomRes(Chain, List&lt;Chain&gt;)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/SeqRes2AtomAligner.html" title="class in org.biojava.bio.structure.io">SeqRes2AtomAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#getMatMatrix()">getMatMatrix()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>
<div class="block">Caution: this matrix is overwriten with very different data at several
 points in the alignment algorithm.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/aaindex/AAIndexFileParser.html#getMatrices()">getMatrices()</a></span> - Method in class org.biojava3.alignment.aaindex.<a href="./org/biojava3/alignment/aaindex/AAIndexFileParser.html" title="class in org.biojava3.alignment.aaindex">AAIndexFileParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/JMatrixPanel.html#getMatrix()">getMatrix()</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/JMatrixPanel.html" title="class in org.biojava.bio.structure.gui">JMatrixPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html#getMatrix()">getMatrix()</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html" title="class in org.biojava.bio.structure.gui">ScaleableMatrixPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#getMatrix(int, int, int, int)">getMatrix(int, int, int, int)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Get a submatrix.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#getMatrix(int[], int[])">getMatrix(int[], int[])</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Get a submatrix.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#getMatrix(int, int, int[])">getMatrix(int, int, int[])</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Get a submatrix.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#getMatrix(int[], int, int)">getMatrix(int[], int, int)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Get a submatrix.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/aaindex/AAIndexProvider.html#getMatrix(java.lang.String)">getMatrix(String)</a></span> - Method in interface org.biojava3.alignment.aaindex.<a href="./org/biojava3/alignment/aaindex/AAIndexProvider.html" title="interface in org.biojava3.alignment.aaindex">AAIndexProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/aaindex/DefaultAAIndexProvider.html#getMatrix(java.lang.String)">getMatrix(String)</a></span> - Method in class org.biojava3.alignment.aaindex.<a href="./org/biojava3/alignment/aaindex/DefaultAAIndexProvider.html" title="class in org.biojava3.alignment.aaindex">DefaultAAIndexProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html#getMatrix()">getMatrix()</a></span> - Method in class org.biojava3.alignment.aaindex.<a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava3.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleSubstitutionMatrix.html#getMatrix()">getMatrix()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleSubstitutionMatrix.html" title="class in org.biojava3.alignment">SimpleSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/SubstitutionMatrix.html#getMatrix()">getMatrix()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/SubstitutionMatrix.html" title="interface in org.biojava3.alignment.template">SubstitutionMatrix</a></dt>
<dd>
<div class="block">Returns entire matrix.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ScoreMatrix.html#getMatrix()">getMatrix()</a></span> - Method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ScoreMatrix.html" title="class in org.biojava3.phylo">ScoreMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html#getMatrixAsString()">getMatrixAsString()</a></span> - Method in class org.biojava3.alignment.aaindex.<a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava3.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleSubstitutionMatrix.html#getMatrixAsString()">getMatrixAsString()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleSubstitutionMatrix.html" title="class in org.biojava3.alignment">SimpleSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/SubstitutionMatrix.html#getMatrixAsString()">getMatrixAsString()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/SubstitutionMatrix.html" title="interface in org.biojava3.alignment.template">SubstitutionMatrix</a></dt>
<dd>
<div class="block">Returns this matrix as a formatted String with <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template"><code>Compound</code></a> labels along the axes.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SubstitutionMatrixHelper.html#getMatrixFromAAINDEX(java.lang.String)">getMatrixFromAAINDEX(String)</a></span> - Static method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SubstitutionMatrixHelper.html" title="class in org.biojava3.alignment">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns any matrix from the AAINDEX database file</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html#getMatrixPanel()">getMatrixPanel()</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html" title="class in org.biojava.bio.structure.gui">ScaleableMatrixPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html#getMax()">getMax()</a></span> - Method in class org.biojava3.alignment.aaindex.<a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava3.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/FuzzyPoint.html#getMax()">getMax()</a></span> - Method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/FuzzyPoint.html" title="class in org.biojava3.core.sequence.location">FuzzyPoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/LocationHelper.html#getMax(java.util.List)">getMax(List&lt;Location&gt;)</a></span> - Static method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/LocationHelper.html" title="class in org.biojava3.core.sequence.location">LocationHelper</a></dt>
<dd>
<div class="block">Scans through a list of locations to find the Location with the
 highest end</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/Location.Tools.html#getMax(java.util.List)">getMax(List&lt;Location&gt;)</a></span> - Static method in class org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/Location.Tools.html" title="class in org.biojava3.core.sequence.location.template">Location.Tools</a></dt>
<dd>
<div class="block">Scans through a list of locations to find the Location with the
 highest end</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getMaxGap()">getMaxGap()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getMaxGapFrag()">getMaxGapFrag()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#getMaxGapSize()">getMaxGapSize()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>
<div class="block">the Max gap size parameter G . default is 30, which was
 described to obtained empirically in the CE paper.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#getMaxGapSize()">getMaxGapSize()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>
<div class="block">CE specific parameter: set the Max gap size parameter G (during AFP extension).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#getMaximumCovalentValence()">getMaximumCovalentValence()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns the maximum valence for this Element.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#getMaximumValence()">getMaximumValence()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns the maximum valence for this Element.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#getMaxIter()">getMaxIter()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/BigSqrt.html#getMaxIterations()">getMaxIterations()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/BigSqrt.html" title="class in org.biojava.bio.structure.secstruc">BigSqrt</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#getMaxNrIterationsForOptimization()">getMaxNrIterationsForOptimization()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>
<div class="block">Get the maximum nr of times the (slow) optimiziation of alignment should iterate.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#getMaxOptRMSD()">getMaxOptRMSD()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>
<div class="block">Returns the maximum RMSD cutoff to be applied during alignment optimization (default: 99 = unlimited)</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#getMaxOptRMSD()">getMaxOptRMSD()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>
<div class="block">(jCE specific): maximum RMSD that shall be calculated for the alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getMaxPenalty()">getMaxPenalty()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#getMaxrefine()">getMaxrefine()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/FractionalIdentityScorer.html#getMaxScore()">getMaxScore()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/FractionalIdentityScorer.html" title="class in org.biojava3.alignment">FractionalIdentityScorer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/FractionalSimilarityScorer.html#getMaxScore()">getMaxScore()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/FractionalSimilarityScorer.html" title="class in org.biojava3.alignment">FractionalSimilarityScorer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/StandardRescoreRefiner.html#getMaxScore()">getMaxScore()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/StandardRescoreRefiner.html" title="class in org.biojava3.alignment">StandardRescoreRefiner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#getMaxScore()">getMaxScore()</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava3.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/Scorer.html#getMaxScore()">getMaxScore()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/Scorer.html" title="interface in org.biojava3.alignment.template">Scorer</a></dt>
<dd>
<div class="block">Returns maximum possible score.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/CaseFreeAminoAcidCompoundSet.html#getMaxSingleCompoundStringLength()">getMaxSingleCompoundStringLength()</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/CaseFreeAminoAcidCompoundSet.html" title="class in org.biojava3.aaproperties.xml">CaseFreeAminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/ModifiedAminoAcidCompoundSet.html#getMaxSingleCompoundStringLength()">getMaxSingleCompoundStringLength()</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/ModifiedAminoAcidCompoundSet.html" title="class in org.biojava3.aaproperties.xml">ModifiedAminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/AminoAcidCompoundSet.html#getMaxSingleCompoundStringLength()">getMaxSingleCompoundStringLength()</a></span> - Method in class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/AminoAcidCompoundSet.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompoundSet.html#getMaxSingleCompoundStringLength()">getMaxSingleCompoundStringLength()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava3.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/CompoundSet.html#getMaxSingleCompoundStringLength()">getMaxSingleCompoundStringLength()</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a></dt>
<dd>
<div class="block">Returns the maximum size of a compound String this set holds</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getMaxTra()">getMaxTra()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>
<div class="block">get the maximum number of Twists that are allowed...</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getMaxTra()">getMaxTra()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">get the maximum nr of Twists that are allowed...</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html#getMaxValue()">getMaxValue()</a></span> - Method in class org.biojava3.alignment.aaindex.<a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava3.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleSubstitutionMatrix.html#getMaxValue()">getMaxValue()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleSubstitutionMatrix.html" title="class in org.biojava3.alignment">SimpleSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/SubstitutionMatrix.html#getMaxValue()">getMaxValue()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/SubstitutionMatrix.html" title="interface in org.biojava3.alignment.template">SubstitutionMatrix</a></dt>
<dd>
<div class="block">Returns the maximum value in this matrix.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/SystemInfo.html#getMessage()">getMessage()</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/SystemInfo.html" title="class in org.biojava.bio.structure.align.gui">SystemInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html#getMethod()">getMethod()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html" title="class in org.biojava.bio.structure.io.mmcif.model">Exptl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#getMethod()">getMethod()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html#getMethod_details()">getMethod_details()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html" title="class in org.biojava.bio.structure.io.mmcif.model">Exptl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html#getMin()">getMin()</a></span> - Method in class org.biojava3.alignment.aaindex.<a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava3.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/FuzzyPoint.html#getMin()">getMin()</a></span> - Method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/FuzzyPoint.html" title="class in org.biojava3.core.sequence.location">FuzzyPoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/LocationHelper.html#getMin(java.util.List)">getMin(List&lt;Location&gt;)</a></span> - Static method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/LocationHelper.html" title="class in org.biojava3.core.sequence.location">LocationHelper</a></dt>
<dd>
<div class="block">Scans through a list of locations to find the Location with the
 lowest start</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/Location.Tools.html#getMin(java.util.List)">getMin(List&lt;Location&gt;)</a></span> - Static method in class org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/Location.Tools.html" title="class in org.biojava3.core.sequence.location.template">Location.Tools</a></dt>
<dd>
<div class="block">Scans through a list of locations to find the Location with the
 lowest start</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#getMinimumValence()">getMinimumValence()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns the minimum valence for this Element.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getMinLen()">getMinLen()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/FractionalIdentityScorer.html#getMinScore()">getMinScore()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/FractionalIdentityScorer.html" title="class in org.biojava3.alignment">FractionalIdentityScorer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/FractionalSimilarityScorer.html#getMinScore()">getMinScore()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/FractionalSimilarityScorer.html" title="class in org.biojava3.alignment">FractionalSimilarityScorer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/StandardRescoreRefiner.html#getMinScore()">getMinScore()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/StandardRescoreRefiner.html" title="class in org.biojava3.alignment">StandardRescoreRefiner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#getMinScore()">getMinScore()</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava3.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/Scorer.html#getMinScore()">getMinScore()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/Scorer.html" title="interface in org.biojava3.alignment.template">Scorer</a></dt>
<dd>
<div class="block">Returns minimum possible score.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html#getMinValue()">getMinValue()</a></span> - Method in class org.biojava3.alignment.aaindex.<a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava3.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleSubstitutionMatrix.html#getMinValue()">getMinValue()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleSubstitutionMatrix.html" title="class in org.biojava3.alignment">SimpleSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/SubstitutionMatrix.html#getMinValue()">getMinValue()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/SubstitutionMatrix.html" title="interface in org.biojava3.alignment.template">SubstitutionMatrix</a></dt>
<dd>
<div class="block">Returns the minimum value in this matrix.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getMisCut()">getMisCut()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getMisLen()">getMisLen()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getMisScore()">getMisScore()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html#getMod_type()">getMod_type()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html" title="class in org.biojava.bio.structure.io.mmcif.model">DatabasePDBrev</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#getModDate()">getModDate()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#getModel(int)">getModel(int)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">retrieve all Chains belonging to a model .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#getModel(int)">getModel(int)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">retrieve all Chains belonging to a model .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ronn/ModelLoader.html#getModel(int)">getModel(int)</a></span> - Method in class org.biojava3.ronn.<a href="./org/biojava3/ronn/ModelLoader.html" title="class in org.biojava3.ronn">ModelLoader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/AtomInfo.html#getModelNumber()">getModelNumber()</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/AtomInfo.html" title="class in org.biojava.bio.structure.align.gui.jmol">AtomInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ModifiedCompound.html#getModification()">getModification()</a></span> - Method in interface org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ModifiedCompound.html" title="interface in org.biojava3.protmod.structure">ModifiedCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ModifiedCompoundImpl.html#getModification()">getModification()</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ModifiedCompoundImpl.html" title="class in org.biojava3.protmod.structure">ModifiedCompoundImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/IPeptideProperties.html#getMolecularWeight(org.biojava3.core.sequence.ProteinSequence)">getMolecularWeight(ProteinSequence)</a></span> - Method in interface org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/IPeptideProperties.html" title="interface in org.biojava3.aaproperties">IPeptideProperties</a></dt>
<dd>
<div class="block">Returns the molecular weight of sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/IPeptideProperties.html#getMolecularWeight(org.biojava3.core.sequence.ProteinSequence, java.io.File)">getMolecularWeight(ProteinSequence, File)</a></span> - Method in interface org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/IPeptideProperties.html" title="interface in org.biojava3.aaproperties">IPeptideProperties</a></dt>
<dd>
<div class="block">Returns the molecular weight of sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/IPeptideProperties.html#getMolecularWeight(org.biojava3.core.sequence.ProteinSequence, java.io.File, java.io.File)">getMolecularWeight(ProteinSequence, File, File)</a></span> - Method in interface org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/IPeptideProperties.html" title="interface in org.biojava3.aaproperties">IPeptideProperties</a></dt>
<dd>
<div class="block">Returns the molecular weight of sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/PeptideProperties.html#getMolecularWeight(java.lang.String)">getMolecularWeight(String)</a></span> - Static method in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/PeptideProperties.html" title="class in org.biojava3.aaproperties">PeptideProperties</a></dt>
<dd>
<div class="block">An adaptor method to return the molecular weight of sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/PeptideProperties.html#getMolecularWeight(java.lang.String, java.io.File, java.io.File)">getMolecularWeight(String, File, File)</a></span> - Static method in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/PeptideProperties.html" title="class in org.biojava3.aaproperties">PeptideProperties</a></dt>
<dd>
<div class="block">An adaptor method to return the molecular weight of sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/PeptideProperties.html#getMolecularWeight(java.lang.String, java.io.File)">getMolecularWeight(String, File)</a></span> - Static method in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/PeptideProperties.html" title="class in org.biojava3.aaproperties">PeptideProperties</a></dt>
<dd>
<div class="block">An adaptor method to return the molecular weight of sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/PeptidePropertiesImpl.html#getMolecularWeight(org.biojava3.core.sequence.ProteinSequence)">getMolecularWeight(ProteinSequence)</a></span> - Method in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/PeptidePropertiesImpl.html" title="class in org.biojava3.aaproperties">PeptidePropertiesImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/PeptidePropertiesImpl.html#getMolecularWeight(org.biojava3.core.sequence.ProteinSequence, java.io.File)">getMolecularWeight(ProteinSequence, File)</a></span> - Method in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/PeptidePropertiesImpl.html" title="class in org.biojava3.aaproperties">PeptidePropertiesImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/PeptidePropertiesImpl.html#getMolecularWeight(org.biojava3.core.sequence.ProteinSequence, java.io.File, java.io.File)">getMolecularWeight(ProteinSequence, File, File)</a></span> - Method in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/PeptidePropertiesImpl.html" title="class in org.biojava3.aaproperties">PeptidePropertiesImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/AminoAcidCompositionTable.html#getMolecularWeight(java.lang.Character)">getMolecularWeight(Character)</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/AminoAcidCompositionTable.html" title="class in org.biojava3.aaproperties.xml">AminoAcidCompositionTable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/CodonCompound.html#getMolecularWeight()">getMolecularWeight()</a></span> - Method in class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/CodonCompound.html" title="class in org.biojava3.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompound.html#getMolecularWeight()">getMolecularWeight()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompound.html" title="class in org.biojava3.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/Compound.html#getMolecularWeight()">getMolecularWeight()</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/CaseInsensitiveCompound.html#getMolecularWeight()">getMolecularWeight()</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava3.core.sequence.transcription">CaseInsensitiveCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/Table.Codon.html#getMolecularWeight()">getMolecularWeight()</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/Table.Codon.html" title="class in org.biojava3.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/IPeptideProperties.html#getMolecularWeightBasedOnXML(org.biojava3.core.sequence.ProteinSequence, org.biojava3.aaproperties.xml.AminoAcidCompositionTable)">getMolecularWeightBasedOnXML(ProteinSequence, AminoAcidCompositionTable)</a></span> - Method in interface org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/IPeptideProperties.html" title="interface in org.biojava3.aaproperties">IPeptideProperties</a></dt>
<dd>
<div class="block">Returns the molecular weight of sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/PeptideProperties.html#getMolecularWeightBasedOnXML(java.lang.String, org.biojava3.aaproperties.xml.AminoAcidCompositionTable)">getMolecularWeightBasedOnXML(String, AminoAcidCompositionTable)</a></span> - Static method in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/PeptideProperties.html" title="class in org.biojava3.aaproperties">PeptideProperties</a></dt>
<dd>
<div class="block">An adaptor method that returns the molecular weight of sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/PeptidePropertiesImpl.html#getMolecularWeightBasedOnXML(org.biojava3.core.sequence.ProteinSequence, org.biojava3.aaproperties.xml.AminoAcidCompositionTable)">getMolecularWeightBasedOnXML(ProteinSequence, AminoAcidCompositionTable)</a></span> - Method in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/PeptidePropertiesImpl.html" title="class in org.biojava3.aaproperties">PeptidePropertiesImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getMolId()">getMolId()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>
<div class="block">Returns the mol id value.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getMolName()">getMolName()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/EntityPolySeq.html#getMon_id()">getMon_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/EntityPolySeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">EntityPolySeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#getMon_id()">getMon_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#getMon_id()">getMon_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getMon_nstd_flag()">getMon_nstd_flag()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getMon_nstd_parent_comp_id()">getMon_nstd_parent_comp_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/Alignments.html#getMultipleSequenceAlignment(java.util.List, java.lang.Object...)">getMultipleSequenceAlignment(List&lt;S&gt;, Object...)</a></span> - Static method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/Alignments.html" title="class in org.biojava3.alignment">Alignments</a></dt>
<dd>
<div class="block">Factory method which computes a multiple sequence alignment for the given <a href="http://java.sun.com/j2se/1.6.0/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util"><code>List</code></a> of <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a>s.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getMutation()">getMutation()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcid.html#getN()">getN()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcid.html" title="interface in org.biojava.bio.structure">AminoAcid</a></dt>
<dd>
<div class="block">get N atom.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcidImpl.html#getN()">getN()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcidImpl.html" title="class in org.biojava.bio.structure">AminoAcidImpl</a></dt>
<dd>
<div class="block">get N atom.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html#getN()">getN()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html" title="class in org.biojava.bio.structure.secstruc">SecStrucGroup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/StructureName.html#getName()">getName()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/StructureName.html" title="class in org.biojava.bio.structure.align.client">StructureName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#getName()">getName()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">get trimmed version of atom name, e.g.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#getName()">getName()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>
<div class="block">Gets this object's name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getName()">getName()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>replaced by <a href="./org/biojava/bio/structure/Chain.html#getChainID()"><code>Chain.getChainID()</code></a></i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getName()">getName()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AuditAuthor.html#getName()">getName()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AuditAuthor.html" title="class in org.biojava.bio.structure.io.mmcif.model">AuditAuthor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getName()">getName()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxEntityNonPoly.html#getName()">getName()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxEntityNonPoly.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxEntityNonPoly</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDescription.html#getName()">getName()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDescription.html" title="class in org.biojava.bio.structure.scop">ScopDescription</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#getName()">getName()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">get biological name of Structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#getName()">getName()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">get biological name of Structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/AminoAcidComposition.html#getName()">getName()</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/AminoAcidComposition.html" title="class in org.biojava3.aaproperties.xml">AminoAcidComposition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/Element.html#getName()">getName()</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/Element.html" title="class in org.biojava3.aaproperties.xml">Element</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/Isotope.html#getName()">getName()</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/Isotope.html" title="class in org.biojava3.aaproperties.xml">Isotope</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/Name2Count.html#getName()">getName()</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/Name2Count.html" title="class in org.biojava3.aaproperties.xml">Name2Count</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html#getName()">getName()</a></span> - Method in class org.biojava3.alignment.aaindex.<a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava3.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/GuideTree.Node.html#getName()">getName()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/GuideTree.Node.html" title="class in org.biojava3.alignment">GuideTree.Node</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleSubstitutionMatrix.html#getName()">getName()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleSubstitutionMatrix.html" title="class in org.biojava3.alignment">SimpleSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/GuideTreeNode.html#getName()">getName()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/GuideTreeNode.html" title="interface in org.biojava3.alignment.template">GuideTreeNode</a></dt>
<dd>
<div class="block">Returns the name of this node.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/SubstitutionMatrix.html#getName()">getName()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/SubstitutionMatrix.html" title="interface in org.biojava3.alignment.template">SubstitutionMatrix</a></dt>
<dd>
<div class="block">Returns the name (short description) of this matrix.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/IUPACParser.IUPACTable.html#getName()">getName()</a></span> - Method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/IUPACParser.IUPACTable.html" title="class in org.biojava3.core.sequence.io">IUPACParser.IUPACTable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/PdbPair.html#getName1()">getName1()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/PdbPair.html" title="class in org.biojava.bio.structure.align.client">PdbPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/SelectPDBPanel.html#getName1()">getName1()</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/SelectPDBPanel.html" title="class in org.biojava.bio.structure.align.gui">SelectPDBPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getName1()">getName1()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/PdbPair.html#getName2()">getName2()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/PdbPair.html" title="class in org.biojava.bio.structure.align.client">PdbPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/SelectPDBPanel.html#getName2()">getName2()</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/SelectPDBPanel.html" title="class in org.biojava.bio.structure.align.gui">SelectPDBPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getName2()">getName2()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#getnAtom()">getnAtom()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPDistanceMatrix.html#getNclose()">getNclose()</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPDistanceMatrix.html" title="class in org.biojava.bio.structure.domain.pdp">PDPDistanceMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/CutSites.html#getNcuts()">getNcuts()</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/CutSites.html" title="class in org.biojava.bio.structure.domain.pdp">CutSites</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#getNdb_seq_num()">getNdb_seq_num()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#getNdb_seq_num()">getNdb_seq_num()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/IPeptideProperties.html#getNetCharge(org.biojava3.core.sequence.ProteinSequence)">getNetCharge(ProteinSequence)</a></span> - Method in interface org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/IPeptideProperties.html" title="interface in org.biojava3.aaproperties">IPeptideProperties</a></dt>
<dd>
<div class="block">Returns the net charge of sequence at pH 7.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/PeptideProperties.html#getNetCharge(java.lang.String)">getNetCharge(String)</a></span> - Static method in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/PeptideProperties.html" title="class in org.biojava3.aaproperties">PeptideProperties</a></dt>
<dd>
<div class="block">An adaptor method to return the net charge of sequence at pH 7.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/PeptidePropertiesImpl.html#getNetCharge(org.biojava3.core.sequence.ProteinSequence)">getNetCharge(ProteinSequence)</a></span> - Method in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/PeptidePropertiesImpl.html" title="class in org.biojava3.aaproperties">PeptidePropertiesImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/Isotope.html#getNeutronsNum()">getNeutronsNum()</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/Isotope.html" title="class in org.biojava3.aaproperties.xml">Isotope</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/XMLHelper.html#getNewDocument()">getNewDocument()</a></span> - Static method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/XMLHelper.html" title="class in org.biojava3.core.util">XMLHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/TreeConstructor.html#getNewickString(boolean, boolean)">getNewickString(boolean, boolean)</a></span> - Method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/TreeConstructor.html" title="class in org.biojava3.phylo">TreeConstructor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/routines/AlignerHelper.html#getNextSubproblem(int[])">getNextSubproblem(int[])</a></span> - Static method in class org.biojava3.alignment.routines.<a href="./org/biojava3/alignment/routines/AlignerHelper.html" title="class in org.biojava3.alignment.routines">AlignerHelper</a></dt>
<dd>
<div class="block">Returns the coordinates for the next subproblem.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/CheckTreeAccuracy.html#getNodeDistance(org.forester.phylogeny.PhylogenyNode, org.forester.phylogeny.PhylogenyNode)">getNodeDistance(PhylogenyNode, PhylogenyNode)</a></span> - Method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/CheckTreeAccuracy.html" title="class in org.biojava3.phylo">CheckTreeAccuracy</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getNormAlignScore()">getNormAlignScore()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#getNotesList()">getNotesList()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#getNrAlignments()">getNrAlignments()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#getNrAtoms(org.biojava.bio.structure.Structure)">getNrAtoms(Structure)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>
<div class="block">Count how many number of Atoms are contained within a Structure object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/SimpleStructureServer.html#getNrCPUs()">getNrCPUs()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/SimpleStructureServer.html" title="class in org.biojava.bio.structure.server">SimpleStructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getNrEQR()">getNrEQR()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">Get the number of structurally equivalent residues</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#getNrGroups(org.biojava.bio.structure.Structure)">getNrGroups(Structure)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>
<div class="block">Count how many groups are contained within a structure object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Domain.html#getNseg()">getNseg()</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Domain.html" title="class in org.biojava.bio.structure.domain.pdp">Domain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SubstitutionMatrixHelper.html#getNuc4_2()">getNuc4_2()</a></span> - Static method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SubstitutionMatrixHelper.html" title="class in org.biojava3.alignment">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Nuc 4.2 matrix by Lowe</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SubstitutionMatrixHelper.html#getNuc4_4()">getNuc4_4()</a></span> - Static method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SubstitutionMatrixHelper.html" title="class in org.biojava3.alignment">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns Nuc 4.4 matrix by Lowe</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html#getNum()">getNum()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html" title="class in org.biojava.bio.structure.io.mmcif.model">DatabasePDBrev</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/EntityPolySeq.html#getNum()">getNum()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/EntityPolySeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">EntityPolySeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/Utils.html#getNumberOfInvalidChar(java.lang.String, java.util.Set, boolean)">getNumberOfInvalidChar(String, Set&lt;Character&gt;, boolean)</a></span> - Static method in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/Utils.html" title="class in org.biojava3.aaproperties">Utils</a></dt>
<dd>
<div class="block">Return the number of invalid characters in sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/Strand.html#getNumericRepresentation()">getNumericRepresentation()</a></span> - Method in enum org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/Strand.html" title="enum in org.biojava3.core.sequence">Strand</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleAlignedSequence.html#getNumGaps()">getNumGaps()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleAlignedSequence.html" title="class in org.biojava3.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AlignedSequence.html#getNumGaps()">getNumGaps()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AlignedSequence.html" title="interface in org.biojava3.alignment.template">AlignedSequence</a></dt>
<dd>
<div class="block">Returns number of gaps in the sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleSequencePair.html#getNumIdenticals()">getNumIdenticals()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleSequencePair.html" title="class in org.biojava3.alignment">SimpleSequencePair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/SequencePair.html#getNumIdenticals()">getNumIdenticals()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/SequencePair.html" title="interface in org.biojava3.alignment.template">SequencePair</a></dt>
<dd>
<div class="block">Returns the number of indices for which both the query and target sequences have an identical <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template"><code>Compound</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getNumRes()">getNumRes()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleSequencePair.html#getNumSimilars()">getNumSimilars()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleSequencePair.html" title="class in org.biojava3.alignment">SimpleSequencePair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/SequencePair.html#getNumSimilars()">getNumSimilars()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/SequencePair.html" title="interface in org.biojava3.alignment.template">SequencePair</a></dt>
<dd>
<div class="block">Returns the number of indices for which both the query and target sequences have a similar <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template"><code>Compound</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcid.html#getO()">getO()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcid.html" title="interface in org.biojava.bio.structure">AminoAcid</a></dt>
<dd>
<div class="block">get O atom.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcidImpl.html#getO()">getO()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcidImpl.html" title="class in org.biojava.bio.structure">AminoAcidImpl</a></dt>
<dd>
<div class="block">get O atom.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html#getO()">getO()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html" title="class in org.biojava.bio.structure.secstruc">SecStrucGroup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#getOccupancy()">getOccupancy()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">get occupancy.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#getOccupancy()">getOccupancy()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getOccupancy()">getOccupancy()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getOccupancy_esd()">getOccupancy_esd()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getOccupancy_max()">getOccupancy_max()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getOccupancy_min()">getOccupancy_min()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModification.html#getOccurrenceType()">getOccurrenceType()</a></span> - Method in interface org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModification.html" title="interface in org.biojava3.protmod">ProteinModification</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationImpl.html#getOccurrenceType()">getOccurrenceType()</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationImpl.html" title="class in org.biojava3.protmod">ProteinModificationImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/CodonCompound.html#getOne()">getOne()</a></span> - Method in class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/CodonCompound.html" title="class in org.biojava3.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/Table.CaseInsensitiveTriplet.html#getOne()">getOne()</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/Table.CaseInsensitiveTriplet.html" title="class in org.biojava3.core.sequence.transcription">Table.CaseInsensitiveTriplet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/Table.Codon.html#getOne()">getOne()</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/Table.Codon.html" title="class in org.biojava3.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getOne_letter_code()">getOne_letter_code()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/chem/ChemCompTools.html#getOneLetterCode(org.biojava.bio.structure.io.mmcif.model.ChemComp, org.biojava.bio.structure.io.mmcif.ChemicalComponentDictionary)">getOneLetterCode(ChemComp, ChemicalComponentDictionary)</a></span> - Static method in class org.biojava.bio.structure.io.mmcif.chem.<a href="./org/biojava/bio/structure/io/mmcif/chem/ChemCompTools.html" title="class in org.biojava.bio.structure.io.mmcif.chem">ChemCompTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompGroupFactory.html#getOneLetterCode(org.biojava.bio.structure.io.mmcif.model.ChemComp)">getOneLetterCode(ChemComp)</a></span> - Static method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompGroupFactory.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompGroupFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/data/sequence/FastaSequence.html#getOnelineFasta()">getOnelineFasta()</a></span> - Method in class org.biojava3.data.sequence.<a href="./org/biojava3/data/sequence/FastaSequence.html" title="class in org.biojava3.data.sequence">FastaSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/MenuCreator.html#getOpenPDBMenuItem()">getOpenPDBMenuItem()</a></span> - Static method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/MenuCreator.html" title="class in org.biojava.bio.structure.align.gui">MenuCreator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleGapPenalty.html#getOpenPenalty()">getOpenPenalty()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleGapPenalty.html" title="class in org.biojava3.alignment">SimpleGapPenalty</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/GapPenalty.html#getOpenPenalty()">getOpenPenalty()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/GapPenalty.html" title="interface in org.biojava3.alignment.template">GapPenalty</a></dt>
<dd>
<div class="block">Returns penalty given when a deletion or insertion gap first opens</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getOptAln()">getOptAln()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">Tracks the Atom positions in the optimal alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getOptLen()">getOptLen()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">The length of each block</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getOptLength()">getOptLength()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getOptRmsd()">getOptRmsd()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getOrgan()">getOrgan()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getOrganelle()">getOrganelle()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getOrganismCommon()">getOrganismCommon()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html#getOrganismName()">getOrganismName()</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>
<div class="block">Get the organism name assigned to this sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getOrganismScientific()">getOrganismScientific()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getOrganismTaxId()">getOrganismTaxId()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html#getOriginal()">getOriginal()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html" title="class in org.biojava.bio.structure.secstruc">SecStrucGroup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#getOriginalHeader()">getOriginalHeader()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ModifiedCompoundImpl.html#getOriginalModificationId()">getOriginalModificationId()</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ModifiedCompoundImpl.html" title="class in org.biojava3.protmod.structure">ModifiedCompoundImpl</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>use getModification().getId()</i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleAlignedSequence.html#getOriginalSequence()">getOriginalSequence()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleAlignedSequence.html" title="class in org.biojava3.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AlignedSequence.html#getOriginalSequence()">getOriginalSequence()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AlignedSequence.html" title="interface in org.biojava3.alignment.template">AlignedSequence</a></dt>
<dd>
<div class="block">Returns the original <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a> before alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleProfile.html#getOriginalSequences()">getOriginalSequences()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleProfile.html" title="class in org.biojava3.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/Profile.html#getOriginalSequences()">getOriginalSequences()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns a <a href="http://java.sun.com/j2se/1.6.0/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util"><code>List</code></a> containing the original <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a>s used for alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#getORmsdThr()">getORmsdThr()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>
<div class="block">Get the Original RMSD threshold from which the alignment optimization is started</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html#getOutFile()">getOutFile()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html" title="class in org.biojava.bio.structure.align">CallableStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#getOutFile()">getOutFile()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/DBSearchGUI.html#getOutFileLocation()">getOutFileLocation()</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/DBSearchGUI.html" title="class in org.biojava.bio.structure.align.gui">DBSearchGUI</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastOutputProperties.html#getOutputFormat()">getOutputFormat()</a></span> - Method in class org.biojava3.ws.alignment.qblast.<a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastOutputProperties.html" title="class in org.biojava3.ws.alignment.qblast">NCBIQBlastOutputProperties</a></dt>
<dd>
<div class="block">Simply returns stream output format for the actual RemoteQBlastOutputProperties object</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastOutputProperties.html#getOutputOption(java.lang.String)">getOutputOption(String)</a></span> - Method in class org.biojava3.ws.alignment.qblast.<a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastOutputProperties.html" title="class in org.biojava3.ws.alignment.qblast">NCBIQBlastOutputProperties</a></dt>
<dd>
<div class="block">Method that returns any value associated to any key for this RemoteQBlastOutputProperties object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/RemotePairwiseAlignmentOutputProperties.html#getOutputOption(java.lang.String)">getOutputOption(String)</a></span> - Method in interface org.biojava3.ws.alignment.<a href="./org/biojava3/ws/alignment/RemotePairwiseAlignmentOutputProperties.html" title="interface in org.biojava3.ws.alignment">RemotePairwiseAlignmentOutputProperties</a></dt>
<dd>
<div class="block">Method that returns the value associated with the key given in parameter.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastOutputProperties.html#getOutputOptions()">getOutputOptions()</a></span> - Method in class org.biojava3.ws.alignment.qblast.<a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastOutputProperties.html" title="class in org.biojava3.ws.alignment.qblast">NCBIQBlastOutputProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/RemotePairwiseAlignmentOutputProperties.html#getOutputOptions()">getOutputOptions()</a></span> - Method in interface org.biojava3.ws.alignment.<a href="./org/biojava3/ws/alignment/RemotePairwiseAlignmentOutputProperties.html" title="interface in org.biojava3.ws.alignment">RemotePairwiseAlignmentOutputProperties</a></dt>
<dd>
<div class="block">Method to get all keys to the information stored in this object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getOverall_FOM_free_R_set()">getOverall_FOM_free_R_set()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getOverall_FOM_work_R_set()">getOverall_FOM_work_R_set()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getOverall_SU_B()">getOverall_SU_B()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getOverall_SU_ML()">getOverall_SU_ML()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getOverall_SU_R_Cruickshank_DPI()">getOverall_SU_R_Cruickshank_DPI()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getOverall_SU_R_free()">getOverall_SU_R_free()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleAlignedSequence.html#getOverlapCount()">getOverlapCount()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleAlignedSequence.html" title="class in org.biojava3.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AlignedSequence.html#getOverlapCount()">getOverlapCount()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AlignedSequence.html" title="interface in org.biojava3.alignment.template">AlignedSequence</a></dt>
<dd>
<div class="block">Returns the maximum number of elements contributed to a column of an alignment by this <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#getOxidationState()">getOxidationState()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns a typical oxidation state for this Element.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFP.html#getP1()">getP1()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFP.html" title="class in org.biojava.bio.structure.align.model">AFP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFP.html#getP2()">getP2()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFP.html" title="class in org.biojava.bio.structure.align.model">AFP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html#getPair()">getPair()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html" title="class in org.biojava.bio.structure.align">CallableStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html#getPair()">getPair()</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html" title="class in org.biojava3.alignment.template">AbstractPairwiseSequenceAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractProfileProfileAligner.html#getPair()">getPair()</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractProfileProfileAligner.html" title="class in org.biojava3.alignment.template">AbstractProfileProfileAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/PairwiseSequenceAligner.html#getPair()">getPair()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/PairwiseSequenceAligner.html" title="interface in org.biojava3.alignment.template">PairwiseSequenceAligner</a></dt>
<dd>
<div class="block">Returns sequence alignment pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/ProfileProfileAligner.html#getPair()">getPair()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/ProfileProfileAligner.html" title="interface in org.biojava3.alignment.template">ProfileProfileAligner</a></dt>
<dd>
<div class="block">Returns alignment profile pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/StandardRescoreRefiner.html#getPairInProfileScorer()">getPairInProfileScorer()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/StandardRescoreRefiner.html" title="class in org.biojava3.alignment">StandardRescoreRefiner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/RescoreRefiner.html#getPairInProfileScorer()">getPairInProfileScorer()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/RescoreRefiner.html" title="interface in org.biojava3.alignment.template">RescoreRefiner</a></dt>
<dd>
<div class="block">Returns class type of pairwise scoring subroutine.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/Alignments.html#getPairwiseAligner(S, S, org.biojava3.alignment.Alignments.PairwiseSequenceAlignerType, org.biojava3.alignment.template.GapPenalty, org.biojava3.alignment.template.SubstitutionMatrix)">getPairwiseAligner(S, S, Alignments.PairwiseSequenceAlignerType, GapPenalty, SubstitutionMatrix&lt;C&gt;)</a></span> - Static method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/Alignments.html" title="class in org.biojava3.alignment">Alignments</a></dt>
<dd>
<div class="block">Factory method which constructs a pairwise sequence aligner.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/Alignments.html#getPairwiseAlignment(S, S, org.biojava3.alignment.Alignments.PairwiseSequenceAlignerType, org.biojava3.alignment.template.GapPenalty, org.biojava3.alignment.template.SubstitutionMatrix)">getPairwiseAlignment(S, S, Alignments.PairwiseSequenceAlignerType, GapPenalty, SubstitutionMatrix&lt;C&gt;)</a></span> - Static method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/Alignments.html" title="class in org.biojava3.alignment">Alignments</a></dt>
<dd>
<div class="block">Factory method which computes a sequence alignment for the given <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a> pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ScoreMatrix.html#getPairwiseScore(java.lang.String, java.lang.String)">getPairwiseScore(String, String)</a></span> - Method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ScoreMatrix.html" title="class in org.biojava3.phylo">ScoreMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ScoreMatrix.html#getPairwiseScore(char, char)">getPairwiseScore(char, char)</a></span> - Method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ScoreMatrix.html" title="class in org.biojava3.phylo">ScoreMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SubstitutionMatrixHelper.html#getPAM250()">getPAM250()</a></span> - Static method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SubstitutionMatrixHelper.html" title="class in org.biojava3.alignment">SubstitutionMatrixHelper</a></dt>
<dd>
<div class="block">Returns PAM 250 matrix by Dayhoff</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#getPAM250(java.lang.String, java.lang.String)">getPAM250(String, String)</a></span> - Static method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#getPAM250(char, char)">getPAM250(char, char)</a></span> - Static method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AFPChainCoordManager.html#getPanelPos(int, int)">getPanelPos(int, int)</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AFPChainCoordManager.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AFPChainCoordManager</a></dt>
<dd>
<div class="block">get the position of the sequence position on the Panel</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/CoordManager.html#getPanelPos(int)">getPanelPos(int)</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/CoordManager.html" title="class in org.biojava.bio.structure.gui.util">CoordManager</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/SigEva.html#getPara(int, int)">getPara(int, int)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/SigEva.html" title="class in org.biojava.bio.structure.align.fatcat.calc">SigEva</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/AbstractStructureAlignment.html#getParameters()">getParameters()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/AbstractStructureAlignment.html" title="class in org.biojava.bio.structure.align">AbstractStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/BioJavaStructureAlignment.html#getParameters()">getParameters()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/BioJavaStructureAlignment.html" title="class in org.biojava.bio.structure.align">BioJavaStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html#getParameters()">getParameters()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce">AbstractUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeMain.html#getParameters()">getParameters()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeMain.html" title="class in org.biojava.bio.structure.align.ce">CeMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeSideChainMain.html#getParameters()">getParameters()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeSideChainMain.html" title="class in org.biojava.bio.structure.align.ce">CeSideChainMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeSideChainUserArgumentProcessor.html#getParameters()">getParameters()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeSideChainUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce">CeSideChainUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeUserArgumentProcessor.html#getParameters()">getParameters()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce">CeUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCatFlexible.html#getParameters()">getParameters()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCatFlexible.html" title="class in org.biojava.bio.structure.align.fatcat">FatCatFlexible</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCatRigid.html#getParameters()">getParameters()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCatRigid.html" title="class in org.biojava.bio.structure.align.fatcat">FatCatRigid</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCatUserArgumentProcessor.html#getParameters()">getParameters()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCatUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.fatcat">FatCatUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWaterman3Daligner.html#getParameters()">getParameters()</a></span> - Method in class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWaterman3Daligner.html" title="class in org.biojava.bio.structure.align.seq">SmithWaterman3Daligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWatermanUserArgumentProcessor.html#getParameters()">getParameters()</a></span> - Method in class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWatermanUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.seq">SmithWatermanUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructureAlignment.html#getParameters()">getParameters()</a></span> - Method in interface org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructureAlignment.html" title="interface in org.biojava.bio.structure.align">StructureAlignment</a></dt>
<dd>
<div class="block">Return the paramers for this algorithm.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/FarmJob.html#getParams()">getParams()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/FarmJob.html" title="class in org.biojava.bio.structure.align">FarmJob</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructurePairAligner.html#getParams()">getParams()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructurePairAligner.html" title="class in org.biojava.bio.structure.align">StructurePairAligner</a></dt>
<dd>
<div class="block">get the parameters.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#getParent()">getParent()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>replaced by <a href="./org/biojava/bio/structure/Atom.html#getGroup()"><code>Atom.getGroup()</code></a></i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#getParent()">getParent()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getParent()">getParent()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Returns the parent Structure of this chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getParent()">getParent()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Returns the parent Structure of this chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#getParent()">getParent()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Get the structure object that this DBRef relates to.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#getParent()">getParent()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>replaced by <a href="./org/biojava/bio/structure/Group.html#getChain()"><code>Group.getChain()</code></a></i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#getParent()">getParent()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">Returns the parent Chain of the Group</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html#getParent(org.biojava.bio.structure.io.mmcif.model.ChemComp)">getParent(ChemComp)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html" title="class in org.biojava.bio.structure.io.mmcif">ChemicalComponentDictionary</a></dt>
<dd>
<div class="block">Get the parent of a component.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/GuideTree.Node.html#getParent()">getParent()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/GuideTree.Node.html" title="class in org.biojava3.alignment">GuideTree.Node</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/GeneSequence.html#getParentChromosomeSequence()">getParentChromosomeSequence()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/GeneSequence.html" title="class in org.biojava3.core.sequence">GeneSequence</a></dt>
<dd>
<div class="block">The parent ChromosomeSequence which contains the actual DNA sequence data</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/AbstractFeature.html#getParentFeature()">getParentFeature()</a></span> - Method in class org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/AbstractFeature.html" title="class in org.biojava3.core.sequence.features">AbstractFeature</a></dt>
<dd>
<div class="block">Get the parent Feature</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/FeatureInterface.html#getParentFeature()">getParentFeature()</a></span> - Method in interface org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/FeatureInterface.html" title="interface in org.biojava3.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">Get the parent feature</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#getParentSequence()">getParentSequence()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopNode.html#getParentSunid()">getParentSunid()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopNode.html" title="class in org.biojava.bio.structure.scop">ScopNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/HBond.html#getPartner()">getPartner()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/HBond.html" title="class in org.biojava.bio.structure.secstruc">HBond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/Alignable.html#getPath()">getPath()</a></span> - Method in interface org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/Alignable.html" title="interface in org.biojava.bio.structure.align.pairwise">Alignable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#getPath()">getPath()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html#getPath()">getPath()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">StrCompAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#getPath()">getPath()</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>
<div class="block">Get the path that is used to cache PDB files.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/LocalCacheStructureProvider.html#getPath()">getPath()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/LocalCacheStructureProvider.html" title="class in org.biojava.bio.structure.io">LocalCacheStructureProvider</a></dt>
<dd>
<div class="block">Returns the path value.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/MMCIFFileReader.html#getPath()">getPath()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/MMCIFFileReader.html" title="class in org.biojava.bio.structure.io">MMCIFFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#getPath()">getPath()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>
<div class="block">Returns the path value.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/StructureIOFile.html#getPath()">getPath()</a></span> - Method in interface org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/StructureIOFile.html" title="interface in org.biojava.bio.structure.io">StructureIOFile</a></dt>
<dd>
<div class="block">get the directory path to the files</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/Alignable.html#getPathSize()">getPathSize()</a></span> - Method in interface org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/Alignable.html" title="interface in org.biojava.bio.structure.align.pairwise">Alignable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html#getPathSize()">getPathSize()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">StrCompAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#getPaulingElectronegativity()">getPaulingElectronegativity()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns the Pauling electronegativity for this Element.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#getPdb1()">getPdb1()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html#getPdb1()">getPdb1()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentResult</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#getPdb2()">getPdb2()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html#getPdb2()">getPdb2()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentResult</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#getPdb_ins_code()">getPdb_ins_code()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#getPdb_ins_code()">getPdb_ins_code()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#getPdb_mon_id()">getPdb_mon_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#getPdb_mon_id()">getPdb_mon_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#getPdb_seq_num()">getPdb_seq_num()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#getPdb_seq_num()">getPdb_seq_num()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#getPdb_strand_id()">getPdb_strand_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#getPdb_strand_id()">getPdb_strand_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getPdbAln()">getPdbAln()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">used temporarily during XML serialization to track the PDB positions of the alignmnet</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModification.html#getPdbccId()">getPdbccId()</a></span> - Method in interface org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModification.html" title="interface in org.biojava3.protmod">ProteinModification</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationImpl.html#getPdbccId()">getPdbccId()</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationImpl.html" title="class in org.biojava3.protmod">ProteinModificationImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/Component.html#getPdbccIds()">getPdbccIds()</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/Component.html" title="class in org.biojava3.protmod">Component</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModification.html#getPdbccName()">getPdbccName()</a></span> - Method in interface org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModification.html" title="interface in org.biojava3.protmod">ProteinModification</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationImpl.html#getPdbccName()">getPdbccName()</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationImpl.html" title="class in org.biojava3.protmod">ProteinModificationImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/util/IOUtils.html#getPDBCharacter(boolean, char, char, boolean, char)">getPDBCharacter(boolean, char, char, boolean, char)</a></span> - Static method in class org.biojava3.core.sequence.io.util.<a href="./org/biojava3/core/sequence/io/util/IOUtils.html" title="class in org.biojava3.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">Creates formatted String for a single character of PDB output</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#getPDBCode()">getPDBCode()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>replaced by <a href="./org/biojava/bio/structure/Group.html#getResidueNumber()"><code>Group.getResidueNumber()</code></a></i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#getPDBCode()">getPDBCode()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>replaced by #getSeqNum</i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureEvent.html#getPDBCode()">getPDBCode()</a></span> - Method in interface org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureEvent.html" title="interface in org.biojava.bio.structure.server">StructureEvent</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureEventImpl.html#getPDBCode()">getPDBCode()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureEventImpl.html" title="class in org.biojava.bio.structure.server">StructureEventImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#getPDBCode()">getPDBCode()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">get PDB code of structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#getPDBCode()">getPDBCode()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">get PDB code of structure .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/PdbPair.html#getPDBCode1()">getPDBCode1()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/PdbPair.html" title="class in org.biojava.bio.structure.align.client">PdbPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/PdbPair.html#getPDBCode2()">getPDBCode2()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/PdbPair.html" title="class in org.biojava.bio.structure.align.client">PdbPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/util/IOUtils.html#getPDBConservation(boolean, char, char, boolean)">getPDBConservation(boolean, char, char, boolean)</a></span> - Static method in class org.biojava3.core.sequence.io.util.<a href="./org/biojava3/core/sequence/io/util/IOUtils.html" title="class in org.biojava3.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">Creates formatted String for displaying conservation in PDB output</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/ConfigPDBInstallPanel.html#getPDBDirField()">getPDBDirField()</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/ConfigPDBInstallPanel.html" title="class in org.biojava.bio.structure.align.gui">ConfigPDBInstallPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#getPdbFilePath()">getPdbFilePath()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#getPdbFilePath()">getPdbFilePath()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/UserConfiguration.html#getPdbFilePath()">getPdbFilePath()</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/UserConfiguration.html" title="class in org.biojava.bio.structure.align.util">UserConfiguration</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/FlatFileInstallation.html#getPDBHeader(java.lang.String)">getPDBHeader(String)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/FlatFileInstallation.html" title="class in org.biojava.bio.structure.server">FlatFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html#getPDBHeader(java.lang.String)">getPDBHeader(String)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html" title="class in org.biojava.bio.structure.server">MMCIFFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/PDBInstallation.html#getPDBHeader(java.lang.String)">getPDBHeader(String)</a></span> - Method in interface org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/PDBInstallation.html" title="interface in org.biojava.bio.structure.server">PDBInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
<div class="block">get the PDB header for a single protein structure</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#getPDBHeader()">getPDBHeader()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">return the header information for this PDB file</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#getPDBHeader()">getPDBHeader()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/StructureName.html#getPdbId()">getPdbId()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/StructureName.html" title="class in org.biojava.bio.structure.align.client">StructureName</a></dt>
<dd>
<div class="block">PDB IDs are always returned as upper case</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#getPdbId()">getPdbId()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/JmolTools.html#getPdbInfo(org.biojava.bio.structure.Atom)">getPdbInfo(Atom)</a></span> - Static method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/JmolTools.html" title="class in org.biojava.bio.structure.align.gui.jmol">JmolTools</a></dt>
<dd>
<div class="block">get jmol style info:
  jmol style: [MET]508:A.CA/1 #3918
  insertion code: [ASP]1^A:A.CA/1 #2</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/JmolTools.html#getPdbInfo(org.biojava.bio.structure.Atom, boolean)">getPdbInfo(Atom, boolean)</a></span> - Static method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/JmolTools.html" title="class in org.biojava.bio.structure.align.gui.jmol">JmolTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/FlatFileInstallation.html#getPDBInfoFile()">getPDBInfoFile()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/FlatFileInstallation.html" title="class in org.biojava.bio.structure.server">FlatFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/SimpleStructureServer.html#getPDBInstallation()">getPDBInstallation()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/SimpleStructureServer.html" title="class in org.biojava.bio.structure.server">SimpleStructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureServer.html#getPDBInstallation()">getPDBInstallation()</a></span> - Method in interface org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureServer.html" title="interface in org.biojava.bio.structure.server">StructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/util/IOUtils.html#getPDBLegend()">getPDBLegend()</a></span> - Static method in class org.biojava3.core.sequence.io.util.<a href="./org/biojava3/core/sequence/io/util/IOUtils.html" title="class in org.biojava3.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">Creates formatted String for displaying conservation legend in PDB output</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#getPDBline()">getPDBline()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>@deprecated replaced by <a href="./org/biojava/bio/structure/PDBRecord.html#toPDB()"><code>PDBRecord.toPDB()</code></a></i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#getPDBline()">getPDBline()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>&nbsp;</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#getPDBName()">getPDBName()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">Get the PDB 3 character name for this group.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#getPDBName()">getPDBName()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">Returns the PDBName.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/StructureGroup.html#getPDBName()">getPDBName()</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/StructureGroup.html" title="class in org.biojava3.protmod.structure">StructureGroup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ModificationLinkage.html#getPDBNameOfPotentialAtomsOnComponent1()">getPDBNameOfPotentialAtomsOnComponent1()</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ModificationLinkage.html" title="class in org.biojava3.protmod">ModificationLinkage</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ModificationLinkage.html#getPDBNameOfPotentialAtomsOnComponent2()">getPDBNameOfPotentialAtomsOnComponent2()</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ModificationLinkage.html" title="class in org.biojava3.protmod">ModificationLinkage</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/JFatCatClient.html#getPdbPairs(java.lang.String, int, java.lang.String)">getPdbPairs(String, int, String)</a></span> - Static method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/JFatCatClient.html" title="class in org.biojava.bio.structure.align.client">JFatCatClient</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#getPDBResidueNumber(org.biojava.bio.structure.Group)">getPDBResidueNumber(Group)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>replaced by  Group.getResidueNumber()</i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/StructureGroup.html#getPDBResidueNumber()">getPDBResidueNumber()</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/StructureGroup.html" title="class in org.biojava3.protmod.structure">StructureGroup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/DisplayAFP.html#getPDBresnum(int, org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[])">getPDBresnum(int, AFPChain, Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/DisplayAFP.html" title="class in org.biojava.bio.structure.align.gui">DisplayAFP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#getPDBresnum1()">getPDBresnum1()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#getPDBresnum2()">getPDBresnum2()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#getPDBserial()">getPDBserial()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">get PDB atom number.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#getPDBserial()">getPDBserial()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>
<div class="block">get PDB atom number.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/ConfigPDBInstallPanel.html#getPdbSplit()">getPdbSplit()</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/ConfigPDBInstallPanel.html" title="class in org.biojava.bio.structure.align.gui">ConfigPDBInstallPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/DBSearchGUI.html#getPDBUploadPanel()">getPDBUploadPanel()</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/DBSearchGUI.html" title="class in org.biojava.bio.structure.align.gui">DBSearchGUI</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html#getPdbx_align_begin()">getPdbx_align_begin()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRef</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getPdbx_ambiguous_flag()">getPdbx_ambiguous_flag()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#getPdbx_auth_seq_align_beg()">getPdbx_auth_seq_align_beg()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#getPdbx_auth_seq_align_end()">getPdbx_auth_seq_align_end()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html#getPdbx_blank_PDB_chainid_flag()">getPdbx_blank_PDB_chainid_flag()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructAsym</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html#getPdbx_CASP_flag()">getPdbx_CASP_flag()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html" title="class in org.biojava.bio.structure.io.mmcif.model">Struct</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getPdbx_data_cutoff_high_absF()">getPdbx_data_cutoff_high_absF()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getPdbx_data_cutoff_high_rms_absF()">getPdbx_data_cutoff_high_rms_absF()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getPdbx_data_cutoff_low_absF()">getPdbx_data_cutoff_low_absF()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html#getPdbx_db_accession()">getPdbx_db_accession()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRef</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#getPdbx_db_accession()">getPdbx_db_accession()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#getPdbx_db_align_beg_ins_code()">getPdbx_db_align_beg_ins_code()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#getPdbx_db_align_end_ins_code()">getPdbx_db_align_end_ins_code()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html#getPdbx_description()">getPdbx_description()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model">Entity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html#getPdbx_descriptor()">getPdbx_descriptor()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html" title="class in org.biojava.bio.structure.io.mmcif.model">Struct</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html#getPdbx_ec()">getPdbx_ec()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model">Entity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getPdbx_formal_charge()">getPdbx_formal_charge()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getPdbx_formal_charge()">getPdbx_formal_charge()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html#getPdbx_fragment()">getPdbx_fragment()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model">Entity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getPdbx_ideal_coordinates_details()">getPdbx_ideal_coordinates_details()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getPdbx_ideal_coordinates_missing_flag()">getPdbx_ideal_coordinates_missing_flag()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getPdbx_initial_date()">getPdbx_initial_date()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getPdbx_isotropic_thermal_model()">getPdbx_isotropic_thermal_model()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructKeywords.html#getPdbx_keywords()">getPdbx_keywords()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructKeywords.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructKeywords</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getPdbx_ls_cross_valid_method()">getPdbx_ls_cross_valid_method()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getPdbx_ls_sigma_F()">getPdbx_ls_sigma_F()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getPdbx_ls_sigma_I()">getPdbx_ls_sigma_I()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getPdbx_method_to_determine_struct()">getPdbx_method_to_determine_struct()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getPdbx_model_coordinates_db_code()">getPdbx_model_coordinates_db_code()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getPdbx_model_coordinates_details()">getPdbx_model_coordinates_details()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getPdbx_model_coordinates_missing_flag()">getPdbx_model_coordinates_missing_flag()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html#getPdbx_model_details()">getPdbx_model_details()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html" title="class in org.biojava.bio.structure.io.mmcif.model">Struct</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html#getPdbx_model_type_details()">getPdbx_model_type_details()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html" title="class in org.biojava.bio.structure.io.mmcif.model">Struct</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html#getPdbx_modified()">getPdbx_modified()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructAsym</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getPdbx_modified_date()">getPdbx_modified_date()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html#getPdbx_mutation()">getPdbx_mutation()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model">Entity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html#getPdbx_number_of_molecules()">getPdbx_number_of_molecules()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model">Entity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AuditAuthor.html#getPdbx_ordinal()">getPdbx_ordinal()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AuditAuthor.html" title="class in org.biojava.bio.structure.io.mmcif.model">AuditAuthor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getPdbx_overall_ESU_R()">getPdbx_overall_ESU_R()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getPdbx_overall_ESU_R_Free()">getPdbx_overall_ESU_R_Free()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getPdbx_overall_phase_error()">getPdbx_overall_phase_error()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#getPdbx_PDB_id_code()">getPdbx_PDB_id_code()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getPdbx_PDB_ins_code()">getPdbx_PDB_ins_code()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getPdbx_PDB_model_num()">getPdbx_PDB_model_num()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getPdbx_processing_site()">getPdbx_processing_site()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getPdbx_R_Free_selection_details()">getPdbx_R_Free_selection_details()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getPdbx_refine_id()">getPdbx_refine_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getPdbx_release_status()">getPdbx_release_status()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getPdbx_replaced_by()">getPdbx_replaced_by()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getPdbx_replaces()">getPdbx_replaces()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#getPdbx_seq_align_beg_ins_code()">getPdbx_seq_align_beg_ins_code()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#getPdbx_seq_align_end_ins_code()">getPdbx_seq_align_end_ins_code()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html#getPdbx_seq_one_letter_code()">getPdbx_seq_one_letter_code()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRef</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getPdbx_solvent_ion_probe_radii()">getPdbx_solvent_ion_probe_radii()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getPdbx_solvent_shrinkage_radii()">getPdbx_solvent_shrinkage_radii()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getPdbx_solvent_vdw_probe_radii()">getPdbx_solvent_vdw_probe_radii()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getPdbx_starting_model()">getPdbx_starting_model()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getPdbx_stereochem_target_val_spec_case()">getPdbx_stereochem_target_val_spec_case()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getPdbx_stereochemistry_target_values()">getPdbx_stereochemistry_target_values()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#getPdbx_strand_id()">getPdbx_strand_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getPdbx_subcomponent_list()">getPdbx_subcomponent_list()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getPdbx_synonyms()">getPdbx_synonyms()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getPdbx_type()">getPdbx_type()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#getPercId()">getPercId()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#getPeriod()">getPeriod()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns the period in the periodic table of this Element.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#getPermutationSize()">getPermutationSize()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/CDSSequence.html#getPhase()">getPhase()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/CDSSequence.html" title="class in org.biojava3.core.sequence">CDSSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#getPhi(org.biojava.bio.structure.AminoAcid, org.biojava.bio.structure.AminoAcid)">getPhi(AminoAcid, AminoAcid)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">phi angle.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/LUDecomposition.html#getPivot()">getPivot()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/LUDecomposition.html" title="class in org.biojava.bio.structure.jama">LUDecomposition</a></dt>
<dd>
<div class="block">Return pivot permutation vector</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#getPmid()">getPmid()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>
<div class="block">Get the value of PMID field.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getPolymerType()">getPolymerType()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/AlignedPosition.html#getPos(int)">getPos(int)</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/AlignedPosition.html" title="class in org.biojava.bio.structure.gui.util">AlignedPosition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#getPos1()">getPos1()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/AlignedPosition.html#getPos1()">getPos1()</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/AlignedPosition.html" title="class in org.biojava.bio.structure.gui.util">AlignedPosition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#getPos2()">getPos2()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/AlignedPosition.html#getPos2()">getPos2()</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/AlignedPosition.html" title="class in org.biojava.bio.structure.gui.util">AlignedPosition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/FuzzyPoint.html#getPosition()">getPosition()</a></span> - Method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/FuzzyPoint.html" title="class in org.biojava3.core.sequence.location">FuzzyPoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/SimplePoint.html#getPosition()">getPosition()</a></span> - Method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/SimplePoint.html" title="class in org.biojava3.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/Point.html#getPosition()">getPosition()</a></span> - Method in interface org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/Point.html" title="interface in org.biojava3.core.sequence.location.template">Point</a></dt>
<dd>
<div class="block">Returns the position held by this object</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/JFatCatClient.html#getPositionInQueue(java.lang.String, java.lang.String, java.lang.String, java.lang.String)">getPositionInQueue(String, String, String, String)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/JFatCatClient.html" title="class in org.biojava.bio.structure.align.client">JFatCatClient</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/JFatCatClient.html#getPositionInQueue(java.lang.String, java.lang.String, java.lang.String, java.lang.String, int)">getPositionInQueue(String, String, String, String, int)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/JFatCatClient.html" title="class in org.biojava.bio.structure.align.client">JFatCatClient</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStruc.html#getPreciseDistance(org.biojava.bio.structure.Atom, org.biojava.bio.structure.Atom)">getPreciseDistance(Atom, Atom)</a></span> - Static method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStruc.html" title="class in org.biojava.bio.structure.secstruc">SecStruc</a></dt>
<dd>
<div class="block">calculate distance between two atoms.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AFPChainCoordManager.html#getPreferredHeight()">getPreferredHeight()</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AFPChainCoordManager.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AFPChainCoordManager</a></dt>
<dd>
<div class="block">Y coordinate size</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AFPChainCoordManager.html#getPreferredWidth()">getPreferredWidth()</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AFPChainCoordManager.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AFPChainCoordManager</a></dt>
<dd>
<div class="block">X coordinate size</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/MenuCreator.html#getPrintMenuItem()">getPrintMenuItem()</a></span> - Static method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/MenuCreator.html" title="class in org.biojava.bio.structure.align.gui">MenuCreator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getProbability()">getProbability()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">The probability (FATCAT) or Z-score (CE) of the alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/FractionalIdentityInProfileScorer.html#getProfile()">getProfile()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/FractionalIdentityInProfileScorer.html" title="class in org.biojava3.alignment">FractionalIdentityInProfileScorer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/FractionalSimilarityInProfileScorer.html#getProfile()">getProfile()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/FractionalSimilarityInProfileScorer.html" title="class in org.biojava3.alignment">FractionalSimilarityInProfileScorer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/GuideTree.Node.html#getProfile()">getProfile()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/GuideTree.Node.html" title="class in org.biojava3.alignment">GuideTree.Node</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/StandardRescoreRefiner.html#getProfile()">getProfile()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/StandardRescoreRefiner.html" title="class in org.biojava3.alignment">StandardRescoreRefiner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#getProfile()">getProfile()</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava3.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/Aligner.html#getProfile()">getProfile()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/Aligner.html" title="interface in org.biojava3.alignment.template">Aligner</a></dt>
<dd>
<div class="block">Returns the alignment <a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template"><code>Profile</code></a> produced by this alignment algorithm.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/GuideTreeNode.html#getProfile()">getProfile()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/GuideTreeNode.html" title="interface in org.biojava3.alignment.template">GuideTreeNode</a></dt>
<dd>
<div class="block">Returns the profile stored at this node.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/PairInProfileScorer.html#getProfile()">getProfile()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/PairInProfileScorer.html" title="interface in org.biojava3.alignment.template">PairInProfileScorer</a></dt>
<dd>
<div class="block">Returns the <a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template"><code>Profile</code></a> from which the aligned pair originated.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/GuideTree.Node.html#getProfileFuture()">getProfileFuture()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/GuideTree.Node.html" title="class in org.biojava3.alignment">GuideTree.Node</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/GuideTreeNode.html#getProfileFuture()">getProfileFuture()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/GuideTreeNode.html" title="interface in org.biojava3.alignment.template">GuideTreeNode</a></dt>
<dd>
<div class="block">Returns the profile future stored at this node, but does not force the calculation, yet.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/StandardRescoreRefiner.html#getProfileProfileAligner()">getProfileProfileAligner()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/StandardRescoreRefiner.html" title="class in org.biojava3.alignment">StandardRescoreRefiner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/PartitionRefiner.html#getProfileProfileAligner()">getProfileProfileAligner()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/PartitionRefiner.html" title="interface in org.biojava3.alignment.template">PartitionRefiner</a></dt>
<dd>
<div class="block">Returns class type of profile-profile alignment subroutine.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/RescoreRefiner.html#getProfileProfileAligner()">getProfileProfileAligner()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/RescoreRefiner.html" title="interface in org.biojava3.alignment.template">RescoreRefiner</a></dt>
<dd>
<div class="block">Returns class type of profile-profile alignment subroutine.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html#getProgram()">getProgram()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemCompDescriptor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html#getProgram_version()">getProgram_version()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemCompDescriptor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/Alignments.html#getProgressiveAlignment(org.biojava3.alignment.GuideTree, org.biojava3.alignment.Alignments.ProfileProfileAlignerType, org.biojava3.alignment.template.GapPenalty, org.biojava3.alignment.template.SubstitutionMatrix)">getProgressiveAlignment(GuideTree&lt;S, C&gt;, Alignments.ProfileProfileAlignerType, GapPenalty, SubstitutionMatrix&lt;C&gt;)</a></span> - Static method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/Alignments.html" title="class in org.biojava3.alignment">Alignments</a></dt>
<dd>
<div class="block">Factory method to run the profile-profile alignments of a progressive multiple sequence alignment concurrently.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#getProperties()">getProperties()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">return properties.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#getProperties()">getProperties()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">return properties.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/DBReferenceInfo.html#getProperties()">getProperties()</a></span> - Method in class org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/DBReferenceInfo.html" title="class in org.biojava3.core.sequence.features">DBReferenceInfo</a></dt>
<dd>
<div class="block">Get the properties</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#getProperty(java.lang.String)">getProperty(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">get a single property .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#getProperty(java.lang.String)">getProperty(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">get a single property .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/TranscriptSequence.html#getProteinCDSSequences()">getProteinCDSSequences()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/TranscriptSequence.html" title="class in org.biojava3.core.sequence">TranscriptSequence</a></dt>
<dd>
<div class="block">Return a list of protein sequences based on each CDS sequence
 where the phase shift between two CDS sequences is assigned to the
 CDS sequence that starts the triplet.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/RNASequence.html#getProteinSequence()">getProteinSequence()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/RNASequence.html" title="class in org.biojava3.core.sequence">RNASequence</a></dt>
<dd>
<div class="block">Get the ProteinSequence from the RNA sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/RNASequence.html#getProteinSequence(org.biojava3.core.sequence.transcription.TranscriptionEngine)">getProteinSequence(TranscriptionEngine)</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/RNASequence.html" title="class in org.biojava3.core.sequence">RNASequence</a></dt>
<dd>
<div class="block">Get the ProteinSequene from the RNA sequence with user defined
 transcription engine</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/TranscriptSequence.html#getProteinSequence()">getProteinSequence()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/TranscriptSequence.html" title="class in org.biojava3.core.sequence">TranscriptSequence</a></dt>
<dd>
<div class="block">Get the protein sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/TranscriptSequence.html#getProteinSequence(org.biojava3.core.sequence.transcription.TranscriptionEngine)">getProteinSequence(TranscriptionEngine)</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/TranscriptSequence.html" title="class in org.biojava3.core.sequence">TranscriptSequence</a></dt>
<dd>
<div class="block">Get the protein sequence with user defined TranscriptEngine</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.html#getProteinSequenceCreator()">getProteinSequenceCreator()</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.html" title="class in org.biojava3.core.sequence.transcription">TranscriptionEngine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/GeneFeatureHelper.html#getProteinSequences(java.util.Collection)">getProteinSequences(Collection&lt;ChromosomeSequence&gt;)</a></span> - Static method in class org.biojava3.genome.<a href="./org/biojava3/genome/GeneFeatureHelper.html" title="class in org.biojava3.genome">GeneFeatureHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/geneid/GeneIDXMLReader.html#getProteinSequences()">getProteinSequences()</a></span> - Method in class org.biojava3.genome.parsers.geneid.<a href="./org/biojava3/genome/parsers/geneid/GeneIDXMLReader.html" title="class in org.biojava3.genome.parsers.geneid">GeneIDXMLReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/AccesionedLocation.html#getProxySequenceReader()">getProxySequenceReader()</a></span> - Method in interface org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/AccesionedLocation.html" title="interface in org.biojava3.core.sequence.location.template">AccesionedLocation</a></dt>
<dd>
<div class="block">Return the proxy reader used to get sequence for this location.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#getProxySequenceReader()">getProxySequenceReader()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#getPsi(org.biojava.bio.structure.AminoAcid, org.biojava.bio.structure.AminoAcid)">getPsi(AminoAcid, AminoAcid)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">psi angle.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModification.html#getPsimodId()">getPsimodId()</a></span> - Method in interface org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModification.html" title="interface in org.biojava3.protmod">ProteinModification</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationImpl.html#getPsimodId()">getPsimodId()</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationImpl.html" title="class in org.biojava3.protmod">ProteinModificationImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModification.html#getPsimodName()">getPsimodName()</a></span> - Method in interface org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModification.html" title="interface in org.biojava3.protmod">ProteinModification</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationImpl.html#getPsimodName()">getPsimodName()</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationImpl.html" title="class in org.biojava3.protmod">ProteinModificationImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#getPublicationDate()">getPublicationDate()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#getPublisher()">getPublisher()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#getPx()">getPx()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/QRDecomposition.html#getQ()">getQ()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/QRDecomposition.html" title="class in org.biojava.bio.structure.jama">QRDecomposition</a></dt>
<dd>
<div class="block">Generate and return the (economy-sized) orthogonal factor</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/QuantityFeature.html#getQuantities()">getQuantities()</a></span> - Method in class org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/QuantityFeature.html" title="class in org.biojava3.core.sequence.features">QuantityFeature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/FractionalIdentityScorer.html#getQuery()">getQuery()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/FractionalIdentityScorer.html" title="class in org.biojava3.alignment">FractionalIdentityScorer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/FractionalSimilarityScorer.html#getQuery()">getQuery()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/FractionalSimilarityScorer.html" title="class in org.biojava3.alignment">FractionalSimilarityScorer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleProfilePair.html#getQuery()">getQuery()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleProfilePair.html" title="class in org.biojava3.alignment">SimpleProfilePair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleSequencePair.html#getQuery()">getQuery()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleSequencePair.html" title="class in org.biojava3.alignment">SimpleSequencePair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html#getQuery()">getQuery()</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html" title="class in org.biojava3.alignment.template">AbstractPairwiseSequenceAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractProfileProfileAligner.html#getQuery()">getQuery()</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractProfileProfileAligner.html" title="class in org.biojava3.alignment.template">AbstractProfileProfileAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/PairwiseSequenceScorer.html#getQuery()">getQuery()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/PairwiseSequenceScorer.html" title="interface in org.biojava3.alignment.template">PairwiseSequenceScorer</a></dt>
<dd>
<div class="block">Returns the first sequence of the pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/ProfilePair.html#getQuery()">getQuery()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/ProfilePair.html" title="interface in org.biojava3.alignment.template">ProfilePair</a></dt>
<dd>
<div class="block">Returns the first <a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template"><code>Profile</code></a> of the pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/ProfileProfileScorer.html#getQuery()">getQuery()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/ProfileProfileScorer.html" title="interface in org.biojava3.alignment.template">ProfileProfileScorer</a></dt>
<dd>
<div class="block">Returns the first profile of the pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/SequencePair.html#getQuery()">getQuery()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/SequencePair.html" title="interface in org.biojava3.alignment.template">SequencePair</a></dt>
<dd>
<div class="block">Returns the first <a href="./org/biojava3/alignment/template/AlignedSequence.html" title="interface in org.biojava3.alignment.template"><code>AlignedSequence</code></a> of the pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/routines/AlignerHelper.Cut.html#getQueryIndex()">getQueryIndex()</a></span> - Method in class org.biojava3.alignment.routines.<a href="./org/biojava3/alignment/routines/AlignerHelper.Cut.html" title="class in org.biojava3.alignment.routines">AlignerHelper.Cut</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/QRDecomposition.html#getR()">getR()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/QRDecomposition.html" title="class in org.biojava.bio.structure.jama">QRDecomposition</a></dt>
<dd>
<div class="block">Return the upper triangular factor</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/JFatCatClient.html#getRandomSleepTime()">getRandomSleepTime()</a></span> - Static method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/JFatCatClient.html" title="class in org.biojava.bio.structure.align.client">JFatCatClient</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html#getRandomUsername()">getRandomUsername()</a></span> - Static method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html" title="class in org.biojava.bio.structure.align.client">FarmJobRunnable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#getRanges()">getRanges()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/EigenvalueDecomposition.html#getRealEigenvalues()">getRealEigenvalues()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/EigenvalueDecomposition.html" title="class in org.biojava.bio.structure.jama">EigenvalueDecomposition</a></dt>
<dd>
<div class="block">Return the real parts of the eigenvalues</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/edits/Edit.Delete.html#getRealEnd()">getRealEnd()</a></span> - Method in class org.biojava3.core.sequence.edits.<a href="./org/biojava3/core/sequence/edits/Edit.Delete.html" title="class in org.biojava3.core.sequence.edits">Edit.Delete</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/edits/Edit.Delete.html#getRealStart()">getRealStart()</a></span> - Method in class org.biojava3.core.sequence.edits.<a href="./org/biojava3/core/sequence/edits/Edit.Delete.html" title="class in org.biojava3.core.sequence.edits">Edit.Delete</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ProteinModificationIdentifier.html#getRecordAdditionalAttachments()">getRecordAdditionalAttachments()</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ProteinModificationIdentifier.html" title="class in org.biojava3.protmod.structure">ProteinModificationIdentifier</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcid.html#getRecordType()">getRecordType()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcid.html" title="interface in org.biojava.bio.structure">AminoAcid</a></dt>
<dd>
<div class="block">Allows to distinguish between amino acids that are provided
                as ATOM records and a SEQRES records.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcidImpl.html#getRecordType()">getRecordType()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcidImpl.html" title="class in org.biojava.bio.structure">AminoAcidImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ProteinModificationIdentifier.html#getRecordUnidentifiableCompounds()">getRecordUnidentifiableCompounds()</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ProteinModificationIdentifier.html" title="class in org.biojava3.protmod.structure">ProteinModificationIdentifier</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#getReducedStructure(org.biojava.bio.structure.Structure, java.lang.String)">getReducedStructure(Structure, String)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>
<div class="block">Reduce a structure to provide a smaller representation .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#getReducedStructure(org.biojava.bio.structure.Structure, int)">getReducedStructure(Structure, int)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>
<div class="block">Reduce a structure to provide a smaller representation.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#getRef()">getRef()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>
<div class="block">Get the value of REF field.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#getRef_id()">getRef_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getRefChainId()">getRefChainId()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>
<div class="block">Returns the ref chain id value.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#getRefn()">getRefn()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>
<div class="block">Get the value of REFN field.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/MyJmolStatusListener.html#getRegistryInfo()">getRegistryInfo()</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/MyJmolStatusListener.html" title="class in org.biojava.bio.structure.align.gui.jmol">MyJmolStatusListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html#getRelevantSubLocations()">getRelevantSubLocations()</a></span> - Method in class org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava3.core.sequence.location.template">AbstractLocation</a></dt>
<dd>
<div class="block">Returns the normalised list of sub locations i.e. only those locations
 which do not have a sub location.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/Location.html#getRelevantSubLocations()">getRelevantSubLocations()</a></span> - Method in interface org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/Location.html" title="interface in org.biojava3.core.sequence.location.template">Location</a></dt>
<dd>
<div class="block">An extension to <a href="./org/biojava3/core/sequence/location/template/Location.html#getSubLocations()"><code>Location.getSubLocations()</code></a> which returns sub-locations
 of sub-locations; this will continue until it runs out of those locations.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html#getRelevantSubSequence(org.biojava3.core.sequence.template.Sequence)">getRelevantSubSequence(Sequence&lt;C&gt;)</a></span> - Method in class org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava3.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/Location.html#getRelevantSubSequence(org.biojava3.core.sequence.template.Sequence)">getRelevantSubSequence(Sequence&lt;C&gt;)</a></span> - Method in interface org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/Location.html" title="interface in org.biojava3.core.sequence.location.template">Location</a></dt>
<dd>
<div class="block">Will return a SequenceReader object which offers a view of all resolved
 locations i.e. those locations which are not complex and define the
 true Sequence represented</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html#getReplaced(org.biojava.bio.structure.io.mmcif.model.ChemComp)">getReplaced(ChemComp)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html" title="class in org.biojava.bio.structure.io.mmcif">ChemicalComponentDictionary</a></dt>
<dd>
<div class="block">if ChemComp is replacing another one, get the old version
 otherwise return the same ChemComp again.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html#getReplaced(java.lang.String)">getReplaced(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html" title="class in org.biojava.bio.structure.io.mmcif">ChemicalComponentDictionary</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBStatus.html#getReplacement(java.lang.String, boolean, boolean)">getReplacement(String, boolean, boolean)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBStatus.html" title="class in org.biojava.bio.structure">PDBStatus</a></dt>
<dd>
<div class="block">Gets the PDB which superseded oldPdbId.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html#getReplacer(org.biojava.bio.structure.io.mmcif.model.ChemComp)">getReplacer(ChemComp)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html" title="class in org.biojava.bio.structure.io.mmcif">ChemicalComponentDictionary</a></dt>
<dd>
<div class="block">if ChemComp is replaced by another one, get the newer version
 otherwise return the same ChemComp again.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html#getReplacer(java.lang.String)">getReplacer(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html" title="class in org.biojava.bio.structure.io.mmcif">ChemicalComponentDictionary</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html#getReplaces()">getReplaces()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html" title="class in org.biojava.bio.structure.io.mmcif.model">DatabasePDBrev</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBStatus.html#getReplaces(java.lang.String, boolean)">getReplaces(String, boolean)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBStatus.html" title="class in org.biojava.bio.structure">PDBStatus</a></dt>
<dd>
<div class="block">Get the ID of the protein which was made obsolete by newPdbId.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/JFatCatClient.html#getRepresentatives(java.lang.String, int)">getRepresentatives(String, int)</a></span> - Static method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/JFatCatClient.html" title="class in org.biojava.bio.structure.align.client">JFatCatClient</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModification.html#getResidId()">getResidId()</a></span> - Method in interface org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModification.html" title="interface in org.biojava3.protmod">ProteinModification</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationImpl.html#getResidId()">getResidId()</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationImpl.html" title="class in org.biojava3.protmod">ProteinModificationImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModification.html#getResidName()">getResidName()</a></span> - Method in interface org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModification.html" title="interface in org.biojava3.protmod">ProteinModification</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationImpl.html#getResidName()">getResidName()</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationImpl.html" title="class in org.biojava3.protmod">ProteinModificationImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/AtomInfo.html#getResidueName()">getResidueName()</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/AtomInfo.html" title="class in org.biojava.bio.structure.align.gui.jmol">AtomInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/AtomInfo.html#getResidueNumber()">getResidueNumber()</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/AtomInfo.html" title="class in org.biojava.bio.structure.align.gui.jmol">AtomInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#getResidueNumber()">getResidueNumber()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">returns a dynamically created ResidueNumber for the group - this
 contains the chainId, resNum and insCode of the group.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#getResidueNumber()">getResidueNumber()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">returns a dynamically created ResidueNumber for the group - this
 contains the chainId, resNum and insCode of the group.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/StructureGroup.html#getResidueNumber()">getResidueNumber()</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/StructureGroup.html" title="class in org.biojava3.protmod.structure">StructureGroup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getResidueType()">getResidueType()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/chem/ResidueType.html#getResidueTypeFromString(java.lang.String)">getResidueTypeFromString(String)</a></span> - Static method in enum org.biojava.bio.structure.io.mmcif.chem.<a href="./org/biojava/bio/structure/io/mmcif/chem/ResidueType.html" title="enum in org.biojava.bio.structure.io.mmcif.chem">ResidueType</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getResNames()">getResNames()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SSBond.html#getResnum1()">getResnum1()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SSBond.html" title="class in org.biojava.bio.structure">SSBond</a></dt>
<dd>
<div class="block">get residue number for first CYS.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SSBond.html#getResnum2()">getResnum2()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SSBond.html" title="class in org.biojava.bio.structure">SSBond</a></dt>
<dd>
<div class="block">get residue number for second CYS.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#getResolution()">getResolution()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/FuzzyPoint.html#getResolver()">getResolver()</a></span> - Method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/FuzzyPoint.html" title="class in org.biojava3.core.sequence.location">FuzzyPoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/ResourceManager.html#getResourceManager(java.lang.String)">getResourceManager(String)</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/ResourceManager.html" title="class in org.biojava.bio.structure.align.util">ResourceManager</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getResScore()">getResScore()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/PdbPair.html#getReverse()">getReverse()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/PdbPair.html" title="class in org.biojava.bio.structure.align.client">PdbPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/DNASequence.html#getReverse()">getReverse()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/DNASequence.html" title="class in org.biojava3.core.sequence">DNASequence</a></dt>
<dd>
<div class="block">Returns a Sequence which runs in the current reverse order</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/Strand.html#getReverse()">getReverse()</a></span> - Method in enum org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/Strand.html" title="enum in org.biojava3.core.sequence">Strand</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/DNASequence.html#getReverseComplement()">getReverseComplement()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/DNASequence.html" title="class in org.biojava3.core.sequence">DNASequence</a></dt>
<dd>
<div class="block">Delegates to <a href="./org/biojava3/core/sequence/template/AbstractSequence.html#getInverse()"><code>AbstractSequence.getInverse()</code></a> for the reverse complement</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/RNASequence.html#getReverseComplement()">getReverseComplement()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/RNASequence.html" title="class in org.biojava3.core.sequence">RNASequence</a></dt>
<dd>
<div class="block">Get reverse complement view of the sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/Frame.html#getReverseFrames()">getReverseFrames()</a></span> - Static method in enum org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/Frame.html" title="enum in org.biojava3.core.sequence.transcription">Frame</a></dt>
<dd>
<div class="block">Returns all frames which are in the reverse orientation</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/JointFragments.html#getRms()">getRms()</a></span> - Method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/JointFragments.html" title="class in org.biojava.bio.structure.align.helper">JointFragments</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentJoiner.html#getRMS(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], org.biojava.bio.structure.align.helper.JointFragments)">getRMS(Atom[], Atom[], JointFragments)</a></span> - Static method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentJoiner.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentJoiner</a></dt>
<dd>
<div class="block">get the RMS of the JointFragments pair frag</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#getRms()">getRms()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SVDSuperimposer.html#getRMS(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">getRMS(Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SVDSuperimposer.html" title="class in org.biojava.bio.structure">SVDSuperimposer</a></dt>
<dd>
<div class="block">Calculate the RMS (root mean square) deviation of two sets of atoms.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFP.html#getRmsd()">getRmsd()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFP.html" title="class in org.biojava.bio.structure.align.model">AFP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#getRmsd()">getRmsd()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getRmsdCut()">getRmsdCut()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>
<div class="block">The cutoff to be used during AFP detection</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#getRmsdThr()">getRmsdThr()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>
<div class="block">RMSD Threshold</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#getRmsdThrJoin()">getRmsdThrJoin()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>
<div class="block">RMSD threshold for joining of AFPs</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.html#getRnaAminoAcidTranslator()">getRnaAminoAcidTranslator()</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.html" title="class in org.biojava3.core.sequence.transcription">TranscriptionEngine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.html#getRnaCompounds()">getRnaCompounds()</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.html" title="class in org.biojava3.core.sequence.transcription">TranscriptionEngine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/RNACompoundSet.html#getRNACompoundSet()">getRNACompoundSet()</a></span> - Static method in class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/RNACompoundSet.html" title="class in org.biojava3.core.sequence.compound">RNACompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/DNASequence.html#getRNASequence()">getRNASequence()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/DNASequence.html" title="class in org.biojava3.core.sequence">DNASequence</a></dt>
<dd>
<div class="block">Return the RNASequence equivalent of the DNASequence using default Transcription Engine.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/DNASequence.html#getRNASequence(org.biojava3.core.sequence.transcription.TranscriptionEngine)">getRNASequence(TranscriptionEngine)</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/DNASequence.html" title="class in org.biojava3.core.sequence">DNASequence</a></dt>
<dd>
<div class="block">Allow a user to pass in a rules engine to do the DNA to RNA translation</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/DNASequence.html#getRNASequence(org.biojava3.core.sequence.transcription.Frame)">getRNASequence(Frame)</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/DNASequence.html" title="class in org.biojava3.core.sequence">DNASequence</a></dt>
<dd>
<div class="block">Allows the user to pass in the Frame shift.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/DNASequence.html#getRNASequence(org.biojava3.core.sequence.transcription.TranscriptionEngine, org.biojava3.core.sequence.transcription.Frame)">getRNASequence(TranscriptionEngine, Frame)</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/DNASequence.html" title="class in org.biojava3.core.sequence">DNASequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.html#getRnaSequenceCreator()">getRnaSequenceCreator()</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.html" title="class in org.biojava3.core.sequence.transcription">TranscriptionEngine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/views/RnaSequenceView.html#getRnaToDna()">getRnaToDna()</a></span> - Method in class org.biojava3.core.sequence.views.<a href="./org/biojava3/core/sequence/views/RnaSequenceView.html" title="class in org.biojava3.core.sequence.views">RnaSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/GuideTree.html#getRoot()">getRoot()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/GuideTree.html" title="class in org.biojava3.alignment">GuideTree</a></dt>
<dd>
<div class="block">Returns the root <a href="./org/biojava3/alignment/GuideTree.Node.html" title="class in org.biojava3.alignment"><code>GuideTree.Node</code></a> which corresponds to the full multiple sequence alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/HierarchicalClusterer.html#getRoot()">getRoot()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/HierarchicalClusterer.html" title="interface in org.biojava3.alignment.template">HierarchicalClusterer</a></dt>
<dd>
<div class="block">Returns the root node of the tree resulting from this clustering algorithm.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#getRot()">getRot()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SVDSuperimposer.html#getRotation()">getRotation()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SVDSuperimposer.html" title="class in org.biojava.bio.structure">SVDSuperimposer</a></dt>
<dd>
<div class="block">Get the Rotation matrix that is required to superimpose the two atom sets.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#getRotationMatrix()">getRotationMatrix()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>
<div class="block">Gets the rotation matrix from the last call to 
 <a href="./org/biojava/bio/structure/align/ce/CECalculator.html#calc_rmsd(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], int, boolean, boolean)"><code>calc_rmsd</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#getRotationMatrix()">getRotationMatrix()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>
<div class="block">returns the rotation matrix that needs to be applied to structure 2 to rotate on structure 1</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html#getRotMax(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">getRotMax(AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html" title="class in org.biojava.bio.structure.align.util">AFPAlignmentDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/IndexPair.html#getRow()">getRow()</a></span> - Method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/IndexPair.html" title="class in org.biojava.bio.structure.align.helper">IndexPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#getRowDimension()">getRowDimension()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Get row dimension.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#getRowPackedCopy()">getRowPackedCopy()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Make a one-dimensional row packed copy of the internal array.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/Alignable.html#getRows()">getRows()</a></span> - Method in interface org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/Alignable.html" title="interface in org.biojava.bio.structure.align.pairwise">Alignable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html#getRows()">getRows()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">StrCompAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html#getRows()">getRows()</a></span> - Method in class org.biojava3.alignment.aaindex.<a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava3.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/SingularValueDecomposition.html#getS()">getS()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/SingularValueDecomposition.html" title="class in org.biojava.bio.structure.jama">SingularValueDecomposition</a></dt>
<dd>
<div class="block">Return the diagonal matrix of singular values</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/JMatrixPanel.html#getSaturation()">getSaturation()</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/JMatrixPanel.html" title="class in org.biojava.bio.structure.gui">JMatrixPanel</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/DefaultMatrixMapper.html#getSaturation()">getSaturation()</a></span> - Method in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/DefaultMatrixMapper.html" title="class in org.biojava.bio.structure.gui.util.color">DefaultMatrixMapper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/MenuCreator.html#getSaveAlignmentMenuItem(org.biojava.bio.structure.align.model.AFPChain)">getSaveAlignmentMenuItem(AFPChain)</a></span> - Static method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/MenuCreator.html" title="class in org.biojava.bio.structure.align.gui">MenuCreator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#getSaveOutputDir()">getSaveOutputDir()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/JMatrixPanel.html#getScale()">getScale()</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/JMatrixPanel.html" title="class in org.biojava.bio.structure.gui">JMatrixPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/SequenceDisplay.html#getScale()">getScale()</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/SequenceDisplay.html" title="class in org.biojava.bio.structure.gui">SequenceDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html#getScale()">getScale()</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html" title="class in org.biojava.bio.structure.gui.util">SequenceScalePanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/BigSqrt.html#getScale()">getScale()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/BigSqrt.html" title="class in org.biojava.bio.structure.secstruc">BigSqrt</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html#getScale()">getScale()</a></span> - Method in class org.biojava3.alignment.aaindex.<a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava3.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/GuiWrapper.html#getScaleableMatrixPanel(org.biojava.bio.structure.jama.Matrix)">getScaleableMatrixPanel(Matrix)</a></span> - Static method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/GuiWrapper.html" title="class in org.biojava.bio.structure.align.ce">GuiWrapper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/JMatrixPanel.html#getScalevalue()">getScalevalue()</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/JMatrixPanel.html" title="class in org.biojava.bio.structure.gui">JMatrixPanel</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/DefaultMatrixMapper.html#getScalevalue()">getScalevalue()</a></span> - Method in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/DefaultMatrixMapper.html" title="class in org.biojava.bio.structure.gui.util.color">DefaultMatrixMapper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/ScopInstallationInstance.html#getSCOP()">getSCOP()</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/ScopInstallationInstance.html" title="class in org.biojava.bio.structure.gui.util">ScopInstallationInstance</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopFactory.html#getSCOP()">getSCOP()</a></span> - Static method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopFactory.html" title="class in org.biojava.bio.structure.scop">ScopFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/ScopDescriptions.html#getScopDescription()">getScopDescription()</a></span> - Method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/ScopDescriptions.html" title="class in org.biojava.bio.structure.scop.server">ScopDescriptions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html#getScopDescriptionBySunid(int)">getScopDescriptionBySunid(int)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">RemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDatabase.html#getScopDescriptionBySunid(int)">getScopDescriptionBySunid(int)</a></span> - Method in interface org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDatabase.html" title="interface in org.biojava.bio.structure.scop">ScopDatabase</a></dt>
<dd>
<div class="block">Return the SCOP description for a node in the hierarchy by its "sunid" id.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#getScopDescriptionBySunid(int)">getScopDescriptionBySunid(int)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/XMLUtil.html#getScopDescriptionFromXML(java.lang.String)">getScopDescriptionFromXML(String)</a></span> - Static method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/XMLUtil.html" title="class in org.biojava.bio.structure.scop.server">XMLUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/XMLUtil.html#getScopDescriptionsXML(java.util.List)">getScopDescriptionsXML(List&lt;ScopDescription&gt;)</a></span> - Static method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/XMLUtil.html" title="class in org.biojava.bio.structure.scop.server">XMLUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/XMLUtil.html#getScopDescriptionXML(org.biojava.bio.structure.scop.ScopDescription)">getScopDescriptionXML(ScopDescription)</a></span> - Static method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/XMLUtil.html" title="class in org.biojava.bio.structure.scop.server">XMLUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/ScopDomains.html#getScopDomain()">getScopDomain()</a></span> - Method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/ScopDomains.html" title="class in org.biojava.bio.structure.scop.server">ScopDomains</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/XMLUtil.html#getScopDomainFromXML(java.lang.String)">getScopDomainFromXML(String)</a></span> - Static method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/XMLUtil.html" title="class in org.biojava.bio.structure.scop.server">XMLUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html#getScopDomainsBySunid(java.lang.Integer)">getScopDomainsBySunid(Integer)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">RemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDatabase.html#getScopDomainsBySunid(java.lang.Integer)">getScopDomainsBySunid(Integer)</a></span> - Method in interface org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDatabase.html" title="interface in org.biojava.bio.structure.scop">ScopDatabase</a></dt>
<dd>
<div class="block">Get a SCOP domain by its sunid</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#getScopDomainsBySunid(java.lang.Integer)">getScopDomainsBySunid(Integer)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/XMLUtil.html#getScopDomainsXML(java.util.List)">getScopDomainsXML(List&lt;ScopDomain&gt;)</a></span> - Static method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/XMLUtil.html" title="class in org.biojava.bio.structure.scop.server">XMLUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/XMLUtil.html#getScopDomainXML(org.biojava.bio.structure.scop.ScopDomain)">getScopDomainXML(ScopDomain)</a></span> - Static method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/XMLUtil.html" title="class in org.biojava.bio.structure.scop.server">XMLUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#getScopId()">getScopId()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#getScopInstallation()">getScopInstallation()</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html#getScopNode(int)">getScopNode(int)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">RemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDatabase.html#getScopNode(int)">getScopNode(int)</a></span> - Method in interface org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDatabase.html" title="interface in org.biojava.bio.structure.scop">ScopDatabase</a></dt>
<dd>
<div class="block">Access a particular ScopNode.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#getScopNode(int)">getScopNode(int)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/ScopNodes.html#getScopNode()">getScopNode()</a></span> - Method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/ScopNodes.html" title="class in org.biojava.bio.structure.scop.server">ScopNodes</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/XMLUtil.html#getScopNodeFromXML(java.lang.String)">getScopNodeFromXML(String)</a></span> - Static method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/XMLUtil.html" title="class in org.biojava.bio.structure.scop.server">XMLUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/XMLUtil.html#getScopNodesXML(java.util.List)">getScopNodesXML(List&lt;ScopNode&gt;)</a></span> - Static method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/XMLUtil.html" title="class in org.biojava.bio.structure.scop.server">XMLUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/XMLUtil.html#getScopNodeXML(org.biojava.bio.structure.scop.ScopNode)">getScopNodeXML(ScopNode)</a></span> - Static method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/XMLUtil.html" title="class in org.biojava.bio.structure.scop.server">XMLUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html#getScopVersion()">getScopVersion()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">RemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDatabase.html#getScopVersion()">getScopVersion()</a></span> - Method in interface org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDatabase.html" title="interface in org.biojava.bio.structure.scop">ScopDatabase</a></dt>
<dd>
<div class="block">Returns the SCOP version</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#getScopVersion()">getScopVersion()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFP.html#getScore()">getScore()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFP.html" title="class in org.biojava.bio.structure.align.model">AFP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/Alignable.html#getScore()">getScore()</a></span> - Method in interface org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/Alignable.html" title="interface in org.biojava.bio.structure.align.pairwise">Alignable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#getScore()">getScore()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>
<div class="block">the alignment score</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html#getScore()">getScore()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">StrCompAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Domain.html#getScore()">getScore()</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Domain.html" title="class in org.biojava.bio.structure.domain.pdp">Domain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Segment.html#getScore()">getScore()</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Segment.html" title="class in org.biojava.bio.structure.domain.pdp">Segment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/FractionalIdentityScorer.html#getScore()">getScore()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/FractionalIdentityScorer.html" title="class in org.biojava3.alignment">FractionalIdentityScorer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/FractionalSimilarityScorer.html#getScore()">getScore()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/FractionalSimilarityScorer.html" title="class in org.biojava3.alignment">FractionalSimilarityScorer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/StandardRescoreRefiner.html#getScore()">getScore()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/StandardRescoreRefiner.html" title="class in org.biojava3.alignment">StandardRescoreRefiner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#getScore()">getScore()</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava3.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/Scorer.html#getScore()">getScore()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/Scorer.html" title="interface in org.biojava3.alignment.template">Scorer</a></dt>
<dd>
<div class="block">Returns score resulting from algorithm.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/GuideTree.html#getScoreMatrix()">getScoreMatrix()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/GuideTree.html" title="class in org.biojava3.alignment">GuideTree</a></dt>
<dd>
<div class="block">Returns the similarity matrix used to construct this guide tree.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#getScoreMatrix()">getScoreMatrix()</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava3.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/MatrixAligner.html#getScoreMatrix()">getScoreMatrix()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/MatrixAligner.html" title="interface in org.biojava3.alignment.template">MatrixAligner</a></dt>
<dd>
<div class="block">Returns the entire score matrix built during alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#getScoreMatrix(java.lang.String)">getScoreMatrix(String)</a></span> - Static method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>
<div class="block">get a ScoreMatrix based on its string name</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#getScoreMatrixAsString()">getScoreMatrixAsString()</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava3.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/MatrixAligner.html#getScoreMatrixAsString()">getScoreMatrixAsString()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/MatrixAligner.html" title="interface in org.biojava3.alignment.template">MatrixAligner</a></dt>
<dd>
<div class="block">Returns a depiction of the score matrix as a <a href="http://java.sun.com/j2se/1.6.0/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang"><code>String</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#getScoreMatrixDimensions()">getScoreMatrixDimensions()</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava3.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html#getScoreMatrixDimensions()">getScoreMatrixDimensions()</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html" title="class in org.biojava3.alignment.template">AbstractPairwiseSequenceAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractProfileProfileAligner.html#getScoreMatrixDimensions()">getScoreMatrixDimensions()</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractProfileProfileAligner.html" title="class in org.biojava3.alignment.template">AbstractProfileProfileAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/ronn/ORonnModel.html#getScores(java.io.File)">getScores(File)</a></span> - Method in class org.biojava3.ronn.<a href="./org/biojava3/ronn/ORonnModel.html" title="class in org.biojava3.ronn">ORonnModel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#getScoringStrategy()">getScoringStrategy()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getSecretion()">getSecretion()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcid.html#getSecStruc()">getSecStruc()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcid.html" title="interface in org.biojava.bio.structure">AminoAcid</a></dt>
<dd>
<div class="block">get secondary structure data .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcidImpl.html#getSecStruc()">getSecStruc()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcidImpl.html" title="class in org.biojava.bio.structure">AminoAcidImpl</a></dt>
<dd>
<div class="block">get secondary structure data .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#getSeedFragmentLength()">getSeedFragmentLength()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#getSeedRmsdCutoff()">getSeedRmsdCutoff()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Domain.html#getSegmentAtPos(int)">getSegmentAtPos(int)</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Domain.html" title="class in org.biojava.bio.structure.domain.pdp">Domain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Domain.html#getSegments()">getSegments()</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Domain.html" title="class in org.biojava.bio.structure.domain.pdp">Domain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/JMatrixPanel.html#getSelectedAlignmentPos()">getSelectedAlignmentPos()</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/JMatrixPanel.html" title="class in org.biojava.bio.structure.gui">JMatrixPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html#getSelectedAlignmentPos()">getSelectedAlignmentPos()</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html" title="class in org.biojava.bio.structure.gui">ScaleableMatrixPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/structure/gui/JmolViewerImpl.html#getSelection()">getSelection()</a></span> - Method in class org.biojava3.structure.gui.<a href="./org/biojava3/structure/gui/JmolViewerImpl.html" title="class in org.biojava3.structure.gui">JmolViewerImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/structure/gui/Selection.html#getSelection()">getSelection()</a></span> - Method in interface org.biojava3.structure.gui.<a href="./org/biojava3/structure/gui/Selection.html" title="interface in org.biojava3.structure.gui">Selection</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/structure/gui/SelectionImpl.html#getSelection()">getSelection()</a></span> - Method in class org.biojava3.structure.gui.<a href="./org/biojava3/structure/gui/SelectionImpl.html" title="class in org.biojava3.structure.gui">SelectionImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/structure/gui/StructureViewer.html#getSelection()">getSelection()</a></span> - Method in interface org.biojava3.structure.gui.<a href="./org/biojava3/structure/gui/StructureViewer.html" title="interface in org.biojava3.structure.gui">StructureViewer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/DBSearchGUI.html#getSelectPDBPanel()">getSelectPDBPanel()</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/DBSearchGUI.html" title="class in org.biojava.bio.structure.align.gui">DBSearchGUI</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#getSeq_align_beg()">getSeq_align_beg()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#getSeq_align_end()">getSeq_align_end()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#getSeq_id()">getSeq_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#getSeq_id()">getSeq_id()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#getSeqBegin()">getSeqBegin()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Initial sequence number of the PDB sequence segment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#getSeqEnd()">getSeqEnd()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Ending sequence number   of the PDB sequence segment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/SequenceTools.html#getSeqeunceFromString(java.lang.String)">getSeqeunceFromString(String)</a></span> - Method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/SequenceTools.html" title="class in org.biojava3.core.util">SequenceTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ResidueNumber.html#getSeqNum()">getSeqNum()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ResidueNumber.html" title="class in org.biojava.bio.structure">ResidueNumber</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AFPChainCoordManager.html#getSeqPos(int, java.awt.Point)">getSeqPos(int, Point)</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AFPChainCoordManager.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AFPChainCoordManager</a></dt>
<dd>
<div class="block">Convert from a X position in the JPanel to alignment position</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/CoordManager.html#getSeqPos(int)">getSeqPos(int)</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/CoordManager.html" title="class in org.biojava.bio.structure.gui.util">CoordManager</a></dt>
<dd>
<div class="block">start counting at 0...</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getSeqResGroup(int)">getSeqResGroup(int)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">return the Group at position X.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getSeqResGroup(int)">getSeqResGroup(int)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">return the Group at position X.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getSeqResGroups(java.lang.String)">getSeqResGroups(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Return a List of all groups of a special type (e.g. amino,
 hetatm, nucleotide).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getSeqResGroups()">getSeqResGroups()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Return all groups of this chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getSeqResGroups(java.lang.String)">getSeqResGroups(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Return a List of all groups of a special type (e.g. amino,
 hetatm, nucleotide).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getSeqResGroups()">getSeqResGroups()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Return all groups of this chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getSeqResLength()">getSeqResLength()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Returns the number of groups in the SEQRES records of the chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getSeqResLength()">getSeqResLength()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getSeqResSequence()">getSeqResSequence()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Get the sequence for all amino acids as it is specified in the SEQRES residues.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getSeqResSequence()">getSeqResSequence()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Get the sequence for all amino acids as it is specified in the SEQRES residues.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getSequence()">getSequence()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>use getAtomSequence instead</i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getSequence()">getSequence()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>use getAtomSequence instead</i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/DNASequenceCreator.html#getSequence(java.lang.String, long)">getSequence(String, long)</a></span> - Method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/DNASequenceCreator.html" title="class in org.biojava3.core.sequence.io">DNASequenceCreator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/DNASequenceCreator.html#getSequence(org.biojava3.core.sequence.template.ProxySequenceReader, long)">getSequence(ProxySequenceReader&lt;NucleotideCompound&gt;, long)</a></span> - Method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/DNASequenceCreator.html" title="class in org.biojava3.core.sequence.io">DNASequenceCreator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/DNASequenceCreator.html#getSequence(java.util.List)">getSequence(List&lt;NucleotideCompound&gt;)</a></span> - Method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/DNASequenceCreator.html" title="class in org.biojava3.core.sequence.io">DNASequenceCreator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/FastaSequenceParser.html#getSequence(java.io.DataInput, int)">getSequence(DataInput, int)</a></span> - Method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/FastaSequenceParser.html" title="class in org.biojava3.core.sequence.io">FastaSequenceParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/FileProxyDNASequenceCreator.html#getSequence(java.lang.String, long)">getSequence(String, long)</a></span> - Method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/FileProxyDNASequenceCreator.html" title="class in org.biojava3.core.sequence.io">FileProxyDNASequenceCreator</a></dt>
<dd>
<div class="block">Even though we are passing in the sequence we really only care about the length of the sequence and the offset
 index in the fasta file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/FileProxyDNASequenceCreator.html#getSequence(org.biojava3.core.sequence.template.ProxySequenceReader, long)">getSequence(ProxySequenceReader&lt;NucleotideCompound&gt;, long)</a></span> - Method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/FileProxyDNASequenceCreator.html" title="class in org.biojava3.core.sequence.io">FileProxyDNASequenceCreator</a></dt>
<dd>
<div class="block">Should be able to extend the same concept to a remote URL call or database connection.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/FileProxyDNASequenceCreator.html#getSequence(java.util.List)">getSequence(List&lt;NucleotideCompound&gt;)</a></span> - Method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/FileProxyDNASequenceCreator.html" title="class in org.biojava3.core.sequence.io">FileProxyDNASequenceCreator</a></dt>
<dd>
<div class="block">Not sure of use case and currently not supported</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/FileProxyProteinSequenceCreator.html#getSequence(java.lang.String, long)">getSequence(String, long)</a></span> - Method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/FileProxyProteinSequenceCreator.html" title="class in org.biojava3.core.sequence.io">FileProxyProteinSequenceCreator</a></dt>
<dd>
<div class="block">Even though we are passing in the sequence we really only care about the length of the sequence and the offset
 index in the fasta file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/FileProxyProteinSequenceCreator.html#getSequence(org.biojava3.core.sequence.template.ProxySequenceReader, long)">getSequence(ProxySequenceReader&lt;AminoAcidCompound&gt;, long)</a></span> - Method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/FileProxyProteinSequenceCreator.html" title="class in org.biojava3.core.sequence.io">FileProxyProteinSequenceCreator</a></dt>
<dd>
<div class="block">Should be able to extend the same concept to a remote URL call or database connection.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/FileProxyProteinSequenceCreator.html#getSequence(java.util.List)">getSequence(List&lt;AminoAcidCompound&gt;)</a></span> - Method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/FileProxyProteinSequenceCreator.html" title="class in org.biojava3.core.sequence.io">FileProxyProteinSequenceCreator</a></dt>
<dd>
<div class="block">Not sure of use case and currently not supported</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/ProteinSequenceCreator.html#getSequence(java.lang.String, long)">getSequence(String, long)</a></span> - Method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/ProteinSequenceCreator.html" title="class in org.biojava3.core.sequence.io">ProteinSequenceCreator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/ProteinSequenceCreator.html#getSequence(java.util.List)">getSequence(List&lt;AminoAcidCompound&gt;)</a></span> - Method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/ProteinSequenceCreator.html" title="class in org.biojava3.core.sequence.io">ProteinSequenceCreator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/ProteinSequenceCreator.html#getSequence(org.biojava3.core.sequence.template.ProxySequenceReader, long)">getSequence(ProxySequenceReader&lt;AminoAcidCompound&gt;, long)</a></span> - Method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/ProteinSequenceCreator.html" title="class in org.biojava3.core.sequence.io">ProteinSequenceCreator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/RNASequenceCreator.html#getSequence(java.lang.String, long)">getSequence(String, long)</a></span> - Method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/RNASequenceCreator.html" title="class in org.biojava3.core.sequence.io">RNASequenceCreator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/RNASequenceCreator.html#getSequence(org.biojava3.core.sequence.template.ProxySequenceReader, long)">getSequence(ProxySequenceReader&lt;NucleotideCompound&gt;, long)</a></span> - Method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/RNASequenceCreator.html" title="class in org.biojava3.core.sequence.io">RNASequenceCreator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/RNASequenceCreator.html#getSequence(java.util.List)">getSequence(List&lt;NucleotideCompound&gt;)</a></span> - Method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/RNASequenceCreator.html" title="class in org.biojava3.core.sequence.io">RNASequenceCreator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/template/SequenceCreatorInterface.html#getSequence(java.lang.String, long)">getSequence(String, long)</a></span> - Method in interface org.biojava3.core.sequence.io.template.<a href="./org/biojava3/core/sequence/io/template/SequenceCreatorInterface.html" title="interface in org.biojava3.core.sequence.io.template">SequenceCreatorInterface</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/template/SequenceCreatorInterface.html#getSequence(org.biojava3.core.sequence.template.ProxySequenceReader, long)">getSequence(ProxySequenceReader&lt;C&gt;, long)</a></span> - Method in interface org.biojava3.core.sequence.io.template.<a href="./org/biojava3/core/sequence/io/template/SequenceCreatorInterface.html" title="interface in org.biojava3.core.sequence.io.template">SequenceCreatorInterface</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/template/SequenceCreatorInterface.html#getSequence(java.util.List)">getSequence(List&lt;C&gt;)</a></span> - Method in interface org.biojava3.core.sequence.io.template.<a href="./org/biojava3/core/sequence/io/template/SequenceCreatorInterface.html" title="interface in org.biojava3.core.sequence.io.template">SequenceCreatorInterface</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/template/SequenceParserInterface.html#getSequence(java.io.DataInput, int)">getSequence(DataInput, int)</a></span> - Method in interface org.biojava3.core.sequence.io.template.<a href="./org/biojava3/core/sequence/io/template/SequenceParserInterface.html" title="interface in org.biojava3.core.sequence.io.template">SequenceParserInterface</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/SequenceLocation.html#getSequence()">getSequence()</a></span> - Method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/SequenceLocation.html" title="class in org.biojava3.core.sequence.location">SequenceLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/data/sequence/FastaSequence.html#getSequence()">getSequence()</a></span> - Method in class org.biojava3.data.sequence.<a href="./org/biojava3/data/sequence/FastaSequence.html" title="class in org.biojava3.data.sequence">FastaSequence</a></dt>
<dd>
<div class="block">Gets the value of sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/GeneSequence.html#getSequence5PrimeTo3Prime()">getSequence5PrimeTo3Prime()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/GeneSequence.html" title="class in org.biojava3.core.sequence">GeneSequence</a></dt>
<dd>
<div class="block">Try to give method clarity where you want a DNASequence coding in the 5' to 3' direction
 Returns the DNASequence representative of the 5' and 3' reading based on strand</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleAlignedSequence.html#getSequenceAsString()">getSequenceAsString()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleAlignedSequence.html" title="class in org.biojava3.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html#getSequenceAsString()">getSequenceAsString()</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava3.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html#getSequenceAsString(java.lang.Integer, java.lang.Integer, org.biojava3.core.sequence.Strand)">getSequenceAsString(Integer, Integer, Strand)</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava3.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/StringProxySequenceReader.html#getSequenceAsString()">getSequenceAsString()</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/StringProxySequenceReader.html#getSequenceAsString(java.lang.Integer, java.lang.Integer, org.biojava3.core.sequence.Strand)">getSequenceAsString(Integer, Integer, Strand)</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html#getSequenceAsString()">getSequenceAsString()</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html#getSequenceAsString(java.lang.Integer, java.lang.Integer, org.biojava3.core.sequence.Strand)">getSequenceAsString(Integer, Integer, Strand)</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/ArrayListSequenceReader.html#getSequenceAsString()">getSequenceAsString()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava3.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/ArrayListSequenceReader.html#getSequenceAsString(java.lang.Integer, java.lang.Integer, org.biojava3.core.sequence.Strand)">getSequenceAsString(Integer, Integer, Strand)</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava3.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/BitSequenceReader.html#getSequenceAsString()">getSequenceAsString()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava3.core.sequence.storage">BitSequenceReader</a></dt>
<dd>
<div class="block">Returns the sequence as a String</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/JoiningSequenceReader.html#getSequenceAsString()">getSequenceAsString()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava3.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/SequenceAsStringHelper.html#getSequenceAsString(java.util.List, org.biojava3.core.sequence.template.CompoundSet, java.lang.Integer, java.lang.Integer, org.biojava3.core.sequence.Strand)">getSequenceAsString(List&lt;C&gt;, CompoundSet&lt;C&gt;, Integer, Integer, Strand)</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/SequenceAsStringHelper.html" title="class in org.biojava3.core.sequence.storage">SequenceAsStringHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html#getSequenceAsString()">getSequenceAsString()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava3.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Delegates to <a href="./org/biojava3/core/sequence/template/SequenceMixin.html#toList(org.biojava3.core.sequence.template.Sequence)"><code>SequenceMixin.toList(org.biojava3.core.sequence.template.Sequence)</code></a></div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#getSequenceAsString(java.lang.Integer, java.lang.Integer, org.biojava3.core.sequence.Strand)">getSequenceAsString(Integer, Integer, Strand)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#getSequenceAsString()">getSequenceAsString()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>
<div class="block">Default case is to assume strand is positive because only CDSSequence can be either positive or negative Strand.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/Sequence.html#getSequenceAsString()">getSequenceAsString()</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a></dt>
<dd>
<div class="block">Returns the String representation of the Sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceProxyView.html#getSequenceAsString()">getSequenceAsString()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceProxyView.html" title="class in org.biojava3.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/views/ComplementSequenceView.html#getSequenceAsString()">getSequenceAsString()</a></span> - Method in class org.biojava3.core.sequence.views.<a href="./org/biojava3/core/sequence/views/ComplementSequenceView.html" title="class in org.biojava3.core.sequence.views">ComplementSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/views/ReversedSequenceView.html#getSequenceAsString()">getSequenceAsString()</a></span> - Method in class org.biojava3.core.sequence.views.<a href="./org/biojava3/core/sequence/views/ReversedSequenceView.html" title="class in org.biojava3.core.sequence.views">ReversedSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/views/RnaSequenceView.html#getSequenceAsString()">getSequenceAsString()</a></span> - Method in class org.biojava3.core.sequence.views.<a href="./org/biojava3/core/sequence/views/RnaSequenceView.html" title="class in org.biojava3.core.sequence.views">RnaSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/SequenceOptimizationHints.html#getSequenceCollection()">getSequenceCollection()</a></span> - Static method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/SequenceOptimizationHints.html" title="class in org.biojava3.core.sequence">SequenceOptimizationHints</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleAlignedSequence.html#getSequenceIndexAt(int)">getSequenceIndexAt(int)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleAlignedSequence.html" title="class in org.biojava3.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AlignedSequence.html#getSequenceIndexAt(int)">getSequenceIndexAt(int)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AlignedSequence.html" title="interface in org.biojava3.alignment.template">AlignedSequence</a></dt>
<dd>
<div class="block">Returns the index in the original <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a> corresponding to the given index within an alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/GuideTree.html#getSequences()">getSequences()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/GuideTree.html" title="class in org.biojava3.alignment">GuideTree</a></dt>
<dd>
<div class="block">Returns the <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a>s which make up the leaves of this tree.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#getSequenceScore()">getSequenceScore()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>
<div class="block">Provide place holder for a metric that indicate a score associated with the sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/SequenceOptimizationHints.html#getSequenceUsage()">getSequenceUsage()</a></span> - Static method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/SequenceOptimizationHints.html" title="class in org.biojava3.core.sequence">SequenceOptimizationHints</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#getSeqWeight()">getSeqWeight()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>
<div class="block">Should sequence conservation be considered as part of the alignment?</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SSBond.html#getSerNum()">getSerNum()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SSBond.html" title="class in org.biojava.bio.structure">SSBond</a></dt>
<dd>
<div class="block">set serial number of this SSBOND in PDB file</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#getServer()">getServer()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html#getServer()">getServer()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">RemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#getShift()">getShift()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>
<div class="block">Gets the shift from the last call to 
 <a href="./org/biojava/bio/structure/align/ce/CECalculator.html#calc_rmsd(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], int, boolean, boolean)"><code>calc_rmsd</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#getShift()">getShift()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>
<div class="block">returns the shift vector that has to be applied on structure to to shift on structure one</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/AminoAcidComposition.html#getShorName()">getShorName()</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/AminoAcidComposition.html" title="class in org.biojava3.aaproperties.xml">AminoAcidComposition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/AbstractFeature.html#getShortDescription()">getShortDescription()</a></span> - Method in class org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/AbstractFeature.html" title="class in org.biojava3.core.sequence.features">AbstractFeature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/FeatureInterface.html#getShortDescription()">getShortDescription()</a></span> - Method in interface org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/FeatureInterface.html" title="interface in org.biojava3.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">Get the short description that can be used to describe the feature</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/CodonCompound.html#getShortName()">getShortName()</a></span> - Method in class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/CodonCompound.html" title="class in org.biojava3.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/NucleotideCompound.html#getShortName()">getShortName()</a></span> - Method in class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompound.html#getShortName()">getShortName()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompound.html" title="class in org.biojava3.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/Compound.html#getShortName()">getShortName()</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/CaseInsensitiveCompound.html#getShortName()">getShortName()</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava3.core.sequence.transcription">CaseInsensitiveCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/Table.Codon.html#getShortName()">getShortName()</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/Table.Codon.html" title="class in org.biojava3.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#getShowDBresult()">getShowDBresult()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/MenuCreator.html#getShowPDBMenuItem()">getShowPDBMenuItem()</a></span> - Static method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/MenuCreator.html" title="class in org.biojava.bio.structure.align.gui">MenuCreator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getSimilarity()">getSimilarity()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">Returns the similarity score for the alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractScorer.html#getSimilarity()">getSimilarity()</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractScorer.html" title="class in org.biojava3.alignment.template">AbstractScorer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractScorer.html#getSimilarity(double)">getSimilarity(double)</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractScorer.html" title="class in org.biojava3.alignment.template">AbstractScorer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/Scorer.html#getSimilarity()">getSimilarity()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/Scorer.html" title="interface in org.biojava3.alignment.template">Scorer</a></dt>
<dd>
<div class="block">Returns score as a similarity between 0.0 and 1.0.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/Scorer.html#getSimilarity(double)">getSimilarity(double)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/Scorer.html" title="interface in org.biojava3.alignment.template">Scorer</a></dt>
<dd>
<div class="block">Returns score as a similarity between 0.0 and scale.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getSimilarity1()">getSimilarity1()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>use getCoverage1() instead</i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getSimilarity2()">getSimilarity2()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>use getCoverage2() instead</i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/SingularValueDecomposition.html#getSingularValues()">getSingularValues()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/SingularValueDecomposition.html" title="class in org.biojava.bio.structure.jama">SingularValueDecomposition</a></dt>
<dd>
<div class="block">Return the one-dimensional array of singular values</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Site.html#getSiteID()">getSiteID()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Site.html" title="class in org.biojava.bio.structure">Site</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#getSites()">getSites()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#getSites()">getSites()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Domain.html#getSize()">getSize()</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Domain.html" title="class in org.biojava.bio.structure.domain.pdp">Domain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleProfile.html#getSize()">getSize()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleProfile.html" title="class in org.biojava3.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/Profile.html#getSize()">getSize()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns the number of rows in this profile.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/MultipleSequenceAlignment.html#getSize()">getSize()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/MultipleSequenceAlignment.html" title="class in org.biojava3.core.sequence">MultipleSequenceAlignment</a></dt>
<dd>
<div class="block">Get the number of sequences in the MSA</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/LightweightProfile.html#getSize()">getSize()</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/LightweightProfile.html" title="interface in org.biojava3.core.sequence.template">LightweightProfile</a></dt>
<dd>
<div class="block">Returns the number of rows in this profile.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getSolvent_model_details()">getSolvent_model_details()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getSolvent_model_param_bsol()">getSolvent_model_param_bsol()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#getSolvent_model_param_ksol()">getSolvent_model_param_ksol()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/AbstractFeature.html#getSource()">getSource()</a></span> - Method in class org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/AbstractFeature.html" title="class in org.biojava3.core.sequence.features">AbstractFeature</a></dt>
<dd>
<div class="block">The feature source</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/FeatureInterface.html#getSource()">getSource()</a></span> - Method in interface org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/FeatureInterface.html" title="interface in org.biojava3.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">The source of the feature.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#getSource()">getSource()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>
<div class="block">Added support for the source of this sequence for GFF3 export
 If a sub sequence doesn't have  source then check for parent source</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getSparse()">getSparse()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#getSpeciesId()">getSpeciesId()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html#getSrc_method()">getSrc_method()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model">Entity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#getSSBonds()">getSSBonds()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">get the list of SSBonds as they have been defined in the PDB files</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#getSSBonds()">getSSBonds()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">get the list of SSBonds as they have been defined in the PDB files</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleAlignedSequence.html#getStart()">getStart()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleAlignedSequence.html" title="class in org.biojava3.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AlignedSequence.html#getStart()">getStart()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AlignedSequence.html" title="interface in org.biojava3.alignment.template">AlignedSequence</a></dt>
<dd>
<div class="block">Returns the <a href="./org/biojava3/core/sequence/location/template/Point.html" title="interface in org.biojava3.core.sequence.location.template"><code>Point</code></a> within an alignment of the first element of the original <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/ProfileView.html#getStart()">getStart()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/ProfileView.html" title="interface in org.biojava3.alignment.template">ProfileView</a></dt>
<dd>
<div class="block">Returns the column index of the viewed <a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template"><code>Profile</code></a> corresponding to the first element in this view</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/edits/Edit.AbstractEdit.html#getStart()">getStart()</a></span> - Method in class org.biojava3.core.sequence.edits.<a href="./org/biojava3/core/sequence/edits/Edit.AbstractEdit.html" title="class in org.biojava3.core.sequence.edits">Edit.AbstractEdit</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html#getStart()">getStart()</a></span> - Method in class org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava3.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/Location.html#getStart()">getStart()</a></span> - Method in interface org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/Location.html" title="interface in org.biojava3.core.sequence.location.template">Location</a></dt>
<dd>
<div class="block">Start of the location</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/TranscriptSequence.html#getStartCodonSequence()">getStartCodonSequence()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/TranscriptSequence.html" title="class in org.biojava3.core.sequence">TranscriptSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#getStartPage()">getStartPage()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html#getStatus()">getStatus()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html" title="class in org.biojava.bio.structure.io.mmcif.model">DatabasePDBrev</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBStatus.html#getStatus(java.lang.String)">getStatus(String)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBStatus.html" title="class in org.biojava.bio.structure">PDBStatus</a></dt>
<dd>
<div class="block">Get the status of the PDB in question.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBStatus.html#getStatus(java.lang.String[])">getStatus(String[])</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBStatus.html" title="class in org.biojava.bio.structure">PDBStatus</a></dt>
<dd>
<div class="block">Get the status of the a collection of PDBs in question in a single query.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/JmolPanel.html#getStatusListener()">getStatusListener()</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/JmolPanel.html" title="class in org.biojava.bio.structure.align.gui.jmol">JmolPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#getStepSize()">getStepSize()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>
<div class="block">how many pairs should be requested for alignment from server?</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/TranscriptSequence.html#getStopCodonSequence()">getStopCodonSequence()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/TranscriptSequence.html" title="class in org.biojava3.core.sequence">TranscriptSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileParsingParameters.html#getStoreEmptySeqRes()">getStoreEmptySeqRes()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileParsingParameters.html" title="class in org.biojava.bio.structure.io">FileParsingParameters</a></dt>
<dd>
<div class="block">a flag to detrermine if SEQRES should be stored, even if alignSeqREs is disabled.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getStrain()">getStrain()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/CDSSequence.html#getStrand()">getStrand()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/CDSSequence.html" title="class in org.biojava3.core.sequence">CDSSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/GeneSequence.html#getStrand()">getStrand()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/GeneSequence.html" title="class in org.biojava3.core.sequence">GeneSequence</a></dt>
<dd>
<div class="block">A gene should have Strand</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html#getStrand()">getStrand()</a></span> - Method in class org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava3.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/Location.html#getStrand()">getStrand()</a></span> - Method in interface org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/Location.html" title="interface in org.biojava3.core.sequence.location.template">Location</a></dt>
<dd>
<div class="block">Strand which the location is located on</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/TranscriptSequence.html#getStrand()">getStrand()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/TranscriptSequence.html" title="class in org.biojava3.core.sequence">TranscriptSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/ResourceManager.html#getString(java.lang.String)">getString(String)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/ResourceManager.html" title="class in org.biojava.bio.structure.align.util">ResourceManager</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/CaseFreeAminoAcidCompoundSet.html#getStringForCompound(org.biojava3.core.sequence.compound.AminoAcidCompound)">getStringForCompound(AminoAcidCompound)</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/CaseFreeAminoAcidCompoundSet.html" title="class in org.biojava3.aaproperties.xml">CaseFreeAminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/ModifiedAminoAcidCompoundSet.html#getStringForCompound(org.biojava3.core.sequence.compound.AminoAcidCompound)">getStringForCompound(AminoAcidCompound)</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/ModifiedAminoAcidCompoundSet.html" title="class in org.biojava3.aaproperties.xml">ModifiedAminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/AminoAcidCompoundSet.html#getStringForCompound(org.biojava3.core.sequence.compound.AminoAcidCompound)">getStringForCompound(AminoAcidCompound)</a></span> - Method in class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/AminoAcidCompoundSet.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompoundSet.html#getStringForCompound(C)">getStringForCompound(C)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava3.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/CompoundSet.html#getStringForCompound(C)">getStringForCompound(C)</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/Strand.html#getStringRepresentation()">getStringRepresentation()</a></span> - Method in enum org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/Strand.html" title="enum in org.biojava3.core.sequence">Strand</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html#getStructure()">getStructure()</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html" title="class in org.biojava.bio.structure.align.gui.jmol">StructureAlignmentJmol</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#getStructure(java.lang.String)">getStructure(String)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>
<div class="block">Request a Structure based on a <i>name</i>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#getStructure()">getStructure()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>
<div class="block">This method will return the parsed protein structure, once the parsing has been finished</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/MMCIFFileReader.html#getStructure(java.lang.String)">getStructure(String)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/MMCIFFileReader.html" title="class in org.biojava.bio.structure.io">MMCIFFileReader</a></dt>
<dd>
<div class="block">Opens filename, parses it and returns
 a Structure object .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/MMCIFFileReader.html#getStructure(java.io.File)">getStructure(File)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/MMCIFFileReader.html" title="class in org.biojava.bio.structure.io">MMCIFFileReader</a></dt>
<dd>
<div class="block">Opens filename, parses it and returns a Structure object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#getStructure(java.lang.String)">getStructure(String)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>
<div class="block">opens filename, parses it and returns
 aStructure object .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#getStructure(java.io.File)">getStructure(File)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>
<div class="block">opens filename, parses it and returns a Structure object</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#getStructure(java.net.URL)">getStructure(URL)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/StructureIOFile.html#getStructure(java.lang.String)">getStructure(String)</a></span> - Method in interface org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/StructureIOFile.html" title="interface in org.biojava.bio.structure.io">StructureIOFile</a></dt>
<dd>
<div class="block">open filename and returns
 a Structure object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/StructureIOFile.html#getStructure(java.io.File)">getStructure(File)</a></span> - Method in interface org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/StructureIOFile.html" title="interface in org.biojava.bio.structure.io">StructureIOFile</a></dt>
<dd>
<div class="block">read file from File and returns
 a Structure object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/FlatFileInstallation.html#getStructure(java.lang.String)">getStructure(String)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/FlatFileInstallation.html" title="class in org.biojava.bio.structure.server">FlatFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html#getStructure(java.lang.String)">getStructure(String)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html" title="class in org.biojava.bio.structure.server">MMCIFFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/PDBInstallation.html#getStructure(java.lang.String)">getStructure(String)</a></span> - Method in interface org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/PDBInstallation.html" title="interface in org.biojava.bio.structure.server">PDBInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
<div class="block">request a structure by its PDB identifier</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureEvent.html#getStructure()">getStructure()</a></span> - Method in interface org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureEvent.html" title="interface in org.biojava.bio.structure.server">StructureEvent</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureEventImpl.html#getStructure()">getStructure()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureEventImpl.html" title="class in org.biojava.bio.structure.server">StructureEventImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/SelectPDBPanel.html#getStructure1()">getStructure1()</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/SelectPDBPanel.html" title="class in org.biojava.bio.structure.align.gui">SelectPDBPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/PDBDirPanel.html#getStructure1()">getStructure1()</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/PDBDirPanel.html" title="class in org.biojava.bio.structure.gui.util">PDBDirPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/PDBServerPanel.html#getStructure1()">getStructure1()</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/PDBServerPanel.html" title="class in org.biojava.bio.structure.gui.util">PDBServerPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/PDBUploadPanel.html#getStructure1()">getStructure1()</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/PDBUploadPanel.html" title="class in org.biojava.bio.structure.gui.util">PDBUploadPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/ScopSelectPanel.html#getStructure1()">getStructure1()</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/ScopSelectPanel.html" title="class in org.biojava.bio.structure.gui.util">ScopSelectPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/StructurePairSelector.html#getStructure1()">getStructure1()</a></span> - Method in interface org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/StructurePairSelector.html" title="interface in org.biojava.bio.structure.gui.util">StructurePairSelector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/SelectPDBPanel.html#getStructure2()">getStructure2()</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/SelectPDBPanel.html" title="class in org.biojava.bio.structure.align.gui">SelectPDBPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/PDBDirPanel.html#getStructure2()">getStructure2()</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/PDBDirPanel.html" title="class in org.biojava.bio.structure.gui.util">PDBDirPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/PDBServerPanel.html#getStructure2()">getStructure2()</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/PDBServerPanel.html" title="class in org.biojava.bio.structure.gui.util">PDBServerPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/PDBUploadPanel.html#getStructure2()">getStructure2()</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/PDBUploadPanel.html" title="class in org.biojava.bio.structure.gui.util">PDBUploadPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/ScopSelectPanel.html#getStructure2()">getStructure2()</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/ScopSelectPanel.html" title="class in org.biojava.bio.structure.gui.util">ScopSelectPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/StructurePairSelector.html#getStructure2()">getStructure2()</a></span> - Method in interface org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/StructurePairSelector.html" title="interface in org.biojava.bio.structure.gui.util">StructurePairSelector</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/AlignmentGui.html#getStructureAlignment()">getStructureAlignment()</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/AlignmentGui.html" title="class in org.biojava.bio.structure.align.gui">AlignmentGui</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/DBSearchGUI.html#getStructureAlignment()">getStructureAlignment()</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/DBSearchGUI.html" title="class in org.biojava.bio.structure.align.gui">DBSearchGUI</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/StructureUtil.html#getStructureAtom(org.biojava.bio.structure.Atom, boolean)">getStructureAtom(Atom, boolean)</a></span> - Static method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/StructureUtil.html" title="class in org.biojava3.protmod.structure">StructureUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/StructureUtil.html#getStructureAtomLinkage(org.biojava.bio.structure.Atom, boolean, org.biojava.bio.structure.Atom, boolean)">getStructureAtomLinkage(Atom, boolean, Atom, boolean)</a></span> - Static method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/StructureUtil.html" title="class in org.biojava3.protmod.structure">StructureUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/LocalCacheStructureProvider.html#getStructureById(java.lang.String)">getStructureById(String)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/LocalCacheStructureProvider.html" title="class in org.biojava.bio.structure.io">LocalCacheStructureProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/MMCIFFileReader.html#getStructureById(java.lang.String)">getStructureById(String)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/MMCIFFileReader.html" title="class in org.biojava.bio.structure.io">MMCIFFileReader</a></dt>
<dd>
<div class="block">Get a structure by PDB code.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#getStructureById(java.lang.String)">getStructureById(String)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>
<div class="block">load a structure from local file system and return a PDBStructure object</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBSRSReader.html#getStructureById(java.lang.String)">getStructureById(String)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBSRSReader.html" title="class in org.biojava.bio.structure.io">PDBSRSReader</a></dt>
<dd>
<div class="block">load a structure from from SRS installation using wgetz
 returns null if no structure found</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/SandboxStyleStructureProvider.html#getStructureById(java.lang.String)">getStructureById(String)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/SandboxStyleStructureProvider.html" title="class in org.biojava.bio.structure.io">SandboxStyleStructureProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/StructureIO.html#getStructureById(java.lang.String)">getStructureById(String)</a></span> - Method in interface org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/StructureIO.html" title="interface in org.biojava.bio.structure.io">StructureIO</a></dt>
<dd>
<div class="block">Get a structure by providing a PDB code.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/StructureIOFile.html#getStructureById(java.lang.String)">getStructureById(String)</a></span> - Method in interface org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/StructureIOFile.html" title="interface in org.biojava.bio.structure.io">StructureIOFile</a></dt>
<dd>
<div class="block">Get a Structure based on its PDB id.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/StructureProvider.html#getStructureById(java.lang.String)">getStructureById(String)</a></span> - Method in interface org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/StructureProvider.html" title="interface in org.biojava.bio.structure.io">StructureProvider</a></dt>
<dd>
<div class="block">get the structure for a PDB ID</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#getStructureForDomain(org.biojava.bio.structure.scop.ScopDomain)">getStructureForDomain(ScopDomain)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>
<div class="block">Returns the representation of a ScopDomain as a BioJava Structure object</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/StructureUtil.html#getStructureGroup(org.biojava.bio.structure.Group, boolean)">getStructureGroup(Group, boolean)</a></span> - Static method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/StructureUtil.html" title="class in org.biojava3.protmod.structure">StructureUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureFetcherRunnable.html#getStructureListeners()">getStructureListeners()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureFetcherRunnable.html" title="class in org.biojava.bio.structure.server">StructureFetcherRunnable</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/SequenceDisplay.html#getStructurePairAligner()">getStructurePairAligner()</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/SequenceDisplay.html" title="class in org.biojava.bio.structure.gui">SequenceDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html#getSubLocations()">getSubLocations()</a></span> - Method in class org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava3.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/Location.html#getSubLocations()">getSubLocations()</a></span> - Method in interface org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/Location.html" title="interface in org.biojava3.core.sequence.location.template">Location</a></dt>
<dd>
<div class="block">Gives access to the sub locations for this location.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleProfile.html#getSubProfile(org.biojava3.core.sequence.location.template.Location)">getSubProfile(Location)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleProfile.html" title="class in org.biojava3.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/Profile.html#getSubProfile(org.biojava3.core.sequence.location.template.Location)">getSubProfile(Location)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns a <a href="./org/biojava3/alignment/template/ProfileView.html" title="interface in org.biojava3.alignment.template"><code>ProfileView</code></a> windowed to contain only the given <a href="./org/biojava3/core/sequence/location/template/Location.html" title="interface in org.biojava3.core.sequence.location.template"><code>Location</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#getSubRanges(org.biojava.bio.structure.Structure, java.lang.String)">getSubRanges(Structure, String)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>
<div class="block">In addition to the functionality provided by getReducedStructure also provides a way to specify sub-regions of a structure with the following 
 specification:
 
 
 ranges can be surrounded by ( and ).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleAlignedSequence.html#getSubSequence(java.lang.Integer, java.lang.Integer)">getSubSequence(Integer, Integer)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleAlignedSequence.html" title="class in org.biojava3.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html#getSubSequence(java.lang.Integer, java.lang.Integer)">getSubSequence(Integer, Integer)</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava3.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/StringProxySequenceReader.html#getSubSequence(java.lang.Integer, java.lang.Integer)">getSubSequence(Integer, Integer)</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html#getSubSequence(java.lang.Integer, java.lang.Integer)">getSubSequence(Integer, Integer)</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html#getSubSequence(org.biojava3.core.sequence.template.Sequence)">getSubSequence(Sequence&lt;C&gt;)</a></span> - Method in class org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava3.core.sequence.location.template">AbstractLocation</a></dt>
<dd>
<div class="block">If circular this will return the sequence represented by the sub
 locations joined.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/Location.html#getSubSequence(org.biojava3.core.sequence.template.Sequence)">getSubSequence(Sequence&lt;C&gt;)</a></span> - Method in interface org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/Location.html" title="interface in org.biojava3.core.sequence.location.template">Location</a></dt>
<dd>
<div class="block">Will return a SequenceReader object which represents the outer bounds
 of this Location</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/ArrayListSequenceReader.html#getSubSequence(java.lang.Integer, java.lang.Integer)">getSubSequence(Integer, Integer)</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava3.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/BitSequenceReader.html#getSubSequence(int, int)">getSubSequence(int, int)</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava3.core.sequence.storage">BitSequenceReader</a></dt>
<dd>
<div class="block">Returns a sub sequence view</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/BitSequenceReader.html#getSubSequence(java.lang.Integer, java.lang.Integer)">getSubSequence(Integer, Integer)</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava3.core.sequence.storage">BitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/JoiningSequenceReader.html#getSubSequence(java.lang.Integer, java.lang.Integer)">getSubSequence(Integer, Integer)</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava3.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html#getSubSequence(java.lang.Integer, java.lang.Integer)">getSubSequence(Integer, Integer)</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava3.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Creates a <a href="./org/biojava3/core/sequence/template/SequenceProxyView.html" title="class in org.biojava3.core.sequence.template"><code>SequenceProxyView</code></a> for the given coordinates</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#getSubSequence(java.lang.Integer, java.lang.Integer)">getSubSequence(Integer, Integer)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/Sequence.html#getSubSequence(java.lang.Integer, java.lang.Integer)">getSubSequence(Integer, Integer)</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a></dt>
<dd>
<div class="block">Returns a portion of the sequence from the different positions.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceProxyView.html#getSubSequence(java.lang.Integer, java.lang.Integer)">getSubSequence(Integer, Integer)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceProxyView.html" title="class in org.biojava3.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#getSubstitutionMatrix()">getSubstitutionMatrix()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>
<div class="block">Sets the  substitution matrix to be used for influencing the alignment with sequence conservation information.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#getSubstitutionMatrix()">getSubstitutionMatrix()</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava3.alignment.template">AbstractMatrixAligner</a></dt>
<dd>
<div class="block">Returns the substitution matrix.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#getSubstitutionScore(int, int)">getSubstitutionScore(int, int)</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava3.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html#getSubstitutionScore(int, int)">getSubstitutionScore(int, int)</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html" title="class in org.biojava3.alignment.template">AbstractPairwiseSequenceAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractProfileProfileAligner.html#getSubstitutionScore(int, int)">getSubstitutionScore(int, int)</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractProfileProfileAligner.html" title="class in org.biojava3.alignment.template">AbstractProfileProfileAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#getSubstitutionScoreVector(int)">getSubstitutionScoreVector(int)</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava3.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#getSubstitutionScoreVector(int, int[])">getSubstitutionScoreVector(int, int[])</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava3.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AFPChainCoordManager.html#getSummaryPos()">getSummaryPos()</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AFPChainCoordManager.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AFPChainCoordManager</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDescription.html#getSunID()">getSunID()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDescription.html" title="class in org.biojava.bio.structure.scop">ScopDescription</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#getSunid()">getSunid()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopNode.html#getSunid()">getSunid()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopNode.html" title="class in org.biojava.bio.structure.scop">ScopNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#getSuperfamilyId()">getSuperfamilyId()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Author.html#getSurname()">getSurname()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Author.html" title="class in org.biojava.bio.structure">Author</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#getSwissprotId()">getSwissprotId()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Get the Swissprot id of this chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#getSwissprotId()">getSwissprotId()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">get the Swissprot id of this chains .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/AminoAcidComposition.html#getSymbol()">getSymbol()</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/AminoAcidComposition.html" title="class in org.biojava3.aaproperties.xml">AminoAcidComposition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/AminoAcidCompositionTable.html#getSymbolSet()">getSymbolSet()</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/AminoAcidCompositionTable.html" title="class in org.biojava3.aaproperties.xml">AminoAcidCompositionTable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getSynonyms()">getSynonyms()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getSynthetic()">getSynthetic()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModification.html#getSystematicName()">getSystematicName()</a></span> - Method in interface org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModification.html" title="interface in org.biojava3.protmod">ProteinModification</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationImpl.html#getSystematicName()">getSystematicName()</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationImpl.html" title="class in org.biojava3.protmod">ProteinModificationImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFP.html#getT()">getT()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFP.html" title="class in org.biojava.bio.structure.align.model">AFP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/IUPACParser.html#getTable(java.lang.String)">getTable(String)</a></span> - Method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/IUPACParser.html" title="class in org.biojava3.core.sequence.io">IUPACParser</a></dt>
<dd>
<div class="block">Returns a table by its name</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/IUPACParser.html#getTable(java.lang.Integer)">getTable(Integer)</a></span> - Method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/IUPACParser.html" title="class in org.biojava3.core.sequence.io">IUPACParser</a></dt>
<dd>
<div class="block">Returns a table by its identifier i.e. 1 means universal codon tables</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.html#getTable()">getTable()</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.html" title="class in org.biojava3.core.sequence.transcription">TranscriptionEngine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/IUPACParser.html#getTables()">getTables()</a></span> - Method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/IUPACParser.html" title="class in org.biojava3.core.sequence.io">IUPACParser</a></dt>
<dd>
<div class="block">Returns a list of all available IUPAC tables</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/DBSearchGUI.html#getTabPane()">getTabPane()</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/DBSearchGUI.html" title="class in org.biojava.bio.structure.align.gui">DBSearchGUI</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/FractionalIdentityScorer.html#getTarget()">getTarget()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/FractionalIdentityScorer.html" title="class in org.biojava3.alignment">FractionalIdentityScorer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/FractionalSimilarityScorer.html#getTarget()">getTarget()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/FractionalSimilarityScorer.html" title="class in org.biojava3.alignment">FractionalSimilarityScorer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleProfilePair.html#getTarget()">getTarget()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleProfilePair.html" title="class in org.biojava3.alignment">SimpleProfilePair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleSequencePair.html#getTarget()">getTarget()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleSequencePair.html" title="class in org.biojava3.alignment">SimpleSequencePair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html#getTarget()">getTarget()</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html" title="class in org.biojava3.alignment.template">AbstractPairwiseSequenceAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractProfileProfileAligner.html#getTarget()">getTarget()</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractProfileProfileAligner.html" title="class in org.biojava3.alignment.template">AbstractProfileProfileAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/PairwiseSequenceScorer.html#getTarget()">getTarget()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/PairwiseSequenceScorer.html" title="interface in org.biojava3.alignment.template">PairwiseSequenceScorer</a></dt>
<dd>
<div class="block">Returns the second sequence of the pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/ProfilePair.html#getTarget()">getTarget()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/ProfilePair.html" title="interface in org.biojava3.alignment.template">ProfilePair</a></dt>
<dd>
<div class="block">Returns the second <a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template"><code>Profile</code></a> of the pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/ProfileProfileScorer.html#getTarget()">getTarget()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/ProfileProfileScorer.html" title="interface in org.biojava3.alignment.template">ProfileProfileScorer</a></dt>
<dd>
<div class="block">Returns the second profile of the pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/SequencePair.html#getTarget()">getTarget()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/SequencePair.html" title="interface in org.biojava3.alignment.template">SequencePair</a></dt>
<dd>
<div class="block">Returns the second <a href="./org/biojava3/alignment/template/AlignedSequence.html" title="interface in org.biojava3.alignment.template"><code>AlignedSequence</code></a> of the pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/routines/AlignerHelper.Cut.html#getTargetIndex(int)">getTargetIndex(int)</a></span> - Method in class org.biojava3.alignment.routines.<a href="./org/biojava3/alignment/routines/AlignerHelper.Cut.html" title="class in org.biojava3.alignment.routines">AlignerHelper.Cut</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/edits/Edit.AbstractEdit.html#getTargetSequence(org.biojava3.core.sequence.template.Sequence)">getTargetSequence(Sequence&lt;C&gt;)</a></span> - Method in class org.biojava3.core.sequence.edits.<a href="./org/biojava3/core/sequence/edits/Edit.AbstractEdit.html" title="class in org.biojava3.core.sequence.edits">Edit.AbstractEdit</a></dt>
<dd>
<div class="block">Returns the Sequence which is our edit.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#getTaxonomy()">getTaxonomy()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#getTechnique()">getTechnique()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#getTempFactor()">getTempFactor()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">get set temp factor.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#getTempFactor()">getTempFactor()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBremark.html#getText()">getText()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBremark.html" title="class in org.biojava.bio.structure.io.mmcif.model">DatabasePDBremark</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructKeywords.html#getText()">getText()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructKeywords.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructKeywords</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/ConcurrencyTools.html#getThreadPool()">getThreadPool()</a></span> - Static method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/ConcurrencyTools.html" title="class in org.biojava3.core.util">ConcurrencyTools</a></dt>
<dd>
<div class="block">Returns current shared thread pool.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#getThreads()">getThreads()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/CodonCompound.html#getThree()">getThree()</a></span> - Method in class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/CodonCompound.html" title="class in org.biojava3.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/Table.CaseInsensitiveTriplet.html#getThree()">getThree()</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/Table.CaseInsensitiveTriplet.html" title="class in org.biojava3.core.sequence.transcription">Table.CaseInsensitiveTriplet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/Table.Codon.html#getThree()">getThree()</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/Table.Codon.html" title="class in org.biojava3.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getThree_letter_code()">getThree_letter_code()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/edits/Edit.AbstractEdit.html#getThreePrime(org.biojava3.core.sequence.template.Sequence)">getThreePrime(Sequence&lt;C&gt;)</a></span> - Method in class org.biojava3.core.sequence.edits.<a href="./org/biojava3/core/sequence/edits/Edit.AbstractEdit.html" title="class in org.biojava3.core.sequence.edits">Edit.AbstractEdit</a></dt>
<dd>
<div class="block">Should return the 3-prime end of the given Sequence according to
 the edit.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/edits/Edit.Delete.html#getThreePrime(org.biojava3.core.sequence.template.Sequence)">getThreePrime(Sequence&lt;C&gt;)</a></span> - Method in class org.biojava3.core.sequence.edits.<a href="./org/biojava3/core/sequence/edits/Edit.Delete.html" title="class in org.biojava3.core.sequence.edits">Edit.Delete</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/edits/Edit.Insert.html#getThreePrime(org.biojava3.core.sequence.template.Sequence)">getThreePrime(Sequence&lt;C&gt;)</a></span> - Method in class org.biojava3.core.sequence.edits.<a href="./org/biojava3/core/sequence/edits/Edit.Insert.html" title="class in org.biojava3.core.sequence.edits">Edit.Insert</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/edits/Edit.Substitute.html#getThreePrime(org.biojava3.core.sequence.template.Sequence)">getThreePrime(Sequence&lt;C&gt;)</a></span> - Method in class org.biojava3.core.sequence.edits.<a href="./org/biojava3/core/sequence/edits/Edit.Substitute.html" title="class in org.biojava3.core.sequence.edits">Edit.Substitute</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#getTime()">getTime()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/JFatCatClient.html#getTimeout()">getTimeout()</a></span> - Static method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/JFatCatClient.html" title="class in org.biojava.bio.structure.align.client">JFatCatClient</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileParser.html#getTimeStamp()">getTimeStamp()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileParser.html" title="class in org.biojava.bio.structure.io">PDBFileParser</a></dt>
<dd>
<div class="block">Returns a time stamp.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getTissue()">getTissue()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html#getTitle()">getTitle()</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html" title="class in org.biojava.bio.structure.align.gui.jmol">StructureAlignmentJmol</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getTitle()">getTitle()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html#getTitle()">getTitle()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html" title="class in org.biojava.bio.structure.io.mmcif.model">Struct</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#getTitle()">getTitle()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>
<div class="block">Get the value of title</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#getTitle()">getTitle()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getTMScore()">getTMScore()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">Returns the tmScore of the alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AFPChainScorer.html#getTMScore(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">getTMScore(AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AFPChainScorer.html" title="class in org.biojava.bio.structure.align.util">AFPChainScorer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SVDSuperimposer.html#getTMScore(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], int, int)">getTMScore(Atom[], Atom[], int, int)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SVDSuperimposer.html" title="class in org.biojava.bio.structure">SVDSuperimposer</a></dt>
<dd>
<div class="block">Calculate the TM-Score for the superposition.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Segment.html#getTo()">getTo()</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Segment.html" title="class in org.biojava.bio.structure.domain.pdp">Segment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompoundTranslator.html#getToCompoundSet()">getToCompoundSet()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompoundTranslator.html" title="class in org.biojava3.core.sequence.template">AbstractCompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastService.html#getTool()">getTool()</a></span> - Method in class org.biojava3.ws.alignment.qblast.<a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastService.html" title="class in org.biojava3.ws.alignment.qblast">NCBIQBlastService</a></dt>
<dd>
<div class="block">Get the tool identifier for QBlast.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getTorsionPenalty()">getTorsionPenalty()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getTotalLenIni()">getTotalLenIni()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getTotalLenOpt()">getTotalLenOpt()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getTotalRmsdIni()">getTotalRmsdIni()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">this is the init-RMSD, not the final RMSD after refinement.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getTotalRmsdOpt()">getTotalRmsdOpt()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">The RMSD of the final alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/BigSqrt.html#getTraceFlag()">getTraceFlag()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/BigSqrt.html" title="class in org.biojava.bio.structure.secstruc">BigSqrt</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#getTrans()">getTrans()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/GeneSequence.html#getTranscript(java.lang.String)">getTranscript(String)</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/GeneSequence.html" title="class in org.biojava3.core.sequence">GeneSequence</a></dt>
<dd>
<div class="block">Get the transcript sequence by accession</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/GeneSequence.html#getTranscripts()">getTranscripts()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/GeneSequence.html" title="class in org.biojava3.core.sequence">GeneSequence</a></dt>
<dd>
<div class="block">Get the collection of transcription sequences assigned to this gene</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/IProfeatProperties.html#getTransition(org.biojava3.core.sequence.ProteinSequence, org.biojava3.aaproperties.profeat.IProfeatProperties.ATTRIBUTE, org.biojava3.aaproperties.profeat.IProfeatProperties.TRANSITION)">getTransition(ProteinSequence, IProfeatProperties.ATTRIBUTE, IProfeatProperties.TRANSITION)</a></span> - Method in interface org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/IProfeatProperties.html" title="interface in org.biojava3.aaproperties.profeat">IProfeatProperties</a></dt>
<dd>
<div class="block">Returns the number of transition between the specified groups for the given attribute with respect to the length of sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/IProfeatProperties.html#getTransition(org.biojava3.core.sequence.ProteinSequence, org.biojava3.aaproperties.profeat.IProfeatProperties.ATTRIBUTE)">getTransition(ProteinSequence, IProfeatProperties.ATTRIBUTE)</a></span> - Method in interface org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/IProfeatProperties.html" title="interface in org.biojava3.aaproperties.profeat">IProfeatProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/IProfeatProperties.html#getTransition(org.biojava3.core.sequence.ProteinSequence)">getTransition(ProteinSequence)</a></span> - Method in interface org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/IProfeatProperties.html" title="interface in org.biojava3.aaproperties.profeat">IProfeatProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/ProfeatProperties.html#getTransition(org.biojava3.core.sequence.ProteinSequence, org.biojava3.aaproperties.profeat.IProfeatProperties.ATTRIBUTE, org.biojava3.aaproperties.profeat.IProfeatProperties.TRANSITION)">getTransition(ProteinSequence, IProfeatProperties.ATTRIBUTE, IProfeatProperties.TRANSITION)</a></span> - Static method in class org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/ProfeatProperties.html" title="class in org.biojava3.aaproperties.profeat">ProfeatProperties</a></dt>
<dd>
<div class="block">An adaptor method which returns the number of transition between the specified groups for the given attribute with respect to the length of sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/ProfeatProperties.html#getTransition(org.biojava3.core.sequence.ProteinSequence, org.biojava3.aaproperties.profeat.IProfeatProperties.ATTRIBUTE)">getTransition(ProteinSequence, IProfeatProperties.ATTRIBUTE)</a></span> - Static method in class org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/ProfeatProperties.html" title="class in org.biojava3.aaproperties.profeat">ProfeatProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/ProfeatProperties.html#getTransition(org.biojava3.core.sequence.ProteinSequence)">getTransition(ProteinSequence)</a></span> - Static method in class org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/ProfeatProperties.html" title="class in org.biojava3.aaproperties.profeat">ProfeatProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/ProfeatProperties.html#getTransition(java.lang.String, org.biojava3.aaproperties.profeat.IProfeatProperties.ATTRIBUTE, org.biojava3.aaproperties.profeat.IProfeatProperties.TRANSITION)">getTransition(String, IProfeatProperties.ATTRIBUTE, IProfeatProperties.TRANSITION)</a></span> - Static method in class org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/ProfeatProperties.html" title="class in org.biojava3.aaproperties.profeat">ProfeatProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/ProfeatProperties.html#getTransition(java.lang.String, org.biojava3.aaproperties.profeat.IProfeatProperties.ATTRIBUTE)">getTransition(String, IProfeatProperties.ATTRIBUTE)</a></span> - Static method in class org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/ProfeatProperties.html" title="class in org.biojava3.aaproperties.profeat">ProfeatProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/ProfeatProperties.html#getTransition(java.lang.String)">getTransition(String)</a></span> - Static method in class org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/ProfeatProperties.html" title="class in org.biojava3.aaproperties.profeat">ProfeatProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/ProfeatPropertiesImpl.html#getTransition(org.biojava3.core.sequence.ProteinSequence, org.biojava3.aaproperties.profeat.IProfeatProperties.ATTRIBUTE, org.biojava3.aaproperties.profeat.IProfeatProperties.TRANSITION)">getTransition(ProteinSequence, IProfeatProperties.ATTRIBUTE, IProfeatProperties.TRANSITION)</a></span> - Method in class org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/ProfeatPropertiesImpl.html" title="class in org.biojava3.aaproperties.profeat">ProfeatPropertiesImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/ProfeatPropertiesImpl.html#getTransition(org.biojava3.core.sequence.ProteinSequence, org.biojava3.aaproperties.profeat.IProfeatProperties.ATTRIBUTE)">getTransition(ProteinSequence, IProfeatProperties.ATTRIBUTE)</a></span> - Method in class org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/ProfeatPropertiesImpl.html" title="class in org.biojava3.aaproperties.profeat">ProfeatPropertiesImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/ProfeatPropertiesImpl.html#getTransition(org.biojava3.core.sequence.ProteinSequence)">getTransition(ProteinSequence)</a></span> - Method in class org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/ProfeatPropertiesImpl.html" title="class in org.biojava3.aaproperties.profeat">ProfeatPropertiesImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html#getTranslation(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">getTranslation(AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AFPAlignmentDisplay.html" title="class in org.biojava.bio.structure.align.util">AFPAlignmentDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SVDSuperimposer.html#getTranslation()">getTranslation()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SVDSuperimposer.html" title="class in org.biojava.bio.structure">SVDSuperimposer</a></dt>
<dd>
<div class="block">Get the shift vector.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html#getTree(org.biojava.bio.structure.scop.ScopDomain)">getTree(ScopDomain)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">RemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDatabase.html#getTree(org.biojava.bio.structure.scop.ScopDomain)">getTree(ScopDomain)</a></span> - Method in interface org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDatabase.html" title="interface in org.biojava.bio.structure.scop">ScopDatabase</a></dt>
<dd>
<div class="block">get the SCOP sub-tree for a particular domain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#getTree(org.biojava.bio.structure.scop.ScopDomain)">getTree(ScopDomain)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/ronn/RonnConstraint.Threshold.html#getTreshold(int)">getTreshold(int)</a></span> - Static method in enum org.biojava3.ronn.<a href="./org/biojava3/ronn/RonnConstraint.Threshold.html" title="enum in org.biojava3.ronn">RonnConstraint.Threshold</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/Table.Codon.html#getTriplet()">getTriplet()</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/Table.Codon.html" title="class in org.biojava3.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getTwi()">getTwi()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatAligner.html#getTwistedGroups()">getTwistedGroups()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatAligner.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/CodonCompound.html#getTwo()">getTwo()</a></span> - Method in class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/CodonCompound.html" title="class in org.biojava3.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/Table.CaseInsensitiveTriplet.html#getTwo()">getTwo()</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/Table.CaseInsensitiveTriplet.html" title="class in org.biojava3.core.sequence.transcription">Table.CaseInsensitiveTriplet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/Table.Codon.html#getTwo()">getTwo()</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/Table.Codon.html" title="class in org.biojava3.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcidImpl.html#getType()">getType()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcidImpl.html" title="class in org.biojava.bio.structure">AminoAcidImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Bond.html#getType()">getType()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Bond.html" title="class in org.biojava.bio.structure">Bond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#getType()">getType()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">get Type of group, e.g. amino, hetatom, nucleotide.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#getType()">getType()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">Returns the type value.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#getType()">getType()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html#getType()">getType()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemCompDescriptor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html#getType()">getType()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model">Entity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/NucleotideImpl.html#getType()">getType()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/NucleotideImpl.html" title="class in org.biojava.bio.structure">NucleotideImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleGapPenalty.html#getType()">getType()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleGapPenalty.html" title="class in org.biojava3.alignment">SimpleGapPenalty</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/GapPenalty.html#getType()">getType()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/GapPenalty.html" title="interface in org.biojava3.alignment.template">GapPenalty</a></dt>
<dd>
<div class="block">Returns <a href="./org/biojava3/alignment/template/GapPenalty.Type.html" title="enum in org.biojava3.alignment.template"><code>GapPenalty.Type</code></a> stored.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/AbstractFeature.html#getType()">getType()</a></span> - Method in class org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/AbstractFeature.html" title="class in org.biojava3.core.sequence.features">AbstractFeature</a></dt>
<dd>
<div class="block">The feature type</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/FeatureInterface.html#getType()">getType()</a></span> - Method in interface org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/FeatureInterface.html" title="interface in org.biojava3.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">The type of the feature.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#getType_symbol()">getType_symbol()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/LUDecomposition.html#getU()">getU()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/LUDecomposition.html" title="class in org.biojava.bio.structure.jama">LUDecomposition</a></dt>
<dd>
<div class="block">Return upper triangular factor</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/SingularValueDecomposition.html#getU()">getU()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/SingularValueDecomposition.html" title="class in org.biojava.bio.structure.jama">SingularValueDecomposition</a></dt>
<dd>
<div class="block">Return the left singular vectors</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/CaseInsensitiveCompound.html#getUnderlyingCompound()">getUnderlyingCompound()</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava3.core.sequence.transcription">CaseInsensitiveCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ProteinModificationIdentifier.html#getUnidentifiableAtomLinkages()">getUnidentifiableAtomLinkages()</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ProteinModificationIdentifier.html" title="class in org.biojava3.protmod.structure">ProteinModificationIdentifier</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ProteinModificationIdentifier.html#getUnidentifiableModifiedResidues()">getUnidentifiableModifiedResidues()</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ProteinModificationIdentifier.html" title="class in org.biojava3.protmod.structure">ProteinModificationIdentifier</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html#getUniprotbaseURL()">getUniprotbaseURL()</a></span> - Static method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>
<div class="block">The current unirpot URL to deal with caching issues. www.uniprot.org is loaded balanced
 but you can access pir.uniprot.org directly.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html#getUniprotDirectoryCache()">getUniprotDirectoryCache()</a></span> - Static method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>
<div class="block">Local directory cache of XML that can be downloaded</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#getUnitv()">getUnitv()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompound.html#getUpperedBase()">getUpperedBase()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompound.html" title="class in org.biojava3.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#getUsed()">getUsed()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#getUsedAtomNames()">getUsedAtomNames()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#getUserCollection()">getUserCollection()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>
<div class="block">Provided for convience if the developer needs to associate data with a sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#getUserConfigHelp()">getUserConfigHelp()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/ConfigStrucAligParams.html#getUserConfigHelp()">getUserConfigHelp()</a></span> - Method in interface org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/ConfigStrucAligParams.html" title="interface in org.biojava.bio.structure.align.ce">ConfigStrucAligParams</a></dt>
<dd>
<div class="block">The help text for each of these parameters.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getUserConfigHelp()">getUserConfigHelp()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWaterman3DParameters.html#getUserConfigHelp()">getUserConfigHelp()</a></span> - Method in class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWaterman3DParameters.html" title="class in org.biojava.bio.structure.align.seq">SmithWaterman3DParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#getUserConfigParameterNames()">getUserConfigParameterNames()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/ConfigStrucAligParams.html#getUserConfigParameterNames()">getUserConfigParameterNames()</a></span> - Method in interface org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/ConfigStrucAligParams.html" title="interface in org.biojava.bio.structure.align.ce">ConfigStrucAligParams</a></dt>
<dd>
<div class="block">The labels to be displayed to the user for each parameter</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getUserConfigParameterNames()">getUserConfigParameterNames()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWaterman3DParameters.html#getUserConfigParameterNames()">getUserConfigParameterNames()</a></span> - Method in class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWaterman3DParameters.html" title="class in org.biojava.bio.structure.align.seq">SmithWaterman3DParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#getUserConfigParameters()">getUserConfigParameters()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/ConfigStrucAligParams.html#getUserConfigParameters()">getUserConfigParameters()</a></span> - Method in interface org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/ConfigStrucAligParams.html" title="interface in org.biojava.bio.structure.align.ce">ConfigStrucAligParams</a></dt>
<dd>
<div class="block">get the list of parameters that the user can change through the user interface.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getUserConfigParameters()">getUserConfigParameters()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWaterman3DParameters.html#getUserConfigParameters()">getUserConfigParameters()</a></span> - Method in class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWaterman3DParameters.html" title="class in org.biojava.bio.structure.align.seq">SmithWaterman3DParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#getUserConfigTypes()">getUserConfigTypes()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/ConfigStrucAligParams.html#getUserConfigTypes()">getUserConfigTypes()</a></span> - Method in interface org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/ConfigStrucAligParams.html" title="interface in org.biojava.bio.structure.align.ce">ConfigStrucAligParams</a></dt>
<dd>
<div class="block">Get the data types of the parameters</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#getUserConfigTypes()">getUserConfigTypes()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWaterman3DParameters.html#getUserConfigTypes()">getUserConfigTypes()</a></span> - Method in class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWaterman3DParameters.html" title="class in org.biojava.bio.structure.align.seq">SmithWaterman3DParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#getUsername()">getUsername()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/AbstractFeature.html#getUserObject()">getUserObject()</a></span> - Method in class org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/AbstractFeature.html" title="class in org.biojava3.core.sequence.features">AbstractFeature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/FeatureInterface.html#getUserObject()">getUserObject()</a></span> - Method in interface org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/FeatureInterface.html" title="interface in org.biojava3.core.sequence.features">FeatureInterface</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/EigenvalueDecomposition.html#getV()">getV()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/EigenvalueDecomposition.html" title="class in org.biojava.bio.structure.jama">EigenvalueDecomposition</a></dt>
<dd>
<div class="block">Return the eigenvector matrix</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/SingularValueDecomposition.html#getV()">getV()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/SingularValueDecomposition.html" title="class in org.biojava.bio.structure.jama">SingularValueDecomposition</a></dt>
<dd>
<div class="block">Return the right singular vectors</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#getValenceElectronCount()">getValenceElectronCount()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns the number of valence electrons for this Element.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/AligMatEl.html#getValue()">getValue()</a></span> - Method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/AligMatEl.html" title="class in org.biojava.bio.structure.align.helper">AligMatEl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/GapArray.html#getValue()">getValue()</a></span> - Method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/GapArray.html" title="class in org.biojava.bio.structure.align.helper">GapArray</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html#getValue(org.biojava3.core.sequence.compound.AminoAcidCompound, org.biojava3.core.sequence.compound.AminoAcidCompound)">getValue(AminoAcidCompound, AminoAcidCompound)</a></span> - Method in class org.biojava3.alignment.aaindex.<a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava3.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleSubstitutionMatrix.html#getValue(C, C)">getValue(C, C)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleSubstitutionMatrix.html" title="class in org.biojava3.alignment">SimpleSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/SubstitutionMatrix.html#getValue(C, C)">getValue(C, C)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/SubstitutionMatrix.html" title="interface in org.biojava3.alignment.template">SubstitutionMatrix</a></dt>
<dd>
<div class="block">Returns value in matrix for conversion from first <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template"><code>Compound</code></a> to the second.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/CRC64Checksum.html#getValue()">getValue()</a></span> - Method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/CRC64Checksum.html" title="class in org.biojava3.core.util">CRC64Checksum</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/ronn/RonnConstraint.Threshold.html#getValues()">getValues()</a></span> - Method in enum org.biojava3.ronn.<a href="./org/biojava3/ronn/RonnConstraint.Threshold.html" title="enum in org.biojava3.ronn">RonnConstraint.Threshold</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#getVariant()">getVariant()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#getVDWRadius()">getVDWRadius()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns the van der Waals radius of this Element.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/AbstractStructureAlignment.html#getVersion()">getVersion()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/AbstractStructureAlignment.html" title="class in org.biojava.bio.structure.align">AbstractStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/BioJavaStructureAlignment.html#getVersion()">getVersion()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/BioJavaStructureAlignment.html" title="class in org.biojava.bio.structure.align">BioJavaStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCPMain.html#getVersion()">getVersion()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCPMain.html" title="class in org.biojava.bio.structure.align.ce">CeCPMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeMain.html#getVersion()">getVersion()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeMain.html" title="class in org.biojava.bio.structure.align.ce">CeMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeSideChainMain.html#getVersion()">getVersion()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeSideChainMain.html" title="class in org.biojava.bio.structure.align.ce">CeSideChainMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCatFlexible.html#getVersion()">getVersion()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCatFlexible.html" title="class in org.biojava.bio.structure.align.fatcat">FatCatFlexible</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCatRigid.html#getVersion()">getVersion()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCatRigid.html" title="class in org.biojava.bio.structure.align.fatcat">FatCatRigid</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#getVersion()">getVersion()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWaterman3Daligner.html#getVersion()">getVersion()</a></span> - Method in class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWaterman3Daligner.html" title="class in org.biojava.bio.structure.align.seq">SmithWaterman3Daligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructureAlignment.html#getVersion()">getVersion()</a></span> - Method in interface org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructureAlignment.html" title="interface in org.biojava.bio.structure.align">StructureAlignment</a></dt>
<dd>
<div class="block">Get the Version information for this Algorithm.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/ProfileView.html#getViewedProfile()">getViewedProfile()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/ProfileView.html" title="interface in org.biojava3.alignment.template">ProfileView</a></dt>
<dd>
<div class="block">Returns the entire <a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template"><code>Profile</code></a> being viewed</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceProxyView.html#getViewedSequence()">getViewedSequence()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceProxyView.html" title="class in org.biojava3.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceView.html#getViewedSequence()">getViewedSequence()</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceView.html" title="interface in org.biojava3.core.sequence.template">SequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/JmolPanel.html#getViewer()">getViewer()</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/JmolPanel.html" title="class in org.biojava.bio.structure.align.gui.jmol">JmolPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#getVolume()">getVolume()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/webstart/WebStartMain.html#getWebStartConfig()">getWebStartConfig()</a></span> - Static method in class org.biojava.bio.structure.align.webstart.<a href="./org/biojava/bio/structure/align/webstart/WebStartMain.html" title="class in org.biojava.bio.structure.align.webstart">WebStartMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/views/WindowedSequence.html#getWindowSize()">getWindowSize()</a></span> - Method in class org.biojava3.core.sequence.views.<a href="./org/biojava3/core/sequence/views/WindowedSequence.html" title="class in org.biojava3.core.sequence.views">WindowedSequence</a></dt>
<dd>
<div class="block">Access the current window size</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#getWinSize()">getWinSize()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>
<div class="block">The window size to look at</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#getX()">getX()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">Get coordinate X.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#getX()">getX()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>
<div class="block">Get the X coordinate.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#getXYZEuler(org.biojava.bio.structure.jama.Matrix)">getXYZEuler(Matrix)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">Convert a rotation Matrix to Euler angles.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#getY()">getY()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">Get coordinate Y.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#getY()">getY()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>
<div class="block">Get the Y coordinate.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#getZ()">getZ()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">Get coordinate Z.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#getZ()">getZ()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>
<div class="block">Get the Z coordinate.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#getZYZEuler(org.biojava.bio.structure.jama.Matrix)">getZYZEuler(Matrix)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">Gget euler angles for a matrix given in ZYZ convention.</div>
</dd>
<dt><a href="./org/biojava3/genome/homology/GFF3FromUniprotBlastHits.html" title="class in org.biojava3.genome.homology"><span class="strong">GFF3FromUniprotBlastHits</span></a> - Class in <a href="./org/biojava3/genome/homology/package-summary.html">org.biojava3.genome.homology</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/homology/GFF3FromUniprotBlastHits.html#GFF3FromUniprotBlastHits()">GFF3FromUniprotBlastHits()</a></span> - Constructor for class org.biojava3.genome.homology.<a href="./org/biojava3/genome/homology/GFF3FromUniprotBlastHits.html" title="class in org.biojava3.genome.homology">GFF3FromUniprotBlastHits</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/genome/parsers/gff/GFF3Reader.html" title="class in org.biojava3.genome.parsers.gff"><span class="strong">GFF3Reader</span></a> - Class in <a href="./org/biojava3/genome/parsers/gff/package-summary.html">org.biojava3.genome.parsers.gff</a></dt>
<dd>
<div class="block">http://www.bioperl.org/wiki/GTF
 Read and write FeatureLists as GFF/GTF formatted files.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/GFF3Reader.html#GFF3Reader()">GFF3Reader()</a></span> - Constructor for class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/GFF3Reader.html" title="class in org.biojava3.genome.parsers.gff">GFF3Reader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/genome/parsers/gff/GFF3Writer.html" title="class in org.biojava3.genome.parsers.gff"><span class="strong">GFF3Writer</span></a> - Class in <a href="./org/biojava3/genome/parsers/gff/package-summary.html">org.biojava3.genome.parsers.gff</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/GFF3Writer.html#GFF3Writer()">GFF3Writer()</a></span> - Constructor for class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/GFF3Writer.html" title="class in org.biojava3.genome.parsers.gff">GFF3Writer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#Gln">Gln</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#Glu">Glu</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#Gly">Gly</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/ColorUtils.html#gold">gold</a></span> - Static variable in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/ColorUtils.html" title="class in org.biojava.bio.structure.gui.util.color">ColorUtils</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/pairwise/Gotoh.html" title="class in org.biojava.bio.structure.align.pairwise"><span class="strong">Gotoh</span></a> - Class in <a href="./org/biojava/bio/structure/align/pairwise/package-summary.html">org.biojava.bio.structure.align.pairwise</a></dt>
<dd>
<div class="block">a class to perform Gotoh algorithm</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/Gotoh.html#Gotoh(org.biojava.bio.structure.align.pairwise.Alignable)">Gotoh(Alignable)</a></span> - Constructor for class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/Gotoh.html" title="class in org.biojava.bio.structure.align.pairwise">Gotoh</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html" title="class in org.biojava.bio.structure.gui.util.color"><span class="strong">GradientMapper</span></a> - Class in <a href="./org/biojava/bio/structure/gui/util/color/package-summary.html">org.biojava.bio.structure.gui.util.color</a></dt>
<dd>
<div class="block">Maps a set of real values onto a gradient.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html#GradientMapper()">GradientMapper()</a></span> - Constructor for class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html" title="class in org.biojava.bio.structure.gui.util.color">GradientMapper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html#GradientMapper(java.awt.Color, java.awt.Color)">GradientMapper(Color, Color)</a></span> - Constructor for class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html" title="class in org.biojava.bio.structure.gui.util.color">GradientMapper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html#GradientMapper(java.awt.Color, java.awt.Color, java.awt.color.ColorSpace)">GradientMapper(Color, Color, ColorSpace)</a></span> - Constructor for class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html" title="class in org.biojava.bio.structure.gui.util.color">GradientMapper</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/gui/util/color/GradientPanel.html" title="class in org.biojava.bio.structure.gui.util.color"><span class="strong">GradientPanel</span></a> - Class in <a href="./org/biojava/bio/structure/gui/util/color/package-summary.html">org.biojava.bio.structure.gui.util.color</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/GradientPanel.html#GradientPanel(org.biojava.bio.structure.gui.util.color.ContinuousColorMapper, double, double)">GradientPanel(ContinuousColorMapper, double, double)</a></span> - Constructor for class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/GradientPanel.html" title="class in org.biojava.bio.structure.gui.util.color">GradientPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html#gradients">gradients</a></span> - Variable in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html" title="class in org.biojava.bio.structure.gui">ScaleableMatrixPanel</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure"><span class="strong">Group</span></a> - Interface in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">This is the data structure for a single Group of atoms.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Feature.html#group()">group()</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Feature.html" title="class in org.biojava3.genome.parsers.gff">Feature</a></dt>
<dd>
<div class="block">Get the first item (everything before first semicolon, if it has one)
 in the attribute field, which is assumed to
 be a group identifer.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/FeatureI.html#group()">group()</a></span> - Method in interface org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/FeatureI.html" title="interface in org.biojava3.genome.parsers.gff">FeatureI</a></dt>
<dd>
<div class="block">Get the group id of the feature.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/convertor/Convertor.html#group1">group1</a></span> - Static variable in class org.biojava3.aaproperties.profeat.convertor.<a href="./org/biojava3/aaproperties/profeat/convertor/Convertor.html" title="class in org.biojava3.aaproperties.profeat.convertor">Convertor</a></dt>
<dd>
<div class="block">Based on Table 2 of http://nar.oxfordjournals.org/content/34/suppl_2/W32.full.pdf<br/>
 An abstract class to convert a protein sequence into representation of different attribute with each attribute having 3 groups.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/convertor/Convertor.html#group2">group2</a></span> - Static variable in class org.biojava3.aaproperties.profeat.convertor.<a href="./org/biojava3/aaproperties/profeat/convertor/Convertor.html" title="class in org.biojava3.aaproperties.profeat.convertor">Convertor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/convertor/Convertor.html#group3">group3</a></span> - Static variable in class org.biojava3.aaproperties.profeat.convertor.<a href="./org/biojava3/aaproperties/profeat/convertor/Convertor.html" title="class in org.biojava3.aaproperties.profeat.convertor">Convertor</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/GroupIterator.html" title="class in org.biojava.bio.structure"><span class="strong">GroupIterator</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">An iterator over all groups of a structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/GroupIterator.html#GroupIterator(org.biojava.bio.structure.Structure)">GroupIterator(Structure)</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/GroupIterator.html" title="class in org.biojava.bio.structure">GroupIterator</a></dt>
<dd>
<div class="block">Constructs a GroupIterator object.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/GroupType.html" title="class in org.biojava.bio.structure"><span class="strong">GroupType</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">contains only the static declaration of which types of Groups are available</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/GroupType.html#GroupType()">GroupType()</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/GroupType.html" title="class in org.biojava.bio.structure">GroupType</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/FeatureList.html#groupValues()">groupValues()</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/FeatureList.html" title="class in org.biojava3.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block">Create a collection of all unique group ids in the list, as defined
 by the group() method of the features.</div>
</dd>
<dt><a href="./org/biojava3/alignment/routines/GuanUberbacher.html" title="class in org.biojava3.alignment.routines"><span class="strong">GuanUberbacher</span></a>&lt;<a href="./org/biojava3/alignment/routines/GuanUberbacher.html" title="type parameter in GuanUberbacher">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava3/alignment/routines/GuanUberbacher.html" title="type parameter in GuanUberbacher">C</a>&gt;,<a href="./org/biojava3/alignment/routines/GuanUberbacher.html" title="type parameter in GuanUberbacher">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/alignment/routines/package-summary.html">org.biojava3.alignment.routines</a></dt>
<dd>
<div class="block">Guan and Uberbacher defined an algorithm for pairwise global sequence alignments (from the first until the last
 <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template"><code>Compound</code></a> of each <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a>).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/routines/GuanUberbacher.html#GuanUberbacher()">GuanUberbacher()</a></span> - Constructor for class org.biojava3.alignment.routines.<a href="./org/biojava3/alignment/routines/GuanUberbacher.html" title="class in org.biojava3.alignment.routines">GuanUberbacher</a></dt>
<dd>
<div class="block">Before running a pairwise global sequence alignment, data must be sent in via calls to
 <a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html#setQuery(S)"><code>AbstractPairwiseSequenceAligner.setQuery(Sequence)</code></a>, <a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html#setTarget(S)"><code>AbstractPairwiseSequenceAligner.setTarget(Sequence)</code></a>, <a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#setGapPenalty(org.biojava3.alignment.template.GapPenalty)"><code>AbstractMatrixAligner.setGapPenalty(GapPenalty)</code></a>, and
 <a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#setSubstitutionMatrix(org.biojava3.alignment.template.SubstitutionMatrix)"><code>AbstractMatrixAligner.setSubstitutionMatrix(SubstitutionMatrix)</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/routines/GuanUberbacher.html#GuanUberbacher(S, S, org.biojava3.alignment.template.GapPenalty, org.biojava3.alignment.template.SubstitutionMatrix)">GuanUberbacher(S, S, GapPenalty, SubstitutionMatrix&lt;C&gt;)</a></span> - Constructor for class org.biojava3.alignment.routines.<a href="./org/biojava3/alignment/routines/GuanUberbacher.html" title="class in org.biojava3.alignment.routines">GuanUberbacher</a></dt>
<dd>
<div class="block">Prepares for a pairwise global sequence alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/routines/GuanUberbacher.html#GuanUberbacher(S, S, org.biojava3.alignment.template.GapPenalty, org.biojava3.alignment.template.SubstitutionMatrix, int)">GuanUberbacher(S, S, GapPenalty, SubstitutionMatrix&lt;C&gt;, int)</a></span> - Constructor for class org.biojava3.alignment.routines.<a href="./org/biojava3/alignment/routines/GuanUberbacher.html" title="class in org.biojava3.alignment.routines">GuanUberbacher</a></dt>
<dd>
<div class="block">Prepares for a pairwise global sequence alignment.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/gui/GUIAlignmentProgressListener.html" title="class in org.biojava.bio.structure.align.gui"><span class="strong">GUIAlignmentProgressListener</span></a> - Class in <a href="./org/biojava/bio/structure/align/gui/package-summary.html">org.biojava.bio.structure.align.gui</a></dt>
<dd>
<div class="block">a GUI that allows to watch progress as multiple alignments are being processed.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/GUIAlignmentProgressListener.html#GUIAlignmentProgressListener()">GUIAlignmentProgressListener()</a></span> - Constructor for class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/GUIAlignmentProgressListener.html" title="class in org.biojava.bio.structure.align.gui">GUIAlignmentProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/alignment/GuideTree.html" title="class in org.biojava3.alignment"><span class="strong">GuideTree</span></a>&lt;<a href="./org/biojava3/alignment/GuideTree.html" title="type parameter in GuideTree">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava3/alignment/GuideTree.html" title="type parameter in GuideTree">C</a>&gt;,<a href="./org/biojava3/alignment/GuideTree.html" title="type parameter in GuideTree">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/alignment/package-summary.html">org.biojava3.alignment</a></dt>
<dd>
<div class="block">Implements a data structure for a guide tree used during progressive multiple sequence alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/GuideTree.html#GuideTree(java.util.List, java.util.List)">GuideTree(List&lt;S&gt;, List&lt;PairwiseSequenceScorer&lt;S, C&gt;&gt;)</a></span> - Constructor for class org.biojava3.alignment.<a href="./org/biojava3/alignment/GuideTree.html" title="class in org.biojava3.alignment">GuideTree</a></dt>
<dd>
<div class="block">Creates a guide tree for use during progressive multiple sequence alignment.</div>
</dd>
<dt><a href="./org/biojava3/alignment/GuideTree.Node.html" title="class in org.biojava3.alignment"><span class="strong">GuideTree.Node</span></a> - Class in <a href="./org/biojava3/alignment/package-summary.html">org.biojava3.alignment</a></dt>
<dd>
<div class="block">Implements a data structure for the node in a guide tree used during progressive multiple sequence alignment.</div>
</dd>
<dt><a href="./org/biojava3/alignment/template/GuideTreeNode.html" title="interface in org.biojava3.alignment.template"><span class="strong">GuideTreeNode</span></a>&lt;<a href="./org/biojava3/alignment/template/GuideTreeNode.html" title="type parameter in GuideTreeNode">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava3/alignment/template/GuideTreeNode.html" title="type parameter in GuideTreeNode">C</a>&gt;,<a href="./org/biojava3/alignment/template/GuideTreeNode.html" title="type parameter in GuideTreeNode">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava3/alignment/template/package-summary.html">org.biojava3.alignment.template</a></dt>
<dd>
<div class="block">Defines a data structure for the node in a guide tree used during progressive multiple sequence alignment.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/gui/GUIFarmJobRunnable.html" title="class in org.biojava.bio.structure.align.gui"><span class="strong">GUIFarmJobRunnable</span></a> - Class in <a href="./org/biojava/bio/structure/align/gui/package-summary.html">org.biojava.bio.structure.align.gui</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/GUIFarmJobRunnable.html#GUIFarmJobRunnable(org.biojava.bio.structure.align.client.FarmJobParameters)">GUIFarmJobRunnable(FarmJobParameters)</a></span> - Constructor for class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/GUIFarmJobRunnable.html" title="class in org.biojava.bio.structure.align.gui">GUIFarmJobRunnable</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/ce/GuiWrapper.html" title="class in org.biojava.bio.structure.align.ce"><span class="strong">GuiWrapper</span></a> - Class in <a href="./org/biojava/bio/structure/align/ce/package-summary.html">org.biojava.bio.structure.align.ce</a></dt>
<dd>
<div class="block">A class to wrap some of the strucutre.gui classes using Reflection</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/GuiWrapper.html#GuiWrapper()">GuiWrapper()</a></span> - Constructor for class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/GuiWrapper.html" title="class in org.biojava.bio.structure.align.ce">GuiWrapper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/InputStreamProvider.html#GZIP_MAGIC">GZIP_MAGIC</a></span> - Static variable in class org.biojava3.core.util.<a href="./org/biojava3/core/util/InputStreamProvider.html" title="class in org.biojava3.core.util">InputStreamProvider</a></dt>
<dd>
<div class="block">The magic number found at the start of a GZIP stream.</div>
</dd>
</dl>
<a name="_H_">
<!--   -->
</a>
<h2 class="title">H</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/Constraints.html#H">H</a></span> - Static variable in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/Constraints.html" title="class in org.biojava3.aaproperties">Constraints</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/MyValidationEventHandler.html#handleEvent(javax.xml.bind.ValidationEvent)">handleEvent(ValidationEvent)</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/MyValidationEventHandler.html" title="class in org.biojava3.aaproperties.xml">MyValidationEventHandler</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/MyJmolStatusListener.html#handlePopupMenu(int, int)">handlePopupMenu(int, int)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/MyJmolStatusListener.html" title="class in org.biojava.bio.structure.align.gui.jmol">MyJmolStatusListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#has3D()">has3D()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">returns true or false, depending if this group has 3D coordinates or not.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#has3D()">has3D()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">returns true or false, depending if this group has 3D coordinates or not.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#hasAltLoc()">hasAltLoc()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">Test if this group has alternate locations.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#hasAltLoc()">hasAltLoc()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#hasAminoAtoms()">hasAminoAtoms()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">calculate if a groups has all atoms required for an amino acid.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#hasAminoAtoms()">hasAminoAtoms()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">calculate if a groups has all atoms required for an amino acid
     this allows to include chemically modified amino acids that
     are labeled hetatoms into some computations ... the usual way
     to identify if a group is an amino acid is getType() !</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#hasAtom(java.lang.String)">hasAtom(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">Teturns flag whether a particular atom is existing within this group .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#hasAtom(java.lang.String)">hasAtom(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">test is an Atom with name is existing.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Feature.html#hasAttribute(java.lang.String)">hasAttribute(String)</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Feature.html" title="class in org.biojava3.genome.parsers.gff">Feature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Feature.html#hasAttribute(java.lang.String, java.lang.String)">hasAttribute(String, String)</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Feature.html" title="class in org.biojava3.genome.parsers.gff">Feature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/FeatureI.html#hasAttribute(java.lang.String)">hasAttribute(String)</a></span> - Method in interface org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/FeatureI.html" title="interface in org.biojava3.genome.parsers.gff">FeatureI</a></dt>
<dd>
<div class="block">Check if the feature has a value defined for the specified key.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/FeatureI.html#hasAttribute(java.lang.String, java.lang.String)">hasAttribute(String, String)</a></span> - Method in interface org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/FeatureI.html" title="interface in org.biojava3.genome.parsers.gff">FeatureI</a></dt>
<dd>
<div class="block">Check if the feature attributes include the specified key/value pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/FeatureList.html#hasAttribute(java.lang.String)">hasAttribute(String)</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/FeatureList.html" title="class in org.biojava3.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block">Check if any feature in list has the specified attribute key.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/FeatureList.html#hasAttribute(java.lang.String, java.lang.String)">hasAttribute(String, String)</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/FeatureList.html" title="class in org.biojava3.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block">Check if any feature in list has the specified attribute key/value pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#hasChain(java.lang.String)">hasChain(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">check if a chain with the id chainId is contained in this structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#hasChain(java.lang.String)">hasChain(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/StructureName.html#hasChainID()">hasChainID()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/StructureName.html" title="class in org.biojava.bio.structure.align.client">StructureName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/CaseFreeAminoAcidCompoundSet.html#hasCompound(org.biojava3.core.sequence.compound.AminoAcidCompound)">hasCompound(AminoAcidCompound)</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/CaseFreeAminoAcidCompoundSet.html" title="class in org.biojava3.aaproperties.xml">CaseFreeAminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/ModifiedAminoAcidCompoundSet.html#hasCompound(org.biojava3.core.sequence.compound.AminoAcidCompound)">hasCompound(AminoAcidCompound)</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/ModifiedAminoAcidCompoundSet.html" title="class in org.biojava3.aaproperties.xml">ModifiedAminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/AminoAcidCompoundSet.html#hasCompound(org.biojava3.core.sequence.compound.AminoAcidCompound)">hasCompound(AminoAcidCompound)</a></span> - Method in class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/AminoAcidCompoundSet.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompoundSet.html#hasCompound(C)">hasCompound(C)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava3.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/CompoundSet.html#hasCompound(C)">hasCompound(C)</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleProfile.html#hasGap(int)">hasGap(int)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleProfile.html" title="class in org.biojava3.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/Profile.html#hasGap(int)">hasGap(int)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns true if any <a href="./org/biojava3/alignment/template/AlignedSequence.html" title="interface in org.biojava3.alignment.template"><code>AlignedSequence</code></a> has a gap at the given index.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/FeatureList.html#hasGaps(int)">hasGaps(int)</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/FeatureList.html" title="class in org.biojava3.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block">Check size of gaps between successive features in list.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/Hashcoder.html#hash(int, boolean)">hash(int, boolean)</a></span> - Static method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/Hashcoder.html" title="class in org.biojava3.core.util">Hashcoder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/Hashcoder.html#hash(int, char)">hash(int, char)</a></span> - Static method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/Hashcoder.html" title="class in org.biojava3.core.util">Hashcoder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/Hashcoder.html#hash(int, int)">hash(int, int)</a></span> - Static method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/Hashcoder.html" title="class in org.biojava3.core.util">Hashcoder</a></dt>
<dd>
<div class="block">Used for ints, bytes and shorts</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/Hashcoder.html#hash(int, long)">hash(int, long)</a></span> - Static method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/Hashcoder.html" title="class in org.biojava3.core.util">Hashcoder</a></dt>
<dd>
<div class="block">long support done by shifting by 32 (using unsigned shift)</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/Hashcoder.html#hash(int, float)">hash(int, float)</a></span> - Static method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/Hashcoder.html" title="class in org.biojava3.core.util">Hashcoder</a></dt>
<dd>
<div class="block">float support done via <a href="http://java.sun.com/j2se/1.6.0/docs/api/java/lang/Float.html?is-external=true#floatToIntBits(float)" title="class or interface in java.lang"><code>Float.floatToIntBits(float)</code></a> which allows
 the encoding of a float as an int.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/Hashcoder.html#hash(int, double)">hash(int, double)</a></span> - Static method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/Hashcoder.html" title="class in org.biojava3.core.util">Hashcoder</a></dt>
<dd>
<div class="block">double support which is done using the same techinque as float hashing
 except we convert to a long not to an int.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/Hashcoder.html#hash(int, java.lang.Object)">hash(int, Object)</a></span> - Static method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/Hashcoder.html" title="class in org.biojava3.core.util">Hashcoder</a></dt>
<dd>
<div class="block"><code>o</code> is a possibly-null object field, and possibly an
 array.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/PdbPair.html#hashCode()">hashCode()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/PdbPair.html" title="class in org.biojava.bio.structure.align.client">PdbPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/StructureName.html#hashCode()">hashCode()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/StructureName.html" title="class in org.biojava.bio.structure.align.client">StructureName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Author.html#hashCode()">hashCode()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Author.html" title="class in org.biojava.bio.structure">Author</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#hashCode()">hashCode()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html#hashCode()">hashCode()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemCompDescriptor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ResidueNumber.html#hashCode()">hashCode()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ResidueNumber.html" title="class in org.biojava.bio.structure">ResidueNumber</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDescription.html#hashCode()">hashCode()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDescription.html" title="class in org.biojava.bio.structure.scop">ScopDescription</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Site.html#hashCode()">hashCode()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Site.html" title="class in org.biojava.bio.structure">Site</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/AccessionID.html#hashCode()">hashCode()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/AccessionID.html" title="class in org.biojava3.core.sequence">AccessionID</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/AminoAcidCompound.html#hashCode()">hashCode()</a></span> - Method in class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/CodonCompound.html#hashCode()">hashCode()</a></span> - Method in class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/CodonCompound.html" title="class in org.biojava3.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/NucleotideCompound.html#hashCode()">hashCode()</a></span> - Method in class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/FuzzyPoint.html#hashCode()">hashCode()</a></span> - Method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/FuzzyPoint.html" title="class in org.biojava3.core.sequence.location">FuzzyPoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/SimplePoint.html#hashCode()">hashCode()</a></span> - Method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/SimplePoint.html" title="class in org.biojava3.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html#hashCode()">hashCode()</a></span> - Method in class org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava3.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/ArrayListSequenceReader.html#hashCode()">hashCode()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava3.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#hashCode()">hashCode()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava3.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/BitSequenceReader.html#hashCode()">hashCode()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava3.core.sequence.storage">BitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html#hashCode()">hashCode()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava3.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompound.html#hashCode()">hashCode()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompound.html" title="class in org.biojava3.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompoundSet.html#hashCode()">hashCode()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava3.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/CaseInsensitiveCompound.html#hashCode()">hashCode()</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava3.core.sequence.transcription">CaseInsensitiveCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/Table.CaseInsensitiveTriplet.html#hashCode()">hashCode()</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/Table.CaseInsensitiveTriplet.html" title="class in org.biojava3.core.sequence.transcription">Table.CaseInsensitiveTriplet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/Table.Codon.html#hashCode()">hashCode()</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/Table.Codon.html" title="class in org.biojava3.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/data/sequence/FastaSequence.html#hashCode()">hashCode()</a></span> - Method in class org.biojava3.data.sequence.<a href="./org/biojava3/data/sequence/FastaSequence.html" title="class in org.biojava3.data.sequence">FastaSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationImpl.html#hashCode()">hashCode()</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationImpl.html" title="class in org.biojava3.protmod">ProteinModificationImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ModifiedCompoundImpl.html#hashCode()">hashCode()</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ModifiedCompoundImpl.html" title="class in org.biojava3.protmod.structure">ModifiedCompoundImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/StructureAtom.html#hashCode()">hashCode()</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/StructureAtom.html" title="class in org.biojava3.protmod.structure">StructureAtom</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/StructureAtomLinkage.html#hashCode()">hashCode()</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/StructureAtomLinkage.html" title="class in org.biojava3.protmod.structure">StructureAtomLinkage</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/StructureGroup.html#hashCode()">hashCode()</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/StructureGroup.html" title="class in org.biojava3.protmod.structure">StructureGroup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/ronn/Jronn.Range.html#hashCode()">hashCode()</a></span> - Method in class org.biojava3.ronn.<a href="./org/biojava3/ronn/Jronn.Range.html" title="class in org.biojava3.ronn">Jronn.Range</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/ronn/ModelLoader.Model.html#hashCode()">hashCode()</a></span> - Method in class org.biojava3.ronn.<a href="./org/biojava3/ronn/ModelLoader.Model.html" title="class in org.biojava3.ronn">ModelLoader.Model</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/util/Hashcoder.html" title="class in org.biojava3.core.util"><span class="strong">Hashcoder</span></a> - Class in <a href="./org/biojava3/core/util/package-summary.html">org.biojava3.core.util</a></dt>
<dd>
<div class="block">Contains helper methods for generating a HashCode without having to resort to
 the commons lang hashcode builders.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/Hashcoder.html#Hashcoder()">Hashcoder()</a></span> - Constructor for class org.biojava3.core.util.<a href="./org/biojava3/core/util/Hashcoder.html" title="class in org.biojava3.core.util">Hashcoder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#hasJournalArticle()">hasJournalArticle()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">return whether or not the entry has an associated journal article
 or publication.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#hasJournalArticle()">hasJournalArticle()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">Return whether or not the entry has an associated journal article
 or publication.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomIterator.html#hasNext()">hasNext()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomIterator.html" title="class in org.biojava.bio.structure">AtomIterator</a></dt>
<dd>
<div class="block">Is there a next atom ?</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/GroupIterator.html#hasNext()">hasNext()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/GroupIterator.html" title="class in org.biojava.bio.structure">GroupIterator</a></dt>
<dd>
<div class="block">is there a group after the current one in the structure?</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/FlatFileInstallation.html#hasNext()">hasNext()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/FlatFileInstallation.html" title="class in org.biojava.bio.structure.server">FlatFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html#hasNext()">hasNext()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html" title="class in org.biojava.bio.structure.server">MMCIFFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/PDBInstallation.html#hasNext()">hasNext()</a></span> - Method in interface org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/PDBInstallation.html" title="interface in org.biojava.bio.structure.server">PDBInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
<div class="block">return if the iteration over all structures will return another structure</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceMixin.SequenceIterator.html#hasNext()">hasNext()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceMixin.SequenceIterator.html" title="class in org.biojava3.core.sequence.template">SequenceMixin.SequenceIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/LocIterator.html#hasNext(int, int)">hasNext(int, int)</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/LocIterator.html" title="class in org.biojava3.genome.parsers.gff">LocIterator</a></dt>
<dd>
<div class="block">Check if next window of specified size is available.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/LocIterator.html#hasNext()">hasNext()</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/LocIterator.html" title="class in org.biojava3.genome.parsers.gff">LocIterator</a></dt>
<dd>
<div class="block">Check if next window of default size is available.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/SimpleStructureServer.html#hasNextStructure()">hasNextStructure()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/SimpleStructureServer.html" title="class in org.biojava.bio.structure.server">SimpleStructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureServer.html#hasNextStructure()">hasNextStructure()</a></span> - Method in interface org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureServer.html" title="interface in org.biojava.bio.structure.server">StructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#hasParent()">hasParent()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/JFatCatClient.html#hasPrecalculatedResult(java.lang.String, java.lang.String, java.lang.String, java.lang.String)">hasPrecalculatedResult(String, String, String, String)</a></span> - Static method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/JFatCatClient.html" title="class in org.biojava.bio.structure.align.client">JFatCatClient</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/JFatCatClient.html#hasPrecalculatedResult(java.lang.String, java.lang.String, java.lang.String, java.lang.String, int)">hasPrecalculatedResult(String, String, String, String, int)</a></span> - Static method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/JFatCatClient.html" title="class in org.biojava.bio.structure.align.client">JFatCatClient</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/xml/HasResultXMLConverter.html" title="class in org.biojava.bio.structure.align.xml"><span class="strong">HasResultXMLConverter</span></a> - Class in <a href="./org/biojava/bio/structure/align/xml/package-summary.html">org.biojava.bio.structure.align.xml</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/HasResultXMLConverter.html#HasResultXMLConverter()">HasResultXMLConverter()</a></span> - Constructor for class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/HasResultXMLConverter.html" title="class in org.biojava.bio.structure.align.xml">HasResultXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/secstruc/HBond.html" title="class in org.biojava.bio.structure.secstruc"><span class="strong">HBond</span></a> - Class in <a href="./org/biojava/bio/structure/secstruc/package-summary.html">org.biojava.bio.structure.secstruc</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/HBond.html#HBond()">HBond()</a></span> - Constructor for class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/HBond.html" title="class in org.biojava.bio.structure.secstruc">HBond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStruc.html#HBONDHIGHENERGY">HBONDHIGHENERGY</a></span> - Static variable in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStruc.html" title="class in org.biojava.bio.structure.secstruc">SecStruc</a></dt>
<dd>
<div class="block">higher limit for H-bond energy</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStruc.html#HBONDLOWENERGY">HBONDLOWENERGY</a></span> - Static variable in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStruc.html" title="class in org.biojava.bio.structure.secstruc">SecStruc</a></dt>
<dd>
<div class="block">Minimal H-bond energy in cal / mol</div>
</dd>
<dt><a href="./org/biojava3/core/exceptions/HeaderParseException.html" title="class in org.biojava3.core.exceptions"><span class="strong">HeaderParseException</span></a> - Error in <a href="./org/biojava3/core/exceptions/package-summary.html">org.biojava3.core.exceptions</a></dt>
<dd>
<div class="block">Currently not used</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/exceptions/HeaderParseException.html#HeaderParseException(java.lang.String)">HeaderParseException(String)</a></span> - Constructor for error org.biojava3.core.exceptions.<a href="./org/biojava3/core/exceptions/HeaderParseException.html" title="class in org.biojava3.core.exceptions">HeaderParseException</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileParser.html#HELIX">HELIX</a></span> - Static variable in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileParser.html" title="class in org.biojava.bio.structure.io">PDBFileParser</a></dt>
<dd>
<div class="block">Helix secondary structure assignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#helix">helix</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#helixmax">helixmax</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#helixmin">helixmin</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/gui/HelpDialog.html" title="class in org.biojava.bio.structure.align.gui"><span class="strong">HelpDialog</span></a> - Class in <a href="./org/biojava/bio/structure/align/gui/package-summary.html">org.biojava.bio.structure.align.gui</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/HelpDialog.html#HelpDialog()">HelpDialog()</a></span> - Constructor for class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/HelpDialog.html" title="class in org.biojava.bio.structure.align.gui">HelpDialog</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/GroupType.html#HETATM">HETATM</a></span> - Static variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/GroupType.html" title="class in org.biojava.bio.structure">GroupType</a></dt>
<dd>
<div class="block">the type for hetero groups</div>
</dd>
<dt><a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure"><span class="strong">HetatomImpl</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">Generic Implementation of a Group interface.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#HetatomImpl()">HetatomImpl()</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/SystemInfo.html#hexPropertyNames">hexPropertyNames</a></span> - Static variable in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/SystemInfo.html" title="class in org.biojava.bio.structure.align.gui">SystemInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#hieFileName">hieFileName</a></span> - Static variable in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/alignment/template/HierarchicalClusterer.html" title="interface in org.biojava3.alignment.template"><span class="strong">HierarchicalClusterer</span></a>&lt;<a href="./org/biojava3/alignment/template/HierarchicalClusterer.html" title="type parameter in HierarchicalClusterer">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava3/alignment/template/HierarchicalClusterer.html" title="type parameter in HierarchicalClusterer">C</a>&gt;,<a href="./org/biojava3/alignment/template/HierarchicalClusterer.html" title="type parameter in HierarchicalClusterer">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava3/alignment/template/package-summary.html">org.biojava3.alignment.template</a></dt>
<dd>
<div class="block">Defines a clustering algorithm that converts a distance matrix into a tree.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#His">His</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/gui/util/color/HSVColorSpace.html" title="class in org.biojava.bio.structure.gui.util.color"><span class="strong">HSVColorSpace</span></a> - Class in <a href="./org/biojava/bio/structure/gui/util/color/package-summary.html">org.biojava.bio.structure.gui.util.color</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/HSVColorSpace.html#HSVColorSpace()">HSVColorSpace()</a></span> - Constructor for class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/HSVColorSpace.html" title="class in org.biojava.bio.structure.gui.util.color">HSVColorSpace</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/util/HTTPConnectionTools.html" title="class in org.biojava.bio.structure.align.util"><span class="strong">HTTPConnectionTools</span></a> - Class in <a href="./org/biojava/bio/structure/align/util/package-summary.html">org.biojava.bio.structure.align.util</a></dt>
<dd>
<div class="block">a class that takes care about opening HttpURLConnections and sets the proper timeouts</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/HTTPConnectionTools.html#HTTPConnectionTools()">HTTPConnectionTools()</a></span> - Constructor for class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/HTTPConnectionTools.html" title="class in org.biojava.bio.structure.align.util">HTTPConnectionTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#hyd">hyd</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#hyd2">hyd2</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#hydmax">hydmax</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#hydmin">hydmin</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#hydrophobic">hydrophobic</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Maths.html#hypot(double, double)">hypot(double, double)</a></span> - Static method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Maths.html" title="class in org.biojava.bio.structure.jama">Maths</a></dt>
<dd>
<div class="block">sqrt(a^2 + b^2) without under/overflow.</div>
</dd>
</dl>
<a name="_I_">
<!--   -->
</a>
<h2 class="title">I</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/Constraints.html#I">I</a></span> - Static variable in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/Constraints.html" title="class in org.biojava3.aaproperties">Constraints</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ProteinModificationIdentifier.html#identify(org.biojava.bio.structure.Structure)">identify(Structure)</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ProteinModificationIdentifier.html" title="class in org.biojava3.protmod.structure">ProteinModificationIdentifier</a></dt>
<dd>
<div class="block">Identify all registered modifications in a structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ProteinModificationIdentifier.html#identify(org.biojava.bio.structure.Structure, java.util.Set)">identify(Structure, Set&lt;ProteinModification&gt;)</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ProteinModificationIdentifier.html" title="class in org.biojava3.protmod.structure">ProteinModificationIdentifier</a></dt>
<dd>
<div class="block">Identify a set of modifications in a structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ProteinModificationIdentifier.html#identify(org.biojava.bio.structure.Chain)">identify(Chain)</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ProteinModificationIdentifier.html" title="class in org.biojava3.protmod.structure">ProteinModificationIdentifier</a></dt>
<dd>
<div class="block">Identify all registered modifications in a chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ProteinModificationIdentifier.html#identify(java.util.List)">identify(List&lt;Chain&gt;)</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ProteinModificationIdentifier.html" title="class in org.biojava3.protmod.structure">ProteinModificationIdentifier</a></dt>
<dd>
<div class="block">Identify all registered modifications in chains.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ProteinModificationIdentifier.html#identify(org.biojava.bio.structure.Chain, java.util.Set)">identify(Chain, Set&lt;ProteinModification&gt;)</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ProteinModificationIdentifier.html" title="class in org.biojava3.protmod.structure">ProteinModificationIdentifier</a></dt>
<dd>
<div class="block">Identify a set of modifications in a a chains.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ProteinModificationIdentifier.html#identify(java.util.List, java.util.Set)">identify(List&lt;Chain&gt;, Set&lt;ProteinModification&gt;)</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ProteinModificationIdentifier.html" title="class in org.biojava3.protmod.structure">ProteinModificationIdentifier</a></dt>
<dd>
<div class="block">Identify a set of modifications in a a list of chains.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#identity(int, int)">identity(int, int)</a></span> - Static method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Generate identity matrix</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html#IDX_COLOR">IDX_COLOR</a></span> - Static variable in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html" title="class in org.biojava.bio.structure.gui.util">SequenceScalePanel</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/helper/IdxComparator.html" title="class in org.biojava.bio.structure.align.helper"><span class="strong">IdxComparator</span></a> - Class in <a href="./org/biojava/bio/structure/align/helper/package-summary.html">org.biojava.bio.structure.align.helper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/IdxComparator.html#IdxComparator()">IdxComparator()</a></span> - Constructor for class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/IdxComparator.html" title="class in org.biojava.bio.structure.align.helper">IdxComparator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#Ile">Ile</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/GapArray.html#index">index</a></span> - Variable in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/GapArray.html" title="class in org.biojava.bio.structure.align.helper">GapArray</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/ronn/RonnConstraint.html#INDEX">INDEX</a></span> - Static variable in class org.biojava3.ronn.<a href="./org/biojava3/ronn/RonnConstraint.html" title="class in org.biojava3.ronn">RonnConstraint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceMixin.html#indexOf(org.biojava3.core.sequence.template.Sequence, C)">indexOf(Sequence&lt;C&gt;, C)</a></span> - Static method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceMixin.html" title="class in org.biojava3.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">Performs a linear search of the given Sequence for the given compound.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/helper/IndexPair.html" title="class in org.biojava.bio.structure.align.helper"><span class="strong">IndexPair</span></a> - Class in <a href="./org/biojava/bio/structure/align/helper/package-summary.html">org.biojava.bio.structure.align.helper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/IndexPair.html#IndexPair()">IndexPair()</a></span> - Constructor for class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/IndexPair.html" title="class in org.biojava.bio.structure.align.helper">IndexPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/IndexPair.html#IndexPair(short, short)">IndexPair(short, short)</a></span> - Constructor for class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/IndexPair.html" title="class in org.biojava.bio.structure.align.helper">IndexPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationRegistry.html#init()">init()</a></span> - Static method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationRegistry.html" title="class in org.biojava3.protmod">ProteinModificationRegistry</a></dt>
<dd>
<div class="block">Initialization the static variables and register common modifications.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationRegistry.html#init(java.io.InputStream)">init(InputStream)</a></span> - Static method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationRegistry.html" title="class in org.biojava3.protmod">ProteinModificationRegistry</a></dt>
<dd>
<div class="block">Initialization the static variables and register common modifications.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/MenuCreator.html#initAlignmentGUIMenu(javax.swing.JFrame)">initAlignmentGUIMenu(JFrame)</a></span> - Static method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/MenuCreator.html" title="class in org.biojava.bio.structure.align.gui">MenuCreator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/SimpleStructureServer.html#initCache()">initCache()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/SimpleStructureServer.html" title="class in org.biojava.bio.structure.server">SimpleStructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/Constraints.html#initExtinctionCoefficient()">initExtinctionCoefficient()</a></span> - Static method in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/Constraints.html" title="class in org.biojava3.aaproperties">Constraints</a></dt>
<dd>
<div class="block">Does the initialization of extinction coefficient based on
 http://au.expasy.org/tools/protparam-doc.html</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/MatrixListener.html#initializeBreakFlag(boolean[][])">initializeBreakFlag(boolean[][])</a></span> - Method in interface org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/MatrixListener.html" title="interface in org.biojava.bio.structure.align.ce">MatrixListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/MenuCreator.html#initMenu(javax.swing.JFrame, org.biojava.bio.structure.align.gui.jmol.StructureAlignmentJmol, org.biojava.bio.structure.align.model.AFPChain)">initMenu(JFrame, StructureAlignmentJmol, AFPChain)</a></span> - Static method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/MenuCreator.html" title="class in org.biojava.bio.structure.align.gui">MenuCreator</a></dt>
<dd>
<div class="block">provide a JMenuBar that can be added to a JFrame</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/MenuCreator.html#initMenu()">initMenu()</a></span> - Static method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/MenuCreator.html" title="class in org.biojava.bio.structure.gui.util">MenuCreator</a></dt>
<dd>
<div class="block">provide a JMenuBar that can be added to a JFrame</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.Builder.html#initMet(boolean)">initMet(boolean)</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava3.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/Constraints.html#initMolecularWeight()">initMolecularWeight()</a></span> - Static method in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/Constraints.html" title="class in org.biojava3.aaproperties">Constraints</a></dt>
<dd>
<div class="block">Does the initialization of molecular weights based on http://au.expasy.org/tools/findmod/findmod_masses.html#AA</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#initSumOfDistances(int, int, int, int, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">initSumOfDistances(int, int, int, int, Atom[], Atom[])</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/util/InputStreamProvider.html" title="class in org.biojava3.core.util"><span class="strong">InputStreamProvider</span></a> - Class in <a href="./org/biojava3/core/util/package-summary.html">org.biojava3.core.util</a></dt>
<dd>
<div class="block">A class that provides an InputStream from a File.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/InputStreamProvider.html#InputStreamProvider()">InputStreamProvider()</a></span> - Constructor for class org.biojava3.core.util.<a href="./org/biojava3/core/util/InputStreamProvider.html" title="class in org.biojava3.core.util">InputStreamProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/XMLHelper.html#inputStreamToDocument(java.io.InputStream)">inputStreamToDocument(InputStream)</a></span> - Static method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/XMLHelper.html" title="class in org.biojava3.core.util">XMLHelper</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/location/InsdcLocations.html" title="class in org.biojava3.core.sequence.location"><span class="strong">InsdcLocations</span></a> - Class in <a href="./org/biojava3/core/sequence/location/package-summary.html">org.biojava3.core.sequence.location</a></dt>
<dd>
<div class="block">A collection of locations which are used whenever we work with INSDC; some
 of which could be deprecated (from INSDC's point of view) yet appear
 in records.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/InsdcLocations.html#InsdcLocations()">InsdcLocations()</a></span> - Constructor for class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/InsdcLocations.html" title="class in org.biojava3.core.sequence.location">InsdcLocations</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/location/InsdcLocations.BondLocation.html" title="class in org.biojava3.core.sequence.location"><span class="strong">InsdcLocations.BondLocation</span></a> - Class in <a href="./org/biojava3/core/sequence/location/package-summary.html">org.biojava3.core.sequence.location</a></dt>
<dd>
<div class="block">Used to represent bond locations equivalent to bond(7,8) or bond(7).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/InsdcLocations.BondLocation.html#InsdcLocations.BondLocation(org.biojava3.core.sequence.location.template.Location...)">InsdcLocations.BondLocation(Location...)</a></span> - Constructor for class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/InsdcLocations.BondLocation.html" title="class in org.biojava3.core.sequence.location">InsdcLocations.BondLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/InsdcLocations.BondLocation.html#InsdcLocations.BondLocation(java.util.List)">InsdcLocations.BondLocation(List&lt;Location&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/InsdcLocations.BondLocation.html" title="class in org.biojava3.core.sequence.location">InsdcLocations.BondLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/location/InsdcLocations.GroupLocation.html" title="class in org.biojava3.core.sequence.location"><span class="strong">InsdcLocations.GroupLocation</span></a> - Class in <a href="./org/biojava3/core/sequence/location/package-summary.html">org.biojava3.core.sequence.location</a></dt>
<dd>
<div class="block">Deprecated in INSDC yet still appears; equivalent to the order()
 directive except no 5' to 3' ordering is defined.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/InsdcLocations.GroupLocation.html#InsdcLocations.GroupLocation(org.biojava3.core.sequence.location.template.Point, org.biojava3.core.sequence.location.template.Point, org.biojava3.core.sequence.Strand, boolean, org.biojava3.core.sequence.location.template.Location...)">InsdcLocations.GroupLocation(Point, Point, Strand, boolean, Location...)</a></span> - Constructor for class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/InsdcLocations.GroupLocation.html" title="class in org.biojava3.core.sequence.location">InsdcLocations.GroupLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/InsdcLocations.GroupLocation.html#InsdcLocations.GroupLocation(org.biojava3.core.sequence.location.template.Point, org.biojava3.core.sequence.location.template.Point, org.biojava3.core.sequence.Strand, org.biojava3.core.sequence.location.template.Location...)">InsdcLocations.GroupLocation(Point, Point, Strand, Location...)</a></span> - Constructor for class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/InsdcLocations.GroupLocation.html" title="class in org.biojava3.core.sequence.location">InsdcLocations.GroupLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/InsdcLocations.GroupLocation.html#InsdcLocations.GroupLocation(int, int, org.biojava3.core.sequence.Strand, org.biojava3.core.sequence.location.template.Location...)">InsdcLocations.GroupLocation(int, int, Strand, Location...)</a></span> - Constructor for class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/InsdcLocations.GroupLocation.html" title="class in org.biojava3.core.sequence.location">InsdcLocations.GroupLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/InsdcLocations.GroupLocation.html#InsdcLocations.GroupLocation(org.biojava3.core.sequence.location.template.Point, org.biojava3.core.sequence.location.template.Point, org.biojava3.core.sequence.Strand, boolean, java.util.List)">InsdcLocations.GroupLocation(Point, Point, Strand, boolean, List&lt;Location&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/InsdcLocations.GroupLocation.html" title="class in org.biojava3.core.sequence.location">InsdcLocations.GroupLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/InsdcLocations.GroupLocation.html#InsdcLocations.GroupLocation(org.biojava3.core.sequence.location.template.Point, org.biojava3.core.sequence.location.template.Point, org.biojava3.core.sequence.Strand, java.util.List)">InsdcLocations.GroupLocation(Point, Point, Strand, List&lt;Location&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/InsdcLocations.GroupLocation.html" title="class in org.biojava3.core.sequence.location">InsdcLocations.GroupLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/InsdcLocations.GroupLocation.html#InsdcLocations.GroupLocation(int, int, org.biojava3.core.sequence.Strand, java.util.List)">InsdcLocations.GroupLocation(int, int, Strand, List&lt;Location&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/InsdcLocations.GroupLocation.html" title="class in org.biojava3.core.sequence.location">InsdcLocations.GroupLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/location/InsdcLocations.OneOfLocation.html" title="class in org.biojava3.core.sequence.location"><span class="strong">InsdcLocations.OneOfLocation</span></a> - Class in <a href="./org/biojava3/core/sequence/location/package-summary.html">org.biojava3.core.sequence.location</a></dt>
<dd>
<div class="block">Deprecated in INSDC; refers to a set of locations of which one
 location could be valid e.g. one-of(location, location, location).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/InsdcLocations.OneOfLocation.html#InsdcLocations.OneOfLocation(org.biojava3.core.sequence.location.template.Location...)">InsdcLocations.OneOfLocation(Location...)</a></span> - Constructor for class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/InsdcLocations.OneOfLocation.html" title="class in org.biojava3.core.sequence.location">InsdcLocations.OneOfLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/InsdcLocations.OneOfLocation.html#InsdcLocations.OneOfLocation(java.util.List)">InsdcLocations.OneOfLocation(List&lt;Location&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/InsdcLocations.OneOfLocation.html" title="class in org.biojava3.core.sequence.location">InsdcLocations.OneOfLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/location/InsdcLocations.OrderLocation.html" title="class in org.biojava3.core.sequence.location"><span class="strong">InsdcLocations.OrderLocation</span></a> - Class in <a href="./org/biojava3/core/sequence/location/package-summary.html">org.biojava3.core.sequence.location</a></dt>
<dd>
<div class="block">Used to describe a 5' to 3' ordering but no firm assurance it is correct</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/InsdcLocations.OrderLocation.html#InsdcLocations.OrderLocation(org.biojava3.core.sequence.location.template.Point, org.biojava3.core.sequence.location.template.Point, org.biojava3.core.sequence.Strand, boolean, org.biojava3.core.sequence.location.template.Location...)">InsdcLocations.OrderLocation(Point, Point, Strand, boolean, Location...)</a></span> - Constructor for class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/InsdcLocations.OrderLocation.html" title="class in org.biojava3.core.sequence.location">InsdcLocations.OrderLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/InsdcLocations.OrderLocation.html#InsdcLocations.OrderLocation(org.biojava3.core.sequence.location.template.Point, org.biojava3.core.sequence.location.template.Point, org.biojava3.core.sequence.Strand, org.biojava3.core.sequence.location.template.Location...)">InsdcLocations.OrderLocation(Point, Point, Strand, Location...)</a></span> - Constructor for class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/InsdcLocations.OrderLocation.html" title="class in org.biojava3.core.sequence.location">InsdcLocations.OrderLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/InsdcLocations.OrderLocation.html#InsdcLocations.OrderLocation(int, int, org.biojava3.core.sequence.Strand, org.biojava3.core.sequence.location.template.Location...)">InsdcLocations.OrderLocation(int, int, Strand, Location...)</a></span> - Constructor for class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/InsdcLocations.OrderLocation.html" title="class in org.biojava3.core.sequence.location">InsdcLocations.OrderLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/InsdcLocations.OrderLocation.html#InsdcLocations.OrderLocation(org.biojava3.core.sequence.location.template.Point, org.biojava3.core.sequence.location.template.Point, org.biojava3.core.sequence.Strand, boolean, java.util.List)">InsdcLocations.OrderLocation(Point, Point, Strand, boolean, List&lt;Location&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/InsdcLocations.OrderLocation.html" title="class in org.biojava3.core.sequence.location">InsdcLocations.OrderLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/InsdcLocations.OrderLocation.html#InsdcLocations.OrderLocation(org.biojava3.core.sequence.location.template.Point, org.biojava3.core.sequence.location.template.Point, org.biojava3.core.sequence.Strand, java.util.List)">InsdcLocations.OrderLocation(Point, Point, Strand, List&lt;Location&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/InsdcLocations.OrderLocation.html" title="class in org.biojava3.core.sequence.location">InsdcLocations.OrderLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/InsdcLocations.OrderLocation.html#InsdcLocations.OrderLocation(int, int, org.biojava3.core.sequence.Strand, java.util.List)">InsdcLocations.OrderLocation(int, int, Strand, List&lt;Location&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/InsdcLocations.OrderLocation.html" title="class in org.biojava3.core.sequence.location">InsdcLocations.OrderLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/location/InsdcParser.html" title="class in org.biojava3.core.sequence.location"><span class="strong">InsdcParser</span></a> - Class in <a href="./org/biojava3/core/sequence/location/package-summary.html">org.biojava3.core.sequence.location</a></dt>
<dd>
<div class="block">Parser for working with INSDC style locations.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/InsdcParser.html#InsdcParser()">InsdcParser()</a></span> - Constructor for class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/InsdcParser.html" title="class in org.biojava3.core.sequence.location">InsdcParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/InsdcParser.html#InsdcParser(org.biojava3.core.sequence.DataSource)">InsdcParser(DataSource)</a></span> - Constructor for class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/InsdcParser.html" title="class in org.biojava3.core.sequence.location">InsdcParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/ColorInterpolator.html#interpolate(java.awt.Color, java.awt.Color, float)">interpolate(Color, Color, float)</a></span> - Method in interface org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/ColorInterpolator.html" title="interface in org.biojava.bio.structure.gui.util.color">ColorInterpolator</a></dt>
<dd>
<div class="block">Interpolates to a color between a and b</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/LinearColorInterpolator.html#interpolate(java.awt.Color, java.awt.Color, float)">interpolate(Color, Color, float)</a></span> - Method in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/LinearColorInterpolator.html" title="class in org.biojava.bio.structure.gui.util.color">LinearColorInterpolator</a></dt>
<dd>
<div class="block">Interpolates to a color between a and b</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/AlignmentCalc.html#interrupt()">interrupt()</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/AlignmentCalc.html" title="class in org.biojava.bio.structure.align.gui">AlignmentCalc</a></dt>
<dd>
<div class="block">stops what is currently happening and does not continue</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/AlignmentCalcDB.html#interrupt()">interrupt()</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/AlignmentCalcDB.html" title="class in org.biojava.bio.structure.align.gui">AlignmentCalcDB</a></dt>
<dd>
<div class="block">stops what is currently happening and does not continue</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/AlignmentCalculationRunnable.html#interrupt()">interrupt()</a></span> - Method in interface org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/AlignmentCalculationRunnable.html" title="interface in org.biojava.bio.structure.align.gui">AlignmentCalculationRunnable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/AlignmentCalc.html#interrupt()">interrupt()</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/AlignmentCalc.html" title="class in org.biojava.bio.structure.gui.util">AlignmentCalc</a></dt>
<dd>
<div class="block">stops what is currently happening and does not continue</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Location.html#intersection(org.biojava3.genome.parsers.gff.Location)">intersection(Location)</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Location.html" title="class in org.biojava3.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Return the intersection, or null if no overlap.</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/IntronSequence.html" title="class in org.biojava3.core.sequence"><span class="strong">IntronSequence</span></a> - Class in <a href="./org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/IntronSequence.html#IntronSequence(org.biojava3.core.sequence.GeneSequence, int, int)">IntronSequence(GeneSequence, int, int)</a></span> - Constructor for class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/IntronSequence.html" title="class in org.biojava3.core.sequence">IntronSequence</a></dt>
<dd>
<div class="block">Place holder for Intron sequence features</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/Table.CaseInsensitiveTriplet.html#intValue()">intValue()</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/Table.CaseInsensitiveTriplet.html" title="class in org.biojava3.core.sequence.transcription">Table.CaseInsensitiveTriplet</a></dt>
<dd>
<div class="block">Attempts to provide an int version of this codon which multiplies
 each position by</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#inverse()">inverse()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Matrix inverse or pseudoinverse</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceMixin.html#inverse(org.biojava3.core.sequence.template.Sequence)">inverse(Sequence&lt;C&gt;)</a></span> - Static method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceMixin.html" title="class in org.biojava3.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">A method which attempts to do the right thing when is comes to a
 reverse/reverse complement</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#invert(org.biojava.bio.structure.Atom)">invert(Atom)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/io/util/IOUtils.html" title="class in org.biojava3.core.sequence.io.util"><span class="strong">IOUtils</span></a> - Class in <a href="./org/biojava3/core/sequence/io/util/package-summary.html">org.biojava3.core.sequence.io.util</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/util/IOUtils.html#IOUtils()">IOUtils()</a></span> - Constructor for class org.biojava3.core.sequence.io.util.<a href="./org/biojava3/core/sequence/io/util/IOUtils.html" title="class in org.biojava3.core.sequence.io.util">IOUtils</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/io/util/IOUtils.ReaderProcessor.html" title="interface in org.biojava3.core.sequence.io.util"><span class="strong">IOUtils.ReaderProcessor</span></a> - Interface in <a href="./org/biojava3/core/sequence/io/util/package-summary.html">org.biojava3.core.sequence.io.util</a></dt>
<dd>
<div class="block">Closure interface used when working with
 <code>IOUtils#processReader(String)</code>.</div>
</dd>
<dt><a href="./org/biojava3/aaproperties/IPeptideProperties.html" title="interface in org.biojava3.aaproperties"><span class="strong">IPeptideProperties</span></a> - Interface in <a href="./org/biojava3/aaproperties/package-summary.html">org.biojava3.aaproperties</a></dt>
<dd>
<div class="block">An interface to generate some basic physico-chemical properties of protein sequences.</div>
</dd>
<dt><a href="./org/biojava3/aaproperties/profeat/IProfeatProperties.html" title="interface in org.biojava3.aaproperties.profeat"><span class="strong">IProfeatProperties</span></a> - Interface in <a href="./org/biojava3/aaproperties/profeat/package-summary.html">org.biojava3.aaproperties.profeat</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/aaproperties/profeat/IProfeatProperties.ATTRIBUTE.html" title="enum in org.biojava3.aaproperties.profeat"><span class="strong">IProfeatProperties.ATTRIBUTE</span></a> - Enum in <a href="./org/biojava3/aaproperties/profeat/package-summary.html">org.biojava3.aaproperties.profeat</a></dt>
<dd>
<div class="block">Enumeration of the seven different attributes</div>
</dd>
<dt><a href="./org/biojava3/aaproperties/profeat/IProfeatProperties.DISTRIBUTION.html" title="enum in org.biojava3.aaproperties.profeat"><span class="strong">IProfeatProperties.DISTRIBUTION</span></a> - Enum in <a href="./org/biojava3/aaproperties/profeat/package-summary.html">org.biojava3.aaproperties.profeat</a></dt>
<dd>
<div class="block">Enumeration of the distribution for the first, first 25%, first 50%, first 75% and 100% of the grouping</div>
</dd>
<dt><a href="./org/biojava3/aaproperties/profeat/IProfeatProperties.GROUPING.html" title="enum in org.biojava3.aaproperties.profeat"><span class="strong">IProfeatProperties.GROUPING</span></a> - Enum in <a href="./org/biojava3/aaproperties/profeat/package-summary.html">org.biojava3.aaproperties.profeat</a></dt>
<dd>
<div class="block">Enumeration of the three different groupings for each attributes</div>
</dd>
<dt><a href="./org/biojava3/aaproperties/profeat/IProfeatProperties.TRANSITION.html" title="enum in org.biojava3.aaproperties.profeat"><span class="strong">IProfeatProperties.TRANSITION</span></a> - Enum in <a href="./org/biojava3/aaproperties/profeat/package-summary.html">org.biojava3.aaproperties.profeat</a></dt>
<dd>
<div class="block">Enumeration of the transition between groupA and groupB</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Location.html#isAfter(org.biojava3.genome.parsers.gff.Location)">isAfter(Location)</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Location.html" title="class in org.biojava3.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Check if this location is entirely after the other location (no overlap).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileParsingParameters.html#isAlignSeqRes()">isAlignSeqRes()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileParsingParameters.html" title="class in org.biojava.bio.structure.io">FileParsingParameters</a></dt>
<dd>
<div class="block">Flag if the SEQRES amino acids should be aligned with the ATOM amino acids.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/data/sequence/SequenceUtil.html#isAmbiguosProtein(java.lang.String)">isAmbiguosProtein(String)</a></span> - Static method in class org.biojava3.data.sequence.<a href="./org/biojava3/data/sequence/SequenceUtil.html" title="class in org.biojava3.data.sequence">SequenceUtil</a></dt>
<dd>
<div class="block">Check whether the sequence confirms to amboguous protein sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/NucleotideCompound.html#isAmbiguous()">isAmbiguous()</a></span> - Method in class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/StructureGroup.html#isAminoAcid()">isAminoAcid()</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/StructureGroup.html" title="class in org.biojava3.protmod.structure">StructureGroup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#isAutoFetch()">isAutoFetch()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#isAutoFetch()">isAutoFetch()</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>
<div class="block">Does the cache automatically download files that are missing from the local installation from the PDB FTP site?</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/MMCIFFileReader.html#isAutoFetch()">isAutoFetch()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/MMCIFFileReader.html" title="class in org.biojava.bio.structure.io">MMCIFFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#isAutoFetch()">isAutoFetch()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/StructureIOFile.html#isAutoFetch()">isAutoFetch()</a></span> - Method in interface org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/StructureIOFile.html" title="interface in org.biojava.bio.structure.io">StructureIOFile</a></dt>
<dd>
<div class="block">Fetch files automatically from FTP server.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html#isAutoFetch()">isAutoFetch()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html" title="class in org.biojava.bio.structure.server">MMCIFFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
<div class="block">should the parser to fetch missing mmCif files from the RCSB FTP server automatically?</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Location.html#isBefore(org.biojava3.genome.parsers.gff.Location)">isBefore(Location)</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Location.html" title="class in org.biojava3.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Check if this location is entirely before other location (no overlap).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html#isBetweenCompounds()">isBetweenCompounds()</a></span> - Method in class org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava3.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/Location.html#isBetweenCompounds()">isBetweenCompounds()</a></span> - Method in interface org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/Location.html" title="interface in org.biojava3.core.sequence.location.template">Location</a></dt>
<dd>
<div class="block">Returns true if the position is meant to represent a point between
 two points such as 78^79.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#isBiologicalAssembly()">isBiologicalAssembly()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">Gets flag that indicates if this structure is a biological assembly</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#isBiologicalAssembly()">isBiologicalAssembly()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">Gets flag that indicates if this structure is a biological assembly</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#isChalcogen()">isChalcogen()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns <CODE>true</CODE> if Element is a chalcogen (O, S, Se, Te, Po).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleAlignedSequence.html#isCircular()">isCircular()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleAlignedSequence.html" title="class in org.biojava3.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleProfile.html#isCircular()">isCircular()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleProfile.html" title="class in org.biojava3.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AlignedSequence.html#isCircular()">isCircular()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AlignedSequence.html" title="interface in org.biojava3.alignment.template">AlignedSequence</a></dt>
<dd>
<div class="block">Returns true if this <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a> wraps around from the last alignment column back to the first.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/Profile.html#isCircular()">isCircular()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns true if any <a href="./org/biojava3/alignment/template/AlignedSequence.html" title="interface in org.biojava3.alignment.template"><code>AlignedSequence</code></a> is circular.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html#isCircular()">isCircular()</a></span> - Method in class org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava3.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/Location.html#isCircular()">isCircular()</a></span> - Method in interface org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/Location.html" title="interface in org.biojava3.core.sequence.location.template">Location</a></dt>
<dd>
<div class="block">Indicates if this location is circular.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/CaseFreeAminoAcidCompoundSet.html#isComplementable()">isComplementable()</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/CaseFreeAminoAcidCompoundSet.html" title="class in org.biojava3.aaproperties.xml">CaseFreeAminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/ModifiedAminoAcidCompoundSet.html#isComplementable()">isComplementable()</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/ModifiedAminoAcidCompoundSet.html" title="class in org.biojava3.aaproperties.xml">ModifiedAminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/AminoAcidCompoundSet.html#isComplementable()">isComplementable()</a></span> - Method in class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/AminoAcidCompoundSet.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompoundSet.html#isComplementable()">isComplementable()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava3.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractNucleotideCompoundSet.html#isComplementable()">isComplementable()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractNucleotideCompoundSet.html" title="class in org.biojava3.core.sequence.template">AbstractNucleotideCompoundSet</a></dt>
<dd>
<div class="block">NucleotideCompounds can always complement</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/CompoundSet.html#isComplementable()">isComplementable()</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html#isComplex()">isComplex()</a></span> - Method in class org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava3.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/Location.html#isComplex()">isComplex()</a></span> - Method in interface org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/Location.html" title="interface in org.biojava3.core.sequence.location.template">Location</a></dt>
<dd>
<div class="block">Returns true if the location is considered to be complex meaning
 the location is actually composed of sub-locations.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/CaseFreeAminoAcidCompoundSet.html#isCompoundStringLengthEqual()">isCompoundStringLengthEqual()</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/CaseFreeAminoAcidCompoundSet.html" title="class in org.biojava3.aaproperties.xml">CaseFreeAminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/ModifiedAminoAcidCompoundSet.html#isCompoundStringLengthEqual()">isCompoundStringLengthEqual()</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/ModifiedAminoAcidCompoundSet.html" title="class in org.biojava3.aaproperties.xml">ModifiedAminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/AminoAcidCompoundSet.html#isCompoundStringLengthEqual()">isCompoundStringLengthEqual()</a></span> - Method in class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/AminoAcidCompoundSet.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompoundSet.html#isCompoundStringLengthEqual()">isCompoundStringLengthEqual()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava3.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/CompoundSet.html#isCompoundStringLengthEqual()">isCompoundStringLengthEqual()</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a></dt>
<dd>
<div class="block">Returns true if all String representations of Compounds are of the
 same length.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#isConnected(org.biojava.bio.structure.AminoAcid, org.biojava.bio.structure.AminoAcid)">isConnected(AminoAcid, AminoAcid)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">test if two amino acids are connected, i.e.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ModificationCategory.html#isCrossLink()">isCrossLink()</a></span> - Method in enum org.biojava3.protmod.<a href="./org/biojava3/protmod/ModificationCategory.html" title="enum in org.biojava3.protmod">ModificationCategory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/Component.html#isCTerminal()">isCTerminal()</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/Component.html" title="class in org.biojava3.protmod">Component</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructurePairAligner.html#isDebug()">isDebug()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructurePairAligner.html" title="class in org.biojava.bio.structure.align">StructurePairAligner</a></dt>
<dd>
<div class="block">check if debug mode is set on</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/SeqRes2AtomAligner.html#isDEBUG()">isDEBUG()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/SeqRes2AtomAligner.html" title="class in org.biojava.bio.structure.io">SeqRes2AtomAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ScoreMatrix.html#isDNA()">isDNA()</a></span> - Method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ScoreMatrix.html" title="class in org.biojava3.phylo">ScoreMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#isDoAngleCheck()">isDoAngleCheck()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#isDoDensityCheck()">isDoDensityCheck()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#isDoDistanceCheck()">isDoDistanceCheck()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#isDoRMSCheck()">isDoRMSCheck()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/DownloadChemCompProvider.html#isDownloadAll()">isDownloadAll()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/DownloadChemCompProvider.html" title="class in org.biojava.bio.structure.io.mmcif">DownloadChemCompProvider</a></dt>
<dd>
<div class="block">By default this provider will download only some of the <a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model"><code>ChemComp</code></a> files.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html#isEmpty()">isEmpty()</a></span> - Method in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html" title="class in org.biojava.bio.structure.gui.util.color">GradientMapper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/AFPChainXMLParser.html#isErrorXML(java.lang.String)">isErrorXML(String)</a></span> - Static method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/AFPChainXMLParser.html" title="class in org.biojava.bio.structure.align.xml">AFPChainXMLParser</a></dt>
<dd>
<div class="block">returns true if the alignment XML contains an error message</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#isFetchCurrent()">isFetchCurrent()</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#isFetchCurrent()">isFetchCurrent()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#isFetchFileEvenIfObsolete()">isFetchFileEvenIfObsolete()</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>
<div class="block">forces the cache to fetch the file if its status is OBSOLETE.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#isFetchFileEvenIfObsolete()">isFetchFileEvenIfObsolete()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>
<div class="block">forces the reader to fetch the file if its status is OBSOLETE.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#isFlexible()">isFlexible()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/QRDecomposition.html#isFullRank()">isFullRank()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/QRDecomposition.html" title="class in org.biojava.bio.structure.jama">QRDecomposition</a></dt>
<dd>
<div class="block">Is the matrix full rank?</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleAlignedSequence.html#isGap(int)">isGap(int)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleAlignedSequence.html" title="class in org.biojava3.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AlignedSequence.html#isGap(int)">isGap(int)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AlignedSequence.html" title="interface in org.biojava3.alignment.template">AlignedSequence</a></dt>
<dd>
<div class="block">Returns true if this <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a> has a gap at a particular alignment column.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/Comparison.html#isGap(char)">isGap(char)</a></span> - Static method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/Comparison.html" title="class in org.biojava3.phylo">Comparison</a></dt>
<dd>
<div class="block">DOCUMENT ME!</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/GuiWrapper.html#isGuiModuleInstalled()">isGuiModuleInstalled()</a></span> - Static method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/GuiWrapper.html" title="class in org.biojava.bio.structure.align.ce">GuiWrapper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#isHalogen()">isHalogen()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns <CODE>true</CODE> if Element is a halogen (F, Cl, Br, I, At).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileParsingParameters.html#isHeaderOnly()">isHeaderOnly()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileParsingParameters.html" title="class in org.biojava.bio.structure.io">FileParsingParameters</a></dt>
<dd>
<div class="block">Parse only the PDB file header out of the files</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#isHeavyAtom()">isHeavyAtom()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns <CODE>true</CODE> is the Element is an not Hydrogen.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#isHeteroAtom()">isHeteroAtom()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns <CODE>true</CODE> if Element is not Hydrogen and not Carbon.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/SystemInfo.html#isHexProperty(java.lang.String)">isHexProperty(String)</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/SystemInfo.html" title="class in org.biojava.bio.structure.align.gui">SystemInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/SimplePoint.html#isHigher(org.biojava3.core.sequence.location.template.Point)">isHigher(Point)</a></span> - Method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/SimplePoint.html" title="class in org.biojava3.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/Point.html#isHigher(org.biojava3.core.sequence.location.template.Point)">isHigher(Point)</a></span> - Method in interface org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/Point.html" title="interface in org.biojava3.core.sequence.location.template">Point</a></dt>
<dd>
<div class="block">Returns true if the point is higher in value to the current point</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#isHydrogen()">isHydrogen()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns true if this Element is Hydrogen.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html#isInitialized()">isInitialized()</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava3.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/AlignmentCalcDB.html#isInterrupted()">isInterrupted()</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/AlignmentCalcDB.html" title="class in org.biojava.bio.structure.align.gui">AlignmentCalcDB</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#isJoinFast()">isJoinFast()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#isJoinPlo()">isJoinPlo()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/GuideTree.Node.html#isLeaf()">isLeaf()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/GuideTree.Node.html" title="class in org.biojava3.alignment">GuideTree.Node</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileParsingParameters.html#isLoadChemCompInfo()">isLoadChemCompInfo()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileParsingParameters.html" title="class in org.biojava.bio.structure.io">FileParsingParameters</a></dt>
<dd>
<div class="block">Should the chemical component information be automatically be downloade from the web?</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#isLocal()">isLocal()</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava3.alignment.template">AbstractMatrixAligner</a></dt>
<dd>
<div class="block">Returns whether alignment finds a region of similarity rather than aligning every compound.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/SimplePoint.html#isLower(org.biojava3.core.sequence.location.template.Point)">isLower(Point)</a></span> - Method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/SimplePoint.html" title="class in org.biojava3.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/Point.html#isLower(org.biojava3.core.sequence.location.template.Point)">isLower(Point)</a></span> - Method in interface org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/Point.html" title="interface in org.biojava3.core.sequence.location.template">Point</a></dt>
<dd>
<div class="block">Returns true if the current point is at a lower position than the
 point given.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#isMetal()">isMetal()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns <CODE>true</CODE> if ElementType is a metal.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ElementType.html#isMetal()">isMetal()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ElementType.html" title="enum in org.biojava.bio.structure">ElementType</a></dt>
<dd>
<div class="block">Returns <CODE>true</CODE> if ElementType is a metal.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#isMetalloid()">isMetalloid()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns <CODE>true</CODE> if ElementType is a metalloid.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ElementType.html#isMetalloid()">isMetalloid()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ElementType.html" title="enum in org.biojava.bio.structure">ElementType</a></dt>
<dd>
<div class="block">Returns <CODE>true</CODE> if ElementType is a metalloid.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Location.html#isNegative()">isNegative()</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Location.html" title="class in org.biojava3.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Check if location is on negative strand.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#isNmr()">isNmr()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">test if this structure is an nmr structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#isNmr()">isNmr()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">is this structure an nmr structure ?</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/data/sequence/SequenceUtil.html#isNonAmbNucleotideSequence(java.lang.String)">isNonAmbNucleotideSequence(String)</a></span> - Static method in class org.biojava3.data.sequence.<a href="./org/biojava3/data/sequence/SequenceUtil.html" title="class in org.biojava3.data.sequence">SequenceUtil</a></dt>
<dd>
<div class="block">Ambiguous DNA chars : AGTCRYMKSWHBVDN // differs from protein in only one
 (!)</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#isNonMetal()">isNonMetal()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns <CODE>true</CODE> if ElementType is a non-metal.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ElementType.html#isNonMetal()">isNonMetal()</a></span> - Method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ElementType.html" title="enum in org.biojava.bio.structure">ElementType</a></dt>
<dd>
<div class="block">Returns <CODE>true</CODE> if ElementType is a non-metal.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/LUDecomposition.html#isNonsingular()">isNonsingular()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/LUDecomposition.html" title="class in org.biojava.bio.structure.jama">LUDecomposition</a></dt>
<dd>
<div class="block">Is the matrix nonsingular?</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/Component.html#isNTerminal()">isNTerminal()</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/Component.html" title="class in org.biojava3.protmod">Component</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#isNucleotide(java.lang.String)">isNucleotide(String)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/SequenceTools.html#isNucleotideSequence(java.lang.String)">isNucleotideSequence(String)</a></span> - Static method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/SequenceTools.html" title="class in org.biojava3.core.util">SequenceTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/data/sequence/SequenceUtil.html#isNucleotideSequence(org.biojava3.data.sequence.FastaSequence)">isNucleotideSequence(FastaSequence)</a></span> - Static method in class org.biojava3.data.sequence.<a href="./org/biojava3/data/sequence/SequenceUtil.html" title="class in org.biojava3.data.sequence">SequenceUtil</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/aaproperties/xml/Isotope.html" title="class in org.biojava3.aaproperties.xml"><span class="strong">Isotope</span></a> - Class in <a href="./org/biojava3/aaproperties/xml/package-summary.html">org.biojava3.aaproperties.xml</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/Isotope.html#Isotope()">Isotope()</a></span> - Constructor for class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/Isotope.html" title="class in org.biojava3.aaproperties.xml">Isotope</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/Isotope.html#Isotope(java.lang.String, int, double)">Isotope(String, int, double)</a></span> - Constructor for class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/Isotope.html" title="class in org.biojava3.aaproperties.xml">Isotope</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#isOutputPDB()">isOutputPDB()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>
<div class="block">When writing the results to a file, don;t write as XML but write aligned PDB file</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileParsingParameters.html#isParseCAOnly()">isParseCAOnly()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileParsingParameters.html" title="class in org.biojava.bio.structure.io">FileParsingParameters</a></dt>
<dd>
<div class="block">the flag if only the C-alpha atoms of the structure should be parsed.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileParsingParameters.html#isParseSecStruc()">isParseSecStruc()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileParsingParameters.html" title="class in org.biojava.bio.structure.io">FileParsingParameters</a></dt>
<dd>
<div class="block">is secondary structure assignment being parsed from the file?</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/MMCIFFileReader.html#isPdbDirectorySplit()">isPdbDirectorySplit()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/MMCIFFileReader.html" title="class in org.biojava.bio.structure.io">MMCIFFileReader</a></dt>
<dd>
<div class="block">Flag that defines if the PDB directory is containing all PDB files or is split into sub dirs (like the FTP site).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#isPdbDirectorySplit()">isPdbDirectorySplit()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>
<div class="block">Flag that defines if the PDB directory is containing all PDB files or is split into sub dirs (like the FTP site).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/StructureIOFile.html#isPdbDirectorySplit()">isPdbDirectorySplit()</a></span> - Method in interface org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/StructureIOFile.html" title="interface in org.biojava.bio.structure.io">StructureIOFile</a></dt>
<dd>
<div class="block">The PDB files are organized hierarchically (as on the PDB - FTP server.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#isPdbDirSplit()">isPdbDirSplit()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#isPdbDirSplit()">isPdbDirSplit()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#isPrintCE()">isPrintCE()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>
<div class="block">Display the output string in CE style</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#isPrintFatCat()">isPrintFatCat()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#isPrintXML()">isPrintXML()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ScoreMatrix.html#isProtein()">isProtein()</a></span> - Method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ScoreMatrix.html" title="class in org.biojava3.phylo">ScoreMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/data/sequence/SequenceUtil.html#isProteinSequence(java.lang.String)">isProteinSequence(String)</a></span> - Static method in class org.biojava3.data.sequence.<a href="./org/biojava3/data/sequence/SequenceUtil.html" title="class in org.biojava3.data.sequence">SequenceUtil</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#isPublished()">isPublished()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#isReady()">isReady()</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava3.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html#isReady()">isReady()</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html" title="class in org.biojava3.alignment.template">AbstractPairwiseSequenceAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractProfileProfileAligner.html#isReady()">isReady()</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractProfileProfileAligner.html" title="class in org.biojava3.alignment.template">AbstractProfileProfileAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastService.html#isReady(java.lang.String, long)">isReady(String, long)</a></span> - Method in class org.biojava3.ws.alignment.qblast.<a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastService.html" title="class in org.biojava3.ws.alignment.qblast">NCBIQBlastService</a></dt>
<dd>
<div class="block">
 This method is used only for the executeBlastSearch method to check for
 completion of request using the NCBI specified RTOE variable
 </div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/RemotePairwiseAlignmentService.html#isReady(java.lang.String, long)">isReady(String, long)</a></span> - Method in interface org.biojava3.ws.alignment.<a href="./org/biojava3/ws/alignment/RemotePairwiseAlignmentService.html" title="interface in org.biojava3.ws.alignment">RemotePairwiseAlignmentService</a></dt>
<dd>
<div class="block">Simple method to check if the specified request has been completed by the service used.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html#isReplaced(org.biojava.bio.structure.io.mmcif.model.ChemComp)">isReplaced(ChemComp)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html" title="class in org.biojava.bio.structure.io.mmcif">ChemicalComponentDictionary</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html#isReplaced(java.lang.String)">isReplaced(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html" title="class in org.biojava.bio.structure.io.mmcif">ChemicalComponentDictionary</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html#isReplacer(org.biojava.bio.structure.io.mmcif.model.ChemComp)">isReplacer(ChemComp)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html" title="class in org.biojava.bio.structure.io.mmcif">ChemicalComponentDictionary</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html#isReplacer(java.lang.String)">isReplacer(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemicalComponentDictionary.html" title="class in org.biojava.bio.structure.io.mmcif">ChemicalComponentDictionary</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#isRunBackground()">isRunBackground()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>
<div class="block">Flag if a job that only runs one parallell job should be run in its own thread or in the main thread.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Location.html#isSameStrand(org.biojava3.genome.parsers.gff.Location)">isSameStrand(Location)</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Location.html" title="class in org.biojava3.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Check if this location is on same strand as other location.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/StructureName.html#isScopName()">isScopName()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/StructureName.html" title="class in org.biojava.bio.structure.align.client">StructureName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#isSequentialAlignment()">isSequentialAlignment()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">Get whether this alignment has the normal topology, ie the residues
 aligned in each block increase sequentially over the original protein.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#isShortAlign()">isShortAlign()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#isShow3d()">isShow3d()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#isShowAFPRanges()">isShowAFPRanges()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#isShowAFPRanges()">isShowAFPRanges()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#isShowMenu()">isShowMenu()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#isSignificantResult()">isSignificantResult()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/CholeskyDecomposition.html#isSPD()">isSPD()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/CholeskyDecomposition.html" title="class in org.biojava.bio.structure.jama">CholeskyDecomposition</a></dt>
<dd>
<div class="block">Is the matrix symmetric and positive definite?</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#isSplit()">isSplit()</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>
<div class="block">Is the organization of files within the directory split, as on the PDB FTP servers,
 or are all files contained in one directory.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/UserConfiguration.html#isSplit()">isSplit()</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/UserConfiguration.html" title="class in org.biojava.bio.structure.align.util">UserConfiguration</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#isStandard()">isStandard()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/chem/ChemCompTools.html#isStandardChemComp(org.biojava.bio.structure.io.mmcif.model.ChemComp)">isStandardChemComp(ChemComp)</a></span> - Static method in class org.biojava.bio.structure.io.mmcif.chem.<a href="./org/biojava/bio/structure/io/mmcif/chem/ChemCompTools.html" title="class in org.biojava.bio.structure.io.mmcif.chem">ChemCompTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/CodonCompound.html#isStart()">isStart()</a></span> - Method in class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/CodonCompound.html" title="class in org.biojava3.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/IUPACParser.IUPACTable.html#isStart(org.biojava3.core.sequence.compound.AminoAcidCompound)">isStart(AminoAcidCompound)</a></span> - Method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/IUPACParser.IUPACTable.html" title="class in org.biojava3.core.sequence.io">IUPACParser.IUPACTable</a></dt>
<dd>
<div class="block">Returns true if the given compound was a start codon in this
 codon table.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/Table.Codon.html#isStart()">isStart()</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/Table.Codon.html" title="class in org.biojava3.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/Table.html#isStart(org.biojava3.core.sequence.compound.AminoAcidCompound)">isStart(AminoAcidCompound)</a></span> - Method in interface org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/Table.html" title="interface in org.biojava3.core.sequence.transcription">Table</a></dt>
<dd>
<div class="block">Returns true if the given compound could have been a start amino acid;
 this does not assert if the codon that actually coded for the amino
 acid was a start codon.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/Table.Codon.html#isStop()">isStop()</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/Table.Codon.html" title="class in org.biojava3.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#isStoringScoreMatrix()">isStoringScoreMatrix()</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava3.alignment.template">AbstractMatrixAligner</a></dt>
<dd>
<div class="block">Returns choice to cache the score matrix or to save memory by deleting score matrix after alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#isStrictSCOP()">isStrictSCOP()</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>
<div class="block">Reports whether strict scop naming will be enforced, or whether this AtomCache
 should try to guess some simple variants on scop domains.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/SimplePoint.html#isUncertain()">isUncertain()</a></span> - Method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/SimplePoint.html" title="class in org.biojava3.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/Point.html#isUncertain()">isUncertain()</a></span> - Method in interface org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/Point.html" title="interface in org.biojava3.core.sequence.location.template">Point</a></dt>
<dd>
<div class="block">Returns a true if the exact point is unknown.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/SimplePoint.html#isUnknown()">isUnknown()</a></span> - Method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/SimplePoint.html" title="class in org.biojava3.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/Point.html#isUnknown()">isUnknown()</a></span> - Method in interface org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/Point.html" title="interface in org.biojava3.core.sequence.location.template">Point</a></dt>
<dd>
<div class="block">Returns true if the current position is unknown but is
 beyond the position encoded for.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileParsingParameters.html#isUpdateRemediatedFiles()">isUpdateRemediatedFiles()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileParsingParameters.html" title="class in org.biojava.bio.structure.io">FileParsingParameters</a></dt>
<dd>
<div class="block">A flag if local files should be replaced with the latest version of remediated PDB files.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/SystemInfo.html#isURLProperty(java.lang.String)">isURLProperty(String)</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/SystemInfo.html" title="class in org.biojava.bio.structure.align.gui">SystemInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#isVerbose()">isVerbose()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>
<div class="block">Flag if the job should be run in verbose mode.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html#isWaitForAlignments()">isWaitForAlignments()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html" title="class in org.biojava.bio.structure.align.client">FarmJobRunnable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#iterator()">iterator()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">get an Atom Iterator.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#iterator()">iterator()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">return an AtomIterator.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/GuideTree.html#iterator()">iterator()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/GuideTree.html" title="class in org.biojava3.alignment">GuideTree</a></dt>
<dd>
<div class="block">Returns a post-order <a href="http://java.sun.com/j2se/1.6.0/docs/api/java/util/Iterator.html?is-external=true" title="class or interface in java.util"><code>Iterator</code></a> that traverses the tree from leaves to root.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleAlignedSequence.html#iterator()">iterator()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleAlignedSequence.html" title="class in org.biojava3.alignment">SimpleAlignedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleProfile.html#iterator()">iterator()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleProfile.html" title="class in org.biojava3.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html#iterator()">iterator()</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava3.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/StringProxySequenceReader.html#iterator()">iterator()</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html#iterator()">iterator()</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html#iterator()">iterator()</a></span> - Method in class org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava3.core.sequence.location.template">AbstractLocation</a></dt>
<dd>
<div class="block">Iterates through all known sub-locations for this location but does
 not descend</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/ArrayListSequenceReader.html#iterator()">iterator()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava3.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/BitSequenceReader.html#iterator()">iterator()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava3.core.sequence.storage">BitSequenceReader</a></dt>
<dd>
<div class="block">Provides basic iterable access to this class</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/JoiningSequenceReader.html#iterator()">iterator()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava3.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>
<div class="block">Iterator implementation which attempts to move through the 2D structure
 attempting to skip onto the next sequence as & when it is asked to</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html#iterator()">iterator()</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava3.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Returns an instance of <code>SequenceMixin.SequenceIterator</code></div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#iterator()">iterator()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceProxyView.html#iterator()">iterator()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceProxyView.html" title="class in org.biojava3.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/views/WindowedSequence.html#iterator()">iterator()</a></span> - Method in class org.biojava3.core.sequence.views.<a href="./org/biojava3/core/sequence/views/WindowedSequence.html" title="class in org.biojava3.core.sequence.views">WindowedSequence</a></dt>
<dd>
<div class="block">Returns an iterator which will return the windows in a sequence in
 sequential order.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Location.html#iterator()">iterator()</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Location.html" title="class in org.biojava3.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Create a location iterator over this location with a window size of 1 and
 an increment of +1 (successive symbols from start to end).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Location.html#iterator(int, int)">iterator(int, int)</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Location.html" title="class in org.biojava3.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Create a location iterator over this location, 
 using specified window size and increment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/IUPACParser.html#IUPAC_LOCATION">IUPAC_LOCATION</a></span> - Static variable in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/IUPACParser.html" title="class in org.biojava3.core.sequence.io">IUPACParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/io/IUPACParser.html" title="class in org.biojava3.core.sequence.io"><span class="strong">IUPACParser</span></a> - Class in <a href="./org/biojava3/core/sequence/io/package-summary.html">org.biojava3.core.sequence.io</a></dt>
<dd>
<div class="block">Available translations

 
 1 - UNIVERSAL
 2 - VERTEBRATE_MITOCHONDRIAL
 3 - YEAST_MITOCHONDRIAL
 4 - MOLD_MITOCHONDRIAL
 5 - INVERTEBRATE_MITOCHONDRIAL
 6 - CILIATE_NUCLEAR
 9 - ECHINODERM_MITOCHONDRIAL
 10 - EUPLOTID_NUCLEAR
 11 - BACTERIAL
 12 - ALTERNATIVE_YEAST_NUCLEAR
 13 - ASCIDIAN_MITOCHONDRIAL
 14 - FLATWORM_MITOCHONDRIAL
 15 - BLEPHARISMA_MACRONUCLEAR
 16 - 2CHLOROPHYCEAN_MITOCHONDRIAL
 21 - TREMATODE_MITOCHONDRIAL
 23 - SCENEDESMUS_MITOCHONDRIAL
 

 Taken from <a
 href="http://www.ncbi.nlm.nih.gov/Taxonomy/Utils/wprintgc.cgi?</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/IUPACParser.html#IUPACParser()">IUPACParser()</a></span> - Constructor for class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/IUPACParser.html" title="class in org.biojava3.core.sequence.io">IUPACParser</a></dt>
<dd>
<div class="block">Default version and uses the classpath based IUPAC table</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/IUPACParser.html#IUPACParser(java.io.InputStream)">IUPACParser(InputStream)</a></span> - Constructor for class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/IUPACParser.html" title="class in org.biojava3.core.sequence.io">IUPACParser</a></dt>
<dd>
<div class="block">Allows you to specify a different IUPAC table.</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/io/IUPACParser.IUPACTable.html" title="class in org.biojava3.core.sequence.io"><span class="strong">IUPACParser.IUPACTable</span></a> - Class in <a href="./org/biojava3/core/sequence/io/package-summary.html">org.biojava3.core.sequence.io</a></dt>
<dd>
<div class="block">Holds the concept of a codon table from the IUPAC format</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/IUPACParser.IUPACTable.html#IUPACParser.IUPACTable(java.lang.String, int, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)">IUPACParser.IUPACTable(String, int, String, String, String, String, String)</a></span> - Constructor for class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/IUPACParser.IUPACTable.html" title="class in org.biojava3.core.sequence.io">IUPACParser.IUPACTable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/IUPACParser.IUPACTable.html#IUPACParser.IUPACTable(java.lang.String, java.lang.Integer, java.lang.String, java.lang.String)">IUPACParser.IUPACTable(String, Integer, String, String)</a></span> - Constructor for class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/IUPACParser.IUPACTable.html" title="class in org.biojava3.core.sequence.io">IUPACParser.IUPACTable</a></dt>
<dd>
<div class="block">Constructor which uses the basic IUPAC codon table format.</div>
</dd>
</dl>
<a name="_J_">
<!--   -->
</a>
<h2 class="title">J</h2>
<dl>
<dt><a href="./org/biojava/bio/structure/align/client/JFatCatClient.html" title="class in org.biojava.bio.structure.align.client"><span class="strong">JFatCatClient</span></a> - Class in <a href="./org/biojava/bio/structure/align/client/package-summary.html">org.biojava.bio.structure.align.client</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/JFatCatClient.html#JFatCatClient()">JFatCatClient()</a></span> - Constructor for class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/JFatCatClient.html" title="class in org.biojava.bio.structure.align.client">JFatCatClient</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/gui/JMatrixPanel.html" title="class in org.biojava.bio.structure.gui"><span class="strong">JMatrixPanel</span></a> - Class in <a href="./org/biojava/bio/structure/gui/package-summary.html">org.biojava.bio.structure.gui</a></dt>
<dd>
<div class="block">a JPanel that can display a difference of distance matrix and paths that have been
 taken for the alignment</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/JMatrixPanel.html#JMatrixPanel()">JMatrixPanel()</a></span> - Constructor for class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/JMatrixPanel.html" title="class in org.biojava.bio.structure.gui">JMatrixPanel</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/gui/events/JmolAlignedPositionListener.html" title="class in org.biojava.bio.structure.gui.events"><span class="strong">JmolAlignedPositionListener</span></a> - Class in <a href="./org/biojava/bio/structure/gui/events/package-summary.html">org.biojava.bio.structure.gui.events</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/events/JmolAlignedPositionListener.html#JmolAlignedPositionListener(org.biojava.bio.structure.gui.BiojavaJmol, org.biojava.bio.structure.align.StructurePairAligner)">JmolAlignedPositionListener(BiojavaJmol, StructurePairAligner)</a></span> - Constructor for class org.biojava.bio.structure.gui.events.<a href="./org/biojava/bio/structure/gui/events/JmolAlignedPositionListener.html" title="class in org.biojava.bio.structure.gui.events">JmolAlignedPositionListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/JmolPanel.html#jmolColorByChain()">jmolColorByChain()</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/JmolPanel.html" title="class in org.biojava.bio.structure.align.gui.jmol">JmolPanel</a></dt>
<dd>
<div class="block">assign a custom color to the Jmol chains command.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/BiojavaJmol.html#jmolInClassPath()">jmolInClassPath()</a></span> - Static method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/BiojavaJmol.html" title="class in org.biojava.bio.structure.gui">BiojavaJmol</a></dt>
<dd>
<div class="block">returns true if Jmol can be found in the classpath, otherwise false.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/gui/jmol/JmolPanel.html" title="class in org.biojava.bio.structure.align.gui.jmol"><span class="strong">JmolPanel</span></a> - Class in <a href="./org/biojava/bio/structure/align/gui/jmol/package-summary.html">org.biojava.bio.structure.align.gui.jmol</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/JmolPanel.html#JmolPanel()">JmolPanel()</a></span> - Constructor for class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/JmolPanel.html" title="class in org.biojava.bio.structure.align.gui.jmol">JmolPanel</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/gui/jmol/JmolTools.html" title="class in org.biojava.bio.structure.align.gui.jmol"><span class="strong">JmolTools</span></a> - Class in <a href="./org/biojava/bio/structure/align/gui/jmol/package-summary.html">org.biojava.bio.structure.align.gui.jmol</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/JmolTools.html#JmolTools()">JmolTools()</a></span> - Constructor for class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/JmolTools.html" title="class in org.biojava.bio.structure.align.gui.jmol">JmolTools</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/structure/gui/JmolViewerImpl.html" title="class in org.biojava3.structure.gui"><span class="strong">JmolViewerImpl</span></a> - Class in <a href="./org/biojava3/structure/gui/package-summary.html">org.biojava3.structure.gui</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/structure/gui/JmolViewerImpl.html#JmolViewerImpl()">JmolViewerImpl()</a></span> - Constructor for class org.biojava3.structure.gui.<a href="./org/biojava3/structure/gui/JmolViewerImpl.html" title="class in org.biojava3.structure.gui">JmolViewerImpl</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/webstart/JNLPProxy.html" title="class in org.biojava.bio.structure.align.webstart"><span class="strong">JNLPProxy</span></a> - Class in <a href="./org/biojava/bio/structure/align/webstart/package-summary.html">org.biojava.bio.structure.align.webstart</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/client/JobKillException.html" title="class in org.biojava.bio.structure.align.client"><span class="strong">JobKillException</span></a> - Exception in <a href="./org/biojava/bio/structure/align/client/package-summary.html">org.biojava.bio.structure.align.client</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/JobKillException.html#JobKillException(java.lang.String)">JobKillException(String)</a></span> - Constructor for exception org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/JobKillException.html" title="class in org.biojava.bio.structure.align.client">JobKillException</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava3.core.sequence.storage"><span class="strong">JoiningSequenceReader</span></a>&lt;<a href="./org/biojava3/core/sequence/storage/JoiningSequenceReader.html" title="type parameter in JoiningSequenceReader">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/core/sequence/storage/package-summary.html">org.biojava3.core.sequence.storage</a></dt>
<dd>
<div class="block">This reader actually proxies onto multiple types of sequence in order
 to allow a number of sequence objects to act as if they are one sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/JoiningSequenceReader.html#JoiningSequenceReader(org.biojava3.core.sequence.template.Sequence...)">JoiningSequenceReader(Sequence&lt;C&gt;...)</a></span> - Constructor for class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava3.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>
<div class="block">Allows creation of the store from Vargs Sequence<C> objects.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/JoiningSequenceReader.html#JoiningSequenceReader(java.util.List)">JoiningSequenceReader(List&lt;Sequence&lt;C&gt;&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava3.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>
<div class="block">Allows creation of the store from List<Sequence<C>>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/JoiningSequenceReader.html#JoiningSequenceReader(org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.template.Sequence...)">JoiningSequenceReader(CompoundSet&lt;C&gt;, Sequence&lt;C&gt;...)</a></span> - Constructor for class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava3.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/JoiningSequenceReader.html#JoiningSequenceReader(org.biojava3.core.sequence.template.CompoundSet, java.util.List)">JoiningSequenceReader(CompoundSet&lt;C&gt;, List&lt;Sequence&lt;C&gt;&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava3.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/helper/JointFragments.html" title="class in org.biojava.bio.structure.align.helper"><span class="strong">JointFragments</span></a> - Class in <a href="./org/biojava/bio/structure/align/helper/package-summary.html">org.biojava.bio.structure.align.helper</a></dt>
<dd>
<div class="block">A utility class that defines which set of atoms are considered
 to be on equivalent positions.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/JointFragments.html#JointFragments()">JointFragments()</a></span> - Constructor for class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/JointFragments.html" title="class in org.biojava.bio.structure.align.helper">JointFragments</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentProgressListener.html#jointFragments(org.biojava.bio.structure.align.helper.JointFragments[])">jointFragments(JointFragments[])</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentProgressListener.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure"><span class="strong">JournalArticle</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">PDB-specific</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#JournalArticle()">JournalArticle()</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/gui/JPrintPanel.html" title="class in org.biojava.bio.structure.align.gui"><span class="strong">JPrintPanel</span></a> - Class in <a href="./org/biojava/bio/structure/align/gui/package-summary.html">org.biojava.bio.structure.align.gui</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/JPrintPanel.html#JPrintPanel()">JPrintPanel()</a></span> - Constructor for class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/JPrintPanel.html" title="class in org.biojava.bio.structure.align.gui">JPrintPanel</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/ronn/Jronn.html" title="class in org.biojava3.ronn"><span class="strong">Jronn</span></a> - Class in <a href="./org/biojava3/ronn/package-summary.html">org.biojava3.ronn</a></dt>
<dd>
<div class="block">This class gives public API to RONN functions.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ronn/Jronn.html#Jronn()">Jronn()</a></span> - Constructor for class org.biojava3.ronn.<a href="./org/biojava3/ronn/Jronn.html" title="class in org.biojava3.ronn">Jronn</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/ronn/Jronn.Range.html" title="class in org.biojava3.ronn"><span class="strong">Jronn.Range</span></a> - Class in <a href="./org/biojava3/ronn/package-summary.html">org.biojava3.ronn</a></dt>
<dd>
<div class="block">Holder for the ranges, contain pointers to starting and ending position 
 on the sequence which comprises a disordered region.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ronn/Jronn.Range.html#Jronn.Range(int, int)">Jronn.Range(int, int)</a></span> - Constructor for class org.biojava3.ronn.<a href="./org/biojava3/ronn/Jronn.Range.html" title="class in org.biojava3.ronn">Jronn.Range</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_K_">
<!--   -->
</a>
<h2 class="title">K</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/Constraints.html#K">K</a></span> - Static variable in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/Constraints.html" title="class in org.biojava3.aaproperties">Constraints</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/RasmolCommandListener.html#keyReleased(java.awt.event.KeyEvent)">keyReleased(KeyEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/RasmolCommandListener.html" title="class in org.biojava.bio.structure.align.gui.jmol">RasmolCommandListener</a></dt>
<dd>
<div class="block">takes care of the cursor up/down keys. triggers copying of stored 
 commands into the current textfield</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/RasmolCommandListener.html#keyReleased(java.awt.event.KeyEvent)">keyReleased(KeyEvent)</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/RasmolCommandListener.html" title="class in org.biojava.bio.structure.gui">RasmolCommandListener</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
<div class="block">takes care of the cursor up/down keys. triggers copying of stored 
 commands into the current textfield</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html#keySet()">keySet()</a></span> - Method in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html" title="class in org.biojava.bio.structure.gui.util.color">GradientMapper</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_L_">
<!--   -->
</a>
<h2 class="title">L</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/Constraints.html#L">L</a></span> - Static variable in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/Constraints.html" title="class in org.biojava3.aaproperties">Constraints</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ModificationCategory.html#label()">label()</a></span> - Method in enum org.biojava3.protmod.<a href="./org/biojava3/protmod/ModificationCategory.html" title="enum in org.biojava3.protmod">ModificationCategory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ModificationOccurrenceType.html#label()">label()</a></span> - Method in enum org.biojava3.protmod.<a href="./org/biojava3/protmod/ModificationOccurrenceType.html" title="enum in org.biojava3.protmod">ModificationOccurrenceType</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceMixin.html#lastIndexOf(org.biojava3.core.sequence.template.Sequence, C)">lastIndexOf(Sequence&lt;C&gt;, C)</a></span> - Static method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceMixin.html" title="class in org.biojava3.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">Performs a reversed linear search of the given Sequence by wrapping
 it in a <a href="./org/biojava3/core/sequence/views/ReversedSequenceView.html" title="class in org.biojava3.core.sequence.views"><code>ReversedSequenceView</code></a> and passing it into
 <a href="./org/biojava3/core/sequence/template/SequenceMixin.html#indexOf(org.biojava3.core.sequence.template.Sequence, C)"><code>SequenceMixin.indexOf(Sequence, Compound)</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/AbstractFeature.html#LENGTH">LENGTH</a></span> - Static variable in class org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/AbstractFeature.html" title="class in org.biojava3.core.sequence.features">AbstractFeature</a></dt>
<dd>
<div class="block">Sort features by length.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Location.html#length()">length()</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Location.html" title="class in org.biojava3.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Get length of range.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#Leu">Leu</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/ColorUtils.html#lighter(java.awt.Color, double)">lighter(Color, double)</a></span> - Static method in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/ColorUtils.html" title="class in org.biojava.bio.structure.gui.util.color">ColorUtils</a></dt>
<dd>
<div class="block">Make a color lighter.</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/template/LightweightProfile.html" title="interface in org.biojava3.core.sequence.template"><span class="strong">LightweightProfile</span></a>&lt;<a href="./org/biojava3/core/sequence/template/LightweightProfile.html" title="type parameter in LightweightProfile">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava3/core/sequence/template/LightweightProfile.html" title="type parameter in LightweightProfile">C</a>&gt;,<a href="./org/biojava3/core/sequence/template/LightweightProfile.html" title="type parameter in LightweightProfile">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava3/core/sequence/template/package-summary.html">org.biojava3.core.sequence.template</a></dt>
<dd>
<div class="block">Defines a minimal data structure for reading and writing a sequence alignment.</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/template/LightweightProfile.StringFormat.html" title="enum in org.biojava3.core.sequence.template"><span class="strong">LightweightProfile.StringFormat</span></a> - Enum in <a href="./org/biojava3/core/sequence/template/package-summary.html">org.biojava3.core.sequence.template</a></dt>
<dd>
<div class="block">List of output formats.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html#LINE_HEIGHT">LINE_HEIGHT</a></span> - Static variable in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html" title="class in org.biojava.bio.structure.gui.util">SequenceScalePanel</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/gui/util/color/LinearColorInterpolator.html" title="class in org.biojava.bio.structure.gui.util.color"><span class="strong">LinearColorInterpolator</span></a> - Class in <a href="./org/biojava/bio/structure/gui/util/color/package-summary.html">org.biojava.bio.structure.gui.util.color</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/LinearColorInterpolator.html#LinearColorInterpolator()">LinearColorInterpolator()</a></span> - Constructor for class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/LinearColorInterpolator.html" title="class in org.biojava.bio.structure.gui.util.color">LinearColorInterpolator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/LinearColorInterpolator.html#LinearColorInterpolator(java.awt.color.ColorSpace)">LinearColorInterpolator(ColorSpace)</a></span> - Constructor for class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/LinearColorInterpolator.html" title="class in org.biojava.bio.structure.gui.util.color">LinearColorInterpolator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/gui/util/color/LinearColorInterpolator.InterpolationDirection.html" title="enum in org.biojava.bio.structure.gui.util.color"><span class="strong">LinearColorInterpolator.InterpolationDirection</span></a> - Enum in <a href="./org/biojava/bio/structure/gui/util/color/package-summary.html">org.biojava.bio.structure.gui.util.color</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/LocalCacheStructureProvider.html#lineSplit">lineSplit</a></span> - Static variable in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/LocalCacheStructureProvider.html" title="class in org.biojava.bio.structure.io">LocalCacheStructureProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/MMCIFFileReader.html#lineSplit">lineSplit</a></span> - Static variable in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/MMCIFFileReader.html" title="class in org.biojava.bio.structure.io">MMCIFFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#lineSplit">lineSplit</a></span> - Static variable in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileParser.html#linkChains2Compound(org.biojava.bio.structure.Structure)">linkChains2Compound(Structure)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileParser.html" title="class in org.biojava.bio.structure.io">PDBFileParser</a></dt>
<dd>
<div class="block">After the parsing of a PDB file the <a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure"><code>Chain</code></a> and  <a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure"><code>Compound</code></a>
 objects need to be linked to each other.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/webstart/PersistentConfig.html#load()">load()</a></span> - Method in class org.biojava.bio.structure.align.webstart.<a href="./org/biojava/bio/structure/align/webstart/PersistentConfig.html" title="class in org.biojava.bio.structure.align.webstart">PersistentConfig</a></dt>
<dd>
<div class="block">loads Config from PersistenceService
  returns null if no PErsistenceService has been created ...</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/MenuCreator.html#LOAD_ALIGNMENT_XML">LOAD_ALIGNMENT_XML</a></span> - Static variable in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/MenuCreator.html" title="class in org.biojava.bio.structure.align.gui">MenuCreator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#LOAD_CHEM_COMP_PROPERTY">LOAD_CHEM_COMP_PROPERTY</a></span> - Static variable in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/MenuCreator.html#LOAD_DB_RESULTS">LOAD_DB_RESULTS</a></span> - Static variable in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/MenuCreator.html" title="class in org.biojava.bio.structure.align.gui">MenuCreator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/GeneFeatureHelper.html#loadFastaAddGeneFeaturesFromGeneIDGFF2(java.io.File, java.io.File)">loadFastaAddGeneFeaturesFromGeneIDGFF2(File, File)</a></span> - Static method in class org.biojava3.genome.<a href="./org/biojava3/genome/GeneFeatureHelper.html" title="class in org.biojava3.genome">GeneFeatureHelper</a></dt>
<dd>
<div class="block">Loads Fasta file and GFF2 feature file generated from the geneid prediction algorithm</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/GeneFeatureHelper.html#loadFastaAddGeneFeaturesFromGeneMarkGTF(java.io.File, java.io.File)">loadFastaAddGeneFeaturesFromGeneMarkGTF(File, File)</a></span> - Static method in class org.biojava3.genome.<a href="./org/biojava3/genome/GeneFeatureHelper.html" title="class in org.biojava3.genome">GeneFeatureHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/GeneFeatureHelper.html#loadFastaAddGeneFeaturesFromGlimmerGFF3(java.io.File, java.io.File)">loadFastaAddGeneFeaturesFromGlimmerGFF3(File, File)</a></span> - Static method in class org.biojava3.genome.<a href="./org/biojava3/genome/GeneFeatureHelper.html" title="class in org.biojava3.genome">GeneFeatureHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/GeneFeatureHelper.html#loadFastaAddGeneFeaturesFromGmodGFF3(java.io.File, java.io.File, boolean)">loadFastaAddGeneFeaturesFromGmodGFF3(File, File, boolean)</a></span> - Static method in class org.biojava3.genome.<a href="./org/biojava3/genome/GeneFeatureHelper.html" title="class in org.biojava3.genome">GeneFeatureHelper</a></dt>
<dd>
<div class="block">Lots of variations in the ontology or descriptors that can be used in GFF3 which requires writing a custom parser to handle a GFF3 generated or used
 by a specific application.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/GeneFeatureHelper.html#loadFastaAddGeneFeaturesFromUpperCaseExonFastaFile(java.io.File, java.io.File, boolean)">loadFastaAddGeneFeaturesFromUpperCaseExonFastaFile(File, File, boolean)</a></span> - Static method in class org.biojava3.genome.<a href="./org/biojava3/genome/GeneFeatureHelper.html" title="class in org.biojava3.genome">GeneFeatureHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoLoadStructure.html#loadStructureFromCache()">loadStructureFromCache()</a></span> - Method in class demo.<a href="./demo/DemoLoadStructure.html" title="class in demo">DemoLoadStructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/XMLHelper.html#loadXML(java.lang.String)">loadXML(String)</a></span> - Static method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/XMLHelper.html" title="class in org.biojava3.core.util">XMLHelper</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/LocalCacheStructureProvider.html" title="class in org.biojava.bio.structure.io"><span class="strong">LocalCacheStructureProvider</span></a> - Class in <a href="./org/biojava/bio/structure/io/package-summary.html">org.biojava.bio.structure.io</a></dt>
<dd>
<div class="block">Provides structures based on locally cached PDB files.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/LocalCacheStructureProvider.html#LocalCacheStructureProvider()">LocalCacheStructureProvider()</a></span> - Constructor for class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/LocalCacheStructureProvider.html" title="class in org.biojava.bio.structure.io">LocalCacheStructureProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/LocationHelper.html#location(java.util.List)">location(List&lt;Location&gt;)</a></span> - Static method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/LocationHelper.html" title="class in org.biojava3.core.sequence.location">LocationHelper</a></dt>
<dd>
<div class="block">Used as a thin wrapper to the <a href="./org/biojava3/core/sequence/location/LocationHelper.html#location(java.util.List, java.lang.String)"><code>LocationHelper.location(java.util.List, java.lang.String)</code></a>
 method to bring the given location list together as a join (the default
 type)</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/LocationHelper.html#location(java.util.List, java.lang.String)">location(List&lt;Location&gt;, String)</a></span> - Static method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/LocationHelper.html" title="class in org.biojava3.core.sequence.location">LocationHelper</a></dt>
<dd>
<div class="block">Builds a location from a List of locations; this can be circular or
 linear joins.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/LocationHelper.html#location(int, int, org.biojava3.core.sequence.Strand, int)">location(int, int, Strand, int)</a></span> - Static method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/LocationHelper.html" title="class in org.biojava3.core.sequence.location">LocationHelper</a></dt>
<dd>
<div class="block">Returns a location object which unlike the location constructors
 allows you to input reverse coordinates and will convert
 these into the right location on the positive strand.</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/location/template/Location.html" title="interface in org.biojava3.core.sequence.location.template"><span class="strong">Location</span></a> - Interface in <a href="./org/biojava3/core/sequence/location/template/package-summary.html">org.biojava3.core.sequence.location.template</a></dt>
<dd>
<div class="block">Sets of integers used to represent the location of features on sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/Location.Tools.html#location(java.util.List, java.lang.Integer, java.lang.String)">location(List&lt;Location&gt;, Integer, String)</a></span> - Static method in class org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/Location.Tools.html" title="class in org.biojava3.core.sequence.location.template">Location.Tools</a></dt>
<dd>
<div class="block">Used for building a location from a series of sub-locations</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/Location.Tools.html#location(int, int, org.biojava3.core.sequence.Strand, int)">location(int, int, Strand, int)</a></span> - Static method in class org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/Location.Tools.html" title="class in org.biojava3.core.sequence.location.template">Location.Tools</a></dt>
<dd>
<div class="block">Returns a location object which unlike the location constructors
 allows you to input reverse coordinates and will convert
 these into the right location on the positive strand.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Feature.html#location()">location()</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Feature.html" title="class in org.biojava3.genome.parsers.gff">Feature</a></dt>
<dd>
<div class="block">Get location of feature.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/FeatureI.html#location()">location()</a></span> - Method in interface org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/FeatureI.html" title="interface in org.biojava3.genome.parsers.gff">FeatureI</a></dt>
<dd>
<div class="block">Get the location of the feature.</div>
</dd>
<dt><a href="./org/biojava3/genome/parsers/gff/Location.html" title="class in org.biojava3.genome.parsers.gff"><span class="strong">Location</span></a> - Class in <a href="./org/biojava3/genome/parsers/gff/package-summary.html">org.biojava3.genome.parsers.gff</a></dt>
<dd>
<div class="block">A location on a sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Location.html#Location(int, int)">Location(int, int)</a></span> - Constructor for class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Location.html" title="class in org.biojava3.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Construct new location from coordinates.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Location.html#Location(org.biojava3.genome.parsers.gff.Location)">Location(Location)</a></span> - Constructor for class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Location.html" title="class in org.biojava3.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Clone other location.</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/location/template/Location.Tools.html" title="class in org.biojava3.core.sequence.location.template"><span class="strong">Location.Tools</span></a> - Class in <a href="./org/biojava3/core/sequence/location/template/package-summary.html">org.biojava3.core.sequence.location.template</a></dt>
<dd>
<div class="block">Helper methods for use with the Location classes.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/Location.Tools.html#Location.Tools()">Location.Tools()</a></span> - Constructor for class org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/Location.Tools.html" title="class in org.biojava3.core.sequence.location.template">Location.Tools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/AbstractFeature.html#LOCATION_LENGTH">LOCATION_LENGTH</a></span> - Static variable in class org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/AbstractFeature.html" title="class in org.biojava3.core.sequence.features">AbstractFeature</a></dt>
<dd>
<div class="block">Sort features by start position and then longest length.</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/location/LocationHelper.html" title="class in org.biojava3.core.sequence.location"><span class="strong">LocationHelper</span></a> - Class in <a href="./org/biojava3/core/sequence/location/package-summary.html">org.biojava3.core.sequence.location</a></dt>
<dd>
<div class="block">Helper methods for use with the Location classes.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/LocationHelper.html#LocationHelper()">LocationHelper()</a></span> - Constructor for class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/LocationHelper.html" title="class in org.biojava3.core.sequence.location">LocationHelper</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/genome/parsers/gff/LocIterator.html" title="class in org.biojava3.genome.parsers.gff"><span class="strong">LocIterator</span></a> - Class in <a href="./org/biojava3/genome/parsers/gff/package-summary.html">org.biojava3.genome.parsers.gff</a></dt>
<dd>
<div class="block">Move a sliding window over a Location.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/LocIterator.html#LocIterator(org.biojava3.genome.parsers.gff.Location, int, int)">LocIterator(Location, int, int)</a></span> - Constructor for class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/LocIterator.html" title="class in org.biojava3.genome.parsers.gff">LocIterator</a></dt>
<dd>
<div class="block">Construct an iterator that slides a window over a Location.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html#log(java.lang.String)">log(String)</a></span> - Static method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html" title="class in org.biojava.bio.structure.align.client">FarmJobRunnable</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/gui/util/color/LogColorMapper.html" title="class in org.biojava.bio.structure.gui.util.color"><span class="strong">LogColorMapper</span></a> - Class in <a href="./org/biojava/bio/structure/gui/util/color/package-summary.html">org.biojava.bio.structure.gui.util.color</a></dt>
<dd>
<div class="block">Performs a log10 transform on input before passing the values off to another
 colorMapper.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/LogColorMapper.html#LogColorMapper(org.biojava.bio.structure.gui.util.color.ContinuousColorMapper)">LogColorMapper(ContinuousColorMapper)</a></span> - Constructor for class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/LogColorMapper.html" title="class in org.biojava.bio.structure.gui.util.color">LogColorMapper</a></dt>
<dd>
<div class="block">Creates a new LogColorMapper with base 10.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/LogColorMapper.html#LogColorMapper(org.biojava.bio.structure.gui.util.color.ContinuousColorMapper, int)">LogColorMapper(ContinuousColorMapper, int)</a></span> - Constructor for class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/LogColorMapper.html" title="class in org.biojava.bio.structure.gui.util.color">LogColorMapper</a></dt>
<dd>
<div class="block">If logspaceMapper maps values x1 to x2, this creates a
 mapper for values base^x1 to base^x2</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/AlignmentCalc.html#logger">logger</a></span> - Static variable in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/AlignmentCalc.html" title="class in org.biojava.bio.structure.align.gui">AlignmentCalc</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/AlignmentCalcDB.html#logger">logger</a></span> - Static variable in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/AlignmentCalcDB.html" title="class in org.biojava.bio.structure.align.gui">AlignmentCalcDB</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/AlignmentGui.html#logger">logger</a></span> - Static variable in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/AlignmentGui.html" title="class in org.biojava.bio.structure.align.gui">AlignmentGui</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#logger">logger</a></span> - Static variable in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentJoiner.html#logger">logger</a></span> - Static variable in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentJoiner.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentJoiner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/webstart/BrowserOpener.html#logger">logger</a></span> - Static variable in class org.biojava.bio.structure.align.webstart.<a href="./org/biojava/bio/structure/align/webstart/BrowserOpener.html" title="class in org.biojava.bio.structure.align.webstart">BrowserOpener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/AlignmentCalc.html#logger">logger</a></span> - Static variable in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/AlignmentCalc.html" title="class in org.biojava.bio.structure.gui.util">AlignmentCalc</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/AlternativeAlignmentFrame.html#logger">logger</a></span> - Static variable in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/AlternativeAlignmentFrame.html" title="class in org.biojava.bio.structure.gui.util">AlternativeAlignmentFrame</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/PDBDirPanel.html#logger">logger</a></span> - Static variable in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/PDBDirPanel.html" title="class in org.biojava.bio.structure.gui.util">PDBDirPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/PDBServerPanel.html#logger">logger</a></span> - Static variable in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/PDBServerPanel.html" title="class in org.biojava.bio.structure.gui.util">PDBServerPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/PDBUploadPanel.html#logger">logger</a></span> - Static variable in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/PDBUploadPanel.html" title="class in org.biojava.bio.structure.gui.util">PDBUploadPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/ScopSelectPanel.html#logger">logger</a></span> - Static variable in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/ScopSelectPanel.html" title="class in org.biojava.bio.structure.gui.util">ScopSelectPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#logger">logger</a></span> - Static variable in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#logger">logger</a></span> - Static variable in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/FlatFileInstallation.html#logger">logger</a></span> - Static variable in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/FlatFileInstallation.html" title="class in org.biojava.bio.structure.server">FlatFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html#logger">logger</a></span> - Static variable in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html" title="class in org.biojava.bio.structure.server">MMCIFFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/PrepareMMcifIndexFile.html#logger">logger</a></span> - Static variable in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/PrepareMMcifIndexFile.html" title="class in org.biojava.bio.structure.server">PrepareMMcifIndexFile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/SimpleStructureServer.html#logger">logger</a></span> - Static variable in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/SimpleStructureServer.html" title="class in org.biojava.bio.structure.server">SimpleStructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#logger">logger</a></span> - Static variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/PrepareIndexFile.html#logPDBInfoFile(java.io.PrintWriter, java.io.PrintWriter, org.biojava.bio.structure.io.StructureIOFile, java.io.File[])">logPDBInfoFile(PrintWriter, PrintWriter, StructureIOFile, File[])</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/PrepareIndexFile.html" title="class in org.biojava.bio.structure.server">PrepareIndexFile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/CountProgressListener.html#logStatus(java.lang.String)">logStatus(String)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/CountProgressListener.html" title="class in org.biojava.bio.structure.align.client">CountProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/events/AlignmentProgressListener.html#logStatus(java.lang.String)">logStatus(String)</a></span> - Method in interface org.biojava.bio.structure.align.events.<a href="./org/biojava/bio/structure/align/events/AlignmentProgressListener.html" title="interface in org.biojava.bio.structure.align.events">AlignmentProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/GUIAlignmentProgressListener.html#logStatus(java.lang.String)">logStatus(String)</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/GUIAlignmentProgressListener.html" title="class in org.biojava.bio.structure.align.gui">GUIAlignmentProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#LOOP_END">LOOP_END</a></span> - Static variable in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#LOOP_START">LOOP_START</a></span> - Static variable in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#lu()">lu()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">LU Decomposition</div>
</dd>
<dt><a href="./org/biojava/bio/structure/jama/LUDecomposition.html" title="class in org.biojava.bio.structure.jama"><span class="strong">LUDecomposition</span></a> - Class in <a href="./org/biojava/bio/structure/jama/package-summary.html">org.biojava.bio.structure.jama</a></dt>
<dd>
<div class="block">LU Decomposition.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/LUDecomposition.html#LUDecomposition(org.biojava.bio.structure.jama.Matrix)">LUDecomposition(Matrix)</a></span> - Constructor for class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/LUDecomposition.html" title="class in org.biojava.bio.structure.jama">LUDecomposition</a></dt>
<dd>
<div class="block">LU Decomposition provides a data structure  to access L, U and piv.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#Lys">Lys</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_M_">
<!--   -->
</a>
<h2 class="title">M</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/Constraints.html#M">M</a></span> - Static variable in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/Constraints.html" title="class in org.biojava3.aaproperties">Constraints</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/AlignmentGuiDemo.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class demo.<a href="./demo/AlignmentGuiDemo.html" title="class in demo">AlignmentGuiDemo</a></dt>
<dd>
<div class="block">e.g. try to align 2hyn vs 1zll</div>
</dd>
<dt><span class="strong"><a href="./demo/ChemCompDistribution.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class demo.<a href="./demo/ChemCompDistribution.html" title="class in demo">ChemCompDistribution</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/CookBook.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class demo.<a href="./demo/CookBook.html" title="class in demo">CookBook</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoAtomCache.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class demo.<a href="./demo/DemoAtomCache.html" title="class in demo">DemoAtomCache</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoCE.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class demo.<a href="./demo/DemoCE.html" title="class in demo">DemoCE</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoChangeChemCompProvider.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class demo.<a href="./demo/DemoChangeChemCompProvider.html" title="class in demo">DemoChangeChemCompProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoCommandLineStartup.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class demo.<a href="./demo/DemoCommandLineStartup.html" title="class in demo">DemoCommandLineStartup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoDomainsplit.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class demo.<a href="./demo/DemoDomainsplit.html" title="class in demo">DemoDomainsplit</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoFATCAT.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class demo.<a href="./demo/DemoFATCAT.html" title="class in demo">DemoFATCAT</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoLoadStructure.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class demo.<a href="./demo/DemoLoadStructure.html" title="class in demo">DemoLoadStructure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoMMCIFReader.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class demo.<a href="./demo/DemoMMCIFReader.html" title="class in demo">DemoMMCIFReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoSCOP.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class demo.<a href="./demo/DemoSCOP.html" title="class in demo">DemoSCOP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoSW3DAligner.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class demo.<a href="./demo/DemoSW3DAligner.html" title="class in demo">DemoSW3DAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/ShowStructureInJmol.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class demo.<a href="./demo/ShowStructureInJmol.html" title="class in demo">ShowStructureInJmol</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/WebStartClientDemo.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class demo.<a href="./demo/WebStartClientDemo.html" title="class in demo">WebStartClientDemo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCPMain.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCPMain.html" title="class in org.biojava.bio.structure.align.ce">CeCPMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeMain.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeMain.html" title="class in org.biojava.bio.structure.align.ce">CeMain</a></dt>
<dd>
<div class="block">Example Parameters:
 
 -pdbFilePath /tmp -autoFetch -printCE -pdb1 1cnv -pdb2 3cna</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeSideChainMain.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeSideChainMain.html" title="class in org.biojava.bio.structure.align.ce">CeSideChainMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/JFatCatClient.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/JFatCatClient.html" title="class in org.biojava.bio.structure.align.client">JFatCatClient</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/FarmJob.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/FarmJob.html" title="class in org.biojava.bio.structure.align">FarmJob</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCat.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCat.html" title="class in org.biojava.bio.structure.align.fatcat">FatCat</a></dt>
<dd>
<div class="block">See demo/FatCatDemo.java for an example how to run.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/AlignmentGui.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/AlignmentGui.html" title="class in org.biojava.bio.structure.align.gui">AlignmentGui</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AligPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/DBResultTable.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/DBResultTable.html" title="class in org.biojava.bio.structure.align.gui">DBResultTable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/DotPlotPanel.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/DotPlotPanel.html" title="class in org.biojava.bio.structure.align.gui">DotPlotPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/AtomInfoParser.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/AtomInfoParser.html" title="class in org.biojava.bio.structure.align.gui.jmol">AtomInfoParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/JmolTools.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/JmolTools.html" title="class in org.biojava.bio.structure.align.gui.jmol">JmolTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html" title="class in org.biojava.bio.structure.align.gui.jmol">StructureAlignmentJmol</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWaterman3Daligner.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWaterman3Daligner.html" title="class in org.biojava.bio.structure.align.seq">SmithWaterman3Daligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructurePairAligner.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructurePairAligner.html" title="class in org.biojava.bio.structure.align">StructurePairAligner</a></dt>
<dd>
<div class="block">example usage of this class</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/webstart/JNLPProxy.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.align.webstart.<a href="./org/biojava/bio/structure/align/webstart/JNLPProxy.html" title="class in org.biojava.bio.structure.align.webstart">JNLPProxy</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/webstart/WebStartDBSearch.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.align.webstart.<a href="./org/biojava/bio/structure/align/webstart/WebStartDBSearch.html" title="class in org.biojava.bio.structure.align.webstart">WebStartDBSearch</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/webstart/WebStartMain.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.align.webstart.<a href="./org/biojava/bio/structure/align/webstart/WebStartMain.html" title="class in org.biojava.bio.structure.align.webstart">WebStartMain</a></dt>
<dd>
<div class="block">If no arguments, shows AlignmentGui for pairwise alignments.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/BiojavaJmol.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/BiojavaJmol.html" title="class in org.biojava.bio.structure.gui">BiojavaJmol</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html" title="class in org.biojava.bio.structure.gui">ScaleableMatrixPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/SequenceDisplay.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/SequenceDisplay.html" title="class in org.biojava.bio.structure.gui">SequenceDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/ColorUtils.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/ColorUtils.html" title="class in org.biojava.bio.structure.gui.util.color">ColorUtils</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html" title="class in org.biojava.bio.structure.gui.util.color">GradientMapper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/HSVColorSpace.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/HSVColorSpace.html" title="class in org.biojava.bio.structure.gui.util.color">HSVColorSpace</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/MMCIFFileReader.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/MMCIFFileReader.html" title="class in org.biojava.bio.structure.io">MMCIFFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">RemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStruc.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStruc.html" title="class in org.biojava.bio.structure.secstruc">SecStruc</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/CommandPrompt.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/CommandPrompt.html" title="class in org.biojava3.aaproperties">CommandPrompt</a></dt>
<dd>
<div class="block">The main method</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/DNASequence.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/DNASequence.html" title="class in org.biojava3.core.sequence">DNASequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/FastaGeneWriter.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/FastaGeneWriter.html" title="class in org.biojava3.core.sequence.io">FastaGeneWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/FastaReader.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/FastaReader.html" title="class in org.biojava3.core.sequence.io">FastaReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/FastaReaderHelper.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/FastaReaderHelper.html" title="class in org.biojava3.core.sequence.io">FastaReaderHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/FastaWriter.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/FastaWriter.html" title="class in org.biojava3.core.sequence.io">FastaWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/GenericFastaHeaderParser.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/GenericFastaHeaderParser.html" title="class in org.biojava3.core.sequence.io">GenericFastaHeaderParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/ProteinSequence.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/ProteinSequence.html" title="class in org.biojava3.core.sequence">ProteinSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/UncompressInputStream.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/UncompressInputStream.html" title="class in org.biojava3.core.util">UncompressInputStream</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/App.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava3.genome.<a href="./org/biojava3/genome/App.html" title="class in org.biojava3.genome">App</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/GeneFeatureHelper.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava3.genome.<a href="./org/biojava3/genome/GeneFeatureHelper.html" title="class in org.biojava3.genome">GeneFeatureHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/homology/GFF3FromUniprotBlastHits.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava3.genome.homology.<a href="./org/biojava3/genome/homology/GFF3FromUniprotBlastHits.html" title="class in org.biojava3.genome.homology">GFF3FromUniprotBlastHits</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/geneid/GeneIDXMLReader.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava3.genome.parsers.geneid.<a href="./org/biojava3/genome/parsers/geneid/GeneIDXMLReader.html" title="class in org.biojava3.genome.parsers.geneid">GeneIDXMLReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Feature.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Feature.html" title="class in org.biojava3.genome.parsers.gff">Feature</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>&nbsp;</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/FeatureList.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/FeatureList.html" title="class in org.biojava3.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>&nbsp;</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/GeneIDGFF2Reader.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/GeneIDGFF2Reader.html" title="class in org.biojava3.genome.parsers.gff">GeneIDGFF2Reader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/GeneMarkGTFReader.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/GeneMarkGTFReader.html" title="class in org.biojava3.genome.parsers.gff">GeneMarkGTFReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/GFF3Reader.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/GFF3Reader.html" title="class in org.biojava3.genome.parsers.gff">GFF3Reader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/GFF3Writer.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/GFF3Writer.html" title="class in org.biojava3.genome.parsers.gff">GFF3Writer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Location.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Location.html" title="class in org.biojava3.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>&nbsp;</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/LocIterator.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/LocIterator.html" title="class in org.biojava3.genome.parsers.gff">LocIterator</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>&nbsp;</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/query/BlastXMLQuery.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava3.genome.query.<a href="./org/biojava3/genome/query/BlastXMLQuery.html" title="class in org.biojava3.genome.query">BlastXMLQuery</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/query/OutputHitsGFF.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava3.genome.query.<a href="./org/biojava3/genome/query/OutputHitsGFF.html" title="class in org.biojava3.genome.query">OutputHitsGFF</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/uniprot/UniprotToFasta.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava3.genome.uniprot.<a href="./org/biojava3/genome/uniprot/UniprotToFasta.html" title="class in org.biojava3.genome.uniprot">UniprotToFasta</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/util/SplitFasta.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava3.genome.util.<a href="./org/biojava3/genome/util/SplitFasta.html" title="class in org.biojava3.genome.util">SplitFasta</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/App.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/App.html" title="class in org.biojava3.phylo">App</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/TreeConstructor.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/TreeConstructor.html" title="class in org.biojava3.phylo">TreeConstructor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/ronn/ModelLoader.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava3.ronn.<a href="./org/biojava3/ronn/ModelLoader.html" title="class in org.biojava3.ronn">ModelLoader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/ronn/ORonn.html#main(java.lang.String[])">main(String[])</a></span> - Static method in class org.biojava3.ronn.<a href="./org/biojava3/ronn/ORonn.html" title="class in org.biojava3.ronn">ORonn</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html#mandatoryArgs">mandatoryArgs</a></span> - Static variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce">AbstractUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/ContinuousColorMapperTransform.html#mapper">mapper</a></span> - Variable in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/ContinuousColorMapperTransform.html" title="class in org.biojava.bio.structure.gui.util.color">ContinuousColorMapperTransform</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/BufferedReaderBytesRead.html#mark(int)">mark(int)</a></span> - Method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/BufferedReaderBytesRead.html" title="class in org.biojava3.core.sequence.io">BufferedReaderBytesRead</a></dt>
<dd>
<div class="block">Marks the present position in the stream.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/CheckTreeAccuracy.html#markPathToRoot(org.forester.phylogeny.PhylogenyNode, boolean)">markPathToRoot(PhylogenyNode, boolean)</a></span> - Method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/CheckTreeAccuracy.html" title="class in org.biojava3.phylo">CheckTreeAccuracy</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/BufferedReaderBytesRead.html#markSupported()">markSupported()</a></span> - Method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/BufferedReaderBytesRead.html" title="class in org.biojava3.core.sequence.io">BufferedReaderBytesRead</a></dt>
<dd>
<div class="block">Tells whether this stream supports the mark() operation, which it does.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/UncompressInputStream.html#markSupported()">markSupported()</a></span> - Method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/UncompressInputStream.html" title="class in org.biojava3.core.util">UncompressInputStream</a></dt>
<dd>
<div class="block">This stream does not support mark/reset on the stream.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/jama/Maths.html" title="class in org.biojava.bio.structure.jama"><span class="strong">Maths</span></a> - Class in <a href="./org/biojava/bio/structure/jama/package-summary.html">org.biojava.bio.structure.jama</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Maths.html#Maths()">Maths()</a></span> - Constructor for class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Maths.html" title="class in org.biojava.bio.structure.jama">Maths</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama"><span class="strong">Matrix</span></a> - Class in <a href="./org/biojava/bio/structure/jama/package-summary.html">org.biojava.bio.structure.jama</a></dt>
<dd>
<div class="block">Jama = Java Matrix class.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#Matrix(int, int)">Matrix(int, int)</a></span> - Constructor for class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Construct an m-by-n matrix of zeros.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#Matrix(int, int, double)">Matrix(int, int, double)</a></span> - Constructor for class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Construct an m-by-n constant matrix.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#Matrix(double[][])">Matrix(double[][])</a></span> - Constructor for class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Construct a matrix from a 2-D array.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#Matrix(double[][], int, int)">Matrix(double[][], int, int)</a></span> - Constructor for class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Construct a matrix quickly without checking arguments.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#Matrix(double[], int)">Matrix(double[], int)</a></span> - Constructor for class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Construct a matrix from a one-dimensional packed array</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleProfile.html#matrix">matrix</a></span> - Static variable in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleProfile.html" title="class in org.biojava3.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/alignment/template/MatrixAligner.html" title="interface in org.biojava3.alignment.template"><span class="strong">MatrixAligner</span></a>&lt;<a href="./org/biojava3/alignment/template/MatrixAligner.html" title="type parameter in MatrixAligner">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava3/alignment/template/MatrixAligner.html" title="type parameter in MatrixAligner">C</a>&gt;,<a href="./org/biojava3/alignment/template/MatrixAligner.html" title="type parameter in MatrixAligner">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava3/alignment/template/package-summary.html">org.biojava3.alignment.template</a></dt>
<dd>
<div class="block">Defines an <a href="./org/biojava3/alignment/template/Aligner.html" title="interface in org.biojava3.alignment.template"><code>Aligner</code></a> which builds a score matrix during computation.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#matrixFromEuler(double, double, double)">matrixFromEuler(double, double, double)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">This conversion uses NASA standard aeroplane conventions as described on page:
   http://www.euclideanspace.com/maths/geometry/rotations/euler/index.htm
   Coordinate System: right hand
   Positive angle: right hand
   Order of euler angles: heading first, then attitude, then bank.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/MatrixListener.html#matrixInOptimizer(double[][])">matrixInOptimizer(double[][])</a></span> - Method in interface org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/MatrixListener.html" title="interface in org.biojava.bio.structure.align.ce">MatrixListener</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/ce/MatrixListener.html" title="interface in org.biojava.bio.structure.align.ce"><span class="strong">MatrixListener</span></a> - Interface in <a href="./org/biojava/bio/structure/align/ce/package-summary.html">org.biojava.bio.structure.align.ce</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#max">max</a></span> - Variable in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava3.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileParser.html#MAX_ATOMS">MAX_ATOMS</a></span> - Static variable in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileParser.html" title="class in org.biojava.bio.structure.io">PDBFileParser</a></dt>
<dd>
<div class="block">the maximum number of atoms we will add to a structure
     this protects from memory overflows in the few really big protein structures.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html#MAX_CUTS">MAX_CUTS</a></span> - Static variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html" title="class in org.biojava.bio.structure.domain.pdp">PDPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/FuzzyPoint.html#MAX_RESOLVER">MAX_RESOLVER</a></span> - Static variable in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/FuzzyPoint.html" title="class in org.biojava3.core.sequence.location">FuzzyPoint</a></dt>
<dd>
<div class="block">Always uses the max point to resolve a location</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/SequenceDisplay.html#MAX_SCALE">MAX_SCALE</a></span> - Static variable in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/SequenceDisplay.html" title="class in org.biojava.bio.structure.gui">SequenceDisplay</a></dt>
<dd>
<div class="block">the maximum value that the scale can get</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html#MAXCONT">MAXCONT</a></span> - Static variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html" title="class in org.biojava.bio.structure.domain.pdp">PDPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html#MAXDOM">MAXDOM</a></span> - Static variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html" title="class in org.biojava.bio.structure.domain.pdp">PDPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html#MAXLEN">MAXLEN</a></span> - Static variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html" title="class in org.biojava.bio.structure.domain.pdp">PDPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html#MAXSIZE">MAXSIZE</a></span> - Static variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html" title="class in org.biojava.bio.structure.domain.pdp">PDPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/FuzzyPoint.html#MEAN_RESOLVER">MEAN_RESOLVER</a></span> - Static variable in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/FuzzyPoint.html" title="class in org.biojava3.core.sequence.location">FuzzyPoint</a></dt>
<dd>
<div class="block">Combines min and max and then gets the mean of it</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/gui/MenuCreator.html" title="class in org.biojava.bio.structure.align.gui"><span class="strong">MenuCreator</span></a> - Class in <a href="./org/biojava/bio/structure/align/gui/package-summary.html">org.biojava.bio.structure.align.gui</a></dt>
<dd>
<div class="block">Create the menu for Fatcat/CE structure alignment stuff</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/MenuCreator.html#MenuCreator()">MenuCreator()</a></span> - Constructor for class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/MenuCreator.html" title="class in org.biojava.bio.structure.align.gui">MenuCreator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/gui/util/MenuCreator.html" title="class in org.biojava.bio.structure.gui.util"><span class="strong">MenuCreator</span></a> - Class in <a href="./org/biojava/bio/structure/gui/util/package-summary.html">org.biojava.bio.structure.gui.util</a></dt>
<dd>
<div class="block">Create the menu for BiojavaJmol</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/MenuCreator.html#MenuCreator()">MenuCreator()</a></span> - Constructor for class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/MenuCreator.html" title="class in org.biojava.bio.structure.gui.util">MenuCreator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#Met">Met</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#midBlue">midBlue</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#min">min</a></span> - Variable in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava3.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html#MIN_DOMAIN_LENGTH">MIN_DOMAIN_LENGTH</a></span> - Static variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html" title="class in org.biojava.bio.structure.domain.pdp">PDPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/FuzzyPoint.html#MIN_RESOLVER">MIN_RESOLVER</a></span> - Static variable in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/FuzzyPoint.html" title="class in org.biojava3.core.sequence.location">FuzzyPoint</a></dt>
<dd>
<div class="block">Always uses the min point to resolve a location</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStruc.html#MINDIST">MINDIST</a></span> - Static variable in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStruc.html" title="class in org.biojava.bio.structure.secstruc">SecStruc</a></dt>
<dd>
<div class="block">the minimal distance between two residues</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html#MINIMUM_HEIGHT">MINIMUM_HEIGHT</a></span> - Static variable in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html" title="class in org.biojava.bio.structure.gui.util">SequenceScalePanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#minus(org.biojava.bio.structure.jama.Matrix)">minus(Matrix)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">C = A - B</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Location.html#minus()">minus()</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Location.html" title="class in org.biojava3.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Return location that is in same position on negative strand.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#minusEquals(org.biojava.bio.structure.jama.Matrix)">minusEquals(Matrix)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">A = A - B</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/UserConfiguration.html#MMCIF_FORMAT">MMCIF_FORMAT</a></span> - Static variable in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/UserConfiguration.html" title="class in org.biojava.bio.structure.align.util">UserConfiguration</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif"><span class="strong">MMcifConsumer</span></a> - Interface in <a href="./org/biojava/bio/structure/io/mmcif/package-summary.html">org.biojava.bio.structure.io.mmcif</a></dt>
<dd>
<div class="block">An interface for the events triggered by a MMcifParser.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html" title="class in org.biojava.bio.structure.server"><span class="strong">MMCIFFileInstallation</span></a> - Class in <a href="./org/biojava/bio/structure/server/package-summary.html">org.biojava.bio.structure.server</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>&nbsp;</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html#MMCIFFileInstallation(java.io.File)">MMCIFFileInstallation(File)</a></span> - Constructor for class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html" title="class in org.biojava.bio.structure.server">MMCIFFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/MMCIFFileReader.html" title="class in org.biojava.bio.structure.io"><span class="strong">MMCIFFileReader</span></a> - Class in <a href="./org/biojava/bio/structure/io/package-summary.html">org.biojava.bio.structure.io</a></dt>
<dd>
<div class="block">How to parse an mmCif file:
 
  public static void main(String[] args){
        String filename =  "/path/to/something.cif.gz" ;

        StructureIOFile reader = new MMCIFFileReader();

        try{
            Structure struc = reader.getStructure(filename);
            System.out.println(struc);
        } catch (Exception e) {
            e.printStackTrace();
        }
    }
    </div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/MMCIFFileReader.html#MMCIFFileReader()">MMCIFFileReader()</a></span> - Constructor for class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/MMCIFFileReader.html" title="class in org.biojava.bio.structure.io">MMCIFFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/MMcifParser.html" title="interface in org.biojava.bio.structure.io.mmcif"><span class="strong">MMcifParser</span></a> - Interface in <a href="./org/biojava/bio/structure/io/mmcif/package-summary.html">org.biojava.bio.structure.io.mmcif</a></dt>
<dd>
<div class="block">Interface that needs to be implemented by an MMcifParser</div>
</dd>
<dt><a href="./org/biojava3/ronn/ModelLoader.html" title="class in org.biojava3.ronn"><span class="strong">ModelLoader</span></a> - Class in <a href="./org/biojava3/ronn/package-summary.html">org.biojava3.ronn</a></dt>
<dd>
<div class="block">Class that loads data from the model files into <a href="./org/biojava3/ronn/ModelLoader.Model.html" title="class in org.biojava3.ronn"><code>ModelLoader.Model</code></a> objects</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ronn/ModelLoader.html#ModelLoader()">ModelLoader()</a></span> - Constructor for class org.biojava3.ronn.<a href="./org/biojava3/ronn/ModelLoader.html" title="class in org.biojava3.ronn">ModelLoader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/ronn/ModelLoader.Model.html" title="class in org.biojava3.ronn"><span class="strong">ModelLoader.Model</span></a> - Class in <a href="./org/biojava3/ronn/package-summary.html">org.biojava3.ronn</a></dt>
<dd>
<div class="block">Represent a RONN model</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ronn/ModelLoader.Model.html#ModelLoader.Model(int, int)">ModelLoader.Model(int, int)</a></span> - Constructor for class org.biojava3.ronn.<a href="./org/biojava3/ronn/ModelLoader.Model.html" title="class in org.biojava3.ronn">ModelLoader.Model</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/ronn/ModelLoader.Threshold.html" title="class in org.biojava3.ronn"><span class="strong">ModelLoader.Threshold</span></a> - Class in <a href="./org/biojava3/ronn/package-summary.html">org.biojava3.ronn</a></dt>
<dd>
<div class="block">Represents a Threshold</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ronn/ModelLoader.Threshold.html#ModelLoader.Threshold(int)">ModelLoader.Threshold(int)</a></span> - Constructor for class org.biojava3.ronn.<a href="./org/biojava3/ronn/ModelLoader.Threshold.html" title="class in org.biojava3.ronn">ModelLoader.Threshold</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/protmod/ModificationCategory.html" title="enum in org.biojava3.protmod"><span class="strong">ModificationCategory</span></a> - Enum in <a href="./org/biojava3/protmod/package-summary.html">org.biojava3.protmod</a></dt>
<dd>
<div class="block">define modification categories.</div>
</dd>
<dt><a href="./org/biojava3/protmod/ModificationCondition.html" title="interface in org.biojava3.protmod"><span class="strong">ModificationCondition</span></a> - Interface in <a href="./org/biojava3/protmod/package-summary.html">org.biojava3.protmod</a></dt>
<dd>
<div class="block">Conditions of a protein modification, e.g. components and atoms.</div>
</dd>
<dt><a href="./org/biojava3/protmod/ModificationConditionImpl.html" title="class in org.biojava3.protmod"><span class="strong">ModificationConditionImpl</span></a> - Class in <a href="./org/biojava3/protmod/package-summary.html">org.biojava3.protmod</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ModificationConditionImpl.html#ModificationConditionImpl(java.util.List, java.util.List)">ModificationConditionImpl(List&lt;Component&gt;, List&lt;ModificationLinkage&gt;)</a></span> - Constructor for class org.biojava3.protmod.<a href="./org/biojava3/protmod/ModificationConditionImpl.html" title="class in org.biojava3.protmod">ModificationConditionImpl</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/protmod/ModificationLinkage.html" title="class in org.biojava3.protmod"><span class="strong">ModificationLinkage</span></a> - Class in <a href="./org/biojava3/protmod/package-summary.html">org.biojava3.protmod</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ModificationLinkage.html#ModificationLinkage(java.util.List, int, int)">ModificationLinkage(List&lt;Component&gt;, int, int)</a></span> - Constructor for class org.biojava3.protmod.<a href="./org/biojava3/protmod/ModificationLinkage.html" title="class in org.biojava3.protmod">ModificationLinkage</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ModificationLinkage.html#ModificationLinkage(java.util.List, int, java.lang.String, int, java.lang.String)">ModificationLinkage(List&lt;Component&gt;, int, String, int, String)</a></span> - Constructor for class org.biojava3.protmod.<a href="./org/biojava3/protmod/ModificationLinkage.html" title="class in org.biojava3.protmod">ModificationLinkage</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ModificationLinkage.html#ModificationLinkage(java.util.List, int, java.util.List, int, java.util.List)">ModificationLinkage(List&lt;Component&gt;, int, List&lt;String&gt;, int, List&lt;String&gt;)</a></span> - Constructor for class org.biojava3.protmod.<a href="./org/biojava3/protmod/ModificationLinkage.html" title="class in org.biojava3.protmod">ModificationLinkage</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ModificationLinkage.html#ModificationLinkage(java.util.List, int, java.util.List, java.lang.String, int, java.util.List, java.lang.String)">ModificationLinkage(List&lt;Component&gt;, int, List&lt;String&gt;, String, int, List&lt;String&gt;, String)</a></span> - Constructor for class org.biojava3.protmod.<a href="./org/biojava3/protmod/ModificationLinkage.html" title="class in org.biojava3.protmod">ModificationLinkage</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/protmod/ModificationOccurrenceType.html" title="enum in org.biojava3.protmod"><span class="strong">ModificationOccurrenceType</span></a> - Enum in <a href="./org/biojava3/protmod/package-summary.html">org.biojava3.protmod</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/aaproperties/xml/ModifiedAminoAcidCompoundSet.html" title="class in org.biojava3.aaproperties.xml"><span class="strong">ModifiedAminoAcidCompoundSet</span></a> - Class in <a href="./org/biojava3/aaproperties/xml/package-summary.html">org.biojava3.aaproperties.xml</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/ModifiedAminoAcidCompoundSet.html#ModifiedAminoAcidCompoundSet(java.util.List, java.util.Map)">ModifiedAminoAcidCompoundSet(List&lt;AminoAcidComposition&gt;, Map&lt;Character, Double&gt;)</a></span> - Constructor for class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/ModifiedAminoAcidCompoundSet.html" title="class in org.biojava3.aaproperties.xml">ModifiedAminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/protmod/structure/ModifiedCompound.html" title="interface in org.biojava3.protmod.structure"><span class="strong">ModifiedCompound</span></a> - Interface in <a href="./org/biojava3/protmod/structure/package-summary.html">org.biojava3.protmod.structure</a></dt>
<dd>
<div class="block">Root interface for all modifications in structure.</div>
</dd>
<dt><a href="./org/biojava3/protmod/structure/ModifiedCompoundImpl.html" title="class in org.biojava3.protmod.structure"><span class="strong">ModifiedCompoundImpl</span></a> - Class in <a href="./org/biojava3/protmod/structure/package-summary.html">org.biojava3.protmod.structure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ModifiedCompoundImpl.html#ModifiedCompoundImpl()">ModifiedCompoundImpl()</a></span> - Constructor for class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ModifiedCompoundImpl.html" title="class in org.biojava3.protmod.structure">ModifiedCompoundImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ModifiedCompoundImpl.html#ModifiedCompoundImpl(org.biojava3.protmod.ProteinModification, org.biojava3.protmod.structure.StructureGroup)">ModifiedCompoundImpl(ProteinModification, StructureGroup)</a></span> - Constructor for class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ModifiedCompoundImpl.html" title="class in org.biojava3.protmod.structure">ModifiedCompoundImpl</a></dt>
<dd>
<div class="block">Create a ModifiedCompoundImpl that has only one involved component.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ModifiedCompoundImpl.html#ModifiedCompoundImpl(org.biojava3.protmod.ProteinModification, java.util.Collection)">ModifiedCompoundImpl(ProteinModification, Collection&lt;StructureAtomLinkage&gt;)</a></span> - Constructor for class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ModifiedCompoundImpl.html" title="class in org.biojava3.protmod.structure">ModifiedCompoundImpl</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/protmod/io/ModifiedCompoundXMLConverter.html" title="class in org.biojava3.protmod.io"><span class="strong">ModifiedCompoundXMLConverter</span></a> - Class in <a href="./org/biojava3/protmod/io/package-summary.html">org.biojava3.protmod.io</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/io/ModifiedCompoundXMLConverter.html#ModifiedCompoundXMLConverter()">ModifiedCompoundXMLConverter()</a></span> - Constructor for class org.biojava3.protmod.io.<a href="./org/biojava3/protmod/io/ModifiedCompoundXMLConverter.html" title="class in org.biojava3.protmod.io">ModifiedCompoundXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/SimpleStructureServer.html#modifiedStructure(org.biojava.bio.structure.server.StructureEvent)">modifiedStructure(StructureEvent)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/SimpleStructureServer.html" title="class in org.biojava.bio.structure.server">SimpleStructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureListener.html#modifiedStructure(org.biojava.bio.structure.server.StructureEvent)">modifiedStructure(StructureEvent)</a></span> - Method in interface org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureListener.html" title="interface in org.biojava.bio.structure.server">StructureListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/LocationHelper.html#modulateCircularIndex(int, int)">modulateCircularIndex(int, int)</a></span> - Static method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/LocationHelper.html" title="class in org.biojava3.core.sequence.location">LocationHelper</a></dt>
<dd>
<div class="block">Takes a point on a circular location and moves it left until it falls
 at the earliest possible point that represents the same base.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/Location.Tools.html#modulateCircularIndex(int, int)">modulateCircularIndex(int, int)</a></span> - Static method in class org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/Location.Tools.html" title="class in org.biojava3.core.sequence.location.template">Location.Tools</a></dt>
<dd>
<div class="block">Takes a point on a circular location and moves it left until it falls
 at the earliest possible point that represents the same base.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanelMouseMotionListener.html#mouseClicked(java.awt.event.MouseEvent)">mouseClicked(MouseEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanelMouseMotionListener.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AligPanelMouseMotionListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/RasmolCommandListener.html#mouseClicked(java.awt.event.MouseEvent)">mouseClicked(MouseEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/RasmolCommandListener.html" title="class in org.biojava.bio.structure.align.gui.jmol">RasmolCommandListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html#mouseClicked(java.awt.event.MouseEvent)">mouseClicked(MouseEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html" title="class in org.biojava.bio.structure.align.gui.jmol">StructureAlignmentJmol</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/RasmolCommandListener.html#mouseClicked(java.awt.event.MouseEvent)">mouseClicked(MouseEvent)</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/RasmolCommandListener.html" title="class in org.biojava.bio.structure.gui">RasmolCommandListener</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/SequenceMouseListener.html#mouseClicked(java.awt.event.MouseEvent)">mouseClicked(MouseEvent)</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/SequenceMouseListener.html" title="class in org.biojava.bio.structure.gui.util">SequenceMouseListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanelMouseMotionListener.html#mouseDragged(java.awt.event.MouseEvent)">mouseDragged(MouseEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanelMouseMotionListener.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AligPanelMouseMotionListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html#mouseDragged(java.awt.event.MouseEvent)">mouseDragged(MouseEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html" title="class in org.biojava.bio.structure.align.gui.jmol">StructureAlignmentJmol</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/SequenceMouseListener.html#mouseDragged(java.awt.event.MouseEvent)">mouseDragged(MouseEvent)</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/SequenceMouseListener.html" title="class in org.biojava.bio.structure.gui.util">SequenceMouseListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanelMouseMotionListener.html#mouseEntered(java.awt.event.MouseEvent)">mouseEntered(MouseEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanelMouseMotionListener.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AligPanelMouseMotionListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/RasmolCommandListener.html#mouseEntered(java.awt.event.MouseEvent)">mouseEntered(MouseEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/RasmolCommandListener.html" title="class in org.biojava.bio.structure.align.gui.jmol">RasmolCommandListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html#mouseEntered(java.awt.event.MouseEvent)">mouseEntered(MouseEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html" title="class in org.biojava.bio.structure.align.gui.jmol">StructureAlignmentJmol</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/RasmolCommandListener.html#mouseEntered(java.awt.event.MouseEvent)">mouseEntered(MouseEvent)</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/RasmolCommandListener.html" title="class in org.biojava.bio.structure.gui">RasmolCommandListener</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/SequenceMouseListener.html#mouseEntered(java.awt.event.MouseEvent)">mouseEntered(MouseEvent)</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/SequenceMouseListener.html" title="class in org.biojava.bio.structure.gui.util">SequenceMouseListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanelMouseMotionListener.html#mouseExited(java.awt.event.MouseEvent)">mouseExited(MouseEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanelMouseMotionListener.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AligPanelMouseMotionListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/RasmolCommandListener.html#mouseExited(java.awt.event.MouseEvent)">mouseExited(MouseEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/RasmolCommandListener.html" title="class in org.biojava.bio.structure.align.gui.jmol">RasmolCommandListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html#mouseExited(java.awt.event.MouseEvent)">mouseExited(MouseEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html" title="class in org.biojava.bio.structure.align.gui.jmol">StructureAlignmentJmol</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/RasmolCommandListener.html#mouseExited(java.awt.event.MouseEvent)">mouseExited(MouseEvent)</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/RasmolCommandListener.html" title="class in org.biojava.bio.structure.gui">RasmolCommandListener</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/SequenceMouseListener.html#mouseExited(java.awt.event.MouseEvent)">mouseExited(MouseEvent)</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/SequenceMouseListener.html" title="class in org.biojava.bio.structure.gui.util">SequenceMouseListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanelMouseMotionListener.html#mouseMoved(java.awt.event.MouseEvent)">mouseMoved(MouseEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanelMouseMotionListener.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AligPanelMouseMotionListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html#mouseMoved(java.awt.event.MouseEvent)">mouseMoved(MouseEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html" title="class in org.biojava.bio.structure.align.gui.jmol">StructureAlignmentJmol</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/SequenceMouseListener.html#mouseMoved(java.awt.event.MouseEvent)">mouseMoved(MouseEvent)</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/SequenceMouseListener.html" title="class in org.biojava.bio.structure.gui.util">SequenceMouseListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html#mouseOverPosition(org.biojava.bio.structure.gui.util.AlignedPosition)">mouseOverPosition(AlignedPosition)</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AligPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/StatusDisplay.html#mouseOverPosition(org.biojava.bio.structure.gui.util.AlignedPosition)">mouseOverPosition(AlignedPosition)</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/StatusDisplay.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">StatusDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/events/AlignmentPositionListener.html#mouseOverPosition(org.biojava.bio.structure.gui.util.AlignedPosition)">mouseOverPosition(AlignedPosition)</a></span> - Method in interface org.biojava.bio.structure.gui.events.<a href="./org/biojava/bio/structure/gui/events/AlignmentPositionListener.html" title="interface in org.biojava.bio.structure.gui.events">AlignmentPositionListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/events/JmolAlignedPositionListener.html#mouseOverPosition(org.biojava.bio.structure.gui.util.AlignedPosition)">mouseOverPosition(AlignedPosition)</a></span> - Method in class org.biojava.bio.structure.gui.events.<a href="./org/biojava/bio/structure/gui/events/JmolAlignedPositionListener.html" title="class in org.biojava.bio.structure.gui.events">JmolAlignedPositionListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanelMouseMotionListener.html#mousePressed(java.awt.event.MouseEvent)">mousePressed(MouseEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanelMouseMotionListener.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AligPanelMouseMotionListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/RasmolCommandListener.html#mousePressed(java.awt.event.MouseEvent)">mousePressed(MouseEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/RasmolCommandListener.html" title="class in org.biojava.bio.structure.align.gui.jmol">RasmolCommandListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html#mousePressed(java.awt.event.MouseEvent)">mousePressed(MouseEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html" title="class in org.biojava.bio.structure.align.gui.jmol">StructureAlignmentJmol</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/RasmolCommandListener.html#mousePressed(java.awt.event.MouseEvent)">mousePressed(MouseEvent)</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/RasmolCommandListener.html" title="class in org.biojava.bio.structure.gui">RasmolCommandListener</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/SequenceMouseListener.html#mousePressed(java.awt.event.MouseEvent)">mousePressed(MouseEvent)</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/SequenceMouseListener.html" title="class in org.biojava.bio.structure.gui.util">SequenceMouseListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanelMouseMotionListener.html#mouseReleased(java.awt.event.MouseEvent)">mouseReleased(MouseEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanelMouseMotionListener.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AligPanelMouseMotionListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/RasmolCommandListener.html#mouseReleased(java.awt.event.MouseEvent)">mouseReleased(MouseEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/RasmolCommandListener.html" title="class in org.biojava.bio.structure.align.gui.jmol">RasmolCommandListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html#mouseReleased(java.awt.event.MouseEvent)">mouseReleased(MouseEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html" title="class in org.biojava.bio.structure.align.gui.jmol">StructureAlignmentJmol</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/RasmolCommandListener.html#mouseReleased(java.awt.event.MouseEvent)">mouseReleased(MouseEvent)</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/RasmolCommandListener.html" title="class in org.biojava.bio.structure.gui">RasmolCommandListener</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/SequenceMouseListener.html#mouseReleased(java.awt.event.MouseEvent)">mouseReleased(MouseEvent)</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/SequenceMouseListener.html" title="class in org.biojava.bio.structure.gui.util">SequenceMouseListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html#mPanel">mPanel</a></span> - Variable in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html" title="class in org.biojava.bio.structure.gui">ScaleableMatrixPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.html#multipleFrameTranslation(org.biojava3.core.sequence.template.Sequence, org.biojava3.core.sequence.transcription.Frame...)">multipleFrameTranslation(Sequence&lt;NucleotideCompound&gt;, Frame...)</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.html" title="class in org.biojava3.core.sequence.transcription">TranscriptionEngine</a></dt>
<dd>
<div class="block">A way of translating DNA in a number of frames</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/MultipleSequenceAlignment.html" title="class in org.biojava3.core.sequence"><span class="strong">MultipleSequenceAlignment</span></a>&lt;<a href="./org/biojava3/core/sequence/MultipleSequenceAlignment.html" title="type parameter in MultipleSequenceAlignment">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava3/core/sequence/MultipleSequenceAlignment.html" title="type parameter in MultipleSequenceAlignment">C</a>&gt;,<a href="./org/biojava3/core/sequence/MultipleSequenceAlignment.html" title="type parameter in MultipleSequenceAlignment">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a></dt>
<dd>
<div class="block">Implements a minimal data structure for reading and writing a sequence alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/MultipleSequenceAlignment.html#MultipleSequenceAlignment()">MultipleSequenceAlignment()</a></span> - Constructor for class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/MultipleSequenceAlignment.html" title="class in org.biojava3.core.sequence">MultipleSequenceAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/alignment/template/MutableAlignedSequence.html" title="interface in org.biojava3.alignment.template"><span class="strong">MutableAlignedSequence</span></a>&lt;<a href="./org/biojava3/alignment/template/MutableAlignedSequence.html" title="type parameter in MutableAlignedSequence">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava3/alignment/template/MutableAlignedSequence.html" title="type parameter in MutableAlignedSequence">C</a>&gt;,<a href="./org/biojava3/alignment/template/MutableAlignedSequence.html" title="type parameter in MutableAlignedSequence">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava3/alignment/template/package-summary.html">org.biojava3.alignment.template</a></dt>
<dd>
<div class="block">Defines a mutable (editable) data structure for an <a href="./org/biojava3/alignment/template/AlignedSequence.html" title="interface in org.biojava3.alignment.template"><code>AlignedSequence</code></a>.</div>
</dd>
<dt><a href="./org/biojava3/alignment/template/MutableProfile.html" title="interface in org.biojava3.alignment.template"><span class="strong">MutableProfile</span></a>&lt;<a href="./org/biojava3/alignment/template/MutableProfile.html" title="type parameter in MutableProfile">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava3/alignment/template/MutableProfile.html" title="type parameter in MutableProfile">C</a>&gt;,<a href="./org/biojava3/alignment/template/MutableProfile.html" title="type parameter in MutableProfile">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava3/alignment/template/package-summary.html">org.biojava3.alignment.template</a></dt>
<dd>
<div class="block">Defines a mutable (editable) data structure for a <a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template"><code>Profile</code></a>.</div>
</dd>
<dt><a href="./org/biojava3/alignment/template/MutableProfilePair.html" title="interface in org.biojava3.alignment.template"><span class="strong">MutableProfilePair</span></a>&lt;<a href="./org/biojava3/alignment/template/MutableProfilePair.html" title="type parameter in MutableProfilePair">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava3/alignment/template/MutableProfilePair.html" title="type parameter in MutableProfilePair">C</a>&gt;,<a href="./org/biojava3/alignment/template/MutableProfilePair.html" title="type parameter in MutableProfilePair">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava3/alignment/template/package-summary.html">org.biojava3.alignment.template</a></dt>
<dd>
<div class="block">Defines a mutable (editable) data structure for a <a href="./org/biojava3/alignment/template/ProfilePair.html" title="interface in org.biojava3.alignment.template"><code>ProfilePair</code></a>.</div>
</dd>
<dt><a href="./org/biojava3/alignment/template/MutableSequencePair.html" title="interface in org.biojava3.alignment.template"><span class="strong">MutableSequencePair</span></a>&lt;<a href="./org/biojava3/alignment/template/MutableSequencePair.html" title="type parameter in MutableSequencePair">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava3/alignment/template/MutableSequencePair.html" title="type parameter in MutableSequencePair">C</a>&gt;,<a href="./org/biojava3/alignment/template/MutableSequencePair.html" title="type parameter in MutableSequencePair">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava3/alignment/template/package-summary.html">org.biojava3.alignment.template</a></dt>
<dd>
<div class="block">Defines a mutable (editable) data structure for the results of pairwise sequence alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Mutator.html#mutate(org.biojava.bio.structure.Structure, java.lang.String, java.lang.String, java.lang.String)">mutate(Structure, String, String, String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Mutator.html" title="class in org.biojava.bio.structure">Mutator</a></dt>
<dd>
<div class="block">creates a new structure which is identical with the original one.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Mutator.html#mutateResidue(org.biojava.bio.structure.AminoAcid, java.lang.String)">mutateResidue(AminoAcid, String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Mutator.html" title="class in org.biojava.bio.structure">Mutator</a></dt>
<dd>
<div class="block">create a new residue which is of the new type.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/Mutator.html" title="class in org.biojava.bio.structure"><span class="strong">Mutator</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">A class that can change one amino acid to another.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Mutator.html#Mutator()">Mutator()</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Mutator.html" title="class in org.biojava.bio.structure">Mutator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/gui/MyAlignmentLoadListener.html" title="class in org.biojava.bio.structure.align.gui"><span class="strong">MyAlignmentLoadListener</span></a> - Class in <a href="./org/biojava/bio/structure/align/gui/package-summary.html">org.biojava.bio.structure.align.gui</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/MyAlignmentLoadListener.html#MyAlignmentLoadListener(org.biojava.bio.structure.align.gui.jmol.StructureAlignmentJmol)">MyAlignmentLoadListener(StructureAlignmentJmol)</a></span> - Constructor for class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/MyAlignmentLoadListener.html" title="class in org.biojava.bio.structure.align.gui">MyAlignmentLoadListener</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/gui/MyDistMaxListener.html" title="class in org.biojava.bio.structure.align.gui"><span class="strong">MyDistMaxListener</span></a> - Class in <a href="./org/biojava/bio/structure/align/gui/package-summary.html">org.biojava.bio.structure.align.gui</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/gui/MyExportListener.html" title="class in org.biojava.bio.structure.align.gui"><span class="strong">MyExportListener</span></a> - Class in <a href="./org/biojava/bio/structure/align/gui/package-summary.html">org.biojava.bio.structure.align.gui</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/gui/jmol/MyJmolStatusListener.html" title="class in org.biojava.bio.structure.align.gui.jmol"><span class="strong">MyJmolStatusListener</span></a> - Class in <a href="./org/biojava/bio/structure/align/gui/jmol/package-summary.html">org.biojava.bio.structure.align.gui.jmol</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/MyJmolStatusListener.html#MyJmolStatusListener()">MyJmolStatusListener()</a></span> - Constructor for class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/MyJmolStatusListener.html" title="class in org.biojava.bio.structure.align.gui.jmol">MyJmolStatusListener</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/gui/MyOpenPdbFileListener.html" title="class in org.biojava.bio.structure.align.gui"><span class="strong">MyOpenPdbFileListener</span></a> - Class in <a href="./org/biojava/bio/structure/align/gui/package-summary.html">org.biojava.bio.structure.align.gui</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/MyOpenPdbFileListener.html#MyOpenPdbFileListener()">MyOpenPdbFileListener()</a></span> - Constructor for class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/MyOpenPdbFileListener.html" title="class in org.biojava.bio.structure.align.gui">MyOpenPdbFileListener</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/gui/MySaveFileListener.html" title="class in org.biojava.bio.structure.align.gui"><span class="strong">MySaveFileListener</span></a> - Class in <a href="./org/biojava/bio/structure/align/gui/package-summary.html">org.biojava.bio.structure.align.gui</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/MySaveFileListener.html#MySaveFileListener(org.biojava.bio.structure.align.model.AFPChain)">MySaveFileListener(AFPChain)</a></span> - Constructor for class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/MySaveFileListener.html" title="class in org.biojava.bio.structure.align.gui">MySaveFileListener</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/gui/MyTableRowSorter.html" title="class in org.biojava.bio.structure.align.gui"><span class="strong">MyTableRowSorter</span></a> - Class in <a href="./org/biojava/bio/structure/align/gui/package-summary.html">org.biojava.bio.structure.align.gui</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/MyTableRowSorter.html#MyTableRowSorter(javax.swing.table.TableModel)">MyTableRowSorter(TableModel)</a></span> - Constructor for class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/MyTableRowSorter.html" title="class in org.biojava.bio.structure.align.gui">MyTableRowSorter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/aaproperties/xml/MyValidationEventHandler.html" title="class in org.biojava3.aaproperties.xml"><span class="strong">MyValidationEventHandler</span></a> - Class in <a href="./org/biojava3/aaproperties/xml/package-summary.html">org.biojava3.aaproperties.xml</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/MyValidationEventHandler.html#MyValidationEventHandler()">MyValidationEventHandler()</a></span> - Constructor for class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/MyValidationEventHandler.html" title="class in org.biojava3.aaproperties.xml">MyValidationEventHandler</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_N_">
<!--   -->
</a>
<h2 class="title">N</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/Constraints.html#N">N</a></span> - Static variable in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/Constraints.html" title="class in org.biojava3.aaproperties">Constraints</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/aaproperties/xml/Name2Count.html" title="class in org.biojava3.aaproperties.xml"><span class="strong">Name2Count</span></a> - Class in <a href="./org/biojava3/aaproperties/xml/package-summary.html">org.biojava3.aaproperties.xml</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/Name2Count.html#Name2Count()">Name2Count()</a></span> - Constructor for class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/Name2Count.html" title="class in org.biojava3.aaproperties.xml">Name2Count</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/Name2Count.html#Name2Count(java.lang.String, int)">Name2Count(String, int)</a></span> - Constructor for class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/Name2Count.html" title="class in org.biojava3.aaproperties.xml">Name2Count</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#nAtomName">nAtomName</a></span> - Static variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastAlignmentProperties.html" title="class in org.biojava3.ws.alignment.qblast"><span class="strong">NCBIQBlastAlignmentProperties</span></a> - Class in <a href="./org/biojava3/ws/alignment/qblast/package-summary.html">org.biojava3.ws.alignment.qblast</a></dt>
<dd>
<div class="block">This class implements RemotePairwiseAlignmentProperties by specifying several
 convenient methods used to wrap the addition of Blast alignment parameters.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastAlignmentProperties.html#NCBIQBlastAlignmentProperties()">NCBIQBlastAlignmentProperties()</a></span> - Constructor for class org.biojava3.ws.alignment.qblast.<a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastAlignmentProperties.html" title="class in org.biojava3.ws.alignment.qblast">NCBIQBlastAlignmentProperties</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastOutputFormat.html" title="enum in org.biojava3.ws.alignment.qblast"><span class="strong">NCBIQBlastOutputFormat</span></a> - Enum in <a href="./org/biojava3/ws/alignment/qblast/package-summary.html">org.biojava3.ws.alignment.qblast</a></dt>
<dd>
<div class="block">The RemoteQBlastOutputFormat enum acts like static fields for specifiying various
 values for certain output options.</div>
</dd>
<dt><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastOutputProperties.html" title="class in org.biojava3.ws.alignment.qblast"><span class="strong">NCBIQBlastOutputProperties</span></a> - Class in <a href="./org/biojava3/ws/alignment/qblast/package-summary.html">org.biojava3.ws.alignment.qblast</a></dt>
<dd>
<div class="block">The actual implementation of the RemotePairwiseAlignmentOutputProperties 
 interface for the QBlast service.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastOutputProperties.html#NCBIQBlastOutputProperties()">NCBIQBlastOutputProperties()</a></span> - Constructor for class org.biojava3.ws.alignment.qblast.<a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastOutputProperties.html" title="class in org.biojava3.ws.alignment.qblast">NCBIQBlastOutputProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastOutputProperties.html#NCBIQBlastOutputProperties()">NCBIQBlastOutputProperties()</a></span> - Method in class org.biojava3.ws.alignment.qblast.<a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastOutputProperties.html" title="class in org.biojava3.ws.alignment.qblast">NCBIQBlastOutputProperties</a></dt>
<dd>
<div class="block">This constructor build the parameters for the default output of the GET command sent to the QBlast service.</div>
</dd>
<dt><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastService.html" title="class in org.biojava3.ws.alignment.qblast"><span class="strong">NCBIQBlastService</span></a> - Class in <a href="./org/biojava3/ws/alignment/qblast/package-summary.html">org.biojava3.ws.alignment.qblast</a></dt>
<dd>
<div class="block">NCBIQBlastService - A simple way of submitting BLAST request to the QBlast
 service at NCBI.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastService.html#NCBIQBlastService()">NCBIQBlastService()</a></span> - Constructor for class org.biojava3.ws.alignment.qblast.<a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastService.html" title="class in org.biojava3.ws.alignment.qblast">NCBIQBlastService</a></dt>
<dd>
<div class="block">The constructor for a QBlast service request.</div>
</dd>
<dt><a href="./org/biojava3/alignment/NeedlemanWunsch.html" title="class in org.biojava3.alignment"><span class="strong">NeedlemanWunsch</span></a>&lt;<a href="./org/biojava3/alignment/NeedlemanWunsch.html" title="type parameter in NeedlemanWunsch">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava3/alignment/NeedlemanWunsch.html" title="type parameter in NeedlemanWunsch">C</a>&gt;,<a href="./org/biojava3/alignment/NeedlemanWunsch.html" title="type parameter in NeedlemanWunsch">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/alignment/package-summary.html">org.biojava3.alignment</a></dt>
<dd>
<div class="block">Needleman and Wunsch defined an algorithm for pairwise global sequence alignments (from the first until the last
 <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template"><code>Compound</code></a> of each <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a>).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/NeedlemanWunsch.html#NeedlemanWunsch()">NeedlemanWunsch()</a></span> - Constructor for class org.biojava3.alignment.<a href="./org/biojava3/alignment/NeedlemanWunsch.html" title="class in org.biojava3.alignment">NeedlemanWunsch</a></dt>
<dd>
<div class="block">Before running a pairwise global sequence alignment, data must be sent in via calls to
 <a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html#setQuery(S)"><code>AbstractPairwiseSequenceAligner.setQuery(Sequence)</code></a>, <a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html#setTarget(S)"><code>AbstractPairwiseSequenceAligner.setTarget(Sequence)</code></a>, <a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#setGapPenalty(org.biojava3.alignment.template.GapPenalty)"><code>AbstractMatrixAligner.setGapPenalty(GapPenalty)</code></a>, and
 <a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#setSubstitutionMatrix(org.biojava3.alignment.template.SubstitutionMatrix)"><code>AbstractMatrixAligner.setSubstitutionMatrix(SubstitutionMatrix)</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/NeedlemanWunsch.html#NeedlemanWunsch(S, S, org.biojava3.alignment.template.GapPenalty, org.biojava3.alignment.template.SubstitutionMatrix)">NeedlemanWunsch(S, S, GapPenalty, SubstitutionMatrix&lt;C&gt;)</a></span> - Constructor for class org.biojava3.alignment.<a href="./org/biojava3/alignment/NeedlemanWunsch.html" title="class in org.biojava3.alignment">NeedlemanWunsch</a></dt>
<dd>
<div class="block">Prepares for a pairwise global sequence alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#negative">negative</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#newAtomSite(org.biojava.bio.structure.io.mmcif.model.AtomSite)">newAtomSite(AtomSite)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#newAtomSite(org.biojava.bio.structure.io.mmcif.model.AtomSite)">newAtomSite(AtomSite)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>
<div class="block">A new AtomSite record has been read.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#newAtomSite(org.biojava.bio.structure.io.mmcif.model.AtomSite)">newAtomSite(AtomSite)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#newAuditAuthor(org.biojava.bio.structure.io.mmcif.model.AuditAuthor)">newAuditAuthor(AuditAuthor)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#newAuditAuthor(org.biojava.bio.structure.io.mmcif.model.AuditAuthor)">newAuditAuthor(AuditAuthor)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>
<div class="block">AuditAuthor contains the info from the PDB-AUTHOR records.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#newAuditAuthor(org.biojava.bio.structure.io.mmcif.model.AuditAuthor)">newAuditAuthor(AuditAuthor)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#newChemComp(org.biojava.bio.structure.io.mmcif.model.ChemComp)">newChemComp(ChemComp)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#newChemComp(org.biojava.bio.structure.io.mmcif.model.ChemComp)">newChemComp(ChemComp)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#newChemComp(org.biojava.bio.structure.io.mmcif.model.ChemComp)">newChemComp(ChemComp)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#newChemCompDescriptor(org.biojava.bio.structure.io.mmcif.model.ChemCompDescriptor)">newChemCompDescriptor(ChemCompDescriptor)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#newChemCompDescriptor(org.biojava.bio.structure.io.mmcif.model.ChemCompDescriptor)">newChemCompDescriptor(ChemCompDescriptor)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#newChemCompDescriptor(org.biojava.bio.structure.io.mmcif.model.ChemCompDescriptor)">newChemCompDescriptor(ChemCompDescriptor)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#newDatabasePDBremark(org.biojava.bio.structure.io.mmcif.model.DatabasePDBremark)">newDatabasePDBremark(DatabasePDBremark)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#newDatabasePDBremark(org.biojava.bio.structure.io.mmcif.model.DatabasePDBremark)">newDatabasePDBremark(DatabasePDBremark)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#newDatabasePDBremark(org.biojava.bio.structure.io.mmcif.model.DatabasePDBremark)">newDatabasePDBremark(DatabasePDBremark)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#newDatabasePDBrev(org.biojava.bio.structure.io.mmcif.model.DatabasePDBrev)">newDatabasePDBrev(DatabasePDBrev)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#newDatabasePDBrev(org.biojava.bio.structure.io.mmcif.model.DatabasePDBrev)">newDatabasePDBrev(DatabasePDBrev)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#newDatabasePDBrev(org.biojava.bio.structure.io.mmcif.model.DatabasePDBrev)">newDatabasePDBrev(DatabasePDBrev)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#newEntity(org.biojava.bio.structure.io.mmcif.model.Entity)">newEntity(Entity)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#newEntity(org.biojava.bio.structure.io.mmcif.model.Entity)">newEntity(Entity)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#newEntity(org.biojava.bio.structure.io.mmcif.model.Entity)">newEntity(Entity)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#newEntityPolySeq(org.biojava.bio.structure.io.mmcif.model.EntityPolySeq)">newEntityPolySeq(EntityPolySeq)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#newEntityPolySeq(org.biojava.bio.structure.io.mmcif.model.EntityPolySeq)">newEntityPolySeq(EntityPolySeq)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#newEntityPolySeq(org.biojava.bio.structure.io.mmcif.model.EntityPolySeq)">newEntityPolySeq(EntityPolySeq)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>
<div class="block">The EntityPolySeq object provide the amino acid sequence objects for the Entities.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#newExptl(org.biojava.bio.structure.io.mmcif.model.Exptl)">newExptl(Exptl)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#newExptl(org.biojava.bio.structure.io.mmcif.model.Exptl)">newExptl(Exptl)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#newExptl(org.biojava.bio.structure.io.mmcif.model.Exptl)">newExptl(Exptl)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#newGenericData(java.lang.String, java.util.List, java.util.List)">newGenericData(String, List&lt;String&gt;, List&lt;String&gt;)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#newGenericData(java.lang.String, java.util.List, java.util.List)">newGenericData(String, List&lt;String&gt;, List&lt;String&gt;)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>
<div class="block">This method is called if no particular handler for the provided cif category
 has been implemented so far.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#newGenericData(java.lang.String, java.util.List, java.util.List)">newGenericData(String, List&lt;String&gt;, List&lt;String&gt;)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/AbstractStructureAlignment.html#newline">newline</a></span> - Static variable in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/AbstractStructureAlignment.html" title="class in org.biojava.bio.structure.align">AbstractStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html#newline">newline</a></span> - Static variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce">AbstractUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCat.html#newline">newline</a></span> - Static variable in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCat.html" title="class in org.biojava.bio.structure.align.fatcat">FatCat</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#newline">newline</a></span> - Static variable in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html#newline">newline</a></span> - Static variable in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html" title="class in org.biojava.bio.structure.align.model">AfpChainWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#newline">newline</a></span> - Static variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#NEWLINE">NEWLINE</a></span> - Static variable in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ModifiedCompoundImpl.html#newline">newline</a></span> - Static variable in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ModifiedCompoundImpl.html" title="class in org.biojava3.protmod.structure">ModifiedCompoundImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/DNACompoundSet.html#newNucleotideCompound(java.lang.String, java.lang.String, java.lang.String...)">newNucleotideCompound(String, String, String...)</a></span> - Method in class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/DNACompoundSet.html" title="class in org.biojava3.core.sequence.compound">DNACompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/RNACompoundSet.html#newNucleotideCompound(java.lang.String, java.lang.String, java.lang.String...)">newNucleotideCompound(String, String, String...)</a></span> - Method in class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/RNACompoundSet.html" title="class in org.biojava3.core.sequence.compound">RNACompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractNucleotideCompoundSet.html#newNucleotideCompound(java.lang.String, java.lang.String, java.lang.String...)">newNucleotideCompound(String, String, String...)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractNucleotideCompoundSet.html" title="class in org.biojava3.core.sequence.template">AbstractNucleotideCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#newPdbxEntityNonPoly(org.biojava.bio.structure.io.mmcif.model.PdbxEntityNonPoly)">newPdbxEntityNonPoly(PdbxEntityNonPoly)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#newPdbxEntityNonPoly(org.biojava.bio.structure.io.mmcif.model.PdbxEntityNonPoly)">newPdbxEntityNonPoly(PdbxEntityNonPoly)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#newPdbxEntityNonPoly(org.biojava.bio.structure.io.mmcif.model.PdbxEntityNonPoly)">newPdbxEntityNonPoly(PdbxEntityNonPoly)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#newPdbxNonPolyScheme(org.biojava.bio.structure.io.mmcif.model.PdbxNonPolyScheme)">newPdbxNonPolyScheme(PdbxNonPolyScheme)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#newPdbxNonPolyScheme(org.biojava.bio.structure.io.mmcif.model.PdbxNonPolyScheme)">newPdbxNonPolyScheme(PdbxNonPolyScheme)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#newPdbxNonPolyScheme(org.biojava.bio.structure.io.mmcif.model.PdbxNonPolyScheme)">newPdbxNonPolyScheme(PdbxNonPolyScheme)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#newPdbxPolySeqScheme(org.biojava.bio.structure.io.mmcif.model.PdbxPolySeqScheme)">newPdbxPolySeqScheme(PdbxPolySeqScheme)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#newPdbxPolySeqScheme(org.biojava.bio.structure.io.mmcif.model.PdbxPolySeqScheme)">newPdbxPolySeqScheme(PdbxPolySeqScheme)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#newPdbxPolySeqScheme(org.biojava.bio.structure.io.mmcif.model.PdbxPolySeqScheme)">newPdbxPolySeqScheme(PdbxPolySeqScheme)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#newRefine(org.biojava.bio.structure.io.mmcif.model.Refine)">newRefine(Refine)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#newRefine(org.biojava.bio.structure.io.mmcif.model.Refine)">newRefine(Refine)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#newRefine(org.biojava.bio.structure.io.mmcif.model.Refine)">newRefine(Refine)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#newStructAsym(org.biojava.bio.structure.io.mmcif.model.StructAsym)">newStructAsym(StructAsym)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#newStructAsym(org.biojava.bio.structure.io.mmcif.model.StructAsym)">newStructAsym(StructAsym)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#newStructAsym(org.biojava.bio.structure.io.mmcif.model.StructAsym)">newStructAsym(StructAsym)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#newStructKeywords(org.biojava.bio.structure.io.mmcif.model.StructKeywords)">newStructKeywords(StructKeywords)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#newStructKeywords(org.biojava.bio.structure.io.mmcif.model.StructKeywords)">newStructKeywords(StructKeywords)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#newStructKeywords(org.biojava.bio.structure.io.mmcif.model.StructKeywords)">newStructKeywords(StructKeywords)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#newStructRef(org.biojava.bio.structure.io.mmcif.model.StructRef)">newStructRef(StructRef)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#newStructRef(org.biojava.bio.structure.io.mmcif.model.StructRef)">newStructRef(StructRef)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#newStructRef(org.biojava.bio.structure.io.mmcif.model.StructRef)">newStructRef(StructRef)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#newStructRefSeq(org.biojava.bio.structure.io.mmcif.model.StructRefSeq)">newStructRefSeq(StructRefSeq)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#newStructRefSeq(org.biojava.bio.structure.io.mmcif.model.StructRefSeq)">newStructRefSeq(StructRefSeq)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#newStructRefSeq(org.biojava.bio.structure.io.mmcif.model.StructRefSeq)">newStructRefSeq(StructRefSeq)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>
<div class="block">create a DBRef record from the StrucRefSeq record:
  
  PDB record                                    DBREF
  Field Name                                    mmCIF Data Item
  Section                                       n.a.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/SimpleStructureServer.html#newStructure(org.biojava.bio.structure.server.StructureEvent)">newStructure(StructureEvent)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/SimpleStructureServer.html" title="class in org.biojava.bio.structure.server">SimpleStructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureListener.html#newStructure(org.biojava.bio.structure.server.StructureEvent)">newStructure(StructureEvent)</a></span> - Method in interface org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureListener.html" title="interface in org.biojava.bio.structure.server">StructureListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomIterator.html#next()">next()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomIterator.html" title="class in org.biojava.bio.structure">AtomIterator</a></dt>
<dd>
<div class="block">Return next atom.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/GroupIterator.html#next()">next()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/GroupIterator.html" title="class in org.biojava.bio.structure">GroupIterator</a></dt>
<dd>
<div class="block">get next Group.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/FlatFileInstallation.html#next()">next()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/FlatFileInstallation.html" title="class in org.biojava.bio.structure.server">FlatFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html#next()">next()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html" title="class in org.biojava.bio.structure.server">MMCIFFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/PDBInstallation.html#next()">next()</a></span> - Method in interface org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/PDBInstallation.html" title="interface in org.biojava.bio.structure.server">PDBInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
<div class="block">iterate over all structures in this Installation that pass the provided filters and 
 return the next one in the list.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceMixin.SequenceIterator.html#next()">next()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceMixin.SequenceIterator.html" title="class in org.biojava3.core.sequence.template">SequenceMixin.SequenceIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/LocIterator.html#next()">next()</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/LocIterator.html" title="class in org.biojava3.genome.parsers.gff">LocIterator</a></dt>
<dd>
<div class="block">Get next window of default size, then increment position by default amount.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/LocIterator.html#next(int, int)">next(int, int)</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/LocIterator.html" title="class in org.biojava3.genome.parsers.gff">LocIterator</a></dt>
<dd>
<div class="block">Get next window of specified size, then increment position by specified amount.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#nextStep(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">nextStep(AFPChain, Atom[], Atom[])</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/phylo/NJTreeProgressListener.html" title="interface in org.biojava3.phylo"><span class="strong">NJTreeProgressListener</span></a> - Interface in <a href="./org/biojava3/phylo/package-summary.html">org.biojava3.phylo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/data/sequence/SequenceUtil.html#NON_AA">NON_AA</a></span> - Static variable in class org.biojava3.data.sequence.<a href="./org/biojava3/data/sequence/SequenceUtil.html" title="class in org.biojava3.data.sequence">SequenceUtil</a></dt>
<dd>
<div class="block">inversion of AA pattern</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/data/sequence/SequenceUtil.html#NON_NUCLEOTIDE">NON_NUCLEOTIDE</a></span> - Static variable in class org.biojava3.data.sequence.<a href="./org/biojava3/data/sequence/SequenceUtil.html" title="class in org.biojava3.data.sequence">SequenceUtil</a></dt>
<dd>
<div class="block">Non nucleotide</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceMixin.html#nonOverlappingKmers(org.biojava3.core.sequence.template.Sequence, int)">nonOverlappingKmers(Sequence&lt;C&gt;, int)</a></span> - Static method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceMixin.html" title="class in org.biojava3.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">Produces kmers of the specified size e.g.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/data/sequence/SequenceUtil.html#NONWORD">NONWORD</a></span> - Static variable in class org.biojava3.data.sequence.<a href="./org/biojava3/data/sequence/SequenceUtil.html" title="class in org.biojava3.data.sequence">SequenceUtil</a></dt>
<dd>
<div class="block">Non word</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#norm1()">norm1()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">One norm</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#norm2()">norm2()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Two norm</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/SingularValueDecomposition.html#norm2()">norm2()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/SingularValueDecomposition.html" title="class in org.biojava.bio.structure.jama">SingularValueDecomposition</a></dt>
<dd>
<div class="block">Two norm</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html#normalizeMatrix(short)">normalizeMatrix(short)</a></span> - Method in class org.biojava3.alignment.aaindex.<a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava3.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleSubstitutionMatrix.html#normalizeMatrix(short)">normalizeMatrix(short)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleSubstitutionMatrix.html" title="class in org.biojava3.alignment">SimpleSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/SubstitutionMatrix.html#normalizeMatrix(short)">normalizeMatrix(short)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/SubstitutionMatrix.html" title="interface in org.biojava3.alignment.template">SubstitutionMatrix</a></dt>
<dd>
<div class="block">Rescales the matrix so that to <a href="./org/biojava3/alignment/template/SubstitutionMatrix.html#getMaxValue()"><code>SubstitutionMatrix.getMaxValue()</code></a> - <a href="./org/biojava3/alignment/template/SubstitutionMatrix.html#getMinValue()"><code>SubstitutionMatrix.getMinValue()</code></a> = scale.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#normF()">normF()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Frobenius norm</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#normInf()">normInf()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Infinity norm</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/AlignedPosition.html#NOT_ALIGNED">NOT_ALIGNED</a></span> - Static variable in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/AlignedPosition.html" title="class in org.biojava.bio.structure.gui.util">AlignedPosition</a></dt>
<dd>
<div class="block">they can be shown in the same column (for a compact display)
 , but they are not structurally equivalent</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/AlignmentGui.html#notifyCalcFinished()">notifyCalcFinished()</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/AlignmentGui.html" title="class in org.biojava.bio.structure.align.gui">AlignmentGui</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/DBSearchGUI.html#notifyCalcFinished()">notifyCalcFinished()</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/DBSearchGUI.html" title="class in org.biojava.bio.structure.align.gui">DBSearchGUI</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/MyJmolStatusListener.html#notifyCallback(int, java.lang.Object[])">notifyCallback(int, Object[])</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/MyJmolStatusListener.html" title="class in org.biojava.bio.structure.align.gui.jmol">MyJmolStatusListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/MyJmolStatusListener.html#notifyEnabled(int)">notifyEnabled(int)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/MyJmolStatusListener.html" title="class in org.biojava.bio.structure.align.gui.jmol">MyJmolStatusListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#nrModels()">nrModels()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">return number of models .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#nrModels()">nrModels()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">return number of models.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/GroupType.html#NUCLEOTIDE">NUCLEOTIDE</a></span> - Static variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/GroupType.html" title="class in org.biojava.bio.structure">GroupType</a></dt>
<dd>
<div class="block">the type for nucelotide groups</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/data/sequence/SequenceUtil.html#NUCLEOTIDE">NUCLEOTIDE</a></span> - Static variable in class org.biojava3.data.sequence.<a href="./org/biojava3/data/sequence/SequenceUtil.html" title="class in org.biojava3.data.sequence">SequenceUtil</a></dt>
<dd>
<div class="block">Nucleotides a, t, g, c, u</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#nucleotide">nucleotide</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/SequenceTools.html#NUCLEOTIDE_LETTERS">NUCLEOTIDE_LETTERS</a></span> - Static variable in class org.biojava3.core.util.<a href="./org/biojava3/core/util/SequenceTools.html" title="class in org.biojava3.core.util">SequenceTools</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound"><span class="strong">NucleotideCompound</span></a> - Class in <a href="./org/biojava3/core/sequence/compound/package-summary.html">org.biojava3.core.sequence.compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/NucleotideCompound.html#NucleotideCompound(java.lang.String, org.biojava3.core.sequence.template.CompoundSet, java.lang.String)">NucleotideCompound(String, CompoundSet&lt;NucleotideCompound&gt;, String)</a></span> - Constructor for class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/NucleotideCompound.html#NucleotideCompound(java.lang.String, org.biojava3.core.sequence.template.CompoundSet, java.lang.String, org.biojava3.core.sequence.compound.NucleotideCompound[])">NucleotideCompound(String, CompoundSet&lt;NucleotideCompound&gt;, String, NucleotideCompound[])</a></span> - Constructor for class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/NucleotideImpl.html" title="class in org.biojava.bio.structure"><span class="strong">NucleotideImpl</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">A nucleotide group is almost the same as a Hetatm group.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/NucleotideImpl.html#NucleotideImpl()">NucleotideImpl()</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/NucleotideImpl.html" title="class in org.biojava.bio.structure">NucleotideImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#nucleotideIndex">nucleotideIndex</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#nucleotideName">nucleotideName</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/ronn/NullOutputStream.html" title="class in org.biojava3.ronn"><span class="strong">NullOutputStream</span></a> - Class in <a href="./org/biojava3/ronn/package-summary.html">org.biojava3.ronn</a></dt>
<dd>
<div class="block">The stream that void its input</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ronn/NullOutputStream.html#NullOutputStream()">NullOutputStream()</a></span> - Constructor for class org.biojava3.ronn.<a href="./org/biojava3/ronn/NullOutputStream.html" title="class in org.biojava3.ronn">NullOutputStream</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/PrettyXMLWriter.html#numericalEntity(char)">numericalEntity(char)</a></span> - Method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/PrettyXMLWriter.html" title="class in org.biojava3.core.util">PrettyXMLWriter</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_O_">
<!--   -->
</a>
<h2 class="title">O</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#oAtomName">oAtomName</a></span> - Static variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/SimpleStructureServer.html#obsoleteStructure(org.biojava.bio.structure.server.StructureEvent)">obsoleteStructure(StructureEvent)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/SimpleStructureServer.html" title="class in org.biojava.bio.structure.server">SimpleStructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureListener.html#obsoleteStructure(org.biojava.bio.structure.server.StructureEvent)">obsoleteStructure(StructureEvent)</a></span> - Method in interface org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureListener.html" title="interface in org.biojava.bio.structure.server">StructureListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/IPeptideProperties.html#obtainAminoAcidCompositionTable(java.io.File)">obtainAminoAcidCompositionTable(File)</a></span> - Method in interface org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/IPeptideProperties.html" title="interface in org.biojava3.aaproperties">IPeptideProperties</a></dt>
<dd>
<div class="block">This method would initialize amino acid composition table based on the input xml files and stores the table for usage in future calls to 
 IPeptideProperties.getMolecularWeightBasedOnXML(ProteinSequence, AminoAcidCompositionTable).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/IPeptideProperties.html#obtainAminoAcidCompositionTable(java.io.File, java.io.File)">obtainAminoAcidCompositionTable(File, File)</a></span> - Method in interface org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/IPeptideProperties.html" title="interface in org.biojava3.aaproperties">IPeptideProperties</a></dt>
<dd>
<div class="block">This method would initialize amino acid composition table based on the input xml files and stores the table for usage in future calls to 
 IPeptideProperties.getMolecularWeightBasedOnXML(ProteinSequence, AminoAcidCompositionTable).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/PeptideProperties.html#obtainAminoAcidCompositionTable(java.io.File)">obtainAminoAcidCompositionTable(File)</a></span> - Static method in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/PeptideProperties.html" title="class in org.biojava3.aaproperties">PeptideProperties</a></dt>
<dd>
<div class="block">An adaptor method would initialize amino acid composition table based on the input xml files and stores the table for usage in future calls to 
 IPeptideProperties.getMolecularWeightBasedOnXML(ProteinSequence, AminoAcidCompositionTable).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/PeptideProperties.html#obtainAminoAcidCompositionTable(java.io.File, java.io.File)">obtainAminoAcidCompositionTable(File, File)</a></span> - Static method in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/PeptideProperties.html" title="class in org.biojava3.aaproperties">PeptideProperties</a></dt>
<dd>
<div class="block">An adaptor method would initialize amino acid composition table based on the input xml files and stores the table for usage in future calls to 
 IPeptideProperties.getMolecularWeightBasedOnXML(ProteinSequence, AminoAcidCompositionTable).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/PeptidePropertiesImpl.html#obtainAminoAcidCompositionTable(java.io.File)">obtainAminoAcidCompositionTable(File)</a></span> - Method in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/PeptidePropertiesImpl.html" title="class in org.biojava3.aaproperties">PeptidePropertiesImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/PeptidePropertiesImpl.html#obtainAminoAcidCompositionTable(java.io.File, java.io.File)">obtainAminoAcidCompositionTable(File, File)</a></span> - Method in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/PeptidePropertiesImpl.html" title="class in org.biojava3.aaproperties">PeptidePropertiesImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/Component.html#of(java.lang.String)">of(String)</a></span> - Static method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/Component.html" title="class in org.biojava3.protmod">Component</a></dt>
<dd>
<div class="block">Get a Component that does not have to occur at terminals.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/Component.html#of(java.lang.String, boolean, boolean)">of(String, boolean, boolean)</a></span> - Static method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/Component.html" title="class in org.biojava3.protmod">Component</a></dt>
<dd>
<div class="block">Get or create a Component.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/Component.html#of(java.util.Set)">of(Set&lt;String&gt;)</a></span> - Static method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/Component.html" title="class in org.biojava3.protmod">Component</a></dt>
<dd>
<div class="block">Get a Component that does not have to occur at terminals.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/Component.html#of(java.util.Set, boolean, boolean)">of(Set&lt;String&gt;, boolean, boolean)</a></span> - Static method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/Component.html" title="class in org.biojava3.protmod">Component</a></dt>
<dd>
<div class="block">Get or create a Component.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/FuzzyPoint.html#offset(int)">offset(int)</a></span> - Method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/FuzzyPoint.html" title="class in org.biojava3.core.sequence.location">FuzzyPoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/SimplePoint.html#offset(int)">offset(int)</a></span> - Method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/SimplePoint.html" title="class in org.biojava3.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/Point.html#offset(int)">offset(int)</a></span> - Method in interface org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/Point.html" title="interface in org.biojava3.core.sequence.location.template">Point</a></dt>
<dd>
<div class="block">Returns a new point offset by the given distance</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/FeatureList.html#omitOverlapping(java.lang.String, org.biojava3.genome.parsers.gff.Location, boolean)">omitOverlapping(String, Location, boolean)</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/FeatureList.html" title="class in org.biojava3.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block">Create a list of all features that do not overlap the specified location on the specified sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/util/IOUtils.html#openFile(java.io.File)">openFile(File)</a></span> - Static method in class org.biojava3.core.sequence.io.util.<a href="./org/biojava3/core/sequence/io/util/IOUtils.html" title="class in org.biojava3.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">For a filename this code will check the extension of the file for a
 .gz extension.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/HTTPConnectionTools.html#openHttpURLConnection(java.net.URL, int)">openHttpURLConnection(URL, int)</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/HTTPConnectionTools.html" title="class in org.biojava.bio.structure.align.util">HTTPConnectionTools</a></dt>
<dd>
<div class="block">open HttpURLConnection.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/HTTPConnectionTools.html#openHttpURLConnection(java.net.URL)">openHttpURLConnection(URL)</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/HTTPConnectionTools.html" title="class in org.biojava.bio.structure.align.util">HTTPConnectionTools</a></dt>
<dd>
<div class="block">open HttpURLConnection.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/JmolPanel.html#openStringInline(java.lang.String)">openStringInline(String)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/JmolPanel.html" title="class in org.biojava.bio.structure.align.gui.jmol">JmolPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/PrettyXMLWriter.html#openTag(java.lang.String, java.lang.String)">openTag(String, String)</a></span> - Method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/PrettyXMLWriter.html" title="class in org.biojava3.core.util">PrettyXMLWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/PrettyXMLWriter.html#openTag(java.lang.String)">openTag(String)</a></span> - Method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/PrettyXMLWriter.html" title="class in org.biojava3.core.util">PrettyXMLWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/XMLWriter.html#openTag(java.lang.String, java.lang.String)">openTag(String, String)</a></span> - Method in interface org.biojava3.core.util.<a href="./org/biojava3/core/util/XMLWriter.html" title="interface in org.biojava3.core.util">XMLWriter</a></dt>
<dd>
<div class="block">Open a new namespace-qualified XML tag.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/XMLWriter.html#openTag(java.lang.String)">openTag(String)</a></span> - Method in interface org.biojava3.core.util.<a href="./org/biojava3/core/util/XMLWriter.html" title="interface in org.biojava3.core.util">XMLWriter</a></dt>
<dd>
<div class="block">Open a new unqualified XML tag.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Location.html#opposite()">opposite()</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Location.html" title="class in org.biojava3.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Return location that is in same position on opposite strand.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/AFPOptimizer.html#optimizeAln(org.biojava.bio.structure.align.fatcat.calc.FatCatParameters, org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">optimizeAln(FatCatParameters, AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/AFPOptimizer.html" title="class in org.biojava.bio.structure.align.fatcat.calc">AFPOptimizer</a></dt>
<dd>
<div class="block">//optimize the alignment by dynamic programming</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/StructureAlignmentOptimizer.html#optimizeResult(int[], int, int[][])">optimizeResult(int[], int, int[][])</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/StructureAlignmentOptimizer.html" title="class in org.biojava.bio.structure.align.fatcat.calc">StructureAlignmentOptimizer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/ColorUtils.html#orange">orange</a></span> - Static variable in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/ColorUtils.html" title="class in org.biojava.bio.structure.gui.util.color">ColorUtils</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a> - package org.biojava.bio.structure</dt>
<dd>
<div class="block">
Interfaces and classes for protein structure (PDB).</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/package-summary.html">org.biojava.bio.structure.align</a> - package org.biojava.bio.structure.align</dt>
<dd>
<div class="block">Classes for the alignment of structures.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/ce/package-summary.html">org.biojava.bio.structure.align.ce</a> - package org.biojava.bio.structure.align.ce</dt>
<dd>
<div class="block">Classes related to the implementation of the CE alignment algorithm, here called jCE.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/client/package-summary.html">org.biojava.bio.structure.align.client</a> - package org.biojava.bio.structure.align.client</dt>
<dd>
<div class="block">This package deals with the server communication for auto-downloading pre-calculated alignments.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/events/package-summary.html">org.biojava.bio.structure.align.events</a> - package org.biojava.bio.structure.align.events</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/fatcat/package-summary.html">org.biojava.bio.structure.align.fatcat</a> - package org.biojava.bio.structure.align.fatcat</dt>
<dd>
<div class="block">Classes related to the implementation of the FATCAT alignment algorithm, here called jFATCAT.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/fatcat/calc/package-summary.html">org.biojava.bio.structure.align.fatcat.calc</a> - package org.biojava.bio.structure.align.fatcat.calc</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/gui/package-summary.html">org.biojava.bio.structure.align.gui</a> - package org.biojava.bio.structure.align.gui</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/gui/aligpanel/package-summary.html">org.biojava.bio.structure.align.gui.aligpanel</a> - package org.biojava.bio.structure.align.gui.aligpanel</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/gui/jmol/package-summary.html">org.biojava.bio.structure.align.gui.jmol</a> - package org.biojava.bio.structure.align.gui.jmol</dt>
<dd>
<div class="block">Utility methods for better interaction with Jmol.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/helper/package-summary.html">org.biojava.bio.structure.align.helper</a> - package org.biojava.bio.structure.align.helper</dt>
<dd>
<div class="block">Helper classes for structural alignment.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/model/package-summary.html">org.biojava.bio.structure.align.model</a> - package org.biojava.bio.structure.align.model</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/pairwise/package-summary.html">org.biojava.bio.structure.align.pairwise</a> - package org.biojava.bio.structure.align.pairwise</dt>
<dd>
<div class="block">Classes for the pairwise alignment of structures.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/seq/package-summary.html">org.biojava.bio.structure.align.seq</a> - package org.biojava.bio.structure.align.seq</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/util/package-summary.html">org.biojava.bio.structure.align.util</a> - package org.biojava.bio.structure.align.util</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/webstart/package-summary.html">org.biojava.bio.structure.align.webstart</a> - package org.biojava.bio.structure.align.webstart</dt>
<dd>
<div class="block">Classes related to Java Web Start</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/xml/package-summary.html">org.biojava.bio.structure.align.xml</a> - package org.biojava.bio.structure.align.xml</dt>
<dd>
<div class="block">Serialization and deserialization for structure alignment results (as AFPChain object) to XML.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/domain/package-summary.html">org.biojava.bio.structure.domain</a> - package org.biojava.bio.structure.domain</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/domain/pdp/package-summary.html">org.biojava.bio.structure.domain.pdp</a> - package org.biojava.bio.structure.domain.pdp</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/gui/package-summary.html">org.biojava.bio.structure.gui</a> - package org.biojava.bio.structure.gui</dt>
<dd>
<div class="block">
A few convenience classes to view protein structures with Jmol (if it is on the classpath),
to calculate a protein structure alignment and to investigate the internals of the protein structure alignment algorithm.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/gui/events/package-summary.html">org.biojava.bio.structure.gui.events</a> - package org.biojava.bio.structure.gui.events</dt>
<dd>
<div class="block">
Some event classes for the protein structure GUIs.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/gui/util/package-summary.html">org.biojava.bio.structure.gui.util</a> - package org.biojava.bio.structure.gui.util</dt>
<dd>
<div class="block">
Some utility classes for the protein structure GUIs.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/gui/util/color/package-summary.html">org.biojava.bio.structure.gui.util.color</a> - package org.biojava.bio.structure.gui.util.color</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/package-summary.html">org.biojava.bio.structure.io</a> - package org.biojava.bio.structure.io</dt>
<dd>
<div class="block">Input and Output of Structures</div>
</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/package-summary.html">org.biojava.bio.structure.io.mmcif</a> - package org.biojava.bio.structure.io.mmcif</dt>
<dd>
<div class="block">Input and Output of mmcif files</div>
</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/chem/package-summary.html">org.biojava.bio.structure.io.mmcif.chem</a> - package org.biojava.bio.structure.io.mmcif.chem</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a> - package org.biojava.bio.structure.io.mmcif.model</dt>
<dd>
<div class="block">Datamodel objects used for processing mmcif files.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/jama/package-summary.html">org.biojava.bio.structure.jama</a> - package org.biojava.bio.structure.jama</dt>
<dd>
<div class="block">Matrix package for from <a href="http://math.nist.gov/javanumerics/jama/">JAMA</a></div>
</dd>
<dt><a href="./org/biojava/bio/structure/scop/package-summary.html">org.biojava.bio.structure.scop</a> - package org.biojava.bio.structure.scop</dt>
<dd>
<div class="block">Parsers and API for <a href="http://scop.mrc-lmb.cam.ac.uk/scop/">SCOP, Structural Classification of Proteins</a>.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/scop/server/package-summary.html">org.biojava.bio.structure.scop.server</a> - package org.biojava.bio.structure.scop.server</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/secstruc/package-summary.html">org.biojava.bio.structure.secstruc</a> - package org.biojava.bio.structure.secstruc</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/server/package-summary.html">org.biojava.bio.structure.server</a> - package org.biojava.bio.structure.server</dt>
<dd>
<div class="block">
classes for easier management of PDB installations

As of BioJava 3.0.2 this package is deprecated.</div>
</dd>
<dt><a href="./org/biojava3/aaproperties/package-summary.html">org.biojava3.aaproperties</a> - package org.biojava3.aaproperties</dt>
<dd>
<div class="block">Set of classes that responsible for providing APIs and Command Prompt access to generate some basic physico-chemical properties of protein sequences.</div>
</dd>
<dt><a href="./org/biojava3/aaproperties/profeat/package-summary.html">org.biojava3.aaproperties.profeat</a> - package org.biojava3.aaproperties.profeat</dt>
<dd>
<div class="block">Set of classes that responsible for providing APIs access to generate some PROFEAT properties of protein sequences.</div>
</dd>
<dt><a href="./org/biojava3/aaproperties/profeat/convertor/package-summary.html">org.biojava3.aaproperties.profeat.convertor</a> - package org.biojava3.aaproperties.profeat.convertor</dt>
<dd>
<div class="block">Set of classes that enable the conversion protein sequences into various attributes.</div>
</dd>
<dt><a href="./org/biojava3/aaproperties/xml/package-summary.html">org.biojava3.aaproperties.xml</a> - package org.biojava3.aaproperties.xml</dt>
<dd>
<div class="block">Set of classes that handles the reading and writing of xml files.</div>
</dd>
<dt><a href="./org/biojava3/alignment/package-summary.html">org.biojava3.alignment</a> - package org.biojava3.alignment</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/alignment/aaindex/package-summary.html">org.biojava3.alignment.aaindex</a> - package org.biojava3.alignment.aaindex</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/alignment/routines/package-summary.html">org.biojava3.alignment.routines</a> - package org.biojava3.alignment.routines</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/alignment/template/package-summary.html">org.biojava3.alignment.template</a> - package org.biojava3.alignment.template</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/exceptions/package-summary.html">org.biojava3.core.exceptions</a> - package org.biojava3.core.exceptions</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a> - package org.biojava3.core.sequence</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/compound/package-summary.html">org.biojava3.core.sequence.compound</a> - package org.biojava3.core.sequence.compound</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/edits/package-summary.html">org.biojava3.core.sequence.edits</a> - package org.biojava3.core.sequence.edits</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/features/package-summary.html">org.biojava3.core.sequence.features</a> - package org.biojava3.core.sequence.features</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/io/package-summary.html">org.biojava3.core.sequence.io</a> - package org.biojava3.core.sequence.io</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/io/template/package-summary.html">org.biojava3.core.sequence.io.template</a> - package org.biojava3.core.sequence.io.template</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/io/util/package-summary.html">org.biojava3.core.sequence.io.util</a> - package org.biojava3.core.sequence.io.util</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/loader/package-summary.html">org.biojava3.core.sequence.loader</a> - package org.biojava3.core.sequence.loader</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/location/package-summary.html">org.biojava3.core.sequence.location</a> - package org.biojava3.core.sequence.location</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/location/template/package-summary.html">org.biojava3.core.sequence.location.template</a> - package org.biojava3.core.sequence.location.template</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/storage/package-summary.html">org.biojava3.core.sequence.storage</a> - package org.biojava3.core.sequence.storage</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/template/package-summary.html">org.biojava3.core.sequence.template</a> - package org.biojava3.core.sequence.template</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/transcription/package-summary.html">org.biojava3.core.sequence.transcription</a> - package org.biojava3.core.sequence.transcription</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/views/package-summary.html">org.biojava3.core.sequence.views</a> - package org.biojava3.core.sequence.views</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/util/package-summary.html">org.biojava3.core.util</a> - package org.biojava3.core.util</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/data/sequence/package-summary.html">org.biojava3.data.sequence</a> - package org.biojava3.data.sequence</dt>
<dd>
<div class="block">Set of classes that responsible for data handling.</div>
</dd>
<dt><a href="./org/biojava3/genome/package-summary.html">org.biojava3.genome</a> - package org.biojava3.genome</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/genome/homology/package-summary.html">org.biojava3.genome.homology</a> - package org.biojava3.genome.homology</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/genome/parsers/geneid/package-summary.html">org.biojava3.genome.parsers.geneid</a> - package org.biojava3.genome.parsers.geneid</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/genome/parsers/gff/package-summary.html">org.biojava3.genome.parsers.gff</a> - package org.biojava3.genome.parsers.gff</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/genome/query/package-summary.html">org.biojava3.genome.query</a> - package org.biojava3.genome.query</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/genome/uniprot/package-summary.html">org.biojava3.genome.uniprot</a> - package org.biojava3.genome.uniprot</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/genome/util/package-summary.html">org.biojava3.genome.util</a> - package org.biojava3.genome.util</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/phylo/package-summary.html">org.biojava3.phylo</a> - package org.biojava3.phylo</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/protmod/package-summary.html">org.biojava3.protmod</a> - package org.biojava3.protmod</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/protmod/io/package-summary.html">org.biojava3.protmod.io</a> - package org.biojava3.protmod.io</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/protmod/structure/package-summary.html">org.biojava3.protmod.structure</a> - package org.biojava3.protmod.structure</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/ronn/package-summary.html">org.biojava3.ronn</a> - package org.biojava3.ronn</dt>
<dd>
<div class="block">
 BioJava provide a module biojava3-protein-disorder for prediction disordered regions 
 from a protein sequence.</div>
</dd>
<dt><a href="./org/biojava3/structure/gui/package-summary.html">org.biojava3.structure.gui</a> - package org.biojava3.structure.gui</dt>
<dd>
<div class="block">
New MVC interface for structure-gui
</div>
</dd>
<dt><a href="./org/biojava3/ws/alignment/package-summary.html">org.biojava3.ws.alignment</a> - package org.biojava3.ws.alignment</dt>
<dd>
<div class="block">This package contains the interfaces that need to be implemented by discrete alignment services.</div>
</dd>
<dt><a href="./org/biojava3/ws/alignment/qblast/package-summary.html">org.biojava3.ws.alignment.qblast</a> - package org.biojava3.ws.alignment.qblast</dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/ronn/ORonn.html" title="class in org.biojava3.ronn"><span class="strong">ORonn</span></a> - Class in <a href="./org/biojava3/ronn/package-summary.html">org.biojava3.ronn</a></dt>
<dd>
<div class="block">Fully re-factored and enhanced version of RONN.</div>
</dd>
<dt><a href="./org/biojava3/ronn/ORonnModel.html" title="class in org.biojava3.ronn"><span class="strong">ORonnModel</span></a> - Class in <a href="./org/biojava3/ronn/package-summary.html">org.biojava3.ronn</a></dt>
<dd>
<div class="block">Fully re-factored version of RONN model.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ronn/ORonnModel.html#ORonnModel(java.lang.String, org.biojava3.ronn.ModelLoader.Model, float)">ORonnModel(String, ModelLoader.Model, float)</a></span> - Constructor for class org.biojava3.ronn.<a href="./org/biojava3/ronn/ORonnModel.html" title="class in org.biojava3.ronn">ORonnModel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/GeneFeatureHelper.html#outputFastaSequenceLengthGFF3(java.io.File, java.io.File)">outputFastaSequenceLengthGFF3(File, File)</a></span> - Static method in class org.biojava3.genome.<a href="./org/biojava3/genome/GeneFeatureHelper.html" title="class in org.biojava3.genome">GeneFeatureHelper</a></dt>
<dd>
<div class="block">Output a gff3 feature file that will give the length of each scaffold/chromosome in the fasta file.</div>
</dd>
<dt><a href="./org/biojava3/genome/query/OutputHitsGFF.html" title="class in org.biojava3.genome.query"><span class="strong">OutputHitsGFF</span></a> - Class in <a href="./org/biojava3/genome/query/package-summary.html">org.biojava3.genome.query</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/query/OutputHitsGFF.html#OutputHitsGFF()">OutputHitsGFF()</a></span> - Constructor for class org.biojava3.genome.query.<a href="./org/biojava3/genome/query/OutputHitsGFF.html" title="class in org.biojava3.genome.query">OutputHitsGFF</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/TreeConstructor.html#outputPhylipDistances(java.lang.String)">outputPhylipDistances(String)</a></span> - Method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/TreeConstructor.html" title="class in org.biojava3.phylo">TreeConstructor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/XMLHelper.html#outputToStream(org.w3c.dom.Document, java.io.OutputStream)">outputToStream(Document, OutputStream)</a></span> - Static method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/XMLHelper.html" title="class in org.biojava3.core.util">XMLHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/XMLHelper.html#outputToStream(org.w3c.dom.Element, java.io.OutputStream)">outputToStream(Element, OutputStream)</a></span> - Static method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/XMLHelper.html" title="class in org.biojava3.core.util">XMLHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceMixin.html#overlappingKmers(org.biojava3.core.sequence.template.Sequence, int)">overlappingKmers(Sequence&lt;C&gt;, int)</a></span> - Static method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceMixin.html" title="class in org.biojava3.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">Used to generate overlapping k-mers such i.e.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Location.html#overlaps(org.biojava3.genome.parsers.gff.Location)">overlaps(Location)</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Location.html" title="class in org.biojava3.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Check if this location and other location overlap.</div>
</dd>
</dl>
<a name="_P_">
<!--   -->
</a>
<h2 class="title">P</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/Constraints.html#P">P</a></span> - Static variable in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/Constraints.html" title="class in org.biojava3.aaproperties">Constraints</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/JmolPanel.html#paint(java.awt.Graphics)">paint(Graphics)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/JmolPanel.html" title="class in org.biojava.bio.structure.align.gui.jmol">JmolPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html#paintComponent(java.awt.Graphics)">paintComponent(Graphics)</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AligPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/JMatrixPanel.html#paintComponent(java.awt.Graphics)">paintComponent(Graphics)</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/JMatrixPanel.html" title="class in org.biojava.bio.structure.gui">JMatrixPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/GradientPanel.html#paintComponent(java.awt.Graphics)">paintComponent(Graphics)</a></span> - Method in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/GradientPanel.html" title="class in org.biojava.bio.structure.gui.util.color">GradientPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html#paintComponent(java.awt.Graphics)">paintComponent(Graphics)</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html" title="class in org.biojava.bio.structure.gui.util">SequenceScalePanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html#pair">pair</a></span> - Variable in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html" title="class in org.biojava3.alignment.template">AbstractPairwiseSequenceAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractProfileProfileAligner.html#pair">pair</a></span> - Variable in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractProfileProfileAligner.html" title="class in org.biojava3.alignment.template">AbstractProfileProfileAligner</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/alignment/template/PairInProfileScorer.html" title="interface in org.biojava3.alignment.template"><span class="strong">PairInProfileScorer</span></a>&lt;<a href="./org/biojava3/alignment/template/PairInProfileScorer.html" title="type parameter in PairInProfileScorer">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava3/alignment/template/PairInProfileScorer.html" title="type parameter in PairInProfileScorer">C</a>&gt;,<a href="./org/biojava3/alignment/template/PairInProfileScorer.html" title="type parameter in PairInProfileScorer">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava3/alignment/template/package-summary.html">org.biojava3.alignment.template</a></dt>
<dd>
<div class="block">Defines an algorithm which computes a score for a sequence alignment pair picked from an alignment <a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template"><code>Profile</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/MenuCreator.html#PAIRS_ONLY">PAIRS_ONLY</a></span> - Static variable in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/MenuCreator.html" title="class in org.biojava.bio.structure.align.gui">MenuCreator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/alignment/template/PairwiseSequenceAligner.html" title="interface in org.biojava3.alignment.template"><span class="strong">PairwiseSequenceAligner</span></a>&lt;<a href="./org/biojava3/alignment/template/PairwiseSequenceAligner.html" title="type parameter in PairwiseSequenceAligner">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava3/alignment/template/PairwiseSequenceAligner.html" title="type parameter in PairwiseSequenceAligner">C</a>&gt;,<a href="./org/biojava3/alignment/template/PairwiseSequenceAligner.html" title="type parameter in PairwiseSequenceAligner">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava3/alignment/template/package-summary.html">org.biojava3.alignment.template</a></dt>
<dd>
<div class="block">Defines an <a href="./org/biojava3/alignment/template/Aligner.html" title="interface in org.biojava3.alignment.template"><code>Aligner</code></a> for a pair of <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a>s.</div>
</dd>
<dt><a href="./org/biojava3/alignment/template/PairwiseSequenceScorer.html" title="interface in org.biojava3.alignment.template"><span class="strong">PairwiseSequenceScorer</span></a>&lt;<a href="./org/biojava3/alignment/template/PairwiseSequenceScorer.html" title="type parameter in PairwiseSequenceScorer">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava3/alignment/template/PairwiseSequenceScorer.html" title="type parameter in PairwiseSequenceScorer">C</a>&gt;,<a href="./org/biojava3/alignment/template/PairwiseSequenceScorer.html" title="type parameter in PairwiseSequenceScorer">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava3/alignment/template/package-summary.html">org.biojava3.alignment.template</a></dt>
<dd>
<div class="block">Defines an algorithm which computes a score for a pair of sequences.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/gui/ParameterGUI.html" title="class in org.biojava.bio.structure.align.gui"><span class="strong">ParameterGUI</span></a> - Class in <a href="./org/biojava/bio/structure/align/gui/package-summary.html">org.biojava.bio.structure.align.gui</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/ParameterGUI.html#ParameterGUI(org.biojava.bio.structure.align.StructureAlignment)">ParameterGUI(StructureAlignment)</a></span> - Constructor for class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/ParameterGUI.html" title="class in org.biojava.bio.structure.align.gui">ParameterGUI</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html#params">params</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce">AbstractUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeMain.html#params">params</a></span> - Variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeMain.html" title="class in org.biojava.bio.structure.align.ce">CeMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/StartCodonSequence.html#parentGeneSequence">parentGeneSequence</a></span> - Variable in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/StartCodonSequence.html" title="class in org.biojava3.core.sequence">StartCodonSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/StopCodonSequence.html#parentGeneSequence">parentGeneSequence</a></span> - Variable in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/StopCodonSequence.html" title="class in org.biojava3.core.sequence">StopCodonSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/AtomInfoParser.html#parse(java.lang.String)">parse(String)</a></span> - Static method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/AtomInfoParser.html" title="class in org.biojava.bio.structure.align.gui.jmol">AtomInfoParser</a></dt>
<dd>
<div class="block">parses e.g.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifParser.html#parse(java.io.BufferedReader)">parse(BufferedReader)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifParser.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifParser</a></dt>
<dd>
<div class="block">Start the actual parsing.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifParser.html#parse(java.io.InputStream)">parse(InputStream)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifParser.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifParser</a></dt>
<dd>
<div class="block">Start the actual parsing.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#parse(java.io.InputStream)">parse(InputStream)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#parse(java.io.BufferedReader)">parse(BufferedReader)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/aaindex/AAIndexFileParser.html#parse(java.io.InputStream)">parse(InputStream)</a></span> - Method in class org.biojava3.alignment.aaindex.<a href="./org/biojava3/alignment/aaindex/AAIndexFileParser.html" title="class in org.biojava3.alignment.aaindex">AAIndexFileParser</a></dt>
<dd>
<div class="block">parse an inputStream that points to an AAINDEX database file</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/InsdcParser.html#parse(java.lang.String)">parse(String)</a></span> - Method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/InsdcParser.html" title="class in org.biojava3.core.sequence.location">InsdcParser</a></dt>
<dd>
<div class="block">Main method for parsing a location from a String instance</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/InsdcParser.html#parse(java.io.Reader)">parse(Reader)</a></span> - Method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/InsdcParser.html" title="class in org.biojava3.core.sequence.location">InsdcParser</a></dt>
<dd>
<div class="block">Reader based version of the parse methods.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/InsdcParser.html#parse(java.io.Reader, org.biojava3.core.sequence.Strand)">parse(Reader, Strand)</a></span> - Method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/InsdcParser.html" title="class in org.biojava3.core.sequence.location">InsdcParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/GenericFastaHeaderParser.html#parseHeader(java.lang.String, S)">parseHeader(String, S)</a></span> - Method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/GenericFastaHeaderParser.html" title="class in org.biojava3.core.sequence.io">GenericFastaHeaderParser</a></dt>
<dd>
<div class="block">Parse the header and set the values in the sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/template/FastaHeaderParserInterface.html#parseHeader(java.lang.String, S)">parseHeader(String, S)</a></span> - Method in interface org.biojava3.core.sequence.io.template.<a href="./org/biojava3/core/sequence/io/template/FastaHeaderParserInterface.html" title="interface in org.biojava3.core.sequence.io.template">FastaHeaderParserInterface</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/InsdcParser.html#parseLocation(java.lang.String, org.biojava3.core.sequence.Strand)">parseLocation(String, Strand)</a></span> - Method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/InsdcParser.html" title="class in org.biojava3.core.sequence.location">InsdcParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/AFPChainXMLParser.html#parseMultiXML(java.lang.String)">parseMultiXML(String)</a></span> - Static method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/AFPChainXMLParser.html" title="class in org.biojava.bio.structure.align.xml">AFPChainXMLParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileParser.html#parsePDBFile(java.io.InputStream)">parsePDBFile(InputStream)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileParser.html" title="class in org.biojava.bio.structure.io">PDBFileParser</a></dt>
<dd>
<div class="block">parse a PDB file and return a datastructure implementing
 PDBStructure interface.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileParser.html#parsePDBFile(java.io.BufferedReader)">parsePDBFile(BufferedReader)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileParser.html" title="class in org.biojava.bio.structure.io">PDBFileParser</a></dt>
<dd>
<div class="block">parse a PDB file and return a datastructure implementing
 PDBStructure interface.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/InsdcParser.html#parseRange(java.util.regex.Matcher, org.biojava3.core.sequence.Strand)">parseRange(Matcher, Strand)</a></span> - Method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/InsdcParser.html" title="class in org.biojava3.core.sequence.location">InsdcParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/exceptions/ParserException.html" title="class in org.biojava3.core.exceptions"><span class="strong">ParserException</span></a> - Exception in <a href="./org/biojava3/core/exceptions/package-summary.html">org.biojava3.core.exceptions</a></dt>
<dd>
<div class="block">General abstraction of different parsing errors</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/exceptions/ParserException.html#ParserException(java.lang.String)">ParserException(String)</a></span> - Constructor for exception org.biojava3.core.exceptions.<a href="./org/biojava3/core/exceptions/ParserException.html" title="class in org.biojava3.core.exceptions">ParserException</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/exceptions/ParserException.html#ParserException(java.lang.Exception)">ParserException(Exception)</a></span> - Constructor for exception org.biojava3.core.exceptions.<a href="./org/biojava3/core/exceptions/ParserException.html" title="class in org.biojava3.core.exceptions">ParserException</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/exceptions/ParserException.html#ParserException(java.lang.String, java.lang.Exception)">ParserException(String, Exception)</a></span> - Constructor for exception org.biojava3.core.exceptions.<a href="./org/biojava3/core/exceptions/ParserException.html" title="class in org.biojava3.core.exceptions">ParserException</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/InsdcParser.html#parseSingle(java.util.regex.Matcher, org.biojava3.core.sequence.Strand)">parseSingle(Matcher, Strand)</a></span> - Method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/InsdcParser.html" title="class in org.biojava3.core.sequence.location">InsdcParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/alignment/template/PartitionRefiner.html" title="interface in org.biojava3.alignment.template"><span class="strong">PartitionRefiner</span></a>&lt;<a href="./org/biojava3/alignment/template/PartitionRefiner.html" title="type parameter in PartitionRefiner">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava3/alignment/template/PartitionRefiner.html" title="type parameter in PartitionRefiner">C</a>&gt;,<a href="./org/biojava3/alignment/template/PartitionRefiner.html" title="type parameter in PartitionRefiner">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava3/alignment/template/package-summary.html">org.biojava3.alignment.template</a></dt>
<dd>
<div class="block">Defines an algorithm which computes a new alignment <a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template"><code>Profile</code></a> by splitting a current alignment and realigning.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileParser.html#PDB_AUTHOR_ASSIGNMENT">PDB_AUTHOR_ASSIGNMENT</a></span> - Static variable in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileParser.html" title="class in org.biojava.bio.structure.io">PDBFileParser</a></dt>
<dd>
<div class="block">Secondary strucuture assigned by the PDB author/</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html#PDB_DIR">PDB_DIR</a></span> - Static variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce">AbstractUserArgumentProcessor</a></dt>
<dd>
<div class="block">the system property PDB_DIR can be used to configure the 
 default location for PDB files.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/ConfigPDBInstallPanel.html#PDB_DIR">PDB_DIR</a></span> - Static variable in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/ConfigPDBInstallPanel.html" title="class in org.biojava.bio.structure.align.gui">ConfigPDBInstallPanel</a></dt>
<dd>
<div class="block">the system property PDB_DIR can be used to configure the 
 default location for PDB files.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/UserConfiguration.html#PDB_DIR">PDB_DIR</a></span> - Static variable in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/UserConfiguration.html" title="class in org.biojava.bio.structure.align.util">UserConfiguration</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/PDBDirPanel.html#PDB_DIR">PDB_DIR</a></span> - Static variable in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/PDBDirPanel.html" title="class in org.biojava.bio.structure.gui.util">PDBDirPanel</a></dt>
<dd>
<div class="block">the system property PDB_DIR can be used to configure the 
 default location for PDB files.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#PDB_FILE_SERVER_PROPERTY">PDB_FILE_SERVER_PROPERTY</a></span> - Static variable in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#pdb_flag">pdb_flag</a></span> - Variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/UserConfiguration.html#PDB_FORMAT">PDB_FORMAT</a></span> - Static variable in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/UserConfiguration.html" title="class in org.biojava.bio.structure.align.util">UserConfiguration</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#pdb_name">pdb_name</a></span> - Variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBStatus.html#PDB_SERVER_PROPERTY">PDB_SERVER_PROPERTY</a></span> - Static variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBStatus.html" title="class in org.biojava.bio.structure">PDBStatus</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/gui/util/PDBDirPanel.html" title="class in org.biojava.bio.structure.gui.util"><span class="strong">PDBDirPanel</span></a> - Class in <a href="./org/biojava/bio/structure/gui/util/package-summary.html">org.biojava.bio.structure.gui.util</a></dt>
<dd>
<div class="block">A class to define where a structure for the alignment is coming from</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/PDBDirPanel.html#PDBDirPanel()">PDBDirPanel()</a></span> - Constructor for class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/PDBDirPanel.html" title="class in org.biojava.bio.structure.gui.util">PDBDirPanel</a></dt>
<dd>
<div class="block">load the PDB files from a local directory</div>
</dd>
<dt><a href="./org/biojava/bio/structure/io/PDBFileParser.html" title="class in org.biojava.bio.structure.io"><span class="strong">PDBFileParser</span></a> - Class in <a href="./org/biojava/bio/structure/io/package-summary.html">org.biojava.bio.structure.io</a></dt>
<dd>
<div class="block">This class implements the actual PDB file parsing.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileParser.html#PDBFileParser()">PDBFileParser()</a></span> - Constructor for class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileParser.html" title="class in org.biojava.bio.structure.io">PDBFileParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io"><span class="strong">PDBFileReader</span></a> - Class in <a href="./org/biojava/bio/structure/io/package-summary.html">org.biojava.bio.structure.io</a></dt>
<dd>
<div class="block">
  The wrapper class for parsing a PDB file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#PDBFileReader()">PDBFileReader()</a></span> - Constructor for class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/server/PDBFilter.html" title="interface in org.biojava.bio.structure.server"><span class="strong">PDBFilter</span></a> - Interface in <a href="./org/biojava/bio/structure/server/package-summary.html">org.biojava.bio.structure.server</a></dt>
<dd>
<div class="block">defines how a filter of PDB files has to behave</div>
</dd>
<dt><a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure"><span class="strong">PDBHeader</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">A class that contains PDB Header information.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#PDBHeader()">PDBHeader()</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/server/PDBInstallation.html" title="interface in org.biojava.bio.structure.server"><span class="strong">PDBInstallation</span></a> - Interface in <a href="./org/biojava/bio/structure/server/package-summary.html">org.biojava.bio.structure.server</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>&nbsp;</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#pdbNumRangeRegex">pdbNumRangeRegex</a></span> - Static variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>
<div class="block">Pattern to describe subranges.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/client/PdbPair.html" title="class in org.biojava.bio.structure.align.client"><span class="strong">PdbPair</span></a> - Class in <a href="./org/biojava/bio/structure/align/client/package-summary.html">org.biojava.bio.structure.align.client</a></dt>
<dd>
<div class="block">A pair for structure alignment</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/PdbPair.html#PdbPair(java.lang.String, java.lang.String)">PdbPair(String, String)</a></span> - Constructor for class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/PdbPair.html" title="class in org.biojava.bio.structure.align.client">PdbPair</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/xml/PdbPairXMLConverter.html" title="class in org.biojava.bio.structure.align.xml"><span class="strong">PdbPairXMLConverter</span></a> - Class in <a href="./org/biojava/bio/structure/align/xml/package-summary.html">org.biojava.bio.structure.align.xml</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/PdbPairXMLConverter.html#PdbPairXMLConverter()">PdbPairXMLConverter()</a></span> - Constructor for class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/PdbPairXMLConverter.html" title="class in org.biojava.bio.structure.align.xml">PdbPairXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/PDBParseException.html" title="class in org.biojava.bio.structure.io"><span class="strong">PDBParseException</span></a> - Exception in <a href="./org/biojava/bio/structure/io/package-summary.html">org.biojava.bio.structure.io</a></dt>
<dd>
<div class="block">An exception during the parsing of a PDB file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBParseException.html#PDBParseException(java.lang.String)">PDBParseException(String)</a></span> - Constructor for exception org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBParseException.html" title="class in org.biojava.bio.structure.io">PDBParseException</a></dt>
<dd>
<div class="block">Constructs a PDBParseException object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBParseException.html#PDBParseException(java.lang.String, java.lang.Throwable)">PDBParseException(String, Throwable)</a></span> - Constructor for exception org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBParseException.html" title="class in org.biojava.bio.structure.io">PDBParseException</a></dt>
<dd>
<div class="block">Constructs a PDBParseException object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBParseException.html#PDBParseException(java.lang.Throwable)">PDBParseException(Throwable)</a></span> - Constructor for exception org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBParseException.html" title="class in org.biojava.bio.structure.io">PDBParseException</a></dt>
<dd>
<div class="block">Constructs a PDBParseException object.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/PDBRecord.html" title="interface in org.biojava.bio.structure"><span class="strong">PDBRecord</span></a> - Interface in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">An interface implemented by all classes that represent PDB records</div>
</dd>
<dt><a href="./org/biojava/bio/structure/gui/util/PDBServerPanel.html" title="class in org.biojava.bio.structure.gui.util"><span class="strong">PDBServerPanel</span></a> - Class in <a href="./org/biojava/bio/structure/gui/util/package-summary.html">org.biojava.bio.structure.gui.util</a></dt>
<dd>
<div class="block">A class to define where a structure for the alignment is coming from</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/PDBServerPanel.html#PDBServerPanel()">PDBServerPanel()</a></span> - Constructor for class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/PDBServerPanel.html" title="class in org.biojava.bio.structure.gui.util">PDBServerPanel</a></dt>
<dd>
<div class="block">load the PDB files from a local directory</div>
</dd>
<dt><a href="./org/biojava/bio/structure/io/PDBSRSReader.html" title="class in org.biojava.bio.structure.io"><span class="strong">PDBSRSReader</span></a> - Class in <a href="./org/biojava/bio/structure/io/package-summary.html">org.biojava.bio.structure.io</a></dt>
<dd>
<div class="block">reads a PDB file from a local SRS installation using getz Actually
 is the same as PDBFileReader, but instead of reading from a file stream, reads from a
 buffered stream.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBSRSReader.html#PDBSRSReader()">PDBSRSReader()</a></span> - Constructor for class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBSRSReader.html" title="class in org.biojava.bio.structure.io">PDBSRSReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/PDBStatus.html" title="class in org.biojava.bio.structure"><span class="strong">PDBStatus</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">Methods for getting the status of a PDB file (current, obsolete, etc)
 and for accessing different versions of the structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBStatus.html#PDBStatus()">PDBStatus()</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBStatus.html" title="class in org.biojava.bio.structure">PDBStatus</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/PDBStatus.Status.html" title="enum in org.biojava.bio.structure"><span class="strong">PDBStatus.Status</span></a> - Enum in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">Represents the status of PDB IDs.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/gui/util/PDBUploadPanel.html" title="class in org.biojava.bio.structure.gui.util"><span class="strong">PDBUploadPanel</span></a> - Class in <a href="./org/biojava/bio/structure/gui/util/package-summary.html">org.biojava.bio.structure.gui.util</a></dt>
<dd>
<div class="block">A JPanel to upload 2 custom PDB files.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/PDBUploadPanel.html#PDBUploadPanel()">PDBUploadPanel()</a></span> - Constructor for class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/PDBUploadPanel.html" title="class in org.biojava.bio.structure.gui.util">PDBUploadPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/PDBUploadPanel.html#PDBUploadPanel(boolean)">PDBUploadPanel(boolean)</a></span> - Constructor for class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/PDBUploadPanel.html" title="class in org.biojava.bio.structure.gui.util">PDBUploadPanel</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxEntityNonPoly.html" title="class in org.biojava.bio.structure.io.mmcif.model"><span class="strong">PdbxEntityNonPoly</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a></dt>
<dd>
<div class="block">A bean for the Pdbx_entity_nonpoly category.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxEntityNonPoly.html#PdbxEntityNonPoly()">PdbxEntityNonPoly()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxEntityNonPoly.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxEntityNonPoly</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model"><span class="strong">PdbxNonPolyScheme</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a></dt>
<dd>
<div class="block">A bean for the PDBX_NONPOLY_SCHEME category, which provides residue level nomenclature
 mapping for non-polymer entities.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#PdbxNonPolyScheme()">PdbxNonPolyScheme()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model"><span class="strong">PdbxPolySeqScheme</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a></dt>
<dd>
<div class="block">A bean for the PDBX_POLY_SEQ_SCHEME category, which provides residue level nomenclature
 mapping for polymer entities.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#PdbxPolySeqScheme()">PdbxPolySeqScheme()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/domain/pdp/PDPDistanceMatrix.html" title="class in org.biojava.bio.structure.domain.pdp"><span class="strong">PDPDistanceMatrix</span></a> - Class in <a href="./org/biojava/bio/structure/domain/pdp/package-summary.html">org.biojava.bio.structure.domain.pdp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPDistanceMatrix.html#PDPDistanceMatrix()">PDPDistanceMatrix()</a></span> - Constructor for class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPDistanceMatrix.html" title="class in org.biojava.bio.structure.domain.pdp">PDPDistanceMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html" title="class in org.biojava.bio.structure.domain.pdp"><span class="strong">PDPParameters</span></a> - Class in <a href="./org/biojava/bio/structure/domain/pdp/package-summary.html">org.biojava.bio.structure.domain.pdp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html#PDPParameters()">PDPParameters()</a></span> - Constructor for class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html" title="class in org.biojava.bio.structure.domain.pdp">PDPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/aaproperties/PeptideProperties.html" title="class in org.biojava3.aaproperties"><span class="strong">PeptideProperties</span></a> - Class in <a href="./org/biojava3/aaproperties/package-summary.html">org.biojava3.aaproperties</a></dt>
<dd>
<div class="block">This is an adaptor class which enable the ease of generating protein properties.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/PeptideProperties.html#PeptideProperties()">PeptideProperties()</a></span> - Constructor for class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/PeptideProperties.html" title="class in org.biojava3.aaproperties">PeptideProperties</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/aaproperties/PeptideProperties.SingleLetterAACode.html" title="enum in org.biojava3.aaproperties"><span class="strong">PeptideProperties.SingleLetterAACode</span></a> - Enum in <a href="./org/biojava3/aaproperties/package-summary.html">org.biojava3.aaproperties</a></dt>
<dd>
<div class="block">Enumeration of 20 standard amino acid code</div>
</dd>
<dt><a href="./org/biojava3/aaproperties/PeptidePropertiesImpl.html" title="class in org.biojava3.aaproperties"><span class="strong">PeptidePropertiesImpl</span></a> - Class in <a href="./org/biojava3/aaproperties/package-summary.html">org.biojava3.aaproperties</a></dt>
<dd>
<div class="block">This class contains the actual implementation of IPeptideProperties and is wrapped around by PeptideProperties for ease of use.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/PeptidePropertiesImpl.html#PeptidePropertiesImpl()">PeptidePropertiesImpl()</a></span> - Constructor for class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/PeptidePropertiesImpl.html" title="class in org.biojava3.aaproperties">PeptidePropertiesImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/SequenceTools.html#percentNucleotideSequence(java.lang.String)">percentNucleotideSequence(String)</a></span> - Static method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/SequenceTools.html" title="class in org.biojava3.core.util">SequenceTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Location.html#percentOverlap(org.biojava3.genome.parsers.gff.Location)">percentOverlap(Location)</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Location.html" title="class in org.biojava3.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Return percent overlap of two locations.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/webstart/WebStartMain.html#persistConfig(org.biojava.bio.structure.align.util.UserConfiguration)">persistConfig(UserConfiguration)</a></span> - Static method in class org.biojava.bio.structure.align.webstart.<a href="./org/biojava/bio/structure/align/webstart/WebStartMain.html" title="class in org.biojava.bio.structure.align.webstart">WebStartMain</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/webstart/PersistentConfig.html" title="class in org.biojava.bio.structure.align.webstart"><span class="strong">PersistentConfig</span></a> - Class in <a href="./org/biojava/bio/structure/align/webstart/package-summary.html">org.biojava.bio.structure.align.webstart</a></dt>
<dd>
<div class="block">a class to store the config using the Java Web Start
 PersistenService.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/webstart/PersistentConfig.html#PersistentConfig()">PersistentConfig()</a></span> - Constructor for class org.biojava.bio.structure.align.webstart.<a href="./org/biojava/bio/structure/align/webstart/PersistentConfig.html" title="class in org.biojava.bio.structure.align.webstart">PersistentConfig</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#Phe">Phe</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/Comparison.html#PID(java.lang.String, java.lang.String)">PID(String, String)</a></span> - Static method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/Comparison.html" title="class in org.biojava3.phylo">Comparison</a></dt>
<dd>
<div class="block">this is a gapped PID calculation</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/Comparison.html#PID(java.lang.String, java.lang.String, int, int)">PID(String, String, int, int)</a></span> - Static method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/Comparison.html" title="class in org.biojava3.phylo">Comparison</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#pidColours">pidColours</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#pidThresholds">pidThresholds</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#plus(org.biojava.bio.structure.Structure, org.biojava.bio.structure.jama.Matrix)">plus(Structure, Matrix)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">calculate structure + Matrix coodinates ...</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#plus(org.biojava.bio.structure.jama.Matrix)">plus(Matrix)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">C = A + B</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Location.html#plus()">plus()</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Location.html" title="class in org.biojava3.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Return location that is in same position on plus strand.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#plusEquals(org.biojava.bio.structure.jama.Matrix)">plusEquals(Matrix)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">A = A + B</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/location/template/Point.html" title="interface in org.biojava3.core.sequence.location.template"><span class="strong">Point</span></a> - Interface in <a href="./org/biojava3/core/sequence/location/template/package-summary.html">org.biojava3.core.sequence.location.template</a></dt>
<dd>
<div class="block">Holds a single point part of a location</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/location/template/Point.Resolver.html" title="interface in org.biojava3.core.sequence.location.template"><span class="strong">Point.Resolver</span></a>&lt;<a href="./org/biojava3/core/sequence/location/template/Point.Resolver.html" title="type parameter in Point.Resolver">T</a> extends <a href="./org/biojava3/core/sequence/location/template/Point.html" title="interface in org.biojava3.core.sequence.location.template">Point</a>&gt; - Interface in <a href="./org/biojava3/core/sequence/location/template/package-summary.html">org.biojava3.core.sequence.location.template</a></dt>
<dd>
<div class="block">Used to resolve a position about a point</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#polar">polar</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/chem/PolymerType.html" title="enum in org.biojava.bio.structure.io.mmcif.chem"><span class="strong">PolymerType</span></a> - Enum in <a href="./org/biojava/bio/structure/io/mmcif/chem/package-summary.html">org.biojava.bio.structure.io.mmcif.chem</a></dt>
<dd>
<div class="block">Enumerates the classification of polymers.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/chem/ResidueType.html#polymerType">polymerType</a></span> - Variable in enum org.biojava.bio.structure.io.mmcif.chem.<a href="./org/biojava/bio/structure/io/mmcif/chem/ResidueType.html" title="enum in org.biojava.bio.structure.io.mmcif.chem">ResidueType</a></dt>
<dd>
<div class="block">The associated <a href="./org/biojava/bio/structure/io/mmcif/chem/PolymerType.html" title="enum in org.biojava.bio.structure.io.mmcif.chem"><code>PolymerType</code></a></div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/chem/PolymerType.html#polymerTypeFromString(java.lang.String)">polymerTypeFromString(String)</a></span> - Static method in enum org.biojava.bio.structure.io.mmcif.chem.<a href="./org/biojava/bio/structure/io/mmcif/chem/PolymerType.html" title="enum in org.biojava.bio.structure.io.mmcif.chem">PolymerType</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/chem/PolymerType.html#POLYNUCLEOTIDE_ONLY">POLYNUCLEOTIDE_ONLY</a></span> - Static variable in enum org.biojava.bio.structure.io.mmcif.chem.<a href="./org/biojava/bio/structure/io/mmcif/chem/PolymerType.html" title="enum in org.biojava.bio.structure.io.mmcif.chem">PolymerType</a></dt>
<dd>
<div class="block">Convenience <tt>Set</tt> of polymer types classified as DNA.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#populate(org.biojava3.core.sequence.template.Sequence)">populate(Sequence&lt;C&gt;)</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava3.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>
<div class="block">Loops through the Compounds in a Sequence and passes them onto
 <a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#setCompoundAt(C, int)"><code>BitSequenceReader.BitArrayWorker.setCompoundAt(Compound, int)</code></a></div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#populate(java.lang.String)">populate(String)</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava3.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>
<div class="block">Loops through the chars in a String and passes them onto
 <a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#setCompoundAt(char, int)"><code>BitSequenceReader.BitArrayWorker.setCompoundAt(char, int)</code></a></div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/ElementTable.html#populateMaps()">populateMaps()</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/ElementTable.html" title="class in org.biojava3.aaproperties.xml">ElementTable</a></dt>
<dd>
<div class="block">Populate the Maps for quick retrieval</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/xml/PositionInQueueXMLConverter.html" title="class in org.biojava.bio.structure.align.xml"><span class="strong">PositionInQueueXMLConverter</span></a> - Class in <a href="./org/biojava/bio/structure/align/xml/package-summary.html">org.biojava.bio.structure.align.xml</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/PositionInQueueXMLConverter.html#PositionInQueueXMLConverter()">PositionInQueueXMLConverter()</a></span> - Constructor for class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/PositionInQueueXMLConverter.html" title="class in org.biojava.bio.structure.align.xml">PositionInQueueXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html#positionSelected(org.biojava.bio.structure.gui.util.AlignedPosition)">positionSelected(AlignedPosition)</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AligPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/StatusDisplay.html#positionSelected(org.biojava.bio.structure.gui.util.AlignedPosition)">positionSelected(AlignedPosition)</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/StatusDisplay.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">StatusDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/events/AlignmentPositionListener.html#positionSelected(org.biojava.bio.structure.gui.util.AlignedPosition)">positionSelected(AlignedPosition)</a></span> - Method in interface org.biojava.bio.structure.gui.events.<a href="./org/biojava/bio/structure/gui/events/AlignmentPositionListener.html" title="interface in org.biojava.bio.structure.gui.events">AlignmentPositionListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/events/JmolAlignedPositionListener.html#positionSelected(org.biojava.bio.structure.gui.util.AlignedPosition)">positionSelected(AlignedPosition)</a></span> - Method in class org.biojava.bio.structure.gui.events.<a href="./org/biojava/bio/structure/gui/events/JmolAlignedPositionListener.html" title="class in org.biojava.bio.structure.gui.events">JmolAlignedPositionListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#positive">positive</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/AFPPostProcessor.html#postProcess(org.biojava.bio.structure.align.fatcat.calc.FatCatParameters, org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">postProcess(FatCatParameters, AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/AFPPostProcessor.html" title="class in org.biojava.bio.structure.align.fatcat.calc">AFPPostProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCPMain.html#postProcessAlignment(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], org.biojava.bio.structure.align.ce.CECalculator)">postProcessAlignment(AFPChain, Atom[], Atom[], CECalculator)</a></span> - Static method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCPMain.html" title="class in org.biojava.bio.structure.align.ce">CeCPMain</a></dt>
<dd>
<div class="block">Circular permutation specific code to be run after the standard CE alignment</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompoundTranslator.html#postProcessCompoundLists(java.util.List)">postProcessCompoundLists(List&lt;List&lt;T&gt;&gt;)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompoundTranslator.html" title="class in org.biojava3.core.sequence.template">AbstractCompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/DNAToRNATranslator.html#postProcessCompoundLists(java.util.List)">postProcessCompoundLists(List&lt;List&lt;NucleotideCompound&gt;&gt;)</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/DNAToRNATranslator.html" title="class in org.biojava3.core.sequence.transcription">DNAToRNATranslator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/RNAToAminoAcidTranslator.html#postProcessCompoundLists(java.util.List)">postProcessCompoundLists(List&lt;List&lt;AminoAcidCompound&gt;&gt;)</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/RNAToAminoAcidTranslator.html" title="class in org.biojava3.core.sequence.transcription">RNAToAminoAcidTranslator</a></dt>
<dd>
<div class="block">Performs the trimming of stop codons and the conversion of a valid start
 amino acid to M</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Location.html#prefix(int)">prefix(int)</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Location.html" title="class in org.biojava3.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">The part of this location before the specified position.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Location.html#prefix(org.biojava3.genome.parsers.gff.Location)">prefix(Location)</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Location.html" title="class in org.biojava3.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">The part of this location before the other location (not inclusive).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCPMain.html#prepareAtomsForAlign(org.biojava.bio.structure.Atom[])">prepareAtomsForAlign(Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCPMain.html" title="class in org.biojava.bio.structure.align.ce">CeCPMain</a></dt>
<dd>
<div class="block">Circular permutation specific code to be run before a standard CE alignment</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/GuiWrapper.html#prepareGroupsForDisplay(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">prepareGroupsForDisplay(AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/GuiWrapper.html" title="class in org.biojava.bio.structure.align.ce">GuiWrapper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/StructureAlignmentDisplay.html#prepareGroupsForDisplay(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">prepareGroupsForDisplay(AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/StructureAlignmentDisplay.html" title="class in org.biojava.bio.structure.align.gui">StructureAlignmentDisplay</a></dt>
<dd>
<div class="block">Rotate the Atoms/Groups so they are aligned for the 3D visualisation</div>
</dd>
<dt><a href="./org/biojava/bio/structure/server/PrepareIndexFile.html" title="class in org.biojava.bio.structure.server"><span class="strong">PrepareIndexFile</span></a> - Class in <a href="./org/biojava/bio/structure/server/package-summary.html">org.biojava.bio.structure.server</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/PrepareIndexFile.html#PrepareIndexFile()">PrepareIndexFile()</a></span> - Constructor for class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/PrepareIndexFile.html" title="class in org.biojava.bio.structure.server">PrepareIndexFile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/PrepareIndexFile.html#prepareIndexFileForInstallation(org.biojava.bio.structure.server.FlatFileInstallation)">prepareIndexFileForInstallation(FlatFileInstallation)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/PrepareIndexFile.html" title="class in org.biojava.bio.structure.server">PrepareIndexFile</a></dt>
<dd>
<div class="block">prepare the index file for this installation</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/PrepareMMcifIndexFile.html#prepareIndexFileForInstallation(org.biojava.bio.structure.server.FlatFileInstallation)">prepareIndexFileForInstallation(FlatFileInstallation)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/PrepareMMcifIndexFile.html" title="class in org.biojava.bio.structure.server">PrepareMMcifIndexFile</a></dt>
<dd>
<div class="block">prepare the index file for this installation</div>
</dd>
<dt><a href="./org/biojava/bio/structure/server/PrepareMMcifIndexFile.html" title="class in org.biojava.bio.structure.server"><span class="strong">PrepareMMcifIndexFile</span></a> - Class in <a href="./org/biojava/bio/structure/server/package-summary.html">org.biojava.bio.structure.server</a></dt>
<dd>
<div class="block">A class to index all files in a PDB installation</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/PrepareMMcifIndexFile.html#PrepareMMcifIndexFile()">PrepareMMcifIndexFile()</a></span> - Constructor for class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/PrepareMMcifIndexFile.html" title="class in org.biojava.bio.structure.server">PrepareMMcifIndexFile</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/util/PrettyXMLWriter.html" title="class in org.biojava3.core.util"><span class="strong">PrettyXMLWriter</span></a> - Class in <a href="./org/biojava3/core/util/package-summary.html">org.biojava3.core.util</a></dt>
<dd>
<div class="block">Implementation of XMLWriter which emits nicely formatted documents
 to a PrintWriter.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/PrettyXMLWriter.html#PrettyXMLWriter(java.io.PrintWriter)">PrettyXMLWriter(PrintWriter)</a></span> - Constructor for class org.biojava3.core.util.<a href="./org/biojava3/core/util/PrettyXMLWriter.html" title="class in org.biojava3.core.util">PrettyXMLWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/Hashcoder.html#PRIME">PRIME</a></span> - Static variable in class org.biojava3.core.util.<a href="./org/biojava3/core/util/Hashcoder.html" title="class in org.biojava3.core.util">Hashcoder</a></dt>
<dd>
<div class="block">The prime number used to multiply any calculated hashcode seed by

 i.e. result = PRIME*result + c

 Where result is the result of the previous calculation (at first this will
 be seed) and c is the calculated int to add to result</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/JPrintPanel.html#print(java.awt.Graphics, java.awt.print.PageFormat, int)">print(Graphics, PageFormat, int)</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/JPrintPanel.html" title="class in org.biojava.bio.structure.align.gui">JPrintPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/MenuCreator.html#PRINT">PRINT</a></span> - Static variable in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/MenuCreator.html" title="class in org.biojava.bio.structure.align.gui">MenuCreator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#print(int, int)">print(int, int)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Print the matrix to stdout.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#print(java.io.PrintWriter, int, int)">print(PrintWriter, int, int)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Print the matrix to the output stream.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#print(java.text.NumberFormat, int)">print(NumberFormat, int)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Print the matrix to stdout.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#print(java.io.PrintWriter, java.text.NumberFormat, int)">print(PrintWriter, NumberFormat, int)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Print the matrix to the output stream.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/PrettyXMLWriter.html#print(java.lang.String)">print(String)</a></span> - Method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/PrettyXMLWriter.html" title="class in org.biojava3.core.util">PrettyXMLWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/XMLWriter.html#print(java.lang.String)">print(String)</a></span> - Method in interface org.biojava3.core.util.<a href="./org/biojava3/core/util/XMLWriter.html" title="interface in org.biojava3.core.util">XMLWriter</a></dt>
<dd>
<div class="block">Prints some textual content in an element.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/PrettyXMLWriter.html#printAttributeValue(java.lang.String)">printAttributeValue(String)</a></span> - Method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/PrettyXMLWriter.html" title="class in org.biojava3.core.util">PrettyXMLWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/PrettyXMLWriter.html#printChars(java.lang.String)">printChars(String)</a></span> - Method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/PrettyXMLWriter.html" title="class in org.biojava3.core.util">PrettyXMLWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/DemoSCOP.html#printDomainsForPDB()">printDomainsForPDB()</a></span> - Method in class demo.<a href="./demo/DemoSCOP.html" title="class in demo">DemoSCOP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/AbstractStructureAlignment.html#printHelp()">printHelp()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/AbstractStructureAlignment.html" title="class in org.biojava.bio.structure.align">AbstractStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/BioJavaStructureAlignment.html#printHelp()">printHelp()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/BioJavaStructureAlignment.html" title="class in org.biojava.bio.structure.align">BioJavaStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/FarmJob.html#printHelp()">printHelp()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/FarmJob.html" title="class in org.biojava.bio.structure.align">FarmJob</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCat.html#printHelp()">printHelp()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCat.html" title="class in org.biojava.bio.structure.align.fatcat">FatCat</a></dt>
<dd>
<div class="block">print the -h help text.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructureAlignment.html#printHelp()">printHelp()</a></span> - Method in interface org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructureAlignment.html" title="interface in org.biojava.bio.structure.align">StructureAlignment</a></dt>
<dd>
<div class="block">Returns some documentation on the command line arguments for this algorithm.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/PrettyXMLWriter.html#println(java.lang.String)">println(String)</a></span> - Method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/PrettyXMLWriter.html" title="class in org.biojava3.core.util">PrettyXMLWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/XMLWriter.html#println(java.lang.String)">println(String)</a></span> - Method in interface org.biojava3.core.util.<a href="./org/biojava3/core/util/XMLWriter.html" title="interface in org.biojava3.core.util">XMLWriter</a></dt>
<dd>
<div class="block">Prints some textual content, terminated with a newline character.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SVDSuperimposer.html#printMatrix(org.biojava.bio.structure.jama.Matrix)">printMatrix(Matrix)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SVDSuperimposer.html" title="class in org.biojava.bio.structure">SVDSuperimposer</a></dt>
<dd>
<div class="block">Simple debug method to print a Matrix object on System.out.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/PrettyXMLWriter.html#printRaw(java.lang.String)">printRaw(String)</a></span> - Method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/PrettyXMLWriter.html" title="class in org.biojava3.core.util">PrettyXMLWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/XMLWriter.html#printRaw(java.lang.String)">printRaw(String)</a></span> - Method in interface org.biojava3.core.util.<a href="./org/biojava3/core/util/XMLWriter.html" title="interface in org.biojava3.core.util">XMLWriter</a></dt>
<dd>
<div class="block">Send raw data to the stream.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastService.html#printRemoteBlastInfo()">printRemoteBlastInfo()</a></span> - Method in class org.biojava3.ws.alignment.qblast.<a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastService.html" title="class in org.biojava3.ws.alignment.qblast">NCBIQBlastService</a></dt>
<dd>
<div class="block">A simple method to check the availability of the QBlast service</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatAligner.html#printTimeStamps">printTimeStamps</a></span> - Static variable in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatAligner.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/AFPChainXMLConverter.html#printXMLEQRInferPositions(org.biojava3.core.util.PrettyXMLWriter, org.biojava.bio.structure.align.model.AFPChain, int, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">printXMLEQRInferPositions(PrettyXMLWriter, AFPChain, int, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/AFPChainXMLConverter.html" title="class in org.biojava.bio.structure.align.xml">AFPChainXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/AFPChainXMLConverter.html#printXMLHeader(org.biojava3.core.util.PrettyXMLWriter, org.biojava.bio.structure.align.model.AFPChain)">printXMLHeader(PrettyXMLWriter, AFPChain)</a></span> - Static method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/AFPChainXMLConverter.html" title="class in org.biojava.bio.structure.align.xml">AFPChainXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#Pro">Pro</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html#process(java.lang.String[])">process(String[])</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/AbstractUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.ce">AbstractUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/UserArgumentProcessor.html#process(java.lang.String[])">process(String[])</a></span> - Method in interface org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/UserArgumentProcessor.html" title="interface in org.biojava.bio.structure.align.ce">UserArgumentProcessor</a></dt>
<dd>
<div class="block">Process user arguments that have been provided from the command line</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/FastaGeneWriter.html#process()">process()</a></span> - Method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/FastaGeneWriter.html" title="class in org.biojava3.core.sequence.io">FastaGeneWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/FastaReader.html#process()">process()</a></span> - Method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/FastaReader.html" title="class in org.biojava3.core.sequence.io">FastaReader</a></dt>
<dd>
<div class="block">The parsing is done in this method</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/FastaWriter.html#process()">process()</a></span> - Method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/FastaWriter.html" title="class in org.biojava3.core.sequence.io">FastaWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/util/IOUtils.ReaderProcessor.html#process(java.lang.String)">process(String)</a></span> - Method in interface org.biojava3.core.sequence.io.util.<a href="./org/biojava3/core/sequence/io/util/IOUtils.ReaderProcessor.html" title="interface in org.biojava3.core.sequence.io.util">IOUtils.ReaderProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/homology/GFF3FromUniprotBlastHits.html#process(java.io.File, double, java.util.LinkedHashMap, java.io.OutputStream)">process(File, double, LinkedHashMap&lt;String, GeneSequence&gt;, OutputStream)</a></span> - Method in class org.biojava3.genome.homology.<a href="./org/biojava3/genome/homology/GFF3FromUniprotBlastHits.html" title="class in org.biojava3.genome.homology">GFF3FromUniprotBlastHits</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/homology/GFF3FromUniprotBlastHits.html#process(java.util.LinkedHashMap, java.util.LinkedHashMap, java.io.OutputStream)">process(LinkedHashMap&lt;String, ArrayList&lt;String&gt;&gt;, LinkedHashMap&lt;String, GeneSequence&gt;, OutputStream)</a></span> - Method in class org.biojava3.genome.homology.<a href="./org/biojava3/genome/homology/GFF3FromUniprotBlastHits.html" title="class in org.biojava3.genome.homology">GFF3FromUniprotBlastHits</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/query/OutputHitsGFF.html#process(java.io.File, java.io.File, java.io.File, double, double, boolean, boolean)">process(File, File, File, double, double, boolean, boolean)</a></span> - Method in class org.biojava3.genome.query.<a href="./org/biojava3/genome/query/OutputHitsGFF.html" title="class in org.biojava3.genome.query">OutputHitsGFF</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/uniprot/UniprotToFasta.html#process(java.lang.String, java.lang.String)">process(String, String)</a></span> - Method in class org.biojava3.genome.uniprot.<a href="./org/biojava3/genome/uniprot/UniprotToFasta.html" title="class in org.biojava3.genome.uniprot">UniprotToFasta</a></dt>
<dd>
<div class="block">Convert a Uniprot sequence file to a fasta file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/CheckTreeAccuracy.html#process(org.forester.phylogeny.Phylogeny, org.forester.phylogenyinference.DistanceMatrix)">process(Phylogeny, DistanceMatrix)</a></span> - Method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/CheckTreeAccuracy.html" title="class in org.biojava3.phylo">CheckTreeAccuracy</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/TreeConstructor.html#process()">process()</a></span> - Method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/TreeConstructor.html" title="class in org.biojava3.phylo">TreeConstructor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/util/SplitFasta.html#processNucleotides(java.io.File, java.lang.String, java.io.File)">processNucleotides(File, String, File)</a></span> - Method in class org.biojava3.genome.util.<a href="./org/biojava3/genome/util/SplitFasta.html" title="class in org.biojava3.genome.util">SplitFasta</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/util/IOUtils.html#processReader(java.io.BufferedReader, org.biojava3.core.sequence.io.util.IOUtils.ReaderProcessor)">processReader(BufferedReader, IOUtils.ReaderProcessor)</a></span> - Static method in class org.biojava3.core.sequence.io.util.<a href="./org/biojava3/core/sequence/io/util/IOUtils.html" title="class in org.biojava3.core.sequence.io.util">IOUtils</a></dt>
<dd>
<div class="block">Takes in a reader and a processor, reads every line from the given
 file and then invokes the processor.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#processUnknownCompound(C, int)">processUnknownCompound(C, int)</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava3.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>
<div class="block">Since bit encoding only supports a finite number of bases
 it is more than likely when processing sequence you will encounter a
 compound which is not covered by the encoding e.g.</div>
</dd>
<dt><a href="./org/biojava3/aaproperties/profeat/ProfeatProperties.html" title="class in org.biojava3.aaproperties.profeat"><span class="strong">ProfeatProperties</span></a> - Class in <a href="./org/biojava3/aaproperties/profeat/package-summary.html">org.biojava3.aaproperties.profeat</a></dt>
<dd>
<div class="block">This is an adaptor class which enable the ease of generating profeat properties.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/ProfeatProperties.html#ProfeatProperties()">ProfeatProperties()</a></span> - Constructor for class org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/ProfeatProperties.html" title="class in org.biojava3.aaproperties.profeat">ProfeatProperties</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/aaproperties/profeat/ProfeatPropertiesImpl.html" title="class in org.biojava3.aaproperties.profeat"><span class="strong">ProfeatPropertiesImpl</span></a> - Class in <a href="./org/biojava3/aaproperties/profeat/package-summary.html">org.biojava3.aaproperties.profeat</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/ProfeatPropertiesImpl.html#ProfeatPropertiesImpl()">ProfeatPropertiesImpl()</a></span> - Constructor for class org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/ProfeatPropertiesImpl.html" title="class in org.biojava3.aaproperties.profeat">ProfeatPropertiesImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#profile">profile</a></span> - Variable in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava3.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template"><span class="strong">Profile</span></a>&lt;<a href="./org/biojava3/alignment/template/Profile.html" title="type parameter in Profile">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava3/alignment/template/Profile.html" title="type parameter in Profile">C</a>&gt;,<a href="./org/biojava3/alignment/template/Profile.html" title="type parameter in Profile">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava3/alignment/template/package-summary.html">org.biojava3.alignment.template</a></dt>
<dd>
<div class="block">Defines a data structure for the results of sequence alignment.</div>
</dd>
<dt><a href="./org/biojava3/alignment/template/Profile.StringFormat.html" title="enum in org.biojava3.alignment.template"><span class="strong">Profile.StringFormat</span></a> - Enum in <a href="./org/biojava3/alignment/template/package-summary.html">org.biojava3.alignment.template</a></dt>
<dd>
<div class="block">List of output formats.</div>
</dd>
<dt><a href="./org/biojava3/alignment/template/ProfilePair.html" title="interface in org.biojava3.alignment.template"><span class="strong">ProfilePair</span></a>&lt;<a href="./org/biojava3/alignment/template/ProfilePair.html" title="type parameter in ProfilePair">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava3/alignment/template/ProfilePair.html" title="type parameter in ProfilePair">C</a>&gt;,<a href="./org/biojava3/alignment/template/ProfilePair.html" title="type parameter in ProfilePair">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava3/alignment/template/package-summary.html">org.biojava3.alignment.template</a></dt>
<dd>
<div class="block">Defines a data structure for the results of the alignment of a pair of <a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template"><code>Profile</code></a>s.</div>
</dd>
<dt><a href="./org/biojava3/alignment/template/ProfileProfileAligner.html" title="interface in org.biojava3.alignment.template"><span class="strong">ProfileProfileAligner</span></a>&lt;<a href="./org/biojava3/alignment/template/ProfileProfileAligner.html" title="type parameter in ProfileProfileAligner">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava3/alignment/template/ProfileProfileAligner.html" title="type parameter in ProfileProfileAligner">C</a>&gt;,<a href="./org/biojava3/alignment/template/ProfileProfileAligner.html" title="type parameter in ProfileProfileAligner">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava3/alignment/template/package-summary.html">org.biojava3.alignment.template</a></dt>
<dd>
<div class="block">Defines an <a href="./org/biojava3/alignment/template/Aligner.html" title="interface in org.biojava3.alignment.template"><code>Aligner</code></a> for a pair of <a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template"><code>Profile</code></a>s.</div>
</dd>
<dt><a href="./org/biojava3/alignment/template/ProfileProfileScorer.html" title="interface in org.biojava3.alignment.template"><span class="strong">ProfileProfileScorer</span></a>&lt;<a href="./org/biojava3/alignment/template/ProfileProfileScorer.html" title="type parameter in ProfileProfileScorer">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava3/alignment/template/ProfileProfileScorer.html" title="type parameter in ProfileProfileScorer">C</a>&gt;,<a href="./org/biojava3/alignment/template/ProfileProfileScorer.html" title="type parameter in ProfileProfileScorer">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava3/alignment/template/package-summary.html">org.biojava3.alignment.template</a></dt>
<dd>
<div class="block">Defines an algorithm which computes a score for a pairing of alignment profiles.</div>
</dd>
<dt><a href="./org/biojava3/alignment/template/ProfileView.html" title="interface in org.biojava3.alignment.template"><span class="strong">ProfileView</span></a>&lt;<a href="./org/biojava3/alignment/template/ProfileView.html" title="type parameter in ProfileView">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava3/alignment/template/ProfileView.html" title="type parameter in ProfileView">C</a>&gt;,<a href="./org/biojava3/alignment/template/ProfileView.html" title="type parameter in ProfileView">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava3/alignment/template/package-summary.html">org.biojava3.alignment.template</a></dt>
<dd>
<div class="block">Defines a data structure for a view of sequence alignment.</div>
</dd>
<dt><a href="./org/biojava3/phylo/ProgessListenerStub.html" title="class in org.biojava3.phylo"><span class="strong">ProgessListenerStub</span></a> - Class in <a href="./org/biojava3/phylo/package-summary.html">org.biojava3.phylo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ProgessListenerStub.html#ProgessListenerStub()">ProgessListenerStub()</a></span> - Constructor for class org.biojava3.phylo.<a href="./org/biojava3/phylo/ProgessListenerStub.html" title="class in org.biojava3.phylo">ProgessListenerStub</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/NJTreeProgressListener.html#progress(java.lang.Object, java.lang.String, int)">progress(Object, String, int)</a></span> - Method in interface org.biojava3.phylo.<a href="./org/biojava3/phylo/NJTreeProgressListener.html" title="interface in org.biojava3.phylo">NJTreeProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/NJTreeProgressListener.html#progress(java.lang.Object, java.lang.String, int, int)">progress(Object, String, int, int)</a></span> - Method in interface org.biojava3.phylo.<a href="./org/biojava3/phylo/NJTreeProgressListener.html" title="interface in org.biojava3.phylo">NJTreeProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ProgessListenerStub.html#progress(java.lang.Object, java.lang.String, int)">progress(Object, String, int)</a></span> - Method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ProgessListenerStub.html" title="class in org.biojava3.phylo">ProgessListenerStub</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ProgessListenerStub.html#progress(java.lang.Object, java.lang.String, int, int)">progress(Object, String, int, int)</a></span> - Method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ProgessListenerStub.html" title="class in org.biojava3.phylo">ProgessListenerStub</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#proline">proline</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#propHash">propHash</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/chem/PolymerType.html#PROTEIN_ONLY">PROTEIN_ONLY</a></span> - Static variable in enum org.biojava.bio.structure.io.mmcif.chem.<a href="./org/biojava/bio/structure/io/mmcif/chem/PolymerType.html" title="enum in org.biojava.bio.structure.io.mmcif.chem">PolymerType</a></dt>
<dd>
<div class="block">Convenience <tt>Set</tt> of polymer types classified as protein.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.Builder.html#proteinCreator(org.biojava3.core.sequence.io.template.SequenceCreatorInterface)">proteinCreator(SequenceCreatorInterface&lt;AminoAcidCompound&gt;)</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava3.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/domain/ProteinDomainParser.html" title="class in org.biojava.bio.structure.domain"><span class="strong">ProteinDomainParser</span></a> - Class in <a href="./org/biojava/bio/structure/domain/package-summary.html">org.biojava.bio.structure.domain</a></dt>
<dd>
<div class="block">Protein Domain Parser is a an algorithm that attempts at assigning domains for 3D protein structures.</div>
</dd>
<dt><a href="./org/biojava3/protmod/ProteinModification.html" title="interface in org.biojava3.protmod"><span class="strong">ProteinModification</span></a> - Interface in <a href="./org/biojava3/protmod/package-summary.html">org.biojava3.protmod</a></dt>
<dd>
<div class="block">This interface defines information about a specific protein
 modification.</div>
</dd>
<dt><a href="./org/biojava3/protmod/structure/ProteinModificationIdentifier.html" title="class in org.biojava3.protmod.structure"><span class="strong">ProteinModificationIdentifier</span></a> - Class in <a href="./org/biojava3/protmod/structure/package-summary.html">org.biojava3.protmod.structure</a></dt>
<dd>
<div class="block">Identify attachment modification in a 3-D structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ProteinModificationIdentifier.html#ProteinModificationIdentifier()">ProteinModificationIdentifier()</a></span> - Constructor for class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ProteinModificationIdentifier.html" title="class in org.biojava3.protmod.structure">ProteinModificationIdentifier</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/protmod/ProteinModificationImpl.html" title="class in org.biojava3.protmod"><span class="strong">ProteinModificationImpl</span></a> - Class in <a href="./org/biojava3/protmod/package-summary.html">org.biojava3.protmod</a></dt>
<dd>
<div class="block">This class contains information about a specific protein
 modification.</div>
</dd>
<dt><a href="./org/biojava3/protmod/ProteinModificationImpl.Builder.html" title="class in org.biojava3.protmod"><span class="strong">ProteinModificationImpl.Builder</span></a> - Class in <a href="./org/biojava3/protmod/package-summary.html">org.biojava3.protmod</a></dt>
<dd>
<div class="block">Uses Builder pattern to build a ProteinModification.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationImpl.Builder.html#ProteinModificationImpl.Builder(java.lang.String, org.biojava3.protmod.ModificationCategory, org.biojava3.protmod.ModificationOccurrenceType, org.biojava3.protmod.ModificationCondition)">ProteinModificationImpl.Builder(String, ModificationCategory, ModificationOccurrenceType, ModificationCondition)</a></span> - Constructor for class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationImpl.Builder.html" title="class in org.biojava3.protmod">ProteinModificationImpl.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationImpl.Builder.html#ProteinModificationImpl.Builder(org.biojava3.protmod.ProteinModification)">ProteinModificationImpl.Builder(ProteinModification)</a></span> - Constructor for class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationImpl.Builder.html" title="class in org.biojava3.protmod">ProteinModificationImpl.Builder</a></dt>
<dd>
<div class="block">Create a Builder from an existing ProteinModification.</div>
</dd>
<dt><a href="./org/biojava3/protmod/ProteinModificationRegistry.html" title="class in org.biojava3.protmod"><span class="strong">ProteinModificationRegistry</span></a> - Class in <a href="./org/biojava3/protmod/package-summary.html">org.biojava3.protmod</a></dt>
<dd>
<div class="block">This class serves as a instance registry by maintaining 
 a pool of ProteinModification instances.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationRegistry.html#ProteinModificationRegistry()">ProteinModificationRegistry()</a></span> - Constructor for class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationRegistry.html" title="class in org.biojava3.protmod">ProteinModificationRegistry</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/protmod/io/ProteinModificationXmlReader.html" title="class in org.biojava3.protmod.io"><span class="strong">ProteinModificationXmlReader</span></a> - Class in <a href="./org/biojava3/protmod/io/package-summary.html">org.biojava3.protmod.io</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/ProteinSequence.html" title="class in org.biojava3.core.sequence"><span class="strong">ProteinSequence</span></a> - Class in <a href="./org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a></dt>
<dd>
<div class="block">The representation of a ProteinSequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/ProteinSequence.html#ProteinSequence(java.lang.String)">ProteinSequence(String)</a></span> - Constructor for class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/ProteinSequence.html" title="class in org.biojava3.core.sequence">ProteinSequence</a></dt>
<dd>
<div class="block">Create a protein from a string</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/ProteinSequence.html#ProteinSequence(java.lang.String, org.biojava3.core.sequence.template.CompoundSet)">ProteinSequence(String, CompoundSet&lt;AminoAcidCompound&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/ProteinSequence.html" title="class in org.biojava3.core.sequence">ProteinSequence</a></dt>
<dd>
<div class="block">Create a protein from a string with a user defined set of amino acids</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/ProteinSequence.html#ProteinSequence(org.biojava3.core.sequence.template.ProxySequenceReader)">ProteinSequence(ProxySequenceReader&lt;AminoAcidCompound&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/ProteinSequence.html" title="class in org.biojava3.core.sequence">ProteinSequence</a></dt>
<dd>
<div class="block">A protein sequence where the storage of the sequence is somewhere else.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/ProteinSequence.html#ProteinSequence(org.biojava3.core.sequence.template.ProxySequenceReader, org.biojava3.core.sequence.template.CompoundSet)">ProteinSequence(ProxySequenceReader&lt;AminoAcidCompound&gt;, CompoundSet&lt;AminoAcidCompound&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/ProteinSequence.html" title="class in org.biojava3.core.sequence">ProteinSequence</a></dt>
<dd>
<div class="block">A protein sequence where the storage of the sequence is somewhere else with user defined
 set of amino acids.</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/io/ProteinSequenceCreator.html" title="class in org.biojava3.core.sequence.io"><span class="strong">ProteinSequenceCreator</span></a> - Class in <a href="./org/biojava3/core/sequence/io/package-summary.html">org.biojava3.core.sequence.io</a></dt>
<dd>
<div class="block">Used to create a ProteinSequence from a String to allow for details
 about the location of the sequence etc.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/ProteinSequenceCreator.html#ProteinSequenceCreator(org.biojava3.core.sequence.template.CompoundSet)">ProteinSequenceCreator(CompoundSet&lt;AminoAcidCompound&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/ProteinSequenceCreator.html" title="class in org.biojava3.core.sequence.io">ProteinSequenceCreator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/template/ProxySequenceReader.html" title="interface in org.biojava3.core.sequence.template"><span class="strong">ProxySequenceReader</span></a>&lt;<a href="./org/biojava3/core/sequence/template/ProxySequenceReader.html" title="type parameter in ProxySequenceReader">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava3/core/sequence/template/package-summary.html">org.biojava3.core.sequence.template</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html#put(java.lang.Double, java.awt.Color)">put(Double, Color)</a></span> - Method in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html" title="class in org.biojava.bio.structure.gui.util.color">GradientMapper</a></dt>
<dd>
<div class="block">Adds a gradient endpoint at the specified position.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/SoftHashMap.html#put(K, V)">put(K, V)</a></span> - Method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/SoftHashMap.html" title="class in org.biojava3.core.util">SoftHashMap</a></dt>
<dd>
<div class="block">Here we put the key, value pair into the HashMap using a SoftValue

 object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html#putAll(java.util.Map)">putAll(Map&lt;? extends Double, ? extends Color&gt;)</a></span> - Method in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html" title="class in org.biojava.bio.structure.gui.util.color">GradientMapper</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_Q_">
<!--   -->
</a>
<h2 class="title">Q</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStruc.html#Q">Q</a></span> - Static variable in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStruc.html" title="class in org.biojava.bio.structure.secstruc">SecStruc</a></dt>
<dd>
<div class="block">constant for electrostatic energy
 
             f  *   q1 *   q2 * scale
 Q = -332 * 0.42 * 0.20 * 1000.0


 q1 and q2 are partial charges which are placed on the C,O
 (+q1,-q1) and N,H (-q2,+q2)</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/Constraints.html#Q">Q</a></span> - Static variable in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/Constraints.html" title="class in org.biojava3.aaproperties">Constraints</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#qr()">qr()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">QR Decomposition</div>
</dd>
<dt><a href="./org/biojava/bio/structure/jama/QRDecomposition.html" title="class in org.biojava.bio.structure.jama"><span class="strong">QRDecomposition</span></a> - Class in <a href="./org/biojava/bio/structure/jama/package-summary.html">org.biojava.bio.structure.jama</a></dt>
<dd>
<div class="block">QR Decomposition.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/QRDecomposition.html#QRDecomposition(org.biojava.bio.structure.jama.Matrix)">QRDecomposition(Matrix)</a></span> - Constructor for class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/QRDecomposition.html" title="class in org.biojava.bio.structure.jama">QRDecomposition</a></dt>
<dd>
<div class="block">QR Decomposition, computed by Householder reflections. provides
   a data structure to access R and the Householder vectors and
   compute Q.</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/features/QuantityFeature.html" title="class in org.biojava3.core.sequence.features"><span class="strong">QuantityFeature</span></a>&lt;<a href="./org/biojava3/core/sequence/features/QuantityFeature.html" title="type parameter in QuantityFeature">S</a> extends <a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a>&lt;<a href="./org/biojava3/core/sequence/features/QuantityFeature.html" title="type parameter in QuantityFeature">C</a>&gt;,<a href="./org/biojava3/core/sequence/features/QuantityFeature.html" title="type parameter in QuantityFeature">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/core/sequence/features/package-summary.html">org.biojava3.core.sequence.features</a></dt>
<dd>
<div class="block">It is common to have a numerical value or values associated with a feature.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/QuantityFeature.html#QuantityFeature(java.lang.String, java.lang.String)">QuantityFeature(String, String)</a></span> - Constructor for class org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/QuantityFeature.html" title="class in org.biojava3.core.sequence.features">QuantityFeature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/SystemInfo.html#quickSort(java.lang.String[], int, int)">quickSort(String[], int, int)</a></span> - Static method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/SystemInfo.html" title="class in org.biojava.bio.structure.align.gui">SystemInfo</a></dt>
<dd>
<div class="block">Stripped-down QuickSort.</div>
</dd>
</dl>
<a name="_R_">
<!--   -->
</a>
<h2 class="title">R</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/Constraints.html#R">R</a></span> - Static variable in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/Constraints.html" title="class in org.biojava3.aaproperties">Constraints</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#radiansPerDegree">radiansPerDegree</a></span> - Static variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">Radians per degree.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html#RAINBOW_GRADIENT">RAINBOW_GRADIENT</a></span> - Static variable in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html" title="class in org.biojava.bio.structure.gui.util.color">GradientMapper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html#RAINBOW_INTENSITY_GRADIENT">RAINBOW_INTENSITY_GRADIENT</a></span> - Static variable in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html" title="class in org.biojava.bio.structure.gui.util.color">GradientMapper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#random(int, int)">random(int, int)</a></span> - Static method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Generate matrix with random elements</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/CollectionTools.html#randomPick(java.util.Collection)">randomPick(Collection&lt;Number&gt;)</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/CollectionTools.html" title="class in org.biojava.bio.structure.align.util">CollectionTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html#rangeSelected(org.biojava.bio.structure.gui.util.AlignedPosition, org.biojava.bio.structure.gui.util.AlignedPosition)">rangeSelected(AlignedPosition, AlignedPosition)</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AligPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/StatusDisplay.html#rangeSelected(org.biojava.bio.structure.gui.util.AlignedPosition, org.biojava.bio.structure.gui.util.AlignedPosition)">rangeSelected(AlignedPosition, AlignedPosition)</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/StatusDisplay.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">StatusDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/events/AlignmentPositionListener.html#rangeSelected(org.biojava.bio.structure.gui.util.AlignedPosition, org.biojava.bio.structure.gui.util.AlignedPosition)">rangeSelected(AlignedPosition, AlignedPosition)</a></span> - Method in interface org.biojava.bio.structure.gui.events.<a href="./org/biojava/bio/structure/gui/events/AlignmentPositionListener.html" title="interface in org.biojava.bio.structure.gui.events">AlignmentPositionListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/events/JmolAlignedPositionListener.html#rangeSelected(org.biojava.bio.structure.gui.util.AlignedPosition, org.biojava.bio.structure.gui.util.AlignedPosition)">rangeSelected(AlignedPosition, AlignedPosition)</a></span> - Method in class org.biojava.bio.structure.gui.events.<a href="./org/biojava/bio/structure/gui/events/JmolAlignedPositionListener.html" title="class in org.biojava.bio.structure.gui.events">JmolAlignedPositionListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#rank()">rank()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Matrix rank</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/SingularValueDecomposition.html#rank()">rank()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/SingularValueDecomposition.html" title="class in org.biojava.bio.structure.jama">SingularValueDecomposition</a></dt>
<dd>
<div class="block">Effective numerical matrix rank</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/gui/jmol/RasmolCommandListener.html" title="class in org.biojava.bio.structure.align.gui.jmol"><span class="strong">RasmolCommandListener</span></a> - Class in <a href="./org/biojava/bio/structure/align/gui/jmol/package-summary.html">org.biojava.bio.structure.align.gui.jmol</a></dt>
<dd>
<div class="block">a utility class that listens to Ramsol script commands in the @link <a href="./org/biojava/bio/structure/gui/BiojavaJmol.html" title="class in org.biojava.bio.structure.gui"><code>BiojavaJmol</code></a> class</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/RasmolCommandListener.html#RasmolCommandListener(org.biojava.bio.structure.align.gui.jmol.JmolPanel, javax.swing.JTextField)">RasmolCommandListener(JmolPanel, JTextField)</a></span> - Constructor for class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/RasmolCommandListener.html" title="class in org.biojava.bio.structure.align.gui.jmol">RasmolCommandListener</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/gui/RasmolCommandListener.html" title="class in org.biojava.bio.structure.gui"><span class="strong">RasmolCommandListener</span></a> - Class in <a href="./org/biojava/bio/structure/gui/package-summary.html">org.biojava.bio.structure.gui</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>use  org.biojava.bio.structure.align.gui.jmol.RasmolCommandListener instead</i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/RasmolCommandListener.html#RasmolCommandListener(org.biojava.bio.structure.align.gui.jmol.JmolPanel, javax.swing.JTextField)">RasmolCommandListener(JmolPanel, JTextField)</a></span> - Constructor for class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/RasmolCommandListener.html" title="class in org.biojava.bio.structure.gui">RasmolCommandListener</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#read(java.io.BufferedReader)">read(BufferedReader)</a></span> - Static method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Read a matrix from a stream.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/BufferedReaderBytesRead.html#read()">read()</a></span> - Method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/BufferedReaderBytesRead.html" title="class in org.biojava3.core.sequence.io">BufferedReaderBytesRead</a></dt>
<dd>
<div class="block">Reads a single character.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/BufferedReaderBytesRead.html#read(char[], int, int)">read(char[], int, int)</a></span> - Method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/BufferedReaderBytesRead.html" title="class in org.biojava3.core.sequence.io">BufferedReaderBytesRead</a></dt>
<dd>
<div class="block">Reads characters into a portion of an array.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/UncompressInputStream.html#read()">read()</a></span> - Method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/UncompressInputStream.html" title="class in org.biojava3.core.util">UncompressInputStream</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/UncompressInputStream.html#read(byte[], int, int)">read(byte[], int, int)</a></span> - Method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/UncompressInputStream.html" title="class in org.biojava3.core.util">UncompressInputStream</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/GeneIDGFF2Reader.html#read(java.lang.String)">read(String)</a></span> - Static method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/GeneIDGFF2Reader.html" title="class in org.biojava3.genome.parsers.gff">GeneIDGFF2Reader</a></dt>
<dd>
<div class="block">Read a file into a FeatureList.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/GeneMarkGTFReader.html#read(java.lang.String)">read(String)</a></span> - Static method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/GeneMarkGTFReader.html" title="class in org.biojava3.genome.parsers.gff">GeneMarkGTFReader</a></dt>
<dd>
<div class="block">Read a file into a FeatureList.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/GFF3Reader.html#read(java.lang.String)">read(String)</a></span> - Static method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/GFF3Reader.html" title="class in org.biojava3.genome.parsers.gff">GFF3Reader</a></dt>
<dd>
<div class="block">Read a file into a FeatureList.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/data/sequence/SequenceUtil.html#readFasta(java.io.InputStream)">readFasta(InputStream)</a></span> - Static method in class org.biojava3.data.sequence.<a href="./org/biojava3/data/sequence/SequenceUtil.html" title="class in org.biojava3.data.sequence">SequenceUtil</a></dt>
<dd>
<div class="block">Reads fasta sequences from inStream into the list of FastaSequence
 objects</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/FastaReaderHelper.html#readFastaDNASequence(java.io.File, boolean)">readFastaDNASequence(File, boolean)</a></span> - Static method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/FastaReaderHelper.html" title="class in org.biojava3.core.sequence.io">FastaReaderHelper</a></dt>
<dd>
<div class="block">Selecting lazySequenceLoad=true will parse the FASTA file and figure out the accessionid and offsets and return sequence objects
 that can in the future read the sequence from the disk.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/FastaReaderHelper.html#readFastaDNASequence(java.io.InputStream)">readFastaDNASequence(InputStream)</a></span> - Static method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/FastaReaderHelper.html" title="class in org.biojava3.core.sequence.io">FastaReaderHelper</a></dt>
<dd>
<div class="block">Read a fasta DNA sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/FastaReaderHelper.html#readFastaDNASequence(java.io.File)">readFastaDNASequence(File)</a></span> - Static method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/FastaReaderHelper.html" title="class in org.biojava3.core.sequence.io">FastaReaderHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/FastaReaderHelper.html#readFastaProteinSequence(java.io.File)">readFastaProteinSequence(File)</a></span> - Static method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/FastaReaderHelper.html" title="class in org.biojava3.core.sequence.io">FastaReaderHelper</a></dt>
<dd>
<div class="block">Read a fasta file containing amino acids with setup that would handle most
 cases.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/FastaReaderHelper.html#readFastaProteinSequence(java.io.InputStream)">readFastaProteinSequence(InputStream)</a></span> - Static method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/FastaReaderHelper.html" title="class in org.biojava3.core.sequence.io">FastaReaderHelper</a></dt>
<dd>
<div class="block">Read a fasta file containing amino acids with setup that would handle most
 cases.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/BufferedReaderBytesRead.html#readLine()">readLine()</a></span> - Method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/BufferedReaderBytesRead.html" title="class in org.biojava3.core.sequence.io">BufferedReaderBytesRead</a></dt>
<dd>
<div class="block">Reads a line of text.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/BufferedReaderBytesRead.html#ready()">ready()</a></span> - Method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/BufferedReaderBytesRead.html" title="class in org.biojava3.core.sequence.io">BufferedReaderBytesRead</a></dt>
<dd>
<div class="block">Tells whether this stream is ready to be read.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/AFPChainXMLParser.html#rebuildAFPChain(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">rebuildAFPChain(AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/AFPChainXMLParser.html" title="class in org.biojava.bio.structure.align.xml">AFPChainXMLParser</a></dt>
<dd>
<div class="block">replace the PDB res nums with atom positions:</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html#RED_BLUE_GRADIENT">RED_BLUE_GRADIENT</a></span> - Static variable in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html" title="class in org.biojava.bio.structure.gui.util.color">GradientMapper</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/ReducedChemCompProvider.html" title="class in org.biojava.bio.structure.io.mmcif"><span class="strong">ReducedChemCompProvider</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/package-summary.html">org.biojava.bio.structure.io.mmcif</a></dt>
<dd>
<div class="block">Unlike the <a href="./org/biojava/bio/structure/io/mmcif/DownloadChemCompProvider.html" title="class in org.biojava.bio.structure.io.mmcif"><code>DownloadChemCompProvider</code></a>, this  <a href="./org/biojava/bio/structure/io/mmcif/ChemCompProvider.html" title="interface in org.biojava.bio.structure.io.mmcif"><code>ChemCompProvider</code></a> does not download any chem comp definitions.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ReducedChemCompProvider.html#ReducedChemCompProvider()">ReducedChemCompProvider()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ReducedChemCompProvider.html" title="class in org.biojava.bio.structure.io.mmcif">ReducedChemCompProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentJoiner.html#reduceFragments(java.util.List, org.biojava.bio.structure.align.pairwise.FragmentPair, org.biojava.bio.structure.jama.Matrix)">reduceFragments(List&lt;FragmentPair&gt;, FragmentPair, Matrix)</a></span> - Static method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentJoiner.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentJoiner</a></dt>
<dd>
<div class="block">In helices often many similar fragments can be found.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#reduceInitialFragments()">reduceInitialFragments()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>
<div class="block">if this is set to false, the time spent to joint the initial fragments (step 2)
 is increased. - particular for large structures this increases calc. time a lot.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#refine(org.biojava.bio.structure.align.StrucAligParameters, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">refine(StrucAligParameters, Atom[], Atom[])</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>
<div class="block">Refinement procedure based on superposition and dynamic programming.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model"><span class="strong">Refine</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#Refine()">Refine()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationRegistry.html#register(org.biojava3.protmod.ProteinModification)">register(ProteinModification)</a></span> - Static method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationRegistry.html" title="class in org.biojava3.protmod">ProteinModificationRegistry</a></dt>
<dd>
<div class="block">Register a new ProteinModification.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/io/ProteinModificationXmlReader.html#registerProteinModificationFromXml(java.io.InputStream)">registerProteinModificationFromXml(InputStream)</a></span> - Static method in class org.biojava3.protmod.io.<a href="./org/biojava3/protmod/io/ProteinModificationXmlReader.html" title="class in org.biojava3.protmod.io">ProteinModificationXmlReader</a></dt>
<dd>
<div class="block">Read protein modifications from XML file and register them.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/LocIterator.html#remainder()">remainder()</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/LocIterator.html" title="class in org.biojava3.genome.parsers.gff">LocIterator</a></dt>
<dd>
<div class="block">Get portion of bounding location that has not yet been retrieved by next() method.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Site.html#remark800toPDB(java.lang.StringBuffer)">remark800toPDB(StringBuffer)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Site.html" title="class in org.biojava.bio.structure">Site</a></dt>
<dd>
<div class="block">Appends the REMARK 800 section pertaining to the site onto the end of the
 StringBuffer provided.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Site.html#remark800toPDB()">remark800toPDB()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Site.html" title="class in org.biojava.bio.structure">Site</a></dt>
<dd>
<div class="block">Provides REMARK 800 section pertaining to the site as a string.</div>
</dd>
<dt><a href="./org/biojava3/ws/alignment/RemotePairwiseAlignmentOutputProperties.html" title="interface in org.biojava3.ws.alignment"><span class="strong">RemotePairwiseAlignmentOutputProperties</span></a> - Interface in <a href="./org/biojava3/ws/alignment/package-summary.html">org.biojava3.ws.alignment</a></dt>
<dd>
<div class="block">RemotePairwiseAlignmentOutputProperties: the simplest representation of an object capable of holding 
 output formatting informations to be fed to a RemotePairwiseAlignmentService-implemented object.</div>
</dd>
<dt><a href="./org/biojava3/ws/alignment/RemotePairwiseAlignmentProperties.html" title="interface in org.biojava3.ws.alignment"><span class="strong">RemotePairwiseAlignmentProperties</span></a> - Interface in <a href="./org/biojava3/ws/alignment/package-summary.html">org.biojava3.ws.alignment</a></dt>
<dd>
<div class="block">RemotePairwiseAlignmentProperties is a interface that contains the barest of 
 methods for setting and getting Alignment properties.</div>
</dd>
<dt><a href="./org/biojava3/ws/alignment/RemotePairwiseAlignmentService.html" title="interface in org.biojava3.ws.alignment"><span class="strong">RemotePairwiseAlignmentService</span></a> - Interface in <a href="./org/biojava3/ws/alignment/package-summary.html">org.biojava3.ws.alignment</a></dt>
<dd>
<div class="block">This interface specifies minimal information needed to execute a pairwise alignment on a remote service.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop"><span class="strong">RemoteScopInstallation</span></a> - Class in <a href="./org/biojava/bio/structure/scop/package-summary.html">org.biojava.bio.structure.scop</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html#RemoteScopInstallation()">RemoteScopInstallation()</a></span> - Constructor for class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">RemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomIterator.html#remove()">remove()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomIterator.html" title="class in org.biojava.bio.structure">AtomIterator</a></dt>
<dd>
<div class="block">does nothing.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/GroupIterator.html#remove()">remove()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/GroupIterator.html" title="class in org.biojava.bio.structure">GroupIterator</a></dt>
<dd>
<div class="block">does nothing .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html#remove(java.lang.Object)">remove(Object)</a></span> - Method in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html" title="class in org.biojava.bio.structure.gui.util.color">GradientMapper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceMixin.SequenceIterator.html#remove()">remove()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceMixin.SequenceIterator.html" title="class in org.biojava3.core.sequence.template">SequenceMixin.SequenceIterator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/SoftHashMap.html#remove(java.lang.Object)">remove(Object)</a></span> - Method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/SoftHashMap.html" title="class in org.biojava3.core.util">SoftHashMap</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/LocIterator.html#remove()">remove()</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/LocIterator.html" title="class in org.biojava3.genome.parsers.gff">LocIterator</a></dt>
<dd>
<div class="block">Unsupported.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructureAlignmentFactory.html#removeAlgorithm(java.lang.String)">removeAlgorithm(String)</a></span> - Static method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructureAlignmentFactory.html" title="class in org.biojava.bio.structure.align">StructureAlignmentFactory</a></dt>
<dd>
<div class="block">Removes the specified algorithm from the list of options</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/MultipleSequenceAlignment.html#removeAlignedSequence(S)">removeAlignedSequence(S)</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/MultipleSequenceAlignment.html" title="class in org.biojava3.core.sequence">MultipleSequenceAlignment</a></dt>
<dd>
<div class="block">Remove a sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/TranscriptSequence.html#removeCDS(java.lang.String)">removeCDS(String)</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/TranscriptSequence.html" title="class in org.biojava3.core.sequence">TranscriptSequence</a></dt>
<dd>
<div class="block">Remove a CDS or coding sequence from the transcript sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/GeneSequence.html#removeExon(java.lang.String)">removeExon(String)</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/GeneSequence.html" title="class in org.biojava3.core.sequence">GeneSequence</a></dt>
<dd>
<div class="block">Remove the exon sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#removeFeature(org.biojava3.core.sequence.features.FeatureInterface)">removeFeature(FeatureInterface&lt;AbstractSequence&lt;C&gt;, C&gt;)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>
<div class="block">Remove a feature from the sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/ChromosomeSequence.html#removeGeneSequence(java.lang.String)">removeGeneSequence(String)</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/ChromosomeSequence.html" title="class in org.biojava3.core.sequence">ChromosomeSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/GeneSequence.html#removeIntron(java.lang.String)">removeIntron(String)</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/GeneSequence.html" title="class in org.biojava3.core.sequence">GeneSequence</a></dt>
<dd>
<div class="block">Remove the intron by accession</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifParser.html#removeMMcifConsumer(org.biojava.bio.structure.io.mmcif.MMcifConsumer)">removeMMcifConsumer(MMcifConsumer)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifParser.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifParser</a></dt>
<dd>
<div class="block">remove a single consumer from the parser</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#removeMMcifConsumer(org.biojava.bio.structure.io.mmcif.MMcifConsumer)">removeMMcifConsumer(MMcifConsumer)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#removeNote(java.lang.String)">removeNote(String)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/TreeConstructor.html#removeProgessListener(org.biojava3.phylo.NJTreeProgressListener)">removeProgessListener(NJTreeProgressListener)</a></span> - Method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/TreeConstructor.html" title="class in org.biojava3.phylo">TreeConstructor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/GeneSequence.html#removeTranscript(java.lang.String)">removeTranscript(String)</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/GeneSequence.html" title="class in org.biojava3.core.sequence">GeneSequence</a></dt>
<dd>
<div class="block">Remove the transcript sequence from the gene</div>
</dd>
<dt><a href="./org/biojava3/structure/gui/RenderStyle.html" title="enum in org.biojava3.structure.gui"><span class="strong">RenderStyle</span></a> - Enum in <a href="./org/biojava3/structure/gui/package-summary.html">org.biojava3.structure.gui</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/structure/gui/JmolViewerImpl.html#repaint()">repaint()</a></span> - Method in class org.biojava3.structure.gui.<a href="./org/biojava3/structure/gui/JmolViewerImpl.html" title="class in org.biojava3.structure.gui">JmolViewerImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/structure/gui/StructureViewer.html#repaint()">repaint()</a></span> - Method in interface org.biojava3.structure.gui.<a href="./org/biojava3/structure/gui/StructureViewer.html" title="interface in org.biojava3.structure.gui">StructureViewer</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/xml/RepresentativeXMLConverter.html" title="class in org.biojava.bio.structure.align.xml"><span class="strong">RepresentativeXMLConverter</span></a> - Class in <a href="./org/biojava/bio/structure/align/xml/package-summary.html">org.biojava.bio.structure.align.xml</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/RepresentativeXMLConverter.html#RepresentativeXMLConverter()">RepresentativeXMLConverter()</a></span> - Constructor for class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/RepresentativeXMLConverter.html" title="class in org.biojava.bio.structure.align.xml">RepresentativeXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/CountProgressListener.html#requestingAlignmentsFromServer(int)">requestingAlignmentsFromServer(int)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/CountProgressListener.html" title="class in org.biojava.bio.structure.align.client">CountProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/events/AlignmentProgressListener.html#requestingAlignmentsFromServer(int)">requestingAlignmentsFromServer(int)</a></span> - Method in interface org.biojava.bio.structure.align.events.<a href="./org/biojava/bio/structure/align/events/AlignmentProgressListener.html" title="interface in org.biojava.bio.structure.align.events">AlignmentProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/GUIAlignmentProgressListener.html#requestingAlignmentsFromServer(int)">requestingAlignmentsFromServer(int)</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/GUIAlignmentProgressListener.html" title="class in org.biojava.bio.structure.align.gui">GUIAlignmentProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/SimpleStructureServer.html#requestNextStructure(org.biojava.bio.structure.server.StructureListener)">requestNextStructure(StructureListener)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/SimpleStructureServer.html" title="class in org.biojava.bio.structure.server">SimpleStructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureServer.html#requestNextStructure(org.biojava.bio.structure.server.StructureListener)">requestNextStructure(StructureListener)</a></span> - Method in interface org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureServer.html" title="interface in org.biojava.bio.structure.server">StructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/webstart/WebStartMain.html#requestUserConfig()">requestUserConfig()</a></span> - Static method in class org.biojava.bio.structure.align.webstart.<a href="./org/biojava/bio/structure/align/webstart/WebStartMain.html" title="class in org.biojava.bio.structure.align.webstart">WebStartMain</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/alignment/template/RescoreRefiner.html" title="interface in org.biojava3.alignment.template"><span class="strong">RescoreRefiner</span></a>&lt;<a href="./org/biojava3/alignment/template/RescoreRefiner.html" title="type parameter in RescoreRefiner">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava3/alignment/template/RescoreRefiner.html" title="type parameter in RescoreRefiner">C</a>&gt;,<a href="./org/biojava3/alignment/template/RescoreRefiner.html" title="type parameter in RescoreRefiner">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava3/alignment/template/package-summary.html">org.biojava3.alignment.template</a></dt>
<dd>
<div class="block">Defines an algorithm which computes a new alignment <a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template"><code>Profile</code></a> by rescoring all pairs in an alignment and
 realigning.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#reset()">reset()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/ConfigStrucAligParams.html#reset()">reset()</a></span> - Method in interface org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/ConfigStrucAligParams.html" title="interface in org.biojava.bio.structure.align.ce">ConfigStrucAligParams</a></dt>
<dd>
<div class="block">Set the parameters to the default.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#reset()">reset()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWaterman3DParameters.html#reset()">reset()</a></span> - Method in class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWaterman3DParameters.html" title="class in org.biojava.bio.structure.align.seq">SmithWaterman3DParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/routines/AnchoredPairwiseSequenceAligner.html#reset()">reset()</a></span> - Method in class org.biojava3.alignment.routines.<a href="./org/biojava3/alignment/routines/AnchoredPairwiseSequenceAligner.html" title="class in org.biojava3.alignment.routines">AnchoredPairwiseSequenceAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#reset()">reset()</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava3.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html#reset()">reset()</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html" title="class in org.biojava3.alignment.template">AbstractPairwiseSequenceAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractProfileProfileAligner.html#reset()">reset()</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractProfileProfileAligner.html" title="class in org.biojava3.alignment.template">AbstractProfileProfileAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/BufferedReaderBytesRead.html#reset()">reset()</a></span> - Method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/BufferedReaderBytesRead.html" title="class in org.biojava3.core.sequence.io">BufferedReaderBytesRead</a></dt>
<dd>
<div class="block">Resets the stream to the most recent mark.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/CRC64Checksum.html#reset()">reset()</a></span> - Method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/CRC64Checksum.html" title="class in org.biojava3.core.util">CRC64Checksum</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#resetAnchors()">resetAnchors()</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava3.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html#resetDisplay()">resetDisplay()</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html" title="class in org.biojava.bio.structure.align.gui.jmol">StructureAlignmentJmol</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#residueNumber">residueNumber</a></span> - Variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/ResidueNumber.html" title="class in org.biojava.bio.structure"><span class="strong">ResidueNumber</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">Everything that is needed to uniquely describe a residue position</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ResidueNumber.html#ResidueNumber()">ResidueNumber()</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ResidueNumber.html" title="class in org.biojava.bio.structure">ResidueNumber</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ResidueNumber.html#ResidueNumber(java.lang.String, java.lang.Integer, java.lang.Character)">ResidueNumber(String, Integer, Character)</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ResidueNumber.html" title="class in org.biojava.bio.structure">ResidueNumber</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo"><span class="strong">ResidueProperties</span></a> - Class in <a href="./org/biojava3/phylo/package-summary.html">org.biojava3.phylo</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/chem/ResidueType.html" title="enum in org.biojava.bio.structure.io.mmcif.chem"><span class="strong">ResidueType</span></a> - Enum in <a href="./org/biojava/bio/structure/io/mmcif/chem/package-summary.html">org.biojava.bio.structure.io.mmcif.chem</a></dt>
<dd>
<div class="block">Enumerates the possible classifications of residues.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentJoiner.html#resizeArray(java.lang.Object, int)">resizeArray(Object, int)</a></span> - Static method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentJoiner.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentJoiner</a></dt>
<dd>
<div class="block">Reallocates an array with a new size, and copies the contents
 of the old array to the new array.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/Point.Resolver.html#resolve(T)">resolve(T)</a></span> - Method in interface org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/Point.Resolver.html" title="interface in org.biojava3.core.sequence.location.template">Point.Resolver</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/util/ResourceManager.html" title="class in org.biojava.bio.structure.align.util"><span class="strong">ResourceManager</span></a> - Class in <a href="./org/biojava/bio/structure/align/util/package-summary.html">org.biojava.bio.structure.align.util</a></dt>
<dd>
<div class="block">A class that manages the Strings that are defined in the spice.properties file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/ResourceManager.html#ResourceManager()">ResourceManager()</a></span> - Constructor for class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/ResourceManager.html" title="class in org.biojava.bio.structure.align.util">ResourceManager</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/FuzzyPoint.html#reverse(int)">reverse(int)</a></span> - Method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/FuzzyPoint.html" title="class in org.biojava3.core.sequence.location">FuzzyPoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/SimplePoint.html#reverse(int)">reverse(int)</a></span> - Method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/SimplePoint.html" title="class in org.biojava3.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/SimplePoint.html#reverse(int, int)">reverse(int, int)</a></span> - Method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/SimplePoint.html" title="class in org.biojava3.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/Point.html#reverse(int)">reverse(int)</a></span> - Method in interface org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/Point.html" title="interface in org.biojava3.core.sequence.location.template">Point</a></dt>
<dd>
<div class="block">Returns the equivalent position on the reverse strand</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/views/ReversedSequenceView.html" title="class in org.biojava3.core.sequence.views"><span class="strong">ReversedSequenceView</span></a>&lt;<a href="./org/biojava3/core/sequence/views/ReversedSequenceView.html" title="type parameter in ReversedSequenceView">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/core/sequence/views/package-summary.html">org.biojava3.core.sequence.views</a></dt>
<dd>
<div class="block">For a given sequence this class will return the base at the reversed
 position i.e. in a sequence of size 10, if you request base 2 you will get
 back the base at position 9.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/views/ReversedSequenceView.html#ReversedSequenceView(org.biojava3.core.sequence.template.Sequence)">ReversedSequenceView(Sequence&lt;C&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.views.<a href="./org/biojava3/core/sequence/views/ReversedSequenceView.html" title="class in org.biojava3.core.sequence.views">ReversedSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html#reverseSequence(org.biojava3.core.sequence.template.Sequence)">reverseSequence(Sequence&lt;C&gt;)</a></span> - Method in class org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava3.core.sequence.location.template">AbstractLocation</a></dt>
<dd>
<div class="block">Reverses and (if possible) complements the Sequence so as to represent
 the reverse strand (if one exists).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html#RG">RG</a></span> - Static variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html" title="class in org.biojava.bio.structure.domain.pdp">PDPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html#RG1">RG1</a></span> - Static variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html" title="class in org.biojava.bio.structure.domain.pdp">PDPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/AlignTools.html#rms_dk_diag(double[], double[], int, int, int, int)">rms_dk_diag(double[], double[], int, int, int, int)</a></span> - Static method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/AlignTools.html" title="class in org.biojava.bio.structure.align.helper">AlignTools</a></dt>
<dd>
<div class="block">Given distance matrix diagonals dk1, dk2, get the rmsd of a fpair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/chem/PolymerType.html#RNA_ONLY">RNA_ONLY</a></span> - Static variable in enum org.biojava.bio.structure.io.mmcif.chem.<a href="./org/biojava/bio/structure/io/mmcif/chem/PolymerType.html" title="enum in org.biojava.bio.structure.io.mmcif.chem">PolymerType</a></dt>
<dd>
<div class="block">Convenience <tt>Set</tt> of polymer types classified as RNA.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.Builder.html#rnaAminoAcidTranslator(org.biojava3.core.sequence.transcription.RNAToAminoAcidTranslator)">rnaAminoAcidTranslator(RNAToAminoAcidTranslator)</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava3.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.Builder.html#rnaCompounds(org.biojava3.core.sequence.template.CompoundSet)">rnaCompounds(CompoundSet&lt;NucleotideCompound&gt;)</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava3.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/compound/RNACompoundSet.html" title="class in org.biojava3.core.sequence.compound"><span class="strong">RNACompoundSet</span></a> - Class in <a href="./org/biojava3/core/sequence/compound/package-summary.html">org.biojava3.core.sequence.compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/RNACompoundSet.html#RNACompoundSet()">RNACompoundSet()</a></span> - Constructor for class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/RNACompoundSet.html" title="class in org.biojava3.core.sequence.compound">RNACompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.Builder.html#rnaCreator(org.biojava3.core.sequence.io.template.SequenceCreatorInterface)">rnaCreator(SequenceCreatorInterface&lt;NucleotideCompound&gt;)</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava3.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/RNASequence.html" title="class in org.biojava3.core.sequence"><span class="strong">RNASequence</span></a> - Class in <a href="./org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a></dt>
<dd>
<div class="block">RNASequence where RNACompoundSet are the allowed values</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/RNASequence.html#RNASequence(java.lang.String)">RNASequence(String)</a></span> - Constructor for class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/RNASequence.html" title="class in org.biojava3.core.sequence">RNASequence</a></dt>
<dd>
<div class="block">Create a RNA sequence from a String</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/RNASequence.html#RNASequence(org.biojava3.core.sequence.template.ProxySequenceReader)">RNASequence(ProxySequenceReader&lt;NucleotideCompound&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/RNASequence.html" title="class in org.biojava3.core.sequence">RNASequence</a></dt>
<dd>
<div class="block">Create a RNA aequence from a proxy reader</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/RNASequence.html#RNASequence(java.lang.String, org.biojava3.core.sequence.template.CompoundSet)">RNASequence(String, CompoundSet&lt;NucleotideCompound&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/RNASequence.html" title="class in org.biojava3.core.sequence">RNASequence</a></dt>
<dd>
<div class="block">Create a RNA sequence from a string with a user defined RNA compound set</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/RNASequence.html#RNASequence(org.biojava3.core.sequence.template.ProxySequenceReader, org.biojava3.core.sequence.template.CompoundSet)">RNASequence(ProxySequenceReader&lt;NucleotideCompound&gt;, CompoundSet&lt;NucleotideCompound&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/RNASequence.html" title="class in org.biojava3.core.sequence">RNASequence</a></dt>
<dd>
<div class="block">Create a RNA sequence from a proxy reader and user defined RNA compound set</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/io/RNASequenceCreator.html" title="class in org.biojava3.core.sequence.io"><span class="strong">RNASequenceCreator</span></a> - Class in <a href="./org/biojava3/core/sequence/io/package-summary.html">org.biojava3.core.sequence.io</a></dt>
<dd>
<div class="block">Used to create a RNA sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/RNASequenceCreator.html#RNASequenceCreator(org.biojava3.core.sequence.template.CompoundSet)">RNASequenceCreator(CompoundSet&lt;NucleotideCompound&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/RNASequenceCreator.html" title="class in org.biojava3.core.sequence.io">RNASequenceCreator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/views/RnaSequenceView.html" title="class in org.biojava3.core.sequence.views"><span class="strong">RnaSequenceView</span></a> - Class in <a href="./org/biojava3/core/sequence/views/package-summary.html">org.biojava3.core.sequence.views</a></dt>
<dd>
<div class="block">Attempts to do on the fly translation of RNA by not requesting the compounds
 until asked.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/views/RnaSequenceView.html#RnaSequenceView(org.biojava3.core.sequence.template.Sequence)">RnaSequenceView(Sequence&lt;NucleotideCompound&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.views.<a href="./org/biojava3/core/sequence/views/RnaSequenceView.html" title="class in org.biojava3.core.sequence.views">RnaSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/views/RnaSequenceView.html#RnaSequenceView(org.biojava3.core.sequence.template.Sequence, org.biojava3.core.sequence.template.CompoundSet)">RnaSequenceView(Sequence&lt;NucleotideCompound&gt;, CompoundSet&lt;NucleotideCompound&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.views.<a href="./org/biojava3/core/sequence/views/RnaSequenceView.html" title="class in org.biojava3.core.sequence.views">RnaSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/transcription/RNAToAminoAcidTranslator.html" title="class in org.biojava3.core.sequence.transcription"><span class="strong">RNAToAminoAcidTranslator</span></a> - Class in <a href="./org/biojava3/core/sequence/transcription/package-summary.html">org.biojava3.core.sequence.transcription</a></dt>
<dd>
<div class="block">Takes a <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a> of <a href="./org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound"><code>NucleotideCompound</code></a> which should
 represent an RNA sequence (<a href="./org/biojava3/core/sequence/RNASequence.html" title="class in org.biojava3.core.sequence"><code>RNASequence</code></a> is good for this) and returns
 a list of <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a> which hold <a href="./org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound"><code>AminoAcidCompound</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/RNAToAminoAcidTranslator.html#RNAToAminoAcidTranslator(org.biojava3.core.sequence.io.template.SequenceCreatorInterface, org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.transcription.Table, boolean, boolean, boolean)">RNAToAminoAcidTranslator(SequenceCreatorInterface&lt;AminoAcidCompound&gt;, CompoundSet&lt;NucleotideCompound&gt;, CompoundSet&lt;Table.Codon&gt;, CompoundSet&lt;AminoAcidCompound&gt;, Table, boolean, boolean, boolean)</a></span> - Constructor for class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/RNAToAminoAcidTranslator.html" title="class in org.biojava3.core.sequence.transcription">RNAToAminoAcidTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/ronn/RonnConstraint.html" title="class in org.biojava3.ronn"><span class="strong">RonnConstraint</span></a> - Class in <a href="./org/biojava3/ronn/package-summary.html">org.biojava3.ronn</a></dt>
<dd>
<div class="block">A collection of various constrain values used by RONN</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ronn/RonnConstraint.html#RonnConstraint()">RonnConstraint()</a></span> - Constructor for class org.biojava3.ronn.<a href="./org/biojava3/ronn/RonnConstraint.html" title="class in org.biojava3.ronn">RonnConstraint</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/ronn/RonnConstraint.Threshold.html" title="enum in org.biojava3.ronn"><span class="strong">RonnConstraint.Threshold</span></a> - Enum in <a href="./org/biojava3/ronn/package-summary.html">org.biojava3.ronn</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#rotate(org.biojava.bio.structure.Atom, double[][])">rotate(Atom, double[][])</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">rotate a single atom aroud a rotation matrix.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#rotate(org.biojava.bio.structure.Structure, double[][])">rotate(Structure, double[][])</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">Rotate a structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#rotate(org.biojava.bio.structure.Group, double[][])">rotate(Group, double[][])</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">rotate a structure .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#rotate(org.biojava.bio.structure.Atom, org.biojava.bio.structure.jama.Matrix)">rotate(Atom, Matrix)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">Rotate an atom around a Matrix object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#rotate(org.biojava.bio.structure.Group, org.biojava.bio.structure.jama.Matrix)">rotate(Group, Matrix)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">Rotate a group object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#rotate(org.biojava.bio.structure.Structure, org.biojava.bio.structure.jama.Matrix)">rotate(Structure, Matrix)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">Rotate a structure object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/ColorUtils.html#rotateHue(java.awt.Color, float)">rotateHue(Color, float)</a></span> - Static method in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/ColorUtils.html" title="class in org.biojava.bio.structure.gui.util.color">ColorUtils</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/Utils.html#roundToDecimals(double, int)">roundToDecimals(double, int)</a></span> - Static method in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/Utils.html" title="class in org.biojava3.aaproperties">Utils</a></dt>
<dd>
<div class="block">Returns a value with the desired number of decimal places.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html#run()">run()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html" title="class in org.biojava.bio.structure.align.client">FarmJobRunnable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/FarmJob.html#run()">run()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/FarmJob.html" title="class in org.biojava.bio.structure.align">FarmJob</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/AlignmentCalc.html#run()">run()</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/AlignmentCalc.html" title="class in org.biojava.bio.structure.align.gui">AlignmentCalc</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/AlignmentCalcDB.html#run()">run()</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/AlignmentCalcDB.html" title="class in org.biojava.bio.structure.align.gui">AlignmentCalcDB</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/GUIFarmJobRunnable.html#run()">run()</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/GUIFarmJobRunnable.html" title="class in org.biojava.bio.structure.align.gui">GUIFarmJobRunnable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/AlignmentCalc.html#run()">run()</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/AlignmentCalc.html" title="class in org.biojava.bio.structure.gui.util">AlignmentCalc</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/AllChemCompProvider.html#run()">run()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/AllChemCompProvider.html" title="class in org.biojava.bio.structure.io.mmcif">AllChemCompProvider</a></dt>
<dd>
<div class="block">Do the actual loading of the dictionary in a thread.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureFetcherRunnable.html#run()">run()</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureFetcherRunnable.html" title="class in org.biojava.bio.structure.server">StructureFetcherRunnable</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/CommandPrompt.html#run(java.lang.String[])">run(String[])</a></span> - Static method in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/CommandPrompt.html" title="class in org.biojava3.aaproperties">CommandPrompt</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/TreeConstructor.html#run()">run()</a></span> - Method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/TreeConstructor.html" title="class in org.biojava3.phylo">TreeConstructor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/StructureAlignmentOptimizer.html#runOptimization(int)">runOptimization(int)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/StructureAlignmentOptimizer.html" title="class in org.biojava.bio.structure.align.fatcat.calc">StructureAlignmentOptimizer</a></dt>
<dd>
<div class="block">run the optimization</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/Alignments.html#runPairwiseScorers(java.util.List)">runPairwiseScorers(List&lt;PairwiseSequenceScorer&lt;S, C&gt;&gt;)</a></span> - Static method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/Alignments.html" title="class in org.biojava3.alignment">Alignments</a></dt>
<dd>
<div class="block">Factory method to run a list of scorers concurrently.</div>
</dd>
</dl>
<a name="_S_">
<!--   -->
</a>
<h2 class="title">S</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/Constraints.html#S">S</a></span> - Static variable in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/Constraints.html" title="class in org.biojava3.aaproperties">Constraints</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/CutValues.html#s_min">s_min</a></span> - Variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/CutValues.html" title="class in org.biojava.bio.structure.domain.pdp">CutValues</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/SandboxStyleStructureProvider.html" title="class in org.biojava.bio.structure.io"><span class="strong">SandboxStyleStructureProvider</span></a> - Class in <a href="./org/biojava/bio/structure/io/package-summary.html">org.biojava.bio.structure.io</a></dt>
<dd>
<div class="block">The "Sandbox" style of organizing files is  to have a directory structure like below, i.e. the files are organized into
 
        directory with two characters, based on the two middle characters of a PDB ID
        directory of PDB ID
        several files that are available for this PDB ID
 
 
 
a1/2a1v/2a1v.cif.gz
a1/2a1v/2a1v.dssp.gz
a1/2a1v/2a1v.pdb-250.jpg.gz
a1/2a1v/2a1v.pdb-500.jpg.gz
a1/2a1v/2a1v.pdb-65.jpg.gz
a1/2a1v/2a1v.pdb-80.jpg.gz
a1/2a1v/2a1v.pdb1-250.jpg.gz
a1/2a1v/2a1v.pdb1-500.jpg.gz
a1/2a1v/2a1v.pdb1-65.jpg.gz
a1/2a1v/2a1v.pdb1-80.jpg.gz
a1/2a1v/2a1v.pdb1.gz
a1/2a1v/2a1v.stride.gz
a1/2a1v/2a1v.xml.gz
a1/2a1v/pdb2a1v.ent.gz
a1/2a1v/r2a1vsf.ent.gz
a1/2a1w/2a1w-deriv.cif.gz
a1/2a1w/2a1w-extatom.xml.gz
a1/2a1w/2a1w-noatom.xml.gz
a1/2a1w/2a1w.cif.gz
a1/2a1w/2a1w.dssp.gz
a1/2a1w/2a1w.pdb-250.jpg.gz
a1/2a1w/2a1w.pdb-500.jpg.gz
a1/2a1w/2a1w.pdb-65.jpg.gz
a1/2a1w/2a1w.pdb-80.jpg.gz
a1/2a1w/2a1w.pdb1-250.jpg.gz
a1/2a1w/2a1w.pdb1-500.jpg.gz
a1/2a1w/2a1w.pdb1-65.jpg.gz
a1/2a1w/2a1w.pdb1-80.jpg.gz
a1/2a1w/2a1w.pdb1.gz
a1/2a1w/2a1w.pdb2-250.jpg.gz
a1/2a1w/2a1w.pdb2-500.jpg.gz
a1/2a1w/2a1w.pdb2-65.jpg.gz
a1/2a1w/2a1w.pdb2-80.jpg.gz
a1/2a1w/2a1w.pdb2.gz
a1/2a1w/2a1w.pdb3-250.jpg.gz
a1/2a1w/2a1w.pdb3-500.jpg.gz
a1/2a1w/2a1w.pdb3-65.jpg.gz
a1/2a1w/2a1w.pdb3-80.jpg.gz
a1/2a1w/2a1w.pdb3.gz
a1/2a1w/2a1w.pdb4-250.jpg.gz
a1/2a1w/2a1w.pdb4-500.jpg.gz
a1/2a1w/2a1w.pdb4-65.jpg.gz
a1/2a1w/2a1w.pdb4-80.jpg.gz
a1/2a1w/2a1w.pdb4.gz
a1/2a1w/2a1w.pdb5-250.jpg.gz
a1/2a1w/2a1w.pdb5-500.jpg.gz
a1/2a1w/2a1w.pdb5-65.jpg.gz
a1/2a1w/2a1w.pdb5-80.jpg.gz
a1/2a1w/2a1w.pdb5.gz
a1/2a1w/2a1w.pdb6-250.jpg.gz
a1/2a1w/2a1w.pdb6-500.jpg.gz
a1/2a1w/2a1w.pdb6-65.jpg.gz
a1/2a1w/2a1w.pdb6-80.jpg.gz
a1/2a1w/2a1w.pdb6.gz
a1/2a1w/2a1w.stride.gz
a1/2a1w/2a1w.xml.gz
a1/2a1w/pdb2a1w.ent.gz
a1/2a1w/r2a1wsf.ent.gz
a1/2a1x/2a1x-deriv.cif.gz
a1/2a1x/2a1x-extatom.xml.gz
a1/2a1x/2a1x-noatom.xml.gz
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/SandboxStyleStructureProvider.html#SandboxStyleStructureProvider()">SandboxStyleStructureProvider()</a></span> - Constructor for class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/SandboxStyleStructureProvider.html" title="class in org.biojava.bio.structure.io">SandboxStyleStructureProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/webstart/PersistentConfig.html#save(org.biojava.bio.structure.align.util.UserConfiguration)">save(UserConfiguration)</a></span> - Method in class org.biojava.bio.structure.align.webstart.<a href="./org/biojava/bio/structure/align/webstart/PersistentConfig.html" title="class in org.biojava.bio.structure.align.webstart">PersistentConfig</a></dt>
<dd>
<div class="block">writes the configuration</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/MenuCreator.html#SAVE_ALIGNMENT_XML">SAVE_ALIGNMENT_XML</a></span> - Static variable in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/MenuCreator.html" title="class in org.biojava.bio.structure.align.gui">MenuCreator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html#SCALE_COLOR">SCALE_COLOR</a></span> - Static variable in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html" title="class in org.biojava.bio.structure.gui.util">SequenceScalePanel</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html" title="class in org.biojava.bio.structure.gui"><span class="strong">ScaleableMatrixPanel</span></a> - Class in <a href="./org/biojava/bio/structure/gui/package-summary.html">org.biojava.bio.structure.gui</a></dt>
<dd>
<div class="block">A JPanel that can display the underlying distance matrix 
 data of the protein structure alignment algorithm.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html#ScaleableMatrixPanel()">ScaleableMatrixPanel()</a></span> - Constructor for class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html" title="class in org.biojava.bio.structure.gui">ScaleableMatrixPanel</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.GradientRenderer.html" title="class in org.biojava.bio.structure.gui"><span class="strong">ScaleableMatrixPanel.GradientRenderer</span></a> - Class in <a href="./org/biojava/bio/structure/gui/package-summary.html">org.biojava.bio.structure.gui</a></dt>
<dd>
<div class="block">A renderer for the the gradient dropdown menu at the top of scaleableMatrixPanes.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.GradientRenderer.html#ScaleableMatrixPanel.GradientRenderer()">ScaleableMatrixPanel.GradientRenderer()</a></span> - Constructor for class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.GradientRenderer.html" title="class in org.biojava.bio.structure.gui">ScaleableMatrixPanel.GradientRenderer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#scaleAdd(float, org.biojava.bio.structure.Atom, org.biojava.bio.structure.Atom)">scaleAdd(float, Atom, Atom)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#scaleColours">scaleColours</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava3.alignment.aaindex"><span class="strong">ScaledSubstitutionMatrix</span></a> - Class in <a href="./org/biojava3/alignment/aaindex/package-summary.html">org.biojava3.alignment.aaindex</a></dt>
<dd>
<div class="block">The biojava-alignment module represents substitution matrices with short values.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html#ScaledSubstitutionMatrix()">ScaledSubstitutionMatrix()</a></span> - Constructor for class org.biojava3.alignment.aaindex.<a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava3.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/aaproperties/xml/SchemaGenerator.html" title="class in org.biojava3.aaproperties.xml"><span class="strong">SchemaGenerator</span></a> - Class in <a href="./org/biojava3/aaproperties/xml/package-summary.html">org.biojava3.aaproperties.xml</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/SchemaGenerator.html#SchemaGenerator(java.lang.String)">SchemaGenerator(String)</a></span> - Constructor for class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/SchemaGenerator.html" title="class in org.biojava3.aaproperties.xml">SchemaGenerator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#SCOP_DOWNLOAD">SCOP_DOWNLOAD</a></span> - Static variable in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/scop/ScopCategory.html" title="enum in org.biojava.bio.structure.scop"><span class="strong">ScopCategory</span></a> - Enum in <a href="./org/biojava/bio/structure/scop/package-summary.html">org.biojava.bio.structure.scop</a></dt>
<dd>
<div class="block">The various categories provided by SCOP.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/scop/ScopDatabase.html" title="interface in org.biojava.bio.structure.scop"><span class="strong">ScopDatabase</span></a> - Interface in <a href="./org/biojava/bio/structure/scop/package-summary.html">org.biojava.bio.structure.scop</a></dt>
<dd>
<div class="block">General API how to interact with SCOP</div>
</dd>
<dt><a href="./org/biojava/bio/structure/scop/ScopDescription.html" title="class in org.biojava.bio.structure.scop"><span class="strong">ScopDescription</span></a> - Class in <a href="./org/biojava/bio/structure/scop/package-summary.html">org.biojava.bio.structure.scop</a></dt>
<dd>
<div class="block">Contains data from
 dir.des.scop.txt_1.75
 
 e.g 
 
 SunID  Cat     Class           Name    Description
 -----  ---     -----           ----    -----------
 26154  px      b.47.1.2        d1nrs.1 1nrs L:,H:
 125030 px      b.47.1.2        d1zgia1 1zgi A:1A-245
 </div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDescription.html#ScopDescription()">ScopDescription()</a></span> - Constructor for class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDescription.html" title="class in org.biojava.bio.structure.scop">ScopDescription</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/scop/server/ScopDescriptions.html" title="class in org.biojava.bio.structure.scop.server"><span class="strong">ScopDescriptions</span></a> - Class in <a href="./org/biojava/bio/structure/scop/server/package-summary.html">org.biojava.bio.structure.scop.server</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/ScopDescriptions.html#ScopDescriptions()">ScopDescriptions()</a></span> - Constructor for class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/ScopDescriptions.html" title="class in org.biojava.bio.structure.scop.server">ScopDescriptions</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop"><span class="strong">ScopDomain</span></a> - Class in <a href="./org/biojava/bio/structure/scop/package-summary.html">org.biojava.bio.structure.scop</a></dt>
<dd>
<div class="block">Container for the information for a domain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#ScopDomain()">ScopDomain()</a></span> - Constructor for class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/scop/server/ScopDomains.html" title="class in org.biojava.bio.structure.scop.server"><span class="strong">ScopDomains</span></a> - Class in <a href="./org/biojava/bio/structure/scop/server/package-summary.html">org.biojava.bio.structure.scop.server</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/ScopDomains.html#ScopDomains()">ScopDomains()</a></span> - Constructor for class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/ScopDomains.html" title="class in org.biojava.bio.structure.scop.server">ScopDomains</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/scop/ScopFactory.html" title="class in org.biojava.bio.structure.scop"><span class="strong">ScopFactory</span></a> - Class in <a href="./org/biojava/bio/structure/scop/package-summary.html">org.biojava.bio.structure.scop</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopFactory.html#ScopFactory()">ScopFactory()</a></span> - Constructor for class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopFactory.html" title="class in org.biojava.bio.structure.scop">ScopFactory</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop"><span class="strong">ScopInstallation</span></a> - Class in <a href="./org/biojava/bio/structure/scop/package-summary.html">org.biojava.bio.structure.scop</a></dt>
<dd>
<div class="block">This class provides access to the SCOP protein structure classification.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#ScopInstallation(java.lang.String)">ScopInstallation(String)</a></span> - Constructor for class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>
<div class="block">Create a new SCOP installation.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#ScopInstallation()">ScopInstallation()</a></span> - Constructor for class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>
<div class="block">Create a new SCOP installation, downloading the file to "the right place".</div>
</dd>
<dt><a href="./org/biojava/bio/structure/gui/util/ScopInstallationInstance.html" title="class in org.biojava.bio.structure.gui.util"><span class="strong">ScopInstallationInstance</span></a> - Class in <a href="./org/biojava/bio/structure/gui/util/package-summary.html">org.biojava.bio.structure.gui.util</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/scop/ScopNode.html" title="class in org.biojava.bio.structure.scop"><span class="strong">ScopNode</span></a> - Class in <a href="./org/biojava/bio/structure/scop/package-summary.html">org.biojava.bio.structure.scop</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopNode.html#ScopNode()">ScopNode()</a></span> - Constructor for class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopNode.html" title="class in org.biojava.bio.structure.scop">ScopNode</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/scop/server/ScopNodes.html" title="class in org.biojava.bio.structure.scop.server"><span class="strong">ScopNodes</span></a> - Class in <a href="./org/biojava/bio/structure/scop/server/package-summary.html">org.biojava.bio.structure.scop.server</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/ScopNodes.html#ScopNodes()">ScopNodes()</a></span> - Constructor for class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/ScopNodes.html" title="class in org.biojava.bio.structure.scop.server">ScopNodes</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/gui/util/ScopSelectPanel.html" title="class in org.biojava.bio.structure.gui.util"><span class="strong">ScopSelectPanel</span></a> - Class in <a href="./org/biojava/bio/structure/gui/util/package-summary.html">org.biojava.bio.structure.gui.util</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/ScopSelectPanel.html#ScopSelectPanel()">ScopSelectPanel()</a></span> - Constructor for class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/ScopSelectPanel.html" title="class in org.biojava.bio.structure.gui.util">ScopSelectPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#score">score</a></span> - Variable in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava3.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Feature.html#score()">score()</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Feature.html" title="class in org.biojava3.genome.parsers.gff">Feature</a></dt>
<dd>
<div class="block">Get score.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#scoreMatrices">scoreMatrices</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/phylo/ScoreMatrix.html" title="class in org.biojava3.phylo"><span class="strong">ScoreMatrix</span></a> - Class in <a href="./org/biojava3/phylo/package-summary.html">org.biojava3.phylo</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/alignment/template/Scorer.html" title="interface in org.biojava3.alignment.template"><span class="strong">Scorer</span></a> - Interface in <a href="./org/biojava3/alignment/template/package-summary.html">org.biojava3.alignment.template</a></dt>
<dd>
<div class="block">Defines an algorithm which computes a score.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#scores">scores</a></span> - Variable in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava3.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/ronn/Jronn.html#scoresToRanges(float[], float)">scoresToRanges(float[], float)</a></span> - Static method in class org.biojava3.ronn.<a href="./org/biojava3/ronn/Jronn.html" title="class in org.biojava3.ronn">Jronn</a></dt>
<dd>
<div class="block">Convert raw scores to ranges.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#SCORING_STRATEGY">SCORING_STRATEGY</a></span> - Static variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html#scroll">scroll</a></span> - Variable in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html" title="class in org.biojava.bio.structure.gui">ScaleableMatrixPanel</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/secstruc/SecStruc.html" title="class in org.biojava.bio.structure.secstruc"><span class="strong">SecStruc</span></a> - Class in <a href="./org/biojava/bio/structure/secstruc/package-summary.html">org.biojava.bio.structure.secstruc</a></dt>
<dd>
<div class="block">Assign secondary structure to a Structure object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStruc.html#SecStruc()">SecStruc()</a></span> - Constructor for class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStruc.html" title="class in org.biojava.bio.structure.secstruc">SecStruc</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html" title="class in org.biojava.bio.structure.secstruc"><span class="strong">SecStrucGroup</span></a> - Class in <a href="./org/biojava/bio/structure/secstruc/package-summary.html">org.biojava.bio.structure.secstruc</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html#SecStrucGroup()">SecStrucGroup()</a></span> - Constructor for class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html" title="class in org.biojava.bio.structure.secstruc">SecStrucGroup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/Hashcoder.html#SEED">SEED</a></span> - Static variable in class org.biojava3.core.util.<a href="./org/biojava3/core/util/Hashcoder.html" title="class in org.biojava3.core.util">Hashcoder</a></dt>
<dd>
<div class="block">An initial value for a <code>hashCode</code>, to which we add
 contributions from fields.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/domain/pdp/Segment.html" title="class in org.biojava.bio.structure.domain.pdp"><span class="strong">Segment</span></a> - Class in <a href="./org/biojava/bio/structure/domain/pdp/package-summary.html">org.biojava.bio.structure.domain.pdp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Segment.html#Segment()">Segment()</a></span> - Constructor for class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Segment.html" title="class in org.biojava.bio.structure.domain.pdp">Segment</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/domain/pdp/SegmentComparator.html" title="class in org.biojava.bio.structure.domain.pdp"><span class="strong">SegmentComparator</span></a> - Class in <a href="./org/biojava/bio/structure/domain/pdp/package-summary.html">org.biojava.bio.structure.domain.pdp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/SegmentComparator.html#SegmentComparator()">SegmentComparator()</a></span> - Constructor for class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/SegmentComparator.html" title="class in org.biojava.bio.structure.domain.pdp">SegmentComparator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/MenuCreator.html#SELECT_EQR">SELECT_EQR</a></span> - Static variable in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/MenuCreator.html" title="class in org.biojava.bio.structure.align.gui">MenuCreator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/FeatureList.html#selectByAttribute(java.lang.String, java.lang.String)">selectByAttribute(String, String)</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/FeatureList.html" title="class in org.biojava3.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block">Create a list of all features that include the specified attribute key/value pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/FeatureList.html#selectByAttribute(java.lang.String)">selectByAttribute(String)</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/FeatureList.html" title="class in org.biojava3.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block">Create a list of all features that include the specified attribute key.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/FeatureList.html#selectByGroup(java.lang.String)">selectByGroup(String)</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/FeatureList.html" title="class in org.biojava3.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block">Create a list of all features that have the specified group id, as defined by
 the group() method of the features.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/FeatureList.html#selectByType(java.lang.String)">selectByType(String)</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/FeatureList.html" title="class in org.biojava3.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block">Create a list of all features that are of the specified type, as defined by
 the type() method of the features.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/FeatureList.html#selectByUserData(java.lang.String, java.lang.Object)">selectByUserData(String, Object)</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/FeatureList.html" title="class in org.biojava3.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block">Create a list of all features that include the specified key/value pair in their userMap().</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/FeatureList.html#selectByUserData(java.lang.String)">selectByUserData(String)</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/FeatureList.html" title="class in org.biojava3.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block">Create a list of all features that include the specified key in their userMap().</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/XMLHelper.html#selectElements(org.w3c.dom.Element, java.lang.String)">selectElements(Element, String)</a></span> - Static method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/XMLHelper.html" title="class in org.biojava3.core.util">XMLHelper</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/structure/gui/Selection.html" title="interface in org.biojava3.structure.gui"><span class="strong">Selection</span></a> - Interface in <a href="./org/biojava3/structure/gui/package-summary.html">org.biojava3.structure.gui</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/structure/gui/SelectionImpl.html" title="class in org.biojava3.structure.gui"><span class="strong">SelectionImpl</span></a> - Class in <a href="./org/biojava3/structure/gui/package-summary.html">org.biojava3.structure.gui</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/structure/gui/SelectionImpl.html#SelectionImpl()">SelectionImpl()</a></span> - Constructor for class org.biojava3.structure.gui.<a href="./org/biojava3/structure/gui/SelectionImpl.html" title="class in org.biojava3.structure.gui">SelectionImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html#selectionLocked()">selectionLocked()</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AligPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/StatusDisplay.html#selectionLocked()">selectionLocked()</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/StatusDisplay.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">StatusDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/events/AlignmentPositionListener.html#selectionLocked()">selectionLocked()</a></span> - Method in interface org.biojava.bio.structure.gui.events.<a href="./org/biojava/bio/structure/gui/events/AlignmentPositionListener.html" title="interface in org.biojava.bio.structure.gui.events">AlignmentPositionListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/events/JmolAlignedPositionListener.html#selectionLocked()">selectionLocked()</a></span> - Method in class org.biojava.bio.structure.gui.events.<a href="./org/biojava/bio/structure/gui/events/JmolAlignedPositionListener.html" title="class in org.biojava.bio.structure.gui.events">JmolAlignedPositionListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html#selectionUnlocked()">selectionUnlocked()</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AligPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/StatusDisplay.html#selectionUnlocked()">selectionUnlocked()</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/StatusDisplay.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">StatusDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/events/AlignmentPositionListener.html#selectionUnlocked()">selectionUnlocked()</a></span> - Method in interface org.biojava.bio.structure.gui.events.<a href="./org/biojava/bio/structure/gui/events/AlignmentPositionListener.html" title="interface in org.biojava.bio.structure.gui.events">AlignmentPositionListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/events/JmolAlignedPositionListener.html#selectionUnlocked()">selectionUnlocked()</a></span> - Method in class org.biojava.bio.structure.gui.events.<a href="./org/biojava/bio/structure/gui/events/JmolAlignedPositionListener.html" title="class in org.biojava.bio.structure.gui.events">JmolAlignedPositionListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/FeatureList.html#selectOverlapping(java.lang.String, org.biojava3.genome.parsers.gff.Location, boolean)">selectOverlapping(String, Location, boolean)</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/FeatureList.html" title="class in org.biojava3.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block">Create a list of all features that overlap the specified location on the specified
 sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/XMLHelper.html#selectParentElement(org.w3c.dom.Element, java.lang.String)">selectParentElement(Element, String)</a></span> - Static method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/XMLHelper.html" title="class in org.biojava3.core.util">XMLHelper</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/gui/SelectPDBPanel.html" title="class in org.biojava.bio.structure.align.gui"><span class="strong">SelectPDBPanel</span></a> - Class in <a href="./org/biojava/bio/structure/align/gui/package-summary.html">org.biojava.bio.structure.align.gui</a></dt>
<dd>
<div class="block">A Panel that allows user to specify PDB & chain ID, as well as sub-ranges</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/SelectPDBPanel.html#SelectPDBPanel()">SelectPDBPanel()</a></span> - Constructor for class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/SelectPDBPanel.html" title="class in org.biojava.bio.structure.align.gui">SelectPDBPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/SelectPDBPanel.html#SelectPDBPanel(boolean)">SelectPDBPanel(boolean)</a></span> - Constructor for class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/SelectPDBPanel.html" title="class in org.biojava.bio.structure.align.gui">SelectPDBPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/XMLHelper.html#selectSingleElement(org.w3c.dom.Element, java.lang.String)">selectSingleElement(Element, String)</a></span> - Static method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/XMLHelper.html" title="class in org.biojava3.core.util">XMLHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/JFatCatClient.html#sendAFPChainToServer(java.lang.String, org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">sendAFPChainToServer(String, AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/JFatCatClient.html" title="class in org.biojava.bio.structure.align.client">JFatCatClient</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastService.html#sendAlignmentRequest(java.lang.String, org.biojava3.ws.alignment.RemotePairwiseAlignmentProperties)">sendAlignmentRequest(String, RemotePairwiseAlignmentProperties)</a></span> - Method in class org.biojava3.ws.alignment.qblast.<a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastService.html" title="class in org.biojava3.ws.alignment.qblast">NCBIQBlastService</a></dt>
<dd>
<div class="block">This method is a wrapper that executes the Blast request via the Put
 command of the CGI-BIN interface with the specified parameters and a
 string representing the sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastService.html#sendAlignmentRequest(org.biojava3.core.sequence.template.Sequence, org.biojava3.ws.alignment.RemotePairwiseAlignmentProperties)">sendAlignmentRequest(Sequence, RemotePairwiseAlignmentProperties)</a></span> - Method in class org.biojava3.ws.alignment.qblast.<a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastService.html" title="class in org.biojava3.ws.alignment.qblast">NCBIQBlastService</a></dt>
<dd>
<div class="block">This method is a wrapper that executes the Blast request via the Put
 command of the CGI-BIN interface with the specified parameters and a
 Sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastService.html#sendAlignmentRequest(int, org.biojava3.ws.alignment.RemotePairwiseAlignmentProperties)">sendAlignmentRequest(int, RemotePairwiseAlignmentProperties)</a></span> - Method in class org.biojava3.ws.alignment.qblast.<a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastService.html" title="class in org.biojava3.ws.alignment.qblast">NCBIQBlastService</a></dt>
<dd>
<div class="block">This method is a wrapper that executes the Blast request via the Put
 command of the CGI-BIN interface with the specified parameters and a
 GenBank GID.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/RemotePairwiseAlignmentService.html#sendAlignmentRequest(org.biojava3.core.sequence.template.Sequence, org.biojava3.ws.alignment.RemotePairwiseAlignmentProperties)">sendAlignmentRequest(Sequence, RemotePairwiseAlignmentProperties)</a></span> - Method in interface org.biojava3.ws.alignment.<a href="./org/biojava3/ws/alignment/RemotePairwiseAlignmentService.html" title="interface in org.biojava3.ws.alignment">RemotePairwiseAlignmentService</a></dt>
<dd>
<div class="block">Doing the actual analysis on the instantiated service using specified parameters and the RichSequence object</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/RemotePairwiseAlignmentService.html#sendAlignmentRequest(java.lang.String, org.biojava3.ws.alignment.RemotePairwiseAlignmentProperties)">sendAlignmentRequest(String, RemotePairwiseAlignmentProperties)</a></span> - Method in interface org.biojava3.ws.alignment.<a href="./org/biojava3/ws/alignment/RemotePairwiseAlignmentService.html" title="interface in org.biojava3.ws.alignment">RemotePairwiseAlignmentService</a></dt>
<dd>
<div class="block">Doing the actual analysis on the instantiated service using specified parameters on the string representation
 of the Sequence object</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/JFatCatClient.html#sendMultiAFPChainToServer(java.lang.String, java.lang.String, java.lang.String)">sendMultiAFPChainToServer(String, String, String)</a></span> - Static method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/JFatCatClient.html" title="class in org.biojava.bio.structure.align.client">JFatCatClient</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html#sendResultsToServer(java.util.List)">sendResultsToServer(List&lt;String&gt;)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html" title="class in org.biojava.bio.structure.align.client">FarmJobRunnable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/CountProgressListener.html#sentResultsToServer(int, java.lang.String)">sentResultsToServer(int, String)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/CountProgressListener.html" title="class in org.biojava.bio.structure.align.client">CountProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/events/AlignmentProgressListener.html#sentResultsToServer(int, java.lang.String)">sentResultsToServer(int, String)</a></span> - Method in interface org.biojava.bio.structure.align.events.<a href="./org/biojava/bio/structure/align/events/AlignmentProgressListener.html" title="interface in org.biojava.bio.structure.align.events">AlignmentProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/GUIAlignmentProgressListener.html#sentResultsToServer(int, java.lang.String)">sentResultsToServer(int, String)</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/GUIAlignmentProgressListener.html" title="class in org.biojava.bio.structure.align.gui">GUIAlignmentProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#seqArraySize(int)">seqArraySize(int)</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava3.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html#seqFont">seqFont</a></span> - Static variable in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html" title="class in org.biojava.bio.structure.gui.util">SequenceScalePanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Feature.html#seqname()">seqname()</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Feature.html" title="class in org.biojava3.genome.parsers.gff">Feature</a></dt>
<dd>
<div class="block">Get the sequence name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/FeatureI.html#seqname()">seqname()</a></span> - Method in interface org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/FeatureI.html" title="interface in org.biojava3.genome.parsers.gff">FeatureI</a></dt>
<dd>
<div class="block">Get the sequence name.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/io/SeqRes2AtomAligner.html" title="class in org.biojava.bio.structure.io"><span class="strong">SeqRes2AtomAligner</span></a> - Class in <a href="./org/biojava/bio/structure/io/package-summary.html">org.biojava.bio.structure.io</a></dt>
<dd>
<div class="block">Aligns the SEQRES residues to the ATOM residues.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/SeqRes2AtomAligner.html#SeqRes2AtomAligner()">SeqRes2AtomAligner()</a></span> - Constructor for class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/SeqRes2AtomAligner.html" title="class in org.biojava.bio.structure.io">SeqRes2AtomAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcid.html#SEQRESRECORD">SEQRESRECORD</a></span> - Static variable in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcid.html" title="interface in org.biojava.bio.structure">AminoAcid</a></dt>
<dd>
<div class="block">Field to distribguish AminoAcids that have been created from SEQRES records and ATOM records.</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><span class="strong">Sequence</span></a>&lt;<a href="./org/biojava3/core/sequence/template/Sequence.html" title="type parameter in Sequence">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava3/core/sequence/template/package-summary.html">org.biojava3.core.sequence.template</a></dt>
<dd>
<div class="block">Main interface for defining a collection of Compounds and accessing them
 using biological indexes</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html#SEQUENCE_COLOR">SEQUENCE_COLOR</a></span> - Static variable in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html" title="class in org.biojava.bio.structure.gui.util">SequenceScalePanel</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/storage/SequenceAsStringHelper.html" title="class in org.biojava3.core.sequence.storage"><span class="strong">SequenceAsStringHelper</span></a>&lt;<a href="./org/biojava3/core/sequence/storage/SequenceAsStringHelper.html" title="type parameter in SequenceAsStringHelper">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/core/sequence/storage/package-summary.html">org.biojava3.core.sequence.storage</a></dt>
<dd>
<div class="block">This is a common method that can be used across multiple storage/proxy implementations to
 handle Negative strand and other interesting elements of sequence data.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/SequenceAsStringHelper.html#SequenceAsStringHelper()">SequenceAsStringHelper()</a></span> - Constructor for class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/SequenceAsStringHelper.html" title="class in org.biojava3.core.sequence.storage">SequenceAsStringHelper</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/SequenceComparator.html" title="class in org.biojava3.core.sequence"><span class="strong">SequenceComparator</span></a> - Class in <a href="./org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a></dt>
<dd>
<div class="block">Used to sort sequences</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/SequenceComparator.html#SequenceComparator()">SequenceComparator()</a></span> - Constructor for class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/SequenceComparator.html" title="class in org.biojava3.core.sequence">SequenceComparator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/io/template/SequenceCreatorInterface.html" title="interface in org.biojava3.core.sequence.io.template"><span class="strong">SequenceCreatorInterface</span></a>&lt;<a href="./org/biojava3/core/sequence/io/template/SequenceCreatorInterface.html" title="type parameter in SequenceCreatorInterface">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava3/core/sequence/io/template/package-summary.html">org.biojava3.core.sequence.io.template</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/gui/SequenceDisplay.html" title="class in org.biojava.bio.structure.gui"><span class="strong">SequenceDisplay</span></a> - Class in <a href="./org/biojava/bio/structure/gui/package-summary.html">org.biojava.bio.structure.gui</a></dt>
<dd>
<div class="block">A sequence display that can show the results of a protein structure alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/SequenceDisplay.html#SequenceDisplay(org.biojava.bio.structure.align.StructurePairAligner)">SequenceDisplay(StructurePairAligner)</a></span> - Constructor for class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/SequenceDisplay.html" title="class in org.biojava.bio.structure.gui">SequenceDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceMixin.html#sequenceEquality(org.biojava3.core.sequence.template.Sequence, org.biojava3.core.sequence.template.Sequence)">sequenceEquality(Sequence&lt;C&gt;, Sequence&lt;C&gt;)</a></span> - Static method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceMixin.html" title="class in org.biojava3.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">A case-sensitive manner of comparing two sequence objects together.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceMixin.html#sequenceEqualityIgnoreCase(org.biojava3.core.sequence.template.Sequence, org.biojava3.core.sequence.template.Sequence)">sequenceEqualityIgnoreCase(Sequence&lt;C&gt;, Sequence&lt;C&gt;)</a></span> - Static method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceMixin.html" title="class in org.biojava3.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">A case-insensitive manner of comparing two sequence objects together.</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava3.core.sequence.loader"><span class="strong">SequenceFileProxyLoader</span></a>&lt;<a href="./org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html" title="type parameter in SequenceFileProxyLoader">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/core/sequence/loader/package-summary.html">org.biojava3.core.sequence.loader</a></dt>
<dd>
<div class="block">This class represents the storage container of a sequence stored in a fasta file where
 the initial parsing of the file we store the offset and length of the sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html#SequenceFileProxyLoader(java.io.File, org.biojava3.core.sequence.io.template.SequenceParserInterface, long, int, org.biojava3.core.sequence.template.CompoundSet)">SequenceFileProxyLoader(File, SequenceParserInterface, long, int, CompoundSet&lt;C&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava3.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/exceptions/SequenceLengthError.html" title="class in org.biojava3.core.exceptions"><span class="strong">SequenceLengthError</span></a> - Error in <a href="./org/biojava3/core/exceptions/package-summary.html">org.biojava3.core.exceptions</a></dt>
<dd>
<div class="block">Currently not used</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/exceptions/SequenceLengthError.html#SequenceLengthError(java.lang.String)">SequenceLengthError(String)</a></span> - Constructor for error org.biojava3.core.exceptions.<a href="./org/biojava3/core/exceptions/SequenceLengthError.html" title="class in org.biojava3.core.exceptions">SequenceLengthError</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/location/SequenceLocation.html" title="class in org.biojava3.core.sequence.location"><span class="strong">SequenceLocation</span></a>&lt;<a href="./org/biojava3/core/sequence/location/SequenceLocation.html" title="type parameter in SequenceLocation">S</a> extends <a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a>&lt;<a href="./org/biojava3/core/sequence/location/SequenceLocation.html" title="type parameter in SequenceLocation">C</a>&gt;,<a href="./org/biojava3/core/sequence/location/SequenceLocation.html" title="type parameter in SequenceLocation">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/core/sequence/location/package-summary.html">org.biojava3.core.sequence.location</a></dt>
<dd>
<div class="block">A location in a sequence that keeps a reference to its parent sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/SequenceLocation.html#SequenceLocation(int, int, S)">SequenceLocation(int, int, S)</a></span> - Constructor for class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/SequenceLocation.html" title="class in org.biojava3.core.sequence.location">SequenceLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/template/SequenceMixin.html" title="class in org.biojava3.core.sequence.template"><span class="strong">SequenceMixin</span></a> - Class in <a href="./org/biojava3/core/sequence/template/package-summary.html">org.biojava3.core.sequence.template</a></dt>
<dd>
<div class="block">Provides a set of static methods to be used as static imports when needed
 across multiple Sequence implementations but inheritance gets in the way.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceMixin.html#SequenceMixin()">SequenceMixin()</a></span> - Constructor for class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceMixin.html" title="class in org.biojava3.core.sequence.template">SequenceMixin</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/template/SequenceMixin.SequenceIterator.html" title="class in org.biojava3.core.sequence.template"><span class="strong">SequenceMixin.SequenceIterator</span></a>&lt;<a href="./org/biojava3/core/sequence/template/SequenceMixin.SequenceIterator.html" title="type parameter in SequenceMixin.SequenceIterator">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/core/sequence/template/package-summary.html">org.biojava3.core.sequence.template</a></dt>
<dd>
<div class="block">A basic sequence iterator which iterates over the given Sequence by
 biological index.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceMixin.SequenceIterator.html#SequenceMixin.SequenceIterator(org.biojava3.core.sequence.template.Sequence)">SequenceMixin.SequenceIterator(Sequence&lt;C&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceMixin.SequenceIterator.html" title="class in org.biojava3.core.sequence.template">SequenceMixin.SequenceIterator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/gui/util/SequenceMouseListener.html" title="class in org.biojava.bio.structure.gui.util"><span class="strong">SequenceMouseListener</span></a> - Class in <a href="./org/biojava/bio/structure/gui/util/package-summary.html">org.biojava.bio.structure.gui.util</a></dt>
<dd>
<div class="block">a mouse listener for the AbstractChainRenderer class
 it listens to all mouse events and triggeres appropriate
 SequenceListener and FeatureListener events</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/SequenceMouseListener.html#SequenceMouseListener(org.biojava.bio.structure.gui.SequenceDisplay)">SequenceMouseListener(SequenceDisplay)</a></span> - Constructor for class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/SequenceMouseListener.html" title="class in org.biojava.bio.structure.gui.util">SequenceMouseListener</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/SequenceOptimizationHints.html" title="class in org.biojava3.core.sequence"><span class="strong">SequenceOptimizationHints</span></a> - Class in <a href="./org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a></dt>
<dd>
<div class="block">A static class that provides optimization hints for memory or performance handling of sequence data.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/SequenceOptimizationHints.html#SequenceOptimizationHints()">SequenceOptimizationHints()</a></span> - Constructor for class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/SequenceOptimizationHints.html" title="class in org.biojava3.core.sequence">SequenceOptimizationHints</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/SequenceOptimizationHints.SequenceCollection.html" title="enum in org.biojava3.core.sequence"><span class="strong">SequenceOptimizationHints.SequenceCollection</span></a> - Enum in <a href="./org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/SequenceOptimizationHints.SequenceUsage.html" title="enum in org.biojava3.core.sequence"><span class="strong">SequenceOptimizationHints.SequenceUsage</span></a> - Enum in <a href="./org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/alignment/template/SequencePair.html" title="interface in org.biojava3.alignment.template"><span class="strong">SequencePair</span></a>&lt;<a href="./org/biojava3/alignment/template/SequencePair.html" title="type parameter in SequencePair">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava3/alignment/template/SequencePair.html" title="type parameter in SequencePair">C</a>&gt;,<a href="./org/biojava3/alignment/template/SequencePair.html" title="type parameter in SequencePair">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava3/alignment/template/package-summary.html">org.biojava3.alignment.template</a></dt>
<dd>
<div class="block">Defines a data structure for the results of pairwise sequence alignment.</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/io/template/SequenceParserInterface.html" title="interface in org.biojava3.core.sequence.io.template"><span class="strong">SequenceParserInterface</span></a> - Interface in <a href="./org/biojava3/core/sequence/io/template/package-summary.html">org.biojava3.core.sequence.io.template</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/template/SequenceProxyView.html" title="class in org.biojava3.core.sequence.template"><span class="strong">SequenceProxyView</span></a>&lt;<a href="./org/biojava3/core/sequence/template/SequenceProxyView.html" title="type parameter in SequenceProxyView">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/core/sequence/template/package-summary.html">org.biojava3.core.sequence.template</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceProxyView.html#SequenceProxyView()">SequenceProxyView()</a></span> - Constructor for class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceProxyView.html" title="class in org.biojava3.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceProxyView.html#SequenceProxyView(org.biojava3.core.sequence.template.Sequence)">SequenceProxyView(Sequence&lt;C&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceProxyView.html" title="class in org.biojava3.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceProxyView.html#SequenceProxyView(org.biojava3.core.sequence.template.Sequence, java.lang.Integer, java.lang.Integer)">SequenceProxyView(Sequence&lt;C&gt;, Integer, Integer)</a></span> - Constructor for class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceProxyView.html" title="class in org.biojava3.core.sequence.template">SequenceProxyView</a></dt>
<dd>
<div class="block">Main constructor for working with SequenceProxyViews</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/template/SequenceReader.html" title="interface in org.biojava3.core.sequence.template"><span class="strong">SequenceReader</span></a>&lt;<a href="./org/biojava3/core/sequence/template/SequenceReader.html" title="type parameter in SequenceReader">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava3/core/sequence/template/package-summary.html">org.biojava3.core.sequence.template</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html" title="class in org.biojava.bio.structure.gui.util"><span class="strong">SequenceScalePanel</span></a> - Class in <a href="./org/biojava/bio/structure/gui/util/package-summary.html">org.biojava.bio.structure.gui.util</a></dt>
<dd>
<div class="block">A class that draws a Sequence as a rectangle, a scale display over it.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html#SequenceScalePanel(int)">SequenceScalePanel(int)</a></span> - Constructor for class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html" title="class in org.biojava.bio.structure.gui.util">SequenceScalePanel</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/util/SequenceTools.html" title="class in org.biojava3.core.util"><span class="strong">SequenceTools</span></a> - Class in <a href="./org/biojava3/core/util/package-summary.html">org.biojava3.core.util</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/SequenceTools.html#SequenceTools()">SequenceTools()</a></span> - Constructor for class org.biojava3.core.util.<a href="./org/biojava3/core/util/SequenceTools.html" title="class in org.biojava3.core.util">SequenceTools</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/data/sequence/SequenceUtil.html" title="class in org.biojava3.data.sequence"><span class="strong">SequenceUtil</span></a> - Class in <a href="./org/biojava3/data/sequence/package-summary.html">org.biojava3.data.sequence</a></dt>
<dd>
<div class="block">Utility class for operations on sequences</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/template/SequenceView.html" title="interface in org.biojava3.core.sequence.template"><span class="strong">SequenceView</span></a>&lt;<a href="./org/biojava3/core/sequence/template/SequenceView.html" title="type parameter in SequenceView">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava3/core/sequence/template/package-summary.html">org.biojava3.core.sequence.template</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#Ser">Ser</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html#serialize(java.io.File)">serialize(File)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentResult</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/ChooseDirAction.html#serialVersionUID">serialVersionUID</a></span> - Static variable in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/ChooseDirAction.html" title="class in org.biojava.bio.structure.align.gui">ChooseDirAction</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBParseException.html#serialVersionUID">serialVersionUID</a></span> - Static variable in exception org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBParseException.html" title="class in org.biojava.bio.structure.io">PDBParseException</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureException.html#serialVersionUID">serialVersionUID</a></span> - Static variable in exception org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureException.html" title="class in org.biojava.bio.structure">StructureException</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/RemotePairwiseAlignmentOutputProperties.html#serialVersionUID">serialVersionUID</a></span> - Static variable in interface org.biojava3.ws.alignment.<a href="./org/biojava3/ws/alignment/RemotePairwiseAlignmentOutputProperties.html" title="interface in org.biojava3.ws.alignment">RemotePairwiseAlignmentOutputProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/RemotePairwiseAlignmentProperties.html#serialVersionUID">serialVersionUID</a></span> - Static variable in interface org.biojava3.ws.alignment.<a href="./org/biojava3/ws/alignment/RemotePairwiseAlignmentProperties.html" title="interface in org.biojava3.ws.alignment">RemotePairwiseAlignmentProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#set(int, int, double)">set(int, int, double)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Set a single element.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/aaindex/AAindexFactory.html#setAAIndexProvider(org.biojava3.alignment.aaindex.AAIndexProvider)">setAAIndexProvider(AAIndexProvider)</a></span> - Static method in class org.biojava3.alignment.aaindex.<a href="./org/biojava3/alignment/aaindex/AAindexFactory.html" title="class in org.biojava3.alignment.aaindex">AAindexFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html#setAbsorpt_coefficient_mu(java.lang.String)">setAbsorpt_coefficient_mu(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html" title="class in org.biojava.bio.structure.io.mmcif.model">Exptl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html#setAbsorpt_correction_T_max(java.lang.String)">setAbsorpt_correction_T_max(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html" title="class in org.biojava.bio.structure.io.mmcif.model">Exptl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html#setAbsorpt_correction_T_min(java.lang.String)">setAbsorpt_correction_T_min(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html" title="class in org.biojava.bio.structure.io.mmcif.model">Exptl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html#setAbsorpt_correction_type(java.lang.String)">setAbsorpt_correction_type(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html" title="class in org.biojava.bio.structure.io.mmcif.model">Exptl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html#setAbsorpt_process_details(java.lang.String)">setAbsorpt_process_details(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html" title="class in org.biojava.bio.structure.io.mmcif.model">Exptl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html#setAccession(org.biojava3.core.sequence.AccessionID)">setAccession(AccessionID)</a></span> - Method in class org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava3.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#setAccession(org.biojava3.core.sequence.AccessionID)">setAccession(AccessionID)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastAlignmentProperties.html#setAdvancedOptions(java.lang.String)">setAdvancedOptions(String)</a></span> - Method in class org.biojava3.ws.alignment.qblast.<a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastAlignmentProperties.html" title="class in org.biojava3.ws.alignment.qblast">NCBIQBlastAlignmentProperties</a></dt>
<dd>
<div class="block">
 This method is to be used if a request is to use non-default values at
 submission.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setAfpAftIndex(int[][])">setAfpAftIndex(int[][])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setAfpBefIndex(int[][])">setAfpBefIndex(int[][])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AFPChainCoordManager.html#setAFPChain(org.biojava.bio.structure.align.model.AFPChain)">setAFPChain(AFPChain)</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AFPChainCoordManager.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AFPChainCoordManager</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html#setAFPChain(org.biojava.bio.structure.align.model.AFPChain)">setAFPChain(AFPChain)</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AligPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/StatusDisplay.html#setAfpChain(org.biojava.bio.structure.align.model.AFPChain)">setAfpChain(AFPChain)</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/StatusDisplay.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">StatusDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setAfpChainLen(int)">setAfpChainLen(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setAfpChainList(int[])">setAfpChainList(int[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setAfpChainTwiBin(double[])">setAfpChainTwiBin(double[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setAfpChainTwiList(double[])">setAfpChainTwiList(double[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setAfpChainTwiNum(int)">setAfpChainTwiNum(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#setAfpDisCut(double)">setAfpDisCut(double)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#setAfpDisCut0(double)">setAfpDisCut0(double)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setAfpIndex(int[][])">setAfpIndex(int[][])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setAfpSet(java.util.List)">setAfpSet(List&lt;AFP&gt;)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">Set the set of AFPs for this alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html#setAlgorithmName(java.lang.String)">setAlgorithmName(String)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html" title="class in org.biojava.bio.structure.align">CallableStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setAlgorithmName(java.lang.String)">setAlgorithmName(String)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html#setAligMap(java.util.List)">setAligMap(List&lt;AlignedPosition&gt;)</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html" title="class in org.biojava.bio.structure.gui.util">SequenceScalePanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/Alignable.html#setAligMat(org.biojava.bio.structure.align.helper.AligMatEl[][])">setAligMat(AligMatEl[][])</a></span> - Method in interface org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/Alignable.html" title="interface in org.biojava.bio.structure.align.pairwise">Alignable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html#setAligMat(int, int, org.biojava.bio.structure.align.helper.AligMatEl)">setAligMat(int, int, AligMatEl)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">StrCompAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html#setAligMat(org.biojava.bio.structure.align.helper.AligMatEl[][])">setAligMat(AligMatEl[][])</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">StrCompAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#setAlign_id(java.lang.String)">setAlign_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#setAlign_se1(int[])">setAlign_se1(int[])</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#setAlign_se2(int[])">setAlign_se2(int[])</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastAlignmentProperties.html#setAlignementOption(java.lang.String, java.lang.String)">setAlignementOption(String, String)</a></span> - Method in class org.biojava3.ws.alignment.qblast.<a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastAlignmentProperties.html" title="class in org.biojava3.ws.alignment.qblast">NCBIQBlastAlignmentProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/RemotePairwiseAlignmentProperties.html#setAlignementOption(java.lang.String, java.lang.String)">setAlignementOption(String, String)</a></span> - Method in interface org.biojava3.ws.alignment.<a href="./org/biojava3/ws/alignment/RemotePairwiseAlignmentProperties.html" title="interface in org.biojava3.ws.alignment">RemotePairwiseAlignmentProperties</a></dt>
<dd>
<div class="block">Method to set the value for a specific alignment parameter using a key to store in a map.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastOutputProperties.html#setAlignmentNumber(int)">setAlignmentNumber(int)</a></span> - Method in class org.biojava3.ws.alignment.qblast.<a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastOutputProperties.html" title="class in org.biojava3.ws.alignment.qblast">NCBIQBlastOutputProperties</a></dt>
<dd>
<div class="block">A method to set the number of alignments to fetch to the GET command.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastOutputProperties.html#setAlignmentOutputFormat(org.biojava3.ws.alignment.qblast.NCBIQBlastOutputFormat)">setAlignmentOutputFormat(NCBIQBlastOutputFormat)</a></span> - Method in class org.biojava3.ws.alignment.qblast.<a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastOutputProperties.html" title="class in org.biojava3.ws.alignment.qblast">NCBIQBlastOutputProperties</a></dt>
<dd>
<div class="block">This method is use to set the alignment output format to get from the QBlast service</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html#setAlignments(org.biojava.bio.structure.align.pairwise.AlternativeAlignment[])">setAlignments(AlternativeAlignment[])</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentResult</a></dt>
<dd>
<div class="block">we only keep the first alternative...</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#setAlignPairs(java.lang.String)">setAlignPairs(String)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setAlignScore(double)">setAlignScore(double)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setAlignScoreUpdate(double)">setAlignScoreUpdate(double)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileParsingParameters.html#setAlignSeqRes(boolean)">setAlignSeqRes(boolean)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileParsingParameters.html" title="class in org.biojava.bio.structure.io">FileParsingParameters</a></dt>
<dd>
<div class="block">define if the SEQRES in the structure should be aligned with the ATOM records
 if yes, the AminoAcids in structure.getSeqRes will have the coordinates set.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setAlnbeg1(int)">setAlnbeg1(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setAlnbeg2(int)">setAlnbeg2(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setAlnLength(int)">setAlnLength(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setAlnseq1(char[])">setAlnseq1(char[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setAlnseq2(char[])">setAlnseq2(char[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setAlnsymb(char[])">setAlnsymb(char[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#setAltAligNumber(int)">setAltAligNumber(int)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/SequenceDisplay.html#setAlternativeAlignment(org.biojava.bio.structure.align.pairwise.AlternativeAlignment)">setAlternativeAlignment(AlternativeAlignment)</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/SequenceDisplay.html" title="class in org.biojava.bio.structure.gui">SequenceDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/AlternativeAlignmentFrame.html#setAlternativeAlignments(org.biojava.bio.structure.align.pairwise.AlternativeAlignment[])">setAlternativeAlignments(AlternativeAlignment[])</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/AlternativeAlignmentFrame.html" title="class in org.biojava.bio.structure.gui.util">AlternativeAlignmentFrame</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/JMatrixPanel.html#setAlternativeAligs(org.biojava.bio.structure.align.pairwise.AlternativeAlignment[])">setAlternativeAligs(AlternativeAlignment[])</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/JMatrixPanel.html" title="class in org.biojava.bio.structure.gui">JMatrixPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html#setAlternativeAligs(org.biojava.bio.structure.align.pairwise.AlternativeAlignment[])">setAlternativeAligs(AlternativeAlignment[])</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html" title="class in org.biojava.bio.structure.gui">ScaleableMatrixPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#setAltLoc(java.lang.Character)">setAltLoc(Character)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">get set alternate Location.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#setAltLoc(java.lang.Character)">setAltLoc(Character)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>
<div class="block">set alternate Location.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/AminoAcidCompositionTable.html#setAminoacid(java.util.List)">setAminoacid(List&lt;AminoAcidComposition&gt;)</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/AminoAcidCompositionTable.html" title="class in org.biojava3.aaproperties.xml">AminoAcidCompositionTable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcid.html#setAminoType(java.lang.Character)">setAminoType(Character)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcid.html" title="interface in org.biojava.bio.structure">AminoAcid</a></dt>
<dd>
<div class="block">set the name of the AA, in single letter code .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcidImpl.html#setAminoType(java.lang.Character)">setAminoType(Character)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcidImpl.html" title="class in org.biojava.bio.structure">AminoAcidImpl</a></dt>
<dd>
<div class="block">set the name of the AA, in single letter code .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/routines/AnchoredPairwiseSequenceAligner.html#setAnchors(int[])">setAnchors(int[])</a></span> - Method in class org.biojava3.alignment.routines.<a href="./org/biojava3/alignment/routines/AnchoredPairwiseSequenceAligner.html" title="class in org.biojava3.alignment.routines">AnchoredPairwiseSequenceAligner</a></dt>
<dd>
<div class="block">Sets the starting list of anchors before running the alignment routine.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setAngleDiff(int)">setAngleDiff(int)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setAniso_B(java.lang.String[][])">setAniso_B(String[][])</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#setAnnotationType(org.biojava3.core.sequence.template.AbstractSequence.AnnotationType)">setAnnotationType(AbstractSequence.AnnotationType)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#setAsym_id(java.lang.String)">setAsym_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#setAsym_id(java.lang.String)">setAsym_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setAtcc(java.lang.String)">setAtcc(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#setAtomGroups(java.util.List)">setAtomGroups(List&lt;Group&gt;)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Set all groups that have been specified in the ATOM section of this chain .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#setAtomGroups(java.util.List)">setAtomGroups(List&lt;Group&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Set all groups that have been specified in the ATOM section of this chain .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/Element.html#setAtomicNumber(int)">setAtomicNumber(int)</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/Element.html" title="class in org.biojava3.aaproperties.xml">Element</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ModifiedCompound.html#setAtomLinkages(java.util.Set)">setAtomLinkages(Set&lt;StructureAtomLinkage&gt;)</a></span> - Method in interface org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ModifiedCompound.html" title="interface in org.biojava3.protmod.structure">ModifiedCompound</a></dt>
<dd>
<div class="block">Set atom linkages</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ModifiedCompoundImpl.html#setAtomLinkages(java.util.Set)">setAtomLinkages(Set&lt;StructureAtomLinkage&gt;)</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ModifiedCompoundImpl.html" title="class in org.biojava3.protmod.structure">ModifiedCompoundImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/AtomInfo.html#setAtomName(java.lang.String)">setAtomName(String)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/AtomInfo.html" title="class in org.biojava.bio.structure.align.gui.jmol">AtomInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html#setAtoms(org.biojava.bio.structure.Atom[])">setAtoms(Atom[])</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html" title="class in org.biojava.bio.structure.align.gui.jmol">StructureAlignmentJmol</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#setAtoms(java.util.List)">setAtoms(List&lt;Atom&gt;)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">Set the atoms of this group.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#setAtoms(java.util.List)">setAtoms(List&lt;Atom&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">set the atoms of this group</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setAuth_asym_id(java.lang.String)">setAuth_asym_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setAuth_atom_id(java.lang.String)">setAuth_atom_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setAuth_comp_id(java.lang.String)">setAuth_comp_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#setAuth_mon_id(java.lang.String)">setAuth_mon_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#setAuth_mon_id(java.lang.String)">setAuth_mon_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setAuth_seq_id(java.lang.String)">setAuth_seq_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#setAuth_seq_num(java.lang.String)">setAuth_seq_num(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#setAuth_seq_num(java.lang.String)">setAuth_seq_num(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#setAuthorList(java.util.List)">setAuthorList(List&lt;Author&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#setAuthors(java.lang.String)">setAuthors(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#setAutoFetch(boolean)">setAutoFetch(boolean)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#setAutoFetch(boolean)">setAutoFetch(boolean)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>
<div class="block">Does the cache automatically download files that are missing from the local installation from the PDB FTP site?</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/UserConfiguration.html#setAutoFetch(boolean)">setAutoFetch(boolean)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/UserConfiguration.html" title="class in org.biojava.bio.structure.align.util">UserConfiguration</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/MMCIFFileReader.html#setAutoFetch(boolean)">setAutoFetch(boolean)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/MMCIFFileReader.html" title="class in org.biojava.bio.structure.io">MMCIFFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#setAutoFetch(boolean)">setAutoFetch(boolean)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/StructureIOFile.html#setAutoFetch(boolean)">setAutoFetch(boolean)</a></span> - Method in interface org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/StructureIOFile.html" title="interface in org.biojava.bio.structure.io">StructureIOFile</a></dt>
<dd>
<div class="block">Tell the parser to fetch missing PDB files from the FTP server automatically.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html#setAutoFetch(boolean)">setAutoFetch(boolean)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/MMCIFFileInstallation.html" title="class in org.biojava.bio.structure.server">MMCIFFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
<div class="block">tell the parser to fetch missing mmCif files from the RCSB FTP server automatically.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setB_iso_max(java.lang.String)">setB_iso_max(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setB_iso_mean(java.lang.String)">setB_iso_mean(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setB_iso_min(java.lang.String)">setB_iso_min(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setB_iso_or_equiv(java.lang.String)">setB_iso_or_equiv(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setB_iso_or_equiv_esd(java.lang.String)">setB_iso_or_equiv_esd(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#setBadRmsd(double)">setBadRmsd(double)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html#setBetweenCompounds(boolean)">setBetweenCompounds(boolean)</a></span> - Method in class org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava3.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#setBioAssemblyFallback(boolean)">setBioAssemblyFallback(boolean)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>
<div class="block">Set bioAssemblyFallback to true, to download the original PDB file in cases that a biological assembly file is not available.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#setBioAssemblyId(int)">setBioAssemblyId(int)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>
<div class="block">Sets the ID of the biological assembly to be loaded.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#setBioBegin(java.lang.Integer)">setBioBegin(Integer)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#setBioEnd(java.lang.Integer)">setBioEnd(Integer)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceProxyView.html#setBioEnd(java.lang.Integer)">setBioEnd(Integer)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceProxyView.html" title="class in org.biojava3.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html#setBiol_id(java.lang.String)">setBiol_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRef</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#setBiologicalAssembly(boolean)">setBiologicalAssembly(boolean)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">Sets a flag to indicate if this structure is a biological assembly</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#setBiologicalAssembly(boolean)">setBiologicalAssembly(boolean)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">Sets a flag to indicate if this structure is a biological assembly</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setBiologicalUnit(java.lang.String)">setBiologicalUnit(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceProxyView.html#setBioStart(java.lang.Integer)">setBioStart(Integer)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceProxyView.html" title="class in org.biojava3.core.sequence.template">SequenceProxyView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastAlignmentProperties.html#setBlastDatabase(java.lang.String)">setBlastDatabase(String)</a></span> - Method in class org.biojava3.ws.alignment.qblast.<a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastAlignmentProperties.html" title="class in org.biojava3.ws.alignment.qblast">NCBIQBlastAlignmentProperties</a></dt>
<dd>
<div class="block">This method set the database to be used with blastall</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastAlignmentProperties.html#setBlastExpect(double)">setBlastExpect(double)</a></span> - Method in class org.biojava3.ws.alignment.qblast.<a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastAlignmentProperties.html" title="class in org.biojava3.ws.alignment.qblast">NCBIQBlastAlignmentProperties</a></dt>
<dd>
<div class="block">This method set the EXPECT parameter to be use with blastall
 
 Example: if you want a EXPECT of 1e-10, try this
 
   setBlastExpect(Double.parseDouble("1e-10"))</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastAlignmentProperties.html#setBlastFromPosition(int)">setBlastFromPosition(int)</a></span> - Method in class org.biojava3.ws.alignment.qblast.<a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastAlignmentProperties.html" title="class in org.biojava3.ws.alignment.qblast">NCBIQBlastAlignmentProperties</a></dt>
<dd>
<div class="block">This method set the QUERY_FROM parameter to be use by blast.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastAlignmentProperties.html#setBlastProgram(java.lang.String)">setBlastProgram(String)</a></span> - Method in class org.biojava3.ws.alignment.qblast.<a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastAlignmentProperties.html" title="class in org.biojava3.ws.alignment.qblast">NCBIQBlastAlignmentProperties</a></dt>
<dd>
<div class="block">This method set the program to be use with blastall.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastAlignmentProperties.html#setBlastToPosition(int)">setBlastToPosition(int)</a></span> - Method in class org.biojava3.ws.alignment.qblast.<a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastAlignmentProperties.html" title="class in org.biojava3.ws.alignment.qblast">NCBIQBlastAlignmentProperties</a></dt>
<dd>
<div class="block">This method set the QUERY_TO parameter to be use by blast.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastAlignmentProperties.html#setBlastWordSize(int)">setBlastWordSize(int)</a></span> - Method in class org.biojava3.ws.alignment.qblast.<a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastAlignmentProperties.html" title="class in org.biojava3.ws.alignment.qblast">NCBIQBlastAlignmentProperties</a></dt>
<dd>
<div class="block">This method set the WORD_SIZE parameter to be use with blastall</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setBlock2Afp(int[])">setBlock2Afp(int[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setBlockGap(int[])">setBlockGap(int[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setBlockNum(int)">setBlockNum(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setBlockNumClu(int)">setBlockNumClu(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setBlockNumIni(int)">setBlockNumIni(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setBlockNumSpt(int)">setBlockNumSpt(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setBlockResList(int[][][])">setBlockResList(int[][][])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setBlockResSize(int[])">setBlockResSize(int[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setBlockRmsd(double[])">setBlockRmsd(double[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setBlockRotationMatrix(org.biojava.bio.structure.jama.Matrix[])">setBlockRotationMatrix(Matrix[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setBlockScore(double[])">setBlockScore(double[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setBlockShiftVector(org.biojava.bio.structure.Atom[])">setBlockShiftVector(Atom[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setBlockSize(int[])">setBlockSize(int[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ProteinModificationIdentifier.html#setbondLengthTolerance(double)">setbondLengthTolerance(double)</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ProteinModificationIdentifier.html" title="class in org.biojava3.protmod.structure">ProteinModificationIdentifier</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html#setC(org.biojava.bio.structure.Atom)">setC(Atom)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html" title="class in org.biojava.bio.structure.secstruc">SecStrucGroup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html#setCA(org.biojava.bio.structure.Atom)">setCA(Atom)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html" title="class in org.biojava.bio.structure.secstruc">SecStrucGroup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html#setCa1(org.biojava.bio.structure.Atom[])">setCa1(Atom[])</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html" title="class in org.biojava.bio.structure.align">CallableStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html#setCa1(org.biojava.bio.structure.Atom[])">setCa1(Atom[])</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AligPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/StatusDisplay.html#setCa1(org.biojava.bio.structure.Atom[])">setCa1(Atom[])</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/StatusDisplay.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">StatusDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setCa1Length(int)">setCa1Length(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html#setCa2(org.biojava.bio.structure.Atom[])">setCa2(Atom[])</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AligPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/StatusDisplay.html#setCa2(org.biojava.bio.structure.Atom[])">setCa2(Atom[])</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/StatusDisplay.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">StatusDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setCa2Length(int)">setCa2Length(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html#setCache(org.biojava.bio.structure.align.util.AtomCache)">setCache(AtomCache)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html" title="class in org.biojava.bio.structure.align">CallableStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#setCacheLocation(java.lang.String)">setCacheLocation(String)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/SimpleStructureServer.html#setCacheSize(int)">setCacheSize(int)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/SimpleStructureServer.html" title="class in org.biojava.bio.structure.server">SimpleStructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureServer.html#setCacheSize(int)">setCacheSize(int)</a></span> - Method in interface org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureServer.html" title="interface in org.biojava.bio.structure.server">StructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
<div class="block">set how many structures should be kept in memory - for quicker access</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setCalculationTime(long)">setCalculationTime(long)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html#setCalculationTime(long)">setCalculationTime(long)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentResult</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/MyJmolStatusListener.html#setCallbackFunction(java.lang.String, java.lang.String)">setCallbackFunction(String, String)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/MyJmolStatusListener.html" title="class in org.biojava.bio.structure.align.gui.jmol">MyJmolStatusListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setCartn_x(java.lang.String)">setCartn_x(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setCartn_x_esd(java.lang.String)">setCartn_x_esd(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setCartn_y(java.lang.String)">setCartn_y(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setCartn_y_esd(java.lang.String)">setCartn_y_esd(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setCartn_z(java.lang.String)">setCartn_z(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setCartn_z_esd(java.lang.String)">setCartn_z_esd(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDescription.html#setCategory(org.biojava.bio.structure.scop.ScopCategory)">setCategory(ScopCategory)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDescription.html" title="class in org.biojava.bio.structure.scop">ScopDescription</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationImpl.Builder.html#setCategory(org.biojava3.protmod.ModificationCategory)">setCategory(ModificationCategory)</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationImpl.Builder.html" title="class in org.biojava3.protmod">ProteinModificationImpl.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setCell(java.lang.String)">setCell(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/JMatrixPanel.html#setCellColor(org.biojava.bio.structure.gui.util.color.ContinuousColorMapper)">setCellColor(ContinuousColorMapper)</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/JMatrixPanel.html" title="class in org.biojava.bio.structure.gui">JMatrixPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html#setCellColor(org.biojava.bio.structure.gui.util.color.ContinuousColorMapper)">setCellColor(ContinuousColorMapper)</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html" title="class in org.biojava.bio.structure.gui">ScaleableMatrixPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setCellLine(java.lang.String)">setCellLine(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setCellularLocation(java.lang.String)">setCellularLocation(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#setCenter1(org.biojava.bio.structure.Atom)">setCenter1(Atom)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#setCenter2(org.biojava.bio.structure.Atom)">setCenter2(Atom)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#setChain(org.biojava.bio.structure.Chain)">setChain(Chain)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">Sets the back-reference to its parent Chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/SequenceMouseListener.html#setChain(org.biojava.bio.structure.Chain)">setChain(Chain)</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/SequenceMouseListener.html" title="class in org.biojava.bio.structure.gui.util">SequenceMouseListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html#setChain(org.biojava.bio.structure.Chain)">setChain(Chain)</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html" title="class in org.biojava.bio.structure.gui.util">SequenceScalePanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#setChain(org.biojava.bio.structure.Chain)">setChain(Chain)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">Sets the back-reference to its parent Chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html#setChain1(java.lang.String)">setChain1(String)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentResult</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html#setChain2(java.lang.String)">setChain2(String)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentResult</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/AtomInfo.html#setChainId(java.lang.String)">setChainId(String)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/AtomInfo.html" title="class in org.biojava.bio.structure.align.gui.jmol">AtomInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#setChainID(java.lang.String)">setChainID(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">get and set the name of this chain (Chain id in PDB file ).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#setChainID(java.lang.String)">setChainID(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">get and set the name of this chain (Chain id in PDB file ).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setChainId(java.util.List)">setChainId(List&lt;String&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>
<div class="block">Sets the list of chain IDs.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#setChainId(java.lang.Character)">setChainId(Character)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">The chain ID of the corresponding chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ResidueNumber.html#setChainId(java.lang.String)">setChainId(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ResidueNumber.html" title="class in org.biojava.bio.structure">ResidueNumber</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/StructureGroup.html#setChainId(java.lang.String)">setChainId(String)</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/StructureGroup.html" title="class in org.biojava3.protmod.structure">StructureGroup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SSBond.html#setChainID1(java.lang.String)">setChainID1(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SSBond.html" title="class in org.biojava.bio.structure">SSBond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SSBond.html#setChainID2(java.lang.String)">setChainID2(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SSBond.html" title="class in org.biojava.bio.structure">SSBond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/FlatFileInstallation.html#setChainInfoFile(java.io.File)">setChainInfoFile(File)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/FlatFileInstallation.html" title="class in org.biojava.bio.structure.server">FlatFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setChainLen(int)">setChainLen(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setChainRmsd(double)">setChainRmsd(double)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">The RMSD of the chain of AFPs.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setChains(java.util.List)">setChains(List&lt;Chain&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#setChains(java.util.List)">setChains(List&lt;Chain&gt;)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">set the chains of a structure, if this is a NMR structure,
 this will only set model 0.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#setChains(int, java.util.List)">setChains(int, List&lt;Chain&gt;)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">set the chains for a model</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#setChains(java.util.List)">setChains(List&lt;Chain&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#setChains(int, java.util.List)">setChains(int, List&lt;Chain&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#setChemComp(org.biojava.bio.structure.io.mmcif.model.ChemComp)">setChemComp(ChemComp)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">Set the Chemical Component that closer describes this group.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#setChemComp(org.biojava.bio.structure.io.mmcif.model.ChemComp)">setChemComp(ChemComp)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompGroupFactory.html#setChemCompProvider(org.biojava.bio.structure.io.mmcif.ChemCompProvider)">setChemCompProvider(ChemCompProvider)</a></span> - Static method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompGroupFactory.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompGroupFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopNode.html#setChildren(java.util.List)">setChildren(List&lt;Integer&gt;)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopNode.html" title="class in org.biojava.bio.structure.scop">ScopNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/AbstractFeature.html#setChildrenFeatures(java.util.List)">setChildrenFeatures(List&lt;FeatureInterface&lt;S, C&gt;&gt;)</a></span> - Method in class org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/AbstractFeature.html" title="class in org.biojava3.core.sequence.features">AbstractFeature</a></dt>
<dd>
<div class="block">Set the children features</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/FeatureInterface.html#setChildrenFeatures(java.util.List)">setChildrenFeatures(List&lt;FeatureInterface&lt;S, C&gt;&gt;)</a></span> - Method in interface org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/FeatureInterface.html" title="interface in org.biojava3.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">Set the children features</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/ChromosomeSequence.html#setChromosomeNumber(int)">setChromosomeNumber(int)</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/ChromosomeSequence.html" title="class in org.biojava3.core.sequence">ChromosomeSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html#setCircular(boolean)">setCircular(boolean)</a></span> - Method in class org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava3.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#setClassId(int)">setClassId(int)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#setClassification(java.lang.String)">setClassification(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDescription.html#setClassificationId(java.lang.String)">setClassificationId(String)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDescription.html" title="class in org.biojava.bio.structure.scop">ScopDescription</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#setClassificationId(java.lang.String)">setClassificationId(String)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#setCluster(int)">setCluster(int)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>
<div class="block">set the number of the cluster this alignment belongs to.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#setCluster(int)">setCluster(int)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/IndexPair.html#setCol(short)">setCol(short)</a></span> - Method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/IndexPair.html" title="class in org.biojava.bio.structure.align.helper">IndexPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/structure/gui/JmolViewerImpl.html#setColor(java.awt.Color)">setColor(Color)</a></span> - Method in class org.biojava3.structure.gui.<a href="./org/biojava3/structure/gui/JmolViewerImpl.html" title="class in org.biojava3.structure.gui">JmolViewerImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/structure/gui/StructureViewer.html#setColor(java.awt.Color)">setColor(Color)</a></span> - Method in interface org.biojava3.structure.gui.<a href="./org/biojava3/structure/gui/StructureViewer.html" title="interface in org.biojava3.structure.gui">StructureViewer</a></dt>
<dd>
<div class="block">Apply this color to the current Selection</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/LinearColorInterpolator.html#setColorSpace(java.awt.color.ColorSpace, org.biojava.bio.structure.gui.util.color.LinearColorInterpolator.InterpolationDirection[])">setColorSpace(ColorSpace, LinearColorInterpolator.InterpolationDirection[])</a></span> - Method in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/LinearColorInterpolator.html" title="class in org.biojava.bio.structure.gui.util.color">LinearColorInterpolator</a></dt>
<dd>
<div class="block">Sets the ColorSpace to use for interpolation.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/LinearColorInterpolator.html#setColorSpace(java.awt.color.ColorSpace)">setColorSpace(ColorSpace)</a></span> - Method in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/LinearColorInterpolator.html" title="class in org.biojava.bio.structure.gui.util.color">LinearColorInterpolator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html#setCols(java.util.List)">setCols(List&lt;AminoAcidCompound&gt;)</a></span> - Method in class org.biojava3.alignment.aaindex.<a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava3.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html#setComp_id(java.lang.String)">setComp_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemCompDescriptor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxEntityNonPoly.html#setComp_id(java.lang.String)">setComp_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxEntityNonPoly.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxEntityNonPoly</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#setCompoundAt(char, int)">setCompoundAt(char, int)</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava3.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>
<div class="block">Converts from char to Compound and sets it at the given biological index</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html#setCompoundAt(C, int)">setCompoundAt(C, int)</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/BitSequenceReader.BitArrayWorker.html" title="class in org.biojava3.core.sequence.storage">BitSequenceReader.BitArrayWorker</a></dt>
<dd>
<div class="block">Sets the compound at the specified biological index</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#setCompounds(java.util.List)">setCompounds(List&lt;Compound&gt;)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">set the compounts</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#setCompounds(java.util.List)">setCompounds(List&lt;Compound&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html#setCompoundSet(org.biojava3.core.sequence.template.CompoundSet)">setCompoundSet(CompoundSet&lt;C&gt;)</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava3.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/StringProxySequenceReader.html#setCompoundSet(org.biojava3.core.sequence.template.CompoundSet)">setCompoundSet(CompoundSet&lt;C&gt;)</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html#setCompoundSet(org.biojava3.core.sequence.template.CompoundSet)">setCompoundSet(CompoundSet&lt;C&gt;)</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/ArrayListSequenceReader.html#setCompoundSet(org.biojava3.core.sequence.template.CompoundSet)">setCompoundSet(CompoundSet&lt;C&gt;)</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava3.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/BitSequenceReader.html#setCompoundSet(org.biojava3.core.sequence.template.CompoundSet)">setCompoundSet(CompoundSet&lt;C&gt;)</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava3.core.sequence.storage">BitSequenceReader</a></dt>
<dd>
<div class="block">Class is immutable & so this is unsupported</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/JoiningSequenceReader.html#setCompoundSet(org.biojava3.core.sequence.template.CompoundSet)">setCompoundSet(CompoundSet&lt;C&gt;)</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava3.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html#setCompoundSet(org.biojava3.core.sequence.template.CompoundSet)">setCompoundSet(CompoundSet&lt;C&gt;)</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava3.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Unsupported</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#setCompoundSet(org.biojava3.core.sequence.template.CompoundSet)">setCompoundSet(CompoundSet&lt;C&gt;)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceReader.html#setCompoundSet(org.biojava3.core.sequence.template.CompoundSet)">setCompoundSet(CompoundSet&lt;C&gt;)</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceReader.html" title="interface in org.biojava3.core.sequence.template">SequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/views/RnaSequenceView.html#setCompoundSet(org.biojava3.core.sequence.template.CompoundSet)">setCompoundSet(CompoundSet&lt;NucleotideCompound&gt;)</a></span> - Method in class org.biojava3.core.sequence.views.<a href="./org/biojava3/core/sequence/views/RnaSequenceView.html" title="class in org.biojava3.core.sequence.views">RnaSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationImpl.Builder.html#setCondition(org.biojava3.protmod.ModificationCondition)">setCondition(ModificationCondition)</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationImpl.Builder.html" title="class in org.biojava3.protmod">ProteinModificationImpl.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setConn(java.lang.Double)">setConn(Double)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#setConnections(java.util.List)">setConnections(List&lt;Map&lt;String, Integer&gt;&gt;)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">sets/gets an List of  Maps which corresponds to the CONECT lines in the PDB file:

       
       COLUMNS         DATA TYPE        FIELD           DEFINITION
       ---------------------------------------------------------------------------------
        1 -  6         Record name      "CONECT"
        7 - 11         Integer          serial          Atom serial number
       12 - 16         Integer          serial          Serial number of bonded atom
       17 - 21         Integer          serial          Serial number of bonded atom
       22 - 26         Integer          serial          Serial number of bonded atom
       27 - 31         Integer          serial          Serial number of bonded atom
       32 - 36         Integer          serial          Serial number of hydrogen bonded
       atom
       37 - 41         Integer          serial          Serial number of hydrogen bonded
       atom
       42 - 46         Integer          serial          Serial number of salt bridged
       atom
       47 - 51         Integer          serial          Serial number of hydrogen bonded
       atom
       52 - 56         Integer          serial          Serial number of hydrogen bonded
       atom
       57 - 61         Integer          serial          Serial number of salt bridged
       atom
       

       the HashMap for a single CONECT line contains the following fields:

        atomserial (mandatory) : Atom serial number
        bond1 .. bond4 (optional): Serial number of bonded atom
        hydrogen1 .. hydrogen4 (optional):Serial number of hydrogen bonded atom
        salt1 .. salt2 (optional): Serial number of salt bridged atom</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#setConnections(java.util.List)">setConnections(List&lt;Map&lt;String, Integer&gt;&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#setContacts(int)">setContacts(int)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html#setContents(java.lang.String)">setContents(String)</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava3.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/StringProxySequenceReader.html#setContents(java.lang.String)">setContents(String)</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html#setContents(java.lang.String)">setContents(String)</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>
<div class="block">Once the sequence is retrieved set the contents and make sure everything this is valid</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/ArrayListSequenceReader.html#setContents(java.lang.String)">setContents(String)</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava3.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/ArrayListSequenceReader.html#setContents(java.util.List)">setContents(List&lt;C&gt;)</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/ArrayListSequenceReader.html" title="class in org.biojava3.core.sequence.storage">ArrayListSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/BitSequenceReader.html#setContents(java.lang.String)">setContents(String)</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/BitSequenceReader.html" title="class in org.biojava3.core.sequence.storage">BitSequenceReader</a></dt>
<dd>
<div class="block">Class is immutable & so this is unsupported</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/JoiningSequenceReader.html#setContents(java.lang.String)">setContents(String)</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/JoiningSequenceReader.html" title="class in org.biojava3.core.sequence.storage">JoiningSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html#setContents(java.lang.String)">setContents(String)</a></span> - Method in class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava3.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Unsupported</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceReader.html#setContents(java.lang.String)">setContents(String)</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceReader.html" title="interface in org.biojava3.core.sequence.template">SequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/views/RnaSequenceView.html#setContents(java.lang.String)">setContents(String)</a></span> - Method in class org.biojava3.core.sequence.views.<a href="./org/biojava3/core/sequence/views/RnaSequenceView.html" title="class in org.biojava3.core.sequence.views">RnaSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/AligMatEl.html#setContig(int)">setContig(int)</a></span> - Method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/AligMatEl.html" title="class in org.biojava.bio.structure.align.helper">AligMatEl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#setCoords(double[])">setCoords(double[])</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">set the coordinates.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#setCoords(double[])">setCoords(double[])</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>
<div class="block">the coordinates.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setCorrelation_coeff_Fo_to_Fc(java.lang.String)">setCorrelation_coeff_Fo_to_Fc(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setCorrelation_coeff_Fo_to_Fc_free(java.lang.String)">setCorrelation_coeff_Fo_to_Fc_free(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#setCovered(int)">setCovered(int)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setCreate_co(float)">setCreate_co(float)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html#setCrystals_number(java.lang.String)">setCrystals_number(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html" title="class in org.biojava.bio.structure.io.mmcif.model">Exptl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/CutSites.html#setCut_sites(int[])">setCut_sites(int[])</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/CutSites.html" title="class in org.biojava.bio.structure.domain.pdp">CutSites</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/routines/AlignerHelper.html#setCuts(int, int[], org.biojava3.alignment.routines.AlignerHelper.Last[][], org.biojava3.alignment.routines.AlignerHelper.Cut[])">setCuts(int, int[], AlignerHelper.Last[][], AlignerHelper.Cut[])</a></span> - Static method in class org.biojava3.alignment.routines.<a href="./org/biojava3/alignment/routines/AlignerHelper.html" title="class in org.biojava3.alignment.routines">AlignerHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/routines/AnchoredPairwiseSequenceAligner.html#setCutsPerSection(int)">setCutsPerSection(int)</a></span> - Method in class org.biojava3.alignment.routines.<a href="./org/biojava3/alignment/routines/AnchoredPairwiseSequenceAligner.html" title="class in org.biojava3.alignment.routines">AnchoredPairwiseSequenceAligner</a></dt>
<dd>
<div class="block">Sets the number of cuts added to each section during each pass.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#setDatabase(java.lang.String)">setDatabase(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Specifies the database value.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/DBReferenceInfo.html#setDatabase(java.lang.String)">setDatabase(String)</a></span> - Method in class org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/DBReferenceInfo.html" title="class in org.biojava3.core.sequence.features">DBReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#setDatabaseReferences(org.biojava3.core.sequence.features.DatabaseReferenceInterface)">setDatabaseReferences(DatabaseReferenceInterface)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html#setDate(java.lang.String)">setDate(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html" title="class in org.biojava.bio.structure.io.mmcif.model">DatabasePDBrev</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html#setDate_original(java.lang.String)">setDate_original(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html" title="class in org.biojava.bio.structure.io.mmcif.model">DatabasePDBrev</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#setDb_align_beg(java.lang.String)">setDb_align_beg(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#setDb_align_end(java.lang.String)">setDb_align_end(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html#setDb_code(java.lang.String)">setDb_code(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRef</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html#setDb_name(java.lang.String)">setDb_name(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRef</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#setDbAccession(java.lang.String)">setDbAccession(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Sequence database accession code.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#setDbIdCode(java.lang.String)">setDbIdCode(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Sequence database          identification code.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#setDBRefs(java.util.List)">setDBRefs(List&lt;DBRef&gt;)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">set the list of database references for this structure</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#setDBRefs(java.util.List)">setDBRefs(List&lt;DBRef&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#setDbSeqBegin(int)">setDbSeqBegin(int)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Initial sequence number of the
    database seqment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#setDbSeqEnd(int)">setDbSeqEnd(int)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">The begin of the sequence position in the database</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructurePairAligner.html#setDebug(boolean)">setDebug(boolean)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructurePairAligner.html" title="class in org.biojava.bio.structure.align">StructurePairAligner</a></dt>
<dd>
<div class="block">set the debug flag</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/SeqRes2AtomAligner.html#setDEBUG(boolean)">setDEBUG(boolean)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/SeqRes2AtomAligner.html" title="class in org.biojava.bio.structure.io">SeqRes2AtomAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/ParameterGUI.html#setDefault()">setDefault()</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/ParameterGUI.html" title="class in org.biojava.bio.structure.align.gui">ParameterGUI</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileParsingParameters.html#setDefault()">setDefault()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileParsingParameters.html" title="class in org.biojava.bio.structure.io">FileParsingParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/routines/AnchoredPairwiseSequenceAligner.html#setDefaultCutsPerSection(int)">setDefaultCutsPerSection(int)</a></span> - Static method in class org.biojava3.alignment.routines.<a href="./org/biojava3/alignment/routines/AnchoredPairwiseSequenceAligner.html" title="class in org.biojava3.alignment.routines">AnchoredPairwiseSequenceAligner</a></dt>
<dd>
<div class="block">Sets the default number of cuts added to each section during each pass.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleGapPenalty.html#setDefaultExtensionPenalty(short)">setDefaultExtensionPenalty(short)</a></span> - Static method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleGapPenalty.html" title="class in org.biojava3.alignment">SimpleGapPenalty</a></dt>
<dd>
<div class="block">Sets the default gap extension penalty.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleGapPenalty.html#setDefaultOpenPenalty(short)">setDefaultOpenPenalty(short)</a></span> - Static method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleGapPenalty.html" title="class in org.biojava3.alignment">SimpleGapPenalty</a></dt>
<dd>
<div class="block">Sets the default gap open penalty.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setDensityCutoff(float)">setDensityCutoff(float)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#setDepDate(java.util.Date)">setDepDate(Date)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#setDescription(java.lang.String)">setDescription(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDescription.html#setDescription(java.lang.String)">setDescription(String)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDescription.html" title="class in org.biojava.bio.structure.scop">ScopDescription</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Site.html#setDescription(java.lang.String)">setDescription(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Site.html" title="class in org.biojava.bio.structure">Site</a></dt>
<dd>
<div class="block">sets the REMARK 800 description of the site</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html#setDescription(java.lang.String)">setDescription(String)</a></span> - Method in class org.biojava3.alignment.aaindex.<a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava3.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleSubstitutionMatrix.html#setDescription(java.lang.String)">setDescription(String)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleSubstitutionMatrix.html" title="class in org.biojava3.alignment">SimpleSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/SubstitutionMatrix.html#setDescription(java.lang.String)">setDescription(String)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/SubstitutionMatrix.html" title="interface in org.biojava3.alignment.template">SubstitutionMatrix</a></dt>
<dd>
<div class="block">Sets the description of this matrix.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/CodonCompound.html#setDescription(java.lang.String)">setDescription(String)</a></span> - Method in class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/CodonCompound.html" title="class in org.biojava3.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/AbstractFeature.html#setDescription(java.lang.String)">setDescription(String)</a></span> - Method in class org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/AbstractFeature.html" title="class in org.biojava3.core.sequence.features">AbstractFeature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/FeatureInterface.html#setDescription(java.lang.String)">setDescription(String)</a></span> - Method in interface org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/FeatureInterface.html" title="interface in org.biojava3.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">Set the description that can be used to describe the feature</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompound.html#setDescription(java.lang.String)">setDescription(String)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompound.html" title="class in org.biojava3.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#setDescription(java.lang.String)">setDescription(String)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/Compound.html#setDescription(java.lang.String)">setDescription(String)</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/CaseInsensitiveCompound.html#setDescription(java.lang.String)">setDescription(String)</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava3.core.sequence.transcription">CaseInsensitiveCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/Table.Codon.html#setDescription(java.lang.String)">setDescription(String)</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/Table.Codon.html" title="class in org.biojava3.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationImpl.Builder.html#setDescription(java.lang.String)">setDescription(String)</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationImpl.Builder.html" title="class in org.biojava3.protmod">ProteinModificationImpl.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ModifiedCompound.html#setDescription(java.lang.String)">setDescription(String)</a></span> - Method in interface org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ModifiedCompound.html" title="interface in org.biojava3.protmod.structure">ModifiedCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ModifiedCompoundImpl.html#setDescription(java.lang.String)">setDescription(String)</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ModifiedCompoundImpl.html" title="class in org.biojava3.protmod.structure">ModifiedCompoundImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastOutputProperties.html#setDescriptionNumber(int)">setDescriptionNumber(int)</a></span> - Method in class org.biojava3.ws.alignment.qblast.<a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastOutputProperties.html" title="class in org.biojava3.ws.alignment.qblast">NCBIQBlastOutputProperties</a></dt>
<dd>
<div class="block">A method to set the number of descriptions to fetch to the GET command.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html#setDescriptor(java.lang.String)">setDescriptor(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemCompDescriptor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setDescriptors(java.util.List)">setDescriptors(List&lt;ChemCompDescriptor&gt;)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setDetails(java.lang.String)">setDetails(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html#setDetails(java.lang.String)">setDetails(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model">Entity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html#setDetails(java.lang.String)">setDetails(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html" title="class in org.biojava.bio.structure.io.mmcif.model">Exptl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setDetails(java.lang.String)">setDetails(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html#setDetails(java.lang.String)">setDetails(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructAsym</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#setDetails(java.lang.String)">setDetails(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setDiagonalDistance(int)">setDiagonalDistance(int)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setDiagonalDistance2(int)">setDiagonalDistance2(int)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#setDisCut(java.lang.Double)">setDisCut(Double)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#setDisFilter(double)">setDisFilter(double)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#setDisSmooth(double)">setDisSmooth(double)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPDistanceMatrix.html#setDist(int[][])">setDist(int[][])</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPDistanceMatrix.html" title="class in org.biojava.bio.structure.domain.pdp">PDPDistanceMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#setDist1(double[][])">setDist1(double[][])</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#setDist2(double[][])">setDist2(double[][])</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setDisTable1(org.biojava.bio.structure.jama.Matrix)">setDisTable1(Matrix)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setDisTable2(org.biojava.bio.structure.jama.Matrix)">setDisTable2(Matrix)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/DistEn.html#setDistance(double)">setDistance(double)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/DistEn.html" title="class in org.biojava.bio.structure.secstruc">DistEn</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#setDistanceIncrement(java.lang.Double)">setDistanceIncrement(Double)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setDistanceMatrix(org.biojava.bio.structure.jama.Matrix)">setDistanceMatrix(Matrix)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">A ca1length x ca2length matrix.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#setDistanceMatrix(org.biojava.bio.structure.jama.Matrix)">setDistanceMatrix(Matrix)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>
<div class="block">The distance matrix this alignment is based on</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/DNASequence.html#setDNAType(org.biojava3.core.sequence.DNASequence.DNAType)">setDNAType(DNASequence.DNAType)</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/DNASequence.html" title="class in org.biojava3.core.sequence">DNASequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setDoAngleCheck(boolean)">setDoAngleCheck(boolean)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setDoDensityCheck(boolean)">setDoDensityCheck(boolean)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setDoDistanceCheck(boolean)">setDoDistanceCheck(boolean)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#setDoi(java.lang.String)">setDoi(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>
<div class="block">Set the value of doi</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#setDomainId(int)">setDomainId(int)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setDoRMSCheck(boolean)">setDoRMSCheck(boolean)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/DownloadChemCompProvider.html#setDownloadAll(boolean)">setDownloadAll(boolean)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/DownloadChemCompProvider.html" title="class in org.biojava.bio.structure.io.mmcif">DownloadChemCompProvider</a></dt>
<dd>
<div class="block">By default this provider will download only some of the <a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model"><code>ChemComp</code></a> files.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setDVar(java.lang.Double)">setDVar(Double)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setEcNums(java.util.List)">setEcNums(List&lt;String&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#setEditorList(java.util.List)">setEditorList(List&lt;Author&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#setElement(org.biojava.bio.structure.Element)">setElement(Element)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">set element of the atom name, e.g.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#setElement(org.biojava.bio.structure.Element)">setElement(Element)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/ElementTable.html#setElement(java.util.List)">setElement(List&lt;Element&gt;)</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/ElementTable.html" title="class in org.biojava3.aaproperties.xml">ElementTable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastService.html#setEmail(java.lang.String)">setEmail(String)</a></span> - Method in class org.biojava3.ws.alignment.qblast.<a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastService.html" title="class in org.biojava3.ws.alignment.qblast">NCBIQBlastService</a></dt>
<dd>
<div class="block">Set the email for QBlast.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html#setEnd(org.biojava3.core.sequence.location.template.Point)">setEnd(Point)</a></span> - Method in class org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava3.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/DistEn.html#setEnergy(double)">setEnergy(double)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/DistEn.html" title="class in org.biojava.bio.structure.secstruc">DistEn</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/HBond.html#setEnergy(double)">setEnergy(double)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/HBond.html" title="class in org.biojava.bio.structure.secstruc">HBond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setEngineered(java.lang.String)">setEngineered(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/EntityPolySeq.html#setEntity_id(java.lang.String)">setEntity_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/EntityPolySeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">EntityPolySeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxEntityNonPoly.html#setEntity_id(java.lang.String)">setEntity_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxEntityNonPoly.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxEntityNonPoly</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#setEntity_id(java.lang.String)">setEntity_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#setEntity_id(java.lang.String)">setEntity_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html#setEntity_id(java.lang.String)">setEntity_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructAsym</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html#setEntity_id(java.lang.String)">setEntity_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRef</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html#setEntry_id(java.lang.String)">setEntry_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html" title="class in org.biojava.bio.structure.io.mmcif.model">Exptl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setEntry_id(java.lang.String)">setEntry_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html#setEntry_id(java.lang.String)">setEntry_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html" title="class in org.biojava.bio.structure.io.mmcif.model">Struct</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructKeywords.html#setEntry_id(java.lang.String)">setEntry_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructKeywords.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructKeywords</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/AlignedPosition.html#setEquivalent(int)">setEquivalent(int)</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/AlignedPosition.html" title="class in org.biojava.bio.structure.gui.util">AlignedPosition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setEvalCutoff(float)">setEvalCutoff(float)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Site.html#setEvCode(java.lang.String)">setEvCode(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Site.html" title="class in org.biojava.bio.structure">Site</a></dt>
<dd>
<div class="block">sets the REMARK 800 EVIDENCE CODE for the site.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setExpressionSystem(java.lang.String)">setExpressionSystem(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setExpressionSystemAtccNumber(java.lang.String)">setExpressionSystemAtccNumber(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setExpressionSystemCell(java.lang.String)">setExpressionSystemCell(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setExpressionSystemCellLine(java.lang.String)">setExpressionSystemCellLine(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setExpressionSystemCellularLocation(java.lang.String)">setExpressionSystemCellularLocation(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setExpressionSystemGene(java.lang.String)">setExpressionSystemGene(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setExpressionSystemOrgan(java.lang.String)">setExpressionSystemOrgan(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setExpressionSystemOrganelle(java.lang.String)">setExpressionSystemOrganelle(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setExpressionSystemOtherDetails(java.lang.String)">setExpressionSystemOtherDetails(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setExpressionSystemPlasmid(java.lang.String)">setExpressionSystemPlasmid(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setExpressionSystemStrain(java.lang.String)">setExpressionSystemStrain(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setExpressionSystemTaxId(java.lang.String)">setExpressionSystemTaxId(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setExpressionSystemTissue(java.lang.String)">setExpressionSystemTissue(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setExpressionSystemVariant(java.lang.String)">setExpressionSystemVariant(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setExpressionSystemVector(java.lang.String)">setExpressionSystemVector(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setExpressionSystemVectorType(java.lang.String)">setExpressionSystemVectorType(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleGapPenalty.html#setExtensionPenalty(short)">setExtensionPenalty(short)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleGapPenalty.html" title="class in org.biojava3.alignment">SimpleGapPenalty</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/GapPenalty.html#setExtensionPenalty(short)">setExtensionPenalty(short)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/GapPenalty.html" title="interface in org.biojava3.alignment.template">GapPenalty</a></dt>
<dd>
<div class="block">Sets penalty given when an already open gap elongates by a single element</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#setFamilyId(int)">setFamilyId(int)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/GUIAlignmentProgressListener.html#setFarmJob(org.biojava.bio.structure.align.FarmJob)">setFarmJob(FarmJob)</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/GUIAlignmentProgressListener.html" title="class in org.biojava.bio.structure.align.gui">GUIAlignmentProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/MySaveFileListener.html#setFatCatOutput(boolean)">setFatCatOutput(boolean)</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/MySaveFileListener.html" title="class in org.biojava.bio.structure.align.gui">MySaveFileListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#setFeaturesKeyWord(org.biojava3.core.sequence.features.FeaturesKeyWordInterface)">setFeaturesKeyWord(FeaturesKeyWordInterface)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#setFetchCurrent(boolean)">setFetchCurrent(boolean)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>
<div class="block">if enabled, the reader searches for the newest possible PDB ID, if not present in he local installation.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#setFetchCurrent(boolean)">setFetchCurrent(boolean)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>
<div class="block">if enabled, the reader searches for the newest possible PDB ID, if not present in he local installation.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#setFetchFileEvenIfObsolete(boolean)">setFetchFileEvenIfObsolete(boolean)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#setFetchFileEvenIfObsolete(boolean)">setFetchFileEvenIfObsolete(boolean)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#setFile1(java.lang.String)">setFile1(String)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#setFile2(java.lang.String)">setFile2(String)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/UserConfiguration.html#setFileFormat(java.lang.String)">setFileFormat(String)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/UserConfiguration.html" title="class in org.biojava.bio.structure.align.util">UserConfiguration</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/LocalCacheStructureProvider.html#setFileParsingParameters(org.biojava.bio.structure.io.FileParsingParameters)">setFileParsingParameters(FileParsingParameters)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/LocalCacheStructureProvider.html" title="class in org.biojava.bio.structure.io">LocalCacheStructureProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#setFileParsingParameters(org.biojava.bio.structure.io.FileParsingParameters)">setFileParsingParameters(FileParsingParameters)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#setFileParsingParameters(org.biojava.bio.structure.io.FileParsingParameters)">setFileParsingParameters(FileParsingParameters)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#setFileParsingParameters(org.biojava.bio.structure.io.FileParsingParameters)">setFileParsingParameters(FileParsingParameters)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/MMCIFFileReader.html#setFileParsingParameters(org.biojava.bio.structure.io.FileParsingParameters)">setFileParsingParameters(FileParsingParameters)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/MMCIFFileReader.html" title="class in org.biojava.bio.structure.io">MMCIFFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileParser.html#setFileParsingParameters(org.biojava.bio.structure.io.FileParsingParameters)">setFileParsingParameters(FileParsingParameters)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileParser.html" title="class in org.biojava.bio.structure.io">PDBFileParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#setFileParsingParameters(org.biojava.bio.structure.io.FileParsingParameters)">setFileParsingParameters(FileParsingParameters)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/SandboxStyleStructureProvider.html#setFileParsingParameters(org.biojava.bio.structure.io.FileParsingParameters)">setFileParsingParameters(FileParsingParameters)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/SandboxStyleStructureProvider.html" title="class in org.biojava.bio.structure.io">SandboxStyleStructureProvider</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/StructureIOFile.html#setFileParsingParameters(org.biojava.bio.structure.io.FileParsingParameters)">setFileParsingParameters(FileParsingParameters)</a></span> - Method in interface org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/StructureIOFile.html" title="interface in org.biojava.bio.structure.io">StructureIOFile</a></dt>
<dd>
<div class="block">Set the parameters that should be used for file parsing</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/StructureProvider.html#setFileParsingParameters(org.biojava.bio.structure.io.FileParsingParameters)">setFileParsingParameters(FileParsingParameters)</a></span> - Method in interface org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/StructureProvider.html" title="interface in org.biojava.bio.structure.io">StructureProvider</a></dt>
<dd>
<div class="block">Set the parameters that should be used for file parsing</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#setFileParsingParams(org.biojava.bio.structure.io.FileParsingParameters)">setFileParsingParams(FileParsingParameters)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/FlatFileInstallation.html#setFilePath(java.io.File)">setFilePath(File)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/FlatFileInstallation.html" title="class in org.biojava.bio.structure.server">FlatFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#setFlexible(boolean)">setFlexible(boolean)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setFocusAfpList(int[])">setFocusAfpList(int[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setFocusAfpn(int)">setFocusAfpn(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setFocusRes1(int[])">setFocusRes1(int[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setFocusRes2(int[])">setFocusRes2(int[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setFocusResn(int)">setFocusResn(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#setFoldId(int)">setFoldId(int)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setFormula(java.lang.String)">setFormula(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationImpl.Builder.html#setFormula(java.lang.String)">setFormula(String)</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationImpl.Builder.html" title="class in org.biojava3.protmod">ProteinModificationImpl.Builder</a></dt>
<dd>
<div class="block">Set the residue formula.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setFormula_weight(java.lang.String)">setFormula_weight(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html#setFormula_weight(java.lang.String)">setFormula_weight(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model">Entity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setFragCompat(float)">setFragCompat(float)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#setFragLen(java.lang.Integer)">setFragLen(Integer)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFP.html#setFragLen(int)">setFragLen(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFP.html" title="class in org.biojava.bio.structure.align.model">AFP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#setFragLenSq(int)">setFragLenSq(int)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setFragment(java.lang.String)">setFragment(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setFragmentLength(int)">setFragmentLength(int)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setFragmentMiniDistance(float)">setFragmentMiniDistance(float)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructurePairAligner.html#setFragmentPairs(org.biojava.bio.structure.align.pairwise.FragmentPair[])">setFragmentPairs(FragmentPair[])</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructurePairAligner.html" title="class in org.biojava.bio.structure.align">StructurePairAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/JMatrixPanel.html#setFragmentPairs(org.biojava.bio.structure.align.pairwise.FragmentPair[])">setFragmentPairs(FragmentPair[])</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/JMatrixPanel.html" title="class in org.biojava.bio.structure.gui">JMatrixPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html#setFragmentPairs(org.biojava.bio.structure.align.pairwise.FragmentPair[])">setFragmentPairs(FragmentPair[])</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html" title="class in org.biojava.bio.structure.gui">ScaleableMatrixPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#setFragScore(double)">setFragScore(double)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Segment.html#setFrom(int)">setFrom(int)</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Segment.html" title="class in org.biojava.bio.structure.domain.pdp">Segment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/ConfigPDBInstallPanel.html#setFromFtp(javax.swing.JCheckBox)">setFromFtp(JCheckBox)</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/ConfigPDBInstallPanel.html" title="class in org.biojava.bio.structure.align.gui">ConfigPDBInstallPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#setFullName(java.lang.String)">setFullName(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">set full name of atom e.g. " CA ".</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#setFullName(java.lang.String)">setFullName(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>
<div class="block">set full name of atom e.g. " CA " .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#setGapCreate(double)">setGapCreate(double)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/Alignable.html#setGapExtCol(float)">setGapExtCol(float)</a></span> - Method in interface org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/Alignable.html" title="interface in org.biojava.bio.structure.align.pairwise">Alignable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html#setGapExtCol(float)">setGapExtCol(float)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">StrCompAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#setGapExtend(double)">setGapExtend(double)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWaterman3DParameters.html#setGapExtend(java.lang.Short)">setGapExtend(Short)</a></span> - Method in class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWaterman3DParameters.html" title="class in org.biojava.bio.structure.align.seq">SmithWaterman3DParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#setGapExtension(java.lang.Double)">setGapExtension(Double)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setGapExtension(float)">setGapExtension(float)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/Alignable.html#setGapExtRow(float)">setGapExtRow(float)</a></span> - Method in interface org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/Alignable.html" title="interface in org.biojava.bio.structure.align.pairwise">Alignable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html#setGapExtRow(float)">setGapExtRow(float)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">StrCompAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setGapLen(int)">setGapLen(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#setGapOpen(java.lang.Double)">setGapOpen(Double)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWaterman3DParameters.html#setGapOpen(java.lang.Short)">setGapOpen(Short)</a></span> - Method in class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWaterman3DParameters.html" title="class in org.biojava.bio.structure.align.seq">SmithWaterman3DParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setGapOpen(float)">setGapOpen(float)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/Alignable.html#setGapOpenCol(float)">setGapOpenCol(float)</a></span> - Method in interface org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/Alignable.html" title="interface in org.biojava.bio.structure.align.pairwise">Alignable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html#setGapOpenCol(float)">setGapOpenCol(float)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">StrCompAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/Alignable.html#setGapOpenRow(float)">setGapOpenRow(float)</a></span> - Method in interface org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/Alignable.html" title="interface in org.biojava.bio.structure.align.pairwise">Alignable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html#setGapOpenRow(float)">setGapOpenRow(float)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">StrCompAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#setGapPenalty(org.biojava3.alignment.template.GapPenalty)">setGapPenalty(GapPenalty)</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava3.alignment.template">AbstractMatrixAligner</a></dt>
<dd>
<div class="block">Sets the gap penalties.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setGene(java.lang.String)">setGene(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#setGroup(org.biojava.bio.structure.Group)">setGroup(Group)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">Sets the back-reference to its parent Group.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#setGroup(org.biojava.bio.structure.Group)">setGroup(Group)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setGroup_PDB(java.lang.String)">setGroup_PDB(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Site.html#setGroups(java.util.List)">setGroups(List&lt;Group&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Site.html" title="class in org.biojava.bio.structure">Site</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ModifiedCompound.html#setGroups(java.util.Set)">setGroups(Set&lt;StructureGroup&gt;)</a></span> - Method in interface org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ModifiedCompound.html" title="interface in org.biojava3.protmod.structure">ModifiedCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ModifiedCompoundImpl.html#setGroups(java.util.Set)">setGroups(Set&lt;StructureGroup&gt;)</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ModifiedCompoundImpl.html" title="class in org.biojava3.protmod.structure">ModifiedCompoundImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html#setH(org.biojava.bio.structure.Atom)">setH(Atom)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html" title="class in org.biojava.bio.structure.secstruc">SecStrucGroup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#setHeader(org.biojava.bio.structure.Compound)">setHeader(Compound)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Set the Header from the PDB file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#setHeader(org.biojava.bio.structure.Compound)">setHeader(Compound)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Set the Header from the PDB file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#setHeader(java.util.Map)">setHeader(Map&lt;String, Object&gt;)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">set the Header data .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#setHeader(java.util.Map)">setHeader(Map&lt;String, Object&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">set the Header data.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileParsingParameters.html#setHeaderOnly(boolean)">setHeaderOnly(boolean)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileParsingParameters.html" title="class in org.biojava.bio.structure.io">FileParsingParameters</a></dt>
<dd>
<div class="block">Parse only the PDB file header out of the files</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setHeaderVars(java.lang.String)">setHeaderVars(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/EntityPolySeq.html#setHetero(java.lang.String)">setHetero(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/EntityPolySeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">EntityPolySeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#setHetero(java.lang.String)">setHetero(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPDistanceMatrix.html#setIclose(int[])">setIclose(int[])</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPDistanceMatrix.html" title="class in org.biojava.bio.structure.domain.pdp">PDPDistanceMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFP.html#setId(long)">setId(long)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFP.html" title="class in org.biojava.bio.structure.align.model">AFP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setId(long)">setId(long)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#setId(long)">setId(long)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>
<div class="block">Set the Hibernate database ID.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#setId(java.lang.Long)">setId(Long)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Set the ID used by Hibernate.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#setId(java.lang.Long)">setId(Long)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Set the ID used by Hibernate.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setId(java.lang.Long)">setId(Long)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>
<div class="block">set the ID used by Hibernate</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#setId(java.lang.Long)">setId(Long)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Set the ID used by Hibernate.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Domain.html#setId(java.lang.String)">setId(String)</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Domain.html" title="class in org.biojava.bio.structure.domain.pdp">Domain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#setId(long)">setId(long)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">the Hibernate database ID</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setId(java.lang.String)">setId(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setId(java.lang.String)">setId(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBremark.html#setId(java.lang.String)">setId(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBremark.html" title="class in org.biojava.bio.structure.io.mmcif.model">DatabasePDBremark</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html#setId(java.lang.String)">setId(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model">Entity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html#setId(java.lang.String)">setId(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructAsym</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html#setId(java.lang.String)">setId(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRef</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#setId(java.lang.Long)">setId(Long)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">set the ID used by Hibernate</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#setId(java.lang.Long)">setId(Long)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">set the ID used by Hibernate</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/DBReferenceInfo.html#setId(java.lang.String)">setId(String)</a></span> - Method in class org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/DBReferenceInfo.html" title="class in org.biojava3.core.sequence.features">DBReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#setIdbnsBegin(char)">setIdbnsBegin(char)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Insertion code of initial residue of the segment, if PDB is the
    reference.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#setIdbnsEnd(char)">setIdbnsEnd(char)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Insertion code of the ending
    residue of the segment, if PDB is
    the reference.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#setIdCode(java.lang.String)">setIdCode(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Set the idCode for this entry.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#setIdCode(java.lang.String)">setIdCode(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>
<div class="block">The PDB code for this protein structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setIdentity(double)">setIdentity(double)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/JointFragments.html#setIdxlist(java.util.List)">setIdxlist(List&lt;int[]&gt;)</a></span> - Method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/JointFragments.html" title="class in org.biojava.bio.structure.align.helper">JointFragments</a></dt>
<dd>
<div class="block">Stores the alignment between the residues of several fragments.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/GapArray.html#setIndex(int)">setIndex(int)</a></span> - Method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/GapArray.html" title="class in org.biojava.bio.structure.align.helper">GapArray</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html#setInitialized(boolean)">setInitialized(boolean)</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava3.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setInitialK(int)">setInitialK(int)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Author.html#setInitials(java.lang.String)">setInitials(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Author.html" title="class in org.biojava.bio.structure">Author</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ResidueNumber.html#setInsCode(java.lang.Character)">setInsCode(Character)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ResidueNumber.html" title="class in org.biojava.bio.structure">ResidueNumber</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/StructureGroup.html#setInsCode(java.lang.Character)">setInsCode(Character)</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/StructureGroup.html" title="class in org.biojava3.protmod.structure">StructureGroup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SSBond.html#setInsCode1(java.lang.String)">setInsCode1(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SSBond.html" title="class in org.biojava.bio.structure">SSBond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SSBond.html#setInsCode2(java.lang.String)">setInsCode2(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SSBond.html" title="class in org.biojava.bio.structure">SSBond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#setInsertBegin(char)">setInsertBegin(char)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Initial insertion code of the PDB sequence segment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#setInsertEnd(char)">setInsertEnd(char)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Ending insertion code of the PDB sequence segment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/LinearColorInterpolator.html#setInterpolationDirection(int, org.biojava.bio.structure.gui.util.color.LinearColorInterpolator.InterpolationDirection)">setInterpolationDirection(int, LinearColorInterpolator.InterpolationDirection)</a></span> - Method in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/LinearColorInterpolator.html" title="class in org.biojava.bio.structure.gui.util.color">LinearColorInterpolator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html#setInterpolator(org.biojava.bio.structure.gui.util.color.ColorInterpolator)">setInterpolator(ColorInterpolator)</a></span> - Method in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html" title="class in org.biojava.bio.structure.gui.util.color">GradientMapper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setIoTime(long)">setIoTime(long)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html#setIoTime(long)">setIoTime(long)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentResult</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/StructureGroup.html#setIsAminoAcid(boolean)">setIsAminoAcid(boolean)</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/StructureGroup.html" title="class in org.biojava3.protmod.structure">StructureGroup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/Element.html#setIsotopes(java.util.List)">setIsotopes(List&lt;Isotope&gt;)</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/Element.html" title="class in org.biojava3.aaproperties.xml">Element</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#setIsPublished(java.lang.Boolean)">setIsPublished(Boolean)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>
<div class="block">Sets the publication state of a JournalArticle - TO BE PUBLISHED == false</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPDistanceMatrix.html#setJclose(int[])">setJclose(int[])</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPDistanceMatrix.html" title="class in org.biojava.bio.structure.domain.pdp">PDPDistanceMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html#setJmolPanel(org.biojava.bio.structure.align.gui.jmol.JmolPanel)">setJmolPanel(JmolPanel)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html" title="class in org.biojava.bio.structure.align.gui.jmol">StructureAlignmentJmol</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setJoinFast(boolean)">setJoinFast(boolean)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setJoinPlo(boolean)">setJoinPlo(boolean)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setJoinRMSCutoff(double)">setJoinRMSCutoff(double)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#setJournalArticle(org.biojava.bio.structure.JournalArticle)">setJournalArticle(JournalArticle)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">set the associated publication as defined by the JRNL records in a PDB
 file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#setJournalArticle(org.biojava.bio.structure.JournalArticle)">setJournalArticle(JournalArticle)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">set the associated publication as defined by the JRNL records in a PDB
 file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#setJournalName(java.lang.String)">setJournalName(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setLabel_alt_id(java.lang.String)">setLabel_alt_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setLabel_asym_id(java.lang.String)">setLabel_asym_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setLabel_atom_id(java.lang.String)">setLabel_atom_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setLabel_comp_id(java.lang.String)">setLabel_comp_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setLabel_entity_id(java.lang.String)">setLabel_entity_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setLabel_seq_id(java.lang.String)">setLabel_seq_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#setLcmp(int)">setLcmp(int)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#setLength(int)">setLength(int)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/CoordManager.html#setLength(int)">setLength(int)</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/CoordManager.html" title="class in org.biojava.bio.structure.gui.util">CoordManager</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html#setLength1(int)">setLength1(int)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentResult</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html#setLength2(int)">setLength2(int)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentResult</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/FastaGeneWriter.html#setLineLength(int)">setLineLength(int)</a></span> - Method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/FastaGeneWriter.html" title="class in org.biojava3.core.sequence.io">FastaGeneWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/FastaWriter.html#setLineLength(int)">setLineLength(int)</a></span> - Method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/FastaWriter.html" title="class in org.biojava3.core.sequence.io">FastaWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/FastaWriter.html#setLineSeparator(java.lang.String)">setLineSeparator(String)</a></span> - Method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/FastaWriter.html" title="class in org.biojava3.core.sequence.io">FastaWriter</a></dt>
<dd>
<div class="block">Allow an override of operating system line separator for programs that needs a specific CRLF or CR or LF option</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileParsingParameters.html#setLoadChemCompInfo(boolean)">setLoadChemCompInfo(boolean)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileParsingParameters.html" title="class in org.biojava.bio.structure.io">FileParsingParameters</a></dt>
<dd>
<div class="block">Sets if chemical component defintions should be loaded from the web</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/AbstractFeature.html#setLocation(org.biojava3.core.sequence.location.SequenceLocation)">setLocation(SequenceLocation&lt;S, C&gt;)</a></span> - Method in class org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/AbstractFeature.html" title="class in org.biojava3.core.sequence.features">AbstractFeature</a></dt>
<dd>
<div class="block">A feature could be a single sequence position like a mutation or a post translational modification of an amino acid.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/FeatureInterface.html#setLocation(org.biojava3.core.sequence.location.SequenceLocation)">setLocation(SequenceLocation&lt;S, C&gt;)</a></span> - Method in interface org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/FeatureInterface.html" title="interface in org.biojava3.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">The new location for this feature.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/MutableAlignedSequence.html#setLocationInAlignment(org.biojava3.core.sequence.location.template.Location)">setLocationInAlignment(Location)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/MutableAlignedSequence.html" title="interface in org.biojava3.alignment.template">MutableAlignedSequence</a></dt>
<dd>
<div class="block">Sets the position of the <a href="./org/biojava3/alignment/template/AlignedSequence.html" title="interface in org.biojava3.alignment.template"><code>AlignedSequence</code></a> to the given <a href="./org/biojava3/core/sequence/location/template/Location.html" title="interface in org.biojava3.core.sequence.location.template"><code>Location</code></a> (start, gaps, end).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/CodonCompound.html#setLongName(java.lang.String)">setLongName(String)</a></span> - Method in class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/CodonCompound.html" title="class in org.biojava3.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompound.html#setLongName(java.lang.String)">setLongName(String)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompound.html" title="class in org.biojava3.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/Compound.html#setLongName(java.lang.String)">setLongName(String)</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/CaseInsensitiveCompound.html#setLongName(java.lang.String)">setLongName(String)</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava3.core.sequence.transcription">CaseInsensitiveCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/Table.Codon.html#setLongName(java.lang.String)">setLongName(String)</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/Table.Codon.html" title="class in org.biojava3.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/webstart/AligUIManager.html#setLookAndFeel()">setLookAndFeel()</a></span> - Static method in class org.biojava.bio.structure.align.webstart.<a href="./org/biojava/bio/structure/align/webstart/AligUIManager.html" title="class in org.biojava.bio.structure.align.webstart">AligUIManager</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setLs_d_res_high(java.lang.String)">setLs_d_res_high(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setLs_d_res_low(java.lang.String)">setLs_d_res_low(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setLs_number_parameters(java.lang.String)">setLs_number_parameters(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setLs_number_reflns_all(java.lang.String)">setLs_number_reflns_all(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setLs_number_reflns_obs(java.lang.String)">setLs_number_reflns_obs(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setLs_number_reflns_R_free(java.lang.String)">setLs_number_reflns_R_free(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setLs_number_restraints(java.lang.String)">setLs_number_restraints(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setLs_percent_reflns_obs(java.lang.String)">setLs_percent_reflns_obs(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setLs_percent_reflns_R_free(java.lang.String)">setLs_percent_reflns_R_free(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setLs_R_factor_all(java.lang.String)">setLs_R_factor_all(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setLs_R_factor_obs(java.lang.String)">setLs_R_factor_obs(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setLs_R_factor_R_free(java.lang.String)">setLs_R_factor_R_free(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setLs_R_factor_R_free_error(java.lang.String)">setLs_R_factor_R_free_error(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setLs_R_factor_R_free_error_details(java.lang.String)">setLs_R_factor_R_free_error_details(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setLs_R_factor_R_work(java.lang.String)">setLs_R_factor_R_work(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setLs_redundancy_reflns_obs(java.lang.String)">setLs_redundancy_reflns_obs(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setLs_wR_factor_R_free(java.lang.String)">setLs_wR_factor_R_free(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setLs_wR_factor_R_work(java.lang.String)">setLs_wR_factor_R_work(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFP.html#setM(org.biojava.bio.structure.jama.Matrix)">setM(Matrix)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFP.html" title="class in org.biojava.bio.structure.align.model">AFP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/Element.html#setMass(double)">setMass(double)</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/Element.html" title="class in org.biojava3.aaproperties.xml">Element</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/Isotope.html#setMass(double)">setMass(double)</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/Isotope.html" title="class in org.biojava3.aaproperties.xml">Isotope</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#setMatMatrix(double[][])">setMatMatrix(double[][])</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/JMatrixPanel.html#setMatrix(org.biojava.bio.structure.jama.Matrix)">setMatrix(Matrix)</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/JMatrixPanel.html" title="class in org.biojava.bio.structure.gui">JMatrixPanel</a></dt>
<dd>
<div class="block">sets the distance matrix to be displayed</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html#setMatrix(org.biojava.bio.structure.jama.Matrix)">setMatrix(Matrix)</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html" title="class in org.biojava.bio.structure.gui">ScaleableMatrixPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#setMatrix(int, int, int, int, org.biojava.bio.structure.jama.Matrix)">setMatrix(int, int, int, int, Matrix)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Set a submatrix.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#setMatrix(int[], int[], org.biojava.bio.structure.jama.Matrix)">setMatrix(int[], int[], Matrix)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Set a submatrix.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#setMatrix(int[], int, int, org.biojava.bio.structure.jama.Matrix)">setMatrix(int[], int, int, Matrix)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Set a submatrix.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#setMatrix(int, int, int[], org.biojava.bio.structure.jama.Matrix)">setMatrix(int, int, int[], Matrix)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Set a submatrix.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html#setMatrix(short[][])">setMatrix(short[][])</a></span> - Method in class org.biojava3.alignment.aaindex.<a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava3.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html#setMax(short)">setMax(short)</a></span> - Method in class org.biojava3.alignment.aaindex.<a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava3.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#setMaxGap(int)">setMaxGap(int)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#setMaxGapFrag(int)">setMaxGapFrag(int)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#setMaxGapSize(java.lang.Integer)">setMaxGapSize(Integer)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>
<div class="block">Set the Max gap size parameter.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#setMaxGapSize(int)">setMaxGapSize(int)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>
<div class="block">CE specific parameter: set the Max gap size parameter G (during AFP extension).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setMaxIter(int)">setMaxIter(int)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/BigSqrt.html#setMaxIterations(int)">setMaxIterations(int)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/BigSqrt.html" title="class in org.biojava.bio.structure.secstruc">BigSqrt</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#setMaxNrIterationsForOptimization(int)">setMaxNrIterationsForOptimization(int)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>
<div class="block">Set the maximum nr of times the (slow) optimiziation of alignment should iterate.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#setMaxOptRMSD(java.lang.Double)">setMaxOptRMSD(Double)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>
<div class="block">set the maximum RMSD cutoff to be applied during alignment optimization.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#setMaxOptRMSD(java.lang.Double)">setMaxOptRMSD(Double)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>
<div class="block">(jCE specific): maximum RMSD that shall be calculated for the alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#setMaxPenalty(double)">setMaxPenalty(double)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setMaxrefine(int)">setMaxrefine(int)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#setMaxTra(java.lang.Integer)">setMaxTra(Integer)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>
<div class="block">set the maximum number of Twists that are allowed...</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setMaxTra(int)">setMaxTra(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">Set the maximum number of Twists that are allowed...</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html#setMethod(java.lang.String)">setMethod(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html" title="class in org.biojava.bio.structure.io.mmcif.model">Exptl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#setMethod(java.lang.String)">setMethod(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html#setMethod_details(java.lang.String)">setMethod_details(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Exptl.html" title="class in org.biojava.bio.structure.io.mmcif.model">Exptl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html#setMin(short)">setMin(short)</a></span> - Method in class org.biojava3.alignment.aaindex.<a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava3.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setMinLen(int)">setMinLen(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#setMisCut(int)">setMisCut(int)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setMisLen(int)">setMisLen(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#setMisScore(double)">setMisScore(double)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html#setMod_type(java.lang.String)">setMod_type(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html" title="class in org.biojava.bio.structure.io.mmcif.model">DatabasePDBrev</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#setModDate(java.util.Date)">setModDate(Date)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#setModel(int, java.util.List)">setModel(int, List&lt;Chain&gt;)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">a convenience function if one wants to edit and replace the
 models in a structure. allows to set (replace) the model at position
 with the new List of Chains.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#setModel(int, java.util.List)">setModel(int, List&lt;Chain&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/AtomInfo.html#setModelNumber(int)">setModelNumber(int)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/AtomInfo.html" title="class in org.biojava.bio.structure.align.gui.jmol">AtomInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ModifiedCompound.html#setModification(org.biojava3.protmod.ProteinModification)">setModification(ProteinModification)</a></span> - Method in interface org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ModifiedCompound.html" title="interface in org.biojava3.protmod.structure">ModifiedCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ModifiedCompoundImpl.html#setModification(org.biojava3.protmod.ProteinModification)">setModification(ProteinModification)</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ModifiedCompoundImpl.html" title="class in org.biojava3.protmod.structure">ModifiedCompoundImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/CodonCompound.html#setMolecularWeight(java.lang.Float)">setMolecularWeight(Float)</a></span> - Method in class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/CodonCompound.html" title="class in org.biojava3.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompound.html#setMolecularWeight(java.lang.Float)">setMolecularWeight(Float)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompound.html" title="class in org.biojava3.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/Compound.html#setMolecularWeight(java.lang.Float)">setMolecularWeight(Float)</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/CaseInsensitiveCompound.html#setMolecularWeight(java.lang.Float)">setMolecularWeight(Float)</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava3.core.sequence.transcription">CaseInsensitiveCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/Table.Codon.html#setMolecularWeight(java.lang.Float)">setMolecularWeight(Float)</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/Table.Codon.html" title="class in org.biojava3.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setMolId(java.lang.String)">setMolId(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>
<div class="block">Set the mol id value.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setMolName(java.lang.String)">setMolName(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/EntityPolySeq.html#setMon_id(java.lang.String)">setMon_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/EntityPolySeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">EntityPolySeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#setMon_id(java.lang.String)">setMon_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#setMon_id(java.lang.String)">setMon_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setMon_nstd_flag(java.lang.String)">setMon_nstd_flag(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setMon_nstd_parent_comp_id(java.lang.String)">setMon_nstd_parent_comp_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setMutation(java.lang.String)">setMutation(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html#setN(org.biojava.bio.structure.Atom)">setN(Atom)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html" title="class in org.biojava.bio.structure.secstruc">SecStrucGroup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#setName(java.lang.String)">setName(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">set trimmed version of atom name, e.g.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#setName(java.lang.String)">setName(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>
<div class="block">trimmed version of atom name, e.g.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#setName(java.lang.String)">setName(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>replaced by <a href="./org/biojava/bio/structure/Chain.html#setChainID(java.lang.String)"><code>Chain.setChainID(String)</code></a></i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#setName(java.lang.String)">setName(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AuditAuthor.html#setName(java.lang.String)">setName(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AuditAuthor.html" title="class in org.biojava.bio.structure.io.mmcif.model">AuditAuthor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setName(java.lang.String)">setName(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxEntityNonPoly.html#setName(java.lang.String)">setName(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxEntityNonPoly.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxEntityNonPoly</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDescription.html#setName(java.lang.String)">setName(String)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDescription.html" title="class in org.biojava.bio.structure.scop">ScopDescription</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#setName(java.lang.String)">setName(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">set biological name of Structure .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#setName(java.lang.String)">setName(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">set biological name of Structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/AminoAcidComposition.html#setName(java.lang.String)">setName(String)</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/AminoAcidComposition.html" title="class in org.biojava3.aaproperties.xml">AminoAcidComposition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/Element.html#setName(java.lang.String)">setName(String)</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/Element.html" title="class in org.biojava3.aaproperties.xml">Element</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/Isotope.html#setName(java.lang.String)">setName(String)</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/Isotope.html" title="class in org.biojava3.aaproperties.xml">Isotope</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html#setName(java.lang.String)">setName(String)</a></span> - Method in class org.biojava3.alignment.aaindex.<a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava3.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleSubstitutionMatrix.html#setName(java.lang.String)">setName(String)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleSubstitutionMatrix.html" title="class in org.biojava3.alignment">SimpleSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/SubstitutionMatrix.html#setName(java.lang.String)">setName(String)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/SubstitutionMatrix.html" title="interface in org.biojava3.alignment.template">SubstitutionMatrix</a></dt>
<dd>
<div class="block">Sets the name (short description) of this matrix.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/PdbPair.html#setName1(java.lang.String)">setName1(String)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/PdbPair.html" title="class in org.biojava.bio.structure.align.client">PdbPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setName1(java.lang.String)">setName1(String)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/PdbPair.html#setName2(java.lang.String)">setName2(String)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/PdbPair.html" title="class in org.biojava.bio.structure.align.client">PdbPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setName2(java.lang.String)">setName2(String)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#setnAtom(int)">setnAtom(int)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPDistanceMatrix.html#setNclose(int)">setNclose(int)</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPDistanceMatrix.html" title="class in org.biojava.bio.structure.domain.pdp">PDPDistanceMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/CutSites.html#setNcuts(int)">setNcuts(int)</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/CutSites.html" title="class in org.biojava.bio.structure.domain.pdp">CutSites</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#setNdb_seq_num(java.lang.String)">setNdb_seq_num(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#setNdb_seq_num(java.lang.String)">setNdb_seq_num(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/Isotope.html#setNeutronsNum(int)">setNeutronsNum(int)</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/Isotope.html" title="class in org.biojava3.aaproperties.xml">Isotope</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#setNmr(boolean)">setNmr(boolean)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">set NMR flag.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#setNmr(boolean)">setNmr(boolean)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setNormAlignScore(double)">setNormAlignScore(double)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#setNotesList(java.util.ArrayList)">setNotesList(ArrayList&lt;String&gt;)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#setNrAlignments(int)">setNrAlignments(int)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/AlignmentCalc.html#setNrCPUs(int)">setNrCPUs(int)</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/AlignmentCalc.html" title="class in org.biojava.bio.structure.align.gui">AlignmentCalc</a></dt>
<dd>
<div class="block">does not do anything here...</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/AlignmentCalcDB.html#setNrCPUs(int)">setNrCPUs(int)</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/AlignmentCalcDB.html" title="class in org.biojava.bio.structure.align.gui">AlignmentCalcDB</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/AlignmentCalculationRunnable.html#setNrCPUs(int)">setNrCPUs(int)</a></span> - Method in interface org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/AlignmentCalculationRunnable.html" title="interface in org.biojava.bio.structure.align.gui">AlignmentCalculationRunnable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Domain.html#setNseg(int)">setNseg(int)</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Domain.html" title="class in org.biojava.bio.structure.domain.pdp">Domain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html#setNum(java.lang.String)">setNum(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html" title="class in org.biojava.bio.structure.io.mmcif.model">DatabasePDBrev</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/EntityPolySeq.html#setNum(java.lang.String)">setNum(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/EntityPolySeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">EntityPolySeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setNumRes(java.lang.String)">setNumRes(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html#setO(org.biojava.bio.structure.Atom)">setO(Atom)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html" title="class in org.biojava.bio.structure.secstruc">SecStrucGroup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#setOccupancy(double)">setOccupancy(double)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">set occupancy.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#setOccupancy(double)">setOccupancy(double)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setOccupancy(java.lang.String)">setOccupancy(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setOccupancy_esd(java.lang.String)">setOccupancy_esd(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setOccupancy_max(java.lang.String)">setOccupancy_max(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setOccupancy_min(java.lang.String)">setOccupancy_min(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationImpl.Builder.html#setOccurrenceType(org.biojava3.protmod.ModificationOccurrenceType)">setOccurrenceType(ModificationOccurrenceType)</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationImpl.Builder.html" title="class in org.biojava3.protmod">ProteinModificationImpl.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setOne_letter_code(java.lang.String)">setOne_letter_code(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleGapPenalty.html#setOpenPenalty(short)">setOpenPenalty(short)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleGapPenalty.html" title="class in org.biojava3.alignment">SimpleGapPenalty</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/GapPenalty.html#setOpenPenalty(short)">setOpenPenalty(short)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/GapPenalty.html" title="interface in org.biojava3.alignment.template">GapPenalty</a></dt>
<dd>
<div class="block">Sets penalty given when a deletion or insertion gap first opens</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setOptAln(int[][][])">setOptAln(int[][][])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setOptLen(int[])">setOptLen(int[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setOptLength(int)">setOptLength(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">The length of the optimal alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setOptRmsd(double[])">setOptRmsd(double[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setOrgan(java.lang.String)">setOrgan(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setOrganelle(java.lang.String)">setOrganelle(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setOrganismCommon(java.lang.String)">setOrganismCommon(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setOrganismScientific(java.lang.String)">setOrganismScientific(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setOrganismTaxId(java.lang.String)">setOrganismTaxId(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html#setOriginal(org.biojava.bio.structure.Group)">setOriginal(Group)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html" title="class in org.biojava.bio.structure.secstruc">SecStrucGroup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#setOriginalHeader(java.lang.String)">setOriginalHeader(String)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#setORmsdThr(java.lang.Double)">setORmsdThr(Double)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>
<div class="block">Set the Original RMSD threshold from which the alignment optimization is started</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html#setOutFile(org.biojava.bio.structure.align.util.SynchronizedOutFile)">setOutFile(SynchronizedOutFile)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html" title="class in org.biojava.bio.structure.align">CallableStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#setOutFile(java.lang.String)">setOutFile(String)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html#setOutputDir(java.io.File)">setOutputDir(File)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html" title="class in org.biojava.bio.structure.align">CallableStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastOutputProperties.html#setOutputFormat(org.biojava3.ws.alignment.qblast.NCBIQBlastOutputFormat)">setOutputFormat(NCBIQBlastOutputFormat)</a></span> - Method in class org.biojava3.ws.alignment.qblast.<a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastOutputProperties.html" title="class in org.biojava3.ws.alignment.qblast">NCBIQBlastOutputProperties</a></dt>
<dd>
<div class="block">This method is use to set the stream output format to get from the QBlast service</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastOutputProperties.html#setOutputOption(java.lang.String, java.lang.String)">setOutputOption(String, String)</a></span> - Method in class org.biojava3.ws.alignment.qblast.<a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastOutputProperties.html" title="class in org.biojava3.ws.alignment.qblast">NCBIQBlastOutputProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/RemotePairwiseAlignmentOutputProperties.html#setOutputOption(java.lang.String, java.lang.String)">setOutputOption(String, String)</a></span> - Method in interface org.biojava3.ws.alignment.<a href="./org/biojava3/ws/alignment/RemotePairwiseAlignmentOutputProperties.html" title="interface in org.biojava3.ws.alignment">RemotePairwiseAlignmentOutputProperties</a></dt>
<dd>
<div class="block">Method to set the value for a specific output parameter using a key to store in a map.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#setOutputPDB(boolean)">setOutputPDB(boolean)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>
<div class="block">When writing the results to a file, don;t write as XML but write aligned PDB file</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setOverall_FOM_free_R_set(java.lang.String)">setOverall_FOM_free_R_set(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setOverall_FOM_work_R_set(java.lang.String)">setOverall_FOM_work_R_set(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setOverall_SU_B(java.lang.String)">setOverall_SU_B(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setOverall_SU_ML(java.lang.String)">setOverall_SU_ML(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setOverall_SU_R_Cruickshank_DPI(java.lang.String)">setOverall_SU_R_Cruickshank_DPI(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setOverall_SU_R_free(java.lang.String)">setOverall_SU_R_free(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFP.html#setP1(int)">setP1(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFP.html" title="class in org.biojava.bio.structure.align.model">AFP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFP.html#setP2(int)">setP2(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFP.html" title="class in org.biojava.bio.structure.align.model">AFP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html#setPaintDefaults(java.awt.Graphics2D)">setPaintDefaults(Graphics2D)</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html" title="class in org.biojava.bio.structure.gui.util">SequenceScalePanel</a></dt>
<dd>
<div class="block">set some default rendering hints, like text antialiasing on</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html#setPair(org.biojava.bio.structure.align.client.PdbPair)">setPair(PdbPair)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html" title="class in org.biojava.bio.structure.align">CallableStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/MutableProfilePair.html#setPair(org.biojava3.alignment.template.Profile, org.biojava3.alignment.template.Profile)">setPair(Profile&lt;S, C&gt;, Profile&lt;S, C&gt;)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/MutableProfilePair.html" title="interface in org.biojava3.alignment.template">MutableProfilePair</a></dt>
<dd>
<div class="block">Sets both <a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template"><code>Profile</code></a>s of the pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/MutableSequencePair.html#setPair(org.biojava3.alignment.template.AlignedSequence, org.biojava3.alignment.template.AlignedSequence)">setPair(AlignedSequence&lt;S, C&gt;, AlignedSequence&lt;S, C&gt;)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/MutableSequencePair.html" title="interface in org.biojava3.alignment.template">MutableSequencePair</a></dt>
<dd>
<div class="block">Sets both <a href="./org/biojava3/alignment/template/AlignedSequence.html" title="interface in org.biojava3.alignment.template"><code>AlignedSequence</code></a>s of the pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/AbstractStructureAlignment.html#setParameters(org.biojava.bio.structure.align.ce.ConfigStrucAligParams)">setParameters(ConfigStrucAligParams)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/AbstractStructureAlignment.html" title="class in org.biojava.bio.structure.align">AbstractStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/BioJavaStructureAlignment.html#setParameters(org.biojava.bio.structure.align.ce.ConfigStrucAligParams)">setParameters(ConfigStrucAligParams)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/BioJavaStructureAlignment.html" title="class in org.biojava.bio.structure.align">BioJavaStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html#setParameters(org.biojava.bio.structure.align.ce.ConfigStrucAligParams)">setParameters(ConfigStrucAligParams)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/CallableStructureAlignment.html" title="class in org.biojava.bio.structure.align">CallableStructureAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeMain.html#setParameters(org.biojava.bio.structure.align.ce.ConfigStrucAligParams)">setParameters(ConfigStrucAligParams)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeMain.html" title="class in org.biojava.bio.structure.align.ce">CeMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeSideChainMain.html#setParameters(org.biojava.bio.structure.align.ce.ConfigStrucAligParams)">setParameters(ConfigStrucAligParams)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeSideChainMain.html" title="class in org.biojava.bio.structure.align.ce">CeSideChainMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCatFlexible.html#setParameters(org.biojava.bio.structure.align.ce.ConfigStrucAligParams)">setParameters(ConfigStrucAligParams)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCatFlexible.html" title="class in org.biojava.bio.structure.align.fatcat">FatCatFlexible</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCatRigid.html#setParameters(org.biojava.bio.structure.align.ce.ConfigStrucAligParams)">setParameters(ConfigStrucAligParams)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCatRigid.html" title="class in org.biojava.bio.structure.align.fatcat">FatCatRigid</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWaterman3Daligner.html#setParameters(org.biojava.bio.structure.align.ce.ConfigStrucAligParams)">setParameters(ConfigStrucAligParams)</a></span> - Method in class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWaterman3Daligner.html" title="class in org.biojava.bio.structure.align.seq">SmithWaterman3Daligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructureAlignment.html#setParameters(org.biojava.bio.structure.align.ce.ConfigStrucAligParams)">setParameters(ConfigStrucAligParams)</a></span> - Method in interface org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructureAlignment.html" title="interface in org.biojava.bio.structure.align">StructureAlignment</a></dt>
<dd>
<div class="block">Set the default parameters for this algorithm to use</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/FarmJob.html#setParams(org.biojava.bio.structure.align.client.FarmJobParameters)">setParams(FarmJobParameters)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/FarmJob.html" title="class in org.biojava.bio.structure.align">FarmJob</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructurePairAligner.html#setParams(org.biojava.bio.structure.align.StrucAligParameters)">setParams(StrucAligParameters)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructurePairAligner.html" title="class in org.biojava.bio.structure.align">StructurePairAligner</a></dt>
<dd>
<div class="block">set the parameters to be used for the algorithm</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#setParent(org.biojava.bio.structure.Group)">setParent(Group)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>replaced by <a href="./org/biojava/bio/structure/Atom.html#setGroup(org.biojava.bio.structure.Group)"><code>Atom.setGroup(Group)</code></a></i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#setParent(org.biojava.bio.structure.Group)">setParent(Group)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#setParent(org.biojava.bio.structure.Structure)">setParent(Structure)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Set the back-reference to its parent Structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#setParent(org.biojava.bio.structure.Structure)">setParent(Structure)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Set the back-reference to its parent Structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#setParent(org.biojava.bio.structure.Structure)">setParent(Structure)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Set the structure object that this DBRef relates to.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#setParent(org.biojava.bio.structure.Chain)">setParent(Chain)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#setParent(org.biojava.bio.structure.Chain)">setParent(Chain)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">Set the back-reference (to its parent Chain)</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/ProteinSequence.html#setParentDNASequence(org.biojava3.core.sequence.template.AbstractSequence, java.lang.Integer, java.lang.Integer)">setParentDNASequence(AbstractSequence, Integer, Integer)</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/ProteinSequence.html" title="class in org.biojava3.core.sequence">ProteinSequence</a></dt>
<dd>
<div class="block">A Protein sequence can be stand alone or loaded from a transcript sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/AbstractFeature.html#setParentFeature(org.biojava3.core.sequence.features.FeatureInterface)">setParentFeature(FeatureInterface&lt;S, C&gt;)</a></span> - Method in class org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/AbstractFeature.html" title="class in org.biojava3.core.sequence.features">AbstractFeature</a></dt>
<dd>
<div class="block">A feature can be the child or contained by a parent feature.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/FeatureInterface.html#setParentFeature(org.biojava3.core.sequence.features.FeatureInterface)">setParentFeature(FeatureInterface&lt;S, C&gt;)</a></span> - Method in interface org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/FeatureInterface.html" title="interface in org.biojava3.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">Set the parent feature</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#setParentSequence(org.biojava3.core.sequence.template.AbstractSequence)">setParentSequence(AbstractSequence&lt;C&gt;)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopNode.html#setParentSunid(int)">setParentSunid(int)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopNode.html" title="class in org.biojava.bio.structure.scop">ScopNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileParsingParameters.html#setParseCAOnly(boolean)">setParseCAOnly(boolean)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileParsingParameters.html" title="class in org.biojava.bio.structure.io">FileParsingParameters</a></dt>
<dd>
<div class="block">the flag if only the C-alpha atoms of the structure should be parsed.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileParsingParameters.html#setParseSecStruc(boolean)">setParseSecStruc(boolean)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileParsingParameters.html" title="class in org.biojava.bio.structure.io">FileParsingParameters</a></dt>
<dd>
<div class="block">a flag to tell the parser to parse the Author's secondary structure assignment from the file
 default is set to false, i.e. do NOT parse.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/HBond.html#setPartner(int)">setPartner(int)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/HBond.html" title="class in org.biojava.bio.structure.secstruc">HBond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/Alignable.html#setPath(org.biojava.bio.structure.align.helper.IndexPair[])">setPath(IndexPair[])</a></span> - Method in interface org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/Alignable.html" title="interface in org.biojava.bio.structure.align.pairwise">Alignable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html#setPath(org.biojava.bio.structure.align.helper.IndexPair[])">setPath(IndexPair[])</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">StrCompAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#setPath(java.lang.String)">setPath(String)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>
<div class="block">Set the path that is used to cache PDB files.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/LocalCacheStructureProvider.html#setPath(java.lang.String)">setPath(String)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/LocalCacheStructureProvider.html" title="class in org.biojava.bio.structure.io">LocalCacheStructureProvider</a></dt>
<dd>
<div class="block">directory where to find PDB files</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/DownloadChemCompProvider.html#setPath(java.lang.String)">setPath(String)</a></span> - Static method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/DownloadChemCompProvider.html" title="class in org.biojava.bio.structure.io.mmcif">DownloadChemCompProvider</a></dt>
<dd>
<div class="block">making sure we use the same path for the PDB installation as is used by the PdbFileReader</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/MMCIFFileReader.html#setPath(java.lang.String)">setPath(String)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/MMCIFFileReader.html" title="class in org.biojava.bio.structure.io">MMCIFFileReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#setPath(java.lang.String)">setPath(String)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>
<div class="block">directory where to find PDB files</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/SandboxStyleStructureProvider.html#setPath(java.lang.String)">setPath(String)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/SandboxStyleStructureProvider.html" title="class in org.biojava.bio.structure.io">SandboxStyleStructureProvider</a></dt>
<dd>
<div class="block">directory where to find PDB files</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/StructureIOFile.html#setPath(java.lang.String)">setPath(String)</a></span> - Method in interface org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/StructureIOFile.html" title="interface in org.biojava.bio.structure.io">StructureIOFile</a></dt>
<dd>
<div class="block">Set path to file / connection string to db.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/Alignable.html#setPathSize(int)">setPathSize(int)</a></span> - Method in interface org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/Alignable.html" title="interface in org.biojava.bio.structure.align.pairwise">Alignable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html#setPathSize(int)">setPathSize(int)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">StrCompAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#setPdb1(java.lang.String)">setPdb1(String)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>
<div class="block">mandatory argument to set the first PDB (and optionally chain ID) to be aligned.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html#setPdb1(java.lang.String)">setPdb1(String)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentResult</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#setPdb2(java.lang.String)">setPdb2(String)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>
<div class="block">mandatory argument to set the second PDB (and optionally chain ID) to be aligned.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html#setPdb2(java.lang.String)">setPdb2(String)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentResult</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#setPdb_ins_code(java.lang.String)">setPdb_ins_code(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#setPdb_ins_code(java.lang.String)">setPdb_ins_code(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#setPdb_mon_id(java.lang.String)">setPdb_mon_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#setPdb_mon_id(java.lang.String)">setPdb_mon_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#setPdb_seq_num(java.lang.String)">setPdb_seq_num(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#setPdb_seq_num(java.lang.String)">setPdb_seq_num(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#setPdb_strand_id(java.lang.String)">setPdb_strand_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#setPdb_strand_id(java.lang.String)">setPdb_strand_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setPdbAln(java.lang.String[][][])">setPdbAln(String[][][])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationImpl.Builder.html#setPdbccId(java.lang.String)">setPdbccId(String)</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationImpl.Builder.html" title="class in org.biojava3.protmod">ProteinModificationImpl.Builder</a></dt>
<dd>
<div class="block">Set the Protein Data Bank Chemical Component ID.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationImpl.Builder.html#setPdbccName(java.lang.String)">setPdbccName(String)</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationImpl.Builder.html" title="class in org.biojava3.protmod">ProteinModificationImpl.Builder</a></dt>
<dd>
<div class="block">Set the Protein Data Bank Chemical Component name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#setPDBCode(java.lang.String)">setPDBCode(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>replaced by <a href="./org/biojava/bio/structure/Group.html#setResidueNumber(org.biojava.bio.structure.ResidueNumber)"><code>Group.setResidueNumber(ResidueNumber)</code></a></i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#setPDBCode(java.lang.String)">setPDBCode(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>replaced by <a href="./org/biojava/bio/structure/HetatomImpl.html#setResidueNumber(org.biojava.bio.structure.ResidueNumber)"><code>HetatomImpl.setResidueNumber(ResidueNumber)</code></a></i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureEvent.html#setPDBCode(java.lang.String)">setPDBCode(String)</a></span> - Method in interface org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureEvent.html" title="interface in org.biojava.bio.structure.server">StructureEvent</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureEventImpl.html#setPDBCode(java.lang.String)">setPDBCode(String)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureEventImpl.html" title="class in org.biojava.bio.structure.server">StructureEventImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#setPDBCode(java.lang.String)">setPDBCode(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">set PDB code of structure .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#setPDBCode(java.lang.String)">setPDBCode(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">set PDB code of structure .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/MMCIFFileReader.html#setPdbDirectorySplit(boolean)">setPdbDirectorySplit(boolean)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/MMCIFFileReader.html" title="class in org.biojava.bio.structure.io">MMCIFFileReader</a></dt>
<dd>
<div class="block">Flag that defines if the PDB directory is containing all PDB files or is split into sub dirs (like the FTP site).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileReader.html#setPdbDirectorySplit(boolean)">setPdbDirectorySplit(boolean)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileReader.html" title="class in org.biojava.bio.structure.io">PDBFileReader</a></dt>
<dd>
<div class="block">Flag that defines if the PDB directory is containing all PDB files or is split into sub dirs (like the FTP site).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/StructureIOFile.html#setPdbDirectorySplit(boolean)">setPdbDirectorySplit(boolean)</a></span> - Method in interface org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/StructureIOFile.html" title="interface in org.biojava.bio.structure.io">StructureIOFile</a></dt>
<dd>
<div class="block">The PDB files are organized hierarchically (as on the PDB - FTP server.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/ConfigPDBInstallPanel.html#setPDBDirField(javax.swing.JTextField)">setPDBDirField(JTextField)</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/ConfigPDBInstallPanel.html" title="class in org.biojava.bio.structure.align.gui">ConfigPDBInstallPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#setPdbDirSplit(boolean)">setPdbDirSplit(boolean)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#setPdbDirSplit(boolean)">setPdbDirSplit(boolean)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#setPdbFilePath(java.lang.String)">setPdbFilePath(String)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>
<div class="block">mandatory argument to set the location of PDB files.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#setPdbFilePath(java.lang.String)">setPdbFilePath(String)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/UserConfiguration.html#setPdbFilePath(java.lang.String)">setPdbFilePath(String)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/UserConfiguration.html" title="class in org.biojava.bio.structure.align.util">UserConfiguration</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#setPDBFlag(boolean)">setPDBFlag(boolean)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">flag if group has 3D data .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#setPDBFlag(boolean)">setPDBFlag(boolean)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">flag if group has 3D data.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#setPDBHeader(org.biojava.bio.structure.PDBHeader)">setPDBHeader(PDBHeader)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">the the header information for this PDB file</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#setPDBHeader(org.biojava.bio.structure.PDBHeader)">setPDBHeader(PDBHeader)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#setPdbId(java.lang.String)">setPdbId(String)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/FlatFileInstallation.html#setPDBInfoFile(java.io.File)">setPDBInfoFile(File)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/FlatFileInstallation.html" title="class in org.biojava.bio.structure.server">FlatFileInstallation</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/SimpleStructureServer.html#setPDBInstallation(org.biojava.bio.structure.server.PDBInstallation)">setPDBInstallation(PDBInstallation)</a></span> - Method in class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/SimpleStructureServer.html" title="class in org.biojava.bio.structure.server">SimpleStructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureServer.html#setPDBInstallation(org.biojava.bio.structure.server.PDBInstallation)">setPDBInstallation(PDBInstallation)</a></span> - Method in interface org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureServer.html" title="interface in org.biojava.bio.structure.server">StructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#setPDBline(java.lang.String)">setPDBline(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>replaced by <a href="./org/biojava/bio/structure/PDBRecord.html#toPDB()"><code>PDBRecord.toPDB()</code></a></i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#setPDBline(java.lang.String)">setPDBline(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>&nbsp;</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcidImpl.html#setPDBName(java.lang.String)">setPDBName(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcidImpl.html" title="class in org.biojava.bio.structure">AminoAcidImpl</a></dt>
<dd>
<div class="block">set three character name of AminoAcid.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#setPDBName(java.lang.String)">setPDBName(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">Set the PDB 3 letter name for this group.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#setPDBName(java.lang.String)">setPDBName(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">set three character name of Group .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/StructureGroup.html#setPDBName(java.lang.String)">setPDBName(String)</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/StructureGroup.html" title="class in org.biojava3.protmod.structure">StructureGroup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/StructureGroup.html#setPDBResidueNumber(org.biojava.bio.structure.ResidueNumber)">setPDBResidueNumber(ResidueNumber)</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/StructureGroup.html" title="class in org.biojava3.protmod.structure">StructureGroup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#setPDBresnum1(java.lang.String[])">setPDBresnum1(String[])</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#setPDBresnum2(java.lang.String[])">setPDBresnum2(String[])</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#setPDBserial(int)">setPDBserial(int)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">set PDB atom number.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#setPDBserial(int)">setPDBserial(int)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>
<div class="block">set PDB atom number.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/ConfigPDBInstallPanel.html#setPdbSplit(javax.swing.JCheckBox)">setPdbSplit(JCheckBox)</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/ConfigPDBInstallPanel.html" title="class in org.biojava.bio.structure.align.gui">ConfigPDBInstallPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html#setPdbx_align_begin(java.lang.String)">setPdbx_align_begin(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRef</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setPdbx_ambiguous_flag(java.lang.String)">setPdbx_ambiguous_flag(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#setPdbx_auth_seq_align_beg(java.lang.String)">setPdbx_auth_seq_align_beg(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#setPdbx_auth_seq_align_end(java.lang.String)">setPdbx_auth_seq_align_end(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html#setPdbx_blank_PDB_chainid_flag(java.lang.String)">setPdbx_blank_PDB_chainid_flag(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructAsym</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html#setPdbx_CASP_flag(java.lang.String)">setPdbx_CASP_flag(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html" title="class in org.biojava.bio.structure.io.mmcif.model">Struct</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setPdbx_data_cutoff_high_absF(java.lang.String)">setPdbx_data_cutoff_high_absF(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setPdbx_data_cutoff_high_rms_absF(java.lang.String)">setPdbx_data_cutoff_high_rms_absF(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setPdbx_data_cutoff_low_absF(java.lang.String)">setPdbx_data_cutoff_low_absF(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html#setPdbx_db_accession(java.lang.String)">setPdbx_db_accession(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRef</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#setPdbx_db_accession(java.lang.String)">setPdbx_db_accession(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#setPdbx_db_align_beg_ins_code(java.lang.String)">setPdbx_db_align_beg_ins_code(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#setPdbx_db_align_end_ins_code(java.lang.String)">setPdbx_db_align_end_ins_code(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html#setPdbx_description(java.lang.String)">setPdbx_description(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model">Entity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html#setPdbx_descriptor(java.lang.String)">setPdbx_descriptor(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html" title="class in org.biojava.bio.structure.io.mmcif.model">Struct</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html#setPdbx_ec(java.lang.String)">setPdbx_ec(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model">Entity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setPdbx_formal_charge(java.lang.String)">setPdbx_formal_charge(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setPdbx_formal_charge(java.lang.String)">setPdbx_formal_charge(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html#setPdbx_fragment(java.lang.String)">setPdbx_fragment(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model">Entity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setPdbx_ideal_coordinates_details(java.lang.String)">setPdbx_ideal_coordinates_details(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setPdbx_ideal_coordinates_missing_flag(java.lang.String)">setPdbx_ideal_coordinates_missing_flag(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setPdbx_initial_date(java.lang.String)">setPdbx_initial_date(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setPdbx_isotropic_thermal_model(java.lang.String)">setPdbx_isotropic_thermal_model(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructKeywords.html#setPdbx_keywords(java.lang.String)">setPdbx_keywords(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructKeywords.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructKeywords</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setPdbx_ls_cross_valid_method(java.lang.String)">setPdbx_ls_cross_valid_method(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setPdbx_ls_sigma_F(java.lang.String)">setPdbx_ls_sigma_F(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setPdbx_ls_sigma_I(java.lang.String)">setPdbx_ls_sigma_I(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setPdbx_method_to_determine_struct(java.lang.String)">setPdbx_method_to_determine_struct(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setPdbx_model_coordinates_db_code(java.lang.String)">setPdbx_model_coordinates_db_code(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setPdbx_model_coordinates_details(java.lang.String)">setPdbx_model_coordinates_details(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setPdbx_model_coordinates_missing_flag(java.lang.String)">setPdbx_model_coordinates_missing_flag(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html#setPdbx_model_details(java.lang.String)">setPdbx_model_details(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html" title="class in org.biojava.bio.structure.io.mmcif.model">Struct</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html#setPdbx_model_type_details(java.lang.String)">setPdbx_model_type_details(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html" title="class in org.biojava.bio.structure.io.mmcif.model">Struct</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html#setPdbx_modified(java.lang.String)">setPdbx_modified(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructAsym</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setPdbx_modified_date(java.lang.String)">setPdbx_modified_date(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html#setPdbx_mutation(java.lang.String)">setPdbx_mutation(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model">Entity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html#setPdbx_number_of_molecules(java.lang.String)">setPdbx_number_of_molecules(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model">Entity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AuditAuthor.html#setPdbx_ordinal(java.lang.String)">setPdbx_ordinal(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AuditAuthor.html" title="class in org.biojava.bio.structure.io.mmcif.model">AuditAuthor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setPdbx_overall_ESU_R(java.lang.String)">setPdbx_overall_ESU_R(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setPdbx_overall_ESU_R_Free(java.lang.String)">setPdbx_overall_ESU_R_Free(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setPdbx_overall_phase_error(java.lang.String)">setPdbx_overall_phase_error(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#setPdbx_PDB_id_code(java.lang.String)">setPdbx_PDB_id_code(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setPdbx_PDB_ins_code(java.lang.String)">setPdbx_PDB_ins_code(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setPdbx_PDB_model_num(java.lang.String)">setPdbx_PDB_model_num(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setPdbx_processing_site(java.lang.String)">setPdbx_processing_site(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setPdbx_R_Free_selection_details(java.lang.String)">setPdbx_R_Free_selection_details(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setPdbx_refine_id(java.lang.String)">setPdbx_refine_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setPdbx_release_status(java.lang.String)">setPdbx_release_status(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setPdbx_replaced_by(java.lang.String)">setPdbx_replaced_by(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setPdbx_replaces(java.lang.String)">setPdbx_replaces(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#setPdbx_seq_align_beg_ins_code(java.lang.String)">setPdbx_seq_align_beg_ins_code(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#setPdbx_seq_align_end_ins_code(java.lang.String)">setPdbx_seq_align_end_ins_code(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html#setPdbx_seq_one_letter_code(java.lang.String)">setPdbx_seq_one_letter_code(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRef</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setPdbx_solvent_ion_probe_radii(java.lang.String)">setPdbx_solvent_ion_probe_radii(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setPdbx_solvent_shrinkage_radii(java.lang.String)">setPdbx_solvent_shrinkage_radii(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setPdbx_solvent_vdw_probe_radii(java.lang.String)">setPdbx_solvent_vdw_probe_radii(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setPdbx_starting_model(java.lang.String)">setPdbx_starting_model(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setPdbx_stereochem_target_val_spec_case(java.lang.String)">setPdbx_stereochem_target_val_spec_case(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setPdbx_stereochemistry_target_values(java.lang.String)">setPdbx_stereochemistry_target_values(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#setPdbx_strand_id(java.lang.String)">setPdbx_strand_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setPdbx_subcomponent_list(java.lang.String)">setPdbx_subcomponent_list(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setPdbx_synonyms(java.lang.String)">setPdbx_synonyms(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setPdbx_type(java.lang.String)">setPdbx_type(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#setPercId(int)">setPercId(int)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setPermutationSize(int)">setPermutationSize(int)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#setPmid(java.lang.String)">setPmid(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>
<div class="block">Set the value of pmid</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setPolymerType(org.biojava.bio.structure.io.mmcif.chem.PolymerType)">setPolymerType(PolymerType)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/AlignedPosition.html#setPos(int, int)">setPos(int, int)</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/AlignedPosition.html" title="class in org.biojava.bio.structure.gui.util">AlignedPosition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#setPos1(int)">setPos1(int)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/AlignedPosition.html#setPos1(int)">setPos1(int)</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/AlignedPosition.html" title="class in org.biojava.bio.structure.gui.util">AlignedPosition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#setPos2(int)">setPos2(int)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/AlignedPosition.html#setPos2(int)">setPos2(int)</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/AlignedPosition.html" title="class in org.biojava.bio.structure.gui.util">AlignedPosition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/SimplePoint.html#setPosition(int)">setPosition(int)</a></span> - Method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/SimplePoint.html" title="class in org.biojava3.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/JMatrixPanel.html#setPreferredSize()">setPreferredSize()</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/JMatrixPanel.html" title="class in org.biojava.bio.structure.gui">JMatrixPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#setPrintCE(boolean)">setPrintCE(boolean)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>
<div class="block">Display the output string in CE style</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileConvert.html#setPrintConnections(boolean)">setPrintConnections(boolean)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileConvert.html" title="class in org.biojava.bio.structure.io">FileConvert</a></dt>
<dd>
<div class="block">enable/disable printing of connections
 connections are sometimes buggy in PDB files
 so there are some cases where one might turn this off.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#setPrintFatCat(boolean)">setPrintFatCat(boolean)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#setPrintXML(boolean)">setPrintXML(boolean)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setProbability(double)">setProbability(double)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/GuideTree.Node.html#setProfile(org.biojava3.alignment.template.Profile)">setProfile(Profile&lt;S, C&gt;)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/GuideTree.Node.html" title="class in org.biojava3.alignment">GuideTree.Node</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/NeedlemanWunsch.html#setProfile(java.util.List, java.util.List)">setProfile(List&lt;AlignedSequence.Step&gt;, List&lt;AlignedSequence.Step&gt;)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/NeedlemanWunsch.html" title="class in org.biojava3.alignment">NeedlemanWunsch</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/routines/AnchoredPairwiseSequenceAligner.html#setProfile(java.util.List, java.util.List)">setProfile(List&lt;AlignedSequence.Step&gt;, List&lt;AlignedSequence.Step&gt;)</a></span> - Method in class org.biojava3.alignment.routines.<a href="./org/biojava3/alignment/routines/AnchoredPairwiseSequenceAligner.html" title="class in org.biojava3.alignment.routines">AnchoredPairwiseSequenceAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleProfileProfileAligner.html#setProfile(java.util.List, java.util.List)">setProfile(List&lt;AlignedSequence.Step&gt;, List&lt;AlignedSequence.Step&gt;)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleProfileProfileAligner.html" title="class in org.biojava3.alignment">SimpleProfileProfileAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SmithWaterman.html#setProfile(java.util.List, java.util.List)">setProfile(List&lt;AlignedSequence.Step&gt;, List&lt;AlignedSequence.Step&gt;)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SmithWaterman.html" title="class in org.biojava3.alignment">SmithWaterman</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#setProfile(java.util.List, java.util.List)">setProfile(List&lt;AlignedSequence.Step&gt;, List&lt;AlignedSequence.Step&gt;)</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava3.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/GuideTreeNode.html#setProfile(org.biojava3.alignment.template.Profile)">setProfile(Profile&lt;S, C&gt;)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/GuideTreeNode.html" title="interface in org.biojava3.alignment.template">GuideTreeNode</a></dt>
<dd>
<div class="block">Stores the given profile.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/GuideTree.Node.html#setProfileFuture(java.util.concurrent.Future)">setProfileFuture(Future&lt;ProfilePair&lt;S, C&gt;&gt;)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/GuideTree.Node.html" title="class in org.biojava3.alignment">GuideTree.Node</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/GuideTreeNode.html#setProfileFuture(java.util.concurrent.Future)">setProfileFuture(Future&lt;ProfilePair&lt;S, C&gt;&gt;)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/GuideTreeNode.html" title="interface in org.biojava3.alignment.template">GuideTreeNode</a></dt>
<dd>
<div class="block">Stores the given profile future.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html#setProgram(java.lang.String)">setProgram(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemCompDescriptor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html#setProgram_version(java.lang.String)">setProgram_version(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemCompDescriptor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#setProperties(java.util.Map)">setProperties(Map&lt;String, Object&gt;)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">properties of this amino acid. currerntly available properties.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#setProperties(java.util.Map)">setProperties(Map&lt;String, Object&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">properties of this amino acid. currerntly available properties.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/DBReferenceInfo.html#setProperties(java.util.LinkedHashMap)">setProperties(LinkedHashMap&lt;String, String&gt;)</a></span> - Method in class org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/DBReferenceInfo.html" title="class in org.biojava3.core.sequence.features">DBReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#setProperty(java.lang.String, java.lang.Object)">setProperty(String, Object)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">set a single property .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#setProperty(java.lang.String, java.lang.Object)">setProperty(String, Object)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">set a single property .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#setProxySequenceReader(org.biojava3.core.sequence.template.SequenceReader)">setProxySequenceReader(SequenceReader&lt;C&gt;)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>
<div class="block">Very important method that allows external mappings of sequence data and features.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationImpl.Builder.html#setPsimodId(java.lang.String)">setPsimodId(String)</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationImpl.Builder.html" title="class in org.biojava3.protmod">ProteinModificationImpl.Builder</a></dt>
<dd>
<div class="block">Set the PSI-MOD ID.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationImpl.Builder.html#setPsimodName(java.lang.String)">setPsimodName(String)</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationImpl.Builder.html" title="class in org.biojava3.protmod">ProteinModificationImpl.Builder</a></dt>
<dd>
<div class="block">Set the PSI-MOD name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#setPublicationDate(int)">setPublicationDate(int)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#setPublished(boolean)">setPublished(boolean)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#setPublisher(java.lang.String)">setPublisher(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#setPx(int)">setPx(int)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/QuantityFeature.html#setQuantities(java.util.List)">setQuantities(List&lt;Number&gt;)</a></span> - Method in class org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/QuantityFeature.html" title="class in org.biojava3.core.sequence.features">QuantityFeature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html#setQuery(S)">setQuery(S)</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html" title="class in org.biojava3.alignment.template">AbstractPairwiseSequenceAligner</a></dt>
<dd>
<div class="block">Sets the query <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractProfileProfileAligner.html#setQuery(org.biojava3.alignment.template.Profile)">setQuery(Profile&lt;S, C&gt;)</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractProfileProfileAligner.html" title="class in org.biojava3.alignment.template">AbstractProfileProfileAligner</a></dt>
<dd>
<div class="block">Sets the query <a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template"><code>Profile</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/MutableProfilePair.html#setQuery(org.biojava3.alignment.template.Profile)">setQuery(Profile&lt;S, C&gt;)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/MutableProfilePair.html" title="interface in org.biojava3.alignment.template">MutableProfilePair</a></dt>
<dd>
<div class="block">Sets the first <a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template"><code>Profile</code></a> of the pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/MutableSequencePair.html#setQuery(org.biojava3.alignment.template.AlignedSequence)">setQuery(AlignedSequence&lt;S, C&gt;)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/MutableSequencePair.html" title="interface in org.biojava3.alignment.template">MutableSequencePair</a></dt>
<dd>
<div class="block">Sets the first <a href="./org/biojava3/alignment/template/AlignedSequence.html" title="interface in org.biojava3.alignment.template"><code>AlignedSequence</code></a> of the pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#setRanges(java.util.List)">setRanges(List&lt;String&gt;)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ProteinModificationIdentifier.html#setRecordAdditionalAttachments(boolean)">setRecordAdditionalAttachments(boolean)</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ProteinModificationIdentifier.html" title="class in org.biojava3.protmod.structure">ProteinModificationIdentifier</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcid.html#setRecordType(java.lang.String)">setRecordType(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcid.html" title="interface in org.biojava.bio.structure">AminoAcid</a></dt>
<dd>
<div class="block">Allows to distinguish between amino acids that are provided
                as ATOM records and a SEQRES records.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcidImpl.html#setRecordType(java.lang.String)">setRecordType(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcidImpl.html" title="class in org.biojava.bio.structure">AminoAcidImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ProteinModificationIdentifier.html#setRecordUnidentifiableCompounds(boolean)">setRecordUnidentifiableCompounds(boolean)</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ProteinModificationIdentifier.html" title="class in org.biojava3.protmod.structure">ProteinModificationIdentifier</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setReduceInitialFragments(boolean)">setReduceInitialFragments(boolean)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#setRef(java.lang.String)">setRef(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>
<div class="block">Set the value of the ref.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#setRef_id(java.lang.String)">setRef_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setRefChainId(java.lang.String)">setRefChainId(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>
<div class="block">Returns the ref chain id value.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#setRefn(java.lang.String)">setRefn(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>
<div class="block">Set the value of the refn</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html#setReplaces(java.lang.String)">setReplaces(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html" title="class in org.biojava.bio.structure.io.mmcif.model">DatabasePDBrev</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationImpl.Builder.html#setResidId(java.lang.String)">setResidId(String)</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationImpl.Builder.html" title="class in org.biojava3.protmod">ProteinModificationImpl.Builder</a></dt>
<dd>
<div class="block">Set the RESID ID.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationImpl.Builder.html#setResidName(java.lang.String)">setResidName(String)</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationImpl.Builder.html" title="class in org.biojava3.protmod">ProteinModificationImpl.Builder</a></dt>
<dd>
<div class="block">Set the RESID name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/AtomInfo.html#setResidueName(java.lang.String)">setResidueName(String)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/AtomInfo.html" title="class in org.biojava.bio.structure.align.gui.jmol">AtomInfo</a></dt>
<dd>
<div class="block">Including insertion code</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/AtomInfo.html#setResidueNumber(java.lang.String)">setResidueNumber(String)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/AtomInfo.html" title="class in org.biojava.bio.structure.align.gui.jmol">AtomInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#setResidueNumber(org.biojava.bio.structure.ResidueNumber)">setResidueNumber(ResidueNumber)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">sets the ResidueNumber for this Group</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#setResidueNumber(java.lang.String, java.lang.Integer, java.lang.Character)">setResidueNumber(String, Integer, Character)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">Utility method to temporarily set a chainID for a group, if a parent chain object does not exist yet.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#setResidueNumber(org.biojava.bio.structure.ResidueNumber)">setResidueNumber(ResidueNumber)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#setResidueNumber(java.lang.String, java.lang.Integer, java.lang.Character)">setResidueNumber(String, Integer, Character)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/StructureGroup.html#setResidueNumber(int)">setResidueNumber(int)</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/StructureGroup.html" title="class in org.biojava3.protmod.structure">StructureGroup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setResidueType(org.biojava.bio.structure.io.mmcif.chem.ResidueType)">setResidueType(ResidueType)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setResNames(java.lang.String)">setResNames(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SSBond.html#setResnum1(java.lang.String)">setResnum1(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SSBond.html" title="class in org.biojava.bio.structure">SSBond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SSBond.html#setResnum2(java.lang.String)">setResnum2(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SSBond.html" title="class in org.biojava.bio.structure">SSBond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#setResolution(float)">setResolution(float)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#setResScore(double)">setResScore(double)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/JointFragments.html#setRms(double)">setRms(double)</a></span> - Method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/JointFragments.html" title="class in org.biojava.bio.structure.align.helper">JointFragments</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#setRms(double)">setRms(double)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>
<div class="block">the rms in the structurally equivalent regions</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#setRms(double)">setRms(double)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFP.html#setRmsd(double)">setRmsd(double)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFP.html" title="class in org.biojava.bio.structure.align.model">AFP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#setRmsdCut(java.lang.Double)">setRmsdCut(Double)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>
<div class="block">The cutoff to be used during AFP detection</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#setRmsdThr(java.lang.Double)">setRmsdThr(Double)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#setRmsdThrJoin(java.lang.Double)">setRmsdThrJoin(Double)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#setRot(org.biojava.bio.structure.jama.Matrix)">setRot(Matrix)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/IndexPair.html#setRow(short)">setRow(short)</a></span> - Method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/IndexPair.html" title="class in org.biojava.bio.structure.align.helper">IndexPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html#setRows(java.util.List)">setRows(List&lt;AminoAcidCompound&gt;)</a></span> - Method in class org.biojava3.alignment.aaindex.<a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava3.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#setRunBackground(boolean)">setRunBackground(boolean)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/JMatrixPanel.html#setSaturation(float)">setSaturation(float)</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/JMatrixPanel.html" title="class in org.biojava.bio.structure.gui">JMatrixPanel</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/DefaultMatrixMapper.html#setSaturation(float)">setSaturation(float)</a></span> - Method in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/DefaultMatrixMapper.html" title="class in org.biojava.bio.structure.gui.util.color">DefaultMatrixMapper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#setSaveOutputDir(java.lang.String)">setSaveOutputDir(String)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/JMatrixPanel.html#setScale(float)">setScale(float)</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/JMatrixPanel.html" title="class in org.biojava.bio.structure.gui">JMatrixPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/CoordManager.html#setScale(float)">setScale(float)</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/CoordManager.html" title="class in org.biojava.bio.structure.gui.util">CoordManager</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html#setScale(float)">setScale(float)</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html" title="class in org.biojava.bio.structure.gui.util">SequenceScalePanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/BigSqrt.html#setScale(int)">setScale(int)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/BigSqrt.html" title="class in org.biojava.bio.structure.secstruc">BigSqrt</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html#setScale(int)">setScale(int)</a></span> - Method in class org.biojava3.alignment.aaindex.<a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava3.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/JMatrixPanel.html#setScalevalue(float)">setScalevalue(float)</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/JMatrixPanel.html" title="class in org.biojava.bio.structure.gui">JMatrixPanel</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/DefaultMatrixMapper.html#setScalevalue(double)">setScalevalue(double)</a></span> - Method in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/DefaultMatrixMapper.html" title="class in org.biojava.bio.structure.gui.util.color">DefaultMatrixMapper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopFactory.html#setScopDatabase(org.biojava.bio.structure.scop.ScopDatabase)">setScopDatabase(ScopDatabase)</a></span> - Static method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopFactory.html" title="class in org.biojava.bio.structure.scop">ScopFactory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/ScopDescriptions.html#setScopDescription(java.util.List)">setScopDescription(List&lt;ScopDescription&gt;)</a></span> - Method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/ScopDescriptions.html" title="class in org.biojava.bio.structure.scop.server">ScopDescriptions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/ScopDomains.html#setScopDomain(java.util.List)">setScopDomain(List&lt;ScopDomain&gt;)</a></span> - Method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/ScopDomains.html" title="class in org.biojava.bio.structure.scop.server">ScopDomains</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#setScopId(java.lang.String)">setScopId(String)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/ScopNodes.html#setScopNode(java.util.List)">setScopNode(List&lt;ScopNode&gt;)</a></span> - Method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/ScopNodes.html" title="class in org.biojava.bio.structure.scop.server">ScopNodes</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopInstallation.html#setScopVersion(java.lang.String)">setScopVersion(String)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopInstallation.html" title="class in org.biojava.bio.structure.scop">ScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFP.html#setScore(double)">setScore(double)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFP.html" title="class in org.biojava.bio.structure.align.model">AFP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/Alignable.html#setScore(float)">setScore(float)</a></span> - Method in interface org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/Alignable.html" title="interface in org.biojava.bio.structure.align.pairwise">Alignable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#setScore(float)">setScore(float)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html#setScore(float)">setScore(float)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">StrCompAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Domain.html#setScore(double)">setScore(double)</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Domain.html" title="class in org.biojava.bio.structure.domain.pdp">Domain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Segment.html#setScore(double)">setScore(double)</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Segment.html" title="class in org.biojava.bio.structure.domain.pdp">Segment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/routines/AlignerHelper.html#setScorePoint(int, int, short, short, short, short[][][])">setScorePoint(int, int, short, short, short, short[][][])</a></span> - Static method in class org.biojava3.alignment.routines.<a href="./org/biojava3/alignment/routines/AlignerHelper.html" title="class in org.biojava3.alignment.routines">AlignerHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/routines/AlignerHelper.html#setScorePoint(int, int, short, short, short[][][])">setScorePoint(int, int, short, short, short[][][])</a></span> - Static method in class org.biojava3.alignment.routines.<a href="./org/biojava3/alignment/routines/AlignerHelper.html" title="class in org.biojava3.alignment.routines">AlignerHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/routines/AlignerHelper.html#setScoreVector(int, short, short, short[], boolean, short[][][])">setScoreVector(int, short, short, short[], boolean, short[][][])</a></span> - Static method in class org.biojava3.alignment.routines.<a href="./org/biojava3/alignment/routines/AlignerHelper.html" title="class in org.biojava3.alignment.routines">AlignerHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/routines/AlignerHelper.html#setScoreVector(int, int[], short, short, short[], boolean, short[][][])">setScoreVector(int, int[], short, short, short[], boolean, short[][][])</a></span> - Static method in class org.biojava3.alignment.routines.<a href="./org/biojava3/alignment/routines/AlignerHelper.html" title="class in org.biojava3.alignment.routines">AlignerHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/routines/AlignerHelper.html#setScoreVector(int, int, int, int, short, short, short[], boolean, short[][][])">setScoreVector(int, int, int, int, short, short, short[], boolean, short[][][])</a></span> - Static method in class org.biojava3.alignment.routines.<a href="./org/biojava3/alignment/routines/AlignerHelper.html" title="class in org.biojava3.alignment.routines">AlignerHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/routines/AlignerHelper.html#setScoreVector(int, short, short[], boolean, short[][][])">setScoreVector(int, short, short[], boolean, short[][][])</a></span> - Static method in class org.biojava3.alignment.routines.<a href="./org/biojava3/alignment/routines/AlignerHelper.html" title="class in org.biojava3.alignment.routines">AlignerHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/routines/AlignerHelper.html#setScoreVector(int, int[], short, short[], boolean, short[][][])">setScoreVector(int, int[], short, short[], boolean, short[][][])</a></span> - Static method in class org.biojava3.alignment.routines.<a href="./org/biojava3/alignment/routines/AlignerHelper.html" title="class in org.biojava3.alignment.routines">AlignerHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/routines/AlignerHelper.html#setScoreVector(int, int, int, int, short, short[], boolean, short[][][])">setScoreVector(int, int, int, int, short, short[], boolean, short[][][])</a></span> - Static method in class org.biojava3.alignment.routines.<a href="./org/biojava3/alignment/routines/AlignerHelper.html" title="class in org.biojava3.alignment.routines">AlignerHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/routines/AlignerHelper.html#setScoreVector(int, short, short, short[], boolean, short[][][], int[], int)">setScoreVector(int, short, short, short[], boolean, short[][][], int[], int)</a></span> - Static method in class org.biojava3.alignment.routines.<a href="./org/biojava3/alignment/routines/AlignerHelper.html" title="class in org.biojava3.alignment.routines">AlignerHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/routines/AlignerHelper.html#setScoreVector(int, int, int, int, short, short, short[], boolean, short[][][], int[], int)">setScoreVector(int, int, int, int, short, short, short[], boolean, short[][][], int[], int)</a></span> - Static method in class org.biojava3.alignment.routines.<a href="./org/biojava3/alignment/routines/AlignerHelper.html" title="class in org.biojava3.alignment.routines">AlignerHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/routines/AlignerHelper.html#setScoreVector(int, short, short[], boolean, short[][][], int[], int)">setScoreVector(int, short, short[], boolean, short[][][], int[], int)</a></span> - Static method in class org.biojava3.alignment.routines.<a href="./org/biojava3/alignment/routines/AlignerHelper.html" title="class in org.biojava3.alignment.routines">AlignerHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/routines/AlignerHelper.html#setScoreVector(int, int, int, int, short, short[], boolean, short[][][], int[], int)">setScoreVector(int, int, int, int, short, short[], boolean, short[][][], int[], int)</a></span> - Static method in class org.biojava3.alignment.routines.<a href="./org/biojava3/alignment/routines/AlignerHelper.html" title="class in org.biojava3.alignment.routines">AlignerHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#setScoringStrategy(java.lang.Integer)">setScoringStrategy(Integer)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>
<div class="block">Set the scoring strategy to use. 0 is default CE scoring scheme. 1 uses
 Side chain orientation.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setSecretion(java.lang.String)">setSecretion(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcid.html#setSecStruc(java.util.Map)">setSecStruc(Map&lt;String, String&gt;)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcid.html" title="interface in org.biojava.bio.structure">AminoAcid</a></dt>
<dd>
<div class="block">Specifies the secondary structure as a Map.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcidImpl.html#setSecStruc(java.util.Map)">setSecStruc(Map&lt;String, String&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcidImpl.html" title="class in org.biojava.bio.structure">AminoAcidImpl</a></dt>
<dd>
<div class="block">set the secondary structure data for this amino acid. the data
 is a Map with the following indeces (@see Secstruc)</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setSeedFragmentLength(int)">setSeedFragmentLength(int)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setSeedRmsdCutoff(float)">setSeedRmsdCutoff(float)</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Domain.html#setSegments(java.util.List)">setSegments(List&lt;Segment&gt;)</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Domain.html" title="class in org.biojava.bio.structure.domain.pdp">Domain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/JMatrixPanel.html#setSelectedAlignmentPos(int)">setSelectedAlignmentPos(int)</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/JMatrixPanel.html" title="class in org.biojava.bio.structure.gui">JMatrixPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html#setSelectedAlignmentPos(int)">setSelectedAlignmentPos(int)</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html" title="class in org.biojava.bio.structure.gui">ScaleableMatrixPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/structure/gui/JmolViewerImpl.html#setSelection(org.biojava3.structure.gui.Selection)">setSelection(Selection)</a></span> - Method in class org.biojava3.structure.gui.<a href="./org/biojava3/structure/gui/JmolViewerImpl.html" title="class in org.biojava3.structure.gui">JmolViewerImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/structure/gui/Selection.html#setSelection(java.util.List)">setSelection(List&lt;AtomInfo&gt;)</a></span> - Method in interface org.biojava3.structure.gui.<a href="./org/biojava3/structure/gui/Selection.html" title="interface in org.biojava3.structure.gui">Selection</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/structure/gui/SelectionImpl.html#setSelection(java.util.List)">setSelection(List&lt;AtomInfo&gt;)</a></span> - Method in class org.biojava3.structure.gui.<a href="./org/biojava3/structure/gui/SelectionImpl.html" title="class in org.biojava3.structure.gui">SelectionImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/structure/gui/StructureViewer.html#setSelection(org.biojava3.structure.gui.Selection)">setSelection(Selection)</a></span> - Method in interface org.biojava3.structure.gui.<a href="./org/biojava3/structure/gui/StructureViewer.html" title="interface in org.biojava3.structure.gui">StructureViewer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#setSeq_align_beg(java.lang.String)">setSeq_align_beg(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#setSeq_align_end(java.lang.String)">setSeq_align_end(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html#setSeq_id(java.lang.String)">setSeq_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxNonPolyScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxNonPolyScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html#setSeq_id(java.lang.String)">setSeq_id(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/PdbxPolySeqScheme.html" title="class in org.biojava.bio.structure.io.mmcif.model">PdbxPolySeqScheme</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#setSeqBegin(int)">setSeqBegin(int)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Initial sequence number of the PDB sequence segment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#setSeqEnd(int)">setSeqEnd(int)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Ending sequence number   of the PDB sequence segment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ResidueNumber.html#setSeqNum(java.lang.Integer)">setSeqNum(Integer)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ResidueNumber.html" title="class in org.biojava.bio.structure">ResidueNumber</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#setSeqResGroups(java.util.List)">setSeqResGroups(List&lt;Group&gt;)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Set the list of SeqResGroups for this chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#setSeqResGroups(java.util.List)">setSeqResGroups(List&lt;Group&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">Set the list of SeqResGroups for this chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/edits/Edit.AbstractEdit.html#setSequence(org.biojava3.core.sequence.template.Sequence)">setSequence(Sequence&lt;C&gt;)</a></span> - Method in class org.biojava3.core.sequence.edits.<a href="./org/biojava3/core/sequence/edits/Edit.AbstractEdit.html" title="class in org.biojava3.core.sequence.edits">Edit.AbstractEdit</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/SequenceOptimizationHints.html#setSequenceCollection(org.biojava3.core.sequence.SequenceOptimizationHints.SequenceCollection)">setSequenceCollection(SequenceOptimizationHints.SequenceCollection)</a></span> - Static method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/SequenceOptimizationHints.html" title="class in org.biojava3.core.sequence">SequenceOptimizationHints</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/MutableProfile.html#setSequences(java.util.List)">setSequences(List&lt;AlignedSequence&lt;S, C&gt;&gt;)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/MutableProfile.html" title="interface in org.biojava3.alignment.template">MutableProfile</a></dt>
<dd>
<div class="block">Sets entirely new set of <a href="./org/biojava3/alignment/template/AlignedSequence.html" title="interface in org.biojava3.alignment.template"><code>AlignedSequence</code></a>s.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#setSequenceScore(java.lang.Double)">setSequenceScore(Double)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/SequenceOptimizationHints.html#setSequenceUsage(org.biojava3.core.sequence.SequenceOptimizationHints.SequenceUsage)">setSequenceUsage(SequenceOptimizationHints.SequenceUsage)</a></span> - Static method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/SequenceOptimizationHints.html" title="class in org.biojava3.core.sequence">SequenceOptimizationHints</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setSequentialAlignment(boolean)">setSequentialAlignment(boolean)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">Set whether this alignment has the normal topology, ie the residues
 aligned in each block increase sequentially over the original protein.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#setSeqWeight(double)">setSeqWeight(double)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>
<div class="block">Should sequence conservation be considered as part of the alignment?</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SSBond.html#setSerNum(int)">setSerNum(int)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SSBond.html" title="class in org.biojava.bio.structure">SSBond</a></dt>
<dd>
<div class="block">get serial number of this SSBOND in PDB file</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#setServer(java.lang.String)">setServer(String)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html#setServer(java.lang.String)">setServer(String)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/RemoteScopInstallation.html" title="class in org.biojava.bio.structure.scop">RemoteScopInstallation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setShortAlign(boolean)">setShortAlign(boolean)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/AbstractFeature.html#setShortDescription(java.lang.String)">setShortDescription(String)</a></span> - Method in class org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/AbstractFeature.html" title="class in org.biojava3.core.sequence.features">AbstractFeature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/FeatureInterface.html#setShortDescription(java.lang.String)">setShortDescription(String)</a></span> - Method in interface org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/FeatureInterface.html" title="interface in org.biojava3.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">Set the short description that can be used to describe the feature</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/AminoAcidComposition.html#setShortName(java.lang.String)">setShortName(String)</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/AminoAcidComposition.html" title="class in org.biojava3.aaproperties.xml">AminoAcidComposition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/CodonCompound.html#setShortName(java.lang.String)">setShortName(String)</a></span> - Method in class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/CodonCompound.html" title="class in org.biojava3.core.sequence.compound">CodonCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompound.html#setShortName(java.lang.String)">setShortName(String)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompound.html" title="class in org.biojava3.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/Compound.html#setShortName(java.lang.String)">setShortName(String)</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/CaseInsensitiveCompound.html#setShortName(java.lang.String)">setShortName(String)</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava3.core.sequence.transcription">CaseInsensitiveCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/Table.Codon.html#setShortName(java.lang.String)">setShortName(String)</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/Table.Codon.html" title="class in org.biojava3.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#setShow3d(boolean)">setShow3d(boolean)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#setShowAFPRanges(boolean)">setShowAFPRanges(boolean)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#setShowAFPRanges(boolean)">setShowAFPRanges(boolean)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#setShowDBresult(java.lang.String)">setShowDBresult(String)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#setShowMenu(boolean)">setShowMenu(boolean)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setSimilarity(double)">setSimilarity(double)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Site.html#setSiteID(java.lang.String)">setSiteID(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Site.html" title="class in org.biojava.bio.structure">Site</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#setSites(java.util.List)">setSites(List&lt;Site&gt;)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#setSites(java.util.List)">setSites(List&lt;Site&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Domain.html#setSize(int)">setSize(int)</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Domain.html" title="class in org.biojava.bio.structure.domain.pdp">Domain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setSolvent_model_details(java.lang.String)">setSolvent_model_details(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setSolvent_model_param_bsol(java.lang.String)">setSolvent_model_param_bsol(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html#setSolvent_model_param_ksol(java.lang.String)">setSolvent_model_param_ksol(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Refine.html" title="class in org.biojava.bio.structure.io.mmcif.model">Refine</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/AbstractFeature.html#setSource(java.lang.String)">setSource(String)</a></span> - Method in class org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/AbstractFeature.html" title="class in org.biojava3.core.sequence.features">AbstractFeature</a></dt>
<dd>
<div class="block">Set the feature source</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/FeatureInterface.html#setSource(java.lang.String)">setSource(String)</a></span> - Method in interface org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/FeatureInterface.html" title="interface in org.biojava3.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">Change the source of the FeatureInterface.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#setSource(java.lang.String)">setSource(String)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>
<div class="block">Added support for the source of this sequence for GFF3 export</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#setSparse(int)">setSparse(int)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#setSpeciesId(int)">setSpeciesId(int)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#setSplit(boolean)">setSplit(boolean)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>
<div class="block">Is the organization of files within the directory split, as on the PDB FTP servers,
 or are all files contained in one directory.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/UserConfiguration.html#setSplit(boolean)">setSplit(boolean)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/UserConfiguration.html" title="class in org.biojava.bio.structure.align.util">UserConfiguration</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html#setSrc_method(java.lang.String)">setSrc_method(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model">Entity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#setSSBonds(java.util.List)">setSSBonds(List&lt;SSBond&gt;)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">set the list of SSBonds for this structure</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#setSSBonds(java.util.List)">setSSBonds(List&lt;SSBond&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">set the list of SSBonds for this structure</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setStandard(boolean)">setStandard(boolean)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html#setStart(org.biojava3.core.sequence.location.template.Point)">setStart(Point)</a></span> - Method in class org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava3.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#setStartPage(java.lang.String)">setStartPage(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html#setStatus(java.lang.String)">setStatus(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html" title="class in org.biojava.bio.structure.io.mmcif.model">DatabasePDBrev</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/routines/AlignerHelper.html#setSteps(int[], java.util.List, java.util.List)">setSteps(int[], List&lt;AlignedSequence.Step&gt;, List&lt;AlignedSequence.Step&gt;)</a></span> - Static method in class org.biojava3.alignment.routines.<a href="./org/biojava3/alignment/routines/AlignerHelper.html" title="class in org.biojava3.alignment.routines">AlignerHelper</a></dt>
<dd>
<div class="block">Sets the alignment path following the list of anchors.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/routines/AlignerHelper.html#setSteps(org.biojava3.alignment.routines.AlignerHelper.Last[][][], boolean, int[], org.biojava3.alignment.routines.AlignerHelper.Last, java.util.List, java.util.List)">setSteps(AlignerHelper.Last[][][], boolean, int[], AlignerHelper.Last, List&lt;AlignedSequence.Step&gt;, List&lt;AlignedSequence.Step&gt;)</a></span> - Static method in class org.biojava3.alignment.routines.<a href="./org/biojava3/alignment/routines/AlignerHelper.html" title="class in org.biojava3.alignment.routines">AlignerHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/routines/AlignerHelper.html#setSteps(org.biojava3.alignment.routines.AlignerHelper.Last[][][], short[][][], java.util.List, java.util.List)">setSteps(AlignerHelper.Last[][][], short[][][], List&lt;AlignedSequence.Step&gt;, List&lt;AlignedSequence.Step&gt;)</a></span> - Static method in class org.biojava3.alignment.routines.<a href="./org/biojava3/alignment/routines/AlignerHelper.html" title="class in org.biojava3.alignment.routines">AlignerHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/routines/AlignerHelper.html#setSteps(org.biojava3.alignment.routines.AlignerHelper.Last[][][], int[], java.util.List, java.util.List)">setSteps(AlignerHelper.Last[][][], int[], List&lt;AlignedSequence.Step&gt;, List&lt;AlignedSequence.Step&gt;)</a></span> - Static method in class org.biojava3.alignment.routines.<a href="./org/biojava3/alignment/routines/AlignerHelper.html" title="class in org.biojava3.alignment.routines">AlignerHelper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#setStepSize(int)">setStepSize(int)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileParsingParameters.html#setStoreEmptySeqRes(boolean)">setStoreEmptySeqRes(boolean)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileParsingParameters.html" title="class in org.biojava.bio.structure.io">FileParsingParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#setStoringScoreMatrix(boolean)">setStoringScoreMatrix(boolean)</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava3.alignment.template">AbstractMatrixAligner</a></dt>
<dd>
<div class="block">Sets choice to cache the score matrix or to save memory by deleting score matrix after alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setStrain(java.lang.String)">setStrain(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/GeneSequence.html#setStrand(org.biojava3.core.sequence.Strand)">setStrand(Strand)</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/GeneSequence.html" title="class in org.biojava3.core.sequence">GeneSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html#setStrand(org.biojava3.core.sequence.Strand)">setStrand(Strand)</a></span> - Method in class org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava3.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#setStrictSCOP(boolean)">setStrictSCOP(boolean)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>
<div class="block">When strictSCOP is enabled, SCOP domain identifiers (eg 'd1gbga_') are
 matched literally to the SCOP database.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/edits/Edit.AbstractEdit.html#setStringSequence(java.lang.String)">setStringSequence(String)</a></span> - Method in class org.biojava3.core.sequence.edits.<a href="./org/biojava3/core/sequence/edits/Edit.AbstractEdit.html" title="class in org.biojava3.core.sequence.edits">Edit.AbstractEdit</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html#setStruct(org.biojava.bio.structure.io.mmcif.model.Struct)">setStruct(Struct)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/ChemCompConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">ChemCompConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html#setStruct(org.biojava.bio.structure.io.mmcif.model.Struct)">setStruct(Struct)</a></span> - Method in interface org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/MMcifConsumer.html" title="interface in org.biojava.bio.structure.io.mmcif">MMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#setStruct(org.biojava.bio.structure.io.mmcif.model.Struct)">setStruct(Struct)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/JmolPanel.html#setStructure(org.biojava.bio.structure.Structure)">setStructure(Structure)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/JmolPanel.html" title="class in org.biojava.bio.structure.align.gui.jmol">JmolPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html#setStructure(org.biojava.bio.structure.Structure)">setStructure(Structure)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html" title="class in org.biojava.bio.structure.align.gui.jmol">StructureAlignmentJmol</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/BiojavaJmol.html#setStructure(org.biojava.bio.structure.Structure)">setStructure(Structure)</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/BiojavaJmol.html" title="class in org.biojava.bio.structure.gui">BiojavaJmol</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/structure/gui/JmolViewerImpl.html#setStructure(org.biojava.bio.structure.Structure)">setStructure(Structure)</a></span> - Method in class org.biojava3.structure.gui.<a href="./org/biojava3/structure/gui/JmolViewerImpl.html" title="class in org.biojava3.structure.gui">JmolViewerImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/structure/gui/StructureViewer.html#setStructure(org.biojava.bio.structure.Structure)">setStructure(Structure)</a></span> - Method in interface org.biojava3.structure.gui.<a href="./org/biojava3/structure/gui/StructureViewer.html" title="interface in org.biojava3.structure.gui">StructureViewer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/events/JmolAlignedPositionListener.html#setStructure1(org.biojava.bio.structure.Structure)">setStructure1(Structure)</a></span> - Method in class org.biojava.bio.structure.gui.events.<a href="./org/biojava/bio/structure/gui/events/JmolAlignedPositionListener.html" title="class in org.biojava.bio.structure.gui.events">JmolAlignedPositionListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/SequenceDisplay.html#setStructure1(org.biojava.bio.structure.Structure)">setStructure1(Structure)</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/SequenceDisplay.html" title="class in org.biojava.bio.structure.gui">SequenceDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/events/JmolAlignedPositionListener.html#setStructure2(org.biojava.bio.structure.Structure)">setStructure2(Structure)</a></span> - Method in class org.biojava.bio.structure.gui.events.<a href="./org/biojava/bio/structure/gui/events/JmolAlignedPositionListener.html" title="class in org.biojava.bio.structure.gui.events">JmolAlignedPositionListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/SequenceDisplay.html#setStructure2(org.biojava.bio.structure.Structure)">setStructure2(Structure)</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/SequenceDisplay.html" title="class in org.biojava.bio.structure.gui">SequenceDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html#setStructureAlignmentJmol(org.biojava.bio.structure.align.gui.jmol.StructureAlignmentJmol)">setStructureAlignmentJmol(StructureAlignmentJmol)</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AligPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/SequenceDisplay.html#setStructurePairAligner(org.biojava.bio.structure.align.StructurePairAligner)">setStructurePairAligner(StructurePairAligner)</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/SequenceDisplay.html" title="class in org.biojava.bio.structure.gui">SequenceDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/AlternativeAlignmentFrame.html#setStructurePairAligner(org.biojava.bio.structure.align.StructurePairAligner)">setStructurePairAligner(StructurePairAligner)</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/AlternativeAlignmentFrame.html" title="class in org.biojava.bio.structure.gui.util">AlternativeAlignmentFrame</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/structure/gui/JmolViewerImpl.html#setStyle(org.biojava3.structure.gui.RenderStyle)">setStyle(RenderStyle)</a></span> - Method in class org.biojava3.structure.gui.<a href="./org/biojava3/structure/gui/JmolViewerImpl.html" title="class in org.biojava3.structure.gui">JmolViewerImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/structure/gui/StructureViewer.html#setStyle(org.biojava3.structure.gui.RenderStyle)">setStyle(RenderStyle)</a></span> - Method in interface org.biojava3.structure.gui.<a href="./org/biojava3/structure/gui/StructureViewer.html" title="interface in org.biojava3.structure.gui">StructureViewer</a></dt>
<dd>
<div class="block">Apply this style to the current selection</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html#setSubLocations(java.util.List)">setSubLocations(List&lt;Location&gt;)</a></span> - Method in class org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava3.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#setSubstitutionMatrix(org.biojava3.alignment.template.SubstitutionMatrix)">setSubstitutionMatrix(SubstitutionMatrix&lt;AminoAcidCompound&gt;)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>
<div class="block">Sets the  substitution matrix to be used for influencing the alignment with sequence conservation information.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#setSubstitutionMatrix(org.biojava3.alignment.template.SubstitutionMatrix)">setSubstitutionMatrix(SubstitutionMatrix&lt;C&gt;)</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava3.alignment.template">AbstractMatrixAligner</a></dt>
<dd>
<div class="block">Sets the substitution matrix.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDescription.html#setSunID(int)">setSunID(int)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDescription.html" title="class in org.biojava.bio.structure.scop">ScopDescription</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#setSunid(java.lang.Integer)">setSunid(Integer)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopNode.html#setSunid(int)">setSunid(int)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopNode.html" title="class in org.biojava.bio.structure.scop">ScopNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#setSuperfamilyId(int)">setSuperfamilyId(int)</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Author.html#setSurname(java.lang.String)">setSurname(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Author.html" title="class in org.biojava.bio.structure">Author</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#setSwissprotId(java.lang.String)">setSwissprotId(String)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">Set the Swissprot id of this chain.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#setSwissprotId(java.lang.String)">setSwissprotId(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">set the Swissprot id of this chains .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/AminoAcidComposition.html#setSymbol(java.lang.String)">setSymbol(String)</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/AminoAcidComposition.html" title="class in org.biojava3.aaproperties.xml">AminoAcidComposition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/Element.html#setSymbol(java.lang.String)">setSymbol(String)</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/Element.html" title="class in org.biojava3.aaproperties.xml">Element</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setSynonyms(java.util.List)">setSynonyms(List&lt;String&gt;)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setSynthetic(java.lang.String)">setSynthetic(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationImpl.Builder.html#setSystematicName(java.lang.String)">setSystematicName(String)</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationImpl.Builder.html" title="class in org.biojava3.protmod">ProteinModificationImpl.Builder</a></dt>
<dd>
<div class="block">Set the systematic name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFP.html#setT(double[])">setT(double[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFP.html" title="class in org.biojava.bio.structure.align.model">AFP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/DBSearchGUI.html#setTabPane(javax.swing.JTabbedPane)">setTabPane(JTabbedPane)</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/DBSearchGUI.html" title="class in org.biojava.bio.structure.align.gui">DBSearchGUI</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html#setTarget(S)">setTarget(S)</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html" title="class in org.biojava3.alignment.template">AbstractPairwiseSequenceAligner</a></dt>
<dd>
<div class="block">Sets the target <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractProfileProfileAligner.html#setTarget(org.biojava3.alignment.template.Profile)">setTarget(Profile&lt;S, C&gt;)</a></span> - Method in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractProfileProfileAligner.html" title="class in org.biojava3.alignment.template">AbstractProfileProfileAligner</a></dt>
<dd>
<div class="block">Sets the target <a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template"><code>Profile</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/MutableProfilePair.html#setTarget(org.biojava3.alignment.template.Profile)">setTarget(Profile&lt;S, C&gt;)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/MutableProfilePair.html" title="interface in org.biojava3.alignment.template">MutableProfilePair</a></dt>
<dd>
<div class="block">Sets the second <a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template"><code>Profile</code></a> of the pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/MutableSequencePair.html#setTarget(org.biojava3.alignment.template.AlignedSequence)">setTarget(AlignedSequence&lt;S, C&gt;)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/MutableSequencePair.html" title="interface in org.biojava3.alignment.template">MutableSequencePair</a></dt>
<dd>
<div class="block">Sets the second <a href="./org/biojava3/alignment/template/AlignedSequence.html" title="interface in org.biojava3.alignment.template"><code>AlignedSequence</code></a> of the pair.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#setTaxonomy(org.biojava3.core.sequence.TaxonomyID)">setTaxonomy(TaxonomyID)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#setTechnique(java.lang.String)">setTechnique(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#setTempFactor(double)">setTempFactor(double)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">get set temp factor .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#setTempFactor(double)">setTempFactor(double)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/AlignmentTextPanel.html#setText(java.lang.String)">setText(String)</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/AlignmentTextPanel.html" title="class in org.biojava.bio.structure.align.gui">AlignmentTextPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBremark.html#setText(java.lang.String)">setText(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBremark.html" title="class in org.biojava.bio.structure.io.mmcif.model">DatabasePDBremark</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructKeywords.html#setText(java.lang.String)">setText(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructKeywords.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructKeywords</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/MyJmolStatusListener.html#setTextField(javax.swing.JTextField)">setTextField(JTextField)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/MyJmolStatusListener.html" title="class in org.biojava.bio.structure.align.gui.jmol">MyJmolStatusListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/ConcurrencyTools.html#setThreadPool(java.util.concurrent.ThreadPoolExecutor)">setThreadPool(ThreadPoolExecutor)</a></span> - Static method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/ConcurrencyTools.html" title="class in org.biojava3.core.util">ConcurrencyTools</a></dt>
<dd>
<div class="block">Sets thread pool to any given <a href="http://java.sun.com/j2se/1.6.0/docs/api/java/util/concurrent/ThreadPoolExecutor.html?is-external=true" title="class or interface in java.util.concurrent"><code>ThreadPoolExecutor</code></a> to allow use of an alternative execution style.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/ConcurrencyTools.html#setThreadPoolCPUsAvailable(int)">setThreadPoolCPUsAvailable(int)</a></span> - Static method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/ConcurrencyTools.html" title="class in org.biojava3.core.util">ConcurrencyTools</a></dt>
<dd>
<div class="block">Sets thread pool to reserve a given number of processor cores for foreground or other use.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/ConcurrencyTools.html#setThreadPoolCPUsFraction(float)">setThreadPoolCPUsFraction(float)</a></span> - Static method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/ConcurrencyTools.html" title="class in org.biojava3.core.util">ConcurrencyTools</a></dt>
<dd>
<div class="block">Sets thread pool to a given fraction of the available processors.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/ConcurrencyTools.html#setThreadPoolDefault()">setThreadPoolDefault()</a></span> - Static method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/ConcurrencyTools.html" title="class in org.biojava3.core.util">ConcurrencyTools</a></dt>
<dd>
<div class="block">Sets thread pool to default of 1 background thread for each processor core.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/ConcurrencyTools.html#setThreadPoolSingle()">setThreadPoolSingle()</a></span> - Static method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/ConcurrencyTools.html" title="class in org.biojava3.core.util">ConcurrencyTools</a></dt>
<dd>
<div class="block">Sets thread pool to a single background thread.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/ConcurrencyTools.html#setThreadPoolSize(int)">setThreadPoolSize(int)</a></span> - Static method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/ConcurrencyTools.html" title="class in org.biojava3.core.util">ConcurrencyTools</a></dt>
<dd>
<div class="block">Sets thread pool to given size.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#setThreads(int)">setThreads(int)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setThree_letter_code(java.lang.String)">setThree_letter_code(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#setTime(int)">setTime(int)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setTissue(java.lang.String)">setTissue(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html#setTitle(java.lang.String)">setTitle(String)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html" title="class in org.biojava.bio.structure.align.gui.jmol">StructureAlignmentJmol</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setTitle(java.lang.String)">setTitle(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/BiojavaJmol.html#setTitle(java.lang.String)">setTitle(String)</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/BiojavaJmol.html" title="class in org.biojava.bio.structure.gui">BiojavaJmol</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html#setTitle(java.lang.String)">setTitle(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html" title="class in org.biojava.bio.structure.io.mmcif.model">Struct</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#setTitle(java.lang.String)">setTitle(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>
<div class="block">Set the value of title</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#setTitle(java.lang.String)">setTitle(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/structure/gui/JmolViewerImpl.html#setTitle(java.lang.String)">setTitle(String)</a></span> - Method in class org.biojava3.structure.gui.<a href="./org/biojava3/structure/gui/JmolViewerImpl.html" title="class in org.biojava3.structure.gui">JmolViewerImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setTMScore(double)">setTMScore(double)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Segment.html#setTo(int)">setTo(int)</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Segment.html" title="class in org.biojava.bio.structure.domain.pdp">Segment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastService.html#setTool(java.lang.String)">setTool(String)</a></span> - Method in class org.biojava3.ws.alignment.qblast.<a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastService.html" title="class in org.biojava3.ws.alignment.qblast">NCBIQBlastService</a></dt>
<dd>
<div class="block">Set the tool identifier for QBlast.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#setTorsionPenalty(double)">setTorsionPenalty(double)</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setTotalLenIni(int)">setTotalLenIni(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setTotalLenOpt(int)">setTotalLenOpt(int)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setTotalRmsdIni(double)">setTotalRmsdIni(double)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">this is the init-RMSD, not the final RMSD after refinement.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setTotalRmsdOpt(double)">setTotalRmsdOpt(double)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>
<div class="block">The RMSD of the final alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/BigSqrt.html#setTraceFlag(boolean)">setTraceFlag(boolean)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/BigSqrt.html" title="class in org.biojava.bio.structure.secstruc">BigSqrt</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#setTrans(org.biojava.bio.structure.Atom)">setTrans(Atom)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setTwi(int[])">setTwi(int[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatAligner.html#setTwistedGroups(org.biojava.bio.structure.Group[])">setTwistedGroups(Group[])</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatAligner.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#setType(java.lang.String)">setType(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html#setType(java.lang.String)">setType(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemCompDescriptor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html#setType(java.lang.String)">setType(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model">Entity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/AbstractFeature.html#setType(java.lang.String)">setType(String)</a></span> - Method in class org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/AbstractFeature.html" title="class in org.biojava3.core.sequence.features">AbstractFeature</a></dt>
<dd>
<div class="block">Set the feature type</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/FeatureInterface.html#setType(java.lang.String)">setType(String)</a></span> - Method in interface org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/FeatureInterface.html" title="interface in org.biojava3.core.sequence.features">FeatureInterface</a></dt>
<dd>
<div class="block">Change the type of this feature.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html#setType_symbol(java.lang.String)">setType_symbol(String)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AtomSite.html" title="class in org.biojava.bio.structure.io.mmcif.model">AtomSite</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/SimplePoint.html#setUncertain(boolean)">setUncertain(boolean)</a></span> - Method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/SimplePoint.html" title="class in org.biojava3.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html#setUniprotbaseURL(java.lang.String)">setUniprotbaseURL(String)</a></span> - Static method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html#setUniprotDirectoryCache(java.lang.String)">setUniprotDirectoryCache(String)</a></span> - Static method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#setUnitv(org.biojava.bio.structure.Atom)">setUnitv(Atom)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/SimplePoint.html#setUnknown(boolean)">setUnknown(boolean)</a></span> - Method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/SimplePoint.html" title="class in org.biojava3.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileParsingParameters.html#setUpdateRemediatedFiles(boolean)">setUpdateRemediatedFiles(boolean)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileParsingParameters.html" title="class in org.biojava.bio.structure.io">FileParsingParameters</a></dt>
<dd>
<div class="block">A flag if local files should be replaced with the latest version of remediated PDB files.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#setUsed(int)">setUsed(int)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#setUsedAtomNames(java.lang.String[])">setUsedAtomNames(String[])</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#setUserCollection(java.util.Collection)">setUserCollection(Collection&lt;Object&gt;)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#setUsername(java.lang.String)">setUsername(String)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/AbstractFeature.html#setUserObject(java.lang.Object)">setUserObject(Object)</a></span> - Method in class org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/AbstractFeature.html" title="class in org.biojava3.core.sequence.features">AbstractFeature</a></dt>
<dd>
<div class="block">Allow the user to associate an object with the feature.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/FeatureInterface.html#setUserObject(java.lang.Object)">setUserObject(Object)</a></span> - Method in interface org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/FeatureInterface.html" title="interface in org.biojava3.core.sequence.features">FeatureInterface</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/AligMatEl.html#setValue(int)">setValue(int)</a></span> - Method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/AligMatEl.html" title="class in org.biojava.bio.structure.align.helper">AligMatEl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/GapArray.html#setValue(int)">setValue(int)</a></span> - Method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/GapArray.html" title="class in org.biojava.bio.structure.align.helper">GapArray</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#setVariant(java.lang.String)">setVariant(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#setVerbose(boolean)">setVerbose(boolean)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#setVersion(java.lang.String)">setVersion(String)</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#setVolume(java.lang.String)">setVolume(String)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html#setWaitForAlignments(boolean)">setWaitForAlignments(boolean)</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html" title="class in org.biojava.bio.structure.align.client">FarmJobRunnable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#setWinSize(java.lang.Integer)">setWinSize(Integer)</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#setX(double)">setX(double)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">Set the X coordinate.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#setX(double)">setX(double)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#setY(double)">setY(double)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">Set the Y coordinate.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#setY(double)">setY(double)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Atom.html#setZ(double)">setZ(double)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Atom.html" title="interface in org.biojava.bio.structure">Atom</a></dt>
<dd>
<div class="block">Set the Z coordinate.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#setZ(double)">setZ(double)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/structure/gui/JmolViewerImpl.html#setZoom(int)">setZoom(int)</a></span> - Method in class org.biojava3.structure.gui.<a href="./org/biojava3/structure/gui/JmolViewerImpl.html" title="class in org.biojava3.structure.gui">JmolViewerImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/structure/gui/StructureViewer.html#setZoom(int)">setZoom(int)</a></span> - Method in interface org.biojava3.structure.gui.<a href="./org/biojava3/structure/gui/StructureViewer.html" title="interface in org.biojava3.structure.gui">StructureViewer</a></dt>
<dd>
<div class="block">Set the Zoom level</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#shift(org.biojava.bio.structure.Structure, org.biojava.bio.structure.Atom)">shift(Structure, Atom)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">shift a structure with a vector.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#shift(org.biojava.bio.structure.Atom, org.biojava.bio.structure.Atom)">shift(Atom, Atom)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">Shift a vector.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#shift(org.biojava.bio.structure.Group, org.biojava.bio.structure.Atom)">shift(Group, Atom)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">Shift a Group with a vector.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/MutableAlignedSequence.html#shiftAtAlignmentLocation(org.biojava3.core.sequence.location.template.Location, int)">shiftAtAlignmentLocation(Location, int)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/MutableAlignedSequence.html" title="interface in org.biojava3.alignment.template">MutableAlignedSequence</a></dt>
<dd>
<div class="block">Slides a part of the <a href="./org/biojava3/alignment/template/AlignedSequence.html" title="interface in org.biojava3.alignment.template"><code>AlignedSequence</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/MutableAlignedSequence.html#shiftAtSequenceLocation(org.biojava3.core.sequence.location.template.Location, int)">shiftAtSequenceLocation(Location, int)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/MutableAlignedSequence.html" title="interface in org.biojava3.alignment.template">MutableAlignedSequence</a></dt>
<dd>
<div class="block">Slides a part of the <a href="./org/biojava3/alignment/template/AlignedSequence.html" title="interface in org.biojava3.alignment.template"><code>AlignedSequence</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/StructureAlignmentDisplay.html#shiftCA2(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.jama.Matrix, org.biojava.bio.structure.Atom, org.biojava.bio.structure.Group[])">shiftCA2(AFPChain, Atom[], Matrix, Atom, Group[])</a></span> - Static method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/StructureAlignmentDisplay.html" title="class in org.biojava.bio.structure.align.gui">StructureAlignmentDisplay</a></dt>
<dd>
<div class="block">only shift CA positions.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/domain/pdp/ShortSegmentRemover.html" title="class in org.biojava.bio.structure.domain.pdp"><span class="strong">ShortSegmentRemover</span></a> - Class in <a href="./org/biojava/bio/structure/domain/pdp/package-summary.html">org.biojava.bio.structure.domain.pdp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/ShortSegmentRemover.html#ShortSegmentRemover()">ShortSegmentRemover()</a></span> - Constructor for class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/ShortSegmentRemover.html" title="class in org.biojava.bio.structure.domain.pdp">ShortSegmentRemover</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/DBResultTable.html#show(java.io.File, org.biojava.bio.structure.align.util.UserConfiguration)">show(File, UserConfiguration)</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/DBResultTable.html" title="class in org.biojava.bio.structure.align.gui">DBResultTable</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/MenuCreator.html#showAboutDialog()">showAboutDialog()</a></span> - Static method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/MenuCreator.html" title="class in org.biojava.bio.structure.align.gui">MenuCreator</a></dt>
<dd>
<div class="block">show some info about this gui</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/GuiWrapper.html#showAlignmentGUI()">showAlignmentGUI()</a></span> - Static method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/GuiWrapper.html" title="class in org.biojava.bio.structure.align.ce">GuiWrapper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/GuiWrapper.html#showAlignmentImage(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], java.lang.Object)">showAlignmentImage(AFPChain, Atom[], Atom[], Object)</a></span> - Static method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/GuiWrapper.html" title="class in org.biojava.bio.structure.align.ce">GuiWrapper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/DisplayAFP.html#showAlignmentImage(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], org.biojava.bio.structure.align.gui.jmol.StructureAlignmentJmol)">showAlignmentImage(AFPChain, Atom[], Atom[], StructureAlignmentJmol)</a></span> - Static method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/DisplayAFP.html" title="class in org.biojava.bio.structure.align.gui">DisplayAFP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/DisplayAFP.html#showAlignmentImage(org.biojava.bio.structure.align.model.AFPChain, java.lang.String)">showAlignmentImage(AFPChain, String)</a></span> - Static method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/DisplayAFP.html" title="class in org.biojava.bio.structure.align.gui">DisplayAFP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/AlternativeAlignmentFrame.html#showAlternative(int)">showAlternative(int)</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/AlternativeAlignmentFrame.html" title="class in org.biojava.bio.structure.gui.util">AlternativeAlignmentFrame</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#showCompound()">showCompound()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/MyJmolStatusListener.html#showConsole(boolean)">showConsole(boolean)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/MyJmolStatusListener.html" title="class in org.biojava.bio.structure.align.gui.jmol">MyJmolStatusListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/AboutDialog.html#showDialog()">showDialog()</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/AboutDialog.html" title="class in org.biojava.bio.structure.align.gui">AboutDialog</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/ConfigPDBInstallPanel.html#showDialog()">showDialog()</a></span> - Static method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/ConfigPDBInstallPanel.html" title="class in org.biojava.bio.structure.align.gui">ConfigPDBInstallPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/HelpDialog.html#showDialog()">showDialog()</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/HelpDialog.html" title="class in org.biojava.bio.structure.align.gui">HelpDialog</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/SystemInfo.html#showDialog()">showDialog()</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/SystemInfo.html" title="class in org.biojava.bio.structure.align.gui">SystemInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/AlternativeAlignmentFrame.html#showDistanceMatrix(int)">showDistanceMatrix(int)</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/AlternativeAlignmentFrame.html" title="class in org.biojava.bio.structure.gui.util">AlternativeAlignmentFrame</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/webstart/BrowserOpener.html#showDocument(java.net.URL)">showDocument(URL)</a></span> - Static method in class org.biojava.bio.structure.align.webstart.<a href="./org/biojava/bio/structure/align/webstart/BrowserOpener.html" title="class in org.biojava.bio.structure.align.webstart">BrowserOpener</a></dt>
<dd>
<div class="block">open a URL in the browser that was used to launch SPICE</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/webstart/BrowserOpener.html#showDocument(java.lang.String)">showDocument(String)</a></span> - Static method in class org.biojava.bio.structure.align.webstart.<a href="./org/biojava/bio/structure/align/webstart/BrowserOpener.html" title="class in org.biojava.bio.structure.align.webstart">BrowserOpener</a></dt>
<dd>
<div class="block">open a URL in the browser that was used to launch SPICE</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/webstart/JNLPProxy.html#showDocument(java.net.URL)">showDocument(URL)</a></span> - Static method in class org.biojava.bio.structure.align.webstart.<a href="./org/biojava/bio/structure/align/webstart/JNLPProxy.html" title="class in org.biojava.bio.structure.align.webstart">JNLPProxy</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#showHeader()">showHeader()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>
<div class="block">Print some debug statements to System.out</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/gui/ShowPDBIDListener.html" title="class in org.biojava.bio.structure.align.gui"><span class="strong">ShowPDBIDListener</span></a> - Class in <a href="./org/biojava/bio/structure/align/gui/package-summary.html">org.biojava.bio.structure.align.gui</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/ShowPDBIDListener.html#ShowPDBIDListener()">ShowPDBIDListener()</a></span> - Constructor for class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/ShowPDBIDListener.html" title="class in org.biojava.bio.structure.align.gui">ShowPDBIDListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#showSource()">showSource()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./demo/ShowStructureInJmol.html" title="class in demo"><span class="strong">ShowStructureInJmol</span></a> - Class in <a href="./demo/package-summary.html">demo</a></dt>
<dd>
<div class="block">Demo how to load and display a structure in Jmol</div>
</dd>
<dt><span class="strong"><a href="./demo/ShowStructureInJmol.html#ShowStructureInJmol()">ShowStructureInJmol()</a></span> - Constructor for class demo.<a href="./demo/ShowStructureInJmol.html" title="class in demo">ShowStructureInJmol</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/MenuCreator.html#showSystemInfo()">showSystemInfo()</a></span> - Static method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/MenuCreator.html" title="class in org.biojava.bio.structure.align.gui">MenuCreator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/MyJmolStatusListener.html#showUrl(java.lang.String)">showUrl(String)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/MyJmolStatusListener.html" title="class in org.biojava.bio.structure.align.gui.jmol">MyJmolStatusListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceMixin.html#shuffle(org.biojava3.core.sequence.template.Sequence)">shuffle(Sequence&lt;C&gt;)</a></span> - Static method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceMixin.html" title="class in org.biojava3.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">Implements sequence shuffling by first materializing the given
 <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a> into a <a href="http://java.sun.com/j2se/1.6.0/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util"><code>List</code></a>, applying
 <a href="http://java.sun.com/j2se/1.6.0/docs/api/java/util/Collections.html?is-external=true#shuffle(java.util.List)" title="class or interface in java.util"><code>Collections.shuffle(List)</code></a> and then returning the shuffled
 elements in a new instance of <code>SequenceBackingStore</code> which behaves
 as a <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/ConcurrencyTools.html#shutdown()">shutdown()</a></span> - Static method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/ConcurrencyTools.html" title="class in org.biojava3.core.util">ConcurrencyTools</a></dt>
<dd>
<div class="block">Disables new tasks from being submitted and closes the thread pool cleanly.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/ConcurrencyTools.html#shutdownAndAwaitTermination()">shutdownAndAwaitTermination()</a></span> - Static method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/ConcurrencyTools.html" title="class in org.biojava3.core.util">ConcurrencyTools</a></dt>
<dd>
<div class="block">Closes the thread pool.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#SIDE_CHAIN_ANGLE_SCORING">SIDE_CHAIN_ANGLE_SCORING</a></span> - Static variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#SIDE_CHAIN_SCORING">SIDE_CHAIN_SCORING</a></span> - Static variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/fatcat/calc/SigEva.html" title="class in org.biojava.bio.structure.align.fatcat.calc"><span class="strong">SigEva</span></a> - Class in <a href="./org/biojava/bio/structure/align/fatcat/calc/package-summary.html">org.biojava.bio.structure.align.fatcat.calc</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/SigEva.html#SigEva()">SigEva()</a></span> - Constructor for class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/SigEva.html" title="class in org.biojava.bio.structure.align.fatcat.calc">SigEva</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/MenuCreator.html#SIMILARITY_COLOR">SIMILARITY_COLOR</a></span> - Static variable in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/MenuCreator.html" title="class in org.biojava.bio.structure.align.gui">MenuCreator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/alignment/SimpleAlignedSequence.html" title="class in org.biojava3.alignment"><span class="strong">SimpleAlignedSequence</span></a>&lt;<a href="./org/biojava3/alignment/SimpleAlignedSequence.html" title="type parameter in SimpleAlignedSequence">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava3/alignment/SimpleAlignedSequence.html" title="type parameter in SimpleAlignedSequence">C</a>&gt;,<a href="./org/biojava3/alignment/SimpleAlignedSequence.html" title="type parameter in SimpleAlignedSequence">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/alignment/package-summary.html">org.biojava3.alignment</a></dt>
<dd>
<div class="block">Implements a data structure for a <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a> within an alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleAlignedSequence.html#SimpleAlignedSequence(S, java.util.List)">SimpleAlignedSequence(S, List&lt;AlignedSequence.Step&gt;)</a></span> - Constructor for class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleAlignedSequence.html" title="class in org.biojava3.alignment">SimpleAlignedSequence</a></dt>
<dd>
<div class="block">Creates an <a href="./org/biojava3/alignment/template/AlignedSequence.html" title="interface in org.biojava3.alignment.template"><code>AlignedSequence</code></a> for the given <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a> in a global alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleAlignedSequence.html#SimpleAlignedSequence(S, java.util.List, int, int)">SimpleAlignedSequence(S, List&lt;AlignedSequence.Step&gt;, int, int)</a></span> - Constructor for class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleAlignedSequence.html" title="class in org.biojava3.alignment">SimpleAlignedSequence</a></dt>
<dd>
<div class="block">Creates an <a href="./org/biojava3/alignment/template/AlignedSequence.html" title="interface in org.biojava3.alignment.template"><code>AlignedSequence</code></a> for the given <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a> in a local alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleAlignedSequence.html#SimpleAlignedSequence(org.biojava3.alignment.template.AlignedSequence, java.util.List)">SimpleAlignedSequence(AlignedSequence&lt;S, C&gt;, List&lt;AlignedSequence.Step&gt;)</a></span> - Constructor for class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleAlignedSequence.html" title="class in org.biojava3.alignment">SimpleAlignedSequence</a></dt>
<dd>
<div class="block">Creates a new <a href="./org/biojava3/alignment/template/AlignedSequence.html" title="interface in org.biojava3.alignment.template"><code>AlignedSequence</code></a> for the given <a href="./org/biojava3/alignment/template/AlignedSequence.html" title="interface in org.biojava3.alignment.template"><code>AlignedSequence</code></a> in a global alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleAlignedSequence.html#SimpleAlignedSequence(org.biojava3.alignment.template.AlignedSequence, java.util.List, int, int)">SimpleAlignedSequence(AlignedSequence&lt;S, C&gt;, List&lt;AlignedSequence.Step&gt;, int, int)</a></span> - Constructor for class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleAlignedSequence.html" title="class in org.biojava3.alignment">SimpleAlignedSequence</a></dt>
<dd>
<div class="block">Creates a new <a href="./org/biojava3/alignment/template/AlignedSequence.html" title="interface in org.biojava3.alignment.template"><code>AlignedSequence</code></a> for the given <a href="./org/biojava3/alignment/template/AlignedSequence.html" title="interface in org.biojava3.alignment.template"><code>AlignedSequence</code></a> in a local alignment.</div>
</dd>
<dt><a href="./org/biojava3/alignment/SimpleGapPenalty.html" title="class in org.biojava3.alignment"><span class="strong">SimpleGapPenalty</span></a> - Class in <a href="./org/biojava3/alignment/package-summary.html">org.biojava3.alignment</a></dt>
<dd>
<div class="block">Implements a data structure for the gap penalties used during a sequence alignment routine.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleGapPenalty.html#SimpleGapPenalty()">SimpleGapPenalty()</a></span> - Constructor for class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleGapPenalty.html" title="class in org.biojava3.alignment">SimpleGapPenalty</a></dt>
<dd>
<div class="block">Creates a new set of gap penalties using the defaults.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleGapPenalty.html#SimpleGapPenalty(short, short)">SimpleGapPenalty(short, short)</a></span> - Constructor for class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleGapPenalty.html" title="class in org.biojava3.alignment">SimpleGapPenalty</a></dt>
<dd>
<div class="block">Creates a new set of gap penalties.</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/location/SimpleLocation.html" title="class in org.biojava3.core.sequence.location"><span class="strong">SimpleLocation</span></a> - Class in <a href="./org/biojava3/core/sequence/location/package-summary.html">org.biojava3.core.sequence.location</a></dt>
<dd>
<div class="block">Very basic implementation of the Location interface which defines a series
 of simple constructors.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/SimpleLocation.html#SimpleLocation(int, int)">SimpleLocation(int, int)</a></span> - Constructor for class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/SimpleLocation.html" title="class in org.biojava3.core.sequence.location">SimpleLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/SimpleLocation.html#SimpleLocation(org.biojava3.core.sequence.location.template.Point, org.biojava3.core.sequence.location.template.Point)">SimpleLocation(Point, Point)</a></span> - Constructor for class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/SimpleLocation.html" title="class in org.biojava3.core.sequence.location">SimpleLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/SimpleLocation.html#SimpleLocation(int, int, org.biojava3.core.sequence.Strand)">SimpleLocation(int, int, Strand)</a></span> - Constructor for class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/SimpleLocation.html" title="class in org.biojava3.core.sequence.location">SimpleLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/SimpleLocation.html#SimpleLocation(org.biojava3.core.sequence.location.template.Point, org.biojava3.core.sequence.location.template.Point, org.biojava3.core.sequence.Strand)">SimpleLocation(Point, Point, Strand)</a></span> - Constructor for class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/SimpleLocation.html" title="class in org.biojava3.core.sequence.location">SimpleLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/SimpleLocation.html#SimpleLocation(org.biojava3.core.sequence.location.template.Point, org.biojava3.core.sequence.location.template.Point, org.biojava3.core.sequence.Strand, org.biojava3.core.sequence.AccessionID)">SimpleLocation(Point, Point, Strand, AccessionID)</a></span> - Constructor for class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/SimpleLocation.html" title="class in org.biojava3.core.sequence.location">SimpleLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/SimpleLocation.html#SimpleLocation(org.biojava3.core.sequence.location.template.Point, org.biojava3.core.sequence.location.template.Point, org.biojava3.core.sequence.Strand, boolean, org.biojava3.core.sequence.AccessionID)">SimpleLocation(Point, Point, Strand, boolean, AccessionID)</a></span> - Constructor for class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/SimpleLocation.html" title="class in org.biojava3.core.sequence.location">SimpleLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/SimpleLocation.html#SimpleLocation(org.biojava3.core.sequence.location.template.Point, org.biojava3.core.sequence.location.template.Point, org.biojava3.core.sequence.Strand, boolean, boolean)">SimpleLocation(Point, Point, Strand, boolean, boolean)</a></span> - Constructor for class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/SimpleLocation.html" title="class in org.biojava3.core.sequence.location">SimpleLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/SimpleLocation.html#SimpleLocation(int, int, org.biojava3.core.sequence.Strand, org.biojava3.core.sequence.location.template.Location...)">SimpleLocation(int, int, Strand, Location...)</a></span> - Constructor for class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/SimpleLocation.html" title="class in org.biojava3.core.sequence.location">SimpleLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/SimpleLocation.html#SimpleLocation(org.biojava3.core.sequence.location.template.Point, org.biojava3.core.sequence.location.template.Point, org.biojava3.core.sequence.Strand, org.biojava3.core.sequence.location.template.Location...)">SimpleLocation(Point, Point, Strand, Location...)</a></span> - Constructor for class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/SimpleLocation.html" title="class in org.biojava3.core.sequence.location">SimpleLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/SimpleLocation.html#SimpleLocation(org.biojava3.core.sequence.location.template.Point, org.biojava3.core.sequence.location.template.Point, org.biojava3.core.sequence.Strand, boolean, org.biojava3.core.sequence.location.template.Location...)">SimpleLocation(Point, Point, Strand, boolean, Location...)</a></span> - Constructor for class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/SimpleLocation.html" title="class in org.biojava3.core.sequence.location">SimpleLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/SimpleLocation.html#SimpleLocation(org.biojava3.core.sequence.location.template.Point, org.biojava3.core.sequence.location.template.Point, org.biojava3.core.sequence.Strand, boolean, java.util.List)">SimpleLocation(Point, Point, Strand, boolean, List&lt;Location&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/SimpleLocation.html" title="class in org.biojava3.core.sequence.location">SimpleLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/SimpleLocation.html#SimpleLocation(org.biojava3.core.sequence.location.template.Point, org.biojava3.core.sequence.location.template.Point, org.biojava3.core.sequence.Strand, boolean, boolean, java.util.List)">SimpleLocation(Point, Point, Strand, boolean, boolean, List&lt;Location&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/SimpleLocation.html" title="class in org.biojava3.core.sequence.location">SimpleLocation</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif"><span class="strong">SimpleMMcifConsumer</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/package-summary.html">org.biojava.bio.structure.io.mmcif</a></dt>
<dd>
<div class="block">A MMcifConsumer implementation that build a in-memory representation of the
 content of a mmcif file as a BioJava Structure object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html#SimpleMMcifConsumer()">SimpleMMcifConsumer()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifConsumer.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifConsumer</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif"><span class="strong">SimpleMMcifParser</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/package-summary.html">org.biojava.bio.structure.io.mmcif</a></dt>
<dd>
<div class="block">A simple mmCif file parser</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#SimpleMMcifParser()">SimpleMMcifParser()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/location/SimplePoint.html" title="class in org.biojava3.core.sequence.location"><span class="strong">SimplePoint</span></a> - Class in <a href="./org/biojava3/core/sequence/location/package-summary.html">org.biojava3.core.sequence.location</a></dt>
<dd>
<div class="block">Basic implementation of the Point interface.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/SimplePoint.html#SimplePoint()">SimplePoint()</a></span> - Constructor for class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/SimplePoint.html" title="class in org.biojava3.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/SimplePoint.html#SimplePoint(int)">SimplePoint(int)</a></span> - Constructor for class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/SimplePoint.html" title="class in org.biojava3.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/SimplePoint.html#SimplePoint(int, boolean, boolean)">SimplePoint(int, boolean, boolean)</a></span> - Constructor for class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/SimplePoint.html" title="class in org.biojava3.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/alignment/SimpleProfile.html" title="class in org.biojava3.alignment"><span class="strong">SimpleProfile</span></a>&lt;<a href="./org/biojava3/alignment/SimpleProfile.html" title="type parameter in SimpleProfile">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava3/alignment/SimpleProfile.html" title="type parameter in SimpleProfile">C</a>&gt;,<a href="./org/biojava3/alignment/SimpleProfile.html" title="type parameter in SimpleProfile">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/alignment/package-summary.html">org.biojava3.alignment</a></dt>
<dd>
<div class="block">Implements a data structure for the results of sequence alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleProfile.html#SimpleProfile(org.biojava3.alignment.template.AlignedSequence, org.biojava3.alignment.template.AlignedSequence)">SimpleProfile(AlignedSequence&lt;S, C&gt;, AlignedSequence&lt;S, C&gt;)</a></span> - Constructor for class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleProfile.html" title="class in org.biojava3.alignment">SimpleProfile</a></dt>
<dd>
<div class="block">Creates a pair profile for the given already aligned sequences.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleProfile.html#SimpleProfile(S)">SimpleProfile(S)</a></span> - Constructor for class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleProfile.html" title="class in org.biojava3.alignment">SimpleProfile</a></dt>
<dd>
<div class="block">Creates a profile from a single sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleProfile.html#SimpleProfile(S, S, java.util.List, int, int, java.util.List, int, int)">SimpleProfile(S, S, List&lt;AlignedSequence.Step&gt;, int, int, List&lt;AlignedSequence.Step&gt;, int, int)</a></span> - Constructor for class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleProfile.html" title="class in org.biojava3.alignment">SimpleProfile</a></dt>
<dd>
<div class="block">Creates a pair profile for the given sequences.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleProfile.html#SimpleProfile(org.biojava3.alignment.template.Profile, org.biojava3.alignment.template.Profile, java.util.List, java.util.List)">SimpleProfile(Profile&lt;S, C&gt;, Profile&lt;S, C&gt;, List&lt;AlignedSequence.Step&gt;, List&lt;AlignedSequence.Step&gt;)</a></span> - Constructor for class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleProfile.html" title="class in org.biojava3.alignment">SimpleProfile</a></dt>
<dd>
<div class="block">Creates a pair profile for the given profiles.</div>
</dd>
<dt><a href="./org/biojava3/alignment/SimpleProfilePair.html" title="class in org.biojava3.alignment"><span class="strong">SimpleProfilePair</span></a>&lt;<a href="./org/biojava3/alignment/SimpleProfilePair.html" title="type parameter in SimpleProfilePair">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava3/alignment/SimpleProfilePair.html" title="type parameter in SimpleProfilePair">C</a>&gt;,<a href="./org/biojava3/alignment/SimpleProfilePair.html" title="type parameter in SimpleProfilePair">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/alignment/package-summary.html">org.biojava3.alignment</a></dt>
<dd>
<div class="block">Implements a data structure for the results of the alignment of a pair of <a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template"><code>Profile</code></a>s.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleProfilePair.html#SimpleProfilePair(org.biojava3.alignment.template.Profile, org.biojava3.alignment.template.Profile, java.util.List, java.util.List)">SimpleProfilePair(Profile&lt;S, C&gt;, Profile&lt;S, C&gt;, List&lt;AlignedSequence.Step&gt;, List&lt;AlignedSequence.Step&gt;)</a></span> - Constructor for class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleProfilePair.html" title="class in org.biojava3.alignment">SimpleProfilePair</a></dt>
<dd>
<div class="block">Creates a pair profile for the given profiles.</div>
</dd>
<dt><a href="./org/biojava3/alignment/SimpleProfileProfileAligner.html" title="class in org.biojava3.alignment"><span class="strong">SimpleProfileProfileAligner</span></a>&lt;<a href="./org/biojava3/alignment/SimpleProfileProfileAligner.html" title="type parameter in SimpleProfileProfileAligner">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava3/alignment/SimpleProfileProfileAligner.html" title="type parameter in SimpleProfileProfileAligner">C</a>&gt;,<a href="./org/biojava3/alignment/SimpleProfileProfileAligner.html" title="type parameter in SimpleProfileProfileAligner">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/alignment/package-summary.html">org.biojava3.alignment</a></dt>
<dd>
<div class="block">Implements a simple (naive) <a href="./org/biojava3/alignment/template/Aligner.html" title="interface in org.biojava3.alignment.template"><code>Aligner</code></a> for a pair of <a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template"><code>Profile</code></a>s.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleProfileProfileAligner.html#SimpleProfileProfileAligner()">SimpleProfileProfileAligner()</a></span> - Constructor for class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleProfileProfileAligner.html" title="class in org.biojava3.alignment">SimpleProfileProfileAligner</a></dt>
<dd>
<div class="block">Before running a profile-profile alignment, data must be sent in via calls to
 <a href="./org/biojava3/alignment/template/AbstractProfileProfileAligner.html#setQuery(org.biojava3.alignment.template.Profile)"><code>AbstractProfileProfileAligner.setQuery(Profile)</code></a>, <a href="./org/biojava3/alignment/template/AbstractProfileProfileAligner.html#setTarget(org.biojava3.alignment.template.Profile)"><code>AbstractProfileProfileAligner.setTarget(Profile)</code></a>, <a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#setGapPenalty(org.biojava3.alignment.template.GapPenalty)"><code>AbstractMatrixAligner.setGapPenalty(GapPenalty)</code></a>, and
 <a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#setSubstitutionMatrix(org.biojava3.alignment.template.SubstitutionMatrix)"><code>AbstractMatrixAligner.setSubstitutionMatrix(SubstitutionMatrix)</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleProfileProfileAligner.html#SimpleProfileProfileAligner(org.biojava3.alignment.template.Profile, org.biojava3.alignment.template.Profile, org.biojava3.alignment.template.GapPenalty, org.biojava3.alignment.template.SubstitutionMatrix)">SimpleProfileProfileAligner(Profile&lt;S, C&gt;, Profile&lt;S, C&gt;, GapPenalty, SubstitutionMatrix&lt;C&gt;)</a></span> - Constructor for class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleProfileProfileAligner.html" title="class in org.biojava3.alignment">SimpleProfileProfileAligner</a></dt>
<dd>
<div class="block">Prepares for a profile-profile alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleProfileProfileAligner.html#SimpleProfileProfileAligner(java.util.concurrent.Future, java.util.concurrent.Future, org.biojava3.alignment.template.GapPenalty, org.biojava3.alignment.template.SubstitutionMatrix)">SimpleProfileProfileAligner(Future&lt;ProfilePair&lt;S, C&gt;&gt;, Future&lt;ProfilePair&lt;S, C&gt;&gt;, GapPenalty, SubstitutionMatrix&lt;C&gt;)</a></span> - Constructor for class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleProfileProfileAligner.html" title="class in org.biojava3.alignment">SimpleProfileProfileAligner</a></dt>
<dd>
<div class="block">Prepares for a profile-profile alignment run concurrently.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleProfileProfileAligner.html#SimpleProfileProfileAligner(org.biojava3.alignment.template.Profile, java.util.concurrent.Future, org.biojava3.alignment.template.GapPenalty, org.biojava3.alignment.template.SubstitutionMatrix)">SimpleProfileProfileAligner(Profile&lt;S, C&gt;, Future&lt;ProfilePair&lt;S, C&gt;&gt;, GapPenalty, SubstitutionMatrix&lt;C&gt;)</a></span> - Constructor for class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleProfileProfileAligner.html" title="class in org.biojava3.alignment">SimpleProfileProfileAligner</a></dt>
<dd>
<div class="block">Prepares for a profile-profile alignment run concurrently.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleProfileProfileAligner.html#SimpleProfileProfileAligner(java.util.concurrent.Future, org.biojava3.alignment.template.Profile, org.biojava3.alignment.template.GapPenalty, org.biojava3.alignment.template.SubstitutionMatrix)">SimpleProfileProfileAligner(Future&lt;ProfilePair&lt;S, C&gt;&gt;, Profile&lt;S, C&gt;, GapPenalty, SubstitutionMatrix&lt;C&gt;)</a></span> - Constructor for class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleProfileProfileAligner.html" title="class in org.biojava3.alignment">SimpleProfileProfileAligner</a></dt>
<dd>
<div class="block">Prepares for a profile-profile alignment run concurrently.</div>
</dd>
<dt><a href="./org/biojava3/alignment/SimpleSequencePair.html" title="class in org.biojava3.alignment"><span class="strong">SimpleSequencePair</span></a>&lt;<a href="./org/biojava3/alignment/SimpleSequencePair.html" title="type parameter in SimpleSequencePair">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava3/alignment/SimpleSequencePair.html" title="type parameter in SimpleSequencePair">C</a>&gt;,<a href="./org/biojava3/alignment/SimpleSequencePair.html" title="type parameter in SimpleSequencePair">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/alignment/package-summary.html">org.biojava3.alignment</a></dt>
<dd>
<div class="block">Implements a data structure for the results of pairwise sequence alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleSequencePair.html#SimpleSequencePair(org.biojava3.alignment.template.AlignedSequence, org.biojava3.alignment.template.AlignedSequence)">SimpleSequencePair(AlignedSequence&lt;S, C&gt;, AlignedSequence&lt;S, C&gt;)</a></span> - Constructor for class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleSequencePair.html" title="class in org.biojava3.alignment">SimpleSequencePair</a></dt>
<dd>
<div class="block">Creates a pair profile for the given already aligned sequences.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleSequencePair.html#SimpleSequencePair(S, S, java.util.List, java.util.List)">SimpleSequencePair(S, S, List&lt;AlignedSequence.Step&gt;, List&lt;AlignedSequence.Step&gt;)</a></span> - Constructor for class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleSequencePair.html" title="class in org.biojava3.alignment">SimpleSequencePair</a></dt>
<dd>
<div class="block">Creates a pair profile for the given sequences with a global alignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleSequencePair.html#SimpleSequencePair(S, S, java.util.List, int, int, java.util.List, int, int)">SimpleSequencePair(S, S, List&lt;AlignedSequence.Step&gt;, int, int, List&lt;AlignedSequence.Step&gt;, int, int)</a></span> - Constructor for class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleSequencePair.html" title="class in org.biojava3.alignment">SimpleSequencePair</a></dt>
<dd>
<div class="block">Creates a pair profile for the given sequences with a local alignment.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/server/SimpleStructureServer.html" title="class in org.biojava.bio.structure.server"><span class="strong">SimpleStructureServer</span></a> - Class in <a href="./org/biojava/bio/structure/server/package-summary.html">org.biojava.bio.structure.server</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>&nbsp;</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/SimpleStructureServer.html#SimpleStructureServer()">SimpleStructureServer()</a></span> - Constructor for class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/SimpleStructureServer.html" title="class in org.biojava.bio.structure.server">SimpleStructureServer</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><a href="./org/biojava3/alignment/SimpleSubstitutionMatrix.html" title="class in org.biojava3.alignment"><span class="strong">SimpleSubstitutionMatrix</span></a>&lt;<a href="./org/biojava3/alignment/SimpleSubstitutionMatrix.html" title="type parameter in SimpleSubstitutionMatrix">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/alignment/package-summary.html">org.biojava3.alignment</a></dt>
<dd>
<div class="block">Implements a data structure which holds the score (penalty or bonus) given during alignment for the exchange of one
 <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template"><code>Compound</code></a> in a sequence for another.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleSubstitutionMatrix.html#SimpleSubstitutionMatrix()">SimpleSubstitutionMatrix()</a></span> - Constructor for class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleSubstitutionMatrix.html" title="class in org.biojava3.alignment">SimpleSubstitutionMatrix</a></dt>
<dd>
<div class="block">Creates a substitution matrix using the defaults.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleSubstitutionMatrix.html#SimpleSubstitutionMatrix(org.biojava3.core.sequence.template.CompoundSet, java.io.File)">SimpleSubstitutionMatrix(CompoundSet&lt;C&gt;, File)</a></span> - Constructor for class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleSubstitutionMatrix.html" title="class in org.biojava3.alignment">SimpleSubstitutionMatrix</a></dt>
<dd>
<div class="block">Creates a substitution matrix by reading in a file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleSubstitutionMatrix.html#SimpleSubstitutionMatrix(org.biojava3.core.sequence.template.CompoundSet, java.io.Reader, java.lang.String)">SimpleSubstitutionMatrix(CompoundSet&lt;C&gt;, Reader, String)</a></span> - Constructor for class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleSubstitutionMatrix.html" title="class in org.biojava3.alignment">SimpleSubstitutionMatrix</a></dt>
<dd>
<div class="block">Creates a substitution matrix by parsing some input.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleSubstitutionMatrix.html#SimpleSubstitutionMatrix(org.biojava3.core.sequence.template.CompoundSet, java.lang.String, java.lang.String)">SimpleSubstitutionMatrix(CompoundSet&lt;C&gt;, String, String)</a></span> - Constructor for class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleSubstitutionMatrix.html" title="class in org.biojava3.alignment">SimpleSubstitutionMatrix</a></dt>
<dd>
<div class="block">Creates a substitution matrix by parsing a String.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleSubstitutionMatrix.html#SimpleSubstitutionMatrix(org.biojava3.core.sequence.template.CompoundSet, short, short)">SimpleSubstitutionMatrix(CompoundSet&lt;C&gt;, short, short)</a></span> - Constructor for class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleSubstitutionMatrix.html" title="class in org.biojava3.alignment">SimpleSubstitutionMatrix</a></dt>
<dd>
<div class="block">Creates an identity substitution matrix from match and replace values.</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava3.core.sequence.storage"><span class="strong">SingleCompoundSequenceReader</span></a>&lt;<a href="./org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html" title="type parameter in SingleCompoundSequenceReader">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/core/sequence/storage/package-summary.html">org.biojava3.core.sequence.storage</a></dt>
<dd>
<div class="block">An implementation of the SequenceReader interface which for every
 call will return only 1 compound (given to it during construction; a String
 is also valid but will require a CompoundSet).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html#SingleCompoundSequenceReader(java.lang.String, org.biojava3.core.sequence.template.CompoundSet, int)">SingleCompoundSequenceReader(String, CompoundSet&lt;C&gt;, int)</a></span> - Constructor for class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava3.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Public constructor to be used with String based constructor</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html#SingleCompoundSequenceReader(C, org.biojava3.core.sequence.template.CompoundSet, int)">SingleCompoundSequenceReader(C, CompoundSet&lt;C&gt;, int)</a></span> - Constructor for class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/SingleCompoundSequenceReader.html" title="class in org.biojava3.core.sequence.storage">SingleCompoundSequenceReader</a></dt>
<dd>
<div class="block">Build the object with a compound rather than a String</div>
</dd>
<dt><a href="./org/biojava/bio/structure/jama/SingularValueDecomposition.html" title="class in org.biojava.bio.structure.jama"><span class="strong">SingularValueDecomposition</span></a> - Class in <a href="./org/biojava/bio/structure/jama/package-summary.html">org.biojava.bio.structure.jama</a></dt>
<dd>
<div class="block">Singular Value Decomposition.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/SingularValueDecomposition.html#SingularValueDecomposition(org.biojava.bio.structure.jama.Matrix)">SingularValueDecomposition(Matrix)</a></span> - Constructor for class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/SingularValueDecomposition.html" title="class in org.biojava.bio.structure.jama">SingularValueDecomposition</a></dt>
<dd>
<div class="block">Construct the singular value decomposition.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/Site.html" title="class in org.biojava.bio.structure"><span class="strong">Site</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">Holds the data of sites presented in PDB files.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Site.html#Site()">Site()</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Site.html" title="class in org.biojava.bio.structure">Site</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Site.html#Site(java.lang.String, java.util.List)">Site(String, List&lt;Group&gt;)</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Site.html" title="class in org.biojava.bio.structure">Site</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/CutValues.html#site2">site2</a></span> - Variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/CutValues.html" title="class in org.biojava.bio.structure.domain.pdp">CutValues</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Group.html#size()">size()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Group.html" title="interface in org.biojava.bio.structure">Group</a></dt>
<dd>
<div class="block">getnumber of atoms.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html#size()">size()</a></span> - Method in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html" title="class in org.biojava.bio.structure.gui.util.color">GradientMapper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html#SIZE">SIZE</a></span> - Static variable in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html" title="class in org.biojava.bio.structure.gui.util">SequenceScalePanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#size()">size()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">getnumber of atoms.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#size()">size()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">return number of Chains in this Structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#size(int)">size(int)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">return number of chains of model.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#size()">size()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">return number of chains , if NMR return number of chains of first model .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#size(int)">size(int)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">return number of chains  of model.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/FlatFileCache.html#size()">size()</a></span> - Method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/FlatFileCache.html" title="class in org.biojava3.core.util">FlatFileCache</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/SoftHashMap.html#size()">size()</a></span> - Method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/SoftHashMap.html" title="class in org.biojava3.core.util">SoftHashMap</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#skalarProduct(org.biojava.bio.structure.Atom, org.biojava.bio.structure.Atom)">skalarProduct(Atom, Atom)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">skalar product.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/BufferedReaderBytesRead.html#skip(long)">skip(long)</a></span> - Method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/BufferedReaderBytesRead.html" title="class in org.biojava3.core.sequence.io">BufferedReaderBytesRead</a></dt>
<dd>
<div class="block">Skips characters.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/UncompressInputStream.html#skip(long)">skip(long)</a></span> - Method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/UncompressInputStream.html" title="class in org.biojava3.core.util">UncompressInputStream</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html#slider">slider</a></span> - Variable in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html" title="class in org.biojava.bio.structure.gui">ScaleableMatrixPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html#SLIDER_STEPS">SLIDER_STEPS</a></span> - Static variable in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html" title="class in org.biojava.bio.structure.gui">ScaleableMatrixPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#small">small</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/BiojavaJmol.html#smartAdapter">smartAdapter</a></span> - Static variable in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/BiojavaJmol.html" title="class in org.biojava.bio.structure.gui">BiojavaJmol</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/structure/gui/JmolViewerImpl.html#smartAdapter">smartAdapter</a></span> - Static variable in class org.biojava3.structure.gui.<a href="./org/biojava3/structure/gui/JmolViewerImpl.html" title="class in org.biojava3.structure.gui">JmolViewerImpl</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/alignment/SmithWaterman.html" title="class in org.biojava3.alignment"><span class="strong">SmithWaterman</span></a>&lt;<a href="./org/biojava3/alignment/SmithWaterman.html" title="type parameter in SmithWaterman">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava3/alignment/SmithWaterman.html" title="type parameter in SmithWaterman">C</a>&gt;,<a href="./org/biojava3/alignment/SmithWaterman.html" title="type parameter in SmithWaterman">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/alignment/package-summary.html">org.biojava3.alignment</a></dt>
<dd>
<div class="block">Smith and Waterman defined an algorithm for pairwise local sequence alignments (best match of sections from each
 <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a>).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SmithWaterman.html#SmithWaterman()">SmithWaterman()</a></span> - Constructor for class org.biojava3.alignment.<a href="./org/biojava3/alignment/SmithWaterman.html" title="class in org.biojava3.alignment">SmithWaterman</a></dt>
<dd>
<div class="block">Before running a pairwise local sequence alignment, data must be sent in via calls to
 <a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html#setQuery(S)"><code>AbstractPairwiseSequenceAligner.setQuery(Sequence)</code></a>, <a href="./org/biojava3/alignment/template/AbstractPairwiseSequenceAligner.html#setTarget(S)"><code>AbstractPairwiseSequenceAligner.setTarget(Sequence)</code></a>, <a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#setGapPenalty(org.biojava3.alignment.template.GapPenalty)"><code>AbstractMatrixAligner.setGapPenalty(GapPenalty)</code></a>, and
 <a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#setSubstitutionMatrix(org.biojava3.alignment.template.SubstitutionMatrix)"><code>AbstractMatrixAligner.setSubstitutionMatrix(SubstitutionMatrix)</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SmithWaterman.html#SmithWaterman(S, S, org.biojava3.alignment.template.GapPenalty, org.biojava3.alignment.template.SubstitutionMatrix)">SmithWaterman(S, S, GapPenalty, SubstitutionMatrix&lt;C&gt;)</a></span> - Constructor for class org.biojava3.alignment.<a href="./org/biojava3/alignment/SmithWaterman.html" title="class in org.biojava3.alignment">SmithWaterman</a></dt>
<dd>
<div class="block">Prepares for a pairwise local sequence alignment.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/seq/SmithWaterman3Daligner.html" title="class in org.biojava.bio.structure.align.seq"><span class="strong">SmithWaterman3Daligner</span></a> - Class in <a href="./org/biojava/bio/structure/align/seq/package-summary.html">org.biojava.bio.structure.align.seq</a></dt>
<dd>
<div class="block">provides a 3D superimposition based on the sequence alignment</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWaterman3Daligner.html#SmithWaterman3Daligner()">SmithWaterman3Daligner()</a></span> - Constructor for class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWaterman3Daligner.html" title="class in org.biojava.bio.structure.align.seq">SmithWaterman3Daligner</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/seq/SmithWaterman3DParameters.html" title="class in org.biojava.bio.structure.align.seq"><span class="strong">SmithWaterman3DParameters</span></a> - Class in <a href="./org/biojava/bio/structure/align/seq/package-summary.html">org.biojava.bio.structure.align.seq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWaterman3DParameters.html#SmithWaterman3DParameters()">SmithWaterman3DParameters()</a></span> - Constructor for class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWaterman3DParameters.html" title="class in org.biojava.bio.structure.align.seq">SmithWaterman3DParameters</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/seq/SmithWatermanUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.seq"><span class="strong">SmithWatermanUserArgumentProcessor</span></a> - Class in <a href="./org/biojava/bio/structure/align/seq/package-summary.html">org.biojava.bio.structure.align.seq</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/seq/SmithWatermanUserArgumentProcessor.html#SmithWatermanUserArgumentProcessor()">SmithWatermanUserArgumentProcessor()</a></span> - Constructor for class org.biojava.bio.structure.align.seq.<a href="./org/biojava/bio/structure/align/seq/SmithWatermanUserArgumentProcessor.html" title="class in org.biojava.bio.structure.align.seq">SmithWatermanUserArgumentProcessor</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/util/SoftHashMap.html" title="class in org.biojava3.core.util"><span class="strong">SoftHashMap</span></a>&lt;<a href="./org/biojava3/core/util/SoftHashMap.html" title="type parameter in SoftHashMap">K</a>,<a href="./org/biojava3/core/util/SoftHashMap.html" title="type parameter in SoftHashMap">V</a>&gt; - Class in <a href="./org/biojava3/core/util/package-summary.html">org.biojava3.core.util</a></dt>
<dd>
<div class="block">A in memory cache using soft references.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/SoftHashMap.html#SoftHashMap()">SoftHashMap()</a></span> - Constructor for class org.biojava3.core.util.<a href="./org/biojava3/core/util/SoftHashMap.html" title="class in org.biojava3.core.util">SoftHashMap</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/SoftHashMap.html#SoftHashMap(int)">SoftHashMap(int)</a></span> - Constructor for class org.biojava3.core.util.<a href="./org/biojava3/core/util/SoftHashMap.html" title="class in org.biojava3.core.util">SoftHashMap</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/CholeskyDecomposition.html#solve(org.biojava.bio.structure.jama.Matrix)">solve(Matrix)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/CholeskyDecomposition.html" title="class in org.biojava.bio.structure.jama">CholeskyDecomposition</a></dt>
<dd>
<div class="block">Solve A*X = B</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/LUDecomposition.html#solve(org.biojava.bio.structure.jama.Matrix)">solve(Matrix)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/LUDecomposition.html" title="class in org.biojava.bio.structure.jama">LUDecomposition</a></dt>
<dd>
<div class="block">Solve A*X = B</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#solve(org.biojava.bio.structure.jama.Matrix)">solve(Matrix)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Solve A*X = B</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/QRDecomposition.html#solve(org.biojava.bio.structure.jama.Matrix)">solve(Matrix)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/QRDecomposition.html" title="class in org.biojava.bio.structure.jama">QRDecomposition</a></dt>
<dd>
<div class="block">Least squares solution of A*X = B</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#solveTranspose(org.biojava.bio.structure.jama.Matrix)">solveTranspose(Matrix)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Solve X*A = B, which is also A'*X' = B'</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/AFPCalculator.html#sortAfps(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">sortAfps(AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/AFPCalculator.html" title="class in org.biojava.bio.structure.align.fatcat.calc">AFPCalculator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/FeatureList.html#sortByStart()">sortByStart()</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/FeatureList.html" title="class in org.biojava3.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block">Create a new list that is ordered by the starting index of the features' locations.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Feature.html#source()">source()</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Feature.html" title="class in org.biojava3.genome.parsers.gff">Feature</a></dt>
<dd>
<div class="block">Get source (aka method).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/FeatureList.html#splice(org.biojava3.core.sequence.DNASequence)">splice(DNASequence)</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/FeatureList.html" title="class in org.biojava3.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block">Concatenate successive portions of the specified sequence
 using the feature locations in the list.</div>
</dd>
<dt><a href="./org/biojava3/genome/util/SplitFasta.html" title="class in org.biojava3.genome.util"><span class="strong">SplitFasta</span></a> - Class in <a href="./org/biojava3/genome/util/package-summary.html">org.biojava3.genome.util</a></dt>
<dd>
<div class="block">Utility to write each Fasta entry to a unique file</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/util/SplitFasta.html#SplitFasta()">SplitFasta()</a></span> - Constructor for class org.biojava3.genome.util.<a href="./org/biojava3/genome/util/SplitFasta.html" title="class in org.biojava3.genome.util">SplitFasta</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/BigSqrt.html#sqrt(java.math.BigInteger)">sqrt(BigInteger)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/BigSqrt.html" title="class in org.biojava.bio.structure.secstruc">BigSqrt</a></dt>
<dd>
<div class="block">Get square root</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/BigSqrt.html#sqrt(java.math.BigDecimal)">sqrt(BigDecimal)</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/BigSqrt.html" title="class in org.biojava.bio.structure.secstruc">BigSqrt</a></dt>
<dd>
<div class="block">Get square root</div>
</dd>
<dt><a href="./org/biojava/bio/structure/gui/util/color/SqrtColorMapper.html" title="class in org.biojava.bio.structure.gui.util.color"><span class="strong">SqrtColorMapper</span></a> - Class in <a href="./org/biojava/bio/structure/gui/util/color/package-summary.html">org.biojava.bio.structure.gui.util.color</a></dt>
<dd>
<div class="block">Performs a sqrt transform on input before passing the values off to another
 colorMapper.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/SqrtColorMapper.html#SqrtColorMapper(org.biojava.bio.structure.gui.util.color.ContinuousColorMapper)">SqrtColorMapper(ContinuousColorMapper)</a></span> - Constructor for class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/SqrtColorMapper.html" title="class in org.biojava.bio.structure.gui.util.color">SqrtColorMapper</a></dt>
<dd>
<div class="block">Creates a new SqrtColorMapper.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/SSBond.html" title="class in org.biojava.bio.structure"><span class="strong">SSBond</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">A simple bean to store disulfid bridge information, the SSBOND records in the PDB files.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SSBond.html#SSBond()">SSBond()</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SSBond.html" title="class in org.biojava.bio.structure">SSBond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#ssHash">ssHash</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/PeptideProperties.html#standardAASet">standardAASet</a></span> - Static variable in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/PeptideProperties.html" title="class in org.biojava3.aaproperties">PeptideProperties</a></dt>
<dd>
<div class="block">Contains the 20 standard AA code in a set</div>
</dd>
<dt><a href="./org/biojava/bio/structure/StandardAminoAcid.html" title="class in org.biojava.bio.structure"><span class="strong">StandardAminoAcid</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">A class that provides a set of standard amino acids.</div>
</dd>
<dt><a href="./org/biojava3/alignment/StandardRescoreRefiner.html" title="class in org.biojava3.alignment"><span class="strong">StandardRescoreRefiner</span></a>&lt;<a href="./org/biojava3/alignment/StandardRescoreRefiner.html" title="type parameter in StandardRescoreRefiner">S</a> extends <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template">Sequence</a>&lt;<a href="./org/biojava3/alignment/StandardRescoreRefiner.html" title="type parameter in StandardRescoreRefiner">C</a>&gt;,<a href="./org/biojava3/alignment/StandardRescoreRefiner.html" title="type parameter in StandardRescoreRefiner">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/alignment/package-summary.html">org.biojava3.alignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/StandardRescoreRefiner.html#StandardRescoreRefiner(org.biojava3.alignment.Alignments.PairInProfileScorerType, org.biojava3.alignment.Alignments.ProfileProfileAlignerType)">StandardRescoreRefiner(Alignments.PairInProfileScorerType, Alignments.ProfileProfileAlignerType)</a></span> - Constructor for class org.biojava3.alignment.<a href="./org/biojava3/alignment/StandardRescoreRefiner.html" title="class in org.biojava3.alignment">StandardRescoreRefiner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Location.html#start()">start()</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Location.html" title="class in org.biojava3.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Get starting index (origin 0).</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/StartCodonSequence.html" title="class in org.biojava3.core.sequence"><span class="strong">StartCodonSequence</span></a> - Class in <a href="./org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a></dt>
<dd>
<div class="block">Used to map the start codon feature on a gene</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/StartCodonSequence.html#StartCodonSequence(org.biojava3.core.sequence.TranscriptSequence, int, int)">StartCodonSequence(TranscriptSequence, int, int)</a></span> - Constructor for class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/StartCodonSequence.html" title="class in org.biojava3.core.sequence">StartCodonSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/webstart/ConfigXMLHandler.html#startElement(java.lang.String, java.lang.String, java.lang.String, org.xml.sax.Attributes)">startElement(String, String, String, Attributes)</a></span> - Method in class org.biojava.bio.structure.align.webstart.<a href="./org/biojava/bio/structure/align/webstart/ConfigXMLHandler.html" title="class in org.biojava.bio.structure.align.webstart">ConfigXMLHandler</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentProgressListener.html#startingAlignment(java.lang.String, org.biojava.bio.structure.Atom[], java.lang.String, org.biojava.bio.structure.Atom[])">startingAlignment(String, Atom[], String, Atom[])</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentProgressListener.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Location.html#startsAfter(org.biojava3.genome.parsers.gff.Location)">startsAfter(Location)</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Location.html" title="class in org.biojava3.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Check if this location starts after the other location starts.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Location.html#startsBefore(org.biojava3.genome.parsers.gff.Location)">startsBefore(Location)</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Location.html" title="class in org.biojava3.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Check if this location starts before other location starts.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce"><span class="strong">StartupParameters</span></a> - Class in <a href="./org/biojava/bio/structure/align/ce/package-summary.html">org.biojava.bio.structure.align.ce</a></dt>
<dd>
<div class="block">a simple bean that contains the parameters that can get set at startup</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#StartupParameters()">StartupParameters()</a></span> - Constructor for class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html#stateChanged(javax.swing.event.ChangeEvent)">stateChanged(ChangeEvent)</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/ScaleableMatrixPanel.html" title="class in org.biojava.bio.structure.gui">ScaleableMatrixPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/SequenceDisplay.html#stateChanged(javax.swing.event.ChangeEvent)">stateChanged(ChangeEvent)</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/SequenceDisplay.html" title="class in org.biojava.bio.structure.gui">SequenceDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/gui/aligpanel/StatusDisplay.html" title="class in org.biojava.bio.structure.align.gui.aligpanel"><span class="strong">StatusDisplay</span></a> - Class in <a href="./org/biojava/bio/structure/align/gui/aligpanel/package-summary.html">org.biojava.bio.structure.align.gui.aligpanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/StatusDisplay.html#StatusDisplay()">StatusDisplay()</a></span> - Constructor for class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/StatusDisplay.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">StatusDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#STOP">STOP</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/StopCodonSequence.html" title="class in org.biojava3.core.sequence"><span class="strong">StopCodonSequence</span></a> - Class in <a href="./org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a></dt>
<dd>
<div class="block">Used to map the stop codon sequence on a gene</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/StopCodonSequence.html#StopCodonSequence(org.biojava3.core.sequence.TranscriptSequence, int, int)">StopCodonSequence(TranscriptSequence, int, int)</a></span> - Constructor for class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/StopCodonSequence.html" title="class in org.biojava3.core.sequence">StopCodonSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/ParameterGUI.html#storeParameters()">storeParameters()</a></span> - Method in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/ParameterGUI.html" title="class in org.biojava.bio.structure.align.gui">ParameterGUI</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileParser.html#STRAND">STRAND</a></span> - Static variable in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileParser.html" title="class in org.biojava.bio.structure.io">PDBFileParser</a></dt>
<dd>
<div class="block">Strand secondary structure assignment.</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/Strand.html" title="enum in org.biojava3.core.sequence"><span class="strong">Strand</span></a> - Enum in <a href="./org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a></dt>
<dd>
<div class="block">Provides a way of representing the strand of a sequence, location
 hit or feature.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#strand">strand</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#strandmax">strandmax</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#strandmin">strandmin</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html" title="class in org.biojava.bio.structure.align.pairwise"><span class="strong">StrCompAlignment</span></a> - Class in <a href="./org/biojava/bio/structure/align/pairwise/package-summary.html">org.biojava.bio.structure.align.pairwise</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html#StrCompAlignment(int, int)">StrCompAlignment(int, int)</a></span> - Constructor for class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/StrCompAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">StrCompAlignment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#STRING_LIMIT">STRING_LIMIT</a></span> - Static variable in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader"><span class="strong">StringProxySequenceReader</span></a>&lt;<a href="./org/biojava3/core/sequence/loader/StringProxySequenceReader.html" title="type parameter in StringProxySequenceReader">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/core/sequence/loader/package-summary.html">org.biojava3.core.sequence.loader</a></dt>
<dd>
<div class="block">An example of a ProxySequenceReader that is created from a String.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/StringProxySequenceReader.html#StringProxySequenceReader(java.lang.String, org.biojava3.core.sequence.template.CompoundSet)">StringProxySequenceReader(String, CompoundSet&lt;C&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align"><span class="strong">StrucAligParameters</span></a> - Class in <a href="./org/biojava/bio/structure/align/package-summary.html">org.biojava.bio.structure.align</a></dt>
<dd>
<div class="block">A class that contains all the parameters of the structure alignment algorithm.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#StrucAligParameters()">StrucAligParameters()</a></span> - Constructor for class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html" title="class in org.biojava.bio.structure.io.mmcif.model"><span class="strong">Struct</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a></dt>
<dd>
<div class="block">a bean to contain the data of the _struct lines</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html#Struct()">Struct()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html" title="class in org.biojava.bio.structure.io.mmcif.model">Struct</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html" title="class in org.biojava.bio.structure.io.mmcif.model"><span class="strong">StructAsym</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a></dt>
<dd>
<div class="block">Contains the data for _struct_asym</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html#StructAsym()">StructAsym()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructAsym.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructAsym</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/StructKeywords.html" title="class in org.biojava.bio.structure.io.mmcif.model"><span class="strong">StructKeywords</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructKeywords.html#StructKeywords()">StructKeywords()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructKeywords.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructKeywords</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html" title="class in org.biojava.bio.structure.io.mmcif.model"><span class="strong">StructRef</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a></dt>
<dd>
<div class="block">A class to containt the _struct_ref field data</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html#StructRef()">StructRef()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRef.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRef</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model"><span class="strong">StructRefSeq</span></a> - Class in <a href="./org/biojava/bio/structure/io/mmcif/model/package-summary.html">org.biojava.bio.structure.io.mmcif.model</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html#StructRefSeq()">StructRefSeq()</a></span> - Constructor for class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/StructRefSeq.html" title="class in org.biojava.bio.structure.io.mmcif.model">StructRefSeq</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure"><span class="strong">Structure</span></a> - Interface in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">Interface for a structure object.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/StructureAlignment.html" title="interface in org.biojava.bio.structure.align"><span class="strong">StructureAlignment</span></a> - Interface in <a href="./org/biojava/bio/structure/align/package-summary.html">org.biojava.bio.structure.align</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/gui/StructureAlignmentDisplay.html" title="class in org.biojava.bio.structure.align.gui"><span class="strong">StructureAlignmentDisplay</span></a> - Class in <a href="./org/biojava/bio/structure/align/gui/package-summary.html">org.biojava.bio.structure.align.gui</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/StructureAlignmentDisplay.html#StructureAlignmentDisplay()">StructureAlignmentDisplay()</a></span> - Constructor for class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/StructureAlignmentDisplay.html" title="class in org.biojava.bio.structure.align.gui">StructureAlignmentDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/StructureAlignmentFactory.html" title="class in org.biojava.bio.structure.align"><span class="strong">StructureAlignmentFactory</span></a> - Class in <a href="./org/biojava/bio/structure/align/package-summary.html">org.biojava.bio.structure.align</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructureAlignmentFactory.html#StructureAlignmentFactory()">StructureAlignmentFactory()</a></span> - Constructor for class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructureAlignmentFactory.html" title="class in org.biojava.bio.structure.align">StructureAlignmentFactory</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html" title="class in org.biojava.bio.structure.align.gui.jmol"><span class="strong">StructureAlignmentJmol</span></a> - Class in <a href="./org/biojava/bio/structure/align/gui/jmol/package-summary.html">org.biojava.bio.structure.align.gui.jmol</a></dt>
<dd>
<div class="block">A class that provides a simple GUI for Jmol</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html#StructureAlignmentJmol()">StructureAlignmentJmol()</a></span> - Constructor for class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html" title="class in org.biojava.bio.structure.align.gui.jmol">StructureAlignmentJmol</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html#StructureAlignmentJmol(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">StructureAlignmentJmol(AFPChain, Atom[], Atom[])</a></span> - Constructor for class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html" title="class in org.biojava.bio.structure.align.gui.jmol">StructureAlignmentJmol</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/fatcat/calc/StructureAlignmentOptimizer.html" title="class in org.biojava.bio.structure.align.fatcat.calc"><span class="strong">StructureAlignmentOptimizer</span></a> - Class in <a href="./org/biojava/bio/structure/align/fatcat/calc/package-summary.html">org.biojava.bio.structure.align.fatcat.calc</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/StructureAlignmentOptimizer.html#StructureAlignmentOptimizer(int, int, org.biojava.bio.structure.Atom[], int, int, org.biojava.bio.structure.Atom[], int, int[][])">StructureAlignmentOptimizer(int, int, Atom[], int, int, Atom[], int, int[][])</a></span> - Constructor for class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/StructureAlignmentOptimizer.html" title="class in org.biojava.bio.structure.align.fatcat.calc">StructureAlignmentOptimizer</a></dt>
<dd>
<div class="block">optimize the structural alignment by update the equivalent residues
      and then run dynamic programming
      input: len1 the length of structure 1; c1: the structure information of 1
             len2 the length of structure 2; c2: the structure information of 2
             iniLen and iniSet is the length and list of initial equivalent residues</div>
</dd>
<dt><a href="./org/biojava3/protmod/structure/StructureAtom.html" title="class in org.biojava3.protmod.structure"><span class="strong">StructureAtom</span></a> - Class in <a href="./org/biojava3/protmod/structure/package-summary.html">org.biojava3.protmod.structure</a></dt>
<dd>
<div class="block">Everything that is needed to uniquely describe a atom.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/StructureAtom.html#StructureAtom(org.biojava3.protmod.structure.StructureGroup, java.lang.String)">StructureAtom(StructureGroup, String)</a></span> - Constructor for class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/StructureAtom.html" title="class in org.biojava3.protmod.structure">StructureAtom</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/protmod/structure/StructureAtomLinkage.html" title="class in org.biojava3.protmod.structure"><span class="strong">StructureAtomLinkage</span></a> - Class in <a href="./org/biojava3/protmod/structure/package-summary.html">org.biojava3.protmod.structure</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/StructureAtomLinkage.html#StructureAtomLinkage(org.biojava3.protmod.structure.StructureAtom, org.biojava3.protmod.structure.StructureAtom, double)">StructureAtomLinkage(StructureAtom, StructureAtom, double)</a></span> - Constructor for class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/StructureAtomLinkage.html" title="class in org.biojava3.protmod.structure">StructureAtomLinkage</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/protmod/io/StructureAtomXMLConverter.html" title="class in org.biojava3.protmod.io"><span class="strong">StructureAtomXMLConverter</span></a> - Class in <a href="./org/biojava3/protmod/io/package-summary.html">org.biojava3.protmod.io</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/io/StructureAtomXMLConverter.html#StructureAtomXMLConverter()">StructureAtomXMLConverter()</a></span> - Constructor for class org.biojava3.protmod.io.<a href="./org/biojava3/protmod/io/StructureAtomXMLConverter.html" title="class in org.biojava3.protmod.io">StructureAtomXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/server/StructureEvent.html" title="interface in org.biojava.bio.structure.server"><span class="strong">StructureEvent</span></a> - Interface in <a href="./org/biojava/bio/structure/server/package-summary.html">org.biojava.bio.structure.server</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/server/StructureEventImpl.html" title="class in org.biojava.bio.structure.server"><span class="strong">StructureEventImpl</span></a> - Class in <a href="./org/biojava/bio/structure/server/package-summary.html">org.biojava.bio.structure.server</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureEventImpl.html#StructureEventImpl(org.biojava.bio.structure.Structure)">StructureEventImpl(Structure)</a></span> - Constructor for class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureEventImpl.html" title="class in org.biojava.bio.structure.server">StructureEventImpl</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/StructureException.html" title="class in org.biojava.bio.structure"><span class="strong">StructureException</span></a> - Exception in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">An exception during the parsing of a PDB file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureException.html#StructureException(java.lang.String)">StructureException(String)</a></span> - Constructor for exception org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureException.html" title="class in org.biojava.bio.structure">StructureException</a></dt>
<dd>
<div class="block">Constructs a StructureException object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureException.html#StructureException(java.lang.String, java.lang.Throwable)">StructureException(String, Throwable)</a></span> - Constructor for exception org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureException.html" title="class in org.biojava.bio.structure">StructureException</a></dt>
<dd>
<div class="block">Constructs a StructureException object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureException.html#StructureException(java.lang.Throwable)">StructureException(Throwable)</a></span> - Constructor for exception org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureException.html" title="class in org.biojava.bio.structure">StructureException</a></dt>
<dd>
<div class="block">Constructs a StructureException object.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/server/StructureFetcherRunnable.html" title="class in org.biojava.bio.structure.server"><span class="strong">StructureFetcherRunnable</span></a> - Class in <a href="./org/biojava/bio/structure/server/package-summary.html">org.biojava.bio.structure.server</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>&nbsp;</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/server/StructureFetcherRunnable.html#StructureFetcherRunnable(org.biojava.bio.structure.server.PDBInstallation)">StructureFetcherRunnable(PDBInstallation)</a></span> - Constructor for class org.biojava.bio.structure.server.<a href="./org/biojava/bio/structure/server/StructureFetcherRunnable.html" title="class in org.biojava.bio.structure.server">StructureFetcherRunnable</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span></div>
&nbsp;</dd>
<dt><a href="./org/biojava3/protmod/structure/StructureGroup.html" title="class in org.biojava3.protmod.structure"><span class="strong">StructureGroup</span></a> - Class in <a href="./org/biojava3/protmod/structure/package-summary.html">org.biojava3.protmod.structure</a></dt>
<dd>
<div class="block">Information of a group (residue or ligand) involved in a modification.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/StructureGroup.html#StructureGroup()">StructureGroup()</a></span> - Constructor for class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/StructureGroup.html" title="class in org.biojava3.protmod.structure">StructureGroup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/StructureGroup.html#StructureGroup(org.biojava.bio.structure.ResidueNumber, java.lang.String, boolean)">StructureGroup(ResidueNumber, String, boolean)</a></span> - Constructor for class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/StructureGroup.html" title="class in org.biojava3.protmod.structure">StructureGroup</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/protmod/io/StructureGroupXMLConverter.html" title="class in org.biojava3.protmod.io"><span class="strong">StructureGroupXMLConverter</span></a> - Class in <a href="./org/biojava3/protmod/io/package-summary.html">org.biojava3.protmod.io</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/io/StructureGroupXMLConverter.html#StructureGroupXMLConverter()">StructureGroupXMLConverter()</a></span> - Constructor for class org.biojava3.protmod.io.<a href="./org/biojava3/protmod/io/StructureGroupXMLConverter.html" title="class in org.biojava3.protmod.io">StructureGroupXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure"><span class="strong">StructureImpl</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">Implementation of a PDB Structure.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#StructureImpl()">StructureImpl()</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">Constructs a StructureImpl object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#StructureImpl(org.biojava.bio.structure.Group)">StructureImpl(Group)</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">construct a Structure object that only contains a single group</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#StructureImpl(org.biojava.bio.structure.Chain)">StructureImpl(Chain)</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">construct a Structure object that contains a particular chain</div>
</dd>
<dt><a href="./org/biojava/bio/structure/io/StructureIO.html" title="interface in org.biojava.bio.structure.io"><span class="strong">StructureIO</span></a> - Interface in <a href="./org/biojava/bio/structure/io/package-summary.html">org.biojava.bio.structure.io</a></dt>
<dd>
<div class="block">Defines the interface how to access Structure (- PDB file) input,output readers, writers.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/io/StructureIOFile.html" title="interface in org.biojava.bio.structure.io"><span class="strong">StructureIOFile</span></a> - Interface in <a href="./org/biojava/bio/structure/io/package-summary.html">org.biojava.bio.structure.io</a></dt>
<dd>
<div class="block">interface StructureIOFile extends the StructureIO interface
 and adds a few File specific methods.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/server/StructureListener.html" title="interface in org.biojava.bio.structure.server"><span class="strong">StructureListener</span></a> - Interface in <a href="./org/biojava/bio/structure/server/package-summary.html">org.biojava.bio.structure.server</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/client/StructureName.html" title="class in org.biojava.bio.structure.align.client"><span class="strong">StructureName</span></a> - Class in <a href="./org/biojava/bio/structure/align/client/package-summary.html">org.biojava.bio.structure.align.client</a></dt>
<dd>
<div class="block">A utility class that makes working with names of strucutres, domains and ranges easier</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/StructureName.html#StructureName(java.lang.String)">StructureName(String)</a></span> - Constructor for class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/StructureName.html" title="class in org.biojava.bio.structure.align.client">StructureName</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/StructurePairAligner.html" title="class in org.biojava.bio.structure.align"><span class="strong">StructurePairAligner</span></a> - Class in <a href="./org/biojava/bio/structure/align/package-summary.html">org.biojava.bio.structure.align</a></dt>
<dd>
<div class="block">Perform a pairwise protein structure superimposition.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StructurePairAligner.html#StructurePairAligner()">StructurePairAligner()</a></span> - Constructor for class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StructurePairAligner.html" title="class in org.biojava.bio.structure.align">StructurePairAligner</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/gui/util/StructurePairSelector.html" title="interface in org.biojava.bio.structure.gui.util"><span class="strong">StructurePairSelector</span></a> - Interface in <a href="./org/biojava/bio/structure/gui/util/package-summary.html">org.biojava.bio.structure.gui.util</a></dt>
<dd>
<div class="block">To be implemented by JPanels that are part of the GUI to trigger structure aligmnents.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/io/StructureProvider.html" title="interface in org.biojava.bio.structure.io"><span class="strong">StructureProvider</span></a> - Interface in <a href="./org/biojava/bio/structure/io/package-summary.html">org.biojava.bio.structure.io</a></dt>
<dd>
<div class="block">A class that can provide a protein structure object from somewhere.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/server/StructureServer.html" title="interface in org.biojava.bio.structure.server"><span class="strong">StructureServer</span></a> - Interface in <a href="./org/biojava/bio/structure/server/package-summary.html">org.biojava.bio.structure.server</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>&nbsp;</div>
</dd>
<dt><a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure"><span class="strong">StructureTools</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">A class that provides some tool methods.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#StructureTools()">StructureTools()</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/protmod/structure/StructureUtil.html" title="class in org.biojava3.protmod.structure"><span class="strong">StructureUtil</span></a> - Class in <a href="./org/biojava3/protmod/structure/package-summary.html">org.biojava3.protmod.structure</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/structure/gui/StructureViewer.html" title="interface in org.biojava3.structure.gui"><span class="strong">StructureViewer</span></a> - Interface in <a href="./org/biojava3/structure/gui/package-summary.html">org.biojava3.structure.gui</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/ConcurrencyTools.html#submit(java.util.concurrent.Callable, java.lang.String)">submit(Callable&lt;T&gt;, String)</a></span> - Static method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/ConcurrencyTools.html" title="class in org.biojava3.core.util">ConcurrencyTools</a></dt>
<dd>
<div class="block">Queues up a task and adds a log entry.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/ConcurrencyTools.html#submit(java.util.concurrent.Callable)">submit(Callable&lt;T&gt;)</a></span> - Static method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/ConcurrencyTools.html" title="class in org.biojava3.core.util">ConcurrencyTools</a></dt>
<dd>
<div class="block">Queues up a task and adds a default log entry.</div>
</dd>
<dt><a href="./org/biojava3/alignment/template/SubstitutionMatrix.html" title="interface in org.biojava3.alignment.template"><span class="strong">SubstitutionMatrix</span></a>&lt;<a href="./org/biojava3/alignment/template/SubstitutionMatrix.html" title="type parameter in SubstitutionMatrix">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Interface in <a href="./org/biojava3/alignment/template/package-summary.html">org.biojava3.alignment.template</a></dt>
<dd>
<div class="block">Defines a data structure which holds the score (penalty or bonus) given during alignment for the exchange of one
 <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template"><code>Compound</code></a> in a sequence for another.</div>
</dd>
<dt><a href="./org/biojava3/alignment/SubstitutionMatrixHelper.html" title="class in org.biojava3.alignment"><span class="strong">SubstitutionMatrixHelper</span></a> - Class in <a href="./org/biojava3/alignment/package-summary.html">org.biojava3.alignment</a></dt>
<dd>
<div class="block">Static utility to access substitution matrices that come bundled with BioJava.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#substract(org.biojava.bio.structure.Atom, org.biojava.bio.structure.Atom)">substract(Atom, Atom)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block"><span class="strong">Deprecated.</span>
<div class="block"><i>use <code>subtract</code> instead.</i></div>
</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#subtract(org.biojava.bio.structure.Atom, org.biojava.bio.structure.Atom)">subtract(Atom, Atom)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">subtract two atoms ( a - b).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Location.html#suffix(int)">suffix(int)</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Location.html" title="class in org.biojava3.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">The part of this location after the specified position.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Location.html#suffix(org.biojava3.genome.parsers.gff.Location)">suffix(Location)</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Location.html" title="class in org.biojava3.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">The part of this location after the other location (not inclusive).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/ProteinDomainParser.html#suggestDomains(org.biojava.bio.structure.Structure)">suggestDomains(Structure)</a></span> - Static method in class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/ProteinDomainParser.html" title="class in org.biojava.bio.structure.domain">ProteinDomainParser</a></dt>
<dd>
<div class="block">Suggest domains for a protein structure</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/ProteinDomainParser.html#suggestDomains(org.biojava.bio.structure.Atom[])">suggestDomains(Atom[])</a></span> - Static method in class org.biojava.bio.structure.domain.<a href="./org/biojava/bio/structure/domain/ProteinDomainParser.html" title="class in org.biojava.bio.structure.domain">ProteinDomainParser</a></dt>
<dd>
<div class="block">Suggest domains for a set of Calpha atoms</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AFPChainCoordManager.html#SUMMARY_POS">SUMMARY_POS</a></span> - Static variable in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AFPChainCoordManager.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AFPChainCoordManager</a></dt>
<dd>
<div class="block">Position at which the alignment summary is printed</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#svd()">svd()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Singular Value Decomposition</div>
</dd>
<dt><a href="./org/biojava/bio/structure/SVDSuperimposer.html" title="class in org.biojava.bio.structure"><span class="strong">SVDSuperimposer</span></a> - Class in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">A class that calculates the superimposition between two sets of atoms
 inspired by the biopython SVDSuperimposer class...</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SVDSuperimposer.html#SVDSuperimposer(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">SVDSuperimposer(Atom[], Atom[])</a></span> - Constructor for class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SVDSuperimposer.html" title="class in org.biojava.bio.structure">SVDSuperimposer</a></dt>
<dd>
<div class="block">Create a SVDSuperimposer object and calculate a SVD superimposition of two sets of atoms.</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/util/SynchronizedOutFile.html" title="class in org.biojava.bio.structure.align.util"><span class="strong">SynchronizedOutFile</span></a> - Class in <a href="./org/biojava/bio/structure/align/util/package-summary.html">org.biojava.bio.structure.align.util</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/SynchronizedOutFile.html#SynchronizedOutFile(java.io.File)">SynchronizedOutFile(File)</a></span> - Constructor for class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/SynchronizedOutFile.html" title="class in org.biojava.bio.structure.align.util">SynchronizedOutFile</a></dt>
<dd>
<div class="block">create a thread safe wrapper for working with this file</div>
</dd>
<dt><a href="./org/biojava/bio/structure/align/gui/SystemInfo.html" title="class in org.biojava.bio.structure.align.gui"><span class="strong">SystemInfo</span></a> - Class in <a href="./org/biojava/bio/structure/align/gui/package-summary.html">org.biojava.bio.structure.align.gui</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/SystemInfo.html#SystemInfo()">SystemInfo()</a></span> - Constructor for class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/SystemInfo.html" title="class in org.biojava.bio.structure.align.gui">SystemInfo</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_T_">
<!--   -->
</a>
<h2 class="title">T</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/Constraints.html#T">T</a></span> - Static variable in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/Constraints.html" title="class in org.biojava3.aaproperties">Constraints</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/transcription/Table.html" title="interface in org.biojava3.core.sequence.transcription"><span class="strong">Table</span></a> - Interface in <a href="./org/biojava3/core/sequence/transcription/package-summary.html">org.biojava3.core.sequence.transcription</a></dt>
<dd>
<div class="block">Provides a way of separating us from the specific <code>IUPACTable</code> even
 though this is the only implementing class for the interface.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.Builder.html#table(java.lang.Integer)">table(Integer)</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava3.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>
<div class="block">Uses the static instance of <a href="./org/biojava3/core/sequence/io/IUPACParser.html" title="class in org.biojava3.core.sequence.io"><code>IUPACParser</code></a> to find instances of
 <a href="./org/biojava3/core/sequence/io/IUPACParser.IUPACTable.html" title="class in org.biojava3.core.sequence.io"><code>IUPACParser.IUPACTable</code></a>s by ID.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.Builder.html#table(java.lang.String)">table(String)</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava3.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>
<div class="block">Uses the static instance of <a href="./org/biojava3/core/sequence/io/IUPACParser.html" title="class in org.biojava3.core.sequence.io"><code>IUPACParser</code></a> to find instances of
 <a href="./org/biojava3/core/sequence/io/IUPACParser.IUPACTable.html" title="class in org.biojava3.core.sequence.io"><code>IUPACParser.IUPACTable</code></a>s by its String name</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.Builder.html#table(org.biojava3.core.sequence.transcription.Table)">table(Table)</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava3.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/transcription/Table.CaseInsensitiveTriplet.html" title="class in org.biojava3.core.sequence.transcription"><span class="strong">Table.CaseInsensitiveTriplet</span></a> - Class in <a href="./org/biojava3/core/sequence/transcription/package-summary.html">org.biojava3.core.sequence.transcription</a></dt>
<dd>
<div class="block">Class used to hold three nucleotides together and allow for equality
 to be assessed in a case insensitive manner.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/Table.CaseInsensitiveTriplet.html#Table.CaseInsensitiveTriplet(org.biojava3.core.sequence.compound.NucleotideCompound, org.biojava3.core.sequence.compound.NucleotideCompound, org.biojava3.core.sequence.compound.NucleotideCompound)">Table.CaseInsensitiveTriplet(NucleotideCompound, NucleotideCompound, NucleotideCompound)</a></span> - Constructor for class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/Table.CaseInsensitiveTriplet.html" title="class in org.biojava3.core.sequence.transcription">Table.CaseInsensitiveTriplet</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/transcription/Table.Codon.html" title="class in org.biojava3.core.sequence.transcription"><span class="strong">Table.Codon</span></a> - Class in <a href="./org/biojava3/core/sequence/transcription/package-summary.html">org.biojava3.core.sequence.transcription</a></dt>
<dd>
<div class="block">Instance of a Codon which is 3 <a href="./org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound"><code>NucleotideCompound</code></a>s, its
 corresponding <a href="./org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound"><code>AminoAcidCompound</code></a> and if it is a start or stop codon.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/Table.Codon.html#Table.Codon(org.biojava3.core.sequence.transcription.Table.CaseInsensitiveTriplet, org.biojava3.core.sequence.compound.AminoAcidCompound, boolean, boolean)">Table.Codon(Table.CaseInsensitiveTriplet, AminoAcidCompound, boolean, boolean)</a></span> - Constructor for class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/Table.Codon.html" title="class in org.biojava3.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/Table.Codon.html#Table.Codon(org.biojava3.core.sequence.transcription.Table.CaseInsensitiveTriplet)">Table.Codon(Table.CaseInsensitiveTriplet)</a></span> - Constructor for class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/Table.Codon.html" title="class in org.biojava3.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/TaxonomyID.html" title="class in org.biojava3.core.sequence"><span class="strong">TaxonomyID</span></a> - Class in <a href="./org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a></dt>
<dd>
<div class="block">A sequence can be associated with a species or Taxonomy ID</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/TaxonomyID.html#TaxonomyID(java.lang.String, org.biojava3.core.sequence.DataSource)">TaxonomyID(String, DataSource)</a></span> - Constructor for class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/TaxonomyID.html" title="class in org.biojava3.core.sequence">TaxonomyID</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#taylor">taylor</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html#TD">TD</a></span> - Static variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html" title="class in org.biojava.bio.structure.domain.pdp">PDPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html#TD1">TD1</a></span> - Static variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/PDPParameters.html" title="class in org.biojava.bio.structure.domain.pdp">PDPParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html#terminate()">terminate()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobRunnable.html" title="class in org.biojava.bio.structure.align.client">FarmJobRunnable</a></dt>
<dd>
<div class="block">Send signal to terminate calculations</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/FarmJob.html#terminate()">terminate()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/FarmJob.html" title="class in org.biojava.bio.structure.align">FarmJob</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/MenuCreator.html#TEXT_ONLY">TEXT_ONLY</a></span> - Static variable in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/MenuCreator.html" title="class in org.biojava.bio.structure.align.gui">MenuCreator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html#TEXT_SCALE_COLOR">TEXT_SCALE_COLOR</a></span> - Static variable in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/SequenceScalePanel.html" title="class in org.biojava.bio.structure.gui.util">SequenceScalePanel</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/features/TextFeature.html" title="class in org.biojava3.core.sequence.features"><span class="strong">TextFeature</span></a>&lt;<a href="./org/biojava3/core/sequence/features/TextFeature.html" title="type parameter in TextFeature">S</a> extends <a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a>&lt;<a href="./org/biojava3/core/sequence/features/TextFeature.html" title="type parameter in TextFeature">C</a>&gt;,<a href="./org/biojava3/core/sequence/features/TextFeature.html" title="type parameter in TextFeature">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/core/sequence/features/package-summary.html">org.biojava3.core.sequence.features</a></dt>
<dd>
<div class="block">A implmentation of AbstractFeature</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/TextFeature.html#TextFeature(java.lang.String, java.lang.String, java.lang.String, java.lang.String)">TextFeature(String, String, String, String)</a></span> - Constructor for class org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/TextFeature.html" title="class in org.biojava3.core.sequence.features">TextFeature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#Thr">Thr</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#time">time</a></span> - Variable in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava3.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/ronn/Timer.html" title="class in org.biojava3.ronn"><span class="strong">Timer</span></a> - Class in <a href="./org/biojava3/ronn/package-summary.html">org.biojava3.ronn</a></dt>
<dd>
<div class="block">A simple timer, calculates the time interval between two events.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ronn/Timer.html#Timer()">Timer()</a></span> - Constructor for class org.biojava3.ronn.<a href="./org/biojava3/ronn/Timer.html" title="class in org.biojava3.ronn">Timer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/ronn/Timer.html#Timer(java.util.concurrent.TimeUnit)">Timer(TimeUnit)</a></span> - Constructor for class org.biojava3.ronn.<a href="./org/biojava3/ronn/Timer.html" title="class in org.biojava3.ronn">Timer</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#times(double)">times(double)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Multiply a matrix by a scalar, C = s*A</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#times(org.biojava.bio.structure.jama.Matrix)">times(Matrix)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Linear algebraic matrix multiplication, A * B</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#timesEquals(double)">timesEquals(double)</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Multiply a matrix by a scalar in place, A = s*A</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#tiny">tiny</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/UserConfiguration.html#TMP_DIR">TMP_DIR</a></span> - Static variable in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/UserConfiguration.html" title="class in org.biojava.bio.structure.align.util">UserConfiguration</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html#toAlignedPairs(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">toAlignedPairs(AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html" title="class in org.biojava.bio.structure.align.model">AfpChainWriter</a></dt>
<dd>
<div class="block">Prints the alignment in the simplest form: a list of aligned residues.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#toCE(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">toCE(Atom[], Atom[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html#toCE(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">toCE(AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html" title="class in org.biojava.bio.structure.align.model">AfpChainWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/HSVColorSpace.html#toCIEXYZ(float[])">toCIEXYZ(float[])</a></span> - Method in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/HSVColorSpace.html" title="class in org.biojava.bio.structure.gui.util.color">HSVColorSpace</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileConvert.html#toDASStructure(org.biojava3.core.util.XMLWriter)">toDASStructure(XMLWriter)</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileConvert.html" title="class in org.biojava.bio.structure.io">FileConvert</a></dt>
<dd>
<div class="block">convert a protein Structure to a DAS Structure XML response .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#toDBSearchResult()">toDBSearchResult()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html#toDBSearchResult(org.biojava.bio.structure.align.model.AFPChain)">toDBSearchResult(AFPChain)</a></span> - Static method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html" title="class in org.biojava.bio.structure.align.model">AfpChainWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/CollectionTools.html#toDoubleArray(java.util.Collection)">toDoubleArray(Collection&lt;Number&gt;)</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/CollectionTools.html" title="class in org.biojava.bio.structure.align.util">CollectionTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#toDssp3State">toDssp3State</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#toFatcat(org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">toFatcat(Atom[], Atom[])</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html#toFatCat(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">toFatCat(AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html" title="class in org.biojava.bio.structure.align.model">AfpChainWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html#toFatCatCore(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], boolean, boolean, boolean, boolean)">toFatCatCore(AFPChain, Atom[], Atom[], boolean, boolean, boolean, boolean)</a></span> - Static method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html" title="class in org.biojava.bio.structure.align.model">AfpChainWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html#toggleSelection(org.biojava.bio.structure.gui.util.AlignedPosition)">toggleSelection(AlignedPosition)</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AligPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/StatusDisplay.html#toggleSelection(org.biojava.bio.structure.gui.util.AlignedPosition)">toggleSelection(AlignedPosition)</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/StatusDisplay.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">StatusDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/events/AlignmentPositionListener.html#toggleSelection(org.biojava.bio.structure.gui.util.AlignedPosition)">toggleSelection(AlignedPosition)</a></span> - Method in interface org.biojava.bio.structure.gui.events.<a href="./org/biojava/bio/structure/gui/events/AlignmentPositionListener.html" title="interface in org.biojava.bio.structure.gui.events">AlignmentPositionListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/events/JmolAlignedPositionListener.html#toggleSelection(org.biojava.bio.structure.gui.util.AlignedPosition)">toggleSelection(AlignedPosition)</a></span> - Method in class org.biojava.bio.structure.gui.events.<a href="./org/biojava/bio/structure/gui/events/JmolAlignedPositionListener.html" title="class in org.biojava.bio.structure.gui.events">JmolAlignedPositionListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/views/ReversedSequenceView.html#toIndex(int)">toIndex(int)</a></span> - Method in class org.biojava3.core.sequence.views.<a href="./org/biojava3/core/sequence/views/ReversedSequenceView.html" title="class in org.biojava3.core.sequence.views">ReversedSequenceView</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/CollectionTools.html#toIntArray(java.util.Collection)">toIntArray(Collection&lt;Number&gt;)</a></span> - Static method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/CollectionTools.html" title="class in org.biojava.bio.structure.align.util">CollectionTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceMixin.html#toList(org.biojava3.core.sequence.template.Sequence)">toList(Sequence&lt;C&gt;)</a></span> - Static method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceMixin.html" title="class in org.biojava3.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">For the given <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a> this will return a <a href="http://java.sun.com/j2se/1.6.0/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util"><code>List</code></a> filled with
 the Compounds of that <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#toPDB(org.biojava.bio.structure.Structure, org.biojava.bio.structure.Structure)">toPDB(Structure, Structure)</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>
<div class="block">converts the alignment to a PDB file
 each of the structures will be represented as a model.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#toPDB()">toPDB()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#toPDB(java.lang.StringBuffer)">toPDB(StringBuffer)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#toPDB()">toPDB()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Convert the DBRef object to a DBREF record as it is used in PDB files</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#toPDB(java.lang.StringBuffer)">toPDB(StringBuffer)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">Append the PDB representation of this DBRef to the provided StringBuffer</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileConvert.html#toPDB()">toPDB()</a></span> - Method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileConvert.html" title="class in org.biojava.bio.structure.io">FileConvert</a></dt>
<dd>
<div class="block">Convert a structure into a PDB file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileConvert.html#toPDB(org.biojava.bio.structure.Atom)">toPDB(Atom)</a></span> - Static method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileConvert.html" title="class in org.biojava.bio.structure.io">FileConvert</a></dt>
<dd>
<div class="block">Prints the content of an Atom object as a PDB formatted line.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/FileConvert.html#toPDB(org.biojava.bio.structure.Atom, java.lang.StringBuffer)">toPDB(Atom, StringBuffer)</a></span> - Static method in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/FileConvert.html" title="class in org.biojava.bio.structure.io">FileConvert</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#toPDB()">toPDB()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>
<div class="block">Return a PDB representation of the PDB Header</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#toPDB(java.lang.StringBuffer)">toPDB(StringBuffer)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>
<div class="block">Appends a PDB representation of the PDB header to the provided StringBuffer</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBRecord.html#toPDB()">toPDB()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBRecord.html" title="interface in org.biojava.bio.structure">PDBRecord</a></dt>
<dd>
<div class="block">Returns a PDB file like representation of this record.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBRecord.html#toPDB(java.lang.StringBuffer)">toPDB(StringBuffer)</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBRecord.html" title="interface in org.biojava.bio.structure">PDBRecord</a></dt>
<dd>
<div class="block">Appends a PDB file like representation of this record to the provided StringBuffer.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ResidueNumber.html#toPDB()">toPDB()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ResidueNumber.html" title="class in org.biojava.bio.structure">ResidueNumber</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Site.html#toPDB()">toPDB()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Site.html" title="class in org.biojava.bio.structure">Site</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Site.html#toPDB(java.lang.StringBuffer)">toPDB(StringBuffer)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Site.html" title="class in org.biojava.bio.structure">Site</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SSBond.html#toPDB()">toPDB()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SSBond.html" title="class in org.biojava.bio.structure">SSBond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/SSBond.html#toPDB(java.lang.StringBuffer)">toPDB(StringBuffer)</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/SSBond.html" title="class in org.biojava.bio.structure">SSBond</a></dt>
<dd>
<div class="block">append the PDB representation of this SSBOND to the provided StringBUffer</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#toPDB()">toPDB()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">create a String that contains the contents of a PDB file .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#toPDB()">toPDB()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">create a String that contains the contents of a PDB file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/HSVColorSpace.html#toRGB(float[])">toRGB(float[])</a></span> - Method in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/HSVColorSpace.html" title="class in org.biojava.bio.structure.gui.util.color">HSVColorSpace</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#toRotMat()">toRotMat()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html#toRotMat(org.biojava.bio.structure.align.model.AFPChain)">toRotMat(AFPChain)</a></span> - Static method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html" title="class in org.biojava.bio.structure.align.model">AfpChainWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#torsionAngle(org.biojava.bio.structure.Atom, org.biojava.bio.structure.Atom, org.biojava.bio.structure.Atom, org.biojava.bio.structure.Atom)">torsionAngle(Atom, Atom, Atom, Atom)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">torsion angle 
 = angle between the normal vectors of the 
 two plains a-b-c and b-c-d.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html#toScoresList(org.biojava.bio.structure.align.model.AFPChain)">toScoresList(AFPChain)</a></span> - Static method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html" title="class in org.biojava.bio.structure.align.model">AfpChainWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/views/WindowedSequence.html#toStartIndex(int)">toStartIndex(int)</a></span> - Method in class org.biojava3.core.sequence.views.<a href="./org/biojava3/core/sequence/views/WindowedSequence.html" title="class in org.biojava3.core.sequence.views">WindowedSequence</a></dt>
<dd>
<div class="block">Calculates start index according to the equation start = ( (index-1) -
 windowSize) +1</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeParameters.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeParameters.html" title="class in org.biojava.bio.structure.align.ce">CeParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/StartupParameters.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/StartupParameters.html" title="class in org.biojava.bio.structure.align.ce">StartupParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/CountProgressListener.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/CountProgressListener.html" title="class in org.biojava.bio.structure.align.client">CountProgressListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/FarmJobParameters.html" title="class in org.biojava.bio.structure.align.client">FarmJobParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/PdbPair.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/PdbPair.html" title="class in org.biojava.bio.structure.align.client">PdbPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/client/StructureName.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.align.client.<a href="./org/biojava/bio/structure/align/client/StructureName.html" title="class in org.biojava.bio.structure.align.client">StructureName</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/FatCatParameters.html" title="class in org.biojava.bio.structure.align.fatcat.calc">FatCatParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCat.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCat.html" title="class in org.biojava.bio.structure.align.fatcat">FatCat</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/AtomInfo.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/AtomInfo.html" title="class in org.biojava.bio.structure.align.gui.jmol">AtomInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/AligMatEl.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/AligMatEl.html" title="class in org.biojava.bio.structure.align.helper">AligMatEl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/IndexPair.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/IndexPair.html" title="class in org.biojava.bio.structure.align.helper">IndexPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/JointFragments.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/JointFragments.html" title="class in org.biojava.bio.structure.align.helper">JointFragments</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFP.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFP.html" title="class in org.biojava.bio.structure.align.model">AFP</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AFPChain.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AFPChain.html" title="class in org.biojava.bio.structure.align.model">AFPChain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlignmentResult.html" title="class in org.biojava.bio.structure.align.pairwise">AlignmentResult</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/AlternativeAlignment.html" title="class in org.biojava.bio.structure.align.pairwise">AlternativeAlignment</a></dt>
<dd>
<div class="block">print the idx positions of this alignment</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.align.pairwise.<a href="./org/biojava/bio/structure/align/pairwise/FragmentPair.html" title="class in org.biojava.bio.structure.align.pairwise">FragmentPair</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/StrucAligParameters.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/StrucAligParameters.html" title="class in org.biojava.bio.structure.align">StrucAligParameters</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcid.html#toString()">toString()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcid.html" title="interface in org.biojava.bio.structure">AminoAcid</a></dt>
<dd>
<div class="block">string representation.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcidImpl.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcidImpl.html" title="class in org.biojava.bio.structure">AminoAcidImpl</a></dt>
<dd>
<div class="block">string representation.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AtomImpl.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AtomImpl.html" title="class in org.biojava.bio.structure">AtomImpl</a></dt>
<dd>
<div class="block">string representation.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Author.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Author.html" title="class in org.biojava.bio.structure">Author</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Bond.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Bond.html" title="class in org.biojava.bio.structure">Bond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Chain.html#toString()">toString()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Chain.html" title="interface in org.biojava.bio.structure">Chain</a></dt>
<dd>
<div class="block">string representation.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ChainImpl.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ChainImpl.html" title="class in org.biojava.bio.structure">ChainImpl</a></dt>
<dd>
<div class="block">String representation.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Compound.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Compound.html" title="class in org.biojava.bio.structure">Compound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/DBRef.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/DBRef.html" title="class in org.biojava.bio.structure">DBRef</a></dt>
<dd>
<div class="block">String representation of a DBRef.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/CutSites.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/CutSites.html" title="class in org.biojava.bio.structure.domain.pdp">CutSites</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/CutValues.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/CutValues.html" title="class in org.biojava.bio.structure.domain.pdp">CutValues</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Domain.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Domain.html" title="class in org.biojava.bio.structure.domain.pdp">Domain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/Segment.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/Segment.html" title="class in org.biojava.bio.structure.domain.pdp">Segment</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/AlignedPosition.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/AlignedPosition.html" title="class in org.biojava.bio.structure.gui.util">AlignedPosition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/AbstractBean.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/AbstractBean.html" title="class in org.biojava.bio.structure.io.mmcif.model">AbstractBean</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemComp.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemComp</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/ChemCompDescriptor.html" title="class in org.biojava.bio.structure.io.mmcif.model">ChemCompDescriptor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/DatabasePDBrev.html" title="class in org.biojava.bio.structure.io.mmcif.model">DatabasePDBrev</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Entity.html" title="class in org.biojava.bio.structure.io.mmcif.model">Entity</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.model.<a href="./org/biojava/bio/structure/io/mmcif/model/Struct.html" title="class in org.biojava.bio.structure.io.mmcif.model">Struct</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/JournalArticle.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/JournalArticle.html" title="class in org.biojava.bio.structure">JournalArticle</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/NucleotideImpl.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/NucleotideImpl.html" title="class in org.biojava.bio.structure">NucleotideImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBHeader.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBHeader.html" title="class in org.biojava.bio.structure">PDBHeader</a></dt>
<dd>
<div class="block">String representation</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ResidueNumber.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ResidueNumber.html" title="class in org.biojava.bio.structure">ResidueNumber</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopCategory.html#toString()">toString()</a></span> - Method in enum org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopCategory.html" title="enum in org.biojava.bio.structure.scop">ScopCategory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDescription.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDescription.html" title="class in org.biojava.bio.structure.scop">ScopDescription</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopDomain.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopDomain.html" title="class in org.biojava.bio.structure.scop">ScopDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopNode.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopNode.html" title="class in org.biojava.bio.structure.scop">ScopNode</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/HBond.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/HBond.html" title="class in org.biojava.bio.structure.secstruc">HBond</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStruc.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStruc.html" title="class in org.biojava.bio.structure.secstruc">SecStruc</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.secstruc.<a href="./org/biojava/bio/structure/secstruc/SecStrucGroup.html" title="class in org.biojava.bio.structure.secstruc">SecStrucGroup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Site.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Site.html" title="class in org.biojava.bio.structure">Site</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Structure.html#toString()">toString()</a></span> - Method in interface org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Structure.html" title="interface in org.biojava.bio.structure">Structure</a></dt>
<dd>
<div class="block">String representation of object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureImpl.html#toString()">toString()</a></span> - Method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureImpl.html" title="class in org.biojava.bio.structure">StructureImpl</a></dt>
<dd>
<div class="block">string representation.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/AminoAcidComposition.html#toString()">toString()</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/AminoAcidComposition.html" title="class in org.biojava3.aaproperties.xml">AminoAcidComposition</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/Element.html#toString()">toString()</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/Element.html" title="class in org.biojava3.aaproperties.xml">Element</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html#toString()">toString()</a></span> - Method in class org.biojava3.alignment.aaindex.<a href="./org/biojava3/alignment/aaindex/ScaledSubstitutionMatrix.html" title="class in org.biojava3.alignment.aaindex">ScaledSubstitutionMatrix</a></dt>
<dd>
<div class="block">Returns in a format similar to the standard NCBI files.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/GuideTree.html#toString()">toString()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/GuideTree.html" title="class in org.biojava3.alignment">GuideTree</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleAlignedSequence.html#toString()">toString()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleAlignedSequence.html" title="class in org.biojava3.alignment">SimpleAlignedSequence</a></dt>
<dd>
<div class="block">Provides standard Java language access to results of <a href="./org/biojava3/alignment/SimpleAlignedSequence.html#getSequenceAsString()"><code>SimpleAlignedSequence.getSequenceAsString()</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleProfile.html#toString(int)">toString(int)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleProfile.html" title="class in org.biojava3.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleProfile.html#toString(org.biojava3.alignment.template.Profile.StringFormat)">toString(Profile.StringFormat)</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleProfile.html" title="class in org.biojava3.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleProfile.html#toString()">toString()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleProfile.html" title="class in org.biojava3.alignment">SimpleProfile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/SimpleSubstitutionMatrix.html#toString()">toString()</a></span> - Method in class org.biojava3.alignment.<a href="./org/biojava3/alignment/SimpleSubstitutionMatrix.html" title="class in org.biojava3.alignment">SimpleSubstitutionMatrix</a></dt>
<dd>
<div class="block">Returns in a format similar to the standard NCBI files.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/Profile.html#toString()">toString()</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns a simple view of the alignment profile.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/Profile.html#toString(int)">toString(int)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns a formatted view of the alignment profile.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/Profile.html#toString(org.biojava3.alignment.template.Profile.StringFormat)">toString(Profile.StringFormat)</a></span> - Method in interface org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/Profile.html" title="interface in org.biojava3.alignment.template">Profile</a></dt>
<dd>
<div class="block">Returns a formatted view of the alignment profile.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/AccessionID.html#toString()">toString()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/AccessionID.html" title="class in org.biojava3.core.sequence">AccessionID</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/features/DBReferenceInfo.html#toString()">toString()</a></span> - Method in class org.biojava3.core.sequence.features.<a href="./org/biojava3/core/sequence/features/DBReferenceInfo.html" title="class in org.biojava3.core.sequence.features">DBReferenceInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html#toString()">toString()</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/SequenceFileProxyLoader.html" title="class in org.biojava3.core.sequence.loader">SequenceFileProxyLoader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/StringProxySequenceReader.html#toString()">toString()</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/StringProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader">StringProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html#toString()">toString()</a></span> - Method in class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/SimplePoint.html#toString()">toString()</a></span> - Method in class org.biojava3.core.sequence.location.<a href="./org/biojava3/core/sequence/location/SimplePoint.html" title="class in org.biojava3.core.sequence.location">SimplePoint</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html#toString()">toString()</a></span> - Method in class org.biojava3.core.sequence.location.template.<a href="./org/biojava3/core/sequence/location/template/AbstractLocation.html" title="class in org.biojava3.core.sequence.location.template">AbstractLocation</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/MultipleSequenceAlignment.html#toString(int)">toString(int)</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/MultipleSequenceAlignment.html" title="class in org.biojava3.core.sequence">MultipleSequenceAlignment</a></dt>
<dd>
<div class="block">Get a string representation of the MSA with a fixed width</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/MultipleSequenceAlignment.html#toString(org.biojava3.core.sequence.template.LightweightProfile.StringFormat)">toString(LightweightProfile.StringFormat)</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/MultipleSequenceAlignment.html" title="class in org.biojava3.core.sequence">MultipleSequenceAlignment</a></dt>
<dd>
<div class="block">Support for different MSA formats</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/MultipleSequenceAlignment.html#toString()">toString()</a></span> - Method in class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/MultipleSequenceAlignment.html" title="class in org.biojava3.core.sequence">MultipleSequenceAlignment</a></dt>
<dd>
<div class="block">String representation of the MSA</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompound.html#toString()">toString()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompound.html" title="class in org.biojava3.core.sequence.template">AbstractCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.html#toString()">toString()</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/LightweightProfile.html#toString()">toString()</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/LightweightProfile.html" title="interface in org.biojava3.core.sequence.template">LightweightProfile</a></dt>
<dd>
<div class="block">Returns a simple view of the alignment profile.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/LightweightProfile.html#toString(int)">toString(int)</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/LightweightProfile.html" title="interface in org.biojava3.core.sequence.template">LightweightProfile</a></dt>
<dd>
<div class="block">Returns a formatted view of the alignment profile.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/LightweightProfile.html#toString(org.biojava3.core.sequence.template.LightweightProfile.StringFormat)">toString(LightweightProfile.StringFormat)</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/LightweightProfile.html" title="interface in org.biojava3.core.sequence.template">LightweightProfile</a></dt>
<dd>
<div class="block">Returns a formatted view of the alignment profile.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceMixin.html#toString(org.biojava3.core.sequence.template.Sequence)">toString(Sequence&lt;C&gt;)</a></span> - Static method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceMixin.html" title="class in org.biojava3.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">Shortcut to <a href="./org/biojava3/core/sequence/template/SequenceMixin.html#toStringBuilder(org.biojava3.core.sequence.template.Sequence)"><code>SequenceMixin.toStringBuilder(org.biojava3.core.sequence.template.Sequence)</code></a>
 which calls toString() on the resulting object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/CaseInsensitiveCompound.html#toString()">toString()</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/CaseInsensitiveCompound.html" title="class in org.biojava3.core.sequence.transcription">CaseInsensitiveCompound</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/Table.CaseInsensitiveTriplet.html#toString()">toString()</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/Table.CaseInsensitiveTriplet.html" title="class in org.biojava3.core.sequence.transcription">Table.CaseInsensitiveTriplet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/Table.Codon.html#toString()">toString()</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/Table.Codon.html" title="class in org.biojava3.core.sequence.transcription">Table.Codon</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/CRC64Checksum.html#toString()">toString()</a></span> - Method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/CRC64Checksum.html" title="class in org.biojava3.core.util">CRC64Checksum</a></dt>
<dd>
<div class="block">Returns a zero-padded 16 character wide string containing the current
 value of this checksum in uppercase hexadecimal format.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/data/sequence/FastaSequence.html#toString()">toString()</a></span> - Method in class org.biojava3.data.sequence.<a href="./org/biojava3/data/sequence/FastaSequence.html" title="class in org.biojava3.data.sequence">FastaSequence</a></dt>
<dd>
<div class="block">Same as oneLineFasta</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Feature.html#toString()">toString()</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Feature.html" title="class in org.biojava3.genome.parsers.gff">Feature</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/FeatureI.html#toString()">toString()</a></span> - Method in interface org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/FeatureI.html" title="interface in org.biojava3.genome.parsers.gff">FeatureI</a></dt>
<dd>
<div class="block">A string representation of the feature.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/FeatureList.html#toString()">toString()</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/FeatureList.html" title="class in org.biojava3.genome.parsers.gff">FeatureList</a></dt>
<dd>
<div class="block">Return a string representation of all features in this list.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Location.html#toString()">toString()</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Location.html" title="class in org.biojava3.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Return a string representation of location.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/LocIterator.html#toString()">toString()</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/LocIterator.html" title="class in org.biojava3.genome.parsers.gff">LocIterator</a></dt>
<dd>
<div class="block">Get string representation of iterator.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/Component.html#toString()">toString()</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/Component.html" title="class in org.biojava3.protmod">Component</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ModificationCategory.html#toString()">toString()</a></span> - Method in enum org.biojava3.protmod.<a href="./org/biojava3/protmod/ModificationCategory.html" title="enum in org.biojava3.protmod">ModificationCategory</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ModificationConditionImpl.html#toString()">toString()</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ModificationConditionImpl.html" title="class in org.biojava3.protmod">ModificationConditionImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ModificationLinkage.html#toString()">toString()</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ModificationLinkage.html" title="class in org.biojava3.protmod">ModificationLinkage</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ModificationOccurrenceType.html#toString()">toString()</a></span> - Method in enum org.biojava3.protmod.<a href="./org/biojava3/protmod/ModificationOccurrenceType.html" title="enum in org.biojava3.protmod">ModificationOccurrenceType</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationImpl.html#toString()">toString()</a></span> - Method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationImpl.html" title="class in org.biojava3.protmod">ProteinModificationImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/ModifiedCompoundImpl.html#toString()">toString()</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/ModifiedCompoundImpl.html" title="class in org.biojava3.protmod.structure">ModifiedCompoundImpl</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/StructureAtom.html#toString()">toString()</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/StructureAtom.html" title="class in org.biojava3.protmod.structure">StructureAtom</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/StructureAtomLinkage.html#toString()">toString()</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/StructureAtomLinkage.html" title="class in org.biojava3.protmod.structure">StructureAtomLinkage</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/structure/StructureGroup.html#toString()">toString()</a></span> - Method in class org.biojava3.protmod.structure.<a href="./org/biojava3/protmod/structure/StructureGroup.html" title="class in org.biojava3.protmod.structure">StructureGroup</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/ronn/Jronn.Range.html#toString()">toString()</a></span> - Method in class org.biojava3.ronn.<a href="./org/biojava3/ronn/Jronn.Range.html" title="class in org.biojava3.ronn">Jronn.Range</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/ronn/ModelLoader.Model.html#toString()">toString()</a></span> - Method in class org.biojava3.ronn.<a href="./org/biojava3/ronn/ModelLoader.Model.html" title="class in org.biojava3.ronn">ModelLoader.Model</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceMixin.html#toStringBuilder(org.biojava3.core.sequence.template.Sequence)">toStringBuilder(Sequence&lt;C&gt;)</a></span> - Static method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceMixin.html" title="class in org.biojava3.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">For the given Sequence this will return a <a href="http://java.sun.com/j2se/1.6.0/docs/api/java/lang/StringBuilder.html?is-external=true" title="class or interface in java.lang"><code>StringBuilder</code></a> object
 filled with the results of <code>Compound#toString()</code>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html#toWebSiteDisplay(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">toWebSiteDisplay(AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html" title="class in org.biojava.bio.structure.align.model">AfpChainWriter</a></dt>
<dd>
<div class="block">Prints the afpChain as a nicely formatted alignment, including alignment
 statistics, the aligned sequences themselves, and information about the 
 superposition.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html#toWebSiteDisplay(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[], boolean)">toWebSiteDisplay(AFPChain, Atom[], Atom[], boolean)</a></span> - Static method in class org.biojava.bio.structure.align.model.<a href="./org/biojava/bio/structure/align/model/AfpChainWriter.html" title="class in org.biojava.bio.structure.align.model">AfpChainWriter</a></dt>
<dd>
<div class="block">Prints the afpChain as a nicely formatted alignment, including alignment
 statistics, the aligned sequences themselves, and information about the 
 superposition.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/UserConfiguration.html#toXML(java.io.PrintWriter)">toXML(PrintWriter)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/UserConfiguration.html" title="class in org.biojava.bio.structure.align.util">UserConfiguration</a></dt>
<dd>
<div class="block">convert Configuration to an XML file so it can be serialized</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/UserConfiguration.html#toXML(org.biojava3.core.util.XMLWriter)">toXML(XMLWriter)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/UserConfiguration.html" title="class in org.biojava.bio.structure.align.util">UserConfiguration</a></dt>
<dd>
<div class="block">convert Configuration to an XML file so it can be serialized
 add to an already existing xml file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/AFPChainXMLConverter.html#toXML(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">toXML(AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/AFPChainXMLConverter.html" title="class in org.biojava.bio.structure.align.xml">AFPChainXMLConverter</a></dt>
<dd>
<div class="block">Convert an afpChain to a simple XML representation</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/AFPChainXMLConverter.html#toXML(org.biojava.bio.structure.align.model.AFPChain, java.io.StringWriter, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">toXML(AFPChain, StringWriter, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/AFPChainXMLConverter.html" title="class in org.biojava.bio.structure.align.xml">AFPChainXMLConverter</a></dt>
<dd>
<div class="block">Write the XML representation to a StringWriter</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/AFPChainXMLConverter.html#toXML(org.biojava.bio.structure.align.model.AFPChain)">toXML(AFPChain)</a></span> - Static method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/AFPChainXMLConverter.html" title="class in org.biojava.bio.structure.align.xml">AFPChainXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/HasResultXMLConverter.html#toXML(boolean)">toXML(boolean)</a></span> - Method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/HasResultXMLConverter.html" title="class in org.biojava.bio.structure.align.xml">HasResultXMLConverter</a></dt>
<dd>
<div class="block">return flag if the server has a result</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/PositionInQueueXMLConverter.html#toXML(int)">toXML(int)</a></span> - Method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/PositionInQueueXMLConverter.html" title="class in org.biojava.bio.structure.align.xml">PositionInQueueXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/xml/RepresentativeXMLConverter.html#toXML(java.util.SortedSet)">toXML(SortedSet&lt;String&gt;)</a></span> - Static method in class org.biojava.bio.structure.align.xml.<a href="./org/biojava/bio/structure/align/xml/RepresentativeXMLConverter.html" title="class in org.biojava.bio.structure.align.xml">RepresentativeXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/ScopDescriptions.html#toXML()">toXML()</a></span> - Method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/ScopDescriptions.html" title="class in org.biojava.bio.structure.scop.server">ScopDescriptions</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/ScopDomains.html#toXML()">toXML()</a></span> - Method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/ScopDomains.html" title="class in org.biojava.bio.structure.scop.server">ScopDomains</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/ScopNodes.html#toXML()">toXML()</a></span> - Method in class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/ScopNodes.html" title="class in org.biojava.bio.structure.scop.server">ScopNodes</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/io/ComponentXMLConverter.html#toXML(org.biojava3.protmod.Component)">toXML(Component)</a></span> - Static method in class org.biojava3.protmod.io.<a href="./org/biojava3/protmod/io/ComponentXMLConverter.html" title="class in org.biojava3.protmod.io">ComponentXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/io/ComponentXMLConverter.html#toXML(org.biojava3.protmod.Component, org.biojava3.core.util.PrettyXMLWriter)">toXML(Component, PrettyXMLWriter)</a></span> - Static method in class org.biojava3.protmod.io.<a href="./org/biojava3/protmod/io/ComponentXMLConverter.html" title="class in org.biojava3.protmod.io">ComponentXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/io/ModifiedCompoundXMLConverter.html#toXML(org.biojava3.protmod.structure.ModifiedCompound)">toXML(ModifiedCompound)</a></span> - Static method in class org.biojava3.protmod.io.<a href="./org/biojava3/protmod/io/ModifiedCompoundXMLConverter.html" title="class in org.biojava3.protmod.io">ModifiedCompoundXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/io/StructureAtomXMLConverter.html#toXML(org.biojava3.protmod.structure.StructureAtom)">toXML(StructureAtom)</a></span> - Static method in class org.biojava3.protmod.io.<a href="./org/biojava3/protmod/io/StructureAtomXMLConverter.html" title="class in org.biojava3.protmod.io">StructureAtomXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/io/StructureAtomXMLConverter.html#toXML(org.biojava3.protmod.structure.StructureAtom, org.biojava3.core.util.PrettyXMLWriter)">toXML(StructureAtom, PrettyXMLWriter)</a></span> - Static method in class org.biojava3.protmod.io.<a href="./org/biojava3/protmod/io/StructureAtomXMLConverter.html" title="class in org.biojava3.protmod.io">StructureAtomXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/io/StructureGroupXMLConverter.html#toXML(org.biojava3.protmod.structure.StructureGroup, org.biojava3.core.util.PrettyXMLWriter)">toXML(StructureGroup, PrettyXMLWriter)</a></span> - Static method in class org.biojava3.protmod.io.<a href="./org/biojava3/protmod/io/StructureGroupXMLConverter.html" title="class in org.biojava3.protmod.io">StructureGroupXMLConverter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#trace()">trace()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Matrix trace.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CECalculator.html#traceFragmentMatrix(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">traceFragmentMatrix(AFPChain, Atom[], Atom[])</a></span> - Method in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CECalculator.html" title="class in org.biojava.bio.structure.align.ce">CECalculator</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.html" title="class in org.biojava3.core.sequence.transcription"><span class="strong">TranscriptionEngine</span></a> - Class in <a href="./org/biojava3/core/sequence/transcription/package-summary.html">org.biojava3.core.sequence.transcription</a></dt>
<dd>
<div class="block">Used as a way of encapsulating the data structures required to parse
 DNA to a Protein sequence.</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava3.core.sequence.transcription"><span class="strong">TranscriptionEngine.Builder</span></a> - Class in <a href="./org/biojava3/core/sequence/transcription/package-summary.html">org.biojava3.core.sequence.transcription</a></dt>
<dd>
<div class="block">This class is the way to create a <code>TranslationEngine</code>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.Builder.html#TranscriptionEngine.Builder()">TranscriptionEngine.Builder()</a></span> - Constructor for class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava3.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/TranscriptSequence.html" title="class in org.biojava3.core.sequence"><span class="strong">TranscriptSequence</span></a> - Class in <a href="./org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a></dt>
<dd>
<div class="block">This is the sequence if you want to go from a gene sequence to a protein sequence.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/TranscriptSequence.html#TranscriptSequence(org.biojava3.core.sequence.GeneSequence, int, int)">TranscriptSequence(GeneSequence, int, int)</a></span> - Constructor for class org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/TranscriptSequence.html" title="class in org.biojava3.core.sequence">TranscriptSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/ContinuousColorMapperTransform.html#transform(double)">transform(double)</a></span> - Method in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/ContinuousColorMapperTransform.html" title="class in org.biojava.bio.structure.gui.util.color">ContinuousColorMapperTransform</a></dt>
<dd>
<div class="block">An arbitrary transform over reals</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/LogColorMapper.html#transform(double)">transform(double)</a></span> - Method in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/LogColorMapper.html" title="class in org.biojava.bio.structure.gui.util.color">LogColorMapper</a></dt>
<dd>
<div class="block">Apply log transform.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/SqrtColorMapper.html#transform(double)">transform(double)</a></span> - Method in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/SqrtColorMapper.html" title="class in org.biojava.bio.structure.gui.util.color">SqrtColorMapper</a></dt>
<dd>
<div class="block">Return sqrt(value).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompoundTranslator.html#translate(F)">translate(F)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompoundTranslator.html" title="class in org.biojava3.core.sequence.template">AbstractCompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/CompoundTranslator.html#translate(F)">translate(F)</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/CompoundTranslator.html" title="interface in org.biojava3.core.sequence.template">CompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.html#translate(org.biojava3.core.sequence.template.Sequence)">translate(Sequence&lt;NucleotideCompound&gt;)</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.html" title="class in org.biojava3.core.sequence.transcription">TranscriptionEngine</a></dt>
<dd>
<div class="block">Quick method to let you go from a CDS to a Peptide quickly.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompoundTranslator.html#translateMany(F)">translateMany(F)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompoundTranslator.html" title="class in org.biojava3.core.sequence.template">AbstractCompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/CompoundTranslator.html#translateMany(F)">translateMany(F)</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/CompoundTranslator.html" title="interface in org.biojava3.core.sequence.template">CompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/RNAToAminoAcidTranslator.html#translateNCodons()">translateNCodons()</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/RNAToAminoAcidTranslator.html" title="class in org.biojava3.core.sequence.transcription">RNAToAminoAcidTranslator</a></dt>
<dd>
<div class="block">Indicates if we want to force exact translation of compounds or not i.e.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.Builder.html#translateNCodons(boolean)">translateNCodons(boolean)</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava3.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/exceptions/TranslationException.html" title="class in org.biojava3.core.exceptions"><span class="strong">TranslationException</span></a> - Exception in <a href="./org/biojava3/core/exceptions/package-summary.html">org.biojava3.core.exceptions</a></dt>
<dd>
<div class="block">Thrown from AbstractCompundTranslator</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/exceptions/TranslationException.html#TranslationException(java.lang.String)">TranslationException(String)</a></span> - Constructor for exception org.biojava3.core.exceptions.<a href="./org/biojava3/core/exceptions/TranslationException.html" title="class in org.biojava3.core.exceptions">TranslationException</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/exceptions/TranslationException.html#TranslationException(java.lang.Exception)">TranslationException(Exception)</a></span> - Constructor for exception org.biojava3.core.exceptions.<a href="./org/biojava3/core/exceptions/TranslationException.html" title="class in org.biojava3.core.exceptions">TranslationException</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/exceptions/TranslationException.html#TranslationException(java.lang.String, java.lang.Exception)">TranslationException(String, Exception)</a></span> - Constructor for exception org.biojava3.core.exceptions.<a href="./org/biojava3/core/exceptions/TranslationException.html" title="class in org.biojava3.core.exceptions">TranslationException</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#transpose()">transpose()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Matrix transpose.</div>
</dd>
<dt><a href="./org/biojava3/phylo/TreeConstructionAlgorithm.html" title="enum in org.biojava3.phylo"><span class="strong">TreeConstructionAlgorithm</span></a> - Enum in <a href="./org/biojava3/phylo/package-summary.html">org.biojava3.phylo</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/phylo/TreeConstructor.html" title="class in org.biojava3.phylo"><span class="strong">TreeConstructor</span></a>&lt;<a href="./org/biojava3/phylo/TreeConstructor.html" title="type parameter in TreeConstructor">C</a> extends <a href="./org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a>&lt;<a href="./org/biojava3/phylo/TreeConstructor.html" title="type parameter in TreeConstructor">D</a>&gt;,<a href="./org/biojava3/phylo/TreeConstructor.html" title="type parameter in TreeConstructor">D</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/phylo/package-summary.html">org.biojava3.phylo</a></dt>
<dd>
<div class="block">Tree constructor uses the forrester tree library to build phylogenetic trees using neighbor joining algorithm.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/TreeConstructor.html#TreeConstructor(org.biojava3.core.sequence.MultipleSequenceAlignment, org.biojava3.phylo.TreeType, org.biojava3.phylo.TreeConstructionAlgorithm, org.biojava3.phylo.NJTreeProgressListener)">TreeConstructor(MultipleSequenceAlignment&lt;C, D&gt;, TreeType, TreeConstructionAlgorithm, NJTreeProgressListener)</a></span> - Constructor for class org.biojava3.phylo.<a href="./org/biojava3/phylo/TreeConstructor.html" title="class in org.biojava3.phylo">TreeConstructor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/TreeConstructor.html#TreeConstructor(org.forester.phylogenyinference.DistanceMatrix, org.biojava3.phylo.TreeType, org.biojava3.phylo.TreeConstructionAlgorithm, org.biojava3.phylo.NJTreeProgressListener)">TreeConstructor(DistanceMatrix, TreeType, TreeConstructionAlgorithm, NJTreeProgressListener)</a></span> - Constructor for class org.biojava3.phylo.<a href="./org/biojava3/phylo/TreeConstructor.html" title="class in org.biojava3.phylo">TreeConstructor</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/phylo/TreeType.html" title="enum in org.biojava3.phylo"><span class="strong">TreeType</span></a> - Enum in <a href="./org/biojava3/phylo/package-summary.html">org.biojava3.phylo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#triggerDocumentEnd()">triggerDocumentEnd()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#triggerDocumentStart()">triggerDocumentStart()</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#triggerGeneric(java.lang.String, java.util.List, java.util.List)">triggerGeneric(String, List&lt;String&gt;, List&lt;String&gt;)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/SequenceMouseListener.html#triggerMouseOverPosition(int, int)">triggerMouseOverPosition(int, int)</a></span> - Method in class org.biojava.bio.structure.gui.util.<a href="./org/biojava/bio/structure/gui/util/SequenceMouseListener.html" title="class in org.biojava.bio.structure.gui.util">SequenceMouseListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#triggerNewChemComp(org.biojava.bio.structure.io.mmcif.model.ChemComp)">triggerNewChemComp(ChemComp)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#triggerNewChemCompDescriptor(org.biojava.bio.structure.io.mmcif.model.ChemCompDescriptor)">triggerNewChemCompDescriptor(ChemCompDescriptor)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#triggerNewEntity(org.biojava.bio.structure.io.mmcif.model.Entity)">triggerNewEntity(Entity)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#triggerNewEntityPolySeq(org.biojava.bio.structure.io.mmcif.model.EntityPolySeq)">triggerNewEntityPolySeq(EntityPolySeq)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#triggerNewPdbxEntityNonPoly(org.biojava.bio.structure.io.mmcif.model.PdbxEntityNonPoly)">triggerNewPdbxEntityNonPoly(PdbxEntityNonPoly)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#triggerNewRefine(org.biojava.bio.structure.io.mmcif.model.Refine)">triggerNewRefine(Refine)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#triggerNewStructAsym(org.biojava.bio.structure.io.mmcif.model.StructAsym)">triggerNewStructAsym(StructAsym)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html#triggerNewStructKeywords(org.biojava.bio.structure.io.mmcif.model.StructKeywords)">triggerNewStructKeywords(StructKeywords)</a></span> - Method in class org.biojava.bio.structure.io.mmcif.<a href="./org/biojava/bio/structure/io/mmcif/SimpleMMcifParser.html" title="class in org.biojava.bio.structure.io.mmcif">SimpleMMcifParser</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanelMouseMotionListener.html#triggerSelectionLocked(boolean)">triggerSelectionLocked(boolean)</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanelMouseMotionListener.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AligPanelMouseMotionListener</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/RNAToAminoAcidTranslator.html#trimStop(java.util.List)">trimStop(List&lt;AminoAcidCompound&gt;)</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/RNAToAminoAcidTranslator.html" title="class in org.biojava3.core.sequence.transcription">RNAToAminoAcidTranslator</a></dt>
<dd>
<div class="block">Imperfect code.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.Builder.html#trimStop(boolean)">trimStop(boolean)</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/TranscriptionEngine.Builder.html" title="class in org.biojava3.core.sequence.transcription">TranscriptionEngine.Builder</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#Trp">Trp</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/PDBFileParser.html#TURN">TURN</a></span> - Static variable in class org.biojava.bio.structure.io.<a href="./org/biojava/bio/structure/io/PDBFileParser.html" title="class in org.biojava.bio.structure.io">PDBFileParser</a></dt>
<dd>
<div class="block">Turn secondary structure assignment.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#turn">turn</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#turnmax">turnmax</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#turnmin">turnmin</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/AFPTwister.html#twistOptimized(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">twistOptimized(AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/AFPTwister.html" title="class in org.biojava.bio.structure.align">AFPTwister</a></dt>
<dd>
<div class="block">superimposing according to the optimized alignment</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/AFPTwister.html#twistPDB(org.biojava.bio.structure.align.model.AFPChain, org.biojava.bio.structure.Atom[], org.biojava.bio.structure.Atom[])">twistPDB(AFPChain, Atom[], Atom[])</a></span> - Static method in class org.biojava.bio.structure.align.<a href="./org/biojava/bio/structure/align/AFPTwister.html" title="class in org.biojava.bio.structure.align">AFPTwister</a></dt>
<dd>
<div class="block">calculate the total rmsd of the blocks
 output a merged pdb file for both proteins                                                                                                                                       
     protein 1, in chain A                                                                                                                                                            
    protein 2 is twisted according to the twists detected, in chain B</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/storage/TwoBitSequenceReader.html" title="class in org.biojava3.core.sequence.storage"><span class="strong">TwoBitSequenceReader</span></a>&lt;<a href="./org/biojava3/core/sequence/storage/TwoBitSequenceReader.html" title="type parameter in TwoBitSequenceReader">C</a> extends <a href="./org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt; - Class in <a href="./org/biojava3/core/sequence/storage/package-summary.html">org.biojava3.core.sequence.storage</a></dt>
<dd>
<div class="block">Implementation of the 2bit encoding.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/TwoBitSequenceReader.html#TwoBitSequenceReader(org.biojava3.core.sequence.template.Sequence)">TwoBitSequenceReader(Sequence&lt;C&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/TwoBitSequenceReader.html" title="class in org.biojava3.core.sequence.storage">TwoBitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/TwoBitSequenceReader.html#TwoBitSequenceReader(java.lang.String, org.biojava3.core.sequence.template.CompoundSet)">TwoBitSequenceReader(String, CompoundSet&lt;C&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/TwoBitSequenceReader.html" title="class in org.biojava3.core.sequence.storage">TwoBitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/TwoBitSequenceReader.html#TwoBitSequenceReader(java.lang.String, org.biojava3.core.sequence.template.CompoundSet, org.biojava3.core.sequence.AccessionID)">TwoBitSequenceReader(String, CompoundSet&lt;C&gt;, AccessionID)</a></span> - Constructor for class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/TwoBitSequenceReader.html" title="class in org.biojava3.core.sequence.storage">TwoBitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/TwoBitSequenceReader.html#TwoBitSequenceReader(org.biojava3.core.sequence.storage.TwoBitSequenceReader.TwoBitArrayWorker)">TwoBitSequenceReader(TwoBitSequenceReader.TwoBitArrayWorker&lt;C&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/TwoBitSequenceReader.html" title="class in org.biojava3.core.sequence.storage">TwoBitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/TwoBitSequenceReader.html#TwoBitSequenceReader(org.biojava3.core.sequence.storage.TwoBitSequenceReader.TwoBitArrayWorker, org.biojava3.core.sequence.AccessionID)">TwoBitSequenceReader(TwoBitSequenceReader.TwoBitArrayWorker&lt;C&gt;, AccessionID)</a></span> - Constructor for class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/TwoBitSequenceReader.html" title="class in org.biojava3.core.sequence.storage">TwoBitSequenceReader</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html" title="class in org.biojava3.core.sequence.storage"><span class="strong">TwoBitSequenceReader.TwoBitArrayWorker</span></a>&lt;<a href="./org/biojava3/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html" title="type parameter in TwoBitSequenceReader.TwoBitArrayWorker">C</a> extends <a href="./org/biojava3/core/sequence/compound/NucleotideCompound.html" title="class in org.biojava3.core.sequence.compound">NucleotideCompound</a>&gt; - Class in <a href="./org/biojava3/core/sequence/storage/package-summary.html">org.biojava3.core.sequence.storage</a></dt>
<dd>
<div class="block">Extension of the BitArrayWorker which provides the 2bit implementation
 code.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html#TwoBitSequenceReader.TwoBitArrayWorker(org.biojava3.core.sequence.template.CompoundSet, int)">TwoBitSequenceReader.TwoBitArrayWorker(CompoundSet&lt;C&gt;, int)</a></span> - Constructor for class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html" title="class in org.biojava3.core.sequence.storage">TwoBitSequenceReader.TwoBitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html#TwoBitSequenceReader.TwoBitArrayWorker(org.biojava3.core.sequence.template.CompoundSet, int[])">TwoBitSequenceReader.TwoBitArrayWorker(CompoundSet&lt;C&gt;, int[])</a></span> - Constructor for class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html" title="class in org.biojava3.core.sequence.storage">TwoBitSequenceReader.TwoBitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html#TwoBitSequenceReader.TwoBitArrayWorker(org.biojava3.core.sequence.template.Sequence)">TwoBitSequenceReader.TwoBitArrayWorker(Sequence&lt;C&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html" title="class in org.biojava3.core.sequence.storage">TwoBitSequenceReader.TwoBitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html#TwoBitSequenceReader.TwoBitArrayWorker(java.lang.String, org.biojava3.core.sequence.template.CompoundSet)">TwoBitSequenceReader.TwoBitArrayWorker(String, CompoundSet&lt;C&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.storage.<a href="./org/biojava3/core/sequence/storage/TwoBitSequenceReader.TwoBitArrayWorker.html" title="class in org.biojava3.core.sequence.storage">TwoBitSequenceReader.TwoBitArrayWorker</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/AminoAcidImpl.html#type">type</a></span> - Static variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/AminoAcidImpl.html" title="class in org.biojava.bio.structure">AminoAcidImpl</a></dt>
<dd>
<div class="block">this is an Amino acid. type is "amino".</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/HetatomImpl.html#type">type</a></span> - Static variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/HetatomImpl.html" title="class in org.biojava.bio.structure">HetatomImpl</a></dt>
<dd>
<div class="block">this is a "hetatm".</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/NucleotideImpl.html#type">type</a></span> - Static variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/NucleotideImpl.html" title="class in org.biojava.bio.structure">NucleotideImpl</a></dt>
<dd>
<div class="block">this is a "nucleotide", a special occurance of a Hetatom.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Feature.html#type()">type()</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Feature.html" title="class in org.biojava3.genome.parsers.gff">Feature</a></dt>
<dd>
<div class="block">Get feature type, such as "exon" or "CDS".</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/FeatureI.html#type()">type()</a></span> - Method in interface org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/FeatureI.html" title="interface in org.biojava3.genome.parsers.gff">FeatureI</a></dt>
<dd>
<div class="block">Get the feature type, for example, "exon", "CDS", etc.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#Tyr">Tyr</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_U_">
<!--   -->
</a>
<h2 class="title">U</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava/bio/structure/jama/Matrix.html#uminus()">uminus()</a></span> - Method in class org.biojava.bio.structure.jama.<a href="./org/biojava/bio/structure/jama/Matrix.html" title="class in org.biojava.bio.structure.jama">Matrix</a></dt>
<dd>
<div class="block">Unary minus</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/UncompressInputStream.html#uncompress(java.lang.String, java.io.FileOutputStream)">uncompress(String, FileOutputStream)</a></span> - Static method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/UncompressInputStream.html" title="class in org.biojava3.core.util">UncompressInputStream</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/util/UncompressInputStream.html" title="class in org.biojava3.core.util"><span class="strong">UncompressInputStream</span></a> - Class in <a href="./org/biojava3/core/util/package-summary.html">org.biojava3.core.util</a></dt>
<dd>
<div class="block">This class decompresses an input stream containing data compressed with
 the unix "compress" utility (LZC, a LZW variant).</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/UncompressInputStream.html#UncompressInputStream(java.io.InputStream)">UncompressInputStream(InputStream)</a></span> - Constructor for class org.biojava3.core.util.<a href="./org/biojava3/core/util/UncompressInputStream.html" title="class in org.biojava3.core.util">UncompressInputStream</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/AtomCache.html#UNDERSCORE">UNDERSCORE</a></span> - Static variable in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Location.html#union(org.biojava3.genome.parsers.gff.Location)">union(Location)</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Location.html" title="class in org.biojava3.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Return the union.</div>
</dd>
<dt><a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader"><span class="strong">UniprotProxySequenceReader</span></a>&lt;<a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html" title="type parameter in UniprotProxySequenceReader">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/core/sequence/loader/package-summary.html">org.biojava3.core.sequence.loader</a></dt>
<dd>
<div class="block">Pass in a Uniprot ID and this ProxySequenceReader when passed to a ProteinSequence will get the sequence data and other data elements
 associated with the ProteinSequence by Uniprot.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html#UniprotProxySequenceReader(java.lang.String, org.biojava3.core.sequence.template.CompoundSet)">UniprotProxySequenceReader(String, CompoundSet&lt;C&gt;)</a></span> - Constructor for class org.biojava3.core.sequence.loader.<a href="./org/biojava3/core/sequence/loader/UniprotProxySequenceReader.html" title="class in org.biojava3.core.sequence.loader">UniprotProxySequenceReader</a></dt>
<dd>
<div class="block">The uniprot id is used to retrieve the uniprot XML which is then parsed as a DOM object
 so we know everything about the protein.</div>
</dd>
<dt><a href="./org/biojava3/genome/uniprot/UniprotToFasta.html" title="class in org.biojava3.genome.uniprot"><span class="strong">UniprotToFasta</span></a> - Class in <a href="./org/biojava3/genome/uniprot/package-summary.html">org.biojava3.genome.uniprot</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/uniprot/UniprotToFasta.html#UniprotToFasta()">UniprotToFasta()</a></span> - Constructor for class org.biojava3.genome.uniprot.<a href="./org/biojava3/genome/uniprot/UniprotToFasta.html" title="class in org.biojava3.genome.uniprot">UniprotToFasta</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#unitVector(org.biojava.bio.structure.Atom)">unitVector(Atom)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">return the unit vector of vector a .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/StructureTools.html#UNKNOWN_GROUP_LABEL">UNKNOWN_GROUP_LABEL</a></span> - Static variable in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/StructureTools.html" title="class in org.biojava.bio.structure">StructureTools</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/convertor/Convertor.html#unknownGroup">unknownGroup</a></span> - Static variable in class org.biojava3.aaproperties.profeat.convertor.<a href="./org/biojava3/aaproperties/profeat/convertor/Convertor.html" title="class in org.biojava3.aaproperties.profeat.convertor">Convertor</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/UnknownPdbAminoAcidException.html" title="class in org.biojava.bio.structure"><span class="strong">UnknownPdbAminoAcidException</span></a> - Exception in <a href="./org/biojava/bio/structure/package-summary.html">org.biojava.bio.structure</a></dt>
<dd>
<div class="block">An exception for use during translating amino acids in a PDB file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/UnknownPdbAminoAcidException.html#UnknownPdbAminoAcidException(java.lang.String)">UnknownPdbAminoAcidException(String)</a></span> - Constructor for exception org.biojava.bio.structure.<a href="./org/biojava/bio/structure/UnknownPdbAminoAcidException.html" title="class in org.biojava.bio.structure">UnknownPdbAminoAcidException</a></dt>
<dd>
<div class="block">Constructs a PDBParseException object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/UnknownPdbAminoAcidException.html#UnknownPdbAminoAcidException(java.lang.String, java.lang.Throwable)">UnknownPdbAminoAcidException(String, Throwable)</a></span> - Constructor for exception org.biojava.bio.structure.<a href="./org/biojava/bio/structure/UnknownPdbAminoAcidException.html" title="class in org.biojava.bio.structure">UnknownPdbAminoAcidException</a></dt>
<dd>
<div class="block">Constructs a UnknownPdbAminoAcidException object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/UnknownPdbAminoAcidException.html#UnknownPdbAminoAcidException(java.lang.Throwable)">UnknownPdbAminoAcidException(Throwable)</a></span> - Constructor for exception org.biojava.bio.structure.<a href="./org/biojava/bio/structure/UnknownPdbAminoAcidException.html" title="class in org.biojava.bio.structure">UnknownPdbAminoAcidException</a></dt>
<dd>
<div class="block">Constructs a UnknownPdbAminoAcidException object.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ProteinModificationRegistry.html#unregister(org.biojava3.protmod.ProteinModification)">unregister(ProteinModification)</a></span> - Static method in class org.biojava3.protmod.<a href="./org/biojava3/protmod/ProteinModificationRegistry.html" title="class in org.biojava3.protmod">ProteinModificationRegistry</a></dt>
<dd>
<div class="block">Remove a modification from registry.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/routines/AlignerHelper.Cut.html#update(int, int[], org.biojava3.alignment.routines.AlignerHelper.Last[][])">update(int, int[], AlignerHelper.Last[][])</a></span> - Method in class org.biojava3.alignment.routines.<a href="./org/biojava3/alignment/routines/AlignerHelper.Cut.html" title="class in org.biojava3.alignment.routines">AlignerHelper.Cut</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/CRC64Checksum.html#update(int)">update(int)</a></span> - Method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/CRC64Checksum.html" title="class in org.biojava3.core.util">CRC64Checksum</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/CRC64Checksum.html#update(byte[], int, int)">update(byte[], int, int)</a></span> - Method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/CRC64Checksum.html" title="class in org.biojava3.core.util">CRC64Checksum</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/CRC64Checksum.html#update(java.lang.String)">update(String)</a></span> - Method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/CRC64Checksum.html" title="class in org.biojava3.core.util">CRC64Checksum</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/SequenceDisplay.html#updateDisplay()">updateDisplay()</a></span> - Method in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/SequenceDisplay.html" title="class in org.biojava.bio.structure.gui">SequenceDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/TreeConstructor.html#updateProgress(java.lang.String, int)">updateProgress(String, int)</a></span> - Method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/TreeConstructor.html" title="class in org.biojava3.phylo">TreeConstructor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/TreeConstructor.html#updateProgress(java.lang.String, int, int)">updateProgress(String, int, int)</a></span> - Method in class org.biojava3.phylo.<a href="./org/biojava3/phylo/TreeConstructor.html" title="class in org.biojava3.phylo">TreeConstructor</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/calc/AFPOptimizer.html#updateScore(org.biojava.bio.structure.align.fatcat.calc.FatCatParameters, org.biojava.bio.structure.align.model.AFPChain)">updateScore(FatCatParameters, AFPChain)</a></span> - Static method in class org.biojava.bio.structure.align.fatcat.calc.<a href="./org/biojava/bio/structure/align/fatcat/calc/AFPOptimizer.html" title="class in org.biojava.bio.structure.align.fatcat.calc">AFPOptimizer</a></dt>
<dd>
<div class="block">to update the chaining score after block delete and merge processed
 the blockScore value is important for significance evaluation</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Location.html#upstream(int)">upstream(int)</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Location.html" title="class in org.biojava3.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Return the adjacent location of specified length directly upstream of this location.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/SystemInfo.html#urlPropertyNames">urlPropertyNames</a></span> - Static variable in class org.biojava.bio.structure.align.gui.<a href="./org/biojava/bio/structure/align/gui/SystemInfo.html" title="class in org.biojava.bio.structure.align.gui">SystemInfo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/HTTPConnectionTools.html#USERAGENT">USERAGENT</a></span> - Static variable in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/HTTPConnectionTools.html" title="class in org.biojava.bio.structure.align.util">HTTPConnectionTools</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/ce/UserArgumentProcessor.html" title="interface in org.biojava.bio.structure.align.ce"><span class="strong">UserArgumentProcessor</span></a> - Interface in <a href="./org/biojava/bio/structure/align/ce/package-summary.html">org.biojava.bio.structure.align.ce</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/util/UserConfiguration.html" title="class in org.biojava.bio.structure.align.util"><span class="strong">UserConfiguration</span></a> - Class in <a href="./org/biojava/bio/structure/align/util/package-summary.html">org.biojava.bio.structure.align.util</a></dt>
<dd>
<div class="block">A container to persist config to the file system</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/UserConfiguration.html#UserConfiguration()">UserConfiguration()</a></span> - Constructor for class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/UserConfiguration.html" title="class in org.biojava.bio.structure.align.util">UserConfiguration</a></dt>
<dd>
<div class="block">Default UserConfiguration:
 
 split directory
 autofetch files
 default download location.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Feature.html#userData()">userData()</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Feature.html" title="class in org.biojava3.genome.parsers.gff">Feature</a></dt>
<dd>
<div class="block">Get HashMap of user data.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/FeatureI.html#userData()">userData()</a></span> - Method in interface org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/FeatureI.html" title="interface in org.biojava3.genome.parsers.gff">FeatureI</a></dt>
<dd>
<div class="block">Get HashMap of user data.</div>
</dd>
<dt><a href="./org/biojava3/aaproperties/Utils.html" title="class in org.biojava3.aaproperties"><span class="strong">Utils</span></a> - Class in <a href="./org/biojava3/aaproperties/package-summary.html">org.biojava3.aaproperties</a></dt>
<dd>
<div class="block">This is a utility class that contains utility methods which will facilitates the coding of other methods</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/Utils.html#Utils()">Utils()</a></span> - Constructor for class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/Utils.html" title="class in org.biojava3.aaproperties">Utils</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_V_">
<!--   -->
</a>
<h2 class="title">V</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/Constraints.html#V">V</a></span> - Static variable in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/Constraints.html" title="class in org.biojava3.aaproperties">Constraints</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#Val">Val</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/helper/GapArray.html#value">value</a></span> - Variable in class org.biojava.bio.structure.align.helper.<a href="./org/biojava/bio/structure/align/helper/GapArray.html" title="class in org.biojava.bio.structure.align.helper">GapArray</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/BondType.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/BondType.html" title="enum in org.biojava.bio.structure">BondType</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ElementType.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ElementType.html" title="enum in org.biojava.bio.structure">ElementType</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/LinearColorInterpolator.InterpolationDirection.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/LinearColorInterpolator.InterpolationDirection.html" title="enum in org.biojava.bio.structure.gui.util.color">LinearColorInterpolator.InterpolationDirection</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/chem/PolymerType.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava.bio.structure.io.mmcif.chem.<a href="./org/biojava/bio/structure/io/mmcif/chem/PolymerType.html" title="enum in org.biojava.bio.structure.io.mmcif.chem">PolymerType</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/chem/ResidueType.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava.bio.structure.io.mmcif.chem.<a href="./org/biojava/bio/structure/io/mmcif/chem/ResidueType.html" title="enum in org.biojava.bio.structure.io.mmcif.chem">ResidueType</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBStatus.Status.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBStatus.Status.html" title="enum in org.biojava.bio.structure">PDBStatus.Status</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopCategory.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopCategory.html" title="enum in org.biojava.bio.structure.scop">ScopCategory</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/CommandPrompt.PropertyName.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/CommandPrompt.PropertyName.html" title="enum in org.biojava3.aaproperties">CommandPrompt.PropertyName</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/PeptideProperties.SingleLetterAACode.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/PeptideProperties.SingleLetterAACode.html" title="enum in org.biojava3.aaproperties">PeptideProperties.SingleLetterAACode</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/IProfeatProperties.ATTRIBUTE.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/IProfeatProperties.ATTRIBUTE.html" title="enum in org.biojava3.aaproperties.profeat">IProfeatProperties.ATTRIBUTE</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/IProfeatProperties.DISTRIBUTION.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/IProfeatProperties.DISTRIBUTION.html" title="enum in org.biojava3.aaproperties.profeat">IProfeatProperties.DISTRIBUTION</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/IProfeatProperties.GROUPING.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/IProfeatProperties.GROUPING.html" title="enum in org.biojava3.aaproperties.profeat">IProfeatProperties.GROUPING</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/IProfeatProperties.TRANSITION.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/IProfeatProperties.TRANSITION.html" title="enum in org.biojava3.aaproperties.profeat">IProfeatProperties.TRANSITION</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/Alignments.PairInProfileScorerType.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava3.alignment.<a href="./org/biojava3/alignment/Alignments.PairInProfileScorerType.html" title="enum in org.biojava3.alignment">Alignments.PairInProfileScorerType</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/Alignments.PairwiseSequenceAlignerType.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava3.alignment.<a href="./org/biojava3/alignment/Alignments.PairwiseSequenceAlignerType.html" title="enum in org.biojava3.alignment">Alignments.PairwiseSequenceAlignerType</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/Alignments.PairwiseSequenceScorerType.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava3.alignment.<a href="./org/biojava3/alignment/Alignments.PairwiseSequenceScorerType.html" title="enum in org.biojava3.alignment">Alignments.PairwiseSequenceScorerType</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/Alignments.ProfileProfileAlignerType.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava3.alignment.<a href="./org/biojava3/alignment/Alignments.ProfileProfileAlignerType.html" title="enum in org.biojava3.alignment">Alignments.ProfileProfileAlignerType</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/Alignments.RefinerType.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava3.alignment.<a href="./org/biojava3/alignment/Alignments.RefinerType.html" title="enum in org.biojava3.alignment">Alignments.RefinerType</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/routines/AlignerHelper.Last.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava3.alignment.routines.<a href="./org/biojava3/alignment/routines/AlignerHelper.Last.html" title="enum in org.biojava3.alignment.routines">AlignerHelper.Last</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AlignedSequence.Step.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AlignedSequence.Step.html" title="enum in org.biojava3.alignment.template">AlignedSequence.Step</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/GapPenalty.Type.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/GapPenalty.Type.html" title="enum in org.biojava3.alignment.template">GapPenalty.Type</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/Profile.StringFormat.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/Profile.StringFormat.html" title="enum in org.biojava3.alignment.template">Profile.StringFormat</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/DataSource.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/DataSource.html" title="enum in org.biojava3.core.sequence">DataSource</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/DNASequence.DNAType.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/DNASequence.DNAType.html" title="enum in org.biojava3.core.sequence">DNASequence.DNAType</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/SequenceOptimizationHints.SequenceCollection.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/SequenceOptimizationHints.SequenceCollection.html" title="enum in org.biojava3.core.sequence">SequenceOptimizationHints.SequenceCollection</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/SequenceOptimizationHints.SequenceUsage.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/SequenceOptimizationHints.SequenceUsage.html" title="enum in org.biojava3.core.sequence">SequenceOptimizationHints.SequenceUsage</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/Strand.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/Strand.html" title="enum in org.biojava3.core.sequence">Strand</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.AnnotationType.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.AnnotationType.html" title="enum in org.biojava3.core.sequence.template">AbstractSequence.AnnotationType</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/LightweightProfile.StringFormat.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/LightweightProfile.StringFormat.html" title="enum in org.biojava3.core.sequence.template">LightweightProfile.StringFormat</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/Frame.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/Frame.html" title="enum in org.biojava3.core.sequence.transcription">Frame</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/TreeConstructionAlgorithm.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava3.phylo.<a href="./org/biojava3/phylo/TreeConstructionAlgorithm.html" title="enum in org.biojava3.phylo">TreeConstructionAlgorithm</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/TreeType.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava3.phylo.<a href="./org/biojava3/phylo/TreeType.html" title="enum in org.biojava3.phylo">TreeType</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ModificationCategory.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava3.protmod.<a href="./org/biojava3/protmod/ModificationCategory.html" title="enum in org.biojava3.protmod">ModificationCategory</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ModificationOccurrenceType.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava3.protmod.<a href="./org/biojava3/protmod/ModificationOccurrenceType.html" title="enum in org.biojava3.protmod">ModificationOccurrenceType</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ronn/RonnConstraint.Threshold.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava3.ronn.<a href="./org/biojava3/ronn/RonnConstraint.Threshold.html" title="enum in org.biojava3.ronn">RonnConstraint.Threshold</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/structure/gui/RenderStyle.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava3.structure.gui.<a href="./org/biojava3/structure/gui/RenderStyle.html" title="enum in org.biojava3.structure.gui">RenderStyle</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastOutputFormat.html#valueOf(java.lang.String)">valueOf(String)</a></span> - Static method in enum org.biojava3.ws.alignment.qblast.<a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastOutputFormat.html" title="enum in org.biojava3.ws.alignment.qblast">NCBIQBlastOutputFormat</a></dt>
<dd>
<div class="block">Returns the enum constant of this type with the specified name.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#valueOfIgnoreCase(java.lang.String)">valueOfIgnoreCase(String)</a></span> - Static method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns the Element that corresponds to the specified element symbol.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/BondType.html#values()">values()</a></span> - Static method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/BondType.html" title="enum in org.biojava.bio.structure">BondType</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Element.html#values()">values()</a></span> - Static method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Element.html" title="enum in org.biojava.bio.structure">Element</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/ElementType.html#values()">values()</a></span> - Static method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/ElementType.html" title="enum in org.biojava.bio.structure">ElementType</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html#values()">values()</a></span> - Method in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html" title="class in org.biojava.bio.structure.gui.util.color">GradientMapper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/LinearColorInterpolator.InterpolationDirection.html#values()">values()</a></span> - Static method in enum org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/LinearColorInterpolator.InterpolationDirection.html" title="enum in org.biojava.bio.structure.gui.util.color">LinearColorInterpolator.InterpolationDirection</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/chem/PolymerType.html#values()">values()</a></span> - Static method in enum org.biojava.bio.structure.io.mmcif.chem.<a href="./org/biojava/bio/structure/io/mmcif/chem/PolymerType.html" title="enum in org.biojava.bio.structure.io.mmcif.chem">PolymerType</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/io/mmcif/chem/ResidueType.html#values()">values()</a></span> - Static method in enum org.biojava.bio.structure.io.mmcif.chem.<a href="./org/biojava/bio/structure/io/mmcif/chem/ResidueType.html" title="enum in org.biojava.bio.structure.io.mmcif.chem">ResidueType</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/PDBStatus.Status.html#values()">values()</a></span> - Static method in enum org.biojava.bio.structure.<a href="./org/biojava/bio/structure/PDBStatus.Status.html" title="enum in org.biojava.bio.structure">PDBStatus.Status</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/ScopCategory.html#values()">values()</a></span> - Static method in enum org.biojava.bio.structure.scop.<a href="./org/biojava/bio/structure/scop/ScopCategory.html" title="enum in org.biojava.bio.structure.scop">ScopCategory</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/CommandPrompt.PropertyName.html#values()">values()</a></span> - Static method in enum org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/CommandPrompt.PropertyName.html" title="enum in org.biojava3.aaproperties">CommandPrompt.PropertyName</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/PeptideProperties.SingleLetterAACode.html#values()">values()</a></span> - Static method in enum org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/PeptideProperties.SingleLetterAACode.html" title="enum in org.biojava3.aaproperties">PeptideProperties.SingleLetterAACode</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/IProfeatProperties.ATTRIBUTE.html#values()">values()</a></span> - Static method in enum org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/IProfeatProperties.ATTRIBUTE.html" title="enum in org.biojava3.aaproperties.profeat">IProfeatProperties.ATTRIBUTE</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/IProfeatProperties.DISTRIBUTION.html#values()">values()</a></span> - Static method in enum org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/IProfeatProperties.DISTRIBUTION.html" title="enum in org.biojava3.aaproperties.profeat">IProfeatProperties.DISTRIBUTION</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/IProfeatProperties.GROUPING.html#values()">values()</a></span> - Static method in enum org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/IProfeatProperties.GROUPING.html" title="enum in org.biojava3.aaproperties.profeat">IProfeatProperties.GROUPING</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/profeat/IProfeatProperties.TRANSITION.html#values()">values()</a></span> - Static method in enum org.biojava3.aaproperties.profeat.<a href="./org/biojava3/aaproperties/profeat/IProfeatProperties.TRANSITION.html" title="enum in org.biojava3.aaproperties.profeat">IProfeatProperties.TRANSITION</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/Alignments.PairInProfileScorerType.html#values()">values()</a></span> - Static method in enum org.biojava3.alignment.<a href="./org/biojava3/alignment/Alignments.PairInProfileScorerType.html" title="enum in org.biojava3.alignment">Alignments.PairInProfileScorerType</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/Alignments.PairwiseSequenceAlignerType.html#values()">values()</a></span> - Static method in enum org.biojava3.alignment.<a href="./org/biojava3/alignment/Alignments.PairwiseSequenceAlignerType.html" title="enum in org.biojava3.alignment">Alignments.PairwiseSequenceAlignerType</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/Alignments.PairwiseSequenceScorerType.html#values()">values()</a></span> - Static method in enum org.biojava3.alignment.<a href="./org/biojava3/alignment/Alignments.PairwiseSequenceScorerType.html" title="enum in org.biojava3.alignment">Alignments.PairwiseSequenceScorerType</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/Alignments.ProfileProfileAlignerType.html#values()">values()</a></span> - Static method in enum org.biojava3.alignment.<a href="./org/biojava3/alignment/Alignments.ProfileProfileAlignerType.html" title="enum in org.biojava3.alignment">Alignments.ProfileProfileAlignerType</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/Alignments.RefinerType.html#values()">values()</a></span> - Static method in enum org.biojava3.alignment.<a href="./org/biojava3/alignment/Alignments.RefinerType.html" title="enum in org.biojava3.alignment">Alignments.RefinerType</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/routines/AlignerHelper.Last.html#values()">values()</a></span> - Static method in enum org.biojava3.alignment.routines.<a href="./org/biojava3/alignment/routines/AlignerHelper.Last.html" title="enum in org.biojava3.alignment.routines">AlignerHelper.Last</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AlignedSequence.Step.html#values()">values()</a></span> - Static method in enum org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AlignedSequence.Step.html" title="enum in org.biojava3.alignment.template">AlignedSequence.Step</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/GapPenalty.Type.html#values()">values()</a></span> - Static method in enum org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/GapPenalty.Type.html" title="enum in org.biojava3.alignment.template">GapPenalty.Type</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/Profile.StringFormat.html#values()">values()</a></span> - Static method in enum org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/Profile.StringFormat.html" title="enum in org.biojava3.alignment.template">Profile.StringFormat</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/DataSource.html#values()">values()</a></span> - Static method in enum org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/DataSource.html" title="enum in org.biojava3.core.sequence">DataSource</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/DNASequence.DNAType.html#values()">values()</a></span> - Static method in enum org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/DNASequence.DNAType.html" title="enum in org.biojava3.core.sequence">DNASequence.DNAType</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/SequenceOptimizationHints.SequenceCollection.html#values()">values()</a></span> - Static method in enum org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/SequenceOptimizationHints.SequenceCollection.html" title="enum in org.biojava3.core.sequence">SequenceOptimizationHints.SequenceCollection</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/SequenceOptimizationHints.SequenceUsage.html#values()">values()</a></span> - Static method in enum org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/SequenceOptimizationHints.SequenceUsage.html" title="enum in org.biojava3.core.sequence">SequenceOptimizationHints.SequenceUsage</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/Strand.html#values()">values()</a></span> - Static method in enum org.biojava3.core.sequence.<a href="./org/biojava3/core/sequence/Strand.html" title="enum in org.biojava3.core.sequence">Strand</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractSequence.AnnotationType.html#values()">values()</a></span> - Static method in enum org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractSequence.AnnotationType.html" title="enum in org.biojava3.core.sequence.template">AbstractSequence.AnnotationType</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/LightweightProfile.StringFormat.html#values()">values()</a></span> - Static method in enum org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/LightweightProfile.StringFormat.html" title="enum in org.biojava3.core.sequence.template">LightweightProfile.StringFormat</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/Frame.html#values()">values()</a></span> - Static method in enum org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/Frame.html" title="enum in org.biojava3.core.sequence.transcription">Frame</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/TreeConstructionAlgorithm.html#values()">values()</a></span> - Static method in enum org.biojava3.phylo.<a href="./org/biojava3/phylo/TreeConstructionAlgorithm.html" title="enum in org.biojava3.phylo">TreeConstructionAlgorithm</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/phylo/TreeType.html#values()">values()</a></span> - Static method in enum org.biojava3.phylo.<a href="./org/biojava3/phylo/TreeType.html" title="enum in org.biojava3.phylo">TreeType</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ModificationCategory.html#values()">values()</a></span> - Static method in enum org.biojava3.protmod.<a href="./org/biojava3/protmod/ModificationCategory.html" title="enum in org.biojava3.protmod">ModificationCategory</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/protmod/ModificationOccurrenceType.html#values()">values()</a></span> - Static method in enum org.biojava3.protmod.<a href="./org/biojava3/protmod/ModificationOccurrenceType.html" title="enum in org.biojava3.protmod">ModificationOccurrenceType</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ronn/RonnConstraint.Threshold.html#values()">values()</a></span> - Static method in enum org.biojava3.ronn.<a href="./org/biojava3/ronn/RonnConstraint.Threshold.html" title="enum in org.biojava3.ronn">RonnConstraint.Threshold</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/structure/gui/RenderStyle.html#values()">values()</a></span> - Static method in enum org.biojava3.structure.gui.<a href="./org/biojava3/structure/gui/RenderStyle.html" title="enum in org.biojava3.structure.gui">RenderStyle</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastOutputFormat.html#values()">values()</a></span> - Static method in enum org.biojava3.ws.alignment.qblast.<a href="./org/biojava3/ws/alignment/qblast/NCBIQBlastOutputFormat.html" title="enum in org.biojava3.ws.alignment.qblast">NCBIQBlastOutputFormat</a></dt>
<dd>
<div class="block">Returns an array containing the constants of this enum type, in
the order they are declared.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/Calc.html#vectorProduct(org.biojava.bio.structure.Atom, org.biojava.bio.structure.Atom)">vectorProduct(Atom, Atom)</a></span> - Static method in class org.biojava.bio.structure.<a href="./org/biojava/bio/structure/Calc.html" title="class in org.biojava.bio.structure">Calc</a></dt>
<dd>
<div class="block">Vector product .</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/domain/pdp/CutDomain.html#verbose">verbose</a></span> - Static variable in class org.biojava.bio.structure.domain.pdp.<a href="./org/biojava/bio/structure/domain/pdp/CutDomain.html" title="class in org.biojava.bio.structure.domain.pdp">CutDomain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/CaseFreeAminoAcidCompoundSet.html#verifySequence(org.biojava3.core.sequence.template.Sequence)">verifySequence(Sequence&lt;AminoAcidCompound&gt;)</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/CaseFreeAminoAcidCompoundSet.html" title="class in org.biojava3.aaproperties.xml">CaseFreeAminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/xml/ModifiedAminoAcidCompoundSet.html#verifySequence(org.biojava3.core.sequence.template.Sequence)">verifySequence(Sequence&lt;AminoAcidCompound&gt;)</a></span> - Method in class org.biojava3.aaproperties.xml.<a href="./org/biojava3/aaproperties/xml/ModifiedAminoAcidCompoundSet.html" title="class in org.biojava3.aaproperties.xml">ModifiedAminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/compound/AminoAcidCompoundSet.html#verifySequence(org.biojava3.core.sequence.template.Sequence)">verifySequence(Sequence&lt;AminoAcidCompound&gt;)</a></span> - Method in class org.biojava3.core.sequence.compound.<a href="./org/biojava3/core/sequence/compound/AminoAcidCompoundSet.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompoundSet.html#verifySequence(org.biojava3.core.sequence.template.Sequence)">verifySequence(Sequence&lt;C&gt;)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompoundSet.html" title="class in org.biojava3.core.sequence.template">AbstractCompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/CompoundSet.html#verifySequence(org.biojava3.core.sequence.template.Sequence)">verifySequence(Sequence&lt;C&gt;)</a></span> - Method in interface org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/CompoundSet.html" title="interface in org.biojava3.core.sequence.template">CompoundSet</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeCPMain.html#version">version</a></span> - Static variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeCPMain.html" title="class in org.biojava.bio.structure.align.ce">CeCPMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/ce/CeMain.html#version">version</a></span> - Static variable in class org.biojava.bio.structure.align.ce.<a href="./org/biojava/bio/structure/align/ce/CeMain.html" title="class in org.biojava.bio.structure.align.ce">CeMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/fatcat/FatCat.html#VERSION">VERSION</a></span> - Static variable in class org.biojava.bio.structure.align.fatcat.<a href="./org/biojava/bio/structure/align/fatcat/FatCat.html" title="class in org.biojava.bio.structure.align.fatcat">FatCat</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/BiojavaJmol.html#viewer">viewer</a></span> - Static variable in class org.biojava.bio.structure.gui.<a href="./org/biojava/bio/structure/gui/BiojavaJmol.html" title="class in org.biojava.bio.structure.gui">BiojavaJmol</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/structure/gui/JmolViewerImpl.html#viewer">viewer</a></span> - Static variable in class org.biojava3.structure.gui.<a href="./org/biojava3/structure/gui/JmolViewerImpl.html" title="class in org.biojava3.structure.gui">JmolViewerImpl</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_W_">
<!--   -->
</a>
<h2 class="title">W</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/Constraints.html#W">W</a></span> - Static variable in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/Constraints.html" title="class in org.biojava3.aaproperties">Constraints</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./demo/WebStartClientDemo.html" title="class in demo"><span class="strong">WebStartClientDemo</span></a> - Class in <a href="./demo/package-summary.html">demo</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./demo/WebStartClientDemo.html#WebStartClientDemo()">WebStartClientDemo()</a></span> - Constructor for class demo.<a href="./demo/WebStartClientDemo.html" title="class in demo">WebStartClientDemo</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/webstart/WebStartDBSearch.html" title="class in org.biojava.bio.structure.align.webstart"><span class="strong">WebStartDBSearch</span></a> - Class in <a href="./org/biojava/bio/structure/align/webstart/package-summary.html">org.biojava.bio.structure.align.webstart</a></dt>
<dd>
<div class="block">A Web Start wrapper for a FarmJobRunnable.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/webstart/WebStartDBSearch.html#WebStartDBSearch()">WebStartDBSearch()</a></span> - Constructor for class org.biojava.bio.structure.align.webstart.<a href="./org/biojava/bio/structure/align/webstart/WebStartDBSearch.html" title="class in org.biojava.bio.structure.align.webstart">WebStartDBSearch</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/align/webstart/WebStartMain.html" title="class in org.biojava.bio.structure.align.webstart"><span class="strong">WebStartMain</span></a> - Class in <a href="./org/biojava/bio/structure/align/webstart/package-summary.html">org.biojava.bio.structure.align.webstart</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/webstart/WebStartMain.html#WebStartMain()">WebStartMain()</a></span> - Constructor for class org.biojava.bio.structure.align.webstart.<a href="./org/biojava/bio/structure/align/webstart/WebStartMain.html" title="class in org.biojava.bio.structure.align.webstart">WebStartMain</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html#WHITE_BLACK_GRADIENT">WHITE_BLACK_GRADIENT</a></span> - Static variable in class org.biojava.bio.structure.gui.util.color.<a href="./org/biojava/bio/structure/gui/util/color/GradientMapper.html" title="class in org.biojava.bio.structure.gui.util.color">GradientMapper</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/data/sequence/SequenceUtil.html#WHITE_SPACE">WHITE_SPACE</a></span> - Static variable in class org.biojava3.data.sequence.<a href="./org/biojava3/data/sequence/SequenceUtil.html" title="class in org.biojava3.data.sequence">SequenceUtil</a></dt>
<dd>
<div class="block">A whitespace character: [\t\n\x0B\f\r]</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/Location.html#window(int, int)">window(int, int)</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/Location.html" title="class in org.biojava3.genome.parsers.gff">Location</a></dt>
<dd>
<div class="block">Enable a "sliding window" iteration over a location 
 to use with Java's "for" loop construct.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html#windowActivated(java.awt.event.WindowEvent)">windowActivated(WindowEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AligPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/StatusDisplay.html#windowActivated(java.awt.event.WindowEvent)">windowActivated(WindowEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/StatusDisplay.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">StatusDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html#windowActivated(java.awt.event.WindowEvent)">windowActivated(WindowEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html" title="class in org.biojava.bio.structure.align.gui.jmol">StructureAlignmentJmol</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html#windowClosed(java.awt.event.WindowEvent)">windowClosed(WindowEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AligPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/StatusDisplay.html#windowClosed(java.awt.event.WindowEvent)">windowClosed(WindowEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/StatusDisplay.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">StatusDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html#windowClosed(java.awt.event.WindowEvent)">windowClosed(WindowEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html" title="class in org.biojava.bio.structure.align.gui.jmol">StructureAlignmentJmol</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html#windowClosing(java.awt.event.WindowEvent)">windowClosing(WindowEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AligPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/StatusDisplay.html#windowClosing(java.awt.event.WindowEvent)">windowClosing(WindowEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/StatusDisplay.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">StatusDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html#windowClosing(java.awt.event.WindowEvent)">windowClosing(WindowEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html" title="class in org.biojava.bio.structure.align.gui.jmol">StructureAlignmentJmol</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html#windowDeactivated(java.awt.event.WindowEvent)">windowDeactivated(WindowEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AligPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/StatusDisplay.html#windowDeactivated(java.awt.event.WindowEvent)">windowDeactivated(WindowEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/StatusDisplay.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">StatusDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html#windowDeactivated(java.awt.event.WindowEvent)">windowDeactivated(WindowEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html" title="class in org.biojava.bio.structure.align.gui.jmol">StructureAlignmentJmol</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html#windowDeiconified(java.awt.event.WindowEvent)">windowDeiconified(WindowEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AligPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/StatusDisplay.html#windowDeiconified(java.awt.event.WindowEvent)">windowDeiconified(WindowEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/StatusDisplay.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">StatusDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html#windowDeiconified(java.awt.event.WindowEvent)">windowDeiconified(WindowEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html" title="class in org.biojava.bio.structure.align.gui.jmol">StructureAlignmentJmol</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/sequence/views/WindowedSequence.html" title="class in org.biojava3.core.sequence.views"><span class="strong">WindowedSequence</span></a>&lt;<a href="./org/biojava3/core/sequence/views/WindowedSequence.html" title="type parameter in WindowedSequence">C</a> extends <a href="./org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt; - Class in <a href="./org/biojava3/core/sequence/views/package-summary.html">org.biojava3.core.sequence.views</a></dt>
<dd>
<div class="block">A sliding window view of a sequence which does not implement any
 interfaces like <a href="./org/biojava3/core/sequence/template/Sequence.html" title="interface in org.biojava3.core.sequence.template"><code>Sequence</code></a> because they do not fit how this works.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/views/WindowedSequence.html#WindowedSequence(org.biojava3.core.sequence.template.Sequence, int)">WindowedSequence(Sequence&lt;C&gt;, int)</a></span> - Constructor for class org.biojava3.core.sequence.views.<a href="./org/biojava3/core/sequence/views/WindowedSequence.html" title="class in org.biojava3.core.sequence.views">WindowedSequence</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html#windowIconified(java.awt.event.WindowEvent)">windowIconified(WindowEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AligPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/StatusDisplay.html#windowIconified(java.awt.event.WindowEvent)">windowIconified(WindowEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/StatusDisplay.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">StatusDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html#windowIconified(java.awt.event.WindowEvent)">windowIconified(WindowEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html" title="class in org.biojava.bio.structure.align.gui.jmol">StructureAlignmentJmol</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html#windowOpened(java.awt.event.WindowEvent)">windowOpened(WindowEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/AligPanel.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">AligPanel</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/aligpanel/StatusDisplay.html#windowOpened(java.awt.event.WindowEvent)">windowOpened(WindowEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.aligpanel.<a href="./org/biojava/bio/structure/align/gui/aligpanel/StatusDisplay.html" title="class in org.biojava.bio.structure.align.gui.aligpanel">StatusDisplay</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html#windowOpened(java.awt.event.WindowEvent)">windowOpened(WindowEvent)</a></span> - Method in class org.biojava.bio.structure.align.gui.jmol.<a href="./org/biojava/bio/structure/align/gui/jmol/StructureAlignmentJmol.html" title="class in org.biojava.bio.structure.align.gui.jmol">StructureAlignmentJmol</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/AbstractCompoundTranslator.html#workingListToSequences(java.util.List)">workingListToSequences(List&lt;List&lt;T&gt;&gt;)</a></span> - Method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/AbstractCompoundTranslator.html" title="class in org.biojava3.core.sequence.template">AbstractCompoundTranslator</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/Frame.html#wrap(org.biojava3.core.sequence.template.Sequence)">wrap(Sequence&lt;C&gt;)</a></span> - Method in enum org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/Frame.html" title="enum in org.biojava3.core.sequence.transcription">Frame</a></dt>
<dd>
<div class="block">Optionally wraps a Sequence in a reverse complementing view (if the
 frame is on the reverse strand) and creates a sub sequence view if
 it is required.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/transcription/DNAToRNATranslator.html#wrapToRna(org.biojava3.core.sequence.template.Sequence)">wrapToRna(Sequence&lt;NucleotideCompound&gt;)</a></span> - Method in class org.biojava3.core.sequence.transcription.<a href="./org/biojava3/core/sequence/transcription/DNAToRNATranslator.html" title="class in org.biojava3.core.sequence.transcription">DNAToRNATranslator</a></dt>
<dd>
<div class="block">Takes in the given DNA Sequence and returns an instance of RNASequence
 which is using <a href="./org/biojava3/core/sequence/views/RnaSequenceView.html" title="class in org.biojava3.core.sequence.views"><code>RnaSequenceView</code></a> as a
 <a href="./org/biojava3/core/sequence/template/ProxySequenceReader.html" title="interface in org.biojava3.core.sequence.template"><code>ProxySequenceReader</code></a>.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/align/util/SynchronizedOutFile.html#write(java.lang.String)">write(String)</a></span> - Method in class org.biojava.bio.structure.align.util.<a href="./org/biojava/bio/structure/align/util/SynchronizedOutFile.html" title="class in org.biojava.bio.structure.align.util">SynchronizedOutFile</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/template/SequenceMixin.html#write(java.lang.Appendable, org.biojava3.core.sequence.template.Sequence)">write(Appendable, Sequence&lt;C&gt;)</a></span> - Static method in class org.biojava3.core.sequence.template.<a href="./org/biojava3/core/sequence/template/SequenceMixin.html" title="class in org.biojava3.core.sequence.template">SequenceMixin</a></dt>
<dd>
<div class="block">Used as a way of sending a Sequence to a writer without the cost of
 converting to a full length String and then writing the data out</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/GeneIDGFF2Reader.html#write(org.biojava3.genome.parsers.gff.FeatureList, java.lang.String)">write(FeatureList, String)</a></span> - Static method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/GeneIDGFF2Reader.html" title="class in org.biojava3.genome.parsers.gff">GeneIDGFF2Reader</a></dt>
<dd>
<div class="block">Write features in FeatureList to file.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/genome/parsers/gff/GFF3Writer.html#write(java.io.OutputStream, java.util.LinkedHashMap)">write(OutputStream, LinkedHashMap&lt;String, ChromosomeSequence&gt;)</a></span> - Method in class org.biojava3.genome.parsers.gff.<a href="./org/biojava3/genome/parsers/gff/GFF3Writer.html" title="class in org.biojava3.genome.parsers.gff">GFF3Writer</a></dt>
<dd>
<div class="block">Output gff3 format for a DNA Sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/ronn/NullOutputStream.html#write(int)">write(int)</a></span> - Method in class org.biojava3.ronn.<a href="./org/biojava3/ronn/NullOutputStream.html" title="class in org.biojava3.ronn">NullOutputStream</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/data/sequence/SequenceUtil.html#writeFasta(java.io.OutputStream, java.util.List, int)">writeFasta(OutputStream, List&lt;FastaSequence&gt;, int)</a></span> - Static method in class org.biojava3.data.sequence.<a href="./org/biojava3/data/sequence/SequenceUtil.html" title="class in org.biojava3.data.sequence">SequenceUtil</a></dt>
<dd>
<div class="block">Writes list of FastaSequeces into the outstream formatting the sequence
 so that it contains width chars on each line</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/data/sequence/SequenceUtil.html#writeFasta(java.io.OutputStream, java.util.List)">writeFasta(OutputStream, List&lt;FastaSequence&gt;)</a></span> - Static method in class org.biojava3.data.sequence.<a href="./org/biojava3/data/sequence/SequenceUtil.html" title="class in org.biojava3.data.sequence">SequenceUtil</a></dt>
<dd>
<div class="block">Writes FastaSequence in the file, each sequence will take one line only</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/FastaWriterHelper.html#writeGeneSequence(java.io.File, java.util.Collection, boolean)">writeGeneSequence(File, Collection&lt;GeneSequence&gt;, boolean)</a></span> - Static method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/FastaWriterHelper.html" title="class in org.biojava3.core.sequence.io">FastaWriterHelper</a></dt>
<dd>
<div class="block">Write a collection of GeneSequences to a file where if the gene is negative strand it will flip and complement the sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/FastaWriterHelper.html#writeGeneSequence(java.io.OutputStream, java.util.Collection, boolean)">writeGeneSequence(OutputStream, Collection&lt;GeneSequence&gt;, boolean)</a></span> - Static method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/FastaWriterHelper.html" title="class in org.biojava3.core.sequence.io">FastaWriterHelper</a></dt>
<dd>
<div class="block">Write a collection of GeneSequences to a file where if the gene is negative strand it will flip and complement the sequence</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/PrettyXMLWriter.html#writeIndent()">writeIndent()</a></span> - Method in class org.biojava3.core.util.<a href="./org/biojava3/core/util/PrettyXMLWriter.html" title="class in org.biojava3.core.util">PrettyXMLWriter</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/FastaWriterHelper.html#writeNucleotideSequence(java.io.File, java.util.Collection)">writeNucleotideSequence(File, Collection&lt;DNASequence&gt;)</a></span> - Static method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/FastaWriterHelper.html" title="class in org.biojava3.core.sequence.io">FastaWriterHelper</a></dt>
<dd>
<div class="block">Write a collection of NucleotideSequences to a file</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/FastaWriterHelper.html#writeNucleotideSequence(java.io.OutputStream, java.util.Collection)">writeNucleotideSequence(OutputStream, Collection&lt;DNASequence&gt;)</a></span> - Static method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/FastaWriterHelper.html" title="class in org.biojava3.core.sequence.io">FastaWriterHelper</a></dt>
<dd>
<div class="block">Write a collection of NucleotideSequences to a file</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/FastaWriterHelper.html#writeProteinSequence(java.io.File, java.util.Collection)">writeProteinSequence(File, Collection&lt;ProteinSequence&gt;)</a></span> - Static method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/FastaWriterHelper.html" title="class in org.biojava3.core.sequence.io">FastaWriterHelper</a></dt>
<dd>
<div class="block">Write collection of protein sequences to a file</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/FastaWriterHelper.html#writeProteinSequence(java.io.OutputStream, java.util.Collection)">writeProteinSequence(OutputStream, Collection&lt;ProteinSequence&gt;)</a></span> - Static method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/FastaWriterHelper.html" title="class in org.biojava3.core.sequence.io">FastaWriterHelper</a></dt>
<dd>
<div class="block">Write collection of protein sequences to a stream</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/FastaWriterHelper.html#writeSequence(java.io.File, org.biojava3.core.sequence.template.Sequence)">writeSequence(File, Sequence&lt;?&gt;)</a></span> - Static method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/FastaWriterHelper.html" title="class in org.biojava3.core.sequence.io">FastaWriterHelper</a></dt>
<dd>
<div class="block">Write a sequence to a file</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/FastaWriterHelper.html#writeSequence(java.io.OutputStream, org.biojava3.core.sequence.template.Sequence)">writeSequence(OutputStream, Sequence&lt;?&gt;)</a></span> - Static method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/FastaWriterHelper.html" title="class in org.biojava3.core.sequence.io">FastaWriterHelper</a></dt>
<dd>
<div class="block">Write a sequence to OutputStream</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/core/sequence/io/FastaWriterHelper.html#writeSequences(java.io.OutputStream, java.util.Collection)">writeSequences(OutputStream, Collection&lt;Sequence&lt;?&gt;&gt;)</a></span> - Static method in class org.biojava3.core.sequence.io.<a href="./org/biojava3/core/sequence/io/FastaWriterHelper.html" title="class in org.biojava3.core.sequence.io">FastaWriterHelper</a></dt>
<dd>
<div class="block">Method which will write your given Sequences to the specified
 <a href="http://java.sun.com/j2se/1.6.0/docs/api/java/io/OutputStream.html?is-external=true" title="class or interface in java.io"><code>OutputStream</code></a>.</div>
</dd>
</dl>
<a name="_X_">
<!--   -->
</a>
<h2 class="title">X</h2>
<dl>
<dt><a href="./org/biojava3/core/util/XMLHelper.html" title="class in org.biojava3.core.util"><span class="strong">XMLHelper</span></a> - Class in <a href="./org/biojava3/core/util/package-summary.html">org.biojava3.core.util</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/core/util/XMLHelper.html#XMLHelper()">XMLHelper()</a></span> - Constructor for class org.biojava3.core.util.<a href="./org/biojava3/core/util/XMLHelper.html" title="class in org.biojava3.core.util">XMLHelper</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava/bio/structure/scop/server/XMLUtil.html" title="class in org.biojava.bio.structure.scop.server"><span class="strong">XMLUtil</span></a> - Class in <a href="./org/biojava/bio/structure/scop/server/package-summary.html">org.biojava.bio.structure.scop.server</a></dt>
<dd>
<div class="block">Utility classes for the XML serialization and de-serialization of SCOP.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava/bio/structure/scop/server/XMLUtil.html#XMLUtil()">XMLUtil()</a></span> - Constructor for class org.biojava.bio.structure.scop.server.<a href="./org/biojava/bio/structure/scop/server/XMLUtil.html" title="class in org.biojava.bio.structure.scop.server">XMLUtil</a></dt>
<dd>&nbsp;</dd>
<dt><a href="./org/biojava3/core/util/XMLWriter.html" title="interface in org.biojava3.core.util"><span class="strong">XMLWriter</span></a> - Interface in <a href="./org/biojava3/core/util/package-summary.html">org.biojava3.core.util</a></dt>
<dd>
<div class="block">Simple interface for building XML documents.</div>
</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#xyMax">xyMax</a></span> - Variable in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava3.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
<dt><span class="strong"><a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html#xyStart">xyStart</a></span> - Variable in class org.biojava3.alignment.template.<a href="./org/biojava3/alignment/template/AbstractMatrixAligner.html" title="class in org.biojava3.alignment.template">AbstractMatrixAligner</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_Y_">
<!--   -->
</a>
<h2 class="title">Y</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava3/aaproperties/Constraints.html#Y">Y</a></span> - Static variable in class org.biojava3.aaproperties.<a href="./org/biojava3/aaproperties/Constraints.html" title="class in org.biojava3.aaproperties">Constraints</a></dt>
<dd>&nbsp;</dd>
</dl>
<a name="_Z_">
<!--   -->
</a>
<h2 class="title">Z</h2>
<dl>
<dt><span class="strong"><a href="./org/biojava3/phylo/ResidueProperties.html#zappo">zappo</a></span> - Static variable in class org.biojava3.phylo.<a href="./org/biojava3/phylo/ResidueProperties.html" title="class in org.biojava3.phylo">ResidueProperties</a></dt>
<dd>&nbsp;</dd>
</dl>
<a href="#_A_">A</a>&nbsp;<a href="#_B_">B</a>&nbsp;<a href="#_C_">C</a>&nbsp;<a href="#_D_">D</a>&nbsp;<a href="#_E_">E</a>&nbsp;<a href="#_F_">F</a>&nbsp;<a href="#_G_">G</a>&nbsp;<a href="#_H_">H</a>&nbsp;<a href="#_I_">I</a>&nbsp;<a href="#_J_">J</a>&nbsp;<a href="#_K_">K</a>&nbsp;<a href="#_L_">L</a>&nbsp;<a href="#_M_">M</a>&nbsp;<a href="#_N_">N</a>&nbsp;<a href="#_O_">O</a>&nbsp;<a href="#_P_">P</a>&nbsp;<a href="#_Q_">Q</a>&nbsp;<a href="#_R_">R</a>&nbsp;<a href="#_S_">S</a>&nbsp;<a href="#_T_">T</a>&nbsp;<a href="#_U_">U</a>&nbsp;<a href="#_V_">V</a>&nbsp;<a href="#_W_">W</a>&nbsp;<a href="#_X_">X</a>&nbsp;<a href="#_Y_">Y</a>&nbsp;<a href="#_Z_">Z</a>&nbsp;</div>
<!-- ======= START OF BOTTOM NAVBAR ====== -->
<div class="bottomNav"><a name="navbar_bottom">
<!--   -->
</a><a href="#skip-navbar_bottom" title="Skip navigation links"></a><a name="navbar_bottom_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="./overview-summary.html">Overview</a></li>
<li>Package</li>
<li>Class</li>
<li>Use</li>
<li><a href="./overview-tree.html">Tree</a></li>
<li><a href="./deprecated-list.html">Deprecated</a></li>
<li class="navBarCell1Rev">Index</li>
<li><a href="./help-doc.html">Help</a></li>
</ul>
<div class="aboutLanguage"><em><script src="http://www.google-analytics.com/urchin.js" type="text/javascript"></script><script type="text/javascript">_uacct = "UA-1326640-1";urchinTracker();</script></em></div>
</div>
<div class="subNav">
<ul class="navList">
<li>Prev</li>
<li>Next</li>
</ul>
<ul class="navList">
<li><a href="./index.html?index-all.html" target="_top">Frames</a></li>
<li><a href="index-all.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_bottom">
<li><a href="./allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_bottom");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<a name="skip-navbar_bottom">
<!--   -->
</a></div>
<!-- ======== END OF BOTTOM NAVBAR ======= -->
<p class="legalCopy"><small>Copyright &#169; 2011 <a href="http://www.biojava.org">BioJava</a>. All Rights Reserved.</small></p>
</body>
</html>
