<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!-- NewPage -->
<html lang="en">
<head>
<!-- Generated by javadoc (version 1.7.0_05) on Fri Nov 30 11:03:56 PST 2012 -->
<meta http-equiv="Content-Type" content="text/html" charset="UTF-8">
<title>Uses of Interface org.biojava3.core.sequence.io.template.FastaHeaderParserInterface (biojava 3.0.5 API)</title>
<meta name="date" content="2012-11-30">
<link rel="stylesheet" type="text/css" href="../../../../../../../stylesheet.css" title="Style">
</head>
<body>
<script type="text/javascript"><!--
    if (location.href.indexOf('is-external=true') == -1) {
        parent.document.title="Uses of Interface org.biojava3.core.sequence.io.template.FastaHeaderParserInterface (biojava 3.0.5 API)";
    }
//-->
</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
<!-- ========= START OF TOP NAVBAR ======= -->
<div class="topNav"><a name="navbar_top">
<!--   -->
</a><a href="#skip-navbar_top" title="Skip navigation links"></a><a name="navbar_top_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../../../../../../../overview-summary.html">Overview</a></li>
<li><a href="../package-summary.html">Package</a></li>
<li><a href="../../../../../../../org/biojava3/core/sequence/io/template/FastaHeaderParserInterface.html" title="interface in org.biojava3.core.sequence.io.template">Class</a></li>
<li class="navBarCell1Rev">Use</li>
<li><a href="../package-tree.html">Tree</a></li>
<li><a href="../../../../../../../deprecated-list.html">Deprecated</a></li>
<li><a href="../../../../../../../index-all.html">Index</a></li>
<li><a href="../../../../../../../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li>Prev</li>
<li>Next</li>
</ul>
<ul class="navList">
<li><a href="../../../../../../../index.html?org/biojava3/core/sequence/io/template/class-use/FastaHeaderParserInterface.html" target="_top">Frames</a></li>
<li><a href="FastaHeaderParserInterface.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_top">
<li><a href="../../../../../../../allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_top");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<a name="skip-navbar_top">
<!--   -->
</a></div>
<!-- ========= END OF TOP NAVBAR ========= -->
<div class="header">
<h2 title="Uses of Interface org.biojava3.core.sequence.io.template.FastaHeaderParserInterface" class="title">Uses of Interface<br>org.biojava3.core.sequence.io.template.FastaHeaderParserInterface</h2>
</div>
<div class="classUseContainer">
<ul class="blockList">
<li class="blockList">
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing packages, and an explanation">
<caption><span>Packages that use <a href="../../../../../../../org/biojava3/core/sequence/io/template/FastaHeaderParserInterface.html" title="interface in org.biojava3.core.sequence.io.template">FastaHeaderParserInterface</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Package</th>
<th class="colLast" scope="col">Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><a href="#org.biojava.bio.structure.io">org.biojava.bio.structure.io</a></td>
<td class="colLast">
<div class="block">Input and Output of Structures</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="#org.biojava3.core.sequence.io">org.biojava3.core.sequence.io</a></td>
<td class="colLast">&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList">
<ul class="blockList">
<li class="blockList"><a name="org.biojava.bio.structure.io">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../../org/biojava3/core/sequence/io/template/FastaHeaderParserInterface.html" title="interface in org.biojava3.core.sequence.io.template">FastaHeaderParserInterface</a> in <a href="../../../../../../../org/biojava/bio/structure/io/package-summary.html">org.biojava.bio.structure.io</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing constructors, and an explanation">
<caption><span>Constructors in <a href="../../../../../../../org/biojava/bio/structure/io/package-summary.html">org.biojava.bio.structure.io</a> with parameters of type <a href="../../../../../../../org/biojava3/core/sequence/io/template/FastaHeaderParserInterface.html" title="interface in org.biojava3.core.sequence.io.template">FastaHeaderParserInterface</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colOne" scope="col">Constructor and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/bio/structure/io/FastaStructureParser.html#FastaStructureParser(java.io.File, org.biojava3.core.sequence.io.template.FastaHeaderParserInterface, org.biojava3.core.sequence.io.template.SequenceCreatorInterface, org.biojava.bio.structure.align.util.AtomCache)">FastaStructureParser</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/io/File.html?is-external=true" title="class or interface in java.io">File</a>&nbsp;file,
                                        <a href="../../../../../../../org/biojava3/core/sequence/io/template/FastaHeaderParserInterface.html" title="interface in org.biojava3.core.sequence.io.template">FastaHeaderParserInterface</a>&lt;<a href="../../../../../../../org/biojava3/core/sequence/ProteinSequence.html" title="class in org.biojava3.core.sequence">ProteinSequence</a>,<a href="../../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;headerParser,
                                        <a href="../../../../../../../org/biojava3/core/sequence/io/template/SequenceCreatorInterface.html" title="interface in org.biojava3.core.sequence.io.template">SequenceCreatorInterface</a>&lt;<a href="../../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;sequenceCreator,
                                        <a href="../../../../../../../org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a>&nbsp;cache)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava/bio/structure/io/FastaStructureParser.html#FastaStructureParser(java.io.InputStream, org.biojava3.core.sequence.io.template.FastaHeaderParserInterface, org.biojava3.core.sequence.io.template.SequenceCreatorInterface, org.biojava.bio.structure.align.util.AtomCache)">FastaStructureParser</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/io/InputStream.html?is-external=true" title="class or interface in java.io">InputStream</a>&nbsp;is,
                                        <a href="../../../../../../../org/biojava3/core/sequence/io/template/FastaHeaderParserInterface.html" title="interface in org.biojava3.core.sequence.io.template">FastaHeaderParserInterface</a>&lt;<a href="../../../../../../../org/biojava3/core/sequence/ProteinSequence.html" title="class in org.biojava3.core.sequence">ProteinSequence</a>,<a href="../../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;headerParser,
                                        <a href="../../../../../../../org/biojava3/core/sequence/io/template/SequenceCreatorInterface.html" title="interface in org.biojava3.core.sequence.io.template">SequenceCreatorInterface</a>&lt;<a href="../../../../../../../org/biojava3/core/sequence/compound/AminoAcidCompound.html" title="class in org.biojava3.core.sequence.compound">AminoAcidCompound</a>&gt;&nbsp;sequenceCreator,
                                        <a href="../../../../../../../org/biojava/bio/structure/align/util/AtomCache.html" title="class in org.biojava.bio.structure.align.util">AtomCache</a>&nbsp;cache)</code>&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava3.core.sequence.io">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../../org/biojava3/core/sequence/io/template/FastaHeaderParserInterface.html" title="interface in org.biojava3.core.sequence.io.template">FastaHeaderParserInterface</a> in <a href="../../../../../../../org/biojava3/core/sequence/io/package-summary.html">org.biojava3.core.sequence.io</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing classes, and an explanation">
<caption><span>Classes in <a href="../../../../../../../org/biojava3/core/sequence/io/package-summary.html">org.biojava3.core.sequence.io</a> that implement <a href="../../../../../../../org/biojava3/core/sequence/io/template/FastaHeaderParserInterface.html" title="interface in org.biojava3.core.sequence.io.template">FastaHeaderParserInterface</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Class and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>class&nbsp;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava3/core/sequence/io/GenericFastaHeaderParser.html" title="class in org.biojava3.core.sequence.io">GenericFastaHeaderParser</a>&lt;S extends <a href="../../../../../../../org/biojava3/core/sequence/template/AbstractSequence.html" title="class in org.biojava3.core.sequence.template">AbstractSequence</a>&lt;C&gt;,C extends <a href="../../../../../../../org/biojava3/core/sequence/template/Compound.html" title="interface in org.biojava3.core.sequence.template">Compound</a>&gt;</strong></code>
<div class="block">The default fasta header parser where some headers are well defined based on the source
 database which allows us to set the source of the protein sequence and the identifier
 that can be used in future implementations to load features from external sources
 
 If the user has a custom header with local data then they can create their own implementation
 of a FastaHeaderParserInterface

 GenBank                           gi|gi-number|gb|accession|locus
 ENA Data Library                  gi|gi-number|emb|accession|locus
 DDBJ, DNA Database of Japan       gi|gi-number|dbj|accession|locus
 NBRF PIR                          pir||entry
 Protein Research Foundation       prf||name
 SWISS-PROT                        sp|accession|name
 Brookhaven Protein Data Bank (1)  pdb|entry|chain
 Brookhaven Protein Data Bank (2)  entry:chain|PDBID|CHAIN|SEQUENCE
 PDB EBI                           PDB:1ECY_A mol:protein length:142  ECOTIN
 Patents                           pat|country|number
 GenInfo Backbone Id               bbs|number
 General database identifier       gnl|database|identifier
 NCBI Reference Sequence           ref|accession|locus
 Local Sequence identifier         lcl|identifier
</div>
</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing constructors, and an explanation">
<caption><span>Constructors in <a href="../../../../../../../org/biojava3/core/sequence/io/package-summary.html">org.biojava3.core.sequence.io</a> with parameters of type <a href="../../../../../../../org/biojava3/core/sequence/io/template/FastaHeaderParserInterface.html" title="interface in org.biojava3.core.sequence.io.template">FastaHeaderParserInterface</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colOne" scope="col">Constructor and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava3/core/sequence/io/FastaReader.html#FastaReader(java.io.File, org.biojava3.core.sequence.io.template.FastaHeaderParserInterface, org.biojava3.core.sequence.io.template.SequenceCreatorInterface)">FastaReader</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/io/File.html?is-external=true" title="class or interface in java.io">File</a>&nbsp;file,
                      <a href="../../../../../../../org/biojava3/core/sequence/io/template/FastaHeaderParserInterface.html" title="interface in org.biojava3.core.sequence.io.template">FastaHeaderParserInterface</a>&lt;<a href="../../../../../../../org/biojava3/core/sequence/io/FastaReader.html" title="type parameter in FastaReader">S</a>,<a href="../../../../../../../org/biojava3/core/sequence/io/FastaReader.html" title="type parameter in FastaReader">C</a>&gt;&nbsp;headerParser,
                      <a href="../../../../../../../org/biojava3/core/sequence/io/template/SequenceCreatorInterface.html" title="interface in org.biojava3.core.sequence.io.template">SequenceCreatorInterface</a>&lt;<a href="../../../../../../../org/biojava3/core/sequence/io/FastaReader.html" title="type parameter in FastaReader">C</a>&gt;&nbsp;sequenceCreator)</code>
<div class="block">If you are going to use the FileProxyProteinSequenceCreator then you need to use this constructor because we need details about
 the location of the file.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colLast"><code><strong><a href="../../../../../../../org/biojava3/core/sequence/io/FastaReader.html#FastaReader(java.io.InputStream, org.biojava3.core.sequence.io.template.FastaHeaderParserInterface, org.biojava3.core.sequence.io.template.SequenceCreatorInterface)">FastaReader</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/io/InputStream.html?is-external=true" title="class or interface in java.io">InputStream</a>&nbsp;is,
                      <a href="../../../../../../../org/biojava3/core/sequence/io/template/FastaHeaderParserInterface.html" title="interface in org.biojava3.core.sequence.io.template">FastaHeaderParserInterface</a>&lt;<a href="../../../../../../../org/biojava3/core/sequence/io/FastaReader.html" title="type parameter in FastaReader">S</a>,<a href="../../../../../../../org/biojava3/core/sequence/io/FastaReader.html" title="type parameter in FastaReader">C</a>&gt;&nbsp;headerParser,
                      <a href="../../../../../../../org/biojava3/core/sequence/io/template/SequenceCreatorInterface.html" title="interface in org.biojava3.core.sequence.io.template">SequenceCreatorInterface</a>&lt;<a href="../../../../../../../org/biojava3/core/sequence/io/FastaReader.html" title="type parameter in FastaReader">C</a>&gt;&nbsp;sequenceCreator)</code>
<div class="block">If you are going to use FileProxyProteinSequenceCreator then do not use this constructor because we need details about
 local file offsets for quick reads.</div>
</td>
</tr>
</tbody>
</table>
</li>
</ul>
</li>
</ul>
</div>
<!-- ======= START OF BOTTOM NAVBAR ====== -->
<div class="bottomNav"><a name="navbar_bottom">
<!--   -->
</a><a href="#skip-navbar_bottom" title="Skip navigation links"></a><a name="navbar_bottom_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../../../../../../../overview-summary.html">Overview</a></li>
<li><a href="../package-summary.html">Package</a></li>
<li><a href="../../../../../../../org/biojava3/core/sequence/io/template/FastaHeaderParserInterface.html" title="interface in org.biojava3.core.sequence.io.template">Class</a></li>
<li class="navBarCell1Rev">Use</li>
<li><a href="../package-tree.html">Tree</a></li>
<li><a href="../../../../../../../deprecated-list.html">Deprecated</a></li>
<li><a href="../../../../../../../index-all.html">Index</a></li>
<li><a href="../../../../../../../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li>Prev</li>
<li>Next</li>
</ul>
<ul class="navList">
<li><a href="../../../../../../../index.html?org/biojava3/core/sequence/io/template/class-use/FastaHeaderParserInterface.html" target="_top">Frames</a></li>
<li><a href="FastaHeaderParserInterface.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_bottom">
<li><a href="../../../../../../../allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_bottom");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<a name="skip-navbar_bottom">
<!--   -->
</a></div>
<!-- ======== END OF BOTTOM NAVBAR ======= -->
<p class="legalCopy"><small>Copyright &#169; 2012 <a href="http://www.biojava.org">BioJava</a>. All Rights Reserved.</small></p>
</body>
</html>
