<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<html lang="en">
<head>
<title>Source code</title>
<link rel="stylesheet" type="text/css" href="../../../../stylesheet.css" title="Style">
</head>
<body>
<div class="sourceContainer">
<pre><span class="sourceLineNo">001</span>/*<a name="line.1"></a>
<span class="sourceLineNo">002</span> * To change this template, choose Tools | Templates<a name="line.2"></a>
<span class="sourceLineNo">003</span> * and open the template in the editor.<a name="line.3"></a>
<span class="sourceLineNo">004</span> */<a name="line.4"></a>
<span class="sourceLineNo">005</span>package org.biojava3.genome;<a name="line.5"></a>
<span class="sourceLineNo">006</span><a name="line.6"></a>
<span class="sourceLineNo">007</span>import java.io.File;<a name="line.7"></a>
<span class="sourceLineNo">008</span>import java.io.FileWriter;<a name="line.8"></a>
<span class="sourceLineNo">009</span>import java.util.ArrayList;<a name="line.9"></a>
<span class="sourceLineNo">010</span>import java.util.Collection;<a name="line.10"></a>
<span class="sourceLineNo">011</span>import java.util.Collections;<a name="line.11"></a>
<span class="sourceLineNo">012</span>import java.util.LinkedHashMap;<a name="line.12"></a>
<span class="sourceLineNo">013</span>import java.util.logging.Logger;<a name="line.13"></a>
<span class="sourceLineNo">014</span>import org.biojava3.core.sequence.AccessionID;<a name="line.14"></a>
<span class="sourceLineNo">015</span>import org.biojava3.core.sequence.CDSSequence;<a name="line.15"></a>
<span class="sourceLineNo">016</span>import org.biojava3.core.sequence.ChromosomeSequence;<a name="line.16"></a>
<span class="sourceLineNo">017</span>import org.biojava3.core.sequence.DNASequence;<a name="line.17"></a>
<span class="sourceLineNo">018</span>import org.biojava3.core.sequence.ExonSequence;<a name="line.18"></a>
<span class="sourceLineNo">019</span>import org.biojava3.core.sequence.io.FastaReaderHelper;<a name="line.19"></a>
<span class="sourceLineNo">020</span>import org.biojava3.core.sequence.io.FastaWriterHelper;<a name="line.20"></a>
<span class="sourceLineNo">021</span>import org.biojava3.core.sequence.GeneSequence;<a name="line.21"></a>
<span class="sourceLineNo">022</span>import org.biojava3.core.sequence.ProteinSequence;<a name="line.22"></a>
<span class="sourceLineNo">023</span>import org.biojava3.core.sequence.Strand;<a name="line.23"></a>
<span class="sourceLineNo">024</span>import org.biojava3.core.sequence.TranscriptSequence;<a name="line.24"></a>
<span class="sourceLineNo">025</span>import org.biojava3.genome.parsers.gff.Feature;<a name="line.25"></a>
<span class="sourceLineNo">026</span>import org.biojava3.genome.parsers.gff.FeatureHelper;<a name="line.26"></a>
<span class="sourceLineNo">027</span>import org.biojava3.genome.parsers.gff.FeatureI;<a name="line.27"></a>
<span class="sourceLineNo">028</span>import org.biojava3.genome.parsers.gff.FeatureList;<a name="line.28"></a>
<span class="sourceLineNo">029</span>import org.biojava3.genome.parsers.gff.GFF3Reader;<a name="line.29"></a>
<span class="sourceLineNo">030</span>import org.biojava3.genome.parsers.gff.GeneIDGFF2Reader;<a name="line.30"></a>
<span class="sourceLineNo">031</span>import org.biojava3.genome.parsers.gff.GeneMarkGTFReader;<a name="line.31"></a>
<span class="sourceLineNo">032</span><a name="line.32"></a>
<span class="sourceLineNo">033</span>/**<a name="line.33"></a>
<span class="sourceLineNo">034</span> *<a name="line.34"></a>
<span class="sourceLineNo">035</span> * @author Scooter Willis &lt;willishf at gmail dot com&gt;<a name="line.35"></a>
<span class="sourceLineNo">036</span> */<a name="line.36"></a>
<span class="sourceLineNo">037</span>public class GeneFeatureHelper {<a name="line.37"></a>
<span class="sourceLineNo">038</span><a name="line.38"></a>
<span class="sourceLineNo">039</span>    private static final Logger log = Logger.getLogger(GeneFeatureHelper.class.getName());<a name="line.39"></a>
<span class="sourceLineNo">040</span><a name="line.40"></a>
<span class="sourceLineNo">041</span>    static public LinkedHashMap&lt;String, ChromosomeSequence&gt; loadFastaAddGeneFeaturesFromUpperCaseExonFastaFile(File fastaSequenceFile, File uppercaseFastaFile, boolean throwExceptionGeneNotFound) throws Exception {<a name="line.41"></a>
<span class="sourceLineNo">042</span>        LinkedHashMap&lt;String, ChromosomeSequence&gt; chromosomeSequenceList = new LinkedHashMap&lt;String, ChromosomeSequence&gt;();<a name="line.42"></a>
<span class="sourceLineNo">043</span>        LinkedHashMap&lt;String, DNASequence&gt; dnaSequenceList = FastaReaderHelper.readFastaDNASequence(fastaSequenceFile);<a name="line.43"></a>
<span class="sourceLineNo">044</span>        for (String accession : dnaSequenceList.keySet()) {<a name="line.44"></a>
<span class="sourceLineNo">045</span>            DNASequence contigSequence = dnaSequenceList.get(accession);<a name="line.45"></a>
<span class="sourceLineNo">046</span>            ChromosomeSequence chromsomeSequence = new ChromosomeSequence(contigSequence.getSequenceAsString());<a name="line.46"></a>
<span class="sourceLineNo">047</span>            chromsomeSequence.setAccession(contigSequence.getAccession());<a name="line.47"></a>
<span class="sourceLineNo">048</span>            chromosomeSequenceList.put(accession, chromsomeSequence);<a name="line.48"></a>
<span class="sourceLineNo">049</span>        }<a name="line.49"></a>
<span class="sourceLineNo">050</span><a name="line.50"></a>
<span class="sourceLineNo">051</span><a name="line.51"></a>
<span class="sourceLineNo">052</span>        LinkedHashMap&lt;String, DNASequence&gt; geneSequenceList = FastaReaderHelper.readFastaDNASequence(uppercaseFastaFile);<a name="line.52"></a>
<span class="sourceLineNo">053</span>        for (DNASequence dnaSequence : geneSequenceList.values()) {<a name="line.53"></a>
<span class="sourceLineNo">054</span>            String geneSequence = dnaSequence.getSequenceAsString();<a name="line.54"></a>
<span class="sourceLineNo">055</span>            String lcGeneSequence = geneSequence.toLowerCase();<a name="line.55"></a>
<span class="sourceLineNo">056</span>            String reverseGeneSequence = dnaSequence.getReverse().getSequenceAsString();<a name="line.56"></a>
<span class="sourceLineNo">057</span>            String lcReverseGeneSequence = reverseGeneSequence.toLowerCase();<a name="line.57"></a>
<span class="sourceLineNo">058</span>            Integer bioStart = null;<a name="line.58"></a>
<span class="sourceLineNo">059</span>            Integer bioEnd = null;<a name="line.59"></a>
<span class="sourceLineNo">060</span>            Strand strand = Strand.POSITIVE;<a name="line.60"></a>
<span class="sourceLineNo">061</span>            boolean geneFound = false;<a name="line.61"></a>
<span class="sourceLineNo">062</span>            String accession = "";<a name="line.62"></a>
<span class="sourceLineNo">063</span>            DNASequence contigDNASequence = null;<a name="line.63"></a>
<span class="sourceLineNo">064</span>            for (String id : dnaSequenceList.keySet()) {<a name="line.64"></a>
<span class="sourceLineNo">065</span>                accession = id;<a name="line.65"></a>
<span class="sourceLineNo">066</span>                contigDNASequence = dnaSequenceList.get(id);<a name="line.66"></a>
<span class="sourceLineNo">067</span>                String contigSequence = contigDNASequence.getSequenceAsString().toLowerCase();<a name="line.67"></a>
<span class="sourceLineNo">068</span>                bioStart = contigSequence.indexOf(lcGeneSequence);<a name="line.68"></a>
<span class="sourceLineNo">069</span>                if (bioStart != -1) {<a name="line.69"></a>
<span class="sourceLineNo">070</span>                    bioStart = bioStart + 1;<a name="line.70"></a>
<span class="sourceLineNo">071</span>                    bioEnd = bioStart + geneSequence.length() - 1;<a name="line.71"></a>
<span class="sourceLineNo">072</span>                    geneFound = true;<a name="line.72"></a>
<span class="sourceLineNo">073</span>                    break;<a name="line.73"></a>
<span class="sourceLineNo">074</span>                } else {<a name="line.74"></a>
<span class="sourceLineNo">075</span>                    bioStart = contigSequence.indexOf(lcReverseGeneSequence);<a name="line.75"></a>
<span class="sourceLineNo">076</span>                    if (bioStart != -1) {<a name="line.76"></a>
<span class="sourceLineNo">077</span>                        bioStart = bioStart + 1;<a name="line.77"></a>
<span class="sourceLineNo">078</span>                        bioEnd = bioStart - geneSequence.length() - 1;<a name="line.78"></a>
<span class="sourceLineNo">079</span>                        strand = Strand.NEGATIVE;<a name="line.79"></a>
<span class="sourceLineNo">080</span>                        geneFound = true;<a name="line.80"></a>
<span class="sourceLineNo">081</span>                        break;<a name="line.81"></a>
<span class="sourceLineNo">082</span>                    }<a name="line.82"></a>
<span class="sourceLineNo">083</span>                }<a name="line.83"></a>
<span class="sourceLineNo">084</span>            }<a name="line.84"></a>
<span class="sourceLineNo">085</span><a name="line.85"></a>
<span class="sourceLineNo">086</span>            if (geneFound) {<a name="line.86"></a>
<span class="sourceLineNo">087</span>                System.out.println("Gene " + dnaSequence.getAccession().toString() + " found at " + contigDNASequence.getAccession().toString() + " " + bioStart + " " + bioEnd + " " + strand);<a name="line.87"></a>
<span class="sourceLineNo">088</span>                ChromosomeSequence chromosomeSequence = chromosomeSequenceList.get(accession);<a name="line.88"></a>
<span class="sourceLineNo">089</span><a name="line.89"></a>
<span class="sourceLineNo">090</span>                ArrayList&lt;Integer&gt; exonBoundries = new ArrayList&lt;Integer&gt;();<a name="line.90"></a>
<span class="sourceLineNo">091</span><a name="line.91"></a>
<span class="sourceLineNo">092</span>                //look for transitions from lowercase to upper case<a name="line.92"></a>
<span class="sourceLineNo">093</span>                for (int i = 0; i &lt; geneSequence.length(); i++) {<a name="line.93"></a>
<span class="sourceLineNo">094</span>                    if (i == 0 &amp;&amp; Character.isUpperCase(geneSequence.charAt(i))) {<a name="line.94"></a>
<span class="sourceLineNo">095</span>                        exonBoundries.add(i);<a name="line.95"></a>
<span class="sourceLineNo">096</span>                    } else if (i == geneSequence.length() - 1) {<a name="line.96"></a>
<span class="sourceLineNo">097</span>                        exonBoundries.add(i);<a name="line.97"></a>
<span class="sourceLineNo">098</span>                    } else if (Character.isUpperCase(geneSequence.charAt(i)) &amp;&amp; Character.isLowerCase(geneSequence.charAt(i - 1))) {<a name="line.98"></a>
<span class="sourceLineNo">099</span>                        exonBoundries.add(i);<a name="line.99"></a>
<span class="sourceLineNo">100</span>                    } else if (Character.isUpperCase(geneSequence.charAt(i)) &amp;&amp; Character.isLowerCase(geneSequence.charAt(i + 1))) {<a name="line.100"></a>
<span class="sourceLineNo">101</span>                        exonBoundries.add(i);<a name="line.101"></a>
<span class="sourceLineNo">102</span>                    }<a name="line.102"></a>
<span class="sourceLineNo">103</span>                }<a name="line.103"></a>
<span class="sourceLineNo">104</span>                if (strand == Strand.NEGATIVE) {<a name="line.104"></a>
<span class="sourceLineNo">105</span>                    Collections.reverse(exonBoundries);<a name="line.105"></a>
<span class="sourceLineNo">106</span>                }<a name="line.106"></a>
<span class="sourceLineNo">107</span><a name="line.107"></a>
<span class="sourceLineNo">108</span><a name="line.108"></a>
<span class="sourceLineNo">109</span>                String geneaccession = dnaSequence.getAccession().getID();<a name="line.109"></a>
<span class="sourceLineNo">110</span>                String note = geneaccession;<a name="line.110"></a>
<span class="sourceLineNo">111</span>                String[] values = geneaccession.split(" ");<a name="line.111"></a>
<span class="sourceLineNo">112</span>                geneaccession = values[0];<a name="line.112"></a>
<span class="sourceLineNo">113</span><a name="line.113"></a>
<span class="sourceLineNo">114</span><a name="line.114"></a>
<span class="sourceLineNo">115</span><a name="line.115"></a>
<span class="sourceLineNo">116</span>                GeneSequence geneSeq = chromosomeSequence.addGene(new AccessionID(geneaccession), bioStart, bioEnd, strand);<a name="line.116"></a>
<span class="sourceLineNo">117</span>                geneSeq.addNote(note);<a name="line.117"></a>
<span class="sourceLineNo">118</span>                geneSeq.setSource(uppercaseFastaFile.getName());<a name="line.118"></a>
<span class="sourceLineNo">119</span>                String transcriptName = geneaccession + "-transcript";<a name="line.119"></a>
<span class="sourceLineNo">120</span>                TranscriptSequence transcriptSequence = geneSeq.addTranscript(new AccessionID(transcriptName), bioStart, bioEnd);<a name="line.120"></a>
<span class="sourceLineNo">121</span><a name="line.121"></a>
<span class="sourceLineNo">122</span>                int runningFrameLength = 0;<a name="line.122"></a>
<span class="sourceLineNo">123</span>                for (int i = 0; i &lt; exonBoundries.size() - 1; i = i + 2) {<a name="line.123"></a>
<span class="sourceLineNo">124</span>                    int cdsBioStart = exonBoundries.get(i) + bioStart;<a name="line.124"></a>
<span class="sourceLineNo">125</span>                    int cdsBioEnd = exonBoundries.get(i + 1) + bioStart;<a name="line.125"></a>
<span class="sourceLineNo">126</span>                    runningFrameLength = runningFrameLength + Math.abs(cdsBioEnd - cdsBioStart) + 1;<a name="line.126"></a>
<span class="sourceLineNo">127</span>                    String cdsName = transcriptName + "-cds-" + cdsBioStart + "-" + cdsBioEnd;<a name="line.127"></a>
<span class="sourceLineNo">128</span><a name="line.128"></a>
<span class="sourceLineNo">129</span>                    AccessionID cdsAccessionID = new AccessionID(cdsName);<a name="line.129"></a>
<span class="sourceLineNo">130</span>                    ExonSequence exonSequence = geneSeq.addExon(cdsAccessionID, cdsBioStart, cdsBioEnd);<a name="line.130"></a>
<span class="sourceLineNo">131</span>                    int remainder = runningFrameLength % 3;<a name="line.131"></a>
<span class="sourceLineNo">132</span>                    int frame = 0;<a name="line.132"></a>
<span class="sourceLineNo">133</span>                    if (remainder == 1) {<a name="line.133"></a>
<span class="sourceLineNo">134</span>                        frame = 2; // borrow 2 from next CDS region<a name="line.134"></a>
<span class="sourceLineNo">135</span>                    } else if (remainder == 2) {<a name="line.135"></a>
<span class="sourceLineNo">136</span>                        frame = 1;<a name="line.136"></a>
<span class="sourceLineNo">137</span>                    }<a name="line.137"></a>
<span class="sourceLineNo">138</span>                    CDSSequence cdsSequence = transcriptSequence.addCDS(cdsAccessionID, cdsBioStart, cdsBioEnd, frame);<a name="line.138"></a>
<span class="sourceLineNo">139</span>                }<a name="line.139"></a>
<span class="sourceLineNo">140</span><a name="line.140"></a>
<span class="sourceLineNo">141</span><a name="line.141"></a>
<span class="sourceLineNo">142</span>            } else {<a name="line.142"></a>
<span class="sourceLineNo">143</span>                if (throwExceptionGeneNotFound) {<a name="line.143"></a>
<span class="sourceLineNo">144</span>                    throw new Exception(dnaSequence.getAccession().toString() + " not found");<a name="line.144"></a>
<span class="sourceLineNo">145</span>                }<a name="line.145"></a>
<span class="sourceLineNo">146</span>                System.out.println("Gene not found " + dnaSequence.getAccession().toString());<a name="line.146"></a>
<span class="sourceLineNo">147</span>            }<a name="line.147"></a>
<span class="sourceLineNo">148</span><a name="line.148"></a>
<span class="sourceLineNo">149</span>        }<a name="line.149"></a>
<span class="sourceLineNo">150</span>        return chromosomeSequenceList;<a name="line.150"></a>
<span class="sourceLineNo">151</span>    }<a name="line.151"></a>
<span class="sourceLineNo">152</span><a name="line.152"></a>
<span class="sourceLineNo">153</span>    /**<a name="line.153"></a>
<span class="sourceLineNo">154</span>     * Output a gff3 feature file that will give the length of each scaffold/chromosome in the fasta file.<a name="line.154"></a>
<span class="sourceLineNo">155</span>     * Used for gbrowse so it knows length.<a name="line.155"></a>
<span class="sourceLineNo">156</span>     * @param fastaSequenceFile<a name="line.156"></a>
<span class="sourceLineNo">157</span>     * @param gffFile<a name="line.157"></a>
<span class="sourceLineNo">158</span>     * @throws Exception<a name="line.158"></a>
<span class="sourceLineNo">159</span>     */<a name="line.159"></a>
<span class="sourceLineNo">160</span>    static public void outputFastaSequenceLengthGFF3(File fastaSequenceFile, File gffFile) throws Exception {<a name="line.160"></a>
<span class="sourceLineNo">161</span>        LinkedHashMap&lt;String, DNASequence&gt; dnaSequenceList = FastaReaderHelper.readFastaDNASequence(fastaSequenceFile);<a name="line.161"></a>
<span class="sourceLineNo">162</span>        String fileName = fastaSequenceFile.getName();<a name="line.162"></a>
<span class="sourceLineNo">163</span>        FileWriter fw = new FileWriter(gffFile);<a name="line.163"></a>
<span class="sourceLineNo">164</span>        fw.write("##gff-version 3\n");<a name="line.164"></a>
<span class="sourceLineNo">165</span>        for (DNASequence dnaSequence : dnaSequenceList.values()) {<a name="line.165"></a>
<span class="sourceLineNo">166</span><a name="line.166"></a>
<span class="sourceLineNo">167</span>            String gff3line = dnaSequence.getAccession().getID() + "\t" + fileName + "\t" + "contig" + "\t" + "1" + "\t" + dnaSequence.getBioEnd() + "\t.\t.\t.\tName=" + dnaSequence.getAccession().getID() + "\n";<a name="line.167"></a>
<span class="sourceLineNo">168</span>            fw.write(gff3line);<a name="line.168"></a>
<span class="sourceLineNo">169</span>        }<a name="line.169"></a>
<span class="sourceLineNo">170</span>        fw.close();<a name="line.170"></a>
<span class="sourceLineNo">171</span>    }<a name="line.171"></a>
<span class="sourceLineNo">172</span><a name="line.172"></a>
<span class="sourceLineNo">173</span>    /**<a name="line.173"></a>
<span class="sourceLineNo">174</span>     * Loads Fasta file and GFF2 feature file generated from the geneid prediction algorithm<a name="line.174"></a>
<span class="sourceLineNo">175</span>     *<a name="line.175"></a>
<span class="sourceLineNo">176</span>     * @param fastaSequenceFile<a name="line.176"></a>
<span class="sourceLineNo">177</span>     * @param gffFile<a name="line.177"></a>
<span class="sourceLineNo">178</span>     * @return<a name="line.178"></a>
<span class="sourceLineNo">179</span>     * @throws Exception<a name="line.179"></a>
<span class="sourceLineNo">180</span>     */<a name="line.180"></a>
<span class="sourceLineNo">181</span>    static public LinkedHashMap&lt;String, ChromosomeSequence&gt; loadFastaAddGeneFeaturesFromGeneIDGFF2(File fastaSequenceFile, File gffFile) throws Exception {<a name="line.181"></a>
<span class="sourceLineNo">182</span>        LinkedHashMap&lt;String, DNASequence&gt; dnaSequenceList = FastaReaderHelper.readFastaDNASequence(fastaSequenceFile);<a name="line.182"></a>
<span class="sourceLineNo">183</span>        LinkedHashMap&lt;String, ChromosomeSequence&gt; chromosomeSequenceList = GeneFeatureHelper.getChromosomeSequenceFromDNASequence(dnaSequenceList);<a name="line.183"></a>
<span class="sourceLineNo">184</span>        FeatureList listGenes = GeneIDGFF2Reader.read(gffFile.getAbsolutePath());<a name="line.184"></a>
<span class="sourceLineNo">185</span>        addGeneIDGFF2GeneFeatures(chromosomeSequenceList, listGenes);<a name="line.185"></a>
<span class="sourceLineNo">186</span>        return chromosomeSequenceList;<a name="line.186"></a>
<span class="sourceLineNo">187</span>    }<a name="line.187"></a>
<span class="sourceLineNo">188</span><a name="line.188"></a>
<span class="sourceLineNo">189</span>    /**<a name="line.189"></a>
<span class="sourceLineNo">190</span>     * Load GFF2 feature file generated from the geneid prediction algorithm and map features onto the chromosome sequences<a name="line.190"></a>
<span class="sourceLineNo">191</span>     *<a name="line.191"></a>
<span class="sourceLineNo">192</span>     * @param chromosomeSequenceList<a name="line.192"></a>
<span class="sourceLineNo">193</span>     * @param listGenes<a name="line.193"></a>
<span class="sourceLineNo">194</span>     * @throws Exception<a name="line.194"></a>
<span class="sourceLineNo">195</span>     */<a name="line.195"></a>
<span class="sourceLineNo">196</span>    static public void addGeneIDGFF2GeneFeatures(LinkedHashMap&lt;String, ChromosomeSequence&gt; chromosomeSequenceList, FeatureList listGenes) throws Exception {<a name="line.196"></a>
<span class="sourceLineNo">197</span>        Collection&lt;String&gt; geneIds = listGenes.attributeValues("gene_id");<a name="line.197"></a>
<span class="sourceLineNo">198</span>        for (String geneid : geneIds) {<a name="line.198"></a>
<span class="sourceLineNo">199</span>            FeatureList gene = listGenes.selectByAttribute("gene_id", geneid);<a name="line.199"></a>
<span class="sourceLineNo">200</span>            FeatureI geneFeature = gene.get(0);<a name="line.200"></a>
<span class="sourceLineNo">201</span>            ChromosomeSequence seq = chromosomeSequenceList.get(geneFeature.seqname());<a name="line.201"></a>
<span class="sourceLineNo">202</span>            geneid = geneid.replaceAll("_", ".G");<a name="line.202"></a>
<span class="sourceLineNo">203</span>            AccessionID geneAccessionID = new AccessionID(geneid);<a name="line.203"></a>
<span class="sourceLineNo">204</span>            GeneSequence geneSequence = null;<a name="line.204"></a>
<span class="sourceLineNo">205</span>            Collection&lt;String&gt; transcriptids = gene.attributeValues("gene_id");<a name="line.205"></a>
<span class="sourceLineNo">206</span>            for (String transcriptid : transcriptids) {<a name="line.206"></a>
<span class="sourceLineNo">207</span>                // get all the individual features (exons, CDS regions, etc.) of this gene<a name="line.207"></a>
<span class="sourceLineNo">208</span>                FeatureList transcriptFeature = listGenes.selectByAttribute("gene_id", transcriptid);<a name="line.208"></a>
<span class="sourceLineNo">209</span>                transcriptid = transcriptid.replaceAll("_", ".G");<a name="line.209"></a>
<span class="sourceLineNo">210</span><a name="line.210"></a>
<span class="sourceLineNo">211</span><a name="line.211"></a>
<span class="sourceLineNo">212</span><a name="line.212"></a>
<span class="sourceLineNo">213</span><a name="line.213"></a>
<span class="sourceLineNo">214</span>                //      String seqName = feature.seqname();<a name="line.214"></a>
<span class="sourceLineNo">215</span>                FeatureI startCodon = null;<a name="line.215"></a>
<span class="sourceLineNo">216</span>                FeatureI stopCodon = null;<a name="line.216"></a>
<span class="sourceLineNo">217</span>                Integer startCodonBegin = null;<a name="line.217"></a>
<span class="sourceLineNo">218</span>                Integer stopCodonEnd = null;<a name="line.218"></a>
<span class="sourceLineNo">219</span>                String startCodonName = "";<a name="line.219"></a>
<span class="sourceLineNo">220</span>                String stopCodonName = "";<a name="line.220"></a>
<span class="sourceLineNo">221</span><a name="line.221"></a>
<span class="sourceLineNo">222</span><a name="line.222"></a>
<span class="sourceLineNo">223</span>                // now select only the coding regions of this gene<a name="line.223"></a>
<span class="sourceLineNo">224</span>                FeatureList firstFeatures = transcriptFeature.selectByType("First");<a name="line.224"></a>
<span class="sourceLineNo">225</span>                FeatureList terminalFeatures = transcriptFeature.selectByType("Terminal");<a name="line.225"></a>
<span class="sourceLineNo">226</span>                FeatureList internalFeatures = transcriptFeature.selectByType("Internal");<a name="line.226"></a>
<span class="sourceLineNo">227</span>                FeatureList singleFeatures = transcriptFeature.selectByType("Single");<a name="line.227"></a>
<span class="sourceLineNo">228</span>                FeatureList cdsFeatures = new FeatureList();<a name="line.228"></a>
<span class="sourceLineNo">229</span>                cdsFeatures.add(firstFeatures);<a name="line.229"></a>
<span class="sourceLineNo">230</span>                cdsFeatures.add(terminalFeatures);<a name="line.230"></a>
<span class="sourceLineNo">231</span>                cdsFeatures.add(internalFeatures);<a name="line.231"></a>
<span class="sourceLineNo">232</span>                cdsFeatures.add(singleFeatures);<a name="line.232"></a>
<span class="sourceLineNo">233</span>                // sort them<a name="line.233"></a>
<span class="sourceLineNo">234</span>                cdsFeatures = cdsFeatures.sortByStart();<a name="line.234"></a>
<span class="sourceLineNo">235</span>                Strand strand = Strand.POSITIVE;<a name="line.235"></a>
<span class="sourceLineNo">236</span>                FeatureI feature = cdsFeatures.get(0);<a name="line.236"></a>
<span class="sourceLineNo">237</span>                if (feature.location().isNegative()) {<a name="line.237"></a>
<span class="sourceLineNo">238</span>                    strand = strand.NEGATIVE;<a name="line.238"></a>
<span class="sourceLineNo">239</span>                }<a name="line.239"></a>
<span class="sourceLineNo">240</span>                if (startCodonBegin == null) {<a name="line.240"></a>
<span class="sourceLineNo">241</span>                    FeatureI firstFeature = cdsFeatures.get(0);<a name="line.241"></a>
<span class="sourceLineNo">242</span>                    if (strand == strand.NEGATIVE) {<a name="line.242"></a>
<span class="sourceLineNo">243</span>                        startCodonBegin = firstFeature.location().bioEnd();<a name="line.243"></a>
<span class="sourceLineNo">244</span>                    } else {<a name="line.244"></a>
<span class="sourceLineNo">245</span>                        startCodonBegin = firstFeature.location().bioStart();<a name="line.245"></a>
<span class="sourceLineNo">246</span>                    }<a name="line.246"></a>
<span class="sourceLineNo">247</span>                }<a name="line.247"></a>
<span class="sourceLineNo">248</span><a name="line.248"></a>
<span class="sourceLineNo">249</span>                if (stopCodonEnd == null) {<a name="line.249"></a>
<span class="sourceLineNo">250</span><a name="line.250"></a>
<span class="sourceLineNo">251</span>                    FeatureI lastFeature = cdsFeatures.get(cdsFeatures.size() - 1);<a name="line.251"></a>
<span class="sourceLineNo">252</span>                    if (strand == strand.NEGATIVE) {<a name="line.252"></a>
<span class="sourceLineNo">253</span>                        stopCodonEnd = lastFeature.location().bioStart();<a name="line.253"></a>
<span class="sourceLineNo">254</span>                    } else {<a name="line.254"></a>
<span class="sourceLineNo">255</span>                        stopCodonEnd = lastFeature.location().bioEnd();<a name="line.255"></a>
<span class="sourceLineNo">256</span>                    }<a name="line.256"></a>
<span class="sourceLineNo">257</span>                }<a name="line.257"></a>
<span class="sourceLineNo">258</span>                //for gtf ordering can be strand based so first is last and last is first<a name="line.258"></a>
<span class="sourceLineNo">259</span>                if (startCodonBegin &gt; stopCodonEnd) {<a name="line.259"></a>
<span class="sourceLineNo">260</span>                    int temp = startCodonBegin;<a name="line.260"></a>
<span class="sourceLineNo">261</span>                    startCodonBegin = stopCodonEnd;<a name="line.261"></a>
<span class="sourceLineNo">262</span>                    stopCodonEnd = temp;<a name="line.262"></a>
<span class="sourceLineNo">263</span>                }<a name="line.263"></a>
<span class="sourceLineNo">264</span><a name="line.264"></a>
<span class="sourceLineNo">265</span>                AccessionID transcriptAccessionID = new AccessionID(transcriptid);<a name="line.265"></a>
<span class="sourceLineNo">266</span>                if (geneSequence == null) {<a name="line.266"></a>
<span class="sourceLineNo">267</span>                    geneSequence = seq.addGene(geneAccessionID, startCodonBegin, stopCodonEnd, strand);<a name="line.267"></a>
<span class="sourceLineNo">268</span>                    geneSequence.setSource(((Feature) feature).source());<a name="line.268"></a>
<span class="sourceLineNo">269</span>                } else {<a name="line.269"></a>
<span class="sourceLineNo">270</span>                    //if multiple transcripts for one gene make sure the gene is defined as the min and max start/end<a name="line.270"></a>
<span class="sourceLineNo">271</span><a name="line.271"></a>
<span class="sourceLineNo">272</span>                    if (startCodonBegin &lt; geneSequence.getBioBegin()) {<a name="line.272"></a>
<span class="sourceLineNo">273</span>                        geneSequence.setBioBegin(startCodonBegin);<a name="line.273"></a>
<span class="sourceLineNo">274</span>                    }<a name="line.274"></a>
<span class="sourceLineNo">275</span>                    if (stopCodonEnd &gt; geneSequence.getBioBegin()) {<a name="line.275"></a>
<span class="sourceLineNo">276</span>                        geneSequence.setBioEnd(stopCodonEnd);<a name="line.276"></a>
<span class="sourceLineNo">277</span>                    }<a name="line.277"></a>
<span class="sourceLineNo">278</span><a name="line.278"></a>
<span class="sourceLineNo">279</span>                }<a name="line.279"></a>
<span class="sourceLineNo">280</span>                TranscriptSequence transcriptSequence = geneSequence.addTranscript(transcriptAccessionID, startCodonBegin, stopCodonEnd);<a name="line.280"></a>
<span class="sourceLineNo">281</span>                if (startCodon != null) {<a name="line.281"></a>
<span class="sourceLineNo">282</span>                    if (startCodonName == null || startCodonName.length() == 0) {<a name="line.282"></a>
<span class="sourceLineNo">283</span>                        startCodonName = transcriptid + "-start_codon-" + startCodon.location().bioStart() + "-" + startCodon.location().bioEnd();<a name="line.283"></a>
<span class="sourceLineNo">284</span>                    }<a name="line.284"></a>
<span class="sourceLineNo">285</span>                    transcriptSequence.addStartCodonSequence(new AccessionID(startCodonName), startCodon.location().bioStart(), startCodon.location().bioEnd());<a name="line.285"></a>
<span class="sourceLineNo">286</span>                }<a name="line.286"></a>
<span class="sourceLineNo">287</span>                if (stopCodon != null) {<a name="line.287"></a>
<span class="sourceLineNo">288</span>                    if (stopCodonName == null || stopCodonName.length() == 0) {<a name="line.288"></a>
<span class="sourceLineNo">289</span>                        stopCodonName = transcriptid + "-stop_codon-" + stopCodon.location().bioStart() + "-" + stopCodon.location().bioEnd();<a name="line.289"></a>
<span class="sourceLineNo">290</span>                    }<a name="line.290"></a>
<span class="sourceLineNo">291</span>                    transcriptSequence.addStopCodonSequence(new AccessionID(stopCodonName), stopCodon.location().bioStart(), stopCodon.location().bioEnd());<a name="line.291"></a>
<span class="sourceLineNo">292</span>                }<a name="line.292"></a>
<span class="sourceLineNo">293</span><a name="line.293"></a>
<span class="sourceLineNo">294</span>                for (FeatureI cdsFeature : cdsFeatures) {<a name="line.294"></a>
<span class="sourceLineNo">295</span>                    Feature cds = (Feature) cdsFeature;<a name="line.295"></a>
<span class="sourceLineNo">296</span>                    String cdsName = cds.getAttribute("transcript_name");<a name="line.296"></a>
<span class="sourceLineNo">297</span>                    if (cdsName == null || cdsName.length() == 0) {<a name="line.297"></a>
<span class="sourceLineNo">298</span>                        cdsName = transcriptid + "-cds-" + cds.location().bioStart() + "-" + cds.location().bioEnd();<a name="line.298"></a>
<span class="sourceLineNo">299</span>                    }<a name="line.299"></a>
<span class="sourceLineNo">300</span>                    AccessionID cdsAccessionID = new AccessionID(cdsName);<a name="line.300"></a>
<span class="sourceLineNo">301</span>                    ExonSequence exonSequence = geneSequence.addExon(cdsAccessionID, cdsFeature.location().bioStart(), cdsFeature.location().bioEnd());<a name="line.301"></a>
<span class="sourceLineNo">302</span>                    CDSSequence cdsSequence = transcriptSequence.addCDS(cdsAccessionID, cdsFeature.location().bioStart(), cdsFeature.location().bioEnd(), cds.frame());<a name="line.302"></a>
<span class="sourceLineNo">303</span>                    cdsSequence.setSequenceScore(cds.score());<a name="line.303"></a>
<span class="sourceLineNo">304</span>                }<a name="line.304"></a>
<span class="sourceLineNo">305</span>            }<a name="line.305"></a>
<span class="sourceLineNo">306</span>        }<a name="line.306"></a>
<span class="sourceLineNo">307</span><a name="line.307"></a>
<span class="sourceLineNo">308</span>    }<a name="line.308"></a>
<span class="sourceLineNo">309</span><a name="line.309"></a>
<span class="sourceLineNo">310</span>    static public LinkedHashMap&lt;String, ChromosomeSequence&gt; getChromosomeSequenceFromDNASequence(LinkedHashMap&lt;String, DNASequence&gt; dnaSequenceList) {<a name="line.310"></a>
<span class="sourceLineNo">311</span>        LinkedHashMap&lt;String, ChromosomeSequence&gt; chromosomeSequenceList = new LinkedHashMap&lt;String, ChromosomeSequence&gt;();<a name="line.311"></a>
<span class="sourceLineNo">312</span>        for (String key : dnaSequenceList.keySet()) {<a name="line.312"></a>
<span class="sourceLineNo">313</span>            DNASequence dnaSequence = dnaSequenceList.get(key);<a name="line.313"></a>
<span class="sourceLineNo">314</span>            ChromosomeSequence chromosomeSequence = new ChromosomeSequence(dnaSequence.getProxySequenceReader()); //we want the underlying sequence but don't need storage<a name="line.314"></a>
<span class="sourceLineNo">315</span>            chromosomeSequence.setAccession(dnaSequence.getAccession());<a name="line.315"></a>
<span class="sourceLineNo">316</span>            chromosomeSequenceList.put(key, chromosomeSequence);<a name="line.316"></a>
<span class="sourceLineNo">317</span>        }<a name="line.317"></a>
<span class="sourceLineNo">318</span>        return chromosomeSequenceList;<a name="line.318"></a>
<span class="sourceLineNo">319</span>    }<a name="line.319"></a>
<span class="sourceLineNo">320</span><a name="line.320"></a>
<span class="sourceLineNo">321</span>    /**<a name="line.321"></a>
<span class="sourceLineNo">322</span>     * Lots of variations in the ontology or descriptors that can be used in GFF3 which requires writing a custom parser to handle a GFF3 generated or used<a name="line.322"></a>
<span class="sourceLineNo">323</span>     * by a specific application. Probably could be abstracted out but for now easier to handle with custom code to deal with gff3 elements that are not<a name="line.323"></a>
<span class="sourceLineNo">324</span>     * included but can be extracted from other data elements.<a name="line.324"></a>
<span class="sourceLineNo">325</span>     * @param fastaSequenceFile<a name="line.325"></a>
<span class="sourceLineNo">326</span>     * @param gffFile<a name="line.326"></a>
<span class="sourceLineNo">327</span>     * @param lazyloadsequences If set to true then the fasta file will be parsed for accession id but sequences will be read from disk when needed to save memory<a name="line.327"></a>
<span class="sourceLineNo">328</span>     * @return<a name="line.328"></a>
<span class="sourceLineNo">329</span>     * @throws Exception<a name="line.329"></a>
<span class="sourceLineNo">330</span>     */<a name="line.330"></a>
<span class="sourceLineNo">331</span>    static public LinkedHashMap&lt;String, ChromosomeSequence&gt; loadFastaAddGeneFeaturesFromGmodGFF3(File fastaSequenceFile, File gffFile,boolean lazyloadsequences) throws Exception {<a name="line.331"></a>
<span class="sourceLineNo">332</span>        LinkedHashMap&lt;String, DNASequence&gt; dnaSequenceList = FastaReaderHelper.readFastaDNASequence(fastaSequenceFile,lazyloadsequences);<a name="line.332"></a>
<span class="sourceLineNo">333</span>        LinkedHashMap&lt;String, ChromosomeSequence&gt; chromosomeSequenceList = GeneFeatureHelper.getChromosomeSequenceFromDNASequence(dnaSequenceList);<a name="line.333"></a>
<span class="sourceLineNo">334</span>        FeatureList listGenes = GFF3Reader.read(gffFile.getAbsolutePath());<a name="line.334"></a>
<span class="sourceLineNo">335</span>        addGmodGFF3GeneFeatures(chromosomeSequenceList, listGenes);<a name="line.335"></a>
<span class="sourceLineNo">336</span>        return chromosomeSequenceList;<a name="line.336"></a>
<span class="sourceLineNo">337</span>    }<a name="line.337"></a>
<span class="sourceLineNo">338</span><a name="line.338"></a>
<span class="sourceLineNo">339</span>    /**<a name="line.339"></a>
<span class="sourceLineNo">340</span>     * Load GFF3 file using mRNA as the gene feature as not all GFF3 files are complete<a name="line.340"></a>
<span class="sourceLineNo">341</span>     * @param chromosomeSequenceList<a name="line.341"></a>
<span class="sourceLineNo">342</span>     * @param listGenes<a name="line.342"></a>
<span class="sourceLineNo">343</span>     * @throws Exception<a name="line.343"></a>
<span class="sourceLineNo">344</span>     */<a name="line.344"></a>
<span class="sourceLineNo">345</span>    static public void addGmodGFF3GeneFeatures(LinkedHashMap&lt;String, ChromosomeSequence&gt; chromosomeSequenceList, FeatureList listGenes) throws Exception {<a name="line.345"></a>
<span class="sourceLineNo">346</span><a name="line.346"></a>
<span class="sourceLineNo">347</span><a name="line.347"></a>
<span class="sourceLineNo">348</span>        // key off mRNA as being a known feature that may or may not have a parent gene<a name="line.348"></a>
<span class="sourceLineNo">349</span><a name="line.349"></a>
<span class="sourceLineNo">350</span><a name="line.350"></a>
<span class="sourceLineNo">351</span>        FeatureList mRNAFeatures = listGenes.selectByType("mRNA");<a name="line.351"></a>
<span class="sourceLineNo">352</span>        LinkedHashMap&lt;String,FeatureList&gt; featureIDHashMap = FeatureHelper.buildFeatureAtrributeIndex("ID", listGenes);<a name="line.352"></a>
<span class="sourceLineNo">353</span>        LinkedHashMap&lt;String,FeatureList&gt; featureParentHashMap = FeatureHelper.buildFeatureAtrributeIndex("Parent", listGenes);<a name="line.353"></a>
<span class="sourceLineNo">354</span><a name="line.354"></a>
<span class="sourceLineNo">355</span>        for (FeatureI f : mRNAFeatures) {<a name="line.355"></a>
<span class="sourceLineNo">356</span>            String geneID;<a name="line.356"></a>
<span class="sourceLineNo">357</span>            String geneNote = null;<a name="line.357"></a>
<span class="sourceLineNo">358</span>            String geneSource = null;<a name="line.358"></a>
<span class="sourceLineNo">359</span>            String sequenceName = null;<a name="line.359"></a>
<span class="sourceLineNo">360</span>            ChromosomeSequence seq = null;<a name="line.360"></a>
<span class="sourceLineNo">361</span>            GeneSequence geneSequence = null;<a name="line.361"></a>
<span class="sourceLineNo">362</span><a name="line.362"></a>
<span class="sourceLineNo">363</span>            Feature mRNAFeature = (Feature) f;<a name="line.363"></a>
<span class="sourceLineNo">364</span>            String mRNAID = mRNAFeature.getAttribute("ID");<a name="line.364"></a>
<span class="sourceLineNo">365</span>            String mRNAsource = mRNAFeature.source();<a name="line.365"></a>
<span class="sourceLineNo">366</span>            String mRNANote = mRNAFeature.getAttribute("Note");<a name="line.366"></a>
<span class="sourceLineNo">367</span>            String mRNAParent = mRNAFeature.getAttribute("Parent");<a name="line.367"></a>
<span class="sourceLineNo">368</span>            if (mRNAParent != null &amp;&amp; mRNAParent.length() &gt; 0) {<a name="line.368"></a>
<span class="sourceLineNo">369</span>               // FeatureList geneFeatureList = listGenes.selectByAttribute("ID", mRNAParent);<a name="line.369"></a>
<span class="sourceLineNo">370</span>                FeatureList geneFeatureList = featureIDHashMap.get(mRNAParent);<a name="line.370"></a>
<span class="sourceLineNo">371</span>                Feature geneFeature = (Feature) geneFeatureList.get(0);<a name="line.371"></a>
<span class="sourceLineNo">372</span>                geneID = geneFeature.getAttribute("ID");<a name="line.372"></a>
<span class="sourceLineNo">373</span>                geneNote = geneFeature.getAttribute("Note");<a name="line.373"></a>
<span class="sourceLineNo">374</span>                geneSource = ((Feature) geneFeature).source();<a name="line.374"></a>
<span class="sourceLineNo">375</span>                sequenceName = geneFeature.seqname();<a name="line.375"></a>
<span class="sourceLineNo">376</span><a name="line.376"></a>
<span class="sourceLineNo">377</span>                //<a name="line.377"></a>
<span class="sourceLineNo">378</span>            } else {<a name="line.378"></a>
<span class="sourceLineNo">379</span>                //deal with cases where no parent gene is given<a name="line.379"></a>
<span class="sourceLineNo">380</span>                geneID = mRNAID;<a name="line.380"></a>
<span class="sourceLineNo">381</span>                geneSource = mRNAsource;<a name="line.381"></a>
<span class="sourceLineNo">382</span>                sequenceName = mRNAFeature.seqname();<a name="line.382"></a>
<span class="sourceLineNo">383</span>            }<a name="line.383"></a>
<span class="sourceLineNo">384</span><a name="line.384"></a>
<span class="sourceLineNo">385</span>            seq = chromosomeSequenceList.get(sequenceName);<a name="line.385"></a>
<span class="sourceLineNo">386</span><a name="line.386"></a>
<span class="sourceLineNo">387</span>            AccessionID geneAccessionID = new AccessionID(geneID);<a name="line.387"></a>
<span class="sourceLineNo">388</span><a name="line.388"></a>
<span class="sourceLineNo">389</span>          //  FeatureList mRNAChildren = listGenes.selectByAttribute("Parent", mRNAID);<a name="line.389"></a>
<span class="sourceLineNo">390</span>            FeatureList mRNAChildren = featureParentHashMap.get(mRNAID);<a name="line.390"></a>
<span class="sourceLineNo">391</span>            FeatureList cdsFeatures = mRNAChildren.selectByType("CDS");<a name="line.391"></a>
<span class="sourceLineNo">392</span>            FeatureI feature = cdsFeatures.get(0);<a name="line.392"></a>
<span class="sourceLineNo">393</span>            Strand strand = Strand.POSITIVE;<a name="line.393"></a>
<span class="sourceLineNo">394</span><a name="line.394"></a>
<span class="sourceLineNo">395</span>            if (feature.location().isNegative()) {<a name="line.395"></a>
<span class="sourceLineNo">396</span>                strand = strand.NEGATIVE;<a name="line.396"></a>
<span class="sourceLineNo">397</span>            }<a name="line.397"></a>
<span class="sourceLineNo">398</span>            cdsFeatures = cdsFeatures.sortByStart();<a name="line.398"></a>
<span class="sourceLineNo">399</span><a name="line.399"></a>
<span class="sourceLineNo">400</span><a name="line.400"></a>
<span class="sourceLineNo">401</span><a name="line.401"></a>
<span class="sourceLineNo">402</span><a name="line.402"></a>
<span class="sourceLineNo">403</span><a name="line.403"></a>
<span class="sourceLineNo">404</span><a name="line.404"></a>
<span class="sourceLineNo">405</span><a name="line.405"></a>
<span class="sourceLineNo">406</span>            String seqName = feature.seqname();<a name="line.406"></a>
<span class="sourceLineNo">407</span>            FeatureI startCodon = null;<a name="line.407"></a>
<span class="sourceLineNo">408</span>            FeatureI stopCodon = null;<a name="line.408"></a>
<span class="sourceLineNo">409</span>            Integer startCodonBegin = null;<a name="line.409"></a>
<span class="sourceLineNo">410</span>            Integer stopCodonEnd = null;<a name="line.410"></a>
<span class="sourceLineNo">411</span>            String startCodonName = "";<a name="line.411"></a>
<span class="sourceLineNo">412</span>            String stopCodonName = "";<a name="line.412"></a>
<span class="sourceLineNo">413</span>            FeatureList startCodonList = mRNAChildren.selectByType("five_prime_UTR");<a name="line.413"></a>
<span class="sourceLineNo">414</span>            if (startCodonList != null &amp;&amp; startCodonList.size() &gt; 0) {<a name="line.414"></a>
<span class="sourceLineNo">415</span>                startCodon = startCodonList.get(0);<a name="line.415"></a>
<span class="sourceLineNo">416</span>                if (strand == Strand.NEGATIVE) {<a name="line.416"></a>
<span class="sourceLineNo">417</span>                    startCodonBegin = startCodon.location().bioEnd();<a name="line.417"></a>
<span class="sourceLineNo">418</span>                } else {<a name="line.418"></a>
<span class="sourceLineNo">419</span>                    startCodonBegin = startCodon.location().bioStart();<a name="line.419"></a>
<span class="sourceLineNo">420</span>                }<a name="line.420"></a>
<span class="sourceLineNo">421</span>                startCodonName = startCodon.getAttribute("ID");<a name="line.421"></a>
<span class="sourceLineNo">422</span>            }<a name="line.422"></a>
<span class="sourceLineNo">423</span><a name="line.423"></a>
<span class="sourceLineNo">424</span>            FeatureList stopCodonList = mRNAChildren.selectByType("three_prime_UTR");<a name="line.424"></a>
<span class="sourceLineNo">425</span><a name="line.425"></a>
<span class="sourceLineNo">426</span>            if (stopCodonList != null &amp;&amp; stopCodonList.size() &gt; 0) {<a name="line.426"></a>
<span class="sourceLineNo">427</span>                stopCodon = stopCodonList.get(0);<a name="line.427"></a>
<span class="sourceLineNo">428</span>                if (strand == Strand.NEGATIVE) {<a name="line.428"></a>
<span class="sourceLineNo">429</span>                    stopCodonEnd = stopCodon.location().bioStart();<a name="line.429"></a>
<span class="sourceLineNo">430</span>                } else {<a name="line.430"></a>
<span class="sourceLineNo">431</span>                    stopCodonEnd = stopCodon.location().bioEnd();<a name="line.431"></a>
<span class="sourceLineNo">432</span>                }<a name="line.432"></a>
<span class="sourceLineNo">433</span>                stopCodonName = stopCodon.getAttribute("ID");<a name="line.433"></a>
<span class="sourceLineNo">434</span><a name="line.434"></a>
<span class="sourceLineNo">435</span>            }<a name="line.435"></a>
<span class="sourceLineNo">436</span><a name="line.436"></a>
<span class="sourceLineNo">437</span><a name="line.437"></a>
<span class="sourceLineNo">438</span><a name="line.438"></a>
<span class="sourceLineNo">439</span><a name="line.439"></a>
<span class="sourceLineNo">440</span>            if (startCodonBegin == null) {<a name="line.440"></a>
<span class="sourceLineNo">441</span>                if (strand == Strand.NEGATIVE) {<a name="line.441"></a>
<span class="sourceLineNo">442</span>                    FeatureI firstFeature = cdsFeatures.get(0);<a name="line.442"></a>
<span class="sourceLineNo">443</span><a name="line.443"></a>
<span class="sourceLineNo">444</span>                    startCodonBegin = firstFeature.location().bioEnd();<a name="line.444"></a>
<span class="sourceLineNo">445</span>                } else {<a name="line.445"></a>
<span class="sourceLineNo">446</span>                    FeatureI firstFeature = cdsFeatures.get(0);<a name="line.446"></a>
<span class="sourceLineNo">447</span><a name="line.447"></a>
<span class="sourceLineNo">448</span>                    startCodonBegin = firstFeature.location().bioStart();<a name="line.448"></a>
<span class="sourceLineNo">449</span>                }<a name="line.449"></a>
<span class="sourceLineNo">450</span>            }<a name="line.450"></a>
<span class="sourceLineNo">451</span><a name="line.451"></a>
<span class="sourceLineNo">452</span>            if (stopCodonEnd == null) {<a name="line.452"></a>
<span class="sourceLineNo">453</span>                if (strand == Strand.NEGATIVE) {<a name="line.453"></a>
<span class="sourceLineNo">454</span>                    FeatureI lastFeature = cdsFeatures.get(cdsFeatures.size() - 1);<a name="line.454"></a>
<span class="sourceLineNo">455</span>                    stopCodonEnd = lastFeature.location().bioStart();<a name="line.455"></a>
<span class="sourceLineNo">456</span>                } else {<a name="line.456"></a>
<span class="sourceLineNo">457</span>                    FeatureI lastFeature = cdsFeatures.get(cdsFeatures.size() - 1);<a name="line.457"></a>
<span class="sourceLineNo">458</span>                    stopCodonEnd = lastFeature.location().bioEnd();<a name="line.458"></a>
<span class="sourceLineNo">459</span>                }<a name="line.459"></a>
<span class="sourceLineNo">460</span>            }<a name="line.460"></a>
<span class="sourceLineNo">461</span>            //for gtf ordering can be strand based so first is last and last is first<a name="line.461"></a>
<span class="sourceLineNo">462</span>            if (startCodonBegin &gt; stopCodonEnd) {<a name="line.462"></a>
<span class="sourceLineNo">463</span>                int temp = startCodonBegin;<a name="line.463"></a>
<span class="sourceLineNo">464</span>                startCodonBegin = stopCodonEnd;<a name="line.464"></a>
<span class="sourceLineNo">465</span>                stopCodonEnd = temp;<a name="line.465"></a>
<span class="sourceLineNo">466</span>            }<a name="line.466"></a>
<span class="sourceLineNo">467</span><a name="line.467"></a>
<span class="sourceLineNo">468</span><a name="line.468"></a>
<span class="sourceLineNo">469</span><a name="line.469"></a>
<span class="sourceLineNo">470</span>            AccessionID transcriptAccessionID = new AccessionID(mRNAID);<a name="line.470"></a>
<span class="sourceLineNo">471</span>            geneSequence = seq.getGene(geneID);<a name="line.471"></a>
<span class="sourceLineNo">472</span>            if (geneSequence == null) {<a name="line.472"></a>
<span class="sourceLineNo">473</span>                geneSequence = seq.addGene(geneAccessionID, startCodonBegin, stopCodonEnd, strand);<a name="line.473"></a>
<span class="sourceLineNo">474</span>                geneSequence.setSource(geneSource);<a name="line.474"></a>
<span class="sourceLineNo">475</span>                if (geneNote != null &amp;&amp; geneNote.length() &gt; 0) {<a name="line.475"></a>
<span class="sourceLineNo">476</span>                    geneSequence.addNote(geneNote);<a name="line.476"></a>
<span class="sourceLineNo">477</span>                }<a name="line.477"></a>
<span class="sourceLineNo">478</span>            } else {<a name="line.478"></a>
<span class="sourceLineNo">479</span><a name="line.479"></a>
<span class="sourceLineNo">480</span>                if (startCodonBegin &lt; geneSequence.getBioBegin()) {<a name="line.480"></a>
<span class="sourceLineNo">481</span>                    geneSequence.setBioBegin(startCodonBegin);<a name="line.481"></a>
<span class="sourceLineNo">482</span>                }<a name="line.482"></a>
<span class="sourceLineNo">483</span>                if (stopCodonEnd &gt; geneSequence.getBioBegin()) {<a name="line.483"></a>
<span class="sourceLineNo">484</span>                    geneSequence.setBioEnd(stopCodonEnd);<a name="line.484"></a>
<span class="sourceLineNo">485</span>                }<a name="line.485"></a>
<span class="sourceLineNo">486</span><a name="line.486"></a>
<span class="sourceLineNo">487</span>            }<a name="line.487"></a>
<span class="sourceLineNo">488</span>            TranscriptSequence transcriptSequence = geneSequence.addTranscript(transcriptAccessionID, startCodonBegin, stopCodonEnd);<a name="line.488"></a>
<span class="sourceLineNo">489</span>            transcriptSequence.setSource(mRNAsource);<a name="line.489"></a>
<span class="sourceLineNo">490</span>            if (mRNANote != null &amp;&amp; mRNANote.length() &gt; 0) {<a name="line.490"></a>
<span class="sourceLineNo">491</span>                transcriptSequence.addNote(mRNANote);<a name="line.491"></a>
<span class="sourceLineNo">492</span><a name="line.492"></a>
<span class="sourceLineNo">493</span>            }<a name="line.493"></a>
<span class="sourceLineNo">494</span>            if (startCodon != null) {<a name="line.494"></a>
<span class="sourceLineNo">495</span>                if (startCodonName == null || startCodonName.length() == 0) {<a name="line.495"></a>
<span class="sourceLineNo">496</span>                    startCodonName = mRNAID + "-start_codon-" + startCodon.location().bioStart() + "-" + startCodon.location().bioEnd();<a name="line.496"></a>
<span class="sourceLineNo">497</span>                }<a name="line.497"></a>
<span class="sourceLineNo">498</span>                transcriptSequence.addStartCodonSequence(new AccessionID(startCodonName), startCodon.location().bioStart(), startCodon.location().bioEnd());<a name="line.498"></a>
<span class="sourceLineNo">499</span>            }<a name="line.499"></a>
<span class="sourceLineNo">500</span>            if (stopCodon != null) {<a name="line.500"></a>
<span class="sourceLineNo">501</span>                if (stopCodonName == null || stopCodonName.length() == 0) {<a name="line.501"></a>
<span class="sourceLineNo">502</span>                    stopCodonName = mRNAID + "-stop_codon-" + stopCodon.location().bioStart() + "-" + stopCodon.location().bioEnd();<a name="line.502"></a>
<span class="sourceLineNo">503</span>                }<a name="line.503"></a>
<span class="sourceLineNo">504</span>                transcriptSequence.addStopCodonSequence(new AccessionID(stopCodonName), stopCodon.location().bioStart(), stopCodon.location().bioEnd());<a name="line.504"></a>
<span class="sourceLineNo">505</span>            }<a name="line.505"></a>
<span class="sourceLineNo">506</span><a name="line.506"></a>
<span class="sourceLineNo">507</span>            for (FeatureI cdsFeature : cdsFeatures) {<a name="line.507"></a>
<span class="sourceLineNo">508</span>                Feature cds = (Feature) cdsFeature;<a name="line.508"></a>
<span class="sourceLineNo">509</span>                String cdsNote = cdsFeature.getAttribute("Note");<a name="line.509"></a>
<span class="sourceLineNo">510</span>                String cdsSource = cds.source();<a name="line.510"></a>
<span class="sourceLineNo">511</span>                String cdsName = cds.getAttribute("ID");<a name="line.511"></a>
<span class="sourceLineNo">512</span>                if (cdsName == null || cdsName.length() == 0) {<a name="line.512"></a>
<span class="sourceLineNo">513</span>                    cdsName = mRNAID + "-cds-" + cds.location().bioStart() + "-" + cds.location().bioEnd();<a name="line.513"></a>
<span class="sourceLineNo">514</span>                }<a name="line.514"></a>
<span class="sourceLineNo">515</span>                AccessionID cdsAccessionID = new AccessionID(cdsName);<a name="line.515"></a>
<span class="sourceLineNo">516</span>                ExonSequence exonSequence = geneSequence.addExon(cdsAccessionID, cdsFeature.location().bioStart(), cdsFeature.location().bioEnd());<a name="line.516"></a>
<span class="sourceLineNo">517</span>                exonSequence.setSource(cdsSource);<a name="line.517"></a>
<span class="sourceLineNo">518</span>                if (cdsNote != null &amp;&amp; cdsNote.length() &gt; 0) {<a name="line.518"></a>
<span class="sourceLineNo">519</span>                    exonSequence.addNote(cdsNote);<a name="line.519"></a>
<span class="sourceLineNo">520</span>                }<a name="line.520"></a>
<span class="sourceLineNo">521</span>                transcriptSequence.addCDS(cdsAccessionID, cdsFeature.location().bioStart(), cdsFeature.location().bioEnd(), cds.frame());<a name="line.521"></a>
<span class="sourceLineNo">522</span>            }<a name="line.522"></a>
<span class="sourceLineNo">523</span>            geneSequence.addIntronsUsingExons();<a name="line.523"></a>
<span class="sourceLineNo">524</span><a name="line.524"></a>
<span class="sourceLineNo">525</span>        }<a name="line.525"></a>
<span class="sourceLineNo">526</span><a name="line.526"></a>
<span class="sourceLineNo">527</span>    }<a name="line.527"></a>
<span class="sourceLineNo">528</span><a name="line.528"></a>
<span class="sourceLineNo">529</span>    static public LinkedHashMap&lt;String, ChromosomeSequence&gt; loadFastaAddGeneFeaturesFromGlimmerGFF3(File fastaSequenceFile, File gffFile) throws Exception {<a name="line.529"></a>
<span class="sourceLineNo">530</span>        LinkedHashMap&lt;String, DNASequence&gt; dnaSequenceList = FastaReaderHelper.readFastaDNASequence(fastaSequenceFile);<a name="line.530"></a>
<span class="sourceLineNo">531</span>        LinkedHashMap&lt;String, ChromosomeSequence&gt; chromosomeSequenceList = GeneFeatureHelper.getChromosomeSequenceFromDNASequence(dnaSequenceList);<a name="line.531"></a>
<span class="sourceLineNo">532</span>        FeatureList listGenes = GFF3Reader.read(gffFile.getAbsolutePath());<a name="line.532"></a>
<span class="sourceLineNo">533</span>        addGlimmerGFF3GeneFeatures(chromosomeSequenceList, listGenes);<a name="line.533"></a>
<span class="sourceLineNo">534</span>        return chromosomeSequenceList;<a name="line.534"></a>
<span class="sourceLineNo">535</span>    }<a name="line.535"></a>
<span class="sourceLineNo">536</span><a name="line.536"></a>
<span class="sourceLineNo">537</span>    static public void addGlimmerGFF3GeneFeatures(LinkedHashMap&lt;String, ChromosomeSequence&gt; chromosomeSequenceList, FeatureList listGenes) throws Exception {<a name="line.537"></a>
<span class="sourceLineNo">538</span>        FeatureList mRNAFeatures = listGenes.selectByType("mRNA");<a name="line.538"></a>
<span class="sourceLineNo">539</span>        for (FeatureI f : mRNAFeatures) {<a name="line.539"></a>
<span class="sourceLineNo">540</span>            Feature mRNAFeature = (Feature) f;<a name="line.540"></a>
<span class="sourceLineNo">541</span>            String geneid = mRNAFeature.getAttribute("ID");<a name="line.541"></a>
<span class="sourceLineNo">542</span>            String source = mRNAFeature.source();<a name="line.542"></a>
<span class="sourceLineNo">543</span><a name="line.543"></a>
<span class="sourceLineNo">544</span>            FeatureList gene = listGenes.selectByAttribute("Parent", geneid);<a name="line.544"></a>
<span class="sourceLineNo">545</span>            FeatureI geneFeature = gene.get(0);<a name="line.545"></a>
<span class="sourceLineNo">546</span>            ChromosomeSequence seq = chromosomeSequenceList.get(geneFeature.seqname());<a name="line.546"></a>
<span class="sourceLineNo">547</span>            AccessionID geneAccessionID = new AccessionID(geneid);<a name="line.547"></a>
<span class="sourceLineNo">548</span>            GeneSequence geneSequence = null;<a name="line.548"></a>
<span class="sourceLineNo">549</span><a name="line.549"></a>
<span class="sourceLineNo">550</span>            FeatureList cdsFeatures = gene.selectByType("CDS");<a name="line.550"></a>
<span class="sourceLineNo">551</span>            FeatureI feature = cdsFeatures.get(0);<a name="line.551"></a>
<span class="sourceLineNo">552</span>            Strand strand = Strand.POSITIVE;<a name="line.552"></a>
<span class="sourceLineNo">553</span><a name="line.553"></a>
<span class="sourceLineNo">554</span>            if (feature.location().isNegative()) {<a name="line.554"></a>
<span class="sourceLineNo">555</span>                strand = strand.NEGATIVE;<a name="line.555"></a>
<span class="sourceLineNo">556</span>            }<a name="line.556"></a>
<span class="sourceLineNo">557</span>            cdsFeatures = cdsFeatures.sortByStart();<a name="line.557"></a>
<span class="sourceLineNo">558</span><a name="line.558"></a>
<span class="sourceLineNo">559</span><a name="line.559"></a>
<span class="sourceLineNo">560</span><a name="line.560"></a>
<span class="sourceLineNo">561</span><a name="line.561"></a>
<span class="sourceLineNo">562</span><a name="line.562"></a>
<span class="sourceLineNo">563</span><a name="line.563"></a>
<span class="sourceLineNo">564</span><a name="line.564"></a>
<span class="sourceLineNo">565</span>            String seqName = feature.seqname();<a name="line.565"></a>
<span class="sourceLineNo">566</span>            FeatureI startCodon = null;<a name="line.566"></a>
<span class="sourceLineNo">567</span>            FeatureI stopCodon = null;<a name="line.567"></a>
<span class="sourceLineNo">568</span>            Integer startCodonBegin = null;<a name="line.568"></a>
<span class="sourceLineNo">569</span>            Integer stopCodonEnd = null;<a name="line.569"></a>
<span class="sourceLineNo">570</span>            String startCodonName = "";<a name="line.570"></a>
<span class="sourceLineNo">571</span>            String stopCodonName = "";<a name="line.571"></a>
<span class="sourceLineNo">572</span>            FeatureList startCodonList = gene.selectByAttribute("Note", "initial-exon");<a name="line.572"></a>
<span class="sourceLineNo">573</span>            if (startCodonList != null &amp;&amp; startCodonList.size() &gt; 0) {<a name="line.573"></a>
<span class="sourceLineNo">574</span>                startCodon = startCodonList.get(0);<a name="line.574"></a>
<span class="sourceLineNo">575</span>                if (strand == Strand.NEGATIVE) {<a name="line.575"></a>
<span class="sourceLineNo">576</span>                    startCodonBegin = startCodon.location().bioEnd();<a name="line.576"></a>
<span class="sourceLineNo">577</span>                } else {<a name="line.577"></a>
<span class="sourceLineNo">578</span>                    startCodonBegin = startCodon.location().bioStart();<a name="line.578"></a>
<span class="sourceLineNo">579</span>                }<a name="line.579"></a>
<span class="sourceLineNo">580</span>                startCodonName = startCodon.getAttribute("ID");<a name="line.580"></a>
<span class="sourceLineNo">581</span>            }<a name="line.581"></a>
<span class="sourceLineNo">582</span><a name="line.582"></a>
<span class="sourceLineNo">583</span>            FeatureList stopCodonList = gene.selectByAttribute("Note", "final-exon");<a name="line.583"></a>
<span class="sourceLineNo">584</span><a name="line.584"></a>
<span class="sourceLineNo">585</span>            if (stopCodonList != null &amp;&amp; stopCodonList.size() &gt; 0) {<a name="line.585"></a>
<span class="sourceLineNo">586</span>                stopCodon = stopCodonList.get(0);<a name="line.586"></a>
<span class="sourceLineNo">587</span>                if (strand == Strand.NEGATIVE) {<a name="line.587"></a>
<span class="sourceLineNo">588</span>                    stopCodonEnd = stopCodon.location().bioStart();<a name="line.588"></a>
<span class="sourceLineNo">589</span>                } else {<a name="line.589"></a>
<span class="sourceLineNo">590</span>                    stopCodonEnd = stopCodon.location().bioEnd();<a name="line.590"></a>
<span class="sourceLineNo">591</span>                }<a name="line.591"></a>
<span class="sourceLineNo">592</span>                stopCodonName = stopCodon.getAttribute("ID");<a name="line.592"></a>
<span class="sourceLineNo">593</span><a name="line.593"></a>
<span class="sourceLineNo">594</span>            }<a name="line.594"></a>
<span class="sourceLineNo">595</span><a name="line.595"></a>
<span class="sourceLineNo">596</span><a name="line.596"></a>
<span class="sourceLineNo">597</span><a name="line.597"></a>
<span class="sourceLineNo">598</span><a name="line.598"></a>
<span class="sourceLineNo">599</span>            if (startCodonBegin == null) {<a name="line.599"></a>
<span class="sourceLineNo">600</span>                if (strand == Strand.NEGATIVE) {<a name="line.600"></a>
<span class="sourceLineNo">601</span>                    FeatureI firstFeature = cdsFeatures.get(0);<a name="line.601"></a>
<span class="sourceLineNo">602</span><a name="line.602"></a>
<span class="sourceLineNo">603</span>                    startCodonBegin = firstFeature.location().bioEnd();<a name="line.603"></a>
<span class="sourceLineNo">604</span>                } else {<a name="line.604"></a>
<span class="sourceLineNo">605</span>                    FeatureI firstFeature = cdsFeatures.get(0);<a name="line.605"></a>
<span class="sourceLineNo">606</span><a name="line.606"></a>
<span class="sourceLineNo">607</span>                    startCodonBegin = firstFeature.location().bioStart();<a name="line.607"></a>
<span class="sourceLineNo">608</span>                }<a name="line.608"></a>
<span class="sourceLineNo">609</span>            }<a name="line.609"></a>
<span class="sourceLineNo">610</span><a name="line.610"></a>
<span class="sourceLineNo">611</span>            if (stopCodonEnd == null) {<a name="line.611"></a>
<span class="sourceLineNo">612</span>                if (strand == Strand.NEGATIVE) {<a name="line.612"></a>
<span class="sourceLineNo">613</span>                    FeatureI lastFeature = cdsFeatures.get(cdsFeatures.size() - 1);<a name="line.613"></a>
<span class="sourceLineNo">614</span>                    stopCodonEnd = lastFeature.location().bioStart();<a name="line.614"></a>
<span class="sourceLineNo">615</span>                } else {<a name="line.615"></a>
<span class="sourceLineNo">616</span>                    FeatureI lastFeature = cdsFeatures.get(cdsFeatures.size() - 1);<a name="line.616"></a>
<span class="sourceLineNo">617</span>                    stopCodonEnd = lastFeature.location().bioEnd();<a name="line.617"></a>
<span class="sourceLineNo">618</span>                }<a name="line.618"></a>
<span class="sourceLineNo">619</span>            }<a name="line.619"></a>
<span class="sourceLineNo">620</span>            //for gtf ordering can be strand based so first is last and last is first<a name="line.620"></a>
<span class="sourceLineNo">621</span>            if (startCodonBegin &gt; stopCodonEnd) {<a name="line.621"></a>
<span class="sourceLineNo">622</span>                int temp = startCodonBegin;<a name="line.622"></a>
<span class="sourceLineNo">623</span>                startCodonBegin = stopCodonEnd;<a name="line.623"></a>
<span class="sourceLineNo">624</span>                stopCodonEnd = temp;<a name="line.624"></a>
<span class="sourceLineNo">625</span>            }<a name="line.625"></a>
<span class="sourceLineNo">626</span><a name="line.626"></a>
<span class="sourceLineNo">627</span><a name="line.627"></a>
<span class="sourceLineNo">628</span><a name="line.628"></a>
<span class="sourceLineNo">629</span>            AccessionID transcriptAccessionID = new AccessionID(geneid);<a name="line.629"></a>
<span class="sourceLineNo">630</span>            if (geneSequence == null) {<a name="line.630"></a>
<span class="sourceLineNo">631</span>                geneSequence = seq.addGene(geneAccessionID, startCodonBegin, stopCodonEnd, strand);<a name="line.631"></a>
<span class="sourceLineNo">632</span>                geneSequence.setSource(source);<a name="line.632"></a>
<span class="sourceLineNo">633</span>            } else {<a name="line.633"></a>
<span class="sourceLineNo">634</span><a name="line.634"></a>
<span class="sourceLineNo">635</span>                if (startCodonBegin &lt; geneSequence.getBioBegin()) {<a name="line.635"></a>
<span class="sourceLineNo">636</span>                    geneSequence.setBioBegin(startCodonBegin);<a name="line.636"></a>
<span class="sourceLineNo">637</span>                }<a name="line.637"></a>
<span class="sourceLineNo">638</span>                if (stopCodonEnd &gt; geneSequence.getBioBegin()) {<a name="line.638"></a>
<span class="sourceLineNo">639</span>                    geneSequence.setBioEnd(stopCodonEnd);<a name="line.639"></a>
<span class="sourceLineNo">640</span>                }<a name="line.640"></a>
<span class="sourceLineNo">641</span><a name="line.641"></a>
<span class="sourceLineNo">642</span>            }<a name="line.642"></a>
<span class="sourceLineNo">643</span>            TranscriptSequence transcriptSequence = geneSequence.addTranscript(transcriptAccessionID, startCodonBegin, stopCodonEnd);<a name="line.643"></a>
<span class="sourceLineNo">644</span>            if (startCodon != null) {<a name="line.644"></a>
<span class="sourceLineNo">645</span>                if (startCodonName == null || startCodonName.length() == 0) {<a name="line.645"></a>
<span class="sourceLineNo">646</span>                    startCodonName = geneid + "-start_codon-" + startCodon.location().bioStart() + "-" + startCodon.location().bioEnd();<a name="line.646"></a>
<span class="sourceLineNo">647</span>                }<a name="line.647"></a>
<span class="sourceLineNo">648</span>                transcriptSequence.addStartCodonSequence(new AccessionID(startCodonName), startCodon.location().bioStart(), startCodon.location().bioEnd());<a name="line.648"></a>
<span class="sourceLineNo">649</span>            }<a name="line.649"></a>
<span class="sourceLineNo">650</span>            if (stopCodon != null) {<a name="line.650"></a>
<span class="sourceLineNo">651</span>                if (stopCodonName == null || stopCodonName.length() == 0) {<a name="line.651"></a>
<span class="sourceLineNo">652</span>                    stopCodonName = geneid + "-stop_codon-" + stopCodon.location().bioStart() + "-" + stopCodon.location().bioEnd();<a name="line.652"></a>
<span class="sourceLineNo">653</span>                }<a name="line.653"></a>
<span class="sourceLineNo">654</span>                transcriptSequence.addStopCodonSequence(new AccessionID(stopCodonName), stopCodon.location().bioStart(), stopCodon.location().bioEnd());<a name="line.654"></a>
<span class="sourceLineNo">655</span>            }<a name="line.655"></a>
<span class="sourceLineNo">656</span><a name="line.656"></a>
<span class="sourceLineNo">657</span>            for (FeatureI cdsFeature : cdsFeatures) {<a name="line.657"></a>
<span class="sourceLineNo">658</span>                Feature cds = (Feature) cdsFeature;<a name="line.658"></a>
<span class="sourceLineNo">659</span>                String cdsName = cds.getAttribute("ID");<a name="line.659"></a>
<span class="sourceLineNo">660</span>                if (cdsName == null || cdsName.length() == 0) {<a name="line.660"></a>
<span class="sourceLineNo">661</span>                    cdsName = geneid + "-cds-" + cds.location().bioStart() + "-" + cds.location().bioEnd();<a name="line.661"></a>
<span class="sourceLineNo">662</span>                }<a name="line.662"></a>
<span class="sourceLineNo">663</span>                AccessionID cdsAccessionID = new AccessionID(cdsName);<a name="line.663"></a>
<span class="sourceLineNo">664</span>                ExonSequence exonSequence = geneSequence.addExon(cdsAccessionID, cdsFeature.location().bioStart(), cdsFeature.location().bioEnd());<a name="line.664"></a>
<span class="sourceLineNo">665</span>                transcriptSequence.addCDS(cdsAccessionID, cdsFeature.location().bioStart(), cdsFeature.location().bioEnd(), cds.frame());<a name="line.665"></a>
<span class="sourceLineNo">666</span>            }<a name="line.666"></a>
<span class="sourceLineNo">667</span><a name="line.667"></a>
<span class="sourceLineNo">668</span>        }<a name="line.668"></a>
<span class="sourceLineNo">669</span><a name="line.669"></a>
<span class="sourceLineNo">670</span>    }<a name="line.670"></a>
<span class="sourceLineNo">671</span><a name="line.671"></a>
<span class="sourceLineNo">672</span>    static public LinkedHashMap&lt;String, ChromosomeSequence&gt; loadFastaAddGeneFeaturesFromGeneMarkGTF(File fastaSequenceFile, File gffFile) throws Exception {<a name="line.672"></a>
<span class="sourceLineNo">673</span>        LinkedHashMap&lt;String, DNASequence&gt; dnaSequenceList = FastaReaderHelper.readFastaDNASequence(fastaSequenceFile);<a name="line.673"></a>
<span class="sourceLineNo">674</span>        LinkedHashMap&lt;String, ChromosomeSequence&gt; chromosomeSequenceList = GeneFeatureHelper.getChromosomeSequenceFromDNASequence(dnaSequenceList);<a name="line.674"></a>
<span class="sourceLineNo">675</span>        FeatureList listGenes = GeneMarkGTFReader.read(gffFile.getAbsolutePath());<a name="line.675"></a>
<span class="sourceLineNo">676</span>        addGeneMarkGTFGeneFeatures(chromosomeSequenceList, listGenes);<a name="line.676"></a>
<span class="sourceLineNo">677</span>        return chromosomeSequenceList;<a name="line.677"></a>
<span class="sourceLineNo">678</span>    }<a name="line.678"></a>
<span class="sourceLineNo">679</span><a name="line.679"></a>
<span class="sourceLineNo">680</span>    static public void addGeneMarkGTFGeneFeatures(LinkedHashMap&lt;String, ChromosomeSequence&gt; chromosomeSequenceList, FeatureList listGenes) throws Exception {<a name="line.680"></a>
<span class="sourceLineNo">681</span>        Collection&lt;String&gt; geneIds = listGenes.attributeValues("gene_id");<a name="line.681"></a>
<span class="sourceLineNo">682</span>        for (String geneid : geneIds) {<a name="line.682"></a>
<span class="sourceLineNo">683</span>            //       if(geneid.equals("45_g")){<a name="line.683"></a>
<span class="sourceLineNo">684</span>            //           int dummy =1;<a name="line.684"></a>
<span class="sourceLineNo">685</span>            //       }<a name="line.685"></a>
<span class="sourceLineNo">686</span>            FeatureList gene = listGenes.selectByAttribute("gene_id", geneid);<a name="line.686"></a>
<span class="sourceLineNo">687</span>            FeatureI geneFeature = gene.get(0);<a name="line.687"></a>
<span class="sourceLineNo">688</span>            ChromosomeSequence seq = chromosomeSequenceList.get(geneFeature.seqname());<a name="line.688"></a>
<span class="sourceLineNo">689</span>            AccessionID geneAccessionID = new AccessionID(geneid);<a name="line.689"></a>
<span class="sourceLineNo">690</span>            GeneSequence geneSequence = null;<a name="line.690"></a>
<span class="sourceLineNo">691</span>            Collection&lt;String&gt; transcriptids = gene.attributeValues("transcript_id");<a name="line.691"></a>
<span class="sourceLineNo">692</span>            for (String transcriptid : transcriptids) {<a name="line.692"></a>
<span class="sourceLineNo">693</span>                // get all the individual features (exons, CDS regions, etc.) of this gene<a name="line.693"></a>
<span class="sourceLineNo">694</span><a name="line.694"></a>
<span class="sourceLineNo">695</span><a name="line.695"></a>
<span class="sourceLineNo">696</span>                FeatureList transcriptFeature = listGenes.selectByAttribute("transcript_id", transcriptid);<a name="line.696"></a>
<span class="sourceLineNo">697</span>                // now select only the coding regions of this gene<a name="line.697"></a>
<span class="sourceLineNo">698</span>                FeatureList cdsFeatures = transcriptFeature.selectByType("CDS");<a name="line.698"></a>
<span class="sourceLineNo">699</span>                // sort them<a name="line.699"></a>
<span class="sourceLineNo">700</span>                cdsFeatures = cdsFeatures.sortByStart();<a name="line.700"></a>
<span class="sourceLineNo">701</span><a name="line.701"></a>
<span class="sourceLineNo">702</span>                FeatureI feature = cdsFeatures.get(0);<a name="line.702"></a>
<span class="sourceLineNo">703</span>                Strand strand = Strand.POSITIVE;<a name="line.703"></a>
<span class="sourceLineNo">704</span><a name="line.704"></a>
<span class="sourceLineNo">705</span>                if (feature.location().isNegative()) {<a name="line.705"></a>
<span class="sourceLineNo">706</span>                    strand = strand.NEGATIVE;<a name="line.706"></a>
<span class="sourceLineNo">707</span>                }<a name="line.707"></a>
<span class="sourceLineNo">708</span><a name="line.708"></a>
<span class="sourceLineNo">709</span>                String seqName = feature.seqname();<a name="line.709"></a>
<span class="sourceLineNo">710</span>                FeatureI startCodon = null;<a name="line.710"></a>
<span class="sourceLineNo">711</span>                FeatureI stopCodon = null;<a name="line.711"></a>
<span class="sourceLineNo">712</span>                Integer startCodonBegin = null;<a name="line.712"></a>
<span class="sourceLineNo">713</span>                Integer stopCodonEnd = null;<a name="line.713"></a>
<span class="sourceLineNo">714</span>                String startCodonName = "";<a name="line.714"></a>
<span class="sourceLineNo">715</span>                String stopCodonName = "";<a name="line.715"></a>
<span class="sourceLineNo">716</span>                FeatureList startCodonList = transcriptFeature.selectByType("start_codon");<a name="line.716"></a>
<span class="sourceLineNo">717</span>                if (startCodonList != null &amp;&amp; startCodonList.size() &gt; 0) {<a name="line.717"></a>
<span class="sourceLineNo">718</span>                    startCodon = startCodonList.get(0);<a name="line.718"></a>
<span class="sourceLineNo">719</span>                    if (strand == Strand.POSITIVE) {<a name="line.719"></a>
<span class="sourceLineNo">720</span>                        startCodonBegin = startCodon.location().bioStart();<a name="line.720"></a>
<span class="sourceLineNo">721</span>                    } else {<a name="line.721"></a>
<span class="sourceLineNo">722</span>                        startCodonBegin = startCodon.location().bioEnd();<a name="line.722"></a>
<span class="sourceLineNo">723</span>                    }<a name="line.723"></a>
<span class="sourceLineNo">724</span>                    startCodonName = startCodon.getAttribute("transcript_name");<a name="line.724"></a>
<span class="sourceLineNo">725</span>                }<a name="line.725"></a>
<span class="sourceLineNo">726</span><a name="line.726"></a>
<span class="sourceLineNo">727</span>                FeatureList stopCodonList = transcriptFeature.selectByType("stop_codon");<a name="line.727"></a>
<span class="sourceLineNo">728</span><a name="line.728"></a>
<span class="sourceLineNo">729</span>                if (stopCodonList != null &amp;&amp; stopCodonList.size() &gt; 0) {<a name="line.729"></a>
<span class="sourceLineNo">730</span>                    stopCodon = stopCodonList.get(0);<a name="line.730"></a>
<span class="sourceLineNo">731</span>                    if (strand == Strand.POSITIVE) {<a name="line.731"></a>
<span class="sourceLineNo">732</span>                        stopCodonEnd = stopCodon.location().bioEnd();<a name="line.732"></a>
<span class="sourceLineNo">733</span>                    } else {<a name="line.733"></a>
<span class="sourceLineNo">734</span>                        stopCodonEnd = stopCodon.location().bioStart();<a name="line.734"></a>
<span class="sourceLineNo">735</span>                    }<a name="line.735"></a>
<span class="sourceLineNo">736</span><a name="line.736"></a>
<span class="sourceLineNo">737</span>                    stopCodonName = stopCodon.getAttribute("transcript_name");<a name="line.737"></a>
<span class="sourceLineNo">738</span><a name="line.738"></a>
<span class="sourceLineNo">739</span>                }<a name="line.739"></a>
<span class="sourceLineNo">740</span><a name="line.740"></a>
<span class="sourceLineNo">741</span><a name="line.741"></a>
<span class="sourceLineNo">742</span><a name="line.742"></a>
<span class="sourceLineNo">743</span><a name="line.743"></a>
<span class="sourceLineNo">744</span>                if (startCodonBegin == null) {<a name="line.744"></a>
<span class="sourceLineNo">745</span>                    if (strand == Strand.NEGATIVE) {<a name="line.745"></a>
<span class="sourceLineNo">746</span>                        FeatureI firstFeature = cdsFeatures.get(0);<a name="line.746"></a>
<span class="sourceLineNo">747</span><a name="line.747"></a>
<span class="sourceLineNo">748</span>                        startCodonBegin = firstFeature.location().bioEnd();<a name="line.748"></a>
<span class="sourceLineNo">749</span>                    } else {<a name="line.749"></a>
<span class="sourceLineNo">750</span>                        FeatureI firstFeature = cdsFeatures.get(0);<a name="line.750"></a>
<span class="sourceLineNo">751</span><a name="line.751"></a>
<span class="sourceLineNo">752</span>                        startCodonBegin = firstFeature.location().bioStart();<a name="line.752"></a>
<span class="sourceLineNo">753</span>                    }<a name="line.753"></a>
<span class="sourceLineNo">754</span>                }<a name="line.754"></a>
<span class="sourceLineNo">755</span><a name="line.755"></a>
<span class="sourceLineNo">756</span>                if (stopCodonEnd == null) {<a name="line.756"></a>
<span class="sourceLineNo">757</span>                    if (strand == Strand.NEGATIVE) {<a name="line.757"></a>
<span class="sourceLineNo">758</span>                        FeatureI lastFeature = cdsFeatures.get(cdsFeatures.size() - 1);<a name="line.758"></a>
<span class="sourceLineNo">759</span>                        stopCodonEnd = lastFeature.location().bioStart();<a name="line.759"></a>
<span class="sourceLineNo">760</span>                    } else {<a name="line.760"></a>
<span class="sourceLineNo">761</span>                        FeatureI lastFeature = cdsFeatures.get(cdsFeatures.size() - 1);<a name="line.761"></a>
<span class="sourceLineNo">762</span>                        stopCodonEnd = lastFeature.location().bioEnd();<a name="line.762"></a>
<span class="sourceLineNo">763</span>                    }<a name="line.763"></a>
<span class="sourceLineNo">764</span>                }<a name="line.764"></a>
<span class="sourceLineNo">765</span>                //for gtf ordering can be strand based so first is last and last is first<a name="line.765"></a>
<span class="sourceLineNo">766</span>                if (startCodonBegin &gt; stopCodonEnd) {<a name="line.766"></a>
<span class="sourceLineNo">767</span>                    int temp = startCodonBegin;<a name="line.767"></a>
<span class="sourceLineNo">768</span>                    startCodonBegin = stopCodonEnd;<a name="line.768"></a>
<span class="sourceLineNo">769</span>                    stopCodonEnd = temp;<a name="line.769"></a>
<span class="sourceLineNo">770</span>                }<a name="line.770"></a>
<span class="sourceLineNo">771</span><a name="line.771"></a>
<span class="sourceLineNo">772</span>                AccessionID transcriptAccessionID = new AccessionID(transcriptid);<a name="line.772"></a>
<span class="sourceLineNo">773</span>                if (geneSequence == null) {<a name="line.773"></a>
<span class="sourceLineNo">774</span>                    geneSequence = seq.addGene(geneAccessionID, startCodonBegin, stopCodonEnd, strand);<a name="line.774"></a>
<span class="sourceLineNo">775</span>                    geneSequence.setSource(((Feature) feature).source());<a name="line.775"></a>
<span class="sourceLineNo">776</span>                } else {<a name="line.776"></a>
<span class="sourceLineNo">777</span>                    //if multiple transcripts for one gene make sure the gene is defined as the min and max start/end<a name="line.777"></a>
<span class="sourceLineNo">778</span><a name="line.778"></a>
<span class="sourceLineNo">779</span>                    if (startCodonBegin &lt; geneSequence.getBioBegin()) {<a name="line.779"></a>
<span class="sourceLineNo">780</span>                        geneSequence.setBioBegin(startCodonBegin);<a name="line.780"></a>
<span class="sourceLineNo">781</span>                    }<a name="line.781"></a>
<span class="sourceLineNo">782</span>                    if (stopCodonEnd &gt; geneSequence.getBioBegin()) {<a name="line.782"></a>
<span class="sourceLineNo">783</span>                        geneSequence.setBioEnd(stopCodonEnd);<a name="line.783"></a>
<span class="sourceLineNo">784</span>                    }<a name="line.784"></a>
<span class="sourceLineNo">785</span><a name="line.785"></a>
<span class="sourceLineNo">786</span>                }<a name="line.786"></a>
<span class="sourceLineNo">787</span>                TranscriptSequence transcriptSequence = geneSequence.addTranscript(transcriptAccessionID, startCodonBegin, stopCodonEnd);<a name="line.787"></a>
<span class="sourceLineNo">788</span>                if (startCodon != null) {<a name="line.788"></a>
<span class="sourceLineNo">789</span>                    if (startCodonName == null || startCodonName.length() == 0) {<a name="line.789"></a>
<span class="sourceLineNo">790</span>                        startCodonName = transcriptid + "-start_codon-" + startCodon.location().bioStart() + "-" + startCodon.location().bioEnd();<a name="line.790"></a>
<span class="sourceLineNo">791</span>                    }<a name="line.791"></a>
<span class="sourceLineNo">792</span>                    transcriptSequence.addStartCodonSequence(new AccessionID(startCodonName), startCodon.location().bioStart(), startCodon.location().bioEnd());<a name="line.792"></a>
<span class="sourceLineNo">793</span>                }<a name="line.793"></a>
<span class="sourceLineNo">794</span>                if (stopCodon != null) {<a name="line.794"></a>
<span class="sourceLineNo">795</span>                    if (stopCodonName == null || stopCodonName.length() == 0) {<a name="line.795"></a>
<span class="sourceLineNo">796</span>                        stopCodonName = transcriptid + "-stop_codon-" + stopCodon.location().bioStart() + "-" + stopCodon.location().bioEnd();<a name="line.796"></a>
<span class="sourceLineNo">797</span>                    }<a name="line.797"></a>
<span class="sourceLineNo">798</span>                    transcriptSequence.addStopCodonSequence(new AccessionID(stopCodonName), stopCodon.location().bioStart(), stopCodon.location().bioEnd());<a name="line.798"></a>
<span class="sourceLineNo">799</span>                }<a name="line.799"></a>
<span class="sourceLineNo">800</span><a name="line.800"></a>
<span class="sourceLineNo">801</span>                for (FeatureI cdsFeature : cdsFeatures) {<a name="line.801"></a>
<span class="sourceLineNo">802</span>                    Feature cds = (Feature) cdsFeature;<a name="line.802"></a>
<span class="sourceLineNo">803</span>                    // for genemark it appears frame of 2 =1 and frame of 1 = 2<a name="line.803"></a>
<span class="sourceLineNo">804</span>                    // doesn't matter when you string cds regions together as one block<a name="line.804"></a>
<span class="sourceLineNo">805</span>                    // but does make a difference when you try to make a protein sequence for each CDS region where<a name="line.805"></a>
<span class="sourceLineNo">806</span>                    // you give up or borrow based on the frame value<a name="line.806"></a>
<span class="sourceLineNo">807</span>                    // compared with gff like files and docs for geneid and glimmer where geneid and glimmer both do it the same<a name="line.807"></a>
<span class="sourceLineNo">808</span>                    // way that appears to match the gff3 docs.<a name="line.808"></a>
<span class="sourceLineNo">809</span>                    int frame = cds.frame();<a name="line.809"></a>
<span class="sourceLineNo">810</span>                    if (frame == 1) {<a name="line.810"></a>
<span class="sourceLineNo">811</span>                        frame = 2;<a name="line.811"></a>
<span class="sourceLineNo">812</span>                    } else if (frame == 2) {<a name="line.812"></a>
<span class="sourceLineNo">813</span>                        frame = 1;<a name="line.813"></a>
<span class="sourceLineNo">814</span>                    } else {<a name="line.814"></a>
<span class="sourceLineNo">815</span>                        frame = 0;<a name="line.815"></a>
<span class="sourceLineNo">816</span>                    }<a name="line.816"></a>
<span class="sourceLineNo">817</span>                    String cdsName = cds.getAttribute("transcript_name");<a name="line.817"></a>
<span class="sourceLineNo">818</span>                    if (cdsName == null || cdsName.length() == 0) {<a name="line.818"></a>
<span class="sourceLineNo">819</span>                        cdsName = transcriptid + "-cds-" + cds.location().bioStart() + "-" + cds.location().bioEnd();<a name="line.819"></a>
<span class="sourceLineNo">820</span>                    }<a name="line.820"></a>
<span class="sourceLineNo">821</span>                    AccessionID cdsAccessionID = new AccessionID(cdsName);<a name="line.821"></a>
<span class="sourceLineNo">822</span>                    ExonSequence exonSequence = geneSequence.addExon(cdsAccessionID, cdsFeature.location().bioStart(), cdsFeature.location().bioEnd());<a name="line.822"></a>
<span class="sourceLineNo">823</span>                    transcriptSequence.addCDS(cdsAccessionID, cdsFeature.location().bioStart(), cdsFeature.location().bioEnd(), frame);<a name="line.823"></a>
<span class="sourceLineNo">824</span>                }<a name="line.824"></a>
<span class="sourceLineNo">825</span>            }<a name="line.825"></a>
<span class="sourceLineNo">826</span>        }<a name="line.826"></a>
<span class="sourceLineNo">827</span><a name="line.827"></a>
<span class="sourceLineNo">828</span>    }<a name="line.828"></a>
<span class="sourceLineNo">829</span><a name="line.829"></a>
<span class="sourceLineNo">830</span>    static public LinkedHashMap&lt;String, ProteinSequence&gt; getProteinSequences(Collection&lt;ChromosomeSequence&gt; chromosomeSequences) throws Exception {<a name="line.830"></a>
<span class="sourceLineNo">831</span>        LinkedHashMap&lt;String, ProteinSequence&gt; proteinSequenceHashMap = new LinkedHashMap&lt;String, ProteinSequence&gt;();<a name="line.831"></a>
<span class="sourceLineNo">832</span>        for (ChromosomeSequence dnaSequence : chromosomeSequences) {<a name="line.832"></a>
<span class="sourceLineNo">833</span>            for (GeneSequence geneSequence : dnaSequence.getGeneSequences().values()) {<a name="line.833"></a>
<span class="sourceLineNo">834</span>                for (TranscriptSequence transcriptSequence : geneSequence.getTranscripts().values()) {<a name="line.834"></a>
<span class="sourceLineNo">835</span>                    //TODO remove?<a name="line.835"></a>
<span class="sourceLineNo">836</span>//                    DNASequence dnaCodingSequence = transcriptSequence.getDNACodingSequence();<a name="line.836"></a>
<span class="sourceLineNo">837</span>//                    System.out.println("CDS=" + dnaCodingSequence.getSequenceAsString());<a name="line.837"></a>
<span class="sourceLineNo">838</span><a name="line.838"></a>
<span class="sourceLineNo">839</span>                    try {<a name="line.839"></a>
<span class="sourceLineNo">840</span>                        ProteinSequence proteinSequence = transcriptSequence.getProteinSequence();<a name="line.840"></a>
<span class="sourceLineNo">841</span>                       <a name="line.841"></a>
<span class="sourceLineNo">842</span>//                        System.out.println(proteinSequence.getAccession().getID() + " " + proteinSequence);<a name="line.842"></a>
<span class="sourceLineNo">843</span>                        if (proteinSequenceHashMap.containsKey(proteinSequence.getAccession().getID())) {<a name="line.843"></a>
<span class="sourceLineNo">844</span>                            throw new Exception("Duplicate protein sequence id=" + proteinSequence.getAccession().getID() + " found at Gene id=" + geneSequence.getAccession().getID());<a name="line.844"></a>
<span class="sourceLineNo">845</span>                        } else {<a name="line.845"></a>
<span class="sourceLineNo">846</span>                            proteinSequenceHashMap.put(proteinSequence.getAccession().getID(), proteinSequence);<a name="line.846"></a>
<span class="sourceLineNo">847</span>                        }<a name="line.847"></a>
<span class="sourceLineNo">848</span>                    } catch (Exception e) {<a name="line.848"></a>
<span class="sourceLineNo">849</span>                        e.printStackTrace();<a name="line.849"></a>
<span class="sourceLineNo">850</span>                    }<a name="line.850"></a>
<span class="sourceLineNo">851</span><a name="line.851"></a>
<span class="sourceLineNo">852</span>                }<a name="line.852"></a>
<span class="sourceLineNo">853</span><a name="line.853"></a>
<span class="sourceLineNo">854</span>            }<a name="line.854"></a>
<span class="sourceLineNo">855</span>        }<a name="line.855"></a>
<span class="sourceLineNo">856</span>        return proteinSequenceHashMap;<a name="line.856"></a>
<span class="sourceLineNo">857</span>    }<a name="line.857"></a>
<span class="sourceLineNo">858</span><a name="line.858"></a>
<span class="sourceLineNo">859</span>    static public LinkedHashMap&lt;String, GeneSequence&gt; getGeneSequences(Collection&lt;ChromosomeSequence&gt; chromosomeSequences) throws Exception {<a name="line.859"></a>
<span class="sourceLineNo">860</span>        LinkedHashMap&lt;String, GeneSequence&gt; geneSequenceHashMap = new LinkedHashMap&lt;String, GeneSequence&gt;();<a name="line.860"></a>
<span class="sourceLineNo">861</span>        for (ChromosomeSequence chromosomeSequence : chromosomeSequences) {<a name="line.861"></a>
<span class="sourceLineNo">862</span>            for (GeneSequence geneSequence : chromosomeSequence.getGeneSequences().values()) {<a name="line.862"></a>
<span class="sourceLineNo">863</span>                geneSequenceHashMap.put(geneSequence.getAccession().getID(), geneSequence);<a name="line.863"></a>
<span class="sourceLineNo">864</span>            }<a name="line.864"></a>
<span class="sourceLineNo">865</span>        }<a name="line.865"></a>
<span class="sourceLineNo">866</span><a name="line.866"></a>
<span class="sourceLineNo">867</span>        return geneSequenceHashMap;<a name="line.867"></a>
<span class="sourceLineNo">868</span>    }<a name="line.868"></a>
<span class="sourceLineNo">869</span><a name="line.869"></a>
<span class="sourceLineNo">870</span>    public static void main(String args[]) throws Exception {<a name="line.870"></a>
<span class="sourceLineNo">871</span>        if (false) {<a name="line.871"></a>
<span class="sourceLineNo">872</span>            LinkedHashMap&lt;String, ChromosomeSequence&gt; chromosomeSequenceList = GeneFeatureHelper.loadFastaAddGeneFeaturesFromGeneMarkGTF(new File("Scaffolds.fna"), new File("genemark_hmm.gtf"));<a name="line.872"></a>
<span class="sourceLineNo">873</span>            LinkedHashMap&lt;String, ProteinSequence&gt; proteinSequenceList = GeneFeatureHelper.getProteinSequences(chromosomeSequenceList.values());<a name="line.873"></a>
<span class="sourceLineNo">874</span>        }<a name="line.874"></a>
<span class="sourceLineNo">875</span><a name="line.875"></a>
<span class="sourceLineNo">876</span>        if (false) {<a name="line.876"></a>
<span class="sourceLineNo">877</span>            LinkedHashMap&lt;String, ChromosomeSequence&gt; chromosomeSequenceList = GeneFeatureHelper.loadFastaAddGeneFeaturesFromGlimmerGFF3(new File("Scaffolds.fna"), new File("glimmerhmm.gff"));<a name="line.877"></a>
<span class="sourceLineNo">878</span>            LinkedHashMap&lt;String, ProteinSequence&gt; proteinSequenceList = GeneFeatureHelper.getProteinSequences(chromosomeSequenceList.values());<a name="line.878"></a>
<span class="sourceLineNo">879</span>            //  for (ProteinSequence proteinSequence : proteinSequenceList.values()) {<a name="line.879"></a>
<span class="sourceLineNo">880</span>            //      System.out.println(proteinSequence.getAccession().getID() + " " + proteinSequence);<a name="line.880"></a>
<span class="sourceLineNo">881</span>            //  }<a name="line.881"></a>
<span class="sourceLineNo">882</span>            FastaWriterHelper.writeProteinSequence(new File("predicted_glimmer.faa"), proteinSequenceList.values());<a name="line.882"></a>
<span class="sourceLineNo">883</span><a name="line.883"></a>
<span class="sourceLineNo">884</span>        }<a name="line.884"></a>
<span class="sourceLineNo">885</span>        if (false) {<a name="line.885"></a>
<span class="sourceLineNo">886</span>            GeneFeatureHelper.outputFastaSequenceLengthGFF3(new File("Scaffolds.fna"), new File("scaffolds.gff3"));<a name="line.886"></a>
<span class="sourceLineNo">887</span>        }<a name="line.887"></a>
<span class="sourceLineNo">888</span><a name="line.888"></a>
<span class="sourceLineNo">889</span><a name="line.889"></a>
<span class="sourceLineNo">890</span><a name="line.890"></a>
<span class="sourceLineNo">891</span>        try {<a name="line.891"></a>
<span class="sourceLineNo">892</span><a name="line.892"></a>
<span class="sourceLineNo">893</span>            if (true) {<a name="line.893"></a>
<span class="sourceLineNo">894</span>                File fastaSequenceFile = new File("Scaffolds.fna");<a name="line.894"></a>
<span class="sourceLineNo">895</span>                File gff3File = new File("geneid-v6.gff3");<a name="line.895"></a>
<span class="sourceLineNo">896</span>                LinkedHashMap&lt;String, ChromosomeSequence&gt; chromosomeSequenceHashMap = GeneFeatureHelper.loadFastaAddGeneFeaturesFromGmodGFF3(fastaSequenceFile, gff3File,true);<a name="line.896"></a>
<span class="sourceLineNo">897</span>            }<a name="line.897"></a>
<span class="sourceLineNo">898</span><a name="line.898"></a>
<span class="sourceLineNo">899</span>        } catch (Exception e) {<a name="line.899"></a>
<span class="sourceLineNo">900</span>            e.printStackTrace();<a name="line.900"></a>
<span class="sourceLineNo">901</span>        }<a name="line.901"></a>
<span class="sourceLineNo">902</span><a name="line.902"></a>
<span class="sourceLineNo">903</span>    }<a name="line.903"></a>
<span class="sourceLineNo">904</span>}<a name="line.904"></a>




























































</pre>
</div>
</body>
</html>
