<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<html lang="en">
<head>
<title>Source code</title>
<link rel="stylesheet" type="text/css" href="../../../../stylesheet.css" title="Style">
</head>
<body>
<div class="sourceContainer">
<pre><span class="sourceLineNo">001</span>/*<a name="line.1"></a>
<span class="sourceLineNo">002</span> * To change this template, choose Tools | Templates<a name="line.2"></a>
<span class="sourceLineNo">003</span> * and open the template in the editor.<a name="line.3"></a>
<span class="sourceLineNo">004</span> */<a name="line.4"></a>
<span class="sourceLineNo">005</span>package org.biojava3.genome;<a name="line.5"></a>
<span class="sourceLineNo">006</span><a name="line.6"></a>
<span class="sourceLineNo">007</span>import java.io.File;<a name="line.7"></a>
<span class="sourceLineNo">008</span>import java.io.FileOutputStream;<a name="line.8"></a>
<span class="sourceLineNo">009</span>import java.util.Collection;<a name="line.9"></a>
<span class="sourceLineNo">010</span>import java.util.LinkedHashMap;<a name="line.10"></a>
<span class="sourceLineNo">011</span>import junit.framework.TestCase;<a name="line.11"></a>
<span class="sourceLineNo">012</span>import junitx.framework.FileAssert;<a name="line.12"></a>
<span class="sourceLineNo">013</span>import org.biojava3.core.sequence.ChromosomeSequence;<a name="line.13"></a>
<span class="sourceLineNo">014</span>import org.biojava3.core.sequence.GeneSequence;<a name="line.14"></a>
<span class="sourceLineNo">015</span>import org.biojava3.core.sequence.ProteinSequence;<a name="line.15"></a>
<span class="sourceLineNo">016</span>import org.biojava3.core.sequence.io.FastaWriterHelper;<a name="line.16"></a>
<span class="sourceLineNo">017</span>import org.biojava3.genome.parsers.gff.FeatureI;<a name="line.17"></a>
<span class="sourceLineNo">018</span>import org.biojava3.genome.parsers.gff.FeatureList;<a name="line.18"></a>
<span class="sourceLineNo">019</span>import org.biojava3.genome.parsers.gff.GFF3Reader;<a name="line.19"></a>
<span class="sourceLineNo">020</span>import org.biojava3.genome.parsers.gff.GFF3Writer;<a name="line.20"></a>
<span class="sourceLineNo">021</span><a name="line.21"></a>
<span class="sourceLineNo">022</span>/**<a name="line.22"></a>
<span class="sourceLineNo">023</span> *<a name="line.23"></a>
<span class="sourceLineNo">024</span> * @author Scooter Willis &lt;willishf at gmail dot com&gt;<a name="line.24"></a>
<span class="sourceLineNo">025</span> */<a name="line.25"></a>
<span class="sourceLineNo">026</span>public class GeneFeatureHelperTest extends TestCase {<a name="line.26"></a>
<span class="sourceLineNo">027</span><a name="line.27"></a>
<span class="sourceLineNo">028</span>    public GeneFeatureHelperTest(String testName) {<a name="line.28"></a>
<span class="sourceLineNo">029</span>        super(testName);<a name="line.29"></a>
<span class="sourceLineNo">030</span>    }<a name="line.30"></a>
<span class="sourceLineNo">031</span><a name="line.31"></a>
<span class="sourceLineNo">032</span>    @Override<a name="line.32"></a>
<span class="sourceLineNo">033</span>    protected void setUp() throws Exception {<a name="line.33"></a>
<span class="sourceLineNo">034</span>        super.setUp();<a name="line.34"></a>
<span class="sourceLineNo">035</span>    }<a name="line.35"></a>
<span class="sourceLineNo">036</span><a name="line.36"></a>
<span class="sourceLineNo">037</span>    @Override<a name="line.37"></a>
<span class="sourceLineNo">038</span>    protected void tearDown() throws Exception {<a name="line.38"></a>
<span class="sourceLineNo">039</span>        super.tearDown();<a name="line.39"></a>
<span class="sourceLineNo">040</span>    }<a name="line.40"></a>
<span class="sourceLineNo">041</span><a name="line.41"></a>
<span class="sourceLineNo">042</span>    public void testZeroLocation() throws Exception{<a name="line.42"></a>
<span class="sourceLineNo">043</span><a name="line.43"></a>
<span class="sourceLineNo">044</span>        FeatureList listGenes = GFF3Reader.read("src/test/resources/amphimedon.gff3");<a name="line.44"></a>
<span class="sourceLineNo">045</span>     //   System.out.println("Features");<a name="line.45"></a>
<span class="sourceLineNo">046</span>     //   for(FeatureI feature : listGenes){<a name="line.46"></a>
<span class="sourceLineNo">047</span>     //       System.out.println(feature);<a name="line.47"></a>
<span class="sourceLineNo">048</span>     //   }<a name="line.48"></a>
<span class="sourceLineNo">049</span>    }<a name="line.49"></a>
<span class="sourceLineNo">050</span><a name="line.50"></a>
<span class="sourceLineNo">051</span>    /**<a name="line.51"></a>
<span class="sourceLineNo">052</span>     * Test of loadFastaAddGeneFeaturesFromUpperCaseExonFastaFile method, of class GeneFeatureHelper.<a name="line.52"></a>
<span class="sourceLineNo">053</span>     * @throws Exception<a name="line.53"></a>
<span class="sourceLineNo">054</span>     */<a name="line.54"></a>
<span class="sourceLineNo">055</span><a name="line.55"></a>
<span class="sourceLineNo">056</span>    public void testLoadFastaAddGeneFeaturesFromUpperCaseExonFastaFile() throws Exception{<a name="line.56"></a>
<span class="sourceLineNo">057</span>//        System.out.println("loadFastaAddGeneFeaturesFromUpperCaseExonFastaFile");<a name="line.57"></a>
<span class="sourceLineNo">058</span>        File fastaSequenceFile = new File("src/test/resources/volvox_all.fna");<a name="line.58"></a>
<span class="sourceLineNo">059</span>        File uppercaseFastaFile = new File("src/test/resources/volvox_all_genes_exon_uppercase.fna");<a name="line.59"></a>
<span class="sourceLineNo">060</span>        boolean throwExceptionGeneNotFound = false;<a name="line.60"></a>
<span class="sourceLineNo">061</span>        LinkedHashMap&lt;String, ChromosomeSequence&gt; chromosomeSequenceHashMap = GeneFeatureHelper.loadFastaAddGeneFeaturesFromUpperCaseExonFastaFile( fastaSequenceFile, uppercaseFastaFile,  throwExceptionGeneNotFound);<a name="line.61"></a>
<span class="sourceLineNo">062</span><a name="line.62"></a>
<span class="sourceLineNo">063</span>        File tmp = File.createTempFile("volvox_all_genes_exon_uppercase", "gff3");<a name="line.63"></a>
<span class="sourceLineNo">064</span>        tmp.deleteOnExit();<a name="line.64"></a>
<span class="sourceLineNo">065</span>        FileOutputStream fo = new FileOutputStream(tmp);<a name="line.65"></a>
<span class="sourceLineNo">066</span>        GFF3Writer gff3Writer = new GFF3Writer();<a name="line.66"></a>
<span class="sourceLineNo">067</span>        gff3Writer.write(fo, chromosomeSequenceHashMap);<a name="line.67"></a>
<span class="sourceLineNo">068</span>        fo.close();<a name="line.68"></a>
<span class="sourceLineNo">069</span><a name="line.69"></a>
<span class="sourceLineNo">070</span>    }<a name="line.70"></a>
<span class="sourceLineNo">071</span><a name="line.71"></a>
<span class="sourceLineNo">072</span>    /**<a name="line.72"></a>
<span class="sourceLineNo">073</span>     * Test of outputFastaSequenceLengthGFF3 method, of class GeneFeatureHelper.<a name="line.73"></a>
<span class="sourceLineNo">074</span>     */<a name="line.74"></a>
<span class="sourceLineNo">075</span>    public void testOutputFastaSequenceLengthGFF3() throws Exception {<a name="line.75"></a>
<span class="sourceLineNo">076</span>//        System.out.println("outputFastaSequenceLengthGFF3");<a name="line.76"></a>
<span class="sourceLineNo">077</span><a name="line.77"></a>
<span class="sourceLineNo">078</span>        File fastaSequenceFile = new File("src/test/resources/volvox_all.fna");<a name="line.78"></a>
<span class="sourceLineNo">079</span>        File gffFile = File.createTempFile("volvox_length", "gff3");<a name="line.79"></a>
<span class="sourceLineNo">080</span>        gffFile.deleteOnExit();<a name="line.80"></a>
<span class="sourceLineNo">081</span>        GeneFeatureHelper.outputFastaSequenceLengthGFF3(fastaSequenceFile, gffFile);<a name="line.81"></a>
<span class="sourceLineNo">082</span>        FileAssert.assertBinaryEquals("volvox_length.gff3 and volvox_length_output.gff3 are not equal", gffFile, new File("src/test/resources/volvox_length_reference.gff3"));<a name="line.82"></a>
<span class="sourceLineNo">083</span><a name="line.83"></a>
<span class="sourceLineNo">084</span>    }<a name="line.84"></a>
<span class="sourceLineNo">085</span><a name="line.85"></a>
<span class="sourceLineNo">086</span>    /**<a name="line.86"></a>
<span class="sourceLineNo">087</span>     * Test if the note from a gff3 file is added to the gene sequence<a name="line.87"></a>
<span class="sourceLineNo">088</span>     * @throws Exception<a name="line.88"></a>
<span class="sourceLineNo">089</span>     */<a name="line.89"></a>
<span class="sourceLineNo">090</span><a name="line.90"></a>
<span class="sourceLineNo">091</span>    public void testAddGFF3Note() throws Exception {<a name="line.91"></a>
<span class="sourceLineNo">092</span>        LinkedHashMap&lt;String, ChromosomeSequence&gt; chromosomeSequenceList = GeneFeatureHelper.loadFastaAddGeneFeaturesFromGmodGFF3(new File("src/test/resources/volvox_all.fna"), new File("src/test/resources/volvox.gff3"),false);<a name="line.92"></a>
<span class="sourceLineNo">093</span>        ChromosomeSequence ctgASequence = chromosomeSequenceList.get("ctgA");<a name="line.93"></a>
<span class="sourceLineNo">094</span>        GeneSequence edenGeneSequence = ctgASequence.getGene("EDEN");<a name="line.94"></a>
<span class="sourceLineNo">095</span>        System.out.println("Note " + edenGeneSequence.getNotesList());<a name="line.95"></a>
<span class="sourceLineNo">096</span><a name="line.96"></a>
<span class="sourceLineNo">097</span>    }<a name="line.97"></a>
<span class="sourceLineNo">098</span><a name="line.98"></a>
<span class="sourceLineNo">099</span>    /**<a name="line.99"></a>
<span class="sourceLineNo">100</span>     * Test of getProteinSequences method, of class GeneFeatureHelper. Used gff3 file that was modified from the volvox gff version. Do not<a name="line.100"></a>
<span class="sourceLineNo">101</span>     * have the reference protein that is generated from each CDS record so subject to being incorrect without a validated test case.<a name="line.101"></a>
<span class="sourceLineNo">102</span>     * Could not find anyone providing a gff3 test case with expected protein output.<a name="line.102"></a>
<span class="sourceLineNo">103</span>     */<a name="line.103"></a>
<span class="sourceLineNo">104</span>    public void testGetProteinSequences() throws Exception {<a name="line.104"></a>
<span class="sourceLineNo">105</span>        LinkedHashMap&lt;String, ChromosomeSequence&gt; chromosomeSequenceList = GeneFeatureHelper.loadFastaAddGeneFeaturesFromGmodGFF3(new File("src/test/resources/volvox_all.fna"), new File("src/test/resources/volvox.gff3"),false);<a name="line.105"></a>
<span class="sourceLineNo">106</span>        LinkedHashMap&lt;String, ProteinSequence&gt; proteinSequenceList = GeneFeatureHelper.getProteinSequences(chromosomeSequenceList.values());<a name="line.106"></a>
<span class="sourceLineNo">107</span>//        for(ProteinSequence proteinSequence : proteinSequenceList.values()){<a name="line.107"></a>
<span class="sourceLineNo">108</span>//            System.out.println("Output=" + proteinSequence.getSequenceAsString());<a name="line.108"></a>
<span class="sourceLineNo">109</span>//        }<a name="line.109"></a>
<span class="sourceLineNo">110</span>        File tmp = File.createTempFile("volvox_all", "faa");<a name="line.110"></a>
<span class="sourceLineNo">111</span>        tmp.deleteOnExit();<a name="line.111"></a>
<span class="sourceLineNo">112</span>        FastaWriterHelper.writeProteinSequence(tmp, proteinSequenceList.values());<a name="line.112"></a>
<span class="sourceLineNo">113</span>        FileAssert.assertEquals("volvox_all_reference.faa and volvox_all.faa are not equal", new File("src/test/resources/volvox_all_reference.faa"), tmp);<a name="line.113"></a>
<span class="sourceLineNo">114</span>    }<a name="line.114"></a>
<span class="sourceLineNo">115</span>   <a name="line.115"></a>
<span class="sourceLineNo">116</span>    /**<a name="line.116"></a>
<span class="sourceLineNo">117</span>     * Test of getGeneSequences method, of class GeneFeatureHelper.<a name="line.117"></a>
<span class="sourceLineNo">118</span>     */<a name="line.118"></a>
<span class="sourceLineNo">119</span>    public void testGetGeneSequences() throws Exception {<a name="line.119"></a>
<span class="sourceLineNo">120</span>//        System.out.println("getGeneSequences");<a name="line.120"></a>
<span class="sourceLineNo">121</span>        LinkedHashMap&lt;String, ChromosomeSequence&gt; chromosomeSequenceList = GeneFeatureHelper.loadFastaAddGeneFeaturesFromGmodGFF3(new File("src/test/resources/volvox_all.fna"), new File("src/test/resources/volvox.gff3"),true);<a name="line.121"></a>
<span class="sourceLineNo">122</span>        LinkedHashMap&lt;String, GeneSequence&gt; geneSequenceHashMap = GeneFeatureHelper.getGeneSequences(chromosomeSequenceList.values());<a name="line.122"></a>
<span class="sourceLineNo">123</span>        Collection&lt;GeneSequence&gt; geneSequences = geneSequenceHashMap.values();<a name="line.123"></a>
<span class="sourceLineNo">124</span>        FastaWriterHelper.writeGeneSequence(new File("src/test/resources/volvox_all_genes_exon_uppercase.fna"), geneSequences, true);<a name="line.124"></a>
<span class="sourceLineNo">125</span>    }<a name="line.125"></a>
<span class="sourceLineNo">126</span><a name="line.126"></a>
<span class="sourceLineNo">127</span><a name="line.127"></a>
<span class="sourceLineNo">128</span>}<a name="line.128"></a>




























































</pre>
</div>
</body>
</html>
