<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<html lang="en">
<head>
<title>Source code</title>
<link rel="stylesheet" type="text/css" href="../../../../../../stylesheet.css" title="Style">
</head>
<body>
<div class="sourceContainer">
<pre><span class="sourceLineNo">001</span>/*<a name="line.1"></a>
<span class="sourceLineNo">002</span> *                    BioJava development code<a name="line.2"></a>
<span class="sourceLineNo">003</span> *<a name="line.3"></a>
<span class="sourceLineNo">004</span> * This code may be freely distributed and modified under the<a name="line.4"></a>
<span class="sourceLineNo">005</span> * terms of the GNU Lesser General Public Licence.  This should<a name="line.5"></a>
<span class="sourceLineNo">006</span> * be distributed with the code.  If you do not have a copy,<a name="line.6"></a>
<span class="sourceLineNo">007</span> * see:<a name="line.7"></a>
<span class="sourceLineNo">008</span> *<a name="line.8"></a>
<span class="sourceLineNo">009</span> *      http://www.gnu.org/copyleft/lesser.html<a name="line.9"></a>
<span class="sourceLineNo">010</span> *<a name="line.10"></a>
<span class="sourceLineNo">011</span> * Copyright for this code is held jointly by the individual<a name="line.11"></a>
<span class="sourceLineNo">012</span> * authors.  These should be listed in @author doc comments.<a name="line.12"></a>
<span class="sourceLineNo">013</span> *<a name="line.13"></a>
<span class="sourceLineNo">014</span> * For more information on the BioJava project and its aims,<a name="line.14"></a>
<span class="sourceLineNo">015</span> * or to join the biojava-l mailing list, visit the home page<a name="line.15"></a>
<span class="sourceLineNo">016</span> * at:<a name="line.16"></a>
<span class="sourceLineNo">017</span> *<a name="line.17"></a>
<span class="sourceLineNo">018</span> *      http://www.biojava.org/<a name="line.18"></a>
<span class="sourceLineNo">019</span> *<a name="line.19"></a>
<span class="sourceLineNo">020</span> * Created on 01-21-2010<a name="line.20"></a>
<span class="sourceLineNo">021</span> */<a name="line.21"></a>
<span class="sourceLineNo">022</span>package org.biojava3.core.sequence.io;<a name="line.22"></a>
<span class="sourceLineNo">023</span><a name="line.23"></a>
<span class="sourceLineNo">024</span>import java.io.BufferedOutputStream;<a name="line.24"></a>
<span class="sourceLineNo">025</span>import java.io.File;<a name="line.25"></a>
<span class="sourceLineNo">026</span>import java.io.FileOutputStream;<a name="line.26"></a>
<span class="sourceLineNo">027</span>import java.io.OutputStream;<a name="line.27"></a>
<span class="sourceLineNo">028</span>import java.util.ArrayList;<a name="line.28"></a>
<span class="sourceLineNo">029</span>import java.util.Collection;<a name="line.29"></a>
<span class="sourceLineNo">030</span>import org.biojava3.core.sequence.DNASequence;<a name="line.30"></a>
<span class="sourceLineNo">031</span>import org.biojava3.core.sequence.GeneSequence;<a name="line.31"></a>
<span class="sourceLineNo">032</span>import org.biojava3.core.sequence.ProteinSequence;<a name="line.32"></a>
<span class="sourceLineNo">033</span><a name="line.33"></a>
<span class="sourceLineNo">034</span>import org.biojava3.core.sequence.compound.AminoAcidCompound;<a name="line.34"></a>
<span class="sourceLineNo">035</span>import org.biojava3.core.sequence.compound.NucleotideCompound;<a name="line.35"></a>
<span class="sourceLineNo">036</span>import org.biojava3.core.sequence.io.template.FastaHeaderFormatInterface;<a name="line.36"></a>
<span class="sourceLineNo">037</span>import org.biojava3.core.sequence.template.Compound;<a name="line.37"></a>
<span class="sourceLineNo">038</span>import org.biojava3.core.sequence.template.Sequence;<a name="line.38"></a>
<span class="sourceLineNo">039</span><a name="line.39"></a>
<span class="sourceLineNo">040</span>/**<a name="line.40"></a>
<span class="sourceLineNo">041</span> * The class that should be used to write out fasta file of a sequence collection<a name="line.41"></a>
<span class="sourceLineNo">042</span> * @author Scooter Willis &lt;willishf at gmail dot com&gt;<a name="line.42"></a>
<span class="sourceLineNo">043</span> */<a name="line.43"></a>
<span class="sourceLineNo">044</span>public class FastaWriterHelper {<a name="line.44"></a>
<span class="sourceLineNo">045</span><a name="line.45"></a>
<span class="sourceLineNo">046</span><a name="line.46"></a>
<span class="sourceLineNo">047</span><a name="line.47"></a>
<span class="sourceLineNo">048</span>    /**<a name="line.48"></a>
<span class="sourceLineNo">049</span>     * Write collection of protein sequences to a file<a name="line.49"></a>
<span class="sourceLineNo">050</span>     *<a name="line.50"></a>
<span class="sourceLineNo">051</span>     * @param file<a name="line.51"></a>
<span class="sourceLineNo">052</span>     * @param proteinSequences<a name="line.52"></a>
<span class="sourceLineNo">053</span>     * @throws Exception<a name="line.53"></a>
<span class="sourceLineNo">054</span>     */<a name="line.54"></a>
<span class="sourceLineNo">055</span>    public static void writeProteinSequence(File file,<a name="line.55"></a>
<span class="sourceLineNo">056</span>            Collection&lt;ProteinSequence&gt; proteinSequences) throws Exception {<a name="line.56"></a>
<span class="sourceLineNo">057</span>        FileOutputStream outputStream = new FileOutputStream(file);<a name="line.57"></a>
<span class="sourceLineNo">058</span>        BufferedOutputStream bo = new BufferedOutputStream(outputStream);<a name="line.58"></a>
<span class="sourceLineNo">059</span>        writeProteinSequence(bo, proteinSequences);<a name="line.59"></a>
<span class="sourceLineNo">060</span>        bo.close();<a name="line.60"></a>
<span class="sourceLineNo">061</span>        outputStream.close();<a name="line.61"></a>
<span class="sourceLineNo">062</span>    }<a name="line.62"></a>
<span class="sourceLineNo">063</span><a name="line.63"></a>
<span class="sourceLineNo">064</span>    /**<a name="line.64"></a>
<span class="sourceLineNo">065</span>     * Write collection of protein sequences to a stream<a name="line.65"></a>
<span class="sourceLineNo">066</span>     * @param outputStream<a name="line.66"></a>
<span class="sourceLineNo">067</span>     * @param proteinSequences<a name="line.67"></a>
<span class="sourceLineNo">068</span>     * @throws Exception<a name="line.68"></a>
<span class="sourceLineNo">069</span>     */<a name="line.69"></a>
<span class="sourceLineNo">070</span><a name="line.70"></a>
<span class="sourceLineNo">071</span>    public static void writeProteinSequence(OutputStream outputStream,<a name="line.71"></a>
<span class="sourceLineNo">072</span>            Collection&lt;ProteinSequence&gt; proteinSequences) throws Exception {<a name="line.72"></a>
<span class="sourceLineNo">073</span><a name="line.73"></a>
<span class="sourceLineNo">074</span>        FastaWriter&lt;ProteinSequence, AminoAcidCompound&gt; fastaWriter = new FastaWriter&lt;ProteinSequence, AminoAcidCompound&gt;(<a name="line.74"></a>
<span class="sourceLineNo">075</span>                outputStream, proteinSequences,<a name="line.75"></a>
<span class="sourceLineNo">076</span>                new GenericFastaHeaderFormat&lt;ProteinSequence, AminoAcidCompound&gt;());<a name="line.76"></a>
<span class="sourceLineNo">077</span>        fastaWriter.process();<a name="line.77"></a>
<span class="sourceLineNo">078</span><a name="line.78"></a>
<span class="sourceLineNo">079</span>    }<a name="line.79"></a>
<span class="sourceLineNo">080</span><a name="line.80"></a>
<span class="sourceLineNo">081</span>        /**<a name="line.81"></a>
<span class="sourceLineNo">082</span>     * Write a collection of GeneSequences to a file where if the gene is negative strand it will flip and complement the sequence<a name="line.82"></a>
<span class="sourceLineNo">083</span>     * @param file<a name="line.83"></a>
<span class="sourceLineNo">084</span>     * @param geneSequences<a name="line.84"></a>
<span class="sourceLineNo">085</span>     * @throws Exception<a name="line.85"></a>
<span class="sourceLineNo">086</span>     */<a name="line.86"></a>
<span class="sourceLineNo">087</span><a name="line.87"></a>
<span class="sourceLineNo">088</span>    public static void writeGeneSequence(File file, Collection&lt;GeneSequence&gt; geneSequences,boolean showExonUppercase) throws Exception {<a name="line.88"></a>
<span class="sourceLineNo">089</span>        FileOutputStream outputStream = new FileOutputStream(file);<a name="line.89"></a>
<span class="sourceLineNo">090</span>        BufferedOutputStream bo = new BufferedOutputStream(outputStream);<a name="line.90"></a>
<span class="sourceLineNo">091</span>        writeGeneSequence(bo, geneSequences,showExonUppercase);<a name="line.91"></a>
<span class="sourceLineNo">092</span>        bo.close();<a name="line.92"></a>
<span class="sourceLineNo">093</span>        outputStream.close();<a name="line.93"></a>
<span class="sourceLineNo">094</span>    }<a name="line.94"></a>
<span class="sourceLineNo">095</span><a name="line.95"></a>
<span class="sourceLineNo">096</span>    /**<a name="line.96"></a>
<span class="sourceLineNo">097</span>     * Write a collection of GeneSequences to a file where if the gene is negative strand it will flip and complement the sequence<a name="line.97"></a>
<span class="sourceLineNo">098</span>     * @param outputStream<a name="line.98"></a>
<span class="sourceLineNo">099</span>     * @param dnaSequences<a name="line.99"></a>
<span class="sourceLineNo">100</span>     * @throws Exception<a name="line.100"></a>
<span class="sourceLineNo">101</span>     */<a name="line.101"></a>
<span class="sourceLineNo">102</span><a name="line.102"></a>
<span class="sourceLineNo">103</span>    public static void writeGeneSequence(OutputStream outputStream, Collection&lt;GeneSequence&gt; geneSequences,boolean showExonUppercase) throws Exception {<a name="line.103"></a>
<span class="sourceLineNo">104</span>        FastaGeneWriter fastaWriter = new FastaGeneWriter(<a name="line.104"></a>
<span class="sourceLineNo">105</span>                outputStream, geneSequences,<a name="line.105"></a>
<span class="sourceLineNo">106</span>                new GenericFastaHeaderFormat&lt;GeneSequence, NucleotideCompound&gt;(),showExonUppercase);<a name="line.106"></a>
<span class="sourceLineNo">107</span>        fastaWriter.process();<a name="line.107"></a>
<span class="sourceLineNo">108</span><a name="line.108"></a>
<span class="sourceLineNo">109</span>    }<a name="line.109"></a>
<span class="sourceLineNo">110</span><a name="line.110"></a>
<span class="sourceLineNo">111</span><a name="line.111"></a>
<span class="sourceLineNo">112</span>    /**<a name="line.112"></a>
<span class="sourceLineNo">113</span>     * Write a collection of NucleotideSequences to a file<a name="line.113"></a>
<span class="sourceLineNo">114</span>     * @param file<a name="line.114"></a>
<span class="sourceLineNo">115</span>     * @param dnaSequences<a name="line.115"></a>
<span class="sourceLineNo">116</span>     * @throws Exception<a name="line.116"></a>
<span class="sourceLineNo">117</span>     */<a name="line.117"></a>
<span class="sourceLineNo">118</span><a name="line.118"></a>
<span class="sourceLineNo">119</span>    public static void writeNucleotideSequence(File file, Collection&lt;DNASequence&gt; dnaSequences) throws Exception {<a name="line.119"></a>
<span class="sourceLineNo">120</span>        FileOutputStream outputStream = new FileOutputStream(file);<a name="line.120"></a>
<span class="sourceLineNo">121</span>        BufferedOutputStream bo = new BufferedOutputStream(outputStream);<a name="line.121"></a>
<span class="sourceLineNo">122</span>        writeNucleotideSequence(bo, dnaSequences);<a name="line.122"></a>
<span class="sourceLineNo">123</span>        bo.close();<a name="line.123"></a>
<span class="sourceLineNo">124</span>        outputStream.close();<a name="line.124"></a>
<span class="sourceLineNo">125</span>    }<a name="line.125"></a>
<span class="sourceLineNo">126</span><a name="line.126"></a>
<span class="sourceLineNo">127</span>    /**<a name="line.127"></a>
<span class="sourceLineNo">128</span>     * Write a collection of NucleotideSequences to a file<a name="line.128"></a>
<span class="sourceLineNo">129</span>     * @param outputStream<a name="line.129"></a>
<span class="sourceLineNo">130</span>     * @param dnaSequences<a name="line.130"></a>
<span class="sourceLineNo">131</span>     * @throws Exception<a name="line.131"></a>
<span class="sourceLineNo">132</span>     */<a name="line.132"></a>
<span class="sourceLineNo">133</span><a name="line.133"></a>
<span class="sourceLineNo">134</span>    public static void writeNucleotideSequence(OutputStream outputStream, Collection&lt;DNASequence&gt; dnaSequences) throws Exception {<a name="line.134"></a>
<span class="sourceLineNo">135</span>        FastaWriter&lt;DNASequence, NucleotideCompound&gt; fastaWriter = new FastaWriter&lt;DNASequence, NucleotideCompound&gt;(<a name="line.135"></a>
<span class="sourceLineNo">136</span>                outputStream, dnaSequences,<a name="line.136"></a>
<span class="sourceLineNo">137</span>                new GenericFastaHeaderFormat&lt;DNASequence, NucleotideCompound&gt;());<a name="line.137"></a>
<span class="sourceLineNo">138</span>        fastaWriter.process();<a name="line.138"></a>
<span class="sourceLineNo">139</span><a name="line.139"></a>
<span class="sourceLineNo">140</span>    }<a name="line.140"></a>
<span class="sourceLineNo">141</span><a name="line.141"></a>
<span class="sourceLineNo">142</span>    /**<a name="line.142"></a>
<span class="sourceLineNo">143</span>     * Write a sequence to a file<a name="line.143"></a>
<span class="sourceLineNo">144</span>     * @param file<a name="line.144"></a>
<span class="sourceLineNo">145</span>     * @param sequence<a name="line.145"></a>
<span class="sourceLineNo">146</span>     * @throws Exception<a name="line.146"></a>
<span class="sourceLineNo">147</span>     */<a name="line.147"></a>
<span class="sourceLineNo">148</span>    public static void writeSequence(File file, Sequence&lt;?&gt; sequence) throws Exception {<a name="line.148"></a>
<span class="sourceLineNo">149</span>        FileOutputStream outputStream = new FileOutputStream(file);<a name="line.149"></a>
<span class="sourceLineNo">150</span>        BufferedOutputStream bo = new BufferedOutputStream(outputStream);<a name="line.150"></a>
<span class="sourceLineNo">151</span>        writeSequences(bo, singleSeqToCollection(sequence));<a name="line.151"></a>
<span class="sourceLineNo">152</span>        bo.close();<a name="line.152"></a>
<span class="sourceLineNo">153</span>        outputStream.close();<a name="line.153"></a>
<span class="sourceLineNo">154</span>    }<a name="line.154"></a>
<span class="sourceLineNo">155</span><a name="line.155"></a>
<span class="sourceLineNo">156</span>    /**<a name="line.156"></a>
<span class="sourceLineNo">157</span>     * Write a sequence to OutputStream<a name="line.157"></a>
<span class="sourceLineNo">158</span>     * @param outputStream<a name="line.158"></a>
<span class="sourceLineNo">159</span>     * @param sequence<a name="line.159"></a>
<span class="sourceLineNo">160</span>     * @throws Exception<a name="line.160"></a>
<span class="sourceLineNo">161</span>     */<a name="line.161"></a>
<span class="sourceLineNo">162</span>    public static void writeSequence(OutputStream outputStream, Sequence&lt;?&gt; sequence) throws Exception {<a name="line.162"></a>
<span class="sourceLineNo">163</span>        writeSequences(outputStream, singleSeqToCollection(sequence));<a name="line.163"></a>
<span class="sourceLineNo">164</span>    }<a name="line.164"></a>
<span class="sourceLineNo">165</span><a name="line.165"></a>
<span class="sourceLineNo">166</span>    /**<a name="line.166"></a>
<span class="sourceLineNo">167</span>     * <a name="line.167"></a>
<span class="sourceLineNo">168</span>     * @param sequence<a name="line.168"></a>
<span class="sourceLineNo">169</span>     * @return<a name="line.169"></a>
<span class="sourceLineNo">170</span>     */<a name="line.170"></a>
<span class="sourceLineNo">171</span><a name="line.171"></a>
<span class="sourceLineNo">172</span>    private static Collection&lt;Sequence&lt;?&gt;&gt; singleSeqToCollection(Sequence&lt;?&gt; sequence) {<a name="line.172"></a>
<span class="sourceLineNo">173</span>        Collection&lt;Sequence&lt;?&gt;&gt; sequences = new ArrayList&lt;Sequence&lt;?&gt;&gt;();<a name="line.173"></a>
<span class="sourceLineNo">174</span>        sequences.add(sequence);<a name="line.174"></a>
<span class="sourceLineNo">175</span>        return sequences;<a name="line.175"></a>
<span class="sourceLineNo">176</span>    }<a name="line.176"></a>
<span class="sourceLineNo">177</span><a name="line.177"></a>
<span class="sourceLineNo">178</span>    /**<a name="line.178"></a>
<span class="sourceLineNo">179</span>     * Method which will write your given Sequences to the specified<a name="line.179"></a>
<span class="sourceLineNo">180</span>     * {@link OutputStream}. This is a very generic method which writes just the<a name="line.180"></a>
<span class="sourceLineNo">181</span>     * AccessionID of the Sequence as the FASTA header.<a name="line.181"></a>
<span class="sourceLineNo">182</span>     *<a name="line.182"></a>
<span class="sourceLineNo">183</span>     * @param outputStream Stream to write to; can be System.out<a name="line.183"></a>
<span class="sourceLineNo">184</span>     * @param sequences The sequences to write out<a name="line.184"></a>
<span class="sourceLineNo">185</span>     * @throws Exception Thrown normally thanks to IO problems<a name="line.185"></a>
<span class="sourceLineNo">186</span>     */<a name="line.186"></a>
<span class="sourceLineNo">187</span>    public static void writeSequences(OutputStream outputStream,<a name="line.187"></a>
<span class="sourceLineNo">188</span>            Collection&lt;Sequence&lt;?&gt;&gt; sequences) throws Exception {<a name="line.188"></a>
<span class="sourceLineNo">189</span><a name="line.189"></a>
<span class="sourceLineNo">190</span>        FastaHeaderFormatInterface&lt;Sequence&lt;?&gt;, Compound&gt; fhfi =<a name="line.190"></a>
<span class="sourceLineNo">191</span>                new FastaHeaderFormatInterface&lt;Sequence&lt;?&gt;, Compound&gt;() {<a name="line.191"></a>
<span class="sourceLineNo">192</span><a name="line.192"></a>
<span class="sourceLineNo">193</span>                    public String getHeader(Sequence&lt;?&gt; sequence) {<a name="line.193"></a>
<span class="sourceLineNo">194</span>                        return sequence.getAccession().toString();<a name="line.194"></a>
<span class="sourceLineNo">195</span>                    }<a name="line.195"></a>
<span class="sourceLineNo">196</span><a name="line.196"></a>
<span class="sourceLineNo">197</span>                    ;<a name="line.197"></a>
<span class="sourceLineNo">198</span>                };<a name="line.198"></a>
<span class="sourceLineNo">199</span><a name="line.199"></a>
<span class="sourceLineNo">200</span>        FastaWriter&lt;Sequence&lt;?&gt;, Compound&gt; fastaWriter =<a name="line.200"></a>
<span class="sourceLineNo">201</span>                new FastaWriter&lt;Sequence&lt;?&gt;, Compound&gt;(outputStream,<a name="line.201"></a>
<span class="sourceLineNo">202</span>                sequences, fhfi);<a name="line.202"></a>
<span class="sourceLineNo">203</span><a name="line.203"></a>
<span class="sourceLineNo">204</span>        fastaWriter.process();<a name="line.204"></a>
<span class="sourceLineNo">205</span>    }<a name="line.205"></a>
<span class="sourceLineNo">206</span>}<a name="line.206"></a>




























































</pre>
</div>
</body>
</html>
