<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<html lang="en">
<head>
<title>Source code</title>
<link rel="stylesheet" type="text/css" href="../../../../stylesheet.css" title="Style">
</head>
<body>
<div class="sourceContainer">
<pre><span class="sourceLineNo">001</span>package org.biojava3.alignment;<a name="line.1"></a>
<span class="sourceLineNo">002</span><a name="line.2"></a>
<span class="sourceLineNo">003</span>import java.io.File;<a name="line.3"></a>
<span class="sourceLineNo">004</span>import java.io.PrintStream;<a name="line.4"></a>
<span class="sourceLineNo">005</span>import java.util.ArrayList;<a name="line.5"></a>
<span class="sourceLineNo">006</span>import java.util.List;<a name="line.6"></a>
<span class="sourceLineNo">007</span><a name="line.7"></a>
<span class="sourceLineNo">008</span>import org.biojava3.alignment.Alignments.PairwiseSequenceScorerType;<a name="line.8"></a>
<span class="sourceLineNo">009</span>import org.biojava3.alignment.Alignments.ProfileProfileAlignerType;<a name="line.9"></a>
<span class="sourceLineNo">010</span>import org.biojava3.alignment.template.GapPenalty;<a name="line.10"></a>
<span class="sourceLineNo">011</span>import org.biojava3.alignment.template.PairwiseSequenceScorer;<a name="line.11"></a>
<span class="sourceLineNo">012</span>import org.biojava3.alignment.template.Profile;<a name="line.12"></a>
<span class="sourceLineNo">013</span>import org.biojava3.alignment.template.SubstitutionMatrix;<a name="line.13"></a>
<span class="sourceLineNo">014</span>import org.biojava3.core.sequence.ProteinSequence;<a name="line.14"></a>
<span class="sourceLineNo">015</span>import org.biojava3.core.sequence.compound.AminoAcidCompound;<a name="line.15"></a>
<span class="sourceLineNo">016</span>import org.biojava3.core.sequence.io.FastaReaderHelper;<a name="line.16"></a>
<span class="sourceLineNo">017</span>import org.biojava3.core.util.ConcurrencyTools;<a name="line.17"></a>
<span class="sourceLineNo">018</span><a name="line.18"></a>
<span class="sourceLineNo">019</span>public class CookbookMSAProfiler {<a name="line.19"></a>
<span class="sourceLineNo">020</span><a name="line.20"></a>
<span class="sourceLineNo">021</span>    private static class Profiler {<a name="line.21"></a>
<span class="sourceLineNo">022</span><a name="line.22"></a>
<span class="sourceLineNo">023</span>        private long maxMemoryUsed, timeCheckpoint;<a name="line.23"></a>
<span class="sourceLineNo">024</span>        private final long timeStart;<a name="line.24"></a>
<span class="sourceLineNo">025</span><a name="line.25"></a>
<span class="sourceLineNo">026</span>        private Profiler() {<a name="line.26"></a>
<span class="sourceLineNo">027</span>            maxMemoryUsed = Runtime.getRuntime().totalMemory();<a name="line.27"></a>
<span class="sourceLineNo">028</span>            timeStart = timeCheckpoint = System.nanoTime();<a name="line.28"></a>
<span class="sourceLineNo">029</span>        }<a name="line.29"></a>
<span class="sourceLineNo">030</span><a name="line.30"></a>
<span class="sourceLineNo">031</span>        private long getMaxMemoryUsed() {<a name="line.31"></a>
<span class="sourceLineNo">032</span>            return maxMemoryUsed = Math.max(maxMemoryUsed, Runtime.getRuntime().totalMemory());<a name="line.32"></a>
<span class="sourceLineNo">033</span>        }<a name="line.33"></a>
<span class="sourceLineNo">034</span><a name="line.34"></a>
<span class="sourceLineNo">035</span>        private long getTimeSinceCheckpoint() {<a name="line.35"></a>
<span class="sourceLineNo">036</span>            return System.nanoTime() - timeCheckpoint;<a name="line.36"></a>
<span class="sourceLineNo">037</span>        }<a name="line.37"></a>
<span class="sourceLineNo">038</span><a name="line.38"></a>
<span class="sourceLineNo">039</span>        private long getTimeSinceStart() {<a name="line.39"></a>
<span class="sourceLineNo">040</span>            return System.nanoTime() - timeStart;<a name="line.40"></a>
<span class="sourceLineNo">041</span>        }<a name="line.41"></a>
<span class="sourceLineNo">042</span><a name="line.42"></a>
<span class="sourceLineNo">043</span>        private void setCheckpoint() {<a name="line.43"></a>
<span class="sourceLineNo">044</span>            maxMemoryUsed = Math.max(maxMemoryUsed, Runtime.getRuntime().totalMemory());<a name="line.44"></a>
<span class="sourceLineNo">045</span>            timeCheckpoint = System.nanoTime();<a name="line.45"></a>
<span class="sourceLineNo">046</span>        }<a name="line.46"></a>
<span class="sourceLineNo">047</span><a name="line.47"></a>
<span class="sourceLineNo">048</span>    }<a name="line.48"></a>
<span class="sourceLineNo">049</span><a name="line.49"></a>
<span class="sourceLineNo">050</span>    public static void main(String[] args) throws Exception {<a name="line.50"></a>
<span class="sourceLineNo">051</span><a name="line.51"></a>
<span class="sourceLineNo">052</span>        if (args.length &lt; 1) {<a name="line.52"></a>
<span class="sourceLineNo">053</span>            System.err.println("The first argument must be a fasta file of protein sequences.");<a name="line.53"></a>
<span class="sourceLineNo">054</span>            return;<a name="line.54"></a>
<span class="sourceLineNo">055</span>        }<a name="line.55"></a>
<span class="sourceLineNo">056</span><a name="line.56"></a>
<span class="sourceLineNo">057</span>        // ConcurrencyTools.setThreadPoolSingle();<a name="line.57"></a>
<span class="sourceLineNo">058</span><a name="line.58"></a>
<span class="sourceLineNo">059</span>        PrintStream fout = new PrintStream("msa.txt");<a name="line.59"></a>
<span class="sourceLineNo">060</span>        Profiler profiler = new Profiler();<a name="line.60"></a>
<span class="sourceLineNo">061</span><a name="line.61"></a>
<span class="sourceLineNo">062</span>        System.out.printf("Loading sequences from %s... ", args[0]);<a name="line.62"></a>
<span class="sourceLineNo">063</span>        List&lt;ProteinSequence&gt; list = new ArrayList&lt;ProteinSequence&gt;();<a name="line.63"></a>
<span class="sourceLineNo">064</span>        list.addAll(FastaReaderHelper.readFastaProteinSequence(new File(args[0])).values());<a name="line.64"></a>
<span class="sourceLineNo">065</span>        if (args.length &gt; 1 &amp;&amp; Integer.parseInt(args[1]) &lt; list.size()) {<a name="line.65"></a>
<span class="sourceLineNo">066</span>            System.out.printf("%s/%d", args[1], list.size());<a name="line.66"></a>
<span class="sourceLineNo">067</span>            list = list.subList(0, Integer.parseInt(args[1]));<a name="line.67"></a>
<span class="sourceLineNo">068</span>        } else {<a name="line.68"></a>
<span class="sourceLineNo">069</span>            System.out.printf("%d", list.size());<a name="line.69"></a>
<span class="sourceLineNo">070</span>        }<a name="line.70"></a>
<span class="sourceLineNo">071</span>        System.out.printf(" sequences in %d ms using %d kB%n%n", profiler.getTimeSinceCheckpoint()/1000000,<a name="line.71"></a>
<span class="sourceLineNo">072</span>                profiler.getMaxMemoryUsed()/1024);<a name="line.72"></a>
<span class="sourceLineNo">073</span><a name="line.73"></a>
<span class="sourceLineNo">074</span>        profiler.setCheckpoint();<a name="line.74"></a>
<span class="sourceLineNo">075</span><a name="line.75"></a>
<span class="sourceLineNo">076</span>        System.out.print("Stage 1: pairwise similarity calculation... ");<a name="line.76"></a>
<span class="sourceLineNo">077</span>        GapPenalty gaps = new SimpleGapPenalty();<a name="line.77"></a>
<span class="sourceLineNo">078</span>        SubstitutionMatrix&lt;AminoAcidCompound&gt; blosum62 = new SimpleSubstitutionMatrix&lt;AminoAcidCompound&gt;();<a name="line.78"></a>
<span class="sourceLineNo">079</span>        List&lt;PairwiseSequenceScorer&lt;ProteinSequence, AminoAcidCompound&gt;&gt; scorers = Alignments.getAllPairsScorers(list,<a name="line.79"></a>
<span class="sourceLineNo">080</span>                PairwiseSequenceScorerType.GLOBAL_IDENTITIES, gaps, blosum62);<a name="line.80"></a>
<span class="sourceLineNo">081</span>        Alignments.runPairwiseScorers(scorers);<a name="line.81"></a>
<span class="sourceLineNo">082</span>        System.out.printf("%d scores in %d ms using %d kB%n%n", scorers.size(),<a name="line.82"></a>
<span class="sourceLineNo">083</span>                profiler.getTimeSinceCheckpoint()/1000000, profiler.getMaxMemoryUsed()/1024);<a name="line.83"></a>
<span class="sourceLineNo">084</span><a name="line.84"></a>
<span class="sourceLineNo">085</span>        profiler.setCheckpoint();<a name="line.85"></a>
<span class="sourceLineNo">086</span><a name="line.86"></a>
<span class="sourceLineNo">087</span>        System.out.print("Stage 2: hierarchical clustering into a guide tree... ");<a name="line.87"></a>
<span class="sourceLineNo">088</span>        GuideTree&lt;ProteinSequence, AminoAcidCompound&gt; tree = new GuideTree&lt;ProteinSequence, AminoAcidCompound&gt;(list,<a name="line.88"></a>
<span class="sourceLineNo">089</span>                scorers);<a name="line.89"></a>
<span class="sourceLineNo">090</span>        scorers = null;<a name="line.90"></a>
<span class="sourceLineNo">091</span>        System.out.printf("%d ms using %d kB%n%n%s%n%n", profiler.getTimeSinceCheckpoint()/1000000,<a name="line.91"></a>
<span class="sourceLineNo">092</span>                profiler.getMaxMemoryUsed()/1024, tree);<a name="line.92"></a>
<span class="sourceLineNo">093</span><a name="line.93"></a>
<span class="sourceLineNo">094</span>        profiler.setCheckpoint();<a name="line.94"></a>
<span class="sourceLineNo">095</span><a name="line.95"></a>
<span class="sourceLineNo">096</span>        System.out.print("Stage 3: progressive alignment... ");<a name="line.96"></a>
<span class="sourceLineNo">097</span>        Profile&lt;ProteinSequence, AminoAcidCompound&gt; msa = Alignments.getProgressiveAlignment(tree,<a name="line.97"></a>
<span class="sourceLineNo">098</span>                ProfileProfileAlignerType.GLOBAL, gaps, blosum62);<a name="line.98"></a>
<span class="sourceLineNo">099</span>        System.out.printf("%d profile-profile alignments in %d ms using %d kB%n%n", list.size() - 1,<a name="line.99"></a>
<span class="sourceLineNo">100</span>                profiler.getTimeSinceCheckpoint()/1000000, profiler.getMaxMemoryUsed()/1024);<a name="line.100"></a>
<span class="sourceLineNo">101</span>        fout.print(msa);<a name="line.101"></a>
<span class="sourceLineNo">102</span>        fout.close();<a name="line.102"></a>
<span class="sourceLineNo">103</span><a name="line.103"></a>
<span class="sourceLineNo">104</span>        ConcurrencyTools.shutdown();<a name="line.104"></a>
<span class="sourceLineNo">105</span><a name="line.105"></a>
<span class="sourceLineNo">106</span>        System.out.printf("Total time: %d ms%nMemory use: %d kB%n", profiler.getTimeSinceStart()/1000000,<a name="line.106"></a>
<span class="sourceLineNo">107</span>                profiler.getMaxMemoryUsed()/1024);<a name="line.107"></a>
<span class="sourceLineNo">108</span><a name="line.108"></a>
<span class="sourceLineNo">109</span>    }<a name="line.109"></a>
<span class="sourceLineNo">110</span><a name="line.110"></a>
<span class="sourceLineNo">111</span>}<a name="line.111"></a>




























































</pre>
</div>
</body>
</html>
