<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!-- NewPage -->
<html lang="en">
<head>
<!-- Generated by javadoc (version 1.7.0) on Fri Sep 02 09:21:04 PDT 2011 -->
<meta http-equiv="Content-Type" content="text/html" charset="UTF-8">
<title>Uses of Package org.biojava3.core.sequence (biojava 3.0.2 API)</title>
<meta name="date" content="2011-09-02">
<link rel="stylesheet" type="text/css" href="../../../../stylesheet.css" title="Style">
</head>
<body>
<script type="text/javascript"><!--
    if (location.href.indexOf('is-external=true') == -1) {
        parent.document.title="Uses of Package org.biojava3.core.sequence (biojava 3.0.2 API)";
    }
//-->
</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
<!-- ========= START OF TOP NAVBAR ======= -->
<div class="topNav"><a name="navbar_top">
<!--   -->
</a><a href="#skip-navbar_top" title="Skip navigation links"></a><a name="navbar_top_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../../../../overview-summary.html">Overview</a></li>
<li><a href="package-summary.html">Package</a></li>
<li>Class</li>
<li class="navBarCell1Rev">Use</li>
<li><a href="package-tree.html">Tree</a></li>
<li><a href="../../../../deprecated-list.html">Deprecated</a></li>
<li><a href="../../../../index-all.html">Index</a></li>
<li><a href="../../../../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li>Prev</li>
<li>Next</li>
</ul>
<ul class="navList">
<li><a href="../../../../index.html?org/biojava3/core/sequence/package-use.html" target="_top">Frames</a></li>
<li><a href="package-use.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_top">
<li><a href="../../../../allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_top");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<a name="skip-navbar_top">
<!--   -->
</a></div>
<!-- ========= END OF TOP NAVBAR ========= -->
<div class="header">
<h1 title="Uses of Package org.biojava3.core.sequence" class="title">Uses of Package<br>org.biojava3.core.sequence</h1>
</div>
<div class="contentContainer">
<ul class="blockList">
<li class="blockList">
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing packages, and an explanation">
<caption><span>Packages that use <a href="../../../../org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Package</th>
<th class="colLast" scope="col">Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><a href="#org.biojava3.aaproperties">org.biojava3.aaproperties</a></td>
<td class="colLast">
<div class="block">Set of classes that responsible for providing APIs and Command Prompt access to generate some basic physico-chemical properties of protein sequences.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="#org.biojava3.aaproperties.profeat">org.biojava3.aaproperties.profeat</a></td>
<td class="colLast">
<div class="block">Set of classes that responsible for providing APIs access to generate some PROFEAT properties of protein sequences.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="#org.biojava3.aaproperties.profeat.convertor">org.biojava3.aaproperties.profeat.convertor</a></td>
<td class="colLast">
<div class="block">Set of classes that enable the conversion protein sequences into various attributes.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="#org.biojava3.alignment">org.biojava3.alignment</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="#org.biojava3.core.sequence">org.biojava3.core.sequence</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="#org.biojava3.core.sequence.io">org.biojava3.core.sequence.io</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="#org.biojava3.core.sequence.loader">org.biojava3.core.sequence.loader</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="#org.biojava3.core.sequence.location">org.biojava3.core.sequence.location</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="#org.biojava3.core.sequence.location.template">org.biojava3.core.sequence.location.template</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="#org.biojava3.core.sequence.storage">org.biojava3.core.sequence.storage</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="#org.biojava3.core.sequence.template">org.biojava3.core.sequence.template</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="#org.biojava3.core.sequence.transcription">org.biojava3.core.sequence.transcription</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="#org.biojava3.genome">org.biojava3.genome</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="#org.biojava3.genome.homology">org.biojava3.genome.homology</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="#org.biojava3.genome.parsers.geneid">org.biojava3.genome.parsers.geneid</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="#org.biojava3.genome.parsers.gff">org.biojava3.genome.parsers.gff</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="#org.biojava3.phylo">org.biojava3.phylo</a></td>
<td class="colLast">&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava3.aaproperties">
<!--   -->
</a>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing classes, and an explanation">
<caption><span>Classes in <a href="../../../../org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a> used by <a href="../../../../org/biojava3/aaproperties/package-summary.html">org.biojava3.aaproperties</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colOne" scope="col">Class and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/ProteinSequence.html#org.biojava3.aaproperties">ProteinSequence</a>
<div class="block">The representation of a ProteinSequence</div>
</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava3.aaproperties.profeat">
<!--   -->
</a>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing classes, and an explanation">
<caption><span>Classes in <a href="../../../../org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a> used by <a href="../../../../org/biojava3/aaproperties/profeat/package-summary.html">org.biojava3.aaproperties.profeat</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colOne" scope="col">Class and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/ProteinSequence.html#org.biojava3.aaproperties.profeat">ProteinSequence</a>
<div class="block">The representation of a ProteinSequence</div>
</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava3.aaproperties.profeat.convertor">
<!--   -->
</a>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing classes, and an explanation">
<caption><span>Classes in <a href="../../../../org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a> used by <a href="../../../../org/biojava3/aaproperties/profeat/convertor/package-summary.html">org.biojava3.aaproperties.profeat.convertor</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colOne" scope="col">Class and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/ProteinSequence.html#org.biojava3.aaproperties.profeat.convertor">ProteinSequence</a>
<div class="block">The representation of a ProteinSequence</div>
</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava3.alignment">
<!--   -->
</a>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing classes, and an explanation">
<caption><span>Classes in <a href="../../../../org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a> used by <a href="../../../../org/biojava3/alignment/package-summary.html">org.biojava3.alignment</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colOne" scope="col">Class and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/AccessionID.html#org.biojava3.alignment">AccessionID</a>
<div class="block">Used in Sequences as the unique indentifier.</div>
</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava3.core.sequence">
<!--   -->
</a>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing classes, and an explanation">
<caption><span>Classes in <a href="../../../../org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a> used by <a href="../../../../org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colOne" scope="col">Class and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/AccessionID.html#org.biojava3.core.sequence">AccessionID</a>
<div class="block">Used in Sequences as the unique indentifier.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/CDSSequence.html#org.biojava3.core.sequence">CDSSequence</a>
<div class="block">Represents a exon or coding sequence in a gene.</div>
</td>
</tr>
<tr class="altColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/ChromosomeSequence.html#org.biojava3.core.sequence">ChromosomeSequence</a>
<div class="block">A ChromosomeSequence is a DNASequence but keeps track of geneSequences</div>
</td>
</tr>
<tr class="rowColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/DataSource.html#org.biojava3.core.sequence">DataSource</a>
<div class="block">GenBank                           gi|gi-number|gb|accession|locus
 ENA Data Library                  gi|gi-number|emb|accession|locus
 DDBJ, DNA Database of Japan       gi|gi-number|dbj|accession|locus
 NBRF PIR                          pir||entry
 Protein Research Foundation       prf||name
 SWISS-PROT UNIPROT                sp|accession|name
 Brookhaven Protein Data Bank (1)  pdb|entry|chain
 Brookhaven Protein Data Bank (2)  entry:chain|PDBID|CHAIN|SEQUENCE
 Patents                           pat|country|number
 GenInfo Backbone Id               bbs|number
 General database identifier       gnl|database|identifier
 NCBI Reference Sequence           ref|accession|locus
 Local Sequence identifier         lcl|identifier</div>
</td>
</tr>
<tr class="altColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/DNASequence.html#org.biojava3.core.sequence">DNASequence</a>
<div class="block">This is class should model the attributes associated with a DNA sequence</div>
</td>
</tr>
<tr class="rowColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/DNASequence.DNAType.html#org.biojava3.core.sequence">DNASequence.DNAType</a>
<div class="block">The type of DNA sequence</div>
</td>
</tr>
<tr class="altColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/ExonSequence.html#org.biojava3.core.sequence">ExonSequence</a>
<div class="block">A gene contains a collection of Exon sequences</div>
</td>
</tr>
<tr class="rowColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/GeneSequence.html#org.biojava3.core.sequence">GeneSequence</a>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/IntronSequence.html#org.biojava3.core.sequence">IntronSequence</a>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/ProteinSequence.html#org.biojava3.core.sequence">ProteinSequence</a>
<div class="block">The representation of a ProteinSequence</div>
</td>
</tr>
<tr class="altColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/RNASequence.html#org.biojava3.core.sequence">RNASequence</a>
<div class="block">RNASequence where RNACompoundSet are the allowed values</div>
</td>
</tr>
<tr class="rowColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/SequenceOptimizationHints.SequenceCollection.html#org.biojava3.core.sequence">SequenceOptimizationHints.SequenceCollection</a>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/SequenceOptimizationHints.SequenceUsage.html#org.biojava3.core.sequence">SequenceOptimizationHints.SequenceUsage</a>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/StartCodonSequence.html#org.biojava3.core.sequence">StartCodonSequence</a>
<div class="block">Used to map the start codon feature on a gene</div>
</td>
</tr>
<tr class="altColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/StopCodonSequence.html#org.biojava3.core.sequence">StopCodonSequence</a>
<div class="block">Used to map the stop codon sequence on a gene</div>
</td>
</tr>
<tr class="rowColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/Strand.html#org.biojava3.core.sequence">Strand</a>
<div class="block">Provides a way of representing the strand of a sequence, location
 hit or feature.</div>
</td>
</tr>
<tr class="altColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/TranscriptSequence.html#org.biojava3.core.sequence">TranscriptSequence</a>
<div class="block">This is the sequence if you want to go from a gene sequence to a protein sequence.</div>
</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava3.core.sequence.io">
<!--   -->
</a>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing classes, and an explanation">
<caption><span>Classes in <a href="../../../../org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a> used by <a href="../../../../org/biojava3/core/sequence/io/package-summary.html">org.biojava3.core.sequence.io</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colOne" scope="col">Class and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/DNASequence.html#org.biojava3.core.sequence.io">DNASequence</a>
<div class="block">This is class should model the attributes associated with a DNA sequence</div>
</td>
</tr>
<tr class="rowColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/GeneSequence.html#org.biojava3.core.sequence.io">GeneSequence</a>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/ProteinSequence.html#org.biojava3.core.sequence.io">ProteinSequence</a>
<div class="block">The representation of a ProteinSequence</div>
</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava3.core.sequence.loader">
<!--   -->
</a>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing classes, and an explanation">
<caption><span>Classes in <a href="../../../../org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a> used by <a href="../../../../org/biojava3/core/sequence/loader/package-summary.html">org.biojava3.core.sequence.loader</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colOne" scope="col">Class and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/AccessionID.html#org.biojava3.core.sequence.loader">AccessionID</a>
<div class="block">Used in Sequences as the unique indentifier.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/Strand.html#org.biojava3.core.sequence.loader">Strand</a>
<div class="block">Provides a way of representing the strand of a sequence, location
 hit or feature.</div>
</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava3.core.sequence.location">
<!--   -->
</a>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing classes, and an explanation">
<caption><span>Classes in <a href="../../../../org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a> used by <a href="../../../../org/biojava3/core/sequence/location/package-summary.html">org.biojava3.core.sequence.location</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colOne" scope="col">Class and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/AccessionID.html#org.biojava3.core.sequence.location">AccessionID</a>
<div class="block">Used in Sequences as the unique indentifier.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/DataSource.html#org.biojava3.core.sequence.location">DataSource</a>
<div class="block">GenBank                           gi|gi-number|gb|accession|locus
 ENA Data Library                  gi|gi-number|emb|accession|locus
 DDBJ, DNA Database of Japan       gi|gi-number|dbj|accession|locus
 NBRF PIR                          pir||entry
 Protein Research Foundation       prf||name
 SWISS-PROT UNIPROT                sp|accession|name
 Brookhaven Protein Data Bank (1)  pdb|entry|chain
 Brookhaven Protein Data Bank (2)  entry:chain|PDBID|CHAIN|SEQUENCE
 Patents                           pat|country|number
 GenInfo Backbone Id               bbs|number
 General database identifier       gnl|database|identifier
 NCBI Reference Sequence           ref|accession|locus
 Local Sequence identifier         lcl|identifier</div>
</td>
</tr>
<tr class="altColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/Strand.html#org.biojava3.core.sequence.location">Strand</a>
<div class="block">Provides a way of representing the strand of a sequence, location
 hit or feature.</div>
</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava3.core.sequence.location.template">
<!--   -->
</a>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing classes, and an explanation">
<caption><span>Classes in <a href="../../../../org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a> used by <a href="../../../../org/biojava3/core/sequence/location/template/package-summary.html">org.biojava3.core.sequence.location.template</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colOne" scope="col">Class and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/AccessionID.html#org.biojava3.core.sequence.location.template">AccessionID</a>
<div class="block">Used in Sequences as the unique indentifier.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/Strand.html#org.biojava3.core.sequence.location.template">Strand</a>
<div class="block">Provides a way of representing the strand of a sequence, location
 hit or feature.</div>
</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava3.core.sequence.storage">
<!--   -->
</a>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing classes, and an explanation">
<caption><span>Classes in <a href="../../../../org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a> used by <a href="../../../../org/biojava3/core/sequence/storage/package-summary.html">org.biojava3.core.sequence.storage</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colOne" scope="col">Class and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/AccessionID.html#org.biojava3.core.sequence.storage">AccessionID</a>
<div class="block">Used in Sequences as the unique indentifier.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/Strand.html#org.biojava3.core.sequence.storage">Strand</a>
<div class="block">Provides a way of representing the strand of a sequence, location
 hit or feature.</div>
</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava3.core.sequence.template">
<!--   -->
</a>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing classes, and an explanation">
<caption><span>Classes in <a href="../../../../org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a> used by <a href="../../../../org/biojava3/core/sequence/template/package-summary.html">org.biojava3.core.sequence.template</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colOne" scope="col">Class and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/AccessionID.html#org.biojava3.core.sequence.template">AccessionID</a>
<div class="block">Used in Sequences as the unique indentifier.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/Strand.html#org.biojava3.core.sequence.template">Strand</a>
<div class="block">Provides a way of representing the strand of a sequence, location
 hit or feature.</div>
</td>
</tr>
<tr class="altColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/TaxonomyID.html#org.biojava3.core.sequence.template">TaxonomyID</a>
<div class="block">A sequence can be associated with a species or Taxonomy ID</div>
</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava3.core.sequence.transcription">
<!--   -->
</a>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing classes, and an explanation">
<caption><span>Classes in <a href="../../../../org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a> used by <a href="../../../../org/biojava3/core/sequence/transcription/package-summary.html">org.biojava3.core.sequence.transcription</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colOne" scope="col">Class and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/RNASequence.html#org.biojava3.core.sequence.transcription">RNASequence</a>
<div class="block">RNASequence where RNACompoundSet are the allowed values</div>
</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava3.genome">
<!--   -->
</a>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing classes, and an explanation">
<caption><span>Classes in <a href="../../../../org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a> used by <a href="../../../../org/biojava3/genome/package-summary.html">org.biojava3.genome</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colOne" scope="col">Class and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/ChromosomeSequence.html#org.biojava3.genome">ChromosomeSequence</a>
<div class="block">A ChromosomeSequence is a DNASequence but keeps track of geneSequences</div>
</td>
</tr>
<tr class="rowColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/DNASequence.html#org.biojava3.genome">DNASequence</a>
<div class="block">This is class should model the attributes associated with a DNA sequence</div>
</td>
</tr>
<tr class="altColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/GeneSequence.html#org.biojava3.genome">GeneSequence</a>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/ProteinSequence.html#org.biojava3.genome">ProteinSequence</a>
<div class="block">The representation of a ProteinSequence</div>
</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava3.genome.homology">
<!--   -->
</a>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing classes, and an explanation">
<caption><span>Classes in <a href="../../../../org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a> used by <a href="../../../../org/biojava3/genome/homology/package-summary.html">org.biojava3.genome.homology</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colOne" scope="col">Class and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/GeneSequence.html#org.biojava3.genome.homology">GeneSequence</a>&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava3.genome.parsers.geneid">
<!--   -->
</a>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing classes, and an explanation">
<caption><span>Classes in <a href="../../../../org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a> used by <a href="../../../../org/biojava3/genome/parsers/geneid/package-summary.html">org.biojava3.genome.parsers.geneid</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colOne" scope="col">Class and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/DNASequence.html#org.biojava3.genome.parsers.geneid">DNASequence</a>
<div class="block">This is class should model the attributes associated with a DNA sequence</div>
</td>
</tr>
<tr class="rowColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/ProteinSequence.html#org.biojava3.genome.parsers.geneid">ProteinSequence</a>
<div class="block">The representation of a ProteinSequence</div>
</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava3.genome.parsers.gff">
<!--   -->
</a>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing classes, and an explanation">
<caption><span>Classes in <a href="../../../../org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a> used by <a href="../../../../org/biojava3/genome/parsers/gff/package-summary.html">org.biojava3.genome.parsers.gff</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colOne" scope="col">Class and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/ChromosomeSequence.html#org.biojava3.genome.parsers.gff">ChromosomeSequence</a>
<div class="block">A ChromosomeSequence is a DNASequence but keeps track of geneSequences</div>
</td>
</tr>
<tr class="rowColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/DNASequence.html#org.biojava3.genome.parsers.gff">DNASequence</a>
<div class="block">This is class should model the attributes associated with a DNA sequence</div>
</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.biojava3.phylo">
<!--   -->
</a>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing classes, and an explanation">
<caption><span>Classes in <a href="../../../../org/biojava3/core/sequence/package-summary.html">org.biojava3.core.sequence</a> used by <a href="../../../../org/biojava3/phylo/package-summary.html">org.biojava3.phylo</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colOne" scope="col">Class and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colOne"><a href="../../../../org/biojava3/core/sequence/class-use/MultipleSequenceAlignment.html#org.biojava3.phylo">MultipleSequenceAlignment</a>
<div class="block">Implements a minimal data structure for reading and writing a sequence alignment.</div>
</td>
</tr>
</tbody>
</table>
</li>
</ul>
</div>
<!-- ======= START OF BOTTOM NAVBAR ====== -->
<div class="bottomNav"><a name="navbar_bottom">
<!--   -->
</a><a href="#skip-navbar_bottom" title="Skip navigation links"></a><a name="navbar_bottom_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../../../../overview-summary.html">Overview</a></li>
<li><a href="package-summary.html">Package</a></li>
<li>Class</li>
<li class="navBarCell1Rev">Use</li>
<li><a href="package-tree.html">Tree</a></li>
<li><a href="../../../../deprecated-list.html">Deprecated</a></li>
<li><a href="../../../../index-all.html">Index</a></li>
<li><a href="../../../../help-doc.html">Help</a></li>
</ul>
<div class="aboutLanguage"><em><script src="http://www.google-analytics.com/urchin.js" type="text/javascript"></script><script type="text/javascript">_uacct = "UA-1326640-1";urchinTracker();</script></em></div>
</div>
<div class="subNav">
<ul class="navList">
<li>Prev</li>
<li>Next</li>
</ul>
<ul class="navList">
<li><a href="../../../../index.html?org/biojava3/core/sequence/package-use.html" target="_top">Frames</a></li>
<li><a href="package-use.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_bottom">
<li><a href="../../../../allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_bottom");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<a name="skip-navbar_bottom">
<!--   -->
</a></div>
<!-- ======== END OF BOTTOM NAVBAR ======= -->
<p class="legalCopy"><small>Copyright &#169; 2011 <a href="http://www.biojava.org">BioJava</a>. All Rights Reserved.</small></p>
</body>
</html>
