<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<html lang="en">
<head>
<title>Source code</title>
<link rel="stylesheet" type="text/css" href="../../../../../stylesheet.css" title="Style">
</head>
<body>
<div class="sourceContainer">
<pre><span class="sourceLineNo">001</span>/*<a name="line.1"></a>
<span class="sourceLineNo">002</span> *                    BioJava development code<a name="line.2"></a>
<span class="sourceLineNo">003</span> *<a name="line.3"></a>
<span class="sourceLineNo">004</span> * This code may be freely distributed and modified under the<a name="line.4"></a>
<span class="sourceLineNo">005</span> * terms of the GNU Lesser General Public Licence.  This should<a name="line.5"></a>
<span class="sourceLineNo">006</span> * be distributed with the code.  If you do not have a copy,<a name="line.6"></a>
<span class="sourceLineNo">007</span> * see:<a name="line.7"></a>
<span class="sourceLineNo">008</span> *<a name="line.8"></a>
<span class="sourceLineNo">009</span> *      http://www.gnu.org/copyleft/lesser.html<a name="line.9"></a>
<span class="sourceLineNo">010</span> *<a name="line.10"></a>
<span class="sourceLineNo">011</span> * Copyright for this code is held jointly by the individual<a name="line.11"></a>
<span class="sourceLineNo">012</span> * authors.  These should be listed in @author doc comments.<a name="line.12"></a>
<span class="sourceLineNo">013</span> *<a name="line.13"></a>
<span class="sourceLineNo">014</span> * For more information on the BioJava project and its aims,<a name="line.14"></a>
<span class="sourceLineNo">015</span> * or to join the biojava-l mailing list, visit the home page<a name="line.15"></a>
<span class="sourceLineNo">016</span> * at:<a name="line.16"></a>
<span class="sourceLineNo">017</span> *<a name="line.17"></a>
<span class="sourceLineNo">018</span> *      http://www.biojava.org/<a name="line.18"></a>
<span class="sourceLineNo">019</span> *<a name="line.19"></a>
<span class="sourceLineNo">020</span> * Created on DATE<a name="line.20"></a>
<span class="sourceLineNo">021</span> *<a name="line.21"></a>
<span class="sourceLineNo">022</span> */<a name="line.22"></a>
<span class="sourceLineNo">023</span>package org.biojava3.core.sequence;<a name="line.23"></a>
<span class="sourceLineNo">024</span><a name="line.24"></a>
<span class="sourceLineNo">025</span>import java.util.ArrayList;<a name="line.25"></a>
<span class="sourceLineNo">026</span>import java.util.Collections;<a name="line.26"></a>
<span class="sourceLineNo">027</span>import java.util.LinkedHashMap;<a name="line.27"></a>
<span class="sourceLineNo">028</span>import java.util.logging.Logger;<a name="line.28"></a>
<span class="sourceLineNo">029</span><a name="line.29"></a>
<span class="sourceLineNo">030</span>import org.biojava3.core.sequence.transcription.TranscriptionEngine;<a name="line.30"></a>
<span class="sourceLineNo">031</span><a name="line.31"></a>
<span class="sourceLineNo">032</span>/**<a name="line.32"></a>
<span class="sourceLineNo">033</span> * This is the sequence if you want to go from a gene sequence to a protein sequence. Need to start with a<a name="line.33"></a>
<span class="sourceLineNo">034</span> * ChromosomeSequence then getting a GeneSequence and then a TranscriptSequence<a name="line.34"></a>
<span class="sourceLineNo">035</span> * @author Scooter Willis<a name="line.35"></a>
<span class="sourceLineNo">036</span> */<a name="line.36"></a>
<span class="sourceLineNo">037</span>public class TranscriptSequence extends DNASequence {<a name="line.37"></a>
<span class="sourceLineNo">038</span><a name="line.38"></a>
<span class="sourceLineNo">039</span>    private static final Logger log = Logger.getLogger(TranscriptSequence.class.getName());<a name="line.39"></a>
<span class="sourceLineNo">040</span>    private final ArrayList&lt;CDSSequence&gt; cdsSequenceList = new ArrayList&lt;CDSSequence&gt;();<a name="line.40"></a>
<span class="sourceLineNo">041</span>    private final LinkedHashMap&lt;String, CDSSequence&gt; cdsSequenceHashMap = new LinkedHashMap&lt;String, CDSSequence&gt;();<a name="line.41"></a>
<span class="sourceLineNo">042</span>    private StartCodonSequence startCodonSequence = null;<a name="line.42"></a>
<span class="sourceLineNo">043</span>    private StopCodonSequence stopCodonSequence = null;<a name="line.43"></a>
<span class="sourceLineNo">044</span>    private GeneSequence parentGeneSequence = null;<a name="line.44"></a>
<span class="sourceLineNo">045</span><a name="line.45"></a>
<span class="sourceLineNo">046</span>    /**<a name="line.46"></a>
<span class="sourceLineNo">047</span>     *<a name="line.47"></a>
<span class="sourceLineNo">048</span>     * @param parentDNASequence<a name="line.48"></a>
<span class="sourceLineNo">049</span>     * @param begin<a name="line.49"></a>
<span class="sourceLineNo">050</span>     * @param end inclusive of end<a name="line.50"></a>
<span class="sourceLineNo">051</span>     */<a name="line.51"></a>
<span class="sourceLineNo">052</span>    public TranscriptSequence(GeneSequence parentDNASequence, int begin, int end) {<a name="line.52"></a>
<span class="sourceLineNo">053</span>        setParentSequence(parentDNASequence);<a name="line.53"></a>
<span class="sourceLineNo">054</span>        this.parentGeneSequence = parentDNASequence;<a name="line.54"></a>
<span class="sourceLineNo">055</span>        setBioBegin(begin);<a name="line.55"></a>
<span class="sourceLineNo">056</span>        setBioEnd(end);<a name="line.56"></a>
<span class="sourceLineNo">057</span><a name="line.57"></a>
<span class="sourceLineNo">058</span>    }<a name="line.58"></a>
<span class="sourceLineNo">059</span><a name="line.59"></a>
<span class="sourceLineNo">060</span>        @Override<a name="line.60"></a>
<span class="sourceLineNo">061</span>    public int getLength() {<a name="line.61"></a>
<span class="sourceLineNo">062</span>        return Math.abs(this.getBioEnd() - this.getBioBegin()) + 1;<a name="line.62"></a>
<span class="sourceLineNo">063</span>    }<a name="line.63"></a>
<span class="sourceLineNo">064</span><a name="line.64"></a>
<span class="sourceLineNo">065</span>    /**<a name="line.65"></a>
<span class="sourceLineNo">066</span>     * @return the strand<a name="line.66"></a>
<span class="sourceLineNo">067</span>     */<a name="line.67"></a>
<span class="sourceLineNo">068</span>    public Strand getStrand() {<a name="line.68"></a>
<span class="sourceLineNo">069</span>        return parentGeneSequence.getStrand();<a name="line.69"></a>
<span class="sourceLineNo">070</span>    }<a name="line.70"></a>
<span class="sourceLineNo">071</span><a name="line.71"></a>
<span class="sourceLineNo">072</span>    /**<a name="line.72"></a>
<span class="sourceLineNo">073</span>     * Remove a CDS or coding sequence from the transcript sequence<a name="line.73"></a>
<span class="sourceLineNo">074</span>     * @param accession<a name="line.74"></a>
<span class="sourceLineNo">075</span>     * @return<a name="line.75"></a>
<span class="sourceLineNo">076</span>     */<a name="line.76"></a>
<span class="sourceLineNo">077</span>    public CDSSequence removeCDS(String accession) {<a name="line.77"></a>
<span class="sourceLineNo">078</span>        for (CDSSequence cdsSequence : cdsSequenceList) {<a name="line.78"></a>
<span class="sourceLineNo">079</span>            if (cdsSequence.getAccession().getID().equals(accession)) {<a name="line.79"></a>
<span class="sourceLineNo">080</span>                cdsSequenceList.remove(cdsSequence);<a name="line.80"></a>
<span class="sourceLineNo">081</span>                cdsSequenceHashMap.remove(accession);<a name="line.81"></a>
<span class="sourceLineNo">082</span>                return cdsSequence;<a name="line.82"></a>
<span class="sourceLineNo">083</span>            }<a name="line.83"></a>
<span class="sourceLineNo">084</span>        }<a name="line.84"></a>
<span class="sourceLineNo">085</span>        return null;<a name="line.85"></a>
<span class="sourceLineNo">086</span>    }<a name="line.86"></a>
<span class="sourceLineNo">087</span><a name="line.87"></a>
<span class="sourceLineNo">088</span>    /**<a name="line.88"></a>
<span class="sourceLineNo">089</span>     * Get the CDS sequences that have been added to the TranscriptSequences<a name="line.89"></a>
<span class="sourceLineNo">090</span>     * @return<a name="line.90"></a>
<span class="sourceLineNo">091</span>     */<a name="line.91"></a>
<span class="sourceLineNo">092</span>    public LinkedHashMap&lt;String, CDSSequence&gt; getCDSSequences() {<a name="line.92"></a>
<span class="sourceLineNo">093</span>        return cdsSequenceHashMap;<a name="line.93"></a>
<span class="sourceLineNo">094</span>    }<a name="line.94"></a>
<span class="sourceLineNo">095</span><a name="line.95"></a>
<span class="sourceLineNo">096</span>    /**<a name="line.96"></a>
<span class="sourceLineNo">097</span>     * Add a Coding Sequence region with phase to the transcript sequence<a name="line.97"></a>
<span class="sourceLineNo">098</span>     * @param accession<a name="line.98"></a>
<span class="sourceLineNo">099</span>     * @param begin<a name="line.99"></a>
<span class="sourceLineNo">100</span>     * @param end<a name="line.100"></a>
<span class="sourceLineNo">101</span>     * @param phase 0,1,2<a name="line.101"></a>
<span class="sourceLineNo">102</span>     * @return<a name="line.102"></a>
<span class="sourceLineNo">103</span>     */<a name="line.103"></a>
<span class="sourceLineNo">104</span>    public CDSSequence addCDS(AccessionID accession, int begin, int end, int phase) throws Exception {<a name="line.104"></a>
<span class="sourceLineNo">105</span>        if (cdsSequenceHashMap.containsKey(accession.getID())) {<a name="line.105"></a>
<span class="sourceLineNo">106</span>            throw new Exception("Duplicate accesion id " + accession.getID());<a name="line.106"></a>
<span class="sourceLineNo">107</span>        }<a name="line.107"></a>
<span class="sourceLineNo">108</span>        CDSSequence cdsSequence = new CDSSequence(this, begin, end, phase); //sense should be the same as parent<a name="line.108"></a>
<span class="sourceLineNo">109</span>        cdsSequence.setAccession(accession);<a name="line.109"></a>
<span class="sourceLineNo">110</span>        cdsSequenceList.add(cdsSequence);<a name="line.110"></a>
<span class="sourceLineNo">111</span>        Collections.sort(cdsSequenceList, new CDSComparator());<a name="line.111"></a>
<span class="sourceLineNo">112</span>        cdsSequenceHashMap.put(accession.getID(), cdsSequence);<a name="line.112"></a>
<span class="sourceLineNo">113</span>        return cdsSequence;<a name="line.113"></a>
<span class="sourceLineNo">114</span>    }<a name="line.114"></a>
<span class="sourceLineNo">115</span><a name="line.115"></a>
<span class="sourceLineNo">116</span>    /**<a name="line.116"></a>
<span class="sourceLineNo">117</span>     * http://www.sequenceontology.org/gff3.shtml<a name="line.117"></a>
<span class="sourceLineNo">118</span>     * http://biowiki.org/~yam/bioe131/GFF.ppt<a name="line.118"></a>
<span class="sourceLineNo">119</span>     * @return<a name="line.119"></a>
<span class="sourceLineNo">120</span>     */<a name="line.120"></a>
<span class="sourceLineNo">121</span>    /**<a name="line.121"></a>
<span class="sourceLineNo">122</span>     * Return a list of protein sequences based on each CDS sequence<a name="line.122"></a>
<span class="sourceLineNo">123</span>     * where the phase shift between two CDS sequences is assigned to the<a name="line.123"></a>
<span class="sourceLineNo">124</span>     * CDS sequence that starts the triplet. This can be used to map<a name="line.124"></a>
<span class="sourceLineNo">125</span>     * a CDS/exon region of a protein sequence back to the DNA sequence<a name="line.125"></a>
<span class="sourceLineNo">126</span>     * If you have a protein sequence and a predicted gene you can take the<a name="line.126"></a>
<span class="sourceLineNo">127</span>     * predict CDS protein sequences and align back to the protein sequence.<a name="line.127"></a>
<span class="sourceLineNo">128</span>     * If you have errors in mapping the predicted protein CDS regions to<a name="line.128"></a>
<span class="sourceLineNo">129</span>     * an the known protein sequence then you can identify possible errors<a name="line.129"></a>
<span class="sourceLineNo">130</span>     * in the prediction<a name="line.130"></a>
<span class="sourceLineNo">131</span>     *<a name="line.131"></a>
<span class="sourceLineNo">132</span>     * @return<a name="line.132"></a>
<span class="sourceLineNo">133</span>     */<a name="line.133"></a>
<span class="sourceLineNo">134</span>    public ArrayList&lt;ProteinSequence&gt; getProteinCDSSequences() {<a name="line.134"></a>
<span class="sourceLineNo">135</span>        ArrayList&lt;ProteinSequence&gt; proteinSequenceList = new ArrayList&lt;ProteinSequence&gt;();<a name="line.135"></a>
<span class="sourceLineNo">136</span>        for (int i = 0; i &lt; cdsSequenceList.size(); i++) {<a name="line.136"></a>
<span class="sourceLineNo">137</span>            CDSSequence cdsSequence = cdsSequenceList.get(i);<a name="line.137"></a>
<span class="sourceLineNo">138</span>            String codingSequence = cdsSequence.getCodingSequence();<a name="line.138"></a>
<span class="sourceLineNo">139</span>            //          System.out.println("CDS " + getStrand() + " "  + cdsSequence.getPhase() + "=" + codingSequence);<a name="line.139"></a>
<span class="sourceLineNo">140</span>            if (this.getStrand() == Strand.NEGATIVE) {<a name="line.140"></a>
<span class="sourceLineNo">141</span>                if (cdsSequence.phase == 1) {<a name="line.141"></a>
<span class="sourceLineNo">142</span>                    codingSequence = codingSequence.substring(1, codingSequence.length());<a name="line.142"></a>
<span class="sourceLineNo">143</span>                } else if (cdsSequence.phase == 2) {<a name="line.143"></a>
<span class="sourceLineNo">144</span>                    codingSequence = codingSequence.substring(2, codingSequence.length());<a name="line.144"></a>
<span class="sourceLineNo">145</span>                }<a name="line.145"></a>
<span class="sourceLineNo">146</span>                if (i &lt; cdsSequenceList.size() - 1) {<a name="line.146"></a>
<span class="sourceLineNo">147</span>                    CDSSequence nextCDSSequence = cdsSequenceList.get(i + 1);<a name="line.147"></a>
<span class="sourceLineNo">148</span>                    if (nextCDSSequence.phase == 1) {<a name="line.148"></a>
<span class="sourceLineNo">149</span>                        String nextCodingSequence = nextCDSSequence.getCodingSequence();<a name="line.149"></a>
<span class="sourceLineNo">150</span>                        codingSequence = codingSequence + nextCodingSequence.substring(0, 1);<a name="line.150"></a>
<span class="sourceLineNo">151</span>                    } else if (nextCDSSequence.phase == 2) {<a name="line.151"></a>
<span class="sourceLineNo">152</span>                        String nextCodingSequence = nextCDSSequence.getCodingSequence();<a name="line.152"></a>
<span class="sourceLineNo">153</span>                        codingSequence = codingSequence + nextCodingSequence.substring(0, 2);<a name="line.153"></a>
<span class="sourceLineNo">154</span>                    }<a name="line.154"></a>
<span class="sourceLineNo">155</span>                }<a name="line.155"></a>
<span class="sourceLineNo">156</span>            } else {<a name="line.156"></a>
<span class="sourceLineNo">157</span>                if (cdsSequence.phase == 1) {<a name="line.157"></a>
<span class="sourceLineNo">158</span>                    codingSequence = codingSequence.substring(1, codingSequence.length());<a name="line.158"></a>
<span class="sourceLineNo">159</span>                } else if (cdsSequence.phase == 2) {<a name="line.159"></a>
<span class="sourceLineNo">160</span>                    codingSequence = codingSequence.substring(2, codingSequence.length());<a name="line.160"></a>
<span class="sourceLineNo">161</span>                }<a name="line.161"></a>
<span class="sourceLineNo">162</span>                if (i &lt; cdsSequenceList.size() - 1) {<a name="line.162"></a>
<span class="sourceLineNo">163</span>                    CDSSequence nextCDSSequence = cdsSequenceList.get(i + 1);<a name="line.163"></a>
<span class="sourceLineNo">164</span>                    if (nextCDSSequence.phase == 1) {<a name="line.164"></a>
<span class="sourceLineNo">165</span>                        String nextCodingSequence = nextCDSSequence.getCodingSequence();<a name="line.165"></a>
<span class="sourceLineNo">166</span>                        codingSequence = codingSequence + nextCodingSequence.substring(0, 1);<a name="line.166"></a>
<span class="sourceLineNo">167</span>                    } else if (nextCDSSequence.phase == 2) {<a name="line.167"></a>
<span class="sourceLineNo">168</span>                        String nextCodingSequence = nextCDSSequence.getCodingSequence();<a name="line.168"></a>
<span class="sourceLineNo">169</span>                        codingSequence = codingSequence + nextCodingSequence.substring(0, 2);<a name="line.169"></a>
<span class="sourceLineNo">170</span>                    }<a name="line.170"></a>
<span class="sourceLineNo">171</span>                }<a name="line.171"></a>
<span class="sourceLineNo">172</span>            }<a name="line.172"></a>
<span class="sourceLineNo">173</span><a name="line.173"></a>
<span class="sourceLineNo">174</span><a name="line.174"></a>
<span class="sourceLineNo">175</span>            //    System.out.println(codingSequence);<a name="line.175"></a>
<span class="sourceLineNo">176</span>            DNASequence dnaCodingSequence = new DNASequence(codingSequence.toString().toUpperCase());<a name="line.176"></a>
<span class="sourceLineNo">177</span>            RNASequence rnaCodingSequence = dnaCodingSequence.getRNASequence(TranscriptionEngine.getDefault());<a name="line.177"></a>
<span class="sourceLineNo">178</span>            ProteinSequence proteinSequence = rnaCodingSequence.getProteinSequence(TranscriptionEngine.getDefault());<a name="line.178"></a>
<span class="sourceLineNo">179</span>            proteinSequence.setAccession(new AccessionID(cdsSequence.getAccession().getID()));<a name="line.179"></a>
<span class="sourceLineNo">180</span>            proteinSequence.setParentDNASequence(cdsSequence, 1, cdsSequence.getLength());<a name="line.180"></a>
<span class="sourceLineNo">181</span>            proteinSequenceList.add(proteinSequence);<a name="line.181"></a>
<span class="sourceLineNo">182</span>        }<a name="line.182"></a>
<span class="sourceLineNo">183</span>        return proteinSequenceList;<a name="line.183"></a>
<span class="sourceLineNo">184</span>    }<a name="line.184"></a>
<span class="sourceLineNo">185</span><a name="line.185"></a>
<span class="sourceLineNo">186</span>    /**<a name="line.186"></a>
<span class="sourceLineNo">187</span>     * Get the stitched together CDS sequences then maps to the cDNA<a name="line.187"></a>
<span class="sourceLineNo">188</span>     * @return<a name="line.188"></a>
<span class="sourceLineNo">189</span>     */<a name="line.189"></a>
<span class="sourceLineNo">190</span>    public DNASequence getDNACodingSequence() {<a name="line.190"></a>
<span class="sourceLineNo">191</span>        StringBuilder sb = new StringBuilder();<a name="line.191"></a>
<span class="sourceLineNo">192</span>        for (CDSSequence cdsSequence : cdsSequenceList) {<a name="line.192"></a>
<span class="sourceLineNo">193</span>            sb.append(cdsSequence.getCodingSequence());<a name="line.193"></a>
<span class="sourceLineNo">194</span>        }<a name="line.194"></a>
<span class="sourceLineNo">195</span>        DNASequence dnaSequence = new DNASequence(sb.toString().toUpperCase());<a name="line.195"></a>
<span class="sourceLineNo">196</span>        dnaSequence.setAccession(new AccessionID(this.getAccession().getID()));<a name="line.196"></a>
<span class="sourceLineNo">197</span>        return dnaSequence;<a name="line.197"></a>
<span class="sourceLineNo">198</span>    }<a name="line.198"></a>
<span class="sourceLineNo">199</span><a name="line.199"></a>
<span class="sourceLineNo">200</span>    /**<a name="line.200"></a>
<span class="sourceLineNo">201</span>     * Get the protein sequence<a name="line.201"></a>
<span class="sourceLineNo">202</span>     * @return<a name="line.202"></a>
<span class="sourceLineNo">203</span>     */<a name="line.203"></a>
<span class="sourceLineNo">204</span>    public ProteinSequence getProteinSequence() {<a name="line.204"></a>
<span class="sourceLineNo">205</span>        return getProteinSequence(TranscriptionEngine.getDefault());<a name="line.205"></a>
<span class="sourceLineNo">206</span>    }<a name="line.206"></a>
<span class="sourceLineNo">207</span><a name="line.207"></a>
<span class="sourceLineNo">208</span>    /**<a name="line.208"></a>
<span class="sourceLineNo">209</span>     * Get the protein sequence with user defined TranscriptEngine<a name="line.209"></a>
<span class="sourceLineNo">210</span>     * @param engine<a name="line.210"></a>
<span class="sourceLineNo">211</span>     * @return<a name="line.211"></a>
<span class="sourceLineNo">212</span>     */<a name="line.212"></a>
<span class="sourceLineNo">213</span>    public ProteinSequence getProteinSequence(TranscriptionEngine engine) {<a name="line.213"></a>
<span class="sourceLineNo">214</span>        DNASequence dnaCodingSequence = getDNACodingSequence();<a name="line.214"></a>
<span class="sourceLineNo">215</span>        RNASequence rnaCodingSequence = dnaCodingSequence.getRNASequence(engine);<a name="line.215"></a>
<span class="sourceLineNo">216</span>        ProteinSequence proteinSequence = rnaCodingSequence.getProteinSequence(engine);<a name="line.216"></a>
<span class="sourceLineNo">217</span>        proteinSequence.setAccession(new AccessionID(this.getAccession().getID()));<a name="line.217"></a>
<span class="sourceLineNo">218</span>        return proteinSequence;<a name="line.218"></a>
<span class="sourceLineNo">219</span>    }<a name="line.219"></a>
<span class="sourceLineNo">220</span><a name="line.220"></a>
<span class="sourceLineNo">221</span>    /**<a name="line.221"></a>
<span class="sourceLineNo">222</span>     * @return the startCodonSequence<a name="line.222"></a>
<span class="sourceLineNo">223</span>     */<a name="line.223"></a>
<span class="sourceLineNo">224</span>    public StartCodonSequence getStartCodonSequence() {<a name="line.224"></a>
<span class="sourceLineNo">225</span>        return startCodonSequence;<a name="line.225"></a>
<span class="sourceLineNo">226</span>    }<a name="line.226"></a>
<span class="sourceLineNo">227</span><a name="line.227"></a>
<span class="sourceLineNo">228</span>    /**<a name="line.228"></a>
<span class="sourceLineNo">229</span>     * @param startCodonSequence the startCodonSequence to set<a name="line.229"></a>
<span class="sourceLineNo">230</span>     */<a name="line.230"></a>
<span class="sourceLineNo">231</span>    public void addStartCodonSequence(AccessionID accession, int begin, int end) {<a name="line.231"></a>
<span class="sourceLineNo">232</span>        this.startCodonSequence = new StartCodonSequence(this, begin, end);<a name="line.232"></a>
<span class="sourceLineNo">233</span>        startCodonSequence.setAccession(accession);<a name="line.233"></a>
<span class="sourceLineNo">234</span>    }<a name="line.234"></a>
<span class="sourceLineNo">235</span><a name="line.235"></a>
<span class="sourceLineNo">236</span>    /**<a name="line.236"></a>
<span class="sourceLineNo">237</span>     * @return the stopCodonSequence<a name="line.237"></a>
<span class="sourceLineNo">238</span>     */<a name="line.238"></a>
<span class="sourceLineNo">239</span>    public StopCodonSequence getStopCodonSequence() {<a name="line.239"></a>
<span class="sourceLineNo">240</span>        return stopCodonSequence;<a name="line.240"></a>
<span class="sourceLineNo">241</span>    }<a name="line.241"></a>
<span class="sourceLineNo">242</span><a name="line.242"></a>
<span class="sourceLineNo">243</span>    /**<a name="line.243"></a>
<span class="sourceLineNo">244</span>     * @param stopCodonSequence the stopCodonSequence to set<a name="line.244"></a>
<span class="sourceLineNo">245</span>     */<a name="line.245"></a>
<span class="sourceLineNo">246</span>    public void addStopCodonSequence(AccessionID accession, int begin, int end) {<a name="line.246"></a>
<span class="sourceLineNo">247</span>        this.stopCodonSequence = new StopCodonSequence(this, begin, end);<a name="line.247"></a>
<span class="sourceLineNo">248</span>        stopCodonSequence.setAccession(accession);<a name="line.248"></a>
<span class="sourceLineNo">249</span>    }<a name="line.249"></a>
<span class="sourceLineNo">250</span>}<a name="line.250"></a>




























































</pre>
</div>
</body>
</html>
