<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<html lang="en">
<head>
<title>Source code</title>
<link rel="stylesheet" type="text/css" href="../../../../../../stylesheet.css" title="Style">
</head>
<body>
<div class="sourceContainer">
<pre><span class="sourceLineNo">001</span>/*<a name="line.1"></a>
<span class="sourceLineNo">002</span> *                  BioJava development code<a name="line.2"></a>
<span class="sourceLineNo">003</span> *<a name="line.3"></a>
<span class="sourceLineNo">004</span> * This code may be freely distributed and modified under the<a name="line.4"></a>
<span class="sourceLineNo">005</span> * terms of the GNU Lesser General Public Licence.  This should<a name="line.5"></a>
<span class="sourceLineNo">006</span> * be distributed with the code.  If you do not have a copy,<a name="line.6"></a>
<span class="sourceLineNo">007</span> * see:<a name="line.7"></a>
<span class="sourceLineNo">008</span> *<a name="line.8"></a>
<span class="sourceLineNo">009</span> *      http://www.gnu.org/copyleft/lesser.html<a name="line.9"></a>
<span class="sourceLineNo">010</span> *<a name="line.10"></a>
<span class="sourceLineNo">011</span> * Copyright for this code is held jointly by the individual<a name="line.11"></a>
<span class="sourceLineNo">012</span> * authors.  These should be listed in @author doc comments.<a name="line.12"></a>
<span class="sourceLineNo">013</span> *<a name="line.13"></a>
<span class="sourceLineNo">014</span> * For more information on the BioJava project and its aims,<a name="line.14"></a>
<span class="sourceLineNo">015</span> * or to join the biojava-l mailing list, visit the home page<a name="line.15"></a>
<span class="sourceLineNo">016</span> * at:<a name="line.16"></a>
<span class="sourceLineNo">017</span> *<a name="line.17"></a>
<span class="sourceLineNo">018</span> *      http://www.biojava.org/<a name="line.18"></a>
<span class="sourceLineNo">019</span> *<a name="line.19"></a>
<span class="sourceLineNo">020</span> * Created on Aug 23, 2007<a name="line.20"></a>
<span class="sourceLineNo">021</span> *<a name="line.21"></a>
<span class="sourceLineNo">022</span> */<a name="line.22"></a>
<span class="sourceLineNo">023</span><a name="line.23"></a>
<span class="sourceLineNo">024</span>package org.biojava.bio.structure.io;<a name="line.24"></a>
<span class="sourceLineNo">025</span><a name="line.25"></a>
<span class="sourceLineNo">026</span><a name="line.26"></a>
<span class="sourceLineNo">027</span>import java.util.ArrayList;<a name="line.27"></a>
<span class="sourceLineNo">028</span>import java.util.HashMap;<a name="line.28"></a>
<span class="sourceLineNo">029</span>import java.util.Iterator;<a name="line.29"></a>
<span class="sourceLineNo">030</span>import java.util.List;<a name="line.30"></a>
<span class="sourceLineNo">031</span>import java.util.Map;<a name="line.31"></a>
<span class="sourceLineNo">032</span><a name="line.32"></a>
<span class="sourceLineNo">033</span><a name="line.33"></a>
<span class="sourceLineNo">034</span>import org.biojava.bio.structure.AminoAcid;<a name="line.34"></a>
<span class="sourceLineNo">035</span>import org.biojava.bio.structure.Atom;<a name="line.35"></a>
<span class="sourceLineNo">036</span>import org.biojava.bio.structure.Chain;<a name="line.36"></a>
<span class="sourceLineNo">037</span>import org.biojava.bio.structure.Group;<a name="line.37"></a>
<span class="sourceLineNo">038</span>import org.biojava.bio.structure.GroupType;<a name="line.38"></a>
<span class="sourceLineNo">039</span>import org.biojava.bio.structure.HetatomImpl;<a name="line.39"></a>
<span class="sourceLineNo">040</span>import org.biojava.bio.structure.NucleotideImpl;<a name="line.40"></a>
<span class="sourceLineNo">041</span>import org.biojava.bio.structure.ResidueNumber;<a name="line.41"></a>
<span class="sourceLineNo">042</span>import org.biojava.bio.structure.Structure;<a name="line.42"></a>
<span class="sourceLineNo">043</span>import org.biojava.bio.structure.StructureException;<a name="line.43"></a>
<span class="sourceLineNo">044</span>import org.biojava.bio.structure.io.mmcif.chem.PolymerType;<a name="line.44"></a>
<span class="sourceLineNo">045</span>import org.biojava.bio.structure.io.mmcif.chem.ResidueType;<a name="line.45"></a>
<span class="sourceLineNo">046</span>import org.biojava.bio.structure.io.mmcif.model.ChemComp;<a name="line.46"></a>
<span class="sourceLineNo">047</span>import org.biojava3.alignment.Alignments;<a name="line.47"></a>
<span class="sourceLineNo">048</span>import org.biojava3.alignment.SimpleGapPenalty;<a name="line.48"></a>
<span class="sourceLineNo">049</span>import org.biojava3.alignment.SubstitutionMatrixHelper;<a name="line.49"></a>
<span class="sourceLineNo">050</span>import org.biojava3.alignment.Alignments.PairwiseSequenceAlignerType;<a name="line.50"></a>
<span class="sourceLineNo">051</span>import org.biojava3.alignment.template.GapPenalty;<a name="line.51"></a>
<span class="sourceLineNo">052</span>import org.biojava3.alignment.template.PairwiseSequenceAligner;<a name="line.52"></a>
<span class="sourceLineNo">053</span>import org.biojava3.alignment.template.SequencePair;<a name="line.53"></a>
<span class="sourceLineNo">054</span>import org.biojava3.alignment.template.SubstitutionMatrix;<a name="line.54"></a>
<span class="sourceLineNo">055</span>import org.biojava3.core.exceptions.CompoundNotFoundError;<a name="line.55"></a>
<span class="sourceLineNo">056</span>import org.biojava3.core.sequence.DNASequence;<a name="line.56"></a>
<span class="sourceLineNo">057</span>import org.biojava3.core.sequence.ProteinSequence;<a name="line.57"></a>
<span class="sourceLineNo">058</span>import org.biojava3.core.sequence.RNASequence;<a name="line.58"></a>
<span class="sourceLineNo">059</span>import org.biojava3.core.sequence.compound.AmbiguityDNACompoundSet;<a name="line.59"></a>
<span class="sourceLineNo">060</span>import org.biojava3.core.sequence.compound.AmbiguityRNACompoundSet;<a name="line.60"></a>
<span class="sourceLineNo">061</span>import org.biojava3.core.sequence.compound.AminoAcidCompound;<a name="line.61"></a>
<span class="sourceLineNo">062</span>import org.biojava3.core.sequence.compound.AminoAcidCompoundSet;<a name="line.62"></a>
<span class="sourceLineNo">063</span>import org.biojava3.core.sequence.compound.DNACompoundSet;<a name="line.63"></a>
<span class="sourceLineNo">064</span>import org.biojava3.core.sequence.compound.NucleotideCompound;<a name="line.64"></a>
<span class="sourceLineNo">065</span>import org.biojava3.core.sequence.template.Compound;<a name="line.65"></a>
<span class="sourceLineNo">066</span>import org.biojava3.core.sequence.template.Sequence;<a name="line.66"></a>
<span class="sourceLineNo">067</span><a name="line.67"></a>
<span class="sourceLineNo">068</span><a name="line.68"></a>
<span class="sourceLineNo">069</span>/** Aligns the SEQRES residues to the ATOM residues.<a name="line.69"></a>
<span class="sourceLineNo">070</span> * The AminoAcids that can be matched between the two of them will be set in the SEQRES<a name="line.70"></a>
<span class="sourceLineNo">071</span> * chains<a name="line.71"></a>
<span class="sourceLineNo">072</span> *<a name="line.72"></a>
<span class="sourceLineNo">073</span> *<a name="line.73"></a>
<span class="sourceLineNo">074</span> * @author Andreas Prlic<a name="line.74"></a>
<span class="sourceLineNo">075</span> *<a name="line.75"></a>
<span class="sourceLineNo">076</span> */<a name="line.76"></a>
<span class="sourceLineNo">077</span>public class SeqRes2AtomAligner {<a name="line.77"></a>
<span class="sourceLineNo">078</span><a name="line.78"></a>
<span class="sourceLineNo">079</span>        boolean DEBUG = false;<a name="line.79"></a>
<span class="sourceLineNo">080</span><a name="line.80"></a>
<span class="sourceLineNo">081</span>        static final List&lt;String&gt; excludeTypes;<a name="line.81"></a>
<span class="sourceLineNo">082</span><a name="line.82"></a>
<span class="sourceLineNo">083</span><a name="line.83"></a>
<span class="sourceLineNo">084</span>        String alignmentString;<a name="line.84"></a>
<span class="sourceLineNo">085</span>        static {<a name="line.85"></a>
<span class="sourceLineNo">086</span>                excludeTypes = new ArrayList&lt;String&gt;();<a name="line.86"></a>
<span class="sourceLineNo">087</span>                excludeTypes.add("HOH"); // we don't want to align water against the SEQRES...<a name="line.87"></a>
<span class="sourceLineNo">088</span>                excludeTypes.add("DOD"); // deuterated water<a name="line.88"></a>
<span class="sourceLineNo">089</span><a name="line.89"></a>
<span class="sourceLineNo">090</span><a name="line.90"></a>
<span class="sourceLineNo">091</span>                //matrix.printMatrix();<a name="line.91"></a>
<span class="sourceLineNo">092</span><a name="line.92"></a>
<span class="sourceLineNo">093</span>        }<a name="line.93"></a>
<span class="sourceLineNo">094</span><a name="line.94"></a>
<span class="sourceLineNo">095</span>        public SeqRes2AtomAligner(){<a name="line.95"></a>
<span class="sourceLineNo">096</span>                alignmentString = "";<a name="line.96"></a>
<span class="sourceLineNo">097</span>        }<a name="line.97"></a>
<span class="sourceLineNo">098</span><a name="line.98"></a>
<span class="sourceLineNo">099</span>        public String getAlignmentString() {<a name="line.99"></a>
<span class="sourceLineNo">100</span>                return alignmentString;<a name="line.100"></a>
<span class="sourceLineNo">101</span>        }<a name="line.101"></a>
<span class="sourceLineNo">102</span><a name="line.102"></a>
<span class="sourceLineNo">103</span>        public boolean isDEBUG() {<a name="line.103"></a>
<span class="sourceLineNo">104</span>                return DEBUG;<a name="line.104"></a>
<span class="sourceLineNo">105</span>        }<a name="line.105"></a>
<span class="sourceLineNo">106</span><a name="line.106"></a>
<span class="sourceLineNo">107</span>        public void setDEBUG(boolean debug) {<a name="line.107"></a>
<span class="sourceLineNo">108</span>                DEBUG = debug;<a name="line.108"></a>
<span class="sourceLineNo">109</span>        }<a name="line.109"></a>
<span class="sourceLineNo">110</span><a name="line.110"></a>
<span class="sourceLineNo">111</span>        public Chain getMatchingAtomRes(Chain seqRes, List&lt;Chain&gt; atomList)<a name="line.111"></a>
<span class="sourceLineNo">112</span>                        throws StructureException {<a name="line.112"></a>
<span class="sourceLineNo">113</span>                Iterator&lt;Chain&gt; iter = atomList.iterator();<a name="line.113"></a>
<span class="sourceLineNo">114</span>                while(iter.hasNext()){<a name="line.114"></a>
<span class="sourceLineNo">115</span>                        Chain atomChain = iter.next();<a name="line.115"></a>
<span class="sourceLineNo">116</span>                        if ( atomChain.getChainID().equals(seqRes.getChainID())){<a name="line.116"></a>
<span class="sourceLineNo">117</span>                                return atomChain;<a name="line.117"></a>
<span class="sourceLineNo">118</span>                        }<a name="line.118"></a>
<span class="sourceLineNo">119</span>                }<a name="line.119"></a>
<span class="sourceLineNo">120</span>                throw new StructureException("could not match seqres chainID &gt;" + seqRes.getChainID() + "&lt; to ATOM chains!");<a name="line.120"></a>
<span class="sourceLineNo">121</span>        }<a name="line.121"></a>
<span class="sourceLineNo">122</span><a name="line.122"></a>
<span class="sourceLineNo">123</span><a name="line.123"></a>
<span class="sourceLineNo">124</span><a name="line.124"></a>
<span class="sourceLineNo">125</span><a name="line.125"></a>
<span class="sourceLineNo">126</span>        public void align(Structure s, List&lt;Chain&gt; seqResList){<a name="line.126"></a>
<span class="sourceLineNo">127</span><a name="line.127"></a>
<span class="sourceLineNo">128</span>                //List&lt;Chain&gt; seqResList = s.getSeqRes();<a name="line.128"></a>
<span class="sourceLineNo">129</span>                List&lt;Chain&gt; atomList   = s.getModel(0);<a name="line.129"></a>
<span class="sourceLineNo">130</span><a name="line.130"></a>
<span class="sourceLineNo">131</span>                for (Chain seqRes: seqResList){<a name="line.131"></a>
<span class="sourceLineNo">132</span>                        try {<a name="line.132"></a>
<span class="sourceLineNo">133</span><a name="line.133"></a>
<span class="sourceLineNo">134</span>                                Chain atomRes = getMatchingAtomRes(seqRes,atomList);<a name="line.134"></a>
<span class="sourceLineNo">135</span><a name="line.135"></a>
<span class="sourceLineNo">136</span>                                mapSeqresRecords(atomRes,seqRes);<a name="line.136"></a>
<span class="sourceLineNo">137</span>                                <a name="line.137"></a>
<span class="sourceLineNo">138</span>                                //chains.add(mapped);<a name="line.138"></a>
<span class="sourceLineNo">139</span>                        } catch (StructureException e){<a name="line.139"></a>
<span class="sourceLineNo">140</span>                                e.printStackTrace();<a name="line.140"></a>
<span class="sourceLineNo">141</span>                        }<a name="line.141"></a>
<span class="sourceLineNo">142</span>                }<a name="line.142"></a>
<span class="sourceLineNo">143</span>                //s.setChains(0,chains);<a name="line.143"></a>
<span class="sourceLineNo">144</span><a name="line.144"></a>
<span class="sourceLineNo">145</span><a name="line.145"></a>
<span class="sourceLineNo">146</span><a name="line.146"></a>
<span class="sourceLineNo">147</span>        }<a name="line.147"></a>
<span class="sourceLineNo">148</span><a name="line.148"></a>
<span class="sourceLineNo">149</span>        /** map the seqres groups to the atomRes chain.<a name="line.149"></a>
<span class="sourceLineNo">150</span>         *  updates the atomRes chain object with the mapped data<a name="line.150"></a>
<span class="sourceLineNo">151</span>         *  seqRes chain shuold not be needed after this and atomRes should be continued to be used.<a name="line.151"></a>
<span class="sourceLineNo">152</span>         *  <a name="line.152"></a>
<span class="sourceLineNo">153</span>         * @param atomRes<a name="line.153"></a>
<span class="sourceLineNo">154</span>         * @param seqRes<a name="line.154"></a>
<span class="sourceLineNo">155</span>         * @throws StructureException <a name="line.155"></a>
<span class="sourceLineNo">156</span>         */<a name="line.156"></a>
<span class="sourceLineNo">157</span>        public void mapSeqresRecords(Chain atomRes, Chain seqRes) throws StructureException {<a name="line.157"></a>
<span class="sourceLineNo">158</span>                List&lt;Group&gt; seqResGroups = seqRes.getAtomGroups();<a name="line.158"></a>
<span class="sourceLineNo">159</span>                List&lt;Group&gt; atmResGroups = atomRes.getAtomGroups();<a name="line.159"></a>
<span class="sourceLineNo">160</span>                <a name="line.160"></a>
<span class="sourceLineNo">161</span>                <a name="line.161"></a>
<span class="sourceLineNo">162</span>                if (DEBUG) {<a name="line.162"></a>
<span class="sourceLineNo">163</span>                        System.err.println("COMPARING " + atomRes.getChainID() + " (" + atmResGroups.size()+") " + seqRes.getChainID() + " (" +seqResGroups.size() +") ");<a name="line.163"></a>
<span class="sourceLineNo">164</span>                }<a name="line.164"></a>
<span class="sourceLineNo">165</span>                <a name="line.165"></a>
<span class="sourceLineNo">166</span>                <a name="line.166"></a>
<span class="sourceLineNo">167</span>                boolean simpleMatchSuccessful = trySimpleMatch(seqResGroups, atmResGroups);<a name="line.167"></a>
<span class="sourceLineNo">168</span><a name="line.168"></a>
<span class="sourceLineNo">169</span>                if ( simpleMatchSuccessful) {<a name="line.169"></a>
<span class="sourceLineNo">170</span>                        // update the new SEQRES list<a name="line.170"></a>
<span class="sourceLineNo">171</span>                        <a name="line.171"></a>
<span class="sourceLineNo">172</span>                        atomRes.setSeqResGroups(seqResGroups);<a name="line.172"></a>
<span class="sourceLineNo">173</span>                        return;<a name="line.173"></a>
<span class="sourceLineNo">174</span>                }<a name="line.174"></a>
<span class="sourceLineNo">175</span><a name="line.175"></a>
<span class="sourceLineNo">176</span>                //System.err.println("Could not map SEQRES to ATOM records easily, need to align...");<a name="line.176"></a>
<span class="sourceLineNo">177</span><a name="line.177"></a>
<span class="sourceLineNo">178</span>                if ( seqRes.getAtomGroups(GroupType.AMINOACID).size() &lt; 1) {<a name="line.178"></a>
<span class="sourceLineNo">179</span><a name="line.179"></a>
<span class="sourceLineNo">180</span>                        if ( seqRes.getAtomGroups(GroupType.NUCLEOTIDE).size() &gt; 1) {<a name="line.180"></a>
<span class="sourceLineNo">181</span>                                if (DEBUG){<a name="line.181"></a>
<span class="sourceLineNo">182</span>                                        System.out.println("chain " + seqRes.getChainID() + " is a nucleotide chain, aligning nucs...");<a name="line.182"></a>
<span class="sourceLineNo">183</span>                                }<a name="line.183"></a>
<span class="sourceLineNo">184</span>                                align2NucleotideChains(seqRes,atomRes);<a name="line.184"></a>
<span class="sourceLineNo">185</span>                                return;<a name="line.185"></a>
<span class="sourceLineNo">186</span>                        } else {<a name="line.186"></a>
<span class="sourceLineNo">187</span><a name="line.187"></a>
<span class="sourceLineNo">188</span>                                if (DEBUG){<a name="line.188"></a>
<span class="sourceLineNo">189</span>                                        System.out.println("chain " + seqRes.getChainID() + " does not contain amino acids, ignoring...");<a name="line.189"></a>
<span class="sourceLineNo">190</span>                                }<a name="line.190"></a>
<span class="sourceLineNo">191</span>                                return;<a name="line.191"></a>
<span class="sourceLineNo">192</span>                        }<a name="line.192"></a>
<span class="sourceLineNo">193</span>                }<a name="line.193"></a>
<span class="sourceLineNo">194</span><a name="line.194"></a>
<span class="sourceLineNo">195</span><a name="line.195"></a>
<span class="sourceLineNo">196</span><a name="line.196"></a>
<span class="sourceLineNo">197</span><a name="line.197"></a>
<span class="sourceLineNo">198</span><a name="line.198"></a>
<span class="sourceLineNo">199</span>                if ( atomRes.getAtomGroups(GroupType.AMINOACID).size() &lt; 1) {<a name="line.199"></a>
<span class="sourceLineNo">200</span>                        if (DEBUG){<a name="line.200"></a>
<span class="sourceLineNo">201</span>                                System.out.println("chain " + atomRes.getChainID() + " does not contain amino acids, ignoring...");<a name="line.201"></a>
<span class="sourceLineNo">202</span>                        }<a name="line.202"></a>
<span class="sourceLineNo">203</span>                        return;<a name="line.203"></a>
<span class="sourceLineNo">204</span>                }<a name="line.204"></a>
<span class="sourceLineNo">205</span>                if ( DEBUG )<a name="line.205"></a>
<span class="sourceLineNo">206</span>                        System.out.println("Alignment for chain "+ atomRes.getChainID() );<a name="line.206"></a>
<span class="sourceLineNo">207</span><a name="line.207"></a>
<span class="sourceLineNo">208</span>                <a name="line.208"></a>
<span class="sourceLineNo">209</span>                boolean noMatchFound = align(seqResGroups,atomRes.getAtomGroups());<a name="line.209"></a>
<span class="sourceLineNo">210</span>                if ( ! noMatchFound){<a name="line.210"></a>
<span class="sourceLineNo">211</span>                        atomRes.setSeqResGroups(seqResGroups);<a name="line.211"></a>
<span class="sourceLineNo">212</span>                }<a name="line.212"></a>
<span class="sourceLineNo">213</span>                <a name="line.213"></a>
<span class="sourceLineNo">214</span>        }<a name="line.214"></a>
<span class="sourceLineNo">215</span><a name="line.215"></a>
<span class="sourceLineNo">216</span>        private void align2NucleotideChains(Chain seqRes, Chain atomRes) throws StructureException {<a name="line.216"></a>
<span class="sourceLineNo">217</span><a name="line.217"></a>
<span class="sourceLineNo">218</span>                if ( atomRes.getAtomGroups(GroupType.NUCLEOTIDE).size() &lt; 1) {<a name="line.218"></a>
<span class="sourceLineNo">219</span>                        if (DEBUG){<a name="line.219"></a>
<span class="sourceLineNo">220</span>                                System.out.println("chain " + atomRes.getChainID() + " does not contain nucleotides, ignoring...");<a name="line.220"></a>
<span class="sourceLineNo">221</span>                        }<a name="line.221"></a>
<span class="sourceLineNo">222</span>                        return;<a name="line.222"></a>
<span class="sourceLineNo">223</span>                }<a name="line.223"></a>
<span class="sourceLineNo">224</span>                if ( DEBUG )<a name="line.224"></a>
<span class="sourceLineNo">225</span>                        System.out.println("Alignment for chain "+ atomRes.getChainID() );<a name="line.225"></a>
<span class="sourceLineNo">226</span><a name="line.226"></a>
<span class="sourceLineNo">227</span>                List&lt;Group&gt; seqResGroups = seqRes.getAtomGroups();<a name="line.227"></a>
<span class="sourceLineNo">228</span>                boolean noMatchFound = alignNucleotideGroups(seqResGroups,atomRes.getAtomGroups());<a name="line.228"></a>
<span class="sourceLineNo">229</span>                if ( ! noMatchFound){<a name="line.229"></a>
<span class="sourceLineNo">230</span>                        atomRes.setSeqResGroups(seqResGroups);<a name="line.230"></a>
<span class="sourceLineNo">231</span>                }<a name="line.231"></a>
<span class="sourceLineNo">232</span><a name="line.232"></a>
<span class="sourceLineNo">233</span>        }<a name="line.233"></a>
<span class="sourceLineNo">234</span><a name="line.234"></a>
<span class="sourceLineNo">235</span>        /** a simple matching approach that tries to do a 1:1 mapping between SEQRES and ATOM records<a name="line.235"></a>
<span class="sourceLineNo">236</span>         *  returns true if this simple matching approach worked fine<a name="line.236"></a>
<span class="sourceLineNo">237</span>         *  <a name="line.237"></a>
<span class="sourceLineNo">238</span>         * @param seqRes<a name="line.238"></a>
<span class="sourceLineNo">239</span>         * @param atomList<a name="line.239"></a>
<span class="sourceLineNo">240</span>         * @return<a name="line.240"></a>
<span class="sourceLineNo">241</span>         */<a name="line.241"></a>
<span class="sourceLineNo">242</span>        <a name="line.242"></a>
<span class="sourceLineNo">243</span>        public boolean trySimpleMatch(List&lt;Group&gt; seqResGroups,List&lt;Group&gt; atmResGroups) {<a name="line.243"></a>
<span class="sourceLineNo">244</span>                // by default first ATOM position is 1<a name="line.244"></a>
<span class="sourceLineNo">245</span>                //<a name="line.245"></a>
<span class="sourceLineNo">246</span>                boolean startAt1 = true;<a name="line.246"></a>
<span class="sourceLineNo">247</span><a name="line.247"></a>
<span class="sourceLineNo">248</span>                for ( int atomResPos = 0 ; atomResPos &lt; atmResGroups.size() ; atomResPos++){<a name="line.248"></a>
<span class="sourceLineNo">249</span><a name="line.249"></a>
<span class="sourceLineNo">250</span>                        //                      if ( DEBUG)<a name="line.250"></a>
<span class="sourceLineNo">251</span>                        //                              System.err.println(" trying to simple match " + atomResPos +"/" + atmResGroups.size());<a name="line.251"></a>
<span class="sourceLineNo">252</span><a name="line.252"></a>
<span class="sourceLineNo">253</span>                        // let's try to match this case<a name="line.253"></a>
<span class="sourceLineNo">254</span>                        Group atomResGroup = atmResGroups.get(atomResPos);<a name="line.254"></a>
<span class="sourceLineNo">255</span><a name="line.255"></a>
<span class="sourceLineNo">256</span>                        // let's ignore waters<a name="line.256"></a>
<span class="sourceLineNo">257</span>                        String threeLetterCode = atomResGroup.getPDBName();<a name="line.257"></a>
<span class="sourceLineNo">258</span>                        if ( excludeTypes.contains(threeLetterCode)){<a name="line.258"></a>
<span class="sourceLineNo">259</span>                                continue;<a name="line.259"></a>
<span class="sourceLineNo">260</span>                        }<a name="line.260"></a>
<span class="sourceLineNo">261</span><a name="line.261"></a>
<span class="sourceLineNo">262</span>                        ResidueNumber atomResNum = atomResGroup.getResidueNumber();<a name="line.262"></a>
<span class="sourceLineNo">263</span><a name="line.263"></a>
<span class="sourceLineNo">264</span>                        int seqResPos = atomResNum.getSeqNum();<a name="line.264"></a>
<span class="sourceLineNo">265</span><a name="line.265"></a>
<span class="sourceLineNo">266</span><a name="line.266"></a>
<span class="sourceLineNo">267</span><a name="line.267"></a>
<span class="sourceLineNo">268</span>                        if ( seqResPos &lt; 0) {<a name="line.268"></a>
<span class="sourceLineNo">269</span>                                if ( DEBUG )<a name="line.269"></a>
<span class="sourceLineNo">270</span>                                        System.err.println("ATOM residue number &lt; 0");<a name="line.270"></a>
<span class="sourceLineNo">271</span>                                return false;<a name="line.271"></a>
<span class="sourceLineNo">272</span>                        }<a name="line.272"></a>
<span class="sourceLineNo">273</span><a name="line.273"></a>
<span class="sourceLineNo">274</span>                        if ( seqResPos == 0){<a name="line.274"></a>
<span class="sourceLineNo">275</span>                                // make sure the first SEQRES is matching.<a name="line.275"></a>
<span class="sourceLineNo">276</span>                                Group seqResGroup = seqResGroups.get(0);<a name="line.276"></a>
<span class="sourceLineNo">277</span>                                if (  seqResGroup.getPDBName().equals(atomResGroup.getPDBName())){<a name="line.277"></a>
<span class="sourceLineNo">278</span>                                        // they match! seems in this case the numbering starts with 0...<a name="line.278"></a>
<span class="sourceLineNo">279</span>                                        startAt1 = false;<a name="line.279"></a>
<span class="sourceLineNo">280</span>                                } else {<a name="line.280"></a>
<span class="sourceLineNo">281</span>                                        if ( DEBUG){<a name="line.281"></a>
<span class="sourceLineNo">282</span>                                                System.err.println("SEQRES position 1  ("+seqResGroup.getPDBName()+<a name="line.282"></a>
<span class="sourceLineNo">283</span>                                                                ") does not match ATOM PDB res num 0 (" + atomResGroup.getPDBName()+")");<a name="line.283"></a>
<span class="sourceLineNo">284</span><a name="line.284"></a>
<span class="sourceLineNo">285</span>                                        }<a name="line.285"></a>
<span class="sourceLineNo">286</span>                                        return false;<a name="line.286"></a>
<span class="sourceLineNo">287</span><a name="line.287"></a>
<span class="sourceLineNo">288</span>                                }<a name="line.288"></a>
<span class="sourceLineNo">289</span>                        }<a name="line.289"></a>
<span class="sourceLineNo">290</span><a name="line.290"></a>
<span class="sourceLineNo">291</span><a name="line.291"></a>
<span class="sourceLineNo">292</span>                        if ( startAt1 )<a name="line.292"></a>
<span class="sourceLineNo">293</span>                                seqResPos--;<a name="line.293"></a>
<span class="sourceLineNo">294</span><a name="line.294"></a>
<span class="sourceLineNo">295</span>                        // another check that would require the alignment approach<a name="line.295"></a>
<span class="sourceLineNo">296</span>                        if ( startAt1 &amp;&amp; seqResPos &gt;=  seqResGroups.size()  )<a name="line.296"></a>
<span class="sourceLineNo">297</span>                        {<a name="line.297"></a>
<span class="sourceLineNo">298</span><a name="line.298"></a>
<span class="sourceLineNo">299</span>                                // could be a HETATOM...<a name="line.299"></a>
<span class="sourceLineNo">300</span>                                if ( atomResGroup instanceof AminoAcid) {<a name="line.300"></a>
<span class="sourceLineNo">301</span>                                        if ( DEBUG )<a name="line.301"></a>
<span class="sourceLineNo">302</span>                                                System.err.println(" ATOM residue nr: " + seqResPos + " &gt; seqres! " + seqResGroups.size() + " " + atomResGroup);<a name="line.302"></a>
<span class="sourceLineNo">303</span>                                        return false;<a name="line.303"></a>
<span class="sourceLineNo">304</span>                                } else if ( atomResGroup instanceof NucleotideImpl) {<a name="line.304"></a>
<span class="sourceLineNo">305</span>                                        if ( DEBUG )<a name="line.305"></a>
<span class="sourceLineNo">306</span>                                                System.err.println(" NUCLEOTIDE residue nr: " + seqResPos + " &gt; seqres! " + seqResGroups.size() + " " + atomResGroup);<a name="line.306"></a>
<span class="sourceLineNo">307</span>                                        return false;<a name="line.307"></a>
<span class="sourceLineNo">308</span>                                } else {<a name="line.308"></a>
<span class="sourceLineNo">309</span>                                        // we won't map HETATOM groups...<a name="line.309"></a>
<span class="sourceLineNo">310</span>                                        continue;<a name="line.310"></a>
<span class="sourceLineNo">311</span>                                }<a name="line.311"></a>
<span class="sourceLineNo">312</span>                        }<a name="line.312"></a>
<span class="sourceLineNo">313</span><a name="line.313"></a>
<span class="sourceLineNo">314</span><a name="line.314"></a>
<span class="sourceLineNo">315</span>                        //                      if ( seqResPos &lt; 0){<a name="line.315"></a>
<span class="sourceLineNo">316</span>                        //<a name="line.316"></a>
<span class="sourceLineNo">317</span>                        //                              System.err.println("What is going on??? " + atomRes.getChainID() + " " + atomResGroup);<a name="line.317"></a>
<span class="sourceLineNo">318</span>                        //                      }<a name="line.318"></a>
<span class="sourceLineNo">319</span><a name="line.319"></a>
<span class="sourceLineNo">320</span>                        if ( seqResPos &gt;= seqResGroups.size()){<a name="line.320"></a>
<span class="sourceLineNo">321</span>                                if ( DEBUG)<a name="line.321"></a>
<span class="sourceLineNo">322</span>                                        System.err.println("seqres groups don't match atom indeces " + seqResPos);<a name="line.322"></a>
<span class="sourceLineNo">323</span>                                if ( atomResGroup instanceof AminoAcid )<a name="line.323"></a>
<span class="sourceLineNo">324</span>                                        return false;<a name="line.324"></a>
<span class="sourceLineNo">325</span>                                else<a name="line.325"></a>
<span class="sourceLineNo">326</span>                                        continue;<a name="line.326"></a>
<span class="sourceLineNo">327</span>                        }<a name="line.327"></a>
<span class="sourceLineNo">328</span><a name="line.328"></a>
<span class="sourceLineNo">329</span>                        Group seqResGroup = seqResGroups.get(seqResPos );<a name="line.329"></a>
<span class="sourceLineNo">330</span><a name="line.330"></a>
<span class="sourceLineNo">331</span>                        if ( ! seqResGroup.getPDBName().trim().equals(atomResGroup.getPDBName().trim())){<a name="line.331"></a>
<span class="sourceLineNo">332</span>                                // a mismatch! something is wrong in the mapping and we need to do an alignment<a name="line.332"></a>
<span class="sourceLineNo">333</span>                                if ( DEBUG )<a name="line.333"></a>
<span class="sourceLineNo">334</span>                                        System.err.println("Mismatch of SEQRES pos " + seqResPos + " and ATOM record: " + atomResGroup + " | " + seqResGroup);<a name="line.334"></a>
<span class="sourceLineNo">335</span>                                return false;<a name="line.335"></a>
<span class="sourceLineNo">336</span>                        }<a name="line.336"></a>
<span class="sourceLineNo">337</span><a name="line.337"></a>
<span class="sourceLineNo">338</span>                        // the two groups are identical and we can merge them<a name="line.338"></a>
<span class="sourceLineNo">339</span>                        // replace the SEQRES group with the ATOM group...<a name="line.339"></a>
<span class="sourceLineNo">340</span>                        if (DEBUG)<a name="line.340"></a>
<span class="sourceLineNo">341</span>                                System.err.println("merging " + seqResPos + " " + atomResGroup);<a name="line.341"></a>
<span class="sourceLineNo">342</span>                        seqResGroups.set(seqResPos, atomResGroup);<a name="line.342"></a>
<span class="sourceLineNo">343</span><a name="line.343"></a>
<span class="sourceLineNo">344</span>                }<a name="line.344"></a>
<span class="sourceLineNo">345</span><a name="line.345"></a>
<span class="sourceLineNo">346</span>                // all atom records could get matched correctly!<a name="line.346"></a>
<span class="sourceLineNo">347</span>                return true;<a name="line.347"></a>
<span class="sourceLineNo">348</span><a name="line.348"></a>
<span class="sourceLineNo">349</span>        }<a name="line.349"></a>
<span class="sourceLineNo">350</span><a name="line.350"></a>
<span class="sourceLineNo">351</span>        /** returns the full sequence of the Atom records of a parent<a name="line.351"></a>
<span class="sourceLineNo">352</span>         * with X instead of HETATMSs. The advantage of this is<a name="line.352"></a>
<span class="sourceLineNo">353</span>         * that it allows us to also align HETATM groups back to the SEQRES.<a name="line.353"></a>
<span class="sourceLineNo">354</span>         * @param groups the list of groups in a parent<a name="line.354"></a>
<span class="sourceLineNo">355</span>         * @param positionIndex a Map to keep track of which group is at which sequence position<a name="line.355"></a>
<span class="sourceLineNo">356</span>         * @return string representations<a name="line.356"></a>
<span class="sourceLineNo">357</span>         */<a name="line.357"></a>
<span class="sourceLineNo">358</span>        public static String getFullAtomSequence(List&lt;Group&gt; groups, Map&lt;Integer, Integer&gt; positionIndex){<a name="line.358"></a>
<span class="sourceLineNo">359</span><a name="line.359"></a>
<span class="sourceLineNo">360</span><a name="line.360"></a>
<span class="sourceLineNo">361</span>                StringBuffer sequence = new StringBuffer() ;<a name="line.361"></a>
<span class="sourceLineNo">362</span>                int seqIndex = 0; // track sequence.length()<a name="line.362"></a>
<span class="sourceLineNo">363</span>                for ( int i=0 ; i&lt; groups.size(); i++){<a name="line.363"></a>
<span class="sourceLineNo">364</span>                        Group g = (Group) groups.get(i);<a name="line.364"></a>
<span class="sourceLineNo">365</span><a name="line.365"></a>
<span class="sourceLineNo">366</span>                        if ( g instanceof AminoAcid ){<a name="line.366"></a>
<span class="sourceLineNo">367</span>                                AminoAcid a = (AminoAcid)g;<a name="line.367"></a>
<span class="sourceLineNo">368</span>                                char oneLetter =a.getAminoType();<a name="line.368"></a>
<span class="sourceLineNo">369</span>                                if ( oneLetter == '?')<a name="line.369"></a>
<span class="sourceLineNo">370</span>                                        oneLetter = 'X';<a name="line.370"></a>
<span class="sourceLineNo">371</span><a name="line.371"></a>
<span class="sourceLineNo">372</span>                                positionIndex.put(seqIndex,i);<a name="line.372"></a>
<span class="sourceLineNo">373</span>                                sequence.append(oneLetter );<a name="line.373"></a>
<span class="sourceLineNo">374</span>                                seqIndex++;<a name="line.374"></a>
<span class="sourceLineNo">375</span>                        } else {<a name="line.375"></a>
<span class="sourceLineNo">376</span><a name="line.376"></a>
<span class="sourceLineNo">377</span>                                // exclude solvents<a name="line.377"></a>
<span class="sourceLineNo">378</span>                                if (  excludeTypes.contains(g.getPDBName()))<a name="line.378"></a>
<span class="sourceLineNo">379</span>                                        continue;<a name="line.379"></a>
<span class="sourceLineNo">380</span><a name="line.380"></a>
<span class="sourceLineNo">381</span>                                // exclude metals<a name="line.381"></a>
<span class="sourceLineNo">382</span>                                if ( g.size() == 1 ) {<a name="line.382"></a>
<span class="sourceLineNo">383</span>                                        try {<a name="line.383"></a>
<span class="sourceLineNo">384</span>                                                Atom a = g.getAtom(0);<a name="line.384"></a>
<span class="sourceLineNo">385</span>                                                if (a.getElement().isMetal())<a name="line.385"></a>
<span class="sourceLineNo">386</span>                                                        continue;<a name="line.386"></a>
<span class="sourceLineNo">387</span><a name="line.387"></a>
<span class="sourceLineNo">388</span>                                                continue;<a name="line.388"></a>
<span class="sourceLineNo">389</span>                                        } catch (StructureException e){<a name="line.389"></a>
<span class="sourceLineNo">390</span>                                                e.printStackTrace();<a name="line.390"></a>
<span class="sourceLineNo">391</span>                                                continue;<a name="line.391"></a>
<span class="sourceLineNo">392</span>                                        }<a name="line.392"></a>
<span class="sourceLineNo">393</span>                                }<a name="line.393"></a>
<span class="sourceLineNo">394</span><a name="line.394"></a>
<span class="sourceLineNo">395</span>                                ChemComp cc = g.getChemComp();<a name="line.395"></a>
<span class="sourceLineNo">396</span>                                if ( <a name="line.396"></a>
<span class="sourceLineNo">397</span>                                                ResidueType.lPeptideLinking.equals(cc.getResidueType()) ||<a name="line.397"></a>
<span class="sourceLineNo">398</span>                                                PolymerType.PROTEIN_ONLY.contains(cc.getPolymerType())  ||<a name="line.398"></a>
<span class="sourceLineNo">399</span>                                                PolymerType.POLYNUCLEOTIDE_ONLY.contains(cc.getPolymerType())<a name="line.399"></a>
<span class="sourceLineNo">400</span>                                                ) {<a name="line.400"></a>
<span class="sourceLineNo">401</span>                                        //System.out.println(cc.getOne_letter_code());<a name="line.401"></a>
<span class="sourceLineNo">402</span>                                        String c = cc.getOne_letter_code();<a name="line.402"></a>
<span class="sourceLineNo">403</span>                                        if ( c.equals("?"))<a name="line.403"></a>
<span class="sourceLineNo">404</span>                                                c = "X";<a name="line.404"></a>
<span class="sourceLineNo">405</span><a name="line.405"></a>
<span class="sourceLineNo">406</span>                                        positionIndex.put(seqIndex,i);<a name="line.406"></a>
<span class="sourceLineNo">407</span>                                        sequence.append(c);<a name="line.407"></a>
<span class="sourceLineNo">408</span>                                        seqIndex++;<a name="line.408"></a>
<span class="sourceLineNo">409</span>                                } else {<a name="line.409"></a>
<span class="sourceLineNo">410</span>                                        //System.out.println(cc);<a name="line.410"></a>
<span class="sourceLineNo">411</span>                                        continue;<a name="line.411"></a>
<span class="sourceLineNo">412</span>                                }<a name="line.412"></a>
<span class="sourceLineNo">413</span><a name="line.413"></a>
<span class="sourceLineNo">414</span><a name="line.414"></a>
<span class="sourceLineNo">415</span>                                //sequence.append("X");<a name="line.415"></a>
<span class="sourceLineNo">416</span>                        }<a name="line.416"></a>
<span class="sourceLineNo">417</span><a name="line.417"></a>
<span class="sourceLineNo">418</span>                }<a name="line.418"></a>
<span class="sourceLineNo">419</span><a name="line.419"></a>
<span class="sourceLineNo">420</span>                return sequence.toString();<a name="line.420"></a>
<span class="sourceLineNo">421</span><a name="line.421"></a>
<span class="sourceLineNo">422</span>        }<a name="line.422"></a>
<span class="sourceLineNo">423</span><a name="line.423"></a>
<span class="sourceLineNo">424</span><a name="line.424"></a>
<span class="sourceLineNo">425</span>        public boolean alignNucleotideGroups(List&lt;Group&gt; seqRes, List&lt;Group&gt; atomRes) throws StructureException{<a name="line.425"></a>
<span class="sourceLineNo">426</span><a name="line.426"></a>
<span class="sourceLineNo">427</span>                Map&lt;Integer,Integer&gt; seqresIndexPosition = new HashMap&lt;Integer, Integer&gt;();<a name="line.427"></a>
<span class="sourceLineNo">428</span>                Map&lt;Integer,Integer&gt; atomIndexPosition   = new HashMap&lt;Integer, Integer&gt;();<a name="line.428"></a>
<span class="sourceLineNo">429</span><a name="line.429"></a>
<span class="sourceLineNo">430</span>                String seq1 = getFullAtomSequence(seqRes, seqresIndexPosition);<a name="line.430"></a>
<span class="sourceLineNo">431</span>                //<a name="line.431"></a>
<span class="sourceLineNo">432</span>                String seq2 = getFullAtomSequence(atomRes, atomIndexPosition);<a name="line.432"></a>
<span class="sourceLineNo">433</span><a name="line.433"></a>
<span class="sourceLineNo">434</span>                if ( DEBUG ) {<a name="line.434"></a>
<span class="sourceLineNo">435</span><a name="line.435"></a>
<span class="sourceLineNo">436</span>                        System.out.println("align seq1 ("+ seq1.length()+") " + seq1);<a name="line.436"></a>
<span class="sourceLineNo">437</span>                        System.out.println("align seq2 ("+ seq2.length()+") " + seq2);<a name="line.437"></a>
<span class="sourceLineNo">438</span>                }<a name="line.438"></a>
<span class="sourceLineNo">439</span><a name="line.439"></a>
<span class="sourceLineNo">440</span><a name="line.440"></a>
<span class="sourceLineNo">441</span>                Sequence s1;<a name="line.441"></a>
<span class="sourceLineNo">442</span>                Sequence s2;<a name="line.442"></a>
<span class="sourceLineNo">443</span><a name="line.443"></a>
<span class="sourceLineNo">444</span>                try {<a name="line.444"></a>
<span class="sourceLineNo">445</span>                        s1 = new DNASequence(seq1,AmbiguityDNACompoundSet.getDNACompoundSet());<a name="line.445"></a>
<span class="sourceLineNo">446</span>                } catch (CompoundNotFoundError e){<a name="line.446"></a>
<span class="sourceLineNo">447</span>                        ///oops perhaps a RNA sequence?<a name="line.447"></a>
<span class="sourceLineNo">448</span>                        try {<a name="line.448"></a>
<span class="sourceLineNo">449</span>                                s1 = new RNASequence(seq1,AmbiguityRNACompoundSet.getRNACompoundSet());<a name="line.449"></a>
<span class="sourceLineNo">450</span>                        } catch (CompoundNotFoundError ex) {<a name="line.450"></a>
<span class="sourceLineNo">451</span>                                System.err.println("Could not determine compound set for sequence 1 " + seq1);<a name="line.451"></a>
<span class="sourceLineNo">452</span>                                return false;<a name="line.452"></a>
<span class="sourceLineNo">453</span>                        }<a name="line.453"></a>
<span class="sourceLineNo">454</span>                }<a name="line.454"></a>
<span class="sourceLineNo">455</span>                try {<a name="line.455"></a>
<span class="sourceLineNo">456</span>                        s2 = new DNASequence(seq2,AmbiguityDNACompoundSet.getDNACompoundSet());<a name="line.456"></a>
<span class="sourceLineNo">457</span>                } catch (CompoundNotFoundError e){<a name="line.457"></a>
<span class="sourceLineNo">458</span>                        ///oops perhaps a RNA sequence?<a name="line.458"></a>
<span class="sourceLineNo">459</span>                        try {<a name="line.459"></a>
<span class="sourceLineNo">460</span>                                s2 = new RNASequence(seq2,AmbiguityRNACompoundSet.getRNACompoundSet());<a name="line.460"></a>
<span class="sourceLineNo">461</span>                        } catch (CompoundNotFoundError ex) {<a name="line.461"></a>
<span class="sourceLineNo">462</span>                                System.err.println("Could not determine compound set for sequence 2 " + seq2);<a name="line.462"></a>
<span class="sourceLineNo">463</span>                                return false;<a name="line.463"></a>
<span class="sourceLineNo">464</span>                        }<a name="line.464"></a>
<span class="sourceLineNo">465</span>                }<a name="line.465"></a>
<span class="sourceLineNo">466</span><a name="line.466"></a>
<span class="sourceLineNo">467</span>                SubstitutionMatrix&lt;NucleotideCompound&gt; matrix = SubstitutionMatrixHelper.getNuc4_2();<a name="line.467"></a>
<span class="sourceLineNo">468</span><a name="line.468"></a>
<span class="sourceLineNo">469</span>                GapPenalty penalty = new SimpleGapPenalty();<a name="line.469"></a>
<span class="sourceLineNo">470</span><a name="line.470"></a>
<span class="sourceLineNo">471</span>                short gop = 8;<a name="line.471"></a>
<span class="sourceLineNo">472</span>                short extend = 1;<a name="line.472"></a>
<span class="sourceLineNo">473</span>                penalty.setOpenPenalty(gop);<a name="line.473"></a>
<span class="sourceLineNo">474</span>                penalty.setExtensionPenalty(extend);<a name="line.474"></a>
<span class="sourceLineNo">475</span><a name="line.475"></a>
<span class="sourceLineNo">476</span>                try {<a name="line.476"></a>
<span class="sourceLineNo">477</span>                        PairwiseSequenceAligner smithWaterman =<a name="line.477"></a>
<span class="sourceLineNo">478</span>                                        Alignments.getPairwiseAligner(s1, s2, PairwiseSequenceAlignerType.LOCAL, penalty, matrix);<a name="line.478"></a>
<span class="sourceLineNo">479</span><a name="line.479"></a>
<span class="sourceLineNo">480</span>                        SequencePair pair = smithWaterman.getPair();<a name="line.480"></a>
<span class="sourceLineNo">481</span><a name="line.481"></a>
<span class="sourceLineNo">482</span><a name="line.482"></a>
<span class="sourceLineNo">483</span><a name="line.483"></a>
<span class="sourceLineNo">484</span>                        if ( pair == null)<a name="line.484"></a>
<span class="sourceLineNo">485</span>                                throw new StructureException("could not align objects!");<a name="line.485"></a>
<span class="sourceLineNo">486</span><a name="line.486"></a>
<span class="sourceLineNo">487</span><a name="line.487"></a>
<span class="sourceLineNo">488</span>                        if ( DEBUG ) {<a name="line.488"></a>
<span class="sourceLineNo">489</span>                                System.out.println(pair.toString(60));<a name="line.489"></a>
<span class="sourceLineNo">490</span>                        }<a name="line.490"></a>
<span class="sourceLineNo">491</span><a name="line.491"></a>
<span class="sourceLineNo">492</span>                        boolean noMatchFound = mapDNAChains(seqRes,atomRes,pair,seqresIndexPosition, atomIndexPosition );<a name="line.492"></a>
<span class="sourceLineNo">493</span><a name="line.493"></a>
<span class="sourceLineNo">494</span>                        return noMatchFound;<a name="line.494"></a>
<span class="sourceLineNo">495</span>                } catch (Exception e){<a name="line.495"></a>
<span class="sourceLineNo">496</span>                        System.err.println("Problem while aligning ATOM and SEQRES records for " + atomRes.get(0).getChain().getParent().getPDBCode() + " chain: "  +atomRes.get(0).getChain().getChainID());<a name="line.496"></a>
<span class="sourceLineNo">497</span>                        //e.printStackTrace();<a name="line.497"></a>
<span class="sourceLineNo">498</span>                        return true;<a name="line.498"></a>
<span class="sourceLineNo">499</span>                }<a name="line.499"></a>
<span class="sourceLineNo">500</span><a name="line.500"></a>
<span class="sourceLineNo">501</span>        }<a name="line.501"></a>
<span class="sourceLineNo">502</span><a name="line.502"></a>
<span class="sourceLineNo">503</span><a name="line.503"></a>
<span class="sourceLineNo">504</span>        /** aligns two chains of groups, where the first parent is representing the<a name="line.504"></a>
<span class="sourceLineNo">505</span>         * list of amino acids as obtained from the SEQRES records, and the second parent<a name="line.505"></a>
<span class="sourceLineNo">506</span>         * represents the groups obtained from the ATOM records (and containing the actual ATOM information).<a name="line.506"></a>
<span class="sourceLineNo">507</span>         * This does the actual alignment and if a group can be mapped to a position in the SEQRES then the corresponding<a name="line.507"></a>
<span class="sourceLineNo">508</span>         * position is repplaced with the group that contains the atoms.<a name="line.508"></a>
<span class="sourceLineNo">509</span>         *<a name="line.509"></a>
<span class="sourceLineNo">510</span>         * @param seqRes<a name="line.510"></a>
<span class="sourceLineNo">511</span>         * @param atomRes<a name="line.511"></a>
<span class="sourceLineNo">512</span>         * @return true if no match has bee found<a name="line.512"></a>
<span class="sourceLineNo">513</span>         * @throws StructureException<a name="line.513"></a>
<span class="sourceLineNo">514</span>         */<a name="line.514"></a>
<span class="sourceLineNo">515</span>        public boolean align(List&lt;Group&gt; seqRes, List&lt;Group&gt; atomRes) throws StructureException{<a name="line.515"></a>
<span class="sourceLineNo">516</span><a name="line.516"></a>
<span class="sourceLineNo">517</span>                Map&lt;Integer,Integer&gt; seqresIndexPosition = new HashMap&lt;Integer, Integer&gt;();<a name="line.517"></a>
<span class="sourceLineNo">518</span>                Map&lt;Integer,Integer&gt; atomIndexPosition   = new HashMap&lt;Integer, Integer&gt;();<a name="line.518"></a>
<span class="sourceLineNo">519</span><a name="line.519"></a>
<span class="sourceLineNo">520</span>                String seq1 = getFullAtomSequence(seqRes, seqresIndexPosition);<a name="line.520"></a>
<span class="sourceLineNo">521</span>                //<a name="line.521"></a>
<span class="sourceLineNo">522</span>                String seq2 = getFullAtomSequence(atomRes, atomIndexPosition);<a name="line.522"></a>
<span class="sourceLineNo">523</span><a name="line.523"></a>
<span class="sourceLineNo">524</span>                if ( DEBUG ) {<a name="line.524"></a>
<span class="sourceLineNo">525</span><a name="line.525"></a>
<span class="sourceLineNo">526</span>                        System.out.println("align seq1 ("+ seq1.length()+") " + seq1);<a name="line.526"></a>
<span class="sourceLineNo">527</span>                        System.out.println("align seq2 ("+ seq2.length()+") " + seq2);<a name="line.527"></a>
<span class="sourceLineNo">528</span>                }<a name="line.528"></a>
<span class="sourceLineNo">529</span>                ProteinSequence s1 = new ProteinSequence(seq1);<a name="line.529"></a>
<span class="sourceLineNo">530</span>                ProteinSequence s2 = new ProteinSequence(seq2);<a name="line.530"></a>
<span class="sourceLineNo">531</span><a name="line.531"></a>
<span class="sourceLineNo">532</span>                SubstitutionMatrix&lt;AminoAcidCompound&gt; matrix = SubstitutionMatrixHelper.getBlosum65();<a name="line.532"></a>
<span class="sourceLineNo">533</span><a name="line.533"></a>
<span class="sourceLineNo">534</span>                GapPenalty penalty = new SimpleGapPenalty();<a name="line.534"></a>
<span class="sourceLineNo">535</span><a name="line.535"></a>
<span class="sourceLineNo">536</span>                short gop = 8;<a name="line.536"></a>
<span class="sourceLineNo">537</span>                short extend = 1;<a name="line.537"></a>
<span class="sourceLineNo">538</span>                penalty.setOpenPenalty(gop);<a name="line.538"></a>
<span class="sourceLineNo">539</span>                penalty.setExtensionPenalty(extend);<a name="line.539"></a>
<span class="sourceLineNo">540</span><a name="line.540"></a>
<span class="sourceLineNo">541</span>                try {<a name="line.541"></a>
<span class="sourceLineNo">542</span>                        PairwiseSequenceAligner&lt;ProteinSequence, AminoAcidCompound&gt; smithWaterman =<a name="line.542"></a>
<span class="sourceLineNo">543</span>                                        Alignments.getPairwiseAligner(s1, s2, PairwiseSequenceAlignerType.LOCAL, penalty, matrix);<a name="line.543"></a>
<span class="sourceLineNo">544</span><a name="line.544"></a>
<span class="sourceLineNo">545</span>                        SequencePair&lt;ProteinSequence, AminoAcidCompound&gt; pair = smithWaterman.getPair();<a name="line.545"></a>
<span class="sourceLineNo">546</span><a name="line.546"></a>
<span class="sourceLineNo">547</span><a name="line.547"></a>
<span class="sourceLineNo">548</span><a name="line.548"></a>
<span class="sourceLineNo">549</span>                        if ( pair == null)<a name="line.549"></a>
<span class="sourceLineNo">550</span>                                throw new StructureException("could not align objects!");<a name="line.550"></a>
<span class="sourceLineNo">551</span><a name="line.551"></a>
<span class="sourceLineNo">552</span><a name="line.552"></a>
<span class="sourceLineNo">553</span>                        if ( DEBUG ) {<a name="line.553"></a>
<span class="sourceLineNo">554</span>                                System.out.println(pair.toString(60));<a name="line.554"></a>
<span class="sourceLineNo">555</span>                        }<a name="line.555"></a>
<span class="sourceLineNo">556</span><a name="line.556"></a>
<span class="sourceLineNo">557</span>                        boolean noMatchFound = mapChains(seqRes,atomRes,pair,seqresIndexPosition, atomIndexPosition );<a name="line.557"></a>
<span class="sourceLineNo">558</span><a name="line.558"></a>
<span class="sourceLineNo">559</span>                        return noMatchFound;<a name="line.559"></a>
<span class="sourceLineNo">560</span>                } catch (Exception e){<a name="line.560"></a>
<span class="sourceLineNo">561</span>                        System.err.println("Problem while aligning ATOM and SEQRES records for " + atomRes.get(0).getChain().getParent().getPDBCode() + " chain: "  +atomRes.get(0).getChain().getChainID());<a name="line.561"></a>
<span class="sourceLineNo">562</span>                        //e.printStackTrace();<a name="line.562"></a>
<span class="sourceLineNo">563</span>                        return true;<a name="line.563"></a>
<span class="sourceLineNo">564</span>                }<a name="line.564"></a>
<span class="sourceLineNo">565</span><a name="line.565"></a>
<span class="sourceLineNo">566</span>        }<a name="line.566"></a>
<span class="sourceLineNo">567</span><a name="line.567"></a>
<span class="sourceLineNo">568</span><a name="line.568"></a>
<span class="sourceLineNo">569</span>        private boolean mapChains(List&lt;Group&gt; seqResGroups, List&lt;Group&gt; atomRes,<a name="line.569"></a>
<span class="sourceLineNo">570</span>                        SequencePair&lt;ProteinSequence, AminoAcidCompound&gt; pair,<a name="line.570"></a>
<span class="sourceLineNo">571</span>                        Map&lt;Integer,Integer&gt; seqresIndexPosition,<a name="line.571"></a>
<span class="sourceLineNo">572</span>                        Map&lt;Integer,Integer&gt; atomIndexPosition<a name="line.572"></a>
<span class="sourceLineNo">573</span>                        ) throws StructureException{<a name="line.573"></a>
<span class="sourceLineNo">574</span><a name="line.574"></a>
<span class="sourceLineNo">575</span><a name="line.575"></a>
<span class="sourceLineNo">576</span><a name="line.576"></a>
<span class="sourceLineNo">577</span>                // at the present stage the future seqREs are still stored as Atom groups in the seqRes parent...<a name="line.577"></a>
<span class="sourceLineNo">578</span><a name="line.578"></a>
<span class="sourceLineNo">579</span><a name="line.579"></a>
<span class="sourceLineNo">580</span>                int aligLength = pair.getLength();<a name="line.580"></a>
<span class="sourceLineNo">581</span><a name="line.581"></a>
<span class="sourceLineNo">582</span>                //              System.out.println("sequence length seqres:");<a name="line.582"></a>
<span class="sourceLineNo">583</span>                //              System.out.println(seqresIndexPosition.keySet().size());<a name="line.583"></a>
<span class="sourceLineNo">584</span>                //              System.out.println("alignment length: " + aligLength);<a name="line.584"></a>
<span class="sourceLineNo">585</span>                // System.out.println(gapSymbol.getName());<a name="line.585"></a>
<span class="sourceLineNo">586</span><a name="line.586"></a>
<span class="sourceLineNo">587</span>                // make sure we actually find an alignment<a name="line.587"></a>
<span class="sourceLineNo">588</span>                boolean noMatchFound = true;<a name="line.588"></a>
<span class="sourceLineNo">589</span><a name="line.589"></a>
<span class="sourceLineNo">590</span>                Compound gapSymbol =  AminoAcidCompoundSet.getAminoAcidCompoundSet().getCompoundForString("-");<a name="line.590"></a>
<span class="sourceLineNo">591</span><a name="line.591"></a>
<span class="sourceLineNo">592</span>                mainLoop:<a name="line.592"></a>
<span class="sourceLineNo">593</span>                        for (int i = 1; i &lt;= aligLength ; i++) {<a name="line.593"></a>
<span class="sourceLineNo">594</span><a name="line.594"></a>
<span class="sourceLineNo">595</span>                                Compound s =  pair.getCompoundAt(1, i);<a name="line.595"></a>
<span class="sourceLineNo">596</span>                                Compound a =  pair.getCompoundAt(2,i);<a name="line.596"></a>
<span class="sourceLineNo">597</span><a name="line.597"></a>
<span class="sourceLineNo">598</span>                                // alignment is using internal index start at 1...<a name="line.598"></a>
<span class="sourceLineNo">599</span>                                int posSeq = pair.getIndexInQueryAt(i)  -1;<a name="line.599"></a>
<span class="sourceLineNo">600</span>                                int posAtom = pair.getIndexInTargetAt(i) -1;<a name="line.600"></a>
<span class="sourceLineNo">601</span><a name="line.601"></a>
<span class="sourceLineNo">602</span>                                if (  s.equals(gapSymbol) || a.equals(gapSymbol)){<a name="line.602"></a>
<span class="sourceLineNo">603</span>                                        continue;                               <a name="line.603"></a>
<span class="sourceLineNo">604</span>                                }<a name="line.604"></a>
<span class="sourceLineNo">605</span><a name="line.605"></a>
<span class="sourceLineNo">606</span>                                if ( s.equals(a)){<a name="line.606"></a>
<span class="sourceLineNo">607</span><a name="line.607"></a>
<span class="sourceLineNo">608</span>                                        // the atom record can be aligned to the SeqRes record!<a name="line.608"></a>
<span class="sourceLineNo">609</span>                                        // replace the SeqRes group with the Atom group!<a name="line.609"></a>
<span class="sourceLineNo">610</span><a name="line.610"></a>
<span class="sourceLineNo">611</span>                                        Group s1 = seqResGroups.get(seqresIndexPosition.get(posSeq));<a name="line.611"></a>
<span class="sourceLineNo">612</span>                                        Group a1 = atomRes.get(atomIndexPosition.get(posAtom));<a name="line.612"></a>
<span class="sourceLineNo">613</span><a name="line.613"></a>
<span class="sourceLineNo">614</span>                                        if ( s1 == null || a1 == null){<a name="line.614"></a>
<span class="sourceLineNo">615</span>                                                /// can't map this position...<a name="line.615"></a>
<span class="sourceLineNo">616</span>                                                System.err.println("can't map " + i + ":" + s + " " + posSeq +" " + s1 + " atom: " + posAtom + " " + a1 );<a name="line.616"></a>
<span class="sourceLineNo">617</span>                                                continue mainLoop;<a name="line.617"></a>
<span class="sourceLineNo">618</span>                                        }<a name="line.618"></a>
<span class="sourceLineNo">619</span><a name="line.619"></a>
<span class="sourceLineNo">620</span>                                        // need to trim the names to allow matching e.g in<a name="line.620"></a>
<span class="sourceLineNo">621</span>                                        // pdb1b2m<a name="line.621"></a>
<span class="sourceLineNo">622</span>                                        String pdbNameS = s1.getPDBName();<a name="line.622"></a>
<span class="sourceLineNo">623</span>                                        String pdbNameA = a1.getPDBName();<a name="line.623"></a>
<span class="sourceLineNo">624</span>                                        if ( pdbNameS == null || pdbNameA == null ){<a name="line.624"></a>
<span class="sourceLineNo">625</span>                                                System.err.println("null value found at " + posSeq + " when trying to align " + s1 + " and " + a1 + " " + posAtom);<a name="line.625"></a>
<span class="sourceLineNo">626</span>                                                throw new StructureException("null value found at group.getPDBName()");<a name="line.626"></a>
<span class="sourceLineNo">627</span>                                        }<a name="line.627"></a>
<span class="sourceLineNo">628</span>                                        if (! pdbNameA.equals(pdbNameS)){<a name="line.628"></a>
<span class="sourceLineNo">629</span>                                                if ( ! pdbNameA.trim().equals(pdbNameS.trim())) {<a name="line.629"></a>
<span class="sourceLineNo">630</span>                                                        System.err.println(s1 + " " + posSeq + " does not align with " + a1+ " " + posAtom + " should be: " + s + " : " + a);<a name="line.630"></a>
<span class="sourceLineNo">631</span>                                                        if ( s1.getType().equals(HetatomImpl.type) &amp;&amp; a1.getType().equals(HetatomImpl.type)){<a name="line.631"></a>
<span class="sourceLineNo">632</span>                                                                System.err.println("they seem to be hetatoms, so ignoring mismatch.");<a name="line.632"></a>
<span class="sourceLineNo">633</span>                                                        }<a name="line.633"></a>
<span class="sourceLineNo">634</span>                                                        else {<a name="line.634"></a>
<span class="sourceLineNo">635</span>                                                                //  System.exit(0);// for debug only<a name="line.635"></a>
<span class="sourceLineNo">636</span>                                                                //System.out.println(lst1.seqString());<a name="line.636"></a>
<span class="sourceLineNo">637</span>                                                                //System.out.println(lst2.seqString());<a name="line.637"></a>
<span class="sourceLineNo">638</span>                                                                System.err.println("could not match residues " + s1 + " " + a1);<a name="line.638"></a>
<span class="sourceLineNo">639</span>                                                                //throw new StructureException("could not match residues " + s1 + " " + a1);<a name="line.639"></a>
<span class="sourceLineNo">640</span>                                                        }<a name="line.640"></a>
<span class="sourceLineNo">641</span><a name="line.641"></a>
<span class="sourceLineNo">642</span>                                                }<a name="line.642"></a>
<span class="sourceLineNo">643</span>                                        }<a name="line.643"></a>
<span class="sourceLineNo">644</span><a name="line.644"></a>
<span class="sourceLineNo">645</span>                                        // do the actual replacing of the SEQRES group with the ATOM group<a name="line.645"></a>
<span class="sourceLineNo">646</span>                                        //                                      if ( s1.getChain().getChainID().equals("A")) {<a name="line.646"></a>
<span class="sourceLineNo">647</span>                                        //                                              System.out.println(" setting " + posSeq +" " + a1);<a name="line.647"></a>
<span class="sourceLineNo">648</span>                                        //                                      }<a name="line.648"></a>
<span class="sourceLineNo">649</span>                                        seqResGroups.set(seqresIndexPosition.get(posSeq),a1);<a name="line.649"></a>
<span class="sourceLineNo">650</span>                                        noMatchFound = false;<a name="line.650"></a>
<span class="sourceLineNo">651</span>                                }<a name="line.651"></a>
<span class="sourceLineNo">652</span>                        }<a name="line.652"></a>
<span class="sourceLineNo">653</span><a name="line.653"></a>
<span class="sourceLineNo">654</span><a name="line.654"></a>
<span class="sourceLineNo">655</span>                // now we merge the two chains into one<a name="line.655"></a>
<span class="sourceLineNo">656</span>                // the Groups that can be aligned are now pointing to the<a name="line.656"></a>
<span class="sourceLineNo">657</span>                // groups in the Atom records.<a name="line.657"></a>
<span class="sourceLineNo">658</span>                if (  noMatchFound) {<a name="line.658"></a>
<span class="sourceLineNo">659</span><a name="line.659"></a>
<span class="sourceLineNo">660</span>                        if ( DEBUG )<a name="line.660"></a>
<span class="sourceLineNo">661</span>                                System.out.println("no alignment found!");<a name="line.661"></a>
<span class="sourceLineNo">662</span>                }<a name="line.662"></a>
<span class="sourceLineNo">663</span>                return noMatchFound;<a name="line.663"></a>
<span class="sourceLineNo">664</span><a name="line.664"></a>
<span class="sourceLineNo">665</span>        }<a name="line.665"></a>
<span class="sourceLineNo">666</span><a name="line.666"></a>
<span class="sourceLineNo">667</span>        private boolean mapDNAChains(List&lt;Group&gt; seqResGroups, List&lt;Group&gt; atomRes,<a name="line.667"></a>
<span class="sourceLineNo">668</span>                        SequencePair&lt;DNASequence, NucleotideCompound&gt; pair,<a name="line.668"></a>
<span class="sourceLineNo">669</span>                        Map&lt;Integer,Integer&gt; seqresIndexPosition,<a name="line.669"></a>
<span class="sourceLineNo">670</span>                        Map&lt;Integer,Integer&gt; atomIndexPosition<a name="line.670"></a>
<span class="sourceLineNo">671</span>                        ) throws StructureException{<a name="line.671"></a>
<span class="sourceLineNo">672</span><a name="line.672"></a>
<span class="sourceLineNo">673</span><a name="line.673"></a>
<span class="sourceLineNo">674</span><a name="line.674"></a>
<span class="sourceLineNo">675</span>                // at the present stage the future seqREs are still stored as Atom groups in the seqRes parent...<a name="line.675"></a>
<span class="sourceLineNo">676</span><a name="line.676"></a>
<span class="sourceLineNo">677</span><a name="line.677"></a>
<span class="sourceLineNo">678</span>                int aligLength = pair.getLength();<a name="line.678"></a>
<span class="sourceLineNo">679</span><a name="line.679"></a>
<span class="sourceLineNo">680</span>                //              System.out.println("sequence length seqres:");<a name="line.680"></a>
<span class="sourceLineNo">681</span>                //              System.out.println(seqresIndexPosition.keySet().size());<a name="line.681"></a>
<span class="sourceLineNo">682</span>                //              System.out.println("alignment length: " + aligLength);<a name="line.682"></a>
<span class="sourceLineNo">683</span>                // System.out.println(gapSymbol.getName());<a name="line.683"></a>
<span class="sourceLineNo">684</span><a name="line.684"></a>
<span class="sourceLineNo">685</span>                // make sure we actually find an alignment<a name="line.685"></a>
<span class="sourceLineNo">686</span>                boolean noMatchFound = true;<a name="line.686"></a>
<span class="sourceLineNo">687</span><a name="line.687"></a>
<span class="sourceLineNo">688</span>                Compound gapSymbol =  DNACompoundSet.getDNACompoundSet().getCompoundForString("-");<a name="line.688"></a>
<span class="sourceLineNo">689</span><a name="line.689"></a>
<span class="sourceLineNo">690</span>                mainLoop:<a name="line.690"></a>
<span class="sourceLineNo">691</span>                        for (int i = 1; i &lt;= aligLength ; i++) {<a name="line.691"></a>
<span class="sourceLineNo">692</span><a name="line.692"></a>
<span class="sourceLineNo">693</span>                                Compound s =  pair.getCompoundAt(1, i);<a name="line.693"></a>
<span class="sourceLineNo">694</span>                                Compound a =  pair.getCompoundAt(2,i);<a name="line.694"></a>
<span class="sourceLineNo">695</span><a name="line.695"></a>
<span class="sourceLineNo">696</span>                                // alignment is using internal index start at 1...<a name="line.696"></a>
<span class="sourceLineNo">697</span>                                int posSeq = pair.getIndexInQueryAt(i)  -1;<a name="line.697"></a>
<span class="sourceLineNo">698</span>                                int posAtom = pair.getIndexInTargetAt(i) -1;<a name="line.698"></a>
<span class="sourceLineNo">699</span><a name="line.699"></a>
<span class="sourceLineNo">700</span>                                if (  s.equals(gapSymbol) || a.equals(gapSymbol)){<a name="line.700"></a>
<span class="sourceLineNo">701</span>                                        continue;                               <a name="line.701"></a>
<span class="sourceLineNo">702</span>                                }<a name="line.702"></a>
<span class="sourceLineNo">703</span><a name="line.703"></a>
<span class="sourceLineNo">704</span>                                if ( s.equals(a)){<a name="line.704"></a>
<span class="sourceLineNo">705</span><a name="line.705"></a>
<span class="sourceLineNo">706</span>                                        // the atom record can be aligned to the SeqRes record!<a name="line.706"></a>
<span class="sourceLineNo">707</span>                                        // replace the SeqRes group with the Atom group!<a name="line.707"></a>
<span class="sourceLineNo">708</span><a name="line.708"></a>
<span class="sourceLineNo">709</span>                                        Group s1 = seqResGroups.get(seqresIndexPosition.get(posSeq));<a name="line.709"></a>
<span class="sourceLineNo">710</span>                                        Group a1 = atomRes.get(atomIndexPosition.get(posAtom));<a name="line.710"></a>
<span class="sourceLineNo">711</span><a name="line.711"></a>
<span class="sourceLineNo">712</span>                                        if ( s1 == null || a1 == null){<a name="line.712"></a>
<span class="sourceLineNo">713</span>                                                /// can't map this position...<a name="line.713"></a>
<span class="sourceLineNo">714</span>                                                System.err.println("can't map " + i + ":" + s + " " + posSeq +" " + s1 + " atom: " + posAtom + " " + a1 );<a name="line.714"></a>
<span class="sourceLineNo">715</span>                                                continue mainLoop;<a name="line.715"></a>
<span class="sourceLineNo">716</span>                                        }<a name="line.716"></a>
<span class="sourceLineNo">717</span><a name="line.717"></a>
<span class="sourceLineNo">718</span>                                        // need to trim the names to allow matching e.g in<a name="line.718"></a>
<span class="sourceLineNo">719</span>                                        // pdb1b2m<a name="line.719"></a>
<span class="sourceLineNo">720</span>                                        String pdbNameS = s1.getPDBName();<a name="line.720"></a>
<span class="sourceLineNo">721</span>                                        String pdbNameA = a1.getPDBName();<a name="line.721"></a>
<span class="sourceLineNo">722</span>                                        if ( pdbNameS == null || pdbNameA == null ){<a name="line.722"></a>
<span class="sourceLineNo">723</span>                                                System.err.println("null value found at " + posSeq + " when trying to align " + s1 + " and " + a1 + " " + posAtom);<a name="line.723"></a>
<span class="sourceLineNo">724</span>                                                throw new StructureException("null value found at group.getPDBName()");<a name="line.724"></a>
<span class="sourceLineNo">725</span>                                        }<a name="line.725"></a>
<span class="sourceLineNo">726</span>                                        if (! pdbNameA.equals(pdbNameS)){<a name="line.726"></a>
<span class="sourceLineNo">727</span>                                                if ( ! pdbNameA.trim().equals(pdbNameS.trim())) {<a name="line.727"></a>
<span class="sourceLineNo">728</span>                                                        System.err.println(s1 + " " + posSeq + " does not align with " + a1+ " " + posAtom + " should be: " + s + " : " + a);<a name="line.728"></a>
<span class="sourceLineNo">729</span>                                                        if ( s1.getType().equals(HetatomImpl.type) &amp;&amp; a1.getType().equals(HetatomImpl.type)){<a name="line.729"></a>
<span class="sourceLineNo">730</span>                                                                System.err.println("they seem to be hetatoms, so ignoring mismatch.");<a name="line.730"></a>
<span class="sourceLineNo">731</span>                                                        }<a name="line.731"></a>
<span class="sourceLineNo">732</span>                                                        else {<a name="line.732"></a>
<span class="sourceLineNo">733</span>                                                                //  System.exit(0);// for debug only<a name="line.733"></a>
<span class="sourceLineNo">734</span>                                                                //System.out.println(lst1.seqString());<a name="line.734"></a>
<span class="sourceLineNo">735</span>                                                                //System.out.println(lst2.seqString());<a name="line.735"></a>
<span class="sourceLineNo">736</span>                                                                System.err.println("could not match residues " + s1 + " " + a1);<a name="line.736"></a>
<span class="sourceLineNo">737</span>                                                                //throw new StructureException("could not match residues " + s1 + " " + a1);<a name="line.737"></a>
<span class="sourceLineNo">738</span>                                                        }<a name="line.738"></a>
<span class="sourceLineNo">739</span><a name="line.739"></a>
<span class="sourceLineNo">740</span>                                                }<a name="line.740"></a>
<span class="sourceLineNo">741</span>                                        }<a name="line.741"></a>
<span class="sourceLineNo">742</span><a name="line.742"></a>
<span class="sourceLineNo">743</span>                                        // do the actual replacing of the SEQRES group with the ATOM group<a name="line.743"></a>
<span class="sourceLineNo">744</span>                                        //                                      if ( s1.getChain().getChainID().equals("A")) {<a name="line.744"></a>
<span class="sourceLineNo">745</span>                                        //                                              System.out.println(" setting " + posSeq +" " + a1);<a name="line.745"></a>
<span class="sourceLineNo">746</span>                                        //                                      }<a name="line.746"></a>
<span class="sourceLineNo">747</span>                                        seqResGroups.set(seqresIndexPosition.get(posSeq),a1);<a name="line.747"></a>
<span class="sourceLineNo">748</span>                                        noMatchFound = false;<a name="line.748"></a>
<span class="sourceLineNo">749</span>                                }<a name="line.749"></a>
<span class="sourceLineNo">750</span>                        }<a name="line.750"></a>
<span class="sourceLineNo">751</span><a name="line.751"></a>
<span class="sourceLineNo">752</span><a name="line.752"></a>
<span class="sourceLineNo">753</span>                // now we merge the two chains into one<a name="line.753"></a>
<span class="sourceLineNo">754</span>                // the Groups that can be aligned are now pointing to the<a name="line.754"></a>
<span class="sourceLineNo">755</span>                // groups in the Atom records.<a name="line.755"></a>
<span class="sourceLineNo">756</span>                if (  noMatchFound) {<a name="line.756"></a>
<span class="sourceLineNo">757</span><a name="line.757"></a>
<span class="sourceLineNo">758</span>                        if ( DEBUG )<a name="line.758"></a>
<span class="sourceLineNo">759</span>                                System.out.println("no alignment found!");<a name="line.759"></a>
<span class="sourceLineNo">760</span>                }<a name="line.760"></a>
<span class="sourceLineNo">761</span>                return noMatchFound;<a name="line.761"></a>
<span class="sourceLineNo">762</span><a name="line.762"></a>
<span class="sourceLineNo">763</span>        }<a name="line.763"></a>
<span class="sourceLineNo">764</span><a name="line.764"></a>
<span class="sourceLineNo">765</span>}<a name="line.765"></a>




























































</pre>
</div>
</body>
</html>
