<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<html lang="en">
<head>
<title>Source code</title>
<link rel="stylesheet" type="text/css" href="../../../../../stylesheet.css" title="Style">
</head>
<body>
<div class="sourceContainer">
<pre><span class="sourceLineNo">001</span>package org.biojava3.core.sequence;<a name="line.1"></a>
<span class="sourceLineNo">002</span><a name="line.2"></a>
<span class="sourceLineNo">003</span>import static org.biojava3.core.sequence.io.util.IOUtils.close;<a name="line.3"></a>
<span class="sourceLineNo">004</span>import static org.hamcrest.CoreMatchers.is;<a name="line.4"></a>
<span class="sourceLineNo">005</span>import static org.junit.Assert.assertThat;<a name="line.5"></a>
<span class="sourceLineNo">006</span><a name="line.6"></a>
<span class="sourceLineNo">007</span>import java.io.InputStream;<a name="line.7"></a>
<span class="sourceLineNo">008</span>import java.util.EnumMap;<a name="line.8"></a>
<span class="sourceLineNo">009</span>import java.util.Map;<a name="line.9"></a>
<span class="sourceLineNo">010</span>import java.util.Map.Entry;<a name="line.10"></a>
<span class="sourceLineNo">011</span><a name="line.11"></a>
<span class="sourceLineNo">012</span>import org.biojava3.core.sequence.compound.AminoAcidCompound;<a name="line.12"></a>
<span class="sourceLineNo">013</span>import org.biojava3.core.sequence.compound.AminoAcidCompoundSet;<a name="line.13"></a>
<span class="sourceLineNo">014</span>import org.biojava3.core.sequence.compound.DNACompoundSet;<a name="line.14"></a>
<span class="sourceLineNo">015</span>import org.biojava3.core.sequence.compound.NucleotideCompound;<a name="line.15"></a>
<span class="sourceLineNo">016</span>import org.biojava3.core.sequence.io.DNASequenceCreator;<a name="line.16"></a>
<span class="sourceLineNo">017</span>import org.biojava3.core.sequence.io.FastaReader;<a name="line.17"></a>
<span class="sourceLineNo">018</span>import org.biojava3.core.sequence.io.GenericFastaHeaderParser;<a name="line.18"></a>
<span class="sourceLineNo">019</span>import org.biojava3.core.sequence.io.IUPACParser;<a name="line.19"></a>
<span class="sourceLineNo">020</span>import org.biojava3.core.sequence.io.ProteinSequenceCreator;<a name="line.20"></a>
<span class="sourceLineNo">021</span>import org.biojava3.core.sequence.io.util.ClasspathResource;<a name="line.21"></a>
<span class="sourceLineNo">022</span>import org.biojava3.core.sequence.template.Sequence;<a name="line.22"></a>
<span class="sourceLineNo">023</span>import org.biojava3.core.sequence.transcription.Frame;<a name="line.23"></a>
<span class="sourceLineNo">024</span>import org.biojava3.core.sequence.transcription.TranscriptionEngine;<a name="line.24"></a>
<span class="sourceLineNo">025</span>import org.junit.Assert;<a name="line.25"></a>
<span class="sourceLineNo">026</span>import org.junit.BeforeClass;<a name="line.26"></a>
<span class="sourceLineNo">027</span>import org.junit.Test;<a name="line.27"></a>
<span class="sourceLineNo">028</span><a name="line.28"></a>
<span class="sourceLineNo">029</span>public class TranslationTest {<a name="line.29"></a>
<span class="sourceLineNo">030</span><a name="line.30"></a>
<span class="sourceLineNo">031</span>    private static DNACompoundSet dnaCs = DNACompoundSet.getDNACompoundSet();<a name="line.31"></a>
<span class="sourceLineNo">032</span>    private static AminoAcidCompoundSet aaCs = AminoAcidCompoundSet.getAminoAcidCompoundSet();<a name="line.32"></a>
<span class="sourceLineNo">033</span>    private static DNASequence brca2Dna;<a name="line.33"></a>
<span class="sourceLineNo">034</span>    private static Sequence&lt;AminoAcidCompound&gt; brca2Pep;<a name="line.34"></a>
<span class="sourceLineNo">035</span>    private static Sequence&lt;NucleotideCompound&gt; volvoxDna;<a name="line.35"></a>
<span class="sourceLineNo">036</span>    private static Sequence&lt;AminoAcidCompound&gt; volvoxPep;<a name="line.36"></a>
<span class="sourceLineNo">037</span><a name="line.37"></a>
<span class="sourceLineNo">038</span>    @BeforeClass<a name="line.38"></a>
<span class="sourceLineNo">039</span>    public static void parseSequences() {<a name="line.39"></a>
<span class="sourceLineNo">040</span>        InputStream cdsIs = new ClasspathResource(<a name="line.40"></a>
<span class="sourceLineNo">041</span>                "org/biojava3/core/sequence/BRCA2-cds.fasta").getInputStream();<a name="line.41"></a>
<span class="sourceLineNo">042</span>        InputStream pepIs = new ClasspathResource(<a name="line.42"></a>
<span class="sourceLineNo">043</span>                "org/biojava3/core/sequence/BRCA2-peptide.fasta").getInputStream();<a name="line.43"></a>
<span class="sourceLineNo">044</span>        InputStream volDnaIs = new ClasspathResource(<a name="line.44"></a>
<span class="sourceLineNo">045</span>                "org/biojava3/core/sequence/volvox-cds.fasta").getInputStream();<a name="line.45"></a>
<span class="sourceLineNo">046</span>        InputStream volPepIs = new ClasspathResource(<a name="line.46"></a>
<span class="sourceLineNo">047</span>                "org/biojava3/core/sequence/volvox-peptide.fasta").getInputStream();<a name="line.47"></a>
<span class="sourceLineNo">048</span><a name="line.48"></a>
<span class="sourceLineNo">049</span>        try {<a name="line.49"></a>
<span class="sourceLineNo">050</span>            FastaReader&lt;DNASequence, NucleotideCompound&gt; dnaReader = new FastaReader&lt;DNASequence, NucleotideCompound&gt;(cdsIs,<a name="line.50"></a>
<span class="sourceLineNo">051</span>                    new GenericFastaHeaderParser&lt;DNASequence, NucleotideCompound&gt;(), new DNASequenceCreator(dnaCs));<a name="line.51"></a>
<span class="sourceLineNo">052</span>            brca2Dna = dnaReader.process().values().iterator().next();<a name="line.52"></a>
<span class="sourceLineNo">053</span>            FastaReader&lt;ProteinSequence, AminoAcidCompound&gt; pReader = new FastaReader&lt;ProteinSequence, AminoAcidCompound&gt;(<a name="line.53"></a>
<span class="sourceLineNo">054</span>                    pepIs, new GenericFastaHeaderParser&lt;ProteinSequence, AminoAcidCompound&gt;(), new ProteinSequenceCreator(<a name="line.54"></a>
<span class="sourceLineNo">055</span>                    aaCs));<a name="line.55"></a>
<span class="sourceLineNo">056</span>            brca2Pep = pReader.process().values().iterator().next();<a name="line.56"></a>
<span class="sourceLineNo">057</span>            <a name="line.57"></a>
<span class="sourceLineNo">058</span>            FastaReader&lt;DNASequence, NucleotideCompound&gt; volvoxDnaReader = new FastaReader&lt;DNASequence, NucleotideCompound&gt;(volDnaIs,<a name="line.58"></a>
<span class="sourceLineNo">059</span>                    new GenericFastaHeaderParser&lt;DNASequence, NucleotideCompound&gt;(), new DNASequenceCreator(dnaCs));<a name="line.59"></a>
<span class="sourceLineNo">060</span>            volvoxDna = volvoxDnaReader.process().values().iterator().next();<a name="line.60"></a>
<span class="sourceLineNo">061</span>            FastaReader&lt;ProteinSequence, AminoAcidCompound&gt; volvoxPepReader = new FastaReader&lt;ProteinSequence, AminoAcidCompound&gt;(<a name="line.61"></a>
<span class="sourceLineNo">062</span>                    volPepIs, new GenericFastaHeaderParser&lt;ProteinSequence, AminoAcidCompound&gt;(), new ProteinSequenceCreator(<a name="line.62"></a>
<span class="sourceLineNo">063</span>                    aaCs));<a name="line.63"></a>
<span class="sourceLineNo">064</span>            volvoxPep = volvoxPepReader.process().values().iterator().next();<a name="line.64"></a>
<span class="sourceLineNo">065</span>        }<a name="line.65"></a>
<span class="sourceLineNo">066</span>        catch (Exception e) {<a name="line.66"></a>
<span class="sourceLineNo">067</span>            e.printStackTrace();<a name="line.67"></a>
<span class="sourceLineNo">068</span>            Assert.fail("Encountered exception");<a name="line.68"></a>
<span class="sourceLineNo">069</span>        }<a name="line.69"></a>
<span class="sourceLineNo">070</span>        finally {<a name="line.70"></a>
<span class="sourceLineNo">071</span>            close(cdsIs);<a name="line.71"></a>
<span class="sourceLineNo">072</span>            close(pepIs);<a name="line.72"></a>
<span class="sourceLineNo">073</span>            close(volDnaIs);<a name="line.73"></a>
<span class="sourceLineNo">074</span>            close(volPepIs);<a name="line.74"></a>
<span class="sourceLineNo">075</span>        }<a name="line.75"></a>
<span class="sourceLineNo">076</span>    }<a name="line.76"></a>
<span class="sourceLineNo">077</span><a name="line.77"></a>
<span class="sourceLineNo">078</span>  @Test<a name="line.78"></a>
<span class="sourceLineNo">079</span>    public void getUniversal() {<a name="line.79"></a>
<span class="sourceLineNo">080</span>        IUPACParser.getInstance().getTable(1);<a name="line.80"></a>
<span class="sourceLineNo">081</span>        IUPACParser.getInstance().getTable("UNIVERSAL");<a name="line.81"></a>
<span class="sourceLineNo">082</span>    }<a name="line.82"></a>
<span class="sourceLineNo">083</span><a name="line.83"></a>
<span class="sourceLineNo">084</span>  @Test<a name="line.84"></a>
<span class="sourceLineNo">085</span>    public void basicTranslation() {<a name="line.85"></a>
<span class="sourceLineNo">086</span>        TranscriptionEngine e = TranscriptionEngine.getDefault();<a name="line.86"></a>
<span class="sourceLineNo">087</span>        DNASequence dna = new DNASequence("ATG");<a name="line.87"></a>
<span class="sourceLineNo">088</span>        RNASequence rna = dna.getRNASequence(e);<a name="line.88"></a>
<span class="sourceLineNo">089</span>        ProteinSequence protein = rna.getProteinSequence(e);<a name="line.89"></a>
<span class="sourceLineNo">090</span>        AminoAcidCompound initMet = protein.getCompoundAt(1);<a name="line.90"></a>
<span class="sourceLineNo">091</span>        assertThat("Initator methionine wrong", initMet.toString(), is("M"));<a name="line.91"></a>
<span class="sourceLineNo">092</span>    }<a name="line.92"></a>
<span class="sourceLineNo">093</span><a name="line.93"></a>
<span class="sourceLineNo">094</span>  @Test<a name="line.94"></a>
<span class="sourceLineNo">095</span>    public void translateN() {<a name="line.95"></a>
<span class="sourceLineNo">096</span>        TranscriptionEngine.Builder b = new TranscriptionEngine.Builder();<a name="line.96"></a>
<span class="sourceLineNo">097</span>        b.translateNCodons(true).initMet(true);<a name="line.97"></a>
<span class="sourceLineNo">098</span>        TranscriptionEngine e = b.build();<a name="line.98"></a>
<span class="sourceLineNo">099</span>        DNASequence dna = new DNASequence("ATN");<a name="line.99"></a>
<span class="sourceLineNo">100</span>        RNASequence rna = dna.getRNASequence(e);<a name="line.100"></a>
<span class="sourceLineNo">101</span>        ProteinSequence protein = rna.getProteinSequence(e);<a name="line.101"></a>
<span class="sourceLineNo">102</span>        assertThat("Ambiguous translation problem", protein.toString(), is("X"));<a name="line.102"></a>
<span class="sourceLineNo">103</span>        DNASequence dna2 = new DNASequence("GTGGTNTAA");<a name="line.103"></a>
<span class="sourceLineNo">104</span>        RNASequence rna2 = dna2.getRNASequence(e);<a name="line.104"></a>
<span class="sourceLineNo">105</span>        ProteinSequence protein2 = rna2.getProteinSequence(e);<a name="line.105"></a>
<span class="sourceLineNo">106</span>        assertThat("Ambiguous translation problem", protein2.toString(), is("VX"));<a name="line.106"></a>
<span class="sourceLineNo">107</span>    }<a name="line.107"></a>
<span class="sourceLineNo">108</span><a name="line.108"></a>
<span class="sourceLineNo">109</span>    @SuppressWarnings("serial")<a name="line.109"></a>
<span class="sourceLineNo">110</span>    @Test<a name="line.110"></a>
<span class="sourceLineNo">111</span>    public void multiFrameTranslation() {<a name="line.111"></a>
<span class="sourceLineNo">112</span>        TranscriptionEngine e = TranscriptionEngine.getDefault();<a name="line.112"></a>
<span class="sourceLineNo">113</span>        DNASequence dna = new DNASequence("ATGGCGTGA");<a name="line.113"></a>
<span class="sourceLineNo">114</span><a name="line.114"></a>
<span class="sourceLineNo">115</span>        Map&lt;Frame, String&gt; expectedTranslations = new EnumMap&lt;Frame, String&gt;(Frame.class) {<a name="line.115"></a>
<span class="sourceLineNo">116</span><a name="line.116"></a>
<span class="sourceLineNo">117</span>            {<a name="line.117"></a>
<span class="sourceLineNo">118</span>                put(Frame.ONE, "MA");<a name="line.118"></a>
<span class="sourceLineNo">119</span>                put(Frame.TWO, "WR");<a name="line.119"></a>
<span class="sourceLineNo">120</span>                put(Frame.THREE, "GV");<a name="line.120"></a>
<span class="sourceLineNo">121</span>                put(Frame.REVERSED_ONE, "SRH");<a name="line.121"></a>
<span class="sourceLineNo">122</span>                put(Frame.REVERSED_TWO, "HA");<a name="line.122"></a>
<span class="sourceLineNo">123</span>                put(Frame.REVERSED_THREE, "TP");<a name="line.123"></a>
<span class="sourceLineNo">124</span>            }<a name="line.124"></a>
<span class="sourceLineNo">125</span>        };<a name="line.125"></a>
<span class="sourceLineNo">126</span><a name="line.126"></a>
<span class="sourceLineNo">127</span>        Map&lt;Frame, Sequence&lt;AminoAcidCompound&gt;&gt; translations =<a name="line.127"></a>
<span class="sourceLineNo">128</span>                e.multipleFrameTranslation(dna, Frame.getAllFrames());<a name="line.128"></a>
<span class="sourceLineNo">129</span><a name="line.129"></a>
<span class="sourceLineNo">130</span>        for (Entry&lt;Frame, Sequence&lt;AminoAcidCompound&gt;&gt; entry : translations.entrySet()) {<a name="line.130"></a>
<span class="sourceLineNo">131</span>            String expected = expectedTranslations.get(entry.getKey());<a name="line.131"></a>
<span class="sourceLineNo">132</span>            Sequence&lt;AminoAcidCompound&gt; protein = entry.getValue();<a name="line.132"></a>
<span class="sourceLineNo">133</span>            assertThat("Checking 6 frame translation", protein.toString(), is(expected));<a name="line.133"></a>
<span class="sourceLineNo">134</span>        }<a name="line.134"></a>
<span class="sourceLineNo">135</span>    }<a name="line.135"></a>
<span class="sourceLineNo">136</span><a name="line.136"></a>
<span class="sourceLineNo">137</span>  @Test<a name="line.137"></a>
<span class="sourceLineNo">138</span>    public void lowerCases() {<a name="line.138"></a>
<span class="sourceLineNo">139</span>        TranscriptionEngine e = TranscriptionEngine.getDefault();<a name="line.139"></a>
<span class="sourceLineNo">140</span>        DNASequence dna = new DNASequence("atgcCt");<a name="line.140"></a>
<span class="sourceLineNo">141</span>        RNASequence rna = dna.getRNASequence(e);<a name="line.141"></a>
<span class="sourceLineNo">142</span>        Sequence&lt;AminoAcidCompound&gt; peptide = rna.getProteinSequence(e);<a name="line.142"></a>
<span class="sourceLineNo">143</span>        assertThat("Checking lower casing is respected", peptide.getSequenceAsString(),<a name="line.143"></a>
<span class="sourceLineNo">144</span>                is("MP"));<a name="line.144"></a>
<span class="sourceLineNo">145</span>    }<a name="line.145"></a>
<span class="sourceLineNo">146</span><a name="line.146"></a>
<span class="sourceLineNo">147</span>  @Test<a name="line.147"></a>
<span class="sourceLineNo">148</span>    public void translateBrca2ExonOne() {<a name="line.148"></a>
<span class="sourceLineNo">149</span>        TranscriptionEngine e = TranscriptionEngine.getDefault();<a name="line.149"></a>
<span class="sourceLineNo">150</span>        DNASequence dna = new DNASequence(<a name="line.150"></a>
<span class="sourceLineNo">151</span>                "ATGCCTATTGGATCCAAAGAGAGGCCAACATTTTTTGAAATTTTTAAGACACGCTGCAACAAAGCA");<a name="line.151"></a>
<span class="sourceLineNo">152</span>        RNASequence rna = dna.getRNASequence(e);<a name="line.152"></a>
<span class="sourceLineNo">153</span>        Sequence&lt;AminoAcidCompound&gt; peptide = rna.getProteinSequence(e);<a name="line.153"></a>
<span class="sourceLineNo">154</span>        assertThat("Initator methionine wrong", peptide.getSequenceAsString(),<a name="line.154"></a>
<span class="sourceLineNo">155</span>                is("MPIGSKERPTFFEIFKTRCNKA"));<a name="line.155"></a>
<span class="sourceLineNo">156</span>    }<a name="line.156"></a>
<span class="sourceLineNo">157</span><a name="line.157"></a>
<span class="sourceLineNo">158</span>    @Test(timeout=2000)<a name="line.158"></a>
<span class="sourceLineNo">159</span>    public void translateBrca2() {<a name="line.159"></a>
<span class="sourceLineNo">160</span>        TranscriptionEngine e =<a name="line.160"></a>
<span class="sourceLineNo">161</span>                new TranscriptionEngine.Builder().decorateRna(true).build();<a name="line.161"></a>
<span class="sourceLineNo">162</span>        for (int i = 0; i &lt; 100; i++) {<a name="line.162"></a>
<span class="sourceLineNo">163</span>            RNASequence rna = brca2Dna.getRNASequence(e);<a name="line.163"></a>
<span class="sourceLineNo">164</span>            ProteinSequence protein = rna.getProteinSequence(e);<a name="line.164"></a>
<span class="sourceLineNo">165</span>            assertThat("BRCA2 does not translate", protein.getSequenceAsString(),<a name="line.165"></a>
<span class="sourceLineNo">166</span>                    is(brca2Pep.getSequenceAsString()));<a name="line.166"></a>
<span class="sourceLineNo">167</span>        }<a name="line.167"></a>
<span class="sourceLineNo">168</span>    }<a name="line.168"></a>
<span class="sourceLineNo">169</span><a name="line.169"></a>
<span class="sourceLineNo">170</span>    @Test<a name="line.170"></a>
<span class="sourceLineNo">171</span>    public void translateInternalStops() {<a name="line.171"></a>
<span class="sourceLineNo">172</span>        TranscriptionEngine e = TranscriptionEngine.getDefault();<a name="line.172"></a>
<span class="sourceLineNo">173</span>        Sequence&lt;AminoAcidCompound&gt; pep = e.translate(volvoxDna);<a name="line.173"></a>
<span class="sourceLineNo">174</span>        assertThat("Ensure internal stops stay", pep.toString(), is(volvoxPep.toString()));<a name="line.174"></a>
<span class="sourceLineNo">175</span>    }<a name="line.175"></a>
<span class="sourceLineNo">176</span><a name="line.176"></a>
<span class="sourceLineNo">177</span>    @Test<a name="line.177"></a>
<span class="sourceLineNo">178</span>    public void translateInitMet() {<a name="line.178"></a>
<span class="sourceLineNo">179</span>        TranscriptionEngine e = TranscriptionEngine.getDefault();<a name="line.179"></a>
<span class="sourceLineNo">180</span>        assertThat("Leucene (CTA) is not changed to init met", e.translate(new DNASequence("CTA")).toString(), is("L"));<a name="line.180"></a>
<span class="sourceLineNo">181</span>        assertThat("Leucene (CTG) is changed to init met", e.translate(new DNASequence("CTG")).toString(), is("M"));<a name="line.181"></a>
<span class="sourceLineNo">182</span><a name="line.182"></a>
<span class="sourceLineNo">183</span>        e = new TranscriptionEngine.Builder().initMet(false).build();<a name="line.183"></a>
<span class="sourceLineNo">184</span>        assertThat("Leucene (CTG) is not changed to init met", e.translate(new DNASequence("CTG")).toString(), is("L"));<a name="line.184"></a>
<span class="sourceLineNo">185</span>    }<a name="line.185"></a>
<span class="sourceLineNo">186</span>}<a name="line.186"></a>




























































</pre>
</div>
</body>
</html>
